# Cloning, expression, crystallization and crystallographic analysis of CouR from Rhodopseudomonas palustris

Received 23 August 2015   
Accepted 8 October 2015

Edited by T. Bergfors,Uppsala University, Sweden

Keywords:MarR family; transcriptional regulator;protein-ligand complex; crystallographic study;lignin; $p$ -coumarate.

Chen Pan,ab Yong-lin Hu,b Xiang-ning JiangaC and Ying Gaia\* aCollege ofBiological ScienceandTechnology,Beijing Forestry UniversityBeijingOoo83,People'sRepublicofChina, bNational Laboratory of Biomacromolecules,Institute of Biophysics, Chinese Academy of Sciences,15 Datun Road, Beijing 10O101,People's RepublicofChina,and‘TheTree andOrnamental Plant Breedingand BiotechnologyLaboratory of Chinese Forestry Administration,Beijing 10oo83,People's Republic of China.\*Corespondence e-mail: gaiying@bjfu.edu.cn

CouR from Rhodopseudomonas palustris is a member of the MarR transcriptional regulator family. It regulates the expression of CouA and CouB, enzymes that are involved in the degradation of $p$ -coumarate. In vivo, CouR binds to a DNA fragment containing the couAB promoter and suppresses the expression of CouA and CouB,while binding of $p$ -coumaroyl-CoA attenuates its affinity towards DNA and activates the expression of CouA and CouB. Here，the crystallization and X-ray diffraction analyses of CouR alone and in complex with $p$ -coumaroyl-CoA are reported.Apo and ligand-complexed CouR crystals diffracted to 2.5 and $3 . 3 \mathring \mathrm { \mathrm { A } }$ resolution,respectively. The crystals of apo CouR belonged to space group $P 2 2 _ { 1 } 2 _ { 1 }$ ,with unit-cell parameters $a = 6 2 . 7 8$ $b = 7 6 . 1 5$ ， $c = 8 7 . 3 8 \mathring \mathrm { ~ A }$ ,whereas the crystals of the CouR-ligand complex belonged to space group $P 2 _ { 1 } 2 _ { 1 } 2 _ { 1 }$ ，with unit-cell parameters $a \ : = \ : 6 1 . 3 7$ ， $b = 6 9 . 8 2$ ， $c = 7 0 . 3 2 \ : \mathring { \mathrm { A } }$ The crystals were predicted to contain two CouR molecules or CouR-ligand complexes per asymmetric unit.

# 1.Introduction

The abundance of aromatic compounds in nature, introduced either by human activities or from decaying plant materials, has led to the evolution of metabolic pathways in soil-dwelling bacteria to utilize these chemicals as a carbon source.Because of their importance in biodegradation,these metabolic pathways have been the focus of research and many pathways have been identified. One area of interest is how these pathways are regulated by the presence of different aromatic compounds. Several families of bacterial transcriptional regulators of the degradation pathways of aromatic compounds have been identified(Tropel& van derMeer,2004).

$p$ -Coumarate is produced in large quantities in green plants via the phenylpropanoid pathway. Most of this compound is converted into lignols,which are subsequently polymerized to form lignin, the second most abundant biopolymer on earth after cellulose.In addition to lignin, the biosynthesis of many compoundssuch asantimicrobials, molecularsignals, aromatics,pigments and UV protectors requires $p$ -coumarate asaprecursor.

![](images/af0997c9c366e373329853e1027b915301a306979040fa648fb8c6ad5a8bdd80.jpg)  
$\copyright$ 2015 International Union of Crystallography

Several soil-dwelling bacteria have evolved metabolic pathways that can use lignin-derived compounds as a carbon source.In Streptomyces coelicolor,for example,the pca genes encode enzymes that are involved in the catabolism of protocatechuate and catechol, and these genes are negatively regulated by the transcriptional factor SCO67O4 in the presence of lignin-derived aromatic compounds (Davis &

Table 1 Macromolecule-production information.   

<html><body><table><tr><td>Source organism</td><td>R. palustris ATGACC TCC TCC AAC CGT ATC ACC TCC</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>DNA sequence</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>CCG GCT ATG ACC GCT TCC AAA ACC</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>GCT GCT GTT</td><td></td><td></td><td>GCT AAA CCG ACC CGT</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>GCT GGT CGT AAA GCT CCG GCT GTT</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>GAA ACC</td><td></td><td>GCT</td><td>CCG GAA</td><td></td><td></td><td>GCT TCC GAA</td><td></td></tr><tr><td></td><td>CTG</td><td>AAA ATG</td><td></td><td>GGT</td><td>GAA</td><td></td><td>CTG TCC GAA</td><td></td></tr><tr><td></td><td>CTG</td><td>CTG</td><td>GGT</td><td>TAC GCT</td><td></td><td></td><td>CTG AAA CGT</td><td></td></tr><tr><td></td><td>GCT</td><td>CAG</td><td>CTG</td><td>CGT</td><td></td><td></td><td>GTT TTC GAA GAC</td><td></td></tr><tr><td></td><td>TTC</td><td>CTG</td><td>CAC</td><td>TGT</td><td>GTT</td><td></td><td>GCT CCG GTT</td><td></td></tr><tr><td></td><td>CAG</td><td>CTG</td><td>ACC</td><td>CCG GCT</td><td></td><td></td><td>CAG TTC TCC</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>GTT CTG CTG CTG CTG GAC GCT AAC</td><td></td></tr><tr><td></td><td>CCG GGT</td><td></td><td>CGT</td><td></td><td></td><td></td><td>AAC CAG ACC GAA ATC</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>GCT ACC ACC CTG GGT ATC CTG CGT</td><td></td></tr><tr><td></td><td>CCG AAC TTC</td><td></td><td></td><td>GTT</td><td></td><td></td><td>GCT ATG CTG GAC</td><td></td></tr><tr><td></td><td>GCT</td><td>CTG GAA</td><td></td><td>GGT</td><td>CGT</td><td></td><td>GGT CTG TGT</td><td></td></tr><tr><td></td><td>GTT</td><td>CGT ACC</td><td></td><td></td><td></td><td></td><td>CGT TCC CCG TCC GAC</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>CGT CGT TCC CAC ATC CTG ATG CTG</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ACC GAC AAA GGT CGT GCT ACC CTG</td><td></td></tr><tr><td></td><td>GCT CGT</td><td></td><td>GCT</td><td>AAA AAA</td><td></td><td></td><td>CTG GTT GCT</td><td></td></tr><tr><td></td><td></td><td>ACC CGT CAC GAA GAC CGT CTG ACC</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td>GAA CTG CTG GGT CGT GAC AAC CGT</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td>GAC GCT CTG CTG TCC ATG CTG GCT</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>ACC ATC GCT CGT GAA TTC TGA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Cloning vector</td><td colspan="8">pET-28a</td></tr><tr><td>Expression vector</td><td colspan="8">pET-28a E.coli strain BL21</td></tr><tr><td>Expression host Complete amino-acid sequence</td><td colspan="8">MTSSNRITSPAMTASKTAAVAKPTRAGRKAPAVE-</td></tr><tr><td>of the construct produced</td><td colspan="8"></td></tr><tr><td rowspan="5"></td><td colspan="5">TAPEASELKMGELSELLGYALKRAQLRVFEDF-</td><td></td><td></td></tr><tr><td></td><td colspan="5">LHCVAPVQLTPAQFSVLLLLDANPGRNQTEIA-</td></tr><tr><td></td><td colspan="5">TTLGILRPNFVAMLDALEGRGLCVRTRSPSDR-</td></tr><tr><td></td><td colspan="3">RSHILMLTDKGRATLARAKKLVATRHEDRLTE-</td><td></td></tr><tr><td colspan="2">LLGRDNRDALLSMLATIAREF</td><td colspan="3"></td></tr></table></body></html>

Sello,201o). Similar systems are also found in Agrobacterium tumefaciens(Parke,1995)，Corynebacterium glutamicum (Brinkrolf et al.,2Oo6) and Acinetobacter baylyi (Siehler et al., 2007).Nevertheless,research on proteins targeting $p$ -coumarate,the starting compound of lignin biosynthesis,has been scarce.

Ina recent study,it has been found that Rhodopseudomonas palustris,a Gram-negative purple nonsulfur bacterium, can utilize $p$ -coumarate as a carbon source.The enoyl-CoA lyase CouB converts $p$ -coumarate into $p$ -coumaroyl-CoA,and the enoyl-CoA hydratase CouA cleaves the ester side chain of $p$ -coumaroyl-CoA to form acetyl-CoA and benzaldehyde (Hirakawa etal.,2O12).CouR,amultiple antibiotic-resistance regulator (MarR) repressor family protein,regulates the gene expression of CouA and CouB in response to $p$ -coumaroylCoA.The couR gene is located downstream of the couAB genes,and the CouR protein binds a 29 bp region upstream of couA and protects it from DNase I digestion. This region encompasses an inverted repeat sequence separated by a 3 bp spacer and overlaps with the $^ { - 1 0 }$ promoter element. The study also found that the binding between CouR and the protected region was not abolished by either $p$ -coumarate or other degradation products,but only by $p$ -coumaroyl-CoA (Hirakawa et al.,2012).A homologous transcriptional regulator has also been identified in Sphingobium sp.strain SYK-6 (Kasai et al.,2O12),in which the MarR-family protein FerC negatively regulates the expression of FerBA,with feruloylCoA, $p$ -coumaroyl-CoA,caffeoyl-CoA and sinapoyl-CoAas inducers.However, the molecular mechanism explaining how these transcriptional regulators interact with $p$ -coumaroylCoA or its derivatives and modulate their DNA-binding affinities accordingly has remained unresolved (Perera & Grove, 2010).

Our research groups are interested in the biosynthesis and degradation of lignin and the phenylpropanoid compounds of the pathway,so we embarked on structural analysis of CouR in order to illustrate how this protein recognizes and binds $p$ -coumaroyl-CoA and the structural basis for the attenuated DNA binding of CouR upon its interaction with $p$ -coumaroylCoA.

Here,we report the expression,purification and crystallization of CouR and its complex with $p$ -coumarate.We also present the crystallographic results of our $\mathbf { X }$ -raydiffraction experiments on these two crystal forms.

# 2.Materialsandmethods

# 2.1.Cloning,expression and purification of CouR

The gene encoding CouR was redesigned to optimize protein expression in Escherichia coli.It was synthesized by Sangon Biotech Company, Shanghai,People's Republic of China and was ligated to a pET-28a vector,which provides an N-terminal $\mathrm { H i s } _ { 6 }$ tag on the expressed protein. The plasmid was transformed into $E _ { \ l }$ ，coli strain BL21 using the heat-shock method (Table 1). The transformed bacteria were grown at $3 1 0 \mathrm { K }$ in the presence of $2 5 \mathrm { m g l ^ { - 1 } }$ kanamycin until the $\mathrm { O D } _ { 6 0 0 }$ reached O.6-O.8. Protein overexpression was induced by adding $0 . 0 5 \mathrm { m } M$ isopropyl $\beta$ -D-1-thiogalactopyranoside and incubating at $2 9 5 \mathrm { K }$ . After $1 2 \mathrm { h }$ of incubation,the cells were harvested by centrifugation at $4 0 0 0 g$ for $3 0 \mathrm { m i n }$ at $2 7 7 \mathrm { K }$

The bacteria were lysed by sonication on ice at $2 0 0 ~ \mathrm { { W } }$ using $\mathrm { 4 ~ s ~ }$ pulseswith9sintervals for 99 cycles.Insoluble debriswas removed by centrifugation at $1 6 0 0 0 g$ for $4 0 \mathrm { m i n }$ at $2 7 7 \mathrm { K }$ .The supernatant was loaded onto $\mathrm { N i } ^ { 2 + } .$ -NTA His·Bind Superflow resin(Novagen,Darmstadt,Germany） pre-equilibrated with lysisbuffer( $5 0 \mathrm { m } M$ Tris-HCl $\mathrm { p H } 7 . 5$ ， $3 0 0 \mathrm { m } M$ NaCl, $1 0 \mathrm { m } M$ imidazole).After washing the resin with wash buffer1( $5 0 \mathrm { m } M$ Tris-HCl $\mathrm { p H } ~ 7 . 5$ ，1 M NaCl, $2 0 { \mathrm { m } } M$ imidazole）and wash buffer2( $5 0 { \mathrm { m } } M$ Tris-HCl pH7.5,1 MNaCl, $5 0 \mathrm { m } M$ imidazole)，the $\mathrm { H i s } _ { 6 }$ -tagged protein was eluted with an elution buffer consisting of $4 2 { \mathrm { m } } M$ ${ \mathrm { N a } } _ { 2 } { \mathrm { H P O } } _ { 4 }$ ， $8 { \mathrm m } M$ ${ \mathrm { N a H } } _ { 2 } { \mathrm { P O } } _ { 4 }$ $3 0 0 \ : \mathrm { m } M \ : \mathrm { N a C l }$ $2 5 0 \mathrm { m } M$ imidazole followed by buffer exchange on a $5 \mathrm { m l }$ HiTrap Desalting column(GE Healthcare,Uppsala, Sweden) with storage buffer ( $2 0 ~ \mathrm { m } M$ Tris-HCl pH7.5, $5 0 \mathrm { m } M$ KCl, $1 { \mathrm { m } } M$ DTT, $8 \%$ glycerol). Purified CouR was concentrated by centrifugation using an Amicon $3 0 \mathrm { k D a }$ centrifugal concentrator (Millipore） and the protein concentration was measured by the absorbance at $2 8 0 \mathrm { n m }$ . The protein was stored in the storage buffer and frozen at $1 9 3 \mathrm { K }$ until further use.

$p$ -Coumaroyl-CoA was enzymatically synthesizedand purified using an improved method based on the work of Till Beuerle(Beuerle&Pichersky，2OO2)，afterwhich itwas lyophilized. The compound wasdissolved in distilled deionized water and its concentration was estimated spectrophotometrically before the co-crystallization experiments.

Table 2 Crystallization.   

<html><body><table><tr><td></td><td>CouR</td><td>CouR-ligand</td></tr><tr><td>Final method</td><td>Hanging-drop vapour diffusion</td><td>Hanging-drop vapour diffusion</td></tr><tr><td>Plate type</td><td>16-well protein crystallization plates</td><td>16-well protein crystallization plates</td></tr><tr><td>Temperature (K)</td><td>293 35</td><td>293 80</td></tr><tr><td>Protein concentration (mg ml-1)</td><td></td><td></td></tr><tr><td>Buffer composition of protein solution</td><td>20 mM Tris-HCl pH7.5, 50 mM KCl, 1 mM DTT,8%_glycerol</td><td>20 mM Tris-HCl pH7.5,50 mMKCl, 1 mM DTT,8% glycerol</td></tr><tr><td>Composition of reservoir solution</td><td>0.1 M Tris-HCl pH 8.0, 3.0 M sodium formate</td><td>0.1Mbis-tris-HCl pH6.5,2%(w/v) Tacsimate pH 6.0, 30%(w/v)PEG 3350</td></tr><tr><td>Volume and ratio of drop</td><td>1 μl drops, 1:1 ratio</td><td>1 μl drops, 1:1 ratio</td></tr><tr><td>Volume of reservoir (μl)</td><td>500</td><td>500</td></tr></table></body></html>

# 2.2.Crystallization

The preliminary crystallization conditions for CouR and the CouR-ligand complex were screened by the sitting-drop vapour-diffusion method at $2 9 3 \mathrm { K }$ using commercial crystallization kits from Hampton Research,Aliso Viejo,California, USA.Each crystallization drop consisted of $1 \mu \mathrm { l }$ protein solution ( $( 2 0 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ CouR） mixed with $1 \mu \mathrm { l }$ screen solution and was equilibrated against $0 . 1 { \mathrm { m l } }$ reservoir solution.During optimization of the crystallization conditions, the method was changed to hanging-drop vapour diffusion. For apo CouR, preliminary crystals were observed in a condition consisting of $0 . 1 M \mathrm { T r i s \mathrm { - } H C l \ p H } 8 . 0 , 3 . 5 M$ sodium formate.By reducing the concentration of sodium formate to $3 . 0 { \ : } M$ and increasing the concentration of CouR to $3 5 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ ，we obtained crystals that were suitable for diffraction data collection (Fig.1a).For crystallization of the CouR-ligand complex,a molar ratio of 1:5 of CouR:ligand was used.Initial complex crystals were obtained in a condition consisting of $0 . 1 \ : M$ bis-tris $\mathsf { p H } 6 . 5$ $2 \% ( \nu / \nu )$ Tacsimate $\mathrm { p H } ~ 6 . 0$ + $2 0 \% ( w / \nu )$ PEG 3350.To obtain crystals that were suitable for data collection,we increased the concentration of PEG 3350 to $3 0 \%$ and the concentration of CouR to $8 0 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ while maintaining the molar ratio of CouR and its ligand.We also added DNase I(Bio Basic Inc., Markham, Canada) at a molar ratio of 1:1OoO (DNase I:CouR) to the protein-ligand solution. The inclusion of DNase I was necessary to optimize the crystals of the protein-ligand complex (Fig. 1b). When preparing the protein-ligand solution,we firstmixed CouRwithDNaseI,followedbyaddition of $p$ -coumarate-CoA.Precipitation resulted when the ligand was added and subsequently cleared on incubation on ice for $^ { 3 \mathrm { h } }$ . Detailed crystallization information is summarized in Table 2.

# 2.3.Data collection and processing

The CouR crystals were directly mounted on nylon CryoLoops (Hampton Research） and flash-cooled in a nitrogen stream at $1 0 0 \mathrm { K }$ .In contrast, the crystals of the CouR-ligand complex were first soaked for several seconds in a cryoprotectant solution that consisted of $9 5 \%$ reservoir solution and $5 \%$ glycerol. They were then transferred into paraffin oil using nylon CryoLoops for approximately 5 s before being flashcooled in a nitrogen stream.Diffraction data were collected at a wavelength of $0 . 9 7 9 3 \mathring { \mathrm { A } }$ on beamline BL17U-MX at

![](images/bccb70654607ce9e0dd6cb9c9decc51ca5ca81d221db25032b9342e9bd84261a.jpg)  
Crystals of CouR and the CouR-ligand complex. (a) Crystals of CouR. The largest is approximately $4 0 0 \times 1 4 \times 1 4 ~ { \mu \mathrm { m } }$ in size. $( b )$ Crystals of the CouRligand complex.

Shanghai Synchrotron Radiation Facility(SSRF) witha MAR DTB detector (Fig.2).All diffraction data sets were processed using MOSFLM(Leslie,20O6) and were scaled using $S C A L A$ (Evans,2006) from the CCP4 suite(Winn et al.,2011).Datacollection statistics are listed in Table 3.

# 3.Results and discussion

The DNA encoding CouR was optimized for $E .$ coli expression and a yield of approximately $2 7 \mathrm { m g }$ protein per litre of overexpression medium was observed in our experiments. During the Ni-NTA afinity purification,a concentration of $1 . 0 \ : M \ : \mathrm { N a C l }$ was required in the presence of 20 and $5 0 \mathrm { m } M$ imidazole to keep the target protein bound while removing other contaminating molecules from the column.After the Ni-NTA affinity column,we used a desalting column to remove the NaCl and imidazole and to change the buffer condition to one that was more suitable for crystallization screening. The purity of the CouR sample after the desalting column was greater than $9 5 \%$ as assayed by SDS-PAGE (Fig. 3).

The initial crystals of apo CouR obtained at a protein concentration of $2 0 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ were clustered thin needles and were difficult to optimize,despite efforts with a wide-ranging screen of additives and varying other conditions such as the $\mathrm { p H }$ and the precipitant concentrations.After we increased the concentration of CouR to $3 5 \mathrm { m g \ m l ^ { - 1 } }$ ,however,we were able to obtain crystals with a thicker needle shape.The crystals used in data collection were cut off from the needle clusters.

The crystallization behaviour of CouR changed dramatically in the presence of its ligand.Protein crystals were only observed in conditions containing PEG 3350, from which apo

Values in parentheses are for the outer shell.

Table 3 Data collection and processing.   

<html><body><table><tr><td></td><td>CouR</td><td>CouR-ligand</td></tr><tr><td>Diffraction source</td><td>Beamline BL17U, SSRF</td><td>Beamline BL17U, SSRF</td></tr><tr><td>Wavelength (A)</td><td>0.97930</td><td>0.97930</td></tr><tr><td>Temperature (K)</td><td>100</td><td>100</td></tr><tr><td>Detector</td><td>MARDTB</td><td>MARDTB</td></tr><tr><td>Crystal-to-detector distance (mm)</td><td>280</td><td>280</td></tr><tr><td>Rotation range per image ()</td><td>1</td><td>1</td></tr><tr><td>Total rotation range (°)</td><td>180</td><td>180</td></tr><tr><td>Exposure time per image (s)</td><td>1</td><td>1</td></tr><tr><td>Space group</td><td>P22121</td><td>P2221</td></tr><tr><td>Unit-cell parameters (A)</td><td>a= 62.78,b= 76.15, c = 87.38</td><td>a = 61.37,b= 69.82, c = 70.32</td></tr><tr><td>Mosaicity()</td><td>0.68</td><td>0.85</td></tr><tr><td>Resolution range (A)</td><td>48.44-2.50 (2.64-2.50)</td><td>49.55-3.30 (3.48-3.30)</td></tr><tr><td>Total No.of reflections</td><td>88646 (12416)</td><td>17842 (2615)</td></tr><tr><td>No.of unique reflections</td><td>15044 (2174)</td><td>4626 (662)</td></tr><tr><td>Completeness (%)</td><td>99.8 (100)</td><td>96.0 (97.0)</td></tr><tr><td>Multiplicity</td><td>5.9 (5.7)</td><td>3.9 (4.0)</td></tr><tr><td>(I/σ(I))</td><td>9.4 (2.8)</td><td>8.0 (2.0)</td></tr><tr><td>Rr.i.m.</td><td>0.052 (0.327)</td><td>0.059 (0.499)</td></tr><tr><td>Overall B factor from Wilson plot (A²)</td><td>40.9</td><td>87.9</td></tr></table></body></html>

CouR had failed to crystallize (Table 2).Based on these observations and the fact that the crystals obtained in the presence of the CouR ligand belonged to a different space group with different unit-cell parameters from those of the apo CouR crystals,we believed that the crystals were of the CouR-ligand complex rather than apo CouR crystals with a different morphology.For crystallization of the CouR-ligand complex, increasing the protein concentration to $8 0 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ reveal the molecular mechanisms of how CouR and other homologous transcriptional regulators recognize and bind $p$ -coumaroyl-CoA and modulate its DNA-binding affinities.

![](images/7c96e526615c858a7315777cf635f578f871342ddf55d8ec93bcddce691946d1.jpg)  
Figure 2 Representativediractionimages.(a)Difractionimageof anapoCoucrystal. (b)ifractionimageofaCouR-ligandcomplexcrystal.

# Acknowledgements

![](images/8622b8c57afc3739bddb9850d750d28ca1a6b45f149954e8877a4de73ba5a829.jpg)  
Figure 3 $12 \%$ SDS-PAGE.Lane 1,protein markers (molecular masses are labelled inkDa).Lane 2,purified CouR.

We are grateful for the technical support of the staff at beamline BL17U of Shanghai Synchrotron Radiation Facility during data collection. This work was supported by grants from the Project of the National Natural Science Foundation of China (No.31300498) to YG and from the National High TechnologyResearchand Development(863Program, 2011AA100203) to XNJ. This work was carried out in the Tree and Ornamental Plant Breeding and Biotechnology Laboratory of the State Forestry Administration and the National Laboratory of Protein Engineering.

also helped to obtain larger crystals,but the crystals were polycrystalline and were of poor quality; integration of the diffraction frames was unsuccessful. After many attempts, we eventually found that the inclusion of DNase Iin the proteinligand mixture at a molar ratio of 1:1Ooo (DNase I:CouR) led to single crystals with an improved diffraction quality.We propose that DNase I degrades the trace amount of DNA that binds to CouR and improves the homogeneity of the protein.

To determine the structures of CouR and the CouR-ligand complex,we plan to solve the phase problem by the molecularreplacement method using homologous MarR protein structures,such as PDB entries 2pex(Newberry et al.,20O7） and 3bpv(Saridakis et al.,2Oo8),as initial models. The results reported here lay a solid foundation for further research to

# References

Beuerle,T.&Pichersky,E.(2002).Anal.Biochem.302,305-312.   
Brinkrolf,K.,Brune,I.& Tauch,A.(2O06).Genet.Mol.Res.5, 773-789.   
Davis,J.R.& Sello,J.K.(2010).Appl.Microbiol.Biotechnol.86, 921-929.   
Evans,P.(2006).Acta Cryst.D62,72-82.   
Hirakawa,H.,Schaefer,A.L.,Greenberg,E.P.&Harwood,C.S. (2012). J. Bacteriol.194,1960-1967.   
Kasai,D.,Kamimura,N.,Tani,K.,Umeda,S.,Abe,T.,Fukuda,M.& Masai,E.(2012).FEMSMicrobiol.Lett.332,68-75.   
Leslie,A. G.W. (2006).Acta Cryst.D62,48-57.   
Newberry,K.J.,Fuangthong,M., Panmanee,W., Mongkolsuk,S.& Brennan,R.G.(2007).Mol. Cell,28,652-664.   
Parke,D. (1995).J.Bacteriol.177,3808-3817.   
Perera,I.C.& Grove,A.(2010).J.Mol.Cell.Biol.2,243-254.   
Saridakis,V.,Shahinas,D.,Xu,X.& Christendat,D.(20o8).J.Mol. Biol.377,655-667.   
Siehler,S.Y.,Dal,S.,Fischer,R.,Patz,P.& Gerischer,U.(2007). Appl.Environ. Microbiol. 73,232-242.   
Tropel,D.& van derMeer,J.R.(20o4).Microbiol.Mol.Biol.Rev.68, 474-500.   
Winn,M.D.et al.(2011). Acta Cryst. D67,235-242.