RESEARCHARTICLE

# Yersinia spp.Identification Using Copy Diversity in the Chromosomal 16S rRNA Gene Sequence

Huijing Hao1@, Junrong Liang1, Ran Duan1@, Yuhuang Chen1, Chang Liu1,2, Yuchun Xiao',Xu Li', Mingming $\mathsf { \pmb { s u } } ^ { 3 }$ , Huaiqi Jing1, Xin Wang1 \*

1 National Institute for Communicable Disease Control and Prevention,Chinese Center for Disease Control andPrevention,StateKeyLaboratoryof Infectious DiseasePreventionand Control,Collaborative Innovation Center for Diagnosisand Treatmentof Infectious Diseases,Beijing,China,2 Departmentof Pathogenic Biology,Schoolof Medical Science,JiangsuUniversity,Zhenjiang,Jiangsu Province,China,3 Instituteof Biophysics, Chinese Academy of Sciences, Beijing, China

These authors contributed equally to this work. \* wangxin @icdc.cn

Citation: Hao H, Liang J, Duan R, Chen Y, Liu C, Xiao Y,et al. (2016) Yersinia spp.Identification Using Copy Diversity in the Chromosomal 16S rRNA Gene Sequence.PLoS ONE11(1): e0147639.doi:10.1371/ journal.pone.0147639

Editor: Dongsheng Zhou,Beijing Institute of Microbiologyand Epidemiology, CHINA

Received:November27,2015

Accepted: January 6,2016

Published: January 25,2016

Copyright: $\circledcirc$ 2016 Hao et al.This is an open access articledistributed under the termsof the Creative Commons Attribution License,which permits unrestricted use,distribution,and reproduction in any medium,provided the original author and source are credited.

DataAvailability Statement:All relevant dataare within the paper and its Supporting Information files.

Funding: The National Natural Science Foundation of China (General Project,no.81470092); the NationalSci-Tech Key Project (2012ZX10004201, 2013ZX10004203-002).The funders had no role in study design,data collection and analysis,decision to publish,or preparation of the manuscript.

Competing Interests:The authors have declared that no competing interests exist.

# Abstract

APl 20E strip test, the standard for Enterobacteriaceae identification,is not sufficient to discriminate some Yersinia species for some unstable biochemical reactions and the same biochemical profile presented in some species, e.g.Yersinia ferderikseniiand Yersinia intermedia,which need a variety of molecular biology methods as auxiliaries for identification. The 16S rRNA gene is considered a valuable tool for assigning bacterial strains to species. However, the resolution of the 16S rRNA gene may be insufficient for discrimination because of the high similarity of sequences between some species and heterogeneity within copies at the intra-genomic level.In this study,for each strain we randomly selected five 16S rRNA gene clones from 768 Yersinia strains,and collected 3,840 sequences of the 16S rRNA gene from 10 species,which were divided into 439 patterns.The similarity among the five clones of 16S rRNA gene is over $9 9 \%$ for most strains. Identical sequences were found in strains of different species.A phylogenetic tree was constructed using the five 16S rRNA gene sequences for each strain where the phylogenetic classifications are consistent with biochemical tests;and species that are difficult to identify by biochemical phenotype can be differentiated. Most Yersinia strains form distinct groups within each species.However Yersinia kristensenii,a heterogeneous species, clusters with some Yersinia enterocoliticaand Yersinia ferderiksenilintermedia strains,while notaffecting the overall efficiency of this species classification.In conclusion,throughanalysis derived from integrated information from multiple 16S rRNA gene sequences,the discrimination ability of Yersinia species is improved using our method.

# Introduction

The genus Yersinia is widely distributed in nature and currently has 17 species [1-3],three of which are pathogenic (Y.enterocolitica,Y. pestis,and Y. pseudotuberculosis) and have been exhaustively researched.The remaining species are referred to as Y.enterocolitica-like bacteria where their etiology in disease is not understood [4,5].Traditionally,bacteria were classified according to similarities and differences in phenotypes,such as morphology and biochemical reactions.The API 2OE strip test is the standard for identifying Enterobacteriaceae; however this test has limitations in identifying Yersinia species.Several Yersinia species can be identified using API 2OE strip test where the accuracy is influenced by passage number,culture conditions,and instability of some biochemical reactions [6].Therefore, sensitive molecular biology methods are needed to assist the traditional approach to identify the Yersinia.

Since the 1980s,the ribosomal RNA gene has been used for phylogenetic studies,and ribosomal RNA-based approaches have been increasingly applied to bacterial classification and identification, especially using the 16S rRNA. The 16S rRNA gene is generally accepted as the best molecular sequence to use for identification because it is functionally constant and shows a mosaic of structure having conserved and variable regions and exists in all organisms; and its length is easily sequenced [7]. These properties make it uniquely well suited for systematics applications.However,the presence of multiple copies of the rRNA operons and intra-genomic heterogeneity in the 16S rRNA gene is regarded as a limiting factor in species identification [8- 10]. In bacteria, three rRNA genes (16S,23S,and 5S) are organized into a gene cluster, which is expressed as a single operon. The total number of rRNA operons in prokaryotes range from one to 15[10].With more and more bacterial genomes being completely sequenced, the heterogeneity of 16S rRNA gene at the intra-genomic level has been discussed. Pei collected 883 prokaryotic genomes from the GenBank database, representing 568 unique species,235 of which have copy diversity in the l6S rRNA gene within a genome ( $0 . 0 6 \%$ to $2 0 . 3 8 \%$ )[9].The diversity in 24 species are close to or over the threshold of the 16S rRNA gene-based operational definition of a species ( $1 \%$ to $1 . 3 \%$ diversity)[9],so these species maybe misclassified into a new species if a different copy is used for identification. Recently,2,013 genomes were analyzed and $2 2 . 5 \%$ were found divergence at over $1 \%$ in the 16S rRNA gene copies [11]. In addition, the 16S rRNA gene shows high similarity comparing some different species where identical sequences were found even[12]. This limits the use of the 16S rRNA gene in identifying bacterial species. Therefore,sequencing the 16S rRNA gene from multiple operons from isolates is recommended to achieve significant phylogenetic information for species identification [13].

Given the reasons above,we used the copy diversity in the 16S rRNA gene to identify Yersinia species by analyzing multiple copies of the 16S rRNA gene from 768 strains within the genus Yersinia.

# MaterialsandMethods

# Source of Strains

We used 768 strains of Yersinia including 10 species of Yersinia.Seven hundred and elevenone isolated strains were widely distributed within 21 provinces of China; and thirty-three reference strains were from Japan,France and the National Institutes for Food and Drug Control (NIFDC) in China. The dates of strain isolation encompass 52 years (1962-2014).Biochemical data were determined using API 2OE test strips (Biomerieux, France),showing 407 strains of Y.ferderiksenii/intermedia,119 strains of Y.kristensenii,123 strains of Y.enterocolitica (72 nonpathogenic and 51 pathogenic),46 strains of Y. pseudotuberculosis and 49 strains of Y. pestis.The source and hosts of strains are shown in Table 1. Twenty-four complete-genome

Table 1. Source and host distribution of Yersinia strains used.   

<html><body><table><tr><td colspan="2">Source and host Y.ferderiksenii/intermedia</td><td rowspan="2">Y. kristensenii</td><td colspan="2">Y. enterocolitica</td><td rowspan="2">Y.pseudotuberculosis</td><td rowspan="2">Y. pestis</td></tr><tr><td></td><td></td><td></td><td>Non-pathogenic pathogenic</td></tr><tr><td>Strains isolated in China</td><td>Chicken</td><td>21</td><td>17</td><td>6</td><td></td><td></td><td></td></tr><tr><td></td><td>Cattle</td><td>6</td><td></td><td>1</td><td></td><td></td><td></td></tr><tr><td></td><td>Dogs</td><td>16</td><td>6</td><td>3</td><td></td><td></td><td></td></tr><tr><td></td><td>Rats</td><td>166</td><td>31</td><td>43</td><td>8</td><td>28</td><td>2</td></tr><tr><td></td><td>Ducks</td><td>2</td><td>4</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Goat</td><td>4</td><td>4</td><td>2</td><td></td><td></td><td></td></tr><tr><td></td><td>Mandarin duck</td><td>5</td><td>7</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Swines</td><td>148</td><td>30</td><td>10</td><td>19</td><td>3</td><td></td></tr><tr><td></td><td> Marmots</td><td></td><td></td><td></td><td></td><td></td><td>31</td></tr><tr><td></td><td>Flies</td><td>2</td><td></td><td>1</td><td>1</td><td></td><td></td></tr><tr><td></td><td>Ticks</td><td></td><td></td><td></td><td></td><td></td><td>2</td></tr><tr><td></td><td>Fleas</td><td></td><td></td><td></td><td></td><td></td><td>6</td></tr><tr><td></td><td> Diarrhea patients</td><td>12</td><td>1</td><td>4</td><td>16</td><td></td><td>4</td></tr><tr><td></td><td>Food</td><td>8</td><td>9</td><td>2</td><td></td><td></td><td></td></tr><tr><td></td><td>Otherse</td><td>9</td><td>4</td><td></td><td>3</td><td></td><td>4</td></tr><tr><td></td><td>Total</td><td>407</td><td>119</td><td>72</td><td>51</td><td>46</td><td>49</td></tr><tr><td>Reference strains</td><td></td><td>8a</td><td>6b</td><td></td><td>4°</td><td>15d</td><td></td></tr></table></body></html>

aAleferencestrnsideaefroNtefrderistrans(235nd6)ndsireYteditrais(7 52244,52248, 52249,and 52250). bAmong the reference strains,four are from NIFDC (52232, 5224, 52246,and 52247),and two from Japan. （204 $^ \mathrm { c } \mathsf { A l l }$ reference strains are from Japan. Amongthese,irefrN(350,35055105123514nd5518)eight(B,P1BB2B01114Ad) from Japan,and one (YPo10) from France. e All bacteria strains were collected from animals, not human subjects.

doi:10.1371/journal.pone.0147639.t001

sequenced strains were selected from the NCBI (http://www.ncbi.nlm.nih.gov/pubmed/) with the GenBank number listed in Table 2.

# Culture and ldentification of Strains

Enrichment was performed using phosphate-buffered saline with sorbitol and bile salts (PSB) a $4 ^ { \circ } \mathrm { C }$ for 21 days. Then strains were inoculated onto Yersinia-selective agar (cefsulodin-irgasan-novobiocin [CIN] agar; Difco). Suspected colonies having a typical bul's-eye appearance (deep red centers surrounded by an outer transparent zone) on CIN agar were selected and allowed to grow on brain heart infusion agar (Beijing Land Bridge Technology Co.,Ltd., China) at $2 5 ^ { \circ } \mathrm { C }$ for 24 to 48 hours to obtain pure strains[14,15]. We used the commercial rapid identification system, API 20E strip test (bioMérieux,France),for the identification of presumptive isolates.The biotypes of Y.enterocolitica strains were identified using the scheme reviewed by Bottone [16].

Table 2. The GenBank numbers of 24 complete-genome-sequenced strains.   

<html><body><table><tr><td>Strains</td><td>Genbank number</td><td> Strains</td><td>Genbank number</td></tr><tr><td>Y.aldovae 670-83</td><td>CP009781.1</td><td>Y. pestis Pestoides F</td><td>CP000668.1</td></tr><tr><td>Y. aleksiciae strain 159</td><td>CP011975.1</td><td> Y. pseudotuberculosis IP31758</td><td>CP000720.1</td></tr><tr><td>Y.ferderiksenii ATCC33641</td><td>KN150731.1</td><td>Y. pseudotuberculosis PB1/+</td><td>CP001048.1</td></tr><tr><td>Y. ferderiksenii Y225</td><td>CP009364.1</td><td>Y. pseudotuberculosis YPIll</td><td>CP000950.1</td></tr><tr><td>Y. intermedia strain Y228</td><td>CP009801.1</td><td>Y. rohdei strain YRA</td><td>CP009787.1</td></tr><tr><td>Y. kristenseni ATCC33639</td><td>CP008955.1</td><td>Y. ruckeri strain YRB</td><td>CP009539.1</td></tr><tr><td>Y. kristensenii Y231</td><td>CP009997.1</td><td>Y. similis strain 228</td><td>CP007230.1</td></tr><tr><td>Y. pestis A1122</td><td>CP002956.1</td><td>Y. enterocolitica strain 2516-87</td><td>CP009838.1</td></tr><tr><td>Y. pestis D106004</td><td>CP001585.1</td><td>Y. enterocolitica strain WA</td><td>CP009367.1</td></tr><tr><td>Y. pestis D182038</td><td>CP001589.1</td><td>Y. enterocolitica subsp. enterocolitica 8081</td><td>AM286415.1</td></tr><tr><td>Y. pestis KIM10+</td><td>AE009952.1</td><td> Y. enterocolitica subsp. palearctica Y11</td><td>FR729477.2</td></tr><tr><td>Y. pestis Nepal516</td><td>CP000305.1</td><td>Y. enterocolitica subsp. palearctica 105.5R(r)</td><td>CP002246.1</td></tr></table></body></html>

doi:10.1371/journal.pone.0147639.t002

# T-vectors Cloning and Sequencing of the 16S rRNA Gene

The bacterial genome was extracted using a DNA nucleic acid extraction kit (Tiangen, China). The l6S rRNA gene was amplified using the universal primers 27F ( $\overrightarrow { 5 }$ -AGAGTTTGATCCTG GCTCAG-3') and1492R( $\cdot \overrightarrow { 5 }$ -GGTTACCTTGTTACGACTT-3')[17]with the Q5 Hot StartHighFidelity $2 \times$ Master Mix(NEB,U.K.).After gel electrophoresis,specific PCR products were purified using a pEASY purification kit (Transgene, China). Connection and transformation were performed using the pEAsYTM-Blunt Simple Cloning Kit (Transgene, China). The transformed bacteria were grown at $3 7 ^ { \circ } \mathrm { C }$ for $1 2 \mathrm { h }$ on Luria-Bertani agar, containing X-gal $\left( 2 0 \mathrm { m g / m l } \right)$ ,IPTG $\left( 2 4 \mathrm { m g / m l } \right)$ ,and kanamycin $\mathrm { ( 5 0 ~ m g / m l ) }$ .Five white single colonies were randomly selected for sequencing. All were sequenced with an ABI Prism BigDye Terminator cycle sequencing ready reaction kit using AmpliTaq DNA polymerase and an ABI Prism 377xl DNA sequencer (Applied Biosystems,Foster City, CA, USA) according to the instructions of the manufacturer at Tsingke BioTech Co.,Ltd., sequencing the amplicons in both directions.

# Sequence Analysis

Sequence alignment was performed using Seqman (Lersergene7.O).A phylogenetic tree based on a single copy of the 16S rRNA gene was constructed using Kimura's 2-parameter distance and the neighbor-joining method (MEGA 6.0). All of the 16S rRNA gene copies from each complete-genome-sequenced strain were coded with a random number,and then five sequences were selected using a random number table.After elimination of redundancies,all the five sequences in the 16S rRNA gene of each strain were obtained corresponding to a unique pattern number.The minimum distance between species was calculated on the basis of a full array between every two strains and each of the five sequences. A phylogenetic tree based on the multi-copy of the diverse l6S rRNA gene was constructed using a distance matrix. The distance of the five sequences within each strain was also calculated.

# Ethics

The sample collection and detection protocols were approved by the Ethics Review Committee [Institutional Review Board (IRB)] of National Institute for Communicable Disease Control and Prevention, Chinese Center for Disease Control and Prevention.The copy of ethical approval documents were provided comply with PLOS ONE requirements. The authors were involved in sample collections involving human subjects and authors can access to personal information before the authors had access to the data.Signed informed consent was obtained from all study participants.For all the patients under 18 years-old,a written consent form was signed by a parent or legal guardian. The field studies did not involve endangered or protected species,so the locations/activities for which specific permission was not required.

# Results

# 16S rRNA Copies at the Intra-Genomic Level

In total, 3,840 sequences of 16S rRNA gene were obtained from 768 strains used in this study. One or more base difference between two sequences was defined as a different 16S rRNA gene

![](images/72ba1fb6cfc2f6aaece90d2c3c76557c55f7ca6f5ec68a090075ef85ba10148b.jpg)  
Fig1.ADisrutiooftetyeumbofAgesi8Yrsas.eolorsitsctioofeartrte typenumberofttttsfurtsfiespeielybine thenumberoftrainsthatveeachindfopisofAgenethepercetaginprentesesepresentsthproprtiooftains.B. TheproportiofchcoyaparingindiferentYrsiniaspces.CThidentical6SApaesthaeistindfrentYeriaies exceptforptisdoublosmsiditmtlpsioeoi species.Numbersinparenthesesrepresent theamountoftotalpattrns incorrespondingspecies.Specificpattesarenotshown.

doi:10.1371/journal.pone.0147639.g001

type.The type number of intra-strain 16S rRNA genes is shown in Fig 1A. In general, $6 0 \%$ of the strains have two or three types, $1 7 \%$ have four, and $1 8 \%$ have one. Only $5 \%$ of the strains have five totally diferent 16S rRNA gene types.Fig 1B shows a comparison of the gene type number between the diferent Yersinia species.The number is primarily one or two in Y. pseudotuberculosis and Y. kristensenii,and two to four in Y.ferderikseni/intermedia. The strains of pathogenic Y.enterocolitica usually have one to three types, whereas the non-pathogenic strains have two to three types where one strain having five identical copies is uncommon at about $8 \%$ of the strains. Y. pestis and Y. pseudotuberculosis do not have five different copy sequences.

# Pairwise Comparison of the 16S rRNA Gene at the Intra-Genomic Level

The sequence similarity comparison between pairwise copies of 16S rRNA genes in each strain is shown in Table 3 classified by Yersinia species.The sequence similarity of pairwise copies in each Yersinia species strain exceeds $9 9 \%$ for most strains.There are only nine Y.ferderiksenii/ intermedia (accounting for $2 . 2 \%$ )of the strains with similarity below $9 8 . 7 \%$ ,and one strain of Y.kristenseni and non-pathogenic Y.enterocolitica of relatively low similarity $9 6 . 7 7 \%$ and $9 7 . 9 4 \%$ ,respectively,whose sequence variation are 47 and 30 bases,respectively.

Table 3.Pairwise comparisonof 16S rRNA gene at the intra-genomic level ineach Yersinia species.   

<html><body><table><tr><td rowspan="2">Species</td><td colspan="3">Similarity*</td><td rowspan="2">Minimum value (%)</td></tr><tr><td>100%</td><td>100-99%</td><td><98.7%</td></tr><tr><td>Y. ferderiksenii/intermedia</td><td>52(12.7%)</td><td>349(85.1%)</td><td>9(2.2%)</td><td>98.15</td></tr><tr><td>Y. kristensenii</td><td>45(37.2%)</td><td>75(62.0%)</td><td>1(0.8%)</td><td>96.77</td></tr><tr><td>Pathogenic Y. enterocolitica</td><td>12(22.6%)</td><td>40(75.5%)</td><td>1(1.9%)</td><td>98.22</td></tr><tr><td>Non-pathogenic Y. enterocolitica</td><td>6(8.0%)</td><td>68(90.7%)</td><td>1(1.3%)</td><td>97.94</td></tr><tr><td>Y. pseudotuberculosis</td><td>15(30.6%)</td><td>34(69.4%)</td><td></td><td>99.45</td></tr><tr><td>Y. pestis</td><td>5(9.1%)</td><td>50(90.9%)</td><td></td><td>99.79</td></tr></table></body></html>

‘ There is notone strain in our study with similrity between $9 9 \% - 9 8 . 7 \%$ in differentcopies of16S rRNA gene,s0 thegroupof $< 9 9 \%$ $2 9 8 . 7 \%$ is not shown in Table 3. The group of $1 0 0 - 9 9 \%$ means the similarity $< 1 0 0 \%$ and $2 9 9 \%$

doi:10.1371/journal.pone.0147639.t003

# Cluster Analysis Based on 16S rRNA Gene Sequence

A phylogenetic tree (Fig 2), constructed on the basis of the five copies of 16S rRNA gene presented in each strain and using the minimum evolution method,can be divided into six groups: la, 1b,2,3,4 and 5.

Group la and group lb comprise Y.ferderiksenii/intermedia strains,220 and 90 strains, respectively.The similarity between strains of each group is over $9 0 \%$ .Group3primarily

![](images/c2b9ea8514b7124fe54592d5a9387e24b6c7cc0f26d5f6aad4a8c9c20a66d464.jpg)  
Fig2.ApylogenetictreconstructedothebsisoftfivecopisofSAgeneneachstranusingtheinmumevotiotd.A. Dotswithittspssostiiit clustering groups.

doi:10.1371/journal.pone.0147639.g002

contains Y.enterocolitica strains; but clustered with three Y.kristeniistrains and one Y.ferderiksenii strain.Y.enterocolitica strains of biotype 1b show a distinct sub-cluster (the dark blue and bold clade in Fig 1A) in group 3,three isolated strains and two complete-genomesequenced strains WA and 8081. Two sub-clusters comprise group 4: one is a combination of Y.ferderikseni/intermedia strains,represented by two reference strains of Y.ferderksenii, ATCC3364l and 52235; another one contains four Y.enterocolitica strains and one completegenome-sequenced strain of Y.rohdei. Group 5 consists of all the strains of Y. pestis and Y. pseudotuberculosis involved in this study,and a complete-genome-sequenced strain of Y. similis, located outside of the group.

Group 2 contains multiple Yersinia species,such as Y.enterocolitica,Y.ferderikseni/intermedia and Y.kristenseni, which can be divided into three clusters:a,b and c. Group 2a includes two reference strains of Y.kristensenii and one complete-genome-sequenced strain of Y.aleksiciae. Group 2b is complicated hence described in three parts. Group 2b-1 is primarily made up of Y. kristeni strains, next is two reference strains of Y.kristenseni, 52246 and 52232,and one complete-genome-sequenced strain of Y.ruckeri. Group 2b-2 has two sub-clusters, one consisting of an Y.enterocolitica strain and a whole-genome-sequenced strain of Y. aldovae,and another with 43Y.ferderiksenii/intermedia strains ofhigh similarity $( > 9 9 . 6 \% )$ , two of which are reference strains of Y.intermedia.Group 2b-3 has three sub-clusters.The first one contains $1 3 Y .$ kristensenii strains, five Y.enterocolitica strains,and one complete-genome-sequenced strain of Y.ferderiksenii Y225 that is distant from the rest of the strains in this sub-cluster.The second contains 33Y.kristensenii strains of high similarity $( > 9 9 . 6 \%$ ),except fortwo Y.ferderiksenii/kristensenii strains.The third with high similarities between strains $( > 9 9 . 6 \%$ ),presents diversity,containing 35 Y.kristensenii (two are complete-genome-sequenced) and 43 Y.ferderiksenii/intermedia strains (three reference strains of Y. intermedia and one complete-genome-sequenced strain of Y. intermedia). There is a single Y.ferderiksenii/intermedia strain alone between group 2-l and group 2b-2. Group 2c contains nine Y. enterocolitica strains, two Y. kristenseni strains and one complete-genome-sequenced strain of Y.ferderiksenii.

# The Dominant Patterns of the 16S rRNA Gene in Each Yersinia Species

Four hundred thirty-nine patterns of 16S rRNA gene sequences, coded as O to 438(S1 File and S1 Table), were obtained after removing the redundant sequences. Table 4 shows each Yersinia species has dominant patterns.Though non-dominant patterns are more than dominant

Table 4. The dominant patterns of the 16S rRNA genes in each Yersinia species.   

<html><body><table><tr><td>Species</td><td>No. strains</td><td>No. sequences of 16S rRNA gene</td><td>No. patterns of 16S rRNA gene</td><td>Dominant 16S rRNA gene pattern and its percentage</td></tr><tr><td>Pathogenic- Y. enterocolitica</td><td>53</td><td>265</td><td>48</td><td>32(51.7%)</td></tr><tr><td>Nonpathogenic- Y. enterocolitica</td><td>75</td><td>375</td><td>90</td><td>19(20.8%) 120(10.4%)</td></tr><tr><td>Y. ferderiksenii/intermedia group 1a</td><td>220</td><td>110</td><td>75</td><td>11(41.5%) 13(17.5%) 23(10.1%)</td></tr><tr><td>Y.ferderiksenii/intermedia group 1b</td><td>91</td><td>455</td><td>38</td><td>142(31.2%) 3(18.7%) 55(12.1%)</td></tr><tr><td>Y.ferderikseni/intermedia group 2b</td><td>88</td><td>440</td><td>58</td><td>10(20.0%) 111(13.2%) 8(13.0%) 9(11.4%)</td></tr><tr><td>Y.ferderiksenii/intermedia group 4</td><td>7</td><td>35</td><td>20</td><td>72(14.3%) 98(11.4%)</td></tr><tr><td>Y. kristensenii</td><td>121</td><td>605</td><td>80</td><td>10(31.9%) 95(14.7%)</td></tr><tr><td>Y. pestis</td><td>55</td><td>275</td><td>45</td><td>1(34.2%)</td></tr><tr><td>Y.pseudotuberculosis</td><td>49</td><td>245</td><td>28</td><td>1(64.1%)</td></tr></table></body></html>

doi:10.1371/journal.pone.0147639.t004

patterns,they appear at a low frequency and usually appear only once. There exists the same dominant gene pattern in the strains of Y. kristenseni and Y.ferderiksenii/intermedia,and Y. pestis and Y. pseudotuberculosis, respectively.

# Identical 16S rRNA Gene Sequences from Different Yersinia Species

Strains of different Yersinia species have identical 16S rRNA gene sequences (Fig 1C).Nonpathogenic Y.enterocolitica strains (biotype 1A) have the same l6S rRNA gene patterns with many kinds of other Yersinia species, with nine same gene patterns with pathogenic Y.enterocolitica,four with Y.kristensenii strains and only one with the other Yersinia species.Y.kristensenii strains have three identical gene patterns with Y.ferderikseni/intermedia strains from group 2b,one of which is dominant,accounting $20 \%$ for Y. kristensenii and $3 1 . 9 \%$ for Y.ferderiksenii/intermedia respectively. There also exist identical 16S rRNA gene patterns among nonpathogenic Y.enterocolitica, Y. kristenseni,and Y.ferderiksenii/intermedia strains from group $^ { 2 \mathrm { b } , }$ and also among non-pathogenic Y.enterocolitica, Y.kristenseni,and pathogenic Y.enterocolitica. Only group la and 1b exist three same l6S rRNA gene patterns among the four groups of Y.ferderikseni/intermedia (la,1b,2 and 4). Y. pestis strains have six identical patterns with Y. pseudotuberculosis (not shown in Fig 1C),one of which is dominant in both species, accounting for $3 4 . 2 \%$ and $6 4 . 1 \%$ ,respectively.

# Discussion

Compared to bacteria identification methods using phenotype,the approach based on genotype stands out for its consistency. One desirable candidate is the 16S rRNA gene, highly conserved and seldom variable within species,and is becoming an important technique for phylogeny research and species classification[7]. Specifically speaking, for Yersinia,the limitation of commercial tests for identification systems such as API 2OE strip test has shown a deficiency in distinguishing some species[18,19],which can be made up through the method based on $1 6 S \mathrm { r R N A }$ gene. However, the similarity of 16S rRNA gene sequence between species is as high as $9 6 . 9 \% 9 9 . 8 \%$ ； it is easy therefore to misclassify species of high homology[20]. Recently,the rRNA operon is shown to have multiple copies with some heterogeneity[17]. In some cases,the diversity between multiple copies of a strain is so high as to be misclassified into different species if a different copy is used to identify a bacterial strain [21,22].This is a limiting factor for species identification using a single 16S rRNA gene.The discrimination ability can be improved by integrating multiple 16S rRNA gene copies. Therefore,here we first report species differentiation using five copies of the 16S rRNA gene from 768 strains of Yersinia.

Previously, the copy number of Yersinial6S rRNA gene was shown to be around seven [11]. According to whole genome sequence strain from NCBI, the number of variable copies is within three. Accordingly, five colonies was randomly selected for 16S rRNA gene sequencing for each strain. Our results show $6 0 \%$ of the strains have two to three genotypes and only $5 \%$ have all 5 types,therefore the selection of 5 colonies is reasonable.Though $8 4 \%$ of the strains have more than one sequence type of 16S rRNA gene,the comparison of the intra-genomic 16S rRNA gene showed that the similarity of $9 8 . 4 \%$ of the strains is above the threshold of species identification ( $9 8 . 7 \%$ to $9 9 \%$ )[9].In other words,for most Yersinia strains,the heterogeneity among different copies does not affect species classification.Whereas for 12 strains, the similarity of intra-genomic 16S rRNA gene is below or close to the threshold of classification. When using a single copy for identification,a different result may occur. Hence in this study, with integrated information from multiple 16S rRNA genes and a comprehensive reflection of phylogeny, species identification is more accurate.

![](images/07d6234066a1ac1a06f941c8f72f53505f799ebf60d89261b58592ed8caf86b1.jpg)  
Fig3.Phylogenetictreebasedonsingle16SNAgenefroY.ferderiksenilintermediastrainsingroupa,1band4andstrasofY. ferderisdis 4;wilsolidtss and group 1b.

doi:10.1371/journal.pone.0147639.g003

A phylogenetic tree (Fig 2) was constructed by combining five l6S rRNA gene sequences for each strain, most clustering of species is accordant with classical identification. Y. pestis and Y. pseudotuberculosis share a large number of identical 16S rRNA gene sequences and are close to each other in the phylogenetic tree,which is consistent with the hypothesis that Y. pestis evolved from O: 1b strains of Y. pseudotuberculosis about 1,500-20,000 years ago[23,24], indirectly supporting our method. Y.enterocolitica strains are in group 3,where Y.enterocolitica subsp.enterocolitica (biotype 1b) and Y.enterocolitica subsp.Palearctica (biotype 1A,2,3,4 and 5) diverge into two clusters, coinciding with previous work[25-27].

The majority of Y. kristensenii strains are within group 2,spreading widely and having multiple branches.In group2b-3-3,Y.kristenseniiand Y.ferderiksenii/intermedia strains are closely clustered with each other,sharing high similarity,due to the fact that the consensus sequence of the two species are also dominant in each species.Notably, very few Y. kristenseni strains cluster with Y.enterocolitica strains. Compared with other species, Y. kristensenii presents higher genetic variability. This may be owing to the uncertainty of biochemical identification that these cross-over’ strains may be classified into other Yersinia species. Sequencing more than five random selected colonies may solve this problem and needs further investigation.

Y.ferderiksenii and Y.intermedia strains are indistinguishable using API 2OE strip test where they are referred to as Y.ferderiksenii/intermedia.In this study,they are gathered within group la,1b,2b and 4.The reference strains of Y.intermedia were allin group 2b,so strains in this group should be classified to be Y. intermedia strains. Consequently, strains in group la, 1b and 4 belong to Y.ferderiksenii. Identified in 198o, Y.ferderiksenii included three geno-species without biochemical differences [28]. Clustering analysis showed sequence types of group la,lb,and 4 clustered with geno-species 2,3,1of Y.ferderiksenii strains,respectively (Fig 3). Therefore,the 16S rRNA gene sequence types can be used for unclassified strains from API 20E strip test, and also for subtyping strains.

Presently there is divergence in species classification for Yersinia.In addition to traditional biochemical reactions,refined molecular biology techniques are needed to compensate for this deficiency. Y. aleksiciae and Y. kristensenii have no difference in their biochemical phenotype but differ into distinct species using 16S rRNA gene sequences[29]. In group 2a, Y. kristensenii reference strains (52242 and 52247) share $9 9 . 9 \%$ similarity with the genome sequenced strain of Y.aleksiciae,a recent phylogenetic relationship.A previous study shows Y.similis is indistinguishable from Y. pseudotuberculosis through biochemical phenotyping that it was regarded as Y. pseudotuberculosis; however,their pathogenicity are significantly different and they differed using l6S rRNA gene sequencing [30]. In this study, genome sequenced strain of Y. similis were inseparably clustered with Y. pseudotuberculosis and Y. pestis in group 5,an outlier (black dot in group 5 Fig 2A) with an outermost evolutionary distance.Further,several comparable subspecies clustered respectively in group2b-1 and group 2b-2: genome sequenced Y. ruckeri strain and four Y.kristenseniistrains formed a sub-branch; Y.aldovae and one Y.enterocolitica strain formed another,far away from the rest of the Y.enterocolitica strains.We suspect this Y. enterocolitica strain may belong to another undescribed Yersinia species.Similarly,in group 4, three Y.enterocolitica strains formed another sub-branch with one genome sequenced strain of Y.rohdei.All of these strains show Yersinia is a genetically diverse genus; the strains involved here may be classified into other species of more diversity.

Based on integrated information from multiple 16S rRNA gene copies and clustering analysis,this study shows comparable species clusters of Yersinia and reduces identification chaos derived from using a single copy of the 16S rRNA gene sequence which is sometimes similar or identical between species,and remarkably increases the discrimination ability of 16S rRNA gene sequence.Further study using our classification method would allow increased differentiation of Yersinia species identification.

# Supporting Information

S1 File. The 439 Patterns (Coded as O to 438) of 16S rRNA Gene Sequences in this study. (PDF)

S1 Table.Patterns Code of Five 16S rRNA Gene Sequences for each Strain. (XLSX)

# Acknowledgments

We thank Liuying Tang and Jim Nelson for critical reading and helpful comments on our manuscript.

# Author Contributions

Conceived and designed the experiments: HJ XW.Performed the experiments: HH RD YC CL YX XL. Analyzed the data: HH JL MS. Contributed reagents/materials/analysis tools: HH JL MS. Wrote the paper: HH JL RD HJ.

# References

1.Souza RA,Imori PF,Falcao JP.Multilocus sequence analysis and 16S rRNA gene sequencing reveal that Yersinia frederikseni genospecies 2is Yersinia massliensis.International journal of systematic and evolutionary microbiology.2013; 63(Pt8):3124-9.doi: 10.1099/ijs.0.047175-0 PMlD: 23908151.   
2.NeubauerH,Sprague LD.Strainsof Yersinia wautersiishould continue to be classifiedas the 'Korean Group'of the Yersinia pseudotuberculosis complex and notas aseparate species.International journal of systematic and evolutionary microbiology.2015; 65(Pt2):732-3.doi: 10.1099/ijs.0.070383-0 PMID: 25505347. 3．Savin C,MartinL,Bouchier C,Filali S,Chenau J,Zhou Z,etal.The Yersinia pseudotuberculosis complex: characterizationand delineation ofanewspecies,Yersinia wautersi. International journal of medical microbiology: IJMM. 2014; 304(3-4):452-63.doi: 10.1016/j.jmm.2014.02.002 PMID: 24598372. 4. Sulakvelidze A.Yersiniae other than Y.enterocolitica,Y.pseudotuberculosis,and Y.pestis:the ignored species. Microbes and infection /Institut Pasteur. 2000; 2(5):497-513. PMlD: 10865195. 5. Mittal S,Mallik S, Sharma S, Virdi JS. Characteristics of beta-lactamases and their genes (blaA and blaB) in Yersinia intermedia and Y.frederiksenii. BMC microbiology.2007; 7:25.doi: 10.1186/1471- 2180-7-25 PMID:17407578; PubMed Central PMCID:PMC1853101. 6. Woo PC,Ng KH,Lau SK,Yip KT,Fung AM,Leung KW,et al. Usefulness of the MicroSeq 500 16S ribosomal DNA-based bacterialidentificationsystemfor identification of clinically significant bacterial isolates with ambiguous biochemical profiles.Journal of clinical microbiology.2003; 41(5):1996-2001. PMID: 12734240; PubMed Central PMCID: PMC154750. 7. Woese CR. Bacterial evolution. Microbiological reviews.1987; 51(2):221-71. PMID: 2439888; PubMed Central PMCID: PMC373105. 8. Fox GE,Wisotzkey JD,Jurtshuk P Jr. How close is close: 16S rRNA sequence identity may notbe sufficientto guarantee species identity. International journal of systematic bacteriology.1992; 42(1):166- 70. PMID: 1371061. 9. Pei AY,Oberdorf WE, Nossa CW,AgarwalA,ChokshiP,Gerz EA,etal. Diversityof16S rRNA genes within individual prokaryotic genomes. Applied and environmental microbiology.2010;76(12):3886- 97.doi: 10.1128/AEM.02953-09 PMID:20418441; PubMed Central PMCID:PMC2893482.   
10.Klappenbach JA,Saxman PR, Cole JR,Schmidt TM.rrndb: the Ribosomal RNA Operon Copy Number Database. Nucleic acids research.2001; 29(1):181-4. PMID: 11125085; PubMed Central PMCID: PMC29826.   
11.Sun DL,Jiang X,Wu QL,Zhou NY.Intragenomic heterogeneityof 16S rRNA genes causes overestimation of prokaryotic diversity. Applied and environmental microbiology.2013;79(19):5962-9.doi: 10. 1128/AEM.01282-13 PMID: 23872556; PubMed Central PMCID: PMC3811346.   
12. Jaspers E,Overmann J.Ecological significance of microdiversity: identical 16S rRNA gene sequences can be found in bacteria with highly divergent genomes and ecophysiologies.Applied and environmental microbiology.2004; 70(8):4831-9. doi: 10.1128/AEM.70.8.4831-4839.2004 PMID: 15294821; PubMed Central PMCID: PMC492463.   
13. Clayton RA,Sutton G, Hinkle PS Jr.,Bult C,Fields C.Intraspecific variation in smal-subunit rRNA sequences in GenBank: why single sequences may not adequately represent prokaryotic taxa.Interna tional journal of systematic bacteriology.1995; 45(3):595-9.PMlD: 8590690.   
14. Wang X,Cui Z,Jin D,Tang L, Xia S,Wang H,etal. Distribution of pathogenic Yersinia enterocoliticain China.European journal of clinical microbiology&infectious diseases: oficial publication of the Euro pean Society of Clinical Microbiology.2009; 28(10):1237-44.doi: 10.1007/s10096-009-0773-x PMID: 19575249.   
15.Liang J,Wang X, XiaoY,CuiZ, XiaS,Hao Q,etal.Prevalence of Yersiniaenterocolicain pigs slaughtered in Chinese abattoirs.Applied and environmental microbiology.2012; 78(8):2949-56.doi: 10. 1128/AEM.07893-11 PMID: 22327599; PubMed Central PMCID: PMC3318836.   
16.Botone EJ.Yersinia enterocolica: a panoramic view of a charismatic microorganism. CRC critical reviews in microbiology. 1977; 5(2):211-41. PMID: 844324.   
17. Moreno C, Romero J,Espejo RT.Polymorphismin repeated 16S rRNA genes is acommon property of type strains and environmental isolates of the genus Vibrio. Microbiology.2002; 148(Pt 4):1233-9. PMID: 11932467.   
18. Neubauer H,SauerT,Becker H,Aleksic S,Meyer H.Comparison of systems foridentification and differentiation of species within the genus Yersinia.Joumal of clinical microbiology.1998;36(11):3366-8. PMID: 9774596; PubMed Central PMCID: PMC105332.   
19.Linde HJ,Neubauer H, Meyer H,Aleksic S,Lehn N.Identificationof Yersinia species bythe Vitek GNI card.Journal of clinical microbiology.1999; 37(1):211-4.PMID:9854094; PubMed Central PMCID: PMC84211.   
20. IbrahimA,Goebel BM,Liesack W,Grifiths M,StackebrandtE.Thephylogenyof the genus Yersinia based on 16S rDNA sequences.FEMS microbiology letters.1993;114(2):173-7.PMID:7506685.   
21.Wang Y, Zhang Z,Ramanan N.Theactinomycete Thermobispora bispora contains two distinct types of transcriptionally active 16S rRNA genes. Journal of bacteriology.1997; 179(10):3270-6.PMID: 9150223; PubMed Central PMCID: PMC179106.   
22. Yap WH, Zhang Z,Wang Y.Distincttypes of rRNA operons existin the genome of the actinomycete Thermomonosporachromogena and evidence for horizontal transfer of anentire rRNAoperon. Journal of bacteriology.1999; 181(17):5201-9.PMID:10464188; PubMed Central PMCID: PMC94023.   
23.Achtman M,Zurth K,Moreli G,TorreaG,Guiyoule A,CarnielE.Yersinia pestis,the causeof plague,is arecently emerged clone of Yersiniapseudotuberculosis.Proceedings of the National Academy of Sciences of the United States of America.1999; 96(24):14043-8.PMID:10570195; PubMed Central PMCID:PMC24187.   
24. Kim W,Song MO,Song W,Kim KJ,Chung SI,ChoiCS,etal.Comparison of 16S rDNA analysis and rep-PCR genomic fingerprinting for molecular identification of Yersinia pseudotuberculosis.Antonie van Leeuwenhoek.2003;83(2):125-33.PMID:12785306.   
25. Neubauer H,Aleksic S,Hensel A,Finke EJ, Meyer H.Yersinia enterocolitica 16S rRNA gene types belong to the same genospecies but form three homology groups.International journal of medical microbiol0gy: IJMM.2000; 290(1):61-4. doi: 10.1016/S1438-4221(00)80107-1 PMID: 11043982.   
26. SihvonenLM,JalkanenK,HuovinenE,ToivonenS,CoranderJ,Kuusi M,etal. Clinicalisolatesof Yersinia enterocolitica biotype1A representtwophylogenetic lineages with difering pathogenicity-related properties. BMC microbiology.2012; 12:208.doi: 10.1186/1471-2180-12-208 PMID: 22985268; PubMed Central PMCID:PMC3512526.   
27．Batzila J, Hoper D,Antonenka U, Heesemann J,Rakin A. Complete genome sequence of Yersinia enterocolitica subsp.palearctica serogroup O:3.Journal of bacteriology.2011;193(8):2067.doi: 10. 1128/JB.01484-10 PMID:21296963; PubMed Central PMCID:PMC3133053.   
28. Jan Ursing DJB, Bercovier Herv4,O.Richard Fanning,Arold G.Steigerwalt, Jacqueline Brault,H.H. Mollaret.Yersinia frederikseni: ANew Speciesof Enterobacteriaceae Composed of Rhamnose-Positive Strains (Formerly Called Atypical Yersinia enterocolitica or Yersinia enterocolitica-Like).CURRENTMICROBIOLOGY.1980;4:213-7.   
29. Sprague LD,Neubauer H.Yersinia aleksiciae sp.nov.International journal of systematic and evolutionary microbiology.2005; 55(Pt2):831-5.doi: 10.1099/ijs.0.63220-0 PMlD: 15774670.   
30.Sprague LD,Scholz HC,Amann S,Busse HJ,Neubauer H.Yersinia similis sp.nov.International journal of systematic and evolutionary microbiology.2008;58(Pt 4):952-8.doi: 10.1099/ijs.0.65417-0 PMID: 18398201.