# Multiplex sequencing of bacterial artificial chromosomes for assembling complex plant genomes

Sebastian Beier1†，Axel Himmelbach1+,Thomas Schmutzer1,Marius Felder²，Stefan Taudien²，Klaus F.X.Mayer, Matthias Platzer²，Nils Stein1,Uwe Scholz' and Martin Mascher1'\*

1Leibniz Institute of Plant Geneticsand Crop Plant Research (IPK) Gatersleben,Stadt Seeland,Germany   
2Leibniz InstituteonAging—Fritz Lipmann Institute (FLI),JenaGermany   
3PlantGoedsteoB)olntuncheaeachnterfoioetaeabuherb

Received14 July2015;   
revised 11 November2015;   
accepted13November2015.   
\*Correspondence (Tel $+ + 4 9$ 39482 5243; fax $+ + 4 9$ 39482 5407;email mascher@ipkgatersleben.de)   
+These authors contributed equally to this work.

# Accession numbers

ERPO06953 (paired-end and mate-pair reads),individual accessionnumbers for scaffolded BAC assemblies presented in this study are listed in Table S1.

Keywords:hierarchical shotgun sequencing,bacterial artificial chromosome,sequence assembly, paired-end sequencing,mate-pair sequencing,physical map,Hordeum vulgare,barley.

# Summary

Hierarchical shotgun sequencing remains the method of choice for assembling high-quality reference sequences of complex plant genomes.The efficient exploitation of current highthroughput technologies and powerful computational facilities for large-insert clone sequencing necessitates the sequencing and assembly of a large number of clones in parallel.We developed a multiplexed pipeline for shotgun sequencing and assembling individual bacterial artificial chromosomes (BACs) using the Ilumina sequencing platform.We illustrate our approach by sequencing 668 barley BACs (Hordeum vulgare L.)in a single Ilumina HiSeq 20oo lane.Using a newly designed parallelized computational pipeline,we obtained sequence assemblies of individual BACs that consist,on average,of eight sequence scaffolds and represent ${ > } 9 8 \%$ of the genomic inserts.Our BAC assemblies are clearly superior to a whole-genome shotgun assembly regarding contiguity,completeness and the representation of the gene space.Our methods may be employed to rapidly obtain high-quality assemblies of a large number of clones to assemble map-based reference sequences of plant and animal species with complex genomes by sequencing along a minimum tiling path.

# Introduction

Hierarchical shotgun sequencing is a divide-and-conquer strategy for genome sequencing that has been successfully applied to produce high-quality reference genome sequences for many plant and animal species,among them Arabidopsis thaliana,rice and maize (Arabidopsis Genome Initiative，2ooo; International Rice Genome Sequencing Project,2005;Schnable et al.，2009).The main steps in hierarchical shotgun sequencing are (i) the construction of large-insert clone libraries,(ii) building a physical map and (ii) sequencing a minimum set of overlapping clones that cover the entire genome (a minimum tiling path).Currently， large-insert clone libraries consist in most cases of bacterial artificial chromosomes(BACs)(Shizuya et al.,1992). Physical maps are constructed bydetermining the size or direct sequencing of restriction fragments (Luo etal.,2003;van Oeveren et al.,2011). Until very recently,sequencing of minimum tiling paths has been performed using automatized Sanger sequencing (Groenen et al.，2012; Howe et al.,2013),although next-generation sequencing (NGS) platforms have been applied for large-scale clone sequencing (Choulet etal.,2014;Li et al.,2015; Zhang et al.,2012).

The main alternative to a map-based genome sequencing strategy is whole-genome shotgun sequencing (WGS),which is nowadays equivalent to deep sequencing of libraries of randomly fragmented genomic DNA with NGS instruments.The main advantage of a map-based sequencing strategy over a wholegenome approach is the reduction of genome complexity by biotechnological means prior to computational analysis,which greatly eases the algorithmic challenges associated with shortreadassembly.Sequence reads that originate from regions that are repeated hundreds or even thousands of times across the whole genome may be present at low copy number on a single BAC clone that is about 100-15O kb in size.Although the initial discussion about the relative benefits of whole-genome and hierarchical methods was led with great intensity (Green，1997; Weber and Myers，1997),it is now accepted that there is no black-and-white distinction between both approaches.

Complementary map-based and whole-genome sequencing approaches are currently employed to develop genomic resources for the Triticeae.This tribe comprises several closely related cereal grasses of great agronomic importance such as bread and durum wheat,barley and rye.In spite of their economic importance,highquality reference genome sequences are still missing for these species,mainly owing to the large size $( > 5 ~ \mathsf { G b } )$ and repeat-richand often polyploid structure of their genomes (Feuillet et al.,2012). In the last 5 years,progress in Triticeae genomics has followed three main paths: (i) shallow-coverage whole-genome sequencing data from flow-sorted chromosomes and transcriptome assemblies from NGS-data were integrated with dense genetic maps and collinearity information to construct so-called GenomeZippers (International Wheat Genome Sequencing Consortium，2014; Martis et al.,2013;Mayer et al.,2011); (ii) gene space assemblies were made from deep-coverage sequencing data on the wholegenome or whole-chromosome level and ultra-dense sequencebased genetic maps (Chapman et al.,2015; International Barley Genome Sequencing Consortium，2012； International Wheat Genome Sequencing Consortium,2014;Mascher et al.,2013a) (ii)in parallel to the first two efforts,collaborative large-scale physical mapping projects were initiated to construct BAC-based physical maps of the whole genome (barley) or individual chromosome arms(wheat)(Feuilletet al.,2012).The genomewide physical map of barley hasbeen completed (Ariyadasaetal.,2014),as has the physical map of several wheat chromosomes or chromosome arms (Breen et al.,2013;Lucas et al.,2013;Paux et al.，2008; Philippe etal.,2013;Poursarebanietal.,2014;Raats etal.,2013). Clone-by-clone sequencing of the entire minimum tiling path of wheat chromosome 3B has been performed using a hybrid approach combining data from the Roche 454 and Illumina NGS platforms (Choulet et al.,2014). Scaling BAC sequencing from a single 1-giga-base-pair chromosome to the 5 Gb barley genome or the remaining twenty wheat chromosomes—each several hundred mega-base pairs in size—will require the efficient utilization of the most advanced high-throughput sequencing technologies.

The aim of the present study was to establish a reliable method to sequence and assemble a large number of BAC clones in a standardized manner.We aimed atassembling BACs individually, because our prior experiences with sequencing barley BACs had indicated that joint assembly of reads from multiple clones results in the formation of chimeric sequence contigs (Taudien et al., 2011)．Thus，sequencing libraries should be constructed for individual BAC clones to track the origin of each sequence read and enable the assembly of reads from only single BACs. Consequently，pooling should be performed only after construction of barcoded libraries as opposed to sequencing pools of BACs without barcoding and then (i) assembling a larger genomic segment formed by overlapping BACs，or (ii) pooling nonoverlapping BACsand relying on the assemblyalgorithms to separate reads from unrelated regions.

Here,an integrated wet laboratoryand computational pipeline forsequencing,assembling and scaffolding individually barcoded BAC clones is presented.The pipeline was employed to sequence BACs from the minimum tiling path of the barley genome.

# Results

# Overview of the pipeline

In the wet laboratory part of our pipeline，multiplexed Illumina sequencing libraries are constructed from DNA of individual BAC clones (Figure 1a).After sequencing，resultant read data are automatically processed by the computational part of the pipeline (Figure 1b).Parallelization is employed to speed up the assembly process.The source code of our pipeline isavailable for download at http://dx.doi.org/10.5447/IPK/2015/6.In the following,we give adetailed walkthrough of our pipeline by following the sequencing and assembly of 668 BACs，whose inserts mostly originate from barley chromosome 4H.

# Shotgun sequencing of barley BACs and quality control

The set of 668 clones included 653 BACs that originated from barley.These BACs contain clones from two of the six independently constructed BAC libraries (Schulte et al.,2011) that have been previously designated as part of the minimum tiling path of the barley genome by restriction fragment-based fingerprinting and subsequent clone overlap analysis (Ariyadasa et al.，2014). Moreover，we chose fifteen‘reference BACs' to be used as internal controls for monitoring the performance ofthe sequencingand assembly process.These'reference BACs' included two human BAC clones (one with six;NCBl accession AF25283O;and the other with eight replicates;NCBl accession AF285443)as wellas one barley BAC clone that had already been shotgun-sequenced repeatedly (NCBl accession AY268139) (Steuernagel et al.，2009;Taudien et al.，2011).The manually curated assemblies of each of the'reference BACs'consisted of a single sequence contig and are available from NCBl (Table S2).

To sequence hundreds of BACs in one lane of the Ilumina HiSeq 2000 sequencing device,we adopted a versatile protocol for deep multiplexing of templates (Meyerand Kircher,201O) that we had previously used for genotyping by sequencing (Wendler et al., 2014).Briefly， the BAC DNA was fragmented and ligated to sequencing adaptors to serve as templates for index PCR,yielding 668 individually barcoded BAC libraries.Full details of the protocols used for library preparation and sequencing are described in the method section.All 668 individual librarieswere subsequently pooled in an equimolar manner and sequenced in a single lane of the Illumina HiSeq 2Ooo instrument using paired-end chemistry $( 2 \times 1 0 0$ cycles).After deconvolution of 418 million raw reads (42 Gb of sequence,Table 1),an average of 625 170 reads could be assigned to each individual BAC (minimum:3o8,maximum: 1 933 970,Figure 2a).Assuming an average genomic insert size of $1 4 5 ~ \mathsf { k b }$ ,the mean read depth was 431. Reads originating from BAC vector backbone, the $E .$ coli genome or reads of low-quality $12 \%$ of readswere below Q3O)were excluded from furtheranalysis.This process removed an average of $2 9 \%$ of raw reads per BAC.After filtering, the average read depth per BAC was 307 (Table 1).

# De novo assembly of paired-end data

We performed initial sequence assemblies separately for each BACwith CLC Assembly Cell software using all filtered pairedend reads.We note that the CLC assembler includesa scaffolding step attempting to link contigs using paired-end information.In the following，we will refer to the initial CLC scaffolds as contigs because no long-distance mate-pair data were used by CLC.The initial set of sequence contigs was further processed by removing remaining vector ends.Flexible coverage thresholds were also set up to remove contigs that were supported by only a very small number of reads and most likely originated from cross-contamination (from BACs in neighbouring wells during library preparation) (Figure 3)or due to errors during computational demultiplexing (wrongly assigned reads) (Kircher et al.,2012).We removed from the assembly of each BAC those contigs that had a coverage below half the mean coverage calculated across the set of contigs after removing $10 \%$ of contigs with the lowest coverage.This step purged the final assemblies of contigs that resulted from reads originating from other BACs and were present with low frequency in the deconvoluted reads.From an initial number of 16 313 contigs,8282 $( 5 0 . 8 \% )$ were removed.The discarded sequence set of contigs was mainly characterized by small sequence length.The average length of low coverage contigs was 1600 bp as compared to 10 208 bp for all contigs.Thus, the cumulative low coverage contig length was $1 3 1 2 8 { \mathrm { ~ } } \mathsf { k b }$ and accounted for $12 \%$ of the cumulative raw assembly length.

![](images/ee4a1712c5d63a289e628c288b9f280160caa4aa90e3d042fd03ee96be22eed8.jpg)  
Figure 1 Overview of the BAC sequencing and assembly pipeline.(a) Wet laboratory workflow. The estimated hands-on times of each step are shown in the right column of panel A and indicate the amount of time each step may require if two technicians process about 650-700 BACs (96-well plate format).The number in brackets in the last row refers to the operating time of the Illumina HiSeq 2ooo device when performing $2 \times 1 0 0$ cycles for paired-end sequencing.(b)Assembly pipeline. Input data (red boxes)are subjected to several processing steps (blue boxes), producing respective output (green boxes).The programs and scripts used in each step are indicated within the boxes.

Table1 Sequencing output for 668 BACs (paired-end and mate-pair sequencing)   

<html><body><table><tr><td>Library</td><td>Insert size</td><td>Raw read length</td><td>Processed length of single reads</td><td>No.of reads (raw)</td><td>No. of bases (raw)</td><td>No. of reads (processed)</td><td>No. of bases (processed)</td><td>Duplication rate*，%</td></tr><tr><td>Paired-end HiSeq</td><td>500 bp</td><td>2 × 100 bp</td><td>97 bp</td><td>417 613 642</td><td>41 970 171 021</td><td>297 035 172</td><td>28 897 856 622</td><td>0.43</td></tr><tr><td>Mate-pair MiSeq library1</td><td>5500 bp</td><td>2 × 250 bp</td><td>152 bp</td><td>5 270 554</td><td>855 965 539</td><td>2 829 040</td><td>429 345 869</td><td>0.69</td></tr><tr><td>Mate-pair MiSeq library 2</td><td>5700 bp</td><td>2 × 250 bp</td><td>147 bp</td><td>5 085 586</td><td>799 162 083</td><td>2 754 524</td><td>405 327 532</td><td>0.85</td></tr><tr><td>Mate-pair HiSeq library 1</td><td>5900 bp</td><td>2 × 100 bp</td><td>82 bp</td><td>34 897 774</td><td>3 472 328 513</td><td>17 171 286</td><td>1 409 064 253</td><td>3.61</td></tr><tr><td>Mate-pair HiSeq library 1</td><td>6200 bp</td><td>2 × 100 bp</td><td>82 bp</td><td>48 555 264</td><td>4 831 248 768</td><td>24 108 068</td><td>1966 403 705</td><td>5.59</td></tr></table></body></html>

\*oprtioopadiactirebistiadl

![](images/52de9190ce6e51e783ace1312c2268227745055d404c55e5f65056cdd22cc74f.jpg)  
Figure 2 Sequencing statistics.(a) Histograms of the number of raw (red) and processed (blue) readsassigned to each BAC.Processed reads were used as input for de novo assembly. (b) Distribution of the number of mate-pair reads assigned to each BAC.Mate pairs were mapped to the initial paired-end BAC assemblies. Only uniquely mapping pairs were assigned to BACs.

After postprocessing,BAC assemblies consisted of an average of twelve contigs (Table 2，Figure 4c)．The mean combined length of all contigs per BAC was $1 4 6 ~ \mathsf { k b } .$ .witha range between 827 bp and $3 4 5 ~ \mathsf { k b }$ (Figure 4b). On average,half of the sequence of each BAC was contained in contigs larger than $4 2 . 3 ~ \mathsf { k b }$ which is similar to our previous assembly results with Roche 454 read data (Figure 4a) (Steuernagel et al.,2oo9;Taudien et al.,2011).

# Scaffolding with Nextera mate-pair reads

Sequencing projects have to employ several library typesand computational tools to improve the contiguity and completeness of sequence assemblies.Scaffolding refers to improving an initial sequence assembly that is fragmented into multiple contigs using long-distance mate-pair reads to connect contig sequences and bridge gaps in the assembly.This process also involves ordering and orienting the set of contigs relative to each other and estimating the sizes of gaps between them.We sequenced 8 kb Nextera mate-pair libraries of BAC pools to improve the contiguity of the initial BAC assemblies made only from short-insert pairedend reads.For preparing mate-pair libraries, the 668 BAC clones were distributed into two nonoverlapping subpools each containing 334 BACs. Both libraries were sequenced twice (Table1): once in one-tenth of a single-lane flow cell of the Ilumina MiSeq, yielding $1 . 6 ~ \mathsf { G b }$ of sequence $( 2 \times 2 5 0$ nt reads),and once in oneeleventh of one lane of the Illumina HiSeq 2ooo,yielding $8 . 3 \ \mathsf { G b }$ of sequence $( 2 \times 1 0 0$ nt reads). Sequencing the libraries on two different platforms combined the advantages of higher read length of the MiSeq instrument with the larger sequencing output of the HiSeq 2000 device.

No individual barcoding of BACs was applied during library preparation.Instead,read pairs were deconvoluted by mapping them to the entire set of initial paired-end assemblies.Pairs mapping uniquely to a single BAC assembly were assigned to this BAC. $5 9 \%$ of MiSeq and $6 5 \%$ of HiSeq reads were assigned to their BAC of origin (Figure 2b).From this set of deconvoluted reads，only those pairs，whose two ends mapped to different contigs of the same BAC,were informative for scaffolding.Thus, onlya minor fraction of read pairs $( 6 . 4 \%$ of MiSeq and $0 . 5 \%$ of HiSeq mate pairs,Table S3)were used as input for the scaffolding process utilizing the SSPACE software (Boetzer et al.，2011). Before scaffolding,BAC assemblies were fragmented on average into12 contigs;after scaffolding,BAC assemblies consisted of an average of eight scaffolds.Scaffolding increased the assembly L50 on average by $5 3 . 1 ~ \mathsf { k b }$ .The mean size difference between the longest scaffold and the longest contig before scaffolding was $4 5 . 0 ~ \mathsf { k b }$ (Figure S1). Thirteen per cent of BACs (88 BACs) were represented by a single sequence scaffold.Half of the assembly length was contained in scaffolds larger than $9 5 ~ { \mathsf { k b } }$ (Table2,information for individual BAC assemblies isavailable in Table S4).As scaffolding only orders and orients existing contigs, but does not introduce new sequence information—stretches of unknown bases('Ns'）are introduced as gaps between linked contigs—the sequence information content remains constant. Manual inspection of sequence alignments supported the good concordance of our final assemblies of the reference BACs and their"gold-standard"assemblies in the public sequence archives (Figure 5).

# Evaluation of assembly quality

In addition to manual inspection of dot plots,we examined the quality of our BAC assemblies with computational tools specifically designed for this purpose.Inconsistencies between our BAC assemblies and their finished reference assemblies were catalogued with QUAST software (Gurevich et al.，2013).QUAST detects larger structural differences (misassemblies) and smaller insertions and deletions ${ < } 5 0$ bp (indels). According to the criteria defined by the Plantagora project (Barthelson et al.，2011), misassemblies were categorized into'global misassemblies'and 'local misassemblies' (see Experimental Procedures for full details). Onaverage， $9 8 . 5 \%$ of reference sequence was covered by contigs of the initial paired-end assemblies.The mean number of global and local misassemblies per BACs were 1 and 17, respectively (Table S5).Scaffolding introduced few errors: the average number of misassemblies increased to 2 (global) and 20 (local) (Table 3).By contrast, the contiguity was greatly improved: theaverage L5O of the fifteen reference BACs increased from $2 7 \ \mathsf { k b }$ to $9 9 ~ \mathsf { k b }$ .In summary，scaffolding with $8 ~ \mathsf { k b }$ mate-pair reads boosted the contiguity of the assembly without compromising its quality.

![](images/429c276d6075630afcfeef2e2ace34e6a210fc277fcc16029b26ca67120671ec.jpg)  
Figure3 Motivation for implementing a coverage threshold to exclude contigswith low read depth.The average read depth is shown for all contigs of the initial paired-end assembly of BAC clone HVVMRXALLeA0276J07.Contigs with coverage below 153 were removed from the assemblies.This threshold was calculated by the algorithm as $50 \%$ of the average contig coverage. BLAST searches revealed that the excluded contigs (c12,c13 and c15) originated from the bacteriophage phiX genome,which is included as an internal quality control for sequencing (recommended by Ilumina as a low concentration spike in). Note that contigs c9,c10,c11 and c14 had been removed because they were entirely composed of sequence originating from the vector or the E. coli genome.

Table 2 Assembly summary statistics\*   

<html><body><table><tr><td></td><td>Before scaffolding</td><td>After scaffolding</td></tr><tr><td>Number of contigs/scaffolds per BAC</td><td>12.2</td><td>7.8</td></tr><tr><td>L50+</td><td>41 346 bp</td><td>94 887 bp</td></tr><tr><td>Size of largest contig/scaffold</td><td>55 072 bp</td><td>100 261 bp</td></tr><tr><td>Assembly length</td><td>146 436 bp</td><td>147 830 bp</td></tr><tr><td>Assembly length in contigs/scaffolds ≥1 kb</td><td>145 135 bp</td><td>146 555 bp</td></tr><tr><td>Assembly length in contigs/scaffolds ≥10 kb</td><td>116 391 bp</td><td>133 281 bp</td></tr></table></body></html>

\*Assembly statistics were calculated for each BAC.Then,the arithmetic mean across all 668 values was computed. +The L50 is maximum value L such that $50 \%$ of the assembly length is contained in contigs whose size is greater than or equal to L.

# Representation of genes

We assessed whether the higher contiguity of our BAC assemblies (ascompared to published draft WGS assembly) improved the representation of genes for which no or only partial sequence informationwasavailable in the WGS assembly.In total,26159 high-confidence gene models have been predicted for barley (International Barley Genome Sequencing Consortium,2012).We searched barley full-length cDNAs (Matsumoto etal.，2011)for coding sequences that were (almost) completely contained in a single sequence scaffold of the BAC assemblies,but spread across multiple WGS contigs or partially absent from the WGS assembly. We found twelve such examples,one of which is shown in Figure 6.The cumulative length of all unique barley BACs assembled in the present study is 95 Mb.Extrapolating from the twelve full-length cDNAs that we found to be represented better in the BAC assemblies,we can estimate that a minimum of five hundred gene models may be fragmented or incomplete in the WGS assembly.This is only a very rough estimate that does not take into account differences in gene density along the genome.Genotypic differences can be excluded as the explanation for‘missing' genes as both the BAC libraries and the WGS assembly were derived from cultivar Morex.To conclude，the construction of a BAC-based reference sequence can be expected to improve the sequence representation of the barley gene space.

# Repetitive sequences

We wished to assess whether the repeat content of single BAC is correlated with final assembly contiguity.Towards this aims,we applied the Kmasker software to identify stretches of highly abundant 21-mers in the assemblies using the reference index for $k$ -mers occurring in the barley genome (Schmutzer et al.,2014). Onaverage， $8 8 . 2 \%$ of the BAC assembly sequences were classified as being composed of 21-mers that are highly abundant in the barley genome (Figure 7a). Screening the BAC sequence scaffolds for the presence of known repeat families from the TriticeaeRepeat Sequence Database (Wicker et al.， 2002) assigned onaverage $6 2 . 5 \%$ of the total sequence of the BAC assemblies to annotated repeats (Figure 7b).There wasno clear correlation between the repeat content of BACs with the contiguity of their sequence assemblies as measured by the L50 value，indicating that overall repeat content is not a good predictor of final BAC assembly quality (Figure 7a and b).

Next,we tested whether the representation of repetitive sequences is better in the BAC assemblies than in the WGS assembly of cv.Morex.Aligning the WGS contigs to repeat structures identified by either the Kmasker approach，or to annotated repeats found in the BAC assemblies showed that $2 4 . 8 \%$ of $k$ -mer masked sequencesand $7 . 9 \%$ of annotated repeat sequences were covered by WGS contigs with a minimal alignment identityof $9 9 \%$ .Ata lowered identity threshold of $80 \%$ ，however, the majority of repeats had matches to WGS assembly (Figure 7cand d),indicating that highly similar copies of repetitive elements were often collapsed into‘consensus contigs inthe WGS assembly,but were better resolved in the single BAC assemblies.

# Discussion

We have established a high-throughput pipeline for sequencing and assembling a large number of BAC clones.The two main improvements of our pipeline over previous efforts for BAC sequencing are (i) deep multiplexing of samples on the Illumina HiSeq 2oo0 platform (ii)and use of updated bioinformatics protocols to accommodate the new structure and vastly increased amount of raw data. Previous BAC sequencing protocols were set up for the Roche 454 sequencing platform (Choulet et al.,2010; Steuernagel et al., 2009; Taudien et al.,2011) or for sequencing smaller numbers of clones on the Ilumina platform using standard multiplexing procedures (Sathuvalli and Mehlenbacher, 2013).

![](images/1b1d300302c2ff03e49e94f0b35f562b93989386f5776838c7c9633f2c412981.jpg)  
Figure4eblatiseaelssoistrbsfOa)sblyendmbtiso (blue)(c)fliabfoieoidi mate-paireadesoinbeL5Oisdefidatengthoftesmalstcotg(safol)amongallzerakedfromlogesttet contigs (or scaffolds) that together make up $50 \%$ or more of the length ofa given assembly.The assembly size is the sumof the lengths of all constituent contigs or scaffolds.

The aim of BAC sequencing can be stated easily,to represent the entire nucleotide sequence of the genomic insert by a single sequence contig.In the present study, this task was complicated by great abundance and high copy of transposable elements in the barley genome，which prevented assembling BACs into a single contig in most cases.Although the assemblies represented onaverage ${ > } 9 8 \%$ of the inserts,they remain fragmented into multiple contigs with an L50 of $9 5 ~ { \mathsf { k b } }$ .This is not unexpected. The most important precedent for assembling a large and repeatrich genome is the human genome project. When the first working draft of the human genome was published, $\sim 3 0 \ 0 0 0$ large-insert clones were sequenced with Sanger sequencing technology and were represented by 40o ooo contigs with an L50 of $8 3 ~ \mathsf { k b }$ (Kentand Haussler,2o01).The finalassembly of the BACs into single contigs required labour-intensive manual curation (International Human Genome Sequencing Consortium, 2004).We anticipate that further improvements to the contiguity and genome representation of BACs comprising the minimum tiling path of the barley genome will proceed along the following lines: (i) generation and use of additional genomewide data sets to improve the contiguity of clone assemblies; (ii) use of long-read sequencingplatforms to close remaining gaps (English et al., 2012).Additional genomewide data sets may include deeply sequenced mate-pair libraries of various insert size，chromatin conformation capture sequencing (HiC) data (Burton et al.,2013; Kaplan and Dekker，2013）or in vitro-derived long-range connectivity information (Adeyetal.，2014).The data setsrequired for these steps cannot be collected at the level of single clones or larger BAC pools because the costs associated with doing so Would be prohibitively high,or because working with BACs,as in the case of chromatin conformation capture sequencing strate gies，may be meaningless.To utilize these genome-scale NGSbased techniques for scaffolding and gap-closing,a genomewide, nonredundant sequence assembly isnecessary.Therefore，a pipeline to automatically detect and visualize overlaps between clones was set up (Colmsee et al.,2015).This sequence overlap information will be used to construct nonredundant sequences on the level of finger-printed contigs，where adjacent clones overlap byapproximately $3 0 ~ \mathsf { k b }$ . It is important to note that $90 \%$ of the assembled sequence of our BACs is contained in contigs $\geq 1 0 \ k \ b ,$ which allows to retrieve long $( > 1 0 ~ \mathsf { k b } )$ ，high-identity $( > 9 9 . 5 \% )$ overlaps between adjacent BACs.These overlaps can be well distinguished from regions of high sequence identity shared between copies of transposable elements on unrelated BACs.Thus, the immediate next goal of the International Barley Genome Sequencing Consortium is to finish the sequencing of all 66 772 clones of the minimum tiling path of the barley genome (Ariyadasa et al.,2o14) and to construct an initial nonredundant clone-based sequence assembly of the entire barley genome.

Newsequencingtechnologiesinvolvingsingle-molecule sequencing (Eid et al.，2oo9) or nanopore devices (Goodwin et al.，2015) hold the potential of producing BAC assemblies of improved contiguity. Current single-molecule sequencingas marketed by Pacific BioSciences Inc. (PacBio) has been shown to deliver high-qualityassemblies of entire bacterial genomes (Koren and Phillippy,2015). Thus,PacBio sequencing can be expected to yield finished single-contig assemblies of BACs using only automated procedures.PacBio continuously improves the performance ofits sequencing devices and has also introduced protocols for multiplex sequencing.Based on currently published results, however,it is not clear to us whether the cost and throughput of the PacBio sequencing platform willbe sufficient for sequencing a large number of BAC clones.Moreover，sequencing entire minimum tiling paths may require the development of new protocols for deep multiplexing protocols for the PacBio platform. Given the progress in the development of algorithms that can efficiently employ long,albeit error-prone reads for the assembly of eukaryotic genomes (Kim et al.，2014; Koren and Phillipy, 2015)，it can be envisaged that a whole-genome shotgun approach combined with single-molecule sequencing can be used to assemble huge plant genomes in the future.

![](images/cc8ce9760aa040356bdd787f14396d74a1f5d48546e904429369233abd7c76c7.jpg)  
Figure 5 Alignment of human control BAC assemblies against their finished reference assembly.Threeassemblies of one human reference BAC clone from independent sequencing libraries were aligned to the manually curated reference assembly obtained from Sanger sequencing data (available as NCBl accession AF25283O).Red diagonals indicate sequence identity of the alignment coordinates between each of our three de novo and the public reference assembly (a-c). Red dots correspond to small gaps in the alignment that are caused by stretches of unknown bases ('Ns').All assemblies represent between $9 9 \%$ and $9 9 . 8 \%$ of the entire length of the reference assembly. Scaffold sc2 in (a)and contig c1 (b) contain a misassembled region,which may have prevented correct scaffolding with mate-pair reads.Among the three de novo assemblies only the one shown in (c) consisted of a single scaffold.

Current standard protocols allow an academic laboratory to manually generate thousands of barcoded libraries from individual BAC DNAs for sequencing without major investments,except the Covaris LE22O instrument required for the fragmentation of DNA.To increase the throughput of library construction further and to reduce the manual labour time, the use of precision liquid handling robots might be beneficial.The standard protocol can beadapted readily to automation.Library preparation isalready parallelized right from the start in the 96-well format and the enzymatic reactions mostly involve pipetting volumes of at least $2 0 ~ \mu \iota .$ .which are suitable for automated processing. Only the setup of the index PCR might require some minor alterations to increase the volumes of index primerand template DNA.Reaction clean-up steps are based on paramagnetic beads (SPRl method) and are amenable for automation as well.If the set-up of a fully automated robot system is impractical,manual easy-to-use 96- well pipetting systems (PlateMaster; Gilson International) are an interesting additional option to accelerate liquid handling.An alternative to the final manual gel-based library purification is the use of the automated BluePippin device (sage science) or doubleSPRI size selection (Lennon et al.,201O; Rodrigue et al.,2010). All these alternatives might be worthwhile additions to further reduce costs or increase throughput of single BAC sequencing.

Development and application of the BAC sequencing pipeline described in this manuscript were motivated by our involvement in the barley genome sequencing project (Schulte et al.,2009). Obviously，the construction of BAC libraries，physical mapping and clone sequencing are not limited to plant genomics. However,map-based genome sequencing is of particular interest to plant research,because of the complex structure of many plant genomes (large genomes ${ > } 5$ Gband abound in repetitive elements)，which is not amenable for whole-genome shotgun approaches.For instance,WGS assemblies of barley(5 Gb),white spruce $( 2 0 ~ \mathsf { G b } )$ and hexaploid bread wheat (17 Gb) achieved only $4 0 \mathrm { - } 7 0 \%$ genome representation.The missing sequence presumably originates predominantly from families of repetitive elements whose individual copies are highly similar to each other and are thus prone to collapse into a single contig in the assembly. Obtaining assemblies that correctly represent repetitive regions is an important goal of reference genome projects in repeat-rich plant genomes.Transposable elements act as driving forces of genome evolution (Fedoroff,2012),and furthermore，they are involved in (i) the diversification of centromeres (Gao et al., 2015),(i) the rapid adaptation to changing environments (Stapley et al.,2o15) and (ii) the expression of genes near their insertion sites (Makarevitch et al.,2015).In addition the benefits of a mapbased reference sequence are not limited to a better resolution of repeat regions,but also improve the representation of genic regions. In the present study,we have shown that in contrast to the barley WGS assembly the sequence of full-length cDNAs is faithfully represented in BAC assemblies.The apparent absence of bona fide genes from the WGS assembly of barley reinforces the conclusion of Denton et al. (2014) that inferences on the number of genes and the sizes of gene families from draft genome assemblies should be treated with caution.

Currently the pipeline described in the present paper is used to sequence and assemble the complete minimum tiling path of the barley genome.Our methods can be readily extended to wheat genome sequencing as all relevant resources—BAC libraries, physical maps and supporting whole-genome data sets—are in place or are being generated.Moreover,we anticipate that our pipeline will be valuable for other sequencing project in species with large genomes.

Table 3 Assembly quality statistics for sequenced reference BACs (scaffolded assemblies)   

<html><body><table><tr><td>BAC</td><td>L50</td><td>No.of misassemblies</td><td>No.of local misassemblies</td><td>Indels</td><td>Genome fraction*</td><td>No. of scaffolds</td></tr><tr><td>NCBI_AF252830_25</td><td>97 756</td><td>2</td><td>19</td><td>10</td><td>99.8</td><td>4</td></tr><tr><td>NCBI_AF252830_26</td><td>99400</td><td>1</td><td>28</td><td>2</td><td>99.0</td><td>5</td></tr><tr><td>NCBI_AF252830_27</td><td>179 959</td><td>0</td><td>24</td><td>9</td><td>99.3</td><td>1</td></tr><tr><td>NCBI_AF252830_28</td><td>76 048</td><td>1</td><td>26</td><td>10</td><td>98.8</td><td>3</td></tr><tr><td>NCBI_AF252830_29</td><td>97 573</td><td>1</td><td>25</td><td>12</td><td>99.3</td><td>6</td></tr><tr><td>NCBI_AF252830_30</td><td>69 370</td><td>1</td><td>18</td><td>15</td><td>99.0</td><td>4</td></tr><tr><td>NCBI_AF285443_23</td><td>39 507</td><td>1</td><td>23</td><td>6</td><td>99.2</td><td>5</td></tr><tr><td>NCBI_AF285443_24</td><td>97 772</td><td>2</td><td>11</td><td>8</td><td>97.2</td><td>4</td></tr><tr><td>NCBI_AF285443_25</td><td>146 380</td><td>3</td><td>22</td><td>11</td><td>99.2</td><td>2</td></tr><tr><td>NCBI_AF285443_26</td><td>47 846</td><td>2</td><td>21</td><td>7</td><td>98.7</td><td>4</td></tr><tr><td>NCBI_AF285443_27</td><td>147 003</td><td>1</td><td>15</td><td>33</td><td>96.2</td><td>3</td></tr><tr><td>NCBI_AF285443_28</td><td>43 512</td><td>4</td><td>19</td><td>18</td><td>98.0</td><td>6</td></tr><tr><td>NCBI_AF285443_29</td><td>150 689</td><td>0</td><td>13</td><td>9</td><td>99.6</td><td>1</td></tr><tr><td>NCBI_AF285443_30</td><td>85 073</td><td>3</td><td>22</td><td>11</td><td>99.5</td><td>3</td></tr><tr><td>HVVMRXALLhA0184G09</td><td>110 185</td><td>1</td><td>9</td><td>5</td><td>97.1</td><td>3</td></tr><tr><td>Mean</td><td>99205</td><td>2</td><td>20</td><td>11</td><td>98.7</td><td>4</td></tr></table></body></html>

\*Fraction of the genomic insert that is represented in the assembly.

![](images/a1d0a3e66b5d09567cfbde8aa00bd2a385679a893326bf164af3d5a5f9c2d80a.jpg)  
Figure6aplfftipleecepreatitteblrltee CDNAcloneNASHv720(NCBlacessinAK3793)canbealgnedtocafoldscofBACHMRXALLeA22A1(alignmentcoodinates: 107539pto3)edregioreepresetedgreectages.trostatadslidtalre greelines.Noieaistetireeeofrtsofrespredacofivfettise indicatedontheleftFurofthesecontigsarecompletelyontaindinHMRXLLeA226A12altoughoneofthem(morexonti04895)is misassembledndjinstistantsequecesdicatedbydasedline)Anotherontigmorecoti599866)canbeigedlyartlyto HVMRXALLeecesatitdaedythatchdeaarebedecesralogssiblyed copies of the gene may have complicated assembly from whole-genome sequencing data.

# Experimental procedures

# Preparation of shotgun sequencing libraries

The isolation of BAC DNA was based on the alkaline lysis method (Birnboim and Doly, 1979). Shotgun sequencing of up to 668 individually barcoded BACs (Ilumina multiplex library) was performed essentially as described previously (Meyer and Kircher,201O). Briefly, BAC DNA was fragmented using focused acoustic energy (Covaris E22O) and cleaned up by precipitation using SPRl-bead suspension (Himmelbach et al.，2014).The DNA waseluted and blunt-end repaired as described (Himmelbach et al.2014).The fragments were provided with two adapters (P5，P7) and SPRl purified (Himmelbach et al.，2014).The adapter ligated DNA was filled in using Bst DNA polymerase (large fragment)，SPRl purified and processed for indexing PCR (Himmelbachet al.， 2014). 672different index primers (Table S6) were created according to the previously published guidelines (Meyer and Kircher，201O).The PCR products were pooled in equimolar manner and size-selected (420-520 bp) using agarose gel electrophoresis (Himmelbach et al.,2014).The structure of the adaptor-ligated DNA fragments is provided as Figure S2.Librarieswerequantified usingreal-time PCR (Mascher et al.，2013b)．Cluster formation (cBot)， $2 \times 1 0 0$ cycles paired-end sequencing and index read (8 cycles)were performed according to the manufacturer (Ilumina,San Diego, California，USA） instructions.Sequences were extracted by CASAVA version 1.8.2 (GenomeAnalysis-Pipeline).

![](images/24cb6876ec8a8f5cbb4e17638389fdd8acece1e022ec802b1a3741f70ca2e930.jpg)  
Figure7 Analysis of repetitive sequence content. The panels a and b show the relation between the amount of repetitive sequences and the L50 of the barley BAC assemblies. In both panels, the red line shows the linear regression curve for these values determined with R command $' | \boldsymbol { \mathrm { m } } ^ { \prime }$ .The $\boldsymbol { \mathsf { r } } ^ { 2 }$ values are indicated in the plots.The box plots in panels c and d show the distribution of fraction of repetitive sequences in the BAC assemblies matchingto WGS for different alignment identity thresholds $( 8 0 - 9 9 \% )$ 1 Repetitive sequences were identified using a $k$ -mer-based approach in panels a and cand a search against known repeat families in panels b and d.

# Preparation of Nextera mate-pair sequencing libraries

Nextera libraries were prepared following essentially the instructions of the manufacturer (llumina,FC-132-10o1). Libraries were quantified and sequenced as described for shotgun sequencing libraries.Up to 12 libraries (each constructed from 384 BACs) were sequenced in one HiSeq 2ooo lane $( 2 \times 1 0 0$ cycles). In addition，up to six libraries were sequenced using the llumina MiSeq $( 2 \times 2 5 0$ cycles).FASTQ sequence files were extracted with CASAVA version 1.8.2 (HiSeq2000) or with the MiSeqReporter Software (llumina,San Diego,California,USA).Full details of all wet laboratory procedures are described in Data S1.

# Initial assembly of paired-end reads

Sequence assembly of data obtained for large BAC pools was performed with a customized pipeline for preprocessing,assembly and postprocessing.The source code of all scripts is available for download using the digital object identifier (DOl) http://dx.doi.org/ 10.5447/IPK/2015/6.This DOl was created with e!DAL (Arend et al.，2014). The pipeline consists of two major modules,one written in Java 1.6 that handles thread initialization and execution while the second module written in Perl 5.8 handles the de novo assembly of individual BAC clones.Users can specify the number of parallel assemblies to be run by the pipeline.Internally,standard Javafunctions(ThreadPoolExecuter，BlockingQueue）ascertain that the number of active threads does not exceed the number of available CPU cores. Each individual BAC was allocated a single execution thread,in which assemblywith CLC and BLAST searches during contamination removal steps were run consecutively.This parallelization layout is based on the ThreadPoolExecuter implementation in Java and works in conjunction with a queue of all tasks (i.e. the number of all FASTQ files in the assembly input folder).

# Pre-assembly processing and quality control

To address the problem of previously reported poor read quality at the $3 ^ { \prime }$ ends of Illumina sequenced reads (Minoche et al.,2011) clc_quality_trim (version 4.0.6 beta) was used on every BAC read set individually with default parameters and only keeping pairs. Genomic E.coli reads were removed through BLASTN filtering with an e-value cut-off of $1 \times 1 0 ^ { - 5 }$ .Vector reads were partially kept for initial assembly if originatingupto $2 1 \times 6$ upor downstream of the cloning site [adapted from (Steuernagel et al.，2oo9)], the remainder of vector reads was discarded (BLASTN with an e-value cut-off of $1 \ \times \ 1 0 ^ { - 5 } ,$ ）

# Postprocessing of assemblies

Assemblies were performed by CLC Assembly Cell (version 4.0.6 beta;CLCbio,Aarhus,Midtjylland, Denmark) using the command clc_novo_assemble. Single BAC assemblies were performed using 10 CPUs (parameter -cpus 1O). Insert size parameters were set to '-p fb ss 200 $1 0 0 0 ^ { \prime }$ according to the span sizes of paired-end libraries.The resultant contigs were then screened for the remaining vector sequences using cross-match.Vector sequences almost exclusively assembled to the ends of a contig making it possible to mark these as‘BAC end contigs'.After clipping vector sequences,only contigs with a length of at least 5oo bp were kept.To account for possible contamination either between BAC clones or other sources,reads were mapped against the clipped version of the assembly by running megaBLAST (e-value cut-off of $1 \times 1 0 ^ { - 5 } )$ .The best hit of each read was regarded as position assigned by the assembler during contig production.Furthermore，the total length of all reads from a single contigwas summed and then divided by the contig length to deduce contig read coverage.Furthermore,the average contig coverage of all contigs from an individual BAC clone was calculated and a filter threshold of $90 \%$ of half of the average coverage was set [to account for uneven coverage in the sequenced region (Dohm et al.，2oo8)].Contigs with a read coverage lower than this threshold were marked as possible contamination or assembly artefactandremovedfromtheassembly.

Finally，assemblies were screened against common contaminants(includingsequencingadaptors， commonphage sequences， common bacterial sequences，vector sequences; DOI:http://dx.doi.org/10.5447/IPK/2015/6) withmegaBLAST (Zhang et al.，2ooo)using a minimum sequence identity of $90 \%$ and a minimum alignment length of 1oo bp.Detected contaminants were removed from the assembly by either clipping contig ends or completely removing the contaminated sequence if the contig length after clipping was below 500 bp.

# Preprocessing and deconvolution of mate-pair reads

Mate-pair reads were processed using a custom PERL script for identifying and discarding duplicate read pairs.The read pairs were screened for the Nextera junction adapter and clipped according to the outline in the Ilumina guideline (http:// www.illumina.com/documents/products/technotes/technote_next era_matepair_data_processing.pdf) using BLASTN (Altschul et al., 1990)with an e-value cut-off of $1 \times 1 0 ^ { - 5 }$ .Reads were clipped according to the orientation of the read and location of the adapter in the read following the procedures described by Leggett et al. (2014).After clipping,only reads with a length ${ > } 1 5$ bp were kept. Unpaired reads were discarded.

Quality trimming was then performed to remove poor quality bases with clc_quality_trim (version 4.0.6 beta)with default parameters.Reads containing vector or E.coli sequences were removed using BLASTN with an e-value cut-off of $1 \times 1 0 ^ { - 5 }$

The draft assemblies of all 658 BACs were concatenated into a single file and used as reference to deconvolute the mate-pair readsin the following fashion.BWA-MEM version O.7.4(Li and Durbin,20o9)was used to map the preprocessed mate-pair reads against the paired-end draft assemblies.Estimated span sizes of read pairs were obtained by filtering reads for mapping quality by SAMtools version O.1.18(Li et al.，2009)and then calculated mean insert sizes bya custom script filtering the mapped data set for high quality (using the command ‘samtools view -q 40'). Only read pairs mapped to the same contig were considered for insert size estimation.A modified custom PERL script based on the SSPACE program sam2tab.pl was then used to assign read pairs to individual BAC assemblies.

# Iterative scaffolding

MiSeq and HiSeq 20oO libraries from the same pool were regarded as two different libraries in scaffolding to prevent potential issues caused by large differences in sequence coverage between the two libraries.The MiSeq library was processed first. Scaffolding was performed with SSPACE_Standard_v3.0 (Boetzer et al.，2011).The value of the SSPACE parameter $- k$ (minimal number of links between contigs) was set to ten for MiSeq and twenty for HiSeq mate pairs.The standard deviation of the insert size was set to $50 \%$ of the mean.Each BAC was scaffolded independently.After scaffolding with the MiSeq data,assemblies were concatenated and used as reference for deconvoluting the HiSeq mate-pair read data.Scaffoldingwith the HiSeq mate-pair readswas performed identical asdescribed for the MiSeq data.

# Sequence comparison of reference BACs

Theprograms 'nucmer'and'mummerplot' from the MUMmer package version 3.23 (Kurtz et al.,2004) were used to align the assemblies (prior and after scaffolding) of the reference BACs to theirsequences in the publicarchives (Tables S5 and3).Assembly quality statistics were obtained with QUAST version 2.3(Gurevich et al.，2013)using default parameters).Misassemblies were classified according to Plantagora(Barthelson et al.,2011).Briefly, a'misassembly'ischaracterized asa position in the contigswhere either: (i) the left and the right flanking sequences align over $1 \ k \mathsf { b }$ away from each other,(ii the flanking sequences overlap for more than 1 kb,or(ii) the flanking sequences align to different strands.A 'local misassembly' is a position that has to fulfil the following three conditions:(i) there are two or more distinct alignments that cover the position,(ii) its flanking sequencesalign less than1 kb apart and (iii) both flanking sequences align to the same strand.Misassemblies that do not qualify as local misassemblies are considered as 'global misassemblies'.For visualization,dot plots were produced using the MUMmer program suite (Kurtz et al.,2004).

# Analysisof repeatcontent

$K \cdot$ -mer frequencies were analysed using the tool Kmasker (Schmutzer et al.,2014) and exported into BED format.Screening of BAC assemblies against TREP (Version 1O) (Wicker et al.，2002) was performed with megaBLAST (Zhang et al.，20oO) and the BLAST output also converted into BED format. The commands 'merge'and ‘intersect' from the BEDtools suite (Quinlan and Hall, 2010)were used to extract and combine features and to compare them to aligned sequences of the barley WGS contigs (International Barley Genome Sequencing et al.,2012).

# Acknowledgements

This work Was supported by the German Ministry of Education and Research projects BARLEX (grant no.FKZ031400O) and TRITEX (grant no.FKZ0315954) to NS，US,KFXM and MP，and the German Leibniz Association Pakt fur Forschung und Innovation (to NS,US).We gratefully acknowledge Thomas Munch and Steffen Flemming for data management， Burkhard Steuernagel for informatics support in the initial phase of the project and Anne Kusserow，Ulrike Beier，Ines Walde，Manuela Knauft，Jelena Perovic and Sandra Driesslein for excellent technical assistance. We thank Doreen Stengel and Daniel Arend for data submission as well as Lothar Altschmied and Ronny Brandt for fruitful discussions.The authors declare that they have no conflict of interest.

# References

Adey，A.，Kitzman,J.O.，Burton,J.N.，Daza,R.，Kumar,A.,Christiansen,L., Ronaghi,M.et al.(2o14) In vitro,long-range sequence information for de

novo genome assembly via transposase contiguity. Genome Res.24,2041- 2049.   
Altschul,S.F.,Gish,W.,Miller,W.,Myers,E.W.and Lipman,D.J.(1990) Basic local alignment search tool.J.Mol. Biol.215,403-410.   
Arabidopsis Genome Initiative. (2ooo) Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.Nature,408,796-815.   
Arend,D.,Lange,M.，Chen,J.，Colmsee，C.，Flemming,S.，Hecht,D.and Scholz,U. (2014) e!DAL-a framework to store,share and publish research data.BMC Bioinformatics,15,214.   
Ariyadasa，R.，Mascher，M.，Nussbaumer，T.，Schulte，D.，Frenkel，Z., Poursarebani,N.,Zhou,R.et al. (2014)A sequence-ready physical map of barley anchored genetically by two million single-nucleotide polymorphisms. Plant Physiol.164,412-423.   
Barthelson,R., McFarlin,A.J.,Rounsley,S.D.and Young,S. (2011) Plantagora: modeling whole genome sequencing and assembly of plant genomes. PLoS ONE,6, e28436.   
Birnboim，H.and Doly，J.(1979)A rapid alkaline extraction procedure for screening recombinant plasmid DNA. Nucleic Acids Res.7,1513-1523.   
Boetzer，M.，Henkel,C.V.,Jansen,H.J.，Butler,D.and Pirovano,W.(2011) Scaffolding pre-assembled contigs using SSPACE.Bioinformatics,27,578- 579   
Breen，J.，Wicker，T.，Shatalina，M.，Frenkel，Z.，Bertin，I.，Philippe，R., Spielmeyer,W.et al.(2013) A physical map of the short arm of wheat chromosOme 1A.PLoS ONE,8,e80272.   
Burton,J.N.,Adey,A.,Patwardhan,R.P., Qiu,R.,Kitzman,J.O.and Shendure,J (2013) Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions. Nat. Biotechnol.31,1119-1125.   
Chapman,J.A., Mascher,M.,Buluc,A.N.,Barry,K.,Georganas,E.,Session,A., Strnadova，V.et al. (2015)A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.Genome Biol. 16, 26.   
Choulet,F.,Wicker,T.,Rustenholz,C., Paux,E., Salse,J., Leroy,P.,Schlub,S. et al. (2o1O) Megabase level sequencing reveals contrasted organization and evolution patterns of the wheat gene and transposable element spaces.Plant Cell, 22, 1686-1701.   
Choulet,F.,Alberti,A.,Theil,S.,Glover,N.,Barbe,V.,Daron,J.，Pingault,L. et al.(2014) Structural and functional partitioning of bread wheat chromosome 3B.Science,345,1249721.   
Colmsee,C.,Beier,S.,Himmelbach,A.,Schmutzer,T.,Stein,N.,Scholz,U.and Mascher,M. (2015) BARLEX- the barley draft genome explorer.Mol.Plant, 8, 964-966.   
Denton,J.F.,Lugo-Martinez,J.,Tucker,A.E.,Schrider,D.R.,Warren,W.C.and Hahn,M.W. (2o14) Extensive error in the number of genes inferred from draft genome assemblies.PLoS Comput. Biol.10,e1003998.   
Dohm,J.C.,Lottaz,C., Borodina,T.and Himmelbauer,H.(20o8) Substantial biases in ultra-short read data sets from high-throughput DNA sequencing. Nucleic Acids Res.36,e105.   
Eid,J.,ehr,A.Gray.uongK,yleJ.toGelusoPetal.(09) Real-time DNA sequencing from single polymerase molecules.Science,323, 133-138.   
English,A.C., Richards,S., Han,Y.,Wang,M.,Vee,V.,Qu,J., Qin,X.et al. (2012)Mind the gap: upgrading genomes with Pacific Biosciences RS longread sequencing technology. PLoS ONE,7,e47768.   
Fedoroff, N.V.(2012) Presidential address.Transposable elements,epigenetics, and genome evolution.Science,338,758-767.   
Feuillt,C.,Stein,N.,ossini,L,Praud,S.,Mayer,K.,Schulman,A.,Eversole,K. et al.(2o12） Integrating cereal genomics to support innovation in the Triticeae.Funct. Integr.Genomics,12,573-583.   
Gao,D.,Jiang,N.,Wing,R.A.,Jiang,J.and Jackson,S.A.(2015) Transposons play an important role in the evolution and diversification of centromeres among closely related species.Front Plant Sci.6,216.   
Goodwin,S.,Gurtowski,J.,Ethe-Sayers,S.,Deshpande,P.,Schatz,M.C.and McCombie，W.R.(2015) Oxford Nanopore sequencing，hybrid error correction,and de novo assembly of a eukaryotic genome.Genome Res. 25, 1750-1756.   
Green,P. (1997) Against a whole-genome shotgun. Genome Res.7,410-417.   
Groenen，M.A.，Archibald，A.L.，Uenishi，H.，Tuggle，C.K.，Takeuchi，Y., Rothschild,M.F.， Rogel-Gailard,C.et al. (2012)Analyses of pig genomes 398.   
Gurevich,A.，Saveliev,V.,Vyahhi,N.and Tesler，G.(2013) QUAST:quality assessment tool for genome assemblies.Bioinformatics,29,1072-1075.   
Himmelbach，A.，Knauft，M.and Stein，N.(2014) Plant sequence capture optimised for Illumina sequencing. Bio Protoc.4,e1166.   
Howe,K., Clark,M.D.,Torroja,C.F.,Torrance,J.,Berthelot,C.,Muffato,M., Collins,J.E.et al. (2013) The zebrafish reference genome sequence and its relationship to the human genome. Nature,496,498-503.   
International Barley Genome Sequencing C,Mayer, K.F.,Waugh,R.,Brown, J.W.，Schulman，A.，Langridge,P.，Platzer，M.Fincher，G.B.，Muehlbauer， G.J.,Sato,K., Close,T.J.,Wise,R.P.and Stein,N.(2012)A physical,genetic and functional sequence assembly of the barley genome.Nature,491,711- 716.   
International Barley Genome Sequencing Consortium T. (2012）A physical, geneticand functional sequence asemblyof thebarley genome.Nature, 491, 711-716.   
International Human Genome Sequencing Consortium. (2o04) Finishing the euchromatic sequence of the human genome. Nature,431,931-945.   
International Rice Genome Sequencing Project.(2Oo5) The map-based sequence of the rice genome.Nature,436,793-800.   
International Wheat Genome Sequencing Consortium.(2014)A chromosomebased draft sequence of the hexaploid bread wheat (Triticum aestivum) genome.Science,345,1251788.   
Kaplan,N.and Dekker,J.(2013) High-throughput genome scaffolding from in vivo DNA interaction frequency. Nat. Biotechnol.31,1143-1147.   
Kent,W.J.and Haussler,D.(2001) Assembly of the working draft of the human genome with GigAssembler. Genome Res.11, 1541-1548.   
Kim,K.E.,Peluso,P., Babayan,P.,Yeadon,P.J.,Yu,C,Fisher,W.W.,Chin,C.S. et al.(2014) Long-read,whole-genome shotgun sequence data for five model organisms.Sci Data,1,140045.   
Kircher，M.，Sawyer，S.and Meyer，M. (2012) Double indexing overcomes inaccuracies in multiplex sequencing on the Ilumina platform.Nucleic Acids Res.40,e3.   
Koren,S.and Philippy,A.M. (2015) One chromosome,one contig: complete microbial genomes from long-read sequencing and assembly. Curr. Opin. Microbiol.23,110-120.   
Kurtz,S.,Phillippy,A., Delcher,A.L., Smoot,M., Shumway,M.,Antonescu,C. and Salzberg,S.L.(2O04) Versatile and open software for comparing large genomes. Genome Biol.5,R12.   
Leggett,R.M., Clavijo,B.J., Clissold,L.,Clark,M.D.and Caccamo,M.(2014) NextClip:an analysis and read preparation tool for Nextera Long Mate Pair libraries.Bioinformatics,30,566-568.   
Lennon,N.J.,Lintner,R.E.,Anderson,S.,Alvarez,P.,Barry,A.,Brockman,W., Daza,R.et al. (2010)A scalable,fully automated process for construction of sequence-ready barcoded libraries for 454. Genome Biol.11,R15.   
Li，H.and Durbin,R. (2O09)Fast and accurate short read alignment with Burrows-Wheeler transform.Bioinformatics,25,1754-1760.   
Li,H.,Handsaker,B.,Wysoker,A.,FennellT., Ruan,J., Homer, N.,Marth,G. et al.(2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics,25,2078-2079.   
Li,F.G.,Fan,G.Y.,Lu,C.R.,Xiao,G.H., Zou,C.S.,Kohel,R.J.,Ma, Z.Y.et al. (2015)Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1)provides insights into genome evolution.Nat. Biotechnol.33,524U242.   
Lucas,S.J.,Akpinar,B.A.，Kantar,M.,Weinstein,Z.,Aydinoglu,F.,Safar,J, Simkova,H.et al. (2013) Physical mapping integrated with syntenic analysis to characterize the gene space of the long arm of wheat chromosome 1A. PLoS ONE,8,e59542.   
Luo,M.C.,Thomas,C.,You,F.M., Hsiao,J.,Ouyang,S.,Buell, C.R.,Malandro, M.etal.(2o03）High-throughput fingerprintingofbacterial artificial chromosomes using the snapshot labeling kit and sizing of restriction fragments by capilary electrophoresis. Genomics,82,378-389.   
Makarevitch,I.,Waters,A.J.West,P.T,titzer,M.,irsch,C.N,Ros-aJ. and Springer,N.M.(2015) Transposable elements contribute to activation of maize genes in response to abiotic stress. PLoS Genet.11, e1004915.   
Martis,M.M.,Zhou,R.,Haseneyer,G.，Schmutzer,T.,Vrana,J. Kubalakova, M., Konig,S.et al. (2013) Reticulate evolution of the rye genome.Plant Cell, 25,3685-3698.   
Mascher,M.,Muehlbauer,G.J.,Rokhsar,D.S,Chapman,J.,Schmutz,J.Barry, K.,Munoz-Amatriain,M.et al. (2013a) Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).Plant J.76,718-727.   
Mascher，M.，Richmond,T.A.，Gerhardt,D.J.，Himmelbach,A.，Clissold,L, Sampath,D.,Ayling,S.et al.(2013b) Barley whole exome capture: a tool for genomic research in the genus Hordeum and beyond. Plant J.76,494-505.   
Matsumoto,T.，Tanaka,T.，Sakai,H.，Amano,N.,Kanamori，H.，Kurita,K., Kikuta,A.et al. (2011) Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries.Plant Physiol.156,20-28.   
Mayer，K.F.，Martis，M.，Hedley，P.E.，Simkova，H.，Liu，H.，Morrs，J.A., Steuernagel, B.et al. (2011) Unlocking the barley genome by chromosomal and comparative genomics.Plant Cell,23,1249-1263.   
Meyer,M.and Kircher,M. (2010) Illumina sequencing library preparation for highly multiplexed target capture and sequencing.Cold Spring Harb. Protoc., 2010,pdb prot5448.   
Minoche, A.E., Dohm,J.C.and Himmelbauer, H.(2011) Evaluation of genomic high-throughput sequencing data generated on Illumina HiSeq and genome analyzer systems. Genome Biol.12, R112.   
van Oeveren,J.,de Ruiter,M.,Jesse,T.,van der Poel,H.,Tang,J.,Yalcin,F., Janssen，A.et al. (2011） Sequence-based physical mapping of complex genomes by whole genome profiling. Genome Res.21, 618-625.   
Paux,E., Sourdille,P., Salse,J., Saintenac,C., Choulet,F.,Leroy,P., Korol,A. et al. (20o8) A physical map of the 1-gigabase bread wheat chromosome 3B. Science,322,101-104.   
Philippe,R.,Paux,E.,Bertin,I.,Sourdille,P., Choulet,F.,Laugier, C.,Simkova, H.et al. (2013)A high density physical map of chromosome 1BL supports evolutionary studies,map-based cloning and sequencing in wheat. Genome Biol. 14, R64.   
Poursarebani,N.,Nussbaumer,T.,Simkova,H.,Safar,J.,Witsenboer,H.,van Oeveren,J.,Dolezel, J.et al.(2014)Whole-genome profiling and shotgun sequencing delivers an anchored, gene-decorated,physical map assembly of bread wheat chromosome 6A.Plant J.79,334-347.   
Quinlan,A.R.and Hall,I.M.(201O) BEDTools:a flexible suite of utilities for comparing genomic features. Bioinformatics,26,841-842.   
Raats,D.,Frenkel, Z., Krugman,T.,Dodek,I., Sela,H., Simkova,H.,Magni,F. etal.(2013) The physical map of wheat chromosome 1BS provides insights into its gene space organization and evolution. Genome Biol.14, R138.   
Rodrigue,S.，Materna, Timberlake,S.C.,Blackburn，M.C.，Malmstrom, R.R.,Alm,E.J.and Chisholm,S.W.(2010) Unlocking short read sequencing formetage PLoS ONE,5,e11840.   
Sathuvalli,V.R.and Mehlenbacher,S.A. (2013) De novo sequencing of hazelnut bacterial artificial chromosomes (BACs) using multiplex Ilumina sequencing and targeted marker development for eastern filbert blight resistance.Tree Genet. Genomes,9,1109-1118.   
Schmutzer,T.，Ma,L.，Pousarebani，N.，BullF.，Stein,N.，Houben,A.and Scholz,U.(2014) Kmasker-a tool forin silico prediction of single-copy FISH probes for the large-genome species Hordeum vulgare.Cytogenet. Genome Res. 142,66-78.   
Schnable,P.S.,Ware,D.,Fulton,R.S.,Stein,J.C., Wei,F.,Pasternak,S.Liang, C.et al. (20o9) The B73 maize genome: complexity, diversity,and dynamics. Science,326,1112-1115.   
Schulte,D.,Close,T.J.,Graner,A.,Langridge,P.,Matsumoto,T.,Muehlbauer, G., Sato,K.et al. (2oo9) The international barley sequencing consortium-at the threshold of efficient access to the barley genome.Plant Physiol.149, 142-147.   
Schulte,D.,Ariyadasa,R.,Shi,B.,Fleury,D.,Saski,C.,Atkins,M.,deJong,P. et al. (2O11) BAC library resources for map-based cloning and physical map construction in barley (Hordeum vulgare L.).BMC Genom.12,247.   
Shizuya, H., Birren,B., Kim, U.J.,Mancino,V., Slepak,T.,Tachiri, Y.and Simon M.(1992) Cloning and stable maintenance of 3o0-kilobase-pair fragments of human DNA in Escherichia coli using an F-factor-based vector.Proc.Natl Acad.Sci. USA,89,8794-8797.   
Stapley, J.,Santure,A.W.and Dennis,S.R.(2015) Transposable elements as agents of rapid adaptation may explain the genetic paradox of invasive species. Mol. Ecol.24,2241-2252.   
Steuernagel,B.,Taudien,S.,Gundlach,H.,Seidel,M.,Ariyadasa,R.,Schulte, D.,Petzold,A.et al. (2009) De novo 454 sequencing of barcoded BAC pools for comprehensive gene survey and genome analysis in the complex genome of barley. BMC Genom.10,547.   
Taudien,S.,Steuernagel,B.,Ariyadasa,R.,Schulte,D.,Schmutzer,T.,Groth, M.，Felder，M.et al.(2O11） Sequencing of BAC pools by different next generation sequencing platforms and strategies. BMC Res. Notes,4,411.   
Weber，J.L.and Myers， E.W.(1997） Human whole-genome shotgun sequencing. Genome Res.7,401-409.   
Wendler,N., Mascher,M., Noh,C., Himmelbach,A.,Scholz, U., Ruge-Wehling, B.and Stein,N. (2014) Unlocking the secondary gene-pool of barley with next-generation sequencing.Plant Biotechnol.J.12,1122-1131.   
Whiteford,N.，Skelly,T.，Curtis,C.，Ritchie,M.E.，Lohr,A.,Zaranek,A.W., Abnizova,l.et al. (2oo9) Swift: primary data analysis for the llumina Solexa sequencing platform.Bioinformatics,25,2194-2199.   
Wicker,T.,Matthews,D.E.and Keller,B.(2OO2) TREP:a database for Triticeae repetitive elements.Trends Plant Sci.7,561-562.   
Zhang,Z.,Schwartz,S.,Wagner,L.and Miller,W.(20oO)A greedy algorithm for aligning DNA sequences.J. Comput. Biol.7,203-214.   
Zhang,G.F.,Fang,X.D.,Guo,X.M.,Li,L.,Luo,R.B.,Xu,F.,Yang,P.C.et al. (2012)The oyster genome reveals stress adaptation and complexity of shell formation.Nature,490,49-54.

# Supporting information

Additional Supporting information may be found in the online version of this article:

Figure S1 Analysis of contig/scaffold L5O as well as gain of longest sequence per BAC and reduction of amount of sequences in relation to contig L50.   
Figure S2 Structure of the sequenced DNA fragments.   
Table S1 Accession numbers of 668 BAC assemblies generated in this study and associated raw read data.   
Table S2 Accession number of reference BACs.   
Table S3 Mapped mate pair reads per BAC.   
Table S4 Assembly statistics for 668 BAC assemblies (paired-end assemblies and scaffolded assemblies).   
Table S5 Assembly quality statistics for sequenced reference BACs (paired-end assemblies prior to scaffolding).   
Table S6 Sequencing indices for multiplex BAC sequencing. DataS1 Detailed description of web-lab methods.