# THEEFFECTOFMECHANICAL TENSIONONDNA POLYMERASEACTIVITYSTUDIED WITH A TWO-STATE MODEL

YAOGEN SHU\*and HUALIN SHIt Institute of Theoretical Physics,The Chinese Academy of Sciences, P.O.Box 2735，Beijing100080,China \*yaogen@itp.ac.cn thlshi@itp.ac.cn

Received 25May 2006

Single-molecule experiments revealed that the replication rate of T7 DNA polymerase has a peak value at tension of about 5.5 pN.However,the mechanism leading to this peak has only been partially investigated.Here,we present a two-state model to investigate the effect of mechanical tension on DNA polymerase activity. The model consists of polymerase and exonuclease.The polymerase's kinetic pathway has been simplified into Michaelis-Menten form with two steps.One is the conformational change from“open" to “close”.The other is the base-stacking. The results are in good agreement with experimental observations.We also predict that the tension on template is beneficial to the fidelity of DNA replication.

Keywords:DNA replication; mechanical tension; two-state model.

# 1．Introduction

DNA polymerase (DNAp) motor is a protagonist in the propagation of genome. It works in a cyclic fashion and moves along a partly single-stranded (ss),partly double-stranded (ds) DNA chain.In each catalytic cycle,a single nucleotide will be added to the daughter strand. If the incoming nucleotide is complementary to that on the template strand,a base pair will form. Otherwise, it will be removed by exonuclease due to the fidelity of DNA replication. $^ { 1 - 3 }$

Crystal structure of T7 DNAp $^ 4$ has revealed that the polymerase resembles a half-open right hand. The“palm” subdomain forms a cleft flanked by the “finger” and “thumb” subdomains.The three subdomains hold the daughter-template DNA in S-shape and position the incoming dNTP for incorporation into DNA. The palm subdomain contains the catalytic site where the nucleotidyl transfer takes place. The fingers subdomain interacts with template and positions the incoming dNTP,while the thumb subdomain primarily binds the duplex DNA in a sequence-independent manner along the minor groove.

T7 DNAp contains a $3 ^ { \prime }$ $5 ^ { \prime }$ exonuclease domain which proofreads newly synthesized DNA and corrects mismatched base pairs to reduce the inherent low error ratio $3 , 5 \mathrm { - } 1 2$ of $1 0 ^ { - 5 }$ to $1 0 ^ { - 7 }$ . On the rare occasion that a mismatch is incorporated into DNA, the DNAp must unwind at least $m = 8 { \ - } - 9$ base pairs and move the daughter strand from the polymerase site to the exonuclease site for editing.3

![](images/035a735ef1a2556b6b6af93efe2e4af4c4b066648c5aa4178b97f4e5378d03d2.jpg)  
Fig.1. The effect of mechanical tension on the net replication rate catalyzed by T7 DNAp. Circular dots and triangles represent our experimental data,13 the dotted line represents one previous model's results,20 and the solid line is our model's $\langle r _ { \mathrm { p } } \rangle$ ，

Single-molecule experiments $^ { 1 3 , 1 4 }$ have investigated the effect of mechanical tension on T7 DNAp activity. The replication rate is strongly dependent on tension and has a peak at a critical tension of about 5.5 pN. The peak value approaches the rate of rate-limiting step in ensemble measurement,15 while the critical tension is located at the vicinity of the crossover force in the force-extension curves for dsDNA and ssDNA. $^ { 1 6 , 1 7 }$ Previous theoretical models13,14,18-21 could not quantitatively explain the origin of the peak.For example, the dotted line $^ { 2 0 }$ in Fig.1 is one result of those models.We will show in this paper that this feature can be naturally explained by a two-state model, the prediction of which is shown by the solid line in Fig.1.In our model, the exonuclease is considered as independent of tension, while the polymerase's kinetic pathway $^ { 1 5 , 2 2 , 2 3 }$ will be simplified into Michaelis-Menten form with two steps. One is the conformational change from “open” to “close”. The other is the base-stacking.We will introduce Van der Waals-like potential to describe the conformational change before the formation of covalent bond between the “O”atom of the $3 ^ { \prime }$ -hydroxyl group of the daughter strand and the “P” atom of the $\alpha$ -phosphate of the incoming dNTP.

$$
\mathrm { \underbrace { \overbrace { ( E _ { \ell } D N A _ { n d } { \overbrace { \mathbf { r } } _ { \ell } } E _ { \ell } D N A _ { n } } ^ { \mathrm { 1 } } ) } _ { \mathrm { \underbrace { \overbrace { ( A _ { \ell } D N A _ { n } \overbrace { \mathbf { k } } _ { \ell - 1 } \overbrace { \mathbf { k } } _ { \ell } D N A _ { n } } ^ { \mathrm { 1 } } ) } _ { \mathrm { \underbrace { \overbrace { ( A _ { \ell } D N A _ { n } \overbrace { \mathbf { r } } _ { \ell - \cdots } \overbrace { \mathbf { r } } _ { \ell - \cdots } \cdots \cdots \cdots \cdots \cdots } } } ^ { \mathrm { \overbrace { ( A _ { \ell } ) } } } } } } } 
$$

Fig.2.The two-state model. State 1 represents the exonuclease,which is independent of tension. State 2 represents the polymerase,which has been simplified into a Michaelis-Menten form with two steps. $k _ { 1 }$ represents the conformational change from“open”to “close”. $k _ { 2 }$ is the base-stacking. Both steps are affected by the tension on the template.

# 2.Two-State Model

Figure 2 shows our two-state model. State 1 represents the exonuclease,while state 2 represents the polymerase.We assume a mechanism exists in T7 DNAp:

(1）In the case of correct incorporation, the potential of state 1 is $\Delta G ^ { \dagger }$ higher than that of state 2.   
(2）Once an incorrect incorporation occurs, the potential difference between two states will be exchanged, that is, the potential of state 2 is $\Delta G ^ { \dagger }$ higher than that of state 1.It can be estimated that $\Delta G ^ { \dagger } \approx m \Delta G ( 0 )$ ，where $\Delta G ( 0 )$ is the free energy required to melt one base pair.

In calculation of net replication rate,the mismatch situation is excluded. We assume the transition from state 2 to state 1 in single-molecule experiment $^ { 1 3 }$ is induced only by the force on the template.At steady state,the probability that DNAp is found in state 1 or 2 is given by

$$
p _ { 1 } = \frac { 1 } { 1 + e ^ { m [ \Delta G ( 0 ) - \Delta g ( f ) ] / k _ { \mathrm { B } } T } } ,
$$

and

$$
p _ { 2 } = \frac { 1 } { 1 + e ^ { m [ \Delta g ( f ) - \Delta G ( 0 ) ] / k _ { \mathrm { B } } T } } ,
$$

respectively.Here $k _ { \mathrm { B } }$ is the Boltzmann constant, $T$ the temperature,and $\Delta g ( f ) =$ $f [ x _ { \mathrm { s s } } ( f ) - x _ { \mathrm { d s } } ( f ) ]$ ，where $x _ { \mathrm { s s } } ( f )$ and $x _ { \mathrm { d s } } ( f )$ are the end-to-end distances per base of ssDNA and dsDNA at tension $f$ ，and can be obtained from measurements of the stretching of DNA strands subjected to tension. $^ { 1 6 , 1 7 }$ Figure 3(a) shows the probabilities versus tension.

# 2.1.State 1:Exonuclease

According to the widely accepted editing mode,12 the melted daughter strand does not contact the template.The depolymerizing rate,thus,can be considered as independent of tension. Experiment data $^ { 1 3 }$ also shows that the exonuclease rate, $r _ { \mathrm { e } }$ ,is saturated if tension is higher than 45 pN.We assume that $r _ { \mathrm { e } }$ is independent of the tension and determined mainly by the rate of motor backward stepping. It can be estimated from experiment data $^ { 1 3 }$ as $r _ { \mathrm { e } } \approx 3 0$ bases/s.

![](images/1ee1f34810784d3cb9b6411772e96351a9c23f9199a554b3b7184359ef3976dd.jpg)  
Fig.3.The probabilities that DNAp is found in (a) two states and (b) the predicted error ratio versus tension on template.

# 2.2.State 2:Polymerase

The pathway of DNA synthesis catalyzed by T7 DNAp has been revealed.15,22,23 Both DNA and the correct dNTP bind polymerase rapidly with high affinity, following which the polymerase undergoes a slow, rate-limiting conformational change from“open” $( \mathrm { E _ { p } \bullet \mathrm { D N A _ { n } } } )$ ）to“close” $( \mathrm { E _ { p } ^ { * } \bullet \mathrm { D N A _ { n } } }$ ),in which the orientation of the nascent base pair will be adjusted optimally in preparation for the nucleophilic attack $^ { 2 4 }$ between the“O” atom of the $3 ^ { \prime }$ -hydroxyl group of the daughter strand and the “P” atom of the $\alpha$ -phosphate of the incoming dNTP. The chemistry step is fast,and is followed by a reversed conformational change,which accompanies the base-stacking and leads to product release and translocation of DNA for the next catalytic cycle.

For saturated DNAp and dNTP, as well as low PP $^ { 1 }$ concentrations in singlemolecule experiment,13 the kinetic DNA synthesis pathway for T7 DNAp $^ { 1 5 , 2 2 , 2 3 }$ can be simply described by the Michaelis-Menten form as Fig.2,where $k _ { 1 }$ represents the conformational change of palm from “open” to “close”,and $k _ { 2 }$ is the basestacking,which accompanies the reversed conformational change.The two processes are inevitably affected by the tension on the template.

![](images/661a868024e1a37e871f2464f8ba81bba35104416648534e1b6e660e92e15f56.jpg)  
Fig.4. The orientation of the nascent（“+1"）base pair. $\theta$ is the angle between“ $+ 1$ ”and“ $_ 0$ ； base pairs; $\mathcal { D }$ ,the diameter of B-DNA; $d$ ,the distance between the“O”atom of $3 ^ { \prime }$ -hydroxyl group of the daughter strand and the “P” atom of $\alpha$ -phosphate of the incoming dNTP；and $d _ { 0 }$ is the sum of theVan der Waals radii ofO and $_ \mathrm { P }$ atoms.

# 2.2.1. Conformational change of palm from “open” to “close”

As the result of evolution,T7 DNAp adjusts optimally the orientation of the nascent base pair in palm to form a covalent bond and achieve an optimal replication rate. In ensemble measurement,15 the DNA primer/template only consists of a synthetic 25/36-mer. In single-molecule experiment, $^ { 1 3 }$ however, the measured chain contains 10,416 base pairs and a constant force was applied on the template.The maximum replication rate $^ { 1 3 , 1 4 }$ implies that DNAp will work most efficiently at critical tension, $f _ { c }$ ，with $x _ { \mathrm { s s } } ( f _ { c } ) = x _ { \mathrm { d s } } ( f _ { c } )$ . We assume the orientation of nascent base pair(“+1") in single-molecule experiment, $^ { 1 3 }$ $\theta$ shown in Fig. 4,will vary with the tension as $[ x _ { \mathrm { s s } } ( f ) - x _ { \mathrm { d s } } ( f ) ]$ ,that is

$$
\theta ( f ) = \theta _ { \mathrm { m a x } } [ x _ { \mathrm { s s } } ( f ) - x _ { \mathrm { d s } } ( f ) ] / d _ { \mathrm { d s } } ,
$$

where $d _ { \mathrm { d s } } \approx 0 . 3 4 ~ \mathrm { n m }$ ，is the distance between the adjacent sugar-phosphates of B-DNA. The distance between the“O” and the“P” atoms before the formation of covalent bond in Fig.4 can be given by

$$
d ( f ) = \sqrt { d _ { 0 } ^ { 2 } + [ 2 D \sin ( \theta / 2 ) ] ^ { 2 } } ,
$$

where $D \approx 2$ nm and $d _ { 0 } \approx 0 . 3 4 2 \ \mathrm { n m }$ ,are the diameter of B-DNA and the sum of the Van der Waals radii of $\mathrm { \Delta O }$ and $\mathrm { P }$ atoms,25 respectively. We introduce the Van

der Waals-like potential

$$
U ( f ) = \varepsilon \left[ \left( \frac { d _ { 0 } } { d ( f ) } \right) ^ { 1 2 } - 2 \left( \frac { d _ { 0 } } { d ( f ) } \right) ^ { 6 } \right]
$$

to describe the conformational change of palm from “open”to “close”,that is, the potential of “open” state is $- U ( f )$ higher than that of“close” state.Because the dsDNA in palm is in an S-shape $^ 4$ whose distance per base approximates to that of ssDNA, the displacement at $f$ direction in this step can be neglected.The conformational change from “open” to“close” in single-molecule experiment $^ { 1 3 }$ can be corresponded to that in ensemble measurement $^ { 1 5 }$ by

$$
\begin{array} { r } { \varepsilon = k _ { \mathrm { B } } T \ln \frac { k _ { \mathrm { 1 } } ( f _ { c } ) } { k _ { - 1 } } , } \\ { k _ { \mathrm { 1 } } ( f ) = k _ { - 1 } e ^ { - U ( f ) / k _ { \mathrm { B } } T } . } \end{array}
$$

The electrostatic interactions in palm is considered as independent of tension and has been taken into account in $\varepsilon$

# 2.2.2. Base-stacking

The reversed conformational change from “close” to “open”15 shown in Eq. (8) accompanies base-stacking, which will organize the adjacent sugar-phosphate units from S-DNA to B-DNA geometry, and has to do additional work $\Delta g ( f )$

$$
\mathrm { E _ { p } ^ { * } \bullet D N A _ { n } } \ \overset { k _ { 2 + } } { \underset { k _ { 2 - } } { \longmapsto } } \ \mathrm { E _ { p } \bullet D N A _ { n + 1 } } \ .
$$

If we assume that the additional work contributes to the activation energy required to reach a transition state from the“close” state,the rate coefficient for this step is

$$
k _ { 2 + } ( f ) = k _ { 2 + } ( 0 ) e ^ { - \Delta g ( f ) / k _ { \mathrm { B } } T } ,
$$

where $k _ { 2 + } ( 0 )$ is contributed mainly by the excess energy in base-stacking. The net rate of base-stacking in Fig.2 is

$$
k _ { 2 } ( f ) = k _ { 2 + } ( f ) - k _ { 2 - } .
$$

The polymerase rate in Fig. 2 can be given by Michaelis Law:

$$
r _ { \mathrm { p } } ( f ) = \frac { k _ { 1 } ( f ) k _ { 2 } ( f ) } { k _ { - 1 } + k _ { 1 } ( f ) + k _ { 2 } ( f ) } .
$$

The average replication rate then can be given by

$$
\left. r _ { \mathrm { p } } \right. = p _ { 2 } r _ { \mathrm { p } } - p _ { 1 } r _ { \mathrm { e } } .
$$

Table 1. The fitted parameters with Eq.(12) to experiment data.13   

<html><body><table><tr><td>△G(0)</td><td>k-1</td><td>0max</td></tr><tr><td>2.1kBT</td><td>10 s-1</td><td>17°</td></tr></table></body></html>

# 3．Discussion

With the proposed parameters from bulk experiments $^ { 3 , 1 5 , 2 2 , 2 3 }$ $m = 8$ ， $k _ { 1 } ( f _ { c } ) =$ （204号 $3 0 0 ~ \mathrm { { s } ^ { - 1 } }$ ， $k _ { 2 + } ( 0 ) = 1 2 0 0 ~ \mathrm { s } ^ { - 1 }$ and $k _ { 2 - } ~ = ~ 1 8 ~ \mathrm { s ^ { - 1 } }$ ，we can use Eq. (12) to fit the experimental data.13 The fitted parameters are listed in Table 1. The solid line in Fig.1 shows that Eq. (12) is in good agreement with the experiment data.13

The assumed mechanism can also be used to explain error ratio. Once a mismatch is incorporated into DNA,DNAp must unwind $m$ base pairs and move the daughter strand from the polymerase site to the exonuclease site for editing.’ The probability that the $3 ^ { \prime }$ -hydroxyl group of the daughter strand still stays at the polymerase site for next cycle is the error ratio, which is about e-m[△G(0)+△g(f)] at steady state. Figure 3(b) shows the predicted error ratio versus tension on template.The referenced $m = 8$ and fitted $\Delta G ( 0 ) = 2 . 1 k _ { \mathrm { B } } T$ are reasonable because the predicted error ratio for free DNA is consistent with the widely accepted data.12 The tension on the template,thus,is beneficial to the fidelity of DNA replication.

Although we used a fitting parameter $\theta _ { \mathrm { m a x } } = 1 7 ^ { \circ }$ ， Fig. 5(a) shows that the angle varies from $- 6 ^ { \circ }$ to $6 ^ { \circ }$ with increasing $f$ . This range awaits testing. The replication rate at low tension,13o bases/s,implies that the orientation to $- 6 ^ { \circ }$ may be constrained by the“close” palm. Figure 5(b) shows that the conformational change from“open” to “close” is a rate-limiting step for all tensions.Near $f _ { c }$ , there is a plateau for $k _ { 1 }$ at $f \in [ 4 . 5 , 6 . 5 ] \ \mathrm { p N }$ ，which may correspond to the situation in ensemble measurement.15

Unlike previous theoretical models,13,14,18-21 our model assumes that T7 DNAp does not work in the conformational change from “open” to “close”. This assumption is consistent with that where the dsDNA in T7 DNAp complex is bent into an S-shape by extensive interactions with the thumb and fingers.4 In the reversed conformational change, T7 DNAp has to do additional work because of base stacking.

# 4.Conclusion

Our model has for the first time explained the peak replication rate observed in experiment.13 The tension on the template not only affects the base-stacking, but also makes a great impact on the conformational change from “open” to “close”, which is described by a Van der Waals-like potential related to the distance between the“O” atom of $3 ^ { \prime }$ -hydroxyl group of the daughter strand and the “P” atom of $\alpha$ -phosphate of the incoming dNTP. The net replication rate of our framework is in good agreement with the experiment data $^ { 1 3 }$ as shown in Fig.1. If the tension on the template in single-molecule experiment $^ { 1 3 }$ is between 4.5 and 6.5 pN, T7

![](images/960eb234cf9f311de004ba2db8b34a6d6bf54340948cd66bd257e0e8d49ee9ae.jpg)  
Fig.5. (a) $\theta$ is the orientation of the nascent base pair, $\Delta g$ ,the work to convert one ds residue into a ss residue,and $U$ ,Van der Waals-like potential,versus tension on the template.(b) Rate constants versus tension. $k _ { 1 }$ represents the conformational change from“open”to“close”,and $k _ { 2 }$ does base-stacking. $f _ { c }$ is the critical value. $\Delta f \approx 2$ pN,is the plateau of optimal replication.

DNA polymerase works most efficiently and its rate approaches that of ensemble experiment.We can also predict that the tension on the template is beneficial to the fidelity of DNA replication.

# Acknowledgments

We acknowledge useful discussions with Ou-Yang Zhong-can， Haijun Zhou and Ming Li. This work was supported by the National Science Foundation of China.

# References

1.A.Kornberg and T.Baker, DNA Replication, 2nd edn.(W.H. Freeman, New York, 1992).   
2.C.Joyce and T. Steitz,Ann. Rev. Biochem.63(1994) 777.   
3．K.A.Johnson,Ann.Rev.Biochem.62(1993) 685.   
4.S.Doublie,S.Tabor,A.M.Long, C.C.Richardson and T.Ellenberger,Nature, 391 (1998) 251.   
5.T.A. Steitz,J.Biol.Chem.274(1999) 17395.   
6.C.A.Brautigam and T.A. Steitz,Curr. Opin. Struct.Biol.8(1998) 54.   
7．J.Jager and J.D.Pata,Curr.Opin. Struct.Biol.9(1999) 21.   
8．L.A.Loeb and T.A.Kunkel,Ann.Rev.Biochem.51(1982) 429.   
9．H. Echols and M. F.Goodman,Ann. Rev.Biochem.60 (1991) 477.   
10．T.A.Kunkel,J. Biol.Chem.267(1992) 18251.   
11.L. B. Bloom, X. Chen,D.K. Fygenson, J. Turner, M. O'Donnell, M.F. Goodman, J.Biol. Chem.272(1997) 27919.   
12.B.Alberts,A.Johnson，J.Lewis,M.Raff,K.Roberts and P.Walter，Molecular Biology of the Cell,4th edn.(Garland Science Publishing,2002),p. 242.   
13.G.J.L.Wuite, S.B.Smith,M. Young,D.Keller and C.Bustamante,Nature, 404 (2000）103.   
14.B.Maier,D.Bensimon and V.Croquette,PNAS,97(2000) 12002.   
15. S. S.Patel, I.Wong and K.A. Johnson,Biochemistry 30 (1991) 511.   
16.S.B. Smith,Y.J.Cui and C.Bustamante, Science 271(1996)795.   
17.I.Rouzina and V.A.Bloomfield,Biophysical J.8O(20o1) 882.   
18．A.Goel,M.D. Frank-Kamenetskii,T.Ellenberger and D.Herschbach，PNAS 98 (2001) 8485.   
19．A.Goel,T.Ellenberger,M.D.Frank-Kamenetskii and D.Herschbach,J. Biomol. Struct.Dyn.19 (2002) 1.   
20．A.Goel,R.D.Astumian and D.Herschbach,PNAS100(2003) 9699.   
21.I.Andricioaei, A. Goel, D.R. Herschbach and M. Karplus,Biophysical J.87(2004)   
1478.   
22.I.Wong,S.S.Patel and K.A.Johnson,Biochemistry 30 (1991) 526.   
23.M. J.Donlin,S.S.Patel and K.A.Johnson,Biochemistry 30(1991） 538.   
24．M.M.Hingorani and M. O'Donnell, Cur. Org. Chem.4(2000) 887.   
25.K.E.van Holde,W.C. Johnson and P.S.Ho,Principles of Physical Biochemistry (Prentice Hall, Upper Saddle River,NJ,1998).