# A Machine Learning Assisted, Label-free, Non-invasive Approach for Somatic Reprogramming in Induced Pluripotent Stem Cell Colony Formation Detection and Prediction

Ke Fan1.2+, Sheng Zhang $^ { 1 , 2 , + }$ , Ying Zhang1.2, Jun Lu1.2, Mike Holcombe34, and Xiao Zhang1,2\*

lCAS Key Laboratoryof Regenerative Biology, Joint Schoolof Life Sciences, Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences,Guangzhou 510530,China;Guangzhou Medical University,Guangzhou 511436, China.   
2Guangdong Provincial Key Laboratory of Biocomputing,Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences, Guangzhou 510530, China   
3Department of Computer Science,Universityof Sheffeld, Sheffield, United Kingdom   
4epiGenesys, Sheffield, United Kingdom   
\*zhang_xiao@gibh.ac.cn   
+these authors contributed equally to this work

# ABSTRACT

Duringcellarreprogramming,themesenchymal-toepithelialtransitionisaccompaniedbychanges inmorphologywhichocur priortoiPSCcolonyformation.Thecurrentapproach fordetecting morphologicalchangesassociatedwith reprogramming purely relies on human experiences,which involve intensiveamounts of upfront training,humaneror withlimitedquality controlandbatch-to-batchvariations.Here,wereportatime-lapse-basedbright-fieldimaginganalysis systemthatalows us to implementalabel-free,non-invasiveapproachtomeasuremorphologicaldynamics.Toautomaticallyanalyseanddetermine iPSCcolonyformation,amachinelearning-basedclaification,segmentation,ndstatisticalmodelingsystemwasdeveloped to guidecolony selection.The systemcandetect and monitortheearliest celular texture changes aftertheinduction of reprogramming inhumansomaticcelsonday7fromthe20-24dayprocess.Moreover,afterdetermining thereprogramming processand iPSCcolonyformationquantitatively,amathematicalmodelwasdevelopedtostatisticallpredictthebestiPSC selectionphase independentofanyotherresources.Allthecomputationaldetectionand predictionexperimentswereevaluated usingavalidationdataset,andbiologicalverficationwasperformed.Thesealgorithm-detectedcoloniesshownosignificant diferences (Pearson Coeffcient) in terms of their biological features compared to the manually processed colonies using standard molecular approaches.

# Introduction

Stemcels area kindof self-replenishing cellpopulation.Their primary functionis to generate progenywhich thendevelop intoterminallydierentiatedcelltypes.Tissue-specificadult stemcelsor progenitors arecommitted toproducing tisse-or lineage-specificcells.Other kinds of stemcellscalld pluripotent or even totipotent stemcells have the potential to giverise to any of the $2 0 0 +$ cell types in all three germ layers. The following are the two types of pluripotent stem cells defined by theirtissue origin: (i) embryonic stem (ES)cellsobtainedfromearlyembryos,typicallyatthe blastulastageand (ij induced pluripotent stem (iPS)cells derived through a reprogramming process whereby terminally diferentiated somatic cels are reprogrammedorinducedback toapluripotent state.iPS technologyis apersonalized approachanddoes notrequire embryos; therefore,unlikeEScels,theethicalissues donotapply.Recently,iPSCs have beenused to treatchronicanddegenerative diseases;forexample,retial degenerative blindnessduetoretinitis pigmentosal.Tus,clinicalapplicationsofiScellscan be generated forpersonalizedformatsforcellreplacementtherapy.Moreover,tesearestrongdemands forquality-assred iPSClines fromboth pharmaceutical industryand basic medicalresearch organizations2.Anon-invasive approach has been establishedtoconverturinalcelsintoiPSCs.This is themostconvenient waytoobtainacellsoureforreprogrammingstudies, and we have implemented this method in this study for data training and statistical modelling3.

One issue that appears to limit iPSC qualityor biological consistency in further applications is colony determination, which leads tote isolationand purificationof iPSCcolonies during thecellarreprogramming process.Thecurrntsolution relies ona judgementcallfrom well-trainedcellculture experts,though the training is time consuming andhasahighcost. Incontrastand inevitably,thenatural instabilityofhumanrecognitionwillcouldallow formisjudgement,whichcancause batch-to-batch variation.The qualitycontrolof iPSCcolonydetermination is extremely importantfordownstream expansion andfor maintainingahomogeneous cultureofundiferentiatedcells.Inconsistencyincolonydeterminationand selection wil causeinsufcientdownstream differentiationintofunctionalcels.Therefore,anautomatedquantitative method withconsistent colony maturation determination wil be a great for assisting biologists during the iPSC formation process.

The traditional approach for identifying and verifying iPSCcolonies is touse immunofluorescence staining orareporter systemto detect pluripotent markers,which could couple wellwith automated fluorescence microscopy provides and provide a dynamic and effctive method4.5.However,this kind of xeno probe-based labelling can only be applied during a late stag inreprogramming;thiscannotbedelayed furtherduetopotentialsafety isses fordownstream clinicalappications67. Furthermore,during celular reprogramming studies,it has been observed thatalong with positive iPSCcolonies,negative colonies contain various morphology subtypes,especially with respect tocellularpolarity.The celular polarityis strongly linkedto thespecificityof geneexpression,cellcycle,andothercellarregulationandmayexplain thedifferentreprogamming mechanisms.For example,thecellpolaritychanges betwee the mesenchymal-to-epithelial transition(MET)orEMT, which is linked with induced pluripotent stemcell(iPSC)colony formationor tumour genesis9-11.Hence,thereprogramming process has astrong link with morphologychanges; theclasificationof the morphologycanbe used asaread-outina quantitative way to ientifyiPSCcoloniesand to monitor the reprogramming process.This provides richer information than the binary fluorescence images and opens the door for a label-free, non-invasive approach.

Here,wereportalabel-fre,non-invasiveandconsistently quantitative methodfor human iPSCdetection inanaccurate way.This automated systeminvolvesacomputer visionrecognitionmodel toassist inthe classificationofiPSCcolonies from bright-field microscopicimages.This modelutilizesaconvolutionneuralnetwork asaclasiferinasliding windowsframework forcolonyrecognition.Subsequently,asemi-supervisedsegmentationmethod wasappliedtolocate thecoloniesanddetecttheir boundaries.Moreover,aHidenMarkov Model (HMM) was trained toestimate the growth phase and maturationtime window ofcolonyformationduringthereprogramming process.Finaly,usingdatafromcolonies traced viatime-lapse,thissystemcan predict the bestselectiontime window for iPSCcolonies to preventrandom diferentiationcaused byovergrowth.Ourresults showthatthis algorithm detected and predicted colonies and demonstrated no significant differences (Pearsoncoeficient r $\textit { i }$ 0.9)in terms ofbiologicalfeatures compared to manuall processedcolonies.This wasevaluatedandcharacterizedbiologicaly using standard immunofluorescence staining,quantitative polymerase chain reaction (QPCR),and RNA-Seq for pluripotent verification.

# Results

# Automated iPSC colony detection using computer vision algorithms and verified via OG mice as Oct-4-GFl reportersystem

During the reprogramming proces,the celular polarization reshaping leads to morphology changes, which indicate iPSC colony formation.A novelcomputer vision assisted method was developed to process the microscopic images.The sliding window approach was performed toscan the potentialcolonyareas,folowedbythe detection,hichgenerates abinary image. Post-processing including discarding the fragmented areas was performed using thebinary images as the template.Therefore, theareas of interestrefering to theiPSCcolonies canbedeterminedbasedonsizeand location(Figure 1B).Examplesof cropped positiveand negative colony texture mosaic windowsfromthe algorithmtraining is shown inFigure1A.The algorithm detects the positive areas in a whole well from a 6-well cell culture plate.

The merits of the algorithms were asessed by verifying the biological signatureof the iPSCs using OG mice with an OCT4- GFP knock-inreporter mice embryonic fibroblast (MEF)system.The activationofthe OCT4 pluripotent gene indicated MEF reprogramming success of $\mathrm { i } \mathrm { P S C s } ^ { 1 2 }$ . This biological fluorescence indication was used as a positive feedback mechanism for algorithmtraining.Subsequentlyweusedthe trainedclasifiertogeneratetheisolatedbinaryimages,where thefluorescence signal areas were masked out as the overlay [FigurelB]. The observations showed that nearly $10 \%$ of the cells from the total population didnot express OCT4-GFP;thus,thesecells werenotselectedbythe algorithm.Thequantitative analysis showed that the correlation between OCT4-GFP expression and the algorithm overlay was highlyrelevant, with anaverage Pearson's Coefficient of $\mathrm { r = } 0 . 8 7 7$ . The r-value was determined from 3 individual reprogramming experiment data sets (other results are shown in Supplemental Figures S2 and S3.)

# Human iPSC colony detection using computer vision based algorithms verified via pluripotency biomark ers

Human PSCregeneration opens the dooras apersonalized cell source forcellreplacement therapy inregenerative medicine. This algorithmhas extended thenon-ivasive andlabel-freecomputer assisted methodforhuman urinalcellreprogramming3. Diferent fromthe OG mice system,since urinalcells OCT4-GFPis notapplicable in thiscase,we built the training dataset using manual cropping processed by wel-trained cellbiologists (Figure 2A). As shown in the previous section,our iPSC detection frameworkconsists ofamosaic sliding window-based classification processIfasmallpatch islocated inapositive iPSCcolony texture,the area was then maskedoutbythe overlay; otherwise,it is anegative patch.The training session was run with numerous manuallycropped samples,followedbythe annotationof thepositivecolonies;this indicated thelocations of regions after post-processing for an entire well in a 6-well plate (Figure 2B).

![](images/05a913b3132a7f27c54eb7c482879fedab2b71c74a45cefd8a6d2ad3a69ab563.jpg)  
Figure1.Computer vision based detection of mice iPSCcolonies.(a)Example oftraining patches.Positive samples cropped out as iPSCcolonies and negative samplesare MEFcels or background.(b) Verificationofcomputer vision based method usingthe OCT4-GFPreporter.Panelsfromlefttorightindicate the brightfieldoriginalimage,amask ofcomputerdetection, an OCT4-GFP fluorescent image, and an overlay between the fluorescence and binary images.

![](images/d288b2886a43c0ac818c17f7c17f39c0deae9e782712a308fb14c8c016b905d4.jpg)  
Figure 2. Computer vision based detection of human urinal cels (UCs)derived iPSC colonies.(a) Example of training patches.Positive samples are parts of iPSC colonies,and negative sample are other cels or background. Scale bar, $2 0 0 \mu \mathrm { m }$ (b) A detection map.Example of iPSCcolonies detection inone 6-wellplate at Day 21 after reprogramming induction. Red regions indicatethepositionofiPSCcolonies,which were detectedbybrightfeld imageclassificationalgorithm.Asmalarea is enlarged to show the details of thedetection.(c)Characterizationof iPSCs,which were manually picked guided bythe binary map. iPSC colonies $\scriptstyle ( \mathtt { n } = 3 )$ ） were picked into cel culture plate based on the binary map. After 4-7 days culturing, iPSCs were stained withantibodiesagainst known pluripotency surface markers (left).The nucleus was stained by DAPI(midle), merged images of two channel (right). Scale bar, $5 0 \mu \mathrm { m }$ (d) Selected iPSCs were analysed for Nanog,SOX2 and OCT4 expression by qRT-PCR.Human ES celline H1 was used as positive control. Error bars indicate $\pm \ : \mathrm { s . d }$ of triplicates. $P$ value is referenced to UCs. \*\* indicates $P < 0 . 0 1$ . (e) PCR detection of exogenous eipsomal DNA in UC-iPSC. UC(Passage 3) and H1(Passage 46)cellwere severedas the negativecontrol, UC-iPSC(Passage21) was stably expanded,UCtransiently transfected with eipsomal vector（pEP4EO2SET2K and pCEP4-miR-302-367 cluster) was served as the positive control.

This computer-guided selection was implemented after combining the generated binary image with the bright-field image, and the colony-pickingdecisions were made onlybasedonthecomputer visionresults.Subsequently,toverifythecomputerguidediPSCcolonydetection,manual picking wasperformed,followedbystandardiPScellcharacterization (Figures2C-D). Theexpandedcolonies thatpassdthecharacterzation,includingkaryotypingbyG-bandanalysis (datanotshown),werefurther characterized for pluripotency.Here,we show iPSCs generated from sample $\mathtt { c 4 p 2 }$ .For example,fluorescent immunostaining was initially performed to demonstrate the expresson of pluripotent surface markers,suchas TRA-1-6O,TRA-1-8l and SSEA4 (Figure 2C).Then,the endogenous pluripotent genes,such as OCT4,SOX2 and NANOG,were full activated,nd were comparable tohuman embryonic stem lineH1cells (Figure 2D).Finall,using genomicPCR,it was confirmed that the UC-iPSCs expanded at passge21andnolonger harbouredthe exogenous reprogramming factors fromthe original episomal plasmid (Figure 2E). Hence,this computer vision approach worked forboth humaniPSC detection as wellas inthe mouse model.

# Quantitative detection of synchronized time-lapse data for iPSC selection prediction in human somatic cellular reprogramming

The somaticcelular reprogramming was indicated bycell polarization changes in morphology.Based on experience,urinal cellreprogramming systems take more than 17 days to reach the matured and complete stage.The computer vision machine learning method allwed us to determine the stage of cellularreprogramming,and this was notlimited bythe experienced developed criteria.To monitor thecompletereprogramming inaquantitative manner,newsets oftime-lapse trainingdata were used to model iPSC formation and to classfy the phases in this process.

First,an individualcolony wasbacktracedas describedearlyon.Combined withthedetection method,theearliest feature for celllar polaritychanges,the formationof the iPSC texturecan berecognized atday7afterreprogramming induction. Anexampleofthedetected time-lapsedataisshowninFigure3A.Subsequently,the individual iPSCtexture features were detected using asegmentedboundary,andeach feature was registered andtagged digitally(Figure 3B).EachiPSCtexture feature was monitored individually during the entire reprogramming process.We can easily exclude the over-grown and under-grown colonies using the average growth rate.This gives aquantitative measurement for colony formation, whichis shown inthe growth phases,andeach curve line representsa qualified iPSCclone (Figure 3C).Moreover,a Hidden Markov Model (HMM) with four quantitative features was applied to analyse this process.This means that under the comparable reprogrammingconditions with thesamecels types,the model classifedfourdiferent phases inthe growthcurvetodescribe thecharacterzationoftheiSCcloneforming mathematicall.Finaly,thismodelgivesaposteriorprobabilityforeachphase; forexample,between days12and17,the probabilityscoreof maturation phase increased fromOto1.The model provides the correlationbetweenthe iPSCharvest ime and theprobabilityscore.The image dataforbest pickingdecision was selected manuallyandfedintothemodeltocalculatetheselectionthreshold.The probabilityscoreofO.3wastheoutputtodescribethe picking threshold in the urinalcellreprogramming system.For example,inthis dataset,thealgorithm gives theclosestscore to 0.3onday14,which means thatcolony picking could betriggered.Therefore,theoptimal picking time windowcan be predicted.Ifthe probabilityfactorreaches1,this meansthecellsareovergrown,whichimpliesariskofrandomdiferentiation.

In separateexperiments,iPSCcolonydetection was synchronized usingaclone-by-clone approach,andthecolony selection was triggeredasdescribedabove.Subsequently,thesecolonies wereverifiedusingasampleenriched forhigh-throughput RNA-Seq gene expresson analysis.The pluripotency- and germ layer-specific marker genes were plotted for comparison.

We analysed the global gene expresion patterns from the algorithm detected inurinal cell-erived iPSCs (AlgiPSCs), UCs and the embryonic stemcellH1.A hierarchicalclustering heat map (Figure 3d) showed that the gene expression inthe Alg_iPSCs is highlyindependentfromthe UCs.Comparisonoftheexpresion profiles betweenthe UCsandAlgiPSCsshowed that 1759 genes were up-regulated inthe Alg_iPSCs,while 1538 genes were up-regulated in the UCs (Figure 3e).

TheGene Ontology project providesacontrolledvocabulary todescribe thediferentially expressed genes.Theup-regulated genes inthe UCs wereenrichedinGOtermsrelatedtoepithelial,angiogenicorfbroblastfunction.Incontrast,theup-rgulated genes inthe AlgiPSCs were relatedto DNA replication and mitotic division.These GO terms also stronglydemonstrated the diferentiallyexpressed genes between the H1celsand UCs (Suppement data FigureS4).Further analysis revealedthat the pluripotency genes were more highly expressd inboth UCLiPSCs and H1cels compare to their low expression in the UCs. Thedatashowednodistinctive expressonbetweentheAlgiPSCsand ManiPSCsorbetween the Alg_iPSCsandtheHlcels. In contrast, the UC gene expression profile is clustered in a totally different manner.

![](images/d682c3ecaf532b106a7dd70c51cc369a260dce9495b63d1edacda3b382b14d53.jpg)  
Figure3.Quantitative detection in synchronized time-lapsedata foriPSCs selection prediction in human somatic celular reprogramming. (a) A series detection map for the entire reprogramming process from day7today 22. Binary indication masked out thecolonies by clasifer. (b)A segmented map. An example result of segmentation ona detection map,each segmentrepresents asinglecolony(left);eachcolonies was individuallyregisteredforfurtherproces(right). (c)Growthcurve of selectedcolony.Different colourindictedfordiferent phases,which was calculatedbythe mathematical model;the purple colour stands forthe mature phase.Solidline is anaverageofalltraining dataanddash lines are someexamplesof training data. (d)Hierarchic clustering analysis of global gene expressionbasedonPearson corelationforAlgiPSCs,H1(Cl-6)and Uri-C(1-2).(e)Diferential gene expressioprofilebetweenAlgiPSCs(bove)andUCs(below).Thedierentiallyexpresd genes (red) are those with an adjusted $\mathrm { \bf P }$ value O.O5 and fold change 3. (f) Functional annotations of genes differentially expresed between Alg_iPSCs (top)and UCs (bottom). Gene ontology(GO) was performed by DAVID,and enriched GO terms (biological processes) for each cell type are plotted with $- l o g _ { 1 0 }$ of the adjusted P values.

# Discussion

Recent developments incomputer vision techniques can provide advance tools for cellular morphology description and clasification43.Theapplicationofacomputervisionsystemprovidesquantitativeanalysis incelliology, whichcanreduce human error andimprove consistency14-20. Historicaly,digital pathology has primarily focusedonlow-levelimage feature extraction2-(e.g.olourandtensityuclearsgmenation,andticalmorpologeatures),followedbyeouctio of classification models using classical machine learning methods,including:Support Vector Machines24,25and Random Forests26. Manual design andthe selection offeatures is based on human observationsand experiences,which make it hard to achieve optimal performance. Moreover, normal classifiers are designed to suit relatively smallsets of training data.

Deplearning hasrecentlyatractedatentionin themachine learning field.The motivationis tousebigdatatodirectlytrain multi-layerneuralnetworks withdifferentdeepstructuresandcombine these networkswithfeature extractionandclasification. Convolution Neural Network (CNN) is a structure introduced by $\mathrm { L e C u n } ^ { 2 7 }$ that is widely used in computer vision area. Its most succesfulapplicationis dealing withimagedatainclassification,segmentationdetectionandretrieval task.Forexample,CNN is used as thecurrent baseline approach in breastcancerclasification anddiagnosis28. Chen etal29 present amulti-feature, label-freecellclassification systemusing deplearning techniques.Thebeautyof thedeeplearning technique is the end-toend learning procedure that combines feature extraction with the clasifer.The performance willbebenefited by expanding training data set.

Here, we present a machine learning-based system for the automated detection and prediction of iPSC formation using a cellular reprogramming process.This model works in both mice and human reprogramming systems,andthe HMM-based model was used for phase probabity estimationto triggeriPSCcolony selection.Our approach, which usesaDeepConvolution Neural Network (DCNN)end-to-end learing framework,can avoid the non-optimal manual design of extractors andclssifiers when faced with complicated celltextures and morphologychanges,which providesoptimized performance and convenience.

In the application point of view,this computer-aided machine learning approach for iPSCscan detect the cels arlies and predicttheoptimal selection time.This means that this toolcanquantitatively evaluate diferent somatic reprogramming approaches,suchas using engineredtranscriptional factororsmallmolecules,for iPSC generationor furter diferentiation. If the time-lapse data hasa high frequency framerate,thisautomated high-throughput analysis could potentiall group the different stages as individual conditions for cellular reprogramming for cellfate mapping modelling.

Finaly,the whole reprogramming processis serum-free,feeder-freeand uses epsomal based induction,this computer vision guided label freeandnon-invasive approach was fully verifiedby standard biological approaches,as well as RNA sequencing.We expect thesecombinedapproach willbecomeaneverydaytechniqueforcellbiologystudies inaquantitative way.It sould not limited incellar reprogramming works.This systemcanbedeveloped further includes to studydownstream celldiffereniation,dcellinedevelopment toientifyappropratecellsiafully traceableand quanitative wayreven guide an automated robotic in application of regenerative medicine.

# Methods

# Ethic Statement

Mus musculus (mice model)were maintainedandcared forinourExperimentalAnimal Centre'sfacility inacordance withthe Guangzhou Institutes of Biomedicine and Health Institutional Animal Care and Use Commitee (approval number N2017039) protocols.All experiments were performed in accordance withthe guidelines setbythe Human Subject Research Ethics Commitee at Guangzhou Institutesof Biomedicine and Health (GIBH)andthe Chinese Academyof Sciences (CAS),and the Commitee approved the experiments (approval number GIBH-IRB07-2017039).Formal informed consent was obtained from all the subjects.

# Overall Computer-Assisted System

TheiPSCcolonies in the bright-feld microscopy images normalyrepresent different morphology features fromothercels. However,itisdiffculttodetectsinglecels inthisimage,because thecelledgesandboundaryoverlapandarefuzzy.Wedid notadoptatraditionalsegmentationandrecognitionprocedure.Ourstrategywas toidentifyatypicalcolonytextureratherthan a single cellfeature.We used the sliding window method to detect the potential colony areas (Figure 4).

A $9 6 \times 9 6$ patch was defined as a fixed detection window,specifically a standard patch from the top left corner "slides” acrossthe whole testing image.Foreachof these windows,we tookthe window region and applied CNN todetermine whether thepatch is an iPSCarea ornot.Finally,abinary mapisobtained that indicates thelocations andsizes ofthe iPSCcolonies in the testing images.Every input image usedin our system has tobe nrmalized toreduce the influence from ilustration conditions.Because thebackground isdominatedbythebrightness oftheentire image,wecanasumethatoncethe brightness is normalized,the diferences betweenimages willbe minimized.Inourrecognitionsystem,the pre-processng process isa modified Autolevels (AL)algorithm(Supplemental data1.1)that normalizes brightnessand enhances the acquired images.I this study, CNN showed the best recognition rate compared to other methods (Supplemental data 1.2).

![](images/6d1480ae8c8b52bcb91386ae85859b768f0c5a0c3c29454ddbd4fdee1c41ad6f.jpg)  
Figure 4.The sliding window method was used todetect colonies from an unknown image.The smallareas were divided intc $9 6 \times 9 6$ pixel patches.

Afterthe binaryimage wascreated,the morphologytransformation was applied;this transformation filled the holes.We rana Gausian fiter and performed a re-binary method onthe image using acertain threshold to remove the single noise points. Utilization of the GFPreporteras the positive labels inthe mice data allowedus traineda good clasifier,which allowed us to directly identifycontoursand boundaries from thebinaryimage.Finaly,theimage was processedtoflterout any sparse residuals that counted as noise.

To locate human urinal cel-derived iPSCcolonies and their boundaries,the initial data colection was selected for day 12,which isaseed frame for segmentation.Forthedatabeforethis decidedtime point,apositive binary windowof points was counted as the area in the contours of certain colonies.In more matured cases,asemi-supervised Random Walker algorithm3 was implemented to expand thearea to complete the boundaries and used labels to mark thedetection. An example of segmentation on day 2O is presented in Figure 3B.

# Growth Curve Modelling

Aftertheboundaryofacolonywas found,eachcolonycanbe measuredin termsof theareaandtimeframeafterreprogramming induction.A growth curvecan be ploted based on the area of detected signals versus time after reprogramming nduction (Figure 3C). Abnormal growthconditions (overgrowth /undergrowth)were manuallydefinedbya stemcellspecialist after reading through the allthe acquired image files after the reprogramming process.Subsequently,te abnormal phase was discrimiated basedonthe meanof first-order diferences inthe growth curve between days 1Oand2O.Theremaining 97 normalsamples were usedto rain the HMMmodelfor predictionusing the following four features: (1)thefirstorderdifference in the growthrate,(2)the second orderdiference inthe growthrate,(3)the area,and (4)the numberof growthdays.Two stages are labeled to find thebestfitHMMmodel.The human experts labelledthe days forthe best picking time for every colony.The periodfrom onedayahead ofthe picking daytothe endis labelled the mature stage.The periodfromdayl to two daysafter the positive sample is shownislabelledfirst stage.Wecanusethe Baum-Welch algorithm to train the HMMmodel with2Odiferentinitializations.Thebestmodelcanfit thelabelleddataith4hiddenstages,andthemiddlepartisdivided into two periods ofsimilarlength.Finaly,theViterbi algorithm3canbeusedtopredictthe hidden stagesof growthcurves,of which an example is shown in Figure 3C.

# Image and Dataset Acquisition

The MEFcells werecultured and imaged via ZeissZ1 microscopy.The image is of acertain area and each wellwas captured every 2 hours.The human celsin each well were imaged with the Solentim Cell Metric once each day. These images were acquired ata 16-bit intensity depth witha MEF resolution of $3 3 2 2 \times 2 4 9 6$ for each region and the human cells were acquired at $1 7 7 0 2 \times 1 7 6 8 4$ for each well More than 5O images for both cel types were acquired from 3 different plates for training. Biologists manuall annotatedthe iPSCcolonies andcropped hundreds of iPSCareas and aconsiderable numberof negative samples fromtheacquired images.Notice thatthe cropped images had various widthsand heights ranging from 200 to400. In each selected area, a $9 6 \times 9 6$ pixel bounding box was randomly placed and cropped to a standard patch. A class label O is assigned to thefeaturevectorofeach iPSC,whileaclasslabel1isasigned toeach non-iPSC.Wefinallyobtained more than （204号 $2 \times 1 0 ^ { 6 }$ standard patches for the two celltypes, which included both iPSCs and non-iPSCs. A repeated random sub-sampling evaluation (Supplement data 1.2)on this dataset showed that the CNN achieved the best accuracy for our study.

# Neural Network Design and Iterative Training

We useda modified AlexNet32 structure as the base ofour neural network design. Because our colony detectionapplication is abinaryclasification problem,the numberof network sizes canbe shrunk to asmallerscale without a performance drop. Abatch-ormalized layer Wasadded beforeeachconvolution layer to accelerate the training speedand slightly improve the performance.To fine-tune the performance ofour network,biologists willcheck theautomatic recognitionresults fora newcapture imageand mark incorrectregions after the initial training.The incorrect data willad tothe trainingsets and be corrected with the right label.After more than 3 fine-tuning processes,the detector willachieve a nearly human level recognition rate for iPSC colonies.

# Mice embryonic fibroblast cell reprogramming

According to the previous OKS protocol33. Oct4-GFP transgenic MEFs were plated at $4 0 0 0 \mathrm { c e l l / c m } 2$ ,and then transfected with retrovirus(retroviral vectors carrying murine cDNAs for Oct4,Sox, Klf4 and c-Myc were purchased from Addgene.）using packed plat-EcellForty-eight hour post-transfection,the viral supernatants were removed,culture medium was added and time point was defined as day O. iPSC-inducing medium contained DMEM $\cdot +$ vitamin C, bFGF, CHIR99021 and other chemical components. Photos of cells in the well were taken bycell metrics or with a SteREO Lumar.V12 (Zeiss)every day.

# Human urinal cell culture and reprogramming

Urine cel (UC)colection,culture and reprogramming were performed as described in Xue et al3.with some modification. Briefly,the primary urine cells were cultured inurine cell medium consisting of an equal proportionofDMEM/F12 mixed with MEF medium containing $10 \%$ foetal bovine serum (FBS, PAA), $0 . 1 \mathrm { m M }$ NEAA (Gibco), $1 \mathrm { m M }$ L-glutamax (Gibco) and a SingleQuot Kit CC-4127 REGM (Lonza). The $1 { - } 2 \mathrm { ~ x ~ } 1 0 6$ urine cells were segregated with $0 . 2 5 \%$ trypsin treatment. After centrifugation, $6 { \mu \mathrm { g } }$ of the T2k vector and $4 \mu \mathrm { g }$ of the microRNA302 vector were co-transfected into the UCs using an AmaxaTM Basic NucleofectorTM Kit (Program,T-020,LONZA)according to the manufacturer's instructions.Transfected UCs were grown on matrigel-coated P6 wellsin UC medium. After 24 hours,the UC medium was replaced by TESR supplement with 4i (4i:CHIR99021 $( 3 ~ \mu \mathrm { M } )$ ,A83-01 $( 0 . 5 \mu \mathrm { M } )$ ,thiazovivin $( 0 . 5 \mu \mathrm { M } )$ ,and PD0325901 $( 1 ~ \mu \mathbf { M } ) _ { \downharpoon } ^ { \prime }$ ). After12 days,the cocktail of four small molecules wereremoved.Bright-fieldimages were takenbyCellMetricseverydayand were analysedbyourdetection system.The systemdetectedand indicated themature iPSCcolonies aroundthe25thday.Colonies were manuallymarkedat their corresponding location, picked,and plated on matrigel-coated 96-well plates containing mTesR medium.

# Immunocytochemical and Quantitative RT-PCR (qRT-PCR) analysis

Immunofluorescence was used tocharacterize the iPScells as previouslydescribed(Anti-TRA-1-60,Milipore;Anti-SEA4, LifeTechnologies).DAPI was used to stain thecell nuclei.Fluorescence-labelledcells were imagedonThe ImageXpres $\textsuperscript { \textregistered }$ Micro Confocal High-Content Imaging System.Total RNA was extracted using the RNeasy Mini kit (Qiagen Cat No.74104). First-strand cDNA was synthesized from $1 \mu \ g$ of total RNA using the GoScriptTM Reverse Transcription System (Promega Cat No.A5000)and qRT-PCR was performed ona CFX96 machine (BIO-RAD)and performed with three biological replicates by the using SYBRPCR Kit (SsoAdvancedTMUniversal SYBR@Green, Cat No.1725272).The CTmethod was used to calculate the relative gene expresion level.Theprimers usedfor genomic PCRand QPCRare listed insuppementaltable (Supplemental Table S2).

# RNA Sequencing

RNA sequencing was performed using a protocol in aprevious publication34. After recognizingand picking the iPS cloes,the iPS clone cells were lysed with $2 0 0 \mu \mathrm { l }$ of Trizol (Invitrogen). Total RNA was prepared with the Direct-zol RNA MiniPrep kit (Zymo Research). RNA was then quantified,purified andused to generate cDNA sequencinglibraries using the TruSeq RNA Sample Prep Kit(Ilumina).The Qubit dsDNA HS Assay kit (Invitrogen) was used to detectthecDNAlibrary concentrations. Sequencing was performed on a MiSeq system with MiSeq Reagent Kits v2 (50 cycles) (Ilumina).

# References

1.Wiley,L.A.etal.GMPproductionfpaientspecificiCsndpotoreceptrprecursorcels totreatretialegeative blindness. Sci Rep 6,30742 (2016).   
2.Zhao,T.etal.Humanized MiceRevealDiffrential ImmunogenicityofCells Derived from Autologous InducedPuripotent Stem Cells. Cell Stem Cell17,353-359 (015).   
3.Xue,Y.et al. Generatinganon-itegrating human induced pluripotent stemcellbank from urine-derivedcels.PLOSONE 8, e70573 (2013).   
4. Zhou,X.,Cao,X.,Perlman,Z.& Wong,S.T. A computerized celular imaging system for high content analysis in Monastrol suppressor screens. J Biomed Inform 39,115-125 (2006).   
5. Wang,M.,Zhou,X.,King,R.W.& Wong,S.T.Contextbased mixture modelfor cellphase identification inautomated fluorescence microscopy. BMC Bioinformatics 8, 32 (2007).   
6.Nattkemper,T.W.,Riter,H.J.&Schubert,W.Aneural classifier enabling high-throughput topologicalanalysisof lymphocytes in tissue sections. IEEE Trans Inf Technol Biomed 5,138-149 (2001).   
7.Nattkemper,T.W.,Twellmann,T.,Riter,H. & Schubert, W. Human vs machine: evaluationoffuorescence micrographs. Comput Biol Med 33,31-43 (2003).   
8.Butler,M.T.& Wallingford,J.B.Planarcell polarity in development and disease.Nat Rev Mol CellBiol (2017).   
9.Li,X.,Pei,D.& Zheng,H.Transitions between epithelialand mesenchymal states duringcellfateconversions.Protein& Cell 5, 580-591 (2014).   
10.Shu,X.&Pei,D.Thefunction and regulationof mesenchymal-to-epithelialtransition in somaticcellreprogramming. Current Opinion in Genetics & Development 28,32-37 (2014).   
11.Rejon, C.A.,Almasri,M.& Mccafrey,L.Cellpolarity proteins in breast cancer progression.Journalof Cellular Biochemistry 117,2215 (2016).   
12.Chen,J.etal. BMPs functionallreplace Klf4 andsupport eficientreprogramming of mouse fibroblasts byOct4 alone. Cell Res 21,205-212 (2011).   
13. Wang,L. & He,D.-C. Texture clasification using texture spectrum. Pattern Recognition 23,905-910(1990).   
14.Tokunaga,K.etal.Computationalimageanalysis ofcolonyand nuclear morphologytoevaluate humaninduced pluripotent stem cells. Sci Rep 4, 6996 (2014).   
15.Maddah,M.,Shoukat-umtaz,U.,Nasirpour,S.&Loewke,K.Asystem forautomated,nonnvasive,morphologybased evaluation of induced pluripotent stem cel cultures. J Lab Autom 19, 454-60 (2014).   
16.Maddah,M.&Loewke,K.Automated, non-invasive characterizationof stemcell-derived cardiomyocytes from phasecontrast microscopy. In medical image computing and computer asisted intervention, vol.17,57-64 (2014).   
17.Kato,R.etal.Parametric analysis ofcolony morphologyof non-labelledlive human pluripotent stemcels forcelquality control. Sci Rep 6, 34009 (2016).   
18.Joutsijoki, H.etal. Histogram-based clasification of ipsc colony images using machine learning methods.Insystems, man and cybernetics,2611-2617 (2015).   
19.Joutsijoki,H.,Haponen,M.,Rasku,J.,Aaltosetala,K.&Juhola,M.Error-correctingoutputcodes inlasicationof human induced pluripotent stem cel colony images. BioMed Research International 2016,3025057 (2016).   
20.Joutsijoki,H.,Haponen,M.,Rasku,J.,Aaltosetala,K.&Juhola,M. Machine Learning Approach tAutomatedQuality Identification of Human Induced Pluripotent Stem CellColony Images. Computational and Mathematical Methods in Medicine 2016,1-15 (2016).   
21.Liu,S., Mundra,P.A.& Rajapakse,J. C.Features for cells and nuclei clasification.In Proceedings of the Annual International Conference of the IEEE Engineering in Medicine and Biology Society EMBS,6601-6604 (IEEE,2011).   
22.Paci,M.etal.Non-Binary Coding forTexture Descriptors in Sub-Cellarand Stem Cel ImageClasification.Current Bioinformatics 8,208-219 (2013).   
23.MasudaA.etal.Automatic detectionof good/bad colonies ofips cels usinglocalfeatures.Ininternational conferenceon machine learning,153-160 (2015).   
24. Mammone,A., Turchi, M.& Cristianini, N. Support vector machines. Engineering 13,18-28 (2009).   
25.Han,J. W.,Breckon,T.P.,Randell,D.a.&andini, G.Theapicationofsupportvectormachineclasificationtodetect cell nuclei for automated microscopy. Machine Vision and Applications 23,15-24 (2010).   
26.Raytchev,B.etal.Detectionofdiferentiatedvs.undifferentiatedcoloniesofipscellsusingrandomforests modeled with the multivariate polyadistribution.InOurselin,S.,Joskowicz,L.,Sabuncu,M.R.,Unal,G.& Wels,W.(eds.)MCCAI 2016, 667-675   
27.Bengio,Y.&Lecun,Y.Convolutional networks forimages,speech,and time-series.Thehandbook ofbrain theoryand neural networks (1995).   
28.Wang,J.et al. Discrimination of breastcancer with microcalcifications on mammography by deep learning.Sci Rep 6,   
27327 (2016).   
29. Chen, C.L. et al. Deep learning in label-free cellclassification. Sci Rep 6,21471 (2016).   
30.Grady,L. Random walks for imagesegmentation.IEEE Trans.Pattern Anal. Mach. Intel.28,1768-1783 (2006).   
31. Viterbi,A.Errorbounds forconvolutionalcodesand an asymptoticallyoptimum decoding algorithm.IEEE Trans.Inf. Theory 13,260-269 (1967).   
32.Krizhevsky,A.,Sutskever,I.&Hinton,G.E.ImageNetclassificationwithdeepconvolutionalneural networks.Inneural information processing systems,1097-1105 (2012).   
33.Chen,J.etal.Towards anoptimizedculture medium forthe generationofmouse induced pluripotent stemcels.Journal of Biological Chemistry 285,31066-31072 (2010).   
34. Wang,L. et al. Tgf $\beta$ signaling regulates the choice between pluripotent and neural fates during reprogramming of human urine derived cells. Scientific Reports 6, 22484 (2016).

# Acknowledgements (not compulsory)

Supportfor this work was providedbythe MinistryofFinance "Life Instrumentation Development Program"(ZDYZ2012-3), theHundred Talents Programof the Chinese Academyof Sciences,the BureauofInternational Cooperationathe Chinese Academy of Sciences(154144KY-SB20130006),andthe Instrument Developing Projectof the Chinese Academyof Sciences, Grant No.Y74HO111K1. We thank Dr. Xiaoye Li for her advice on the manuscript.

# Author contributions statement

K.F.,S.Z.and X.Z. conceived the experiments.K.F.and M.H. designed and implemented the computational system.S.Z. performed thecellculture and validation experiments.K.F.,Y.Z.andJ.L.analysed theresults.K.F.,S.Z.andX.Z.wrote the manuscript. All the authors reviewed the manuscript.

# Additionalinformation

The authors declare no competing financial interests.