# ORIGINAL ARTICLE A long noncoding RNA critically regulates Bcr-Abl-mediated cellular transformation by acting as a competitive endogenous RNA

G Guo',Q Kang1,XZhu12,QChen²,XWang1,YChen',JOuyang',LZhang4,HTan5,RChen,SHuang7andJ-LChen13

Aberrant expresionoflong noncoding RNAs IncRNAs)isassociatedwithvarious humancancers.However,teroleof IncRNAs in Bcr-Abl-mediatedchronicmyeloidleukemia (CML)isunknown.InthisstudyweperformedacomprehensiveanalysisofIAs in human CMLcels usinganlIncRNAcDNA microarryand identifedan IncRNA termed IncRNA-BGL3 thatactedasa keyregulatorof Bcr-Abl-mediatedcellartransformation.Notably,weobservedthat IncRNA-BGL3was highlyinduced inresponse todisruptionof Bcr-Abl expressonor by inhbiting Bcr-Abl kinaseactivity in K562celsand leukemiccellsderivedfrom CMLpatients.Ectopic expression of IncRNA-BGL3sensitized leukemic cels toundergo apoptosis and inhibited Bcr-Abl-induced tumorigenesis. Furthermore,transgenic (TG)miceexpressng IncRNA-BGL3weregenerated.Wefound thatTGexpressionof lncRNA-BGL3alone in mice wassuficienttoimpairprimaryonemarrowtransformationbyBcr-Abl.IterestinglyweidentifedthatIncRNA-BL3 wasa targetof miR-17,miR-93,miR-20a,miR-20b,miR-106aandmR-106b,microRNAs thatrepress mRNAof phosphataseandtensin homolog (PTEN).Further experiments demonstratedthat IncRNA-BGL3functionedasacompetitive endogenous RNAforbinding these microRNAs tocros-rgulate PTENexpression.Aditionally，our experiments havebegun toaddress the mecanismof howIncRNA-BGL3 is regulated in the leukemic celsand showed that Bcr-Abl repressed IncRNA-BGL3 expression through c-Myc-dependentDNA methylation.Taken together,theseresultsrevealthatBcr-Abl-mediatedcelular transformationcriticaly requiresilenceoftumor-suppresorlncRNA-BGL3andsuggestapotentialstrategyforthetreatmentofBcr-Abl-positivelukemia.

Oncogene (2015) 34, 1768-1779; doi:10.1038/onc.2014.131; published online 19 May 2014

# INTRODUCTION

Chronic myeloid leukemia (CML) is a hematological malignancy associated with a translocation between chromosome 9 and 22 that results in the formation of bcr-ab/ hybrid gene which occurs in over $9 0 \%$ of CML cases.1 Bcr-Abl-induced tumorigenesis is a complicated process,involving the alteration of multiple signaling pathways that regulate cell survival and proliferation,including phosphatidylinositide 3-kinases/phosphatase and tensin homolog (PTEN)/AKT,RAS and Janus kinases/signal transducer and activator of transcription (STAT).2-4 Although progress has been made in the understanding of signal transduction in the Abl transformation,the role of IncRNAs in these processes remains unknown.

Human genome is pervasively transcribed,and the majority does not have apparent protein-coding potential,which is defined as noncoding RNA (ncRNA).5-7 LncRNAs are $> 2 0 0$ nucleotides in length，transcribed by RNA polymerase I orIll and poorly conserved." Numerous functional IncRNAs have been identified and recognized as critical regulators of various physiological processesviacontrollingmultiplelevelsofthegene expression.9,10LncRNAsfunctionbyvirtueofthemotifs embedded in their sequences that enable the specific association between IncRNAsand DNA,RNA or_protein.11,12 Importantly, aberrant expression of IncRNAs has been referred to as the hallmark of diverse human cancers.13-15 For instance,the IncRNA, HOX antisense intergenic RNA,is greatly upregulated in both primary and metastatic breast tumors,which is correlated with highcancermetastasisand poorprognosis.16,17 Inaddition, elevated expression of metastasis-associated lung adenocarcinoma transcript1was originallyidentified asa powerful predictor of high metastatic potential and_poor prognosis in non-small-cell lung adenocarcinoma patients.18 It has now been observed in a variety of human tumors,including those derived from breast, prostate，colon,liver and uterus.19-21 Recently， IncRNA-LET has also been implicated in various_tumor formations and may function as a tumor suppressor.14 Interestingly，a number of IncRNAs,such as T-UCRs (transcribed ultraconserved regions), HOTTIP (HOXA transcript at the distal tip)，ANRlL (antisense noncoding RNA in the INK4 locus), lincRNA-p21 (long intergenic noncoding RNA-p21）and MEG3 (maternally expressed 3),have been found to be associated with leukemia.2223 However， the molecular mechanisms underlying the involvement of IncRNAs in leukemia development are poorly understood.

Recently,a competitive endogenous RNA (ceRNA) hypothesis has been proposed.24 In this hypothesis, RNAs can crosstalk with eachother by competing for shared miRNAs. The tumorsuppressor PTEN has been validated to compete with several ceRNAs in diverse cancers.For example,VAPA (vesicle-associated membrane protein-associated protein A) and CNOT6L (CCR4-NOT transcription complex,subunit 6-like) were identified as ceRNAs for PTEN and displayed tumor-suppressive properties in prostate cancer.25 ZEB2 (zinc finger E-box-binding homeobox 2) has been revealed to actasa decoyof miRNAs targeting PTEN and thereby regulates PTEN levels in melanoma.26 Interestingly,a recent study has demonstrated that Linc-MD1 (long intergenic noncoding RNAmuscle differentiation 1） could control muscle differentiation by acting as a ceRNA.Linc-MD1 may sequester miR-133 and miR-135 toregulate their targets MAML1 (mastermind-like protein 1) and MEF2C (myocyte-specific enhancer factor 2C),transcription factors associated with the activation of muscle-specific genes.27 PTENP1 (PTEN pseudogene 1）was found to affect PTEN level by functioning as a decoy of PTEN-targeting miRNAs.28 These observations suggest that the ceRNA-based regulatory circuitry may represent a general phenomenon in physiological and pathological processes.29

![](images/499cd3f2bcb64ae269bf77a2c418e0051926136a2312a1873b1a7a2f58782dd4.jpg)  
Figur1Lcientidsfuctioaloeexprioiswelatdcr-b.(a)r-expoi6 celexpressingciferasesRA(s-uc)orBr-blsRA(s-Br-bl)asexaedywestblotig(a)ndquantitatieealtePCR (b). Results in panel (b) are presented as mean $\pm s . \mathsf { e } . \mathsf { m }$ ： $\left( n = 3 \right)$ 1 $^ { * } P < 0 . 0 5$ (c) Shown is hierarchical clustering analysis of IncRNAs differentially expressed in K562 cels expressing luciferase shRNA or Bcr-Abl shRNA (greater than twofold differences; $\textstyle P < 0 . 0 5$ ).Thevalues from three independent experimentsar micr oarrayanalysisrevealed 338upregulated IncRNAs and 1o8 downregulated IncRNAs after silencing Bcr-Abl. (d) Cell s gselecte ncRNAsore mpty vect orcontrol (EV) treatedwith imatinib. Cell viabilitywasassayed b pidiu iodide ottedare independent experiments.Error barsrepresent s.e.m $\begin{array} { r } { n = 3 , \ { } ^ { * } P < 0 . 0 5 . } \end{array}$ ofIncRNA-BGL3b the hemoglobin gamma A (HBG1) gene and the $( H \dot { B } B P \mathcal { I } )$ sh 5fHBG1 HBBP1 enesare indicated by blackboxesandblack A-BGL3isde tr ndicates thedirection of the transcription.(f)Northeri blo fIncRNA-BGL3 K562 cells.Egual amount of RNAs isolated from K562 cells expressing luciferase shRNA B r-Abl shRNAwere examinedwith $^ { 3 2 } \mathsf { P }$ -labeled IncRNA-BGL3-specific probe.

In this study,we found that numerous IncRNAs differentially expressed in K562 leukemic cells in response to silencing of Bcr-Abl.One of these，IncRNA-BGL3 (NCBl accession number KF110790,AY034471） was greatly upregulated by disruption of Bcr-Abl expression or inactivation of Abl kinase in CML cell line and primary CML samples.LncRNA-BGL3 acted as a tumor suppressor during Bcr-Abl-induced tumorigenesis.There existed a miRNA-dependent reciprocal regulation mode between IncRNABGL3 and PTEN.These observations demonstrate the functional involvement of IncRNA-BGL3 in Bcr-Abl-mediated leukemogenesisand provide novel insights into complicated mechanisms underlying Bcr-Abl-induced hematopoietic malignancies.

# RESULTS

Human IncRNA-BGL3 is identified as a functional IncRNA whose expression is downregulated by Bcr-Abl

In an attempt to provide insights into the mechanisms by which Bcr-Abl causes leukemogenesis and to identify IncRNAs involved in Bcr-Abl-mediated cellular transformation,microarray of cDNAs encoding IncRNAs was used to analyze differentially expressed IncRNAs in human K562 leukemic cells stably expressing specific short hairpin RNAs (shRNAs) targeting Bcr-Abl or luciferase control. The interference efficiency of these shRNAs in cellswas determined (Figures 1a and b).Following disruption of Bcr-Abl expression,all the IncRNAs whose expressions were altered by at least twofold were clustered and displayed (Figure 1c).The microarray analysis revealed 338 upregulated IncRNAs and 108 downregulated IncRNAs after silencing Bcr-Abl.

To identify functional IncRNAs that are important in Bcr-Ablmediated transformation，we selected the most significantly affected IncRNAs for further studies.These IncRNAs were examined by open reading frame finder from NCBl and no protein-coding potential was found.Furthermore, their expression Was confirmed in Bcr-Abl knockdown K562 cells (Supplementary Figure S1a).Additionally,imatinib,an Abl tyrosine kinase inhibitor, was used to determine the effect of Bcr-Abl kinase inactivation on their expression.Interestingly，similar expression patterns were observed in the cells treated with imatinib (Supplementary Figure S1a).Next, the functional involvement of these IncRNAs in Bcr-Abl-mediated transformation was investigated.We generated K562 cell lines stably overexpressing the selected IncRNAs, respectively，and analyzed effects of these IncRNAs on cell survival. K562 cells underwent apoptosis following imatinib treatment,and approximately $5 4 \%$ of the control cells remained viable after incubation with this inhibitor for $4 0 \mathsf { h }$ under our culture condition.In contrast,only approximately $3 5 \%$ of IncRNABGL3-expressing cells were viable under the same condition (Figure 1d).Importantly,overexpression of IncRNA-BGL3 had the most significant effect on K562 cell survival.Thus,it was pursued forstudies in detail.

![](images/0692737df46703a9a439a3c7340b1095c7bfea080cecd84d1bf070c11890b24f.jpg)  
Figure2cGsss562celtdegatois.(aalysisfexpeioin562ceesi diferntBrAspistatarg-usipnt(soepplemetargurea).cdaail timePCRnalysisfIcRBLexprsioninK62celsifecteditlntiviuencodingsr-bfortendatedtiRsiae (c) are presented as mean $\pm$ s.e.m. $( n = 3 )$ $^ { * } P < 0 . 0 5$ (d)RT-PCR analysis of IncRNA-BGL3 expression in primary CML cels from_patientsafter exposure to imatinib $( 1 0 \mu \mathrm { M } )$ for $2 4 \mathsf { h }$ (e,f) Quantitative real-time PCR was performed to examine the expression of IncRNA-BGL3 in stable K562cellsectopicallyoverexpressing IncRNA-BGL3(e)orexpressing IncRNA-BGL3shRNA(f).Resultsare presentedasmean $\pm s . \mathsf { e } . \mathsf { m }$ ： $( n = 3 )$ $^ { * } P < 0 . 0 5$ (g,h)Survialof562celstablyexpreinglR-G3,ncRseisoontrosaaalydbetry after treatment with imatinib. Results are presented as mean $\pm s . \mathsf { e . m }$ ： $( n = 3 )$ 1 $^ { * } P < 0 . 0 5$ ：(i)Western blotting and RT-PCR were performed to examine the protein and mRNA levels as indicated.

Previous studies indicated that BGL3 (Beta Globin Locus 3) was anoncoding RNA that may have a role in the regulation of Y-globin expression during development. BGL3 participates in Y-globin long-range contact with the locus control region and transcription activation.30,31 However，precise function of BGL3 remains largelyunknown.Previous data showed that BGL3 wasa transcript of 993 nt (GenBank AY034471).We found that BGL3 was included in NONCODE $\mathsf { v } 3 . 0 ,$ apublicly comprehensiveand systematic ncRNA database,² and noted that it is located between the hemoglobin gamma A (HBG1） gene and the hemoglobin beta pseudogene 1 (HBBP1） on chromosome 11; 5265784-5269453 (Figure 1e).Consistent with the prediction based on in silico analysis of genomic sequences，northern blotting analysis demonstrated that BGL3 was a transcript of approximately 3670 nt and was markedly upregulated by silencing the Bcr-Abl in K562 cells (Figure 1f).Thus,this novel transcript was cloned by reversetranscriptase-PCR(RT-PCR)， sequenced, renamed as IncRNA-BGL3 and submitted to GenBank database (GenBank KF110790). The open reading frames in the sequence were examined by open reading frame finder from ${ \mathsf { N C B l } } ^ { \dot { 3 } 3 }$ and demonstrated that IncRNA-BGL3 had poor Kozak strength and no protein-coding potential (Supplementary Figure S1b).We also determined the coding potential calculator score of IncRNA-BGL3 and performed a codon substitution frequency analysis using PhyloCSF.34,35LncRNA-BGL3 showed very low coding potential calculator and codon substitution frequency scores $( - 1 . 1 3 5 6 9$ and -2189.6o95,respectively),indicating that it is a non-coding RNA.

# LncRNA-BGL3 sensitizes Bcr-Abl-positive K562 leukemic cells to undergo apoptosis induced by imatinib

To further confirm that expression of IncRNA-BGL3 is regulated by Bcr-Abl,K562 cells were infected with different lentiviral shRNAs targeting Bcr-Abl or infected in a time course with the lentivirus. We found that silencing Bcr-Abl expression markedly increased the expression of IncRNA-BGL3 in K562 cells (Figures 2a-c, Supplementary Figure S2a). Similarly， treatment of thecells withimatinib led to increased expression of IncRNA-BGL3 (Supplementary Figures S2b-e).Moreover，four Bcr-Abl-positive CML samples derived from patients in chronic phases at diagnosis were utilized to determine the expression of IncRNA-BGL3. Strikingly,we observed that imatinib treatment greatly induced the expression of IncRNA-BGL3 in the leukemic cells from the CML patients but had no effect on normal control cells (Figure 2d).

Because expression of IncRNA-BGL3 is downregulated by Bcr-Abl kinase in both the leukemic cell line and primary CML cells,we hypothesized that IncRNA-BGL3 might have a key role inthe oncogenic transformation.To test this possibility，we conducted gain-of-function studies by generating stable K562 cell lines overexpressing IncRNA-BGL3 and designed shRNA targeting IncRNA-BGL3 to disrupt its expression in the cells (Figures 2e and f; Supplementary Figures S2f and g).Subsequently,the effect of altered IncRNA-BGL3 expression on survival of K562 cells was investigated.We observed that $5 7 . 7 4 \%$ of control K562 cells remained viable after treatment with imatinib for $4 0 \mathsf { h }$ under our culture condition.However，only $3 3 . 7 3 \%$ of the cells overexpressing IncRNA-BGL3 were viable under the same condition (Figure 2g)，although there were only slight decreases in the survival and proliferation of cells expressing IncRNA-BGL3 in the absence of imatinib (Supplementary Figures S2h and i).These data indicate that IncRNA-BGL3 sensitizes K562 cells to undergo apoptosis in response to the drug treatment. Experiments testing the effect ofsilencing IncRNA-BGL3 on cell survivalalso demonstrated that depletion of IncRNA-BGL3 significantly promoted survival of K562 cells in the presence of imatinib (Figure2h)．Similarly，IncRNA-BGL3 overexpressionalso sensitized v-Abl-transformed cells to undergo imatinib-induced apoptosis (Supplementary Figures S2j-l).These results suggest that IncRNA-BGL3 is critical for imatinib-induced apoptosis in Abl-transformed cells.

In an attempt to provide insights into the mechanism by which IncRNA-BGL3 regulates survival of Bcr-Abl-expressing cells,we evaluated the activation of STAT5 and expression of Bcl-xL thatare critical regulators of apoptosis in the leukemic cells.36 The results showed that overexpression of IncRNA-BGL3 significantly inhibited STAT5 phosphorylation and decreased the levels of Bcl-xL protein in the cells (Figure 2i).

Altering IncRNA-BGL3 expression significantly affects tumor formation induced by K562 leukemic cells in xenograft mouse model

Next,we asked whether Bcr-Abl-mediated repression of IncRNABGL3 expression is required for Bcr-Abl-induced tumorigensis in vivo.To this end,we injected nude mice subcutaneously with K562 cells stably overexpressing IncRNA-BGL3 or empty vector control.Tumor growth was examined every week after inoculation.Strikingly, tumors formed by control cells grew much faster than those formed by cells expressing IncRNA-BGL3,whereas no tumors were induced by Bcr-Abl-silencing K562 cells (Figure 3a; Supplementary Figures S3a-c).By statistical analysis,we found that tumor growth was significantly impeded by ectopic expression of IncRNA-BGL3 (Figures 3b and c). At least three independent experiments were performed to ensure the specificity and consistency of these results.These observations were further confirmed by bioluminescent imaging analysis (Figure 3d), and overexpression of IncRNA-BGL3 in the tumors was determined by quantitative real-time PCR (Figure 3e). In addition, nudemice were injected subcutaneously with K562 cells stably expressing shRNAs targeting IncRNA-BGL3 or luciferase control.As expected,depletion of IncRNA-BGL3 promoted K562 xenografted tumor growth in nude mice (Figures 3f-h).

To further confirm the effect of IncRNA-BGL3 expression on Bcr-Abl-mediated tumorigensis,we used another vector， the lentiviral vector pll3.7，to ectopically express IncRNA-BGL3 in K562 cells.Similarly，IncRNA-BGL3 was shown to significantly promote the cells to undergo imatinib-induced apoptosis and inhibit Bcr-Abl-induced tumor growth in vivo(Figures 3i-k; SupplementaryFiguresS3d-g).Theseresultsdemonstrated an inhibitory role of IncRNA-BGL3 in Bcr-Abl-induced tumorigenesis.

Expression of IncRNA-BGL3 in transgenic (TG) mice significantly inhibits Bcr-Abl-mediated primary bone marrow transformation Tofurther define the inhibitory role ofIncRNA-BGL3in Bcr-Abl-mediated cellular transformation,we were interested in establishing a more physiological model system to analyze the involvement of IncRNA-BGL3 in malignant transformation by BcrAbl oncogene.For this,we generated IncRNA-BGL3 expressing TG mice as previously described.4,37 The TG founders with high expression efficiency were selected (Figure 4a; Supplementary Figure S4).Primary bone marrow cells derived from IncRNA-BGL3- expressing TG mice or their wild-type (WT) littermateswere infected with the retrovirus encoding Bcr-Abl,and the abilityof the virus to transform bone marrow cells was measured by counting the number of Bcr-Abl-transformed cell clonesas previously described.4,38 As shown in Figure 4b,WT cells infected with the retrovirus displayed Bcr-Abl transformation,with an average of 12.5 wells showing growth of transformed cell clones per 96-well plate.However，under the same condition，Bcr-Abl transformation efficiency significantly decreased to 6.25 wells per 96-well plate when bone marrow cells derived from TG were infected with equal titer of the virus.

![](images/b562cb3e210771edacad004787f9da16640bfa37fc0b528a7eb284d0c3c4091a.jpg)  
Figure3AlteringlcRNA-BL3expressionsignifcantlyffectstumorfmationiducedbyK562eukemiccellixenogaftmosede (a)NudemiceweresubcutaneouslyinjectedwithK562celloverexpresingIncRNA-BGL3(ncRNA-BGL3)oremptyvector(EV).etumor volumesweremeasuredattheindicatedtimepoints.Plottedareresultsfromthreeindependentexperiments.Errorbarsrepresent s.e. $n = 1 0$ $^ { * } P < 0 . 0 5$ (b)Tumors were excised from mic Shown are ntative images from four independent experimentswith similar results.(c) Relative volum ressing IncRNA-BGL3 orcontrol.Theaverage volumeoftumorsinduc $n = 1 0 ,$ $^ { * } P < 0 . 0 5$ 28-day period after inoculation,tumors caused by IncRNA-BGl are representative images fromatleastthre umors formedbyK562cells overex $\pm$ s.e.m $\left( n = 3 \right)$ 1 $^ { * } P < 0 . 0 5$ ： $( \pmb { \mathscr { f } } )$ Nude IA-BGL3 shRNA (sh-IncRNABGL3). The nents.Errorbars represer $n = 1 0 ,$ $^ { * } P < 0 . 0 5$ vithsimilarresults. (h) Relative volu inducedbycontrol cellsis1.Erro $\displaystyle n = 1 0 , { \ast } P < 0 . 0 5$ K562cellsexpressing IncRNA-BGL3 $\pm s . e . \mathsf { m }$ $( n = 3 )$ $^ { * } P < 0 . 0 5$ utaneouslyinjected withK562 cells .Plotted are results fromthree independent e $n = 1 0$ $^ { * } P < 0 . 0 5$ orscausedby K562 cells overexpressing IncRNA-BGL3 or pll3.7. ay bars represents.e. $\begin{array} { r } { n = 1 0 , } \end{array}$ $^ { * } P < 0 . 0 5$

![](images/97b04e499802b47a002773d4e153206b25a4a751b58a14de585d8a0fb4426878.jpg)  
Figure4Expreioofc-inGmiceignantlinbitsB-bmeiatedprimarmaroB)trasformatona)A BGL3expressoninepresentativetssues(BM)fromIncNA-BGL-expressngTGmice(TG)andWlitermatesasexaminedbyR-CR. (b)BMcellfoWndTGmicereinfectdwithetroviusesexpressingBr-blansfoatiofncyaoredasscribedin Materials and Methods. Resultsare presented as mean $\pm s . \mathsf { e . m }$ $( n = 3 )$ $^ { * } P < 0 . 0 5$ (c,d) Lentiviral vectors expressing Bcr-Abl and either GFP or IncRNA-BGL3weecostructed(candexpreiofBcr-andA-BG3sexamedyestebotigadRCR(d.e)Bcls fromWTandTGmicewereinfectedwiththelentivirusesexpressingBr-AblandeitherGForIncRNA-BGL.ottedareresultsfromthree independent experiments.Error bars represent s.e.m., $n = 3 .$ $^ { * } P < 0 . 0 5$

To confirm this observation,we generated lentiviral constructs expressing Bcr-Abl and either GFP (green fluorescent protein) or IncRNA-BGL3 (Figures 4c and d). Indeed,we found that overexpression of IncRNA-BGL3 significantly reduced Bcr-Abl-mediated cellular transformation efficiency when bone marrow cells derived from WT mice were infected with the lentiviruses (Figure 4e). Similar results were obtained from experiments using the TG mice. Taken together, these results suggest that human IncRNA-BGL3 is functional in mice，and it has profound inhibitory effect on Bcr-Abl-mediated primary bone marrow transformation.

LncRNA-BGL3 is a target of miR-17,miR-20a,miR-20b,miR-93, miR-106a and miR-106b,microRNAs that repress messenger RNA of PTEN

To understand the mechanism by which IncRNA-BGL3 suppresses Bcr-Abl-mediated transformation,we used computational and experimental methodsto identify the functional sitesin IncRNA-BGL3 sequences as previously described.39 Surprisingly, bioinformatics prediction using FINDTAR3 (http://bio.sz.tsinghua. edu.cn/） for miRNA recognition sequences in IncRNA-BGL3 indicated the presence of a putative binding site for a set of miRNAs,including miR-17，miR-20a,miR-20b，miR-93，miR-106a and miR-106b,which share the same seed sequence (CGUGAAA) (Figure 5a).We determined the expression levels of these miRNAs in K562 cells.Consistent with previous studies,we observed that theywere all expressed in K562 leukemic cells (Supplementary Figure S5a).40 To validate that the predicted microRNA target site isfunctional, we generated luciferase reporter constructs (Figure 5b).WT IncRNA-BGL3 or IncRNA-BGL3 mutant devoid of the specific miRNA-binding site was cloned downstream of the Renilla luciferase gene and transfected into 293T cells together with specific miRNAs mimics or the negative control mimic. Our resultsshowed that luciferase expression was significantly reduced in cells transfected with IncRNA-BGL3 and specific miRNA mimics as compared with that in cells transfected with IncRNABGL3and negative control. However,luciferase expression in cells transfected with IncRNA-BGL3 mutant and the miRNA mimics was comparable to that of control cells (Figure 5c).These data suggest that IncRNA-BGL3 is a target of miR-17,miR-93,miR-20a,miR-20b, miR-106a and miR-106b.

Importantly,these miRNAs have been shown to repress the mRNA of PTEN.28 Thus，we compared the miRNA-dependent regulation of IncRNA-BGL3 with that of PTEN.Notably，highly similar miRNA-dependent regulation pattern was observed in cells expressing these targets with the miRNA mimics (Figure 5d).Then, the localization of IncRNA-BGL3 in K562 and human peripheral blood mononuclear cells was examined.41 Cell fractionation experiments showed that IncRNA-BGL3 was localized both in the cytoplasm and nucleus of these cells,and there existed significant number of IncRNA-BGL3 molecules in the cytoplasm (Figure 5e，Supplementary Figures S5b and S5c).We next performed RNA immunoprecipitation to detect IncRNA-BGL3 and PTEN bound to Ago2,an essential component of a functional RNA-induced silencing complex.42 We observed IncRNA-BGL3 and PTEN enrichment in the precipitate (Figure 5f).These results suggest that IncRNA-BGL3 and PTEN are both directly targeted by the shared miRNAs.

LncRNA-BGL3 regulates PTEN levels by functioning as a ceRNA Recently,it has been proposed that ceRNAs sequester miRNAs to regulate mRNA transcripts carrying common miRNA response elements (MREs).25,26 Our results presented above revealed that IncRNA-BGL3 and PTEN shared the same MRE in their sequences and displayed similar miRNA-dependent regulation pattern in

IncRNA-BGL3 5'-1446 ACCGUUUCAGAAUAUGCACUUUC 1468-3' 二 hsa-miR-106a 3'GAUGGACGUGACAUUCGUGAAAA 5' hsa-miR-17 ${ \boldsymbol { 3 } } ^ { \prime }$ GAUGGACGUGACAUUCGUGAAAC 5' hsa-miR-20a 3’GAUGGACGUGAUAUUCGUGAAAU 5' hsa-miR-20b 3'GAUGGACGUGAUACUCGUGAAAC 5' hsa-miR-93 ${ \mathfrak { z } } ^ { , }$ GAUGGACGUGCUUGUCGUGAAAC 5' hsa-miR-106b 3'UAGACGUGACAGUCGUGAAAU 5'

TEN 5'-2500 GGAUUAAUAAAGAUGGCACUUUC 2522-3'   

<html><body><table><tr><td>Promoter</td><td>Renilla Luciferase</td><td>IncRNA-BGL3 (1098-1729)</td><td>SV40 p(A)</td></tr><tr><td>Promoter</td><td>LuRenilase</td><td>(232-2N28)</td><td>SV40 p(A)</td></tr></table></body></html>

![](images/e20701bd2b14a0df1ff5e0f3c432f301569f17eb5cef531d638edaeda79018a5.jpg)  
Figure5.LncRNA-BL3isatargetofmiR-17mR-93,mR20amiR-2bmR106aandmR-106bmicroRNAsthatrepressRAofEN (a)NucleotidersolutionofiRNA-bindingsitesinncRN-BGL3andPEN.Asinicated,IRN-BL3containsaperfectseedence matchwithmiR-17,miR-93,miR-20a,miR-2Ob,miR-106aandmiR-106bthatsharethesameseedsequence.ThesemiRAsarevalidatedPN targetingmiRAs.(b)chematicrepresentatiooftheconstructsgeneratedforuciferaseasss.(c)LncRNA-BGL3W(ncRNA-wtits mutantdevoidofspecificmiRNA-bindingsites(ncRNA-BGL3-mu)inwhichseedmatchesofIncRNA-BGL3/PTEN-targetingmiRNAswere mutagenizedfrom'GCACT'toCTCATAwasclonddownstreamofRenillluciferaseene(Luc)inthevectorpRL-Tandtransfectedinto 293Tcelstogeterwitspecifcmsmiicsortheegativecontrolmimic(NC)uciferaseassyasperfordasdescribdinterials and Methods.Plotted are results from three independent experiments.Error bars represent s.e.m., $n = 3 ,$ $^ { * } P < 0 . 0 5$ .(d)LncRNA-BGL3 or PTEN-3UTRwascloneddownstreamofRenillluciferasegene,transfectedinto293TcellstogetherwithspecificmiRNAsmimicsor controlandexadyiferassibdiael().(6lrefractioatedtleardolaactio RT-PCRWasperformedtoexaminethelevelsofIncRNA-BGL3，nuclearcontroltranscript(U6)andcytoplasmiccontroltranscript (glyceraldehyde3-phosphatedehydrogenase(GAPH))inthesefractions.(f）Ago2wasimmunoprecipitated(IP)fromK562celsand co-precipitatedRNAsweredetectedbyquantitativereal-timePCRusingprimersforIncRNA-BGL3,PTENandGAPDHIPenrichmentis determinedbytheamountofRNAinprecipitateasociatedwithAgo2relativetothatinnormallgGontrolResultsarepresentedas mean $\pm s . \mathsf { e } . \mathsf { m }$ ： $\left( n = 3 \right)$ 一

a NC Mmix b □IncRNA-BGL3 C □EV d IncRNA-BGL3 1.5 OPTEN\* 3 OIncRNA-BGL3 Ev2L Actin 一 rli reelriieler 1 H 2 p-AKT RT-PCR 1.5 AKT PTEN 0.5 1 士 PTEN ： 0.5 Actin Actin 0 0+ WB NC Mmix WT MU   
e f EV control-1 g h □IncRNA-BGL3-wt 1EV2 InGE5sAu Frillleei leiieley 2 工 \* sh-ucGLA 士   
p-AKT p-AKT   
PAKT 1.5 T 0.6 AKT 0.5 0.2   
Actin Actin 0 0 PTEN Phosphoprotein AKT sh-luc h-nGeERAshBG   
： sh-luc i □No imatinib k 1 □No imatinib □sh-IncRNA- BGL3 120 □With imatinib 100 120 7 □Withimatinib 3 (%) iel nl 100 \* \* 80 \* \*   
25 251 80 (%)fliele l 80 60 T 60 60 40 40 \* 40 20 20 - PYEN 20 0 0 01 0 E 分 。 公 3GL3 O hr20 hr30 hr40 hr 公 neRNABGL3 IncRNA-BGL3 InRNAEGL q ncRN G In

luciferase reporter assay. These observations prompted us to examine whether there is a ceRNA-mediated regulation mode between IncRNA-BGL3 and PTEN.RT-PCR and western blotting were performed to determine endogenous levels of IncRNA-BGL3 and PTEN in K562 cells transfected with the mimics of these miRNAs. Indeed，we found that IncRNA-BGL3 exhibited concordant decreased expression pattern with PTEN in treated cells (Figures 6a and b).Furthermore，we generated stable K562 cell lines expressing PTEN 3'UTR $( 3 ^ { \prime }$ untranslated region）or specific shRNA that targets PTEN.Noticeably,overexpression or knockdown of PTEN correlated perfectly with the increased ordecreasedexpressionofIncRNA-BGL3，respectively (Supplementary Figures S5d and g).

Next,we investigated the ability of IncRNA-BGL3 to regulate the expression of endogenous PTEN in K562 cells.Our results showed that ectopic expression of WT IncRNA-BGL3 led to increased expression of PTEN,whereas expression of IncRNA-BGL3 mutant devoid of the MRE had no effect on PTEN level, indicating that the miRNA recognition sequences in IncRNA-BGL3 are required for IncRNA-BGL3-mediated regulation of PTEN expression (Figures 6c-f;Supplementary Figures S5h and i).By contrast,mRNA and protein levels of PTEN were markedly reduced in IncRNA-BGL3 knockdown cels (Figures 6g-i; Supplementary Figure S5j). Taken together, these data suggest that IncRNA-BGL3 and PTEN crossregulate each other probably through sequestration of shared miRNAs.

Furthermore,we examined the functional involvement of the miRNA recognition sequences of IncRNA-BGL3 in leukemic cell survival.K562 cells expressing IncRNA-BGL3 and its mutant devoid of the MRE were analyzed for cell survival.Indeed, the IncRNABGL3 mutant exhibited impaired ability to affect K562 cell apoptosis (Figure 6j,Supplementary Figure S6a).Furthermore, several IncRNA-BGL3 truncated mutants were constructed based onthe predicted secondary structure of IncRNA-BGL3 using RNAfold analysis (Supplementary Figure S6b). K562 cell lines stably overexpressing the mutants were generated and analyzed for cell survival (Supplementary Figures S6c and d).Interestingly,mutant (nt 1319-3670) containing the MRE had similar ability as IncRNABGL3 WT to sensitize K562 cells to undergo apoptosis，but remarkably,deletion of the region containing the MRE abolished this ability (Supplementary Figure S6e).These findings indicate that miRNA recognition sequences of IncRNA-BGL3 are essential for its function.However，expression of a smaller form (nt 1319-2713) containing the MRE was not sufficient to affect K562 cell survival (Supplementary Figures S6f and g), suggesting that expressing this part of IncRNA-BGL3 may alter its structure and thus affect its function.Additionally,our results showed that PTEN overexpression promoted imatinib-induced apoptosis of K562 cells,and depletion of PTEN significantly attenuated IncRNA-BGL3- mediated pro-apoptotic effect on K562 cells (Figures 6k and l; Supplementary Figures S6h and i),suggesting that IncRNA-BGL3 regulates cell survival,at least in part, through upregulation of PTEN levels in a miRNA-dependent manner.

# Bcr-Abl represses IncRNA-BGL3 expression through c-Myc-dependent DNA methylation

Because disruption of Bcr-Abl expression or inhibition of Abl kinase led to a significant increase in IncRNA-BGL3 level,we asked next how IncRNA-BGL3 is downregulated by Bcr-Abl in CML cells. It is well recognized that epigenetic silence of tumor suppressors isone of key mechanisms that cause tumorigenesis，and upregulated expression of DNA methyltransferases DNMT1，3A and 3B has been described in CML.43 Thus,we explored the role of DNA methylation in regulating the expression of IncRNA-BGL3 that functioned as a tumor suppressor in Bcr-Abl transformation. To this end,we evaluated the effect of 5-Aza-2-deoxycytidine (5-Aza-dc),a specific DNA methyltransferase inhibitor,on IncRNABGL3 expression.Of interest,we observed that treatment with 5-Aza-dc significantly increased the level of IncRNA-BGL3 in K562 cells (Figures 7a and b).Furthermore,5-Aza-dc treatment also resulted in upregulation of IncRNA-BGL3 expression in some human cancer cells,including acute T-cell leukemia cell line Jurkat and hepatocellular carcinoma cell lines 7402 and HepG2 but not in othercell lines, including 293T,hepatocellular carcinoma cell lines 7703and Huh7，as well asbreast adenocarcinoma MCF7 (Figure 7c; Supplementary Figure S7a),suggesting that expression and methylation of IncRNA-BGL3 may be cell line dependent.

![](images/c0ab587b31fd22cafce578c4b8fc6c62b8cbe719d10b0fe57065ca042be59580.jpg)  
Figure 7.Bcr-Abl repressesIncRNA-BGL3 expression through $\mathsf { c }$ -Myc-dependent DNA methylation.(a,b) RT-PCR and real-time PCR were performed to examine IncRNA-BGL3 expression in K562 cells treated with 5-Aza-dc $( 1 0 \mu \mathrm { M } )$ for $7 2 \mathsf { h }$ .Resultsare presented as mean $\pm s . e . \mathsf { m }$ ： $( n = 3 )$ $^ { * } P < 0 . 0 5$ . (c)RT-PCR analysis of IncRNA-BGL3 expression in 293T,Jurkat,7703,7402, HepG2,Huh7 and MCF7 cells treated with 5-Aza-dc.(d,e) Expression of c-Myc and IncRNA-BGL3 in Bcr-Abl knockdown or control K562 cells (d) or in cells treated with orwithout imatinib (e)was examined.Shown are data from realtime PCR analysis.Results are presented as mean $\pm$ s.e.m. $( n = 3 )$ _ $^ { * } P < 0 . 0 5$ ：(f）The levels of IncRNA-BGL3 and DNMT3B in c-Myc knockdown or control K562 cells were examined by real-time PCR.Results are_presented as mean $\pm s . e . \mathsf { m }$ ： $( n = 3 )$ 、 $^ { * } P < 0 . 0 5$ (g)Expression of IncRNA-BGL3 and DNMT3B in control or c-Mycoverexpressing K562 cells treated with imatinib alone,or imatinib and 5-Aza-dc,was examined by real-time PCR.Results are presented as mean $\pm \ : s . e . \mathsf { m }$ ： $( n = 3 )$ 1 $^ { * } P < 0 . 0 5$

It is well known that $\mathsf { c }$ -Myc is essential for transformation by Abl oncogenes，and c-Myc and Bcr-Abl cooperate to endow the transformed cells with cytokine-independent growth.44,45 In addition,c-Myc contributes to enhanced expression of DNMT3A and 3B in different cancers.46,47 Thus,we tested whether Bcr-Abl repressed IncRNA-BGL3 expression through c-Myc-dependent pathway.As expected, loss of Bcr-Abl expression or Abl kinase activity caused decline of c-Myc levels (Figures 7d ande; Supplementary Figures S7b and c).Importantly,silencing c-Myc expression alone in K562 cells remarkably increased the level of

IncRNA-BGL3 with simultaneously reduced level of DNMT3B (Figure 7f;Supplementary Figure S7d). In contrast， ectopic expression of c-Myc abrogated the imatinib-induced upregulation of IncRNA-BGL3 (Figure 7g; Supplementary Figure S7e). Notably, the addition of 5-Aza-dc could rescue the c-Myc-blocked IncRNABGL3upregulation in the presence of imatinib (Figure $\mathsf { 7 9 } ;$ Supplementary Figure S7e).Together, these data suggest that Bcr-Abl represses IncRNA-BGL3 expression probably through c-Myc-dependent DNA methylation.

# DISCUSSION

It iswell documented that malignant transformation of myeloid and lymphoid cells by Bcr-Abl involves the dysregulation or mutation of a variety of protein-coding genes that are normally involved in regulating the proliferation and survival of hematopoieticcels.2,48,49However，the functional relevanceof IncRNAs and their in vivo regulation during Bcr-Abl-mediated tumorigenesis remain enigmatic.In this study,we identified that IncRNABGL3 was markedly upregulated by altering Bcr-Abl expression or its kinase activity in leukemic cells.We revealed that IncRNA-BGL3 overexpression sensitized leukemic cells to undergo apoptosis and inhibited Bcr-Abl-induced tumorigenesis in vivo.LncRNA-BGL3 TG mice exhibited impaired Bcr-Abl-mediated primary bone marrow transformation.These data provide strong evidence that IncRNABGL3 acts as a tumor suppressor during the Bcr-Abl-induced leukemogenesis. Importantly，upregulation of IncRNA-BGL3by imatinib treatment was also detected in several primary CML patient samples,which express Bcr-Abl.We understand that our CML sample size is limited.Another large-scale study could increase the statistical power of our results obtained from CML samples.These remain an ongoing task.

Recently, growing evidence has demonstrated the critical roles ofIncRNAs_in_various physiological and pathophysiological processes.127Forexample,sayctdge decoys for miRNAs to affect their distribution on specific targets. Here we show that IncRNA-BGL3 is a target of miR-17 family, which has also been found to repress mRNA of PTEN.28 Because PTEN and its downstream AKT signaling have critical roles in Abl transformation,50 we determined whether IncRNA-BGL3 might regulate Bcr-Abl-mediated transformation by competing for miRNAs targeting PTEN. Indeed，we found that altering the expression of IncRNA-BGL3 but not its mutant devoid of miRNAbinding sites had profound effect on the level of PTEN in K562 cells,indicating that the miRNA recognition sequences of IncRNABGL3are required for IncRNA-BGL3-mediated regulation of PTEN. Interestingly,we observed that altering IncRNA-BGL3 expression had also effects on the levels of CNOT6L，VAPA and CDKN1A, three known，targets of the miR-17 family (Supplementary Figure S8),25.51uggestingthatIcRNA-BGL3ouldaectole networks of multiple targets.This miRNA-dependent reciprocal regulation mode shed a new light on the mechanisms underlying Bcr-Abl-induced tumorigenesis.On the other hand,our results suggest that IncRNA-BGL3 may modulate STAT5-dependent expression of anti-apoptotic Bcl-xL protein.A previous study has provided _a direct link between the PTEN-AKT and STAT5 pathways.52 However, further work is required to address the relationship of IncRNA-BGL3,PTEN-AKT and STAT5 pathway.

Recent studies revealed the existence of a previously unrecognized gene regulatory layer characterized by ceRNAs that compete for miRNA binding to cross-regulate the miRNA targets. CeRNAsbased gene regulation may form robust networks that when perturbed may lead to cancer.29 Remarkably,it has been shown that regulation of the tumor-suppressor PTEN by ceRNA activity frequently occurs _in several cancers,such as melanoma and prostate cancer.25,26 The present report demonstrates for the first time that IncRNA-BGL3 functions as a ceRNA to suppress Bcr-Abl-positive leukemic growth through regulating PTEN and its downstream AKT signaling,and silencing its expression appears to downregulate PTEN level, tipping cells towards cancer progression.These results suggest a potential strategy for treatment of Bcr-Abl-induced malignancies.

Although BGL3 has been previously described,30,31 little is known about its in vivo regulation.Here,we show that IncRNABGL3 is downregulated in the cells by DNA methylation.We observed that expression of c-Myc in K562 cells was Bcr-Abl dependent,and silencing c-Myc expression alone markedly increased the level of IncRNA-BGL3，with reduced levels of DNMT3B.Ectopic expression of c-Myc abrogated the imatinibinduced expression of IncRNA-BGL3,whereas 5-Aza-dc reversed the inhibitory effect of c-Myc on IncRNA-BGL3 upregulation by imatinib.These observations indicate that Bcr-Abl represses IncRNA-BGL3 expression likely through c-Myc-dependent DNA methylation.Together,the results reveal a mechanism of how Bcr-Abl may overcome IncRNA-BGL3 suppression to constitutively activate the AKT-dependent signaling.However，it is unclear whether regulation of IncRNA-BGL3 is important in physiological (non-CML) setting.There has been a report of an enhancer-like function for IncRNA in human cels.5³ Given the role of the BGL3 in regulating Y-globin transcription activation,it is interesting to explore the possible role of IncRNA-BGL3 as an enhancer IncRNA. Further studies are needed to address these issues and to determine the therapeutic significance of IncRNA-BGL3in leukemia.

# MATERIALSANDMETHODS

Microarray and data analysis

The IncRNA cDNA microarray Was from Arraystar (Arraystar, Rockville, MD, USA).Total RNAs from three independent groups of K562 cells expressing luciferase shRNA or Bcr-Abl shRNA were prepared using Trizol reagent (Invitrogen, Carlsbad,CA, USA).cDNA synthesis, labeling,hybridization and data analysis were carried out as previously described.54 Our microarray data have been deposited in the NCBl Gene Expression Omnibus (www. ncbi.nlm.nih.gov/geo/;access number GSE42269).

# Cell lines and cell culture

Cell lines K562,293T and Jurkat were purchased from American Type Culture Collection (Manassas,VA, USA). Hepatocellular carcinoma cell lines QGY-7703,BEL-7402,HepG2 and Huh7 and breast carcinoma cell MCF-7 were purchased from National Platform of Experimental Cell Resources for Sci-Tech (http://cellresource.cn,Beijing，China). Cells were grown in Dulbecco's modified Eagle medium or RPMl1640 supplemented with $1 0 \%$ fetal bovine serum and antibiotics (penicilin and streptomycin) as previously described.4 LncRNA-BGL3 overexpressing K562 cellswere generated by infecting the cells with viruses encoding IncRNA-BGL3 using the viral vector pMSCV-GFP or pll3.7 as previously described.4 shRNA-expressing stable K562 cells were generated by infection of the cells with lentiviruses expressing specific shRNAs as described previously.4 The shRNA sequences are described in Supplementary Materials and Methods.

# Apoptosis assay

Cellswere treated with imatinib $( 1 0 \mu \mathrm { m } )$ for the indicated time, then stained with propidium iodode (Pl)/Annexin Vor Pl and analyzed by fluorescenceactivated cell sorter (BD Biosciences,San Jose, CA,USA).

# Nude mice injection

Nude-mouse injection was performed as previously described.4,38 Tumor growth was monitored and measured in volume (length $\times$ height $\times$ width) at the indicated time after inoculation.Bioluminescent imaging was used to probe tumor growth from GFP-expressing cells.Images were quantified as photons/s using the Indigo software (Berthold Technologies, Bad Wildbad, Germany).

# Luciferase assay

LncRNA-BGL3 (IncRNA-BGL3-wt),its mutant devoid of specific miRNAbinding sites (IncRNA-BGL3-mu) or PTEN-3'UTR was cloned into $3 ^ { \prime } \mathsf { U T R }$ of the Renilla luciferase gene in the vector pRL-TK (Promega,Madison,Wl, USA).Each plasmid was transfected into cells,together with specific miRNAs mimics or with a negative control mimic (RiboBio,Guangzhou, China).Firefly luciferase gene in the vector pGL3-control (Promega) was used as a control for transfection efficiency.Luciferase assays were performed using the dual-luciferase reporter assay system kit (Promega) according to the manufacturer's instructions.Luciferase expression was analyzed by Modulus single-tube multimode reader (Promega).The relative luciferase expression equals the expression of Renilla luciferase (pRL-TK) divided by the expression of firefly luciferase.

# Generation of IncRNA-BGL3-expressing TG mice

The IncRNA-BGL3-expressing TG mice were generated by the microinjectionmethod aspreviously described.437 RT-PCR wasperformed to determine the expression efficiency.The TG founders with high expression efficiency were selected and maintained on a C57BL/6J genetic background.

Primary murine bone marrow transformation assay

Bcr-Abl-mediated bone marrow transformation was carried outas previously described.3,55 Transformation efficiency was scored by counting the number of the wells that displayed cytokine-independent growth 3 weeks postinfection.

Antibodies and reagents,DNA construction,cellular fractionation, RT-PCR, quantitative real-time PCR,western blotting, RNA immunoprecipitation and statistical analysis

Antibodies and reagents,DNA construction,cellular fractionation,RT-PCR, quantitative real-time PCR,western blotting, RNA immunoprecipitation and statistical analysis are described in the Supplementary Materialsand Methods.

# ABBREVIATIONS

IncRNA-BGL3, long noncoding RNA-Beta Globin Locus 3; PTEN, phosphatase and tensin homolog; miRNA,microRNA.

# REFERENCES

1 Chen Y,Peng C, Sullivan C, Li D,Li S. Critical molecular pathways in cancer stem cells of chronic myeloid leukemia. Leukemia 2010;24:1545-1554.   
2 Cilloni D,Saglio G.Molecular pathways:BCR-ABL.Clin Cancer Res 2012;18:   
930-937.   
3 Guo G,Qiu X,Wang S,Chen Y,Rothman PB,Wang Z et al.Oncogenic E17K mutation in the pleckstrin homology domain of AKT1 promotes v-Abl-mediated pre-B-cell transformation and survival of Pim-deficient cells. Oncogene 2010;29:   
3845-3853.   
4 Yang J,Wang J,Chen K,Guo G, Xi R, Rothman PB et al.elF4B phosphorylation by pim kinases plays a critical role in cellular transformation by Abl oncogenes. Cancer Res 2013;73:4898-4908.   
5Kapranov P,Cheng J,Dike S,Nix DA, Duttagupta R,Willingham AT et al. RNA maps reveal new RNA classes and a possible function for pervasive transcription. Science 2007;316:1484-1488.   
6 GuttmanM,GarberM,LevinJZ,DonagheyJ,RobinsonJ,diconis Xetal.Abiti reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs.Nat Biotechnol 2010;28:756-756   
7 Guttman M,Amit I, Garber M,French $\mathsf { C } ,$ Lin MF,Feldser D et al.Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals. Nature 2009;458: 223-227.   
8 Ponting CP,Oliver PL, Reik W. Evolution and functions of long noncoding RNAs. Cell 2009; 136: 629-641.   
9 Klattenhoff CA,Scheuermann JC,Surface LE, BradleyRK,Fields PA,Steinhauser M etal. Braveheart,a long noncoding RNA required forcardiovascular lineage commitment. Cell 2013;152: 570-583.   
10 Sigova AA，Mullen AC,Molinie B,Gupta S,Orlando DA,Guenther MG et al. Divergent transcription of long noncoding RNA/mRNA gene pairs in embryonic stem cells.Proc Natl Acad Sci USA 2013;110: 2876-2881.   
11 Wang KC,Chang HY. Molecular mechanisms of long noncoding RNAs.Mol Cell 2011; 43: 904-914.   
12 Yoon JH,AbdelmohsenK,SrikantanS,Yang X,MartindaleJL,DeSetal.LinRNAp21 suppresses target mRNA translation.Mol Cel 2012; 47: 648-655.   
13 Prensner JR,lyer MK,Balbin OA,Dhanasekaran SM, Cao $\scriptstyle \mathbf { Q } ,$ Brenner JC et al. Transcriptome sequencing across a prostate cancer cohort identifies PCAT-1,an unannotated lincRNA implicated in disease progresson. Nat Biotechnol 2011;29: 742-749.   
14 Yang F,HuoXS,Yuan SX,ZhangL,ZouWP,WangFetal.Repressionofthelong noncoding RNA-LET by histone deacetylase 3 contributes to hypoxia-mediated metastasis. Mol Cell2013; 49:1083-1096.   
15 Tsai MC,Spitale RC,Chang HY.Long intergenic noncoding RNAs: new links in cancer progression. Cancer Res 2011; 71: 3-7.   
16 GuptaRA,ShahN,Wang KC,KimJ,HorlingsHM,WongDJetal.Longnonoding RNA HOTAlR reprograms chromatin state to promote cancer metastasis.Nature 2010; 464: 1071-1076.   
17Tsai MC,Manor O,Wan Y,Mosammaparast N,Wang JK,Lan Fet al.Long noncoding RNA as modular scaffold of histone modification complexes.Science 2010; 329: 689-693.   
18JiP,Diederichs S,Wang W,Boing S,Metzger R,Schneider PM et al. MALAT-1, a novel noncoding RNA,and thymosin beta4 predict metastasis and survival in early-stage non-small cell lung cancer. Oncogene 2003;22: 8031-8041.   
19TripathiV,ElisJD,ShenZ,SongDY,PanQ,WattATetal.henuclear-etand noncoding RNA MALAT1 regulates alternative splicing by modulating SR splicing factor phosphorylation. Mol Cell 2010; 39: 925-938.   
20 Guffanti A, lacono M,Pelucchi P, Kim N, Solda G,Croft LJ etal.A transcriptional sketch of a primary human breast cancer by 454 deep sequencing.BMC Genomics 2009; 10: 163.   
21Luo JH, Ren B, Keryanov S,Tseng GC, Rao UN, Monga SP et al.Transcriptomic and genomic analysis of human hepatocellular carcinomas and hepatoblastomas. Hepatology 2006; 44:1012-1024.   
22 Heuston EF,Lemon KT,Arceci RJ.The beginnngof the road for non-coding RNAs in normal hematopoiesis and hematologic malignancies.Front Genet 2011; 2: 94.   
23 Wang KC,YangYW,LiuB,SanyalA,Corces-Zimmerman RChenYetal.Long noncoding RNA programs active chromatin domain to coordinate homeotic gene expression.J Invest Dermatol 2011;131: S63-S63.   
24SalmenaLPolisenoLayYKats,PandolfiPP.AceRNAhypothesis:theoa stone of ahidden RNA language? Cell2011;146:353-358.   
25 Tay Y, Kats L,Salmena L,Weiss D,Tan SM,Ala U et al. Coding-independent regulation of the tumor suppressor PTEN by competing endogenous mRNAs. Cell 2011; 147: 344-357.   
26 Karreth FA,Tay Y,Perna D,Ala U,Tan SM,Rust AG etal.In vivo identification of tumor- suppressive PTEN ceRNAs in an oncogenic BRAF-induced mouse model of melanoma. Cell 2011; 147: 382-395.   
27Cesanaceiitiieio noncoding RNA controlsmuscle diferentiation by functioning asacompeting endogenous RNA. Cell 2011; 147: 358-369.   
28PolisenoLmenaLZangJCaerB，avemanWPandol.ding independent function of gene and pseudogene mRNAsregulates tumour biology. Nature 2010; 465: 1033-1038.   
29deGiogoA,KrellJ,rdingVtebbingJstellanoLEergingrolof incancer:insightsfromtheregulationofPTN.MolCellBol2013;33:3976-3982.   
30 Kiefer CM,Lee J, Hou C, Dale RK,LeeYT,Meier ER etal. Distinct Ldb1/NLI complexes orchestrate gamma-globin repression and reactivation through ETO2 in human adult erythroid cells. Blood 2011; 118: 6200-6208.   
31 Chan CM,Fulton J,Montiel-Duarte C, Colins HM,Bharti N,Wadelin FR et al. A signature motif mediating selective interactions of BCL11A with the NR2E/F subfamily of orphan nuclear receptors.Nucleic Acids Res 2013;41: 9663-9679.   
32 Bu D,YuK,Sun S, XieC,Skogerbo G,MiaoRetal.NONCODEv3.0:integrative annotation of long noncoding RNAs. Nucleic Acids Res 2012; 40: D210-215.

# CONFLICTOFINTEREST

The authors declare no conflict of interest.

# ACKNOWLEDGEMENTS

We thank Dr Paul B Rothman from the School of Medicine,The Johns Hopkins University for valuable discussions.This work was supported by the National Key Technologies Research and Development Program of China (2013ZX10004-611), National Basic Research Program (973) of China (2014CB541804), Intramural grant of the Chinese Academy of Sciences (KJZD-EW-LO1-3),Natural Science Foundation of China (81171943,U1305212) and Hundreds of Talents Program of Chinese Academy of Sciences 2009-2014 to J-LC.

33 Nishikawa T,Ota T, Isogai T.Prediction whether a human cDNA sequence contains initiation codon by combining statistical information and similarity with protein sequences. Bioinformatics 2000;16:960-967.   
34 Kong L, Zhang Y,Ye ZQ,Liu XQ, Zhao SQ,Wei L et al.CPC:assess the proteincoding potential of transcripts using sequence features and support vector machine.Nucleic Acids Res 2007;35:W345-W349.   
35 Lin MF,Jungreis I，Kellis M. PhyloCSF:a comparative genomics method to distinguish protein coding and non-coding regions.Bioinformatics 2011；27: 1275-1282.   
36 Gesbert F，Griffn JD.Bcr/Abl activates transcription of the Bcl-X gene through STAT5. Blood 2000; 96:2269-2276.   
37 Lu D,Ma Y,Zhang W,Bao D,Dong W, Lian H et al. Knockdown of cytochrome P450 2E1 inhibits oxidative stress and apoptosis in the cTnT(R141W) dilated cardiomyopathy transgenic mice. Hypertension 2012;60:81-89.   
38 Qiu X,Guo G,Chen K, Kashiwada M, Druker BJ,Rothman PB et al.A requirement forSOCS-1 and SOCS-3 phosphorylation in Bcr-Abl-induced tumorigenesis. Neoplasia 2012;14: 547-558.   
39 Chen Y,Chen J,Wang H,Shi J,Wu K, Liu S et al. HCV-induced miR-21 contributes to evasion of host immune system by targeting MyD88 and IRAK1.PLoS Pathog 2013;9:e1003248.   
40 Venturini L, Battmer K,Castoldi M, Schultheis B, Hochhaus A,Muckenthaler MU et al. Expression of the miR-17-92 polycistron in chronic myeloid leukemia (CML) CD34(+) cells. Blo0d 2007;109:4399-4405.   
41 Prasanth KV,Prasanth SG,Xuan Z,Hearn S,Freier SM,Bennett CF et al. Regulating gene expression through RNA nuclear retention. Cell 20o5;123: 249-263.   
42 Ma F,Xu S, Liu X,Zhang Q,XuX, Liu M et al.The microRNA miR-29 controls innate and adaptive immune responses to intracellular bacterial infection by targeting interferon-gamma.Nat Immunol 2011;12:861-869.   
43 Mizuno S,Chijiwa T,Okamura T,Akashi K,Fukumaki Y，Niho Y et al. Expression of DNA methyltransferases DNMT1,3A,and 3B in normal hematopoiesis and in acute and chronic myelogenous leukemia. Blood 2001;97: 1172-1179.   
44 Sawyers CL, Callahan W,Witte ON.Dominant negative Myc blocks transformation by Abl oncogenes. Cell 1992; 70: 901-910.   
45Xie S, Lin H, Sun T,Arlinghaus RB.Jak2 is involved in c-Myc induction by Bcr-Abl. Oncogene 2002;21:7137.   
46Mott JL,Kurita S,Cazanave SC,Bronk SF,Werneburg NW,Fernandez-Zapico ME. Transcriptional suppression of mir-29b-1/mir-29a promoter by c-Myc, hedgehog, and NF-kappaB. J Cell Biochem 2010;110:1155-1164.   
47 Mishra A,Liu S,Sams GH,Curphey DP,Santhanam R,Rush L et al.Aberrant overexpression of IL-15 initiates large granular lymphocyte leukemia through chromosomal instability and DNA hypermethylation.Cancer Cell 2012;22: 645-655.   
48 Barcelona SL,Thompson AA, Cote CJ. Intraoperative pediatric blood transfusion therapy: a review of common issues. Part I: Hematologic and physiologic differences from adults;metabolic and infectious risks.Pediatr Anesth 2005;15: 716-726.   
49 O'Hare T,Deininger MW, Eide CA, Clackson T,Druker BJ.Targeting the BCR-ABL signalingpathway in therapy-resistant Philadelphia chromosome-positive leukemia. Clin Cancer Res 2011;17:212-221.   
50 Peng C,Chen YY,Yang ZF,Zhang HJ,Osterby L,Rosmarin AG et al.PTEN is a tumor suppressor in CML stem cells and BCR-ABL-induced leukemias in mice. Blood 2010;115:626-635.   
51Fontana L,Fiori ME,Albini S,Cifaldi L, Giovinazzi S,Forloni M et al.Antagomir-17- 5pabolishes the growth of therapy-resistant neuroblastoma through p21 and BIM.PLoS One 2008;3:e2236.   
52 Chen CC, Stairs DB, Boxer RB, Belka GK, Horseman ND,Alvarez JVetal.Autocrine prolactin induced by the Pten-Akt pathway is required for lactation initiation and provides a direct link between the Akt and Stat5 pathways.Genes Dev 2012; 26: 2154-2168.   
53 Qrom UA, Derrien T,Beringer M,Gumireddy K, Gardini A, Bussotti G et al. Long noncoding RNAs with enhancer-like function in human cells. Cell 2010;143: 46-58.   
54 Wang S,Li H, Chen YH,Wei HT,Gao GF,Liu HQ et al.Transport of influenza virus neuraminidase (NA) to host cell surface is regulated by ARHGAP21 and Cdc42 proteins.J Biol Chem 2012;287: 9804-9816.   
55 Chen JL, Limnander A,Rothman PB.Pim-1 and Pim-2 kinases are required for efficient pre-B-cell transformation by v-Abl oncogene. Blood 2008;111: 1677-1685.