# Structure of Drosophila Oskar reveals a novel RNA binding protein

Na Yanga,b.1.2, Zhenyu $\yen 1$ , Menglong ${ \mathsf { H } } { \mathsf { u } } ^ { \mathsf { a } , 1 , 3 }$ ,Mingzhu Wanga, Ruth Lehmannd.2,and Rui-Ming $\mathbf { x } \mathbf { u } ^ { \mathbf { a } , \mathbf { b } , 2 }$ tib ofChieseb 5choolof Medicie,NeYrk,0016;ndDepartmentofCelogNeorkUnvrsthoolofdicie,NeYk16

ContributedbyuthLhmann,August7015(sentforreviewJuly16,015;reviewedbyAnthonyP.MahowaldandDinshaJatel)

Oskar (Osk) protein plays critical roles during Drosophila germ cell development,yet its functions in germ-line formation and body patterning remain poorly understood. This situation contrasts sharply with the vast knowledge about the function and mechanism of osk mRNA localization. Osk is predicted to have an N-terminal LOTUS domain (Osk-N),which has been suggested to bind RNA,and a C-terminal hydrolase-like domain (Osk-C) of unknown function.Here,we report the crystal structures of Osk-N and Osk-C.Osk-N shows a homodimer of winged-helix-fold modules,but without detectable RNA-binding activity. Osk-C has a lipase-fold structure but lacks critical catalytic residues at the putative active site. Surprisingly,we found that Osk-C binds the 3'UTRs of osk and nanos mRNA in vitro. Mutational studies identified a region of Osk-C important formRNA binding. These results suggest possible functions of Osk in the regulation ofstability，regulation of translation,and localization of relevant mRNAs through direct interaction with their 3'UTRs,and provide structural insights into a novel protein-RNA interaction motif involvingahydrolase-related domain.

Oskar|structure丨RNA-binding|3'UTR丨germ cell

Spraialadteacralotrel if mlocalzatind polarity,and neuronal function (1,2).The assembly of germplasm in Drosophila is illustrative of such strict regulations.During oogenesis,the germ plasm is assembled at the posterior pole of the oocyte.Germ plasm is composed ofRNAs and proteins,which are specifically required for the formation of germ cells in the early embryo and specification of germ cell fate(3).Germplasm isalso the source of the Nanos protein gradient required for embryonic polarity.Aberrant mRNA localization and untimely protein synthesis during oogenesis can lead to germ cell formation defects and mis-patterning,causing embryonic lethalityand sterility (4).

A critical component of germ plasm is oskar.Oskar RNA is synthesized during oogenesis in the nurse cells, the sistercells of the oocyte,and transported toward the oocyte's posterior pole by a process involving the exon junction complex, the oskar 3'UTR,and microtubule-based movement (5,6).Oskar protein translation is repressed during transport by the RNA-binding protein Bruno and this repression is released by the binding of activators,such as Orb, once the RNA reaches the posterior pole (7-11). Oskar organizes germ plasm by recruiting other proteins, such as Vasa, Tudor, and Aubergine (12-15).An important function of germ plasm is the localization of 50-200 germ plasm-associated RNAs (16,17). Among these,nanos (nos),germ cell-less (gcl)，and polar granule component (pgc) have important roles in abdominal patterning and germ cell specification,germ cell formation,and germ cell transcriptional repression,respectively (18-22).All these functions fail in the absence of Osk and can be initiated at an ectopic location upon mislocalization of Osk (4,23).

Germ plasm-associated RNAs are translationally repressed outside of the germ plasm and are translated in the germ plasm or germ cells.Osk has been implicated not only in the localization but also in the translational activation of nos mRNA,possiblyby releasing the inhibitor Smaug (24).It is not known how

Osk disrupts the interaction between Smaug and nos $3 ^ { \prime } \mathrm { U T R }$ and whether Osk is also responsible for translational regulation of other germ plasm-enriched RNAs.

Although the function and the mechanism of osk mRNA localization have been studied in great detail,the role of Osk protein in germ-line formation and body patterning remains poorly understood.Translation of osk mRNA generates two protein isoforms,a long form and a short Osk protein spanning residues 1-606 and 139-606,respectively. The two isoforms have different localization patterns and the short Osk alone can rescue the defects of osk mutants,whereas the long Osk cannot (25).Here we focus our study on the short Osk protein,which is composed of an N-terminal LOTUS/OST-HTH domain (Osk-N) and a C-terminal hydrolase-like domain (Osk-C) (Fig.1A).We have solved the crystal structure of each domain,and our combined structural and biochemical studies offer mechanistic insights into the molecular function of Oskar.

# Results

The LOTUS Domain of Osk Is a Winged-Helix-Fold Dimerization Module. An $\sim 8 0$ -residue N-terminal segment of short Osk,termed the LOTUS domain,was predicted to form an RNA-binding domain resembling the structure of a winged-helixDNA-binding domain.

# Significance

A profound question in animal biology concerns the origin of germ cells. In the model organism Drosophila melanogaster, oskar directs the assembly of germ plasm that forms and specifies germ cells, but the precise function of its gene product, the Oskar protein, remains poorly understood. Using X-ray crystallography, we determined the 3D structures of an N-terminal fragment (Osk-N) and a C-terminal domain of Oskar (Osk-C). Surprisingly, we discovered that Osk-C has RNA binding function. It binds the 3'UTR of its own mRNA, as well as that of nanos, another gene important for germ-line development. This discovery should direct future mechanistic studies of Oskar for a better understanding of its function in germ cell development.

Author contributions:N.Y.,Z.Y.,M.H.,R..,and R.-M.X.designed research;N.Y.,Z.Y., M.H.,and M.W.performed research; N.Y.,Z.Y.,M.H.,M.W., R.L.,and R.-M.X.analyzed data; and N.Y., R.L.,and R.-M.X. wrote the paper.

Reviewers:A.P.M.，The University of Chicago;and D.J.P.,Memorial Sloan-Kettering Cancer Center.

The authors declare no conflict of interest.

Freely available online through the PNAS open access option.

Data deposition: Atomic coordinates and diffraction data of the structures have been deposited the Protein Data Bank,Www.pdb.org (PDB ID codes 5CD8,5CD7,and 5CD9 for Osk-N,its L199M mutant,and Osk-C,respectively).

'N.Y.,Z.Y,and M.H.contributed equally to this work.

2To whom correspondence may be_addressed.Email: yangna@moon.ibp.ac.cn, Ruth.   
Lehmann@med.nyu.edu,rmxu@sun5.ibp.ac.cn.

3Present addres: School of Biomedical Sciences, Universityof Hong Kong, Hong Kong   
999077, China.

This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1515568112/-/DCSupplemental

![](images/466d241a24d6ba509aeb12215ad1b41cf95bdae6f5bdb2dd8b04499ed71b1434.jpg)  
Fig.1.Structure of Osk-N.(A)A schematic drawing of domain organization of Osk.The shaded regions indicate Osk-N (cyan) and $\cos k - c$ (yellow). Starting positions of long and short forms of Osk are indicated with red arrows. Residue numbers at domain boundaries are labeled.(B) Overall structure of Osk-N homodimer.Two Osk-N molecules,Mol A and Mol B,are shown as a ribbon model colored in cyan and green,respectively.(C) Dimerization interactions involving intermolecular hydrogen bonding (dashed lines) between main-chain groups of two $\beta \uparrow$ stands. $\beta \uparrow$ stands are shown in a stick model (carbon,cyan and green; nitrogen,blue;oxygen,red; sulfur,yellow). $( D )$ Intermolecular interactions between the two α3 helixes.Residues involved in interactions are shown in a stick model.

We crystallized two Osk fragments encompassing the delineated region, one wild-type fragment spanning residues 15O-240 and a L199M mutant (animo acids 150-224),which is used for the SeMetbased single wavelength anomalous dispersion(SAD） method of structure determination. The two structures have been determined to $3 . 0 \mathrm { \AA }$ and $2 . 5 \mathrm { \AA } ,$ respectively (Table S1). Despite being crystallized in different space groups, there are six protein monomers per asymmetric unit for both the wild-type and the mutant Osk fragments that can be separated into three homodimers with the same protein-protein interaction interface (Fig. S1).Furthermore,the ordered protein regions end within residues 22O-223,and there are no significant differences,with an rmsd of $1 . 5 \mathrm { ~ \AA ~ }$ ，between the structure models. Thus,the two structures will be collectively referredto as Osk-Nhenceforth,unless explicitly specified.

The Osk-N monomer adopts a winged-helix-fold,a typical DNA or RNA binding motif,with two antiparallel $\beta$ -strands (β1 and β2) and three $\alpha$ -helixes(α1-α3) forming the“wing”and “helix,”respectively (Fig.1B). Osk-N is predicted to be a RNAbinding motif, termed the LOTUS domain (26,27).The structure shows that although Osk-N assumesa fold capable of RNA binding, it homodimerizes through the wing and helix $\alpha 3$ , masking the potential nucleic acid binding surface (Fig. S2). The dimerization interface buries a total surface area of $8 \mathbf { \check { 8 } } \mathbf { 1 } \mathbf { \mathring { A } } ^ { 2 }$ , indicating amoderately strong intermolecular interaction. The dimeric interface hasanobvioushydrophobicpatch involvingLeu2Oo(located on $\propto 3$ ）andAla207(located on $\beta 1$ )fromboth monomers.Principal dimerization interactions occur between the two $\beta 1$ -strands through intermolecular hydrogen bonding of main-chain groups (Fig. 1C). Ala207 is situated next to the axis of the pseudo twofold symmetry, and they interact with each other througha pair of intermolecular hydrogen bonds between the main-chain carbonyl and amide groups. Ser21O is located in a loop connecting $\beta 1$ and $\beta 2$ ，which appears to be in a location to influence proper positioning of $\beta 1$ . In addition, $\mathrm { A r g } 2 1 5$ located on β2 interacts with Asp197 located on $\propto 3$ of the partner monomer (Fig. $1 D$ ）

To confirm homodimerization of Osk-N in solution,we analyzed the molecular massof Osk-N(150-224) by multiangle light scattering (MALS).The measured molecular mass of Osk-N(150-224) is $1 3 . 2 \mathrm { k D a }$ (Fig. S3A), which is ${ \sim } 5 4 \%$ larger than the calculated molecular weight of $8 . 5 5 \mathrm { k D a }$ ,suggesting that dimeric and monomeric forms of Osk-N are in equilibrium in solution.To ensure that the dimeric Osk-N results from specific association,we hoped to identify Osk-N mutants that will disrupt the dimerization interaction.Based on the structural information,we designed a S210P mutant of Osk-N,with the expectation that introduction of a proline will alter the main-chain conformation of $\beta 1$ that leads to the disruption of the dimer.MALS measurements show that S210P has a molecular mass of $8 . 7 7 \mathrm { \ k D a }$ ,a value close to the calculated mass of monomeric Osk-N (Fig. S3A). This result indicates that the observed Osk-N dimer is a result of specific association of two monomers.

Overall Structure of Osk-C.A $2 . 1 \mathring { \mathrm { A } }$ structure of the C-terminal domain of Oskar (Osk-C,amino acids 393-606) was solved,and the structure reveals a globular domain composed of seven $\alpha$ -helices $( \alpha 1 - \alpha 7 )$ ，two $\beta$ -strands (β1 and $\beta 2$ )，andseveral longloops. $\beta 1$ is locatedbetween $\mathbf { \alpha } \propto 2$ and $\alpha 3$ ,and $\beta 2$ is inserted between $\boldsymbol { \alpha } 4$ and α5. The two $\beta$ -strands forma parallel sheet. Three long loops stand out in the structure: they are the N-terminal loop (LN), the one connecting $\mathbf { \alpha } _ { \mathrm { { \alpha } } } \mathbf { \alpha } _ { \mathrm { { \alpha } } }$ and $\alpha 2$ (L1),and another connecting α6 and $\propto 7$ (L2) (Fig.2A). These loops are longer than 2O residues and some are embedded with short secondary structural elements.Structural alignment shows that Osk-Chasa lipase-like fold,havinganrmsd of $2 . { \overset { \sim } { 7 } } \mathrm { \AA }$ when superimposed with a GDSL-like lipase (PDB ID code 3P94). Overall, secondary structural elements (α1-α7 and $_ { { \beta } 1 - { \beta } 2 ) }$ aligned well between the two structures,but significant differences occur in the loop segments (Fig.2B).The N-terminal ends of Osk-C and the lipase protrude out of the core domain at different directions,and the short turn in the N-terminal loop is replaced by a distorted helix in the lipase structure (Fig.2B).In addition, a pair of short strands- $- \beta \mathbf { a }$ preceding α1 and $\beta \mathbf { b }$ between α5 and $\alpha 6 - \mathrm { j o i n } \beta 1$ and $\beta 2$ to form an extended $\beta$ -sheet,and an extrashort helix(αa) is found C-terminal to $\alpha 1$ in the lipase structure.Furthermore,the lipase L1 loop and the loop connecting β2 and α5 are longer than the corresponding ones in Osk-C,and these loops take significantly different conformations in the two structures.In contrast, the L2 loop in Osk-C is much longer and extends over a significant portion of the protein surface (Fig.2B).

The catalytic active site of the lipase is located on the loop corresponding to loop L2 in Osk-C. The lipase loop is much shorter,and the catalytic triad residues Ser2Oo,Asp2O2,and His205 are spatially adjacent. The Osk-C residues corresponding to these positions are Phe571, Leu586,and Phe587 (Fig.2B). Clearly,these hydrocarbon side-chains of the Osk-C residues are incapable to carryout any catalytic activities,as theylack polar functional groups suitable for carrying out nucleophillic atack and electron transfers.Despite the protein-fold similarity,no lipase or similar enzymatic activities are anticipated for Oskar.

![](images/ff9b02f02ff26792b95d00373a0cf4b8fdb6ff9f7dd68a21e24966d3d4a4bbf2.jpg)  
Fig.2.Structureof Osk-C.(A)OverallStructure of Osk-Cshown asa ribbon model(x-helixes and loops in yellow and $\beta$ -sheets in purple).The three long loops are labeled as LN (N-terminal loop),L1 (loop between α1 and $\propto$ ),and L2 (loop between α6 and α7).(B) Structural alignment of Osk-Cwith a GDSL-like lipase (PDBIDcode)OCslodtheaesielisisoirestwthrucuresi notably those of LN,L1,L2,and the one connecting β2 and $\propto 5$ .The area shaded in pink indicates the location of the lipase active site.An enlarged view of the activeispidialds residues are shown in green.

RNA Binding Propertiesof Oskar.Previous studies have shown that themRNAlocalizationsignalsof oskarand nanosare situated in the 3'UTR region of theirmRNAs (28,29).The 3'UTR region often harbors the binding sites of RNA localization and translational control factors.Gene products of cappuccino,orb,staufen, bruno,and oskar itself, have been implicated as posttranscriptional regulators of oskar mRNA (29).For nanos mRNA, Smaug and Osk are major regulators for repression and activation of translation (24,3O).However, directbiochemical evidence of Osk asan RNA-binding protein is lacking.

Two incidental observations suggest that Osk-C may be an RNA binding module: (i) during protein purification we noticed that Osk-C always associates with nucleic acids;and $( \ddot { u } )$ the protein surface is highly basic,a number of sulfate ions from the crystallization buffer are bound in the structure (Fig. S4). The sulfate ions may mimic the negatively charged phosphate backbone ofRNA.Prompted by these observations,we set out to biochemically test the binding of Osk-C and Osk-N fragments to RNA.We first generated the 3'UTR of osk mRNAby in vitro transcription,and tested whether Osk-N and Osk-Care able to bind it in an EMSA. The results show that Osk-Cbinds osk3'UTRavidly,but Osk-Nshowsno detectable binding under our assay conditions (Fig.3A).Next,we tested whether Osk-C specificallybindsRNAor indiscriminatelyassociates with nucleic acids, such as double-stranded DNA (dsDNA), using a competition assay.We generated a fluorescein-labeled osk 3'UTR by incorporation of fluorescein-12 dUTP into the probe during in vitro transcription (Fig.3B).Addition of fourfold excessofunlabeled osk3'UTR completelyabolished thebinding of labeled RNA,whereas dsDNA could not compete for binding even at eightfold excess.Therefore,we conclude that Osk-C is an RNA binding domain that can interact with its own 3'UTR.We further tested the binding of Osk-C to the $3 ^ { \prime } \mathrm { U T R s }$ of nos and pgc(Fig.3C).Osk-C bound to the translational control element(TCE) of nos 3'UTR,but not to the 3'UTR of pgc under the same assay condition and a protein:RNA molar ratio up to 30:1. Osk-C appeared to bind to the $3 ^ { \prime } \mathrm { U T R }$ of nos stronger than to that of oskar (Fig.3C).By increasing the protein to RNA ratio we could detect a shift of the 3'UTR of pgc,but the implication isunclear.It shouldbe emphasized that ourobservation does not exclude the possibility that Osk-C may bind full-length pgc vigorously.

RNA Binding Surface of Osk-C. The structure of Osk-C reveals several positively charged patches on the surface that may be RNA binding sites (Fig. 4A).We chose the residues forming positively charged patches and involved in binding sulfate ions for mutational studies.Eleven single or combinational substitution mutants—R448E，R470E，R490E，R567E，R576E，R593E, R593A，R595E，R436E/R442E,R445E/H446D，andR538E/ K541E/R544E—were generated.We were able to express and purify all but the R593E mutant,and they were subjected to EMSA analysisusing oskarand nanos3'UTRsas probes (Fig.4B and Fig. S5). Compared with wild-type Osk-C,two mutants,R436E/R442E andR576E，showed reduced affinities toward oskar3'UTR. R436E/R442E almost completely lost the ability to bind oskar 3'UTR.Similar levels of reduction of binding abilities of these two mutants toward nos 3'UTR were observed (Fig.4C).This analysis identifies the regions occupied by the three residues,Arg436, Arg442,and Arg576 as part of the RNAbinding surface.Arg436 is located on α1,and Arg442 and Arg576 are situated in the long loops between α1 and $\alpha 2$ and $\propto$ andα6,respectively.These three residues lie on the same side of the Osk-C surface on a positively charged narrow ridge,which appears adequate for accommodating RNA (Fig. 4D).

![](images/b0ff199d7b8e64a4388655a8a0b15cab0c88b358abb8993945dfdfc40591ae4c.jpg)  
Fig.3.EMSA analysis of Osk-RNA interaction.(A) EMSA results show that Osk-Cbinds osk 3'UTR but Osk-N shows no detectable binding.(B)Fluorescein-labeled osk 3'UTR,shows as green bands,shifted by increasing concentration of Osk-C and the binding is competed by fourfold-excess of unlabeled osk 3'UTR but not dsDNA,even at eightfold excess.(C) Binding of Osk-C to the 3'UTRs of osk,TCE of nanos,and pgc.

![](images/23c10ca3faf6bc216bfe8c0415b4f65c1a095cc0b8feaecf17771cadeb0d7a38.jpg)  
Fig4RAingurfaceofOk-C.(oitivledsisoOCurfceleutaaaalyesreoinicee imposed with the surface representation of $\cos k - c$ colored according to electrostatic potential distribution.(B and C)Two Osk-C mutants,R436E/R442E and R576E,displadduddnitrdthsof(dspdittatfidtO.(g4 Arg576 form a narrow positively charged ridge,which may be involved in RNA binding.

# Discussion

Osk is critically important for germ cell development in Drosophila.Most studies to date have focused on the function and regulation of osk mRNA,and little is known about the biochemical function of Osk protein.It has been reported that Osk interacts with Vasa,Staufen,and Tudor proteins (12),but we were not able to detect robust interactions between Osk and these proteins in our hands by coelution in a gel-filtration column using purified proteins (Fig. S6). Speculations regarding the biochemical properties of Osk from bioinformatic analyses predicted Osk-C to have a hydrolase-like fold and Osk-N to form a LOTUS winged-helix domain with RNA binding properties. Our structural analysislargely confirms the protein-fold predictions, and offers additional and novel insights into the structure and function of Osk.First, the winged-helix LOTUS domain forms a dimerand displaysno detectable RNA binding activity.We also do not detect RNA binding activity of monomeric Osk-N using the S21OP mutant (Fig. S3B).However, it should be cautioned that we could not absolutely rule out possible RNA binding activities of Osk-N because we only tested a specific set of RNAs. The role of the LOTUS domain and the function of the predicted Oskar protein dimer need yet to be determined in vivo. Besides stop codon mutations that lead to protein truncations, specific point mutations mapping to this region and affecting Oskar's posterior patterning function have not been identified.

It comes as a surprise that Osk-C binds RNA. To our knowledge,this is the first time that a lipase-like domain has been reported to be a nucleic acid binding domain.Interestingly,a number of metabolic enzymes have recently been shown to “moonlight” as RNA binding proteins (31).We further find that the 3'UTRs of osk and nanos are potential binding partners of Osk-C.This discovery provides new insights into the biochemical functions of Oskar in embryonic patterning and germ cell development.Mutational studies also identified surface regions of Osk-C that are important for RNA binding.Interestingly,the biochemically identified residues that are key for RNA binding areinproximity to several genetically identified osk alleles defective in posterior pattering (Fig. S7 and Table S2) (32-35). This correlation provides a possible link between the biochemical and developmental functions of Oskar.To gain further insights into the RNA binding property of Osk,we attempted to determine the RNA sequence specificity for Osk-C binding. However,we were unable to discern a pattern by simple deletion analysis.It is possible that there may not be strict RNA sequence specificity for Osk binding,as other factors,such as RNA secondary structures,may regulate RNA binding.Nevertheless,a clearer picture of RNAbinding,includinga structure of Osk in complex with RNA，should greatly benefit mechanistic understandings of the biological functions of Osk.

# Experimental Procedures

Protein Expression and Purification.The short isoform of Oskar,generated from translation initiated from an internal start codon,spans residues 139- 606.cDNA fragments encoding Osk-N (residues 150-240) and a L199M mutant (residues 150-224) were cloned into a pET28a-SUMO vector between the BamHl and Sall sites and expressed in the BL21-Codon Plus (DE3)-RIL strain of Escherichia coli.Once the cell density reached ${ \mathsf { O D } } _ { 6 0 0 } \sim 0 . 8$ ，protein expression was induced with $0 . 3 ~ \mathsf { m M }$ isfopropyl $\cdot \{ \beta$ -D-thiogalactopyranoside (IPTG) at $1 6 ^ { \circ } C$ for $1 8 { \mathsf { h } }$ .Cells were harvested by centrifugationat $4 , 6 7 0 \times g$ and lysed by passing through an Emulsiflex C3 cell disruptor in the lysis buffer $( 2 0 \mathsf { m } \mathsf { M }$ Tris·HCI pH 8.0,500 mM NaCl, $1 0 ~ \mathsf { m M }$ imidazole).The homogenate was centrifuged at $2 6 { , } 4 0 0 \times { } g$ and supernatant was loaded onto a Ni-NTA (Novagen) column pre-equilibrated with the lysis Buffer.After extensive washing with the washing buffer $( 2 0 ~ \mathsf { m M }$ Tris·HCI pH 8.0, $5 0 0 ~ { \mathsf { m M } }$ NaCl, $2 0 ~ \mathsf { m M }$ imidazole), the hisSUMO tagged protein was eluted with the elution buffer (20 mM Tris:HCl pH 8.0, $5 0 0 ~ \mathsf { m } \mathsf { M }$ NaCl, $5 0 0 ~ \mathsf { m } \mathsf { M }$ imidazole).The his-SUMO tag was cleaved with SUMO protease during dialysisagainst bufferA(20 mM MES pH 6.0,100 mM NaCl, $0 . 1 \%$ $\beta$ -ME).The tag-free protein was loaded onto a heparin column pre-equilibrated with bufferAand eluted with a $1 0 0 { - } 1 , 0 0 0 \ \mathsf { m M N a C l }$ gradient.The desired fractions were pooled,concentrated and further purified on a Superdex 75 10/60 XK column (GE Healthcare) in a buffer containing 20 mM Tris·HCl pH 8.0,500 mM NaCl,and $0 . 1 \%$ $\beta$ -ME.

Osk-C and its mutants were expressed using a modified pET28a-SUMO vector with the coding sequences inserted between the BamHl and Sall sites. Protein expression and purification follow the same procedures as that for Osk-N,except that buffer A used in the Heparin column contains $2 0 ~ \mathsf { m M }$ Tris·HCl pH 8.0,150 mM NaCl, $0 . 1 \%$ $\beta$ -ME, $0 . 5 \mathsf { m M }$ EDTA,and the final step of size-exclusion column chromatography was carried out using a Superdex 7510/3o0 GL column (GE Healthcare) in a buffer containing $2 0 ~ \mathsf { m M }$ Tris-HCl ${ \mathsf { p H } } 8 . 0$ ,300 mM NaCl, $0 . 1 \%$ $\beta$ -ME,and $0 . 5 ~ \mathsf { m M }$ EDTA.

Crystallization and Structure Determination. Osk-N concentrated to $\sim 5 \ : \mathrm { m g / m L }$ was crystallized usinga reservoir buffer containing $1 0 0 ~ \mathsf { m } \mathsf { M }$ sodium acetate ${ \mathsf { p H } } 5 . 6$ $20 \%$ (vol/vol) PEG 400 and $2 0 0 ~ \mathsf { m M }$ magnesium acetate by hangingdrop vapor diffusion at $1 6 ^ { \circ } \mathsf { C }$ .The L199M mutant was concentrated to $6 . 5 \mathrm { \ m g / m L }$ and the crystallization buffer contains $1 0 0 ~ \mathsf { m M }$ sodium acetate ${ \mathsf { p H } } 5 . 6 ,$ （2 $1 7 - 2 2 \%$ (vol/vol) PEG 3350 and 150 mM NaCl.Osk-C was concentrated to $\mathord { \sim } 1 5 \mathrm { \ m g / m L }$ and crystals for X-ray diffraction were obtained from a condition containing $1 0 0 ~ \mathsf { m M }$ sodium acetate ${ \mathsf { p H } } 4 . 6$ $8 0 0 ~ \mathsf { m M }$ ammonium sulfate and $2 0 0 ~ \mathsf { m } \mathsf { M }$ lithium sulfate,also by hanging-drop vapor diffusion at $1 6 ^ { \circ } \mathsf C$

X-raydiffraction data were collected at Beamline BL17U of Shanghai Synchrotron Radiation Facility using an ADSC Q315r detector.All diffraction data were processed using HKL2000 (36).The $2 . 5 \mathring { \mathsf { A } }$ SADdata for the SeMet L199M mutant were collected at wavelength $0 . 9 7 9 0 \mathrm { \AA } .$ The crystal belongs to the P21 space group and there are six molecules in one asymmetric unit. The 2.1 A SAD data of SeMet Osk-C were collected at wavelength $0 . 9 7 8 8 \mathring { \mathsf { A } }$ The crystal belongs to the P3121 space group and there is one molecule per asymmetry unit.In both cases,six Se sites were found using SHELXD (37)and phasing was carried out with PHENiX (38).A $3 . 0 \mathrm { ~ \AA ~ }$ dataset of SeMet wildtype Osk-N(150-240) was collected at wavelength $0 . 9 6 4 1 \mathrm { ~ \AA ~ }$ .The crystal belongs to the P32 space group and there are six molecules in one asymmetric unit.The structure was solved by molecular replacement with PHASER (39)using the L199M structure as the search model. Initial model building, model fitting,and refinement of all of the three structures were done with PHENIX and COOT (40).Detailed statistics for data collection and refinement are shown in Table S1.

MALS. Osk-N was diluted to $2 \ : \mathrm { m g / m L }$ in a buffer containing $2 0 ~ \mathsf { m } \mathsf { M }$ Tris-HCl ${ \mathsf { p H } } 8 . 0$ and $1 5 0 \ : \mathrm { m M } \ : \mathsf { N a C l }$ .After being centrifuged at $1 5 , 8 7 1 \times g$ for 15 min,a $5 0 \mathrm { - \mu L }$ sample was loaded onto a WTC-o3os5 size-exclusion column (Wyatt Technologies) connected to a MALS instrument (DAWN HELEOS II;Wyatt Technologies).Molecular mass of particles in a single elution peak was calculated based on light scattering data using the ASTRA software package.

In Vitro Transcription of RNA.cDNAs encoding osk 3'UTR,nanos 3'UTR,and pgc 3'UTR were cloned into a pBluescript Il $\mathsf { S K } ( + )$ vector between Kpnl and Hindlll sites for in vitro transcription.The recombinant plasmid DNAswere linearized with Hindll downstream of the insert to be transcribed as the template.The RNAs were transcribed with T7 polymerase by MEGAscript Kit (AM1334,Ambion) following the procedures recommended by the manufacturer.For in vitro transcription of fluorescein-labeled osk3'UTR,fluorescein12-UTP (Roche) was added in the reaction together with the unlabeled one. The transcribed RNAs were purified with MEGAclear Purification Kit (Ambion). Quality and quantity of RNAs were examined both by UV spectroscopy and $5 \%$ denaturing polyacrylamide gel.

EMSA.In each EMSA reaction (total volume, $2 0 \mu \updownarrow ,$ ，an ${ 8 0 \mathsf { n g } }$ heat-denatured RNA probe of osk 3'UTR,nanos 3'UTR,or pgc 3'UTR was incubated

1. Besse F,Ephrussi A (20o8) Translational control of localized mRNAs: Restricting protein synthesis in space and time.Nat Rev Mol Cell Bio/9(12):971-980.   
2.Kugler JM, Lasko P (2oo9) Localization,anchoring and translational control of oskar, gurken,bicoid and nanos mRNA during Drosophila oogenesis.Fly (Austin)3(1):15-28.   
3.Mahowald AP (2001) Assembly of the Drosophila germ plasm.Int Rev Cytol 203: 187-213.   
4.Ephrussi A,Lehmann R (1992) Induction of germ cell formation by oskar.Nature 358(6385):387-392.   
5.Ghosh S,Marchand V,Gaspar I, Ephrussi A (2012) Control of RNP motility and localization by a splicing-dependent structure in oskar mRNA.Nat Struct Mol Biol 19(4): 441-449.   
6.Zimyanin VL,et al.(2o08) In vivo imaging of oskar mRNA transport reveals the mechanism of posterior localization. Ce// 134(5):843-853.   
7.Kim-Ha J,Kerr K,Macdonald PM (1995) Translational regulation of oskar mRNA by bruno,an ovarian RNA-binding protein,is essential. Ce// 81(3):403-412.   
8.Chekulaeva M, Hentze MW,Ephrussi A (2oo6) Bruno acts as a dual repressor of oskar translation,promoting mRNA oligomerization and formation of silencing particles. Cell 124(3):521-533.   
9.Chang JS,Tan L,Schedl P (1999) The Drosophila CPEB homolog,orb,is required for oskar protein expression in oocytes. Dev Biol 215(1):91-106.   
10.Kim G,et al.(2015) Region-specific activation of oskar mRNA translation by inhibition ofBruno-mediated repression.PLoS Genet11(2):e1004992.   
11.Nakamura A,Sato K, Hanyu-Nakamura K (2004) Drosophila cup is an elF4E binding protein that associates with Bruno and regulates oskar mRNA translation in oogenesis. Dev Cell 6(1):69-78.   
12.Breitwieser W,Markussen FH,Horstmann H,Ephrussi A(1996) Oskar protein interaction with Vasa represents an essential step in polar granule assembly.Genes Dev 10(17):2179-2188.   
13.Lasko P (2011) Posttranscriptional regulation in Drosophila oocytes and early embryos.Wiley Interdiscip Rev RNA 2(3):408-416.   
14.Anne J(201o) Targeting and anchoring Tudor in the pole plasm of the Drosophila oocyte.PLoS One 5(12):e14362.

individually with various quantities of Osk or Osk mutants ina binding buffer containing $2 0 ~ \mathsf { m M }$ Tris·HCl,pH7.9,1mM DTT,10 mMKCI,15 mMNaCl, $4 0 \mu \ g / \mathrm { m L }$ calf BSA,and $5 \%$ glycerol for $3 0 ~ \mathrm { m i n }$ on ice.After incubation,protein-bound and free RNAswere separated by electrophoresis on nondenaturing $6 \%$ polyacrylamide gels (80:1) running in a $0 . 5 { \times } \mathsf { T B E }$ buffer at $1 0 \ \mathsf { V } / \mathsf { c m } ^ { - 1 }$ at $4 ^ { \circ } \mathsf C$ For unlabeled RNA probes, gels were stained by SYBR Gold (Life Technologies) and scanned by FluorChem M System (ProteinSimple).For fluorescein-labeled $R N A s ,$ （204 gels were directly scanned at $6 0 7 ~ \mathsf { n m }$ by FluorChem M System (ProteinSimple). The RNA probes were quantified by UV spectrophotometer Nanodrop 2000 (Thermo Scientific) at wavelength $2 6 0 ~ \mathsf { n m }$ ：

Accession Numbers.Atomic coordinates and diffraction data of the structures have been deposited with PDB under accession codes 5CD8,5CD7 and 5CD9 for Osk-N,its L199M mutant,and Osk-C,respectively.Structural figures were prepared using Pymol (www.pymol.org).

Note Added in Proof.While this manuscript was being reviewed,a report by Jeske et al.describing similar studies of Osk was published online (41). Crystal structures of Osk fragments equivalent to Osk-N and Osk-C denoted in our work were reported,and the structures are nearly identical (rmsd $0 . 3 4 \mathring { \mathsf { A } }$ and $0 . 6 9 \mathring { \mathsf { A } }$ for Osk-C and Osk-N,respectively),supporting the dimeric form of Osk-N from both of our studies.Using UV cross-linking and immunoprecipitation,Jeske et al.(41) showed that full-length short Oskar binds to nos,pgc,and $g c I \cap \mathsf { R N A s }$ in vivo,and the RNA binding activity was attributed to Osk-C based on in vitro pulldown results.In our work,we showed that purified Osk-C directly binds the 3'UTR regionsof osk and nos in vitro.We detected no meaningful binding of Osk-C to the 3'UTR of pgc in our assay, but we cannot rule out that Osk-C may bind full-length pgc mRNA.Another difference between the two studies is that Osk-N was shown to interact with Vasa by Jeske et al.using GST-pulldown,isothermal titration calorimetry, and yeast two-hybrid assays.We were not able to detect such interaction by gel-filtration column chromatography using purified proteins.

ACKNOWLEDGMENTS.We thank the beamline scientists at Shanghai Synchrotron Radiation Facility for technical support during data collection; Xiaoxia Yu for assistance with mulitangle light-scattering experiments;and Dr.Thomas Hurd for critically reading the manuscript.This work was supported by the Natural Science Foundation of China (Grants 31370734, 31430018,and 31400670);the Ministry of Science and Technology of China (Grant 2015CB856202); the National Key New Drug Creation and Manufacturing Program of China (Grant 2014ZX09507002); the Strategic Priority Research Program (Grant XDB08010100);and the Key Research Program (Grant KJZDEW-Lo5) of the Chinese Academy of Sciences.N.Y.is also supported by the Youth Innovation Promotion Association of the Chinese Academy of Sciences.R.L.is a Howard Hughes Medical Institute investigator.

15.Thomson T,Liu N,Arkov A,Lehmann R,Lasko P (2oo8) Isolation of new polar granule components in Drosophila reveals P body and ER associated proteins.Mech Dev 125(9-10):865-873.   
16.Jambor H,et al.(2015) Systematic imaging reveals features and changing localization ofmRNAsin Drosophila development.eLife4:e05003.   
17.Lécuyer E,et al.(2oo7) Global analysis of mRNA localization reveals a prominent role in organizing cellular architecture and function. Ce// 131(1):174-187.   
18.Wang C,Lehmann R (1991) Nanos is the localized posterior determinant in Drosophila.Cell 66(4):637-647.   
19.Cinali RM, Lehmann R(2013)A spindle-independent cleavage pathway controls germ cell formation in Drosophila. Nat Cell Biol 15(7):839-845.   
20.Jongens TA,Hay B,Jan LY,Jan YN (1992) The germ cell-less gene product: A posteriorly localized component necessary for germ cell development in Drosophila. Cell 70(4):569-584.   
21.Hanyu-Nakamura K,Sonobe-Nojima H,Tanigawa A,Lasko P,Nakamura A (2008) Drosophila Pgc protein inhibits P-TEFb recruitment to chromatin in primordial germ cells.Nature 451(7179):730-733.   
22.Martinho RG, Kunwar PS,Casanova J,Lehmann R (20o4)A noncoding RNA is required for the repression of RNApolll-dependent transcription in primordial germ cells. Curr Biol 14(2):159-165.   
23.Smith JL,Wilson JE,Macdonald PM (1992) Overexpression of oskar directs ectopic activation of nanos and presumptive pole cellformation in Drosophila embryos. Cell 70(5):849-859.   
24.Dahanukar A,Walker JA,Wharton RP(1999) Smaug,a novel RNA-binding protein that operates a translational switch in Drosophila.Mol Cell 4(2):209-218.   
25.Markussen FH,Michon AM, Breitwieser W, Ephrussi A(1995) Translational control of oskar generates short OsK,the isoform that induces pole plasma assembly. Development 121(11):3723-3732.   
26.Callebaut I，Mornon JP (2010) LOTUS,a new domain associated with small RNA pathways in the germline.Bioinformatics 26(9):1140-1144.   
27.Anantharaman V,Zhang D,Aravind L (2010) OST-HTH:A novel predicted RNAbinding domain. Biol Direct 5:13.   
28.Gavis ER,Lehmann R (1994) Translational regulation of nanos by RNA localization. Nature 369(6478):315-318.   
29.Rongo C,Lehmann R (1996) Regulated synthesis, transport and assembly of the Drosophila germ plasm.Trends Genet 12(3):102-109.   
30.Dahanukar A,Wharton RP (1996) The Nanos gradient in Drosophila embryos is generated by translational regulation.Genes Dev 10(20):2610-2620.   
31. Castello A,et al.(2012) Insights into RNA biology from an atlas of mammalian mRNAbinding proteins.Ce// 149(6):1393-1406.   
32.Lehmann R, Nüsslein-Volhard C(1986) Abdominal segmentation, pole cell formation, and embryonic polarity require the localized activity of oskar,a maternal gene in Drosophila.Cell 47(1):141-152.   
33.Ephrussi A,Dickinson LK,Lehmann R (1991) Oskar organizes the germ plasm and directs localization of the posterior determinant nanos.Ce/ 66(1):37-50.   
34.Kim-Ha J,Smith JL,Macdonald PM (1991) oskar mRNA is localized to the posterior pole of the Drosophila oocyte. Cell 66(1):23-35.   
35.Rongo C(1996) The role of RNA localization and translational regulation in Drosophila germ cell determination.PhD thesis (Massachusetts Institute of Technology, Cambridge,MA).   
36.Otwinowski Z,Minor W(1997) Processing of X-ray diffraction data collected in oscillation mode.Methods Enzymol 276:307-326.   
37.Sheldrick GM(2008)A short history of SHELX.Acta Crystallogr A 64(Pt1):112-122.   
38.Adams PD,et al.(2010) PHENIX:A comprehensive Python-based system for macromolecular structure solution.Acta CrystallogrD Biol Crystallogr 66(Pt 2):213-221.   
39.Collaborative Computational Project, Number 4 (1994) The CCP4 suite: Programs for protein crystallography.Acta CrystallogrD Biol Crystallogr 50(Pt5):760-763.   
40.Emsley P,Cowtan K(2o04) Coot: Model-building tools for molecular graphics.Acta CrystallogrD Biol Crystallogr 60(Pt12Pt1):2126-2132.   
41.Jeske M,et al.(2015) The crystal structure of the Drosophila germline inducer Oskar identifies two domains with distinct Vasa helicase- and RNA-binding activities. Cell Reports 12(4):587-598.