# Terpene metabolic engineering via nuclear or chloroplast genomes profoundly and globally impacts off-target pathways through metabolite signalling

Elise K.Pasoreck¹， Jin ${ \mathsf { S u } } ^ { 1 }$ , lan M. Silverman²，Sager J.Gosai²,Brian D.Gregory², Joshua S.Yuan³ anc Henry Daniell1 \*

1Departmentof Biochemistry,Schoolof Dental Medicine,Universityof Pennsylvania,Philadelphia,PAUSA   
2Departmentof Biology， Universityof Pennsylvania,Philadelphia,PA,USA   
3Department of Plant Pathologyand Microbiology,TexasA&M University,Collge Station,TX,USA

Received 11 November2015;   
revised3February2016;   
accepted8 February 2016.   
\*Correspondence (Tel 215 746 2563; fax 215 898 3695;email hdaniell@upenn.edu) GEO accession number for RNA-seq data: GSE74103.

# Summary

The impact of metabolic engineering on nontarget pathways and outcomes of metabolic engineering from different genomes are poorly understood questions.Therefore,squalene bioSyntheSis genes FARNESYL DIPHOSPHATE SYNTHASE(FPS) and SQUALENE SYNTHASE (SQS) were engineered via the Nicotiana tabacum chloroplast (C),nuclear (N)or both (CN) genomes to promote squalene biosynthesis.SQS levels were \~4300-fold higher in C and CN lines than in N, but all accumulated $\mathord { \sim } 1 5 0$ -fold higher squalene due to substrate or storage limitations.Abnormal leaf and flower phenotypes,including lower polen production and reduced fertility,were observed regardless of the compartment or level of transgene expression.Substantial changes in metabolomes of allines were observed: levels of 65-120 unrelated metabolites,including the toxic alkaloid nicotine,changed by as much as 32-fold.Profound effects of transgenesis on nontarget gene expresson included changes in the abundance of 19 O76 transcripts by up to 2000-fold in CN;7784 transcripts by up to 1400-fold in N; and 5224 transcripts by as much as 2200-fold in C.Transporter-related transcripts were induced,and cell cycle-associated transcripts were disproportionally repressed in all three lines.Transcriptome changes were validated by qRTPCR.The mechanism underlying these large changes likely involves metabolite-mediated anterograde and/or retrograde signalling irrespective of the level of transgene expression or end product, due to imbalance of metabolic pools,offering new insight into both anticipated and unanticipated consequences of metabolic engineering.

Keywords: Secondary metabolism, retrograde/anterograde signaling, transgenic/transplastomic plants, squalene.

# Introduction

Many metabolic engineering strategies rely on transformation of the nuclear genome.For example，avian FARNESYL DIPHOSPHATE SYNTHASE(FPS)and yeaSt SQUALENE SYNTHASE (SQS) have been expressed in Nicotiana tabacum to promote squalene accumulation (Wu et al.，2012).New metabolic pathways have also been engineered,for example by introducing the pathway for the cyanogenic glucoside dhurrin into Arabidopsis (Tattersall et al.,2oo1). Several other studies have found that directing key enzymes to chloroplasts can substantially increase the content of metabolites such as $\beta$ -carotene—used to produce‘golden rice' (Ye et al.,2ooo)—and terpenes,including patchoulol (Wu et al., 2006)and squalene (Wu et al.,2012).

However，nuclear transformation is associated with several disadvantages.Positional effects can reduce or silence transgene expression,making it difficult to establish a stable population that consistently accumulates high levels of the desired product (Jin and Daniell,2015;Verma and Daniell,2o07).Additionally,lack of transgene containment raises regulatory issues and makes commercialization more challenging (Chan and Daniell,2015). Some of these problems can be overcome by expressing transgenes via thechloroplast genome. Chloroplast transformation，which occurs through homologous recombination,contains transgenes (Verma et al.，2008),and its high ploidy—up to 10 o00 copies per cell—permits transgene products to reach exceptionally high levels (Ruhlman et al.，201O).For example,tobacco chloroplasts expressing a multigene operon encoding Bt toxin allowed the protein to accumulate up to $\sim 5 0 \%$ of total leaf protein and to form cuboidal crystals within chloroplasts (De Cosa et al.,2001); chloroplast-expressed $\beta$ -glucosidase accumulated to 160-fold higher levels than in untransformed plants (Jin et al.，2011); and proinsulin could make up nearly $70 \%$ of total transplastomic leaf protein (Ruhlman et al., 2010).

Chloroplast genome transformation has also been successfully used in metabolic engineering (Jin and Daniell,2015):chloroplasts expressing chorismate pyruvate-lyase accumulated the biopolymer $p$ -hydroxybenzoic acid to ${ \sim } 2 6 . 5 \%$ dry weight—the highest level reported for any bioproduct—without compromisingplant health (Viitanen et al.，2oo4).In later studies，the chloroplast genome was engineered to express all six genes of the cytoplasmic mevalonate pathway for terpenoid synthesis (Kumar et al.，2O12) or thirteen artemisinin pathway biosynthesis genes (Saxena et al.,2o14),although in the latter case,expression was inadequate to meet target levels.However,chloroplast genome transformation requires further optimization,anda comparison of metabolic engineering outcomes via engineering different cellular compartments has not been performed.

The chloroplast genome is highly reduced,with many genes lost or transferred to the nucleus (Jensen and Leister，2014). Consequently,chloroplast function requires the import of thousands of nuclear-encoded proteins,many of which work in concert with plastid-encoded gene products and require proper stoichiometry (Jinand Daniell,2015).Therefore, the expression of nuclear- and plastid-encoded genes must be coordinately regulated,and this occursvia anterograde signalling from the nucleus to plastids and retrograde signalling from plastids to the nucleus. Whereas anterograde signalling is well understood,retrograde signalling isstill enigmatic.Chloroplasts may regulate nuclear gene expression via proteins (Jin and Daniell,2014;Singh et al., 2008),redox state (Nott et al.，20o6) ormetabolites (Chi et al., 2015；Estavillo et al.，2011;Woodson et al.，2011; Xiao et al., 2012),but these signals act only under specific circumstances, and mechanisms by which they are conveyed remain elusive. However，high accumulation ofproteins expressed viathe chloroplast genome and compartmentalization within chloroplasts make chloroplast genetic engineering an excellent system to study retrograde signalling.

Much work on metabolic engineering and synthetic biology has focused on engineering pathways to generate high-value metabolites,but the global impact of such engineering has not yet been explored despite the potential for unintended consequences (Bobik and Burch-Smith，2015)．The availabilityof modern tools to study the metabolome and transcriptome facilitates global evaluation of the effect of these introduced pathways on native genes through metabolite-mediated anterograde or retrograde signalling. Here,we used chloroplast genetic engineering and an existing nuclear transgenic line to uncover potential unintended consequences of expressing metabolic genes from different compartments.We focused on squalene because of its importance in steroid biosynthesis,its industrial applications in cosmetics and nutraceuticals (Kim and Karadeniz, 2012)and its use as a vaccine adjuvant (O'Hagan et al.,2o11),as well as the availability of nuclear transgenic lines expressing squalene biosynthetic enzymes for comparative investigations (Wu et al.，2012).We found profound but similar off-target effects on both the metabolome and transcriptome regardless of the compartment from which transgenes were expressed.By focusing on global effects of intercompartmental signalling rather than attempting to identify yet another signalling molecule,we provide a framework for future studies on large-scale effects of metabolite-mediated intercompartmental signalling.

# Results

# Characterization of transplastomic lines expressing FPS and sQs

Toengineer squalene biosynthesis through different organellar genomes，we generated two tobacco chloroplast expression vectors，one encoding Flag-tagged SQS (pLD-SQS) and one encoding both Flag-tagged SQS and His-tagged FPS (pLD-FPSSQS) (Figure 1a).Encoded amino acid sequenceswere identical to those reported previously (Wu et al.，2012)，but corresponding DNA sequences were codon-optimized for enhanced chloroplast expression (Daniell et al.，2009). In pLD-SQS，Flag-SQS was regulated by the tobacco psbA promoter, $5 ^ { \prime }$ -UTRand $3 ^ { \prime }$ -UTR, and isoleucine tRNA (trnl) and alanine tRNA (trnA) flanking sequences were included for integration into the chloroplast genome via homologous recombination (Verma andDaniell, 2007).In pLD-FPS-SQS,expression of His-FPS was controlled by theplastid rRNA operon promoter (Prrn)，the $5 ^ { \prime }$ translation control element of bacteriophage T7 gene 1O and the tobacco rbcL $3 ^ { \prime }$ -UTR (Dhingra et al.,2004).

Two N. tabacum lines were used: the wild-type (WT)1068 introduction,which has abundant glandular trichomes that may besqualene sinks (Wu et al.,2o12),and a transgenic line in the WT background expressing chloroplast-targeted SQS and FPS via the nuclear genome (Wu et al.，2012),hereafter referred to as 'N'.Leaves of N were bombarded with pLD-SQS to generate transplastomic lines expressing SQS from the chloroplast genome; theresulting line is referred to as‘CN'.After selection and regeneration on spectinomycin-containing media,several independent CN lines were regenerated from ten bombardments. PCR analysis with the 3P/3M primer set (Verma et al.，2008) indicated that the Flag-SQS cassette had been stably integrated into the chloroplast genome via homologous recombination (Figure S1).After two additional rounds of selection on spectinomycin-containing media, CN lines were confirmed by Southern blot.As shown in Figure 1a,b，several independent CN lines showed a $6 . 6 3 \cdot \vert \times \vert )$ band but not the 4.43-kb WT band,confirming that homoplasmic plants had been generated.We also bombarded N with pLD-FPS-SQS,but no shoots survived.When pLDFPS-SQS was used to transform WT tobacco,several independent transplastomic lines，referred to as‘C'，were obtained，as confirmed by PCR (Figure S1) and Southern blot (Figure 1b, left). Two bands with correct sizes of 8.42 and $3 . 4 8 ~ \mathsf { k b }$ (due to the presence of a Hindll site in the His-FPS cassette)，but not the 7.67-kb WT band,were observed in Hindll-digested DNA from C lines,confirming that homoplasmic C lines had been generated (Figure 1b，right). Because the engineered His tag was not detectable in western blots,we confirmed expression of His-FPS via the chloroplast genome using northern blot for the rbcL $3 ^ { \prime }$ UTR (Figure 1a).In addition to the endogenous rbcL signal in each line,the C line showed an additional band corresponding to the expressed His-FPS transgene (Figure 1c).Notably，the His-FPS transcript was as abundant as that of rbcL，which is the most abundant protein on earth (Dhingra et al.,2004).

Expression of Flag-SQS protein was detected by western blot usinga monoclonal anti-Flag antibody.A Flag signal could be detected in both transgenic and transplastomic lines.However, detection of the signal in N required loading as much as 120-fold more protein (Figure 1d). To quantify this difference，we performed densitometric analysis.After accounting for differences in the amount of loaded protein and normalizing to N,we found that CN expressed 2813-4372 times more Flag-SQS than N and that C expressed 1399-2309 times more than N.

# The impact of FPSand SQS expression on leaf and flower development

Regardless of the genome from which transgenes were expressed or levels of Flag-SQS,expression of FPS and SQS had a profound effect on leaf and flower development,but CN displayed most severe leaf and flower phenotypes.At the time of transfer to soil, CN leaves were half as long as WT (Figure 2a) and remained small,both after transferring to the glasshouse (Figure 2b) and at the onset of flowering (Figure 2c).In particular, leaves of adult CN plants were shorter,narrower and more curled (Figure 2c). Leaves of Nwere also initially shorter than WT (Figure 2a,b),but as they aged, leaf morphology more closely resembled that of WT (Figure 2c).A similar phenomenon was observed for C plants, whose pleiotropic phenotypes became less pronounced with age (Figure 2b,c).

![](images/a30a643dc2de90639fbc4a7f0db31d0716112cfb53b72d7be7ce3d621c83c53c.jpg)  
Figure1Chloroplasttransformatiovectorsandcaracteratiooftransplastomiclines.a)hematicsofthetbaocoplaseo transftisdiitstotradt; bottom, pLD-FPS-SQS.(b) Genomic DNA $( 0 . 5 ~ \mu \ g )$ from independent transplastomiclines was digested withAfl (CNlines,left)or Hindll(Clines,rightand probed with a DIG-labelled probe specific for the trnl/trnA flanking region.(c) Total RNA ( $1 - 5 ~ \mu \ g )$ Was hybridized with a DlG-labelled TrbcL probe.Northern blotswereoleditiFt indicated amOUnts of total protein.FPS,FARNESYL DIPHOSPHATE SYNTHASE;SQS,SQUALENE SYNTHASE.

Inflorescences of engineered lines were more clustered than those of WT (Figure 2d),and many buds aborted prematurely (Figure 2e)；these buds contained tiny,underdeveloped stigmas and desiccated petals and stamens (Figure 2e). In CN flowers, stamens were on long,curved filaments that produced minimal pollen (Figure 2f).By contrast, Nand C flowers displayed stamens on straight filaments with anthers that contained noticeably more pollen (Figure 2g,h),although N filaments were generally longer and anthers contained more pollen than C (compare Figure 2g with Figure 2h). Unlike in engineered lines,WT stamens were the same length as stigmas,and anthers produced abundant pollen (Figure 2i).

(a) (b)   
火 WT N CN WT N CN C   
(c)   
武 WT N CN C   
(e) (f） （g） CN (h) i

Notably，because of differences in antibiotic selection when plantswere initially grown (WT and N on antibiotic-free media and CN and C on spectinomycin-containing media),it is more meaningful to compare CN with C and WT with N.Given this, differences between CN and C are quite striking; despite both expressing high levels of Flag-SQS,CN had much more severe phenotypes.Similarly,N's phenotypes were as clear as those of CN and C and occurred despite expressing comparatively little Flag-SQS.

# Evaluation of metabolic profiles of transgenic/ transplastomic lines expressing FPS and SQs

Although our original intention was to produce transplastomic plants that accumulated squalene,phenotypes of all three lines suggested that more than just squalene levels might be altered. Therefore,we used nodal cutting to grow all lines in vitro under identical physiological conditions and harvested the third youngest leaf from plants at an identical developmental stage to analyse global metabolic profiles.All transplastomic plants were from the

TO generation.The metabolomic analysis used gas chromatography/mass spectrometry (GC/MS)and two types of ultra highperformance liquid chromatography/tandem mass spectrometry (UHPLC/MS/MS)，one optimized for acidic species and one optimized for basic species (Clarke et al.，2013；Evans et al., 2009).Principal component analysis(PCA）of quadruplicate samples showed tight clustering within each line (Figure 3a). PCA also showed that metabolic profiles of CN and C were distinct from N and WT but were almost indistinguishable from each other,whereas N showed a small but clear separation from WT (Figure 3a).

Metabolomic-based analysis of squalene levels indicated that despite up to 4OoO-fold higher Flag-SQS levels in CN and C than in N (Figure 1d),all three lines accumulated approximately 150- fold more squalene than WT (Figure 3b).As predicted, however, the abundance of dozens of other metabolites changed statistically significantly in all three engineered lines,and they encompassed multiple functional categories,including amino acids, carbohydrates，nucleotides and lipids (Figure 3c，Table S1).

![](images/3a24753de996ebd2ef641a3b5ecd91cb7295dea20cbaf6d8548d35f44756dff4.jpg)  
Figure3 Impact of FPS/SQS expression on levels of cellular metabolites.(a) PCA showing clustering of quadruplicate samples from each line. (b) Boxplot showing squalene levels. (c) Heat map showing functional categories of metabolites. Each box represents one sample. (d) Graphs showing the accumulation of selected metabolites compared with WT,including aromatic amino acidderivatives (left) and TCA cycle-related metabolites (right). Error bars indicate SEM. Abbreviations for statistical analysis are clustered in groups of three and are ordered N, CN,C.a, statistically significant increase over WT;b, statistically significant decrease relative to WT;-, difference with WT is not statistically significant. In the left panel, three letter abbreviations are those of amino acids;Phe.a,phenethylamine;Trp.a, tryptamine;Tyr.a, tyramine.FPS,FARNESYL DIPHOSPHATE SYNTHASE;PCA,principal Component analysis;SQS,SQUALENE SYNTHASE; WT,wild-type.   
$\circledcirc$ 2016ta

Notably，when ametabolite'sabundancewassignificantly changed in all three lines, the direction of change was generally identical (Figure 3c,Tables 1and S1).

In both CN and C，approximately 120 metabolites changed significantly (Figure 3c,Table S1),and several accumulated more than10-fold (Figure 3c,d,Table 1).Similarly，a transplastomic line in the Petit Havana (PH) background expressing only SQS significantlyaffected the abundance of nearly 1oo metabolites (Table S2).Of these，the most notable was nicotine，which accumulated to nearly 11-fold higher levels than in untransformed plants (Table S2).Among those metabolites that accumulated in plants engineered in the WT 1o68 background, three —tryptamine,tyramine and phenethylamine—increased without a corresponding increase in their aromatic amino acid precursors (Figure 3d，Table1).Two TCA cycle-associated metabolites, citrate and succinate，accumulated to 15.2-fold and fourfold higher levels,respectively,in CN than in WT (Figure 3d,Table 1). In C，these metabolites accumulated to 5.4-fold and twofold higher levels than in WT (Figure 3d,Table1).We also noted a concomitant reduction in photosynthetic Calvin cycle outputs fructose 6-phosphate (repressed up to fivefold)and glucose 6-phosphate (repressed up to 6.25-fold) (Table S1).In N，the abundance of approximately 65 metabolites changed statistically significantly (Figure 3c，Table S1)，but with the exception of squalene (Figure 3b)，they changed only two- to threefold at most (Figure 3c,Tables 1 and S1).

# The impact of FPS and SQS expression on the transcription of nuclear genes

The metabolic changes and physiological phenotypes in all three lines prompted us to investigate transcriptional mechanisms that underlie these changes.Therefore,we performed poly(A)-selected RNA sequencing (RNA-seq) on leaf tissue from plants of each line. Leaf material was distinct from that used for the metabolomic analysis but was grown inan identical manner and harvested atan identical developmental stage,with transplastomic plants in the T0 generation.Importantly，poly(A)selection enriches for nuclearencoded transcripts and depletes chloroplast transcripts，which lack an extended poly(A) tail. High-throughput sequencing reads were mapped to the tobacco genome，and transcripts were identified and quantified as previously described (Sierro et al., 2014).Hierarchical clustering analysis of expression counts for predicted transcripts demonstrated that expression patterns of each replicate for the three different transgenic lines were similar (Figure 4a).PCA showed close clusteringwithin lines (Figure 4b).

Unlike the metabolomic analysis,which showed N clustering more closely with WT and transplastomic CN and C lines clustering together (Figure 3a),transcriptomic analysis showed that overall gene expression profiles of each line were fairly separate. However, transcriptomes of engineered lines were more similar to each other than to WT (Figure 4b).

To annotate predicted transcripts,we first identified likely open reading frames (ORFs) and then used BLASTP with the NCBI Nicotiana database.This analysis led to many transcripts being assigned‘unknown'or‘uncharacterized'annotations or given no annotation.Additionally，multiple transcripts were often annotated to the same gene.Therefore,to increase the number of uniquely annotated ORFs，we used BLASTP with the TAIR10 version of the Arabidopsis transcriptome and assigned Arabidopsis-based annotations to the predicted transcripts.This analysis permitted more detailed functional categorization of each transcript and allowed the assignment of the most closely matched annotation to each transcript model.

At the $9 9 . 9 \%$ confidence level，more than twice as many transcripts were differentially expressed (DE) in CN:19 076 compared with 7784 in N and 5224 in C. First,we found a 4300-fold increase in FPS expression and a 72oo-fold increase inSQS expression in N compared with WT. In CN，FPS expression increased 36oo-fold and nuclear transgenic SQs expression increased 1300-fold relative to WT. Next，we determined which transcripts were most highly changed in each line compared with WT，ranked them by overall fold change independent of the line(s)，then identified the fold change in other two lines.Despite identifying DE transcripts in aline-independent manner， the majority of most highly induced(Table 2） andmosthighly repressedtranscripts (Table 3)were common to all three lines and changed in the same direction．Strikingly，nearly half of most highly induced transcripts (9/2O)were predicted to encode transporters of varying functions,including transporters for sucrose (SWEETs), phosphate， metals and auxin． In addition，transcripts with predicted roles in growth and development，metabolism and defence were induced. Consistent with these annotations, Gene Ontology (GO) analysis of biological process-related terms showed enrichment for terms related to stress and stimulus responses (Figure 4c).We also observed enrichment for transcriptsinvolved in secondarymetabolism(Figure 4c)． GO analysis of cellular component- and molecular function-specific termsshowedslight enrichmentforphotosynthesis-related components and processes (Figures S2 and S3).

Table1 List of metabolites whose abundance changes at least 10-fold compared with wild-type (WT)   

<html><body><table><tr><td>Metabolite</td><td>Category</td><td>Family</td><td>N/WT</td><td>CNAWT</td><td>CAWT</td></tr><tr><td>Squalene</td><td>Secondary metabolism</td><td>Terpenoids</td><td>154.10*</td><td>158.57*</td><td>158.86*</td></tr><tr><td>3-(2-piperidinyl)pyridine</td><td>Secondary metabolism</td><td>Alkaloids</td><td>-1.17</td><td>32.16*</td><td>31.82*</td></tr><tr><td>Pheophorbide A</td><td>Cofactors,prosthetic groups,electron carriers</td><td>Chlorophyll and haem metabolism</td><td>1.00</td><td>25.52*</td><td>2.58</td></tr><tr><td>Gluconate</td><td>Carbohydrate</td><td>Amino sugar and nucleotide sugar</td><td>1.55</td><td>24.85*</td><td>10.42*</td></tr><tr><td>Citrate</td><td>Carbohydrate</td><td>TCA cycle</td><td>1.13</td><td>14.16*</td><td>5.41*</td></tr><tr><td>Tryptamine</td><td>Amino acid</td><td>Aromatic amino acid metabolism (PEP derived)</td><td>1.76*</td><td>9.35*</td><td>13.48*</td></tr><tr><td>4-hydroxybutyrate (GHB)</td><td>Amino acid</td><td>Glutamate family (alpha-ketoglutarate derived)</td><td>-1.75</td><td>12.54*</td><td>-1.50</td></tr><tr><td>Tyramine</td><td>Amino acid</td><td>Aromatic amino acid metabolism (PEP derived)</td><td>-1.05</td><td>11.78*</td><td>11.21*</td></tr><tr><td>Phenethylamine</td><td>Amino acid</td><td>Aromatic amino acid metabolism (PEP derived)</td><td>1.57*</td><td>11.11*</td><td>11.55*</td></tr><tr><td>1-palmitoyl-GPI (16:0)</td><td>Lipid</td><td>Phospholipid</td><td>1.01</td><td>11.27*</td><td>3.93*</td></tr></table></body></html>

\*Statistically significant difference in the accumulation of the indicated metabolite compared with WT.

![](images/6272b287b854d4b93b0fef009c3b55694d786192145ca0674e9d368fc246112b.jpg)  
Figure4easfpnts.aeracalusteralyisd)Aplemes.atoie biological processes for up-and down-regulated transcripts in each line.Indicated values are $- \mathsf { l o g } _ { 1 0 }$ of the Benjamini adjusted p-value.(d,e) RNA-seq data fortrasritfrfsadstrssesis cofiatifddi error bars represent mean $\pm \mathsf { S E M }$ of two biologicalreplicates withthree technicalreplicateseach.FPS,FARNESYLDIPHOSPHATESYNTHASE;PCA,principal compOnent analysis; SQS,SQUALENE SYNTHASE;WT,wild-type.

Fora few transcripts，for example SWEET1 (transcript 10), expression was induced similarly across all three lines (Table 2). However，for many other induced transcripts，although the largest change in expression differed，the greatest induction occurred most often in CN.For example,COPT1 (transcript 6) and PIN5 (transcript 8)were both induced approximately 3oo-fold in CN,whereas in N,COPT1 was induced only20-fold and PiN5was not significantly changed; and in C,COPT1 was induced 70-fold and PiN5 was induced 5O-fold.By contrast,only one transcript wasmost highly induced each in C (Nicotiana tomentosiformis NCED4，transcript 15）and N (Nicotiana sylvestris TONOPLAST DICARBOXYLATE TRANSPORTER，transcript 17)，aIthough the N.tomentosiformis 14-kDa protein (a lipid transporter,transcript 1）was also induced very highly in N—more than 800-fold (Table 2).Additionally,a subset of transcripts Was induced to a similar level in both N and CN,including the N. tomentosiformis

Table2Topoveallostiglinducedtransriptsdteirexprsioiothrttrasgicrsplastomilinsroupedyfuctioal categoryWhenposibleachcusinudesaaotatiforNoatomentosifois(NtadaaotatiforNotiaasystriss Values without asterisks indicate $9 9 . 9 \%$ confidence.Cells without values were not significantly changed for the indicated transcript in the indicated line   

<html><body><table><tr><td colspan="7">imiuicatedlne</td></tr><tr><td></td><td>RankNCBl gene ID/At number</td><td>Nicotiana/Arabidopsisannotation</td><td>N/Vild-type (WT)</td><td>CN/WT</td><td>C/WT</td></tr><tr><td colspan="6">Transporters</td></tr><tr><td>1</td><td>LOC104087257/At4g12510</td><td>Nt 14kDa PROLINE-RICH PROTEIN DC2.15-like/2S albumin superfamily protein</td><td>806.987</td><td>940.608</td><td>172.152</td></tr><tr><td></td><td>LOC104242032/At4g12510</td><td>Ns14 kDa PROLINE-RICH PROTEIN DC2.15-like/2S albumin superfamily protein</td><td>73.826</td><td>56.626</td><td>18.003</td></tr><tr><td>4</td><td></td><td>LOC104088673/At3g48740Nt N3 SUGAR TRANSPORTER-like/SWEET11</td><td>87.235</td><td>201.150</td><td>65.779</td></tr><tr><td></td><td>LOC104243017/At3g48740</td><td>Ns SWEET12-like/SWEET11</td><td>148.741</td><td>327.890</td><td>72.298</td></tr><tr><td>5</td><td></td><td>LOC104115159/At3g54700Nt probable INORGANIC PHOSPHATE1-7/PHT1;7</td><td>11.423*</td><td></td><td>15.762</td></tr><tr><td></td><td></td><td>LOC104216800/At3g54700Ns probable INORGANIC PHOSPHATE TRANSPORTER 1-2/PHT1;7</td><td>10.502*</td><td>323.984</td><td>299.250</td></tr><tr><td>6</td><td></td><td>LOC104086110/At5g59030Nt COPPER TRANSPORTER 1-like/COPT1</td><td>20.421</td><td>318.867</td><td>55.805</td></tr><tr><td></td><td>LOC104244096/At5g59030</td><td>Ns COPPER TRANSPORTER1-like/COPT1</td><td>23.937</td><td>271.475</td><td>68.523</td></tr><tr><td>8</td><td></td><td>LOC104084825/At5g16530Nt putative AUXIN EFFLUX CARRIER COMPONENT 8/PIN5</td><td></td><td>299.984</td><td>48.512</td></tr><tr><td></td><td></td><td>LOC104249472/At5g16530Ns putative AUXIN EFFLUX CARRIER COMPONENT8/PIN5</td><td></td><td>201.077</td><td>32.503</td></tr><tr><td></td><td>10LOC104086686/At1g21460 Nt SWEET1-like/SWEET1</td><td></td><td>248.285</td><td>220.603</td><td>259.497</td></tr><tr><td></td><td>LOC104225662/At1g21460</td><td>Ns SWEET1-like/SWEET1</td><td>63.264</td><td>79.362</td><td>57.860</td></tr><tr><td></td><td></td><td>12LOC104101946/At5g59520Nt ZINC TRANSPORTER 2/ZIP2</td><td></td><td>32.226</td><td></td></tr><tr><td></td><td></td><td>LOC104211888/At5g59520Ns ZINCTRANSPORTER 2/ZIP2</td><td>13.010</td><td>249.781</td><td>15.498</td></tr><tr><td></td><td></td><td>14LOC104096904/At5g53190Nt bidirectional sugar transporter SWEET3b/SWEET3</td><td></td><td>182.521</td><td>140.590</td></tr><tr><td></td><td></td><td>LOC104241809/At5g53190Ns bidirectional sugar transporter SWEET3/SWEET3</td><td></td><td>236.357</td><td>107.210</td></tr><tr><td></td><td>17LOC104118881/At5g47560</td><td>Nt TONOPLASTDICARBOXYLATETRANSPORTER/TDT</td><td>37.066</td><td>9.426</td><td></td></tr><tr><td></td><td>LOC104242854/At5g47560</td><td>NsTONOPLASTDICARBOXYLATETRANSPORTER/TDT</td><td>221.581</td><td>33.801</td><td></td></tr><tr><td colspan="6">Growth and development</td></tr><tr><td>2</td><td></td><td>Nt CEN-like 1/BFT</td><td></td><td>538.996</td><td></td></tr><tr><td></td><td>LOC104105402/At5g62040</td><td>Ns CEN-like 1/BFT</td><td></td><td>431.913</td><td>14.363</td></tr><tr><td>3</td><td>LOC104229471/At5g62040</td><td>LOC104089336/At5g10625Nt FLOWERING PROMOTING FACTOR 1-like 3/FPF1-like</td><td>115.961</td><td>356.184</td><td></td></tr><tr><td></td><td></td><td>Ns FLOWERING PROMOTING FACTOR 1-like 3/FPF1-like</td><td></td><td>440.531</td><td>16.807</td></tr><tr><td>7</td><td>LOC104232534/At5g10625</td><td>Nt PEROXIDASE 15/PA2</td><td>20.544</td><td></td><td>12.179*</td></tr><tr><td></td><td>LOC104104199/At5g06720</td><td>Ns LIGNON-FORMING ANIONIC PEROXIDASE-Iike/PA2</td><td></td><td>312.576</td><td>80.377</td></tr><tr><td></td><td>LOC104217681/At5g06720</td><td>13LOC104243032/At4g25820Ns XYLOGLUCAN ENDOTRANSGLUCOSYLASE/HYDROLASE 24-Iike/XTR9</td><td>197.748</td><td>81.758 242.975</td><td></td></tr><tr><td colspan="6">Secondary metabolism</td></tr><tr><td>9</td><td>LOC104099208/At4g14090</td><td>Nt CROCETIN GLUCOSYLTRANSFERASE/ANTHOCYANIDIN-5-O-</td><td>94.367</td><td>272.367</td><td>18.904</td></tr><tr><td></td><td></td><td>GLUCOSYLTRANSFERASE LOC104227958/At4g14090Ns CROCETIN GLUCOSYLTRANSFERASE/ANTHOCYANIDIN-5-O-</td><td>9.567</td><td>12.620</td><td></td></tr><tr><td></td><td></td><td>GLUCOSYLTRANSFERASE</td><td></td><td></td><td>4.611</td></tr><tr><td></td><td></td><td>15LOC104104691/At4g19170Nt CAROTENOID CLEAVAGE DIOXYGENASE4/NCED4</td><td>131.189 214.326</td><td>101.045</td><td>234.293</td></tr><tr><td></td><td>19LOC104107213/At4g15480</td><td>Nt CINNAMATE BETA-D-GLUCOSYLTRANSFERASE/UGT84A1</td><td></td><td>200.087</td><td>142.143</td></tr><tr><td>Hormone</td><td></td><td>Nt GIBBERELLIN-REGULATED1-Iike/GASA1</td><td>58.475</td><td></td><td></td></tr><tr><td colspan="6">11LOC104088764/At1g75750</td></tr><tr><td></td><td>Primary metabolism</td><td></td><td></td><td>256.662</td><td>11.149*</td></tr><tr><td></td><td>16LOC104096882/At5g48850Nt uncharacterized/SDl1</td><td></td><td>38.021</td><td>227.743</td><td></td></tr><tr><td colspan="6">Cytoskeleton</td></tr><tr><td></td><td></td><td>18LOC104116945/At1g70810Nt probable ADP-ribosylation factor GTPase-activating protein AGD11/CAR7</td><td>9.630</td><td>91.183</td><td>12.693</td></tr><tr><td></td><td></td><td>LOC104249050/At1g70810Ns probable ADP-ribosylation factor GTPase-activating protein AGD11/CAR7</td><td>52.144</td><td>216.566</td><td>32.270</td></tr><tr><td>Defence</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>20LOC104103886/At2g38870</td><td>Nt PROTEINASE INHIBITOR I-B/PR6-like</td><td>79.968</td><td>100.779</td><td>23.172</td></tr><tr><td></td><td>LOC104231145/At2g38870</td><td>Ns PROTEINASE INHIBITOR I-A/PR6-like</td><td>131.969</td><td>210.761</td><td>35.884</td></tr></table></body></html>

\*Value reported at the $9 9 . 0 \%$ confidence level.

14-kDa protein (transcript 1,900-fold in CN and 800-fold in N) and both N.tomentosiformisand N.sylvestris PR6-likeproteinase inhibitor (transcript 20,100-200-fold in CN and ${ \sim } 1 0 0$ -fold in N) (Table 2).

Table 3 lists the 2O most highly repressed transcripts,with more than half (12/2O) predicted to function in the cell cycle,cell wall remodellingand DNA remodelling,and additional functional categories included kinases，metabolism and growth and development.GOanalysisfor‘biologicalprocess'-specific terms confirmed substantial enrichment among repressed transcripts for those with predicted functions in the cell cycle (Figure 4c). GO analysis for 'cellular component'-related terms showed enrichment for cell wall,，chromosomes and cytoskeletal components among down-regulated transcripts (Figure S2),and GO analysis for‘molecular function'-related terms showed enrichment for transcripts predicted to be involved in binding nucleosides and cytoskeletal proteins (Figure S3).

Table3Tpoverallosthglepresedtransciptsdteirexpressthertotrasenic/transplastomicinesroupeductioal categoryWhenpibleachlocusinudesaaotatiforNotianatmentosifois(tndaotatioforcotiantri)   

<html><body><table><tr><td>Rank NCBl gene ID/At number</td><td></td><td>Nicotiana annotation/Arabidopsis annotation</td><td>N/Wild-type (WT)</td><td>CNAWT</td><td>CAWT</td></tr><tr><td>Cell cycle</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1</td><td>LOC10409419/At1g30690 Nt PATELLIN-4/Sec14-like</td><td></td><td>-816.166</td><td>-263.187</td><td>-2262.789</td></tr><tr><td></td><td>LOC10421949/At1g30690</td><td>Ns PATELLIN-4/Sec14-like</td><td>-493.844</td><td>-746.213</td><td>-666.637</td></tr><tr><td>7</td><td>LOC10408601/At5g06150</td><td>Nt CYCLIN S13-7-like/CYCB1;2</td><td>-726.831</td><td>-364.512</td><td>-559.138</td></tr><tr><td></td><td>LOC10423757/At5g06150</td><td>Ns CYCLIN S13-7-like/CYCB1;2</td><td>-1052.573</td><td>-524.124</td><td>-1325.489</td></tr><tr><td></td><td>10LOC10421173/At4g11080</td><td>Ns HIGH MOBILITY GROUP B 6-like/3XHMG-BOX1</td><td>-531.236</td><td>-317.492</td><td>-1124.502</td></tr><tr><td>11</td><td></td><td>LOC10424297/At1g44110 Ns CYCLIN-A1-4/CYCA1;1</td><td>-257.417</td><td>-401.190</td><td>-989.323</td></tr><tr><td></td><td>12 LOC10409411/At3g20150</td><td>Nt kinesin-like KIN12A/kinesin family protein</td><td>-732.380</td><td>-361.282</td><td>-918.263</td></tr><tr><td></td><td>LOC10424399/At3g20150</td><td>Ns kinesin-like KIN12A/kinesin family protein</td><td>-368.772</td><td>-322.527</td><td>-453.958</td></tr><tr><td></td><td>13LOC10411222/At1g03780</td><td>Nt TPX2-like/TPX2</td><td>-584.023</td><td>-504.647</td><td>-308.380</td></tr><tr><td></td><td>LOC10421712/At1g03780</td><td>Ns TPX2-like/TPX2</td><td>-915.733</td><td>-790.737</td><td>-694.173</td></tr><tr><td></td><td>15LOC10410080/At2g26760</td><td>Nt CYCLIN S13-7-like/CYCB1;4</td><td>-682.727</td><td>-593.531</td><td>-850.024</td></tr><tr><td></td><td>LOC10422394/At2g26760</td><td>Ns CYCLIN S13-7-like/CYCB1;4</td><td>-248.516</td><td>-119.077</td><td>-306.997</td></tr><tr><td></td><td>16LOC10409883/At1g08560</td><td>Nt KNOLLE/KNOLLE</td><td>−654.447</td><td>-125.074</td><td>-298.701</td></tr><tr><td></td><td>LOC10422292/At1g08560</td><td>Ns KNOLLE/KNOLLE</td><td>-647.369</td><td>-163.412</td><td>-847.873</td></tr><tr><td>Cell wall/cuticle</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>3</td><td>LOC10411994/At4g22010</td><td>Nt L-ASCORBATE OXIDASE homologue/SKS4</td><td>-459.175</td><td>-660.733</td><td>-594.408</td></tr><tr><td></td><td></td><td>LOC10423150/At4g22010 Ns L-ASCORBATE OXIDASE homologue/SKS4</td><td>-481.000</td><td>-1001.534</td><td>−1473.924</td></tr><tr><td>6</td><td>LOC10410009/At5g47500</td><td>Nt probable PECTINESTERASE 68/PME5</td><td>-642.662</td><td>-370.989</td><td>-1355.422</td></tr><tr><td></td><td></td><td>LOC10421908/At5g47500Ns probable PECTINESTERASE 68/PME5</td><td>-345.366</td><td>-518.392</td><td>-831.930</td></tr><tr><td></td><td>19LOC10409611/At1g02205 Nt ECERIFERUM 1-like/CER1</td><td></td><td>-38.939</td><td>-91.297</td><td>-78.052</td></tr><tr><td></td><td>LOC10424030/At1g02205 Ns ECERIFERUM1-like/CER1</td><td></td><td>-199.464</td><td>-546.898</td><td>-799.481</td></tr><tr><td>Kinase</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>4</td><td></td><td>LOC10411606/At3g5174ONt probably inactive leucine-rich repeat receptor-like protein kinase IMK2/IMK2-1446.626</td><td></td><td>-764.462</td><td>-457.671</td></tr><tr><td></td><td></td><td>LOC10423705/At3g51740Ns probable leucine-rich repeat receptor-like protein kinase IMK3/MK3</td><td>-305.037</td><td>-265.833</td><td>-219.477</td></tr><tr><td></td><td></td><td>17LOC10409992/At5g43020 Nt probable inactive receptor kinase At5g67200/LRR kinase family</td><td>-168.644</td><td>-572.847</td><td>-841.587</td></tr><tr><td></td><td></td><td>LOC10423543/At5g43020 Ns probable inactive receptor kinase At5g67200/LRR kinase family</td><td>-32.928</td><td></td><td></td></tr><tr><td>Miscellaneous</td><td></td><td></td><td></td><td>-96.819</td><td>-44.635</td></tr><tr><td></td><td></td><td>18LOC10409693/At4g31840 Nt EARLY NODULIN-LIKE 1/ENODL15</td><td>-395.968</td><td>-118.288</td><td>-821.479</td></tr><tr><td></td><td></td><td>LOC10421187/At4g31840 Ns EARLY NODULIN-LIKE3/ENODL15</td><td>-343.122</td><td>-120.464</td><td>-623.076</td></tr><tr><td></td><td></td><td>20LOC10412083/At5g23420 Nt HIGH MOBILITY GROUP B 7-like/HMGB6</td><td>-500.882</td><td>-134.517</td><td>-476.056</td></tr><tr><td></td><td></td><td>LOC10424371/At5g23420Ns HIGH MOBILITY GROUP B 7-like/HMGB6</td><td>-262.125</td><td>-222.445</td><td>-790.240</td></tr><tr><td>Lipid metabolism</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>2</td><td></td><td>LOC10412153/At5g33370Nt GDSL esterase/lipase At5g33370-like/GDSL-motif esterase/lipase</td><td>-546.326</td><td>-2106.034</td><td>-1221.907</td></tr><tr><td></td><td></td><td>LOC10422041/At5g33370Ns GDSL esterase/lipase At5g33370-like/GDSL-motif esterase/lipase</td><td>-79.807</td><td>-93.596</td><td>-224.898</td></tr><tr><td>Secondary metabolism</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>5</td><td></td><td>LOC10408627/At5g66230Nt uncharacterized/chalcone-flavanone isomerase</td><td>-828.169</td><td>-136.904</td><td>-490.657</td></tr><tr><td></td><td></td><td>LOC10421950/At5g66230Ns uncharacterized/chalcone-flavanone isomerase</td><td>-686.380</td><td>-310.377</td><td>-1419.234</td></tr><tr><td>Growth and development</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>8</td><td></td><td>LOC10409319/At2g42840 Nt PROTODERMAL FACTOR 1-like/PDF1</td><td>-1003.212</td><td>-869.240</td><td>-1255.300</td></tr><tr><td></td><td></td><td>LOC10422232/At2g42840 Ns PROTODERMAL FACTOR 1-like/PDF1</td><td>-486.103</td><td>-145.233</td><td>-302.012</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>DNA remodelling</td><td></td><td>LOC10412078/At3g23890Nt DNA TOPOISOMERASE I-like/TOPII</td><td>-615.688</td><td>-1172.774</td><td>-442.749</td></tr><tr><td>9</td><td></td><td>LOC10424672/At3g23890Ns DNATOPOISOMERASE I-like/TOPII</td><td>-189.431</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>-287.318</td><td>-247.807</td></tr><tr><td>Cytoskeleton</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>14LOC10408502/At1g03470 Nt uncharacterized/KIP1-like</td><td></td><td>-688.519</td><td>-235.486</td><td>-866.163</td></tr><tr><td></td><td>LOC10422085/At1g03470</td><td>Ns uncharacterized/KIP1-like</td><td>-306.668</td><td>-252.734</td><td>-287.424</td></tr></table></body></html>

Without exception,each of the most highly repressed transcripts were repressed across all three lines,and there were some—for example the N.tomentosiformis TPX2-like gene (transcript13)and the N. sy/vestris KiP1-family gene (transcript 14)—whose expressionwas comparable in all three lines. However, there were other transcripts for which the magnitude of change did vary.Strongest repression was often found in C,as in the case of the N. tomentosiformis PATELLIN-4 (transcript1，repressed more than 2000- fold)and the N.sylvestris HMG PROTEIN6(transcript10,repressed 1000-fold). In other instances,as for a transcript annotated to a N.tomentosiformis GDSL-motifesterase/lipase (transcript 2) and fora N.tomentosiformis DNA TOPOISOMERASE I(transcript 9)，CN clearly showed the strongest repression:2ooo-fold and

1100-fold,respectively. CN also showed a unique enrichment for down-regulated transcripts predicted to be involved in reproductive development (Figure 4c).The N line also had a large effect on the expression of certain transcripts,most notably that for the N.tomentosiformis IMK2 kinase (transcript 4，repressed 1400- fold).Two other transcripts,N. sylvestris CYCLIN B1;2 and N. tomentosiformis PDF1 (transcripts7and 8,respectively)were also repressed more than 10oO-fold in N(Table 3).Interestingly,in cases in which a transcript was most highly repressed in two lines, the two with the strongest effect tended to be Nand C(e.g.for N. sy/vestris CYCLIN B1;2 and N. tomentosiformis kinesin KIN12A).

To further confirm the validity of RNA-seq data,we performed qRT-PCR for select induced and repressed transcripts using biologically independent samples at an identical developmental stage.All transcripts predicted to be induced by RNA-seq were verified to be induced by qRT-PCR (Figure 4d)，and with one exception (TOPl/ in N)，transcripts predicted by RNA-seq to be repressed were also repressed relative to WT in qRT-PCR (Figure 4e). Plotting the $\mathsf { l o g } _ { 2 }$ fold change of each transcript using qRT-PCR versus RNA-seq showed a good correlation between results from both methods,with R values ranging from O.690 to 0.899 (Figure S4).

# Discussion

To date,few studies on plant metabolic engineering have looked beyond the pathway of interest to assess unanticipated consequences，and no report has compared metabolomic and transcriptomic effects of expressing transgenes from different organellar genomes. Here，we demonstrate that expressing chloroplast-targeted FPS and/or SQS from the nuclear genome (N line),the chloroplast genome (C line) or both (CN line)has broad off-target effects on the metabolome and transcriptome. However,because many of the same metabolites and transcripts are changed in a similar direction acrossall three lines,FPS and SQS expression may cause broad but somewhat predictable offtarget effects.

Because the chloroplast genome promotes higher transgene expression than the nuclear genome (Verma and Daniell,2007), one goal of this study was to increase squalene production by transplastomically expressing codon-optimized FPS and/or SQS. However，neither transplastomic line accumulated significantly more squalene than N (Figure 3b,Tables 1 and S1),suggesting a substrate or storage limit. Consistent with this,in the background of PH，which has fewer glandular trichomes than 1068 (Wu et al.，2012)，we could generate plants expressing SQS only; transforming PH chloroplasts with pLD-FPS-SQS yielded shoots that did not survive，and similar problems were encountered using nuclear transformation constructs that targeted FPS and SQS to chloroplasts. Inadequate trichomes (sinks) in PH may have caused this,perhaps through feedback inhibition.Indeed, the 2- C-methyl-D-erythritol 4-phosphate (MEP) pathway of terpene biosynthesis is under feedback inhibition by its own products, dimethylallyl diphosphate (DMAPP) and isopentenyl diphosphate (IPP) (Banerjee et al.,2013),and IPP is the substrate of FPS.One strategy to increase squalene could be to increase sink capacity by sequestering squalene in storage compartments such as lipid droplets (Wang et al.，2015）or additional trichomes (Lange, 2015).Trichome density could be increased via transgenic expression of $\beta$ -glucosidase (Jin et al.，2011)，which improves artemisinin yieldsin transgenic Artemisia annua (Singh et al., 2015).

Among the 65-120 metabolites changed across engineered lines,in transplastomic lines,we noted an increase in TCA cyclerelated metabolites,including citrate and succinate (Figure 3d) anda reductioninfructose 6-phosphate and glucose 6-phosphate (Table S1). Fructose 6-phosphate and glucose 6-phosphate are both photosynthetic outputs from the Calvin cycleand are synthesized from glyceraldehyde 3-phosphate，a three-carbon Calvin cycle output. In chloroplasts， squalene biosynthesis is initiated by the conversion of glyceraldehyde 3- phosphate and pyruvate into 1-deoxy-D-xylulose-5-phosphate in a committed step catalysed by 1-deoxy-D-xylulose-5-phosphate synthase.Therefore，the 15o-fold increase in squalene levels and the decrease in fructose 6-phosphate and glucose 6- phosphate biosynthesis observed in engineered lines suggest that glyceraldehyde 3-phosphate is repartitioned from the synthesis of sugars into the production of terpenes (Wang et al.,2015).

The chloroplast amino acid pool is limited (Bally et al.,2009), and chloroplast transgene products are produced at the expense of resident proteins such as Rubisco (Bally et al.,2O11;Ruhlman et al.，2010).Therefore，given the accumulation of Flag-SQS in transplastomic lines (Figure 1d)，it isnot surprising that amino acid metabolism was altered in CN and C (Figure 3c and d, Table1). Decarboxylated aromatic amino acids accumulated to at least eightfold higher levels in CN and C than in WT without an increase in their corresponding precursors (Figure 3d,Table 1). This result demonstrates that transplastomic lines are not impairedin amino acid biosynthesis and suggests the diversion of carbon into squalene biosynthesis, leading to an imbalance between carbon and nitrogen that explains the accumulation of these amino acid derivatives.Additionally，the abundance of variousnucleotideswasaffected (Figure 3c;Table S1)，possibly due to large quantities of $F P S$ and/or SQS mRNA.

Transcriptomic analysis revealed thousands of DE transcripts between each line and WT (Figure 4a). Induction of transporterrelated transcripts (Table 2） may reflect the need to move additional metabolites from sites of synthesis to sites of use or storage.Additionally， highly repressed cell cycle transcripts (Table 3，Figure 4c) correlated well with growth phenotypes (Figure 2c). Importantly，all most highly DE transcripts were changed in the same direction acrossall three lines (Tables2and 3，Figure 4d,e)，regardless of the compartment(s) from which transgenes were expressed.

Wealso observed differences in transcriptomes that varied with the compartment(s) from which transgenes were expressed, including the number and magnitude of change of DE transcripts. Importantly， transcriptomic data also showed no correlation between transgene expression and the effect on the expression of nuclear-encoded genes; the N line expressed up to 4oo0-fold less Flag-SQS than transplastomic lines (Figure 1d)but affected more nuclear-encoded transcripts than C， and it had the strongest influence on the expression of certain transcripts (Tables 2 and 3).Additionally，when a transcript was most strongly affected in two lines,one of those lines was generally N. Therefore,alterationsin gene expressionare likely the result of metabolites and not transgene products. Indeed, levels of FlagSQS did not vary much between transplastomic lines,but the effect on nuclear-encoded genes was quite significant.

The number of DE transcripts reported here is considerably greater than those reported in previous studies (Ricroch et al., 2011).For example，transgenic rice expressing choline oxidase showed only 165 DE transcripts between engineered and parental lines (Kathuria et al.,2oo9). In transgenic potatoes with elevated or knocked down expression of sucrose synthase,50 and 357DE genes were found,respectively (Baroja-Fernandez et al., 2009).In transgenic rice expressinganthranilate synthase,after correcting for variation between samples,only 22 genes met authors' criteria for differential expression (Dubouzet et al., 2007).Compared with DNA microarrays,the greater sensitivity of RNA-seq may facilitate the detection of far more transcriptomic changes (Wang et al.，2oo9)，and transgene expression from different cellular compartments likely also increased the number of DE transcripts (discussed below).

One potential explanation for strong phenotypes observed in CN is disparate levels of FPS and SQS transgene products. Transplastomic CN and C lines accumulate at least 2ooo-fold more Flag-SQS than N (Figure 1d),and expression of FPS from the nuclear genome alone may not produce enough transgene product to match levels of SQS in CN (Figure 1c).Therefore, phenotypes observed in CN may be mitigated by incorporating FPS into the chloroplast genome.Similarly,in transplastomic PH plants expressing only SQS， the abundance of dozens of metabolites changed by ${ > } 1 0$ -fold (Table S2)．Notably，one of these metabolites was nicotine，which accumulated to nearly 11-fold higher levels than in untransformed plants (Table S2), supporting the idea that metabolic imbalance can result in substantial off-target effects on entirely unrelated pathways.

Because of transgene compartmentalization，transplastomic plants are excellent tools to study retrograde signalling.In CN, because the parental N line only contained approximately 8000 DEtranscripts，additional 11 Ooo DE transcripts mayhave changed due to retrograde signalling.In the C line，because transgenes and resulting products are totally contained within chloroplasts,all changes in the expression of the more than 5000 nuclear-encoded transcripts must be the result of retrograde signalling.

Although our data do not permit us to identify the retrograde signal,it is likely not squalene;otherwise,similar squalene levels (Figure 3b)would cause similar changes in gene expression. However,not only the number of DE transcripts differed,but also theirexpression levels (Tables2and3).The signal may therefore be a different metabolite or metabolites,perhaps one affected by the imbalance between SQS and FPS levels in CN (Figure 3c, Table S1),but investigation of this possibility is beyond the scope of this study. Interestingly，an upstream molecule in the MEP pathway，methylerythritol cyclodiphosphate (MEcPP)，acts asa retrograde signalling molecule under high light and wounding; these stresses induce accumulation of MEcPP to \~two- to threefold higher levels than in controls (Xiao et al.，2012). Altered flux through the MEP pathway may change the abundance of MEcPP，thereby causing at least a subset of observed transcriptomic changes in engineered plants.The effect of FPS and SQS expression could extend downstream of MEcPP andparallel to farnesyl diphosphate (FPP)and/or squalene,and these metabolites may also serve as retrograde signals. Both IPP and DMAPP are precursors for the synthesis of carotenoids such as $\beta$ -carotene (Nisar etal.，2015)，and a volatile $\beta$ -carotene derivative, $\beta \mathrm { . }$ -cyclocitral，has been proposed asa stress-induced retrograde signalling molecule (Ramel et al.，2012). Increased demand for IPPbyFPSorfor FPPby SQS mayaffect carotenoid metabolism，and，consequently， $\beta$ -cyclocitral levels，causing altered retrograde signalling.Although none of these metabolites weresignificantly changed between WT and engineered lines in ouranalysis，it does not exclude the possibility that the abundance of one or more of them was altered，leading to changes in phenotype and gene expression.If such a difference had occurred, small changes needed to induce signalling (e.g. for MEcPP)and differences in detection and analytical methods may haveexcluded these metabolites from analysis,especially if differences were statistically insignificant.

Gene products have also been suggested to act as highly specific retrograde signallingmolecules，and transgenes engineered via the chloroplast genome that are unrelated to metabolism can influence the expression of nuclear genes. For example，chloroplast expression of Arabidopsis TIC40 or γ- tocopherol methyltransferase has been shown to promote massive proliferation of the chloroplast inner membrane and the up-regulation of associated nuclear-encoded inner membrane proteins without affecting nuclear-encoded proteins that are targeted to the outer or thylakoid membranes (Jin and Daniell, 2014; Singh et al.,2oo8).Additionally, proteins have been shown to be released from intact chloroplasts (Kwon et al.，2013b), providing yet another observation in support of the idea that proteins themselves can be retrograde signals.Therefore，the retrograde signal(s) acting in this case could be a metabolite or a gene product.

Previous studies on genetically modified plants have reported that environmental factors and cultivar-specific differences play largerrolesinaltering the transcriptomeand metabolome than does transgene expression (Baker et al.，2oo6；Clarke et al., 2013；Kogel et al.，2010;Ricroch et al.，2011)．FPS and SQS expression may cause clear transcriptomic，metabolomicand phenotypic changes because they are metabolism related.For example,transplastomic tobacco expressing twelve genes for the biosynthesis of artemisinic acid (AA)accumulated only approximately $0 . 1 \mathrm { \ m g / g }$ fresh weight (FW)AA，but growth was still reduced (Saxena et al.，2014)，and nuclear transgenic plants accumulating linalool and nerolidol accumulated $1 . 5 ~ \mu \ g / \ g$ FWof these terpenes but still displayed delayed growth (Aharoni et al., 2003).These observations may reflect the partitioning of carbon away from biomassand into the desired metabolite (Melis,2013). By contrast,transgenic/transplastomic expression of genes that areunrelated to metabolism has few to no phenotypic effects and,in the case of transplastomic plants,can often be rescued by adding exogenous nitrogen (Bally et al.，2009，2011;De Cosa et al.，2001； Ruhlman et al.，2010;Vitanen et al.，2004). Therefore，metabolite engineering and not protein engineering likely accounts for these phenotypes.

Using cutting-edge -omics technology，we show that metabolic engineering via the nuclear and/or chloroplast genomes can result in broad off-target effects in the metabolomeand transcriptome.However,these effects may be predictable and can therefore be minimized or exploited.By focusing on global consequences of metabolic engineering rather than simply searching for another individual signalling molecule,our results provide a unique,holistic view of metabolite-mediated intercompartmental signalling that can be used as a framework for future studies on both metabolic engineering and metabolite-mediated anterograde and retrograde signalling.

# Experimental procedures

Vector construction,plant transformation and characterization via PCR,Southern blot,northern blot andwestern blot

The WT 1068 introduction and the N nuclear transgenic line were as previously described (Wu et al.,2012).All N plants were homozygous siblings derived from the same transformation event.To construct the vectors used to generate transplastomic lines,DNA sequences encoding Flag-tagged yeast SQS (GenBank accession NM001179321) and 4xHis-tagged avian FPS (GenBank accession Po8836) were codon-optimized for enhanced chloroplast expression (Daniell et al.，2oo9) and synthesized by Integrated DNA Technologies (Coralville， IA).The optimized Flag-SQS coding sequence was cloned into the Ndel and Xbal sites of pLD-ctv under the control of the psbA promoter, $5 ^ { \prime }$ -UTR and $3 ^ { \prime }$ -UTR (Daniell et al.，2005).To construct pLD-FPS-SQS,an intermediate vector was assembled that contained the Prrn-g10/ His-FPS/TrbcL cassette.This vector Was digested with Sall，and the released cassette was ligated into Sa/l-digested pLD-SQS. Both vectors were confirmed by sequencing. Chloroplast transformation of WT with pLD-FPS-SQS and of N with pLD-SQS was performed as described previously (Verma et al.,2oo8). PCRand western blot of putative transplastomic plants were carried out as described (Kwon et al.,2013a; Verma et al.,2008).Western blot was performed using an anti-DYKDDDDK antibody (LifeTein， SouthPlainfield， NJ) ata $1 : 1 0 0 0$ dilutionanda horseradish peroxidase-conjugated anti-mouse secondary antibody (Southern Biotech，Birmingham，AL，USA）at a $1 : 4 0 0 0$ dilution.Densitometric analysis was carried out using the gel analysis feature of ImageJ，Bethesda，MD，USA.Southern blot was performed using the DIG High Prime DNA Labeling and Detection Kit (Roche Diagnostics, Indianapolis, IN) according to the manufacturer's instructions.In brief, $0 . 5 ~ \mu \ g$ of total tobacco DNA was digested with Af/lll or Hindlll and resolved on a $0 . 8 \%$ agarose gel.The DNA was blotted onto a positively charged nylon membrane (Nytran SPC；GE Healthcare，Marlborough, MA).Hybridization with DIG-labelled probe for the trn/-trnA flanking region (Figure 1a) was conducted at $4 1 ^ { \circ } \mathsf C$ in a UVP HB-1000 hybridizer (UVP LLC，Upland,CA)，and signals were detected with CSPD substrate and $\mathsf { X }$ -ray film.Probe for northern blot analysis was synthesized using the DIG High Prime DNA Labeling and Detection Kit with a PCR product containing the rbcL $3 ^ { \prime }$ -UTR.Northern blot was performed using total RNA (see below) run on a $0 . 9 \%$ denaturing agarose gel. The RNA was blotted onto a Nytran membrane and hybridized with the probe at $4 2 ^ { \circ } C$ .Signals were detected as for the Southern blot. For phenotypic analysis of homoplasmic transplastomic lines，data are representative of at least two independent transformation events.

# Metabolomeanalysis

Samples were harvested from the third leaf of plants of the same developmental stage grown via nodal cutting in sterile tissue culture conditions.Harvested tissue was lyophilized in a Genesis lyophilizer (SP Scientific，Warminster，PA)，and quadruplicate samples were sent to Metabolon (Durham,NC).Samples were analysed using a platform consisting of GC/MS and two UHPLC/ MS/MS analyses,one optimized for acidic species and one for basic species (Clarke et al.，2013;Evans et al.，2009).Metabolomicdata are available in Tables S1and S2.

# RNA sequencing and analysis and GO analysis

mRNA-seq was performed as previously described (Elliott et al., 2013).Briefly,total RNA was purified from the third youngest leaf of two plants from each line at the same developmental stage grown in sterile tissue culture conditions using a miRNeasy kit (Qiagen,Valencia,CA).Poly(A) $^ +$ RNA was isolated using oligo(dT) beads(Thermo Fisher Scientific，Waltham，MA).RNAwas fragmented for 7 min using Fragmentation Reagent (Thermo Fisher Scientific)．mRNA-seq libraries were generated using an Illumina mRNA-seq kit (Illumina, San Diego, CA). Sequencing was performed at the University of Pennsylvania Next Generation Sequencing Core. Reads were trimmed with Cutadapt and mapped to the tobacco genome with Tophat2 (Kim et al., 2013;Martin，2011;Sierro et al.，2014;Trapnell et al.，2009). Cufflinks was used to predict transcripts (Trapnell et al.，2010), HTSeq was used to quantify expression of each transcript (Anders et al.，2015)，and DESeq2 was used to perform differential expression analysis and data normalization (Love et al.，2014). We used TransDecoder (Haas et al.,2013) to identify ORFs within predicted transcripts and annotated them to TAiR1O proteins. RNA-seq data have been deposited in GEO under accession number GSE74103.

To perform GO analysis,transcripts were separated into upand down-regulated and then merged with identical TAiR10 annotations.These gene lists were used as input for the DAVID functional annotation tool (Huang et al.,2oo9). Heat maps were generated using only level 2 and 3 terms with Benjaminiadjusted p-values $< 0 . 0 5$ ：

# Total RNA extraction,cDNA synthesis and qRT-PCR

Total RNA was isolated from the third youngest leaf of two plants per line grown in sterile tissue culture conditions using a PureLink RNA Mini Kit(Thermo Fisher Scientific) with on-column DNase treatment. Three micrograms of total RNA was reverse-transcribed using random hexamers and a Reverse Transcription System (Promega,Madison,Wl) according to the manufacturer's recommendations.The cDNA was diluted 5O-fold in nuclease-free water,and $6 . 5 ~ \mu \ L$ of diluted cDNA was used as template for qRTPCRs.

qRT-PCR was performed in a volume of $2 0 ~ \mu \iota$ using Power SYBR Green PCR master mix (Thermo Fisher Scientific).Primers used for RT-PCR analysis are listed in Table S3. Reactions were run onaStepOnePlus Real-Time PCR system (Thermo Fisher Scientific). Calculations were performed using the $2 ^ { \Delta \Delta \mathsf { C t } }$ method and normalized to ACT and EF1a.

# Acknowledgements

Authors acknowledge contributions from Drs. Dheeraj Verma, Anirudha Dixit， Man Zhou and John Cupp in the Daniell laboratory during early stages of this project.We thank Dr. Joe Chappell (University of Kentucky) for N seeds and Drs.Danny Alexander and Lining Guo (Metabolon, Inc.）for contributions to the metabolomic study. This work was supported in part by a grant from the U.S. Department of Energy ARPA-E programme to H.D.and J.S.Y.and NIH grants R01 HL107904 and R01 HL109442 to H.D.

# Author contributions

E.K.P.performed the research and analysed/interpreted data in Figures 1c and 2-4,Tables 1-3 and the supplementary data and wrote this manuscript. J.S. contributed data in Figure 1b,d and related text in Results and Methods sections.I.M.S.，S.J.G. and B.D.G.analysed/interpreted data in Figure 4,Tables 2and3 and Figures S2-S4 and contributed related text in Resultsand Materialsand methods sections.J.S.Y.analysed/interpreted data and contributed text in the Introduction and Discussion sections.H.D.

conceived and designed the study,analysed/interpreted data and wrote this manuscript.

# Conflict of interest

H.D.is an inventor in several U.S.and international patents related to chloroplast transformation technology.

# References

Aharoni,A.,Giri,A.P.,Deuerlein,S.,Griepink,F.，de Kogel,W.J.,Verstappen, F.W.,Verhoeven,H.A.et al.(20o3) Terpenoid metabolism in wild-type and transgenic Arabidopsis plants.Plant Cell,15,2866-2884.   
Anders,S.,Pyl,P.T.and Huber,W. (2015) HTSeq—a Python framework to work with high-throughput sequencing data. Bioinformatics,31,166-169.   
Baker,J.M.,Hawkins,N.D.,Ward,J.L.,Lovegrove,A.,Napier,J.A.,Shewry,P.R. and Beale,M.H. (2oo6)A metabolomic study of substantial equivalence of field-grown and genetically modified wheat. Plant Biotechnol.J.4,381-392.   
Bally,J.,Nadai,M.,Vitel,M.,Rolland,A.,Dumain,R.and Dubald,M.(2009) Plant physiological adaptations to the massive foreign protein synthesis occurring in recombinant chloroplasts.Plant Physiol.150,1474-1481.   
Bally，J.，Job,C.，Belghazi，M.and Job,D.(2011）Metabolic adaptation in transplastomic plants massively accumulating recombinant proteins.PLoS ONE,6,e25289.   
Banerjee,A.,Wu,Y.,Banerjee,R.,Li,Y.,Yan,H.and Sharkey,T.D.(2013) Feedback inhibition of deoxy-D-sylulose-5-phosphate synthase regulates the methylerythritol 4-phosphate pathway.J. Biol. Chem.288,16926-16936.   
Baroja-Fernández,E.，Munoz,F.J.，Montero,M.，Etxeberria,E.,Sesma,M.T., Ovecka,M.， Bajahi,A.et al.(2oo9) Enhancing sucrose synthase activity in transgenic potato (Solanum tuberosum L.) tubers results in increased levels of starch,ADPglucose and UDPglucose and total yield.Plant Cell Physiol.50, 1651-1662.   
Bobik,K.and Burch-Smith,T.M.(2015) Chloroplast signaling within,between and beyond cells.Front. Plant Sci.6,781.   
Chan,H.-T.and Daniell,H. (2015) Plant-made oral vaccines against human infectious disease are we there yet? Plant Biotechnol.J.13,1056-1070.   
Chi,W.，Feng, 、and Zhang,L. (2015)Metabolites and chloroplast retrograc ling Curr.Opin.Plant Biol.25,32-38.   
Clarke,J.D.,Alexander,D.C.,Ward,D.P.,Ryals,J.A.,Mitchell,M.W.,Wulff,J.E. and Guo,L. (2013) Assessment of genetically modified soybean in relation to natural variation in the soybean seed metabolome.Sci. Rep.3,3082.   
Daniell, H., Ruiz, O.N.and Dhingra,A. (2oo5) Chloroplast genetic engineering to improve agronomic traits.Methods Mol. Biol.286,111-138.   
Daniell，H.，Ruiz，G.，Denes，B.，Sandberg，L．and Langridge，W.(2009) Optimization of codon composition and regulatory elements for expression of human insulin like growth factor-1 in transgenic chloroplasts and evaluation of structural identity and function. BMC Biotechnol.9,33.   
De Cosa，B.，Moar，W.，Lee，S.B.，Miller，M.and Daniell，H.(2001) Overexpresson of the Bt cry2Aa2 operon in chloroplasts leads to formation of insecticidal crystals. Nat. Biotechnol.19,71-74.   
Dhingra,A.，Portis，A.R. Jr and Daniell，H. (2OO4) Enhanced translation of a chloroplast-expressed RbcSgene restores smallsubunit levelsand photosynthesis in RbcS antisense plants. Proc.Natl Acad.Sci.USA,101, 6315-6320.   
Dubouzet,J.G.,lshihara,A., Matsuda,F.,Miyagawa,H.,Iwata,H.and Wakasa, K.(2007） Integrated metabolomic and transcriptomic analyses of hightryptophan rice expressing a mutant anthanilate synthase alpha subunit.J. Exp. Bot.58,3309-3321.   
Ellott,R.,LiF.,Dragomir,I.,Chua,M.M.,Gregory,B.D.and Weis,S.R.(2013) Analysis of the host transcriptome from demyelinating spinal cord of murine coronavirus-infected mice.PLoS ONE,8,e75346.   
Estavillo,G.M., Crisp,P.A.,Pornsiriwong,W.,Wirtz,M.,Collinge,D.,Carrie,C., Giraud,E.etal.(2011) Evidence for a SAL1-PAP chloroplast retrograde pathway that functions in drought and high light signaling in Arabidopsis. Plant Cell,23,3992-4012.   
Evans,A.M.,DeHaven,C.D.,Barrett,T.,Mitchell,M.and Milgram,E.(2009) Integrated，nontargeted ultrahigh performance liquid chromatography/ IeltUsprdy IUILdUIU Iuen IIIdsy Cuomeuy pldtiUIII uie identification and relative quantification of the small-molecule complement of biological systems. Anal. Chem.81,6656-6667.   
Haas,B.J.,Papanicolaou,A., Yassour,M.,Grabherr,M.,Blood,P.D., Bowden, J.,Couger,M.B.et al. (2013) De novo sequence reconstruction from RNASeq using the Trinity platform for reference generation and analysis. Nat. Protoc. 8, 1494-1512.   
Huang，D.W.，Sherman，B.T.and Lempicki，R.A.(2009) Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucleic Acids Res.37,1-13.   
Jensen，P.E.and Leister，D. (2O14) Chloroplast evolution，structure and functions.F100o Prime Rep.6,40.   
Jin,S.and Daniell,H. (2014) Expression of $\gamma \cdot$ -tocopherol methyltransferase in chloroplastsresultsin massive proliferationofthe inner envelope membrane and decreases susceptibilitytosalt and metal-induced Oxidative stresses by reducing reactive oxygen species.Plant Biotechnol. J.12,1274-1285.   
Jin,S.and Daniell，H.(2O15） The engineered chloroplast genome just got smarter.Trends Plant Sci.20,622-640.   
Jin,S.,Kanagaraj,A.,Verma,D.,Lange,T.and Daniell，H. (2O11) Release of hormones from conjugates: chloroplast expression of $\beta$ glucosidase results in elevated phytohormone levels associated with significant increase in biomass and protection from aphids or whiteflies conferred by sucrose esters. Plant Physiol. 155,222-235.   
Kathuria,H.,Giri,J.,Nataraja,K.,Murata,N., Udayakumar,M.and Tyagi,A. (2009）Glycinebetaine-induced water-stress tolerance in codA-expressing transgenic indica rice is associated with up-regulation of several stress responsive genes.Plant Biotechnol.J.7,512-526.   
Kim,S.-K.and Karadeniz,F. (2012) Biological importance and applications of squalene and squalane.Adv. Food Nutr. Res.65, 223-233.   
Kim,D.，Pertea,G.，Trapnell，C.，Pimentel，H.，Kelley，R.and Salzberg，S.L. (2013)TopHat2:accurate alignment of transcriptomes in the presence of insertions,deletions and gene fusions. Genome Biol.14,R36.   
Kogel,K.-H.,Voll,L.M., Schäfer,P.,Jansen,C.,Wu,Y.,Langen,G.,Imani,J. et al.(2010） Transcriptome and metabolome profiling of field-grown transgenic barley lack induced differences but show cultivar-specific variances.Proc.Natl Acad.Sci. USA,107,6198-6203.   
Kumar,S.,Hahn,F.M.,Baidoo,E.,Kahlon,T.S.,Wood,D.F.,McMahan,C.M., Cornish,K.et al. (2012) Remodeling the isoprenoid pathway in tobacco by expressing the cytoplasmic mevalonate pathway in chloroplasts.Metab.Eng. 14, 19-28.   
Kwon,K.C., Nityanandam,R.,New,J.S.and Daniell,H.(2013a)Oral deliveryof bioencapsulated exendin-4 expressed in chloroplasts lowers blood glucose levelin mice and stimulates insulin secretion in beta-TC6 cells.Plant Biotechnol. J. 11,77-86.   
Kwon,K.C.,Verma,D., Jin,S., Singh,N.D.and Daniell,H.(2013b) Release of proteins from intact chloroplasts induced by reactive oxygen species during biotic and abiotic stress. PLoS ONE,8,e67106.   
Lange,B.M.(2015) The evolution of plant secretory structures and emergence of terpenoid chemical diversity.Annu. Rev. Plant Biol. 66,139-159.   
Love，M.I.， Huber,W.and Anders,S.(2014) Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.Genome Biol.15,550.   
Martin,M.(2011) Cutadapt removes adapter sequences from high-throughput sequencing reads.EMBnet.journal,17,10-12. Melis,A. (2013) Carbon partitioning in photosynthesis. Curr. Opin.Chem. Biol. 17, 453-456.   
Nisar，N.，Li，L.，Lu，S.，Khin,N.C.and Pogson,B.J.(2015） Carotenoid metabolism in plants.Mol. Plant,8,68-82.   
NottA.,Jung,H.S.，Koussevitzky，S.and Chory,J. (2006) Plastid-to-nucleus retrograde signaling.Annu. Rev.Plant Biol.57,739-759.   
O'Hagan,D.T., Rappuoli,R., De Gregorio,E., Tsai,T.and Del Giudice,G.(2011) MF59 adjuvant: the best insurance against influenza strain diversity.Expert Rev.Vaccines,10,447-462.   
Ramel,F., Birtic,S.,Ginies,C.,Soubigou-Taconnat,L.,Triantaphylides,C.and Havaux,M.(2012) Carotenoid oxidation products are stress signals that mediate gene responses to singlet oxygen in plants. Proc.Natl Acad.Sci. USA,   
Ricroch，A.E.，Bergé,J.B.and Kuntz，M.(2O11） Evaluation of genetically engineered crops using transcriptomic, proteomic,and metabolomic profiling techniques.Plant Physiol.155,1752-1761.   
Ruhlman,T.，Verma，D.，Samson，N．and Daniell，H.(201O)The role of heterologous chloroplast sequence elements in transgene integration and expression.Plant Physiol.152,2088-2104.   
Saxena,B.,Subramaniyan,M.,Malhotra,D.,Bhavesh,N.S.,Potlakayala,S.D. and Kumar,S. (2014) Metabolic engineering of chloroplasts for artemisinic acid biosynthesis and impact on plant growth.J. Biosci.39,33-41.   
Sierro,N., Battey,J.N.,Ouadi,S., Bakaher,N., Bovet,L.,Willig,A.,Goepfert,S. et al. (2014) The tobacco genome sequence and its comparison with those of tomato and potato.Nat. Commun.5,3833.   
Singh,N.D.，Li,M.，Lee,S.B.，Schnell,D.and Daniell,H.(20o8) Arabidopsis Tic40 expression in tobacco chloroplasts results in massive proliferation of the inner envelope membrane and upregulation of associated proteins. Plant Cell, 20,3405-3417.   
Singh，N.D.，Kumar,S.and Daniell,H.(2015)Expression of $\beta$ -glucosidase increases trichome density and artemisinin content in transgenic Artemisia annua plants.Plant Biotechnol.J.14,1034-1045.   
Tattersall, D.B., Bak,S.,Jones, P.R.,Olsen,C.E.,Nielsen,J.K., Hansen,M.L,Hoj, P.B.et al. (2oo1) Resistance to an herbivore through engineered cyanogenic glucoside synthesis. Science,293,1826-1828.   
Trapnell,C.，Pachter,L.and Salzberg,S.L.(20o9) TopHat:discovering splice junctions with RNA-seq. Bioinformatics,25,1105-1111.   
Trapnell,C.,Wiliams,B.A.，Pertea,G.，Mortazavi,A.，Kwan,G.,van Baren, M.J.,Salzberg,S..et al. (2O10) Transcript assembly and quantification by RNA-seq reveals unannotated transcripts and isoform switching during cel differentiation.Nat. Biotechnol.28,511-515.   
Verma,D.and Daniell,H.(2007) Chloroplast vector systems for biotechnology applications.Plant Physiol.145,1129-1143.   
Verma，D.，Samson,N.P.，Koya，V.and Daniell,H.(20o8)A protocol for expression of foreign genes in chloroplasts. Nat. Protoc.3,739-758.   
Viitanen,P.V., Devine,A.L.,Khan,M.S, Deuel, D.L., Ven Dyk,D.E.and Daniell,H. (2004) Metabolic engineering of the chloroplast genome using the Echerichia coli ubiC gene reveals that chorismate is a readily abundant plant precursor for p-hydroxybenzoic acid biosynthesis. Plant Physiol.136,4048-4060.   
Wang,Z.,Gerstein,M.and Snyder,M. (20o9) RNA-seq: a revolutionary tool for transcriptomics. Nat. Rev. Genet. 10, 57-63.   
Wang,X.,Ort,D.and Yuan, J.S.(2015) Photosynthetic terpene hydrocarbon production for fuels and chemicals. Plant Biotechnol.J.13,137-146.   
Woodson,J.D.,Perez-Ruiz, J.M.and Chory,J. (2011) Heme synthesis by plastid ferrochelatase 丨 regulates nuclear gene expression in plants. Curr. Biol.21, 897-903.   
Wu,S.,Schalk,M.,Clark,A.,Miles,R.B.,Coates,R.and Chappell,J.(2006) Redirection of cytosolic or plastidic isoprenoid precursors elevates terpene production in plants. Nat. Biotechnol.24,1441-1447.   
Wu,S.，Jiang，Z.，Kempinski,C.，Nybo,S.E.，Husodo,S.，Williams，R．and Chappell,J.(2012) Engineering triterpene metabolism in tobacco.Planta, 236,867-877.   
Xiao,Y.,Savchenko,T.,Baidoo,E.E.,Chehab,W.E.,Hayden,D.M.,Tolstikov, V., Corwin,J.A.et al. (2012) Retrograde signaling by the plastidial metabolite MEcPP regulates expression of nuclear stress-response genes.Cell,149, 1525-1535.   
Ye,X.，Al-Babili,S.，Kloti,A.,Zhang,J.,Lucca,P.，Beyer,P.and Potrykus,I. (2000) Engineering the provitamin A (β-carotene) biosynthetic pathway into (carotenoid-free) rice endosperm.Science,287,303-305.

# Supporting information

Additional Supporting information may be found in the online version of thisarticle:

Figure S1 PCR analysis of transgene integration in transplastomic plants.   
Figure S2 GO analysis for‘cellular component'-related terms. Figure S3 GO analysis for'molecular function'-related terms. Figure S4 Plot of $\mathsf { l o g } _ { 2 }$ fold changes for selected transcripts as determined by qRT-PCR versus RNA-seq.   
Table S1 Metabolites with significantly changed abundance in each line relative to WT.   
Table S2 List of all metabolites whose abundance changes significantlyand byat least twofold in PH-SQS compared with PH. Table S3 List of qRT-PCR primers used in this study.