# Recognition of H3K9 methylation by GLP is required for efficient establishment of H3K9 methylation, rapid target gene repression, and mouse viability

Nan Liu,1,236 Zhuqiang Zhang,36Hui Wu,46 Yonghua Jiang,2 Lingjun Meng,2 Jun Xiog Zuodong Zhao,² Xiaohua Zhou,² Jia Li,² Hong Li,² Yong Zheng,² She Chen,² Tao Cai,² Shaorong Gao, and Bing Zhu2,3

lCollegeofLifeiences,BengralUiversityeijing875,hina;NatioalIstituteofiologicalSiensg 102206,China;National Laboratoryof Biomacromolecules,Instituteof BiophysicsChinese Academyof ciences,Beijing 100101,China;NatioalEginengboratorforVacne,hoolofifeiences,lUversityCgchl Provincel30ol2,China;Schoolof Life Sciences and Technology,Tongji UniversityShanghai20o092,China

GLP and G9a are major H3K9 dimethylases and are essential for mouse early embryonic development. GLP and G9a both harbor ankyrin repeat domains that are capable of binding H3K9 methylation.However,the functional significance of their recognition of H3K9 methylation is unknown.Here,wereport that the histone methyltransferase activities of GLPand G9a are stimulated by neighboring nucleosomes that are premethylated at H3K9.These stimulation events function in cis and are dependent on the H3K9 methylation binding activities of ankyrn repeat domains of GLP and G9a.Disruption of the H3K9 methylation-binding activity of GLP in micecauses growth retardationof embryos,osificationdefects ofcalvaria,and postnatal lethalitydueto starvationof the pups.In mouse embryonic stem cels (ESCs)harboring amutant GLP that lacks H3K9me1-binding activity,critical pluripotent genes,including Oct4 and Nanog,display inefficient establishment of H3K9me2 and delayed gene silencing during differentiation.Collctively,our study reveals anew activation mechanism for GLPand G9a that plays an important role in ESC differentiation and mouse viability.

[Keywords: GLP; G9a; H3K9 methylation; gene repression; cell differentiation]

Supplemental material is available for this article.

Received October17,20l4;revised version accepted January 6,2015.

GLP and G9a are SET domain-containing histone methyltransferases that play vital roles during early mouse development (Tachibana et al. 200l,2002,2005) and are also involved in many other developmental processes,such as hematopoiesis,adipogenesis,and retrovirus silencing. (Chen et al.2012; Maksakova et al.20l3;Wang etal.2013; Balemans et al.2Ol4). GLP and G9a are widely expressed and are responsible for the majority of H3K9me2 at facultative heterochromatin regions (Mermoud et al. $2 0 0 2 ;$ Tachibana et al. $2 0 0 2 ;$ Trojer and Reinberg 2007). Facultative heterochromatin regions often span up to millions of base pairs and,unlike the centromeric constitutive heterochromatin, vary among cell types (Wen et al. 2009;Lienert etal.2Ol1).During cell fate transition events such as the differentiation of the embryonic stem cells (ESCs)or X-chromosome inactivation，new blocks of facultative heterochromatin are established (Mermoud et al.2002;Chow and Heard 2009；Wen et al.2009). However，it remains challenging to pinpoint different mechanisms involved in efficiently establishing the facultative heterochromatin marks during cell fate transition.H3K27me3,another histone modification marking distinct facultative heterochromatic regions,can be efficiently established by two different allosteric activation mechanisms of the H3K27-specific histone methyltransferase complex PRC2,which can sense repressive histone modifications (Margueron et al.2oo9)and dense chromatin (Yuan et al.2012).H3K9me2 covers more than onethird of the genome and undergoes drastic changes during cell fate transitions (Wen et al. 2OO9;Lienert et al. 2011; Chen et al.20l2). However, GLP and G9a exhibit very lowactivities toward nucleosomalH3 in vitro(Tachibana et al.2O05). This leads to questions of whether any activation mechanisms are involved in efficiently establishing H3K9me2 during cell fate transitions and whether the enzymatic activities of GLP and G9a are also regulated by chromatin environment.

Intriguingly,in addition to the catalytic activities for methylating H3K9,GLP and G9a also carry the ankyrin repeat domains that specificallyrecognizeH3K9mel and H3K9me2，respectively (Brent and Marmorstein 2008; Collins et al. 2O08). We reason that GLP and G9a may use theirH3K9 methylation-binding activities to sense methylated H3K9 and achieve their full activity as well as maintain H3K9me2 levels during mitotic divisions or efficiently establish H3K9me2 at genomic regions that are to be silenced during cell fate transition events.

Using oligonucleosome substrates containing unmodified nucleosomes and nucleosomes premodified at the H3K9 position to various states,we observed strong stimulation of the enzymatic activities of GLP and G9a by neighboring nucleosomes containing H3K9mel and H3K9me2, respectively.ThebindingabilitiestoH3K9melandH3K9me2of GLP and G9a were required for the enzymatic stimulation. ESCs harboring the H3K9mel-binding mutant form of GLP failed to efficiently establish H3K9me2 during retinoic acid (RA)-induced differentiation across a large number of genes, and a subset of these genes displayed delayed transcription repression.Notably, several genes encoding critical pluripotent factors, including Oct4 and Nanog,were among them, suggesting that GLP's H3K9 methylation-binding activity is required for efficient establishment of H3K9me2 and rapid shutdown of the pluripotency transcription program in ESCs undergoing differentiation. Homozygous knock-in mice carrying a H3K9mel-binding mutant form of GLP exhibited postnatal lethality.These mice displayed embryonic growth retardation and defects in calvaria bone formation.Together，these results provided evidences for the significance of GLP's H3K9mel-binding activity in efficient H3K9me2 establishment and gene silencing during cell differentiation and mouse viability but not in mitotic inheritance of H3K9 methylation.

# Results

Histone methyltransferase activities of GLP and G9a are stimulated by neighboring nucleosomes premethylated atH3K9 inan ankyrin repeat-dependentmanner

To test whether GLP and G9a can be stimulated by neighboring nucleosomes premethylated at H3K9,we developed a histone methyltransferase activity stimulation assay.We prepared premethylated histones using chemical reactions to install methyl-lysine analogs (MLAs) at the K9 position of histone H3 (Simon et al. 2007).The MLAs installed by MLA reactions preserve most of the features of the methyl-lysine, including correct recognition by specific methyl-lysine-binding proteins (Simon et al. 2Oo7）and compatibility in serving as substrates for histone methyltransferases and demethylases (Jia et al.2OO9).Histone octamers that contained H3 premodified at the K9 position with various modifications $\mathrm { \Delta } ( \mathrm { K c { 9 m e 0 } } / 1 / 2 / 3$ or unreacted K9C) and histone octamers that contained unmodified H3 were assembled separately and then pooled together at a l:1 ratio.The octamers were assembled with DNA into oligonucleosomes.The assembled oligonucleosomes were purified by gel filtration on a Sepharose CL-4B column to remove free histone octamers.The final substrates contained unmodified nucleosomes that were flanked by nucleosomes harboring premodified H3K9 (Fig. 1A). To distinguish the two types of histone H3, the premodified H3 histones were tagged and therefore migrated slower on SDS-PAGE gels (Fig. 1B).

Recombinant GLP and G9a proteins covering ankyrin repeats and SET domains were purified from Escherichia coli and assayed for methyltransferase activity using the above-described substrates.Strikingly,despite the fact that equal amounts of enzymes and unmodified H3 histones were present in the reactions,G9a displayed 2O-fold more activity on the unmodified nucleosomes that were flanked by H3Kc9me2-containing nucleosomes than on unmodified nucleosomes that were flanked by nucleosomes containingH3K9C,H3Kc9me0,or H3Kc9me3 (Supplemental Fig.lA).These results are consistent with the fact that H3K9me2 is G9a's favored binding partner (Collins et al. 2008). Similarly, GLP displayed the best activity on unmodified nucleosomes flanked by nucleosomes containing H3Kc9mel (Fig.1B),which was in agreement with its preferred binding to H3K9mel (Collins et al.2O08).Catalytic mutants of GLP(C1201A) or G9a (C1007A) displayed no detectable activity on any of the substrates (Fig. $\textstyle 1 { \mathrm { C } } ;$ Supplemental Fig. S1B).Notably, G9a also displayed much higher enzymatic activity at the tagged premethylated H3 histones in the case of H3Kc9me2 (Supplemental Fig.1A, lane 4).Similar observations were also obtained for GLP at the tagged premethylated H3 histones in the case of H3Kc9mel (Fig.1B, lane 3). This is most likely due to G9a's and GLP's in vitro catalytic activities at H3K27,as previously reported (Tachibana et al.2OOl; Wu et al. 2011). Indeed, when residue Lys27of H3 was mutated to alanine, the stimulation activity was observed at only the unmodified H3 histones but not the tagged premethylated H3 histones (Supplemental Fig.1C,lane 2). Furthermore, neither GLP nor G9a activity was stimulated by nucleosomes premethylated at H3K27 (Fig. 1D;Supplemental Fig.lD),in agreement with their inability to recognize H3K27 methylation (Collins et al.2008). Together, these results suggested that the activities of GLP and G9a on nucleosomal histones could be strongly stimulated when their substrates were flanked by neighboring nucleosomes harboring H3K9 premethylation.

Ankyrin repeats of GLP and G9a were responsible for their recognition toward H3K9 methylation (Collins et al. 2008).GLP W877A/W882A/E885A and G9a W791A/ W796A/E799A mutants (abbreviated as GLP 3A and G9a 3A mutants,respectively) lost methylated H3K9-binding activity due to the disruption of their aromatic cages and hydrogen bonds within the ankyrin repeats that confine

A pool of premoitiedai octamers WinDnm unmodified H3 unmodified octamers O octamerassembly 0000o mixing at 1:1 ratio unmodified nucleosomes with H4,H2B, H2A flanked by nucleosomes containing pre-modified H3K9   
B C GLP C1201A GLP (Ankyrin+SET) activity on pre-modified His-H3 -pre-modified His-H3● 6000 activity on unmodified H3 -unmodified H3 O 4800   
2400 autoradiography 1200 junmodified H3 -pre-modified His-H3● ○ 0 -H2B/H2A quantified activity -H4 -pre-modified His-H3● coomassie -unmodified H3 O 1 2 3 4 5 autoradiography F33ngr ka3nmr 23e 233mr pre-modification -pre-modified His-H3● 6 at K9 of His-H3 unmodified H3 O -H2B/H2A -H4 coomassie E GLP3A 1 2 3 4 5 6000 activity on pre-modified His-H3 kaengn engr 23m 233egr pre-modification 4800 activity on unmodified H3 26K at K9 of His-H3● 040 m3600   
D GLP (Ankyrin+SET) 1200 □ 1 quantified activity -pre-modified His-H3● -unmodified H3 O -pre-modified His-H3● autoradiography -unmodified H3 O -pre-modified His-H3● autoradiography junmodified H3 O -H4 -H2B/H2A unmodified H3 -pre-modified His-H3● O -H2B/H2A coomassie -H4 1 2 3 4 5 X33meor 3a3mmmr 223r 233eer pre-modification 1 2 coomassi4 5 X222 at K27 of His-H3● oeue lewe 23mgr 3egr pre-modification X65 KX5n 0 atK9 of His-H3

the methylated lysine (Collins et al. 2OO8)，which was confirmed by isothermal titration calorimetry (ITC) (Supplemental Fig.2).Neither GLP 3A (Fig.1E) nor G9a 3A (Supplemental Fig.1E)was stimulated by premethylated neighboring nucleosomes,indicating that recognition of premethylated neighboring nucleosomes by ankyrin repeats is an essential step for stimulation.

# The stimulation of GLP and G9a activities bypremethylated nucleosome functions in cis

A second set of substrates was assembled to allow simultaneous testing of stimulation events at nucleosomes existing in cis and in trans. Oligonucleosomes containing Flag-H3 were mixed with the substrates described in Figure 1A at a 1:1 ratio (Fig.2A).In the final substrate mixture,the unmodified H3 histones existed in cis with the premodified His-H3 histones,while the Flag-H3 histones existed in trans with the premodified His-H3 histones (Fig. 2A).These three types of H3 histones could be distinguished in SDS-PAGE gels due to their different migration velocities (Fig.2B). GLP (Fig.2B) and $\mathrm { G } \mathcal { G } \mathrm { a }$ (Supplemental Fig. 3A)displayed robust activity on unmodified H3 histones existing in cis with nucleosomes containing H3Kc9mel and $\mathrm { H } 3 \mathrm { K c } 9 \mathrm { m e } 2 ,$ respectively,but not on Flag-H3 nucleosomes existing in trans with the premodified nucleosomes.

![](images/90864982f7c686c6a4e523ba6bd9c66d7ee94cedf01c4fba6d2ff4ad07ff24ec.jpg)  
Figure 2. Stimulation of GLP activity by premethylated neighboring nucleosomes is_a cis event. $\left. A \right.$ Substrate description. (B) MethyltransferaseactivityassayofGLPat nucleosomes existing incisandin trans with premodified nucleosomes.(C)Methylated H3K9 peptides did not stimulate the methyltransferase activity of GLP.(One microgram of recombinant enzymes, $0 . 5 ~ { \mu \mathrm { g } }$ of peptide, and $1 ~ { \mu \mathrm { g } }$ of recombinant oligonucleosomes were used in the assay.)

To exclude that the Flag tag might affect the enzymatic activities,control experiments were performed using substrates in which Flag-H3 histones were placed in cis with the pre-modified His-H3 histones,and untagged H3 histones were placed in trans with the premodified His-H3 histones.GLP and G9a displayed robust activities on FlagH3 nucleosomes existing in cis with the premethylated H3 and minimal activities on H3 nucleosomes existing in trans with the premethylated H3 (Supplemental Fig.3B,C). These results indicated that the stimulation events were dependent on spatial position of the nucleosomes and functioned in cis.

H3K27 methyltransferase (PRC2） can be allosterically activated by peptides containing H3K27me3 (Margueron et al.20O9).However, neither GLP nor G9a was activated by the addition of methylated H3K9 peptides (Fig.2C; Supplemental Fig. 3D),which suggests that GLP and G9aadapted distinct strategies for spreading chromatin modifications different from the allosteric activation approach that PRC2 uses.Taken together,GLP and G9a were able to sense H3K9mel/2 through their ankyrin repeats and facilitate their H3K9 methylation activities in vitro.

# GLP 3A mutant mice were postnatal-lethal with growth retardation and craniofacial defects

To understand the in vivo role of the H3K9 methylation recognition activities of GLP and G9a，we generated knock-in mice bearing the GLP 3A or G9a 3A mutations. Mutations were introduced into R1 ESCs by homologous recombination (Supplemental Fig.4).We confirmed that the targeting events did not affect the expression level of the targeted $G L P$ and $G 9 a$ alleles in the corresponding heterozygote ESC lines (Supplemental Fig. 4). F1 heterozygous mice of the GLP 3A and G9a 3A mutants (termed $\mathrm { G L P ^ { + / M } }$ and ${ \mathrm { G } } { \mathrm { 9 a } } ^ { + / { \mathrm { M } } } { \rceil }$ were healthy and fertile. Surprisingly, homozygous mice $| \mathrm { G L P ^ { M / M } }$ and ${ \cal G } 9 a ^ { { \cal M } / { \cal M } } )$ displayed strikingly different phenotypes. $\mathrm { G } 9 \mathrm { a } ^ { \mathrm { M } / \mathrm { M } }$ mice were born at the Mendelian ratio (Supplemental Fig. 5A) and were healthy and fertile. In contrast, ${ \bf G L P } ^ { { \bf M } / { \bf M } }$ mice were mostly postnatal-lethal (Fig. 3A). All $\mathrm { G L P ^ { M / M } }$ mice were born smaller in size (Fig.3B),and 12O out of 128 of them died within $2 \mathrm { d }$ of birth (Fig. 3A). Only eight $\mathrm { G L P } ^ { \mathrm { M / M } }$ mice survived to adulthood,and they suffered from severe growth retardation within the first 4 wk after birth (Supplemental Fig. 5B).GLP and G9a double-mutant mice displayed a phenotype similar to that of ${ \bf G L P } ^ { { \bf M } / { \bf M } }$ mice (Fig. 3C), suggesting that the H3K9 methylation recognition activity of GLP played a dominant role in vivo.Therefore,we focused our study on $\mathrm { G L P } ^ { \mathrm { M / M } }$ in all subsequent experiments.

Consistent with the smaller size of $\dot { \mathrm { G L P } } ^ { \mathrm { M } / \mathrm { M } }$ newborns, the $\mathrm { G L P ^ { M / M } }$ embryos also showed growth retardation as early as embryonic day 1O.5 (E1O.5)(Supplemental Fig.5C). Although no drastic abnormalities of the main organs were observed and no catastrophic events occurred in breathing

A B 2.00 1.80   
DelveredG21 G2PG2PM 中GLP(n=10） →GLPMM（n=11; n=5 a P0.5 119 220 97 01.00 because the other6 P2 119 220 8 0.80 GLP+ GLPMM 0.60 1h 6h 24h   
C 39393   
GLP genotype M/M +/+ +/M M/M +/+ +/M   
G9a genotype M/M +/M +/M +/M +/M M/M   
D p=0.409 E p=1.3X105 1.0 1 1 1e 0.8 pege fail 0.4 succeed 0 WT Hetero Homo WT Hetero Homo (N=26) (N=22) (N=18) (N=26) (N=22) (N=18) Nasal bones (Pomice)   
F DoDO G Calvaria(POmice) M/M +/+ M/M 2months

Fig.5D).The most severe delayed ossification occurred in the sagittal and metopic sutures as well as in the nasal bones (Fig. 3F). Even for the few surviving adult GLPM/M mice,bone loss was observed in their metopic sutures (Fig.3G; Supplemental Fig.5E),which was likely caused by ossification defects during embryonic development. In addition, several of the surviving adult $\mathrm { G L P ^ { M / \hat { M } } }$ mice (two out of eight) displayed an obviously bent nose and malocclusion (Supplemental Fig.5E,F).

# The H3K9me1-bindingactivity of GLP is largely dispensable formaintenanceof globalH3K9me2levels in ESCs and tissues

To study the in vivo role of H3K9 methylation-binding activity of GLP at molecular level， we derived wild-type and GLP 3A mutant ESCs (termed wild-type and GLP 3A ESCs,respectively) from littermate embryos at the blastocyst stage.These ESCs displayed the correct karyotypes (Supplemental Fig. 6A) and similar expression levels of GLP (Supplemental Fig. 6B).Western analysis revealed comparable bulk H3K9me2 levels between wild-type and GLP 3A ESCs (Supplemental Fig. 6C).Next,we analyzed brain and liver samples harvested from multiple littermate pairs of wild-type and $\mathsf { \bar { G } L P } ^ { \mathrm { M / M } }$ newborn pups (Supplemental Fig. 6D). We did not observe consistent changes of H3K9me2 between wildtype and ${ \bf G L P } ^ { { \bf M } / { \bf M } }$ livers or brains (Supplemental Fig. 6D). Considering that Western analysis is not an ideal quantitative assay, especially for subtle changes,we performed H3K9me2 ChIP-seq (chromatin immunoprecipitation [ChIP] combined with deep sequencing) experiments using ESCs and livers from newborn pups with wild-type and GLP mutant backgrounds.We first divided the mouse genome into $1 0 \mathrm { - k b }$ windows.This window size was chosen because H3K9me2 often exist in large blocks of chromatin regions termed $\scriptstyle \mathrm { { L O C K s } , }$ and the minimal size of LOCKs is ${ \sim } 2 0 ~ \mathrm { k b }$ (Wen et al.2009).We plotted the H3K9me2 fold enrichment changes between mutant and wild-type samples for all of the 10-kb windows.In ESCs and livers from newborn pups, only $1 \%$ and $2 . 2 \%$ of thewindows displayed H3K9me2 level changes ${ > } 1 . 5$ -fold,respectively (Supplemental Fig. 6E).The above results collectively suggest that the H3K9mel-binding activity of GLP is largely dispensable for maintenance of global H3K9me2 levels in ESCs and tissues.

and heart beating, we consistently observed that no milk was present in the stomachs of $\dot { \mathrm { G L P } } ^ { \mathrm { M } / \mathrm { M } }$ pups, so it was highly possible that ${ \bf G L P } ^ { { \bf M } / { \bf M } }$ mice died of starvation. To understand at which stage the $\mathrm { G L P } ^ { \mathrm { M / M } }$ pups failed to be fed,we adapted an assay to dissect multiple steps of suckling processes (Hongo et al. 2000). The $\dot { \mathrm { G L P } ^ { \mathrm { M / M } } }$ mice were capable of actively searching and licking the nipples of mother mice (Fig. 3D), suggesting that their olfaction and sensory-motor systems were probably fine.However, these pups were unable to remain stably attached to the nipples; while most wild-type and heterozygote pups can be stably attached to the nipples for up to $2 0 ~ \mathrm { { m i n } } .$ ，the GLpM/M pups failed to remain attached for even just 1 min in $\sim 7 0 \%$ of the attempts (Fig.3E),which suggested that they might suffer from craniofacial development abnormalities (Turgeon and Meloche 2Oo9). Consistent with this hypothesis,skeletal preparations at various developmental stages showed obvious delayed calvaria and nasal bone ossification in $\mathrm { G L P } ^ { \mathrm { M } / \mathrm { M } }$ mice (Fig. 3F; Supplemental

Nevertheless,among the 3940 1O-kb windows that displayed a reduction of H3K9me2 in mutant newborn livers,2430 localized to genes $( 6 2 \% )$ . In contrast, among the 77O 1O-kb windows that displayed an increase of H3K9me2 in mutant newborn livers,196 localized to genes $( 2 5 \% )$ In $\mathrm { E S C s } ,$ ${ > } 7 0 \%$ of 10-kb windows with increased or decreased H3K9me2 levels localized to genes.These results suggest that H3K9me2 levels at genic regions are preferentially affected upon the loss of H3K9mel-binding activity of GLP.

# TheH3K9me1-bindingactivityofGLPis critical forefficient establishment ofH3K9me2 and gene silencingat the Oct4 gene during ESC differentiation

Considering that the H3K9mel-binding activity of GLP stimulated its catalytic activity in vitro (Fig.1),it displayed a minimal effect in its contribution to total H3K9me2 levels in several cell types (Supplemental Fig. 6D,E).We hypothesized that the H3K9 methylation-binding activity of GLP and its enzymatic stimulation may be more important for the kinetics of H3K9me2 establishment than the maintenance of H3K9me2.To test this hypothesis,we adapted a well-studied process involving dynamic establishment of H3K9me2，which is the RA-induced differentiation of ESCs (Wen et al.2OO9；Lienert et al. 2011).During RA-induced differentiation of ESCs，the expression of Oct4 gene is quickly reduced,and its final silencing requires GLP,G9a,and H3K9me2 establishment at the Oct4 locus (Feldman et al. 2OO6;Epsztejn-Litman etal.2Oo8).We first examinedthe H3K9me2 establishment and gene silencing at the Oct4 gene during RAinduced differentiation of ESCs with wild-type and GLP 3Abackgrounds.Prior to RA treatment,H3K9me2levels were comparably low at the Oct4 gene in wild-type and GLP 3A ESCs (Fig. 4A).After RA treatment, H3K9me2 levels were significantly increased at the Oct4 locus in wild-type cells (Fig. 4B),in agreement with previous reports (Feldman et al.2OO6).However,in RA-treated GLP 3A cells,H3K9me2 levels at the Oct4 locus were much lower than that in the wild-type cells (Fig.4B). In contrast, such a difference was not observed at the Magea2 gene (Fig.4A,B),which is constitutively repressed by GLP and G9a (Tachibana et al. 2002).

Wild-type and GLP 3A ESCs displayed comparable levels of Oct4 expression (Fig.4C)，and，in both cells，Oct4 expression was quicklyreduced within the first $3 \mathrm { ~ d ~ }$ of RA treatment, which most likely reflects changes of transcription factor association at the Oct4 gene.However, GLP 3A cells displayed an obvious delay in shutting down Oct4 expression,and complete silencing of Oct4 transcription was delayed for ${ \sim } 6$ d in GLP 3A cells (Fig. 4C),although establishment of DNA methylation at the Oct4 promoter was largely normal in these cells during RA-induced differentiation (Fig. 4D). To rule out a potential clonal effect of $\operatorname { E S C s } ,$ the above experiments were repeated in ESCs derived from a separate litter of mice,and similar results were observed (Supplemental Fig.7A,B).

We also monitored H3K9me2 establishment and Oct4 repression in a related differentiation system involving the formation of embryoid bodies (EBs) from ESCs,which to some extent mimics the in vivo post-implantation embryo development (Desbaillets et al. 2OoO).In this system,we observed similar defects in H3K9me2 establishment and transcription repression at the Oct4 gene (Supplemental Fig.7C-E). These results collectively sug gested that the H3K9mel-binding activity of GLP was required for efficient establishment of H3K9me2 and rapid repression of the Oct4 gene during differentiation.

TheH3K9me1-binding activity of GLP is required forefficient establishmentofH3K9me2,butnot H3K9mel,at genic regions during ESC differentiation

We then asked whethera similarrole of GLP's H3K9melbinding activity occurred at the genome-wide level. Given that GLPis essentialforH3K9me2levelsin vivo(Tachibana et al.2OO5)，we first performed ChIP-seq experiments for H3K9me2 with ESCs andRA-treated cells at $2 . 5 \mathrm { d } ,$ at which stage the GLP protein levels were comparable between wild-type and GLP 3A cells (Supplemental Fig. 8A). Three biological replicates were performed for each analysis.We divided the mouse genome into lO-kb windows.For each window, the fold enrichment of H3K9me2 was calculated asthe ratio of normalizedread density between ChIP and input samples,and the mean H3K9me2 fold enrichment value was calculated with data from the three replicates. Windows with at least one case of more than twofold change between any pair of samples (33O7 in total) were selected for clustering analysis (Fig. 5A). The majority of these regions $( 8 6 \% )$ displayed an increase in H3K9me2 during RA-induced differentiation in wild-type cells (Fig. ${ 5 \mathrm { A } } ,$ groups 1 and 2), suggesting that $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ wasactively established at these regions in normal differentiating cells. Although wild-type and GLP 3A cells contained similar H3K9me2 levels at these regions in ESCs, $\sim 7 5 \%$ (2495 of 3307)of these regions displayed obviously lower H3K9me2 levels in RA-treated GLP 3A cells in comparison with their wild-type counterparts (Fig.5A, group 1). On the other hand, only $1 1 \%$ (362 of 3307) of these regions displayed comparableorhigher $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ levels inRA-treated GLP3A cells (Fig. ${ 5 \mathrm { A } } ,$ group 2). These results clearly indicate a global defect in establishing H3K9me2 in GLP 3A cells.

![](images/d210ce2c5d247fc58a32c5c5b273b2dedfbc7d4f5e0f4c7bcf78a948539323bd.jpg)  
Figure 4. H3K9me2 establishment and transcription silencing of Oct4 during ESC differentiation were delayed in GLP 3Acells. $\scriptstyle { \left. A , B \right. }$ ChIPqPCR results showing H3K9me2 occupancy at the indicated loci in ESCs $| A |$ orRA-treated cells at the indicated time points (B). $| C |$ RT-PCR results showing $O c t { 4 }$ expression levels during differentiation. $| D |$ DNA methylation levels at the Oct4 promoter during differentiation.Error bars represent the standard error.

![](images/7d4d3f091222267f2d1a408ddf6aa2be05923c8ae0ac9e75480295e889e763ea.jpg)  
Figure5.TheH3K9methylation-bndingactivityofGLPwasrequiredforeficientH3K9me2establishmentduringESCdfrentiatioat a genome-wide level. $| A |$ Clustering of the $1 0 \mathrm { - k b }$ genomic windows according to their H3K9me2 levels for wild-type (WT) and GLP 3A cells withorwithoutRAtrament.Tefdichmentvausoftreblogicalepcatesereeaged,enteedtoteafows normalized to standard deviation by rows $Z$ -score in A-C). $\left. B \right.$ Clustering of the $1 0 \mathrm { - k b }$ genomic windows according to their H3K9me2 levels for all replicates in wild-type and GLP 3A cells with or without RA treatment. $| C |$ Clustering of the $3 3 0 7 1 0 { \cdot } \mathrm { k b }$ genomic windows (shown in A)acordingtotheirH3K9mellevelsforwild-typeandGLP3AcelswithorwithoutRAtreatment.Thevaluesareaveragesof three biological replicates. $\scriptstyle | D , E |$ Accumulative distribution of H3K9me2 $| D |$ and H3K9mel $\mathbf { \Psi } ( E )$ fold changes between RA-treated cells and ESCs at genic regions. $\left( F \right)$ Number of genes with significantly changed H3K9me2 in wild-type or GLP 3A cells after RA treatment. $| G |$ MA plot of H3K9me2foldchangesagainsttheaverageH3K9me2foldenrichment.ReddotsrepresentgeneswithsignificantlyincreasedH3K9me2 levels after RA treatment in wild-type cells. $\left. H \right.$ Box plot of H3K9me2 fold changes between ESCs and RA-treated cells at genes acquired by H3K9me2 (shown in $G \}$ in wild-type and GLP 3A cells. $\left( I \right)$ H3K9me2 profile at genic regions of genes acquired by H3K9me2 (shown in $G$ in ESCs or RA-treated wild-type (left) and GLP 3A (right) cells. $^ { \prime \prime } \mathrm { R e p } \ 1 { - } 3 ^ { \prime \prime }$ represents the three biological repeats. Lengths of genes were normalized to 1O0 percentiles and extended $20 \%$ upstream of and downstream from the gene body, respectively.

To test the reproducibility of the above analysis,we performed clustering analysis for the same 3307 10-kb windows for all 12 experiments (two cell types $\times$ two treatments $\times$ three replicates). Essentially,almost all replicates clustered together, indicating good reproduc ibility among them (Fig. 5B). The only exception was that one experiment of the GLP 3A cells at the ESC stage was grouped together with the wild-type ESCs,which was in agreement with our observation that wild-type and GLP 3AESCs werehighly similar for theirH3K9me2 patterns (Fig. 5B).These observations were also supported by quantitative results calculating the correlation coefficiency between each experiment pair (Supplemental Fig 8B).Interestingly,RA-treated GLP 3A cells,which failed to establish H3K9me2 at most of these regions,were clustered together with ESCs but not RA-treated wildtype cells (Fig. 5B)，implying that the differentiation process was impaired.

Given that GLP can also generate H3K9mel products and used them as substrates for further methylation (Tachibana et al. 2O05),we performed ChIP-seq experiments for H3K9mel using the same samples.The same above-mentioned 3307 10-kb genomic regions were first chosen for analysis,and $\sim 6 6 \%$ of them displayed obvious gain of H3K9mel after RA treatment in wild-type cells (Fig. 5C)．Most of these regions displayed correctly established H3K9mel inGLP3A cells after RA treatment (Fig.5C).However, some of the regions that are defective in establishingH3K9me2duringdifferentiation displayed a reduction of H3K9mel at the ESC stage of the GLP 3A cells (Fig.5C).To analyze whether the reduced H3K9mel is correlated with reduced H3K9me2 in GLP 3A cells before or after RA-induced differentiation,we first selected all1O-kb regions with more than twofold increase of H3K9me2 during RA-induced differentiation in wildtype cells, categorized them into five subgroups according to their H3K9mel ratio (GLP 3A/wild type) at the ESC stage,and calculated theirH3K9me2 ratio (GLP 3A/wild type)before or after RA treatment.Regions with lower H3K9mel levels tended to display relatively lower H3K9me2 levels in GLP 3A ESCs (Supplemental Fig. S8C). On the other hand,all of these regions displayed similar degrees of defect in theirH3K9me2 levels in GLP 3A cells after RA treatment regardless of their initial levels of H3K9mel defect at the ESC stage.

We also directly analyzed the H3K9mel levels in the four samples above.All iO-kb windows with at least one case of more than twofold change between any pair of samples (5642 in total) were selected for clustering analysis.Clearly, the main differences among the four samples resided between ESCs and RA-treated cells but not between wild-type and GLP 3A cells (Supplemental Fig. S8D).Taken together, the above results collectively support that the primary defect of GLP 3A cells during RA treatment was the efficient establishment of H3K9me2 but not of H3K9mel.

Notably, $89 \%$ (2955 of 3307) of the regions with the highest $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ variation among the four samples were genic regions of 3079 genes $\begin{array} { r } { | P < 1 \mathtt { E } \ – 3 0 0 , } \end{array}$ ，binomial test), which was consistent with previous studies reporting that GLP and G9a primarily functioned at genic regions during differentiation of ESCs (Lienert et al. 2011). Therefore，we focused on the establishment of H3K9 methylation at genic regions for further analysis.Globally,H3K9me2 levels at genic regions were comparable between wild-type and GLP 3A ESCs (Supplemental Fig. 8E).However,afterRA treatment,GLP 3Acells displayed significantlylower H3K9me2 levels $\left( P < 2 . 2 \mathrm { E } { - } 1 6 \right)$ at genic regions than wild-type cells (Supplemental Fig 8F).We also calculated the H3K9me2 ratio between RA-treated and untreated cells and plotted the accumulative percentage curve against the fold change of the H3K9me2 ratio.GLP 3A cells displayed a clear shift in comparison with wild-type cells (Fig. 5D),indicatinga wide range of defects in establishing H3K9me2 at genic regions in GLP 3A cells.Such a defect was not observed for H3K9mel; instead，the GLP 3A cells displayed modestly higher H3K9mel enrichment at genic regions after RA treatment than their wild-type counterparts (Fig. 5E),and this probably reflected that less H3K9mel was converted to H3K9me2 in the GLP 3A cells.

To further analyze the defect of H3K9me2 establishment at genes in detail,we calculated the H3K9me2 ratio between RA-treated cells and ESCs for each gene and defined genes with significantly changed H3K9me2 levels during RA-induced differentiation in wild-type cells and GLP 3A cells $( P < 0 . 0 5$ in three biological replicates with at least 1.5-fold change). Clearly, far more genes gained H3K9me2 than those lost H3K9me2 (3091 vs.l01). In contrast,in GLP 3A cells,only 295 genes gained H3K9me2,while 1463 genes displayed decreased H3K9me2 levels (Fig. 5F).

We next plotted the H3K9me2 changes against the average H3K9me2 fold enrichment of ESCsandRA-treated cells.First,the overall distribution of the plot for the wildtype cells was much higher than the one for GLP 3A cells (Fig. 5G),indicating that the GLP 3A cells suffered from a wide range of defects in establishingH3K9me2 at genes during differentiation.Strikingly, $9 3 . 9 \%$ of the genes that acquired H3K9me2 in wild-type cells displayed obviously reduced H3K9me2 levels in GLP 3A cells (Fig. $^ { 5 \mathrm { G } , }$ right panel),and the average difference in H3K9me2 changes between wild-type and GLP 3A cells at these genes was 0.8 $\left( \log _ { 2 } \right)$ (Fig.5H).We next plotted the average fold enrichment of H3K9me2 at these genes forall12 samples,and theresultsclearly indicated defectiveH3K9me2 establishment in GLP 3A cells (Fig. 5I).

We also analyzed genes with significantly changed H3K9me2 levels during differentiation in GLP 3A cells. The results showed that a large number of genes in GLP 3A cells failed to maintain the H3K9me2 levels during differentiation (Supplemental Fig. 9A). These results again implicated that the GLP 3A cells failed to establish H3K9me2 at a wide range of genes during differentiation. Notably,at the 295 genes that acquired H3K9me2 in GLP 3A cells,the average difference in $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ changes between wild-type and GLP 3A cells was only 0.2 $\left( \log _ { 2 } \right)$ (Supplemental Fig. 9B),much more modest than the difference observed in Figure 5H.

Together, these results indicated that GLP3Acells had normal H3K9me2 levels at the ESC stage but failed to correctly establish H3K9me2 levels at genic regions during RA-induced cell differentiation.

# TheH3K9me1-binding activity of GLP is required for efficient silencing of pluripotent genes duringESCdifferentiation

To examine whether the defective establishment of H3K9me2 in GLP3Acells was associated with inefficient gene repression,we performed RNA sequencing (RNAseq)experiments for the above-mentioned samples.We chose genes acquired by H3K9me2 during RA-induced differentiation in wild-type cells (Fig.5G,red dots) and plotted their expression fold changes against their expression levels.The majority of these genes did not show more than twofold changes during RA treatment in wildtype cells,probably because H3K9me2 changes do not lead to strict linear changes in gene expression (Lienert et al.2011)，and sometimes H3K9me2 can also occur during transcription (Vakoc et al. 2OO5,20O6;Lee et al. 2006; Yuan et al.2OO7; Chaturvedi et al. 2009).Nevertheless,55l genes were repressed (Fig. $6 \mathrm { A } ,$ red bars),while only 2l5 genes were activated (Fig.6A，green bars) during RA treatment in wild-type cells,in line with a role of H3K9me2 establishment in silencing a subset of genes during differentiation. Remarkably， these 551 genes displayed similar expression levels in wild-type and GLP 3A ESCs (Fig.6B),but 11O of them were expressed at much higher levels (more than twofold) in GLP3A cells than in wild-type cells afterRA treatment (Fig. 6C).Moreover,even for the rest of the 44l genes that did not cross the twofold threshold,we observed an overall trend of upshift,which suggests that inefficient establishment of H3K9me2 contributed to inefficient silencing of these genes.

As a control, we also studied genes displaying significantly elevated H3K9me2 levels during RA-induced dif ferentiation in GLP 3A cells (Supplemental Fig.9A,red dots).Among these genes (295 in total; 151 have detectable expression), 70 genes were repressed in GLP 3A cells during RA-induced differentiation (Supplemental Fig.9C red bars),and their expression differences between GLP 3Aand wild-type cells did not show a particularly biased shift before or after RA treatment (Supplemental Fig.9D). Notably,among the 11O genes that failed to be efficiently repressed in GLP 3A cells,the gene ontology $| \mathrm { G O } |$ term "stem cell maintenance" was enriched for ${ > } 4 0$ -fold (Fig.7A),and several critical pluripotent genes,including Oct4，Nanog，and $F g / 4 ,$ were found within the list (Supplemental Table 1).The silencing defect of these genes in GLP 3A cells duringRA-induced differentiation was validated by RT-qPCR results (Fig. 7B). ChIP-seq results at these gene loci are shown in Figure 7C and were verified by ChIP-qPCR (Fig. 7D).

In conclusion, $> 1 0 0$ genes，including several critical pluripotent genes that should have been repressed during RA-induced differentiation,were not efficiently repressed due to impaired H3K9me2 establishment when the H3K9mel-binding activity of GLP was lost.

We also found that 329 genes failed to be activated in GLP 3Acells during RA-induced differentiation,unlike in the wild-type cells.Genes with GO terms related to differentiation such as “embryonic morphogenesis"and "pattern specification process" were enriched among these genes,including Sox1,Kdr,foxa1,Bmp,and Wnt family genes,indicatinga defect in differentiation (Sup plemental Fig.9E).We reason that these results are in line with the impairment in efficiently shutting down the pluripotent genes during RA-induced differentiation in GLP 3A cells.

# Discussion

GLP and G9a have relatively poor activities on nucleosomes in vitro;whether they catalyze H3K9 methylation on nucleosome substrates or histone octamers prior to nucleosome assembly during replication remains an open question (Shinkai and Tachibana 2Ol1). Our results demonstrated that GLP and G9a are capable of exerting strong catalytic activities on nucleosomal substrates when they are flanked by neighboring nucleosomes carrying H3K9 methylation (Fig.1). This suggests that GLP and G9a can use nucleosomes as their optimal substrates.

The enzymatic activation of GLP and G9a by neighboring nucleosomes methylated at H3K9 leads to several interesting questions:Is this a directional,processive methylation-spreadingactivity ora dose effect?Does this contribute to the mitotic inheritance of H3K9 methylation? What are the functional roles of such H3K9 methylation-mediated activation ofH3K9 methyltransferases?

# The activation of G9a and GLP by neighboring methylated nucleosomesisunlikely to be a directional, processive methylation-spreading activity

Our observation that GLP and G9a can sense neighboring nucleosomes premethylated at H3K9 suggests that GLP and G9a can efficiently spread H3K9 methylation.This partially resembles the H3K9 methylation-spreading ac tivity of Clr4 in fission yeast (Al-Sady et al. 2013). However,unlike $\mathrm { C l r 4 , }$ .which guides the spreading toward one specific direction (Al-Sady et al. 2Ol3),we believe that the spreading activity of GLP is most probably not a directional, processive event. Otherwise,it would be predicted that we could observe unaffected nucleating centers flanked by impaired H3K9me2 establishment in their surrounding regions in GLP 3A cells during RAinduced ESC differentiation,which we did not observe. Instead,we observed a general reduction of H3K9me2 within the affected regions (Fig.7C).This is not surprising,becausea mechanism that can reinforce a directional and processive spreading activity would require much more sophisticated machinery.

![](images/d1cb749250ddd8e109aef74366fae71575b85f9871c92313f2f06d6cf9179878.jpg)  
Figure 6. Inefficient gene repression in GLP 3A cells during RA-induced differentiation. $\vert A \vert$ Histogram showing the expression change of genes acquired by $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ (Fig.5G) during diferentiation in wild-type cells. (Black bars)Unchanged; (red bars)repressed; (green bars)activated. $| B , C |$ Expression comparison of the genes acquired by $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ in wild-type cells (Fig. 5G) between wild-type and GLP 3A ESCs (B)or RA-treated cells $| C |$ .Note: Red dots in $B$ and $C$ represent the same set of genes,and all of the dots shown represent the red dots in Figure 5G.

![](images/b624a80af2322ee0619551eab5d7a3a99f1160656a8b7d6a269855781b67526e.jpg)  
Figure 7.Examples of pluripotent genes with defects in $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ establishment and gene repression. $| A |$ GO term analysis of the 110 genes that failed to be eficiently repressed in RA-treated GLP 3A cells.(F.E.)Fold enrichment. $\left( B \right)$ RT-qPCR results showing the expression levels of the indicated genes in ESCs and RA-treated cells. $| C |$ Normalized $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ profiles at the indicated gene loci. The red lines represent amplified regions used in ChIP-qPCR validation experiments. $| D |$ ChIP-qPCR verified the H3K9me2 level at the indicated genes in ESCs and RA-treated cells.Error bars represent the standard error.

Instead, we favor the idea that the stimulation activities observed were due to the stabilization of GLP and G9a association with nucleosomes bearing H3K9 methylation，which leads to increased local concentration of GLP and G9a and efficient methylation of surrounding nucleosomes.Interestingly,a recent study reported that heterozygote GLP knockout mice displayed developmental delay and cranial abnormalities (Balemans et al. 2014). These phenotypes，including the bent nose phenotype observed in $\sim 3 0 \%$ of the heterozygote GLP knockout mice (Balemans et al.2Ol4),are highly similar to the phenotypes of the ${ \mathrm { G L P } } ^ { { \mathrm { M } } / { \mathrm { M } } }$ mice observed in our study, which suggests that both dosage and H3K9mel-mediated stimulation activityofGLPare critical forbone formationand craniofacial development.It remains unclear why bone formation appears to be one of the development processes most sensitive to the impairment of GLP-mediated H3K9 methylation. It is highly conceivable that the loss of H3K9mel binding and the subsequent stimulation activity of GLP mayaffect biological processes similar to those affected by the dose reduction of GLP. This can be well explained by our understanding that the H3K9mel-binding activity of GLPhelps to stabilize its association with nucleosomes bearingH3K9mel and increase its local concentration for efficient methylation. Moreover, we note that the heterozygote GLP knockout mice display much milder defects than the ${ \bf G L P } ^ { { \bf M } / { \bf M } }$ mice in this study. The heterozygote GLP knockout mice are viable despite some cranial abnormalities (Balemans etal.2Ol4),which indicates that the defect in H3K9mel-binding activity of GLP has a greater impact than losing one allele of GLP.

# Mitotic inheritance of H3K9 methylation does not require GLP's and G9a's methylation-spreading activity

H3K9me2 occupies more than $3 0 \%$ of the genome (Lienert et al.2O1l；Voigt et al.2Ol2)but can be rapidly reestablished during mitosis (Xu et al. 2Ol1). Considering that G9a localizes at the replication foci (Esteve et al.2006) and G9a and GLP are activated by premethylated neighboring nucleosomes,it is very attractive to hypothesize that GLP and G9a may travel with the replication machinery and copy-paste H3K9 methylation from the parental nucleosomes to the newly deposited nucleosomes.Knockout mice for either GLP or G9a are embryonic-lethal (Tachibana et al. 2OO2,2OO5),indicating that proper establishment and maintenance of H3K9 methylation are critical for mouse embryonic development.However,double-mutant mice losing H3K9 methylation-binding activities of both G9a and GLP can still undergo fairly normal embryonic development and be born alive (Fig. 3C).This suggests that the H3K9-binding activity and H3K9 methylation-mediated stimulation activity of G9a and GLP are largely dispensable for mitotic inheritance of H3K9me2 because defects in mitotic inheritance of histone modifications should lead to significant reduction of the modification and more severe phenotypes. Indeed, the changes of H3K9me2 levels in GLP 3A ESCs and mouse tissues were quite modest at the global level (Supplemental Fig. S6D,E).Therefore,it is much more likely that the fast re-establishment of H3K9 methylation during replication was mediated by the association between G9a, GLP, the replication machinery,and DNMT1 via a reinforcement mechanism (Esteve et al.2Oo6; Shinkai and Tachibana 2Oll； Zhu and Reinberg 2O1l） rather than copying of the existing marks from the parental histones. This conclusion is consistent with our previous observations that histone lysine methylations are generally not inherited at high resolution (Xu et al. 20l0,201l; Chen et al. 201l; Huang et al. 2013).

# Efficient establishment ofH3K9me2duringESC differentiationrequires GLP'sH3K9me1-mediated stimulation activity

Besides mitotic inheritance of H3K9 methylation，anotherscenarioinvolvingrapidestablishmentof H3K9me2 occurs during cell differentiation.As a facultative heterochromatin marker， H3K9me2 undergoes global changes during the differentiation processes of ESCs,mesenchymal stem cells,and hematopoietic stem cells (Tan et al.2009;Wen et al.2009;Lienert et al.2011; Chen et al.20l2;Wang et al. 20l3). Our results demonstrated that the H3K9mel-mediated stimulation of GLP played an important role in rapid establishment of H3K9me2 during RA-induced differentiation of ESCs.

Upon loss of H3K9mel binding and subsequent enzymatic stimulation of GLP,many genes,including several critical pluripotent genes,displayed defective H3K9me2 establishment and delayed repression during differentiation of ESCs.In vivo, the $\mathrm { G L P } ^ { \mathrm { \mathbf { \hat { M } } / M } }$ embryos went through the entire embryonic development stage，which may argue against a role of an H3K9me2 establishment defect observed in the in vitro ESC differentiation system.There are_several potential explanations.First，the GLpM/M embryos may indeed suffer froma delayed differentiation process even at the early development stage because the $\mathsf { \bar { G } L P } ^ { \mathrm { M / M } }$ embryos were lighter than their wild-type litrmates starting from E1O.5,which may be a consequence of delayed differentiation processes at earlier stages. Second,although bone ossification appears to be the most vulnerable process,milder but similar impacts could well be perceived in other cell types.It would be interesting to interrogate other cell differentiation systems,especially the osteoblasts differentiation process,which is highly relevant to the observed bone ossification defects in ${ \bf G } { \bf L } { \bf P } ^ { { \bf M } / { \bf M } }$ mice.

# Distinct roles of GLP and G9a

GLP and G9a share high sequence similarity and similar biochemical activities and are both required for H3K9me2 levels in vivo (Tachibana et al. 2OO5; Collins et al. 2008). However, GLP 3A mutant mice and G9a 3A mutant mice displayed dramatic phenotype differences.The H3K9 methylation-binding activity of GLP appeared to be much more important than that of G9a.It remains unclear whether this is due to their differential expression profile or their binding affinity difference toward H3K9mel and H3K9me2.The ankyrin repeat domain of G9a preferentially associates with H3K9me2，whereas the ankyrin repeat domain of GLP preferentially associates with H3K9mel (Collins et al.20o8). Engineering a GLP knock-in mouse line bearing an H3K9me2-binding ankyrin repeat domain may help to resolve this question in future studies.Another potential explanation is the distinct expression profile between GLP and G9a; although these two proteins can formheterodimers (Tachibana et al. 2005),they often display different tissue-specific expression profiles (Ohno et al. 20l3), suggesting that they may have functions independent of each other.

# Roleof GLP's H3K9me1-mediated stimulation activity in bone ossification

Bone ossification is mediated by osteocytes,which are originated from sequential differentiation of mesenchymal stem cells and osteoblasts.Chromatin modifications, including histone deacetylation and H3K9 methylation, regulate bone formation (Tan et al.2OO9；Bradley et al. 2011).During mouse embryonic development, histone H3K9 methyltransferases G9a and GLP emerge on E16.5 in prehypertrophic and hypertrophic chondrocytes in the growth plate as well as mesenchyme cells in the tooth germ，resulting in increased H3K9 methylation,which maybe important in regulating chondrocyte differentiation and tooth development (Ideno et al.2Ol3; Kamiunten et al.2Ol4).It is highly probable that during the differentiation process of mesenchymal stem cells and/or osteoblasts in $\mathrm { G L P ^ { M / M } }$ mice,defective establishment of $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ at genes that should be repressed led to inefficient gene silencing， incomplete differentiation of osteocytes,and defective ossification.

Finally, we note the potential relevance between the nonhistone substrates of GLP or G9a and the phenotypes that we observed in this study.In addition to histone H3, G9a has been reported to methylate other substrates, including histone Hl.4 and a number of nonhistone proteins (Sampath et al.2007; Rathert et al.2008; Collins and Cheng 2OlO). It is easily conceivable that GLP may also have additional substrates.To the best of our knowledge，no other methylated proteins have been reported to be specifically recognized by the ankyrin repeats of GLP or G9a.Considering H3K9 methylation remains the only known methyl mark that GLP recognizes via its ankyrin repeats,it is most likely that the cellular and developmental defects of GLP 3A mutants observed in this study are the consequences of impaired H3K9 methylation recognition and subsequent stimulation.However,we certainly cannot completely rule out the possibility that a fraction of the defects observed in this study may be attributed to the impaired association between GLP and an unknown methylated protein.

# Materialsandmethods

# Preparation of recombinant enzymes and substrates

Forhistone methyltransferase activity assays，His-tagged recombinant G9a (amino acids 601-1210，NP_O06700)and recombinant GLP (amino acids 635-1296,NP_001012536) covering both the SET domain and ankyrin repeats were expressed and purified from E.coli.Briefly,His-tagged G9a orHis-tagged GLP were expressed in BL21 (DE3) cells upon IPTG induction.The cells were pelleted and resuspended in lysis buffer $\mathrm { \Omega } ^ { ( 2 0 \mathrm { \ m M } }$ TrisHCl at $\mathrm { \ p H { 8 . 0 } } ,$ 150 mMNaCl, $1 \mathrm { m } \mathrm { M }$ EDTA, $1 ~ \mathrm { m M }$ PMSF)and then sonicated to release cellular proteins.The cell lysates were incubated with Ni-NTA beads (Qiagen).The beads were washed with 5O column volumes of lysis buffer containing $1 5 \mathrm { ~ m M }$ imidazole,and recombinant proteins were then eluted with lysis buffer containing $2 5 0 ~ \mathrm { m M }$ imidazole.The purified proteins were dialyzed against lysis buffer,snap-frozen,and stored at $- 8 0 ^ { \circ } \mathrm { C }$

All histones were purified under denatured conditions.Histone H3K9C/C1l0A with a C-terminal Flag-His dual tag was purified with Ni-NTAbeads and then used for MLA reactions.

Recombinant oligonucleosomes were assembled via sequential salt dialysis using pG5E4 plasmid DNA and histone octamers (Yuan et al.2Ol2).All oligonucleosomes were purified by gel filtration with a 3-mL CL-4B gel filtration column to remove free histone octamers.

# ITC

Recombinant G9a (amino acids 730-965，NP_665829)and recombinant GLP (amino acids 763-997,NP_001012536) covering the ankyrin repeats were expressed and purified from E.coli. The proteins were dialyzed againstITC buffer ( $1 0 \mathrm { m M }$ Tris-HCl at $\mathrm { p H 8 . 0 } ,$ $5 0 ~ \mathrm { m M }$ NaCl). The following peptides were synthesized withScilight-Peptide,Inc.:H3K9meO,ARTKQ TARKS TGGKA P；H3K9mel，ARTKQ TARK(me1)S TGGKA P；H3K9me2,

ARTKQ TARK(me2)S TGGKA P;and H3K9me3，ARTKQ TARK(me3)S TGGKA P.

ITC experiments were performed with ITC2o0 (MicroCal). Briefly, titration cells and syringes were washed with distilled waterandITC buffer.A total volume of $2 2 0 ~ \mu \mathrm { L }$ of protein (80- $1 2 0 \mu \mathrm { M } { \mu \mathrm { M } }$ was injected into the cell, $4 0 ~ \mu \mathrm { L }$ of peptides $\left( \sim 1 . 6 \ : \mathrm { m M } \right)$ was loaded into the syringe,and the system was equilibrated at $2 5 ^ { \circ } \mathrm { C }$ .The titration was performed with the following settings: 2 $\mu \mathrm { L }$ of peptide and 4-sec duration for each injection,2O injections in total, 18O-sec interval between each injection,and a stirring speed of $1 0 0 0 \mathrm { r p m }$ .The acquired data were processed with Origin software as instructed.

# MLA reactions

The MLA reactions were performed as described previously (Simon et al.2007).Briefly, $5 ~ \mathrm { m g }$ of histones was dissolved in 1 MHEPES $\mathrm { \{ p H 8 . 0 \} }$ 1 $4 \mathrm { ~ M ~ }$ urea,and $1 0 ~ \mathrm { m M }$ methionine and then reduced with DTT for1h.Next, the chemicals (N-methylethylamines with different methyl states)were added into the solutions for the alkylating reactions.Finally, $\beta$ -mercaptoetha nolwasadded to $5 \%$ to quench the reactions.All MLA reaction products were verified by MALDI-TOF mass spectrometry analysis for quality assurance.

# Histone methyltransferase assays

The reactions were performed as previously described (Yuan et al. 2012).A $4 0 \mathrm { - } \mu \mathrm { L }$ reaction mixture containing S-[methl- $^ 3 \mathrm { H } ]$ adenosyl-methionine (PerkinElmerLife Sciences),recombinant oligonucleosomes,and enzymes in HKMTassaybuffer $\mathrm { \Omega } ^ { \prime } 5 0 \mathrm { m M }$ Trisat $\mathrm { p H } 8 . 5 .$ $2 0 \ \mathrm { m M \ K C l } _ { , }$ 10mM $\begin{array} { r } { \mathbf { M g C l } _ { 2 } , } \end{array}$ $1 0 ~ \mathrm { m M }$ DTT,250 mM sucrose) was incubated for $^ \mathrm { ~ 1 ~ h ~ }$ at $2 5 ^ { \circ } \mathrm { C }$ .The reaction products were separated by $1 3 \%$ SDS-PAGE，transferred to PVDF membranes,and then subjected to autoradiography. For quantification,the membranes were stained by Coomassie blue G250 followed by liquid scintillation counting for each stained histone bands.

# Generation of knock-in mouse model

The targeting vectors were constructed according to previously published methods (Liu et al.2Oo3).Vectors were verified by sequencing and electrotransfected into R1 ESCs.After selection with G4l8 and ganciclovir, surviving clones were picked and then screened by genomic PCR.To confirm that the targeting events resulted in a functional and correctly mutated allele, a large fragment that covered the targeting region was amplified from cDNA and sequenced.The correct ESC clones were used to produce chimeras.F1 heterozygous mice were obtained by crossing chimera with C57BL/6 female mice.

# ESC derivation and cell culture

ESCs were derived from littermate blastocysts obtained by intercrossing $\mathrm { G L P ^ { + / M } }$ .The obtained clones were genotyped and karyotyped.Two pairs of ESC lines were obtained.ESCs were maintained in the presence of feeders in standard ES medium.Prior to ChIP-seq and RNA-seq experiments,ESCs were cultured in feeder-free conditions in ES medium supplemented with 2i for two passages to eliminate feeder cells.For RA induction, $1 ~ \mu \mathrm { M }$ all-trans RA was added into the medium, and LIF was withdrawn simultaneously. The RA induction experiments were performed in triplicate.For EB differentiation, $2 0 \mathrm { - } \mu \mathrm { L }$ liquid drops containing lOoo ESCs were cultured for $^ \textrm { \scriptsize 2 d }$ in ES medium without LIF.Next, the cell aggregates were transferred to ultralow attachment plates and cultured for various days in the presence of 1 ${ \mathbf { \mu } } _ { \mathbf { \mu } } \mathbf { M }$ RA.

# ChIP-seq and ChIP-qPCR

ChIP was performed according to the Diagenode iDeal ChIP-seq kit with minor modifications.Briefly，2 million cells were harvested in PBS and cross-linked with $1 \%$ formaldehyde for 2 min.After sonication，chromatin from 1 million cells was subjected to ChIP using the iDeal ChIP-seq kit (Diagenode). For ChIP-qPCR assays，the $\mathrm { H } 3 \mathrm { K } 9 \mathrm { m e } 2$ enrichment was normalized to Olfrl8 (a constitutively inactive gene) promoter (Magklara et al.2Oll).For ChIP-seq, libraries were constructed with KAPA library preparation kits and then single-end-sequenced with HiSeq 2Ooo. Three biological replicate experiments were performed using independently cultured cells.The antibodies used are listed in Supplemental Table 2.

# RT-qPCR and RNA-seq

RNA samples were extracted using Trizol reagent,and genomic DNA contaminations were eliminated with Turbo DNase (Life Technologies).cDNA samples were synthesized using the Takara PrimeScript first strand cDNA synthesis kit.The qPCR results were normalized against a housekeeping gene, Gapdh. For high-throughput sequencing, total RNA were extracted,and the subsequent library preparation steps were carried out according to standard Illumina procedures at our in-house sequencing center and subjected to single-end sequencing with HiSeq 2000. Two biological replicates were performed.

# Bisulfite sequencing

Bisulfite conversion was performed with the EpiTect bisulfite kit (Qiagen). The Oct4 promoter region was amplified with nested primers,and the PCR fragments were purified and ligated to pMD18-T vectors.At least eight clones per sample were sequenced.

# Bioinformatics analysis

Sequencing depths were between 24 million and 6l million reads for the four RNA-seq experiments and between 43 million and 180 millon reads for ChIP-seq experiments.The sequencing depth and mappingpercentage are listed in Supplemental Table 3.

H3K9me2 ChIP-seq reads were mapped to a mouse genome $\left. \mathrm { m m } 9 \right.$ ,and only uniquely mapped and nonredundant reads were kept for further analysis.The mapped reads were extended to the average fragment size and binned into 25-base-pair (bp)windows for visualization in the Integrative Genomics Viewer (IGV) browser.For the analysis of genomic regions, the mouse genome was divided into $1 0 \mathrm { - k b }$ windows,and the fold enrichment of H3K9me2 for each window was calculated as the ratio of normalized read numbers between ChIP and input samples. Any window that had $< 1 0 0$ reads in input samples was filtered because of insufficient information.For each window,the fold enrichment of H3K9me2 was calculated as the ratio of normalized read density between ChIP and input samples,and the final H3K9me2 enrichment value was an average of values from three replicate experiments.Ten-kilobase windows with at least one case of more than a twofold change between any pair of samples were selected for average linkage clustering performed with Cluster 3.0 software.

RNA-seq data were mapped to the mouse genome $\left. \mathrm { m m } 9 \right.$ with TopHat (version l.4.1),and the transcriptionlevels were quantified with Cufflinks (version 2.O.2) software with two replicates.The ChIP-seq and RNA-seq data were deposited in the Gene Expression Omnibus database of NCBI under accession number GSE54412.

# Mouse phenotype analysis

All of our study procedures complied with the guideline of the National Institute of Biological Sciences,Beijing,for the care and use of laboratory animals.

MicroCT scanning was performed with an Aloka Latheta LCT2Oo.Skeleton preparation and Alizarin red staining were carried out according to a previous study (Dobreva et al. 2006). Briefly,sacrificed animals were skinned and fixed in EtOH and then stained in $0 . 1 \%$ Alizarin red solution for at least $3 \mathrm { ~ d ~ }$ Next,the samples were transferred into $1 \%$ KOH to remove tissues.The calvaria of animals were carefully dissected and photographed.

Mouse suckling assays were adapted from previous studies (Hongo et al.2OoO).Briefly,shortly after giving birth to newborns,female mice were given $0 . 0 5 ~ \mathrm { m g / g }$ pentobarbital sodium intraperitoneally for anesthesia and laid on a heating block $( 3 7 ^ { \circ } \mathrm { C } )$ ， $\mathrm { G L P ^ { + / + } }$ and $\mathrm { G L P ^ { M / M } }$ pups were put on their mother's abdomen at $0 . 5 ~ \mathrm { c m }$ distance from nipples.We divided the suckling process into two steps: (1) searching and licking nipples (known as the rooting reflex) and (2) stably attaching to the nipples with rhythmic suckling. Successfully accomplishing each step within1 min was scored as l,otherwise,it was scored as O.Each newborn mouse was given two to three trials.At least seven newborns of each genotype were tested.The final "search and lick score" and “attachment score" were calculated as summed scores divided by trials.Statistical significance was tested using Fisher's exact test.

# Acknowledgments

We thank Y.Shinkaiat RIKEN forcDNA and discussion.This work was supported by China National Science Foundation grant 314250l3,Chinese Ministry of Science and Technology grants 2015CB856200 and 2011CB965300，Strategic Priority Research Program of the Chinese Academy of Sciences grant XDBO8OlOlO3,and the Howard Hughes Medical Institute International Early Career Scientist Program to B.Z.This work was also supported by China National Science Foundation grant 81301803 to H.W.,and grant 31401114 to Z.Z.

# References

Al-SadyB,MadhaniHitenD,Narlikar GeetaJ.2Ol3.Division of laborbetween the chromodomains of HP1 and Suv39 methylase enables coordination of heterochromatin spread.Mol Cell 51: 80-91.   
Balemans MCM，Ansar M,Oudakker $\mathsf { A R } ,$ van Caam APM, Bakker B,Vitters $\operatorname { E L } ,$ van der Kraan PM,de Bruijn DRH, Janssen SM,Kuipers AJ,et al.2Ol4.Reduced euchromatin histone methyltransferase 1 causes developmental delay, hypotonia,and cranial abnormalities associated with increased bone gene expression in Kleefstra syndrome mice. Dev Biol 386: 395-407.   
Bradley EW,McGee-Lawrence ME,Westendorf JJ.2011.Hdacmediated control of endochondral and intramembranous ossification. Crit Rev Eukaryot Gene Expr 21:l0l-113.   
Brent MM，Marmorstein R.2Oo8.Ankyrin for methylated lysines.Nat Struct Mol Biol15:221-222.   
Chaturvedi CP,Hosey AM,Palii C,Perez-Iratxeta C,Nakatani Y,Ranish JA,DilworthFJ,Brand M.2OO9.Dual role for the methyltransferase G9a in the maintenance of $\beta$ -globin gene 106: 18303-18308.   
Chen X, Xiong J, Xu M, Chen S,Zhu B. 2011. Symmetrical modification within a nucleosome is not required globally for histone lysine methylation. EMBO Rep 12: 244-251.   
Chen X, Skutt-Kakaria K,Davison J, Ou Y-L,Choi E,Malik P, Loeb K, Wood B, Georges G, Torok-Storb B, et al. 2012. G9a/ GLP-dependent histone H3K9me2 patterning during human hematopoietic stem cell lineage commitment.Genes Dev 26: 2499-2511.   
Chow J, Heard E. 2O09. X inactivation and the complexities of silencing a sex chromosome.Curr Opin Cell Biol 21: 359-366.   
Collins R, Cheng X.2010.A case study in cross-talk: the histone lysine methyltransferases G9a and GLP.Nucleic Acids Res 38:3503-3511.   
Collins RE, Northrop JP, Horton JR,Lee DY, Zhang X,Stallcup MR,Cheng X. 2008. The ankyrin repeats of G9a and GLP histone methyltransferases are mono- and dimethyllysine binding modules.Nat Struct Mol Biol 15: 245-250.   
Desbaillets I,Ziegler U,Groscurth ${ \bf P } ,$ Gassmann M. 2000. Embryoid bodies: an in vitro model of mouse embryogenesis. Exp Physiol 85: 645-651.   
Dobreva $\mathbf { G } ,$ Chahrour M,Dautzenberg M, Chirivella L, Kanzler B,Farinas I,Karsenty $\mathbf { G } ,$ Grosschedl R．2006．SATB2 is a multifunctional determinant of craniofacial patterning and osteoblast differentiation. Cell 125: 971-986.   
Epsztejn-Litman S,Feldman N,Abu-Remaileh M,Shufaro ${ \mathrm { Y , } }$ （20 Gerson A, Ueda J, Deplus $\begin{array} { r } { \mathbf { R } , } \end{array}$ Fuks F, Shinkai Y, Cedar ${ \mathrm { H } } ,$ et al. 2008.De novo DNA methylation promoted by G9a prevents reprogramming of embryonically silenced genes.Nat Struct Mol Biol 15:1176-1183.   
Esteve PO, Chin HG, Smallwood A, Feehery GR,Gangisetty ${ \bf O } ,$ （204号 Karpf AR,Carey MF,Pradhan S.2OO6.Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication. Genes Dev 20: 3089-3103.   
Feldman N, Gerson $\mathbf { A } ,$ Fang J, LiE, Zhang Y, Shinkai Y,CedarH, Bergman Y. 2Oo6.G9a-mediated irreversible epigenetic inactivation of Oct-3/4 during early embryogenesis. Nat Cell Biol 8: 188-194.   
Hongo T，Hakuba A， Shiota K,Naruse I.2O00. Suckling dysfunction caused by defects in the olfactory system in genetic arhinencephaly mice.Neonatology 78:293-299.   
Huang C, Xu M, Zhu B.20l3. Epigenetic inheritance mediated by histone lysine methylation: maintaining transcriptional states without the precise restoration of marks? Philos Trans R Soc Lond B Biol Sci 368: 20110332.   
Ideno H, ShimadaA, Imaizumi K, Kimura H, Abe M, Nakashima $\mathbb { K } ,$ （20 Nifuji A.2Ol3.Predominant expression of H3K9 methyltransferases in prehypertrophic and hypertrophic chondrocytes during mouse growth plate cartilage development. Gene Expr Patterns 13:84-90.   
Jia $\mathbf { G } ,$ Wang W, LiH, Mao Z, Cai G, Sun J, Wu H, Xu M, Yang P, Yuan W, et al. 20o9.A systematic evaluation of the compatibility of histones containing methyl-lysine analogues with biochemical reactions. Cell Res 19:12i7-1220.   
Kamiunten $\mathrm { \Delta T , }$ Ideno ${ \mathrm { H } } ,$ Shimada A, Nakamura Y,Kimura ${ \mathrm { H } } ,$ （204号 Nakashima $\mathbb { K } ,$ Nifuji A.2Ol4.Coordinated expression of H3K9 histone methyltransferases during tooth development in mice.Histochem Cell Biol doi: 10.1107/s00418-014-1284-0.   
Lee DY, Northrop JP, Kuo MH, Stallcup MR.2006.Histone H3 lysine 9 methyltransferase G9a is a transcriptional coactivator for nuclear receptors.JBiol Chem 281: 8476-8485.   
Lienert F,Mohn F, Tiwari VK,Baubec T,Roloff TC,Gaidatzis D, Stadler MB,Schubeler D.2Ol1.Genomic prevalence of heternrhromatir H3KOme2 and tranerrintinn dn nnt dicrriminate pluripotent from terminally differentiated cells.PLoS Genet 7: e1002090.   
Liu P,Jenkins N,Copeland N.2OO3.A highly efficient recombineering-based method for generating conditional knockout mutations. Genome Res 13: 476-474.   
Magklara $\mathbf { A } ,$ Yen A, Colquitt Bradley $M ,$ Clowney EJ,Allen W, Markenscoff-Papadimitriou $\scriptstyle \mathbf { E } ,$ Evans Zoe $\mathbf { A } ,$ Kheradpour ${ \bf P } ,$ （204号 Mountoufaris $\mathbf { G } ,$ Carey $\scriptstyle \mathbf { C } ,$ et al. 20l1.An epigenetic signature for monoallelic olfactory receptor expression. Cell 145: 555-570.   
Maksakova I, Thompson P, Goyal P,Jones S, Singh P, Karimi M, Lorincz M.2013.Distinct roles of KAPl,HPl and G9a/GLP in silencing of the two-cell-specific retrotransposon MERVL in mouse ES cells. Epigenet Chromatin 6: 15.   
Margueron $\scriptstyle \mathrm { \mathrm { ~ R , ~ } }$ Justin N,Ohno K, Sharpe ML, Son J, Drury Ii WJ, Voigt P,Martin SR,Taylor WR,De Marco V,et al. 2O09.Role of the polycomb protein EED in the propagation of repressive histone marks. Nature 461: 762-767.   
Mermoud JE, Popova B, Peters AHFM,Jenuwein T, Brockdorff N. 2002.Histone H3 Lysine 9 methylation occurs rapidly at the onset of random X chromosome inactivation.Curr Biol 12: 247-251.   
Ohno $\operatorname { H } ,$ Shinoda $\scriptstyle \mathrm { \mathrm { K } , }$ Ohyama $\mathrm { K , }$ Sharp $\scriptstyle \mathbf { L } Z ,$ Kajimura S.2013. EHMTl controls brown adipose cell fate and thermogenesis through the PRDMl6 complex. Nature 504: 163-167.   
Rathert P,Dhayalan A，Murakami M，Zhang X，Tamas $\begin{array} { r } { \mathbf { R } , } \end{array}$ Jurkowska R,Komatsu Y, Shinkai ${ \mathrm { Y , } }$ Cheng $\mathbf { X } ,$ Jeltsch A. 2008.Protein lysine methyltransferase G9a acts on nonhistone targets. Nat Chem Biol 4: 344-346.   
Sampath SC, Marazzi I, Yap KL,Krutchinsky AN,Mecklenbrauker I Viale A,Rudensky E,Zhou MM, Chait BT, Tarakhovsky A. 2007.Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly. Mol Cell 27: 596-608.   
Shinkai Y, Tachibana M.2O11.H3K9 methyltransferase G9a and the related molecule GLP. Genes Dev 25: 781-788.   
Simon MD, Chu F,Racki LR,de la Cruz CC,Burlingame AL, Panning B,Narlikar GJ, Shokat KM. 20O7. The site-specific installation of methyl-lysine analogs into recombinant histones. Cell 128:1003-1012.   
Tachibana M, Sugimoto $\textstyle \mathrm { K } ,$ Fukushima T, Shinkai Y.2001. Set domain-containing protein, G9a, is a novel lysine-preferring mammalian histone methyltransferase with hyperactivity and specific selectivity to lysines 9 and 27 of histone H3.J Biol Chem 276: 25309-25317.   
Tachibana M, Sugimoto K, Nozaki M, Ueda J, Ohta T, Ohki M, Fukuda M, Takeda N,Niida H, Kato $\textstyle \mathrm { H } ,$ et al. 2002.G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis.Genes Dev 16:1779-1791.   
Tachibana M, Ueda J, Fukuda M, Takeda N, Ohta T, Iwanari H, Sakihama T,Kodama T,Hamakubo T,Shinkai Y. 2005. Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9. Genes Dev 19: 815-826.   
Tan J,Lu J, Huang W,Dong Z, Kong C, LiL,Gao $\scriptstyle { \mathrm { ~ L , } }$ Guo J, Huang B. 2009.Genome-wide analysis of histone H3 lysine9 modifications in human mesenchymal stem cell osteogenic differentiation.PLoS ONE 4:e6792.   
Trojer P, Reinberg D.2OO7.Facultative heterochromatin:is there a distinctive molecular signature? Mol Cell 28: 1-13.   
Turgeon B,Meloche S. 2OO9. Interpreting neonatal lethal phenotypes in mouse mutants: insights into gene function and human diseases.Physiol Rev 89:1-26.   
Vakoc CR，Mandat SA，Olenchock BA，Blobel GA．2005. with transcription elongation through mammalian chromatin. Mol Cell19: 381-391.   
Vakoc CR,Sachdeva MM,Wang H,Blobel GA.2OO6.Profile of histone lysine methylation across transcribed mammalian chromatin.Mol Cell Biol 26:9185-9195.   
VoigtP,LeRoy G,Drury IiiWilliamJ,Zee BarryM, SonJ,Beck David B, Young Nicolas $\scriptstyle { \mathrm { ~ L , } }$ Garcia Benjamin A,Reinberg D. 2012．Asymmetrically modified nucleosomes.Cell 151: 181-193.   
Wang L,Xu S,Lee JE,Baldridge A,Grullon S,Peng W,Ge K. 2013.Histone H3K9 methyltransferase G9a represses PPARy expression and adipogenesis.Embo /32:45-59.   
Wen B,Wu H, Shinkai Y, Irizarry RA,Feinberg AP.2OO9.Large histone H3 lysine 9 dimethylated chromatin blocks distinguish differentiated from embryonic stem cells.Nat Genet 41: 246-250.   
Wu H, Chen X,XiongJ,LiY,LiH,Ding X,Liu S,Chen S,Gao S, Zhu B.2Ol1.Histone methyltransferase G9a contributes to H3K27 methylation in vivo. Cell Res 21: 365-367.   
Xu M,Long C,Chen X,Huang C,Chen S,Zhu B.2010. Partitioning of histone H3-H4 tetramers during DNA replication-dependent chromatin assembly. Science 328:94-98.   
Xu M,Wang W,Chen S,Zhu B.20ll.A model for mitotic inheritance of histone lysine methylation. EMBO Rep 13: 60-67.   
Yuan X,Feng W,Imhof A,GrummtI,Zhou Y.20O7.Activation of RNA polymerase I transcription by cockayne syndrome group B protein and histone methyltransferase G9a.Mol Cell 27: 585-595.   
Yuan W,Wu T,Fu H,Dai C,Wu H,Liu N,LiX,Xu M,Zhang Z, Niu T,et al.2Ol2.Dense chromatin activates polycomb repressive complex 2 to regulate H3 Lysine 27 methylation. Science 337: 971-975.   
Zhu B,Reinberg D.2Ol1. Epigenetic inheritance: uncontested? Cell Res 21: 435-441.

# Recognition of H3K9 methylation by GLP is required for efficient establishment of H3K9 methylation, rapid target gene repression, and mouse viability

Nan Liu, Zhuqiang Zhang, Hui Wu, et al.

Genes Dev. 2015 29: 379-393 originally published online January 30, 2015   
Access the most recent version at doi:10.1101/gad.254425.114

# Supplemental Material

http://genesdev.cshlp.org/content/suppl/2015/01/29/gad.254425.114.DC1.html

# References

Thisarticle cites 49 articles,17 of which can be accessed free at: http://genesdev.cshlp.org/content/29/4/379.full.html#ref-list-1

# Creative Commons License

This article is distributed exclusively by Cold Spring Harbor Laboratory Press for the first sixmonths after the full-issue publication date (see   
http://genesdev.cshlp.org/site/misc/terms.xhtml). After six months,it is available under a Creative Commons License (Atribution-NonCommercial 4.0 International),as described at http://creativecommons.org/licenses/by-nc/4.0/.

# Email Alerting Service

Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or click here.