# A highly recombined, high-density, eight-founder wheat MAGlC map reveals extensive segregation distortion and genomic locations of introgression segments

Keith A. Gardner1,\*†, Lukas M.Wittern2,† and lan J. Mackay1

1The John Bingham Laboratory,National Institute of Agricultural Botany (NIAB),Cambridge, UK   
²Department of Plant Sciences, University of Cambridge,Cambridge,UK

Received6August2015;   
revised 13 October2015;   
accepted 24 October 2015.   
\*Correspondence (Tel +44(0)1223342484; fax +44(0)1223277602;email   
keith.gardner@niab.com)   
+These two authors wish to be considered a: equal first authors.

Keywords:Multiparent Advanced Generation Intercross (MAGIC),highdensity map,wheat,recombination, segregation distortion,introgression.

# Summary

Multiparent Advanced Generation Intercross (MAGlC) mapping populations offer unique opportunities and challnges for marker and QTL mapping in crop species.We have constructed the first eight-parent MAGlC genetic map for wheat, comprising 18 601 SNP markers.We validated the accuracy of our map against the wheat genome sequence and found an improvement in accuracy compared to published genetic maps.Our map shows a notable increase in precision resulting from the three generations of intercrossing required to create the population.This is most pronounced in the pericentromeric regions of the chromosomes. Sixteen percent of mapped markers exhibited segregation distortion (SD) with many occurring in long $( > 2 0 ~ \mathsf { c M } )$ blocks.Some of the longest and most distorted blocks were collinear with noncentromeric high-marker-density regions of the genome,suggesting they were candidates for introgression fragments introduced into the bread wheat gene pool from other grass species. We investigated two of these linkage blocks in detail and found strong evidence that one on chromosome 4AL,showing SD against the founder Robigus,is an interspecific introgression fragment.The completed map isavailable from http://www.niab.com/pages/id/326/Resources.

# Introduction

Multiparent mapping populations are now being widely developed in plant species,and combine high genetic recombination with high diversity. Examples include nested association mapping populations (NAM,Yu et al.,20o8), the Arabidopsis multiparent RIL population (AMPRIL，Huang et al.，2011） and Multiparent Advanced Generation Intercross(MAGIC） populations. The MAGIC approach was first advocated for crops in2007 (Cavanagh et al.，2008;Mackay and Powell,2007) and MAGIC populations have subsequently been developed in a range of species,such as rice (Bandillo et al.，2o13),barley (Sannemann et al.， 2015)， tomato (Pascual et al.， 2015) and wheat (Huang et al.，2012;Mackay et al.，2014;Milner et al.，2015; Thepot et al.,2015).

Multiparent Advanced Generation Intercross populations are multifounder equivalents of the advanced intercross introduced by Darvasi and Soller (1995)and are closely related to the heterogeneous stock and composite cross-populations used in mouse genetics (Mott et al.，2ooo;Threadgill and Churchill, 2012).They are created by several generations of intercrossing among multiple founder lines leading to greater accumulation of recombination events and hence greater mapping precision. Multiple founders contribute more allelic and phenotypic diversity than captured in typical biparental mapping populations,raising the numbers of QTL that segregate and the types of trait and locus interactions that can be investigated.A MAGlC population founded from good representation of a breeder's gene pool also offers the opportunity to explore patterns of genomic diversity in that gene pool,such as identifying linkage blocks under fertility or viability selection and locating introgression fragments introduced from other species in the breeding process.

The NIAB eight-parent winter wheat MAGlC population (Mackay et al.，2014) was developed in partnership with UK breeders to represent the diversity of UK wheat germplasm. Founder varieties were Alchemy，Brompton，Claire，Hereward, Rialto,Robigus, Soissons and Xi19.Here,we describe the creation and validation of a high-density genetic map in this population, based on the llumina Infinium iSelect 8OK SNP array (http:// www.illumina.com/).This is the first publically available eightfounder MAGlC map created for wheat. We estimate the precision and accuracy of our map by reference to the wheat genome sequence and in comparison with existing wheat highdensity genetic maps.Finally,we map recombination rates,blocks of segregation distortion (SD) and blocks of high marker density, andinfer the genomic locations ofblocks of interspecific introgression.

# Results

# Genotyping

A total of 643 F4 MAGlC lines (assayed as F5 progeny bulks) passed stringent quality control and were used for mapping,and 20 639 SNP markers were scorable and polymorphic,compared to 25 499 markers in a comprehensive UK wheat association mapping panel ('WAGTAIL'，520 varieties，similarly genotyped and scored by K.A.Gardner),suggesting the MAGlC population has captured ${ > } 8 0 \%$ of the genetic diversity of UK wheat germplasm.Fifty-three markers with heterozygotes or missing data in the founder lines could not be used for mapping with R/ mpMap (Huang and George，2011).Of the remaining 20 586 markers，18750 $( 9 1 \% )$ were scored as codominant and 1836 $( 9 \% )$ as dominant; 664 of the dominant loci were nulls $( 3 . 2 \% )$ compared to $5 . 4 \%$ of single-locus scorable SNPs showing null alleles in Wang et al. (2014).For codominant markers, $2 . 4 \%$ residual heterozygosity was observed,compared to an expectationof $2 . 2 \%$ ．Four PCR markers were also genotyped (Appendix S1).

# Linkage map

In total,18 601 markers were mapped across all chromosomes. The 2042 unmapped markers (Table S1) are a highly nonrandom sample: $30 \%$ are dominant (compared to $3 \%$ of mapped markers), $1 6 \%$ are nulls $( 2 \%$ mapped）and $1 7 . 5 \%$ show SD witha false discovery rate('fdr') $< 0 . 0 1$ 1 $9 . 7 \%$ mapped)．All unmapped markers were treated as traitsand QTL mapped to the finished map，and their flankingmarkers were blasted against both wheat genome sequences (International Wheat Genome Sequencing Consortium，2014，http://www.wheatgenome.org/, 'IWGSC';Chapman et al.，2015，‘CHAPMAP');1118 ofthe unmapped markers had matching QTL and blast hits (with $2 9 8 \%$ identity，Table S1).From the distribution of these unmapped markers (Table S1，'data analysis')，both SD and translocations correlate with failure to map.About $20 \%$ of these markers were aligned to chromosome 5B or 7B,acrossa segregating translocation breakpoint and $12 \%$ were from chromosome 3B,which has two major SD blocks.Several large blocks of co-localized segregation distorted markers were unmappable，includinga 104-marker block with SD against Robigus from the distal end of 4A.

Our NIAB MAGIC linkage map，'NIAB2015'，is dense and compact (Figure 1，data in Table S2).Summary statistics are shown in Table 1.The number of markers mapped per chromosome varied from 80 (4D) to 2327 (1B). About $3 7 \%$ 1 $50 \%$ ，and $13 \%$ of markers were mapped to the A,B and D genomes, respectively.The map length totalled $5 4 0 5 ~ { \textcircled { \times } } { \mathrm { ~ M ~ } }$ ，with individual chromosomes ranging from 126 to $3 8 6 ~ { \textcircled { M } }$ .Aand B genome chromosomes together showed relatively low variation in length and were about $60 \%$ longer than D genome chromosomes. Altogether, the 18 601 markers were mapped to 4578 unique sites across the genome,with $4 1 \%$ $4 7 \%$ and $12 \%$ of the unique sites mapping to the A，B and D genomes，respectively.The number of unique sites per cM isapproximately one for the Aand B genomes (0.94 and1.02,respectively) but only 0.41 per cM for the D genome,reflecting the well-documented lower polymorphism of the D genome of hexaploid wheat (Wang et al., 2014). For A and B genomes,chromosome map length in cM is strongly associated with the number of unique sites (adjusted $R ^ { 2 }$ 0.69, $P = 0 . 0 0 0 1 3 2 ;$ ，witha gradient of O.53cM/uniquesite （204 $( 9 5 \% \ C \vert \ 0 . 3 2 – 0 . 7 4 )$ ,very similar to the default minimum genetic distance of 0.5oo3 cM between proximal recombination fraction bins in mpMap (corresponding to a recombination fraction,rf,of 0.005). Indeed, $6 9 \%$ of proximal unique sites in the AB genomes are $0 . 5 ~ { \mathsf { c M } }$ apart,confirming the compactness of NlAB2015.

# Internal validations

The genomewide heatmap of the recombination fraction matrix and logarithm of odds (LOD) values for NlAB2015 shows distinctly separated chromosomes and very few off-diagonal low-value (rf 0-0.1) colours (Figure 2).Two exceptions are evident.Firstly,the well-known rye introgression on chromosome 1B (Villareal et al., 1991;Worland and Snape,20o1) is segregating in the population (present in founders Brompton and Rialto),causing long range linkage disequilibrium (ina similarmanner to Huang et al.,2012). During mapping，markers that are segregating within native wheat chromosomes in the region of the rye introgression will be 'pushed to the side'of the markers which are only segregating between rye and wheat.This is evident in our map,where we haveopted to include all these markers.Secondly，thereis segregation for the occurrence of a common wheat translocation between chromosomes 5BS and 7BS (Badaeva et al.，2007), although with unknown parental frequencies.The largest distinct group of unmapped markers are from around the translocation breakpoint.

![](images/0df78d9ecdc0ad58cc53bc894244a6d0a729484f51bc9eac4645dc9f964d6b4a.jpg)  
Figure 1 The NIAB2015 MAGIC genetic map. Short arm of each chromosome at top (O cM).

Table 1 MAGlC map summary statistics and comparison to other maps (see Table 2 for description of other maps)   

<html><body><table><tr><td></td><td></td><td colspan="4">Other maps</td><td colspan="3">NIAB2015 overlap*</td></tr><tr><td></td><td>NIAB 2015</td><td>CM2014</td><td>9KCONS</td><td>9KMAGIC</td><td>SynOp</td><td>CM2014</td><td>9KCONS</td><td>9KMAGIC</td></tr><tr><td>Total length (cM)</td><td>5405</td><td>11 185</td><td>5192</td><td>3722</td><td>3242</td><td></td><td></td><td></td></tr><tr><td>Max chromosome length</td><td>386</td><td>743</td><td>396</td><td>300</td><td>225</td><td></td><td></td><td></td></tr><tr><td>Min chromosome length</td><td>126</td><td>295</td><td>99</td><td>73</td><td>67</td><td></td><td></td><td></td></tr><tr><td>Average chromo length (A)+</td><td>287</td><td>619</td><td>284</td><td>203</td><td>156</td><td></td><td></td><td></td></tr><tr><td>Average chromo length (B)</td><td>301</td><td>552</td><td>277</td><td>201</td><td>155</td><td></td><td></td><td></td></tr><tr><td>Average chromo length (D)</td><td>184</td><td>427</td><td>80</td><td>128</td><td>152</td><td></td><td></td><td></td></tr><tr><td>Total No markers</td><td>18 601</td><td>40 267</td><td>7497</td><td>4300</td><td>N/A</td><td>15 672</td><td>2840</td><td>1663</td></tr><tr><td>Max markers/chromosome</td><td>2327</td><td>3471</td><td>768</td><td>451</td><td>N/A</td><td>1906</td><td>284</td><td>162</td></tr><tr><td>Min markers/chromosome</td><td>80</td><td>296</td><td>38</td><td>14</td><td>N/A</td><td>67</td><td>5</td><td>4</td></tr><tr><td>% markers A</td><td>37</td><td>38</td><td>46</td><td>48</td><td>N/A</td><td>37</td><td>46</td><td>50</td></tr><tr><td>% markers B</td><td>50</td><td>46</td><td>46</td><td>45</td><td>N/A</td><td>50</td><td>46</td><td>43</td></tr><tr><td>% markers D</td><td>13</td><td>16</td><td>8</td><td>7</td><td>N/A</td><td>13</td><td>8</td><td>7</td></tr><tr><td>Total unique sites</td><td>4578</td><td>5564</td><td>3010</td><td>1813</td><td>1446</td><td>2897</td><td>1436</td><td>966</td></tr><tr><td>Max unique sites/chromo</td><td>452</td><td>387</td><td>265</td><td>202</td><td>114</td><td>234</td><td>119</td><td>88</td></tr><tr><td>Min unique sites/chromo</td><td>39</td><td>101</td><td>30</td><td>11</td><td>28</td><td>32</td><td>4</td><td>4</td></tr><tr><td>Average unique sites (A)</td><td>270</td><td>282</td><td>188</td><td>114</td><td>66</td><td>161</td><td>93.6</td><td>65</td></tr><tr><td>Average unique sites (B)</td><td>309</td><td>328</td><td>195</td><td>125</td><td>89</td><td>188</td><td>92.1</td><td>62</td></tr><tr><td>Average unique sites (D)</td><td>75</td><td>185</td><td>47</td><td>20</td><td>52</td><td>65</td><td>19.4</td><td>11</td></tr><tr><td>Unique sites/cM</td><td>0.85</td><td>0.50</td><td>0.60</td><td>0.49</td><td>0.44</td><td></td><td></td><td></td></tr><tr><td>Unique sites/cM (A)</td><td>0.94</td><td>0.46</td><td>0.66</td><td>0.56</td><td>0.42</td><td></td><td></td><td></td></tr><tr><td>Unique sites/cM (B)</td><td>1.02</td><td>0.59</td><td>0.70</td><td>0.62</td><td>0.57</td><td></td><td></td><td></td></tr><tr><td>Unique sites/cM (D)</td><td>0.41</td><td>0.43</td><td>0.26</td><td>0.15</td><td>0.34</td><td></td><td></td><td></td></tr></table></body></html>

\*These comparisons only include markers variable in NIAB2015. +(A) refers to A genome. $^ { \ddagger _ { > 1 } }$ million contigs mapped in SynOp using GbS—see text.

The heat map for 2A (Figure 2)is typical for a cleanly mapped AB genome chromosome with only the centromeric region being relatively poorly defined.The heat map for 2D (Figure 2) is typical of D genome chromosomes with blocks of tightly linked markers separated by tracts of low variation,although our MAGlC map has high within-block recombination．We also used principal component analysis (PCA) of the recombination fraction matrix to qualitatively assess whether chromosomes were well ordered. Figure 2 shows the first two principal components for chromosome 2B,demonstrating a horseshoe shape characteristic of a successfully mapped chromosome (Cheema and Dicks，2009; Curtis，1994)．The curve shape of PCA plots is particularly sensitive to SD； deviations provided informative supporting evidence for our SD analysis. Figure S1 has heat maps and PCA figures forall chromosomes.

# Genetic map comparisons

Table1 compares NlAB2015 to four other genetic maps,listed in Table 2.CM2014 contains 40 267 markers (5564 unique sites) and is $1 1 1 8 5 \mathsf { c M }$ in length.However,looking only at markers segregating in our MAGlC population, there are $1 . 1 9 \times$ as many markers but $1 . 5 8 \times$ as many unique sites as CM2014;this difference is much more pronounced for the A $( 1 . 6 8 \times )$ and B $( 1 . 6 4 \times )$ genomes than for the D genome $( 1 . 1 5 \times )$ .The difference is probably related to the presence of the ‘Synthetic'part of the

SynOp cross included in CM2014; SynOp and CM2014 both have a proportionally higher number of unique sites in the D genome than NlAB2015.Our total number of unique sites is much higher than the other three maps,as expected; the 9K maps have far fewer markers,and SynOp is based on a single biparental cross and far fewer progeny lines.NIAB2015 is $5 1 \%$ of the length of CM2014,despite having $8 1 \%$ of the unique sites. In fact, for the A and B genomes,our number of unique sites per cM (0.85) is higher than any of the other maps (range O.44-O.6o,Table 1). However,for the D genome,the unique sites/cM is roughly the same as all other maps except 9KCONS and 9KMAGIC.

Figure 3 compares NIAB2015 chromosome 3A to the CM2014, 9KCONS and SynOp maps and also to the pseudomolecule POPSEQ data from IWGSC2，derived from SynOp.Other chro mosomes are in Figure S2.There isa very low gradient in the middle of most chromosomes,that is NlAB2015 is much longer in this region than CM2014 and SynOp，as a result of more recombination events.This effect is either absent or weak (Figure S2)in the 9KCONS comparisons.There is still very low recombination directly at the centromere in NIAB2015 (Figure 3f)，but the surrounding nonrecombining region is smaller than other non-MAGlC maps.Figure 3b clearly demonstrates the centromeric effect in combination with the much more compact linkage map;at the distal ends of the chromosome,the markers fan outwards from NIAB2015 (left of the figure) to CM2014 (on theright)whereas in the centromeric region，the markers fan outwards from CM2014 to NIAB2015.Further examination of all chromosomes (Figure S2) reveals: (i) between NIAB2015 and CM2014，nonlocal map order in the D genome is less well conserved than in the A and B genomes; (ii) anomalous near vertical lines apparent on some chromosomes (e.g.1B 100 cM, 4A $1 7 5 \ { \mathrm { c M } } )$ ）reflect blocks of low recombination inthe MAGlC population (possible introgression fragments)； and (ii) two instances of missing chromosomal fragments.On chromosome 5A，the first $8 5 ~ { \mathsf { c M } }$ of NIAB2015 is largely absent in SynOp, whereas the first $1 5 \subset { \mathsf { M } }$ of chromosome 5B in SynOp is largely missing in NIAB2015.The latter is the unmapped region around the translocation breakpoint of 5BS-7BS.

![](images/2676e4e20c1f8e90d6c105aa36ee6c0ddbba6d757e5210aea90ac26e3e3d173f.jpg)  
Figure 2 NlAB2015 map diagnostics. (a) Overall heatmap of recombination fraction matrix,with problematic chromosomes highlighted.Redto yellow $\mathbf { \Sigma } = \mathbf { \Sigma }$ low (O) to high recombination (O.5). (b,c) heat maps for chromosomes 2A,2D.(d) First two principal components (xaxis,y-axis) of recombination fraction in principal components analysis for 2B (see text).

Table 2 List of high-density wheat genetic maps used for comparison to NIAB2015 MAGIC map   

<html><body><table><tr><td>Map</td><td>Markers</td><td>Population</td><td>Reference</td></tr><tr><td>SynOp</td><td>Genotyping-by-</td><td>Synthetic W7894 × Opata M85</td><td>Poland et al. (2012)</td></tr><tr><td>9KMAGIC</td><td>sequencing 9K SNP array</td><td>4-founder MAGIC</td><td>Cavanagh et al. (2013)</td></tr><tr><td>9KCONS</td><td>9K SNP array</td><td>9KCONS+6</td><td>Cavanagh et al. (2013)</td></tr><tr><td></td><td></td><td>bi-parentals (inc.SynOp)</td><td></td></tr><tr><td>CM2014</td><td>80K SNP array</td><td>8 bi-parentals (inc. SynOp)</td><td>Wang et al. (2014)</td></tr></table></body></html>

![](images/771bbbddee15541d48a4eabf469398bf44671d2425f90d0aa620f3cc01519d50.jpg)  
Figure 3 NlAB2015 chromosome 3A compared to four other genetic maps (a,b) CM2014 (c,d) SynOp (e) 9KCONS (f) IWGSC2 pseudomolecule.(a,c,e) cM-cM genetic map comparison,NlAB2015 on $x \cdot$ -axis.(b,d) direct comparison between chromosome diagrams, NIAB2015 on left (f) comparison of NIAB2015 (cM, $x \cdot$ -axis) to IWGSC2 pseudomolecule (base pairs).

Figure 4 compares LD decay for chromosome 2D in NlAB2015 with 2D in CM2014，as measured by $D ^ { \prime }$ and $R ^ { 2 }$ .For other chromosomes,see Figure S3.Graphically,it is apparent that the pattern of LD decay in the MAGlC map is considerably cleaner than in CM2014,with many more long-distance high LD values apparent in CM2014.

# Alignment to 3B reference sequence

Comparison of the alignment of CM2014 and NIAB2015 to the physical map shows differences in quality of marker alignment aswell as recombination landscape (Figure 5). $2 . 9 \%$ of the markers in NIAB2015 which align to the 3B pseudomolecule had inconsistent orders with respect to the reference sequence (identified by a red cross in Figure 5),while in CM2014 this is $1 0 . 4 6 \%$ ．Nineteen ofthe23 misalignedmarkersfrom NIAB2015 are also found in CM2014，which showed identical inconsistencies.This suggests the alignment problems for these markersmay lie with their physical positions withinthe reference sequence，rather than the genetic maps.Consistent with this explanation,nine of the 23 anomalous markers also mapped to duplicated genes with different physical positions on 3B.

![](images/bed8650d3b41b8c2357f3f1d223ab6845edd07759bf7a4a857306dc4ad552de0.jpg)  
Figure 4 Linkage disequilibrium decay for chromosome 2D.(a,b) NIAB2015 (c, d) CM2014. Red line is best fit lowess curve with smoothing spanparameter $= 0 . 1 0$

Detected crossovers on chromosome 3B vary by method of analysis: haplotype analysis in R/mpMap using R/happy.hbrem detects 2655 crossovers，equalling 1.46 crossovers per line per Morgan，while haplotype analysisusing R/qtl detects 2920 crossovers or 1.60 per line per Morgan.The countXO function in R/qtl detects 4117 crossovers，equalling 2.26 per line per Morgan.Choulet et al.(2014)report 787 crossovers in the biparental Chinese Spring $\times$ Renan population.The distribution of crossovers in NIAB2015 varies significantly over the chromosome with average chromosome-wide recombination rates of 0.30 cM/Mb and a maximum of 1.34 cM/Mb.The recombination landscape between NIAB2015 and CM2014 differs with proportionally higher recombination ratesin the two distal chromosome regions (identified as O-68 Mb and 715-774 Mb) in CM2014 (1.18 cM/Mb and 1.47 cM/Mb) comparedto NIAB2015(1 cM/Mb and 1 cM/Mb)，but lower recombination rates in the large proximal regions (O.07 cM/Mb compared to 0.15 cM/Mb).Choulet et al. (2014) report distal recombination rates to be O.60 and 0.96 cM/Mb and the large proximal region 0.05cM/Mb.We observe falls in recombination ratesaround SD loci.SD loci associated with the founder Soissons are at $1 0 7 -$ 127 Mb and 242-414 Mb,while Robigus SD loci are at 145- 150Mb，538-554 Mb and641-663Mb.

![](images/eab461e6f32fa24203ae53adb8d9fee6b8722b9f83c6a88a57d97393ae925df1.jpg)  
Figure 5 Chromosome 3B physical map comparison.(a,c) physical vs map distance for NIAB2015,CM2014.Note CM2014 scaled to be same length as NIAB2015.(b,d) recombination profile for NIAB2015,CM2014.

# Further genome sequence comparisons

For chromosomes where no physical map is currently available, we compared NIAB2015 map locations of markers which had top BLAST hits $( > 9 9 \%$ sequence identity) to the same CHAPMAP genomic sequence contig (Table 3，Appendix S1)． Sixty-six percent of markers could be assigned to a contig in CHAPMAP, but only $3 9 \%$ were in shared contigs.Twenty percent( $2 2 \%$ for CM2014) of markers in singleton contigs were mapped to a different chromosome than the BLASTn hit but for markers in shared contigs, this number was only $5 \%$ $( 7 \%$ for CM2014). However, $7 6 \%$ of singleton and $8 1 \%$ of shared contig disagreements mapped to homeologous chromosomes. Combined with the high percentage of no hits,this strongly suggests that the incomplete sequence coverage of the genome (estimated 10.1/ 17 Gb,Chapman et al.，2015) explains most of the NlAB2015- BLASTn discrepancies.Of markers in shared contigs on the same chromosome, $82 \%$ were fully consistent with our map order (same or adjacent location),a further $6 \%$ had a single nonshared unique site (usually a single marker) between the shared contig markers and the remaining $12 \%$ were further apart (median distance $6 . 6 ~ { \mathsf { c M } } _ { \prime } ^ { \cdot }$ ）although $5 3 \%$ of these occurrences corresponded to separation by only 2-5 unique sites.

Table 3 Results of BLASTn analysis against Chapman et al. (2015)   

<html><body><table><tr><td></td><td>NIAB2015</td><td>CM2014*</td></tr><tr><td>Grouping</td><td></td><td></td></tr><tr><td>No hit</td><td>34%</td><td></td></tr><tr><td>Singleton</td><td>27%</td><td></td></tr><tr><td>Same chr</td><td>80%</td><td>78%</td></tr><tr><td>Diff chr</td><td>20%</td><td>22%</td></tr><tr><td>Shared</td><td>39%</td><td></td></tr><tr><td>Same chr</td><td>95%</td><td>93%</td></tr><tr><td>Diff chr</td><td>5%</td><td>7%</td></tr><tr><td>Ordering</td><td></td><td></td></tr><tr><td>Same/adjacent map position</td><td>82%</td><td></td></tr><tr><td>1-site gap</td><td>6%</td><td></td></tr><tr><td>Multisite gap</td><td>12%</td><td></td></tr><tr><td>Multisite gap median cM dist</td><td>6.6cM</td><td></td></tr><tr><td>Multisite gap ≤5 sites</td><td>53%</td><td></td></tr></table></body></html>

\*CM2014 comparison used only markers also mapped in NIAB2015.

# Genome diversity analysis

There has been no intentional trait selection within the MAGIC population except for the removal of double dwarf lines (lines carrying dwarfing alleles at both the RhtB and RhtD height loci). Mapping SD in the MAGlC population is thus a powerful approach to locate genomic regions causing meiotic segregation problems or under fertility or viability selection.Figure 6 shows the distribution of marker density and SD across the genome.A substantial fraction of mapped markers show statistically significant SD (Table 4):2887 $( 1 5 . 5 \% )$ atthe fdr ${ < } 0 . 0 5$ level,1764 $( 9 . 5 \% )$ atfdr ${ \bf \ < } 0 . 0 1$ .If unmapped markers are included,these figuresare higher: $1 6 . 4 \%$ fdr ${ < } 0 . 0 5$ 1 $1 0 . 3 \%$ fdr ${ \bf \ < } 0 . 0 1$ .Markers exhibiting SD are nonrandomly distributed: $52 \%$ of mapped markers showing SD at fdr ${ < } 0 . 0 1$ map to chromosome 1B,mostly showingSD against the rye introgression (i.e.alleles from founders carrying the introgression are at lower than expected frequencies),and a further $12 \%$ of SD markers are found on each of chromosomes 3B and 6B.Otherwise，only 4A $( 6 \% )$ shows statistically significantly more SD loci than expected across the whole data setat the fdr ${ < } 0 . 0 1$ level (Table 4).

Some blocks of SD for particular founders occur over long distances along a chromosome,often $> 2 0$ cM.Frequently,two single founders individually show weak SD along separate chromosome blocks but when these blocks overlap and the founders co-occur,a much stronger SD effect is seen (e.g.on 2B a weak SD against Claire block starting at $3 3 \ c M$ overlaps at 98.4- $9 8 . 9 ~ \mathsf { c M }$ with a weak SD against Soissons block ending at $1 3 0 ~ { \textcircled { \mathsf { M } } }$ ：the combined peak has stronger SD)． Conversely, counteracting blocks (SD in favour of founder A but against founder B) cancel when overlapping.From these observations,we conclude that for many founder combinations,SD extends over considerable distances in the MAGlC population.We inferred 63 SDblocks at fdr $< 0 . 0 1$ (96 for fdr $< 0 . 0 5$ ,ofwhich39consisted of at least two unique sites.This may be an underestimate: (i) it was difficult to conclude whether some widely separated SD blocks with identical founder patterns were one block or two; we treated them as one block if no contradictory founder pattern was found in between them (ii) as we can detect SD blocks only when they are adequately discriminated by the available markers, wewill have by chance missed small SD blocks where the appropriate founder combination did not happen to occur (ii)a high frequency of strong SD markers are unmapped.Table 5 shows the 39 blocks with fdr $< 0 . 0 1$ ，covering more than one unique site.Blocks containing the dwarfing loci RhtD and RhtB rank 4th and 6th,with the direction of SD as expected (dwarfing alleleRhtB- $1 b$ is found only in Robigus and Soissons,RhtD-1b is found in the other six founders but not in Robigus and Soissons).

Marker density peaks around the centromere in many chromosomes but noncentromeric high marker density (HMD)blocks are also visible on 1D，2D，3B,4A，5A，5B，6B，7A and most notably on 1B in the centre of the region containing the rye introgression (SNP density up to 793 markers/10 cM).Twentyseven noncontiguous blocks of markers contain more than 100 markers/10 cM.Of these,11 are centromeric without significant SD and three are both centromeric and overlap SD blocks.Six SD blocks are associated with noncentromeric HMD blocks (Table 5, bolded).These include the blocks with the 2nd and 3rd highest chi-squared peaks: a block with SD favouring Robigus alleles on 3B and the 1B-1R rye introgression (SD against Brompton and

![](images/418ba06f95ec46a35bb62da623b931110a79d02be649184ffc7ad504e01760c3.jpg)  
Figure6 Genomewide patterns of marker density and segregation distortion in NIAB2015. Markers above the dotted line in the lower figure show significant SD at fdr $< 0 . 0 1$ ：   
$\circledcirc$ 2015tiso

Table 4 Summary of segregation distortion results by chromosome   

<html><body><table><tr><td></td><td colspan="2">Marker numbers</td><td colspan="2">Percentage</td></tr><tr><td>Chrom</td><td>SD, fdr<0.05</td><td>SD, fdr<0.01</td><td>%SD, fdr<0.05</td><td>%SD, fdr <0.01</td></tr><tr><td>ALL</td><td>2887</td><td>1764</td><td></td><td></td></tr><tr><td>1A</td><td>12</td><td>3</td><td>0.42</td><td>0.17</td></tr><tr><td>1B</td><td>1033</td><td>912</td><td>35.78</td><td>51.70</td></tr><tr><td>1D</td><td>3</td><td>2</td><td>0.10</td><td>0.11</td></tr><tr><td>2A</td><td>37</td><td>8</td><td>1.28</td><td>0.45</td></tr><tr><td>2B</td><td>27</td><td>8</td><td>0.94</td><td>0.45</td></tr><tr><td>2D</td><td>7</td><td>5</td><td>0.24</td><td>0.28</td></tr><tr><td>3A</td><td>23</td><td>0</td><td>0.80</td><td>0.00</td></tr><tr><td>3B</td><td>392</td><td>207</td><td>13.58</td><td>11.73</td></tr><tr><td>3D</td><td>1</td><td>1</td><td>0.03</td><td>0.06</td></tr><tr><td>4A</td><td>138</td><td>114</td><td>4.78</td><td>6.46</td></tr><tr><td>4B</td><td>60</td><td>50</td><td>2.08</td><td>2.83</td></tr><tr><td>4D</td><td>14</td><td>9</td><td>0.48</td><td>0.51</td></tr><tr><td>5A</td><td>162</td><td>84</td><td>5.61</td><td>4.76</td></tr><tr><td>5B</td><td>32</td><td>4</td><td>1.11</td><td>0.23</td></tr><tr><td>5D</td><td>5</td><td>1</td><td>0.17</td><td>0.06</td></tr><tr><td>6A</td><td>122</td><td>38</td><td>4.23</td><td>2.15</td></tr><tr><td>6B</td><td>502</td><td>213</td><td>17.39</td><td>12.07</td></tr><tr><td>6D</td><td>82</td><td>18</td><td>2.84</td><td>1.02</td></tr><tr><td>7A</td><td>159</td><td>51</td><td>5.51</td><td>2.89</td></tr><tr><td>7B</td><td>60</td><td>36</td><td>2.08</td><td>2.04</td></tr><tr><td>7D</td><td>16</td><td>0</td><td>0.55</td><td>0.00</td></tr></table></body></html>

SD,segregation distortion; fdr, false discovery rate. ${ \mathsf { B o l d } } =$ significantly more SD markers than expected for given chromosome （204 $( P < 0 . 0 1 )$ ：

Rialto)on 1B．Block 24，against Robigus on 4A，is underrepresented in this table: 74 unmapped markers showing significant SD against Robigus were mapped as traits to this location,with lower SD P-values,which would make this the 2nd strongest block.Furthermore,these six SD-HMD blocks are very long:they are the six blocks with the highest number of SD markers in Table 5,3 of the top 6 with most unique sites (all 6 within top 12) and the two longest cM ranges (5 of top 1O).These large SD regions posed considerable mapping challenges (chromosomes 1B,3B and 6B,Figure S1),caused PCA plot distortions (1B,3B,4A,Figure S1),or were clearly visible in map comparisons (1B,4A,Figure S2).

Robigus is thought to have emmer wheat (Triticum dicoccoides) in its pedigree (P.Werner,pers.comm.),but the location of any introgressions has not previously been reported.We examined the occurrence of the Robigus alleles in the two SD-HMD Robigus blocks using the Bristol University 82OK SNP array database (http:// www.cerealsdb.uk.net,Wilkinson et al.,2012).In the 820K data set,72O2 SNPs were found in Robigus and its descendants but in no more than $1 5 \%$ of lines with known non-Robigus pedigree,and could be assigned an IWGSC2 location with high certainty using BLASTn (Appendix S1). Of these,481 mapped to the MAGlC 3B SDHMD block,most densely in the distal part of the block containing the peak SD markers. Almost al SNPs in this region had 4-6 nonRobigus pedigree lines carrying Robigus alleles: Oratorio,Moisson, Garcia and Bacanora consistently carried Robigus alleles,while Dekan and Highbury did so sporadically.In addition,Robigus alleles for most markers in the peak region are found in the variety

Glasgow,also suspected to have T.dicoccoides in its pedigree (R. Jennaway,pers.comm.).Within the core 3B SD block,10‘perfect match' Robigus SNPs (i.e.found in O non-Robigus pedigree lines) are found; however, they are not the peak markers and all have 'no call' as the Robigus allle.In contrast,155'perfect match'Robigus markers mapped exactly to the MAGlC 4A SD-HMD block (175- $2 1 8 ~ { \mathsf { c M } }$ in MAGlC map)，none of which had null alleles.This represents $23 \%$ of all perfect,null-free Robigus markers which could be assigned an IWGSC location in the entire $8 2 0 1$ data set. Furthermore,when we grouped all the perfect,null-free Robigus markers into blocks $_ { ( < 0 . 7 5 }$ Mb separation between markers in IWGSC2 pseudomolecule) along chromosomes,this block was the largest, containing nearly three times as many markers as the 2nd largest block.

# Discussion

# Linkage map

We have constructed the first eight-parent MAGlC genetic map for wheat,comprising 18 601 mapped markers.This required strict quality control of marker calling and extensive manual curation.During map construction,we dropped $10 \%$ of markers which could not be cleanly placed along a chromosome or which could not be placed without a large increase in map distance; these were biased towards dominant，null and SD loci and included a block of loci from around the translocation breakpoint of chromosomes 5BS and 7BS.To the best of our knowledge, this small block is the only specific missing chromosomal block in our map，and 5BS-7BS was the only large-scale chromosomal rearrangement we wereableto detect.

There are two major practical benefits from constructing a genetic map in an eight-founder MAGlC population.Firstly,we obtain anincrease inprecisionfrom alargernumberof accumulated recombination events via three rounds of intercrossing.Secondly，while all high-density genetic maps are by necessity an average across several parental combinations,a MAGIC map produced in a single mapping experiment should have greater accuracy (lower error rate) than a map produced by merging data from several separate biparental crossing populations. Comparing our eight-founder MAGlC genetic map ('NIAB2015'） to published high-density wheat genetic maps, including the current reference standard CM2014 (Wang et al., 2014),we can see that NlAB2015 has either considerably more total unique sites or,compared to CM2014,more unique sites for mutually shared markers (Table 1,Figures 3 and S2). Confirming the advantage of increased recombinationinmultiparental populations,we have a considerably higher number of unique sites per cM in the A and B genomes than all other maps; NIAB2015 is much more compact. The most visible evidence of increased recombination in NlAB2015 is in the pericentromeric regions (Figures 3,5 and S2) where NIAB2015 has many more unique sites than other maps.Of all map comparisons our map has most in common with the four-parent wheat MAGlC map (Cavanagh et al.,2o13),which has a close to linear relationship with NlAB2015 for most chromosomes,(i.e.an overall similar recombination pattern along the chromosome),confirming the expectation that MAGlC maps exhibit greater precision.

We quantified recombination rates in NlAB2015 using chromosome 3B,for which a physical map is available. Broman (2005) calculates for an eight-founder MAGlC population the expected number of informative crossovers per Morgan per line to be on average 4.Our estimates were 1.46 using R/happy.hbrem and

Table5MajosegregtiodistortionDbocksdetectedinteB5ppulatideredyalsediscoveryte(fdruefaarke   

<html><body><table><tr><td>Rank</td><td>PEAK SD_fdr</td><td>No SNPs</td><td>No sites</td><td>Direction</td><td>Minority founder</td><td>Ch</td><td>Start (cM)</td><td>Finish (cM)</td><td>Range (cM)</td><td>HMD</td><td>Map distort</td><td>Known Loc</td></tr><tr><td>1</td><td>6.48E-14</td><td>55</td><td>23</td><td>FOR</td><td>CL, SO, XI</td><td>6B</td><td>0.0</td><td>24.5</td><td>24.5</td><td></td><td>Yes</td><td></td></tr><tr><td>2</td><td>2.32E-09</td><td>268</td><td>89</td><td>FOR</td><td>RO</td><td>3B</td><td>86.6</td><td>198.5</td><td>111.9</td><td>HMD</td><td>Yes</td><td></td></tr><tr><td>3</td><td>2.67E-09</td><td>931</td><td>80</td><td>AGAINST</td><td>BR, RI</td><td>1B</td><td>0.0</td><td>119.0</td><td>119.0</td><td>HMD</td><td>Yes</td><td>1BR</td></tr><tr><td>4</td><td>1.69E-08</td><td>8</td><td>2</td><td>FOR</td><td>RO,SO</td><td>4D</td><td>32.2</td><td>40.1</td><td>7.9</td><td></td><td></td><td>RhtD</td></tr><tr><td>5</td><td>7.35E-08</td><td>15</td><td>6</td><td>FOR</td><td>AL, CL</td><td>7A</td><td>216.3</td><td>219.8</td><td>3.5</td><td>Edge</td><td>Yes</td><td></td></tr><tr><td>6</td><td>2.78E-07</td><td>16</td><td>10</td><td>AGAINST</td><td>RO,SO</td><td>4B</td><td>10.4</td><td>52.7</td><td>42.2</td><td></td><td></td><td>RhtB</td></tr><tr><td>7</td><td>7.85E-07</td><td>17</td><td>5</td><td>AGAINST</td><td>RO,SO</td><td>7A</td><td>219.8</td><td>221.8</td><td>2.0</td><td></td><td></td><td></td></tr><tr><td>8</td><td>2.63E-06</td><td>4</td><td>2</td><td>AGAINST</td><td>SO</td><td>4D</td><td>3.1</td><td>4.1</td><td>1.0</td><td></td><td></td><td></td></tr><tr><td>9</td><td>1.92E-05</td><td>2</td><td>2</td><td>AGAINST</td><td>SO, XI</td><td>2B</td><td>372.4</td><td>376.0</td><td>3.5</td><td></td><td></td><td></td></tr><tr><td>10</td><td>4.79E-05</td><td>33</td><td>13</td><td>AGAINST</td><td>CL</td><td>7B</td><td>194.9</td><td>226.2</td><td>31.4</td><td></td><td></td><td></td></tr><tr><td>11</td><td>5.37E-05</td><td>88</td><td>27</td><td>FOR</td><td>XI</td><td>5A</td><td>58.1</td><td>118.6</td><td>60.5</td><td></td><td></td><td></td></tr><tr><td>12</td><td>5.43E-05</td><td>351</td><td>56</td><td>FOR</td><td>BR, RO, SO</td><td>6B</td><td>81.7</td><td>149.6</td><td>67.8</td><td>HMD</td><td>Yes</td><td></td></tr><tr><td>13</td><td>7.96E-05</td><td>13</td><td>7</td><td>FOR</td><td>SO, XI</td><td>7A</td><td>347.1</td><td>352.6</td><td>5.6</td><td>Edge</td><td></td><td></td></tr><tr><td>14</td><td>1.71E-04</td><td>13</td><td>9</td><td>FOR</td><td>RI, SO, XI</td><td>6A</td><td>84.6</td><td>113.0</td><td>28.4</td><td></td><td></td><td></td></tr><tr><td>15</td><td>7.28E-04</td><td>12</td><td>6</td><td>AGAINST</td><td>BR,SO</td><td>4A</td><td>145.5</td><td>158.7</td><td>13.2</td><td></td><td></td><td></td></tr><tr><td>16</td><td>7.39E-04</td><td>70</td><td>9</td><td>AGAINST</td><td>SO</td><td>6D</td><td>192.7</td><td>204.6</td><td>11.9</td><td></td><td></td><td></td></tr><tr><td>17</td><td>7.86E-04</td><td>47</td><td>17</td><td>FOR</td><td>XI</td><td>6B</td><td>227.7</td><td>252.3</td><td>24.6</td><td></td><td></td><td></td></tr><tr><td>18</td><td>8.16E-04</td><td>18</td><td>5</td><td>AGAINST</td><td>XI</td><td>5A</td><td>242.5</td><td>249.2</td><td>6.6</td><td></td><td></td><td></td></tr><tr><td>19</td><td>8.46E-04</td><td>4</td><td>3</td><td>FOR</td><td>RI,SO</td><td>2A</td><td>127.6</td><td>130.7</td><td>3.0</td><td></td><td></td><td></td></tr><tr><td>20</td><td>1.06E-03</td><td>11</td><td>6</td><td>FOR</td><td>HE, RO,SO</td><td>1B</td><td>183.6</td><td>210.6</td><td>27.0</td><td></td><td></td><td></td></tr><tr><td>21</td><td>1.12E-03</td><td>7</td><td>5</td><td>FOR</td><td>HE, SO,XI</td><td>5B</td><td>251.9</td><td>265.0</td><td>13.2</td><td></td><td></td><td></td></tr><tr><td>22</td><td>1.45E-03</td><td>16</td><td>8</td><td>FOR</td><td>AL</td><td>7B</td><td>207.8</td><td>216.4</td><td>8.6</td><td></td><td></td><td></td></tr><tr><td>23</td><td>1.46E-03</td><td>40</td><td>4</td><td>FOR</td><td>RI</td><td>4B</td><td>54.7</td><td>61.3</td><td>6.6</td><td></td><td></td><td></td></tr><tr><td>24</td><td>1.63E-03</td><td>99</td><td>14</td><td>AGAINST</td><td>RO</td><td>4A</td><td>170.1</td><td>210.2</td><td>40.1</td><td>HMD</td><td>Yes</td><td></td></tr><tr><td>24*</td><td>1.35E-11</td><td>62</td><td>n/a</td><td>AGAINST</td><td>RO</td><td>4A</td><td>218.3</td><td>218.3</td><td>0.0</td><td></td><td></td><td></td></tr><tr><td>25</td><td>1.70E-03</td><td>87</td><td>36</td><td>AGAINST</td><td>AL, CL, RO</td><td>7A</td><td>240.1</td><td>310.3</td><td>70.2</td><td></td><td></td><td></td></tr><tr><td>26</td><td>1.77E-03</td><td>5</td><td>3</td><td>AGAINST</td><td>CL, SO</td><td>2B</td><td>98.5</td><td>101.0</td><td>2.5</td><td></td><td></td><td></td></tr><tr><td>27</td><td>2.06E-03</td><td>2</td><td>2</td><td>FOR</td><td>HE，RI</td><td>4D</td><td>104.5</td><td>106.0</td><td>1.5</td><td></td><td></td><td></td></tr><tr><td>28</td><td>2.06E-03</td><td>22</td><td>5</td><td>AGAINST</td><td>RO,SO, XI</td><td>5A</td><td>310.0</td><td>313.6</td><td>3.5</td><td></td><td></td><td></td></tr><tr><td>29</td><td>2.21E-03</td><td>96</td><td>12</td><td>FOR</td><td>HE, SO, XI</td><td>6A</td><td>125.9</td><td>137.0</td><td>11.1</td><td>HMD</td><td></td><td></td></tr><tr><td>30</td><td>3.12E-03</td><td>7</td><td>4</td><td>AGAINST</td><td>XI</td><td>6D</td><td>11.1</td><td>19.3</td><td>8.1</td><td></td><td></td><td></td></tr><tr><td>31</td><td>3.43E-03</td><td>12</td><td>7</td><td>AGAINST</td><td>BR</td><td>2A</td><td>152.5</td><td>207.6</td><td>55.1</td><td>HMD</td><td>Yes</td><td></td></tr><tr><td>32 33</td><td>3.73E-03 4.09E-03</td></table></body></html>

HMD-SD('High marker density segregation distortion') blocks in bold.Minority founder $\mathbf { \Sigma } = \mathbf { \Sigma }$ origin of minority allele: AL Alchemy, BR Brompton, CL Claire, HE Hereward, RI Rialto,RO Robigus,SO Soissons,Xi Xi19.Ch $\mathbf { \Psi } = \mathbf { \Psi }$ chromosome，HMD $\mathbf { \Sigma } = \mathbf { \Sigma }$ overlaps with high-density block (HMD $\mathbf { \Sigma } = \mathbf { \Sigma }$ in high-density block,edge $\mathbf { \Sigma } = \mathbf { \Sigma }$ border of HMDblock).Map distort $\mathbf { \Sigma } = \mathbf { \Sigma }$ visual evidence of map distortion. \*Additional unmapped markers almost certainly belong here (see text).

1.60 using R/qtl in mpMap with the mpprob function,and 2.26 using the countXO function in R/qtl.Similar to our countXO estimate， Huang et al. (2012)also observed about half the number of recombinations expected from simulations in a fourparent MAGlC population. In the case of using inferred haplotypes from ‘mpprob'to calculate crossovers,this underestimation can partially be attributed to the extent of missing haplotype data $1 8 . 6 1 \%$ using R/qtl, $3 7 . 4 6 \%$ using R/happy.hbrem).The other main reason for underestimation of crossovers is that each SNP is not fully informative in a multiparent population,making it not possible to count all recombination events directly.Nevertheless, using R/qtl (Broman et al.,2003),we detected over $5 \times$ as many crossovers on Chromosome 3B (using the function countXO)as in the Chinese Spring $\times$ Renan population of Choulet et al. (2014).

A comparison between recombination rates on chromosome 3B using NIAB2015 and CM2014 shows that NIAB2015 has a much cleaner estimation of recombination rate as recombination ratesrarely drop below O cM/Mb (Figure 5).NIAB2015 has a slightly higher rate of recombination in the central chromosomal region compared to CM2014,and an $1 8 - 4 2 \%$ lower estimated recombination rate in the distal regions.A possible explanation is that our map distances are inflated in the proximal regions,due to the increased number of unique proximal recombination fraction bins compared to CM2014.Genetic mapping errors that will lead to over- or underestimation of recombination rates,especially in the pericentromeric regions， could also be a confounding problem.Furthermore，the two SD blocks on 3B contributed to lower recombination estimates，a potential problem forall estimates of recombination rates from multifounder populations. Interestingly，Maccaferri et al. (2015)also report that CM2014 shows a much lower recombination rate throughout the centromeric-pericentromeric regions compared to tetraploid wheat.

Theoretically,an increase in unique sites in our map compared to others may result from erroneous ordering falsely breaking up linkage blocks,or a higher genotyping error rate，rather than from increased recombination.Several lines of evidence suggest this is not the case.For genotype calling,we explicitly aimed for high accuracy by a combination of strict QC and manual calling, and estimated error rates at several stages of the process are very low (Appendix S1).Furthermore，very few recombination bins contain only 1 marker.More generally,our detailed physical map comparison to chromosome 3B (Figure 5) shows a considerably lower rate of disagreement $( 2 . 9 \% )$ than in CM2014 $( 1 0 . 3 \% )$ . Our marker order on 3B，especially in the pericentromeric region, matches the physical map,which would not be expected if many unique sites resulted from errors.Local inversions would also show up as negative recombination values in Figure 5 (as they do in CM2014).For other chromosomes,assessing absolute accuracy of our map，especially of grouping，is hampered bythe incompleteness of the wheat genome sequences. Comparatively, grouping was better than in CM2014.For ordering,our results are in strong agreement with the 3B results.Although $1 8 \%$ of markers in shared contigs were not in identical or adjacent map positions,it is highly unlikely that both markers are erroneously mapped,so the best estimate of our error rate is $9 \%$ ，of which $6 \%$ is very local (1-5 unique sites).The remaining $3 \%$ of longerrange error matches the $2 . 9 \%$ visible component seen in Figure 5 for chromosome 3B.For 3B,our evidence suggests that some of this error results from duplicate genes. In summary，we have some minor local ordering issues common to most high-density genetic maps，but have greatly improved longer-range error compared to CM2014.A caveat of using MAGlC populations for mapping is that map accuracy may be diminished in SD regions. Shah et al. (2014) developed an approach to minimize this effect for distortions involving a single founder SD locus,which we used successfully on chromosome 3B.Until a more general methodologyis developed，we believe that including and noting the introgressions,as we do here,is the optimal approach.Overall, we are confident that our map represents a significant improvement in both precision and map accuracy over previously published maps.The completed map is available for download from http://www.niab.com/pages/id/326/Resources.

# Genome diversity

Given the lack of deliberate selection on all but one trait,plant height,we found a remarkably extensive and diverse array of linkage blocks showing SD in our MAGlC population (Figure 6, Table 5).Many of these extend for substantial distances along the chromosome ( $1 6 \mathsf { b l o c k s } > 2 0 \mathsf { c m } )$ ，suggesting that the underlying cause was operating from the earliest intercrossing generations. This unanticipated degree of SD may reflect negative interactionsbetween linkageblocks from differentgenetic backgroundsina multifounder population，or simplyreflect lower detection ability in previous biparental and low-markerdensity populations.We assume that most of these blocks either cause meiotic problems orare subject to some form of viability or fertility selection.There is evidence in wheat for meiotic problems causing SD，particularlyin interspecific crosses.For example, gametocidal genes from Aegilops speciesare expressed in crosses to bread wheat (Endo，199O)and in general crosses between evenclosely related species such as durum wheat and T.dicoccoides show considerable SD (Avni et al.,2014).

Several interspecitic introgression tragments have been incorporated into bread wheat germplasm in an attempt to broaden the genetic base and improve specific traits such as disease resistance and yield (e.g.1B rye introgression).Often such fragmentsare known to exhibit SD.Such cases are likely to be characterized by a high marker density as well as significant SD for two reasons: (i) a higher frequency of markers will be polymorphic between species than within species (ii) reduced recombination between the introgressed fragment and native chromosomes.Six major SD fragments were also associated with HMD blocks in our data set, including 3 of the4 strongest and 5 of the 10 longest SD blocks.The top 3 are the well-known rye introgression on 1B (Worland and Snape,2oo1),with strong SD against the introgression,and two blocks with Robigus alone as the minority founder.Robigus is thought to have T. dicoccoides in its pedigree,but the locations of introgressed fragments in Robigus and its descendent lines have been a matter of speculation. One of the Robigus SD-HMD blocks at the distal end of chromosome 4AL(170-210 cM) is a perfect fit to the model of an interspecific introgressed fragment,with strong SD against Robigus in MAGlC centred on an HMD block，and restriction to Robigus and Robigus descendants only in the 820K SNP array data set from Bristol University.In the 82OK data set, this block is by far the largest linkage block of Robigus-restricted markers,including $23 \%$ of all perfect match Robigus SNPs on the array.The other SD-HMD Robigus linkage block in MAGlC，on chromosome 3B,is being strongly selected for in the MAGIC population (allele frequency $1 . 8 \times$ expectation). The presence of a SD-HMD block in MAGlC,the long chromosomal range of the effect and the occurrence of Robigus alleles in Glasgow all suggest the presence of a T. dicoccoides introgression fragment. On the other hand, this SD block is clearly found in some older UKwheat lines and some of their more recent descendants,suggesting it is either of hexaploid wheat origin or there has been at least one more,older introgression of the fragment into the UK wheat gene pool.In the 82OK array,alleles of both these Robigus fragments occur sporadically in accessions of wheat wild relatives, including a single T. dicoccoides accession,so definitive assignment to T.dicoccoides could not be confirmed from this source.Forthcoming genomic resources for T. dicoccoides (F.Leigh,M. Caccamo,pers. comm.)will help resolve this question.

Knowledge of the location of interspecific introgressions and other linkage blocks containing useful alleles but showing negative interactions in common genetic backgrounds is important in disentangling their desirable and undesirable effects and in tracking their inheritance during breeding.Identifying the two Robigus SD blocks is an exemplar of the potential of the MAGIC approach to mapping to aid in this process.We are presently researching the underlying genetic causes and phenotypic effects of these complex SD-HMD and translocation patterns in our MAGIC population in further detail.

# Methods

# Plantmaterial

Details of the MAGlC population construction are given in Mackay et al. (2014) and Appendix S1. Genotyping was

performed using the Illumina Infinium iSelect 8O Ooo SNP wheat array ('8oK array',http://www.illumina.com/),described in Wang et al. (2014).

# SNP genotype calling

The polyploid module of Genome Studio V2011.1 (llumina,San Diego,CA) developed for wheat by Wang et al. (2014) is poorly suited to genotype calling in our MAGlC population,due to the presence of a low-density heterozygote cluster in most assays. We used a strategy of strict QC/error quantification followed by manual curation of all assays that failed to pass QC—about half themarkers in the final data set.This data set isa larger,higher quality version of that presented in Mackay et al. (2o14).See Appendix S1 for details.

# Map construction

The MAGlC map Was constructed in two steps using the R package mpMap version 1.25 (Huang and George，2011) available from github (https://github.com/behuang/mpMap).For the first round of mapping,we used a subset of 18 750 SNP markers scored as co-dominant. All heterozygote calls were set to missing.These markers were then filtered for missing founder genotypes as well as SD with P-values <1e-5.Recombination fractions between all pairs of markers were calculated using the function‘mpestrf'at default values.Markers were grouped hierarchically using the ‘mpgroup' function into 3oo linkage groups.These linkage groups were merged using R/mpMaplnteractive (https://github.com/rohan-shah/mpMaplnteractive) to produce larger groups which could then be assigned chromosome names based on marker groupings in previous genetic maps (Cavanagh et al.，2013;Wang et al.，2014;Genomezipper v5 (http://wheat-urgi.versailles.inra.fr/Seq-Repository/Genes-annotations)and in the Kansas deletion lines genotyped for the $8 0 \mathsf { K }$ arrayby Bristol University (http://www.cerealsdb.uk.net),aswell as top BLASTn (Altschul et al.,199O) hits to IWGSC contigs listed in Wang et al. (2014, Table S6).

Within linkage groups,markers were ordered using two-point ordering implemented in the function ‘mporder' using default settings.Fine orderingwas performed interactively usingR/ mpMaplnteractive.Map distances were computed using the Haldane mapping function using‘computemap'.Once a draft map was built,previously excluded markers,including all those scored as dominant,were mapped as traits to the existing draft chromosomes,that is we treated the unmapped markers as phenotypes in a QTL analysis,similarly to Rostoks et al. (2006). Founder haplotype probabilities were computed with the ‘mpprob'function in mpMap implemented in R/qtl (Broman et al., 2003)with a threshold of O.6 and QTL were calculated using single-stage QTL mapping in‘mplM'. Linkage groups were then reordered following the same steps as previously but including all the previously excluded loci which could be mapped to a chromosome with $- \mathrm { l o g } _ { 1 0 } P > 1 6$ .In the construction of this final map,manual curation was used throughout the process and loci which could not be cleanly fitted into the ordered chromosomes were dropped.

# Mapvalidation

Recombination fractions were visualized using R/mpMap,and by PCA of the recombination fraction matrix to qualitatively assess if chromosomes were well ordered.We plotted the decay of linkage disequilibrium (LD) against map distance in our MAGIC map,‘NIAB2015',and in the Wang et al. (2014) consensus map, $' C M 2 0 1 4 ^ { \prime }$ ,using R/popgen (Marchini,2013) to calculate LD,then fitting a lowess curve with smoothing span parameter $= 0 . 1 0$ .For cross-chromosome comparisons to other published maps，we used a modified version of the‘plotMap'function in R/qtl (Broman et al.，2oo3).We validated our map against both published wheat genome sequences (Chapman et al.，2015; International Wheat Genome Sequencing Consortium，2014). Flanking sequences of SNP markers in our map were initially blasted against both published genomes using BLASTn (Altschul et al.,1990) with an e-value cut-off of 1e-20 and ${ \tt 2 9 9 }$ identity. For SNP markers with multiple blast hits,we retained the top hit and all other hits which had an equal match or one mismatch worse than the top hit.Where the marker assay had only one segregating SNP and one of the multiple BLAST hits was the same as our map location，we selected this hit as our top hit.For IWGSC data，we converted IWGSC2 coordinates to IWGSC1 coordinates using EnsemblPlants (www.plants.ensembl.org).

# Alignment to 3B reference sequence

Flanking marker sequences for both NlAB2105 and CM2014 werealigned to the 3B pseudomolecule (Choulet et al.，2014; https://urgi.versailles.inra.fr/download/wheat/3B/)using BLASTn. We only included alignments which were part of the final 3B pseudomolecule and had no mismatches.The strong Robigus SD locus was controlled for using the method described in Shah et al.(2o14),and map distances were re-estimated using the Haldane mapping function.This 783-marker long MAGlC chromosome 3B had a reduced length of $2 3 0 ~ { \textcircled { M } }$ compared to the original of $2 8 4 ~ { \mathsf { c M } }$ with 1408 markers.The number of crossovers per line was calculated using the function‘mpprob'in mpMap with either the options program $= ^ { \prime } { \mathsf { q t } } | ^ { \prime \prime }$ (Broman et al.,2003) or program $\mathbf { \Sigma } = \mathbf { \Sigma }$ 'happy'(Mott et al.,20oO) and a threshold of 0.5.In R/qtl the number of crossovers was calculated using the function 'calculateX $\mathrm { \Omega ^ { \prime } }$ on default settings for eight-parent RILs. Recombination per physical distance (cM/Mb)was calculated and visualized using R/MareyMap1.3.0 (Rezvoy etal.,2007).A cubicspline with a smoothing parameter of spar 0.65 was fitted to calculate local recombination rates.Markers with inconsistent alignments were excluded from the fitting.Chromosome 3B of CM2014was aligned to the 3B pseudomolecule as described above (with a total of 1406 markers),and its genetic distance rescaled to match the reduced 3B MAGlC map length of 230 cM.

# Genome diversity analysis

Marker density was calculated in a 1O-cM sliding window along each chromosome to identify HMD blocks. Segregation distortion wasestimated with chi-squared tests using fdr correction for multiple comparisons.SD markers were assigned into linkage blocks by scanning along chromosomes for consistent patterns. We confirmed the SD assessment of all these blocks by manually rescoring up to 5 individual markers per SD block and recalculating SD. Flanking sequences of SNP markers in the $8 2 0 1$ array data set were blasted against the IWGSC genome sequence in a similar manner to our map validation and then aligned to NIAB2015 via our IWGSC top hits.Details are givenin Appendix S1.

# Acknowledgements

Development of the NIAB MAGlC population Was led by lan Mackay and Phil Howell. Richard Horsnell,Alison Bentley, Gemma Rose，Tobias Barber，Claire Pumfrey and Nick Gosman contributed to the creation of the population.Rhian Howells and RAGT seeds UK,Ltd. provided genotyping data for mapped PCR markers.Fundingfor development ofthepopulationwas provided by BBSRC Crop Science Initiative grant BB/E007201/1 and funding for genotyping with the 8OK array was provided by the NIAB Trust. LMW Was funded by BBSRC iCASE grant BB/ K011790/1 awarded to Alex Webb (University of Cambridge), Matthew Hannah (Bayer CropScience) and Andy Greenland (NIAB).Many thanks to Emma Huang and Rohan Shah for helpful discussions and comments，especially with the use of mpMap and associated software.The WAGTAIL Consortium kindly gave us permission to cite the number of scorable markers in the WAGTAIL panel for comparison to MAGIC.

# Conflict of interest

The authors have no conflict of interest to declare.

# References

Altschul,S.F.,Gish,W.,Miller,W.,Myers,E.W.and Lipman,D.J.(199O) Basic local alignment search tool.J.Mol. Biol.215,403-410.   
Avni,R.,Nave,M., Eilam,T., Sela,H.,Alekperov,C.,Peleg,Z., Dvorak,J.et al. (2014)Ultra-dense genetic map of durum wheat x wild emmer wheat developed using the 90K iSelect SNP genotyping assay.Mol. Breed.34, 1549-1562.   
Badaeva,E.D., Dedkova, O.S.,Gay,G.,Pukhalskyi,V.A., Zelenin,A.V., Bernard, S.and Bernard,M. (2oo7) Chromosomal rearrangements in wheat: their types and distribution. Genome,50, 907-926.   
Bandillo，N.，Raghavan，C.，Muyco，P.A.，Sevilla，M.A.L.，Lobina,I.T.，DillaErmita,C.J.，Tung,C.W.et al.(2013)Multi-parent advanced generation inter-cross (MAGlC) populations in rice:progress and potential for genetics research and breeding. Rice,6,15.   
Broman, K.W.(2oo5) The genomes of recombinant inbred lines.Genetics,169, 1133-1146.   
Broman, K.W.,Wu, H.,Sen,S.and Churchil,G.A. (2003) R/qtl: QTL mapping in experimental crosses. Bioinformatics,19,889-890.   
Cavanagh,C., Morell,M.,Mackay,I.and Powell,W. (20o8) From mutations to MAGIC: resources for gene discovery,validation and delivery in crop plants. Curr.Opin. Plant Biol.11,215-221.   
Cavanagh,C.R.,Chao,S.M.,Wang,S.C.,Huang,B.E.,Stephen,S.,Kiani,S., Forrest，K.et al. (2013）Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.Proc.Natl Acad. Sci. USA,110,8057-8062.   
Chapman,J.A.,Mascher,M.， Buluc,A.,Barry,K.,Georganas,E.,Session,A., Strnadova，V.et al. (2o15)A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.Genome Biol. 16,26.   
Cheema,J.and Dicks,J.(2oo9) Computational approaches and software tools for genetic linkage map estimation in plants. Brief. Bioinform.10,595-608.   
Choulet,F.,Alberti,A.,Theil,S.,Glover,N.,Barbe,V.,Daron,J.,Pingault,L. et al.(2o14) Structural and functional partitioning of bread wheat chromosome 3B.Science,345,7.   
Curtis,D.(1994) Another procedure for the preliminary ordering of loci based on 2 point LOD scores.Ann. Hum. Genet.58,65-75.   
Darvasi,A.and Soller,M.(1995)Advanced intercross lines,an experimental population for fine genetic-mapping. Genetics,141, 1199-1207.   
Endo，T.R.(199o) Gametocidal chromsomesand their inductionof chromosome mutations in wheat. Jpn.J.Genet.65,135-152.   
Huang, B.E.and George,A.W. (2011) R/mpMap: a computational platform for the genetic analysis of multiparent recombinant inbred lines. Bioinformatics, 27, 727-729.   
Huang,X.Q.,Paulo,M.J.，Boer,M.,Effgen,S.，Keizer,P.,Koornneef,M.and van Eeuwijk,F.A.(2011) Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.Proc. Natl Acad. Sci. USA,108,4488-4493.   
Huang,B.E.,George,A.W.,Forrest, K.L.,Kilian,A.,Hayden,M.J.,Morell, M.K. and Cavanagh,C.R.(2012)A multiparent advanced generation inter-cross population for genetic analysis in wheat. Plant Biotechnol.J.10,826-839.   
International Wheat Genome Sequencing Consortium. (2014)A chromosomebased draft sequence of the hexaploid bread wheat (Triticum aestivum) genome.Science,345,11.   
Maccaferri,M.,Ricci,A.,Salvi,S.,Milner,S.G.,Noli,E.,Martell,P.L,Casadio R. et al. (2015)A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding. Plant Biotechnol.J.13, 648-663.   
Mackay,l.and Powell,W. (2oo7) Methods for linkage disequilibrium mapping in crops. Trends Plant Sci.12,57-63.   
Mackay,I.J., Bansept-Basler,P., Barber,T., Bentley,A.R., Cockram,J., Gosman N.,Greenland,A.J.et al. (2014)An eight-parent multiparent advanced generationinter-crosspopulationforwinter-sownwheat:creation, properties,and validation. G3-Genes Genomes Genet.,4,1603-1610.   
Marchini,J.L.(2013) Popgen:Statistical and Population Genetics.R package version 1.0-3.https://cran.r-project.org/web/packages/popgen/popgen.pdf.   
Milner,G.M., Maccaferri, M.,Huang, B.E.,Mantovani,P.,Massi,A.,Frascaroli E.,Tuberosa,R.et al. (2015)A multiparental cross population for mapping QTL for agronomic traits in durum wheat (Triticum turgidum ssp.durum). Plant Biotechnol.J.13,648-663.   
Mott,R.,Talbot, C.J.,Turri, M.G., Collins,A.C.and Flint,J.(2OoO)A method for fine mapping quantitative trait loci in outbred animal stocks.Proc.Natl Acad. Sci. USA,97,12649-12654.   
Pascual,L.,Desplat,N., Huang,B.E, Desgroux,A., Bruguier,L.,Bouchet,J.P., Le,Q.H.et al. (2015) Potential of a tomato MAGlC population to decipher the genetic control of quantitative traits and detect causal variants in the resequencing era.Plant Biotechnol.J.13,565-577.   
Poland,J.A.,Brown,P.J.,Sorels,M.E.and Jannink,J.L.(2o12) Developmentof high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach. PLoS ONE,7,8.   
Rezvoy,C, Charif,D.,Gueguen,L.and Marais,G.A.B.(2007) MareyMap: an Rbased tool with graphical interface for estimating recombination rates. Bioinformatics,23,2188-2189.   
Rostoks,N. Ramsay，L.，Mackenzie，K.,Cardle,L.，Bhat，P.R.,，Roose，M.L., Svensson,J.T.et al. (2oo6) Recent history of artifical outcrossing facilitates whole-genome association mapping in elite inbred crop varieties. Proc.Natl Acad.Sci. USA,103,18656-18661.   
Sannemann,W.，Huang,B.E.，Mathew,B.and Léon,J.(2015)Multi-parent advanced generation inter-cross in barley: high-resolution quantitative trait locus mapping for flowering time as a proof of concept.Mol. Breeding,35, 86. Shah,R.,Cavanagh,C.R.and Huang,B.E.(2014) Computationally efficient map construction in the presence of segregation distortion. Theor.Appl. Genet. 127,2585-2597.   
Thepot,S.,Restoux,G.,Goldringer,I.,，Hospital,F.,Gouache,D.,Mackayl.and Enjalbert,J. (2O15)Eficiently tracking selection ina multiparental population: the case of earliness in wheat. Genetics,199,609-623.   
Threadgill,D.W.and Churchill,G.A. (2012)Ten yearsof the collaborative cross. G3-Genes Genomes Genet.,2, 153-156.   
Villareal, R.L., Rajaram,S.,Mujeebkazi,A.and Deltoro,E.(1991) The effect of chromosome 1B/1R translocation on the yield potential of certain spring wheats (Trtiticum aestivum).Plant Breed.106,77-81.   
Wang,S.,Wong,D.,Forrest,K.,Allen,A.,Chao,S.,Huang,B.E.,Maccaferri, M.et al. (2014) Characterization of polyploid wheat genomic diversity using ahigh-density9o,ooo single nucleotidepolymorphism array.Plant Biotechnol. J.12,787-796.   
Wilkinson，P.A.，Winfield，M.O.，Barker，G.LA.，Alln，A.M.，Burrdge，A. Coghill,J.A., Burridge,A.et al. (2012) CerealsDB 2.0:an integrated resource for plant breeders and scientists. BMC Bioinformatics,13,219.   
Worland,A.J.and Snape, J.W.(20o1) Genetic basis of worldwide wheat varietal improvement.In The World Wheat Book (Bonjean,A.P.，Angus,W.J., eds), pp.59-100.Paris,France: Lavoisier.   
Yu,J.M.,Holland,J.B.,McMullen,M.D.andBuckler,E.S. (2008)Geneticdesign and statistical power of nested association mapping in maize. Genetics,178, 539-551.

# Supporting information

Additional Supporting information may be found in the online version of this article:

Figure S1 Heat maps of individual chromosomes in NlAB2015. Figure S2 Graphical comparison of all chromosomes to four genetic maps.

Figure S3 LD decay (measured as $R ^ { 2 }$ and $D ^ { \prime }$ )comparedacrossall chromosomes for NIAB2015 and CM2014. Table S1 List of unmapped markers,with QTL positions,blast hits (see 'Notes'worksheet). Table S2 The NIAB2015 MAGIC map (see ‘Notes' worksheet). Appendix S1 Methods (Genotype calling including genotype error calculation,mapping error rate estimation,SD analysis).