# Interactome analysis identifies a new paralogue of XRCC4 in non-homologous end joining DNA repair pathway

Mengtan Xingl\*， Mingrui Yang2,3,\*， Wei Huol\*,Feng Feng2，Leizhen Wei4，Wenxia Jiang5,Shaokai Ning, ZhenxinYanlWen Lil,Qingsong Wang],Mei Hou’,Chunxia Dong1,Rong Guo],Ge Gaol,Jianguo Ji1,ShanZha5, Li Lan4, Huanhuan Liang² & Dongyi $\mathsf { X } \mathsf { u } ^ { 1 }$ （204号

Non-homologous end joining (NHEJ) is a major pathway to repair DNA double-strand breaks (DSBs),which can display different types of broken ends. However,it is unclear how NHEJ factors organize to repair diverse types of DNA breaks. Here,through systematic analysis of the human NHEJfactor interactome,we identify PAXXas a direct interactor of Ku.The crystal structure of PAXX is similar to those of XRCC4 and XLF.Importantly, PAXX-deficient cells are sensitive to DSB-causing agents. Moreover， epistasis analysis demonstrates that PAXX functions together with XLF in response to ionizing radiation-induced complex DSBs,whereas they function redundantly in response to Topo2 inhibitor-induced simple DSBs. Consistently, PAXX and XLF coordinately promote the ligation of complex but not simple DNA ends in vitro. Altogether,our data identify PAXX as a new NHEJ factor and provide insight regarding the organization of NHEJ factors responding to diverse types of DSB ends.

D NAdoublorsdDADs)sedt replication and repair,programmed recombination and exposure to exogenous agentsused in cancer therapies. Unrepaired DSBs can lead to apoptosis or senescence,whereas misrepaired DSBs can cause developmental defects,accelerated ageing and cancerl.Non-homologous end joining (NHEJ） is a major pathway for detection and repair of DSBs in mammalian cells². The core factors mediating NHEJ are the $\mathrm { K u } 7 0 / 8 0$ heterodimer (Ku)，DNA-dependent protein kinase catalytic subunit (DNA-PKcs)，XRCC4，DNA ligase IV (Lig4） and XRCC4-like factor (XLF,also called Cernunnos)².For simple DSBs,these core factors are sufficient to recognize,align and ligate a pair of broken ends. However, for complex or‘dirty breaks,which have mismatched or covalently modified DNA ends，additional factors are required to modify the ends and facilitate their ligation.These factors include Artemis, DNA polymerases $\lambda$ and $\mu { : }$ ，terminal dinucletidyltrasferase, polynucleotidekinase-phosphatase, aprataxin(APTX)and APTX-polynucleotidekinase-phosphatase-likefactor1．It remains unclear how these auxiliary factors are targeted to specific DSBs,in particular those with ends that contain densely clustered damage.It has been speculated thata fraction ofNHEJ proceeds by a‘trial and error’process,in which sequential attemptsatligationand end processing bydifferent enzymesare repeated until the substrate is cleaned for ligationl.This process requires a stable NHEJ complex to maintain the end-alignment. Many DSB repair proteins have been implicated in this role, including XRCC4 and its paralogue, ${ \mathrm { X L F } } ^ { 3 }$ ：

The XRCC4-like proteins, including XRCC4, XLF and SAS6, function as homodimers composed of an amino-terminal globular head domain and a carboxy-terminal coiled-coil stalk. All of these proteins play important scaffold roles by forming various high-order structures4-9. XRCC4 and XLF can form super-helical filaments6-9，which have beenshown to bridge, align or protect DNA ends3.In addition, they interact with Ku10 and Lig4(refs 11,12)，respectively，and can stimulate Lig4 activities13-i5.Moreover, XLF preferentially stimulates the end joining of non-cohesive ends over cohesive ends16,17,suggesting thatit maybe specifically needed for the repair of complexDNA ends.

We have previously identified Rifl as a novel NHEJ regulator18. Here we perform systematic interactome analyses of human NHEJ factors and identify a novel XRCC4-like protein named paralogue of XRCC4 and XLF (PAXX). The crystal structure study reveals that PAXX forms a homodimer through its N-terminal domain that is structurally similar to XRCC4 and XLF.In contrast to XRCC4 and XLF,PAXX directly and strongly interacts with the Ku complex.Interestingly,PAXX and XLF cooperate to promote the ligation of non-cohesive but not blunt ends in vitro.Importantly,the epistasis analysis suggests that PAXX is upstream of XRCC4 and Lig4 in DSB repair,while PAXX and XLF display both epistatic and non-epistatic interactions that are dependent on the complexity of the DSB ends.Thus,our data reveal thatPAXXisa newcore NHEJ factor that playsa key role in the organization ofNHEJ complexes for the repair of diverse types of DSB ends.

# Results

PAXX is a new component of NHEJ. We systematically expressed 19 Flag-tagged NHEJ proteins in HEK293 cells and immunoprecipitated their associated complexesusingan anti-Flag antibody (Fig.la, Table1 and Supplementary Data1).A silver-stained gel of partial samples confirmed the high quality ofthe immunoprecipitations (Fig. la). Mass spectrometry identified a novel protein, C9orfl42,in the immunoprecipitates from 4 (XLF,Lig4, $ { \mathrm { K u } } 7 0$ and $\mathtt { K u 8 0 }$ ）of the 5NHEJ core proteins (Fig.la,lanes 2，3 and 5-7)and 2 (DNA polymerases $\lambda$ and APTX-polynucleotide kinase-phosphatase-like factor） of the 14 non-core NHEJ proteins (Supplementary Data 1). Immunoblotting of the Ku80-associated complex further validated this finding (Fig.1b).Based on the sequence and structural similarity to XRCC4 and XLF (see below),we renamed C9orf142 as PAXX.Silver staining (Fig.la, lane 4),mass spectrometry (Table 1） and immunoblotting (Fig. lc） of the reciprocal immunoprecipitation(IP） revealed the presence of major NHEJproteins,including XLF,XRCC4,Lig4,Ku70,Ku80 and DNA-PKcs in the PAXX-associated complexes,demonstrating that PAXX is a new component of the NHEJ complex.

NHEJ factors have been reported to relocalize to laser-induced DSB sites in vivol9. We expressed green fluorescent protein-fused PAXX in U2OS cells and induced DSBs with a micro-laser. Green fluorescent protein-PAXX relocalized to the DSB sites as expected (Fig.1d),consistent with the biochemical results that PAXX isan NHEJ factor.

PAXXformsastable complexwithKu.We noted thatPAXX is enriched between 3-and 17-fold over other core NHEJ proteins examined in the $\mathtt { K u 8 0 }$ -associated complexes (Fig. lb),suggesting that PAXX may interact with $\mathtt { K u }$ more strongly than other NHEJ factors.In agreement with this notion,we found that the interaction between PAXX and $\mathtt { K u 8 0 }$ was more resistant to high salt treatment than those between PAXX and other NHEJ proteins (Fig.lc, lanes 4-7).Moreover, immunoblotting of the immunoprecipitates of endogenous proteins showed that PAXX but not XRCC4 and XLF co-immunoprecipitated with $\mathtt { K u 8 0 }$ (Fig.le).Interestingly, the three members of the XRCC4 family did not co-immunoprecipitate with one another in detectable quantitiesunderourexperimental conditions, whichis inconsistent with a previous report showing that XRCC4 and XLF co-immunoprecipitate20. These data suggest that PAXX forms a more stable complex with Ku than with XRCC4,XLF and other NHEJ core factors.

PAXX is structurally similar to XRCC4 and XLF. Sequence homology searches using the BLAST algorithm showed that PAXX has no obvious similarity to any known proteins in the NCBI database.However, three-dimensional structure prediction with HHprep(http://toolkit.tuebingen.mpg.de/hhpred#) suggested that PAXX belongs to the same family as XRCC4 and XLF (Fig.2a and Supplementary Fig. la,b). Specifically, PAXX protein is predicted to possess an N-terminal domain structure that is identical to XRCC4,XLF and SAS6 (Fig. 2a and Supplementary Fig.la,b);all of these proteins_contain a globular head domain followed bya coiled-coil stalk4,5,21,22To test this hypothesis, we determined the crystal structure of the PAXX N-terminal fragment (1-145）diffracted at $2 . 6 \mathring \mathrm { A }$ （PDB 4WJA） using the SAD(singlewavelength anomalousscattering） and MR (molecular replacement) methods (Fig. 2b and Table 2).The space groups for both data sets were $\mathrm { P } 6 _ { 5 } 2 2$ ,with two molecules in each asymmetric unit.As predicted, the N-terminal domain of PAXX was highly similar to those of XLF and XRCC4 (Fig. 2b and Supplementary Fig.1b). Seven $\beta$ -strands (S1-S7)and three helixes (H1-H3)were observed in the PAXX1-145 structure,in which a globular head domain and a coiled-coil stalk formed a dimer (Fig.2b),consistent with the observation that PAXX1-145 exists primarily as a dimer in solution (Supplementary Fig.2).

Compared with XLF, PAXX showed more structural similarities to XRCC4(Fig. $2 { \mathrm { b } } , \mathrm { c }$ ).Different values for the angle between the head domain and H3 have been reported for XLF and XRCC4. The C terminus of XLF folds back to the linkage point between the head domain and H3,and elevates the head domain, forming a $9 0 ^ { \circ }$ angle between them (Fig. 2b)²1.In XRCC4 and PAXX, the angle between the head domain and H3 is only $\sim 4 5 ^ { \circ }$ due to a lack of tension (Fig. 2c)22.In addition, compared with XRCC4 and XLF,PAXX has a much shorter coiled-coil stalk (Fig.2c and Supplementary Fig.1b).The extended stalk of

![](images/c2d47d6706f0afaa045a6debb4625645029a2372ba1abe2c05a1eb5ef8dfb20c.jpg)  
Figure1|PAXXisonecompoentoftheEJcomplex.a）Ailver-stainedSDSgelowingthepolypeptidesthatereimmunopuriedwithFlag antibody from the extracts of HEK293 cells expressing Flag-tagged XRCC4,XLF, PAXX, Lig4, ${ \sf K u 7 0 }$ or $\mathsf { K u 8 0 }$ .Asa control,a mock IP(Mock) was performed usingregularHK293celsthatdidotexpreFlag-taggedproteneprotesientiedusingmassspectrometryandteumberofpepties discoveredforachproterestediblagprotsinegeedicadydods.)munoblotgat PAXX co-immunoprecipitates with Flag- $\mathsf { K u 8 0 }$ .(c) Immunobloting shows Flag-PAXX IP under different wash conditions with the indicated sodium concentratiosmbsinddateeatdtesitftde.dcuaifefuresoteiat thelasermicroadiatioiduceddamasitesinUOcel-Ku8siudedtrol.agesereobadforeltaeldn after (right panel) irradiation. Scale bars, $2 \mu \mathrm { m }$ (e) Immunobloting of the IP with endogenous antibodies.

<html><body><table><tr><td colspan="7">Table 1 | The proteins identified in the purified NHEJ complexes using mass spectrometry.</td></tr><tr><td rowspan="2">Identified proteins</td><td colspan="6"></td></tr><tr><td>Mock</td><td>XRCC4</td><td>XLF</td><td>Flag-IP PAXX</td><td>LIG4</td><td>Ku70</td><td>Ku80</td></tr><tr><td>DNA-PKcs</td><td>0</td><td>0.5</td><td>222.5</td><td>286.5</td><td>270</td><td>212</td><td>206.5</td></tr><tr><td></td><td>0</td><td>1</td><td>55</td><td>68</td><td>71</td><td>76.5</td><td>62.5</td></tr><tr><td>Ku70</td><td></td><td>0</td><td>58</td><td>62.5</td><td>70</td><td>73</td><td>86</td></tr><tr><td>Ku80</td><td>0</td><td></td><td>44.5</td><td>23</td><td>94</td><td></td><td></td></tr><tr><td>LIG4 XRCC4</td><td>0</td><td>22.5</td><td></td><td>14.5</td><td>68</td><td>0</td><td>1</td></tr><tr><td></td><td>0</td><td>39</td><td>57</td><td>0</td><td>11.5</td><td>4</td><td>1</td></tr><tr><td>XLF</td><td>0</td><td>13</td><td>27 5.5</td><td>24</td><td>8.5</td><td>0 9</td><td>0</td></tr><tr><td>PAXX</td><td>0 0</td><td>0 0</td><td>2.5</td><td>1.5</td><td>8</td><td>1.5</td><td>8.5</td></tr><tr><td>APLF PNKP</td><td>0</td><td>29</td><td>0</td><td>1</td><td>0</td><td>0</td><td>1 0</td></tr><tr><td>APTX</td><td>0</td><td>13</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>WRN</td><td>0</td><td>0</td><td>0</td><td>37.5</td><td>44</td><td>15.5</td><td>30</td></tr></table></body></html>

APLF,APTX-polynucleotide kinase-phosphatase-like factor; $\mathsf { A P T X } ,$ aprataxin; DNA-PKc,DNA-dependent protein kinasecatalyticsubunit; IP,immunoprecipitation;Lig4,DNAligaseIV;NHEJ, Non-homologous end joining; PKNP,polynucleotide kinase-phosphatase; XLF,XRCC4-like factor. The numbers of peptidesare averaged over two experiments.

XRCC4 is required for Lig4 binding，implicating that the N-terminal domain of PAXX may lack the ability to bind Lig4.

The database of the Human Protein Atlas (http://www.proteinatlas.org/） shows that PAXX is widely expressed in all human tissues，with particularly high expression in the blood and immune system.PAXX is present in all vertebrates but is absent in most invertebrates and in yeast (Fig.2a).This evolutionary distributionofPAXXdiffers from thatofXRCC4 and XLF,which are conserved from yeast to vertebrates.PAXX appears to coevolve with DNA-PKcs and Artemis,which implies that PAXX may function in a manner similar to DNA-PKcs and Artemis in joining the complex but not the simple DNA ends23.

![](images/233cb72afd06290242c4b9b6b4bee93a655a52da35dd4f779f9ee19cdfc28390.jpg)  
Figure2|TheN-termaldomainofPAXXistructuralysiilartothoseofXCC4andXLF.(aDomainmapsofXCC4LFandPAitesof protein-proteinteractiosarkdderlidthnteractigprotesdiadeloovesaiftprotensispldi therightpalsusfo Worm(Cenohbitisesndstcharomseseetidusingthgrittoahtatabaeadt NCBIYexists; homodimerstructureisshownincartoonrepresentation.Onemonomerisshowinspectrumandtheotherisshowninpalecyan.Thespectrum ranginfrombluetodepresentsesiduesfroteNtoCtermius.eNerius,Ctermiudsecodarytructuraelementslbeldtom panel:electrostaticpotetialsurfacerepresetatioofAimreitivgatieadutraegisreoinbludit respectielyOverferaddoasfPAA-4-9ndCAA4-49),n purple,respectively.

PAXX interacts with NHEJ factors via its C-terminal region. Unlike the N-terminal domain structure that is shared among XRCC4 family members,C-terminal regions (CTRs） are more divergent and show little sequence similarity (Fig. 2a).The C-terminal-conserved region of PAXX,similar to that of XLF, localizes at the end of C terminus (Fig.2a and Supplementary Fig.la)lo.A previous study has shown that the CTR domain of XLFis required forXLF to interact with $\mathrm { K u }$ and XRCC4,and for the localization at $\mathrm { D S B s } ^ { 1 0 }$ . We found that the CTR motif of PAXX is similarly required for these associations,because the deletion (AA 1-170） and point mutation (RRR177-179AAA, IN186-187AA or F201A) of the PAXX-CTR completely abolished its interaction with Ku and XRCC4,as well as Lig4 and DNA-PKcs (Fig.3a,b and Supplementary Fig.la).Moreover, the F201A mutation also disrupted the localization of PAXX at laser-induced DSBs and compromised its exclusively nuclear localization (Fig. 3c). These data support the notion that PAXX and XLF may interact with common factors in the NHEJ pathway and imply that the two proteins may have overlapping functions.

PAXX promotes cellular resistance to agents that induce DSBs. To investigate the function of PAXX in vivo,we generated PAXXnull DT40 cells by gene targeting (Supplementary Fig. 3a,b). These cells displayed a modest sensitivity to a variety of agents that induce DSBs,including ionizing radiation (IR),bleomycin (a radiomimetic agent that can induce a greater number of homogeneousDSBsthan $\mathrm { I R } ^ { 2 4 }$ ），VP16(etoposide,a Topoisomerase I(Topo2） inhibitor that directly induces DSBs with less complex ends25,26) and ICRF193(a Topo2 inhibitor that indirectlyinduces DSB via an unknownmechanism;

<html><body><table><tr><td colspan="3">Table 2 | Data collection and refinement statistics (SAD).</td></tr><tr><td colspan="3">Native SeMet</td></tr><tr><td>Data collection</td><td></td><td></td></tr><tr><td>Space group</td><td>P6522</td><td>P6522</td></tr><tr><td>Cell dimensions</td><td></td><td></td></tr><tr><td>a,b,c (A)</td><td>90.062,90.062,153.505</td><td>91.825, 91.825, 152.468</td></tr><tr><td>α,β,y(°）</td><td>90.000,90.000,120.000</td><td>90.000,90.000,120.000</td></tr><tr><td>Wavelength</td><td>0.979</td><td>0.979</td></tr><tr><td>Resolution (A)</td><td>50-2.64 (2.69-2.64)</td><td>50-2.80 (2.86-2.80)</td></tr><tr><td>Rsym Or Rmerge</td><td>0.069 (0.634)</td><td>0.081 (0.646)</td></tr><tr><td>/ol</td><td>69.64 (7.52)</td><td>56.62 (8.17)</td></tr><tr><td>Completeness (%)</td><td>99.3 (100)</td><td>100 (100)</td></tr><tr><td>Redundancy</td><td>20.0 (20.9)</td><td>22.3 (22.5)</td></tr><tr><td>Refinement</td><td></td><td></td></tr><tr><td>Resolution (A)</td><td>50-2.64</td><td></td></tr><tr><td>No.reflections</td><td>10813</td><td></td></tr><tr><td>Rwork/Rfree</td><td>23.49/27.06</td><td></td></tr><tr><td>No. atoms</td><td></td><td></td></tr><tr><td>Protein</td><td>1947</td><td></td></tr><tr><td>Ligand/ion</td><td></td><td></td></tr><tr><td>Water</td><td>6</td><td></td></tr><tr><td>B-factors</td><td></td><td></td></tr><tr><td>Protein</td><td>91.1</td><td></td></tr><tr><td>Ligand/ion</td><td></td><td></td></tr><tr><td>Water</td><td>67.4</td><td></td></tr><tr><td>Root mean squared deviations</td><td></td><td></td></tr><tr><td>Bond lengths (A)</td><td></td><td></td></tr><tr><td></td><td>0.013</td><td></td></tr><tr><td>Bond angles (°)</td><td>1.570</td><td></td></tr><tr><td>SAD, single wavelength anomalous scatering.</td><td></td><td></td></tr></table></body></html>

![](images/d1fc53dce448134e82edb817d8aa3652ec790b9be686c814dea6ea83bf3d8eed.jpg)  
Figure3|PAXXinteractsiththeKucompleandtheLF-XCC4-Lig4complexthoughitsCR.(a）SheaticepresetatiofinX mutants(left）ndtheiabittoomnoprecipitatewithKuandLF-XCC4-Lg4omplexefroHK293extractsigt)utatiite indicateduplementaryFig)AnI-WalysisteterneethrXXutatsicomuoprecipiateitdirentNpotei (c)lmagingshothatthePAXXutantFOAloseitslocalizatioatlasermicroradiatioiduceddamagsitesRF-Ku8Oasideds control.Images were obtained before and 3 min after irradiation.Scale bars, $2 \mu \mathrm { m }$

Supplementary Fig.4),demonstrating that PAXX is required for normal DSB repair in vivo.As NHEJ-deficient cells are much more sensitive to IR when arrested in G1 or early S phase27, we analysed $P A X X ^ { - / - }$ cells under these conditions and found that theyare hypersensitive to IR (Fig.4a),consistent with the idea that PAXX plays a role in NHEJ.Among other DSB-inducing agents,ICRF193 sensitivity is usually observed in cells defective in NHEJ but not homologous recombination (HR） pathways28. The finding that $P A X X ^ { - / - }$ cells possess ICRF193 sensitivity (Supplementary Fig.4） is consistent with the biochemistry data,suggesting that PAXX plays a role in NHEJ. Conversely, $P A X X ^ { - 7 ^ { \circ } - }$ cells showed no obvious sensitivity to camptothecin (Supplementary Fig.4),a TopoisomeraseIinhibitor that induces replication-dependent DSBs that are repaired primarily by the HR pathway29. These data are consistent with the notion that PAXX participates in NHEJ but not HR.We further examined the function of PAXX in the DSB repair in mammalian cells and generated PAXX-deficient human HCT116 cells using CRISPR (Supplementary Fig.5a,b).Consistent with the results in DT40 cells,PAXX-deficient HCT116 cells were hypersensitive to IR and VP16 (Supplementary Fig. 5c), which suggested that PAXX is also important for the DSB repair pathways in mammalian cells.

Both $\mathbf { N } _ { - }$ and C-terminal domains are required for PAXX function. To determine which domain is important for the function of PAXX in DSB repair,we performed genetic rescue experiments using PAXX-/- DT40 cells transfected with variants of PAXX. Re-expression of wild-type human PAXX protein in $P A X X ^ { - / - }$ cells largely rescued the IR sensitivity phenotype, whereas re-introduction of the Ku-interactiondeficient mutant PAXX(F2o1A）did not,which indicated that $\mathtt { K u }$ -binding activity is critical for the function of PAXX in DSB repair (Fig. 4a).

The S6-loop-S7 region of the global head domain in XRCC4 or XLF is important for their mutual interactions and for theirfunctionsin NHEJ6-9.We generatedacombined mutation (Nmut:L96D,L98D,L105D and L109D） in this region of PAXX (Supplementary Fig. la),which is expected to disrupt the hydrophobic interface.Nmut also failed to rescue the IR sensitivity phenotype (Fig. 4a),suggesting that both the $\mathrm { N } \mathrm { - }$ and C-terminal domains are important for PAXX promotion of DSB repair.

PAXX acts upstream of the XRCC4-Lig4 complex. To examine how PAXX interacts genetically with other NHEJ factors,we performed an epistasis analysis by generating $L i g 4 ^ { - \dot { / } - }$ ， $X R C C 4 ^ { - }$ and $X L F ^ { - / - }$ single-mutant DT40 cells and then inactivating PAXX in these cells to create double knockouts (Supplementary Fig.3c-h).In agreement with an earlier report,XRCC4 and Lig4 single knockout cells displayed a strong hypersensitivity to IR (Fig. 4b, left panel)30.The IR sensitivity was not as severe in $P A \bar { X } X ^ { - / - }$ cells as in $X R C C 4 ^ { - }$ and $L i \dot { g } 4 ^ { - / - }$ cells， suggesting that PAXX is not as essential as XRCC4 and Lig4 for DSB repair. Interestingly, the inactivation of PAXX dramatically suppressed IR sensitivity in both $L i g 4 ^ { - / - }$ and $X R C C 4 ^ { - }$ cells (Fig. 4b,left panel） and also partially suppressed the slow proliferation phenotype of $L i g \dot { 4 } ^ { - / - }$ cells (Supplementary Table 1).This phenotype mimics that of $\mathrm { K u } 7 0$ ，the mutation of which also suppresses the IR sensitivity of $L i \dot { g } 4 ^ { - / - } ~ \mathsf { c e l l s } ^ { 3 0 }$ . As Ku functions upstream in the NHEJ pathway, these data imply that PAXX may, similar to $\mathrm { K u } ,$ ，function upstream of the XRCC4-Lig4 complex (Fig.4c,left panel).These data are also consistent with the results of our biochemistry analysis,which indicated that PAXX has a more stable interaction with $\mathtt { K u }$ than with other NHEJ core factors.

![](images/3009a1074a1dddeee0e2d36bc1299ed4216b5db66c9ac05cba3d44ad493e8b9b.jpg)  
Figure 4|PAXX works in both the same and paralel DNA repair pathways with XLF. (a) Sensitivity Curves for $P A X X ^ { - \diagdown { - } }$ DT40 cells complemented by wild-type,2OAorNutvariantsofhumanPAXXinG/Sphase.CellswerearestedinG/SphasebeforeIRtreatmentMutantitesinNmutare indicatedupmetarig.sistalyegeeiciteactisofitCCL)ndFdiyos cels.Survieeaedusineandtepedetexptoatooti genetic relationships of PAXX with XRCC4, Lig4 and XLF under diferent treatment conditions.

Following treatment with the DSB-inducing agents bleomycin, ICRF193 and VP16, $L i g 4 ^ { - / - }$ and $X R C C 4 ^ { - }$ single-mutant cells did not differ significantly in terms of sensitivity compared with their respective PAXX double mutant cells (Fig. 4b,middle panel and Supplementary Fig. 6a,b)，which suggested that PAXX functions in the same NHEJ pathway as XRCC4 and Lig4 to repair DSBs caused by these reagents (Fig. 4c,right panel).

# PAXX and XLF function in both common and parallel pathways.

The $P A X X ^ { - / - } / X L F ^ { - / - }$ double-mutant cells, such as the $P A X X ^ { - / - } / X R C C 4 ^ { - }$ and $P A X X ^ { - / - } / L i g 4 ^ { - / - }$ cells， showed reduced IR sensitivity compared with the ${ \check { X } } L F ^ { - / - }$ single-mutant cells,indicating that PAXX also functions upstream of XLF in the repair of IR-induced DSBs (Fig. $^ { 4 c , \mathrm { d } , }$ leftpanels).Similarly, $X L F ^ { - 7 - }$ single- and $P A X X ^ { - / - } / \bar { X } L F ^ { - / - }$ double-mutant cells showed no significant differences in bleomycin and ICRF193 sensitivity (Supplementary Fig. 6c)，suggesting that PAXX and XLF function in the same pathway to protect against cell death caused bythese agents.Importantly,the $P A X X ^ { - / - } / X L F ^ { - / - }$ doublemutant cells displayed an increased sensitivity to VP16 than did either one of the single-mutant cells (Fig.4d,middle panel).To test whether this phenomena was due to the different cell survival assays for IR and VP16, we also examined the VP16 sensitivity using a colony formation assay.The results were similar to those obtained using the MTS (3-(4,5-dimethylthiazol-2-yl)-5- (3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium) assay (Supplementary Fig.7).Moreover,the double-mutant cells also displayed a slower proliferation than each single mutant (Supplementary Table1).Together, these data suggest that the two XLF paralogues act in parallel pathways to resist VP16-induced cell death and to promote cell proliferation.

To study whetherPAXX and XLF work in parallel pathways to resist other DSB-causing agents，we assayed the effects of doxorubicin,a Topo2 inhibitor similar to VP16.We found that the $P A X X ^ { - / - } / X \dot { L } F ^ { - / - }$ double-mutant cells displayed increased sensitivity to doxorubicin than either of the single-mutant cells (Fig.4d,right panel).By comparison,the doxorubicin sensitivity of $P A X X ^ { - / - ^ { \bot } } / X R C C 4 ^ { - }$ and $P A X X ^ { - / - } / L i g 4 ^ { - / - }$ doublemutant cells was comparable to that of the $X R \bar { C } C 4 ^ { - }$ or $L i g 4 ^ { - / }$ single-mutant cells (Fig. 4b,right panel).These data suggest that PAXX acts in a parallel pathway to XLF but in the same pathwayas XRCC4 and Lig4,to repair DSBs caused by Topo2 inhibitors (Fig. 4c, right panel).

PAXX forms a complex with DNA-bound Ku. It has been shown that the interaction between XLF and the Ku complex is mediated by $\mathrm { D N A } ^ { 3 1 }$ . We found that the addition of ethidium bromide,whichintercalates into DNAand interfereswithDNAprotein interactions， to cell lysates strongly reduced the association between PAXX and other NHEJ proteins, including $\mathrm { K u }$ (Fig.5a).These data suggest that PAXX and XLF share another common feature-they both use DNA to mediate their interactions with Ku.

We investigated how DNA mediates the interaction between PAXX and Ku. One possibility is that DNA acts as a bridge between proteins. We do not favour this possibility， because PAXX showed no detectable binding activity for DNA up to 315bps in length (Fig. 5b and Supplementary Fig.8a,b). This feature difers from its paralogues XRCC4 and XLF,which possess length-dependent DNA-binding activity14,15 (see DNAbinding activity of XLF in Supplementary Fig. 8c,d). The finding that PAXX differs from its paralogues in terms of the DNAbinding characteristics implies that it may have a unique function in the NHEJ pathway.

Another mechanismby which DNA may facilitate the PAXX$\mathrm { K u }$ interaction is that DNA binding may alter the conformation of $\mathrm { K u }$ to favour an interaction with PAXX. One prediction of this hypothesis is thatPAXX would be capable of interactingwith the DNA end-bound Ku complex. Consistent with this prediction, PAXX supershifted the DNA-bound Ku complex (Fig. 5c lane 2) but not free DNA in electrophoretic mobility shift assays (Fig. 5c, lanes 3-6)．As a control,no supershift was observed for the PAXX-CTR domain mutant F201A，which is defective in its association with Ku in co-IP assays (Fig.5c,lanes 7-1O). These results suggest that PAXX can interact with DNA-bound Ku through its CTR domain to form a DNA-Ku-PAXX tertiary complexatDNA ends.This featureresembles thatofXLF,which can form similar DNA-Ku-XLF tertiary complexes31. Thus, the two XLF paralogues form similar complexes at DNA ends to promote DNA end joining,which may explain why they have redundant functions in repairing Topo2 inhibitor-induced DSBs.

PAXX promotes the assembly of the NHEJ complex. The above data suggest that XLF and PAXX can each form a tertiary complex with DNA-bound Ku. This phenomenon prompted us to investigate whether each protein can be assembled into the tertiary complex pre-formed by its paralogue to produce a quaternary complex.Consistent with a previous report 31，XLF supershifted the band corresponding to the DNA-bound Ku complex to produce a band corresponding to the DNA-Ku-XLF tertiary complex (Fig. 5d,lanes 2-5 andlane 12).PAXX further supershifted this band to producea slower mobility band,which probablyrepresentsthe DNA-Ku-XLF-PAXX quaternary complex (Fig.5d,lanes 13-15 and Fig.5e)．Notably，the addition of PAXX significantly reduced the amount of tertiary complex but not of the DNA-Ku complex (Fig. 5e), suggesting that the incorporation of XLF into the DNA-Ku complex promotes the subsequent binding of PAXX.In the reciprocal experiment,XLF similarly supershifted a preformed DNA-KuPAXX tertiary complex (Fig. 5d, lane 6) to produce the DNAKu-XLF-PAXX quaternary complex (Fig.5d, lanes 7-10). Moreover,XLF substantially reduced the amount of tertiary complex but only modestly affected the amount of the binary (DNA-Ku) complex (Fig. 5d,lanes 7-10 and Fig. 5f), indicating that the assembly of PAXX into the DNA-Ku complex also facilitated subsequent binding by XLF. Together, these data suggest that the two XLF paralogues can cooperate to assemble a DNA-Ku-XLF-PAXX quaternary complex at DNA ends. This quaternary complex is more stable than either tertiary complex, which explainswhy both XLF and PAXX are required fora stable NHEJ complex to repair IR-induced DSB ends.

PAXX stimulates the ligation of non-cohesive DNA ends.XLF has been shown to robustly stimulate the ligation of mismatched and non-cohesive DNA ends in vitrol6.We found that the presence of PAXX had no significant effect on the ligation of DNAsubstrateswithnon-cohesiveends(EcoRV-KpnI overhangs）using the same assay (Fig.6a,left panel).As a control, XLF strongly enhanced this ligation reaction (11.4-fold at $2 . 5 \mathrm { n M }$ concentration; Supplementary Fig. 9). Interestingly, in the presence of $2 . 5 \mathrm { n M }$ XLF,PAXX stimulated the ligation by 7.6-fold (Fig.6a,middle panel),demonstrating that PAXX has an XLFdependent stimulatory effect on the ligation of non-cohesive DNA ends.Moreover,this stimulatory activity was not detected for the PAXX-F201A mutant (Fig. 6a,right panel),which has a deficient association withKu.This characteristic resembles that of $\mathrm { X L F } ^ { 3 2 }$ ， which suggests that the interaction with $\mathtt { K u }$ is essential for both XLF paralogues to stimulate the ligation of DNA ends. Furthermore,neither PAXX nor its mutant (F201A）had a significant effect on the ligation of blunt DNA ends (EcoRVEcoRV) in the absence or the presence of XLF (Fig.6b),which implies that PAXX preferentially stimulates the joining of complex but not simple DNA ends.

![](images/351b15ae7e926408564df26b1eeb88756240c5c72b81bffed8ed6825be9266a6.jpg)  
Figure5|PAXXpromotestheaemblyofLFintotheDAend-boundKucomplex.(a）lmmunoblotingshowingFlag-AXithowitouttB (b)Electrophoretcobityhifts)showigthatAXdsnotbndReactioscontain5Cy-lbelld3O-bpad 100or300nMPAXX.(c)EMSAaashowingthatPAXbutnotitsmutantF2OAsupershiftsteDNA-KucomplexReactionswereperforedas inbexdinfnadlmotesa.g that PAXX and XLF mutually promote their recruitment to the DNA- ${ \bf \nabla } \cdot \sf K u$ complex. Reactions contain 5nM Cy3-labelled 3O-bp dsDNA and the indicated proteins.The series of concentrations was 5O,1OO,200 and $4 0 0 { \mathsf { n } } M$ for XLF,and 30,100 and $3 0 0 { \mathsf { n } } M$ for PAXX.(e,f) Quantifications of reduced complexes.Tereducedcompleesindes6-fandes-5(e)wereuantifedeomplequanitiesinnesndeetat $100 \%$ for the respective experiments.

# Discussion

In this study,we performed interactome analyses of the human NHEJ pathway and identified PAXX as a new component of this pathway.PAXX stably interacts with the DNA end-bound $\mathtt { K u }$ complex,cooperates with XLF to stimulate non-cohesive DNA end ligation in vitro and is required for cellular resistance to DSBcausingagents in vivo.Genetic analyses further demonstrated that

PAXX and the XRCC4-Lig4 complex work in a common pathway to repair DSBs induced by IR,bleomycin and Topo2 inhibitors.

PAXX shares structural and functional similarities with its paralogues XRCC4 and XLF; they all possess the same N-terminal domains,can stably assemble into a DNA-end-bound NHEJ complex, can rapidly relocalize to DSBs and can promote cellular resistance to DSB-inducing agents. Structurally, the N-terminal domain of PAXX is more similar to that of XRCC4,as both exhibit a $4 5 ^ { \circ }$ angle between the head domain and the helix stalk. Conversely,the C-terminal domain of PAXX is more similar to that of XLF,as indicated by the fact that both can directly interact with DNA-bound $\mathtt { K u }$

The finding that the inactivation of PAXX,XRCC4 and XLF in DT40 cells resulted in cellular hypersensitivity to agents causing DSBs indicates that each protein must have at least one unique role in the NHEJ pathway that cannot be substituted by its paralogues. One unique feature of PAXX is that it shows a strong association with $\mathtt { K u }$ but no detectable direct association with Lig4. This characteristic differs from XRCC4,which stably associates withLig4,and from XLF,which weakly associateswith both Ku and XRCC4.Another unique feature of PAXX is that it is present invertebrates but absent in yeast and most invertebrates,unlike XRCC4 and XLF,which are present in all eukaryotes.Another NHEJ factor that shows a similar evolutionary distribution is DNA-PKcs,which mediates the joining of complex but not simple ends in V(D)J recombination23.It is possible that PAXX co-evolved with DNA-PKcs to repair DSBs with more sophisticated endsinhigher eukaryotes.

![](images/2d68d53d7e30f044865c3dbc1ba05849bfe4ee15524f6070da4a7c01db883e10.jpg)  
Figur6|PAXXpromotesnon-cohesiveendjoining.(b）PAXXstimulatesnon-cohesiveendligationinthepresenceofXLFThelinarDA substrates contain blunt- ${ \boldsymbol { \cdot } } 3 ^ { \prime }$ overhangs(EcoRV-Kpnl)(a)or blunt-blunt (EcoRV-EcoRV)(b)ends.The reactions contain 5nM Ku7O/80,2.5nM XRCC4-Lig4 and the indicated wild-type PAXX or its mutant F2O1A,with or without $2 . 5 { \mathsf { n M } }$ XLF.Data from three independent experiments are represented as the mean ± s.e.m.

The observation that PAXX and Ku can form a stable complex suggests that they work together in the NHEJ pathway. Consistent with this suggestion,our genetic data showed that similar to Ku,PAXX may act upstream of the XRCC4-Lig4 complex.This hypothesis is supported by the finding that mutations in PAXX suppress the IR sensitivity of XRCC4 or Lig4 mutantDT40 cells (Fig.4b),which mimics previous findings that the Ku mutation suppresses the phenotype of Lig4 mutations in DT40 celsand mice30,33One model to explain the observed genetic suppression is that without Lig4 or XRCC4,the NHEJ pathway is inactive and DSBs are channelled to alternative DSB repair pathways,including HR and microhomology-mediated end joining30,33However,binding of the PAXX-Kucomplex to DNA ends protects the ends against resection,and thus inhibits the alternative pathways.The deletion of either PAXX or $\mathtt { K u }$ releases the block and allows resection and alternative repair reactions to occur, thus suppressing the mutant phenotypes of Lig4 or XRCC4.

PAXX isclosely related to XLF; both proteins can interact with the DNA end-bound Ku complex through their C-terminal domains and stimulate the ligation of non-cohesive DNA ends. Interestingly,genetic studies have revealed that the two proteins work through different mechanisms to promote cellular resistance to different types of DSBs induced by various agents.For IR-induced DSBs that carry DNA ends that are more complex or ‘dirty’(with many chemical modifications)24, the two proteins work in the same pathway (Fig.7a).However, for DSBs with DNA ends that are‘clean'or have fewer modifications,such as those produced by Topo2 inhibitors25,26 and processed by TDP2 (ref.34; Fig.7b),the two proteins act in parallel pathways. One hypothesis to explain the difference is that the ‘dirty' ends require a more stable complex to maintain their alignment and to allow a ‘trial and error’ process by different enzymes to remove the complex covalent modifications1.Both PAXX and XLF may be required to constitute such a stable complex (Fig.7a). PAXX may firstbe recruited to DNA-bound Ku,because PAXX has a more stable association with the Ku complex than XLF (Fig. ${ \mathrm { 1 b , c , e } }$ ） This process results in the subsequent recruitment of XLF to form the PAXX-DNA-Ku-XLF complex,which stabilizes the complex and enables theassemblyofotherNHEJ factors.For‘clean'ends, the‘trial and error'process is not needed and, therefore,one XLF orPAXX protein may be sufficient to maintain the end alignment for ligation (Fig. 7b).This finding is consistent with our biochemical observation that PAXX and XLF can be assembled individually into a DNA-bound Ku complex (Fig. 5c,d). In support of this hypothesis,our biochemistry data showed that PAXX and XLF cooperated to stimulate the ligation of complex (non-cohesive） but not simple (blunt) DNA ends.Thus,PAXX and XLF have both cooperative and redundant functions that are dependent on the complexity of the DSB ends.

Altogether,our data provide new insight into the process of NHEJ factor organization and assembly in response to different DSBs that arise in a variety of ways.

# Methods

Cell culture and transfection.HeLa,U2OS,HCT116 and HEK293 cells were purchased from ATCC.DT40 cells were a gift from the laboratory of Dr Shunichi Takeda.HeLa and U2OS cells were cultured in DMEM supplemented with $1 0 \%$ FCS at $3 7 ^ { \circ } \mathrm { C }$ with $5 \%$ $\mathrm { C O } _ { 2 }$ .HCT116 cells were cultured in RPMI1640. Suspension HEK293 cells were cultured in Freestyle medium (Invitrogen) plus $1 \%$ Gibco FCS and $1 \%$ glutamine in an incubator with shaking at $1 3 0 \mathrm { r . p . m }$ .Plasmid transfections were conducted with Lipofectamine 2ooo (Invitrogen) and PEI (polyethyleneimine, Polysciences) for U2OS and suspension HEK293 cells,respectively.

![](images/0443bebe31b0193def5782c51b0e56fb38056479293025fa47820f9e724cb0c5.jpg)  
Figure7|AmodelshowingtheelatioshipofPAXandLFinDBrepair.(a）inducesDBswithomplexendsontaningmultpleodificatio ('dirtyeefuegotXdtblgtdsfotrdrsi muliplemdfs.piubillucetatioofsede Topo2 peptide willbe removed by TDP2 to generate a cohesive end that is phosphorylated at its $5 ^ { \prime }$ -end.These ‘clean'ends can be quickly ligated even by an incomplete NHEJ complex lacking XLF or PAXX.

DT40 cells were cultured in RPMI 1640 medium supplemented with $1 0 \%$ FCS, $1 \%$ chicken serum, $1 0 \mathrm { m M }$ HEPES and $1 \%$ penicillin-streptomycin mixture at $3 9 . 5 ^ { \circ } \mathrm { C }$ with $5 \%$ $\mathrm { C O } _ { 2 }$ .Transfection was performed by electroporation using the Lonza Nucleofector 4D.For selection, growth medium containing G418 $( 2 \mathrm { m g } \mathrm { m l } ^ { - 1 } )$ ,puromycin $( 0 . 5 \mu \mathrm { g } \mathrm { m l } ^ { - 1 } ,$ ,Blasticidin $( 2 5 \mu \mathrm { g } \mathrm { m l } ^ { - 1 } ,$ or histidinol $( 1 \mathrm { m g } \mathrm { m l } ^ { - 1 } ,$ wasused.

Antibodies.Rabbit PAXX polyclonal antibodies were generated against an MBPfused protein containing full-length PAXX and were used at dilution of 1:1,00o for western blotting(WB).Anti-Ku80 mouse monoclonal antibody was purchased from Santa Cruz (sc-56132,1:200 for WB),anti-XLF from Bethyl (A300-730A, 1:1,000 for WB),anti- $\cdot \beta$ -actin mouse monoclonal antibody from MBL(M177-3, 1:2,000 for WB),anti-XRCC4 rabbit polyclonal antibodies from Abcam (ab145, 1:1,000 for WB),anti-DNA-PKcs goat polyclonal antibodies from NeoMarkers Fremont $^ { \prime } 4 2 3 \times 8 1 2$ ,1:1,000 for WB),anti-Lig4 rabbit polyclonal antibodies from Proteintech Group (12695-I-AP,1:1,000 for WB)and anti-Flag M2 monoclonal antibody from Sigma-Aldrich (F3165,1:1,0o0 for WB).Uncropped WBs are shown in Supplementary Fig.10.

Immunoprecipitation.The complementary DNAs of The NHEJ genes were obtained from hORFemone (V8.1).Mammalian expression plasmids were generated by shuttling cDNAs to the destination plasmid pDEST26-Flag using an LR reaction (Invitrogen).

For immunoprecipitation,suspension HEK293 cells were transfected with expression plasmids using PEI.After $6 4 \mathrm { h }$ ,the cells were harvested and the pellets were directly lysed with NTEN buffer $2 0 \mathrm { m M }$ Tris-HCl, pH 7.5,150 mM NaCl, $1 0 \%$ glycerol, $0 . 5 \%$ NP40, $1 0 \mathrm { m M }$ NaF, $1 \mathrm { m M }$ phenylmethylsulfonyl fluoride (PMSF), $1 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ leupeptin, $1 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ aprotinin).Following ultra-centrifugation at $^ { 4 4 0 , 0 0 0 g }$ for $1 5 \mathrm { m i n }$ at ${ 4 } ^ { \circ } \mathrm { C } ,$ the supernatant was incubated with anti-Flag M2- conjugated agarose beads for $3 \sim 4 \mathrm { h }$ .The beads were then spun down and washed four times with IP buffer ( $2 0 \mathrm { m M }$ Tris-HCl, $\mathrm { p H } 7 . 5$ ,150 mMNaCl, $\mathrm { M g C l } _ { 2 }$ （204号 $5 \mathrm { m M }$ $1 0 \%$ glycerol, $0 . 1 \%$ NP40,1mM dithiothreitol (DTT) and 1mMPMSF). Subsequently,the complexes were eluted with IP buffer containing $4 0 0 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ （20 $3 \times \mathrm { F l a g }$ peptide and analysed by SDS-PAGE and mass spectrometry.

Protein purification.For XLF,PAXX and its mutants,glutathione S-transferasefused proteins were expressed in Escherichia coli (Transetta cells,TransGen) from a pCPD9 vector. Cells were grown at $3 7 ^ { \circ } \mathrm { C }$ until an $\mathrm { O D } 6 0 0 = 0 . 8$ and induced with $0 . 3 \mathrm { m M }$ isopropylthiogalactoside at $2 8 ^ { \circ } \mathrm { C }$ for $^ { 3 \mathrm { h } }$ .The cell pellet from the 0.5-1 culture was lysed using a French Press in $3 0 \mathrm { m l }$ ofPBST buffer(PBS buffer $\mathrm { \Omega } ^ { \prime } 1 0 \mathrm { m M }$ phosphate,pH 7.4, $1 3 7 \mathrm { m M } \mathrm { N a C l }$ and $2 . 7 \mathrm { m M }$ KCl) supplemented with $1 0 \%$ glycerol, $0 . 1 \%$ Triton100, $0 . 2 \mathrm { m M }$ DTT and $0 . 5 \mathrm { m M }$ PMSF). The lysate was

centrifuged at ${ 3 5 , 0 0 0 g }$ for $2 5 \mathrm { m i n }$ and the supernatant was incubated with $1 \mathrm { m l }$ Glutathione Sepharose 4B beads (GE Healthcare) at $4 ^ { \circ } \mathrm { C }$ for $^ { 3 \mathrm { h } }$ .The beadswere washed twice with high-salt PBST buffer $( 5 0 0 \mathrm { m M } \mathrm { N a C l } )$ andtwicewith PBST buffer.Protein was eluted using TEV $( 5 0 \mu \mathrm { g } \mathrm { m l } ^ { - 1 } ,$ inTEVcleavagebuffer( $\mathrm { 1 0 ~ m M }$ Tris Cl, $\mathrm { \ p H \ 8 . 0 }$ ，100 mMNaCl, $0 . 5 \mathrm { m M }$ EDTA, $0 . 0 1 \%$ NP40, $1 0 \%$ glycerol and $1 \mathrm { m M }$ DTT)and loaded ona Superdex 75(1o/3oo) column (GE Healthcare) that had been pre-equilibrated with buffer ( $\mathrm { 5 0 } \mathrm { m M }$ Tris,pH7.4,0.5M NaCl,1 mM EDTA, $1 0 \%$ glycerol, $1 0 \mathrm { m M }$ $\beta$ -mercaptoethanol).Peak fractions containing the purified protein were flash-frozen and stored at $\mathrm { ~ - ~ } 8 0 ^ { \circ } \mathrm { C }$

XRCC4-Lig4 and $\mathrm { K u } 7 0 / 8 0$ complexes were purified as described³5.Briefly, Sf9 insect cells infected with baculoviruses containing XRCC4 and Ligase IV-His (a gift from Dale Ramsden) were lysed,and XRCC4-Lig4 was purified with Ni-NTA beads (GE Healthcare),followed by Superdex-2o0 (26/60)and Mono Q $( 5 / 5 0 ~ \mathrm { G L } )$ columns (GE Healthcare).Ku was purified from the lysate of Sf9 cells expressing Ku70-His and Ku86(a gift from Dale Ramsden) using Ni-NTA beads (GE Healthcare) and by gel filtration using a Superose-12 (HR 10/30) column (GE Healthcare).

Forcrystallization,PAXX1-145 protein with a ${ \mathrm { H i s } } _ { 6 }$ -tagat its N terminuswas expressed using the pETDuet-1 vector in $E$ ，coli.The expression of the PAXX 1-145 protein was induced using $0 . 2 \mathrm { m M }$ isopropylthiogalactoside at $1 6 ^ { \circ } \mathrm { C }$ The pelleted cellswere suspended in binding buffer $2 0 \mathrm { m M }$ Tris-HCl, pH 8.0, $2 0 0 \mathrm { m M }$ $\mathrm { \Delta N a C l ) }$ and crushed by sonication.After centrifugation at ${ 3 5 , 0 0 0 g }$ for $3 0 \mathrm { m i n }$ at ${ 4 } ^ { \circ } \mathrm { C } ,$ （204号 the supernatant was gently mixed with nickel beads (GE Healthcare).After elution from the nickel beads,PAXX1-145 protein was further purified using a $\mathrm { \Delta Q }$ ion exchange column (HiTrap QHP,GE Healthcare) and gel filtration chromatography (HiLoad_16/60_Superdex200_prep grad,GE Healthcare) using AKTApurifier (GE healthcare).The purified protein was dissolved in crystallization buffer $\mathrm { \Omega } ^ { \mathrm { 2 0 } \mathrm { m M } }$ Tris-HCl, $\mathfrak { p H } 8 . 0$ ,200 mMNaCl),identified using SDS-PAGE and crystallized at $1 0 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ .As the methionine content of PAXXi-145 is limited,a double-mutated PAXX1-145L57M/L105M was designed to prepare the Se-Metlabelled protein.The selenium derivative was expressed using $E .$ coli grown in M9 culture with selenio-methionine and purified using the same methods.

Crystalization and structure determination.Both native and Se-derivative PAXX 1-145 proteins were crystallized by vapour diffusion.The native protein was crystallized in a solution of 1.5 M $( \mathrm { N H _ { 4 } } ) _ { 2 } \mathrm { S O _ { 4 } } ,$ 0.1 M Tris-HCl, $\mathrm { p H } 8 . 5$ and $12 \%$ （204 glycerol. Se-derivative crystals were observed in a solution of $2 . 2 4 \mathrm { M }$ （20 $( \mathrm { N H _ { 4 } } ) _ { 2 } \mathrm { S O _ { 4 } }$ and $0 . 1  { \mathrm { M } }$ Bis-Tris, $\mathrm { p H } 5 . 5$ ，After dehydration and cryo-protection in $5 0 \%$ glycerol, the crystals were flash-cooled using liquid nitrogen.The diffraction data were collected at $1 0 0 \mathrm { K }$ at SSRF (Shanghai Synchrotron Radiation Facility) beamline BL17U1 and processed using HKL2ooo software.The selenium sites were discovered via SHELX using the single-wavelength anomalous scattering method.The initial phase was generated using AutoSol in PHENIX.AutoBuild was used

alternately to improve the electron density map and generate the model. Approximately $9 0 \%$ of the residues in the protein were built automatically and additional missing residues were added manually in COOT and refined with PHENIX.The native structure was determined via Phaser using the MR method. The Se-derivative structure was used as the initial model. The final structure was refined using PHENIX.In the Ramachandran plot generated using PROCHECK, $9 4 . 4 \%$ and $4 . 8 \%$ of the amino acids in the final atomic model were in the most favourable and additional allowed regions,respectively.Allof the structural figures were prepared using COOT and PyMOL.

Generation of DT4O knockout strains.DT40 knockout constructs for XLF, PAXX,XRCC4 and Lig4 were generated as previously described36 using a MultiSite Gateway Three-Fragment Vector Construction Kit. The $5 ^ { \prime }$ and $3 ^ { \prime }$ arms were amplified from genomic DNA using the primers listed in Supplementary Table 2. They were cloned into the pDONR P4-P1R and pDONR P2R-P3 vectors, respectively.The knockout constructs were generated by LR recombination of the pDONR- ${ . 5 ^ { j } }$ arm,pDONR- $3 ^ { \prime }$ arm,resistance gene cassette-containing pDONR-211 and pDEST R4-R3 destination vector. The knockout constructs were linearized before transfection.The primers used for genomic DNA PCR screening are listed in Supplementary Table 2.

Generation of HCT116 knockout strains.PAXX-deficient HCT116 cells were generated using CRISPR.Briefly, two guide sequences $5 ^ { \prime }$ -CGGGGCGGCTC GGGGCCCGG $3 ^ { \prime }$ and $5 ^ { \prime }$ -AGCTGTGGCTCTGACTCTGC- $3 ^ { \prime }$ ,targeting two different sites of the human PAXX gene,respectively,were inserted into the pX330 vector37.The guide sequence-containing $\mathtt { p X } 3 3 0$ plasmids were transfected into HCT116 cells. Single colonies were picked after 8-10 days of incubation. The genomic fragments of the PAXX gene were amplified by PCR using primers $5 ^ { \prime }$ -CCGGCTCCGGAAGTCGTTCTTCC- ${ \boldsymbol { \cdot } } 3 ^ { \prime }$ and $5 ^ { \prime }$ -CGGCGTGAAGCAGGT GCTCCAAAG- $3 ^ { \prime }$ ,and $5 ^ { \prime }$ -GGTGAGACCCAAGCAGGGAGGAAG- $3 ^ { \prime }$ and $5 ^ { \prime }$ -GGACCCCAGTGAAGGTATGAGGTG- $3 ^ { \prime }$ ,and digested with ApaI and PstI, respectively. Colonies containing the expected PCR fragments were further sequenced and examined by WB analysis.

Cell survival assay.The cell survival curves for the DT4O cells were measured using the MTSassayasdescribed previously38.Briefly,1,500-3,0o0cels were plated into each well of a 96-well plate with a range of doses of ICRF193 or camptothecin.After a 48-h incubation,the cells were pulsed with CellTiter 96 Aqueous One Solution Reagent (Promega) for $\mathrm { 4 h }$ .Cell viability was measured with aluminometer and each dose point was measured in triplicate.For bleomycin and VP16,a density of 30o-1,0oo cells per well and a 72-h incubation were used.

The IR sensitivity assay for DT40 cells was performed using a colony formation assay.Briefly,200-20,0oo cells were seeded in each well of $^ 6$ -well plateswith $0 . 7 \%$ methylcellulose medium.The plates were exposed to the appropriate dose of X-ray irradiation.Afterincubatingat $3 9 . 5 ^ { \circ } \mathrm { C }$ for 6 days,the number of colonies was counted.To measure the Gi/S phase-specific radiation sensitivity,DT40 cells were cultured in medium containing $0 . 5 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ nocodazole for $^ { 8 \mathrm { h } }$ and then washed three times with PBS containing $5 \%$ calf serum. The cells were further cultured in medium containing $0 . 8 \mathrm { m M }$ mimosine (Sigma) for $^ { 8 \mathrm { h } }$ and then washed three times as described above.The synchronized cells then were seeded on $0 . 7 \%$ methylcellulose medium for IR.

Colony formation assay of DT40 cells to VP16 was performed as described previously28. Briefly,the appropriate number of cels was plated in $0 . 7 \%$ （204号 methylcellulose (Sigma)-containing medium with a range of doses of VP16.After incubation at $3 9 . 5 ^ { \circ } \mathrm { C }$ for 7-14 days,the number of colonies was counted. Cell survival curves for HCT116 cels to IR and VP16 was performed as described39. Appropriate cell number was plated in six-well plates,cultured for $2 4 \mathrm { h }$ and exposed to the appropriate dose of X-ray irradiation.For VPl6,the cells were cultured for $2 4 \mathrm { h }$ and then the indicated dose of VP16 was added to the medium. Colonies were stained with methylene blue and counted after an additional 9-14 days of incubation.

Electrophoretic mobility shift assay. The DNA substrates were generated by annealing the oligos H40f $5 ^ { \prime }$ -GTGACCGTCTCCGGGAGCTGGAAACGCGCGA GACGAAAGG- $3 ^ { \prime }$ and H40r $5 ^ { \prime }$ -CCTTTCGTCTCGCGCGTTTCCAGCTCCCGG AGACGGTCAC- $3 ^ { \prime }$ ；H60f $5 ^ { \prime }$ -GACGCTGCCGAATTCTACCAGTGCCTTGCT GGACATCTTTGCCCACCTGCAGGTTCACCC- $3 ^ { \prime }$ and H60r $5 ^ { \prime }$ -GGGTGAA CCTGCAGGTGGGCAAAGATGTCCAGCAAGGCACTGGTAGAATTCGG CAGCGTC- $3 ^ { \prime }$ ：HL-111 $5 ^ { \prime }$ -GGGCTACCGTCAAGTAAGATGCAGATACGGAAC ACAGCTGGCACAGTGGTAGTACTCCACTGTCTGGCTGTACAAAAACCC TCGGGATCT- $3 ^ { \prime }$ and HL-112 $5 ^ { \prime }$ -AGATCCCGAGGGTTTTTGTACAGCCAG ACAGTGGAGTACTACCACTGTGCCAGCTGTGTTCCGTATCTGCATCTTA CTTGACGGTAGCCC- $3 ^ { \prime }$ ),respectively.Before annealing, the $5 ^ { \prime }$ ends of the oligos H40f,H6of and HL-11l were labelled with $^ { 3 2 } \mathrm { P }$ using T4 polynucleotide kinase.The 30-bp Cy3-labelled double-stranded DNA (dsDNA) was generated as previously described40.The 315-bp dsDNA was generated by PCR with primers c9orfl42PCRu and c9orfl42PCRd,using chicken genomic DNA.The indicated amounts of protein and ${ 5 \mathrm { n M } } ^ { 3 2 } { \mathrm { P } } { \mathrm { - } }$ or Cy3-labelled DNA substrate was incubated at $2 5 ^ { \circ } \mathrm { C }$ in $1 0 \mu \mathrm { l }$ reaction buffer ( $2 5 \mathrm { m M }$ Tris-HCl at pH7.5,5 mM $\mathrm { \ M g C l } _ { 2 }$ ， $1 0 0 \mathrm { m M }$

NaCl, $1 \mathrm { m M }$ DTT, $0 . 0 5 \%$ Triton X-100, $1 0 0 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ BSA and $5 \%$ glycerol) for $1 5 \mathrm { m i n }$ .The reaction mixture was loaded and resolved in a $5 \%$ TBE gel.

DNAligationassay. The DNA ligation assay was performed as described previouslyl6.Briefly,the reactions were performed in buffer consisting of $2 5 \mathrm { m M }$ Tris, $\mathrm { p H ~ 8 . \dot { 0 } / 1 0 0 ~ m M ~ N a C l / 0 . 1 ~ m M ~ E D T { \dot { A } } / 5 0 ~ \mu g ~ m l ^ { - 1 } ~ B S A / 0 . 0 5 \% }$ Triton $\mathrm { X } { - } 1 0 0 / 2 \mathrm { m M }$ $\mathrm { D T T } / 5 \%$ (wt/vol) PEG8ooo.The proteins were incubated with DNA substrates for $1 0 \mathrm { m i n }$ before the reaction at $2 5 ^ { \circ } \mathrm { C }$ ，To initiate ligation, $5 \mathrm { m M } \mathrm { M g } \mathrm { C l } _ { 2 }$ and $0 . 1 \mathrm { m M }$ ATP were added to the mixture.The reactions were conducted in $2 0 \mathrm { - } \mu \mathrm { l }$ volumes containing $1 \mathrm { n M }$ DNA(DNA1 and DNA2) and the indicated proteins at $3 7 ^ { \circ } \mathrm { C }$ for $3 0 \mathrm { m i n }$ .The reaction was stopped by adding $2 \mu \mathrm { l }$ of $0 . 5 \mathbf { M }$ EDTA $\mathrm { ( p H ~ 8 . 0 ) }$ . The produced DNA was purified on a QIAquick column (Qiagen) and measured by quantitative PCR.

# References

1.Waters,C.A., Strande,N.T.,Wyatt,D.W.,Pryor,J.M.& Ramsden,D.A. Nonhomologous end joining:a good solution for bad ends.DNA Repair (Amst) 17,39-51 (2014).   
2.Lieber,M.R. The mechanism of double-strand DNA break repair by the nonhomologous DNA end-joining pathway. Annu. Rev.Biochem.79,181-211 (2010).   
3.Mahaney,B.L.,Hammel, M.,Meek,K.,Tainer,J.A.& Lees-Miller, S.P. XRCC4 and XLF form long helical protein flaments suitable for DNA end protection and alignment to facilitate DNA double strand break repair. Biochem. Cell Biol.91,31-41 (2013).   
4.van Breugel,M.et al. Structures of SAS-6 suggest its organization in centrioles. Science 331,1196-1199 (2011).   
5．Kitagawa,D.et al. Structural basis of the 9-fold symmetry of centrioles.Cell 144,364-375 (2011).   
6.Wu, Q.et al. Non-homologous end-joining partners ina helical dance: structural studies of XLF-XRCC4 interactions. Biochem. Soc. Trans.39, 1387-1392,suppl 2 p following 1392 (2011).   
7．Ropars,V.et al. Structural characterization of flaments formed by human Xrcc4-Cernunnos/XLF complex involved in nonhomologous DNA endjoining.Proc.Natl Acad.Sci. USA 108,12663-12668 (2011).   
8.Hammel,M.et al.XRCC4 protein interactions with XRCC4-like factor (XLF) create an extended grooved scaffold for DNA ligation and double strand break repair.J.Biol.Chem.286,32638-32650 (2011).   
9．Andres,S.N.et al.A human XRCC4-XLF complex bridges DNA.Nucleic Acids Res.40,1868-1878 (2012).   
10.Yano,K.,Morotomi-Yano,K.,Lee,K.J.& Chen,D.J.Functional significance of the interaction with Ku in DNA double-strand break recognition of XLF.FEBS Lett. 585,841-846 (2011).   
11.Grawunder,U.et al.Activity of DNA ligase IV stimulated by complex formation with XRCC4 protein in mammalian cells.Nature 388,492-495 (1997).   
12.Sibanda,B.L.et al. Crystal structure of an Xrcc4-DNA ligase IV complex.Nat. Struct.Biol.8,1015-1019 (2001).   
13.Riballo,E.et al. XLF-Cernunnos promotes DNA ligase IV-XRCC4 readenylation following ligation.Nucleic Acids Res.37,482-492 (2009).   
14.Lu,H.,Pannicke,U.,Schwarz,K.& Lieber,M.R.Length-dependent binding of human XLF to DNA and stimulation of XRCC4.DNA ligase IV activity.J. Biol. Chem. 282,11155-11162 (2007).   
15.Modesti,M.,Hesse,J.E.& Gellert,M.DNA binding of Xrcc4 protein is associated with V(D)J recombination but not with stimulation of DNA ligase IV activity.EMBO J.18,2008-2018 (1999).   
16.Tsai, C.J.,Kim, S.A.& Chu,G.Cernunnos/XLF promotes the ligation of mismatched and noncohesive DNA ends.Proc. Natl Acad.Sci. USA 104, 7851-7856 (2007).   
17. Gu,J.,Lu,H., Tsai, A.G.,Schwarz,K.& Lieber,M.R. Single-stranded DNA ligation and XLF-stimulated incompatible DNA end ligation by the XRCC4- DNA ligase IV complex: influence of terminal DNA sequence.Nucleic Acids Res.35,5755-5762 (2007).   
18.Escribano-Diaz,C.et al.A cellcycle-dependent regulatory circuit composed of 53BP1-RIF1 and BRCA1-CtIP controls DNA repair pathway choice.Mol. Cell 49,872-883 (2013).   
19.Bekker-Jensen, S.et al. Spatial organization of the mammalian genome surveillance machinery in response to DNA strand breaks.J. Cell Biol.173, 195-206 (2006).   
20.Ahnesorg,P.,Smith,P.& Jackson,S.P.XLF interacts with the XRCC4-DNA ligase IV complex to promote DNA nonhomologous end-joining. Cell 124, 301-313 (2006).   
21.Andres,S.N.,Modesti, M.,Tsai, C.J.,Chu, G.& Junop,M.S.Crystal structure of human XLF:a twist in nonhomologous DNA end-joining.Mol Cell 28, 1093-1101 (2007).   
22.Junop,M. S.et al. Crystal structure of the Xrcc4 DNA repair protein and implications for end joining.EMBO J.19,5962-5970 (2000).   
23.Kumar,V.,Alt,F.W.& Oksenych,V.Functional overlaps between XLF and the ATM-dependent DNA double strand break response.DNA Repair (Amst)16, 11-22 (2014).   
24.Povirk,L.F.Processing of damaged DNA ends for double-strand break repair in mammalian cells.ISRN Mol.Biol.(doi:10.5402/2012/345805) (2012).   
25.Shibata,A.et al. Factors determining DNA double-strand break repair pathway choice in G2 phase.EMBO J.30,1079-1092 (2011).   
26.Riballo,E.et al.A pathway of double-strand break rejoining dependent upon ATM,Artemis,and proteins locating to gamma-H2AX foci.Mol. Cell 16, 715-724 (2004).   
27.Takata,M. et al. Homologous recombination and non-homologous end-joining pathways of DNA double-strand break repair have overlapping roles in the maintenance of chromosomal integrity in vertebrate cells.EMBO J.17, 5497-5508 (1998).   
28.Adachi, N., Suzuki,H.,Iizumi, S.& Koyama,H. Hypersensitivity of nonhomologous DNA end-joining mutants to VP-16 and ICRF-193: implications for the repair of topoisomerase II-mediated DNA damage.J. Biol. Chem. 278,35897-35902 (2003).   
29.Adachi, N.,So, S.& Koyama,H.Loss of nonhomologous end joining confers camptothecin resistance in DT4O cells.Implications for the repair of topoisomerase I-mediated DNA damage.J.Biol. Chem.279,37343-37348 (2004).   
30.Adachi,N.,Ishino,T., Ishii,Y.,Takeda,S.& Koyama,H.DNA ligase IVdeficient cells are more resistant to ionizing radiation in the absence of $\mathrm { K u } 7 0$ Implications for DNA double-strand break repair.Proc.Natl Acad.Sci. USA 98, 12109-12113 (2001).   
31.Yano,K.et al.Ku recruits XLF to DNA double-strand breaks.EMBO Rep.9, 91-96 (2008).   
32.Tsai, C.J.& Chu,G.Cooperative assembly of a protein-DNA flament for nonhomologous end joining.J. Biol. Chem.288,18110-18120 (2013).   
33.Karanjawala,Z.E.et al. The embryonic lethality in DNA ligase IV-deficient mice is rescued by deletion of Ku: implications for unifying the heterogeneous phenotypes of NHEJ mutants.DNA Repair (Amst)1,1017-1026 (2002).   
34.Gomez-Herreros,F.et al. TDP2-dependent non-homologous end-joining protects against topoisomerase II-induced DNA breaks and genome instability in cells and in vivo.PLoS Genet.9,el003226 (2013).   
35.Riballo,E.et al. Cellular and biochemical impact of a mutation in DNA ligase IV conferring clinical radiosensitivity.J. Biol. Chem.276,31124-31132 (2001).   
36.Iiizumi, S.et al. Simple one-week method to construct gene-targeting vectors: application to production of human knockout cell lines.Biotechniques 41, 311-316 (2006).   
37.Cong,L. et al. Multiplex genome engineering using CRISPR/Cas systems. Science 339,819-823 (2013).   
38.Xu,D.et al.RMI,a new OB-fold complex essential forBloom syndrome protein to maintain genome stability.Genes Dev.22,2843-2855 (2008).   
39.Katsube,T.et al.Differences in sensitivity to DNA-damaging Agents between XRCC4-and Artemis-deficient human cells.J.Radiat.Res.52,415-424 (2011).   
40.Grundy,G.J.et al.APLF promotes the assembly and activity of nonhomologous end joining protein complexes.EMBO J. 32,112-125 (2013).

# Acknowledgements

We thank Weidong Wang for advice and modification of the manuscript.This work was supported in part by the National Basic Research Program of China (2013CB91l002 and 2010CB912203),the National Natural Science Foundation of China (31271435,31370836 and 3127o872),the Peking University-Tsinghua University Center for Life Sciences and the Seeding Grant for Medicine and Life Sciences of Peking University (2Ol4-MB-18).

# Author contributions

M.X.,M.Y.,W.H.,L.W., S.N., Z.Y.,W.L.,Q.W.,C.D.andR.G.conducted the experiments. G.G.,J.J.,L.L., Y.Land D.X. supervised the project. H.L.,F.F.,M.X.,M.Y., W.H.,M.H.,G.G.,J.J.,L.L.,Y.L.and D.X.analysed the data.M.X.,M.Y.,H.L.,F.F.,Y.L. and D.X.wrote the manuscript,with input from the other authors.

# Additional information

Accession codes.The atomic coordinates and structure factors for PAXX N-terminal fragment have been deposited in the Protein Data Bank with accession code 4WJA.

Supplementary Information accompanies this paper at http://www.nature.com/ naturecommunications

Competing financial interests:The authors declare no competing financial interests.

Reprints and permission information is available online at http://npg.nature.com/ reprintsandpermissions/

How to cite this article:Xing,M.et al.Interactome analysis identifies a new paralogue of XRCC4 in non-homologous end-joining DNA repair pathway. Nat. Commun. 6:6233 doi:10.1038/ncomms7233 (2015).

This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license,visit http://creativecommons.org/licenses/by/4.0/