# SCIENTIFIC REPORTS

# OPEN

Received: 1g May 2015   
Accepted: 14 July 2015   
Published: 14 October 2015

# Three-dimensional superresolution protein localization correlated with vitrified cellular context

Bei Liu1,2,\*, Yanhong Xue¹,\*,Wei Zhao¹,3,\*, Yan Chen¹,3, Chunyan Fan¹,3, Lusheng $\mathsf { G u } ^ { \mathsf { 1 } , \mathsf { 2 } }$ Yongdeng Zhang¹,2, Xiang Zhang¹, Lei Sun¹/4, Xiaojun Huang',4,Wei Ding¹4, Fei Sun1,3/4, Wei Ji134&Tao $\times ( 1 ^ { 2 } , 2 , 3 , 4$ （204号

We demonstrate the Use of cryogenic super-resolution correlative light and electron microscopy (csCLEM)to precisely determine the spatial relationship between proteinsand their native cellular structures.Several fluorescent proteins (FPs) were found to be photoswitchable and emitted far more photons under our cryogenic imaging condition,resulting in higher localization precision which is comparable to ambient super-resolution imaging.Vitrified specimens were prepared by high pressure freezing and cryo-sectioning to maintain a near-native state with better fluorescence preservation. A 2-3-fold improvement of resolution over the recent reports was achieved due to the photon budget performance of screening out Dronpa and optimized imaging conditions, even with thin sections which is at a disadvantage when calculate the structure resolution from label density. We extended csCLEM to mammalian cels by introducing cryo-sectioning and observed good correlation of a mitochondrial protein with the mitochondrial outer membrane at nanometer resolution in three dimensions.

A major goal in cell biology is to locate and directly visualize macromolecules in their native celular context, i.e., visual proteomics1.Although immuno-electron microscopy (EM) can be employed to label proteins, it suffers from low label density and poor preservation of cellular morphology. Recently developed super-resolution fluorescence microscopy (i.e., STED,(f)PALM/STORM, SIM) techniques have achieved nanometer resolution of fluorescence-labeled proteins2.However,super-resolution fluorescence microscopy cannot reveal the ultra-structural context.The integration of super-resolution fluorescence microscopy with EM may allow the correlation of fluorescently labeled individual proteins with the cellular ultrastructure at nanometer precision and holds great promise for meeting the technique demands of biological researches.

Indeed, STED and (f)PALM have been combined with EM to correlate protein localization and ultrastructural features3-5. Unfortunately, the chemical fixation and staining procedures during EM sample preparation tend to distort native cellular structures67 and quench fluorophores8,9. High pressure freezing followed by freeze substitution has been proved to maintain fluorescence,and meanwhile largely keep the celllar structural details5. However, this procedure needs to balance the contrast of EM imaging and fluorescence preserving during the staining process. Normaly, post staining after fluorescence imaging is

1National Laboratory of Biomacromolecules，Institute of Biophysics， Chinese Academy of Sciences，Beijing, 100101,China.²Colege of Life Science and Technology, Huazhong Universityof ScienceandTechnology, Wuhan, Hubei, $4 3 0 0 7 4 ,$ China.3Colege of Life Sciences, University of the Chinese Academy of Sciences, Beijing,100049, China.4Center for Biological Imaging，Institute of Biophysics, Chinese Academy of Sciences, Beijing, $\scriptstyle \mathtt { 1 0 0 1 0 1 } ,$ （204号 China.\*Theseauthors contributed equaly to this work.Correspondence and requests for materials should be addressed to W.J. (email: jiwei@moon.ibp.ac.cn) or T.X. (email: xutao@ibp.ac.cn)

required to enhance the contrast in EM,which may cause further distortion of the sample and affect the precision of correlation59.Recently,an improvement of the sample preparation procedure of high pressure freezing,freeze substitution and resin embedding was described to better preserve fluorescence and photo-switching of standard fluorescent proteins,such as mGFP, mVenus and mRuby210.This improved procedure enabled the correlation of fluorescently labeled structures to the ultrastructure in the same cell at higher precision and superior structural preservation.An alternative sample preparation method is vitrification,which preserves the structures in a near-native state in glaslike amorphous ice with no compromise of fluorescence preservingl,12. Another advantage of vitrification is the decreased photobleaching at cryo-temperatures13-15. Hence, there is now an emerging demand for fluorescence microscopy under cryo-conditions (cryo-FM) and its integration with other techniques including cryo- $\mathrm { \cdot ^ { E M ^ { 1 5 - 1 7 } } }$ ：

The limited sample thickness of cryo-EM and the low resolution of cryo-FM hamper the application of CLEM with vitrified samples. Cryo-electron microscopy of vitreous sections (CEMOVIS) is the most promising technique to visualize the three-dimensional (3D)architecture of hydrated cels and tissues which is too thick for cryo-EM18.19. Cryo-FM have been used to guide EM data acquisition with vitreous sections20.21. Several technical challenges limit the resolution of cryo-FM, such as low NA objectives,poor mechanical stability of the cryo-stage,and undesired performance of fluorophores under cryo-conditions.The adaptation of super-resolution methods to cryo-conditions is expected to overcome the resolution limit of cryo-FM.Indeed,two recent studies along this direction have extended cryo-FM to super-resolution imaging with a 3-5-fold improvement in lateral resolution2,23. Here,we further improve the resolution of cryo-FM to be comparable with ambient super-resolution imaging and extend csCLEM to 3D for mammalian cells by employing cryo-sectioning.

# Results

Cryo-nanoscopy setup.Our goal is to correlate genetically labeled proteins with subcelllar structures of mammalian cels at nanometer precision.Mammalian cels are generally too thick for EM. Therefore,we cryo-sectioned mammalian cells to slices of ${ \sim } 2 0 0 \mathrm { n m }$ thickness and employed CEMOVIS, which enables us to observe the internal architecture of cells or even subcellular organelles in their fully hydrated state18,19. We employed the strategy of single-molecule localization microscopy (SMLM, i.e., (f) PALM/STORM) for super-resolution imaging under cryo-conditions. The long imaging time required for SMLM necesitates long-term mechanical and thermal stability of the cryo-stage to hold the cryo-sections.To meet the above requirements, we built a cryo-nanoscopy system,as depicted in Fig.la. We used a special designed upright microscope coupled with a commercialized cryo-chamber (EM FC6,Leica) (Fig. la and Methods,Figures S1, S2). The upper part of the objective was wrapped with heating wires to keep the lens warm and free of frost. The sample holder was fixed on a copper block by magnetic butons.The cryo-chamber includes a self-pumping flled Dewar connected to the liquid nitrogen inlet and feedback systems to stabilize the temperature and nitrogen gas airflow. The sample holder was separated from the Dewar and immerged in cooled nitrogen gas, which greatly reduced vibrations caused by the flow and boiling of liquid nitrogen.We further introduced an active drift correction system to stabilize the position of the sample with nanometer precision $( \sim 5 \mathrm { n m } )$ during long-term data acquisition as described in Methods section (Fig. 1b-e).

Photophysical properties of FPs under cryogenic temperature. Another key requirement of cryo-SMLM is the ability offluorescent probes to be photoswitched.To find suitable photoswitchable FPs that function at cryogenic temperature (CT),we first investigated the temperature-dependent behavior of FPs including Dronpa24, mGeos-M, mEos3.2, PATagRFP and PAmCherry.We found that mEos3.2,an irreversible photoconvertor with the best performance at room temperature $( \mathrm { R T } ) ^ { 2 5 }$ ,required ${ \tt > } 1 0 0 \$ -fold more power to be efficiently photoconverted from green to red by a $4 0 5 \mathrm { n m }$ laser at CT (Methods, Figure S3).For the reversibly switchable fluorescent proteins,such as Dronpa, reversible photoswitching was maintained at CT (Fig. 2a). We also compared the ensemble photobleaching of Dronpa at RT and CT under the same laser power. Dronpa showed much greater anti-bleaching ability at CT than at RT (Fig. 2b). The temperature dependence of the fluorescence emission spectra of Dronpa was measured to guide the selection of suitable filter sets (see Methods section for experimental detail).The emission spectrum of Dronpa displayed a blue shift and narrowing band with decreasing temperature,a behavior also observed for GFP proteins26 (Fig.2c,d).EGFP also displayed reversible photoswitching behavior at CT, as previously reported for various GFP mutants23.27,28 (Methods, Figure S4).

We screened the photon budget performance of these photoswitchable FPs which are commonly used under our CT imaging conditions with purified fluorescent proteins (see Methods section). The setting temperature of cryo-chamber is $1 1 3 \mathrm { K }$ which was verified with a tiny $\mathtt { P t l 0 0 }$ temperature sensor placed around the sample. The real temperature may be slightly higher than the seting point due to laser illumination. We found that, among these candidates, Dronpa exhibited the best performance in terms of photon budget and localization precision for single-molecule imaging (Table i). Hence,we selected Dronpa for the following experiments.

Verifying resolving capability of cryo-nanoscopy system. We further tested the performance of Dronpa in an intracellular context at CT and verified the resolving capability of our cryo-nanoscopy system.We constructed a marker for the mitochondrial outer membrane by fusing Dronpa with the first 33 amino acids of TOM20 (see Methods section),which is responsible for localizing TOM20 to the mitochondria5.The proper localization of TOM20-Dronpa in HEK293 cels was verified using confocal microscopy (Methods,Figure S5).

![](images/5e5f2cbea6237ad3d0734d2ae052b737d7a202360598166e4c1bf14272be9b70.jpg)  
Figure1.Schematic drawingand drift correction capability of the system.(a) Schematic drawing of the cryo-nanoscopy system.A dark red fluorescent bead was monitored for $5 ~ \mathrm { m i n }$ without active fedback (b) and with active feedback (c).Lateral X(d) and Y(e) position distribution of allpoints in (c).Data show that the active feedback stabilization system could effectively suppressthe drift.Figure la was drawn by Wei Ji.

![](images/e994d981876af6e6e2ff8955c2b9042146686bf5069d1dc20417768987fca360.jpg)  
Figure 2. Photophysical properties of Dronpa at CT.(a) Photoswitching capability of Dronpa at CT (113 K).Purified Dronpa proteins were first switched off by a $4 8 8 \mathrm { n m }$ laser, followed by alternative illumination with $4 0 5 \mathrm { n m }$ and $4 8 8 \mathrm { n m }$ lasers. Each purple arrow indicates a $5 s 4 0 5 \mathrm { n m }$ laser pulse. (b) Fluorescence decay kinetics of Dronpa atRT and CT under the irradiation of a $1 . 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ $4 8 8 \mathrm { n m }$ laser. The decaycurves at RT and CT were both fitted with a bi-exponential function27,28.The time constants/amplitudes at RT are $0 . 0 6 s / 0 . 8 7$ (fast phase)and $2 . 3 7 \mathrm { s } / 0 . 0 4$ (slow phase),and the corresponding values at CT are $0 . 8 7 ~ s / 0 . 3 7$ (fast phase) and $8 . 3 2 ~ \mathrm { s } / 0 . 3 3$ (slow phase).Please note that at RT, the amplitude of the slow phase is very small $( 4 \% )$ ,indicating fast bleachingand off-switching mainly in the fast phase $( \mathbf { c , d } )$ .(c) shows the evolution of the emission spectra of Dronpa subjected to a temperature gradient from $2 9 3 \mathrm { K }$ to $1 4 3 \mathrm { K } ,$ (d) is the normalized presentation,and the inset is the magnified view of the red area in (d).

Table1．Single molecule statistics of selected FPs under cryogenic conditions.Fluorescent protein samples were prepared as described in Methods.   

<html><body><table><tr><td rowspan="2">Fluorescent Proteins</td><td colspan="2">Total Photon Number</td><td colspan="2">Localization Error (nm)</td></tr><tr><td>Mean</td><td>Median</td><td>Mean</td><td>Median</td></tr><tr><td>EGFP</td><td>895±165</td><td>584±108</td><td>14.7±0.5</td><td>13.7±0.6</td></tr><tr><td>mEos3.2 (green)</td><td>843±227</td><td>312±153</td><td>14.0 ± 1.1</td><td>13.0±0.6</td></tr><tr><td>mEos3.2 (red)</td><td>1334±502</td><td>614±370</td><td>13.5±4.6</td><td>12.1 ± 3.1</td></tr><tr><td>PATagRFP</td><td>621±60</td><td>260±18</td><td>17.8±0.3</td><td>18.3±0.1</td></tr><tr><td>PAmCherry</td><td>1026± 24</td><td>454±18</td><td>15.0±0.1</td><td>14.5± 0.1</td></tr><tr><td>Dronpa</td><td>2203±316</td><td>1576± 227</td><td>10.1± 1.2</td><td>8.9±1.4</td></tr><tr><td>mGeos-M</td><td>1814± 164</td><td>954±35</td><td>10.6± 2.2</td><td>9.0±1.7</td></tr></table></body></html>

After high-pressure freezing and cryo-sectioning (experimental procedurals described in Methods section),we collected ${ \sim } 2 0 0 \mathrm { n m }$ thick cryo-sections of HEK293 cells expressing TOM20-Dronpa (Fig.3a) on TEM finder grids and imaged them using our cryo-nanoscopy. The challnge is to maintain the vitrified state of the specimens during super-resolution fluorescence imaging. The fluorescence collection efficiency of our cryo-nanoscopy was lower than standard super-resolution microscopy at RT due to the use of an air objective $\left( \mathrm { N A } \ 0 . 8 \right)$ ，so we increased the excitation laser power to achieve sufficiently bright single-molecule signals.However, higher laser power tends to heatthe cryo-sections and devitrify the sample,as shown by the appearance of ice crystals. To reduce the laser absorption-induced heat, we replaced the high absorbance carbon film with a low background fluorescence formvar film (see Methods section).Due to low absorbance of the formvar film, we could increase the power of the $4 8 8 \mathrm { n m }$ laser up to $1 . 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ at the specimen without devitrifying the samples (Methods,Figure S6). The reconstructed super-resolution images from $\sim 2 0 { , } 0 0 0$ frames of single-molecule data clearly revealed ring shapes representative of the outer membrane of mitochondria (Fig.3b).To obtain 3D single-molecule localization information, we introduced acylindrical lens as previously reported29(Methods,Figure S7). The single molecules were distributed evenly along the $\mathbf { z }$ -axis with a full width at half maximum (FWHM) of $2 4 0 \mathrm { n m }$ ，corresponding nicely to the thickness $( \sim 2 0 0 \mathrm { n m } )$ of the cryo-sections plus the uncertainty caused by limited photon numbers.We divided the total single-molecule data into $8 \mathrm { ~ z ~ }$ -sections of ${ \sim } 3 0 \mathrm { n m }$ thickness.For a single z-section in the middle,we estimated an average localization precision $s$ of ${ \sim } 7 . 4 \mathrm { n m }$ in the lateral direction (Fig. 3c). The $\textbf { z }$ -axis resolution was estimated to be ${ \sim } 4 0 \mathrm { n m }$ by analyzing the z distribution of long-lasting single molecules29. Further resolution improvement can be achieved by using beter-localized fluorophores and applying an acceptable precision cutoff, as recently demonstrated30. The apparent width of a line profile of TOM20-Dronpa along the mitochondrial outer membrane could reach σ ${ \sim } 1 3 \mathrm { n m }$ for a given $\textbf { z }$ -section (Fig.3d),which represents the summed consequence of limited photon count, residual mechanical drift,and labeling uncertainty of the TOM20-Dronpa protein. The label density was estimated to be $1 8 5 0 / 7 4 4 \mu \mathrm { m } ^ { - 2 }$ (mean/median) (Fig. 3e),leading to a Nyquist resolution of ${ \sim } 4 6 / 7 4 \mathrm { n m }$ (Fig. 3f),which is a 2-3-fold improvement over the recent reports22,23.

![](images/7ebf8e313418fe433a9270aa6b74d3b0164445022b8cb8b9f0a6e00bd7d5b3c4.jpg)  
Figure 3. Verification of cryo-nanoscopy's resolving capability. (a,b) Conventional fluorescence image (a) and PALM image (b) of a $2 0 0 \mathrm { n m }$ cryo-section of a TOM20-Dronpa-labeled HEK293 cell. Scale bar, $1 \mu \mathrm { m }$ (c) Distribution of the localization error in the middle layer of $\mathbf { z }$ -sections obtained from 3D data in (b).(d) Line profile of the position marked in ${ \bf ( b ) }$ (red).The middle layer was used to generate the line profile.(e) Distribution of label density in (b), $1 8 5 0 / 7 4 4 \mu \mathrm { m } ^ { - 2 }$ (mean/median). (f) Distribution of Nyquist resolution calculated from the label density in (e), $4 6 / 7 4 \mathrm { n m }$ (mean/median).

$3 D$ Protein localization correlated with vitrified cellular context. Next,we correlated 3D fluorescence super-resolution imaging with 3D electron micrographs.We verified that the electron beam could bleach the FPs even at CT,so the correct order is fluorescence imaging first, followed by cryo-electron tomography (cryo-ET). After cryo-fluorescence super-resolution imaging, we transferred the specimens to a commercial cryo-TEM (Titan Krios 300KV, FEI). We used the EM finder grids to locate the cels on which we had performed fluorescence imaging and collected EM data (Fig. 4a,b, Methods). Because formvar film is less electron conductive than carbon films,charge-induced movement and radiation damage of the electron beam may become issues of concern.We thus used low dose radiation and selected areas near the grid bar for tilted imaging to facilitate charge conducting. No clear charge-induced movement or radiation damage was observed under our experimental condition. The fine alignment between the fluorescent and electron images was achieved by three steps involving different fiducial markers,and the final registration error could be controlled at ${ \sim } 1 2 \mathrm { n m }$ (Figure S8), and the comprehensive process was described in Methods section. As shown in Fig.4c,a section of the mitochondrial outer membrane and cristae was clearly visualized by 3D reconstruction from cryo-ET (Methods,Movie S1),while super-resolution imaging of Tom20-Dronpa molecules showed correlated localization to the outer membrane at nanometer resolution (Fig. 4c,d).

# Discussion

Pioneering work has been conducted to correlate protein localization with ultrastructural features revealed by $\mathrm { E M } ^ { 4 , 5 , 1 \bar { 9 } , 2 2 }$ .Compared to previous methods, our csCLEM method has the following advantages. 1) It avoids chemical fixation and contrast enhancement procedures that may cause artifacts in EM or damage the $\mathrm { F P s } ^ { 5 }$ . 2） The cryo-condition not only preserves the specimen to near-native state but also enhances the photon budget due to reduced photobleaching. By using low fluorescence background formvar film instead of high absorbance carbon film,we were able to increase the power of the excitation laser and thus obtain ten times more photons (Table 1） than recently reported under cryogenic conditions22. The higher power of the activation laser also helps to identify a larger number of cryo-photoswitchable FPs.3)Nanometer resolution is obtained in 3D for both cryo-SMLM and cryo-ET.Two identified red PA-FPs candidates,PATagRFP and PAmCherry, may also be combined with Dronpa or EGFP to achieve dual-color cryogenic superresolution.4) The use of CEMOVIS enables the study of thick biological samples and offers very low out-of-focus background for super-resolution fluorescence imaging. In summary, we have extended CLEM to vitrified sections of mammalian cells at higher localization precision and within near-native cellular context. Our csCLEM method harnesses the super-resolution power of both SMLM and cryo-ET. The enhancement of axial resolution by introducing axial interferometry and the development of new dedicated fluorophores under CT could further extend the application of this approach. We envision that csCLEM willfacilitate quantitative deciphering of functional protein organization and the relative spatial distribution to native cell architecture at unprecedented 3D resolution.

# Methods

Protein expression and purification. The FPs were cloned into a pRSET vector using the appropriate restriction sites and expressed in the Escherichia coli strain BL21 (DE3).The bacteria were cultured at $3 1 0 \mathrm { K }$ to $\mathrm { O D } { = } 0 . 6$ The expression of proteins was induced by adding $0 . 1 ~ \mathrm { m M }$ IPTG and incubating overnight at $2 9 5 \mathrm { K }$ 、The proteins were purified with a Ni-NTA His-Bind resin (Qiagen).Then,further purification of the proteins was performed through a gel-filtration step using a Superdex 2Oo column (GE Healthcare).Purified proteins were concentrated by ultrafiltration and diluted in PBS for additional analysis.

Spectra measurement. The measurement of spectra of Dronpa at different temperatures was performed using a spectrometer (FLS920,Edinburgh Instrument） equipped with a liquid nitrogen cryostat (OptistatDN-V2, Oxford Instruments).The slow-freezing of the samples from $2 9 3 \mathrm { K }$ to $1 4 3 \mathrm { K }$ was achieved by meticulous control of the liquid nitrogen flow. Purified Dronpa was diluted to ${ \sim } 0 . 0 2 \mathrm { m M }$ This concentration was necessary to avoid impurity peaks.Moreover, glycerol $( 6 0 \% )$ was used as a cryo-protectant to prevent crystalline ice formation. The evolution of the emission spectra of Dronpa subjected to a temperature gradient with excitation wavelength at $4 4 5 \mathrm { n m }$ was observed (Fig. $^ { 2 \mathrm { c } , \mathrm { d } } ,$ ）

![](images/8e8298662b3f33522fc674a035b56d426b025fdf360670fff71c0f60ef6095b0.jpg)  
Figure 4.3D correlative images from cryo-sections of HEK293 cells expressing TOM20-Dronpa. (a) Summed image from single-molecule data. Scale bar, $1 \mu \mathrm { m }$ .(b) Correlative isosurface reconstruction of a PALM image with low magnification TEM data.(c) Each column in c was from a single layer of 3D PALM (top),single layer of cryo-ET (middle)and correlative representations (bottom),respectively. The structural resolution of the whole 3D data set was estimated to be $7 3 . 7 \pm 5 \mathrm { n m }$ employing the Fourier ring correlation method35,36, corresponding to a density of 749 signals. $/ \mu \mathrm { m } ^ { 2 }$ .Scale bar, $2 0 0 \mathrm { n m }$ (d) Correlative 3D segmentation of cryo-ET data with single-molecule localization of fluorescent proteins.Mitochondrial outer membrane and cristae are denoted by purple and blue,respectively.Dronpa molecules are denoted by green dots.For clear demonstration,we kept those signals that have higher localization precision $( 2 . 9 / 3 . 0 \mathrm { n m }$ ， mean/median),which rejects $9 0 \%$ of the signals. Scale bar, $2 0 0 \mathrm { n m }$

Plasmid construction. The Dronpa gene was cloned and inserted into the pEGFP-N1 (Clontech) vector using BamHI and NotI to replace the EGFP gene,which generated pDronpa-N1. The first 33 amino acids of TOM20 (Homo sapiens,NP_055580) were PCR-amplified and inserted into pDronpa-N1. The correct localization of TOM20-Dronpa in HEK293 cells was verified using confocal microscopy (Methods,Figure S5)

Cell culture.HEK293 cells were cultured in Dulbecco's Modifed Eagle Medium (DMEM)(Gibco) containing $1 0 \%$ fetal bovine serum and maintained at $3 1 0 \mathrm { K }$ and $5 \% \mathrm { C O } _ { 2 }$ ina humidified incubator. Cells were then transiently transfected using LipofectamineTM 2ooo (Invitrogen） for $2 4 \mathrm { h }$ =

Cryo-nanoscopy setup. The cryo-nanoscopy system (Fig. la, Methods,Figure S1） consists of a cryo-chamber and a home-built upright microscopy (Figure S2a). We used a commercial cryo-chamber from Leica (Leica EM FC6) which is widely used for ultrathin cryo-sectioning of biological samples and has been shown to keep samples vitrified. The cryo-chamber uses cryogenic nitrogen gas to build up a stable cryogenic environment surrounding the sample stage and objective (Olympus , $1 0 0 \mathrm { x }$ NA 0.8). Inside the cryo-chamber,a copper block (made of oxygen free copper) (Methods,Figure S2b),which is connected with the piezo stage by two glass rods, is used to support a copper sample holder (Methods, Figure S2c).The sample holder (made of oxygen free copper) has been designed to adopt four EM finder grids and is kept in place on the copper block by four small magnets integrated inside both parts.The glass rods is connected to a 3-axies piezo stage placed outside of the cryo-chamber. The piezo stage is attached to a XY stage.The goal of the design is to reduce the heat conduction from the cryo-chamber to sample stages and to provide high mechanical stability by avoiding the vibration of cryo-chamber due to liquid nitrogen pump. A heating wire is attached to a copper tube connecting the objective to prevent water condensation at the objective.For fluorescence activation and excitation,we used a $4 0 5 \mathrm { n m }$ diode laser(OBIS,Coherent)and two solid-state lasers (OPSL,Coherent） with wavelengths of $4 8 8 \mathrm { n m }$ and 56l nm coupled into a single-mode polarization-maintaining optical fiber. The laser beams were combined using DM1 and DM2 (LM01-427-25,LM01-503-25,SEMROCK) and delivered into the objective with DM3 $( \mathrm { D i } 0 1 { - } \mathrm { R } 4 0 5 / 4 8 8 / 5 6 1 / 6 3 5 { - } 2 5 \times 3 6 ,$ SEMROCK),which reflected all laser beams but allowed the transmission of emitted fluorescence.An AOTF (AOTFnC-400.650,AA) was used for flexible control of the wavelength and intensity of the lasers delivered by the fber.

Two EMCCDs (iXon DV-897 BV,Andor） were used in this implementation to ensure that the super-resolution imaging system (EMCCD1) and the fedback stabilization system (EMCCD2) remained independent of each other during data acquisition.A green $( 5 3 2 \mathrm { n m } )$ light-emitting diode (LED） facilitated brightfield imaging. The fluorescence of the dark red beads ( $2 0 0 \mathrm { n m }$ ,Invitrogen） excited by the $4 8 8 \mathrm { n m }$ laser was split by DM4 (DM690, Olympus), which reflected parts of the bead signal to EMCCD2 and allowed transmission of the leftover bead signal and the fluorescence of Dronpa.The image of the dark red beads and Dronpa on EMCCD1 was split by DM5 $( \mathrm { F F } 5 6 0  – \mathrm { D i } 0 1 – 2 5 \times 3 6 ,$ ，SEMROCK) and cleaned by EM2 and EM3 (FF03-525/50-25,FF01-697/58-25),then projected onto two regions of EMCCD1. The bead image on EMCCD1 was used for correlation with TEM images,and the bead image on EMCCD2 was used for drift correction.The major advantage of the two-EMCCD system is the flexibility of optimizing the feedback stabilization system and fluorescence imaging system separately. Two cylindrical lenses were placed before EMCCD1 and EMCCD2 to enable imaging in 3D. A feedback loop that tracked the image of the dark red beads on EMCCD2 in real time actuated a piezo stage (P-733.3 DD,E-516 controller,Physik Instrument) and locked the sample at a fixed set point during data acquisition.Two computers were used to control EMCCD1 and EMCCD2 independently.A fast synchronization unit was designed to communicate with PC2 and the piezo stage controler to synchronize imaging with piezo stage movement for fast fedback.The fast feedback stabilizes the position of the sample with $\sim 5 \mathrm { n m }$ precision during data acquisition (Fig. lb-e).

Cryo-nanoscopy imaging and data analysis. EM finder grids with cryo-sections were first transferred to a sample holder that can support up to 4 grids in liquid nitrogen.Then,the sample was transferred into the cryo-chamber and mounted onto a copper block by four tiny magnetic buttons. A tiny $\mathrm { P t l } 0 0$ temperature sensor was inserted into the copper block around the samples to monitor the temperature. The cryo-chamber was usually pre-cooled to $1 1 3 \mathrm { K }$ for more than $3 0 \mathrm { m i n }$ before specimens were loaded.We first identified regions with fluorescence under low-power ilumination with a $4 8 8 \mathrm { n m }$ laser and further verified the regions of interest (ROIs)by the bright-field image on EMCCD1.Then, we ensured the presence of fiducial markers (fluorescent beads) on the red channel of EMCCDl and EMCCD2 and locked the specimens at set positions through our feedback stabilization system. Next, we gradually increased the $4 8 8 \mathrm { n m }$ laser power to $1 . 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ at the specimen to switch the ensemble Dronpa fluorescence to the single-molecule level. The $1 . 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ power was chosen as a compromise of imaging speed and avoiding devitrification of the specimen. Finally, the single-molecule images were obtained at $\bar { 2 } 0 \mathrm { H z }$ The activation laser pulses ranged from $0 . 0 0 5 { - } 0 . 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ at the specimen during data recording with a $4 \mathrm { H z }$ repeat rate and 0.01s width. To avoid build-up of heat, we divided our imaging process into several sessions.Each session lasts for $5 \mathrm { { m i n } }$ or less,with 10-20 seconds of breaks in between with no laser illumination.Inside the cryo-chamber, samples are surrounded by continuous replenished, gently flowing cryogenic nitrogen gas. The heat caused by laser-sample interaction would be quickly conducted during the breaks.At the end of the experiments,we have checked our sample and verified that there is no obvious crystalline ice formation. One example with $1 7 \mathrm { m i n }$ total laser illumination is presented in Movie S1.

Data analysis and super-resolution image reconstruction were performed as previously described with some modifications3,25,3iWe first appliedanatrous wavelet filter2 to the original fluorescence images to extract all the single molecules.Then,a trajectory linking algorithm (http://physics.georgetown.edu/ matlab/) was used to find the corresponding single molecule in successive frames.Each identified molecule was background-subtracted and fit to an ellptical Gaussian function to obtain its precise centroid position and Gaussian width ( $\mathbf { \dot { W } _ { X } }$ and ${ \bf w } _ { \mathrm { y } } ^ { \mathrm { } } ,$ ).Based on the calibration curve(Methods,Figure S7),the molecules whose ellpticity was too large or too small were rejected. The remaining molecules were divided linearly into $8 \mathrm { ~ z ~ }$ -sections on the basis of $\mathbf { z }$ -positions.Meanwhile, sample drift during image acquisition in all three dimensions was further estimated by analyzing the trace of fiducial markers which are imaged on red channel of EMCCD1.Finaly, every drift-corrected molecule was represented by a Gaussian spot whose FWHM was determined by the localization error of each molecule,and the final PALM image wasrendered.

To analyze the properties of FPs under CT,purified FPs were first diluted to ${ \sim } 1 \mathrm { m g / m l }$ and fas frozen under a Vitrobot Mark IV (FEI). The excitation laser power was maintained at ${ \sim } 2 . 0 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ Single-molecule data were recorded at CT and analyzed as previously described33.

To analyze the behaviors of fluorescent proteins under CT, we collcted fluorescence images under diferent ilumination protocols (Fig. 2a,b,Methods and Figures S3,S4).We developed software for the control of AOTF and the generation of sophisticated time sequences of different lasers (https://github. com/shepherd87/pyAOTF).

Preparation of support film. The traditional support film on an EM grid has a high background fluorescence that afects the performance of fluorescence nanoscopy. We found that most of the background fluorescence was generated by contaminants on the surface of the glass slides that were used to generate the thin formvar film.We applied a chemical cleaning method to wash the glass slides.Briefly, glass slides were washed in a mixture of $2 5 0 \mathrm { m l }$ of clean water, $5 0 \mathrm { m l }$ of ammonium hydroxide,and $5 0 \mathrm { m l }$ of hydrogen peroxide in a $5 0 0 \mathrm { m l }$ beaker for $3 \mathrm { h }$ at $3 8 3 \mathrm { K }$ ，After rinsing 3 times in $\begin{array} { r } { \mathrm { d d H H } _ { 2 } \mathrm { O } , } \end{array}$ the glass slides were carefully transferred to a beaker containing anhydrous ethyl alcohol overnight and dried ona clean bench.A cleaned glass slide was dipped into a $0 . 2 \%$ formvar/chloroform solution in a volumetric beaker and dried above the liquid surface.After scoring the edge of the slides,the flm was floated onto the water surface,and 200 mesh EM-finder copper grids (Gilder Grids,England) were placed on the film and dried on filter paper.

For fine correlation between light and EM,we deposited $2 0 0 \mathrm { n m }$ fluorescent beads and $5 0 \mathrm { n m }$ gold particles on the surface of the formvar film before applying to vitreous cryo-sections.Briefly, $3 \mu \mathrm { l }$ of 1:300 diluted dark red $2 0 0 \mathrm { n m }$ fluorescent beads was added to a glow-discharged grid for $1 \mathrm { m i n }$ ,and extra liquid was then blotted using filter paper.After the grid was dried,another $3 \mu \mathrm { l }$ of a $\mathrm { d d H } _ { 2 } \mathrm { O }$ -diluted solution of $5 0 \mathrm { n m }$ gold particles (Microspheres-Nanospheres,790116-010） was applied onto the grid for $1 \mathrm { m i n }$ ， and extra liquid was blotted using filter paper. The grids were then dried and stored until use.

High-pressure freezing.Transfected HEK293 cels were harvested enzymatically by trypsin and centrifugation at $6 0 0 { \mathrm { g } }$ ，then re-suspended in an equal volume mixture of $4 0 \%$ w/v dextran $\left( \mathrm { M _ { r } } { \sim } 4 0 , 0 0 0 \right.$ Sigma) and 1:2 diluted dark red $2 0 0 \mathrm { n m }$ fluorescent beads. Finally, the samples were vitrified using a high-pressure freezing apparatus (EMPACT2,Leica Microsystems, Germany) as described previously18

Cryo-electron microscopy of vitreous specimens. Cryo-EM of vitreous specimens has been thoroughly described18 and has been correlated with conventional fluorescence imaging20,21. In the current study, the vitrified sample was mounted in the holder of an ultramicrotome (UC6/FC6, Leica Microsystems,Germany) at $1 1 3 \mathrm { \bar K }$ 、After trimming the sides of the copper tube with a cryo $4 5 ^ { \circ }$ diamond trimming knife (Diatome,Biel, Switzerland),the sample was sectioned using a cryo $2 5 ^ { \circ }$ diamond knife (Diatome,Biel, Switzerland) with a nominal thickness of $2 0 0 \mathrm { n m }$ .Ribbons of sections were transferred onto the above-prepared grid using an eyelash glued to a wooden stick,and the sections were pressed and then atached to the formvar film using two polished ceramic plates. The grids with cryo-sections were stored in liquid nitrogen prior to observation.

After fluorescence imaging, TEM sample grids containing vitreous cryo-sections of HEK293 cells were transferred and imaged by a cryo-electron microscope (FEI Titan Krios) operated at $3 0 0 ~ \mathrm { k V }$ and equipped with a $4 \mathrm { K ^ { * } 4 K }$ CCD camera (Gatan UltraScan 40oo).For cryo-ET data collection,a defocus of approximately $- 1 5 \mu \mathrm { m }$ was used with a total electron dose $< 7 0 0 0 { \mathrm { e / n m } } ^ { 2 }$ ：

Single-axis tilt series were collected covering the angular range of $- 6 0 ^ { \circ } \sim + 6 0 ^ { \circ }$ in $2 ^ { \circ }$ increments with a nominal magnification of $1 8 0 0 0 \times$ . Tilt series were aligned,and tomograms were generated using the Inspect3D software (FEI).During reconstruction,raw images were binned once,resulting in a final voxel size of $1 . 9 2 \mathrm { n m }$ ：

To confirm that the cryo-sections remained vitreous after fluorescence microscopy imaging, we performed an illumination-dependent devitrification assay verified by electron diffraction (Methods,Figure S6).For this purpose,the sections were loaded onto a Gatan 626 cryo-holder maintained at $9 3 \mathrm { ~ K ~ }$ and then transferred to a TEM(FEI Talos F20oC) operated at $2 0 0 \mathrm { k V }$ and equipped with a field emission gun and a $4 \mathrm { K ^ { * } 4 K }$ Ceta camera. TEM images were obtained under a low-dose condition at a dose rate of $\sim 5 0 0 \mathrm { { e / n m ^ { 2 } } }$ . Then,diffraction images were recorded with a selective area aperture of $2 0 0 \mu \mathrm { m } .$ camera length of $5 3 0 \mathrm { m m }$ and exposure time of 1s.We found that the sample remained mostly vitrified when the illumination intensity was up to $1 . 7 5 \mathrm { k W } { \cdot } \mathrm { c m } ^ { - 2 }$ at the specimen.When the laser intensity was increased to $2 . 0 \mathrm { k W } \cdot \mathrm { c m } ^ { - 2 }$ ,we began to observe devitrification,with clear diffraction spots from smallice crystals.

Employing formvar film instead of carbon film may lead to charging and beam-induced specimen movement.We used the folowing methods to minimize this problem.(1) We tried to image samples near the grid bar where cryo-sections attached better than at the center of the hole.Moreover,cryo-sections near the grid bar have beter conductivity than the ones at the center of the hole.(2)Brightfield light microscopy images and fluorescence images helped us to locate the same regions with high precision, which significantly reduced the searching time under EM and thereby minimized the charging and radiation damage of the specimen. (3) Low dose radiation was adopted during EM data collection.

Alignment of cryo-SMLM with EM. Currently,two methods have been introduced to correlate cryo-SMLM images with EM images.The first method utilizes fiducial markers,whereas the second method uses the image details from both light microscopy and EM.

The first method is usually applied for the alignment of 2D cryo-SMLM images with a single TEM projection image.We used electron dense fluorescent beads,which were visible under light microscopy (LM) and EM,as well as gold particles (Methods,Figure S8). The entire registration progress involves three steps: registration between two fluorescent channels,registration between dark red fluorescent signals and low-magnification electron micrographs,and registration between low-magnification and high-magnification micrographs. Each time before cooling the cryo-SMLM system,we deposited diluted four-color fluorescent beads onto a cover glass and used it for the registration between the single-molecule channel and the dark red channel. This type of registration error is denoted by $\mathfrak { E } _ { \mathrm { l } }$ .During correlative imaging,we used electron dense fluorescent beads to map the dark red fluorescent channel to low-magnification EM micrographs.This type of registration error is denoted by $\mathfrak { E } _ { 1 - \mathrm { e } } .$ （ Finally, we used fluorescent beads and gold particles to map low-magnification electron micrographs to high-magnification electron micrographs,and this type of registration error is denoted by $\mathfrak { E } _ { \mathrm { e } . }$ The final registration error is calculated as follows:

$$
\mathfrak { e } _ { \mathrm { t } } = \sqrt { \mathfrak { e } _ { 1 } { } ^ { 2 } + \mathfrak { e } _ { 1 - \mathrm { e } } { } ^ { 2 } + \mathfrak { e } _ { \mathrm { e } } { } ^ { 2 } } < 1 2 \ \mathrm { n m }
$$

For the alignment of 3D cryo-SMLM images with electron tomography, we developed a simple but efficient registration method.Briefly,we first extracted the outer membrane of the mitochondrion of interest from EM images using segmented line tools in ImageJ (denoted as a closed curve). Then, we calculated the distance between each single molecule using the extracted outer membrane (denoted by $D i$ 0 $1 \leq i \leq N _ { : }$ where $\mathbf { N }$ is the total number of single molecules).We shifted $( \Delta \mathbf { x } , \Delta \mathbf { y } )$ and rotated(0) the extracted line to find the best parameters that minimize the sum of all distances $( \Sigma D i )$ . The description above can be submitted to the following optimization problem:

$$
{ \mathrm { M i n i m i z e } } \colon { \mathrm { f } } ( \Delta \mathbf { x } , \ \Delta \mathbf { y } , \ { \mathrm { \Theta } } { \mathrm { \Theta } } ) \ = \sum { D i \ } ( \Delta \mathbf { x } , \ \Delta \mathbf { y } , { \mathrm { \Theta } } { \mathrm { ) } } , \ 1 \leq i \leq N .
$$

The molecules that were too far from the extracted outer membrane were excluded from calculation. For the $\textbf { z }$ -axis,we first calculated the histogram of the $\textbf { z }$ -position of all the single molecules,which followed a Gaussian distribution.Then, we used the peak position and FWHMof the Gaussian distribution to indicate the middle layer and the thickness of the vitreous sections,respectively. The $\textbf { z }$ -distribution of single molecules was $2 4 0 \mathrm { n m }$ ，which was comparable with the thickness calculated from the electron tomogram of $2 2 0 \mathrm { n m }$

In practice,we randomly selected $2 / 3$ of all detected single molecules to perform the registration procedure.Then, we calculated the distances between the remaining molecules and the extracted outer mitochondrial membrane.The histogram of these distances shows that the average distance between Dronpa molecules and the outer mitochondrial membrane is approximately $8 . 6 \mathrm { n m }$ ,which is comparable to the immune-gold staining results34.

# References

1.Nickel,.KferCeis,.&mestrWsuaproachroteicsNatevol.Celo-(). 2.Lippincot-chwartz,J.&ManleyS.Putingsuper-resolutionfluorescencemicroscopytoork.Nat.Methods6,2-2(009). 3.Betzig,E.etal.Imaging intracellularfluorescent proteins at nanometer resolution.Science 313,1642-l645 (2006). 4.Sochacki,K.A.,tengel,GEgeleburg,,,H.&raska,JWCrelaiveperresoltionfuorce metal-replica transmission electron microscopy. Nat.Methods 11,305-308 (2014). 5.Watanabe,S.etal.Proteinlocalizationinelectron micrographs usingfluorescence nanoscopy.Nat.Methods 8,80-84(2011). 6.Bleck,C.etal.Comparisonof diferent methodsforthin section EManalysisof Mycobacteriumsmegmatis.J.Microsc.237, 23-38 (2010). 7.Weinhausen,B.etal.ScaningX-raynanodifractionolivingeukaryoticcelsinmicrofldicenvironments.hysRevet, 088102 (2014). 8.Pedie,C.J.etal.Corelativeand integrated lightandelectronmicrosopyof in-resinGFPfluorescenceusedtolocalise diacylglycerol in mammalian cells.Ultramicroscopy 143,3-14 (2014). 9.Perkovic,M.etal.Correlative Light-and Electron Microscopywithchemical tags.J.Struct.Biol.186,205-213(2014). 10.Johnson,E.etal.Corelativeiresinsuper-resolutionandelectronmicrosopyusingstandardfluorescent proteins.SciRep.5 (2015). 11.Hurbain,I&chse,M.Thefutureisold:cryo-preparationmethodsfortransmissioneectronmicrosopyofcels.ol.Cl 103,405-420 (2011). 12.Schneide,utma,Pebe,S.We.&Fllth,.CroX-racosoewiapleeometryfoaie fluorescence and nanoscale tomographic imaging. J. Struct. Biol.177,212-223 (2012). 13.Le Gros,,eot,G,Uida,,ohel,C.&rbellCigaertrecgenicligtcrooyJcoc. 1-8 (2009). 14.Moerner,W.& Orrit,M.Illuminating single molecules in condensed mater.Science 283,1670-1676 (1999). 15.Schwartz,C.L,arbash,V.AtaullkhanovFcintosh,J..&icastroD.Cryo-fluorescencemicrocopyciliate correlationsbetweenlightandryo-electron microscopyandreduces therateofphotobleaching.J.Micros.27,98-109 (200). 16.Agronskia,A.V.etal.IntgratedforescenceadtrasmsionelectroncroopyJ.Struct.Biol.64,3-9(08). 17.Sartori,A.etal.Corelativemicroscopy:bdgingthegapbetweenfluorescencelightmicroscopyadcyo-electrontoography J. Struct.Biol.160,135-145 (2007). 18.Al-Amoudi,A.et al. Cryo-electron microscopy of vitreous sections.EMBO J. 23,3583-3588 (2004).

19.Hsieh,C.ELeith,A,Manela,C..Frank,J&MarkoM.Towardsig-resolutionthree-dimensionalimagingfntie mammalian tissue: electron tomography of frozen-hydrated rat liver sections.J. Struct. Biol.153,1-13 (2006).   
20.Briegel,A.etal.Chapterirteen-CorelatedLightandElectron Cryo-Microscopy.MethodEnzymol481,17-34(2010).   
21.Gruska,M.,Medalia,O.,Baumeister,W.&Leis,A.Electrontomographyofvitreoussectionsfromculturedmammalianells.J. Struct. Biol.161,384-392 (2008).   
22.Chang,Y-W.etal.Corelatedcryogenicphotoactivatedlocalizationmicrosopyandcryo-electrontomography.Natetods 11,737-739 (2014).   
23.Kaufmann,R.talSuper-resolutionmicroscopyusingstandarduorescentproteinsinintactcellsundercry-conditios.Nan Lett 14,4171-4175 (2014).   
24.Habuchi,S.etal.Reversible single-molecule photoswitchingintheGFP-likefluorescentprotei Dronpa.Proc.Natl.Acd.Sci. USA 102,9511-9516 (2005).   
25.Zhag,M.etal.Rationaldesignoftrue monomericandbrightphotoactivatablefluorescentproteins.Nat.Methods9,727-729 (2012).   
26.Leiderman,P,Huppert,D.&Agmon,N.Transitioninthetemperature-dependencefGFPfuorescence:fromprotoniresto proton exit.Biophys.J.90,1009-1018 (2006).   
27.Adam,V.etal.tructuralbasisofX-rayducedtrasietpotobleachinginapotoactivatablegrenfuorescentproteinAm Chem Soc 131,18063-18065 (2009).   
28.Creemers,T,ock,A.ubramana,V,Jovin,T.&Vkerotopisdoicalsitchngingreenforestoten mutants.Proc.Natl.Acad.Sci.USA 97,2974-2978 (2000).   
29.Huang,B.,Wang,W,Bates,M.&Zhuang,X.Tree-dimensionalsuper-resolutionimaging bystochasticopticalreconstruction microscopy. Science 319,810-813 (2008).   
30.Pertsinidis,A.etal.Ultraigh-resolutionimagingreveals formationofneuronal SNARE/Munc18complexes insitu.Proc.Natl. Acad.Sci.USA 110,E2812-E2820,doi: 10.1073/pnas.1310654110 (2013).   
31.Chang,H.etal.Auniqueseriesofreversiblyswitchablefuorescentproteinswithbeneficialpropertiesforvariousapplications. Proc.Natl. Acad. Sci. USA 109,4455-4460 (2012).   
32.Olivo-Marin,J.-C.Extractionofspotsinbiological images usingmultiscale products.Patern Recogn.35,1989-6 (2002).   
33.Annibale,,VanniSarsell,,othlisberger,U.&Radenovic,A.Quantitativepotoactivatedlocalzatincoo: unraveling the effects of photoblinking.PLoS ONE 6,e22678 (2011).   
34.Vogel,F,orovd,CNeupert,W.&Reichert,A..Dyamicsubcompartmentalzationoftheitochondrialinrebrane. J Cell Biol.175,237-247 (2006).   
35.Banterle,,BuiK.H,Lmke,E.A.&ckM.Furiegcorrelatiossolutioniteionfouperesoutioy J. Struct. Biol.183,363-367 (2013).   
36.Nieuwenhuizen,R.P.et al.Measuring image resolution inoptical nanoscopy.Nat.Methods10,557-562 (2013).

# Acknowledgements

We thank Gang Ji (Center for Biological Imaging)and Tao Meng (Core Facility for Protein Research) for their asistance in cryo-EM.Funding: This work is supported by grants from the National Science Foundation of China (31127901，31370850,31100615,31130065 and 31127002)，Youth Innovation Promotion Association CAS,the Instrument Developing Project of the Chinese Academy of Sciences (yz2009-154-27,yg2012029),and the Major State Basic Research Program of the People's Republic of China (2010CB833701 and 2014CB910700).

# Author Contributions

T.X.and W.J. conceived of the project and wrote the manuscript. B.L. and W.J. constructed the cryonanoscopy. Y.X. performed high pressure freezing and cryosection. B.L.contributed to fluorescence imaging and data analysis.W.Z.contributed to cryo-EM imaging and data analysis.Y.C.constructed the plasmids and cultured the cell. C.F. purified fluorescent proteins and performed spectra measurements. B.L.,X.Z.,L.G.and Y.Z.contributed to active drift correction system.L.S.,X.H.,W.D.and F.S.contributed to electron diffraction experiments.F.S.contributed to the critical revision of the manuscript.All authors discussed the results and commented on the manuscript.

# Additional Information

Supplementary information accompanies this paper at http://www.nature.com/srep

Competing financial interests: The authors declare no competing financial interests.

How to cite this article: Liu,B.et al.Three-dimensional super-resolution protein localization correlated with vitrified cellular context. Sci.Rep.5,13017; doi:10.1038/srepl3017 (2015).

This work is licensed under a Creative Commons Attribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license,users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/