# New insights into the enzymatic role of EF-G in ribosome recycling

Dejiu Zhang1,2,t, Kaige Yan3,t, Yiwei Zhang1,4,t, Guangqiao Liu1, Xintao Cao1,2, Guangtao Song1, Qiang Xie4,\*, Ning Gao3\* and Yan Qin1,2,\*

1KeyLaboratory of RNA Biology,Institute of Biophysics,Chinese Academy of Sciences,Beijing 100101,China, 2Universityof ChineseAcademy of Sciences,Beijing 10o049,China,3MinistryofEducation KeyLaboratoryof Protein Sciences,CenterforStructural Biology,SchoolofLifeSciences,Tsinghua University,Beijing084,Cina and 4College of Life Sciences, Nankai University, Tianjin 30oo71, China

Received May 07,2015;Revised September 07,2015;Accepted September19,2015

# ABSTRACT

During translation, elongation factor G (EF-G) playsa catalyticroleintRNAtranslocationandafacilitative role in ribosome recycling. By stabilizing the rotated ribosome and interacting with ribosome recycling factor (RRF), EF-G was hypothesized to induce the domain rotations of RRF, which subsequently performs the function of splitting the major intersubunit bridges and thus separates the ribosome into subunits for recycling. Here,with systematic mutagenesis,FRET analysis and cryo-EM single particle approach,we analyzed the interplaybetween EF-G/RRF and post termination complex (PoTC). Our data reveal that the two conserved loops (loop land Il) at the tip region of EF-G domain IV possess distinct roles in tRNA translocation and ribosome recycling. Specifically, loop Il might be directly involved in disrupting the main intersubunit bridge B2a between helix 44 (h44 from the 3oS subunit) and helix 69 (H69 from the 50S subunit) in PoTC.Therefore, our data suggest a newribosome recyclingmechanismwhich requires an active involvement of EF-G. In addition to supporting RRF, EF-G plays an enzymatic role in destabilizing B2a via its loop Il.

# INTRODUCTION

Protein translation is a cyclic process which could be divided into four steps: initiation, elongation, termination and ribosomerecycling(1,2).Each step is assisted bya translational GTPase (trGTPase) in combination with other translation factors. EF-G is the only trGTPase involved in two distinct steps, elongation and ribosome recycling.In the elongation step, EF-G translocates the (tRNA)·mRNA complex by one codon in length on the ribosome, moving the peptidyl-tRNA from the A to the P site and the deacylated tRNA from the $\mathrm { ~ \bf ~ P ~ }$ to the E site, respectively (Figure 1A) (3).In the ribosome recycling step,EF-G together with RRF disassembles the post termination complex (PoTC) into two ribosomal subunits and releasesmRNA and deacylated tRNA for the next round of translation (Figure 1B) (4-8).Subsequently, translation initiation factor 3 (IF3) binds to 3oS subunit preventing reassociation of the free subunits (7,8). Both the translocation and ribosome recyclingrequire EF-G-catalyzed GTPhydrolysis (9,10).When GTP hydrolysis is blocked by replacing GTP with its nonhydrolyzable analog GDPNP, the ribosome recycling is completely blocked(7,8),while a single round of translocation still take place, albeit slowly (9,11).

EF-G is composed of five domains which could be divided into two superdomains. Superdomain I includes domains I-II and superdomain I includes domains III-V.During translocation, the two superdomains could move relatively to eachother.Upon GTPhydrolysis, the distal end of superdomain II, namely domain IV tip region,has a movement of $1 0 { - } 1 5 \mathring \mathrm { A }$ and the whole factor thus transforms from the pre-translocational (PRE) to the post-translocational (POST) conformation(12).Domain IV of EF-G is a factorspecific domain (13).The structural integrity of domain IV is important for both translocation and ribosome recycling (14-18).

The tip region of domain IV contains two highly conserved loops,i.e.loop Iand loop II(19).During the process of translocation, these two loops insert into the decoding center (DC) to override the hydrogen bonds between the DC nucleotides,mainly A1492 and A1493,and A-site tRNA, hence to disturb the interactions between mRNA-tRNA2 and the 30S subunit (Figure 1C-D) (20,21). This allows the swiveling motion of the 3oS head and the subsequent conformational change of the ribosome, and eventually leads to the concomitant translocation of mRNA-tRNAz (1,21). During ribosome recycling,binding of RRF-PoTC first induces a movement of the main intersubunit bridge B2a, which locates adjacent to the DC(Figure 1E) (22-27).With subsequent loading of EF-G,RRF was hypothesized to undergo domain rotations and hence to split the B2a completely (4,25-27). In this model, EF-G plays the role of facilitator, yet the detailed mechanism remains elusive. Considering the close location of domainIV tip in the vicinity of B2a, we assume that in addition to support RRF, EF-G might play a more active role in the splitting process.

![](images/6add663b67f8024a41fc33c860927dc3304077694ee4045167b2b53c514977f5.jpg)  
Figure1.FunctiooF-Guringtraslocatioadibooeecing.()riefodeofF-Gataldtrasloatioheprocssfroto POSTstateispromtedbdrolysis.rifoelofosoecigprocessE-GndFuctiotgtrtsocatee intoubuis.(CteacosofGoue)dRe)ibaltebeeotaisaf rRNA and $\mathtt { h 4 4 }$ (lavender)of16SrRNA.BothEF-GandRRFinteractwiththisbridge.The modelof thePOST-stateribosome(C,D)wasextracted fromapreviouscystalructure(BV(19))emodeofF(E)wasalsofrocystalograpdy(DBD26).ectof loop I deletion and single site mutants on ribosome breakdown.The ribosome sedimentation patern was analyzed by $1 5 \text{‰}$ sucrose density gradient ultracentrifugation.Fractions were collcted from the bottom to the top of the tube and measured at $\mathbf { A } _ { 2 5 4 }$ .The peaks corresponding to the 70S ribosome, disoetrs mutants are efective in both recycling and tRNA translocation,while the blue ones only in tRNA translocation.

In this study,we employed in vivo and in vitro assays to investigate the effect of the conserved loops of EF-G domainIV on the tRNA translocationand PoTC disassembly. Our earlier study has revealed that the two loops of EF-G domain IV, especially loop II, is essential for triggering the process of translocation (21).Extension of this study has now revealed that the loop II,which is away from the EFG:RRF interface,unexpectedly,is crucial for ribosome recycling.In the cryo-electron microscopy (cryo-EM) reconstitution of the PoTC·EF-G.GDPNP complex at the resolution of $4 . 3 \mathring \mathrm { A }$ ,we observed a conformational difference of loop II compared with EF-G structures in other recycling or elongational complexes. These data suggest a catalytic role ofEF-G in destabilizing the intersubunit bridges of the 70S ribosome and a common aspect of EF-G function in translation and recycling.

# MATERIALSANDMETHODS

# Strains, plasmids, growth conditions and reagents

Bacterial strains and plasmids used in this study were prepared as described in our previous study (21).LB broth orplates were used for bacterial growth.The media was supplemented with ampicillin $( 1 0 0 ~ \mu \mathrm { g / m l } )$ orkanamycin $( 5 0 ~ \mu \mathrm { g / m l } )$ .IPTG(isopropyl- $\boldsymbol { \beta }$ -D-thiogalactopyranoside) was added to induce protein expression. GTP and GDPNP(guanidine $5 ^ { \prime } { \cdot } \beta$ ， $\gamma$ -imido tri-phosphate),a nonhydrolysable GTP analogue,were purchased from Sigma (USA).All otherchemical were from Merck(GER) or AMRESCO(USA).

# Polysome breakdown assay

Polysome was prepared from $E .$ coli MRE600 as previously described (28).Polysome $( 0 . 2 \mathrm { - } 0 . 6 \mathrm { A } _ { 2 6 0 }$ units) was incubated with puromycin,RRF,EF-G and GTP in $5 5 0 ~ \mu \mathrm { l }$ ofRRF buffer( $2 0 ~ \mathrm { m M }$ HEPES $\mathrm { p H } 7 . 6$ ， $8 . 2 ~ \mathrm { m M }$ （204 $\mathrm { M g } \mathrm { S O } _ { 4 }$ ， $8 0 ~ \mathrm { m M }$ $\mathrm { \Delta N H _ { 4 } C l }$ ,4mM $\beta$ -Mercaptoethanol) at $3 0 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { m i n }$ .The sedimentation profiles of reactions were obtained using sucrose density gradient centrifugation ( $1 5 \text{‰}$ ,w/v) sucrose in RRF buffer, Beckman SW40 rotor, $3 6 ~ 0 0 0 ~ \mathrm { r p m }$ ， $3 . 5 \mathrm { ~ h ~ }$ $4 ^ { \circ } \mathrm { C }$ The results were analyzed by $\mathbf { A } _ { 2 5 4 }$ measurement.

# RESULTS

# Conserved loop II of EF-G domain IV isrequired forribosome recycling

In our previous work,we showed that both loops Iand I of EF-G domain IV are essential for translocation by systematic mutagenesis of EF-G(21).Here,we further tested the capacity of these EF-G mutants in ribosome recycling. Two sets of constructs were prepared, including loop deletions and single site mutations.Three loop deletion mutants were EF-G △lo0p I(507-513), △loop I(584-586) and △loop II (586-589) (Supplementary Figure S1). Single site mutants were createdbyreplacingtheresidueswithanaminoacid (aa) of different charge or the size of side chain (Supplementary Figure S1).

The effects of EF-G mutants on PoTC dissociation were determined by measuring the reduction of naturally produced poly-PoTC complexes, which were puromycin treated polysomes with a deacylated tRNA at the P-site (4). In the presence of wild type(WT) EF-G(EF-Gwt) and RRF,almost all the polysomes were disassembled into subunits, which quickly reassociated into monosomes and resulted in a sharp increase of 7oS(Figure1F,left part).However, when EF-Gwt was substituted by the loop deletion mutants, only △loop I was able to convert polysomes into monosomes, while the two loop II deletions abolished the activity (Figure 1F,middle part).This clearly indicates that loop I is indispensable for ribosome recycling,in contract to the negligible role of loop I.

The contribution of the individual aa residue to the disassemblyactivity ofEF-G was nextassayed.As expected from △loopI data,all loopI single site mutants displayed no apparent defects in the polysome breakdown experiments (SupplementaryFigure S2A).Similarly, most single site mutants of loop II showed no defects (Supplementary Figure S2B),except for the S588P mutant(Figure 1F,right part). In the presence of S588P, the increase of the monosome peak wassignificantly smaller,andresidual 3Xand4Xforms of polysome could be identified,indicating that the recycling activity of EF-G was considerably compromised.It is interesting to note that previous data suggested that the interactions between the upper region of EF-G domain IV and RRFis important for RRF domain rotation (29,30).Here, our data indicate that the distal loopIof domainIV, which locates far away from EF-G:RRF interface (25),is essential for ribosome recycling.Among the seven residues of loop II, S588 might be of particular importance.

# Loop I single site mutants retard cell growth and protein synthesis

Considering the necessity of loop II for ribosome recycling, we next analyzed the total effect of loop I mutants on cell growth and protein synthesis in vivo.We constructed overexpressionplasmids ofEF-G mutants and transfected each plasmid into $E .$ coli BL21 cells,which could overexpress the corresponding EF-G mutant upon IPTG induction.As shown in Figure 2A,the overexpression of EF-Gwt, compared to the control (pEF-28a), did not inhibit cell growth. In contrast,the single mutantsH583K,S588PandE589A were all toxic.We further investigated the effect of these mutants on total protein synthesis by using $^ { 3 5 } \mathrm { S }$ -methionine incorporation assay. With SDS-PAGE and subsequent radioautography, the total proteins synthesized in vivo after $^ { 3 5 } \mathrm { S }$ methionine incorporation were visualized (Figure 2B). It is clear that the three mutations had dominant negative effect on total protein synthesis and this effect could not be compensated by a prolonged incubation (Figure 2C). These results provide explanations for the observations in Figure 2A, that overexpression of EF-G loop I mutants retarded cell growth due to their inhibitive effect on translationprocess.However, EF-G has dual functions of translocation and recycling in translation.To specify the reason, we next performed the fluorescence measurements, biochemical analyses,and cryo-EM structural studies subsequently to determine the effect of loop II mutants in translocation and recycling.

# Loop II mutants inhibit (tRNA)2•mRNA translocation

We recently identified His583,Ser588 and $\mathrm { \ G l u { 5 8 9 } }$ as the catalytic residues for EF-G translocase activity by tRNA foot-printing studies (21).Here, we employed a steady-state fluorescence measurement to monitor the active motion of mRNA upon EF-G proteins binding onto the PRE state ribosomes (Figure 2D). In the pre-experiment of puromycin assay,we first tested the translocation efficiency of the tRNAs in the PRE complexes,which were programmed with MF-mRNAs, a deacylated $\mathrm { { t R N A } _ { \mathrm { { f } } } \mathrm { { M e t } } }$ at the $\mathrm { \bf P }$ site and an $\mathrm { A c } [ ^ { 1 4 } \mathrm { C } ] \mathrm { P h e - t R N A ^ { P h e } }$ at the A site (21,31). After addition of EF-G proteins,a rapid increase in puromycin activity was observed in the EF-Gwt samples but not in the samples of single site mutants (Supplementary Figure S3).In the main experiment of steady-state fluorescence measurement,mRNA movement was monitored by tracing a fluorescence dye pyrene attached to the $3 ^ { \prime }$ endof the mRNA (32,33).Upon mRNA translocation, the pyrene will enter the mRNA tunnel in the ribosome, thus the fluorescent signal will be quenched(Figure 2D).In this assay, PRE state ribosome complexes were prepared in a similar manner as in the puromycin assay, except that the mRNA was replaced with pyrene-labeled mRNA.When the PRE ribosomes were mixed with EF-Gwt, the pyrene signal was quenched very quickly. Contrarily, little fluorescent change was observed upon addition of EF-G mutants, demonstrating that the mRNA was not translocated (Figure 2E). Together with our previous data, these results firmly establish that the loop II mutants,even the single site mutants H583K,S588P and E589A,dramatically impaired the translocation activity of EF-G. Their pronounced toxicity to the cells as observed in Figure 2A-C is likely caused mainly by their deficiency in translocation, the main step of translation elongation. However, considering the importance of loop II, especially the residue Ser588,in recycling (Figure 1F),we next explored the impact of loop II mutants on RRF in PoTC.

![](images/ce07b57569b148a559907be638b52d4e98e9d062dfe6748b40ffdf8b31b274ab.jpg)  
Figure2.fecoeutsoalpotsra.(veeota growthofE.colckrdicathiatIsdd.(dCsof-oveepssoototalprotsi $\dot { m }$ （204号 vivo.Total soluble proteins from E.coli BL21 overexpressing EF-G wt or mutants were analyzed by SDS-PAGE(B). After $^ { 3 5 } \mathrm { S }$ -methionine addition, the ratio of proteins containing $^ { 3 5 } \mathrm { S } .$ -methionine increasedover time as new proteins were continued tobesynthesized incels(C).(D)Brief llustrationof the pyrene-modifedmRNAtranslocationassay.Thetranslocatioactionwasmonitoredbytracingthefuorescenceofpyreneatachedtothe $3 ^ { \prime }$ end of the mRNA.Tefoeseceessgedtaforesoatiodreassackausoatio.(ERelativef state ribosomes programmed by the labeled mRNA with incorporation of EF-Gwt or mutants.Error bars,s.e.m.( $\mathrm { \tilde { n } } = 3$ technical replicates). $^ { * * * } P < 0 . 0 0 1$ ：

# Loop I mutants do not retard RRF in PoTC

The binding property of recycling-defective mutants of EFG,i.e. △loop II,△loop II' and S588P, to PoTC was first determined by sucrose cushion assay.Figure 3A shows that at the molecular ratio of 10 (EF-G to 70S), the occupancy of EF-G wt on PoTC has been saturated at ${ \sim } 9 5 \%$ Similarly, every mutant could saturate its binding on PoTC to an extent of $9 5 \%$ at the ratio of 10 (Figure 3B).It indicates that the binding of EF-G loop II mutants to PoTC was not impaired by mutations. For binding condition optimization, different nucleotides(GTP,GTP+FA,GDPNP) were also tested,and similar results were obtained (Supplementary Figure S4).

In PoTC,binding of RRF will be replaced uponaddition ofEF-Gdue to the observed steric hindrancebetweenRRF and EF-G (in GTP-like state) in the post termination state of the ribosome (30,34).Here, when the wt EF-G was exchanged into the recycling-defective mutants, the same results were obtained: upon addition of each EF-G mutant, RRF was no long detected in PoTC (Figure 3C). It suggests that all EF-G loop I mutants maintained the natural conformation of the factor, in agreement with Figure 3A and 3B,and therefore interacted normally with RRF in PoTC under the optimized condition in vitro.To determine the situation in vivo,we next analyzed the polysomes of $E .$ coli cells overexpressing S588P.By collecting the fractions with detectable amount of both EF-G and RRF(Figure 3D), we enriched the potential PoTC with both factors.However, when we performed cryo-EM reconstitution,we did not find ribosome particles with both EF-G and RRF.Thus,all the in vitro and in vivo conditions we have tried did not support binding of RRF to PoTC in the presence ofEF-G or itsrecycling-defectivemutants.Nevertheless, these results reflect an essential aspect of recyclingmechanism, that the loop I region of EF-G domain IV dose not interact with or influence RRF in PoTC,and very possibly they function independently from each other. Thus,loop II must play distinctive role inPoTC.To specify the role ofEF-GloopII, we performed further molecular dynamic and cryo-EM analyses.

![](images/32fe5bbaffe99eb5ba9e4f22ffd28ca39ea10aba5db5fed53ceddd0d8bfdb61c.jpg)  
Figure3.ConditionoptizatiotachevePF-GFomplex.(AandB)tratiofF-Gwt(A)orutants(B)toCPreeforeucoe cushionultracftuatateossactfatiQuatiFaoisoa gel.The optimized ratio is colored in red.Error bars,s.e.m.( $\mathrm { { \acute { n } } } = 3$ technical replicates).(C) Titration of RRF to PoTC with subsequent addition of EF-G proteinstpcoflofracoatndcteeytotiespdo RRF and small ribosomalsubunit protein S2.In thefractionof disome (red rectangle),both EF-Gand RRFwere detected.

# Loop I mutants stabilize rotated PoTC

After the release of the newly synthesized polypeptide from the ribosome, the PoTC carries the mRNA and a deacylated tRNAat the P-site, fluctuating between $\mathbf { P } / \mathbf { P }$ and $\mathrm { P / E }$ positions. This fluctuation is coupled to the classical and the rotated states of the ribosome, respectively (25,26). The rotated state of the PoTC could be stabilized by either EFG or RRF (20,26,35,36) and is required for the subsequent steps of disassembly. For binding condition optimization, different nucleotides (GTP,GTP+FA,GDPNP) were also tested,and similar results were obtained,we constructed PoTC·EF-G·GDPNP complexes and studied the conformational changes of the ribosome using Forster resonance energy transfer (FRET) assay.

We applied a FRET strategy described earlier (37,38), where L9 and S6 were modified with the donor and acceptor dyes AF555 and AF647,respectively.A counterclockwise rotation of the 3oS subunit relative to the 50S subunit will cause a longer separation distance between the two fluorophores and a consequent lower FRET efficiency (Figure 4A).In the experiments,binding of EF-Gwt to the PoTC in the presence of GDPNP induced a decrease in energytransfer(Figure 4B).Similar results were also observed in △loop II, △loop II' or S588P samples, indicating that all these recycling-defective mutants were able to stabilize the rotated state of the PoTC(Figure 4C).Hence,loopIIshould not contribute to the hybrid state stabilizer role of EF-G in recyclingphase.

# Loop II neighbors the groove of $\mathbf { H 6 9 } / \mathbf { h 4 4 }$ close to B2a

To obtain the structural insights into the interaction betweenloopII and thePoTC,we performed cryo-EM structural analysesof the PoTC·EF-G(S588P) GDPNP complex.Thirty thousand raw particles were collected and were analyzed by RELION (39) and the final structure was solved at the resolution of $\lvert 4 . 3 \mathring \mathrm { ~ A ~ }$ (Supplementary Figure S5).The PoTC·EF-G(S588P)·GDPNP complex was featured with a rotated ribosome anda hybrid $\mathrm { P / E }$ -sitetRNA (Figure 5A). Compared to the previous structures (40), the S588P-induced ribosomal conformation exhibited a very similar 30S rotationas observed in ribosome·EF-Gwt complexes (Figure 5C).Whenthe structure wasfurther zoomed in,we observed a distinct conformation of loop II,which locates next to the groove of H69/h44,i.e. the major intersubunit bridge B2a of PoTC (Figure 5B).When we compared our result with a recent cryo-EM structure of 7oS·EF-G (41),whose EF-G was GTPase center mutated (H91A) but with an intact loop II(Figure 5D),we found the S588P mutation caused a much less extended conformation of loop II (Figure 5E),which is also true when a recent crystal structure of 70S.EF-G (35) (Figure 5F) was used for comparison.

# DISCUSSION

The translocation and recycling are essential steps for protein synthesis in both prokaryotes and eukaryotes.In these two steps, the ribosome undergoes similar conformational changes (25,29,36) but applies different mechanisms. In the translocation step,EF-G stabilizes therotated state of the

![](images/002943616eab7bad41ffb0784e6c509e94c2e7fac3ff2cc2024ae2e4808d84d7.jpg)  
Figure4FREalfalato.(catisfeceedusidbitsallolote S6walabelediodbolbprotewasbeeditFetoodaf rotation of PoTC induced by EF-G binding.(B)Fluorescence emission of the $S 6 / \mathrm { L 9 }$ ribosome constructs.Black curve is the spectrum of vacant ribosome; blue curve is the PoTC complex in the presence of EF-GGDPNP.The excitation wavelength was $5 5 0 \mathrm { n m }$ .Addition of EF-G wt led to a significant decrease iFR EF-G mutants △loop II,△loop II'and, S588P.

PRE ribosomes and catalyzes the (tRNA)2·mRNA translocation by disrupting the interaction between the ribosomal DC and the codon-anticodon duplex at the A site(1,21,42). In the recycling step,EF-G stabilizes the rotated PoTC similarlyas in PRE state,and facilitates the domain rotation of RRF,which consequently catalyzes the dissociation of the ribosome into subunits and the release of mRNA and deacylated tRNA fora new round of translation (43,44).

The interplay between RRF and PoTC has been studied by foot-printing (45) and various structural approaches (22-24,26,27,29.30,45-48). However, the exact role of RRF in splitting the PoTC remains unclear.For example, in a POST·RRF complex, the inclusion of RRF did not induce any changes in bridges connecting the two subunits,suggesting that RRF cannot take full responsibility for breaking the intersubunit bridges (22).More importantly, it is difficult to visualize the functional state of PoTC with both RRF and EF-Gloaded concurrently,because the whole 70S ribosome will easily be split into subunits in the presence ofboth factors (7,8,49).A recent cryo-EM structural study suggested that EF-G and RRF from different species could coexist in PoTC(25).In this structure, EF-G domains III, IVandVcontactedRRF interdomainhinges,head domain and C-terminus,respectively,and hence induced a series of domain rotations of RRF in the splitting process.With reoriented head domain and C-terminus,RRF destabilized the main intersubunit bridge B2a (26,32). But, the final step ofB2a separation remains elusive.

The distal end of EF-G domain IV has been reported to interact with the DC and the neighbor B2a region in h44 of 16S rRNA and undergo flexible conformations (Supplementary Figure S6) during translocation (19,20,36).Here, in the structure of PoTC·EF-G (S588P).GDPNP complex we observed the deformed EF-G loop II(S588P) neighboring the groove of $\mathrm { H } 6 9 / \mathrm { h } 4 4$ ，which locates adjacent to the B2a but far from the interface between EF-G and RRF. In native conformation, the loop II of EF-Gwt might be directly involved in splitting the B2a bridge in PoTC(Figure 5E and F).In polysome breakdown study, WT RRF with EF-G loop II deletion or S588P mutants abolished largely the disassemblyactivity.Taken together, these results demonstrate that EF-G is involved in the final splitting process byplaying an active role in addition to facilitating RRF orientation. Through the highly conserved loop II,EF-G together with RRF disassembles the PoTC into the subunits.These findings shed new light on the molecular mechanism of EF-G/RRF catalyzed ribosome recycling:upon binding ofRRF to PoTC state ribosomes,it promotes the rotated conformation of the ribosome and destabilizes the interactions between H69 and h44. Subsequently, loading ofEF-G·GTP facilitates RRF reorientation and actively splits the B2a bridge.Finally,upon GTP hydrolysis,EFG transforms into a GDP conformation and thus separates the 70S ribosome into two subunits completely (Figure 6). In this process,loop II plays a mechanistic role to disrupt the interactions in the vicinity of DC,in a similar manner as during translocation.Thus, the interactions of loop I with the DC region constitute a threshold event in both the translocation and recycling processes.It reflects a common aspect of EF-G function in these two translational phases.

![](images/3c955078e8ba98f4c9cde19398bdc86f937084fcc6bb550be0b2c8c9513e3fa6.jpg)  
Figure5.CryoEMructurefEF-G(58)GD(A)OvervieoftheCwitEF-G(58)(edandacylated(yaine P/stateToelspletosealure)d)oofedf H69and4(eaitieits EF-G(588P)binding.Compared tothecasicalstate (gray,PDB4V51(50),30Ssowsacounter-clockwiserotatioby $7 ^ { \circ }$ (viewed from the solvent side of0Subculdsheeepeipofsiactu (H91A)withtrdatomcodel(DB(41).(E)uperipositiofousiaoftheF-G(88PGDpleithte atomicmodelfroJA1.F)upermpostioofourdensitymapofteTCEF-G(S588)GDPNPcomplexwiththeatomicodelfrom4I() Compared to (B), the orientation in (D-F) has been turned ${ \sim } 1 8 ^ { \circ }$ along the long axis of EF-G domain IV.

![](images/a7c1bbfa5a16f93027a76c856cbb220b44e09c3e735fae53915c596fa76e9047.jpg)  
Figure6.Scheptalifsat conformatioalcageof9adtstableBaUpEFdig,loI(eloatesineviciityf2aTeei loop II disassembles the ribosome into the subunits.

# ACCESSIONNUMBERS

The cryo-EM map has been deposited at EMDB under accession code EMD-6474.

# SUPPLEMENTARYDATA

Supplementary Data are available at NAR Online.

# ACKNOWLEDGEMENTS

We thank Profs. Akira Kaji (University of Pennsylvania) and Hideko Kaji (Thomas Jefferson University） for help and discussions.We also thank the China National Center forProtein Sciences (Beijing) and Tsinghua National Laboratory for Information Science and Technology("Explorer $1 0 0 ^ { \circ }$ cluster system) for providing computation resource.

# FUNDING

Major State Basic Research ofChina 973 [2012CB911001 and 2013CB531200 to Y.Q.]; National Natural Science Foundation of China [31170756,31270847and 31322015 to Y.Q., 31170677 and 31422016 to N.G.]. Funding for open access charge:Major State Basic Research of China 973 [2012CB911001 to Y.Q].

Conflictof interest statement.None declared.

# REFERENCES

1.Yamamoto,H.,Qin,Y.,Achenbach,J.,Li,C.,Kijek,J., Spahn,C.M. and Nierhaus,K.H. (2014) EF-G and EF4: translocation and back-translocation on the bacterial ribosome.Nat.Rev. Microbiol., 12, 89-100.   
2.Ramakrishnan,V.(2oo2) Ribosome structure and the mechanism of translation. Cell,108,557-572.   
3.Agirrezabala,X.and Frank,J.(2oo9) Elongation in translation as a dynamic interaction among the ribosome, tRNA,and elongation factors EF-G and EF-Tu.Q.Rev.Biophys.,42,159-200.   
4.Hirokawa,G.,Demeshkina,N.,Iwakura,N.,Kaji,H.and Kaji,A. (2006) The ribosome-recycling step: consensus or controversy? Trends Biochem. Sci.,31,143-149.   
5.Frank,J.,Gao,H.,Sengupta,J.,Gao,N.and Taylor,D.J.(2o07) The process of mRNA-tRNA translocation. Proc. Natl. Acad. Sci. U.S.A., 104,19671-19678.   
6. Hirashima,A.and Kaji,A.(1973) Role of elongation factor G and a protein factor on the release of ribosomes from messenger ribonucleic acid.J.Biol. Chem.,248,7580-7587.   
7.Zavialov,A.V.,Hauryliuk,V.V.and Ehrenberg,M.(2oo5) Splitting of the posttermination ribosome into subunits by the concerted action ofRRFandEF-G.Mol.Cell,18,675-686.   
8.Peske,F.,Rodnina,M.V.and Wintermeyer,W.(2oo5) Sequence of Steps in Ribosome Recyclingas Defined by Kinetic Analysis.Mol. Cell,18,403-412.   
9.Hirokawa,G.,Iwakura,N.,Kaji,A.and Kaji,H.(20o8) The role of GTP in transient spliting of 70S ribosomes byRRF (ribosome recycling factor) and EF-G (elongation factor G).Nucleic Acids Res., 36,6676-6687.   
10.Agrawal,R.K.,Heagle,A.B.,Penczek,P.,Grassucci,R.A.and Frank,J. (1999) EF-G-dependent GTP hydrolysis induces translocation accompanied by large conformational changes in the 7oS ribosome. Nat. Struct. Biol.,6,643-647.   
11.Rodnina,M.V., Savelsbergh,A.,Katunin,V.I.and Wintermeyer,W. (1997) Hydrolysis ofGTP by elongation factor G drives tRNA movement on the ribosome.Nature,385,37-41.   
12.Brilot,A.F.,Korostelev,A.A.,Ermolenko,D.N.and Grigorieff,N. (2013) Structure of the ribosome with elongation factor G trapped in the pretranslocation state.Proc.Natl. Acad. Sci. U.S.A.,110, 20994-20999.   
13. Zhang,D.and Qin,Y.(2013) The paradox of elongation factor 4: highly conserved,yet of no physiological significance? Biochem. J., 452,173-181.   
14.Ito,K.,Fujiwara,T.,Toyoda,T.and Nakamura,Y.(2oo2) Elongation factor G participates in ribosome disassembly by interacting with ribosome recycling factor at their tRNA-mimicry domains.Mol. Cell, 9,1263-1272.   
15.Savelsbergh,A.,Matassova,N.B.,Rodnina,M.V.andWintermeyer,W. (2000) Role of Domains 4and 5 in Elongation Factor G Functions on the Ribosome.J.Mol.Biol.,300,951-961.   
16.Martemyanov,K.A. and Gudkov,A.T. (1999) Domain IV of elongation factor G from Thermus thermophilus is strictly required for translocation.FEBSLett.,452,155-159.   
17.Martemyanov,K.A.,Yarunin,A.S.,Liljas,A.and Gudkov,A.T.(1998) An intact conformation at the tip of elongation factor G domain IV is functionally important.FEBS Lett., 434,205-208.   
18.Guo,X.,Peisker,K.,Backbro,K., Chen,Y., Koripella,R.K., Mandava,C.S., Sanyal,S.and Selmer,M. (2012) Structure and function of FusB:an elongation factor G-binding fusidic acid resistance protein active in ribosomal translocation and recycling. Open Biol.,2,120016.   
19. Gao,Y.G., Selmer,M.,Dunham,C.M., Weixlbaumer,A., Kelly,A.C. and Ramakrishnan,V.(2oo9) The structure of the ribosome with elongation factor G trapped in the posttranslocational state. Science, 326,694-699.   
20. Zhou,J.,Lancaster,L.,Donohue,J.P.and Noller,H.F. (2013) Crystal structures of EF-G-ribosome complexes trapped in intermediate states of translocation. Science,34o,1236086.   
21.Liu,G.Q., Song,G.T., Zhang,D.Y., Zhang,D.J.,Li,Z.K.,Lyu,Z.X. Dong,J.S.,Achenbach,J., Gong,W.M., Zhao,X.S. et al. (2014) EF-G catalyzes tRNA translocation by disrupting interactions between decoding center and codon-anticodon duplex.Nat. Struct. Mol. Biol., 21, 817-824.   
22. Weixlbaumer,A.,Petry,S.,Dunham,C.M.,Selmer,M.,Kelley,A.C. and Ramakrishnan,V. (20o7) Crystal structure of the ribosome recycling factor bound to the ribosome.Nat. Struct. Mol. Biol.,14, 733-737.   
23.Barat,C.,Datta,P.P,Raj,V.S., Sharma,M.R., Kaji,H., Kaji,A.and Agrawal,R.K. (2007) Progression of the Ribosome Recycling Factor through the Ribosome Dissociates the Two Ribosomal Subunits. Mol. Cell,27,250-261.   
24.Agrawal,R.K., Sharma,M.R.,Kiel,M.C.,Hirokawa,G.,Booth,T.M., Spahn,C.M., Grassucci,R.A., Kaji,A. and Frank,J. (2004) Visualization of ribosome-recycling factor on the Escherichia coli 70S ribosome: functional implications.Proc.Natl.Acad.Sci. U.S.A.,101, 8900-8905.   
25. Yokoyama,T., Shaikh,T.R.,Iwakura,N., Kaji,H., Kaji,A.and Agrawal,R.K. (2012) Structural insights into initial and intermediate steps of the ribosome-recycling process. EMBO J.,31,1836-1846.   
26.Dunkle,J.A.,Wang,L.,Feldman,M.B.,Pulk,A.,Chen,V.B., Kapral,G.J.,Noeske,J.,Richardson,J.S.,Blanchard,S.C.and Cate,J.H.(2O11) Structures of the bacterial ribosome in classical and hybrid states of tRNA binding. Science,332, 981-984.   
27.Pai,R.D., Zhang,W.,Schuwirth,B.S.,Hirokawa,G.,Kaji,H.,Kaji,A. and Cate,J.H.D. (2008) Structural Insights into Ribosome Recycling Factor Interactions with the 7OS Ribosome.J.Mol.Biol.,376, 1334-1347.   
28.Fujiwara,T., Ito,K.and Nakamura,Y. (2o01) Functional mapping of ribosome-contact sites in the ribosome recycling factor: a structural view from a tRNA mimic.RNA,7,64-70.   
29. Gao,N., Zavialov,A.V., Ehrenberg,M.and Frank,J. (2007) Specific interaction between EF-G and RRF and its implication for GTP-dependent ribosome splitting into subunits.J. Mol. Biol.,374, 1345-1358.   
30. Gao,N., Zavialov,A.V.,Li,W., Sengupta,J.,Valle,M., Gursky,R.P., Ehrenberg,M.and Frank,J. (2oo5) Mechanism for the disassembly of the posttermination complex inferred from cryo-EM studies. Mol. Cell,18, 663-674.   
31. Qin,Y.,Polacek,N., Vesper,O.,Staub,E.,Einfeldt,E,Wilson,D.N. and Nierhaus,K.H. (2oo6) The highly conserved LepA is a ribosomal elongation factor that back-translocates the ribosome.Cell,127, 721-733.   
32.Ermolenko,D.N.and Noller,H.F. (2011) mRNA translocation occurs during the second step of ribosomal intersubunit rotation. Nat. Struct.Mol.Biol.,18,457-462.   
33.Studer,S.M.,Feinberg,JSandJosephS.(203)Rapidineticaalis of EF-G-dependent mRNA translocation in the ribosome.J. Mol. Biol.,327,369-381.   
34. Zavialov,A.V., Hauryliuk,V.V.and Ehrenberg,M. (2005) Splitting of the posttermination ribosome into subunits by the concerted action of RRF and EF-G.Mol. Cell,18,675-686.   
35.Pulk,A.andCate,JH.(2013)Controlofribosomalsubunitrotation by elongation factor G. Science,340,1235970.   
36.Tourigny,D.S.,Fernandez,I.S.,Kelley,A.C.andRamakrishnan,V. (2013) Elongation factor G bound to the ribosome in an intermediate state of translocation.Science,340,1235490.   
37.Hickerson,R.,Majumdar,Z.K.,Baucom,A.,Clegg,R.M.and Noller,H.F.(2oo5) Measurement of internal movements within the 30 S ribosomal subunit using Forster resonance energy transfer.J. Mol. Biol.,354,459-472.   
38.Ermolenko,D.N.,Majumdar,Z.K.,Hickerson,R.P., Spiegel,P.C., Clegg,R.M.and Noller,H.F.(2o07) Observationof intersubunit movement of the ribosome in solution using FRET.J. Mol. Biol.,   
370,530-540.   
39.Scheres,S.H.(2012)A Bayesian view on cryo-EM structure determination.J. Mol. Biol.,415,406-418.   
40.Valle,M.,Zavialov,A.,Li,W., Stagg,S.M., Sengupta,J.,Nielsen,R.C., Nissen,P.,Harvey,S.C.,Ehrenberg,M.andFrank,J.(2003) Incorporation of aminoacyl-tRNA into the ribosome as seen by cryo-electron microscopy. Nat. Struct. Biol.,10, 899-906.   
41.Li,W.,Liu,Z.,Koripella,R.K.,Langlois,R., Sanyal,S.and Frank,J. (2015) ActivationofGTPhydrolysis in mRNA-tRNA translocation by elongation factor G. Sci. Adv.,1,e1500169.   
42.Frank,J.and Agrawal,R.K.(2ooo)A ratchet-like inter-subunit reorganization of the ribosome during translocation.Nature,406,   
318-322.   
43.Karimi,R.,Pavlov,M.Y.,Buckingham,R.H.andEhrenberg,M.(1999) Novel roles forclassical factors at the interface between translation terminationand initiation. Mol. Cell,3,6O1-609.   
44.Kaji,A.,Kiel,M.C.,Hirokawa,G.,Muto,A.R.,Inokuchi,Y.and Kaji,H. (2oo1) The fourth step of protein synthesis: disassembly of the posttermination complex is catalyzed by elongation factor G and ribosome recycling factor, a near-perfect mimic of tRNA. Cold Spring Harb.Symp. Quant.Biol.,66,515-529.   
45.Lancaster,L.,Kiel,M.C.,Kaji,A.andNoller,H.F.(2002) Orientation of ribosome recycling factor in the ribosome from directed hydroxyl radical probing. Cell,111,129-140.   
46.Wilson,D.N., Schluenzen,F.,Harms,J.M.,Yoshida,T., Ohkubo,T., Albrecht,R.,Buerger,J.,Kobayashi,Y.and Fucini,P. (20o5)X-ray crystallography study on ribosome recycling: the mechanism of binding and action of RRF on the 5OS ribosomal subunit.EMBO J., 24,251-260.   
47.Borovinskaya,M.A.,Pai,R.D.,Zhang,W.,Schuwirth,B.S., Holton,J.M.,Hirokawa,G.,Kaji,H.,Kaji,A.and Cate,JH. (2007) Structural basis for aminoglycoside inhibition of bacterial ribosome recycling. Nat. Struct. Mol.Biol.,14,727-732.   
48.Dunkle,J.A.and Cate,J.H. (2010) Ribosome structure and dynamics during translocation and termination.Annu. Rev. Biophys.,39, 227-244.   
49.Hirokawa,G.,Nijman,R.M.,Raj,V.S.,Kaji,H., Igarashi,K.and Kaji,A.(2oo5) The role of ribosome recycling factor in dissociation of 70S ribosomes into subunits.RNA,11, 1317-1328.   
50.Selmer,M.,Dunham,C.M.,Murphy,F.V.t.,Weixlbaumer,A.,Petry,S., Kelley,A.C.,Weir,J.R.andRamakrishnan,V.(2oo6) Structure of the 70S ribosome complexed with mRNA and tRNA. Science, 313, 1935-1942.