# Genome-scale analysis of the cotton KCS gene family revealed a binary mode of action for gibberellin A regulated fiber growth

Guang-Hui Xiao13, Kun Wang²,Gai Huang' and Yu-Xian Zhu1,2\*

1The State Key Laboratoryof Proteinand Plant Gene Research,College of Life Sciences,Peking University,Beijing $1 0 0 8 7 1$ , China,²Institute for AdvancedStucsityaifsai 710062,China.\*Correspondence: zhuyx2@pku.edu.cn

Abstract Production of $\beta$ -ketoacyl-CoA,which is catalyzed by 3-ketoacyl-CoA synthase (KCS),is the first step in very long chain fatty acid (VLCFA) biosynthesis. Here we identified $5 8 \mathsf { K C S }$ genes from Gossypium hirsutum，31 from G.arboreum and 33 from G.raimondii by searching the assembled cotton genomes.The gene family was divided into the plant-specific FAE1-type and the more general ELO-type.KCS transcripts were widely expressed and 32 of them showed distinct subgenome-specific expressions in one or more cotton tissues/organs studied.Six GhKCS genes rescued the lethality of elo2△elo3△ yeast double mutant, indicating that this gene family possesses diversified functions. Most KCS genes with GA-responsive elements (GAREs) in the promotersweresignificantlyupregulated bygibberellin $A _ { 3 }$ (GA). Exogenous $\mathsf { G A } _ { 3 }$ not only promoted fiber length，but also increased the thickness of cell walls significantly.GAREs present also in the promoters of several cellulose synthase (CesA) genes required for cell wall biosynthesis and they were all induced significantly by $\mathsf { G A } _ { 3 }$ .Because GA treatment resulted in longer cotton fibers with thicker cell walls and higher dry weight per unit cell length,we suggest that it may regulate fiber elongation upstream of the VLCFA-ethylene pathway and also in the downstream steps towards cell wall synthesis.

Keywords:3-Ketoacyl-CoA synthase; cotton; expression patterns; gibberellins;very long chain fattyacids   
Citation: Xiao GH,Wang K,Huang G,Zhu YX (2o15) Genome-scale analysis of the cotton KCS gene family revealed a binary mode of action for gibberellin A regulated fiber growth.JIntegr Plant Biol XX: XX-XX doi: 10.1111jipb.12429   
Edited by: Chun-Ming Liu, Institute of Botany, CAS,China   
Received Sept.2,2015; Accepted Sept.22,2015   
Available online on Sept.24,2015 at www.wileyonlinelibrary.com/ journal/jipb   
$\mathfrak { O }$ 2015 Institute of Botany, Chinese Academy of Sciences

# INTRODUCTION

Cotton fibers are single-celled trichomes differentiated from the ovule epidermis and considered asa model system for studying cell elongation and cell wall biogenesis (Kim and Triplett 2001; Wang et al. 2004; Pang et al. 201o; Qin and Zhu 2011; Zhu and Li 2o13; Li et al. 2o14). Many genes are involved in various steps during fattyacid biosynthesis and elongation is highly upregulated during cotton fiber initiation and elongation (Ji et al. 2003; Qin et al. 2005; Shi et al. 20o6; Yang Samuel et al. 2006; Gou et al. 2007; Qin et al. 2007; Shang et al. 2013; Zou et al.2013; Liu et al. 2015). Very long chain fatty acids (VLCFAs) (C2o:o to C3o:o) significantly promoted cotton fiber cell elongation with several KCS genes highly upregulated during cotton fiber development (Qin et al.2007).

Very long chain fatty acids are generated in the endoplasmic reticulum with four successive reactions catalyzed by four diverse enzymes: Condensation of a long chain acyl-CoA with a malonyl-CoA by 3-ketoacyl-CoA synthase (KCS);reduction of 3-ketoacyl-CoA by 3-ketoacyl-CoA reductase (KCR); dehydration of $3$ -hydroxyacyl-CoA by 3-hydroxyacyl-CoA dehydratase (HCD);and further reduction of trans2-enoyl-CoA by trans-2-enoyl-CoA reductase (ECR) to form a two carbon elongated acyl-CoA (Fehling et al. 1991; Riezman 2007).In yeast，Elo1p is responsible for the elongation of medium-chain fattyacids,Elo2p participates in the elongation to C24,and Elo3p is essential for the elongation of C24 to C26 (Toke et al.1996; Oh et al.1997; Dittrich et al.1998). Yeast cells with elo2 and elo3 double deletion is constitutive lethal (Oh et al.1997).The functions of KCS genes in other organisms are often characterized with lethality restoration of the double mutant yeast cell line.

KCS is regarded as the rate-limiting enzyme that also determines the substrate and tissue specificities of fattyacid elongation in higher plants (Lassner et al.1996;Millar et al. 1997;Denic and Weissman 2oo7). VLCFAs were found in sphingolipids,seed oil and epicuticular waxes in plant.In the Arabidopsis genome,21 FAE1-type and 2 ELO-type KCS genes werereported,with the functions of ELO-type KCSs largely unknown (Dunn et al. 2oo4; Costaglioli et al. 20o5; Joubés et al.2oo8).The biological functions and substrate specificities (based on differences in carbon chain length and degrees of unsaturation) of several KCS genes have gradually been elucidated in the past several decades using various Arabidopsis mutants.For example，FAE1 isolated from the Arabidopsis thaliana mutant fae1 was shown to catalyze C20 and C22 VLCFA biosynthesis for storage lipids in the seeds (Kunst et al. 1992; James et al.1995; Rossak et al. 2001). CER6/ CUT1 was involved in the biosynthesis of C26 and longer VLCFAs for cuticular waxes in epidermis and during root hair development (Hooker et al. 20o2; Pang et al. 201o; Xu et al. 2012).Loss of FDH1 (FIDDLEHEAD1) function resulted in ectopic organ fusions indicating a crucial role of KCS in organ development (Lolle et al. 1997; Yephremov et al. 1999;

Pruitt et al.2ooo; Voisin et al.2oo9). In addition, complete loss of KCS2o or KCS2 function led to the accumulation of C20 VLCFA suggesting that these two KCSs may be required for cuticular wax and root suberin biosynthesis (Franke et al. 2009; Lee et al. 2009).

Phytohormones,mainly ethylene，brassinosteroid (BR) and auxin are known to play important roles in cotton fiber cell initiation and elongation (Shi et al. 2oo6; Zhang et al.2011). GA was recently reported to be able to regulate cellulose synthesis through the DELLA-NAC signaling cascade in rice (Huang et al. 2o15). Exogenous application of $\mathsf { G A } _ { 3 }$ or indole3-acetic acid (IAA） to the flowers resulted in significant increases of fiber number per ovule (Seagull et al.2oo4) and $\mathsf { G A } _ { 3 }$ promoted significant fiber cell growth as well (Mei et al. 2010). Previously, GA was suggested to act downstream of the ethylenepathway since the GA biosynthesis inhibitor, paclobutrazol,nullified the biological functions of externally applied ethylene and C24:o, the later is known to mediate ethylene production (Qin et al. 2007; Mei et al. 2010). GAs may function through interaction with $\mathsf { G h H O X 3 } ,$ 、an HD-ZIP IV transcription factor,to upregulate the expression of GhRDL1 and GhEXP1 to promote fiber elongation (Shan et al.2014).

To fully understand the possible functions and the evolution of this important gene family among the three cotton species,we identified $5 8 \ K C S$ genes from G. hirsutum, 31 from G.arboreum and 33 from G.raimondii,based on the whole genome sequences (Paterson et al. 2o12; Wang et al. 2012; Li et al. 2014, 2015). We focused on the phylogeny and expression pattern analysis，and used the yeast double mutant cell line to characterize systematically the cotton KCS genes.We searched for the presence of GAREs and L1 box, which is required for GhHOX3 function in the GA signaling pathway(Shan et al. 2o14),in the promoter regions of KCS and CesA genes to clarify whether and how GAs are involved in the control of cotton fiber cell growth.

# RESULTS

# Genome-wide identification of KCS genes in three different cotton species

To identify all the KCS genes in G. hirsutum,G.arboreum and G.raimondii genome，HMMER and BLAST search were performed using 23 KCS genes from Arabidopsis and three from yeast as the query. A sum of 47,27,29 FAE1-type and 11,4, 4ELO-type KCS genes were found in G. hirsutum,G.arboreum and G. raimondii，respectively.Among all 58 KCS genes identified in the G. hirsutum genome,25 showed At origin and 33 Dt origin (Table S1).Most G. hirsutum KCS genes have no intron throughout their whole open reading frames,with only seven possessing one intron and two with two introns.

# Phylogenetic analysis of KCS genes

Two phylogenetic trees were independently constructed using full length amino acid sequences and the MEGA 5.0 software that employs the neighbor-joining method.All KCS proteins collected from the three cotton species plus the Arabidopsis genome fell in two distinct groups，namely the FAE1-type and ELO-type.FAE1-type KCS family proteins were further divided into eight subclasses: α, $\beta , \gamma , \delta , \varepsilon , \zeta , \eta$ and 0(Figure 1A) and the ELO-type KCS proteins were classified into four different subgroups: I,II, Ill and IV (Figure 1B). Subgroup m from the FAE1-type and subgroups Il, Ill,IV from ELO-type were only found in Gossypium,indicating that KCS genes in cotton might have undergone specific gene duplication and have more complicated functions than in Arabidopsis. Evolutionary analysis using19 species from lower aquatic to higher terrestrial plants showed that FAE1-type KCS first appeared in Volvox carteri and the number dramatically increased in Physcomitrella patens,then stayed relatively stable until the allotetraploid cotton evolved (Figure 1C). The ELO-type KCS was composed of only one to four genes in all otherspeciesanalyzed and itsnumber increased dramatically in G. hirsutum as well(Figure 1D).This finding is consistent with a whole-genome duplication (WGD) event in the Gossypium genus after its separation from T. cacao (Wang et al.2012; Li et al. 2014).

# Subgenome-specific transcription activation of 32 GhKCs genes

We further investigated the expression patterns of KCS genes between the allotetraploid and the corresponding diploid cottons using real-time quantitative polymerase chain reaction (qRT-PCR). Of the 58 analyzed KCS genes,32 genes showed subgenome-specific expression patterns (Figure 2). Furthermore,26 KCSs among the subgenome-specific genes were highly expressed in reproductive organs such as flowers and ovules (Figure 2). Compared to their orthologs from the DD genome,11 KCS genes from Dt subgenome expressed notably higher during various stages of fiber development. Three Dt-derived KCS genes showed flower-specific expression,while DtKCS4 and DtKCS8 were dramatically expressed in roots.DtKCS5 was expressed in a stem-specific manner when compared with its ortholog from the DD genome.By contrast, DDKCS9 was expressed in a highly leaf-preferential manner (Figure 2A). Eight of the At-derived KCS genes were markedly higher in different stages of fiber development compared with their orthologs in the AA genome.Four KCS genes that originated from the At subgenome were specifically expressed in flowersand two other KCS genes were significantly expressed in roots.AtkCS58 was highly expressed in leaves (Figure 2B).Transcripts of another 15 KCS genes either from the At or Dt subgenome showed no subgenome-specificity when compared to their corresponding diploid orthologs and 10 KCS genes were always expressed at extremely low levels in the tissues studied (Figure S1).Subgenome-specific high-level expression of various KCS genes may somehow be related to the successful development of longand spinnable fibers in the allotetraploid cotton.

# Potential regulatory mechanisms of cotton KCS genes

Sequence analysis showed that deletions of a 121-bp fragment on KCS55 and a 237-bp fragment on KCS43 promoter in the AA genome resulted in the loss of a few MYB binding sites (Figure 3A). Likewise,deletions of two large DNA fragments on KCS6 promoter from the DD genome resulted in the acquisition of two additional MYC-binding sites on DtKCS6 promoter (Figure 3A).These changes correlate with the low level expression both in G.arboreum and G.raimondii,when compared with what was found either in the At or the Dt subgenomes.Also，two additional MYC binding sites on AtKCS38 promoterand two new MYB binding sites on DtKCS1 promoter were generated potentially as results of point mutations (Figure 3B). The presence of additional or new transcription factor binding sites can be responsible for the highly fiber-specific expression of the Dt or At originated KCS genes.Also,we observed four additional GT-1 binding sites on AtkCS48 promoter (Figure 3B)，we suggest that they may facilitate the flower-specific expression from the At-subgenome gene.One each of the Copia element,which isa long terminal repeat(LTR) retrotransposon,was inserted in the promoters of AtKCS44 or DtKCS28 (Figure 3C).Further, depletion of two WRKY binding sites known to have inhibitory effects on target gene expression,on the DtkCS18 promoter region，may lead to its high-level flower-specific expression (Figure 3D).

![](images/9ae31b466a18114f06e71bfcfdbdec4c7f1ccbd47d705fa1eca172002cd83855.jpg)  
Figure1.Evolution and diversification of the KCS gene family among different organisms   
(A)Theunrooted phylogenetic treeofFAE1-type KCSattheaminoacid sequence levelusing theneighbor-joining method in MEGA 5.0.(B)Phylogenetic tree of ELO-type KCS proteins.The color keys are used universally between (A)and (B). (C) Comparisons ofFAE1-type KCS gene numbersacrossa wide range of organisms. (D) Comparisons of ELO-type KCS gene numbersacrossideangeofgaism.p,cromoaspusill;OstOstreococstauri;Volvcarteri;Ppitrell patens;S,ginleledfi;Abo;o;p;Csis;t Populustrichocarpa;Ca,Capsicumannuum;At,Arabidopsis thaliana;Mt,Medicago truncatula;Sb,Sorghumbicolor;Os,Orya SativaJaponica; Gm,Glycinemax; Hv,Hordeumvulgare;St,Solanum tuberosum;Zm,Zea mays;GaGossypiumarboretum;Gr, Gossypium raimondii; Gh,Gossypium hirsutum.

![](images/49e6943ccb3882bff83b07424fff40e84a5dc5e2620bebb67334bcbf079af371.jpg)  
Figure 2. Continued.

# Functional characterization of cotton KCS genes

We next constructed elo2△elo3△ double mutant strain carrying the pYES2-ScELO3 plasmid.We confirmed the double mutation by successful amplification of elo2:kanMX4 and elo3::natMX4 cassettes using genomic DNA prepared from the mutant (Figure 4A).Wild-type and all mutant strains were able to growon $" 2 \%$ galactose $+ 0 . 0 5 \%$ glucose"plates (Figure 4B, top left panel).The elo2△elo3△ double mutant was unable to grow on $2 \%$ glucose medium,owing to the suppression of ELO3 expression which was driven by galactose (Figure 4B,top right).Both the elo2 single mutant and the elo2△elo3△ double mutant were able to grow on geneticin (G418) due to the presence of the elo2:kanMX4 cassette (Figure 4B,mid left). On the same line,both elo3 and elo2△elo3△were able to grow on Clonat due to the presence of the elo3:natMX4 cassette (Figure 4B,mid right). Only the elo2△elo3△ double mutant was able to grow on ${ \mathsf { G } } 4 1 8 +$ Clonat plates (Figure 4B,bottom left).By contrast， the double mutant failed to grow on 5-fluoroorotic acid(FOA) medium as a result of the ejection of the pYES2 plasmid,while all other strains grew well(Figure 4B, bottom right)． Subsequently， two FAE1-type and four ELO-type KCSs from G. hirsutum were found to restore the viability of the double mutant (Figure 5A). KCS12, KCS14, KCS53 and KCS26 produced almost wild-type growth rate as observed on YPD solid plates (Figure 5B,left) and asobserved by continuous culture in liquid medium as well (Figure 5B, right).KCS12 and KCS26 genes showed fiber-specific expression and KCS18 was highly expressed in flower (Figure 2), indicating that these KCS genes potentially regulated cotton development.

Cells expressing ELO-type cotton KCS12,KCS14, KCS53 and KCS26 genes contained almost identical amounts of C26:o as compared with the wild-type (Figure 5C; Table S2), indicating that this subgroup may be directly involved in VLCFA biosynthesis up to C26:o,which is the longest fatty acid in yeast cells. Heterologous expression of KCS18 produced large amounts of C22:o，with undetectable levels of C24:o and C26:0.Cells rescued by KCS2o accumulated high levels of C24:0,with a negligible amount of C26:o (Figure 5C; Table S2). Considering their low complementation efficiency reported in Figure 5B and their VLCFA accumulation patterns reported in Table S2,we suggest that KCS18 and KCS2o are involved in synthesis of C22:o from C20:0 and C24:0 from C22:0, respectively.

# $\mathsf { \pmb { G } } \mathsf { \pmb { A } } _ { 3 }$ rapidly promotes the expression of many KCS and CesA genes during fiber development

When analyzing the promoter regions of all 19 KCS genes preferentially expressed during different fiber growth stages, 11 of them were found to contain GAREs with two of them showing L1 boxes as well (Figure 6A). qRT-PCR analysis showed that the transcript levels of eight cotton KCSs increased significantly shortly after application of $1 \mu \mathsf { M } \mathsf { G } \mathsf { A } _ { 3 }$ (Figure 6B).For KCS28 and KCS23 that have three consecutive GAREs on each promoter,approximately 3o-fold increases in transcript levels $2 4 \mathsf { h }$ after $\mathsf { G A } _ { 3 }$ application was recorded. For $K C S 1 3$ and $K C S 3 8$ that possessed one GARE and one L1 box on their promoters,about $2 0 - 3 0$ fold increases in transcript levelsafter $\mathsf { G A } _ { 3 }$ treatment were observed (Figure 6A,B). $\mathsf { G A } _ { 3 }$ also induced significant expression of KCS36 (with two GAREs)，KCS37，KCS40 and KCS1，which showed one GARE on each of the promoters at greater than 1kb from the putative translation start sites.Exogenous $\mathsf { G A } _ { 3 }$ failed to activate the transcription of KCS2，KCS35 and $K C S 3 0$ ，all possessed with one GARE on each of the promoters at less than 1kb from the putative translation start sites (Figure 6A，B).Further analysis showed that five of the cotton KCSs contained BR-insensitive-EMS-suppressor1(BES1) elements (Figure S2A) and two contained auxin responsive elements (AuxREs) (Figure S2C). Exogenous applications of BR and auxin induced the transcription of all but one of the KCS genes with corresponding elements (Figure S2B，D). GAREs and L1-boxes presented also on CesA gene promoters (Figure 6C). qRT-PCR experiments indicated that all four of these CesA genes were significantly upregulated by $\mathsf { G A } _ { 3 }$ (Figure 6D).Applications of $\mathsf { G A } _ { 3 }$ did not induce the expression of the other two major CesA genes without GARE or L1-box (Figure 6E). Exogenous C26:o resulted in significant transcript increases for two CesA genes $( C e s A 6 - 4$ and CesA6-7),whereas exogenous ethylene did not stimulate the expression of all CesA genes (Table S3).

# Deciphering the mode of GA actions

When C26:o,ethylene and $\mathsf { G A } _ { 3 }$ were applied to standard ovule culture media for 7 d，significant fiber elongation was observed (Figure 7A). Surprisingly，only exogenous $\mathsf { G A } _ { 3 }$ resulted in significant fiber dry weight increase (calculated

(A) Seventeen KCS genes from the Dt subgenomes were expressed higherin various tissues when compared to their correspondingdiploidorthologs.Blackcolumns indicatetheexpressionofgenefromtheDgenome.Slashcolumnsindicate the expresion of genefromthe Dtsubgenome.(B)Fifteen KCS genes from theAtsubgenome were expressed higher in various tissues whencompared to teir corresponding diploid orthologs.Black columns indicate the expression of gene from the A genome.Slash columns indicate the expression of gene from the At subgenome.qRT-PCR analyses were performed in triplicateusing diferentcottonmaterialsanderorbars inthis figurerepresent the SDfrom threeindependent experiments.

at the per cm bases),whereas C26:o treatment showed a slight increase of the dry weight and ethylene treated ovules produced similar to untreated wild-type levels (Figure 7B). Fiber cell wall thickness and fiber diameters were measured under an electron microscope using semi-thin sections prepared from fiber cells closest to the ovule epidermal layer. Again,fiber cells harvested from $\mathsf { G A } _ { 3 }$ treated ovules showed most significant cell wall growth,as indicated by both cell wall thicknessand by the diameter of each fiber,followed by C26:0 that stimulated cell wall growth to some degree (significant at the $0 . 0 5 \%$ level) (Figure 7C). Fibers from ethylene treated ovules showed no significant increase in cell wall thickness (Figure 7C).These changes were consistent with significant increase of cellulose content in fibers from $\mathsf { G A } _ { 3 }$ treated ovules, amoderate increase after C26:o treatment and no increase after ethylene treatment (Figure 7D).

# DISCUSSION

6   
A   
AAKCS55 中 DATG -2,000 -1,62-1,502 -1,000 +1   
AtKCS55 Ⅲ T DATG -2.000 -1,000 I MYB site +1   
AAKCS43 □ X Y ATG -2,000 -1,000 -656 -419 +1   
AtKCS43 直 DATG -2.000 -1.000 I MYB site +1   
DDKCS6 T 1 DATG -2.000 -1,311 -1,000 （204号 $+ 1$ （2   
DtKCS6 T 工 H DATG -2,000 -1,000 I MYC site +1   
B   
AAKCS38 DATG T -2.000 -1,000 （204号 $+ 1$   
AtKCS38 == 干 DATG -2.000 -1,000 I MYC site +1   
DDKCS1-2.00 -1.000 ATG   
DtKCS1 X 工 DATG -2,000 -1,000 I MYB site +1   
AAKCS48 Y 二 DATG -2,000 -1,000 +1   
AtKCS48（ 二 共 日 DATG -2,000 -1,000 GT-1 site +1   
C   
AAKCS44 DATG -2,000 -1,000 +1   
AtKCS44 T T DATG -2.000 -1,350 -1,000 Copia -299 +1   
DDKCS28 十 DATG -2,000 -1,000 +1   
DtKCS28 DATG -2,000 -1,435 -1,000 -770 copia +1   
D   
DDKCS18 甘 T ATG T -2,000 -1,000 +1   
DtKCS18 0 ATG -2.000 -1,000 I WRKY site +1

The genome of Arabidopsis contains 21 FAE1-type KCS genes and two ELO-type genes (Costaglioli et al. 2oo5). In yeast cells, only three ELO-type KCS genes have been found so far (Toke and Martin 1996). Here 47 FAE1-type KCS and 11 ELO-type KCS in G.hirsutum,27 FAE1-type KCS and four ELO-type KCS in G.arboreum,and 29 FAE1-type KCS and four ELO-type KCS in G.raimondii were identified using comparative genomics approaches (Table S1). Since four new subclasses (m, Il, Ill and IV) and many more KCS gene family members exist in various cotton species,we suggest that additional KCS functions may be required for this genus and gene duplication events occurred in this plant after its separation from T.cacao.

Thirty-two of the $5 8 ~ G h K C S$ genes displayed subgenomespecific transcription activation over a wide range of tissues studied,with only one KCS gene expressed at higher level from G. raimondii (the DD genome) in leaves (Figure 2). Interestingly enough,19 of these GhKCS genes were activated at the transcription level during various fiber developmental stages,which indicates that the allotetraploidization event plays an important role in regulating KCS gene expression and cotton fiber elongationas well. Evolution of more cis-elements and insertion of retrotransposons in nine KCS promoter regions of At or Dt subgenomes seemed to have contributed to their highly specific expressions (Figure 3).These results may reflect the necessity of KCS genes for proper fiber development in the allotetraploid species. Also, the presence of extra regulatory elements in promoter regions of functionally important genes may be considered resultant of different selection pressures among different cotton species,which was consistent with the previous notion that the allotetraploid genome enjoyed lower selection pressure than the diploid genomes (Li et al.2015).

![](images/52b4d2c4954fa75cc3b7a75e5fd7d0372f3c0a89d1cf875db166c744b45f54b6.jpg)  
Figure 4. Continued.   
Figure4.Preparationsof Saccharomyces cerevisiae haploid elo2Δelo3△ deletion strain over-expressing ELO3

Four of the Arabidopsis FAE1-type KCS proteins were able to rescue the lethality of elo2△elo3△ yeast double mutant (Trenkamp 2oo4; Paul etal.2oo6). Complete loss of KCS20 and KCS2 decreased the total wax content (Lee et al.2oo9) and CER6/CUT1 mutation affected lipid content on the surface of pollen and stamens in this plant (Fiebig etal.2ooo). Currently we find that two FAE1-type and four ELO-type KCSs from G.hirsutum were able to substitute the yeast Elo2p/Elo3p proteins for biosynthesis of VLCFA.Mutant yeast cells expressing four of the ELO-type genes including KCS12, KCS14,KCS53 and KCS26 resulted in close-to-wild-type growth rate,whereas markedly lower efficiency was found for cells expressing KCS18 or KCS2o，both FAE1-types (Figure 5B). Accordingly，wild-type levels of C26:o were observed from mutant cells carrying all four ELO-type genes，with cells expressing KCS18 or KCS2o accumulating relatively large amounts of C22:o or C24:0,respectively (Figure 5C),suggesting that cotton KCS proteins have discrete substrate specificities.

In conclusion,we produced a binary mode of GA action as depicted in Figure 7E.GAs may act upstream of the VLCFAethylene pathway via activation of a large set of GhKCS genes (either At or Dt origin) (Figure 6B).GAs may function concomitantly in downstream steps that resulted in cellulose biosynthesis by promoting CesA expressions (Figure 6D). Clarification of this binary mode of GA action,in combination with progress in cotton genomics，may help elucidate molecular mechanisms controlling cotton growth and development.

# MATERIALS AND METHODS

Materials and growth condition

Gossypium hirsutum cv.Xuzhou 142,Gossypium arboreum cv. Shixiya1 (SXY1) and Gossypium raimondii acc.D5-3 (CMD10)

(A) PCR experiments to verify the presence of elo2:kanMX4 or elo3::natMX4 cassettes using genomic DNA isolated from elo2△，elo3△ or the elo2△elo3△ double mutant strains. Primers specific to ELO2 promoterand kanMX4 were designed as SP2 and AP2 (See Table S4 for primer sequences); Primers specific to ELO3 promoterand natMX4 were designed and $\mathsf { S P 3 }$ and AP3.(B) Validating the prepared mutant strains on solid plates containing various chemicals or antibiotics,including Gal (galactose),Glu (glucose),G418,Clonat or FOA.

![](images/671d06c3fddaa2f9842f25e138581a1994d172d3f32768afe88eaa52f304f902.jpg)  
Figure 5. Continued.

were grown in a climate-controlled greenhouse with a 16 h light and $^ { 8 \mathfrak { h } }$ dark cycle at $2 5 ^ { \circ } C$ (Liu et al.2015). For in vitro ovule culture,1 day post-anthesis (DPA) fresh ovules were excised from bolls on cotton plant and cultured directly on the surface of the liquid media (Shi et al.2oo6). For RNA extraction,ovules collected from different growth stages and various tissues were frozen and stored in liquid nitrogen immediatelyafter harvest.

# Phylogenetic and evolution analysis

The sequence data used in this study were either sequenced asdescribed above or collected from the National Center for Biotechnology Information (NCBl).A phylogenetic tree of deduced amino acid sequences was constructed using the neighbor-joiningalgorithmwithdefault parametersinMEGA 5.0 (www.megasoftware.net). Multiple sequence alignments of the nucleotide were performed using ClustalX(Thompson et al. 1997).

# RNA extraction and qRT-PCR analysis

Cotton ovules harvested at the indicated time were first frozen in liquid nitrogen before being ground to fine powder with a mortar and pestle using a modified hot borate method (Ji et al. $2 0 0 3 \mathrm { ; }$ ：Jin et al.2013).Total RNA was extracted from ovules of three cotton species and cDNA was reversetranscribed from $5 \mu \mathrm { g }$ of total RNA as reported(Shi et al.2006; Jin et al.2o13).For chemical treatment experiments,RNA samples were prepared from three independent ovule cultures in the presence of $1 \mu \mathrm { M } \mathsf { G A } _ { 3 } , 5 \mu \mathrm { M } \mathsf { C } 2 6 { : 0 }$ or without chemical supplementation (CK) for the indicated time.Primers for qRT-PCR analysis were listed in Table S4. Cotton UBQ7 was used as the internal control forall related figures.

# In vitro ovule culture

Cotton bolls at 1 DPA were collected and sterilized with $10 \%$ sodium hypochlorite for 15 min.Ovules were excised from bollsand placed in liquid medium as described previously (Shi et al. 2006). $1 \mu \mathsf { M } \mathsf { G } \mathsf { A } _ { 3 }$ or $5 \mu \mathrm { M }$ C26:0 was added to the medium and ovules were cultured at $3 0 ^ { \circ } C$ in darkness for the indicated time without agitation.2.2 ppm ethylene was added in the head space of the flask used for ovule culture experiments as described previously (Shi et al.2oo6).The cultured ovuleswere immediatelycollected forRNAextraction.

trp1-1,ura3-1,elo3::natMX4,elo2::kanMX4)，elo3::natMX4 was amplified using genomic DNA prepared from BY4741 elo3△ strain (Mat a; his3 $\varDelta$ ，leu2△，met15△，ura3△，elo3:: kanMX4）and subsequently transformed into S.cerevisiae W303-1A elo2△ strain (Mat a; ade2△,can1-10o,his3-11, leu2-3, trp1-1,ura3-1,elo2:kanMX4) carrying the pYES2-ScELO3 vector. The elo2△elo3 $\varDelta$ double mutant cells containing pYES2-ScELO3 were selected on the plate supplemented with $2 0 0 ~ { \mu \mathrm { g / m L } }$ G418， $1 0 0 ~ { \mu \mathrm { g / m L } }$ Clonat and $2 \%$ (w/v）galactose，and subsequently transformed with individual KCS gene ligated into a ${ \mathsf { T R P 1 } }$ -marked pYADE4 under the control of the ADH promoter.The transformants were selected on synthetic complete medium lacking tryptophan (SC-Trp） plates but containing $0 . 1 \%$ FOA.Cells were plated by 1o-fold dilutionsand were grown on the medium for 2 days at $3 0 ^ { \circ } C$ before photographswere taken.

# Fattyacid extractions and GC-MS analysis

One gram of fresh weight of wild type cells or elo2△elo3△ double deletion mutant cells complemented by individual cotton KCSs were ground to a fine powder with a mortar and pestle.Fatty acid were extracted by sulfuric acid: methanol (1:4o,v/v) and derivatized by heating to $8 5 ^ { \circ } C$ for $2 h$ (Browse et al.1986).After cooling to room temperature, fatty acid methyl esters were extracted three times with hexane,dried under nitrogen gas and then concentrated to $0 . 5 \mathsf { m L }$ . C17:0 fatty acid (heptadecanoic acid; Sigma-Aldrich), which does not exist in plants,was added before extraction to monitor sample recovery and also for quantitative analysis. $1 \mu \ L$ of each sample was analyzed on the Agilent 6890N GC system with the DB-225MS column (J&W) that operated with hydrogen carrier gas and a splitless inlet (injection temperature, $2 8 0 ^ { \circ } C )$ ,according to the manufacturer's instructions.All fatty acids and hydroxylated fatty acids were identified by the Agilent 68goN GC system coupled to an HP 5973 mass selective detector using both the National Institute of Standards,and Technology and Wiley databases.

# Analysis of Cellulose Content

Five milligram fiber cells separated from cultured ovules were ground to a fine powder with a mortar and pestle.Samples were treated with acetic nitric reagent for $3 0$ min at $1 0 0 ^ { \circ } \mathsf { C }$ to obtain the insoluble residues.The pellets were washed with distilled water twice and hydrolyzed with $67 \%$ sulfuric acid. The cellulose content was quantified with anthrone reagent and suitable spectrophotometer at a wave-length of $6 2 0 \mathsf { n m }$ against a reagent blank.

(A)elo2Aelo3△double mutant yeastcels regain their abilityto grow onsolidFOA mediumonly whentransformed with the identifiedGhKCS genes (labeledbeloweachoftheplates).WT,wild-typeyeaststrain W30-1Ausedforgenetictransformations. (B)Detailedgrowthperformanceanalysisofeloelo3△doublemutantyeastcellines expressngvariouscottonKCSgeneswith comparisonstoWT.Left,Aseriesof10-folddilutionthat showedthedensityofthecolonies expressng diferentcotton KCS genes onsolidFOA media; Right,Growth curvesof the same yeast celines grown inliquid FOA media.(C)GC-MSanalyses of totalfattyacids extractedfromWTyeastcelsortheelo△elo3doublemutantcarryingvariouscotonKCSgenes.Theeffiency oftheextraction processwas monitoredbyexternaladditionoffatyacidC17:0andweadjustedthefinalsamplingquantityfor eachcelline acordingly.The experimentwasrepeatedthree times with independentsamplesandonlyone representative graph was shown for each treatment.

![](images/59b77795fdc7efbc380d961cc1a8ec3fb5cb78df762d43833150199ddd4bd37f.jpg)  
Figure 6. Exogenous gibberellin $\pmb { A } _ { 3 }$ 1 $( \mathsf { G A } _ { 3 } )$ promotes the transcription of several GhKCSs and GhCesAs with GAREs on their promoters

(A)Analysis of GhKCSswith GAREsandL1boxes present in their promoterregions. (B)GAactivated significanttranscriptionof most $G h K C S$ genes with GAREs and L1boxes in their promoter regions.(C) Analysis of GhCesAs with GAREs and L1boxes present in their promoterregions.GAREs(marked by green color)and L1-box(blue color)were predicted using the PLACEand MEME softwares. (D) $\mathsf { G A } _ { 3 }$ promoted significant transcription activation of all four GhCesA genes with GAREs and L1 boxes. (E) The expression of the other two major CesA genes without L1box or GARE were not induced by $\mathsf { G A } _ { 3 }$ .Relative expression levels were determinedafter normalizing alldata to thatofCKovules,whichwassetto1.0.Statisticalsignificance was determined using one-way analysis of variance software combined with Tukey's test. $^ { * } P < 0 . 0 5$ ： $^ { * * } P < 0 . 0 1$ $^ { * * * } P < 0 . 0 0 1$ ：

# Electron Microscopic analyses of fiber cell wall thickness

Electron microscopic analyses were conducted essentially as previously described (Pugh et al.2o1o).Wild-type cotton ovules harvested at1 DPA were cultured for7 d with either $1 \mu \mathsf { M } \mathsf { G A } _ { 3 } , 5 \mu \mathsf { M }$ C26:0 or 2.2 ppm ethylene added to the media. Cotton fibers from various treatments were fixed for $1 2 \mathsf { h }$ in $2 . 5 \%$ glutaraldehyde.After three washes with distilled and deionized water, the fibers were incubated in $1 . 5 \%$ osmium tetraoxide for $4 \ h$ at room temperature. Samples were dehydrated in a series of ethanol $( 5 0 \% - 1 0 0 \% )$ at 30min intervalsand epoxy ethaneand then embedded in Epon resin for $4 8 \mathsf { h }$ at $6 0 ^ { \circ } \mathsf { C } .$ Fibers at the base of the ovule were cut into $5 0 \mathsf { n m }$ thin sections,stained with uranyl acetate and lead citrate,then photographed under a transmission electron microscope (Tecnai G2 2o TWIN,FEI).

![](images/dbd2b19b3d77e39e2ad3c59368bcc180aba0432951a615661fabbabd17045d89.jpg)  
Figure 7. Continued.   
Figure7. Depicting the mode of gibberellin A(GA) action in an elongating cotton fiber cell

# Dry weight measurements

Wild-type cotton ovules were cultured as specified in the previous subsection for7 d before harvest.The ovules were first rinsed in distilled and deionized water, heated to $1 0 0 ^ { \circ } \mathsf { C }$ for $5 \min$ ，transferred to $70 \%$ ethanol after cooling to room temperature and dried.The number of fibers (N) from each ovule treated with different chemicals was counted as described (Zhang et al.2o11) and fibers were removed from ovule surface for weight measurements (W1).According to the fiber length (L)，dry weight per unit cell length was determined by the formula: $W { = } W \imath / ( N { \times } L )$ ：

# ACKNOWLEDGEMENTS

We thank Dr.Yongmei Qin and Dr. Yingchun Hu for their constructive suggestions to this work. This work was supported by grants from the China National Basic Research Program (2o1oCB126ooo） and the National Natural Science Foundation of China (9o717009).

# AUTHOR CONTRIBUTIONS

G.X.,G.H.and Y.Z. designed the phylogenetic analysis. G.X., K.W.and G.H. prepared RNA samples and performed PCR analysis.G.X.and G.H.performed complementary experiments in yeast.G.X.,K.W.and G.H.performed the observation of cell wall.G.X. and K.W.wrote the manuscript.Y.Z. conceived and directed the project.

(A) Exogenous applications of ${ \mathsf { G A } } _ { 3 } ( 1 \mu \mathsf { M } ) , { \mathsf { C } } 2 6 { : 0 } \left( 5 \mu \mathsf { M } \right)$ and ethylene (2.2 ppm) promoted significant cotton fiber elongation.Cotton ovules were harvested at 1 DPA and cultured for 7 d before being fixed and measured under a light microscope. (B) Comparisons of dry fiber weight from ovule cultures after the same treatment as in (A).The experiment was repeated three times with five ovules each time and reported as mean $\pm S D$ for both (A) and (B).(C) Comparisons of cell wall thicknessamong fibers after the same treatment as in (A). CWT,cell wall thickness in $\mu \mathsf { m }$ ；DIA,diameterof fibercells in $\mu \mathrm { m }$ .Thirty fibers from three ovules were measured for each sample and each fiber was measured three times from different positions.The experiment was repeated three times and reportedas mean $\pm { \mathsf { S D } }$ .Scalebars $= 2 \mu \mathrm { m }$ (D)Measurement of fiber cellulose content from ovule cultures after the same treatment as in (A).The experiment was repeated three timesand reported asmean $\pm S D$ (E)Proposed binarymodel of GA action during cotton fiber elongation.Statistical significance was determined using one-way analysis of variance software combined with Tukey's test. $^ { * } P < 0 . 0 5$ $^ { * * } P < 0 . 0 1$ $^ { * * * } P < 0 . 0 0 1$ ：

REFERENCES   
Browse J, McCourt PJ,Somerville CR (1986) Fatty acid composition of leaf lipids determined after combined digestion and fatty acid methyl ester formation from fresh tissue.Anal Biochem 152: 141-145   
Costaglioli P,JoubésJ,Garcia C,Stef M,Arveiler B,Lessire R,Garbay B (2005) Profiling candidate genes involved in wax biosynthesis in Arabidopsis thaliana by microarray analysis.Biochim Biophys Acta 1734:247-258   
Denic V，Weissman JS (2oo7） Molecular caliper mechanism for determining very-long chain fatty acid length. Cell13o: 663-677   
Dittrich F,Zajonc D,Huhne K,Hoja U, Ekici A,Greiner E,Klein H, Hofmann J, Bessoule JJ,Sperling P,Schweizer E(1998) Fatty acid elongation in yeast-biochemical characteristics of the enzyme system and isolation of elongation-defective mutants.Eur J Biochem 252: 477-485   
Dunn TM,Lynch DV, Michaelson LV,Napier JA (2oo4) A post-genomic approach to understanding sphingolipid metabolism in Arabidopsis thaliana. Ann Bot 93: 483-497   
Fehling E, Mukherjee KD(1991) Acyl-CoA elongase from a higher plant (Lunaria annua):Metabolic intermediates of very-long chain acylCoA products and substrate specificity.Biochim Biophys Acta 1082: 239-246   
Fiebig A,Mayfield JA,Miley NL,Chau S,Fischer RL,Preuss D (2000) Alteration in CER6,a gene identical to CUT1,differentially affect long chain lipid content on the surface of pollen and stems.Plant Cell 12: 2001-2008   
Franke R, Hofer R, Briesen I, Emsermann M,Efremova N, Yephremov A,Schreiber L(2oo9) The DAISY gene from Arabidopsis encodes a fatty acid elongase condensing enzyme involved in the biosynthesis of aliphatic suberin in roots and the chalaza-micropyle region of seeds. Plant J 57: 80-95   
Gou JY,Wang L,Chen SP, Hu WL,Chen XY (2oo7) Gene expression and metabolite profiles of cotton fiber during cell elongation and secondary cell wall synthesis. Cell Res 17: 422-434   
Huang DB,Wang SG,Zhang BC,Shang-Guan KK,Shi YY,Zhang DM, Liu XL,Wu K,Xu ZP,Fu XD,Zhou YH(2o15)A gibberellins-mediated DELLA-NAC signaling cascade regulates cellulose synthesis in rice. Plant Cell 27: 1681-1696   
Hooker TS, Millar AA, Kunst L(2oo2) Significance of the expression of the CER6 condensing enzyme for cuticular wax production in Arabidopsis.Plant Physiol 129: 1568-1580   
James DW,Lim E,Keller J,Plooy I，Ralston E,Dooner HK (1995) Directed tagging of the Arabidopsis FATTY ACID ELONGATION1 (FAE1) gene with the maize transposon activator. Plant Cell 7: 309-319   
Joubés J，Raffaele S,Bourdenx B,Garcia C, Laroche-Traineau J, Moreau P, Domergue F,Lessire R (2oo8) The VLCFA elongase gene family in Arabidopsis thaliana:Phylogenetic analysis, 3D modeling and expression profling. Plant Mol Biol 67: 547-566   
Ji SJ,Lu YC, Feng JX,Wei G,Li J,Shi YH,Fu Q, Liu D, Luo JC, Zhu YX (2003) Isolation and analyses of genes preferentially expressed during early cotton fiber development by subtractive PCR and CDNA array. Nucleic Acids Res 31: 2534-2543   
Jin X,Li Q, Xiao GH,Zhu YX(2013) Using genome-reference expressed sequence tag assembly to analyze the origin and expression patterns of Gossypium hirsutum transcripts.JInter Plant Biol 55: 576-585   
Kim HJ,Triplett BA (2oo1) Cotton fiber growth in planta and in vitro. Models for plant cell elongation and cell wall biogenesis.Plant Physiol 127: 1361-1366 developing seeds of Arabidopsis thaliana.Plant Physiol 30: 425-434   
Lassner MW,Lardizabal K,Metz JG (1996) A jojoba $\beta$ -ketoacyl-CoA synthase cDNA complements the canola fatty acid elongation mutation in transgenic plants. Plant Cell 8: 281-292   
Lee SB,Jung SJ,Go YS,Kim HU, Kim JK,Cho HJ,Park OK,Suh MC (2009）Two Arabidopsis 3-ketoacyl CoA synthase genes,KCS20 and KCS2/DAISY,are functionally redundant in cuticular wax and root suberin biosynthesis, but differentially controlled by osmotic Stress. Plant J 6o: 462-475   
Li F, Fan G,Lu C, Xiao G, Zou C, Kohel RJ, Ma Z, Shang H, Ma X,Wu J, LiangX,HuangG,Percy RG,LiuK,YangW,Chen W,DuX,ShiC,Yuan Y,Ye W,Liu X,Zhang X,Liu W,Wei H,WeiS, Huang G,Zhang X,Zhu S, Zhang H,Sun F,Wang S,Liang J,Wang J, He Q, Huang L,Wang J, Cui J, Song G,Wang K, Xu X,Yu JZ, Zhu Y,Yu S (2015) Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution. Nat Biotech 33: 524-530   
Li F,Fan G,Wang K,Sun F,Yuan Y,Song G, Ma Z,LiQ,Lu C,Zou C,Chen W,Liang X,Shang H, Liu W,Shi C, Xiao G, Gou C, Ye W, Xu X, Zhang X,Wei H,Li Z,Zhang G,Wang J,LiuK, Kohel R,Percy R,Yu J,Zhu Y, Wang J, Yu S (2o14) Genome sequence of the cultivated cotton Gossypium arboreum. Nat Genet 46: 567-572   
Li Q, Xiao GH, Zhu YX (2014) Single-nucleotide resolution mapping of the Gossypium raimondii transcriptome reveals a new mechanism for alternative splicing of introns. Mol Plant 7: 829-840   
Liu GJ,Xiao GH,Liu NJ,Liu D,Chen PS,Qin YM,Zhu YX(2015)Targeted lipidomics studies reveal that linolenic acid promotes cotton fiber elongation by activating phosphatidylinositol and phosphatidylinositol monophosphate biosynthesis. Mol Plant 8: 911-921   
Lolle SJ,Berlyn GP,Engstrom EM,Krolikowski KA,Reiter WD,Pruitt RE (1997）Developmental regulation of cell interactions in the Arabidopsis fiddlehead-1 mutant:A role for the epidermal cell wall and cuticle. Dev Biol 189: 311-321   
Mei WQ,Qin YM,Zhu YX(2o1o) Cross-talk among ethylene,very longchain fatty acids,reactive oxygen species,brassinosteroid and gibberellins mediates cotton fiber elongation.Chin Bull Life Sci 22: 7-14   
Millar AA,Kunst L (1997) Very-long-chain fatty acid biosynthesis is controlled through the expression and specificity of the condensing enzyme. Plant J12: 121-131   
Oh CS，Toke DA，Mandala S，Martin CE(1997） ELO2 and ELO3, homologues of the Saccharomyces cerevisiae ELO1 gene,function in fatty acid elongation and are required for sphingolipid formation. J Biol Chem 272: 17376-17384   
Pang CY,Wang H, Pang Y,Xu C,Jiao Y,Qin YM,Western TL,Yu SX, Zhu YX (2010) Comparative proteomics indicate that biosynthesis of pectic precursors is important for cotton fiber and Arabidopsis root hair elongation. Mol CellProteomics 9: 2019-2033   
Paterson AH,Wendel JF,Gundlach H,Guo H,JenkinsJ,Jin D,Llewellyn D,Showmaker KC,Shu S, UdallJ,Yoo MJ,Byers R,Chen W, DoronFaigenboim A, Duke MV,Gong L, Grimwood J,Grover C, Grupp K, Hu G,Lee TH, Li J, Lin L,Liu T,Marler BS,Page JT,Roberts AW, Romanel E,Sanders W,SzadkowskiE,TanX,TangH,XuC,WangJ, Wang Z,Zhang D,ZhangL,Ashrafi H, Bedon F,Bowers JE, Brubaker CL,Chee PW, Das S,Gingle AR, Haigler CH, Harker D, Hoffmann LV, Hovav R,Jones DC,Lemke C,Mansoor S,ur Rahman M, Rainville LN,Rambani A,Reddy UK,Rong JK,Saranga Y,Scheffler BE, Scheffler JA,Stelly DM,Triplett BA,Van Deynze A,Vaslin MF, Waghmare VN,Walford SA,Wright RJ,Zaki EA,ZhangT,Dennis ES, Mayer KF,Peterson DG,Rokhsar DS,Wang X, Schmutz J. (2012) Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibers. Nature 492: 423-427

Paul S,Gable K, Beaudoin F,Cahoon E,Jaworski J,Napier JA,Dunn TM (2006）MemDers Ot tne ArabiaopsIs FAE1-like 3-ketoacyl-COA synthase gene family substitute for the Elop proteins of Saccharomyces cerevisiae.J Biol Chem 281: 9018-9029

Pruitt RE,Viele-Calzada JP,Ploense SE,Grossniklaus U,Lolle SJ(2000) FIDDLEHEAD，a gene required to suppress epidermal cel interactions in Arabidopsis,encodes a putative lipid biosynthetic enzyme. Proc Natl Acad Sci USA 97: 1311-1316   
Pugh DA,Offler CE,Talbot MJ, Ruan YL(2o1o) Evidence for the role of transfer cells in the evolutionary increase in seed and fiber biomass yield in cotton. Mol Plant 3: 1075-1086   
Qin YM,Hu CY, Pang Y, Kastaniotis AJ,Hiltunen JK,Zhu YX (2007) Saturated very-long-chain fatty acids promote cotton fiber and Arabidopsis cell elongation by activating ethylene biosynthesis. Plant Cell 19: 3692-3704   
Qin YM,Pujol FM,ShiYH,Feng JX,Liu YM,Kastaniotis AJ, Hiltunen JK, Zhu YX(2oo5） Cloning and functional characterization of two cDNAs encoding NADPH-dependent 3-ketoacyl-CoA reductase from developing cotton fibers. Cell Res 15: 465-473   
Qin YM,Zhu YX (2o11) How cotton fibers elongate: A tale of linear cellgrowth mode. Curr Opin Plant Biol 14: 106-111   
Rossak M,Smith M,Kunst L (2oo1) Expression of the FAE1 gene and FAE1 promoter activity in developing seeds of Arabidopsis thaliana. Plant Mol Biol 46:717-725   
Riezman H (2oo7) The long and short of fatty acid synthesis.Cell 130: 587-588   
Seagull RW,Giavalis (2oo4） Pre- and post-anthesis application of exogenous hormones alters fiber production in Gossypium hirsutum L. cultivar Maxxa GTO.J Cotton Sci 8: 105-111   
Shan CM, Shang XX, Zhao B, Zhang XF,Chao LM,Yang CQ,Wang , Zhu HY, Zeng YD,Guo WZ,Zhou BL,Hu GJ,Guan XY,Chen ZJ, Wendel JF,Zhang TZ, Chen XY (2014) Control of cotton fibre elongation by a homeodomain transcription factor GhHOX3. Nat Commun 5: 5519   
Shang H,Li W, Zou C, Yuan L(2o13) Analyses of the NAC transcription factor gene family in Gossypium raimondii Ulbr: Chromosomal location,structure,phylogeny,and expression patterns. J Inter Plant Biol 55: 663-676   
Shi YH,Zhu SW, Mao XZ,Feng JX, Qin YM,Zhang L, Cheng J, Wei LP, Wang ZY，Zhu YX (2oo6） Transcriptome profiling，molecular biological，and physiological studies reveal a major role for ethylene in cotton fiber cell elongation.Plant Cell 18: 651-664   
Thompson JD,GibsonTJ,Plewniak F,Jeanmougin F,Higgins DG(1997) The Clustal_X windows interface: Flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res 25: 4876-4882   
Toke DA,Martin CE (1996) Isolation and characterization of a gene affecting fatty acid elongation in Saccharomyces cerevisiae.JBiol Chem 271: 18413-18422   
Trenkamp S,Martin W,Tietjen K(2oo4）Specific and differential inhibition of very-long-chain fattyacid elongase from Arabidopsis thaliana by different herbicides.Proc Natl Acad Sci USA 101: 11903-11908   
Voisin D，Nawrath C，Kurdyukov S，Franke RB，Reina-Pinto JJ, Efremova N,Will I,Schreiber L,Yephremov A (2oo9) Dissection of the complex phenotype in cuticular mutants of Arabidopsis reveals a role of SERRATE as a mediator. PLoS Genet 5: e1000703   
Wang K, Wang Z, Li F, Ye W, Wang J,Song G, Yue Z, Cong L, Shang H, Zhu S,Zou C, Li Q, Yuan Y, Lu C,Wei H, Gou C, Zheng Z, Yin Y, Zhang X, Liu K,Wang B,Song C,Shi N,Kohel RJ,Percy RG, Yu JZ, Zhu YX,Wang J,Yu S (2o12) The draft genome of a diploid cotton Gossypium raimondii. Nat Genet 44: 1098-1103   
Wang S,Wang JW,Yu N,LiCH,Luo B,Gou JY,WangLJ,Chen XY(2004) Control of plant trichome development bya cotton fiber MYB gene. Plant Cell 16: 2323-2334   
Xu JY,Carlsson AS,FrancisT,Zhang M,HoffmanT,Giblin ME,Taylor DC(2012）Triacylglycerol synthesis by PDAT1 in the absence of DGAT1 activity is dependent on re-acylation of LPC by LPCAT2. BMC Plant Biol 12: 4   
Yang Samuel S,Cheung F,Lee JJ,Ha M,Wei NE,Sze SH,Stelly DM, Thaxton P，Triplett B，Town CD，Jeffrey Chen Z (2006) Accumulation of genome-specific transcripts,transcription fac tors and phytohormonal regulators during early stages of fiber cell development in allotetraploid cotton. Plant J47: 761-775   
Yephremov A,Wisman E,Huijser P,Huijser C,Wellesen K,Saedler H (1999) Characterization of the FIDDLEHEAD gene of Arabidopsis revealsa link between adhesion response and cell differentiation in the epidermis.Plant Cell 11: 2187-2201   
Zhang M, Zheng X,Song S, Zeng Q, Hou L,Li D, Zhao J,WeiY, Li X, Luo M,Xiao Y,Luo X,Zhang J,Xiang C,PeiY(2o11) Spatiotemporal manipulation of auxin biosynthesis in cotton ovule epidermal cells enhances fiber yield and quality.Nat Biotech 29: 453-458   
Zhu YX,Li FG (2o13) The Gossypium raimondii genome,a huge leap forward in cotton genomics.JInter Plant Biol 55: 570-571   
Zou C,Lu C,Shang H,Jing X,Cheng H,ZhangY,Song G(2o13) Genomewide analysis of the Sus gene family in cotton.JInter Plant Biol 55: 643-653

# SUPPORTING INFORMATION

Additional supporting information may be found in the online version of thisarticle at the publisher's web-site.   
Figure S1. Expression profiles of the other 25 KCS genes between the allotetraploid and the corresponding diploid genomes   
(A) Transcripts of 15 KCS genes from the At or Dt subgenomes showed no genome- or subgenome-specificity when compared to their corresponding diploid orthologs in the tissues studied.(B) Ten KCS genes always expressed at almost undetectable levels in different cotton genomes or subgenomes in the tissues studied.R,root; S,stem; L,leaf; F,flower. Figure S2. Exogenous BR and NAA promoted the expression of several GhKCS Genes   
(A) Analysis of five GhKCSs with BES1 elements present in their promoter regions.(B) BR promoted significant transcription activation to four of these GhKCSs.(C) Analysis of two GhKCSs with AuxRE element present in their promoter regions. (D) NAA promoted significant transcription activation to these GhKCSs.Sequences highlighted in blue boxes are BES1element and red boxesare AuxRE element predicted by the PLACEand MEME software.Relative expression levelsweredetermined after normalizing all data to that of CK ovules,which was set to 1.0. $^ { * } P < 0 . 0 5$ ： $^ { * * } P < 0 . 0 1$ ： $^ { * * * } P < 0 . 0 0 1$ ：   
Table S1. Analysis of Gossypium hirsutum KCS gene family and its orthologs in AA and DD cotton genomes   
Table S2.Quantitative analysis of fatty acid composition from wild-type and elo2△elo3△ double mutant yeast cells genetically complemented with different cotton KCSs as reported in Figure 5   
Table S3.Relative expression of CesA genes in ovulesafter application of C26:o (A) and ethylene (B)   
Table S4.A list of primers used in qRT-PCR experiments