# Subtractive SELEX using agar beads for screening DNA aptamers with specific affinity to HIV gp41 antigen

LI $\mathbf { K u n ^ { 1 * } }$ , XIUChenlin', GAOLiming²\*, SHI Ming', ZHAI Yue3

DepartmentofiologicalEngineeringCollgeofironment&emicalEngineering'Departmentoftenaledicinesn UniversityHospitalsnUesitindona;epmntoflstple'sspitalof Qinhuangdao 066004, China

Abstract: Objective To obtain DNA aptamers with a highly specific afinity to HIV $\mathrm { g p 4 1 }$ antigen using SELEX screening for detection of HIV. Methods The specific DNA aptamers of HIV gp41 antigen were screened from the double-stranded DNA derived from the single-stranded DNA (ssDNA) library with agarose beads as the supportive medium and HIV $\mathrm { g p 4 1 }$ antigen as the target molecule using SELEX techniqueand real-time quantitative PCR.Results The secondary ssDNAlibrary obtained after 6 rounds of screening was amplified by PCR to obtain dsDNA. The dsDNA was linked with $\mathrm { \ p M D ^ { \mathrm { T M } } }$ 18-T vector, cloned and sequenced to obtain 4 aptamers of HIV $\mathrm { g p 4 1 }$ antigen. The affinities of the 4 aptamers $( K _ { \mathrm { d } } )$ all reached the nanomolar level. Among the 4aptamers,theNo.15aptamer showed the strongest afinity.Specificityanalysis oftheaptamersrevealed thatall these 4 aptamers had specific affinity to $\mathrm { H I V \ g p 4 1 }$ antigen with no affinity to other non-specific proteins. Conclusion We successfully obtained DNA aptamers with highly specific afinity to the HIV $\mathrm { g p 4 1 }$ antigen from random single-stranded oligonucleotide library, and the obtained aptamers have the ability to antagonize HIV gp41 antigen.

Key words: HIV gp41 antigen; subtractive SELEX technique; DNA aptamers

# INTRODUCTION

HIV infection is known to stimulate the production of antibodies against such antigens as envelope proteins (gp120 and gp41) and core protein (P24) of HIV [1.2] Typically,HIV canbedetected two or threeweeksafter its infection,and the antigens in the blood can serve as serum markers during the window period of AIDS to allow early diagnosis and treatment [3,4]. Currently the two main strategies for recognition and detection of the antigens rely on the detection of the interactions between antibodyand antigen and between DNA aptamersand antigen [5].Compared with antibody/ antigeninteraction,DNAaptamershavemany advantages including a higher stability due to their low d cycle (oneortwomonths，alommungcit potential of repeatable application due to their quick denaturation and renaturation,absence of the function of effector,abettersensitivityandfacilechemical modificatioandalowercostSofarvarietyof DNA aptamers with specific binding affinity to $\mathrm { g p } \ 1 2 0$ and P24 have been screened for detecting the antigens for early diagnosis of HIV infection[14.15],but no sucha DNA aptamer has been reported with a highly specific affinity to the antigen $\mathrm { g p } \ 4 1$ ，which plays an important role in the process of virus packaging and maturation of HIV[16, 17]

Herein we report,for the first time,the screening of 4 DNA aptamers with specific binding affinity to gp41 antigen through subtractive SELEX screeningwith commercial carboxyl-enriched agar magnetic beads as the separation medium and subsequent conventional quantitativePCR(qPCR)techniqueandgel electrophoresis.These aptamers provide a basis for efficient,rapid and accurate detection of gp41 antigen forearly diagnosisof HIVi8.19]and may greatlyfacilitate the development of future high-throughput screening of HIV with a clear molecular mechanism[20].

# MATERIALSANDMETHODS

# Materials

The single-stranded DNA (ssDNA; 5'-CTATAGCAATGGTACGGTACTTCC40N-CAAAAGTGCACGCTACTT-TGCTAA-3'), P6 primer ( $5 ^ { \prime }$ -TAATACGACTCACTATAGCAATGGTACGGTACTTCC-3')， P9 primer( $5 ^ { \prime }$ -TTAGCAAAGTAGCGTGCACTTTTG-3')，biotin-P9 primer,universal primer M13-47 ( $5 ^ { \prime }$ -CGCCAGGGTTTTCCCAGTCACGAC-3')，and PCR mix werepurchased fromTaKaRa.pMD $\mathrm { T M }$ 18-Tvector vector and the Gel Extraction

Kitwere obtained from KGI Nanjing Biological Technology Development Co., Ltd.

# Buffer solutions

PBS $( 0 . 0 1 \ \mathrm { \ m o l / L } , \ \mathrm { p H } \ 7 . 4 )$ was purchased from Jiangsu Bollywood Biotechnology Co.,Ltd,which contained 137 mmol/L of NaCl, $2 . 7 \ \mathrm { \ m m o l / L }$ of KCl, $6 . 5 \ \mathrm { \ m m o l / L }$ of ${ \mathrm { N a } } _ { 2 } { \mathrm { H P O } } _ { 4 }$ ， $1 . 8 \ \mathrm { m m o l / L }$ of ${ \mathrm { N a H } } _ { 2 } { \mathrm { P O } } _ { 4 }$ ， $2 . 5 \ \mathrm { m m o l / L }$ of $\mathrm { M g C l } _ { 2 }$ $1 \ \mathrm { m m o l / L }$ of $\mathrm { C a C l } _ { 2 }$ ，and $1 \%$ (mass ratio) bovine serum albumin (BSA) and used as the $1 \times$ screening buffer.The $1 \times$ binding buffer contained PBS $\mathrm { ( 0 . 0 1 \ m o l / L }$ , pH 7.4), $1 \ \mathrm { m m o l / L }$ of $\mathrm { C a C l } _ { 2 }$ and $1 ~ \mathrm { m g / m L }$ of BSA. The blocking solution contained $0 . 0 1 \mathrm { \ m o l / L }$ of PBS,1 mg/mL of BSA, and $\mathrm { 1 ~ m g / m L }$ of salmon sperm DNA.

Aptamerselection

ssDNA library pretreatmentssDNA library was centrifuged at $1 0 ~ 0 0 0 ~ \mathrm { { r / m i n } }$ for $5 ~ \mathrm { m i n }$ followed by addition of $6 0 0 ~ \mu \mathrm { L }$ binding buffer,and the mixture was well mixed.

Two rounds of positive screening with HIV gp4l antigen In the first round of positive screening, $0 . 3 ~ \mathrm { \ m L }$ of magnetic beads and $3 0 0 ~ \mu \mathrm { L }$ of HIV gp41 antigen (diluted at $1 \colon 1 0 0$ according to the instruction） were added in a $1 . 5 \mathrm { - m L }$ EP tube and incubated for $2 \mathrm { ~ h ~ }$ at $3 7 \mathrm { ~ \textdegree C }$ . The product was magnetically separated and rinsed with the screening buffer for 3 times,followed by the addition of $5 0 0 ~ \mu \mathrm { L }$ of the blocking solution.After incubation at $3 7 \mathrm { ~ \textdegree C }$ for $^ { 1 \mathrm { ~ h ~ } }$ ，the product was separated and washed with the screening buffer twice (for $1 ~ \mathrm { m i n }$ each time) and with the binding buffer once for $1 ~ \mathrm { m i n }$ before the addition of the primary ssDNA library. The above mixture was treated at $9 5 ~ \mathrm { { ^ { \circ } C } }$ for $5 ~ \mathrm { m i n }$ ， $4 \ \mathrm { { ^ \circ C } }$ for $1 0 ~ \mathrm { m i n }$ ,and $3 7 \ \mathrm { { ^ { \circ } C } }$ for $5 \mathrm { m i n }$ ,and then incubated for $^ { \textrm { 1 h } }$ at $3 7 \ \mathrm { { ^ { \circ } C } }$ .The composites of magnetic beads and ssDNA library were separated and washed for 3 times with the screening buffer (1 min each time); $2 0 0 ~ \mu \mathrm { L }$ deionized water was then added and the mixture was centrifuged at $9 5 ~ \mathrm { { ^ { \circ } C } }$ for $1 0 ~ \mathrm { m i n }$ . The supernatant was collected for use as the secondary library for the next round of screening. The second round of positive screening was carried out following the identical procedure in the first round of screening except that the amount of magnetic beads was $0 . 1 5 ~ \mathrm { m L }$ ：

Amplification of ssDNA into dsDNAbyqPCR The PCR template $( 1 6 0 ~ \mu \mathrm { L } )$ was added in half of the above solution and mixed thoroughly. The homogeneous mixture was then divided into 8 equal portions $( 3 0 ~ \mu \mathrm { L } )$ for qPCR until the amplification fluorescence reached the highest point of the S-curve.

Preparation of the secondary ssDNA libraryStreptavidinbiotin magnetic beads $( 0 . 1 ~ \mathrm { \ m L }$ ）and the amplified double-stranded DNA (dsDNA) were added into a 1.5-mL EP tube and incubated for 3O min at $3 7 \ \mathrm { { ^ \circ C } }$ . The mixture of the recovered beads and $5 0 0 ~ \mu \mathrm { L }$ of $1 \%$ (mass ratio) TPBS was incubated at $3 7 \ \mathrm { ^ { \circ } C }$ for 4 min,and the procedure was repeated for 3 times.The recovered beads were washed with water followed by the addition of $2 0 0 ~ \mu \mathrm { L }$ of the binding buffer.After incubation at $9 5 \ \mathrm { { ^ \circ C } }$ for $5 \ \mathrm { m i n }$ ，the recovered supernatant was added into a counter-screening tube and incubated at $3 7 \ \mathrm { { ^ { \circ } C } }$ for $4 5 \mathrm { \ m i n }$ . The recovered supernatant was subsequently added into a positive-screening tube and incubated for $^ { \textrm { 1 h } }$ to remove unbound ssDNA library.All the recovered beads after positive-screening and counter-screening were washed with the screening buffer for 3 times ( $3 \mathrm { m i n }$ each time),after which $2 0 0 ~ \mu \mathrm { L }$ of deionized water was added.After incubation at $9 5 \ \mathrm { ~ \textdegree C }$ for $5 ~ \mathrm { \ m i n }$ ，the supernatant was recovered for use as the secondary libraries in the next round of screening.

Enrichment effect monitoringPCR template $( 1 0 ~ \mu \mathrm { L } )$ was added into $2 0 ~  { \mu \mathrm { L } }$ of recovered supernatant after positive-screening and counter-screening,respectively. Thecorrespondinghomogeneoussolutionswere amplified by qPCR, with $2 0 ~  { \mu \mathrm { L } }$ of PCR template in $1 0 ~ \mu \mathrm { L }$ of deionized water as the control sample.

SELEX screening with HIV P24 antigen as counterscreening target molecule SELEX screening with HIV P24 antigen as the counter-screening target molecule was conducted following identical screening procedures described above except for the replacement of gp41 antibody by HIV P24 antigen in the third round of screening.The conditions in each round of SELEX screening were listed in Tab.1.

# Cloning and sequencing of selected aptamers

dsDNA obtained through secondary library The recovered supernatant in the positive-screening tube after 6 rounds of selection was used as the template to form $3 0 0 ~ \mu \mathrm { L }$ of PCR reaction system,which consisted of $3 0 ~ \mu \mathrm { L }$ of $1 0 \times$ buffer containing $\mathrm { M g } ^ { 2 \ + }$ ， $4 5  { \mu \mathrm { L } }$ of template, $2 . 4 ~ \mu \mathrm { L }$ of dNTP, $3 ~ \mu \mathrm { L }$ of $\mathrm { P } 6$ primer, $3 ~ \mu \mathrm { L }$ of P9 primer, $2 ~ \mu \mathrm { L }$ of rTaq enzyme,and $2 1 4 . 6 ~ \mu \mathrm { L }$ of deionized water. The homogeneous PCR reaction was divided into 6 equal portions and amplified by routine PCRfor 26 thermal cycles of $9 5 ~ \mathrm { { ^ { \circ } C } }$ for $5 ~ \mathrm { m i n }$ ， $9 4 ~ \mathrm { { ^ circ C } }$ for 30 s, $5 5 \ \mathrm { { ^ \circ C } }$ for $3 0 \mathrm { ~ s ~ }$ ，and $7 2 \ \mathrm { { ^ \circ C } }$ for $3 0 \mathrm { ~ s ~ }$ .The dsDNA fragment was recovered through gel electrophoresis from the thermally-cycled solution.

Connection of dsDNA fragment with $p M D ^ { \scriptscriptstyle T M } I \& T$ vector The recovered dsDNA was dissolved in deionized water and connected with $\mathrm { p M D ^ { \mathrm { T M } } 1 8 – T }$ vector to prepare the connection solution,which consisted of $0 . 5 ~ \mu \mathrm { L }$ of $\mathrm { p M D } ^ { \mathrm { T M } } 1 8$ -Tvector, $2 ~ \mu \mathrm { L }$ of inserted fragment, $5 ~ \mu \mathrm { L }$ of solution and $3 8 3 . 5 \quad \mu \mathrm { L } \quad$ ofdeionizedwater. The connection solution was reacted at $1 6 ~ \mathrm { { ^ circ C } }$ for $3 0 ~ \mathrm { m i n }$ to obtainthesecondarylibrarydsDNArecombinant plasmid.

E. coli transformation and selection of positive clones The original strain of $E$ ，coli DH5α was cultured and shaken till logarithmic growth (with absorbance at $6 0 0 \ \mathrm { n m }$ of O.6).Competent bacterial cells $( 2 0 0 ~ \mu \mathrm { L } )$ were prepared by treatment with O.1 mol/L $\mathrm { C a C l } _ { 2 }$ and transferred into a $1 . 5 \mathrm { - m L }$ EP tube for storage at $- 8 0 \ \mathrm { { ^ circ C } }$ beforeuse.For $E$ ，colitransformation, $1 0 ~ \mu \mathrm { L }$ of recombinant DNA or $1 0 ~ \mu \mathrm { L }$ of deionized water were added into the thawed competent cells,which were coated on the plate and cultured on a solidLB medium containing ampicillin to select the positive clones.

Tab.1 Conditions of 6 rounds of SELEX screening   

<html><body><table><tr><td>Round</td><td>Molumtic mead</td><td>DNA input (pmol)</td><td>Teput (um RNA</td><td>Washing times</td><td>Bonding time (h)</td><td>HiVding4 antiney</td></tr><tr><td>1</td><td>0.30</td><td>2500</td><td>10</td><td>3</td><td>1</td><td>15%</td></tr><tr><td>2</td><td>0.15</td><td>1000</td><td>10</td><td>3</td><td>1</td><td></td></tr><tr><td>3</td><td>0.15</td><td>300</td><td>10</td><td>4</td><td>1</td><td>32.6%</td></tr><tr><td>4</td><td>0.15</td><td>300</td><td>10</td><td>5</td><td>1</td><td>41.7%</td></tr><tr><td>5</td><td>0.15</td><td>300</td><td>10</td><td>5</td><td>1</td><td>52.9%</td></tr><tr><td>6</td><td>0.15</td><td>300</td><td>10</td><td>6</td><td>1</td><td>63%</td></tr></table></body></html>

Stagger extension process(StEP） to identify positive clonesThe monoclonal bacteria preserved at $- 8 0 \ \mathrm { ~ \textdegree C }$ was used as a template system,and the blank group was set. Staggered PCR system of the universal primers M13-47 and P6/P9 consisted of $1 0 ~ \mu \mathrm { L }$ of $1 0 \times \mathrm { P C R }$ buffer containing $\mathrm { M g } ^ { 2 + }$ $2 ~ \mu \mathrm { L }$ of template, $8 ~ \mu \mathrm { L }$ of dNTP, $1 ~ \mu \mathrm { L }$ of primer $\mathrm { P 6 / P 9 }$ ， $1 . 5 ~ \mu \mathrm { L }$ of rTaq enzyme and $8 0 . 7 ~ \mu \mathrm { L }$ of deionized water. The homogeneous PCR reaction was divided into 1O equal portions and amplified by routine PCR for 3O thermal cycles ( $9 5 ~ \mathrm { { ^ { \circ } C } }$ for $5 ~ \mathrm { m i n }$ ， $9 4 ~ \mathrm { { ^ { \circ } C } }$ for 30 s, $5 5 ~ \mathrm { { ^ { \circ } C } }$ for $3 0 ~ \mathrm { s }$ ，and $7 2  { \mathrm { ~ C ~ } } 3 0  { \mathrm { ~ s ~ } }$ ).Thedirection of the targetfragmentinsertedintotheplasmid was determinedbynon-denaturingpolyacrylamidegel electrophoresis.

Identification of specific affinity of HIV gp41 antigen aptamer to positiveclones The positive clones identified by StEP technique were used as the template to synthesize ssDNA using asymmetric PCR method.Agar beads (O.2 mL) were packed in 4 tubes labeled with "+" "1 -","2-", "3-",which were coupled with HIV gp41 antigen,anti-HBsAg antibodies,HIV P24 antigen,and BSA,respectively.The specific affinities of the aptamers to these proteins were determined with qPCR analysis.

Flow cytometry analysisThe specific affinities of the four aptamers to HIV gp41 antigen were measured to identify the aptamer with the strongest specificity.To monitor the enrichment of the aptamers after selection, $1 \times 1 0 ^ { 5 }$ beads binding to HIV gp41 antigen and $1 \times 1 0 ^ { 5 }$ beads binding to HIV P24 antigen were incubated with FITC-labeled ssDNA in the binding buffer ( $\dot { C _ { \mathrm { f } } }$ =200 nmol/L, $V _ { \mathrm { f } } { = } 2 0 0 ~ \mu \mathrm { L }$ ）at $3 7 \mathrm { ~ \textdegree C }$ for $4 5 ~ \mathrm { { \ m i n } }$ .The fluorescence intensity of the beads was monitored with a FACS Aria cytometer by counting $1 0 ~ 0 0 0$ events.The FITC-labeled unselected ssDNA library served as the negative control. The binding affinities of the aptamers were evaluated by flowing cytometry,with the initial FITC-labeled library as the negative control to determine the background binding.All the binding assays were performed 3 times.

The $K _ { \mathrm { d } }$ of the interaction between the aptamer and the $\mathrm { H I V \ \ g p 4 1 }$ antigen was determined using the equation $Y { = } B _ { \mathrm { m a x } } X / ( K _ { \mathrm { d } } { + } X )$

The process of aptamer selection by SELEX was illustrated in Fig.1.

# RESULTS

# SelectionofDNA aptamers against HIV gp4l antigen

Through two rounds of positive screening and 4 rounds of negative screening,4 highly specific aptamers of HIV $\mathrm { g p 4 1 }$ antigen were obtained. The qPCR resultsof positiveselectionandnegativeselectioninthe third-round counter-selection (Fig.2) showed enrichment of the selected aptamers in the positive-selection tube. Although the specific aptamers binding with the fixed target gradually increased in ssDNA library,aptamer enrichment did not reach saturation. Further screening with an increased screening pressure was needed to obtain the highly specific aptamers binding with HIV $\mathrm { g p 4 1 }$ antigen.

Asshown in Fig.3，qPCR results ofpositive selection and counter selection in the fourth round of counter selection showed that the enrichment of the specific aptamers binding with HIV gp41 antigen reachedsaturationinthepositive-selectiontube, suggesting that the non-specific ssDNA could be quickly eliminated by magnetic bead-based screening to increase the enrichment of the specific aptamers.

Sequencing result and secondary structures of the 4 aptamers   
The products of the sixth round of screening were cloned and sequenced to obtain the sequences of the 4 aptamers (Tab.2).The secondary structures of the 4 aptamers of HIV gp41 antigen were shown in Fig.4. Fig.5 shows the enrichment of the DNA sequences that bind to HIV P24 and gp41 antigens.

# Positive clones identified by StEP technique

The positive clones were identified by StEP technique. A clear combined band of 17O-180 bp appeared in some clones but not in the other clones,demonstrating successful transformation by the recombinant plasmids. The selected combination of primers M13-47 and $\mathrm { P } 7 \mathrm { x }$ with nucleic acid bands at 17O-18O bp demonstrated that the fragment was inserted forwardly into the vector, and the selected combination of the primers M13-47 and P1lx with bands at 17O-18O bp indicated a reverse insertion of the fragment in the vector; no such bands occurred in the blank group following amplification with the two primers.A total of 9 positive monoclones were identified,including 4 with positive insertion and 5 with reverse insertion as shown by electrophoresis analysis (Fig.6).

![](images/a760760d1392acae0bbe228517e608a471a4f6d007b75f67030d6300a6720abe.jpg)  
Fig.1 Process of aptamer selection by SELEX.

![](images/a68564e84f949ee665221082e075f5004f32ecc498e1617e530bf956465e4323.jpg)  
Fig.2 ${ \mathsf { q P C R } }$ detection of the third round of negative selection.

# Specific binding between HIV gp4l antigen aptamer and positive clones

Fig.7 shows successful conversion of the positive clones with specific binding with HIV gp41. The No.15 positive clone had a highly specific affinity to HIV gp41 without adsorption to other non-specific proteins.

![](images/996837e9a3d7069356463fecd159b09ea82ada7ffa35c0cbae7cd9ec69450e73.jpg)  
Fig.3qPCR detection after the fourth and fifth rounds of counter-selection.

# DISCUSSION

SELEX technique is the first requirement for designing and synthesizing random ssDNA libraries of suitable length (6O-1OO bp). The central segment of the library is a random sequence containing 2O-6O base pairs flanked by a primer annealing sequence as the lateral segments. Theoretically,a random sequence contains 2O bases of random ssDNA library,in which at least $1 \times 1 0 ^ { 1 5 }$ different sequences could be formed so that the storage capacity of the library could reach atleast $1 \times 1 0 ^ { 1 5 }$

Magnetic beads can be used as a good separation medium due to their affinity to a variety of target proteins.Ligands bound to the beads could be rapidly separated from unbound ssDNA library by washing the beads with the screening buffer. Asbiotinand streptavidin have the strongest interaction known in nature,commercial streptavidin-biotin magnetic beads were utilized to screen HIV $\mathrm { g p 4 1 }$ antigen aptamers.

Tab.2 Sequences of the 4 aptamers binding with HIV gp41 antigen   

<html><body><table><tr><td>Name</td><td>Size (bp)</td><td>Sequence (5'-3')</td><td>Kd (nmol/L)</td></tr><tr><td>1</td><td>40</td><td>ATCGGATGGGGACTCACCAGCTGACAGCTTGCCCTTTAGC</td><td>98+7.2</td></tr><tr><td>7</td><td>40</td><td>CCCCTGCGTTGAATCGGGTCGCCACGCCTGGTTGGAAGGA</td><td>51+4.9</td></tr><tr><td>10</td><td>40</td><td>TCGGCACGTTCTCAGTAGCGCTCGCTGGTCATCCCACAGC</td><td>32+9.6</td></tr><tr><td>15</td><td>40</td><td>GGACGCGGTTAGGGGAGAATTTCCATACCTTTGGGTTGTA</td><td>26+6.6</td></tr></table></body></html>

![](images/8d17ad037e9415d97e76f00ac4379e65b10d3f69ba26173d822e27f55e32e996.jpg)  
Fig.4 Stem-loop structures of the4 selected aptamers binding with HIVgp41 antigenaspredicted byRNA Structure 3.7 software.

![](images/9fd3e41c1dd85ab6a957af1701d7cc982204f7e3268b028a728a53b33679cfb8.jpg)  
Fig.5Flowcytometryto monitor the specific affinity of the beads to HIVantigens P24 (A)and gp41 (B).

The library enrichment is an important indicator of SELEX screening. The aptamers with affinity to HIV gp41antigen were successfully obtained after two roundsofpositive-screeningand4roundsof counter-screening. Analysis of positive-screening and counter-screening using qPCR in the third round of counter-screening showed that the screened aptamer beganto be enriched in positive-screening tube. Although the specific aptamers binding with the fixed target gradually increased in ssDNA library,aptamer enrichment did not reach saturation.After the fourth roundofcounter-selection, thespecificaptamers binding with HIV $\mathrm { g p 4 1 }$ antigen reached saturation in the positive-selection tube.Subsequent StEP and $K _ { \mathrm { d } }$ value measurement indicated that the aptamers obtained by magnetic beads-based SELEX technique possessed a high specificity to HIV $\mathrm { g p 4 1 }$ antigen.The utilization of qPCR,gelelectrophoresisand otherconventional techniques simplifies the procedures for screening the aptamers and shortens screening time.The aptamers obtained may facilitate future high-throughput screening and future development of rapid test kits of HIV with a low cost.

![](images/7e9f85298177a652c27137354d75e112b66d14d28f6f88d49d880c121196ea34.jpg)  
Fig.6Electrophoresis of 9 positive monoclines identified using StEP technique.Lanes 6,11:Blank control group.M:Marker.

![](images/bd20c7e22c88c56fa6e70822342ae5df2cef37c908f1d4fcdb1a5c82fd30fd50.jpg)  
Fig.7 Identification of the specific aptamer of HlV gp41.

In this study, the aptamers with a highly specific affinity to HIV gp41 antigen were successfully obtained after 6 rounds of screening with magnetic beads.To reduce the non-specific adsorption of ssDNA as much as possible to obtain highly specific aptamers,the coupled targets were replaced in the counter-selection tube in the third round of screening.The specific aptamer obtained provides an efficient,rapid and accurate probe for early diagnosis of HIV by detecting the gp41 antigen.

# REFERENCES

[1]Wang CY,Wu CY,Hung TC. Sequence-constructive SELEX: a new strategy for screening DNA aptamer binding to Globo H[J].Biochem Bioph Res Commun,2014,452(3): 484-9.   
[2]Shao NS,LI SH, Huang YP,et al.Advances in the SELEX technique and aptamer[J].Prog Biochem Biophys,2006,33(4): 329-35.   
[3]Ghamar SD,Mohammad JR,Ali ML,et al. Selection of DNA aptamers against human cardiac troponin I for colorimetric sensor based dot blot application[J].J Biotechnol, 2015,208:80-6.   
[4]Reda E,Mohamed S,Mohammed Z,et al.DNA aptamers selection and characterization for development of label-free impedimetric aptasensor for neurotoxin anatoxin-a[J].Biosens Bioelectron,2015, 68:295-302.   
[5]Nina M,Marko V,Jan M,et al. Optimization of SELEX: comparison of different methods for monitoring the progress of in vitro selection of aptamers[J].JPharmaceut Biomed,2014,91:151-9.   
[6]Moon J,Kim G,Lee S,et al.Identification of Salmonella Typhimurium-specific DNAaptamers developed using whole-cell SELEX and FACS analysis[J].JMicrobiol Meth,2013,95(2):162-6.   
[7]Xin H, Zhou ZX,Yuan L,et al. Selective collection and detection of MCF-7 breast cancer cells using aptamer-functionalized magnetic beads and quantum dots based nano-bio-probes[J].Anal Chim Acta, 2013,788(14): 135-40.   
[8]Elyse D,Maria C,et al.Development of a bead-based aptamer/ antibody detection system for C-reactive protein|J].Anal Biochem, 2015,472: 67-74.   
[9]Gulshan S,Poornima V,Neetika R,et al.Bio-capture of S. Typhimurium from surfacewater byaptamer for culture-free quantification[J]. Ecotoxicol Environ Saf,2012,78(78): 320-6.   
[10] Wu SJ, Zhang H, Shi Z,et al.Aptamer-based fluorescence biosensor for chloramphenicol determination using upconversion nanoparticles [J].Food Control,2015,50: 597-604.   
[11] Georg M,Ruth M,Bilha S,et al.Multimerization of ERBB2/HER2 specific aptamer leads to improved receptor binding [J].Biochem Bioph Res Commun,2015,465(2): 218-24.   
[12]Heo K,Min SW, Sung HJ.An aptamer-antibody complex (oligobody) asanovel delivery platform for targeted cancer therapies Jl.J Control Release,2016,229:1-9.   
[13]Eric O,Eric T,Karen C,et al.A simple method for eliminating fixed-region interference of aptamer binding during SELEX[J]. Biotechnol Bioeng,2014,11: 2265-79.   
[14]Li X,Deng JC,Yan SH,et al.Fabrication of endothelial progenitor cell capture surface via DNA aptamer modifying dopamine/ polyethyleneimine copolymer film[J].Appl Surf Sci,2O16,386: 138-150.   
[15]Reda E,Mohamed S,Mohammed Z,et al.In vitro selection, characterization,andbiosensingapplicationofhigh-affinity cylindrospermopsin-targeting aptamers[J].Anal Chem,2O14, 86(18): 9196-203.   
[16] Camille LH,Peng HY,Wang ZX,et al.An improved SELEX technique for selection of DNA aptamers binding to M-type 11 of Streptococcus pyogenes[J].Methods,2016,97:51-7.   
[17]Mariia D,Silvie R,Helena G,et al. Current approaches in SELEX: An update to aptamer election technology[J].Biotechnol Adv,2015, 33: 1141-61.   
[18] Ozer A,Pagano JM,Lis JT. New technologies provide quantum changes in the scale,speed,and success of SELEX methods and aptamer characterization[J].Mol Ther Nucleic Acids,2O14,3(8): e183.   
[19]Mona A,Mohammad R,Khalil A,etal.In vitro and in vivo evaluation of therapy targeting epithelial-cell adhesion-molecule aptamers for non-small cell lung cancer[J]. JControl Release,2015,2O9: 88-100.   
[20] Nasa S,Jonathan N,Koichi A,et al. Selection of DNA aptamers against uropathogenic Escherichia coli NSM59 by quantitative PCR controlled Cell-SELEX[J].JMicrobiol Meth,2014,104:94-100.

# 琼脂磁珠消减SELEX技术筛选HIVgp41抗原适配体

栗坤},修尘林'，高立明²,石 明¹,翟 越³  
燕山大学'环境与化学工程学院，燕山大学校医院内科,河北 秦皇岛 066004；秦皇岛市第一人民医院肿瘤  
科,河北 秦皇岛 066004

摘要：目的 通过消减 SELEX技术筛选得到高特异性、强亲和力的 $\mathrm { H I V } \ \mathrm { g p 4 1 }$ 抗原适配体，为HIV的早期诊断提供了新的检测技术。方法 以琼脂磁珠为载体,HIV $\mathrm { g p 4 1 }$ 抗原为靶标分子,利用消减 SELEX技术和实时定量-PCR技术,筛选得到 $\mathrm { H I V \ g p 4 1 }$ 抗原适配体。结果通过6轮筛选，筛选得到的次级ssDNA库通过PCR扩增得到dsDNA,dsDNA与pMDTM18载体链接，进行克隆、测序,获得4条HIV gp41抗原适配体。获得的4条适配体亲和力(Kd)均在纳摩尔水平,其中15号适配体的亲和力最强，特异性检测表明筛选得到的适配体几乎只与 $\mathrm { H I V \ g p 4 1 }$ 抗原结合,不与其他非特异性蛋白结合。结论 利用随机单链寡核苷酸文库成功获得与HIV $\mathrm { g p 4 1 }$ 抗原特异性结合的适配体，所获得的适配体具有括抗HIV $\mathrm { g p 4 1 }$ 抗原的能力。

关键词：HIV $\mathrm { g p 4 1 }$ 抗原;消减SELEX技术;DNA适配体识别

收稿日期：2016-04-13  
基金项目：秦皇岛市科学技术研究与发展计划(201501B034，201502A193)  
作者简介：栗坤,教授，E-mail:thebestlikun $@$ 163.com  
通信作者:栗坤,E-mail: thebestlikun $ @ 1 6 3 . \mathrm { c o m }$ ；高立明,E-mail:17778889387@163.com