# SCIENTIFIC REPORTS

# OPEN

# Comparative proteomics reveals abnormal binding of ATGL and dysferlin on lipid droplets from pressure overload-induced dysfunctional rat hearts

Received:17July 2015   
Accepted:17 December 2015   
Published:22 January 2016

Linghai Li1,3,\*, Huina Zhang2,3,\*, WeiyiWang4,Yun Hong5, Jifeng Wang²,Shuyan Zhang3, Shimeng $\mathbf { \times } \mathbf { u } ^ { 3 , 6 }$ QingboShu3,6, Juanfen Li7,FuquanYang³,Min Zheng5,Zongjie Qian7& Pingsheng Liu³

Excessive retention of neutrallipids in cardiac lipid droplets (LDs)is a common observation in cardiomyopathy.Thus,the systematic investigation ofthe cardiac LD proteome willhelp to dissectthe underlying mechanisms linking cardiac steatosis and myocardialdysfunction.Here,after isolation of LDsfrom normal and dysfunctional Sprague-Dawley rat hearts, we identified 752 heart-associated LD proteins using iTRAQ quantitative proteomic method, including 451 proteins previously unreported on LDs.The most noteworthy finding was the identification ofthe membrane resealing protein,dysferlin. An analysis of dysferlin truncation mutants indicated that its C2 domain was responsible for its LD localization.Quantitative proteomic results furtherdetermined that 27 proteins were increased and 16 proteins were decreasedin LDs from post pressure overload-induced dysfunctional hearts,compared with normal hearts. Notably,adipose triacylglycerollipase (ATGL)was dramatically decreased and dysferlin was substantially increased on dysfunctional cardiac LDs.This studyforthefirst time reveals the dataset ofthe heartLD proteome in healthy tissue and the variationofit under cardiac dysfunction. These findings highlight an association between the altered LD protein localization of dysferlin and ATGL and myocardial dysfunction.

The heart is a major consumer of energy through lipid utilization1. However, under certain pathological conditions asociated with cardiac dysfunction,excess neutral lipids are deposited in cardiomyocytes as the consequence of insufficient fatty acid $\beta$ -oxidation2-4.Lipid droplets (LDs),a ubiquitous organelle distributed among most celltypes,serve as a neutral lipid reservoir and provide fatty acids to fuel cellular $\beta$ -oxidative processes.LDs stringently govern the storage and turnover of intracelular neutral lipids through the actions of LD-associated proteins, including both lipid metabolic enzymes as welas LD structural proteins of the perilipin family (PLINs)6.The altered expression and activity of these LD-associated proteins are reported to influence cardiac lipid homeostasis and,subsequently,cardiac function7.For instance,the cardiac targeted overexpression of adipose triacylglycerol lipase (ATGL)protects against pressre overload-induced cardiac dysfunction8, ameliorates diabetes-induced cardiomyopathy9,and even prevents obesity-relatedcardiac steatosisand dilated cardiomyopathyl0.Thus,obtaining a global view of the cardiacLD protein profile under diferent physiological

1DepartmentofAnesthesiology,BeijingChestHospital,CapitalMedicalUniversity,BeijingTuberculosisandoracic Tumor Research Institute,Beijing,China.²BeijingAn Zhen Hospital,CapitalMedicalUniversity,KeyLaboratoryof UpperAirway Dysfunction-related Cardiovascular Diseases,Beijing Instituteof HeartLung and BloodVesselDisease, Beijing,China.NationalLaboratoryof Bomacromolecules,Institute ofBiophysics,ChineseAcademyofSciences, Beijing,China.Depatmentofardiovasculariseases,CivilAvationGeneralosital,ekingUiversityeing, China.5DepartmentofGastroenterology,the FirstAfliated Hospital,Collegeof Medicine,Zhejang University Hangzhou,China.UniversityofChineseAcademyofSciences,Beijing,China.7DepartmentofCardiology,Affliated HospitalofGuilinMedicalUniversityGuilin,China.\*Theseauthorscontributedequallytothis work.Corespondence andrequests formaterials should be addressedtoH.Z.(email: whinnerzhn@126.com)orP.L.(email: pliu@ibp.ac.cn)

and pathological conditions willhelp to extend our understanding of heart lipid metabolism and the underlying mechanisms maintaining cardiac lipid homeostasis as wellas provide insight into etiology of various cardiac pathological states.

Besides their role inneutrallpid metabolismll,LDs are also involved in diverse intracellular processes including signal transductionl2,protein storagel3,and membrane traffcking14 through the mediation ofLD proteins either embedded in or associated with the organelle.Accumulated LD proteomic results suggest that proteins associated with membrane repair such as the SNARE complex15,Caveolin- $3 ^ { 1 6 }$ ,Rab proteins17,MG-53/TRIM7218, and dysferlinl9 are located on $\mathrm { L D } s ^ { 2 0 , 2 1 }$ . Other evidence also points to a potential relationship between membrane repairand lipid metabolism.For example,the membrane traffc inhibitor BFA notonly blocks membrane repair-2, butalso blocks intracellular neutrallipid storage23.Moreover,a deficiencyofdysferlin,akey protein inmembrane repair,induces aberrant TAG accumulation24.It is well established that membrane repair proteins playa pivotal rolein sustaining normalcardiac function,since rapid andeficient membrane resealing is crucial for maintaining cardiac plasma membrane integrityas wellas normal cardiac contractionand relaxationl6.However,the relationship between membrane sealing and lipid metabolism in cardiomyocytes remains obscure and needs further investigation.Therefore,the examination oftheheartLD proteomewill provide clues to illminate the roleofthe organelle in cardiac membrane repair,and to dissect the mechanisms linking lipid metabolism,membrane repair, and cardiac function.

Inthis study,we investigated cardiac LD proteome in normaland pressure overload-induced dysfunctionalrar heart.752 proteins were identified.Ofthese,43 proteins were found with significant variation in heartLD under differentconditions.These findings provide usefulinformation forfuture studies regardingthe functions ofheart LDs and give some novel clues to promote the development of clinical treatments for cardiopathy.

# Results

Morphology of lipid droplets in rat myocardium.Transmisson electron microscopy(TEM) observation of mature rat heart revealed that cardiac LDs were dispersed in cardiomyocytes and were tightly associated with mitochondria (Fig.lAa).HeartLDs were isolated from five rat hearts according toa modified protocol, as reported previously21.Nile red-stained fluorescence micrographs showed that heart LDs appeared spherical shape.Apart froma few large LDs,most isolatedLDs were smaller than $1 \mu \mathrm { m }$ in diameter (Fig.1Ab).Consistently, the electronmicrographs fromboth negativeand positive staining modes revealed the integrityofisolated cardiac LDs with most of them $< 1 \mu \mathrm { m }$ in diameter (Fig.1Ac,Ad).In addition to the morphological evaluation,the purity of the isolated LDs was also determined using more stringent biochemical measurements. Results from silver staining of electrophoreticallyseparated proteins demonstrated that the protein patrnfrom theisolated LDs was distinctlydiffrent from that of post-nuclear supernatant (PNS),total membranes (TM)and cytosol(Cyto), suggesting the significant enrichmentofLD-specific proteins (Fig.1B).The purity ofLDs was further assessed by the relative abundance of LD-resident proteins including adipocyte diferentiation related protein (ADRP/PLIN2), TIP47/PLIN3,OXPAT/PLIN5,and by the deficiency of proteins of other intracellular compartments(i.e.EEA1, endosome; LAMP1,lysosome).The cellmembrane protein Annexin A2 and muscle specific caveola protein caveolin-3 were also foundintheLD fraction,consistent with previous observations21.Some mitochondrial proteins (COX IV and TIM23)and ER proteins (BIP and p62) were also found in the LD fraction (Fig.1C),indicating the LD fraction isolated from rat heart was largely free of other organelles except for ER and mitochondria, which was in agreement with the proposed argument that LD was originally from $\mathrm { E R } ^ { 2 5 }$ and frequently associated with mitochondria²1.

Normal and dysfunctional rat heart lipid droplets.The establishment ofa method to isolated cardiac LDs allowed us to compare theLD protein composition between normal and dysfunctional hearts to reveal molecular details associated with disorders of cardiomyocyte lipid storage and metabolism.We first compared the histological morphology of normaland dysfunctional hearts after successfully establishing the pressure overload-induceddysfunctional heart model inrats (Fig.2A,B).Images takenofheart ventricle cryo-sections demonstrated increased lipid accumulation in dysfunctional heart with stronger Oil red O(ORO) staining than normal tissue (Fig.2Ca).Furthermore,TEM images clearly showed that LDs in dysfunctional myocardium were larger and more abundant compared with the normal group (Fig.2Cb).Consistent with the microscopy, asemi-quantitative analysis of triacylglycerol demonstrated that approximately $2 5 \%$ more of the neutral lipid deposited in dysfunctional hearts than the normal ones (Fig.2Cc).The significant diferences inLD morphology suggested that there may be accompanying distinctions in the protein profiles between these two groups.To examine this,the proteins extracted from normaland dysfunctional heartLDs were silver stained after electrophoretic separation on polyacrylamide gels.Indeed,some diferences were observed in the LD protein profiles from the two groups; notably,some proteins bands in the gels from the dysfunctional group had substantially stronger staining than the controls (Fig.2D,blue arrows).

Quantitative proteomic analysis of cardiac lipid droplets. After validating the purity and protein profile ofLDs isolated from normal and dysfunctional hearts,quantitative proteomic analysis was performed.The proteins extracted from normal and dysfunctional heart LDs were digested with trypsin,labeled byiTRAQ,and thenanalyzed by nano LC-MS/MS (Fig.3A).To obtain reliable results,two biological replicatesand two technical replicates were carred out. One biological group contained proteins from normalcardiac LDs (labeled with iTRAQ reagents,114or 116)and dysfunctional cardiacLDs (labeled with iTRAQ reagents,115 or 117). Only one protein (Isoform 4of NADHdehydrogenase) was identified without anyiTRAQlabeling.Therefore,we set our acceptance criteria to include proteins with atleasttwo unique labeled peptides (Fig.3A). Underthis criterion, we identified a totalof752rat cardiac LD-associated proteins after combining four replicates (R1-R4; 688,688,680 and 680 proteins respectively).Of these,592 proteins were reproducibly detected as shown in the Venn diagram (Fig.3B),withasubsetof30l proteins havingbeen reported previously (Supplementary,Table S1),indicating the accuracy of the LD purification and proteomics techniques.

![](images/cccd4bfdd29bc1ad511507ce7893db3c49494aa8e04576540a93419223cc23cd.jpg)  
Figure1.Isolating LDs from rat heart.(Aa) Ultra-thin sections ofrat heart were observed by TEM.Asterisks indicate LDs.M,mitochondrial.B ${ \mathfrak { x } } = 1 \mu \mathrm { m }$ .(Ab)LD fraction isolated from rat heart was stained with Nile red and imaged by fluorescence microscopy. Ba $\mathrm { r } = 5 0 \mu \mathrm { m }$ ,Heart LDs were imaged by TEM after negative staining (Ac) or positive staining (Ad). $\mathrm { B a r } = 1 { \mu \mathrm { m } }$ .(B) Equal amount of proteins of different cellular fractions were separated by SDS-PAGE folowed by silver staining.TM,total membrane; Cyto,cytosol; PNS,post-nuclear supernatant. (C) Equal amount of proteins from isolated LDs,total membrane (TM),cytosol(Cyto),and postnuclear supernatant (PNS) were processed for immunoblotting with indicated antibodies.

All identified proteins were categorized into 10 groups according to their function and their assigned sub-cellular locations based on PATRHE and Uniprot KB websites (Fig.3C and Supplementary, Table S1). Examining these groups,mitochondrial proteins had the greatest representation ( $2 4 \%$ ,185 proteins),which was also observed in our previous LD proteomics from skeletal muscle²1,and brownadipose tissue26,suggesting the intimate metabolic relationship between LDs and mitochondria in cardiomyocytes. Another major group was proteins involved in DNA replication,transcription and protein translation and modification (about $20 \%$ ,153 proteins),including several histone proteins and 21 chaperones.This finding raised the possibility thatLDs may be involved in intracellular protein expresion and activity by modulating protein maturation or degradation. Some recent studyindeed proved thatLD played arole inhistone storageand stabilization7,andalso participated in the degradation of ApoBl00 and 3-hydroxy-3-methylglutaryl CoA (HMG CoA)through the mediation of LD-related Derlin-1 and Ubxd828.Roughly $1 1 \%$ of the proteome from heart LD samples was cell signaling proteins (83 proteins),supporting the hypothesis thatLDs participate in intracellular signal transduction.Another notable group was composed of 71 lipid metabolic proteins (about $9 \%$ of the total),including the enzymes implicated in metabolismof riacylglycerol(TAG),fattyacids,phospholipids,nd sterol esters.This was inlinewiththe roleofLDs as alipid metabolic compartmentto sustain the intracellularlipid homeostasis.Membrane trafficking proteins accounted for approximately $6 \%$ of the total proteins.Besides the previously reported LD-related Rab proteins,SNAREs and coatomers29-31,the most unexpected finding was the identification ofthe muscle-specific membrane-sealing protein dysferlin and its associated proteins,MG-53/TRIM7232,caveolin- $3 ^ { 3 3 }$ and Annexin $A 2 ^ { 3 4 }$ . These identifications imply that cardiomyocyte LDs may be participated in membrane repairing during the constant processof myocardial contraction and relaxation.Besides the abundant mitochondrial proteins,the heart LD proteome also included proteins from other organelles and intracellular structures such as ER and the cytoskeleton,which is consistent withadynamicinteraction betweenLDs and other intracellularcompartments. Finally, there were a substantial number of proteins (158; $2 1 \%$ of the total) not belonging to the other 9 categories, indicating some functions of LDs are still unknown and will require further study.

![](images/fa133cd33289707678748227d78f2f609681de72178a4ba5065a88c5e82c9c9b.jpg)  
Figure 2.Morphology and protein profile of normal and dysfunctional rat heart LDs.(A) The morphological images of representative normal and dysfunctional heart with integral (left)or longitudinal section view (right).(B) Cardiac function of normal (upper)and dysfunctional heart (lower) was detected by echocardiography assy.(C) The tissue morphology of normal and dysfunctional heart was observed by microscope after Oil Red O staining(a, $\mathtt { B a r } = 2 0 \mu \mathrm { m }$ )orbyTEM( $\mathsf { b } , \mathsf { B a r } = 1 \mu \mathrm { m }$ ).Thecorresponding TAG content was quantified by the relative ratio of TAG to protein (c).Results are means $\pm \operatorname { S E M }$ （20 $( \mathrm { n } = 5 )$ 1 $^ { * } p < 0 . 0 5$ vs. Nor-heart.(D)Diferent protein pattern of normal and dysfunctional cardiac LD.Arows indicated the higher intensity protein bands in dysfunctional cardiac samples.Nor-heart, normal heart; Dys-heart,dysfunctional heart.

The STRING web database was employed to analyze the physical and functionalinteractions of allthe identified LD proteins.Among the total 752 identified proteins,671 had entries in the STRING database with 6173 associations as depicted by specificcoloredlines (SupplementaryFig.S1).Examining onlythelipid metabolicand PLIN family proteins,the database depicted 57 proteins and presented 146 associations (SupplementaryFig.S2). The protein association networks enabled us toanalyze the identified proteins systematically.From this complicated global view,some proteins showed close relationship with each other and formed unique protein clusters.

![](images/0516333e20b34b4e2f1cb61ba8a182facdfadf1a7da8945e324816e38a4f4959.jpg)  
Figure3.Proteome analysis of isolated heartLD.(A)Flowchart showedthe experimentdesign for heart LD comparative proteomics.(B)The Venn diagram showed the overlap of the identified LD proteins in two technical replicates (blue and red)and two biological replicates (115/114and117/116).(C)The identified heart LD proteins were categorized by sub-cellular distributions and functions based on PANTHER and Uniprot KB sources.

Consistent withour functional clasification,the densest interaction networks contained lipid metabolic proteins (orangecolor),and mitochondrial proteins (yellow color).Three other notable interaction networks were composedof cytoskeletal proteins (green color),membrane traffcking proteins (redcolor),and proteins involved in DNA replication,transcription,protein translation,and modification (blue color). On the other hand,the protein clusters can also lead insights into unknown protein functionsand to help deeply understanding the relationship between proteins.Taking three enzymes in TAG mobilization as example,besides the internalconnection with each other, there were specific linkagebelonging to each enzyme,such as the interaction of Pnpla2(ATGL)with Abhd5 (CGI-58); Lipe (HSL) with apolipoproteins (Apob and Apoe) and LDL receptor (Ldlr); and Mgll (MGL) with Lss(lanosterol synthase)(SupplementaryFig.S2),suggesting that except fortherole inTAG hydrolysis that the three enzymes deal with together, there are unique functions owned by each of them.

Verification of proteomic results.To verifythe proteomic results, immunofluorescence assays were carried outto detect some of the identifed LD-associated proteins in oleate-treated primary cardiomyocytes.The perilipin family protein ADRP/PLIN2 surrounded LDs, forming a ring-shaped image (Fig.4A), while OXPAT/ PLIN5 was present on the LDs as wellas in the cytoplasm (Fig. 4B),implying the diverse characteristics of these two proteins in lipid metabolism and in maintaining the LD structure.Lipid metabolic proteins such as ATGL,ACSL and CGI-58 were also observed both on the LDs and in the cytoplasm (Fig.4C-E).As expected, dysferlin (Fig.4F)and dysferlin-associated Annexin A2,Annexin A11,caveolin-3,and VCP(Fig.4G-J) were observed partiallylocalized on cardiomyocyte LDs. But no green signal was observed in negative control images (Supplementary Figure S3).

![](images/a70d04e6e535252a5b112b285af558c093c12d9ca80b2979ef94e8c93907a90f.jpg)  
Figure 4. Subcellular distribution ofLD-associated proteins in primarycardiomyocytes.The primary cardiomyocytes isolated from neonatal rat were cultured in DMEM supplemented with $1 0 \%$ FBS. $1 0 0 \mu \mathrm { m o l / L }$ oleate was added in the medium for 24 hours before the immunostaining analysis.(A-J)Tenofthe identified heartLD proteins including ADRP, OXPAT,ATGL, ACSL, CGI-58,dysferlin, Annexin A2,Annexin A11,VCP, caveolin-3 were detected by immunofluorescent assay. Column a(blue): nucleus stained with Hochest33258; column b (green): LD-associated proteins stained with FITC; column c (red): LD stained with LipidTOX Deep Red; column d: the corresponding merged picture of column a,band c;column e: the enlarged image of the white frame from column d pictures.The white arrowheads represented the LD residence of indicated proteins. Bar $= 5 { \mu \mathrm { m } }$ ，

![](images/1f91f58d03014ca720840a649e67f01a5d372c6d40651f113b0040a510e5a1e5.jpg)  
Figure 5.C2 domain-dependentLD targeting of dysferlin and attenuated TAG accumulation by dysferlin. (A) Sub-celllar distribution of dysferlin was detected by immunoblotting assay. Colloidal Blue-staining gel was utilized to present proteinloading.(B) Domain diagram offull-length and truncation mutant constructs of dysferlin with GFP in-frame fusion in the C-terminal.(C)A series of truncation mutants of dysferlin expresion vectors were transiently transfected to HEK293A along with oleate treatment for 24hours and observed by an Olympus FV1000.Column a (blue): nuclear stained with Hochest33258; column b (green): GFP; column c (red): LD stained with LipidTOX Deep Red; column d: the corresponding merged picture of column a,b and C; column e: the enlarged image ofthe white frame from column d pictures.The white arrowheads represented the LD localization of indicated proteins.Bai $: = 5 \mu \mathrm { m }$ .(D) TAG content was quantified by the relative ratio of TAG to protein after transient transfection of dysferlin and aseries of truncation mutants of dysferlin expreion vectors to HEK293A cellsalongwith oleate( $\mathrm { { 1 0 0 \mu m o l / L } }$ treatment for 24hours.Results are means $\pm$ SEM （204 $( \mathrm { n } = 4 )$ ） $^ { * } p < 0 . 0 5$ vs. GFP-Con.

Lipid droplet targeting domain of dysferlin and overexpression of dysferlin decreases TAG accumulation. Immunoblotting assays were performed to confirm the localization of dysferlin.As shown in Fig.5A,dysferlin was dramaticallenrichedonLDs rather than total membrane.Dysferlinconsists of6cytosolic C2 domains (C2A-F)and a single-passtransmembrane domain atits C terminus5 (Fig.5B).The C2 domains of dysferlin are thought to function in membrane repair,vesicle traficking,and membrane fusion36 by interaction with phospholipids and with other proteins.Therefore,we hypothesized thatthe C2 domains may be involved in theLD targeting of dysferlin.To test this hypothesis,aseries of dysferlin C2 domain truncation mutants fused with a GFP reporter were over-expressed in oleate-treated HEK293A cells.Fluorescence confocal micrographs showed that although different truncations had distinctive intracellularlocations (e.g.,GFP-C2-TMl and GFP-TM had more cytoplasmic signal,whereas GFP-C2-N2,GFP-C2-N1and GFP-C2-TM2 signals were equally dispersed throughout the cytoplasmand nucleus),if the C2 domain was present in the structure,there was stil merged fluorescence (yelow)signal observed on theLDs.This indicates thatthe C2 domain,nottransmembrane domain,was responsible fortargeting dysferlinto thesurfaceofLDs (Fig.5C).In addition,the peptide sequences identified fordysferlin in the LD proteome included many amino acids belonging to C2 domain (Supplementary Fig.S4),although not completely identical (which may because the hydrophobic amino acids of C2 domains37 are not easy to be identified by mass spectrometry),giving another support for the pivotal role of C2 domain in dysferlin LD targeting.Additionall,to determine the relationship between dysferlin and lipid storage,WT dysferlinand its truncation mutants were transiently transfected into HEK293Awiththe similar effciency,lone with oleate treatment for 24 hours,and then TAG level was measured.Results showed that only WT dysferlin attenuated TAG amount after overexpression in HEK293A (Fig. 5D).

![](images/f31a80f48c3b58fae50839df58b09d526ae35080b4a9a111bd26ca23159bbe69.jpg)  
Figure 6. Comparative proteomics of the normal and dysfunctional heart LD.(A) The Venn diagram showed the number of unchanged,up-regulated and down-regulated proteins from cardiac LD comparative proteome. (B)AfteriTRAQ-quantification,some of the identified proteins including perilipin family proteins (ADRP,OXPAT),membrane traffc proteins (dysferlin,Rab 5)and metabolic proteins (ATGL,CGI-58) were detected in different cellular fractions by Western blotting.(C) Statistical data ofthe expressionofDysferlin, ATGL,OXPAT, CGI-58,ADRP and Rab 5on normal and dysfunctional cardiac LDs.Protein expression was normalized to gel staining. Results are means $\pm$ SEM $( \boldsymbol { \mathrm { n } } = 4 )$ ： $^ { * } p < 0 . 0 5$ vs.Nor-heart.(D) Immunofluorescence results demonstrated the localization of dysferln in cryosection of normal and dysfunctional hearts. Blue, nuclear stained with Hochest33258; green: dysferlin stained with FITC; red: LD stained with LipidTOX Deep Red. Bar $= 2 0 \mu \mathrm { m }$ ：

Differential protein expression on lipid droplets from normal and dysfunctional hearts.Using iTRAQ-based quantitative proteomics with a Q Exactive mass spectrometer (Thermo Scientific, USA) equipped with an Easy n-LC1,0 HPLC(Thermo Scientific,USA),we were notonlyableto identify more LD-associated proteins than previously reported,but alsoable to obtain quantification data for comparative proteomics.To obtain reliable quantification results,only proteins quantified at least twice (iTRAQlabeled twice)were accepted. Sincethe expression ratio of most proteins was unchanged and the mean value of the total protein expression ratio was 1.033 with standard deviation (SD)value of0.225,a1.96-fold change was chosen as the threshold for significant changes in protein expresson. Using these criteria,the expression levels of 43 proteins were changed in dysfunctional cardiac LDs compared with normal ones. Of these,l6 proteins were down-regulated and 27 proteins were up-regulated (Fig. 6A, Table 1). Supplementary Figure S5 in presented a representative MS/MS spectrum of the dysferlin peptide with the diferent expression level in normal and dysfunctional cardiac LD samples afteriTRAQlabeling. Consistent with iTRAQ quantification results,Western blotting validation showed that ADRP/PLIN2 were up-regulated, whereas the ATGL,CGI-58 as wellas OXPAT/PLIN5 were reduced on dysfunctional cardiac LDs (Fig.6B,C),suggesting that the excessive accumulation of lipid in dysfunctional hearts was directlydueto thelower hydrolysis ofTAG.Notably,dysferlinanddysferlin-associated partners VCP, caveolin-3,and Annexin A2 were enriched on dysfunctional cardiac LDs compared with normal LDs (Fig. 6B,C and Supplementary Fig.S6).Consistent with the immunobloting results,immunostaining demonstrated that more dysferlin signal was apparent in the cytoplasm of dysfunctional cardiac myocytes in comparison to normal ones (Fig. 6D).The enrichment of these proteins on LDs may be related to the insufficient membrane healing knownto exist indysfunctional hearts.Meanwhile,in accordance with the comparative proteome identification, Western bloting demonstrated thatthere is no difference in Rab5 targeting on dysfunctional cardiac LDs and its counterpart (Fig.6B,C).As to the identified proteins with altered expresson levelon heartLDs under diferent physiological and pathological statuses,43 proteins entered STRING database and 60 links were ilustrated (Fig.7).In the STRING network, membrane trafc proteins (purple),lipid metabolism proteins (green) and glycolysis proteins (red)formed 3 main protein links that revealed the relevantalterations of these intracelular processes between normal and dysfunctional hearts.

Table 1.The differentLD proteins in normal and dysfunctional heart identified by quantitative mass spectrometry.iTRAQ ratios was presented with mean $\pm$ SD. $9 5 \%$ confidence intervals $\mathbf { \bar { z } } \ s c o r e = 1 . 9 6 )$ were used to determine the cutoff values for proteins with changes.Significant test for the intergroup variables were analyzed with the Student's $\mathrm { \Delta t }$ -testwith $\textstyle P \leq 0 . 0 5$ considered statistically significant.   

<html><body><table><tr><td>UniProt KB</td><td>Description</td><td>Gene Symbol</td><td>iTRAQ ratio (dysfunctional/normal)</td><td>P value</td></tr><tr><td>P04638</td><td>Apolipoprotein A-II</td><td>Apoa2</td><td>0.36±0.07</td><td>1.20E-05</td></tr><tr><td>P41350</td><td>Caveolin-1</td><td>Cav1</td><td>0.47±0.10</td><td>9.10E-06</td></tr><tr><td>G3V8L9</td><td>Polymerase Iand transcript release factor</td><td>Ptrf</td><td>0.51±0.05</td><td>8.25E-07</td></tr><tr><td>P04639</td><td>Apolipoprotein A-I</td><td>Apoal</td><td>0.51±0.02</td><td>1.00E-09</td></tr><tr><td>P02651</td><td>Apolipoprotein A-IV</td><td>Apoa4</td><td>0.57±0.01</td><td>8.33E-05</td></tr><tr><td>Q75Q41</td><td>Mitochondrial import receptor subunit TOM22 homolog</td><td>Tomm22</td><td>0.57±0.03</td><td>1.81E-08</td></tr><tr><td>D4A115</td><td>Protein Col6a3</td><td>Col6a3</td><td>0.57±0.02</td><td>1.57E-05</td></tr><tr><td>Q6QA69</td><td>1-acylglycerol-3-phosphate O-acyltransferase ABHD5</td><td>Abhd5 (CGI-58)</td><td>0.58±0.07</td><td>1.00E-07</td></tr><tr><td>Q66H98</td><td> Serum deprivation-response protein</td><td>Sdpr</td><td>0.58±0.05</td><td>1.06E-07</td></tr><tr><td>G3V8D4</td><td>Apolipoprotein C-II (Predicted)</td><td>Apoc2</td><td>0.58±0.04</td><td>6.26E-08</td></tr><tr><td>Q9Z1H9</td><td>Protein kinase Cdelta-binding protein</td><td>Prkcdbp</td><td>0.58± 0.01</td><td>8.60E-07</td></tr><tr><td>P0C548</td><td>Patatin-like phospholipase domain-containing protein 2</td><td>Pnpla2 (ATGL)</td><td>0.58± 0.05</td><td>4.79E-08</td></tr><tr><td>O08590</td><td>Membrane primary amine oxidase</td><td>Aoc3</td><td>0.58±0.03</td><td>1.07E-07</td></tr><tr><td>M0R7Z9</td><td>Protein Plin5</td><td>Plin5(OXPAT)</td><td>0.58±0.03</td><td>7.85E-06</td></tr><tr><td>F1LM84</td><td>Nidogen-1</td><td>Nid1</td><td>0.58±0.03</td><td>4.06E-09</td></tr><tr><td>P55159</td><td>Serum paraoxonase/arylesterase 1</td><td>Ponl</td><td>0.58± 0.07</td><td>1.08E-03</td></tr><tr><td>Q5U2U5</td><td>Adipose differentiation related protein</td><td>Plin2 (ADRP)</td><td>1.48 ± 0.07</td><td>5.50E-06</td></tr><tr><td>P04642</td><td>L-lactate dehydrogenase A chain</td><td>Ldha</td><td>1.49 ± 0.03</td><td>1.29E-06</td></tr><tr><td>M0R757</td><td>Elongation factor l-alpha</td><td>LOC100360413</td><td>1.49± 0.05</td><td>2.70E-03</td></tr><tr><td>B2GV06</td><td>cin</td><td>Oxct1</td><td>1.49 ± 0.07</td><td>3.22E-09</td></tr><tr><td>G3V8B0</td><td>Myosin-7</td><td>Myh7</td><td>1.49 ± 0.10</td><td>1.33E-03</td></tr><tr><td>Q63151-2</td><td>Isoform Short of Long-chain-fatty-acid-CoA ligase 3</td><td>Acsl3</td><td>1.50± 0.07</td><td>4.95E-07</td></tr><tr><td>035244</td><td>Peroxiredoxin-6</td><td>Prdx6</td><td>1.50± 0.09</td><td>2.61E-05</td></tr><tr><td>P42123</td><td>L-lactate dehydrogenase B chain</td><td>Ldhb</td><td>1.51±0.05</td><td>1.17E-10</td></tr><tr><td>Q80ZF7</td><td>Retinol dehydrogenase 10</td><td>Rdh10</td><td>1.52 ± 0.09</td><td>1.33E-05</td></tr><tr><td>Q9QZ76</td><td>Myoglobin</td><td>Mb</td><td>1.52 ± 0.10</td><td>2.08E-02</td></tr><tr><td>P46462</td><td>Transitional endoplasmic reticulum ATPase</td><td>Vcp</td><td>1.53 ± 0.04</td><td>3.24E-10</td></tr><tr><td>P05065</td><td>Fructose-bisphosphate aldolase A</td><td>Aldoa</td><td>1.53 ± 0.07</td><td>1.31E-08</td></tr><tr><td>P55797</td><td>Apolipoprotein C-IV</td><td>Apoc4</td><td>1.58± 0.06</td><td>1.72E-06</td></tr><tr><td>P97541</td><td>Heat shock protein beta-6</td><td>Hspb6</td><td>1.58± 0.08</td><td>8.02E-05</td></tr><tr><td>P20059</td><td>Hemopexin</td><td>Hpx</td><td>1.64±0.06</td><td>1.51E-08</td></tr><tr><td>Q6AY30</td><td> Saccharopine dehydrogenase-like oxidoreductase</td><td>Sccpdh</td><td>1.64± 0.09</td><td>4.98E-06</td></tr><tr><td>D4A6X1</td><td>Dysferlin (Predicted), isoform CRA_a</td><td>Dysf</td><td>1.64 ± 0.08</td><td>3.40E-05</td></tr><tr><td>Q6PDU1</td><td>Serine/arginine-rich splicing factor 2</td><td>Srsf2</td><td>1.65 ± 0.09</td><td>6.46E-06</td></tr><tr><td>Q6UK00</td><td>Promethin</td><td>Tmem159</td><td>1.70 ± 0.10</td><td>2.56E-09</td></tr><tr><td>P20070-3</td><td>Isoform 3 of NADH-cytochrome b5 reductase 3</td><td>Cyb5r3</td><td>1.72 ± 0.05</td><td>5.43E-07</td></tr><tr><td>F1LUV9</td><td>Neural cell adhesion molecule 1 (Fragment)</td><td>Ncam1</td><td>1.73± 0.01</td><td>4.98E-06</td></tr><tr><td>F7F469</td><td>Protein Irgm2</td><td>Igtp</td><td>1.74±0.08</td><td>3.40E-05</td></tr><tr><td>G3V9Z3</td><td>Amine oxidase [flavin-containing] A</td><td>Maoa</td><td>1.77 ± 0.09</td><td>6.46E-06</td></tr><tr><td>035987</td><td>NSFL1 cofactor p47</td><td>Nsfl1c</td><td>1.81± 0.05</td><td>2.56E-09</td></tr><tr><td>E9PTT7</td><td>Protein Dhrsx</td><td>Dhrsx</td><td>1.95 ± 0.09</td><td>1.04E-03</td></tr><tr><td>D4A6C5</td><td>Protein Arhgap1</td><td>Arhgap1</td><td>2.32±0.04</td><td>1.47E-10</td></tr><tr><td>Q5M840</td><td>G0/Gl switch protein 2</td><td>G0S2</td><td>2.51±0.09</td><td>8.29E-10</td></tr></table></body></html>

![](images/27c97bdc3301a3654b1f90da8c96905c1dd2092bab62f6c6124549da7db957be.jpg)  
Figure 7. The association network of LD proteins with varying expression in normal and dysfunctional heart.The relationshipofheartLD proteins with different expression under physicaland pathological status was predicted by the website program STRING against rat database.Network edges represent predicted functional associations with diferent line colors standing for various types of evidence used for predicting: red, fusion evidence; green,neighborhood evidence; blue,co-occurrence evidence; purple,experimental evidence; yellw, text-mining evidence; black,co-expresson evidence.Purple circle embraced the proteins associated with membrane traffc process; red circle comprised the glycolysis-related proteins and green circle contained the lipid metabolism-related proteins.

# Discussion

In the research presented here,we have enumerated the proteome of cardiac LDs and have made a comparative analysis between normal and dysfunction hearts.We identified 752 LD proteins by 2D-LC-MS/MS, including 451 previously unreported proteins.Dysferlin was found closely associated with LDs in cardiomyocytes and its C2 domain was found to be required for LD targeting. A quantitative proteomic comparison between normal and dysfunctional heartLDs identified 43 proteins with altered expression levels in LDs.Notably, this suggests that thereduction of ATGLand the enrichment of dysferlin on the defectivecardiac LDs maybe relevant to the development of cardiac dysfunction.

Membrane repair is an elemental process for maintaining cell integrityand function. Cardiomyocytes in particular require unique mechanisms to efciently and efectively repair the plasma membrane damaged due to frequent disruptions caused by persistent mechanical contraction.Transient membrane damage existing in normalcardiomyocytes is exacerbated by various pathophysiological stresses when the capacity of membrane repair machinerycannot meet the cellular requirement.Dysferlin is a well-recognized membrane-associated protein that is mplicated in the repair ofsarcolemma membrane in skeletal muscles38.Recent studies have found that dysferlin-deficient mice display defective cardiomyocyte membrane repair capacity that is linked to increased susceptibilityto cardiomyopathy39,40.Ithas been postulated that dysferlin exerts its membrane repair function by binding withan unidentified lysosomal-exclusive vesicular compartment39.However,the nature ofthesevesicles andthe exact mechanism underlying dysferlin-mediated membrane repair remain to be elucidated41. Ourresults showed that dysferlinand its partners,VCP,AnnexinA2,andcaveolin-3,were localizedonheartLDs,implying thatLDs contribute to cardiac membrane resealing.Truncation mutants revealed that the phospholipid binding structure,the C2 domain,may be responsible fordysferlinLD targeting.Moreover,theidentificationof the dysferlin-interacting cardiac membrane-repair proteins,MG-53/TRIM723² and its partner PTRF42 in the heart LD proteome further supports the hypothesis that LDs are involved in membrane repair. Beyond our current observations,there is precedence in previous research also pointing to a role for LDs in membrane resealing through the synthesis of phospholipids (phospholipids synthases localized on $\mathrm { L D } ^ { 2 1 }$ ),or by providing a source of pre-existing membrane ipids either folowing homotypic fusion43,or direct fusion with the plasma membrane at thedisruptedlocation4430,However,there is nodirect experimentalevidence insupportofthisand no molecular mechanism has been demonstrated.The more interesting finding is the decreased TAGaccumulationby dysferlin overexpression in HEK293Acels,whilethe increased expressionofdysferlinand itsassociated proteins Annexin A2 and caveolin-3on dysfunctional cardiac LDs according to the comparative proteomics study.These observations letus hypothesize that dysferlinand its partners posibly promotethe lipid conversion from TAG to phospholipid under normal situation, thus decreasing the accumulation of intracelular TAG when overexpressing dysferin. While under cardiac dysfunction,LDs may only act asa docking site of these membrane carrier proteins since energy generation is insufficient for membrane-resealing and lipid conversion.

Synthesis and catabolismofintracellular TAGare tightly controlled under normal conditions through regulation ofthe expression,locationand activity oflipid metabolic enzymes.Under circumstances where lipid storage and energy utilization areunbalanced,the pathologic accumulationofTAG in LDs can occur.Forinstance,under normal conditions,lipotropic hormones promote CGI-58 translocation to LDs,leading to the recruitment of ATGL,and a subsequent enhancement of ATGL activity hydrolyzing TAG. At the same time,increased phosphorylation of PLIN1 through lipotropic hormone-induced pathways releases hormone sensitive lipase (HSL), triggering its relocation fromthe cytoplasm toLDs for itsaction45.However,inadiabetic state,elevated TNF- $\propto$ reduces PLIN1 expression inadipocytes,attnuating theactivityof HSL,and resulting inabnormal neutral lipid stored inadipocyte $\mathrm { L D s ^ { 4 6 } }$ .Compared with other tissues such as liver and adipose,the homeostasis of lipid metabolisminthe heart is more criticaldueto the extremelyhigh rateoflipid turnover in the tissue.In the cardiac LD proteome presented here,we found that the TAG metabolic enzyme,ATGL,and its co-factor CGI-58 were associated with LDs.More strikingly,ourcomparative proteomicsand subsequent verification demonstrated that the amount of ATGL and CGI-58 on LDs was reduced under cardiac dysfunction. Another significant finding from the comparative proteome data is the enhanced targeting of Go/G1 switch gene 2(GoS2)on dysfunctional cardiac LDs. GoS2 can inhibit ATGL activity,and this suggests another mechanism explaining ATGLatenuation in cardiac dysfunction47.All these findings are reasonable since the insufficient $\beta$ -oxidation does not require so much fatty acid hydrolyzed by ATGL under cardiac dysfunction,thus leading to the excess retention of TAG in cardiac LDs.Our Western bloting results also revealed the increased binding of ADRP/PLIN2on dysfunctional cardiac LDs,which may be required to stabilize the enlarged LDs.By contrast,another PLIN family protein OXPAT/PLIN5 was significantly reduced in the dysfunctional cardiac LD fraction,perhaps reflecting unique functions of OXPAT/PLIN5 in the modulation of myocardial TAG deposition48. Collctively, the discovery of these alterations in lipid metabolism-related proteins on dysfunctional cardiac LDs willfacilitate the development of a deeper understanding of the changes in lipid metabolism associated with various physiological and pathological situations.

Our study provides the first dataset of heart LD proteins as wellas their variation under a pathological condition.It highlights potentialassociations between the ectopic LD location of dysferlin and ATGL with cardiac dysfunction.These results ofer new insights into the intricate metabolicfunctions in heartand may promote new directions in future research concerning the etiologyand treatmentofheartfailure.Meanwhile,some unresolved problems of this study are also proposed.For example,the exact mechanism of elevated dysferlinon defective cardiac LD still remains elusive.To obtain the dynamics and comprehensive cardiac LD proteomics,other situation suchas myocardial ischemia,cardiomyopathy,and diferent stages of pressure overload-induced defective heart should be measured to help deeply understanding the lipid metabolism of heartunder various situations.

Generation of rat cardiac dysfunction model.This investigation was approved by the Animal Care and Use Commitee of Institute of Biophysics,Chinese Academy of Sciences,Beijing,China,which has a permission of conducting animal experiments,SYXK (SPF 2009-111).All experimental protocols were conformed to the Guide for the Care and Use of Laboratory Animals (NIHPublication Eighth Edition,updated 20l1).The rat cardiac dysfunction model was constructed by clipping the ascending aorta under anesthesia by intraperitoneal injection of pentobarbital $\left( 4 0 \mathrm { m g / k g } \right)$ .During the operation procedure,cardiovascular and respiratory effects and variable responses were monitored. To minimize the pain,morphine ( $\mathrm { \langle 1 0 m g / k g . }$ subcutaneous injection) was administered after surgery. Heart function was evaluated by echocardiography assay (Andover,MA, USA).

The rats with heart ejection fraction (EF) ratio $< 5 0 \%$ were categorized into the heart dysfunction group and were sacrificed for LD isolation.

LD isolation.Rats were sacrificed under anesthesia with i.p. pentobarbital $( 4 0 \mathrm { m g / k g ) }$ . Five rat hearts were pooled for each LD isolation performed as described²1.

Protein digestion, iTRAQ labeling,LC-MS/MS analysis,protein identification and quantification analysis.The samples for iTRAQ quantitative analysis were prepared according to iTRAQTM Reagents Protocol (Applied Biosystems,USA).The labeled proteins were analyzed by nanoLC-MS/MS usinga QExactive equipped with an Easyn-LC 1000 HPLC system(Thermo Scientific).The raw data and protein quantification data wereanalyzed with Proteome Discovery version 1.4.The fold change threshold for upor down regulation was set as mean $\pm 1 . 9 6 0 \sigma$ ：

Statistical analysis. Results represent means $\pm$ SEM $( \mathrm { n } = 4  – 5 )$ ),unless specified.Student's t-test (two-tailed) was used when two groups of means were compared. Protein expression was quantified by Quantity One software (Bio-Rad) and normalized to gel staining.TAG concentration was normalized to corresponding protein level. Statistical significance was accepted at $p { \leq } 0 . 0 5 \left( { ^ \ast } \right)$ ：

# References

1.Goldberg,I.J,Trent,C.M.&Schulze,P.C.LipidMetabolismandToxicityintheHeart.CellMetab 15,805-812(2012).   
2.Stanley,W.Cecchia,FA.&opaschuk,G.D.MyocardialsubstratemetabolisinteoraandfilingheartPioes85, 1093-1129 (2005).   
3.Marfell,Retalyocardalidumulatoninpatientswithressureverladedeartdmeabolcsydrome.LidRes, 2314-2323 (2009).   
4.FujitaS.etalMarkedlyncreaediacellalidoletsdsruptiofinterelarjuntiosiopedmyodfro a patient with arrhythmogenic right ventricular cardiomyopathy.Heart Vessels 23,440-444 (2008).   
5.DucharmeN.A.&B.PMinireview:Lipiddropletsinlipogenesisandlipolysis.Endocrinology.149,942-949 (2008).   
6.Yang,L.etal.Teproteomicsofipiddroplets:structure,dynamicsandfunctionsoftheorganelleonseedfrombacteriato humans.JLipid Res 53,1245-1253 (2012).   
7.Haemmerle,G.etal.ATGL-mediatedfatatabolismregulatescardiacmitochondrialfunctionviaPPAR-alphaandPGC-1.NatMed 17,1076-1085 (2011).   
8.Kienesberger,P.C.etal.MyocardialATGLOverexpresionDecreases theRelianceonFatyAcidOxidationandProtectsgainst Pressure Overload-Induced Cardiac Dysfunction.Mol Cell Biol 32,740-750 (2012).   
9.Pulinilkunnil,T.tal.MyocardialAdiposeTrglycerideLipaseOverexpressionProtectsDiabeticMiceFromtheDevelopentf Lipotoxic Cardiomyopathy. Diabetes 62,1464-1477,(2013).   
10.Pulinilkunil,Tetal.Cardacpeifdioe triglyridelpaseoverexpresiprotectsfroadiacteatoisdted cardiomyopathy following diet-induced obesity.Int JObesity 38,205-215 (2014).   
11.Ducharme,N.A.&ckel,P.E.Minireview:Lipiddropletsinlipogenesisandlipolysis.Endocrinology49,94-949 (008).   
12.Yu,W.G.,Cassra,J.&Weller,P.FPhosphatidylinositide3-kinaselocalizestocytoplasmic lipidbodies inuman polymorphonuclear leukocytes and other myeloid-derived cells.Blood 95,1078-1085 (2000).   
13.Cermeli,SGu,ros,SP.&Welte,.A.eddropletprteoeevealsaropletsreproteintorageu Biol 16,1783-1795 (2006).   
14.Zehmer,J. K.etal.Arole for lipid droplets in inter-membrane lipid traffc.Proteomics 9,914-921(2009).   
15.Schapire,A.L.talArabidopsisSyaptotagmi1RequiredrtheantenanceofPlasaMembraneIntegrityndCelVability. Plant Cell20,3374-3388 (2008).   
16.HausenloyD.J.&Yelon,D.M.CellembraneRepaisaechansmforhemicreconditionng?Circulaton,4249 (2010).   
17.Los,FC.O.etal.RAB-5-andRAB-11-Dependent Vesicle-TrafckingPathwaysAreRequiredforPlasmaMembraneRepairafter Attack byBacterialPore-Forming Toxin.Cell Host Microbe9,147-157 (2011).   
18.Cai,C.et al.MG53 nucleates assemblyof cellmembrane repair machinery.Nat CellBiol11,56-64 (2009).   
19.Bansal,D.etal.Defective membranerepair indysferlin-deficientmusculardystrophy.Nature423,168-17 (2003).   
20.Zehmer,J.K.etal.Arole forlipid droplets in inter-membrane lipid trafic.Proteomics 9,914-921 (2009).   
21.Zhang,H.etlProteomeofsetalusceliidroletrevealsasocatoithiohondriadpolipopotein-.rem Res 10,4757-4768 (2011).   
22.TogoT,KrasievaT&rdt.reaseraetsonecedesseulellbraeepiroll 11,4339-4346 (2000).   
23.Bartz,R.etalEvidencetatmon-ADP-rbosyationofCt/BARSregulateslpidsorage.MoliolCell1015-305(00).   
24.Grounds,M.D.et al.Lipid accumulation in dysferlin-deficient muscles.AmJPathol184,i668-1676,(2014).   
25.Martin,S.&Parton,R.G.Lipiddroplets:aunifiedviewofadynamicorganele.NatRevMolCellBio773-378(2006).   
26.Yu,J.etalLipidoletremodelingandinteractionithohondriainouserodiosetisseuringcoldtreatntBBAMol Cell Res,doi:10.1016/j.bbamcr.2015.01.020 (2015).   
27.Li,Z.etal.Lipiddroplets controlthe materalhistonesupplyofDrosophilaembryos.CurBiol22,104-2113(012).   
28.SuzukiM.talDerln-ndUBXDareengagedindocatioanddegradationofidatedApoB-atlipiddropletsMolol Cell 23,800-810 (2012).   
29.Ozeki,S.etalab8localestolipiddropletsandiducsteirloseappositiontoteendoplasmiceticlum-driedbane. JCell Sci118,2601-2611 (2005).   
30.Bostrom,Petal.AREproteinsmedatefusioetwencytosoiclipiddropletsandareiplcatedininsulinsnsitivityatCl Biol9,1286-U1139 (2007).   
31.Soni,K.G.et al.Coatomer-dependent protein delivery to lipiddroplets.JCellSci122,1834-1841 (2009).   
32.Matsuda,C.et al.The C2A domain in dysferlin is important for association with MG53(TRIM72).PLoS Curr 4, e5035add5038caff5034(2012).   
33.Matsuda,C.etal.Tesarcolemmalproteinsdsferlinandcaveolin-3interactinskeletalmuscle.HumMolGenet1,76-76 (2001).   
34.Lennon,N.J.etal.Dysferlininteracts withAnnexinsAlandAandmediatessarcolemmal wound-healing.BiolChem278, 50466-50473 (2003).   
35.Bansal,D&mbellK.eldbraeeuulatroedsCell04).   
36.Davis,D.,DohertyK.R,elmonte,A.J.&NallE..Calcium-sesitivepospolipidindingpropertiesofnol mutant ferlin C2 domains.J Biol Chem 277,22883-22888 (2002).   
37.Marty,N.J,,C.ull,&JCeCstofefelelle lipid bilayers. Biochemistry 52,5585-5592 (2013).   
38.Clarke,..,Caldwel,W,aoH,ake,K&neil,LCtractio-ucedCellWoundingdReleasefobst Growth-Factor in Heart. Circ Res 76,927-934 (1995).   
39.Han,R.etal.Dysferl-mediatedmembranerepairprotcts theeartfromstres-nducedleftventricularijury.JClinIvest (2007).   
40.Wenzel,K.et al.Dysfunction of dysferlin-deficient hearts.J Mol Med 85,1203-1214(2007).   
41.McNeil,P.L.&Kirchhausen,T.Anemergencyresponseteamformembranerepair.NatRevMol CellBio6,499-505 (2005).   
42.Zhu,H.etal.PolymeraseTranscriptaseReleaseFactor(PRF)AnchrsMG53ProteintoCellInjuryitefritiationofembane Repair. JBiol Chem 286,12820-12824 (2011).   
43.Murphy,S.,Martin,S.&Parton,R.G.QuantitativeAnalysisofLipidDropletFusion:IneficientSteadyStateFusionbutRapid Stimulation by Chemical Fusogens.Plos One 5,doi:ARTN e15030 (2010).   
44.Detrait,E.etal.Axolemmalrepairrequires proteinsthatmediate synapticvesiclefusio.JNeurobiol4,82-391(000).   
45.WattM.J.&Steinberg,G.R.Regulationandfunctionoftracylgycerolipasesincelularmetabolism.BochemJ414,13-32 (2008).   
46.Souza,S.CetalOverexpressoofrlinAndBblocksthlitoftumorecosisctorlatoinceaselosis-l adipocytes.J Biol Chem 273,24665-24669 (1998).   
47. Yang, X. et al. The ${ \mathrm { G } } ( 0 ) / { \mathrm { G } } ( 1 )$ switch gene 2 regulates adipose lipolysis through association with adipose triglyceride lipase. Cell Metab 11,194-205 (2010).   
48.Granea,JGe,H.P,otilo.Pu,Z&ou,Lteactisperili-5()doergl JBiol Chem 286,5126-5135 (2011).

# Acknowledgements

This work was supported by the following grants the National Natural Science Foundation of China (No. 81270932,81471082,U1402225),the Ministry of Science and Technology of China (No.2011CBA00906),and the Beijing Natural Science Foundation (No.5122028).

# AuthorContributions

L.L.and H.Z. performed all the experiments including sample preparation, mass spectrometry analysis,and data analysis.Y.H., W.W.and M.Z. were involved in data analysis.J.W., Q.S.andFY. were involved in proteomics study. S.Z.,S.X.,JL.and Z.Q. were in involved in the sample preparation. H.Z.and P.L.outlined the research plan and H.Z.developed the main body of the manuscript and all authors participated in revising the manuscript.

# Additional Information

Supplementary information accompanies this paper at http://www.nature.com/srep

Competing financial interests: The authors declare no competing financial interests.

Howto citethis article:Li,L.etal.Comparative proteomics revealsabnormal bindingof ATGLanddysferlin on lipid droplets from pressure overload-induced dysfunctionalrat hearts.Sci.Rep.6,19782; doi: 10.1038/ srep19782 (2016).

This work is licensed undera Creative Commons Attribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is notincluded under the Creative Commons license, users will need to obtain permisson from the license holder to reproduce the material.To viewa copy of this license,visit http://creativecommons.org/licenses/by/4.0/