# RESEARCH ARTICLE

# Human BDCA2+CD123+CD56+ dendritic cells (DCs) related to blastic plasmacytoid dendritic cell neoplasm represent a unique myeloid DC subset

Haisheng $\mathsf { Y u } ^ { 1 , 3 }$ , Peng Zhang2.3, Xiangyun Yin1.3, Zhao Yin4, Quanxing $\mathsf { \pmb { s h i } } ^ { 4 }$ , Ya Cui², Guanyuan Liu5, Shouli Wang4, Pier Paolo Piccaluga, Taijiao Jiang², Liguo Zhang1

1 Key Laboratoryof Immunityand Infection,Institute of Biophysics,Chinese AcademyofSciences,Beijing 100101,China   
2 Key Laboratory of Protein and Peptide Pharmaceuticals,Institute of Biophysics, Chinese Academy of Sciences, Beijing 100101,China   
3 Graduate School of the Chinese Academy of Sciences,Bejing 100080,China   
4 DepartmentofCardiology306thHospitalof PLABeijing10001,China   
5DeparmentofGecolondbstetricsejingaoyangHospital,CapitaleicalUiversityeingoin   
6DepartmentofExperimentaliagnosticandpecialtyedicineHematopathologyHematologyectionsolecular Pathology Laboratory, S. Orsola-Malpighi Hospital, Bologna University, Bologna 40126,Italy   
区Correspondence: taijiao@moon.ibp.ac.cn (T. Jiang),liguozhang@ibp.ac.cn (L. Zhang)   
Received December10,2014 Accepted December25,2014

# ABSTRACT

Dendritic cells (DCs) comprise two functionally distinct subsets: plasmacytoid DCs (pDCs) and myeloid DCs (mDCs). pDCs are specialized in rapid and massive secretion of type l interferon (IFN-l) in response to nucleic acids through Toll like receptor (TLR)-7 or TLR-9. In this report, we characterized a ${ \mathsf { C D 5 6 } } ^ { + }$ DC population that express typical pDC markers including CD123 and BDCA2 but produce much less IFN-l comparing with pDCs. In addition， CD56+ DCs cluster together with mDCs but not pDCs by genome-wide transcriptional profiling. Accordingly, CD56+ DCs functionally resemble mDCs by producing IL-12 upon TLR4 stimulation and priming naive Tcells without prior activation.These data suggest that the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs represent a novel mDC subset mixed with some pDC features.A ${ \mathsf { C D } } 4 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ hematological malignancy was classified as blastic plasmacytoid dendritic cell neoplasm (BPDCN) due to its expression of characteristic molecules of pDCs.

However, we demonstrated that BPDCN is closer to CD56+ DCs than pDCs by global gene-expression profiling. Thus,we propose that the ${ \mathsf { C D } } 4 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ neoplasm may be a tumor counterpart of CD56+ mDCs but not pDCs.

KEYWORDS dendritic cells, ${ \mathsf { C D 5 6 } } ^ { + }$ DC,pDC,mDC, BPDCN

# INTRODUCTION

Dendritic cells (DCs） are professional antigen-presenting cells found in virtually all tissues.The main function of DCs is toinduce T-cell activation， polarization and expansion against invading pathogens while maintaining tolerance to self antigens (Steinman，2007).Three DC subsets have been identified in the human blood:plasmacytoid dendritic cell (pDC) and two subsets of myeloid DC (mDC) expressing CD1c (BDCA1) and CD141(BDCA3) respectively(Dzionek et al.,2000; Ziegler-Heitbrock etal.,2010). pDCs are characterized by their specific expression of CD123，BDCA2, BDCA4 and ILT7 (Dzionek et al.， 200O; Cao et al., 2006; Rissoan et al., 2002). In addition,pDCs express high levels ofinterferon regulatory factor 7 (IRF7)，Toll like receptor (TLR)-7 and TLR-9. Accordingly， they produce large amounts of type 丨 interferon (IFN-l) upon stimulation with nucleic-acid ligandsofTLR7/TRL9 (Siegal etal., 1999; Cella et al.，1999).Transcription factors E2-2 and Spi B are specifically expressed in pDCs and play important roles in their development and maintenance (Cisse et al.，2008; Schotte et al., 2004).

Blastic plasmacytoid dendritic cell neoplasm (BPDCN) is arare hematological malignancy characterized by the clonal proliferation of pDC-like cells (Facchetti et al.,20o8). Neoplastic cells from BPDCN patients express pDC specific molecules，suchasBDCA2，CD123，CD4，TCL1，Bcl11A CD2AP and Spi B (Petrella and Facchetti， 2010). Additionally,BPDCN cells can produce IFN-I upon TLR7 and TLR9 ligands stimulation although at much lower level comparing withpDCs (Chaperot et al.，2001). Interestingly，BPDCN cells also express CD56,which is not present on the majority of pDCs (Grouard et al., 1997). A small population of DCs in normal or FLT3 mobilized human blood do express CD56, which have been proposed as a pDC subpopulation that gives rise to BPDCN (Petrella etal.,2002;Comeau,2002). Inthisreport， we characterizedthese BDCA2+ $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ DCs from human peripheral blood.Although they express some pDC featured molecules,the transcriptomic and functional characterizations indicate that they belong to mDCs rather than pDCs.

# RESULTS

# CD56+ DCs are functionally distinct from pDCs

BDCA2 and CD123 are commonly used for the identification of human pDCs in the blood and lymphoid tissues.A subpopulation of ${ \mathsf { B D C A 2 ^ { + } C D 1 2 3 ^ { + } C D 5 6 ^ { + } } }$ $( \mathsf { C D } 5 6 ^ { + } )$ DCs have been reported in human blood and proposed as a pDC subpopulation related to BPDCN (Petrella et al.，2002; Comeau, 2002).We first confirmed the presence of ${ \mathsf { C D 5 6 } } ^ { + }$ subset in the HLA-DR+,CD11c",BDCA2+ and CD1 $2 3 ^ { + }$ DCs in human blood (Fig. 1A and 1B). The ${ \mathsf { C D 5 6 } } ^ { + }$ subset representsabout $5 \%$ of the BDCA2 and CD123 double positive cells(Fig. 1B, $n = 1 5$ ). Itwas reported that pDCs can be divided into two populations by their CD2 expression (Matsui et al., 2009). We found that the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs universally express CD2 (Fig. 2A).

Next, we purified the ${ \mathsf { C D 5 6 } } ^ { + }$ and $\mathsf { C D 5 6 } ^ { - }$ cells to compare their IFN-l production byTLR7 orTLR9 stimulation.In order to get higher purity,we added CD2 in our purification protocol， thus $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 2 ^ { + } \mathsf { C D } 5 6 ^ { + }$ and $\mathsf { C D 1 2 3 ^ { + } C D 2 ^ { - } C D 5 6 ^ { - } }$ DCs were sorted by FACS for further analysis.Usually the purity of ${ \mathsf { C D 2 ^ { - } C D 5 6 ^ { - } } }$ cells was over $9 5 \%$ and the purity of ${ \mathsf { C D } } 2 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ cells was over $80 \%$ with contamination of some ${ \mathsf { C D 2 ^ { + } C D 5 6 ^ { - } } }$ cells (Fig.2B). High level IFNα production upon TLR7 or TLR9 stimulation is the key characteristic of pDCs. However，purified ${ \mathsf { C D 5 6 } } ^ { + }$ DCs produced much less IFNα upon CpG A stimulation comparing to $\mathsf { C D 5 6 } ^ { - }$ cells from same donors (around $1 0 \%$ ）(Fig.2C，left panel).Similar results were also observed when the purified DCs were stimulated with influenza virus (Fig.2C,right panel).In order to exclude the effect of CD56 antibody binding during purification,we also compared the IFNα production capacity of ${ \mathsf { C D 5 6 } } ^ { + }$ and $\mathsf { C D 5 6 } ^ { - }$ DCs by intracellular staining without prior purification.When enriched DCs were stimulated with CpG A and stained IFNα intracellularly,above $70 \%$ of $\mathsf { C D 5 6 } ^ { - }$ cells but only $10 \%$ ${ \mathsf { C D 5 6 } } ^ { + }$ cells expressed IFNα (Fig.2D，left panel). Similar results were observed by R848,a TLR7 Iigand stimulation (Fig.2D,right panel).

# CD56+ DCs clustered together with mDCs but not pDCs bytranscriptomic analysis

In order to compare the gene expression of ${ \mathsf { C D 5 6 } } ^ { + }$ DCs and other DC subsets at whole genome level,we analyzed the transcriptome of both $\mathsf { C D 5 6 } ^ { - }$ pDCs and ${ \mathsf { C D 5 6 } } ^ { + }$ DCs by RNA sequencing (RNA-seq).Next,we compared the transcriptional profile of ${ \mathsf { C D 5 6 } } ^ { + }$ DCs with cDNA array data of human DC subsets retrieved from public databases (Robbins et al., 2008).There were 772 genes expressed in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs at higher level (over 2 times) than those of $\mathsf { C D 5 6 } ^ { - }$ DCs and 2398 genes vice versa. Among those，648 genes highly expressed in ${ \mathsf { C D 5 6 } } ^ { + }$ cells (Fig.3A,left panel) and 1557 lowly expressed ones (Fig.3B，left panel) were present in the cDNA array data of human blood DC subsets (Robbins et al., 2008).And the names of those differentially expressed genes are provided in Table S1 and S2. Intriguingly, 547 $( 9 1 \% )$ ${ \mathsf { C D 5 6 } } ^ { + }$ DC highly expressed genes present at higher level in mDCs than pDCs based on cDNA array data (Fig.3A, right panel; Table S1).Similarly，1292 $( 8 3 \% )$ ${ \mathsf { C D 5 6 } } ^ { + }$ DCs lowly expressed genes presented at lower level in mDCs than pDCs (Fig.3B, right panel; Table S2).

To clarify the relationship of ${ \mathsf { C D 5 6 } } ^ { + }$ DCs and other DC subsets,we normalized our RNA-Seq data with cDNA data, then did hierarchical clustering with complete linkage and principal component analysis (PCA)(11,372 genes). Both analyses showed that $\mathsf { C D 5 6 } ^ { - }$ DCs clustered together with pDCs as expected,while the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs clustered together with BDCA3 and BDCA1 mDCs (Fig.3C and 3D).

# CD56+ DCs are functionally analogous to mDCs

TLR expression patterns of pDCs and mDCs are drastically different.pDCs preferentially express TLR7 and TLR9while mDCs express higher levels of TLR2 and TLR4 (Liu 2005; Cerboni et al.,2013).We found that the expression level of TLR4 (Fig. 4A) and the amount of TNFα produced upon LPS stimulation (Fig.4B and 4C) were similarbetween CD56+DCs andmDCs.Moreover, ${ \mathsf { C D 5 6 } } ^ { + }$ DCs could produce IL-12 at a comparable level to mDCs at both RNA (Fig. 4D) and protein level (Fig.4E).While pDCs did not produce IL-12 (Fig.4D and 4E)as previously reported (Kadowaki etal., 2001).

Different from pDCs，mDCs could prompt T-cell proliferation without prior stimulation or activation.In order to further characterize the relationship between ${ \mathsf { C D 5 6 } } ^ { + }$ DCs, pDCs and mDCs,we examined more gene expression that related to antigen presentation.MHC Il molecules and CD86 were highly expressed on ${ \mathsf { C D 5 6 } } ^ { + }$ DCs compared to pDCs at RNA level (Table S3).These were confirmed at protein level byFACS analysis (Fig.4F).Gamma-interferon-inducible lysosomal thiolreductase(GlLT) isan essential enzyme for antigen processing,(Singh and Cresswell 2010;Maric et al., 2001）and we found that GlLT and cathepsins were expressed at much higher levels in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs than pDCs (Table S3).We next examined the antigen presentation by ${ \mathsf { C D 5 6 } } ^ { + }$ DCs,pDCs and mDCs.Allogeneic naive Tcells were co-cultured with purified DCs and analyzed by tritium incor poration．The ${ \mathsf { C D 5 6 } } ^ { + }$ DCs stimulated comparable T-cell proliferation to mDCs,while pDCs did not induce significant T-cell expansion (Fig. 4G).

![](images/2d727aaf0c5e5c8bf969319e45fafe62dbd16b26a79e00b36a3f33b92fb6c17c.jpg)  
Figure1.CD56+DCsinhumanblood.(A)Human PBMC werefirstgatedbasedonlightscatters,anddeadcellswereexcluded by yellw fluorescentreactivedye(L/Ddye),thenLinHLA-DR+BDCA2+CD123+dendriticcels(DCs)weregatedforfurtheranalysis. (B)GatedLinHLA-DR+BDCA2+CD13+bloodDCs weredividedinto two subsets bytheirCD56 expresion.(C)Summarieddata show relative abundance of $\mathsf { C D 5 6 } ^ { - }$ and CD56+ populations in BDC ${ \tt S A 2 ^ { + } C D 1 2 3 ^ { + } }$ DCs (Mean $\pm$ SD, $n = 1 5$ ）

![](images/526871d3d4715b0e84ab55b9f9c810a1503acef507a24c918e3ae82b4b7bf348.jpg)  
Figure 2.CD56\* DCs are functionaly distinctfrompDCs.(A) Human PBMC were gated on Lin"HLA-DR+CD11c-CD123+ DCs, andthegated DCs wereanalyzedbytheirCD2and CD56 expression.(B)Lineage depleted PBMC weregatedon LinHLA$\mathsf { D R } ^ { \mathrm { + } } \mathsf { C D } 1 1 \mathsf { c } ^ { \mathrm { - } } \mathsf { C D } 1 2 3 ^ { \mathrm { + } }$ DCs, then ${ \mathsf { C D } } 2 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ and ${ \mathsf { C D } } 2 { \mathsf { C D } } 5 6 ^ { - }$ DCs were sorted by FACS.The purities of sorted cels are shown in each plot. (C) 10,000 purified $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { - }$ and $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ DCs were stimulated with CpG A or influenza virus (Flu) for $2 4 \mathsf { h }$ theninterferonα(IFNa)inthe supernatants wereanalyzedbyELISA.Data summarizedfrom 3independent donors.(D)Lineage depleted PBMCs were stimulatedwith CpGAor R848thenanalyzedbyFACSfor surface markersand IFNαintracelularly. $\mathsf { L i n ^ { - } H L A D R ^ { + } C D 1 1 c ^ { - } C D } 1 2 3 ^ { + }$ cells were gated and ploted for CD56 and IFNα.The numbers in each quadrant represent percentage of total gated cells.One out of four independent experiments is shown.

![](images/25af7f373c85be1ae889f8c0ef13807579dd74632bbf2075e76e726ce52725ea.jpg)  
Figure 3. Comparing gene expression of CD56+ DCs and other DC subsets. (A) 648 genes expressed in $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ DCs at least two fold higher than in $\mathsf { C D } 1 2 3 ^ { - } \mathsf { C D } 5 6 ^ { - }$ DCs obtained by RNA-Seq (left panel) and their expressions in pDCs, BDCA3 mDCs and BDCA1 mDCs extracted from cDNA array data (right panel). (B) 1557 lowly expressed genes in $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ relative to $\mathtt { C D 1 2 3 } ^ { - }$ $\mathsf { C D 5 6 } ^ { - }$ DCs(left panel)and their expressions in pDCs,BDCA3 mDCsand BDCA1 mDCs extracted from cDNA arraydata (right panel). (C) Clustering of $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ and CD123" CD56" DCs with other immune cellineages isolated from human peripheral blood based on their geneexpression profiles.(D）The principal component analysis(PCA）results.Mono，monocytes；Neu, neutrophils;NK,natural killer cells.

![](images/169e19a62902335d283d7dcc971cf7d80f7aec7bcf868acfabab7f7ea78862df.jpg)  
Figure4.CD56+DCsfunctionallyresemble mDCs.(A)TheexpressionofTLR4 wasquantifiedbyRT-PCRwith purifiedDCs.Error barsrepresent standardderivationfromduplicatedPCR samplesofthe samedonorData shownare fromonerepresentativeof 3 independent donors.(B) Lineage depleted PBMC were stimulated with LPS for $6 \mathfrak { h }$ ，then the stimulated cells were collected and stained with surface markers followed by intracellular staining of TNFα. $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { - }$ (pDC) $\mathsf { C D 1 2 3 ^ { + } C D 5 6 ^ { + } \left( C D 5 6 ^ { - } \right. }$ )and $\mathtt { C D 1 2 3 } ^ { - }$ ${ \mathsf { C D } } 1 1 { \mathsf { C } } ^ { + }$ (mDC)in the Lin" HLA-DR+gate wereanalyzed for TNFα expression.(C) Summarized dataof (B)from 5 independent donors.(D) Purified DCs were stimulated with mixture of LPS,CpG and IFNy for $6 \mathfrak { h }$ ,the expression of IL-12 p35 and $\mathsf { p } 4 0$ were analyzedbyRT-PCR.Datashownarerepresentativeof3independent experiments.(E)PurifiedDCs were stimulated mixtureof LPS,CpG and IFNy for $2 4 \ h$ ，and the expression of IL-12 p7O were analyzed by ELISA. Data shown are representative of 2 independentexperiments.(F)ExpressionofHLA-DRandCD86ondiferentbloodDCsubsetswereanalyzedbyFACS.(G)Purified allogenic naive T cells were cocultured with different numbers of $\mathsf { C D 1 2 3 ^ { + } C D 5 6 ^ { - } }$ (pDC,circle) ( $\mathtt { C D 1 2 3 ^ { + } C D 5 6 ^ { + } }$ 0 $\cos 6 ^ { + }$ ，square)and $\mathsf { C D 1 2 3 ^ { - } C D 1 1 C ^ { + } }$ (mDC,triangle) DCs.After 5 days,the cocultured cells were pulsed with 1 mCi $[ ^ { 3 } \mathsf { H } ]$ thymidine for $1 8 \mathrm { ~ h ~ }$ before harvest. Data shown are one representative of 4 experiments.

# CD56+ DCs are unique mDCs mixed with some pDC features

${ \mathsf { C D 5 6 } } ^ { + }$ DCs expressed BDCA2 and CD123，which were considered as hallmarks of pDCs (Fig.1B). In order to further characterize the relationship between ${ \mathsf { C D 5 6 } } ^ { + }$ DCs and pDCs,we examined more pDC-specific genes at both RNA and protein levels in other DC subsets.And we found that BDCA4 and ILT7，two additional pDC specific surface markers，were expressed on ${ \mathsf { C D 5 6 } } ^ { + }$ DCs but not mDCs (Fig.5A).Both E2-2 and SpiB,the two important transcription factors for pDC lineage development,were expressed in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs at slightly lower levels than pDCs in the RNAseq data (Table S4) and confirmed by RT-PCR(Fig.5B, left two panels). It is worth to pointing out that the expression levels of E2-2 and SpiB in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs were much higher than in mDCs (Fig.5B, left two panels). In addition，pDC highly expressed genes IRF7， BCL11A and CD2AP (Marafioti etal.,20o8) were also expressed in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs at a slightly lower level compared to pDCs (Table S4).However，other pDC specific genes，such as Granzyme B, TCL1A (Herling et al., 2003), PACSIN1 (Esashi etal., 2012) and BAD-LAMP (Defays et al.，2011)，were expressed at least 10 times higher in pDCs than in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs in RNAseq data (Table S4).

We have already shown that ${ \mathsf { C D } } 5 6 ^ { + }$ DCs produced very low level IFNα upon CpG stimulation.And this is correlated with the lower expression level of TLR9 in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs than pDCs.However， the expression of TLR9 in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs is clearly higher than in mDCs (Fig.5B,far right panel). Interestingly， ${ \mathsf { C D 5 6 } } ^ { + }$ DCs produced comparable levels of TNFα to those of pDCs upon CpG B stimulation (Fig.5C-E).These results argue that ${ \mathsf { C D 5 6 } } ^ { + }$ DCs are unique comparing other mDCs by maintaining certain pDC characteristic features.

# BPDCN is closely related to CD56+ DCs, but not pDCs by global gene expression profiling

Ina previous report,we have found that BPDCN shows both myeloid and pDC characteristics (Cerboni et al.，2013).In order to clarify the relationship among BPDCN，pDC and ${ \mathsf { C D 5 6 } } ^ { + }$ DCs,we analyzed the differentially expressed genes between BPDCN and pDCs.We identified 127 genes express at higher levels in BPDCN than in pDC in cDNA array data (Fig.6A,left panel; Table S7),but 93 $( 7 3 \% )$ of them are highly expressed in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs compared to pDCs in our RNA-seq data (Fig.6A,right panel). Similarly, there are 1143 genes expressed at lower levels in BPDCN cells than in pDCs (Fig.6B, left panel; Table S8),and 778 $( 6 8 \% )$ of them are highly expressed in the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs than pDCs (Fig. 6B, right panel). The global gene expression profiling data suggests that BPDCN is more closely related to the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs than pDCs.

# DISCUSSION

${ \mathsf { C D 4 ^ { + } C D 5 6 ^ { + } } }$ hematologic malignancy with primary cutaneous presentation has been well documented and is proposed as the malignant counterpart of pDC precursors (Chaperot et al.，2001；Adachi et al.，1994；Brody 1995; Uchiyama 1998;Petrella et al.，1999)and was named as blastic plasmacytoid dendritic cell neoplasm(BPDCN) (Facchetti etal., 20o8). Here in this report,we characterized aBDC $\mathtt { A } 2 ^ { + } \mathtt { C } \mathtt { D } 1 2 3 ^ { + } \mathtt { C } \mathtt { D } 5 6 ^ { + }$ blood DC subset,which has been considered as a subpopulation of pDCs (Petrella et al., 2002).We demonstrated that the ${ \mathsf { C D 5 6 } } ^ { + }$ DCs clustered with mDCs by transcriptional profiling at whole genome level. Accordingly, ${ \mathsf { C D 5 6 } } ^ { + }$ DCs responded to TLR4 stimulation and prime naive Tcells without prior stimulation.And this agrees with our previous finding that BPDCN are characterized by their mixed pDC and myeloid signature (Sapienza et al., 2014).However, ${ \mathsf { C D 5 6 } } ^ { + }$ DCs are distinguished from other mDCs by expressing pDC specific molecules．Thus，we propose that ${ \mathsf { C D 5 6 } } ^ { + }$ DCs represent a unique mDC subset and their function in immune response warrant further characterization.

![](images/fa9e35aec43017625a7f86e2613112397631bf45f5d260e8800a0ceb1164ab39.jpg)  
Figure 5.CD56DCsadoptcertaincharacteristicsattributed topDCs.(A)Theexpressionof BDCA4andILT7indifferentDC subpopulations from PBMC were analyzed by FACS. Data shown are one representative of 3 donors.(B) $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { - }$ (pDC), （204号 $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ (CD56\*) and $\mathsf { C D } 1 2 3 ^ { - } \mathsf { C D } 1 1 \mathsf { C } ^ { + }$ (mDC) were purified from PBMC,E2-2, Spi-B and TLR9 expression in those three DC populations wereanalyzedbyRT-CR.Erorbarsrepresentstandardderivationfromduplicated PCRsamplesofsamedonor.Data shown are one representative of 3 donors.(C) Purified DCs were stimulated with $\mathsf { C p G B }$ for $2 4 \ h$ ,and the expression of TNFα were analyzedbyELISA.Datashownaresummarizedfrom3independent experiments.(D)Lineage depletedPBMCswere stimulated with CpG B for $6 \mathrm { ~ h ~ }$ ，then stimulated cells were collected and stained with Lin,HLA-DR,CD123,CD11C and CD56 followed by intracellular staining of TNFα.The Lin" HLA-DR+ cells were divided into $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { - }$ (pDC), $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ (CD56+) and $\mathtt { C D 1 2 3 ^ { - } }$ ${ \mathsf { C D } } 1 1 { \mathsf { C } } ^ { + }$ (mDC)subsets,TNFα expression in each DC subset were analyzed.(E) Summarized data of (D)from 5 independent donors.

![](images/ea22974b9579a255b201db7a37b5ebe06b9c86f369674980222f0fdec27dca81.jpg)  
Figure 6.Global gene expression of BPDCN,pDCs and CD56+DCs.(A) Heat mapof 127 genes highly expressd in BPDCN comparing with pDCs by cDNA array (left panel) and their expressions in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs relative to $\mathsf { C D 5 6 } ^ { - }$ pDCs by RNA-Seq (right panel).(B)Heat mapof 1143 genes highly expressed inpDCscomparing with BPDCNbycDNAarry(left panel)and their expressions in ${ \mathsf { C D 5 6 } } ^ { + }$ DCs relative to $\mathsf { C D 5 6 } ^ { - }$ pDCs by RNA-Seq (right panel).

Besides BPDCN,another pDC related tumoral condition, pDC proliferations in patients with myeloid disorders (PPMD), hasalso been reported (Petrellaand Facchetti 201O).The tumor cells of BPDCN and PPDM share the common characteristic phenotype of pDCs:lin"， $\mathsf { C D 4 } ^ { + }$ ， ${ \mathsf { C D 6 8 } } ^ { + }$ ， ${ \mathsf { C L A } } ^ { + }$ ${ \mathsf { C D } } 1 2 3 ^ { + }$ ，BDCA2+, ${ \mathsf { B c l } } 1 1 { \mathsf { A } } ^ { + }$ and ${ \mathsf { C D } } 2 { \mathsf { A P } } ^ { + }$ ．However,CD56 expression,which is the hallmark of BPDCN, is not detected inPPDM (Petrella and Facchetti 2010;Vermi etal.,2004). Granzyme B (GrB) expresses at high level in pDCs (Rissoan et al.,2002) and is also positive in PPMD(Vermi etal.,2004). However，most reported BPDCN cases are GrB negative (Chaperot etal.,2001; Petrella et al.,2002; Pilichowska et al., 2007;An et al., 2013). Here in this study，we show ${ \mathsf { C D 5 6 } } ^ { + }$ DCs express much lower level of GrB compared to pDC (Table S4).We speculate that PPMD may be the tumor counterpart of pDC,while BPDCN may represent the counterpartof $\mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 5 6 ^ { + }$ DCs.This hypothesis needs to be further investigated in the future by transcription profiling at whole genome level for both DC subsets and their tumor counterparts.

It is worth pointing out that the human ( $\mathtt { C D 1 2 3 ^ { + } C D 5 6 ^ { + } }$ DCs are similar to a $\mathsf { C X } _ { 3 } \mathsf { C R } 1 ^ { + } \mathsf { C D } 8 \mathsf { a } ^ { + }$ mouse DC subset,which also exhibits mixed characteristics of pDC and mDC (Ghosh et al.， 2010). The $\mathsf { C X } _ { 3 } \mathsf { C R } 1 ^ { + } \mathsf { C D } 8 \mathsf { a } ^ { + }$ mouse DCs are developmentally dependent on E2-2 and express pDC specific genes (Spi-B,Siglec H,Bst2).However, $\mathsf { C X } _ { 3 } \mathsf { C R } 1 ^ { + }$ ${ \mathsf { C D 8 } } { \mathsf { a } } ^ { + }$ DCs do not produce IFNα upon TLR7 and TLR9 stimulation (Ghosh et al., 2010). The $\mathsf { C X } _ { 3 } \mathsf { C R } 1 ^ { + } \mathsf { C D } \mathsf { 8 } \mathsf { a } ^ { + } \mathsf { I }$ DCis transcriptionally similar to mouse $\mathsf { C D 8 } ^ { - }$ mDCs,but not to $\mathsf { C D 8 ^ { + } }$ mDCs (Ghosh et al., 2010). The developmental relationship betweenhuman ${ \mathsf { C D 5 6 } } ^ { + }$ DCsandmouse $\mathsf { C X } _ { 3 } \mathsf { C R } 1 ^ { + } \mathsf { C D } 8 \mathsf { a } ^ { + } \mid$ DCsneeds further investigation.

Taken together， we have demonstrated that human ${ \mathsf { C D 5 6 } } ^ { + }$ DCs were very close to mDC by both transcriptomic analysis and functional characterization.Thus,we propose that ${ \mathsf { C D 5 6 } } ^ { + }$ DC should be classified as mDC, but not pDCs. And we propose that BPDCN should be classified as mDC butnot pDC related leukemia.

# MATERIALSANDMETHODS

# FACS analysis of DC subsets in human blood

Foranalysis of DC subsets in human peripheral blood,single cells were first gated by light scatters and dead cells were excluded by yellow fluorescent reactive live/dead cell dye staining (Life Technologies,Carlsbad,USA), then the lineage positive cellswere gated out by specific markers(CD3，CD14，CD16，CD19)．HLA${ \mathsf { D } } { \mathsf { R } } ^ { + } { \mathsf { C } } { \mathsf { D } } 1 2 3 ^ { + } { \mathsf { B } } { \mathsf { D } } { \mathsf { C } } { \mathsf { A } } 2 ^ { + }$ cellswere further divided by their CD2 and CD56 expression.Antibody stained peripheral blood mononuclear cells (PBMC)were analyzed with LSRfortessa (BD Biosciences, Franklin Lakes,NJ,USA) and data were analyzed with Summit 4.3 (DAKO,Denmark).Detailed informationofantibodiesused for flow cytometry analysis was shown in Table S5.

# DC enrichment and in vitro stimulation

DCs were enriched by lineage (CD3,CD14, CD16,CD19) depletion, then lineage negative cells( $2 \times 1 0 ^ { 5 }$ in $2 0 0 ~ \mu \iota$ culture medium) were stimulated with TLR ligands or viruses for $4 \mathfrak { h }$ followed byanother $2 h$ inthe presence of Golgi Blocker (BD Biosciences,Franklin Lakes, NJ,USA).Cells were stained with surface markers and permeablized and stained with antibodies against various cytokines.TLR ligands were purchased from Invivogen (San Diego，CA，USA) and used at following concentrations: LPS $( 1 ~ \mu \mathfrak { g } / \mathfrak { m } \ L )$ ，CpG 2216 $( 2 \ \mu \mathsf { g } / \mathsf { m L } )$ ，R848 $( 2 \ \mu \mathsf { g } / m L )$ ．Heat inactivated influenza virus A/PR8/34 was usedat10 MOl for stimulation.

# DC purification and stimulation

The lineage (CD3,CD14,CD16,CD19,CD20) depletedPBMC were stained with HLA-DR (APC-Cy7)，CD2 (PE-Cy7)，CD11c (APC), CD56(PerCP-cy5.5) and CD123(BV421)．Lin-HLA$\mathsf { D R } ^ { + } \mathsf { C D } 1 2 3 ^ { + } \mathsf { C D } 1 1 \mathsf { c } ^ { - } \mathsf { C D } 2 ^ { + } \mathsf { C D } 5 6 ^ { + }$ and Lin-HLA-DR+CD123+CD11c ${ \mathsf { C D } } 2 { \mathsf { C D } } 5 6 ^ { - }$ DCs were sorted with BD FACSArialll (BD Biosciences, Franklin Lakes， NJ，USA). In some experiments， Lin-HLA$\mathsf { D R ^ { + } C D 1 2 3 ^ { - } C D 1 1 c ^ { + } }$ mDC were sorted as control. Purified DCs 0 $1 \times 1 0 ^ { 4 }$ in $2 0 0 ~ \mu \iota$ culture medium) were stimulated with diferent TLR ligands at the concentration mentioned above.For IL-12 production，purified DCs were stimulated with LPS，CpG 2006 （204号 $( 1 ~ \mu \mathrm { m o l / L } )$ and IFNγ $( 5 0 \mathrm { \ n g / m L } )$ .IFNα, IL-12and TNFα levels in the supernatantwere quantified by ELISA.

# RNA-seq and data analysis

RNA extraction and sequencing were done by BGl Tech (Shenzhen, Guangdong,China). The gene expression level was measured by the number of uniquely mapped reads per kilobase of exon region per million mappable reads (RPKM). The RNA-seq raw data was aligned to the reference genome (hg19) (Trapnell etal.,20o9).The gene expressions of other cell types used in our study were obtained from cDNA array data by Scott H Robbins (Robbins et al., 2008). The target gene regions of microarray probes were retrieved from the annotation file for human U133 Plus 2.0 (http://www.affymetrix. com).To compare gene expressions from both RNA-seq and cDNA arrayexperiments,we only considered the reads that haveat least one nucleotide overlap with the gene regions targeted by microarray probes.Heat maps were generated with differentially expressed genes between ${ \mathsf { C D } } 2 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ and ${ \mathsf { C D } } 2 { \mathsf { C D } } 5 6 ^ { - }$ DCs (larger than 2 times difference).Transcriptome comparison of ${ \mathsf { C D } } 2 ^ { + } { \mathsf { C D } } 5 6 ^ { + }$ DCs and other blood cells lineages were carried out with two different mathematical methods, hierarchical clustering with complete linkage (Eisen et al.,1998) and principal component analysis (PCA) (Alter et al.,20oO). Detailed methods of RNA-seq and data analysis were provided in the supplemental file.

BPDCN microarray data of 6 cryopreserved tissue samples (Sapienza etal.,2014)by Affymetrixmicroarray platformwere normalized with the rank invariant method using Lumi (Bioconductor) (Du et al.,20o8). Statistics for differential expression were applied by

Limma (Bioconductor), (Smyth 2004) and we defined BPDCN significant differential expressed genes on normalized dataasadjusted $P .$ -value less than O.05.One hundred and twenty seven BPDCN highly expressed genes and 1143 BPDCN lowly expressed genes were matched to our RNA-seq datasets of pDC and ${ \mathsf { C D 5 6 } } ^ { + }$ DCs (genes with RPKM less than 10 were deleted).Heat maps were displayed using the R software.

# RT-PCR

RNA was extracted from purified DCs and reverse transcribed with Oligo dT primers.Quantitative real time PCR (RT-PCR) was performed using cDNA with EF1α gene as internal control.Primers for RT-PCR are listed in Table S6.

# T-cell proliferation assay

Human $\mathsf { C D 4 ^ { + } C D 4 5 R A ^ { + } }$ naive T cells were purified from PBMC by negative selection with magnetic beads (Miltenyi Biotech, Germany). Different numbers of FACS purified DCs were cultured with $5 \times 1 0 ^ { 4 }$ allogeneic naive Tcells in 96-well round-bottomed plates in $2 0 0 ~ \mu \iota$ culture medium.After 5 days of DC and Tcell co-culture,cells were pulsed with $1 \mathrm { \ m C i }$ （204号 $[ ^ { 3 } \mathsf { H } ]$ -thymidine for $1 8 \mathsf { h }$ before harvest.Radioactive uptake was measured with a TopCount NXT micro-plate scintillation and luminescence counter (PerkinElmer).

# ACKNOWLEDGMENTS

This work was supported by grants from Beijing Municipal Science & Technology Commission (SCW 2014-09)，Chinese Academy of Science(KJZD-EW-L10-02)andMinistryofHealth (2013ZX10001002) to L.Z.The authors thank Dr.Lishan Su for critical reading this manuscript and Mr.Anthony Curtis for English editing.H.Y.and L.Z.designed the project,analyzed data and wrote the paper; P.Z., Y.C.and T.J.analyzed and interpreted the RNA-seq dataand wrote the paper;P.P.participated in the analysis of BPDCN data;X.Y.,Z.Y.,Q.S.,S.W.and G.L. performed experimentsand analyzed data.Theauthorsdeclareno financial conflict of interest in relationto thiswork.

# ABBREVIATIONS

BPDCN, blastic plasmacytoid dendritic cell neoplasm; DCs,dendritic cells;IFN-l， type 丨interferon;IRF7,interferon regulatory factor 7; mDC,myeloid DC;PBMC,peripheral blood mononuclear cell; PCA, principal component analysis;pDC,plasmacytoid DC;TLR,toll like receptor.

# COMPLIANCEWITHETHICSGUIDELINES

Haisheng Yu,Peng Zhang,Xiangyun Yin, Zhao Yin,Quanxing Shi YaCui,Guanyuan Liu,Shouli Wang,Pier Paolo Piccaluga,Taijiao Jiang and Liguo Zhang declare that they have no conflict of interest. All procedures followed were in accordance with the ethical standardsof the responsible committee on human experimentation (institutional and national) and with the Helsinki Declaration of 1975, asrevised in 2ooo (5).Informed consent was obtained from all patients for being included in the study.

# OPENACCESS

This article is distributed under the terms of the Creative Commons Attribution License whichpermits anyuse， distribution，and reproduction in any medium，provided the original author(s)and the source are credited.

# REFERENCES

Adachi M,Maeda K,Takekawa M et al (1994) High expression of CD56(N-CAM） in a patient with cutaneous CD4-positive lymphoma.Am J Hematol 47(4):278-282   
Alter O, Brown PO, Botstein D (20oo) Singular value decomposition for genome-wide expression data processing and modeling. Proc Natl Acad Sci USA 97(18):10101-10106   
An HJ,Yoon DH,Kim S et al (2013) Blastic plasmacytoid dendritic cell neoplasm:a single-center experience.Ann Hematol 92 (3):351-356   
Brody JP,Allen S,Schulman P et al (1995) Acute agranular CD4- positive natural killer cell leukemia.Comprehensive clinicopathologic studies including virologic and in vitro culture with inducing agents.Cancer 75(10):2474-2483   
Cao W,Rosen DB,Ito T et al (20o6) Plasmacytoid dendritic cellspecific receptor ILT7-Fc epsilonRl gamma inhibits Toll-like receptor-induced interferon production. J Exp Med203 (6):1399-1405   
Cella M，Jarrossay D，Facchetti F et al (1999） Plasmacytoid monocytes migrate to inflamed lymph nodes and produce large amounts of type I interferon. Nat Med 5(8):919-923   
Cerboni S,Gentili M, Manel N (2013) Diversity of pathogen sensors in dendritic cells.Adv Immunol 120:211-237   
Chaperot L,Bendriss N,Manches O etal (2oo1) ldentification of a leukemic counterpart of the plasmacytoid dendritic cells.Blood 97 (10):3210-3217   
Cisse B,Caton ML,Lehner M etal (20o8) Transcription factor E2-2 is anessential and specific regulator of plasmacytoid dendritic cell development. Cel 135(1):37-48   
Comeau MR (2002) Van der Vuurst de Vries AR,Maliszewski CR, Galibert L.CD123bright plasmacytoid predendritic cells:progenitors undergoing cel fate conversion? J Immunol 169(1):75- 83   
Defays A, David A,de Gassart A et al (2011) BAD-LAMP is a novel biomarker of nonactivated human plasmacytoid dendritic cells. Blood 118(3):609-617   
Du P, Kibbe WA,Lin SM (2008) lumi: a pipeline for processing Illumina microarray. Bioinformatics 24(13):1547-1548   
Dzionek A, Fuchs A, Schmidt P et al (200O) BDCA-2, BDCA-3,and BDCA-4:three markers for distinct subsets of dendritic cels in human peripheral blood. J Immunol 165(11):6037-6046   
Eisen MB，Spellman PT，Brown PO，Botstein D (1998) Cluster analysis and display of genome-wide expression patterns.Proc Natl Acad Sci USA 95(25):14863-14868   
Esashi E，Bao M，Wang YH，Cao W,Liu YJ (2012) PACSIN1 regulates the TLR7/9-mediated type I interferon response in plasmacytoid dendritic cells.Eur J Immunol 42(3):573-579   
Facchetti F，Jones D，Petrella T (2O08）Blastic plasmacytoid dendritic cell neoplasm.In: Swerdlow S，Campo E，Haris N et al (eds) WHO classification of tumours of haematopoietic and lymphoid tissues,4th edn. International Agency for Research on Cancer (IARC), Lyon,pp 145-147   
Ghosh HS, Cisse B, Bunin A,Lewis KL, Reizis B (2010) Continuous expression of the transcription factor e2-2 maintains the cel fate of mature plasmacytoid dendritic cells. Immunity 33(6):905-916   
Grouard G, Rissoan MC,Filgueira L, Durand I, Banchereau J,Liu YJ (1997)The enigmatic plasmacytoid Tcells develop into dendritic cells with interleukin (IL)-3 and CD40-ligand. J Exp Med 185 (6):1101-1111   
Herling M,Teitell MA,Shen RR,Medeiros LJ,Jones D (2003) TCL1 expression in plasmacytoid dendritic cells (DC2s) and the related $\mathsf { C D 4 ^ { + } C D 5 6 ^ { + } }$ blastic tumors of skin. Blood 101(12):5007-5009   
Kadowaki N,Ho S,Antonenko S et al (2001) Subsets of human dendritic cell precursors express different toll-like receptors and respond to different microbial antigens.J Exp Med 194(6):863- 869   
Liu YJ(2005) IPC: professional type 1 interferon-producing cells and plasmacytoid dendritic cell precursors.Annu Rev Immunol 23:275-306   
Marafioti T,Paterson JC,Ballabio E et al (2008) Novel markers of normal and neoplastic human plasmacytoid dendritic cells.Blood 111(7):3778-3792   
Maric M,Arunachalam B,Phan UT et al (2001) Defective antigen processing in GILT-free mice. Science 294(5545):1361-1365   
Matsui T,Connolly JE,Michnevitz M et al (2009) CD2 distinguishes two subsets of human plasmacytoid dendritic cells with distinct phenotype and functions. J Immunol 182(11):6815-6823   
Petrella T，Facchetti F (2010） Tumoral aspects of plasmacytoid dendritic cells:what do we know in 2009? Autoimmunity 43 (3):210-214   
Petrella T, Dalac S, Maynadie M et al (1999) ${ \mathsf { C D 4 } } ^ { + } { \mathsf { C D 5 6 } } ^ { + }$ cutaneous neoplasms:a distinct hematological entity? Groupe Francais d'Etude des Lymphomes Cutanes (GFELC).Am JSurg Pathol 23 (2):137-146   
Petrella T,Comeau MR,Maynadie M etal (2002) ‘Agranular $\mathsf { C D 4 ^ { + } }$ CD56+ hematodermic neoplasm’ (blastic NK-cell lymphoma) originates from a population of CD56+ precursor cells related to plasmacytoid monocytes. Am J Surg Pathol 26(7):852-862   
Pilichowska ME,Fleming MD,Pinkus JL,Pinkus GS (2007) $\mathsf { C D 4 ^ { + } } /$ CD56+ hematodermic neoplasm ("blastic natural killer cel lymphoma"):neoplastic cells express the immature dendritic cell marker BDCA-2 and produce interferon.Am J Clin Pathol 128 (3):445-453   
Rissoan MC，Duhen T，Bridon JM et al (2002) Subtractive hybridization reveals the expression of immunoglobulin-like transcript 7，Eph-B1，granzyme B,and 3 novel transcripts in human plasmacytoid dendritic cells. Blood 100(9):3295-3303   
Robbins SH,Walzer T, Dembele D et al (2O08) Novel insights into the relationships between dendritic cell subsets in human and mouse revealed by genome-wide expression profiling. Genome Biol 9(1):R17   
Sapienza MR,Fuligni F,Agostineli C et al (2014) Molecular profiling of blastic plasmacytoid dendritic cell neoplasm reveals a unique pattern and suggests selective sensitivity to NF-kB pathway inhibition.Leukemia 28:1606-1616   
Schotte R,Nagasawa M,Weijer K,Spits H, Blom B (2004) The ETS transcription factor Spi-B is required for human plasmacytoid dendritic cell development.J Exp Med 200(11):1503-1509   
SiegalFP,KadowakiN,ShodellM etal (1999)Thenature of the principal type 1 interferon-producing cells in human blood. Science 284(5421):1835-1837   
Singh R,Cresswell P (2010) Defective cross-presentation of viral antigens in GILT-free mice.Science 328(5984):1394-1398   
Smyth GK (20o4) Linear models and empirical bayes methods for assessing differential expression in microarray experiments.Stat Appl GenetMol Biol 3:1-25   
Steinman RM (2007）Lasker Basic Medical Research Award. Dendritic cells:versatile controllers of the immune system.Nat Med 13(10):1155-1159

Trapnell C，Pachter L，Salzberg SL (2009）TopHat:discovering splice junctions with RNA-Seq. Bioinformatics 25(9):1105-1111 Uchiyama N, Ito K,Kawai K,Sakamoto F,Takaki M, Ito M(1998) CD2', $\mathsf { C D 4 ^ { + } }$ ${ \mathsf { C D 5 6 } } ^ { + }$ agranular natural killer cell lymphoma of the skin.Am J Dermatopathol 20(5):513-517 Vermi W,Facchetti F,Rosati S etal (2oo4) Nodal and extranodal tumor-formingaccumulationof plasmacytoidmonocytes/interferon-producing cells associated with myeloid disorders.Am J Surg Pathol 28(5):585-595 Ziegler-HeitbrockL,AncutaP,Crowe S etal (2010) Nomenclature of monocytes and dendritic cells in blood.Blood 116(16):e74-e80