# PERSPECTIVE

# CRISPR/Cas9 and TALE: beyond cut and paste

Liping Deng1, Ruotong Ren1, Jun $\boldsymbol { \mathsf { W } } \boldsymbol { \mathsf { u } } ^ { 2 }$ , Keiichiro Suzuki², Juan Carlos lzpisua Belmote², Guang-Hui Liu1.3,4区

1National Laboratoryof Biomacromolecules,Instituteof Biophysics，Chinese Academyof Sciences,Beijing 100101China   
2GeneExptsitidll   
3 Beijing Institute for Brain Disorders,，Beijing 100069,China   
4 Center for Molecular and Translational Medicine,CMTM, Beijing 100101, China   
区Correspondence: belmonte@salk.edu (J. C. Izpisua Belmote), ghliu@ibp.ac.cn (G.-H. Liu)

# ABSTRACT

Nuclease-based genome editing has proven to be a powerful and promising tool for disease modeling and gene therapy. Recent advances in CRlSPR/Cas and TALE indicate that they couldalso beused as a targeted regulator of gene expression, as well as being utilized for illuminating specific chromosomal structures or genomic regions.

KEYWORDS TALE,CRISPR/Cas9, genome editing, geneexpression, transcription regulation

Nuclease-based genome editing tools,including zinc finger nuclease (ZFN), transcriptional activator-like effector nucleases (TALENs),and the recentlydeveloped CRISPR (clustered regularly interspaced short palindromic repeat)-Cas system,have been progressingatan unprecedented pace for gene therapy and disease modeling(Gaj etal.2013; Pan et al. 2011;Liuetal.2014;Li etal.2014).Recentadvances have carved out new paths leading to novel applications of these genome engineering tools includingvisualization of specific lociof the genome and targeted regulation of gene expression.

In addition to their genome-editing capabilities,TALE and Cas9 have also been harnessed for targeted regulation of gene expression.Several studies exploited TALE and Cas9's DNA binding abilities and converted them into synthetic transcriptional factors or epigenetic modifiers to modulate gene expression. Synthetic transcription factors created by fusingTALE or catalytically dead Cas9 (dCas9) to effector domains were successfully used to gain transcriptional control of gene expression.Binding of dCas9 to DNA alone could repress transcription (CRlSPRi)，possibly through stalling transcription elongation. Fusing dCas9 to protein domains that can recruit repressive chromatin-modifying complexes,e. g. the KRAB domain of Kox12, can further enhance CRlSPRi. To activate genes,in one study Therizols et al.(2014) fused TALE to VP64,a tetramerof the VP16 acidic transcriptional activator and used the fusion protein to ectopically activate genes normally silenced, leading to novel insights of nuclear reorganization in embryonic stem cells (ESCs). Similarly, Gilbert et al.fused dCas9 to the activation domains of either VP64 or p65 to activate targeted genes (Gilbert et al. 2013). Synthetic epigenetic modifiers were created in Maeder et al.'s study(2013)by fusinga hydroxylase catalytic domain of TET1 to TALE,leading to targeted demethylation of specific promoter CpGs. Removal of the methylation from key promoter CpGs can result in enhanced transcription of endogenous genes.Tethering Tet1 hydroxylase domain to a target promoter thus constitutes a proof-of-concept in epigenetic activation of specific gene transcription.To epigenetically repress transcription of target genes,Mendenhall et al. (2013) fused TALE with the LSD1 histone demethylase which enabled targeted demethylation of enhancer-associated histone modifications thereby repressing the proximal genes.Furthermore,a recent study achieved spatiotemporal transcriptional regulation by combiningTALE and a popular optogenetic approach and created a LITE (light-inducible transcriptional effectors) system (Konermann etal.2013).The LITE system includes two parts:a customizable TALE DNAbinding domain，fused with light-sensitive cryptochrome 2 (CRY2),and transcriptional regulator-fused CiB1 (an interacting partner of CRY2).The LITE system allows for precise spatiotemporal control of genetic and epigenetic factors contributing to a variety of biological processes in vivo. Very recently，two studies further expanded the application of CRISPR/Cas9 to genome-wide interrogation of gene function through systematic transcriptional perturbation. One study by Zalatan et al. (2014) incorporated modular RNA domains into the sgRNA design and repurposed sgRNA as scaffolding molecules that encode both target and function in a single scaffold RNA (scRNA). scRNA enables simultaneous multidirectional regulation of multiple target genes.In the second study,Konermann etal. (2014) showed that following structure-guided design,a novel CRlSPR-Cas9 complex could drive transcriptional activation at endogenous gene loci efficiently.Moreover, genome-wide transcriptional activation was realized witha librarycomposed of 70,290 engineering guides targeting all coding isoforms of human RefSeq，with which they screened genes exhibiting resistance toa BRAF inhibitor upon activation.

Interestingly,TALE and Cas9 have also been brought to service to visualize specific genomic loci in live cells,which provide a novel way to uncover the functional relevance between chromatin spatial organization and genome function.Traditionally fluorescent in situ hybridization (FISH)is the method of choice to label DNA.However, FlSH requires sample fixation and is incompatible for monitoring live cellular processes.Three recent studies turned TALE or Cas9 into powerful live cell imaging tools. In Miyanari et al.'s (2013)，fluorescent TALE was designed to visualize major satellites in cultured mouse cells, including centromeric and telomeric elements in the genome.Moreover， the TALEbased approach exhibited high specificity allowing for distinguishing single-nucleotide polymorphisms (SNPs).Ma et al.(2013) successfully labeled telomeres in human cells by fusing fluorescent protein(FP） to a TALE targeting a telomeric sequence.They also designed unique centromeric sequences specifically associated with certain chromosomes to visualize individual chromosomes in human cells Meanwhile,since signals generated by FP-TALE positively correlated with telomere length,they could measure telomere length in human cells.Results from the above methods are consistent with those obtained by DNA-FISH, suggesting their potential in labeling specific genomic sequences with high accuracy. Therefore，TALE-based strategies hold immense promise to gain insight into the chromatin dynamics associated with different cellular physiologies by visualizing genomic DNA repetitive sequences. Importantly，an optimized CRISPR-Cas system with structure-guided sgRNA was recently employed for efficiently labeling arbitrary genomic sequences in live mammalian cells (Chen et al. 2013). It provedto bearobustmethod for imagingof both repetitive elements and coding genes.Although genome-scale imaging has not been implemented with a library of sgRNA,visualization of specific genomic loci paves the way for further study of dynamic organization of the human genome.

The advances in targeted genome engineering technolo gies via CRlSPR/Cas9 and TALE may lead to a revolution in cell biology research．Several considerations nevertheless should be taken into account for future applications.(1) Whether potential of-target effects can contribute to certain biasormisinterpretationinobtained results.While thecurrent whole-genome sequencing has indicated that gene editing mediated by well-designed sgRNA and TALEN leads to minimal mutational load at global level (Hsu etal.2013; Smith et al. 2014; Suzuki et al. 2014; Veres et al. 2014), precautions need to be taken to minimize of-targets caused by poor sgRNA and TALE design.(2) Choice of TALE vs Cas9 in specific application? From the aspect of construct size,TALE isrelatively smaller than Cas9 and is easier to be delivered into cells;engineered TALE may be advantageous in sitespecific transcriptional manipulation by serving as a direct transcriptional regulatoror,alternatively,establishingabridge between protein and DNA or between two genomic DNA loci without involving exogenous gRNA molecules.By contrast, while Cas9 with larger size is difficult to get into cells,it holds the potential for multiplex targeting. (3) It still remains unclear whether binding of TALEor Cas9-gRNA complex to the target sites would perturb the function of the native DNA-binding proteins and cause unwanted perturbation of cellular function. (4)Whether intra-nuclear localization, three-dimensional organization,orepigenetic modificationsof targetDNA element potentiallyaffect the binding of engineered TALE and Cas9- gRNA warrants further investigations.Also how these factors contribute to unbiased interpretation of the observed results needs to be considered.

Collectively，despite some current concerns for using engineered Cas9 and TALE forexploringuncharted cellular events,esp.in a physiologically relevant context, these tools undoubtedly open a new avenue to uncover novel events underlying gene expression regulation and dynamic nuclear organization of chromatin,and may greatly facilitate translation of basic studies into clinical therapies.

# NOTES

This work was supported by the National Basic Research Program (973 Program) (Nos.2015CB964800 and 2014CB964600),the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA01020312), National Natural Science Foundation of China (Grant Nos．81271266，31222039，31201111，81300261 and 81300677), Beijing Natural Science Foundation (7141oo5), National Laboratoryof Biomacromolecules (2013kf05，2013kf11and 2014kf02)，and State Key Laboratory of Drug Research (SIMM1302KF-17).JCIB was supported by G.Harold and Leila Y.Mathers Charitable Foundation,The Leona M.and Harry B.Helmsley Charitable Trust(2012-PG-MED002), The Glenn Foundation and The Ellison Medical Foundation.

Liping Deng，Ruotong Ren，Jun Wu，Keiichiro Suzuki，Juan Carlos lzpisua Belmonte,and Guang-Hui Liu declare no conflict of interest.

# OPENACCESS

This article is distributed under the terms of the Creative Commons Attribution License which permits any use， distribution，and reproduction in any medium，provided the original author(s)and the source are credited.

# REFERENCES

Chen B et al (2013) Dynamic imaging of genomic loci in living human cells by an optimized CRlSPR/Cas system.Cell 155:1479-1491   
GajT，Gersbach CA，Barbas CF 3rd(2013)ZFN，TALEN，and CRISPR/Cas-based methods for genome engineering. Trends Biotechnol 31:397-405   
Gilbert LA et al (2013)CRISPR-mediated modular RNA-guided regulation of transcriptionin eukaryotes.Cell 154:442-451   
Hsu PD etal (2013) DNA targeting specificity of RNA-guided Cas9 nucleases.Nat Biotechnol 31:827-832   
Konermann S et al (2013) Optical control of mammalian endogenous transcription and epigenetic states.Nature 5o0:472-476   
Konermann S et al (2014) Genome-scale transcriptional activation by an engineered CRiSPR-Cas9 complex.Nature 512(7515):441-444   
Li M,Suzuki K,Kim NY,Liu GH, Izpisua Belmonte JC (2014) A cut above the rest: targeted genome editing technologies in human pluripotent stem cells.JBiol Chem 289:4594-4599   
Liu GH et al (2014) Modelling Fanconi anemia pathogenesisand therapeutics using integration-free patient-derived iPSCs.Nat Commun 5:4330   
Maeder ML etal (2013) Targeted DNA demethylation and activation of endogenous genes using programmable TALE-TET1 fusion proteins.Nat Biotechnol 31:1137-1142   
Ma H，Reyes-Gutierrez P，Pederson T (2013)Visualization of repetitive DNA sequences in human chromosomeswith transcription activator-like effectors.Proc Natl Acad Sci USA 110:21048-21053   
Mendenhall EM et al (2013）Locus-specific editing of histone modifications at endogenous enhancers.Nat Biotechnol 31: 1133-1136   
Miyanari Y, Ziegler-Birling C,Torres-Padilla ME(2013) Live visualization of chromatin dynamics with fluorescent TALEs.Nat Struct Mol Biol20:1321-1324   
Pan H,Zhang W,Liu GH (2011) Find and replace:editing human genome in pluripotent stem cells.Protein Cell 2:950-956   
Smith C et al (2014) Whole-genome sequencing analysis reveals high specificity of CRISPR/Cas9 and TALEN-based genome editing in human iPSCs.Cell Stem Cell 15:12-13   
Suzuki K et al (2014) Targeted gene correction minimally impacts whole-genome mutational load in human-disease-specific induced pluripotent stem cell clones.Cell Stem Cell 15:31-36   
Therizols P et al (2014) Chromatin decondensation is suficient to alternuclear organization in embryonic stem cells.Science 346:1238-1242   
Veres A et al (2014)Low incidence of off-target mutations in individual CRiSPR-Cas9 and TALEN targeted human stem cell clones detected by whole-genome sequencing. Cell Stem Cell 15:27-30   
Zalatan JG et al (2014) Engineering complex synthetic transcriptional programs with CRlSPR RNA scaffolds.Cell 160:339-350