# Identification of a novel EXT1 mutation in patients with hereditary multiple exostosis by exome sequencing

HONGJIE LIU3.4.9\*， SONG WU1,2\*\*， LI DUAN1\*，WEIMING ZHU1\*， SHIQUAN ZHANG1\*， XIAOXIAO HU3\*,WENLONG JIA,GUOSHENG YANG1，CHUNXIAO $\mathrm { L I U } ^ { 1 }$ ，WEIPING $\boldsymbol { \mathrm { L I } } ^ { 1 }$ ， LEI YANG1， LIJUN GUO1,YOUCHENG LIN8， YONGQIANG WANG1.7， MEIJIAN $\mathrm { H E } ^ { 1 }$ , ZHAO YANG6.9,YINGYING $\mathrm { H E } ^ { 1 }$ ， ZHIMING CA11\*\* \*\* and DAPING WANG1\*\*

lShenzhen Second People's Hospital,The First Afiliated Hospitalof Shenzhen University,Shenzhen 518000;   
2Zhongshan School of Medicine,Sun Yat-Sen University,Guangzhou 510oo; College of Life Science, Universityof Chinese Academy of Sciences College, Beijing 100049; $^ 4 \mathrm { { B G I } }$ Education Centre,University of Chinese Academy of Sciences, Beishan Industrial Zone,Shenzhen 518083;BGI Tech Solutions Co.,Ltd.,Beishan Industrial Zone,Shenzhen 518083;   
6Key Laboratoryof Infection and Immunity of CAS,Institute of Biophysics,Chinese Academy of Sciences,Beijing 100101;   
7Anhui Medical University,Hefei 230ooo; &Departmentof Urology,Zhujiang Hospital of Southern Medical University, Guangzhou 51000o; 9University of Chinese Academy of Sciences,Beijing 100049,P.R. China

Received September14,2014;Accepted October 20,2014

DOI:10.3892/0r.2014.3610

Abstract.Hereditary multiple exostosis(HME) is an autosomal inherited skeletal disease whose etiology is not fully understood.To further understand the genetic spectrum of the disease,we analyzed a five-generation Chinese family with HME that have observable inheritance.Exome sequencing wasperformed on three HME individuals and three unaffected individuals from the family.A downstream study confirmed a new C deletion at codon 442 on exon 5 of the exostosin-1 (EXT1) gene as the only pathogenic site which generated a stop codon and caused the truncation of the protein.We rediscovered the deletionin otheraffected individuals but not in the unaffected individuals from the family.Upon immunohistochemistry assay,we found that the EXT1 protein level of the patients with the novel mutation in our study was less than the level in the patients without the EXT1 mutation from another unrelated family.For a deeper understanding,we analyzed the mutation spectrum of the EXT1 gene.The present study should facilitate a further understanding of HME.

# Introduction

Hereditary multiple exostosis(HME)is an autosomal dominant disorder with an incidence of1in 5O.OoO(1).Thus,itis one of the most common inherited musculoskeletal disorders. Approximately $20 \%$ of reported cases have no family history of multiple exostosis (2).There are 3 features of HME.First, patients show multiple benign exostoses, typically located at the juxta-epiphyseal region of bones. Second, it has an obvious hereditary property.Third,and the most severe complication, is the characteristic of malignant transformation (3).

Previous studies have reported that the EXT family is responsible for HME.The EXT family was reported to encode proteins linked to the biosynthesis of heparan sulfate (HS) (4). HS is an essential molecule and dysfunction could cause HME. The EXT family consists of exostosin-1(EXT1) (5),exostosin-2 (EXT2) (6),exostosin-3 (EXT3) (7),exostosin-like1 (EXTL1) (8),exostosin-like 2 (EXTL2) (9) and exostosin-like 3 (EXTL3) (10).Among these,EXT1 has been suggested to be responsible to a large extent for HME (9,11).

With the rapid development of DNA sequencing technology, the power of exome sequencing for gene mapping of diseases has been demonstrated (12-14). Exome sequencing can help to identify novel causal genetic variants,as it covers all exon regions.

In the present study, we performed exome sequencing on 3 affected and 3 unaffected individuals from a HME family. After filtering out the mutations from the dbSNP database (build 132), the candidate mutations were validated on all of the patients from the HME family by Sanger sequencing. Further validation was also performed on three members from another unrelated HME family.Additionally,immunohistochemisty was conducted to provide a deeper insight into the importance of the identified causal gene mutation.Furthermore, the mutation spectrum of the EXT1 gene was analyzed.

Table I. Clinical characteristics of the subjects for exome sequencing   

<html><body><table><tr><td>Sample no.</td><td>Position</td><td>Clinical diagnosis</td><td>Blood</td><td>Age (years)</td><td>Gender</td><td>Analyzed by WES</td></tr><tr><td>13JX00006WB3</td><td>I10</td><td>/</td><td>Yes</td><td>65</td><td>Male</td><td>Yes</td></tr><tr><td>13JX00007WB2</td><td>19</td><td>HME</td><td>Yes</td><td>62</td><td>Female</td><td>Yes</td></tr><tr><td>13JX000011LC2</td><td>113</td><td>HME</td><td>Yes</td><td>27</td><td>Male</td><td>Yes</td></tr><tr><td>13JX00002LC2</td><td>112</td><td>/</td><td>Yes</td><td>29</td><td>Male</td><td>Yes</td></tr><tr><td>13JX000017WB2</td><td>II19</td><td>HME</td><td>Yes</td><td>19</td><td>Male</td><td>Yes</td></tr><tr><td>13JX000019WB2</td><td>1126</td><td>/</td><td>Yes</td><td>39</td><td>Female</td><td>Yes</td></tr></table></body></html>

‘HME',affected members; $\cdot _ { / } ,$ ,unaffected members;HME,hereditary multiple exostosis.

# Materialsandmethods

Subjects.For exome sequencing,we selected 3 affected cases and 3 unaffected individuals(Table I) from a five-generation HME family(Fig.1A) from the Henan Province of China. Sixteen affected cases,13 unaffected individuals in the family and 3 individuals(2 affected cases and 1 unaffected) from another unrelated family(Fig.1B) were selected for further validation.All of the participants were informed and consented with the specimen collection.

After filling in the informed consent form,all of the individuals were carefully examined by at least2 experienced doctors.All of the cases were diagnosed as HME,while all of the unaffected individuals were healthy.

Venous blood samples were collected from all members who participated.Genomic DNA was extracted from peripheral blood by standard procedures for exome sequencing. In this study,all steps were conducted according to the Declaration of Helsinki Principles.The use of human subjects in this report was approved by the Shenzhen Second People's Hospital Institutional Review Committee.

Exome capture and sequencing.Exome capture was performed with the Human All Exon V3 (Agilent Technologies,Santa Clara,CA,USA) and the sequencing was conducted with the HiSeq 20oO platform (Illumina Inc., San Diego, CA,USA). Genomic DNA samples were randomly fragmented for library conducted and base-pair peaks of 15O-2OO bp were selected and then adapters were ligated to both ends of the fragments.The adapter-ligated templates were purified by Agencourt AMPure SPRI beads,and fragments with insert size ${ \sim } 2 0 0$ bp were excised.Extracted DNA was amplified byligation-mediated polymerase chain reaction (LM-PCR), purified and hybridized to the SureSelect Biotinylated RNA Library(BAITS) for enrichment.Hybridized fragments were bound to the strepavidin beads whereas non-hybridized fragments were washed out after $2 4 \mathrm { ~ h ~ }$ . The capturedLM-PCR product was subjected to the Agilent 21oo Bioanalyzer to estimate the magnitude of enrichment.All of the steps were performed according to the manufacturer's recommendations. Each captured library was then loaded onto the Hiseq 2000 platform and sequencing was performed with read lengths of $9 0 ~ \mathrm { b p }$ ，which provided atleast a $5 0 \mathrm { x }$ average depth for each sample.

Read mapping and variant detection.Reads with adapters, the N's percent $> 1 0 \%$ and reads whose low quality bases (quality value $\leq 5$ )were ${ > } 5 0 \%$ were filtered.The clean reads of each individual were aligned to the human reference genome (NCBI Build 36.3,hg19) using Burrows-Wheeler transform (BWA; Cambridge,UK); the parameter were set as‘-o 1 -e 50 （20 $\mathrm { - m ~ 1 0 0 , 0 0 0 \mathrm { ~ - t ~ } 4 \mathrm { ~ - i ~ } 1 5 \mathrm { ~ - q ~ } 1 0 ^ { \circ } }$ Reads that had duplicated aligned sites were removed; the remaining reads mapped on or near the target were collected for subsequent analyses and variant calling.

For the SNP calling,we used Short Oligonucleotide Analysis Package (SOAPsnp,China)(l5).A consensus genotype with Phred-like quality of at least 2O and at least $4 \mathrm { x }$ coverage depth was considered to be a high-confidence genotype.The genotypes that were different from the reference were extracted as candidate SNP and the SNP list was filtered with the followed criterion: Phred-like SNP quality $\scriptstyle \geq 2 0$ ,overall depth of 4 to $1 { , } 0 0 0 \mathrm { x }$

Genome Analysis Toolkit (GATK;Broad Institute, Cambridge,MA,USA)(l6) was used to call InDel and the results were filtered as follows: After gap aligning the sequence reads with default parameters to the human reference (hg19), local realignment of the BWA-aligned reads using the GATK IndelRealigner were performed and the final InDels were called by GATK IndelGentotyperV2 according to the recommendations of the software.

Functional annotation of genetic variants.With ANNOVAR (USA) (l7) the variants were annotated and categorized into missense,nonsense,synonymous,splice-site and insertion/ deletion mutations.After filtering out synonymous and nonframeshift,weobtainedmutationswhichwerelikelyto be deleterious and then filtered against Chinese Han SNP data available in the dbSNP database (build 132).We chose those variants that were shared byall affected but not anyunaffected individualascandidate casualvariants.

Sanger sequencing.Sanger sequencing was performed to confirm the identified variants found by exome sequencing including 18 affected and 14 unaffected individuals. Sanger sequencing was performed according the standard protocol. The primer sequences were as follows: 5'-GATGGACCCCAT TAGAGTAG- $3 ^ { \prime }$ and 5'-AGAGTAGTGACTCTACCCTC-3'. Sequence comparisons and analyses were performed using

![](images/bfe65b4364fdf6db0eecfa03560340f84e011659cb762b4af382a57f79f56fa9.jpg)  
FigurelPdigedlilolsi hereditary multiple exostosis (HME) in the study. $\cdot _ { + } ,$ indicates those individuals examined and sequenced.The patient indicated by the arrow represents the probdac linesrepresteadoltetdttedaub.ota faily(d.lod The upward arrows indicate the perichondrium and the downward arrow indicates the cartilage cap.

Chromas2 (Technelysium Pty Ltd.，Tewantin,QLD, Australia).

Histochemical staining and immunohistochemistry.The paraformaldehyde-fixed chondroma tissues from the HME patient were fixed with $1 \%$ paraformaldehyde (PFA;Sigma-Aldrich), rinsed and decalcified and embedded in paraffin. Paraffin-embedded chondroma tissues were sectioned $( 5 - \mu \mathrm { m }$ thick) and placed on glass slides.For histochemistry,the tissue slidesofHME were dewaxed in xylene,hydrated with graded ethanol and stained by toluidine blue(TB) and hematoxylin and eosin (H&E),respectively.

For immunohistochemistry, the dewaxed and hydrated chondroma sections of HME were treated with $3 \%$ hydrogen peroxide solution for $1 0 \mathrm { m i n }$ ,rinsed with PBS followed by $5 \%$ donkey serum blocking and incubated with rabbit polyclonal anti-EXT1 antibody (1:5O working dilution; Abcam Plc, UK) at $4 ^ { \circ } \mathrm { C }$ overnight.The chondroma sections were then incubated with the secondary antibody (Golden Bridge Biotechnology, Zhongshan,China) at $3 7 ^ { \circ } \mathrm { C }$ for $3 0 ~ \mathrm { m i n }$ and stained using the DAB substrate kit（Vector Laboratories,Burlingame, CA, USA) and Meyer's hematoxylin.

EXTl mutation spectrum.We downloaded the mutation data according to the variant type of substitution,deletion, insertion,inversion,insertion or deletion and translocation from the Multiple Osteochondroma Mutation Database. Subsequently,we summarized the mutations which were located in the exon regions and drew the mutation spectrum with Circos plot.

![](images/af100d12118f5ca3405e5c4e48fc149935282a3d4e4724a910b0fcc273138de5.jpg)  
Figure2d)as((d (9dt genecaleplatebelstideidatscatioifrotesecdutated (CandDualdroiplofobd exostosis (HME)patient withoutEXT mutationfromanother family.The arrowrepresentsachondrocyte with functionalEXT1.

# Results

We observed that the tissue structure (Fig.1C) of HME was represented with a cartilage cap,covered by fibrous perichondrium and merged into an underlying spongy bone. The stained chondroma sections with H&E and TB are presented in Fig. 1D.

We used Human All Exon V3 Agilent $( 5 0 ~ \mathrm { M } )$ to capture theexome of 3 affected(I9,I3 andI19) and3 unaffected (I10,II2 and I26) individuals in the HME family (Fig.1A). After performing parallel sequencing, ${ \sim } 1 2 0$ million bases and 80 million reads per individual in average were generated.The mapping rates against the human being reference (hgl9) of all individuals achieved $9 9 \%$ . Approximately $92 \%$ of the target regions were sequenced at least 1O times (depth $\mathord { \geq } 1 0 \mathrm { x }$ ）ineach individual.For every individual, ${ \sim } 8 0 { , } 0 0 0$ variants were identified within the captured regions.

![](images/d04c616f50b1e01548c98fa755dec1d4a64e2a4e2f39244a56093012de4f3ecc.jpg)  
Figure3utatoattas repnt indicates the deletion in the present study.

We focused on those variants that were located in exon, UTR and splicing sites.Taking the familial dominant model of inheritance into account, the variants that appeared in all 3 affected individuals but not in any unaffected individual were selected.We then fltered against the available public databases (dbSNP).Furthermore,we only focused on nonsynonymous,stopgain, stoplose and frameshift variants.After all the previous filtrations,only 3 heterozygous variants,ATXN3, TRPM3 and C1lorf4O,remained and only one heterozygous deletion (c.1325delC) in EXT1 was probably damaging, as the EXT family has been reported to be associated with HME.No reports were found at this position in the Multiple Osteochondroma Mutation Database.Thus,we selected it for further study.

To eliminate false-positive mutations,we examined and validated the mutations by Sanger sequencing in the source individuals. The heterozygous deletion g.chr8: 118834796 delC (Fig.2A) was confirmed to be shared by all of the 3 studied patients but was absent in the normal individuals.The mutation led to a frameshift at codon 442 resulting in a change of serine to isoleucine and generated a stop codon at codon 443 causing the glycosyltransferase domain loss (Fig.2B).

To further verify that the deletion was the disease-causing gene mutation in the HME family,we performed PCR and Sanger sequencing on all the family members and another

3 individuals (I1,I2 and II7） from another unrelated family (Fig.1B).The mutation was confirmed to be shared by all the affected individuals except patient I22 and was absent in all of the healthy family members.

Immunohistochemistry was used to detect the protein expression level of EXT1 in these 2unrelated families.We found that the EXT1 protein level of a HME patient (Fig.2C) with the novel mutation was less than the level in an HME patient without the EXT1 mutation from another unrelated family (Fig. 2D).

To further analyze the mutations in EXT1,we gathered all of the mutations in its 11 exons from the Multiple Osteochondroma Mutation Database.No inversion and translocation were found on all of the 11 exons.Substitutions and deletions were evenly distributed on the 11 exons,while small insertions were rarely locate in exon 2,9 and11 (Fig.3).

# Discussion

The gene EXT1 locates on 8q24.1 and consists of 1l exons. EXT1 encodes a protein which possesses the activity of glucuronic acid sugar-based transfer enzyme and $\mathbf { N }$ acetyl glucosamine glycosyltransferase (4) which catalyzes the polymerization of HS chains at the endoplasmic reticulum and Golgi apparatus (18). HS is widely expressed on the cell surface,and as the component of extracellular matrix glycoprotein it mediates cell adhesion,signal transduction and the receptor ligand binding process.Previous studies have indicated that HS is an essential molecule and dysfunction could cause HME (19,20).

Herein,we found a novel c.1325delC mutation located in exons 5.This frameshift mutation was validated to be shared by all of the affected individuals except patient II22 and was absent in the healthy family members.Additionally, the mutation was not discovered in individuals from another HME family.As this damaging deletion mutation was not present in patient I22,further study will be carried out.

The c.1325delC mutation locates between the exostosin and glycosyltransferase domains.It led to a frameshift at codon 442 and resulted in a change of a serine to an isoleucine. The promptly premature stop codon at codon 443 caused loss of the glycosyltransferase domain,and the decrease in EXT1 protein expression was confirmed by immunohistochemisty. Because of the truncation, the EXT1 products may be unable to fold correctly and would rapidly degrade (21).This result is consistent with the reduced EXT1 protein observed in HME patients in previous studies (22,23).According to the statistics of all mutations on EXT1 in the Multiple Osteochondroma Mutation Database (24),all variant types were nearly equally distributed on the 1l exons.

Taken together,we report a novel frameshift mutation c.1325delC in the gene EXT1 by whole-exome sequencing. The mutation was validated by Sanger sequencing and its negative influence on the expression of EXT1 was examined via immunohistochemistry. Our results provide a new mutation in EXT1,further emphasizing the dysfunction of the EXT gene family as a cause of HME.This finding may be helpful for the early diagnosis and prenatal genetic screening of HME.

# Acknowledgements

We would like to thank all members for their participation in the study. This study was mainly supported by Grant support from: The Shenzhen Science and Technology Project (project nos.CXZZ20130321152713220, GJHZ20130412153906739，CXZZ20120614160234842, CXZZ20130321152218032，JCYJ20130401114928183, JCYJ20130401114715714 and JCYJ20130401112006994) and SOAPsnp (Short Oligonucleotide Analysis Package); The Natural Science Foundation of China (nos.81260161, 8100046O and 813O174O),The Natural Science Foundation of Guangdong Province, China (no.S2012010008129), The China Postdoctoral Science Foundation Funded Project (no. 2013M530385）and The Promotion Program for Shenzhen KeyLaboratory, Shenzhen,China (nos.CXB201104220049A and ZDSY20120614154551201).

# References

1.Schmale GA,Conrad EUIII and Raskind WH:The natural history of hereditary multiple exostoses.J Bone Joint Surg Am 76:986-992,1994.

2. Chen WC, Chi CH, Chuang CC and Jou IM: Three novel EXT1 and EXT2 gene mutations in Taiwanese patients with multiple exostoses.J Formos Med Assoc 105: 434-437,2006. 3.Wicklund CL,Pauli RM, Johnston D and Hecht JT: Natural history_study of hereditary multiple exostoses.Am J Med Genet 55: 43-46,1995. 4. Busse M,FetaA,Presto J,et al: Contribution of EXT1,EXT2 and EXTL3 to heparan sulfate chain elongation. JBiol Chem 282: 32802-32810,2007.   
5. Ahn J,Ludecke HJ,Lindow S,et al: Cloning of the putative tumour suppressor gene for hereditary multiple exostoses (EXT1). Nat Genet 11: 137-143,1995.   
6. Stickens D, Clines G, Burbee D, et al: The EXT2 multiple exostoses gene defines a family of putative tumour suppressor genes.Nat Genet 14: 25-32,1996.   
7. Le Merrer M,Legeai-Mallet L,Jeannin PM, et al: A gene for hereditary multiple exostoses maps to chromosome 19p. Hum Mol Genet 3: 717-722,1994.   
8.Wise CA, Clines GA, Massa H, Trask BJ and Lovett M: Identification and localization of the gene for EXTL,a third member of the multiple exostoses gene family. Genome Res 7: 10-16, 1997.   
9. Wuyts W and Van Hul W: Molecular basis of multiple exostoses: mutations in the EXT1 and EXT2 genes.Hum Mutat 15: 220-227, 2000.   
10. Van Hul W,Wuyts W,Hendrickx J,et al: Identification of a third EXT-like gene (EXTL3) belonging to the EXT gene family. Genomics 47: 230-237,1998.   
11. Signori E,Massi E,Matera MG,et al: A combined analytical approach reveals novel EXT1/2 gene mutations in a large cohort of Italian multiple_osteochondromas patients.Genes Chromosomes Cancer 46: 470-477,2007.   
12. Wu S,Sun X,Zhu W, et al: Evidence for GAL3ST4 mutation as the potential cause of pectus excavatum. Cell Res 22: 1712-1715, 2012.   
13. Zhang SQ,Jiang T,LiM,et al: Exome sequencing identifies MVK mutations in disseminated superficial actinic porokeratosis.Nat Genet 44: 1156-1160,2012.   
14. Daoud H, Dobrzeniecka S, Camu W,et al: Mutation analysis of PFN1 in familial amyotrophic lateral sclerosis patients. Neurobiol Aging 34: e1311-e1312,2013.   
15.Li R, Yu C,Li Y,et al: SOAP2: an improved ultrafast tool for short read alignment. Bioinformatics 25:1966-1967,2009.   
16.McKenna A,Hanna M,Banks E,et al: The genome analysis toolkit:a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res 20:1297-1303,2010.   
17. Wang K,Li M and Hakonarson H: ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res 38:el64,2010.   
18.Lind T, Tufaro F,McCormick C,Lindahl_U and Lidholt K: The putative tumor suppressors EXT1 and EXT2 are glycosyltransferases required for the biosynthesis of heparan sulfate.JBiol Chem 273: 26265-26268,1998.   
19. Duncan G, McCormick C and Tufaro F: The link between heparan sulfate and hereditary bone disease: finding a function for the EXT family of putative tumor suppressor proteins. J Clin Invest 108: 511-516,2001.   
20.Busse-Wicher M,Wicher KB and Kusche-Gullberg M: The extostosin family: Proteins with many functions.Matrix Biol 35: 25-33,2014.   
21. Wuyts W,Van Hul W,De Boulle K,et al: Mutations in the EXT1 andEXT2 genes in hereditary multiple exostoses.AmJHum Genet 62: 346-354,1998.   
22.Bernard MA,Hall CE,Hogue DA,et al: Diminished levels of the putative tumor suppressor proteins EXT1 and EXT2 in exostosis chondrocytes. Cell Motil Cytoskeleton 48:149-162,2001.   
23. Zhang F,Liang J,Guo X,et al: Exome sequencing and functional analysis identifies a novel mutation in EXT1 gene that causes multiple osteochondromas.PLoS One 8: e72316,2013.   
24. Jennes I, Pedrini E,Zuntini M,et al: Multiple osteochondromas: mutation update and description of the multiple osteochondromas mutation database (MOdb).Hum Mutat 30: 1620-1627.2009.