# Structure of Ljungan virus provides insight into genome packaging of this picornavirus

Ling $Z \mathsf { h u } ^ { 1 , 2 , \star }$ ， Xiangxi Wang3,\*, Jingshan Ren', Claudine Portal,2，Hannah Wenham², Jens-Ola Ekström4, Anusha Panjwani2, Nick J. Knowles2， Abhay Kotechal, C. Alistair Siebert1, A. Michael Lindberg4, Elizabeth E. Fry1, Zihe Rao3.5,Tobias J. Tuthill² & David I. Stuart1,6

Picornaviruses are responsible for a range of human and animal diseases,but how their RNA genome is packaged remains poorly understood.A particularly poorly studied group within this family are those that lack the internal coat protein,VP4. Here we report the atomic structure of one such virus,Ljungan virus,the type member of the genus Parechovirus B,which has been linked to diabetes and myocarditis in humans.The 3.78-A resolution cryo-electron microscopy structure shows remarkable features, including an extended VP1 C terminus, forming a major protuberance on the outer surface of the virus,and a basic motif at the N terminus of VP3,binding to which orders some $12 \%$ of the viral genome.This apparently charge-driven RNA attachment suggests that this branch of the picornaviruses uses a diferent mechanism of genome encapsidation,perhaps explored early in the evolution of picornaviruses.

E packaging viral genomes involve either filling a preformed procapsid with progeny genome or nucleation of capsid assembly around viral nucleic acids1,2. We have a convincing picture of how some bacteriophages package the viral genome into a preassembled procapsid using packaging motor proteins³,and how plant viruses govern RNA packaging through proteinprotein or protein-RNA interactions4；however，picornavirus assembly remains much less clear.Picornaviruses,small $( 3 0 0 \mathring { \mathrm { A } }$ diameter） single-stranded,positive-sense,RNA viruses,with a genome of between 7.1 and $\bar { 9 } . 7 \mathrm { k b }$ ,are responsible fora variety of human and animal diseases and include poliovirus (PV),human rhinovirus,enterovirus A71,hepatitis A virus(HAV） and foot-and-mouth disease virus5.The selective encapsidation of the viral genome,and not cellular RNAs,requires the specific recognition of packaging signals-sequences or structures unique to viral nucleic acid. Numerous attempts to identify an RNA encapsidation signal in enterovirus genomeshave failed6, although a recent study reported that PV (Enterovirus genus) encapsidation is facilitated by the interactions of capsid proteins and the RNA replication complex7 and Aichi virus (Kobuvirus genus）was reported to contain a $5 ^ { \prime }$ -terminal RNA stem-loop critical for viral RNA encapsidation8.However，noneof these studies have provided structural evidence to explain encapsidation.

Ljungan virus (LV),a picornavirus originally isolated from voles，has been proposed as a zoonotic virus， potentially associated with type-1 diabetes mellitus, myocarditisand Guillain-Barré syndrome in humans9.Currently， four LV genotypes have been characterized (LV-1 to LV-4),two isolated in Sweden and another two in the United States10.LV belongs to the Parechovirus genus and is closely related to human parechoviruses $( { \mathrm { H P e V } } )$ $( \sim 4 8 \%$ amino-acid identity between the VP0 proteins)11,i2. Parechoviruses exhibit several distinctive features,including the lack of the final,assumed RNA-catalysed, maturation cleavage of the capsid protein precursor VPO to VP2 and VP4 (refs 12-15)，the existence of an $\sim 2 0$ -amino-acid extension enriched with basic residues to the $\mathrm { ~ N ~ }$ terminus of VP3 (ref.15) and distinctive 2A proteins with unknown functionl6. The lack of VP4 in these viruses,which is implicated in forming membrane pores crucial for viral uncoatingl7,suggests that the parechoviruses mayuncoat viaa different mechanism.Compared with $\mathrm { H P e V }$ ，theN terminus ofLVVPO is further shortened by some 3O residues and does not contain the myristylation signal GXXX(S/T) typical of the majority of picornaviruses18.A further major difference between HPeV and LV is located at the C terminus of VP1-LV does not possess the arginine-glycineaspartic acid(RGD)motif important for cell surface interaction with integrin receptors in most $\mathrm { H P e V s } ^ { 1 9 , 2 0 }$ , but instead contains a unique 42-amino-acid extension.To date,no receptors have been identified for LV.

For many picornaviruses,two predominant types of viral particles are produced during a natural infection,mature virus (containing packaged RNA） and empty procapsids (without RNA),which may be separated using continuous sucrose density gradient ultracentrifugation21. However， such natural empty particles have not been observed for either LV or $\mathrm { H P e V } ^ { 2 2 }$ . The reasons for the absence of empty particles are unclear,but may reflect a distinct assembly mechanism.It is conceivable that the basic N-terminal A/LRM (arginine/lysine-rich motif) of VP3 mightbe involved inRNA-protein interactionsand this,together with the fact thatviral RNA does not cleave VPO into VP2and VP4,could reflect an altered mechanism of genome encapsidation forparechoviruses.Assembled LV particles are robust,so that even heat treatment,which tends to produce modified enterovirus particles23，does not produce detectable alteration in LV particles22.Furthermore，LV particlesare reported tobe resistant to acidic $\mathsf { p H }$ ，detergents and oxidizing environments with complete inactivation of crude extracts requiring heating to $9 0 ^ { \circ } \mathrm { C }$ (ref. 24).

Althoughan $8 . 5 { - } \mathring \mathrm { A }$ cryo-electron microscopy (cryo-EM) structure of $\mathrm { H P e V - 1 }$ has been reported15,there is no highresolution structural information available for parechoviruses. Here we report the determination of an atomic model of LV from a 3.78-A resolution cryo-EM analysis. Structural comparisons place the virus close to HAV within the evolutionary hierarchy of picornaviruses,while substantial differences on both the inner and outer capsid surfaces,including the visualization of $12 \%$ of the RNA genome forming icosahedrally ordered interactions with the capsid,suggest the use of a charge-driven mechanism for genome encapsidation.

# Results

Characterization and structure determination.An infectious complementary DNA (cDNA） clone was previously constructed from the_cytolytically replicating cell-culture-adapted LV variant $8 7 – 0 1 2 \mathrm { G } ^ { 2 5 }$ .Virus generated from this cDNA clone produced a clear cytopathic effect (CPE）at 3 days post infection.LV was grown in GMK cells and purified by centrifugation,linear and discontinuous sucrose gradient ultracentrifugation and ultrafiltration (See Methods).Unlike other picornaviruses,for example,enterovirusA71,CVA16 and HAV,only one band was observed following rate-zonal centrifugation and characterized by SDS-polyacrylamidegelelectrophoresis(Supplementary Fig.ia,b).The PaSTRy assay26，260/2280 absorbance ratios (1.64）and negative-stain transmission electron microscopy (Supplementary Fig. 1c） indicated that this band comprised mature virions,containing RNA.

Cryo-EM micrographs of purified LV virions were recorded using a FEI Polara electron microscope equipped with a Gatan K2 Summit detector (see Methods)．A total of 5,558 virions were selected from the cryo-EM images and subjected to two-dimensional (2D）alignment and three-dimensional (3D) reconstruction by single-particle techniques(Fig. 1） using EMAN2 andRelion27,28 Imageprocessingand3D reconstruction adopted truly independent refinements²9 and the resolution was assessed using the‘gold'standard Fourier shell correlation $( \mathrm { F S C } ) = 0 . 1 4 3$ criterion30,31The final resolution was $3 . 7 8 \mathring { \mathrm { A } }$ (Table 1; Supplementary Fig.2). The polypeptide main chain and side chains were well resolved for most of the capsid (Fig.2),allowing an atomic model for the majority of the three capsid proteins to be manually built in.The model was refined32 using standard X-ray crystallographic metrics,including $R$ -factor, real-space correlation and Ramachandran plots,to validate the refined structure (Table 2).

Unusual surface features.The LV capsid proteins are mostly well ordered,apart from residues1-30 ofVP1,1-36 ofVP0 and1-3 of VP3.There are very marked ( $\mathrm { \sim } 2 0 \mathring \mathrm { A }$ high）protrusions displaced some $4 5 \mathrm { \AA }$ from the icosahedral five-fold axes,but otherwise the external surface is broadly similar to that of other picornaviruses;being relatively smooth it is most similar to $\mathrm { \dot { H } A V } ^ { 2 1 }$ ,although less angular (Fig.3a,b). There is no trace of the enteroviruscanyonoftenthesiteofreceptorbinding,since shortened VP1 and VPo loops ablate the north and south walls of the canyon,in a very similar fashion to HAV.The transverse central sections of the LV density maps (Fig. lc,d) emphasize the internal differences,revealing genomic RNA involved in visible contacts with the protein shell at the five-fold axes.

![](images/90f8811c0b0b494e5e8bf21b9768a3b957a72665eb1b86270db19958786b053f.jpg)  
FigureSgleparticecosrctisofLVarafLcosructiofVartivddotefdis surfaceiscolouedbdusfrouetougentodfroteestothstdoadafifdtfoddtde arelablleddspetiveyfetactfrtiedtis.dtiof the LV reconstruction rendered at higher contour level ( $4 \sigma$ above the mean),viewed alonga five-fold axis of symmetry and depth queued to reveal the five high-density protrusions inside the capsid under each vertex.

<html><body><table><tr><td>Table 1| Cryo-EM imaging and data processing statistics.</td></tr><tr><td>Micrographs (total) 292; micrographs (used) 288</td></tr><tr><td>Particlesselected 5,867</td></tr><tr><td>Particles included in final reconstruction 5,558</td></tr><tr><td>Sampling,A per pixel 1.35 1.5-3.0</td></tr><tr><td>Defocus range,μm Resolution,A (gold standard FSC= O.143 criterion) 3.78</td></tr><tr><td></td></tr><tr><td>FSC, Fourier shell correlation.</td></tr></table></body></html>

Stability of LV.LV capsids from crude extracts have been reported to be able to withstand high temperature and low $\mathrm { \ p H }$ (retaining infectivity above $7 0 ^ { \circ } \mathrm { C }$ and at $\mathrm { p H s }$ down to about 4), which would make them equally robust with HAV capsids21. However,our results (Supplementary Fig.3） show that genomic RNA becomes exposed at a significantly lower temperature $( \sim 6 0 ^ { \circ } \mathrm { C } )$ with little variation in stability with $\mathsf { p H }$ ，this being $\sim 1 0 ^ { \circ } \mathrm { C }$ lower than the capsid dissociation temperature of HAV. In fact,this is broadly in line with previous experiments with purified $\mathrm { L V } ^ { 2 4 }$ ， which suggests that_an exogenous factor may stabilize the particles in infected cells.In enteroviruses,capsid protein melting occurs in two distinct steps indicative of a twostage transition in protein conformation，whereas LV only undergoes the higher-temperature conformational transition, consistent with the observation that alternative particles of parechoviruses cannot be produced by heat treatment²².These results,along with the lack of VP4,indicate that LV probably uncoats via a novel mechanism,which is different from those utilized by most other picornaviruses21. In line with this, monitoring of LV replication by immunofluorescence staining suggested direct cell-to-cell transmission25，which was also proposed for HAV21.

The organization of VP1-3 resembles that observed in HAV. The capsid proteins (VPo,VP1 and VP3）adopt the expected eight-stranded anti-parallel $\beta$ -barrel configuration andare arranged with pseudo $T = 3$ symmetry (where $T$ is the triangulation number),such that VP1 surrounds the five-fold axes and VPO and VP3 alternate about the two- and three-fold axes. Structure-based sequence alignment allows us to make a robust sequence alignment revealing limited similarity $( \sim 2 5 \%$ identity) to other picornaviruses (Supplementary Fig. 4). Overall,HAV has the most similar known structure to LV and the fold of the three proteins in the two viruses are compared in Fig.3d-g. Perhaps correlated with the absence of a canyon,LV does not harbour a continuous hydrophobic pocket in VP1 (Supplementary Fig. 5a), such as that seen in enteroviruses34 where expulsion of a lipidic pocket factor is a pre-requisite for uncoating35.LV VP1 most closely resembles that ofHAV,whichlikewise does not havea pocket factor (root mean squared deviation for $8 3 \%$ of Cαs is $\dot { 2 } . 0 \mathring \mathrm { A } )$ ; similar characteristics between the two viruses in VP1 are: a lengthened $\beta \mathrm { H }$ strand and alternative conformation of the GH loop blocking the entrance to the pocket,while the ${ \beta } { \mathrm { C } } ,$ βE, $\beta \mathrm { F }$ and $\beta \mathrm { H }$ strands shift so that it can no longer accommodate a lipidic pocket factor.There are nonetheless differences in the conformation of the CD,EF and GH loops between the LV and HAV VP1 structures (Fig. 3e). The LV GH loop is three residues shorter (one of the shortest in any picornavirus) and the CD and EF loops are nine and seven residues shorter,respectively. The major difference between LV and HAV VP1,however,occurs at the C terminus,owing to the $\sim 5 0$ residue extension in LV (Supplementary Fig. 4 and next section). LV VPO is strongly similar to HAV VP2 (root mean squared deviation for $2 0 6 ~ \mathrm { C } \mathfrak { X } s$ $( 9 5 \% )$ is $1 . 6 \mathring { \mathrm { A } } \rangle$ .Sequence alignment indicates a short counterpart 0 $\cdot \sim 2 7$ residues) of HAV VP4 in LV VP0 (Supplementary Fig. 4), although there is no reported cleavage of VPO into VP2 and VP4 in the genus Parechovirus14. Both HAV VP4 and the LV VPO N terminus (residues 1-36) are disordered. The N terminus of HPeV VPO is extended by some 30 residues compared with LV. LV uses the VP2 N-terminal domain swap across the icosahedral two-fold axis,frst observed in a picornavirus for $\mathrm { H A V } ^ { 2 1 }$ (and seen in the insect picorna-like viruses36),possibly imparting extra stability at this interface (Fig. $\mathrm { 3 d , f ) }$ ．This arrangement is presumably common to all members of the genus.Furthermore, the interaction region (VP2 αA helix) adjacent to the icosahedral two-fold axes,which__separates during the initial stages of enterovirusuncoating35,37，shows tight packing in LV $3 . 7 \mathring \mathrm { A }$ separation),as was seen in HAV2l,possibly further contributing to the stability of the LV particles.The overall fold of VP3 is similar to other picornaviruses;however,LV and parechoviruses have an N-terminal extension, which is responsible for interacting with RNA (see below).LV VP3 has a puff structure connecting the αA helix and $\beta \mathrm { B }$ strand,which is absent in HAV,but the latterhas an additional puff structure between the $\beta \mathrm { C }$ strand and $\alpha \mathrm { B }$ helix of VP3 (Supplementary Fig.4).There are also differences between the two viruses in the conformation of the VP3 GH loop (Fig. 3).We would expect strong similarity between the structure ofLVand HPeVs; however,antibodies raised against the VPo and VP1 capsid proteins of LV,while recognizing the LV virion,do not recognize HPeV-1 by immunofluorescence of infected cells38.

![](images/ced860dd5ece9de66c3252c9112b19f56c6f410b292fb2d90fd07b25ab21f2a7.jpg)  
Figure2|Electron density maps.(a) From a section of VP1.(b) From a section of VPO.

<html><body><table><tr><td>Table 2|Model refinement statistics.</td><td></td></tr><tr><td>R.m.s.d. (bond angles,°)</td><td>1.573</td></tr><tr><td>R.m.s.d. (bond lengths,A)</td><td>0.010</td></tr><tr><td>Ramachandran statistics Most favoured (%)</td><td>90.7</td></tr><tr><td>Allowed Outliers</td><td>8.2 1.1</td></tr><tr><td>Rwork/Rfree (%)</td><td>29.7/31.3</td></tr><tr><td></td><td></td></tr><tr><td>r.m.s.d., root mean squared deviation.</td><td></td></tr></table></body></html>

Surface projections surround the five-fold axes. The external surface of the LV capsid possesses unusual protrusions surrounding the five-fold axes,which extend the radius of the LV particle to $\mathrm { \sim 1 7 0 \AA }$ (Fig. 3b).Although these are poorly ordered and best seen in lower-resolution reconstructions,it is unambiguous that they are formed from the C terminus of VP1. The disorder precludes tracing the protein backbone for this domain (residues 242-297) but the predicted protein secondary structure would suggest three helices.Such a protrusion formed from a C-terminal extension is unique in picornaviruses； however,in Seneca valley virus,there is a similarly sized and placed projection formed by loops of VP1 and VP2,and in the insect picorna-like virus Triatoma virus $( \mathrm { T r } \mathrm { V } ^ { 3 9 } )$ there are such projections formed from insertions within the loops and $\beta$ -sheets of VP1 and VP3, although displaced by $\mathrm { \sim } 1 3 \mathring \mathrm { \mathrm { A } }$ (Fig. 3). The 55 amino acids contributing to this structure have a high proportion of charged residues (13 negatively and 8 positively charged residues). Some other picornaviruses,for example,CV-A9 and $\mathrm { H P e V - 1 }$ ，possess an arginine-glycine-aspartic acid (RGD) motif at the C terminus of VP1 involved in integrin recognition; however, this motif is not present in the LV VP1 extension,which appears to contain no distinctive sequence motifs (Supplementary Fig. 4),although it shares $4 0 \%$ identity with the universal stress protein of Streptomycesauratus.Cell-culture-adaptedvirusableto replicate more efficiently has been found to have three mutations in this region: Y289H in VP1 and A162T and S172G in VPO (ref.40) (Supplementary Fig. 5b). The VP0 mutations are located at either end of the EF loop and while not surface exposed, they may well alter the surface conformation, especially since $\mathrm { A l } 6 2 \mathrm { T }$ is close to a disulphide bond between Cys 191 (GH loop） and Cys 124(CD loop)of VP1.Y289H is located in the VP1 C-terminal extension (Supplementary Fig. 5b),disordered density for which is located above the junction of the three capsid proteins within a biological protomer (Fig.3d) and is therefore adjacent to the VPo EF loop.LV encodes two 2A proteins,2A1 and 2A2. The 20-residue-long aphthovirus-like 2A1 protein possesses a DvExNPGP motif proposed to mediate cleavage of the LV polyprotein in vitro and may constitute the carboxy terminus of the structural protein VP1 (ref.13).It is not yet resolved as to whether this forms partof the extra densitywe see in the five-fold protrusion.However, in HAV protein VPX $( \mathrm { V P } 1 + 2 \mathrm { A } )$ is present during particle assembly and the 2A protein is believed to have a role in directing assembly,before being cleaved off.It is possible that the 2A1 of LV also has a role in particle assembly,and may likewise be cleaved on particle maturation.

![](images/263a568a6a4065066dcc0a49cbf69c856ec8cd192c1dc23bd732b2b9822d51e0.jpg)  
Figure3OveraltructurfLV(a-cmparisofapsidfL)ithHV(aan).ecapsidsareolouredbydufroued accordingtotalebo.dbagrafteLotoerOdoinueeddspcti C-terminal55residuestatfothesurfaceprotrusioaroundthefivefoldaxisaresobyaowerresolutioectrodenitmap.bon diagrams showing the LV capsid proteins VP1(e),VPO (f)and $\mathsf { V P 3 } \left( \pmb { \mathsf { g } } \right)$ with the individual strands and termini labelled,compared with those of HAV(grey), the N-and C termini of the LV proteins are labelled in blue and red,respectively.

The N-terminal region of VP3 binds an RNA stem-loop. The extended N-terminal region of VP3 was clearly defined in the density map and was modelled starting from residue 4 onwards (Fig.4b).The sequence is markedly basic,reminiscent of the RNA-binding domains of $T = 3$ plant viruses4l (Supplementary Fig.4).Proceeding from residue 64 to residue 4,the polypeptide leaves the VP3 eight-stranded antiparallel $\beta$ -barrel and passes along close to a five-fold axis to insert into a symmetry-related VP3 on the inner surface of the protein shell (Fig. 4). There are fiveprominent, $\sim 5$ -nm-long spiralling regions of density extending radially from each five-fold axis at the inner surface of the capsid (Fig.4).There is a considerable amount of density in addition to that corresponding to the residues of the VP3 N-terminal extension,which we identify as RNA in part based on the shape of the electron density (Fig. 4a; Supplementary Fig. 6).

We note that a feature ata similarlocation was also observed in human parechovirus,although at lower resolution,such that it was not possible to identify the nature of the protein-RNA interaction15.Electrostatic interactions dominate the contact between the $\mathrm { V P 3 ~ N }$ terminus and the RNA stem-loop (Fig. 4c). The N-terminal polypeptide (residues 1-64) comprises 13 basic resides,most of whichare clustered within the first20 residues.A total of 65 basic residues from mainly the N-terminal segments of the VP3 subunits around the five-fold axes,but also including a region of VP1，produce a positively charged environment that partly counteracts the abundant negative charge of the RNA phosphate groups.Residues including Arg 6,Lys 7,Lys 9,Lys 12, Lys14,Arg 49,Arg 62,Lys 64 ofVP3 and Arg 225 ofVP1 are engaged in interactions with the ordered RNA (Fig.4b);of these, most of the VP3 residues are conserved across the genus,while residue 225 of VP1 is strictly conserved.

Our interpretation of the RNA density places 15 bases of RNA in close association with the $\mathrm { ~ N ~ }$ terminus of VP3,although theresolution is insufficient to identify them individually. The strength of the icosahedrally averaged density implies that the RNA is essentially fully occupied (Fig. 4a; Supplementary Fig. 6). Throughout the capsid the ordered RNA comprises some 900 bases, $12 \%$ of the viral genome. We note that Seal et al.42 observed that $\mathrm { H P e V - 1 }$ contains a large $( 4 0 \% )$ fraction of hairpin RNA compared with PV RNA and that modelling of the genomic $5 ^ { \prime }$ end_showed unique secondary structure folding of this region25; we propose that this structural constraint isa common feature of parechoviruses.

![](images/fc9d1c990f4d591a39603b41e9981b0db10ff24a650287cb436a758448fb4de6.jpg)  
Figure4|ThepartialtructurefthVgenoeadirusecapsdati.(aEectronstforteNeriniofVanddeedodte fivefldisd residue4labelldtsie6beldndteirproximittomage).)eelectrostaticsurfefteLVpentamowingte predominanceofpositivagetatiteractitthbcboefteomicAmagetaics).dCartoodiagamepresetingthLVoic RNAasaspherewihoderedfragmentsfoingpentamericprotrusios(mageta);thepentamersofasidproteinasociatewiththegenoeRA through electrostatic interaction.

![](images/acedf9a63e5a82adbc246206133e584c9d6c239ad4c7c07e8b4de944886bdf1d.jpg)  
Figure5Phgey.(aupepositiofLVOi2flvusDVHVdCesfLVfsialV andCrPVbutdiferetltotpovirusarasteEFootatfomspartftsothalloftoinlors.)tructured ()qcedgiitie HRV14;e;ss fottsi

Evolution and implications for assembly. Overall, the structure of LV is most similar to that of HAV (Fig. 5).Furthermore,LV possesses a number of structural features also observed in insect viruses,including the VP2 N-terminal domain swap (Fig. 5a),the presence of a basic VP3 N-terminal ‘arm'43 and the observation of well-ordered genome RNA with numerous protein/RNA interactions44.Indeed， both structure and sequence-based phylogeny suggests that LV is the closest picornavirus to the insect picorna-like viruses (Fig. 5b,c).

# Discussion

One of the key distinguishing features of LV is the long extension at the C terminus of VP1.Other picornaviruses such as CV-A9 and the closely related HPeV also have VP1 C-terminal extensions,although shorter,and these function in cell attachment15.This is the most likely role of this extension,presumably giving the virus scope to attach to a different receptor,although a role in particle assembly is also possible as in HAV21.In line with this,mutations that increase replication efficiency map to this region or the adjacent VPO EF loop.Increased replication efficiency was also associated with induction of an apoptotic response in GMK cells,as indicated by detection of activated caspase-3 and DNA fragmentation40. It is conceivable that this extension,with a rich collection of basic residues might also possess an alternative function in regulating host gene expression.

The observation of 15 bases of relatively well-ordered RNA associated with the N termini of VP3 near the five-fold axes is also significant in terms of the Picornaviridae family.Wellordered genome RNA has thus far only been observed in parechoviruses15. For other genera of the family,only occasional isolated bases have been observed. This suggests that members of the Parechovirus genus may use a mechanism of passive genome encapsidation whereby VP3 acts asan RNA chaperone, organizing the genome into a condensed structure compatible with the icosahedral symmetry of the particle through numerous largely electrostatic interactions with short RNA segments with preformed stem-loop structures.Such a mechanism would be significantly different from the model for enterovirus packaging, which involves interaction between capsid proteins and nonstructural proteins in the RNA replication complex,as described for $\mathrm { P V } ^ { 7 }$ ，but is not inconsistent with the observation of RNA secondary structures described as packaging signals in Aichi virus(genus Kobuvirus,see Fig. 5c) and the icosahedral RNA plant virus,satellite tobacco necrosis virus4.This may also be relevant to the reported lack of VPo cleavage in parechoviruses, since thismaturation cleavage isthought to happen autocatalytically in the presence of RNA,and may be a final quality assurance step required,for example,by genera such as enteroviruses.Substantial portions of ordered RNA have been reportedininsect， plant， otheranimalvirusesand bacteriophages45-48，and basicregions，often located at the $\mathrm { N } .$ or C termini of the coat proteins,are found in several families of icosahedral viruses where they are thought to be involved in RNA-protein interactions responsible for encapsidating the viral genomel. Similar to some insect viruses,LV bears such a basic A/LRM motif at the $\mathrm { V P 3 ~ N }$ terminus,which is seen to bind viral RNA.This binding would not seem to be directly sequence dependent, but governed by electrostatic effects in conjunction with local RNA structure.A role for the highly basic 20

N-terminal residues of LV VP3,in association with RNA,in directing the assembly and dimensioning of the pseudo $T = 3$ particle49， would be consistent with the lack of observation of empty parechovirus particles in the viral life cycle.In the absence of the RNA the charge on the individual protomers may be repulsive,perhaps blocking the assembly of the pentamer4.It has been shown for nodaviruses47,50 that selection of non-viral RNA for packaging results in the formation of more dynamic particles. It may be that more recently evolved picornaviruses use more specific encapsidation signals in the $\mathrm { \mathop { R N A } } ^ { 5 1 }$ rather than an electrostatic effect.Indeed,other evidence presented here such as the VP2 domain swap and absence of a VP1 pocket suggests that LV,similar to HAV,retainsa number of structural and functional features characteristic of primordial picornaviruses,which may have had their origins in insect viruses.Understanding the physical characteristics of LV and its assembly should help in devising methods to prevent and alleviate the infection and spread of this virus.

# Methods

InfectiouscDNAinvitro transcriptionand transfection.An infectiouscDNA clone,pLV87-012G,containing the full genome sequence of LV-1 87-012G,was linearized using the NotI site encoded downstream of the poly-A tail and transcribed in vitro usinga MEGAscript T7 Transcription kit (Ambion).RNAwas transfected into subconfluent Vero cells using Escort IV reagent (Sigma).CPE was initially weak and increased CPE was observed over five passages following which the cell lysates were used to infect GMK cells.

Virus production and purification.LV was grown in $2 8 \times 1 7 5 \mathrm { - } \mathrm { c m } ^ { 2 }$ flasks of GMK cells infected at a multiplicityof infection of 2.The cells were cultured in DMEM (Sigma) supplemented with $1 \%$ fetal bovine serum (Gibco) until $9 0 \%$ of cells exhibited a CPE.Both cells and virus-containing supernatant were collected,frozen and thawed three times and then centrifuged to remove cell debris.The supernatants were centrifuged for $^ { 3 \mathrm { h } }$ at $9 5 , 0 0 0 g$ and viral pellets resuspended in PBS buffer $+ 0 . 0 1 \%$ Triton X-1oo,then loaded onto $1 5 \mathrm { - } 4 5 \%$ （2 $\scriptstyle \left( \mathbf { w } / \mathbf { v } \right)$ sucrose density gradients and centrifuged at $2 2 2 , 0 0 0 g$ max for $1 . 5 \mathrm { h }$ in an SW41 rotor at $4 { } ^ { \circ } \mathrm { C }$ # Further details have been described previouslyl2.

Negative-stain electron microscopy.Aliquots of purified samples were diluted to $0 . 2 \mathrm { \bar { m g } m l ^ { - 1 } }$ and deposited onto glow-discharged,Formvar/carbon-coated copper grids (Electron Microscopy Sciences,Hatfield,USA).After $3 0 \mathrm { s }$ incubation, the excess sample was blotted away and the grids washed twice with deionized water. Samples were stained with $1 \%$ （204号 $\scriptstyle \left( \mathbf { w } / \mathbf { v } \right)$ uranyl acetate for $4 5 s$ and excess stain was removed by blotting.Grids were examined ona Tecnai T12 (FEI,Hillsboro, OR) transmission electron microscope operated at $8 0 \mathrm { k V }$ .Images were acquired on a $4 \times 4$ -k-High-sensitivity FEIEagle camera(FEI) at a nominal magnification of $\times 6 7 { , } 0 0 0$

Cryo-EM and data collection.A $3 \mathrm { - } \mu \mathrm { l }$ aliquot of purified LV virions $( \sim 1 \mathrm { m g } \mathrm { m l } ^ { - 1 } )$ was applied to a freshly glow-discharged 400-mesh holey carboncoated copper grid (C-flat,CF-2/1-2C; Protochips Inc.) and blotted for $3 s$ in $7 0 \%$ relative humidity for plunge-freezing(Vitrobot;FEI) in liquid ethane.Cryo-EM data were collected using an FEI Tecnai G2 Polara microscope (FEI) operated at $3 0 0 \mathrm { k V }$ ,equipped with an energy filter(GIF Quantum; Gatan,Pleasanton, CA) operating in zero-loss mode ( $2 0 \mathrm { { e V } }$ energy selecting slit width) and a direct electron detector (K2 Summit; Gatan).Micrographs were collected with a defocus between 1.5 and $3 \mu \mathrm { m }$ as movies (25 frames,each $0 . 2 s \mathrm { \dot { } }$ ）in single electron counting mode using SerialEM52 at a calibrated magnification of $\times 3 7 , 0 2 7$ ,resulting ina pixel size of $1 . 3 5 \mathring { \mathrm { A } }$

Image processing and 3D reconstruction.The micrographs were corrected for beam-induced drift by aligning and averaging the individual frames of each movie using MOTIONCORR53,54，Particles were Selected automatically using ETHAN54 and then manually screened with the boxer program in EMAN55.Contrast transfer function(CTF) parameters for drift corrected micrographs were estimated using a GPU accelerated program Gctf (Kai Zhang,Gctf: real-time CTF determination and correction,2015,http://www.biorxiv.org/content/early/2015/07/12/022376). Micrographs showing signs of astigmatism or significant drift were discarded and not used for further analysis.A total of 5,558 particles from 288 micrographs were subjected to 2D alignment and 3D reconstruction using Relion 1.3 (ref. 28) by following recommended gold standard refinement procedures28 and applying icosahedral symmetry.Two initial models for LV were generated,one was created byEMAN2 (ref.27);another was generated by filtering the reconstructed HAV structure to $3 0 \mathrm { \AA }$ ；the independent initial models all converged to the same result.

Several cycles of 2D classification and 3D refinement were used to further select the particles for final refinement.The final resolution was evaluated using the ‘gold' standard Fourier shell correction (threshold $\phantom { - } = 0 . 1 4 3$ criterion)3031of the two truly independent reconstructions (Supplementary Fig.2).The data set and refinement statistics are summarized in Table 1.

Model buildingand refinement.The atomic modelsof LV VPo,VP1 and VP3 were built de novo into density with the structures of HAV capsid proteins as a guide,using $\mathrm { C O O T } ^ { 5 6 }$ .REFMAC57 was used to calculate the difference map that highlighted the areas where the model was incorrect.Models were further improved by iterative positional and B-factor refinement in real space using Phenix3² and rebuilding in $\mathrm { C O O T } ^ { 5 6 }$ .Only coordinates were refined,the maps were kept constant. Each round of model optimization was guided by cross-correlation between the map and the model.Refinement statistics are given in Table 2 as evaluated by Molprobity58 functions integrated in Phenix.

Thermal stability assay. $\mathrm { P a S T R y } ^ { 2 6 }$ experiments were performed with an MX3005p RT-PCR instrument (Agilent) using both SYTO9 and SYPRO-Red (Invitrogen) dyes to detect the presence of RNA and the exposed hydrophobic regions of proteins,respectively.Fifty-microlitre reactions were set up in a thinwalled PCR plate (Agilent),containing $1 \mu \ g$ of the LV virus, $5 \mathrm { m M }$ SYTO9 and $3 \times$ SYPRO-Red in PBS $\mathrm { ( p H ~ 7 . 4 ) }$ ,and the temperature ramped from 25 to $9 9 ^ { \circ } \mathrm { C }$ Fluorescence was recorded in triplicate at $1 { ^ { - } \mathrm { { ^ { \circ } C } } }$ intervals.

# References

1．Rao,A.L.Genome packaging by spherical plant RNA viruses.Annu. Rev. Phytopathol.44,61-87 (2006).   
2．Ni, P.& Cheng Kao,C.Non-encapsidation activities of the capsid proteins of positive-strand RNA viruses.Virology 446,123-132 (2013).   
3.Casjens,S.R.The DNA-packaging nanomotor of tailed bacteriophages. Nat. Rev.Microbiol.9,647-657 (2011).   
4.Ford,R.J.et al. Sequence-specific,RNA-protein interactions overcome electrostatic barriers preventing assembly of satellite tobacco necrosis virus coat protein.J.Mol.Biol.425,1050-1064 (2013).   
5.Whitton,J.L.,Cornell,C.T.& Feuer,R.Host and virus determinants of picornavirus pathogenesis and tropism.Nat.Rev.Microbiol.3,765-776 (2005).   
6．Jiang,P.,Liu,Y.,Ma,H.C.,Paul,A.V.& Wimmer,E.Picornavirus morphogenesis. Microbiol. Mol. Biol. Rev.78,418-437 (2014).   
7.Liu,Y.et al.Direct interaction between two viral proteins,the nonstructural protein 2C and the capsid protein VP3,is required for enterovirus morphogenesis.PLoS Pathog.6,el001066 (2010).   
8．Sasaki,J.& Taniguchi,K. The $5 ^ { \prime }$ -end sequence of the genome of Aichi virus,a picornavirus,contains an element critical for viral RNA encapsidation.J. Virol. 77,3542-3548 (2003).   
9.Niklasson, B.Current views on Ljungan virus and its relationship to human diabetes.J.Med.Virol.83,1673 (2011).   
10.Tolf, C.et al. Molecular characterization of a novel Ljungan virus (Parechovirus; Picornaviridae) reveals a fourth genotype and indicates ancestral recombination.J.Gen.Virol.90,843-853 (2009).   
11. Lindberg,A.M.& Johansson,S.Phylogenetic analysis of Ljungan virus and A-2 plaque virus,new members of the Picornaviridae.Virus Res.85,61-70 (2002).   
12.Johansson,E. S.et al. Cell culture propagation and biochemical analysis of the Ljungan virus prototype strain. Biochem. Biophys. Res. Commun. 317, 1023-1029 (2004).   
13.Johansson, S.,Niklasson,B.,Maizel,J.,Gorbalenya,A.E.& Lindberg,A.M. Molecular analysis of three Ljungan virus isolates reveals a new,close-to-root lineage of the Picornaviridae with a cluster of two unrelated 2A proteins. J.Virol. 76,8920-8930 (2002).   
14.Stanway,G.& Hyypia,T.Parechoviruses.J. Virol.73,5249-5254 (1999).   
15.Seitsonen,J.et al. Interaction of alphaVbeta3 and alphaVbeta6 integrins with human parechovirus 1.J. Virol. 84,8509-8519 (2010).   
16.Hughes,P.J.& Stanway,G.The 2A proteins of three diverse picornaviruses are related to each other and to the H-revlO7 family of proteins involved in the control of cell proliferation.J. Gen.Virol.81,201-207 (2000).   
17.Panjwani,A.et al. Capsid protein VP4 of human rhinovirus induces membrane permeability by the formation of a size-selective multimeric pore.PLoS Pathog. 10,e1004294 (2014).   
18.Chow,M.et al. Myristylation of picornavirus capsid protein VP4 and its structural significance. Nature 327,482-486 (1987).   
19.Pull,T.,Koivunen,E.& Hypia,T.Cell-surface interactions of echovirus 22. J.Biol. Chem. 272,21176-21180 (1997).   
20.Triantafilou,K.,Triantafilou,M.,Takada,Y.& Fernandez,N.Human parechovirus 1 utilizes integrins alphavbeta3 and alphavbetal as receptors. J.Virol. 74,5856-5862 (2000).   
21.Wang,X.et al. Hepatitis A virus and the origins of picornaviruses.Nature 517, 85-88 (2015).   
22.Ekstrom,J.-O.Doctoral Dissertation (University of Kalmar,2007).   
23.Levy,H.C., Bostina,M.,Filman,D.J.& Hogle,J.M.Catching a virus in the act of RNA release: a novel poliovirus uncoating intermediate characterized by cryo-electron microscopy.J. Virol. 84,4426-4441 (2010).   
24.Ekstrom,J.O.,Tolf,C.,Edman,K.A.& Lindberg,A.M.Physicochemical properties of the Ljungan virus prototype virion in different environments: inactivated by heat but resistant toacidic pH,detergents and non-physiological environments such as Virkon-containing solutions.Microbiol. Immunol. 51, 841-850 (2007).   
25.Ekstrom,J. O.et al. Replication of Ljungan virus in cell culture: the genomic $5 ^ { \prime }$ -end,infectious cDNAclones and host cellresponse toviral infections.Virus Res.130,129-139 (2007).   
26.Walter,T.S.et al.A plate-based high-throughput assay for virus stability and vaccine formulation.J. Virol.Methods 185,166-170 (2012).   
27.Tang, G.et al. EMAN2: an extensible image processng suite for electron microscopy.J. Struct. Biol.157,38-46 (2007).   
28.Scheres,S.H.W.RELION: Implementation of a Bayesian approach to cryo-EM structure determination.J. Struct. Biol. 180,519-530 (2012).   
29.Guo,F.& Jiang,W. Single particle cryo-electron microscopy and 3-D reconstruction of viruses.Methods Mol. Biol.1117,401-443 (2014).   
30.Henderson,R.et al. Outcome of the first electron microscopy validation task force meeting.Structure 20,205-214 (2012).   
31.Scheres,S.H.W.& Chen,S.X.Prevention of overfitting in cryo-EMstructure determination.Nat. Methods 9,853-854 (2012).   
32.Afonine,P.V.et al. Towards automated crystallographic structure refinement with phenix.refine.Acta Crystallogr.D Biol.Crystallogr.68,352-367(2012).   
33.Rossmann,M.G.et al. Structure of a human common cold virus and functional relationship to other picornaviruses.Nature 317,145-153 (1985).   
34.Hogle,J.M., Chow,M.&Filman,D.J. Three-dimensional structure of poliovirus at 2.9 A resolution. Science 229,1358-1365 (1985).   
35.Wang, X.et al. A sensor-adaptor mechanism for enterovirus uncoating from structures of EV71.Nat. Struct.Mol.Biol.19,424-429 (2012).   
36.Tate,J.et al. The crystal structure of cricket paralysis virus: the first view of a new virus family.Nat. Struct.Biol.6,765-774 (1999).   
37.Dang,M. et al. Molecular mechanism of SCARB2-mediated attachment and uncoating of EV71.Protein Cell 5,692-703 (2014).   
38.Tolf,C.et al. Characterizationof polyclonalantibodiesagainst thecapsid proteins of Ljungan virus.J.Virol. Methods 150,34-40 (2008).   
39.SquiresGetlucureoftsaidtaCtalol Crystallogr.69,1026-1037 (2013).   
40.Tolf, C., Gulberg,M.,Ekstrom,J. O.,Jonsson, N.& Michael Lindberg,A. Identification of amino acid residues of Ljungan virus VPO and VPl associated with cytolytic replication in cultured cels.Arch. Virol.154,1271-1284 (2009).   
41.Larson, S.B., Lucas,R.W., Greenwood,A.& McPherson,A. The RNA of turnip yellow mosaic virus exhibits icosahedral order.Virology 334,245-254 (2005).   
42.Seal,L.A.& Jamison,R.M.Evidence for secondary structure within the virion RNA of echovirus 22.J. Virol. 50,641-644 (1984).   
43.Hogle,J.M,Maeda,A.& Harrson,S.C.Structure and assemblyof tuip crinkle virus.I. X-ray crystallographic structure analysis at 3.2A resolution. J.Mol. Biol.191,625-638 (1986).   
44.Sorger,P.K,ockley,P.G.&arronS.C.ructreadassembyf crinkle virus.II. Mechanism of reassembly in vitro.J. Mol. Biol.19i,639-658 (1986).   
45.Larson,S.B.et al. Double-helical RNA in satelite tobacco mosaic virus.Nature 361,179-182 (1993).   
46.Tang, L.et al. The structure of pariacoto virus reveals a dodecahedral cage of duplex RNA. Nat.Struct.Biol.8,77-83 (2001).   
47.Fisher,A.J.& Johnson,J.E.Ordered duplex RNA controls capsid architecture in an icosahedral animal virus.Nature 361,176-179 (1993).   
48.Toropova,K.,Basnak, G.,Twarock,R. Stockley,P.G.& Ranson,N.A. The three-dimensional structure of genomic RNA in bacteriophage MS2: implications for assembly.J.Mol. Biol. 375,824-836 (2008).   
49. Sangita, V. et al. $\mathrm { { T } } = 1$ capsid structures of Sesbania mosaic virus coat protein mutants: determinants of $\mathrm { T } = 3$ and $\mathrm { { T } } = 1$ capsid assembly.J. Mol. Biol. 342, 987-999 (2004).   
50.Dong,X.F.,Natarajan,P.,Tihova,M.,Johnson,J.E.& Schneemann,A.Particle polymorphism caused by deletion of a peptide molecular switch in a quasiequivalent icosahedral virus.J.Virol.72,6024-6033 (1998).   
51.Stockley,P.G.etalPackagingsignalsinsingle-strandedRAvises:aures alternative to a purely electrostatic assembly mechanism.J.Biol.Phys.39, 277-287 (2013).   
52. Mastronarde,D. N. Automated electron microscope tomography using robust prediction of specimen movements.J. Struct. Biol.152,36-51 (2005).   
53.Li, X.et al. Electron countingand beam-induced motion correction enable near atomic-resolution single-particle cryo-EM.Nat.Methods 10,584-590 (013).   
54.Kivioja,T.,Ravanti,J.,Verkhovsky,A.,Ukkonen,E.& Bamford,D.Local average intensity-based method for identifying spherical particles in electron micrographs.J. Struct. Biol.131,126-134 (2000).   
55.Ludtke,S.J.,Baldwin,P.R.& Chiu,W.EMAN: semiautomated software for high-resolution single-particle reconstructions.J. Struct. Biol. 128,82-97 (1999).   
56.Emsley,P.& Cowtan,K.Coot: model-building tools for molecular graphics. Acta Crystallogr.D Biol.Crystallogr.60,2126-2132 (2004).   
57.Murshudov,G.N.,Vagin,A.A.& Dodson,E.J.Refinement of macromolecular structures by the maximum-likelihood method.Acta. Crystallogr.D Biol. Crystallogr.53,240-255 (1997).   
58.Chen,V.B.et al.MolProbity:all-atom structure validation for macromolecular crystallography.Acta Crystallogr.D Biol.Crystallogr.66,12-21 (2010).   
59.Yamashita,T.et al. Complete nucleotide sequence and genetic organization of Aichi virus,a distinct member of the Picornaviridae associated with acute gastroenteritis in humans.J.Virol.72,8408-8412 (1998).

# Acknowledgements

We thank Kai Zhang for advice on data processing, Jun Dong,Robert Esnouf and Jonathan Diprose for IT support and Dave Rowlands for reading the manuscript. The OPIC electron microscopy facility was founded by a Wellcome Trust JIF award (060208/ $Z / 0 0 / Z )$ and is supported by a WT equipment grant (093305/Z/10/Z).The Wellcome Trust,MRC and BBSRC also supports the National EM facility,eBIC,which enabled provision of the K2 detector.Work was also supported by the Ministry of Science and Technology 973 Project (grant no.2014CB54280o),National Science Foundation grant no.8l330036,no.31570717 and 81520108019 and the Strategic Priority Research Program of the Chinese Academy of Sciences,grant no.XDB080202o0.H.W.and A.P.are supported by the MRC(MR/K020811/1) and The Pirbright Institute receives strategic funding from the BBSRC.J.R.is Wellcome Trust-supported,EEF and DIS are supported by the MRC (grant no Glooo99),and D.I,S.is supported as a Jenner Investigator.The work of the Wellcome Trust Centre in Oxford is supported by the Wellcome Trust core award 09053/Z/09/Z.

# Author contributions

L.Z.,X.W., C.P.,H.W.,A.P.,T.J.T.and A.M.L. prepared samples.L.Z.,X.W.,C.P.,J.R., J.-O.E.,A.K.and C.A.S.assisted in research.E.E.F.,T.J.T.,N.J.K.and D.I.S.designed the study.All authors analysed data and L.Z.,X.W.,J.R.,E.E.F.,T.J.T., Z.R.and D.I.S.wrote the manuscript.

# Additional information

Accession codes:Coordinates are deposited in the Protein Data Bank under accession code 3JB4.Cryo-EM reconstructions are deposited in the EMData Bank under accession codes EMD-6394 $( 3 . 7 8 \mathrm { \AA } )$ and EMD-6395 (4.3 A).

Supplementary Information accompanies this paper at http://www.nature.com/ naturecommunications

Competing financial interests: The authors declare no competing financial interests.

Reprints and permission information is available online at http://npg.nature.com/ reprintsandpermissions/

How to cite this article: Zhu,L.et al. Structure of Ljungan virus provides insight into genome packaging of this picornavirus.Nat. Commun. 6:8316 doi:10.1038/ncomms9316 (2015).

This work is licensed under a Creative Commons Attribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license,visit http://creativecommons.org/licenses/by/4.0/