# CRASP: CFP reconstitution across synaptic partners

Yiming Li a, b, Aike Guo a, C, Hao Li a,

Istitutee   
China   
b Universityof Chinese Academyof Sciences,Beijing10049,China   
‘StateKeyLaboratoryofranandCognitiveScience,Istituteofiophsics,ChineseAcademyofSiences,BeijingCina

# ARTICLEINFO

# ABSTRACT

Article history: Received 27 November 2015 Accepted 2 December 2015 Available online 10 December 2015

Keywords:   
CRASP   
GRASP   
CFP reconstitution   
GFP reconstitution   
Connectome   
Drosophila

Mapping the pattern of connectivity between neurons is widely regarded to be critical for understanding the nervous system. GRASP(GFP reconstitution across synaptic partners)has been used as a promising method for mapping neuronal connectivity,but is currentlyavailable in the greencolor only,limiting its potential applications. Here we demonstrate CRASP(CFP reconstitution across synaptic partners),a cyancolored version of GRASP.We validated the system in HEK 293T cels,and generated transgenic Drosophila lines to show that the system could reliably detect neuronal contacts in the brain.Furthermore,we showed that the CRASP signal could be selectively amplified using standard immunohistochemistry methods.The CRASP system adds to the toolkit available to researchers for mapping neuronal connectivity,and substantially expands the potential application of GRASP-like strategies.

$\circledcirc$ 2015 Elsevier Inc.All rights reserved.

# 1.Introduction

The brain is composed of enormous number of neurons that are intricately wired with each other,with its functions thought to be largely determined by the connectivity pattern between the individual neurons [1].Mapping the pattern of connectivity between neurons is therefore critical for the understanding of the nervous system. However, the fine size of neuronal processes and the extreme complexity of the nervous system pose formidable technical difficulties for connectivity mapping.Up to date,Caenorhabditis elegans is the only species for whicha comprehensive connectivity diagram of its 3O2 neurons has been obtained using electron microscopy reconstruction methods [2].However,even with the latest generation of automated electron microscopy image acquisition and reconstruction techniques,these projects remain extremely labor- and time-intensive,requiring millions to even billions of man-hours to complete [3].

GRASP(GFP reconstitution across synaptic partners）isa recently developed strategy[4] that offers a simpler alternative method for mapping neuronal connectivity (Fig.1A).The strategy is based on the split-GFP methods[5],where the GFP molecule was split into two fragments that were not fluorescent when isolated from each other.Interestingly,when the two complementary parts were in physical proximity,the fragments self-assemble into functional proteins that emitted green fluorescence when excited. In the GRASP strategy, the two fragments of split-GFP were separately expressed on the extracellular surface of two groups of neurons of interest.Green fluorescence could be observed only when these two groups of neurons were in physical contact with each other,providing a measure of whether the neurons are connected.Since its introduction in 2oo8,GRASP has been widely used for mapping neuronal connectivity in nematodes,fruitflies and mice [4,6-13].

A majorlimitation of the method,however,is that the system is available in only one color (green） at present.As a result,many useful genetically encoded proteins that emit green fluorescence, suchas GFP itself or the calcium indicator G-CaMP[14],could not be used simultaneously with the GRASP system,because signals from the proteins could not be easily separated from each other. We reasoned that such a problem could be overcome by creating variants of GRASP in other colors.

Here we demonstrate a cyan-colored variant of GRASP that we have named CRASP(CFP Reconstitution Across Synaptic Partners). Based on the bright cyan fluorescent protein Cerulean [15],we made five point-mutations to the split-GFP constructs,obtaining a pair of split-Cerulean constructs spCerulean1-1O and spCerulean11.

![](images/b9c1bc6ff8f98e331368fb8ba8dfd99bb031b711d3c76573769d1f96a495e1dc.jpg)  
Fig.1.The GRASP and CRASP system.(A) Schematic illustration of the GRASP system. Green fluorescence could be observed only when cells expressing the two split-GFP fragments are in direct contact with each other.(B) Split-GFP and split-Cerulean. The two split-GFP fragments,termed spGFP1-10 and spGFP1,are evolved from the 1-214 and 215-230 fragments of GFP.Five point mutations were made to spGFP1-10 to obtain spCerulean1-10.(For interpretation of the references to colour in this figure legend,the reader is referred to the web version of this article.)

We validated that the pair of split-Cerulean reconstitute into functional proteins emitting cyan fluorescence in HEK 293T cells. We then generated transgenic Drosophila lines carrying the constructs,and showed that the CRASP strategy could be used to detect neuronal contacts between synaptic partners in vivo.Furthermore, we found that the reconstituted CRASP signal could be selectively amplified with immunohistochemistry (IHC).The CRASP system expands the toolkit available to researchers for mapping neuronal connectivity，and opens the possibility for a number of new applications.

# 2.Material and methods

# 2.1.Molecular biology

# 2.1.1.pCAG-mCRASP-spCerulean1-10

We designed spCerulean1-10 by making five point-mutations (Y66W/S72A/F145A/N146I/H148D） to spGFP1-10[4,16].The spCerulean1-1O fragment was chemically synthesized (GenScript, Nanjing）to replace the spGFP1-10 fragment in paavCAG-postmGRASP-2A-dTomato (Addgene # 34912)[11] via the BamHI and Xhol sites, generating pCAG-mCRASP-spCerulean1-10.

# 2.1.2.pCAG-mCRASP-spCerulean11

This construct was modified from paavCAG-pre-mGRASPmCerulean(Addgene # 34910)[11],which contains the spGFP11 fragment. Because spGFP11 is equivalent to spCerulean11 (see Results)，no modification was necessary to the spGFP11 fragment. However,the mCerulean fragment has to be removed to avoid any potential interference of the CRASP signal.This was done by cutting the mCerulean fragment between the BglI and HindIII sites,followed by blunting and ligating the vector.

# 2.1.3.pUAST-CD4-spCerulean1-10

ThespCerulean1-1O fragment wasused to replace the spGFP1- 10 fragment in pUAST-CD4-spGFP1-10 [9] between the Nhel and Sall sites.

# 2.2. Fly genetics

Transgenic fliescarryingUAS-CD4-spCerulean1-10were generated by embryo injection of pUAST-CD4-spCerulean1-10 following standard procedures. Transgenic lines with strong expression level and minimal leaky expression were chosen for the subsequent experiments.

Flies were reared on standard medium at $2 5 ^ { \circ } \mathsf C .$ Fly stocks used   
include: GH146-GAL4 (II), 247-LexA (II), UAS-CD4-spCerulean1-10   
(III),LexAop-CD4-spGFP11 (II)，OK107-GAL4 (IV),GH146-QF (II),   
QUAS-CD4-spGFP11 (II). Fly genotype for Fig.3A were (from left to right): GH146-GAL4/LexAop-CD4-spGFP11; 247-LeXA/UAS-CD4-   
spCerulean1-10 GH146-QF/QUAS-CD4-spGFP11；UAS-CD4-spCerulean1- $\cdot 1 0 / +$ ：   
OK107-GAL4 $I +$ GH146-GAL4 $I _ { + }$ ; 247-LexA/UAS-CD4-spCerulean1-10 GH146-GAL4/LexAop-CD4-spGFP11;247-LexA/+ Fly genotype for Fig.3B were (from left to right): GH146-GAL4/LexAop-CD4-spGFP11; 247-LexA/UAS-CD4-   
spCerulean1-10 GH146-GAL4 $^ { \prime } +$ ; 247-LexA/UAS-CD4-spCerulean1-10 GH146-GAL4/LexAop-CD4-spGFP11;247-LexA/+

# 2.3.Cell culture and transfection

Human embryonic kidney(HEK) 293T cells were cultured in DMEM(Life Technologies,11995-065)supplemented with $10 \%$ FBS (Life Technologies,10099-141）at $3 7 ^ { \circ } \mathsf C$ under $5 \% \thinspace { \mathsf { C O } } _ { 2 }$ .Cellswere transfected at $\sim 7 0 \%$ confluence with Lipofectamine 2ooo (Life Technologies）in 6-well plates according to the manufacturer's protocol. Cells were fixed for imaging1 day after transfection.

# 2.4.Immunohistochemistry

HEK 293T cells or Drosophila brains were fixed with $4 \%$ formaldehyde in PBS for $^ { 3 \mathrm { ~ h ~ } }$ on ice and blocked with $10 \%$ normal goat serum(Vector Labs,S-1000) in $0 . 3 \%$ PBS-Triton.Primaryantibodies wererabbit anti-GFP(Invitrogen,A11122）used at1:1000,mouse anti-GFP(Invitrogen,A1112O) and mouse anti-GFP(Sigma,G6539) usedat1:1oo,stained overnight at $4 ^ { \circ } \mathsf C$ Secondaryantibodies were Alexa 568 goat anti-rabbit IgG(Invitrogen,A11o36) and Alexa 568 goat anti-mouse IgG (Invitrogen,A11031), both used at 1:500 for staining overnight at $4 ^ { \circ } \mathsf C$ The specimen were then cleared with SlowFade Gold (Invitrogen,S36936） and mounted for confocal imaging.

# 2.5. Confocal imaging

A Nikon FN1 confocal microscope witha $2 5 \times$ $\mathsf { N A } = 1 . 1$ ora $4 0 \times$ $\mathsf { N A } = 0 . 8$ objective was used for imaging.For imaging the CFP fluorescence,we used $4 4 0 \mathrm { n m }$ excitation laser light in combination with a 45O/50 emission filter.For imaging the GFP fluorescence,we used $4 8 8 ~ \mathrm { n m }$ excitation laser light in combination with a 525/50 emission flter.Images were acquired at the resolution of $5 1 2 \times 5 1 2$ or $1 0 2 4 \times 1 0 2 4$ ,with pixel dwell time adjusted to produce images with optimal signal-to-noise ratio.

# 3.Results

# 3.1.Design ofthe CRASP constructs

We chose to develop a cyan-colored version of GRASP,because CFPs not only share close sequence homology with GFP,but also havea distinct spectral profile that could be easily separated from that of GFP and other fluorescent proteins [17].We based our design on Cerulean,a bright cyan fluorescent protein that was evolved from ECFP[15]. Incorporating the Cerulean mutations into the corresponding locations of split-GFP [4,16],we constructed spCerulean1-10 with five point mutations (Y66W/S72A/F145A/ N146I/H148D) from spGFP1-10 (Fig.1B).The spGFP11 fragment was used as spCerulean11 with no modification(and referred to interchangeably in this work)，because the peptide sequence was identical between GFP and Cerulean in this part.

![](images/a1690aa055a0c96defa4dac9cbf4713f6262815d3e2b1128e8b0d2a4d745762b.jpg)  
gl expressedindividualy. (B)ReconstitutedCeruleanshoweddistinctspectral profilewencomparedwithreconstitutedGFP.Scalebar $1 0 0 \mu \mathrm { m } .$ (For interpretation of the references to colour in this figure legend,the reader is referred to the web version of this article.)

# 3.2.Validationof theCRASPsysteminHEK293Tcells

We first tested whether functional Cerulean proteins could be reconstituted from spCerulean1-10 and spCerulean11 by expressing them in HEK293Tcells.We specifically tethered the spCerulean fragments to the extracellular surface of the cells using a backbone previouslyused for mammalian GRASP(see Material and methods) [11].When we transfected cells with only spCerulean1-10 (pCAGmCRASP-spCerulean1-1O)，or with only spCerulean11 (pCAGmCRASP-spCerulean11)，no cyan fluorescence could observed. However,cyan fluorescence was observed when the two constructs were co-expressed,indicating that functional Cerulean protein was successfully reconstituted from the two fragments (Fig.2A).

Furthermore,when these cells were imaged using the filter set for GFP,no fluorescence could be observed,demonstrating that the excitation and emission spectrum of the CRASP system was clearly distinct from that of the GRASP system (Fig.2B).

# 3.3.Application of the CRASP systemto the Drosophila nervous system

We then tested whether the CRASP system could be used to detect neuronal contacts in vivo.In Drosophila,the connections between the projection neurons(PNs) of the antennal lobe and the Kenyon cells(KCs) of the mushroom body have been widely studied asa model system[18-21].The axons of PNs made synaptic contactswith the dendrites of KCs only at the calyx area of the mushroom body.Therefore,if the two spCerulean fragments are expressed separately at PNs and KCs,cyan fluorescence should be observedat the calyxarea only.

To express the spCerulean proteins on the extracellular side of the membrane,we tethered them to the extracellular domain of human CD4 [4,9].We generated transgenic flies carrying UAS-CD4- spCerulean1-1O,and directed its expression in PNs.Simultaneously, we directed the expression of LexAop-CD4-spCerulean11 in KCs. Consistent to expectation,cyan fluorescence was observed only from the calyx area of the mushroom body (Fig.3A). Similar results were obtained when the two fragments were expressed in a reverse manner,with spCerulean1-1O expressed in KCs and spCerulean11 in PNs (Fig. 3A).As a control,when either the spCerulean1-1O or the spCerulean11 fragment was not expressed, no cyan fluorescence was observed (Fig.3A). Collectively, these results demonstrate that the CRASP system could be used to detect neuronal contacts in intact neuronal circuits in Drosophila

![](images/1ebdb1913dece14c55e10938056c675604ac267eb1c454af8331c4fa59f56563.jpg)  
FigApli Ceruaasepiptle $3 0 ~  { \mu \mathrm { m } }$ (B)CRASPsignalcouldbeselectivelyamplifiedwiththe monoclonalantibodyGFP-2butnotwith twootherantibodies.Scalebar $1 0 0 ~ { \mu \mathrm { m } }$ .(For interpretation of the references to colour in this figure legend,the reader is referred to the web version of this article.)

# 3.4.The CRASP signal couldbe selectivelyamplifiedwith IHC

A major problem facing the adoption of GRASP-like strategy is that the reconstituted fluorescence is often weak and somewhat difficult to detect. Although IHC could be potentially used to amplifythe signal,high structural similarity between the spCerulean1-1O fragment and the reconstituted Cerulean necessitates careful screening of the antibodies to be used.It has been previously reported that the monoclonal antibody GFP-2O (Sigma, G6539） selectively recognize the reconstituted GFP and not the split-GFP fragments [9].We therefore tested this monoclonal antibody，together with two other antibodies，to determine whetherthe CRASP signal could be selectivelyamplified.We found that the monoclonal antibody GFP-2O selectively amplified the reconstituted CRASP signal,while another monoclonal antibody 3E6（A-1112o,Invitrogen）and a polyclonal antibody(A-11122, Invitrogen) amplified both the reconstituted CRASP signal and the spCerulean1-1O signal (Fig.3B). Therefore,the CRASP signal could beselectively amplified using the GFP-2O antibody following routine IHC methods.

# 4.Discussion

In this study we demonstrated CRASP,a cyan-colored version of GRASP for the study of connectivity between neurons.We showed that reconstituted cyan fluorescence could be observed from the pair of split-Cerulean proteins,both in HEK 293T cells and in Drosophila brains.Inaddition,the CRASP signal could be selectively amplified with routine IHC methods,further increasing the sensitivity of the system.

It should be possible to generate additional color variants of the GRASP system using strategies similar to this work.Yellow fluorescent proteins （YFP） are obvious candidates,although the potential spectral overlap between YFP and GFP is greater than that between CFP and GFP.In addition,a number of red fluorescent proteins and even the photo-switchable protein Dronpa have been reported to be able to reconstitute in a similar manner to split-GFP [22].However,these constructs were developed for intracellular reconstitution,and further optimization for reconstitution in an extracellular membrane-tethered environment is probably necessary for them [4,16].

With the CRASP system,new potential applications that were previously difficult or impossible for GRASP emerge.For example, CRASP maybe combined with GRASP to study the synaptic contacts between more than two groups of neurons.As another example, thewidely used genetically encoded calcium indicators G-CaMP [14] or R-GEcO[2O,23] may now be used simultaneously with the CRASP system,to selectively detect the calcium signals originating from synaptic contacts.

# Acknowledgments

We thank Qian Hu,Qianru Ma and Ying Yang of the Optical Imaging Facility at the Institute of Neuroscience for support with optical imaging;Core Facility of Drosophila Resource and Technique at the Institute of Biochemistryand Cell Biology for support with the generation of transgenic Drosophila lines;Can Yang for help with cell culture and Muming Poo for providing cell culture equipment;Kristin Scott and Nannan Chen for the CD4-spGFP flies; Bloomington Stock Center for providing fly stocks;and Jinhyun Kim for the mGRASP constructs (Addgene plasmid # 34910 and 34912). This work was supported by grants from the 973 Program (2011CBAoo4oO to A.G.),the Natural Science Foundation of China (30921064，90820008，and 31130027 to A.G.）and the Strategic

Priority Research Program of the Chinese Academy of Sciences (XDB02040100 t0 A.G.).

# Appendix A. Supplementary data

Supplementary data related to this article can be found athttp:// dx.doi.0rg/10.1016/j.bbrc.2015.12.011.

# References

[1] S.Seung, Connectome :How the Brain's Wiring Makes Us Who We Are, Houghton Mifflin Harcourt, Boston, 2012. [2] J.G.White, E. Southgate, J.N. Thomson, S. Brenner, The structure of the nervous system of the nematode Caenorhabditis elegans, philosophical transactions of the royal society of London, Ser. B Biol. Sci. 314 (1986), 1-340.   
[3] M. Helmstaedter, Cellular-resolution connectomics: challenges of dense neural circuit reconstruction, Nat. methods 10(2013) 501-507. [4] E.H. Feinberg,M.K.Vanhoven,A. Bendesky,G. Wang,R.D. Fetter,K. Shen, C.I. Bargmann, GFP reconstitution across synaptic partners (GRASP) defines cell contacts and synapses in living nervous systems,Neuron 57 (2008) 353-363.   
[5] I. Ghosh, A.D. Hamilton, L. Regan, Antiparallel leucine zipper-directed protein reassembly:application to the green fluorescent protein, J. Am. Chem. Soc. 122 (2000) 5658-5659.   
[6] S. Druckmann, L. Feng, B. Lee, C. Yook, T. Zhao, J.C. Magee, J. Kim, Structured synaptic connectivity between hippocampal regions， Neuron 81 (2014) 629-640.   
[7] P.Fan, D.S. Manoli, O.M. Ahmed, Y. Chen, N. Agarwal, S. Kwong, A.G. Cai, J. Neitz, A. Renslo, B.S. Baker, N.M. Shah, Genetic and neural mechanisms that inhibit Drosophila from mating with other species, Cell154 (2013) 89-102.   
[8] D.D. Frank, G.C. Jouandet, P.J. Kearney, L.J. Macpherson, M. Gallio,Temperature representation in the Drosophila brain, Nature 519 (2015) 358-361.   
[9] M.D. Gordon, K. Scott, Motor control in a Drosophila taste circuit, Neuron 61 (2009) 373-384.   
[10] T. Karuppudurai, T.Y. Lin, C.Y. Ting，R. Pursley， K.V. Melnattur, F. Diao, B.H. White,L.J. Macpherson,M. Gallio, T. Pohida, C.H. Lee, A hard-wired glutamatergic circuit pools and relays UV signals to mediate spectral preference in Drosophila, Neuron 81 (2014) 603-615.   
[11] J. Kim, T. Zhao,R.S. Petralia, Y. Yu, H. Peng, E. Myers, J.C. Magee,mGRASP enables mapping mammalian_synaptic connectivity with light microscopy, Nat. methods 9 (2012) 96-102.   
[12] M. Yamagata, J.R. Sanes, Transgenic strategy for identifying synaptic connections in mice by fluorescence complementation (GRASP), Front. Mol. Neurosci. 5 (2012) 18.   
[13] Z. Zhang, X. Li, J. Guo, Y. Li, A. Guo, Two clusters of GABAergic ellipsoid body neurons modulate olfactory labile memory in Drosophila, J. Neurosci. Off. J. Soc. Neur0sci. 33 (2013) 5175-5181.   
[14] T.W. Chen, T.J. Wardill, Y. Sun， S.R. Pulver，S.L. Renninger, A. Baohan, E.R. Schreiter, R.A. Kerr, M.B. Orger, V. Jayaraman,LL. Looger,K. Svoboda, D.S. Kim, Ultrasensitive fluorescent proteins for imaging neuronal activity, Nature 499 (2013) 295-300.   
[15] M.A. Rizzo, G.H. Springer, B. Granada, D.W. Piston,An improved cyan fluorescent protein variant useful for FRET, Nat. Biotechnol. 22 (2004) 445-449.   
[16] S. Cabantous, T.C. Terwilliger, G.S. Waldo, Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein, Nat. Biotechnol. 23 (2005) 102-107.   
[17] R.Y. Tsien,The green fluorescent protein, Annu. Rev. Biochem. 67 (1998) 509-544.   
[18] S.J. Caron, V. Ruta,L.F. Abbot, R. Axel, Random convergence of olfactory inputs in the Drosophila mushroom body, Nature 497 (2013) 113-117.   
[19] G.S. Jefferis,E.C. Marin, R.F. Stocker,L. Luo, Target neuron prespecification in the olfactory map of Drosophila, Nature 414 (2001) 204-208.   
[20] H. Li, Y. Li, Z. Lei, K. Wang, A. Guo, Transformation of odor selectivity from projection neurons to single mushroom body neurons mapped with dualcolor calcium imaging, Proc. Natl. Acad. Sci. U. S. A. 110 (2013) 12084-12089.   
[21] A.M. Wong, J.W. Wang, R. Axel, Spatial representation of the glomerular map in the Drosophila protocerebrum, Cell 109 (2002) 229-241.   
[22] Y. Kodama, C.D. Hu, Bimolecular fluorescence complementation (BiFC): a 5- year update and future perspectives, BioTechniques 53 (2012) 285-298.   
[23] Y. Zhao, S. Araki, J. Wu, T. Teramoto, Y.F. Chang, M. Nakano, A.S. Abdelfattah, M. Fujiwara, T. Ishihara,T. Nagai, R.E. Campbell, An expanded palette of genetically encoded ${ \mathsf { C a } } ( 2 ) ( + )$ indicators, Science 333 (2011) 1888-1891.