# Crystal structure of cyclic nucleotide-binding-like protein from Brucella abortus

Zheng He a,b, Yuan Gao a, Jing Dong C, Yuehua Ked, Xuemei Lia, Zeliang Chen d, Xuejun C. Zhang a,\*

Natiolbo   
Beijing,100101, China   
b University of Chinese AcademyofSciences,Beijing,10o049,China   
C Yangtze River Fisheris Research Institute,Chinese Academy ofFishery Sciences,Wuhan,430224, China   
dInstituteofseaseotroldevenioncdemofitaryedicalienceeingo

# ARTICLEINFO

# ABSTRACT

Article history: Received 28 October 2015 Accepted 2 November 2015 Availableonline6November 2015

Keywords:   
Brucella abortus   
Cyclic nucleotide binding(CNB) domain   
Crystal structure

The cyclic nucleotide-binding(CNB)-like protein (CNB-L） from Brucella abortus shares sequence homology with CNB domain-containing proteins.We determined the crystal structure of CNB-L at $2 . 0 \mathrm { ~ \AA ~ }$ resolution in the absence of its C-terminal helix and nucleotide.The 3D structure of CNB-L is in a twofold symmetric form.Each protomer shows high structure similarity to that of cGMP-binding domaincontaining proteins,and likely mimics their nucleotide-free conformation.A key residue,Glu17,mediates the dimerization and prevents binding of cNMP to the canonical ligand-pocket.The structurally observed dimer of CNB-L is stable in solution,and thus is likely to be biologically relevant.

$\circledcirc$ 2015 Elsevier Inc.All rights reserved.

# 1.Introduction

Brucella abortus isa Gram-negative bacterium found in cattle populations[1] and is the cause of brucellosis which often leads to the premature abortion of cattle fetus.Its potential infection to humans is a serious health concern in ranch areas [2].Therefore, significant efforts have been made for the development of new therapeutics against B.abortus.Cyclic nucleotide monophosphate (cNMP)-binding proteins are one of such potential drug targets due to their essential roles in many cellular functions[3,4].Some cNMP analogs have beenused to study the functions of cyclic nucleotide binding(CNB） proteins and have showed promising application prospect [5-7].

The CNB-like（CNB-L） protein (GenBank ID:AE017223.1） identified from B.abortus is a putative cNMP-binding protein for its conserved CNB domain.In both prokaryotes and eukaryotes,CNB domains are the critical components of cellular machineries that regulate multiple inner cellular processes [8].For examples, these protein domains have been found in several protein families,such as cAMP receptor proteins (CRP)，cAMP- and cGMP-dependent protein kinases (e.g. PKA and PKG) [9],and the ether-a-go-go (EAG) channel [1O].CNB domains are responsible for the binding of $3 ^ { \prime }$ = $5 ^ { \prime }$ -cyclic adenosine monophosphate(cAMP) and $3 ^ { \prime }$ = $5 ^ { \prime }$ -cyclic guanosine monophosphate (cGMP)， two universal secondary messengers.Generally,a CNB domain consists of approximately 120 amino acid residues,formingan eight-stranded antiparallel ${ \mathfrak { \beta } }$ 1 barrel which accommodates the cyclic nucleotide molecule and is flankedbyavariablenumberof $\mathfrak { a }$ -helices[11].A signaturemotif referred to as the phosphate binding cassette(PBC) is situated between β6 and $\boldsymbol { \beta 7 }$ strands [12].Upon ligand binding,the flexible $\mathfrak { a }$ 1 helix subdomain usually undergoes a dramatic structural transformation to deliver the allosteric signal [13].

In the current work,we report the crystal structure of B.abortus CNB-L protein.It is in a homo-dimeric form,resembling the nucleotide-free form of PKG CNB domains.However, the nucleotide binding site is blocked by a residue in the dimer interface,suggesting that B. abortus CNB-L dimer does not function as a signaling molecule.

# 2.Material and methods

# 2.1. Cloning

The target gene of CNB-L was amplified from B.abortus bv.1 strain 9-941 by the polymerase chain reaction (PCR)with forward primer $5 ^ { \prime }$ -CGGAA TTCAT GGCGC TAG- ${ \cdot } 3 ^ { \prime }$ and reverse primer $5 ^ { \prime }$ 1 CCGCT CGAGT CAGTC GCGGT TT- ${ \cdot } 3 ^ { \prime }$ .The PCR product was digested with EcoRI and Xhol and subsequently cloned into the expression vector pGEX-6P-1(Qiagen, USA).The construct Was verified by DNA sequencing and then transformed into E.coli strain BL21(DE3) cell for protein expression.

# 2.2.Overexpression and purification

The seleno-methionine (SeMet） protein was overexpressed in E. coli BL21(DE3） strain at $1 6 ~ ^ { \circ } \mathsf { C }$ for $^ { 1 2 \mathrm { ~ h ~ } }$ ，containing $1 0 0 \mathrm { - } \mu \mathrm { g } / \mathrm { m l }$ ampicillin.The cell was induced with $0 . 5 ~ \mathrm { m M }$ IPTG when OD600 reached O.8 and was grown further for $^ { 1 8 \mathrm { ~ h ~ } }$ at $1 6 ^ { \circ } \mathsf C .$ The cells were harvested by centrifugation at $4 0 0 0 \ { \mathrm { g } }$ for $1 5 { \mathrm { ~ m i n } }$ .Then,the cell pellets were resuspended in buffer A( $2 0 ~ \mathrm { m M }$ Tris-HCl $\left( \mathsf { p H 8 . 0 } \right)$ 1 $4 0 0 \ \mathrm { m M \ N a C l }$ and $1 0 \%$ glycerol） and lysed using high pressure breaking method (EF-C3,from Avestin) at15,ooO psi.The lysate was centrifugedat $2 2 { , } 0 0 0 { \mathrm { g } }$ for $3 0 \mathrm { m i n }$ at $4 ^ { \circ } C$ to remove unbroken cells and debris,and the supernatant was collected.A slurry of Glutathione Sepharose 4B(GE Healthcare) pre-equilibrated with buffer A was added into the supernatant and agitated by end-over-end rotation for $^ \mathrm { ~ 1 ~ h ~ }$ at $4 ~ ^ { \circ } C .$ Then the mixture was transferred to a $1 0 \ \mathrm { m l }$ column and washed extensively with buffer A to remove contaminant proteins.To remove the GST tag，the slurry was incubated with Prescission protease in buffer A at an enzyme:- protein ratio of $1 { : } 2 0 \ ( \mathrm { w / w } )$ for $^ { 4 \mathrm { ~ h ~ } }$ at $4 ^ { \circ } C$ .The flow-through containing the target protein was collected and subjected to sizeexclusion chromatography (SEC)(Superdex 2oo,GE Healthcare) inbufferB( $2 0 ~ \mathrm { m M }$ Tris-HCl( $\mathrm { \tt { p H 8 . 0 } } $ 一， $1 0 0 \ \mathrm { m M \ N a C l } ,$ and $5 \%$ glycerol).Peak fractions were collected (monitored by $2 8 0 \ \mathrm { n m }$ absorption) and concentrated to ${ \sim } 2 0 ~ \mathrm { m g / m l }$ for crystallization.Purity of the protein sample was estimated to be $9 8 \%$ by using SDS-PAGE.

# 2.3.Crystallization

Initial crystallization experiments were carried out using commercial crystallization screening kits from Hampton Research (US) and sitting drop method.Crystals appeared in 4-6 weeks under multiple conditions.Hanging-drop vapor-diffusion method was further used to optimize crystallization conditions by mixing $1 ~ { \mu \mathrm { l } }$ protein solution with $1 ~ \mu \mathrm { l }$ reservoir solution and equilibrating against $2 0 0 \mathrm { - } \mu \mathrm { l }$ reservoir solution at $2 0 ^ { \circ } \mathsf C$ The crystals used for data collection were grown in $0 . 2 \mathrm { M }$ potassium sodium tartrate $\left( \mathsf { p H 7 . 4 } \right)$ $0 . 1 \mathrm { ~ M ~ }$ KCl, and $2 0 \% \mathrm { ( w / v ) }$ polyethylene glycol 3350.Crystals grew to full size ( $1 5 0 \times 1 5 0 \times 2 0 0 \mu \mathrm { m } ^ { 3 } )$ in $3 0 { \ - } 4 0 \mathrm { d }$

# 2.4.Data collection and structure determination

The crystals were cryo-protected with $2 0 \% \ ( \mathrm { v / v ) }$ glycerol in reservoir solution and flash-cooled in a liquid nitrogen stream at $1 0 0 ~ \mathrm { K } .$ The single-wavelength anomalous diffraction (SAD） data were collected at beamline BL17U of Shanghai Synchrotron Radiation Facility(SSRF) with the ADSC Quantum 315r CCD detector.Xray diffraction data were collected from single crystals over a range of $3 6 0 ^ { \circ }$ with an oscillation angle of $1 ^ { \circ }$ at the wavelength of 0.9762A.The data were processed and scaled with the program package of HKL2ooo [14]. The initial phase calculation,density modification,and partial model building were carried out with Phenix.autosolve [15].The structure was completed manually with

COOT[16],and the refinement was carried out with Phenix.refine [15]. Statistics of data collection and refinement are summarized in Table 1. All the structure figures were prepared with Pymol [17].

# 2.5.Truncation and site-directed mutagenesis

A plasmid to express a truncated form of CNB-L (tCNB-L) was constructed by removing nucleotides coding for the last 31 amino acid residues from the C-terminus of the full-length CNB-L(flCNBL)[18].Then,based on tCNB-L a site-directed mutation of Glu17 into tryptophan (tCNB-L/E17W)was constructed.Both tCNB-Land tCNB-L/E17W were over-expressed and purified with the same method as flCNB-L.The purified protein samples were analyzed using SEC.

# 2.6.Isothermal titration calorimetry

ITC experiments were performed at $2 5 ^ { \circ } \mathsf { C }$ using the iTC200 (MicroCal,Northampton,MA,USA).Protein and cyclic nucleotides were prepared in the same buffer,i.e.ITC buffer containing $1 0 ~ \mathrm { m M }$ Tris $\left( \mathsf { p H } 8 . 0 \right)$ and $1 0 0 \ \mathrm { m M \ N a C l }$ 、The protein was placed in the sample cell at a concentration of $0 . 4 ~ \mathrm { m M }$ in the ITC buffer.Cyclic nucleotides were placed in the injection syringe at a concentration of $1 0 ~ \mathrm { m M }$ .The injection volume was $2 \mu \mathrm { l }$ and the interval between injections was $2 \ \mathrm { m i n }$ .All data were corrected using the heat changes arising from injection of the cyclic nucleotides into the buffer.Data were processed using the Origin software with a manufacturer-supplied custom-add-on ITC sub-routine.The reported results were repeated in at least duplicate.

# 3.Results

# 3.1.Overall structure of CNB-L

The structure of CNB-L was resolved at $2 . 0 \mathrm { \AA }$ resolution in space group of I222 using merged SAD data collected from one crystal, and was refined with the same data set(Table 1).As predicted from the sequence homology (Fig.1),the CNB-L structure contains a typical CNB-folding and forms homo-dimers.In the following,we will use the PGK $\mathrm { I } \beta$ CNB nomenclature [19] to describe the secondary structural elements of CNB-L.Each asymmetric unit of the crystal contains two independent CNB-L molecules,termed Aand B,which forma dual-symmetrical dimer. In each protomer, CNB-L containsaneight-stranded antiparallel $\beta$ -barrel(β1-8),whichis preceded by helices αX:N and αA at the N-terminus and followed by $\mathsf { \alpha } \mathsf { a } \mathsf { B }$ at the C-terminus.The signature motif PBC of the CNB folding is comprised of a short helix αP andaloop,and islocated between strands $\beta 6$ and $\boldsymbol { \beta 7 }$ (Fig.2A).At the C-terminal end of each CNB-L molecule,approximately 3O amino acid residues were invisible in the electron density map.SDS-PAGE analysis of the crystal sample confirmed that these residues were missing,probably because of degradation. Interestingly,an unusual cis-peptide between Ile73 and Arg74,which is a signature feature of a cGMP binding pocket in CNB domains[19],remains in both protomers of the CNB-L dimer. In addition,two short anti-parallel $\beta$ strands（βs1 and βs2）in molecule Bare stabilized by four hydrogen bonds (Fig.S1),while in molecule A only two hydrogen bonds were observed in the corresponding region. The $\beta s 2$ strand is located near the loop region of PBC.

Table 1 Statistics of data collection and refinement.   

<html><body><table><tr><td>Data</td><td>Se (peak)</td></tr><tr><td>Data Processing</td><td></td></tr><tr><td>Wavelength (A)</td><td>0.9762</td></tr><tr><td>Space group</td><td>[222</td></tr><tr><td>Cell dimensions a,b and c (A)</td><td>56.1,82.9,103.9</td></tr><tr><td>Resolution (A)</td><td>34.6-2.0 (2.1-2.0)a</td></tr><tr><td>Completeness (%)</td><td>98.8 (100)</td></tr><tr><td>Rmerge (%)b</td><td>8.5 (38.1)</td></tr><tr><td>I/σ(1)</td><td>45.1 (3.0)</td></tr><tr><td>Unique reflections</td><td>16,773 (1519)</td></tr><tr><td>Redundancy</td><td>20 (15)</td></tr><tr><td>Refinement</td><td></td></tr><tr><td>Resolution (A)</td><td>24.6-2.0</td></tr><tr><td>No.of reflections (test)</td><td>16,665 (833)</td></tr><tr><td>Rwork/Rfree (%)</td><td>18.5/23.3</td></tr><tr><td>Number of atoms</td><td>1798</td></tr><tr><td>Water</td><td>171</td></tr><tr><td>Average B factor (A²)</td><td>28</td></tr><tr><td>Water</td><td>44</td></tr><tr><td>Root-mean-square-deviation</td><td></td></tr><tr><td>Bond lengths (A)</td><td>0.007</td></tr><tr><td>Bond angles (°)</td><td>0.73</td></tr><tr><td>Ramachandran plot (%)d</td><td></td></tr><tr><td>Favored region</td><td>99.13</td></tr><tr><td>Allowed region</td><td>0.87</td></tr></table></body></html>

a Values in parentheses are for the highest resolution shell. b $\mathrm { R } _ { \mathrm { m e r g e } } = \sum _ { h l k } \sum _ { i } | I _ { i } - ( I ) | / | ( I ) |$ where $I _ { i }$ istheintesioeat ofanequivalent reflection with indices h,k and 1. c $\begin{array} { r } { \mathbb { R } = \sum | F _ { o } - F _ { c } | / \sum | F _ { o } | } \end{array}$ ，where $F _ { o }$ and $F _ { c }$ are the observed and calculated structure factors,respectively. d Calculated using MolProbity.

# 3.2.CNB-L exists asα stable dimerin solution

The CNB-L dimer appears to be distinctive from that of other known structures of CNB proteins.This difference is probably due to the absence of the C-terminal 3O residues in CNB-L,which are supposed to form two $\boldsymbol { \mathfrak { a } }$ -helixes,αC and αS (Fig.1).Despite the lack of αC,the interface between the two CNB-L subunits is extensive, with a buried area of ${ \sim } 1 9 0 0 \mathrm { ~ \AA ~ } ^ { 2 }$ . This interface is mediated by a network of hydrogen bondsas well as hydrophobic interactions (Fig.1). In many available structures of CNB domain-containing proteins (e.g.bacterial CRP,PDB ID:3MZH)，αC is the core elementthatboth mediates the dimerizationand contributes to the formation of the binding pocket of cAMP [20-22].In addition,in eukaryotes (e.g.Plasmodium falciparum PKG)，αC is observed to serve as an allosteric switch that triggers kinase activation upon cGMP binding[23].Due to the dimer-interface difference,the CNBL dimer is ina head-to-tail configuration rather than the side-byside one as observed in a typical CNB dimer (Fig.S2). Notably, in each CNB-L protomer the canonical cNMP binding pocket is sealed up by the side chain of Glu17 from its symmetry mate (Fig. 3).

Toverify the biological relevance of the structurally observed dimer interface,a site-directed mutation of Glu17 to tryptophan, termed as tCNB-L/E17W,was made based ona C-terminal truncated form of CNB-L.Here,we refer to the variant of C-terminal 30- residue truncation as tCNB-L,while the full-length WT CNB-L is referred to as flCNB-L.All the three proteins,flCNB-L,tCNB-L and tCNB-L/E17W were purified and subjected to SEC under the same condition.The result showed that the flCNB-L ran the fastest closely followed by tCNB-L,and the tCNB-L/E17W sample ran the slowest (Fig.S3). It indicates that both flCNB-L and tCNB-L form stable homo-dimers,while the tCNB-L/E17W sample stays as monomers in solution.Furthermore, the purified tCNB-L sample was able to be crystallized under the same condition as flCNB-Lwithin $4 8 \mathrm { h }$ (much faster than that required for flCNB-L),and the crystal morphology remained the same.This observation supports that it was the truncated CNB-L sample that crystallized in our original crystals. Taken together,the structurally observed dimer is likely to be stable in solution rather than being a crystallization artifact.

# 3.3.Testing of cGMPbindingto CNB-L

Despite the high similarity in the protomer structure of CNB-L compared with other CNB domain-containing proteins (Fig. 4A), neither cAMP nor cGMP showed a reliable binding signal with purified flCNB-L,tCNB-L and tCNB/E17W in our ITC experiments. However,tCNB-L/E17W showed an assessable binding signal with cGMP(Fig.S4),suggesting that CNB-L may bind cGMP weakly in its monomeric form (Table S1). In consistence,co-crystallization of fICNB-L with either cAMP orcGMP did not show bound nucleotides. While the blockage of the ligand binding site by Glu17 explains the lack of binding in both flCNB-L and tCNB-L,the absence of αC in the dimer interface and the instabilityas monomer maybe partially account for the reduced bindingaffinity of tCNB-L/E17W compared to other CNB domains.

# 4.Discussion

The structure basis of cyclic nuclide binding specificity has been illustrated in the CNB domains of proteins,PKG,PKA,and CRP. In general,all the CNB domains share a similar overall folding across various species,and three main binding sites are responsible for the cyclic nucleotide binding:The site-1 is composed of two highly conserved residues glutamate and alanine,and functions in the same bindingmode forboth cAMPand cGMP.The site-2 and site-3 can bind cGMP specifically in the so-called syn conformation [19]. InPKG Iβ CNB-A(PDB ID:3ODO),mutations of the Thr193 in site-2 into either alanine or valine resulted in a decrease of the binding specificity of cGMP and a nearly 3O-fold decrease in the $K _ { a }$ value of cGMP in terms of kinase activation.In comparison,substitution

αX:N αAeeeeece eecececceTE β2 β3 β4CNB-L 1MALDDDIRILGTVGLFE .SFTPEQLRLLAFGAERLVLRAGRELFREGQSADCAYIIVTGTITLFHEGD..EGR70  
PKGIβCNB-B222IKHTEYMEFLKSVPTFQ .SLPEEILSKLADVLEETHYENGEYIIRQGARGDTFFIISKGTVNVTRDDSPSEDP293  
PKGIBCNB-A 94VTLPFYPKSPQSKDLIKEAILDNDFMKNLELSQIQEIVDCMYPVEYGKDSCIIKEGDVGSLVYVMEDGKVEVTKEG. · 169αp αB αC αSβ5 6 β7 β8 CCCCCCCCClCCCCCCCCCCCCCCCCCCCCCCCClCCCT： · PBC  
PKGIBCNB-B 294VR MALIAEDTRTTNAYADVAVEVIVISRSIRRLIGGLDDVAALHaHISRNLVELIGRIEOVAPRLAN 153  
PKG Iβ CNB-A 170VKLCTMGPGKVF T ILYNCTRTATVKTLVNVKLWAIDRQCEQTIMMRTGLIKHTEY 227A ★★ ：

![](images/d156acdda2e9a728d632df89d7e5063960e20aa19b08b75b9dc72bd2c12185ff.jpg)  
Fig.2.Overallstructureof CNB-L. (A) Ribbon diagramof CNB-L molecule Awith its secondary elements labeled. $\boldsymbol { \mathfrak { a } }$ -Helicesare colored in pale-green, $\beta$ -barrelsin blue,and the PBC ti to the web version of this article.)

![](images/48bef53a0535346f0917f9c4c5b68754440eca0b0dab34386d7a782d6614a458.jpg)  
Fig.3.CNB-Ldimerinterface.RbbondiagramoftheCNB-Ldimer,consistingofmoleculeA(coloredaccordingsecondarystructures: $\alpha$ -helices in pale-green and $\beta$ -barrels in blue) aoel si version of this article.)

with serine only resulted in a four-fold decrease in the $\mathrm { K _ { a } }$ value [12]. In CNB-L,the position analogous to site-2 is replaced by Leu94 which is followed by Thr95,and we cannot rule out the possibility that Thr95 may replace the role of Thr193 of PKG $\mathrm { I } \beta$ CNB-A during an allosteric process upon the cGMP binding.Moreover, site-3 is composed ofLeu172and Cys173 in PKG $\mathrm { I } \beta$ CNB-A,of Leu296 and Arg297 inPKGIβ CNB-B (PDB ID:4KU7),and of Ile73 and Arg74 in CNB-L.We speculate that Ile73 in CNB-L can make a hydrophobic contact with cGMP similar to Leu296 in PKG $\mathrm { I } \beta$ CNB-B.Importantly, the conserved cis-peptide of site-3 in CNB-L enables both side chains of the adjacent residues to point toward the binding pocket making contacts with the ligand (Fig. 4B).

During evolution,the CNB domain-containing proteins have undergone a great functional divergence by means of residue variations in conserved motifs,specific insertions and deletions [24]. A typical example is the ether-a-go-go(EAG) channel,in which CNB homologous domains bind neither cAMP nor cGMP but respond to changes in membrane voltage,witha short $\beta$ -strand at the end of αC acting as an ‘intrinsic ligand'[1O]. Although the sequence alignment and structural comparison suggest that CNB-L may maintain the ability to bind cGMP,our ITC results showed practically no binding signal of flCNB-L towards eithercGMP or cAMP.It is possible that in CNB-L,the short helix αS(which is missing in the crystal structure）mayalso serve asan ‘intrinsic ligand',like the short $\beta$ -strand in EAG channel,preventing CNB domain from binding cNMP.In the light that all known CNB domains play regulatory roles in some complex proteins,we speculate that the B.abortus CNB-L is a subunit of a protein complex,and its function remains to be elucidated.

![](images/67626db15e20df66f9432419d14218be4863434bab600c9e8ac01913e1c58847.jpg)  
Fig.4. Superposition of CNB-L with PKG Iβ.(A) Structural comparison of CNB-L(blue) with human PKG $\mathrm { I } \beta$ CNB-A (purple)and CNB-B (cyan).Conserved domains are labeled in dote)i ress references to color in this figure legend,the reader is referred to the web version of this article.)

# Protein databank accession code

Coordinates of the crystal structure of CNB-L have been deposited into the Protein Data Bank under the accession code 5D1I.

# Conflictsofinterest

There are no conflicts of interest.

# Acknowledgments

We thank Drs.Bo Huang and Yan Zhao for their help in data processingand structure refinement and Mr.Lei Han for his critical discussion.We also thank the staff of the Protein Research Core

Facilityat the Institute of Biophysics,Chinese Academyof Sciences (CAS),and the staff of SSRF(China) synchrotron facilities for their excellent technical assistance.This work was supported by the National Natural Science Foundation of China (31470745 to XCZ).

# Appendix A. Supplementary data

Supplementary data related to this article can be found athttp:// dx.doi.0rg/10.1016/j.bbrc.2015.11.005.

# References

[1] P.G. Detilleux, B.L. Deyoe,N.F. Cheville,Penetration and intracellular growth of Brucella-abortus in nonphagocytic cells-invitro, Infect. Immun. 58 (1990) 2320-2328.   
[2]D.A. Ashford, J. di Pietra, J. Lingappa, C.Woods,H. Noll, B. Neville, R. Weyant, S.L.Bragg,R.A. Spiegel, J. Tappero,B.A. Perkins,Adverse events in humans associated with accidental exposure to the livestock brucellosis vaccine RB51, Vaccine 22 (2004) 3435-3439.   
[3] Y.S. Chochung,T. Clair,P. Tagliaferri, S. Ally, D. Katsaros,G.Tortora,L. Neckers, T.L.Avery, G.W. Crabtree，R.K. Robins, Site-selective cyclic-amp analogsas new biological tools in growth-control, differentiation,and proto-oncogene regulation, Cancer Investig.7(1989)161-177.   
[4] A.Merckx, G. Bouyer, S.L.Y. Thomas,G. Langsley, S. Egee,Anion channels in Plasmodium-falciparum-infected erythrocytes and protein kinase A, Trends Parasitol. 25 (2009)139-144.   
[5] JM. Enserink,A.E. Christensen,J. de Rooij，M. van Triest,F. Schwede, H.G. Genieser, S.O. Doskeland, J.L. Blank, J.L. Bos, A novel Epac-specific cAMP analogue demonstrates independent regulation of Rap1 and ERK, Nat. Cell Biol. 4(2002) 901-906.   
[6] G.X. Kang, J.W. Joseph, O.G. Chepurny,M. Monaco,M.B.Wheeler, J.L. Bos, F. Schwede,H.G. Genieser, G.G. Holz, Epac-selective cAMP analog 8-pCPT-2'- O-Me-cAMP as a stimulus for $\mathsf { C a 2 + }$ -induced $\mathsf { C a 2 + }$ release and exocytosis in pancreatic beta-cells, J. Biol. Chem. 278 (2003) 8279-8285.   
[7]M.Almahariq,T.Tsalkova,F.C.Mei,H.J. Chen,J. Zhou, S.K. Sastry,F. Schwede, X.D. Cheng, A novel EPAC-specific inhibitor suppresses pancreatic cancer cell migration and invasion, Mol. Pharmacol. 83 (2013) 122-128.   
[8]N.Kalman,J.Wu,S.Yooseph, A.F. Neuwald,J.C. Venter, S.S.Taylor, Evolution of allostery in the cyclic nucleotide binding module:a comparative genomics study, Faseb J.22 (2008).   
[9] J.B. Shabb, J.D. Corbin, Cyclic nucleotide-binding domains in proteins having diverse functions, J. Biol. Chem. 267(1992) 5723-5726.   
[10] T.I. Brelidze, A.E. Carlson, B.Sankaran,W.N. Zagotta, Structure of the carboxyterminal region of a KCNH channel, Nature 481 (2012), 530-U147.   
[11]H. Rehmann, A. Wittinghofer, J.L. Bos, Capturing cyclic nucleotides in action: snapshots from crystallographic studies,Nat. Rev. Mol. Cell Biol. 8 (2007) 63-73.   
[12] JJ.Kim,D.E. Casteel,G.Huang,T.H. Kwon,R.K.Ren,P. Zwart, J.J. Headd, N.G. Brown,D.C. Chow,T.Palzkill, C. Kim, Co-crystal structures of PKG Ibeta (92-227）with cGMP and cAMP reveal the molecular details of cyclicnucleotide binding,Plos One 6 (2011) e18413.   
[13] C. Kim, N.H. Xuong, S.S. Taylor, Crystal structure of a complex between the catalytic and regulatory (Rl alpha） subunits of PKA,Science 307 (2005) 690-696.   
[14] Z. Otwinowski,W.Minor, Processing of X-ray diffraction data collected in oscillation mode,MethodsEnzymol. 276(1997) 307-326.   
[15] P.D.Adams,P.V. Afonine,G.Bunkoczi,V.B. Chen,LW.Davis,N. Echols, J.J.Headd，LW. Hung，G.J. Kapral， R.W. Grosse-Kunstleve，A.J. McCoy, N.W.Moriarty，R. Oeffner，R.J.Read，D.C. Richardson，J.S. Richardson, T.C. Terwilliger,P.H. Zwart, PHENIX:a comprehensive python-based system for macromolecular structure solution，Acta Crystallogr. Sect.D Biol. Crystallogr. 66 (2010) 213-221.   
[16] P. Emsley, K. Cowtan, Coot: model-building tools for molecular graphics,Acta Crystallogr. Sect.D Biol. Crystallogr. 60 (2004) 2126-2132.   
[17] E.H. Baugh, S. Lyskov, B.D.Weitzner,JJ. Gray, Real-time PyMOL visualization forRosetta and PyRosetta, Plos One 6(2011) e21931.   
[18]R.C. Davidson, J.R. Blankenship,P.R. Kraus,M. de Jesus Berrios, C.M. Hull, C.D'Souza,P.Wang, J. Heitman, A PCR-based strategy to generate integrative targeting alleles with large regions of homology，Microbiology 148 (2002) 2607-2615.   
[19] G.Y. Huang, J.J. Kim, A.S.Reger, R. Lorenz, E.W. Moon, C. Zhao,D.E. Casteel, D.Bertinetti, B. VanSchouwen,R. Selvaratnam, J.W. Pflugrath, B. Sankaran, G.Melacini,F.W. Herberg, C. Kim,Structural basis for cyclic-nucleotide selectivity and cGMP-selective activation of PKG I， Structure 22 (2014) 116-124.   
[20] S.Q. An,K.H. Chin,M. Febrer, Y. McCarthy,J.G.Yang, C.L. Liu, D. Swarbreck, J. Rogers,J. Maxwell Dow, S.H. Chou,R.P. Ryan,A cyclic GMP-dependent signalling pathway regulates bacterial phytopathogenesis，EMBO J. 32 (2013) 2430-2438.   
[21]M.C.M. Reddy，S.K. Palaninathan,J.B. Bruning,C. Thurman，D. Smith, J.C.Sacchettini, Structural insights into the mechanism of the allosteric transitions of Mycobacterium tuberculosis cAMP receptor protein, J. Biol. Chem. 284 (2009) 36581-36591.   
[22]H.S.Won,Y.S. Lee,S.H. Lee,B.J. Lee, Structural overview on the allosteric activation of cyclic AMP receptor protein, Bba Proteins Proteom. 1794 (2009) 1299-1308.   
[23]JJ. Kim,C. Flueck,E. Franz,E. Sanabria-Figueroa,E. Thompson,R. Lorenz, D.Bertinetti，D.A. Baker,F.W.Herberg,C. Kim, Crystal structures of the carboxyl cGMP binding domain of the Plasmodium falciparum cGMPdependent protein kinase reveal a novel capping triad crucial for merozoite egress, Plos Pathog.11 (2015).   
[24] S. Mohanty，E.J. Kennedy，F.W.Herberg，R. Hui, S.S. Taylor,G. Langsley, N. Kannan, Structural and evolutionary divergence of cyclic nucleotide binding domains in eukaryotic pathogens: implications for drug design, Biochim. Biophys. Acta 1854 (2015)1575-1585.   
[25]F. Corpet,Multiple sequence alignment with hierarchical-clustering, Nucleic Acids Res.16(1988) 10881-10890.   
[26] X. Robert,P. Gouet, Deciphering key features in protein structures with the new ENDscript server,Nucleic Acids Res. 42 (2014) W320-W324.