# CRISPR/Cas9-Mediated Deletionof Foxn1 in NOD/SCID/IL2rg Mice Results in Severe

# Immunodeficiency

4 Xinru Wei1,2.3, Yunxin Lai2,3, Baiheng Li2.3,Le $\mathrm { Q i n } ^ { 2 , 3 }$ , Youdi $\mathrm { X u } ^ { 1 , 2 , 3 }$ ，Simiao $\operatorname { L i n } ^ { 2 , 3 }$ ，   
5 Suna Wang2,3, Qiting Wu2.3, Qiubin Liang4, Guohua Huang5, Qiuhua Deng5, Pentao   
6 Liu,Donghai Wu2,3,Liangxue Lai2, 3, Yao Yao2,3\*, Peng Li2, 3\*   
，   
8 Affiliations:   
9 lSchool of Life Sciences,University of Science and Technology of China, Hefei,   
10 230027, China;   
11 ²Key Laboratory of Regenerative Biology， South China Institute for Stem Cell   
12 Biology and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health,   
13 Chinese Academy of Sciences, Guangzhou, 510530, China;   
14 3Guangdong Provincial Key Laboratory of Stem Cell and Regenerative Medicine,   
15 South China Institute for Stem Cell Biology and Regenerative Medicine, Guangzhou   
16 Institutes of Biomedicine and Health,Chinese Academy of Sciences,Guangzhou,   
17 510530, China;   
18 4Shenzhen InVivo Biomedicine Co. Ltd, Shenzhen, 518000, China;   
19 5Department of Respiratory medicine, Nanfang Hospital, Southern Medical University,   
20 Guangzhou, 510515, China;   
21 6Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, England, UK.   
22   
23 \* Correspondence: Peng Li, PhD, Guangzhou Institutes of Biomedicine and Health,   
24 Chinese Academy of Sciences，19O Kaiyuan Avenue, Science Park, Guangzhou,   
25 Guangdong, 510530, China; phone: +86-20-32093613; fax: +86-20-32093613, email:   
26 li_peng@gibh.ac.cn; Yao Yao,PhD, Guangzhou Institutes of Biomedicine and Health,   
27 Chinese Academy of Sciences，19O Kaiyuan Avenue，Science Park, Guangzhou,   
28 Guangdong, 510530, China; phone: +86-20-32093616; fax: +86-20-32093616, email:   
29 yao_yao@ gibh.ac.cn.

# Abstract

Immunodeficient mice engrafted with either normal or cancerous human cells are widely used in basic and translational research. In particular, NOD/SCID/IL2rg'/ mice can support the growth of various types of human cancer cells. However, the hairs of these mice interfere with the observation and imaging of engrafted tissues. Therefore, novel hairless strains exhibiting comparable immunodeficiency would be beneficial. Recently, the CRISPR/Cas9 system has been used for efcient multiplexed genome editing. In the present study， we generated a novel strain of nude NOD/SCID/IL2rg-/- (NSIN) mice by knocking out Foxnl using the CRISPR/Cas9 system from $\mathrm { N O D / S C I D / I L 2 r g } ^ { - / - }$ (NSI) mice. NSIN mice were deficient in B, T, and NK cells,which showed impaired T cell reconstitution and thymus regeneration after allogeneic bone marrow nucleated cell transplantation， and exhibited improved capacities of grafting both leukemic and solid tumor cells compared with NSI, NOG and NDG mice. Moreover, NSIN mice facilitated the monitoring and in vivo imaging of both leukemia and solid tumors. Therefore,our NSIN mice provide a new platform for xenograft mouse models in basic and translational research.

# 1 Introduction

2 The development of immunodeficient mice engrafted with human cells or tissues 3 ("humanized’ mice), contributed significantly to translational biomedical research 1-3. 4 The discovery of athymic nude mice4，which were first reported in 1966 as a 5 spontaneously occurring phenotype，enabled the modeling of human tumors in 6 immunodeficient mice5. Subsequent improvements include the severe combined 7 immune deficient (SCID)mutation, targeted mutations in recombination-activating 8 genes 1 and 2 $\left( \mathrm { R a g 1 } ^ { - / - } \right.$ and $\mathbf { R a g 2 } ^ { - / - } ) ^ { 7 , 8 }$ that severely cripple the adaptive immune 9 response of the murine host, and a mutation in the gene encoding the common $\gamma$ chain .0 of the interleukin 2 (IL2） receptor $( \mathrm { I L } 2 \mathrm { r g } )$ .NOD/SCID background with IL2rg 1 mutations, such as $\mathrm { N O D . C g - } P r k d c ^ { s c i d } I l 2 r g ^ { t m I W j l } \qquad \mathrm { ( N S G ) } ^ { 9 }$ （20 and .2 NODShi.Cg-Prkdcscidil2rgtmlSug(NOG) mice10，are able to grow almost all types of 3 human cancers in vivo, including most solid tumors and hematological malignancies, 4 and can be engrafted with functional human immune cels11-14 In our previous Work, 5 we generated a strain of NOD/SCID/II $2 \mathrm { r g } ^ { - / - }$ (NSI) micel5-19.However, the hairs 6 present on these strains impair the observation and imaging of engrafted tumors, .7 necessitating the generation of nude NOD/SCID/IL2rg- mice. The nude 4gene Foxnl 8 (forkhead box N1) encodes a transcription factor for forkhead family proteins20.21. 9 Foxnl is continuously expressed in the thymus,and is necessary for initial thymus ：0 organogenesis and the maintenance of cortical and medullary thymic epithelial cells 1 (cTECs and mTECs) 2²in both embryonic23 and postnatal mice24-26.Mutations in ：2 Foxnl cause inborn dysgenesis of the thymus and hairless skin ²7.

A variety of methods have been developed for genome modification, including designer zinc fingers (ZFs)， transcription activator-like effectors (TALEs)，and the type II bacterial CRISPR/Cas9 system. Recently, the CRISPR/Cas9 system has been show tobesuitableformultiplexedgenomeediting28,9Theeaseof design, construction, and delivery of multiple sgRNAs by co-microinjection30-32 of Cas9 mRNA suggest that this system can be used to generate a variety of novel immunodeficient mouse strains.

In the present study,we derived a Foxnlmutated NOD/SCID/IL2rg/- mouse (NSIN） strain using the CRISPR/Cas9 system. Cas9 mRNA and gRNA targeting Foxnl were injected into the cytoplasm of pronuclear-stage NSI mouse embryos. The mutant offspring were mated to generate homozygous NSIN mice. The NSIN mice were hairless and deficient in B,T,and NK cells,exhibited enhanced engrafting capacities for both leukemia and solid tumors compared with NSI and NOG mice. Moreover, the hairlessness facilitated the observation and imaging of tumors. Our study shows that NSIN mice can be used to generate ideal models for basic and translational research.

# Results

# Efficient modification of Foxn1 in PLO8 cells in vitro using CRISPR/Cas9

First, to test the targeting accuracy and efficiency of our CRISPR/Cas9 system, we designed gRNA targeting the first exon 33of murine Foxnl (Figure 1A） and transfected plasmids expressing mammalian-codon optimized Cas9 and gRNA into a murine PLO8 cell line 16 (Figure 1B). $2 4 \mathrm { ~ h ~ }$ later, transfected cells were selected with 72 hours treatment of500 $\mu \ : \mathrm { g / m l }$ of G418 and cell clones were picked.DNA was extracted from twenty cell clones to determine their genotypes in each experiment. DNA sequencing revealed cell clones that carried the expected mutation at the target locus (Figure 1C). The knock-out efficiency of Foxnl in PLO8 cells was about $20 \%$ (Figure 1D). These data demonstrated the specific and efficient targeting of Foxnl by our CRISPR/Cas9 system.

# Generation of NSIN mice by deletion of Foxnl

Then，with the CRISPR/Cas9 system with multiplexable genome engineering capabilities,we attempted to knock out the Foxnl gene from the NOD/SCID/IL2rg background (NSI mice). Foxnl deleted mice were generated through direct embryo manipulation (Figure 2A). After in vitro transcription, a mixture of Cas9 mRNA (20 $\mathrm { n g / \mu l } )$ and gRNA for Foxnl $( 2 0 \mathrm { \ n g / \mu l } )$ was microinjected into the cytoplasm of

1 pronuclear-stage embryos of NSI mice28.Blastocysts derived from the injected   
2 embryos were transplanted into foster mothers and 14 newborn pups were obtained   
3 (Table 1). Genomic DNA was extracted from the pups for PCR amplification. DNA   
4 sequencing revealed that one mouse carried the expected mutation at the target locus   
5 (Figure 2B). A new AluI restriction enzyme recognition site was generated by deleting   
6 a thymidine (Figure 2B). Then, a 123 bp fragment spanning the target site was   
7 amplified using PCR. The PCR products were digested with the AluI enzyme. The   
8 AluI digestion of the PCR products of wild-type，heterozygous and homozygous offspring generated fragments with lengths of 123 bp, $1 2 3 \substack { + 9 8 + 2 5 }$ bp,and $9 8 \substack { + 2 5 }$ bp, respectively (Figure 2C). Using this restriction fragment length polymorphism (RFLP) assay, we were able to rapidly and accurately identify the genotypes of offspring mice. Due to the loss of Foxnl, homozygous NSIN females are poor breeders and fail to lactate.Heterozygous $( F o x n I ^ { + / - } \mathrm { N S I } )$ ） females breed well,which were mated with homozygous male NSIN mice to generate the NSIN offspring. The average litter size was about 6 to 8 per litter. Overall, about a half of the mice offspring were NSIN mice. The NSIN mice were athymic and hairless (Figure 2D). Furthermore, B,T and NK cells were absent in NSIN mice, similar to NSI, NOG and NDG mice (Figure 2E). In summary, we generated a novel strain of nude mice by knocking out the Foxnl gene by CRISPR/Cas9 from the background of NOD/SCID/IL2rg-/- (NSI).

# Loss of Foxnl impaired T cell development and thymus regeneration in NSIN mice

Foxnl is a critical regulator of thymus and T cell development. To compare the capacities of adoptive T cell development between NSI and NSIN mice, we detected the reconstitution of allogeneic bone marrow nucleated cells (BMNCs) (Figure 3A). After 4 weeks, the reconstitution of total donor cells was similar between NSI and NSIN mice (Figure 3B); however, T cell development from BMNCs was markedly impaired in NSIN mice compared to NSI mice (Figure 3C),as indicated by the percentages of donor T cells in the peripheral blood (PB) (Supplemental Figure 1A), spleen (SP) and bone marrow (BM).Moreover, both CD4 and CD8 T cels were present in NSIN and NSI mice, but the CD4-CD8- compartment increased in NSIN mice compared to NSI mice (Supplemental Figure 1B). Strikingly,4 weeks after BMNCs transplantation, thymuses were regenerated in NSI mice,but not in NSIN mice (Supplemental Figure 1C)，confirming that Foxnl is a critical regulator of thymus development. In the regenerated thymus, most thymocytes were derived from the donor mice and most T cells were $_ \mathrm { C D 4 + C D 8 + }$ (Supplemental Figure 1D).

# Foxn1 deletion resulted in improved engraftment of leukemic cells in NSIN mice

Immunodeficiency is positively correlated with tumor engraftment capacity12.34.35. NSI mice have been widely used in translational studies of human immunology and hematological malignancies6. To evaluate the effect of Foxnl deletion on the engraftment capacity of hematological cells，we compared the engraftment of a human B cell acute lymphoblastic leukemia (B-ALL) cell line, Nalm6, in NDG, NOG, NSI and NSIN mice.First, Nalm6 cells were labeled with GFP-luciferase using lentiviral transfection followed by FACS sorting on $\mathrm { G F P ^ { + } }$ cells (Figure 4A). Subsequently, Nalm6-GL cells were injected into in NDG, NOG, NSI and NSIN mice at low ( $_ { \mathrm { n } = 5 }$ , for each strain) (Figure 4B),medium ( $\mathrm { \Delta } \mathrm { n } { = } 5$ , for each strain) (Figure 4C), and high ( $_ { \mathrm { n } = 5 }$ ，for each strain） (Figure 4D） doses via the tail vein without preconditioning. Upon exhibiting symptoms of paralysis,the mice were sacrificed. The proportions of $\mathrm { G F P + }$ Nalm6-GL cells in PB,SP and BM were higher in NSIN mice than in NDG, NOG and NSI mice (Figure 4B-D, Supplementary Figure 2). To ensure that $\mathrm { G F P + }$ cells were Nalm6-GL cells,we also analyzed human CD19 and found that all ${ \mathrm { G F P + } }$ cells were tumor cells (Figure 4E). Thus,NSIN mice exhibited enhanced capacity of engrafting leukemic cells compared to NDG, NOG and NSI mice.

Foxn1 deletion in NSIN mice resulted in improved engraftment of solid tumors To evaluate the solid tumor engraftment capacity of NSIN mice,we compared the engraftment of a human lung adenocarcinoma cel line, A549, in NDG, NOG, NSI and NSIN mice. A549-GL cells were generated similarly to Nalm6-GL cells (Figure 5A). A549-GL cells were subcutaneously injected into NDG, NOG, NSI and NSIN mice at low ( $\mathrm { \Delta } \mathrm { n } { = } 5$ ,for each strain),medium ( $\mathrm { \tilde { n } } { = } 5$ ,for each strain) and high doses（ $\mathrm { \Delta } ( { \mathrm { n } } { = } 5$ for each strain). The A549-GL tumors were significantly larger in NSIN mice than in NDG,NOG and NSI mice,at all three doses (Figure 5B-D). NOG, NSI and NSIN mice bearing subcutaneous A549-GL tumors in the medium and high dose groups were shown (Figure 5E-F) and tumors were most clearly visible in NSIN mice. In contrast to NOG and NSI mice, the blood vessels of the tumors were also clearly visible in NSIN mice.

# NSIN mice exhibited higher TEI scores

Previously， we developed a tumor engraftment index (TEI) and a statistical formula 15 for the simple and accurate quantification of the immunodeficiency of mouse strains. The TEI scores for NDG, NOG, NSI and NSIN mice engrafting Nalm6-GL cells were calculated as detailed previously 15 (Figure 6A). The differences between the TEI scores of these three strains were small， indicating significant engrafting of human acute leukemia cells by all of these strains,which is consistent with previous results12. However, the TEI score of NSIN mice engrafting A549-GL cells was higher than those of NDG, NOG and NSI strains (Figure 6B). The overall TEI scores for hematological and solid cancers were also calculated, which indicated an overall increase in the immunodeficiency of NSIN mice,compared with NDG, NOG and NSI strains (Figure 6C).

# NSIN mice facilitated fluorescence live imaging of xenografts

The hairless phenotype of NSIN mice could be advantageous in tumor watch and imaging. Indeed,another human large cell lung cancer cell line, H46O-GL cells, were injected into NSIN and NSI mice,and the subcutaneous tumors showed no difference between NSIN and NSI mice in terms of tumor size and luciferase-derived bioluminescence; however, tumors in NSIN mice emanated more condensed and sharper GFP fluorescence than NSI mice (Figure 7A).Likewise, the $\mathrm { C D } 1 2 3 +$ human acute myeloid leukemia cell line Molm13-GL were intravenously injected into NSIN and NSI mice, and GFP signals emanating from the spleen and bone marrow of NSIN mice were more distinct than that in NSI mice; moreover, the tumor burden of Molm13-GL cells in NSIN mice were higher than that in NSI mice (Figure 7B). These results indicated that NSIN mice are more suitable hosts for live imaging of autofluorescence derived from xenografts.

# Discussion

In basic and translational research，complex biological processes are to be analyzed in vivo, which leads to a demand for new animal models. NSI mice have been used in translational biomedical research in our lab.However, a nude strain with comparable immunodeficiency has not been previously reported. In the present study, we generated nude NSI mice using the CRISPR/Cas9 system on the background of NSI mice by embryonic co-microinjection with Cas9 mRNA and gRNA. The mutant offspring were mated to generate homozygous NSIN mice. These mice were hairless and exhibited more severe immunodeficiency than NSI mice.Hence,NSIN mice provide a new platform for basic and translational research.

Athymic nude mice bearing deletion of Foxnl have been widely used in cancer research. Foxnl is required to maintain the postnatal thymus,and changes in Foxnl expression in TECs may contribute to thymus involution during aging24. Our study also revealed that loss of Foxnl resulted in blockade of thymus development following allogeneic BMNCs transplantation，which confirmed a critical role of Foxnl in thymus function. The postnatal thymus is the primary source of T cells in vertebrates,and thymocyte development requires interactions with TECs. Specifically, Foxnl controls the transcription of genes involved in the attraction and lineage commitment of T cell precursors,and regulates the expression of genes involved in antigen processing and thymocyte selection25. In consistence， T cell development from allogeneic BMNCs in NSIN mice was also impaired by deletion of Foxnl in our study. Extrathymic development of T cells has been well established 37,which could be responsible for the reduced level, but still presence of T cells in NSIN mice, especially in the spleen.

Nude mice are suitable for research requiring whole-body fluorescence-based imaging techniques38-40. The novel NSIN strain derived using the CRISPR/Cas9 system exhibited a hairless phenotype. This was advantageous for observation and imaging of tumors by GFP or other fluorescence.For example,the blood vessels of subcutaneous tumors could be seen through the skin of NSIN mice,making NSIN mice a potentially ideal model for xenograft angiogenesis studies. In addition, NSIN mice exhibited higher immunodeficiency than NSI mice,as indicated by their TEI scores. The elevation of TEI scores caused by Foxnl deletion was more significant for subcutaneous solid tumors than for hematological cancers (Figure 5). In our study, NSI mice engrafted tumors more efficiently than NOG mice，which could be attributed to the presence of extracellular domain of $\mathrm { I L 2 r g }$ in NOG mice, but complete IL2rg deletion in NSI mice 3. Foxnl inactivity in nude mice maintains skin in an immature state,resembling neoteny41-43. Thus, NSIN mice mayalso be useful for generating humanized skin models to study human skin regeneration.

In sum, our study generated a new mouse strain and demonstrated the advantages of this strain in xenograft models generation and tumor surveillance.

# Materialsand methods

# Mice

All of the animal experiments were performed in the Laboratory Animal Center of the Guangzhou Institutes of Biomedicine and Health (GIBH).All of the experimental protocols were performed in accordance with the instructional guidelines of the China Council on Animal Care and approved by the Ethics Committee of Animal Experiments at GIBH. C57BL/6J (B6) (RRID: IMSR_JAX:000664), ICR/HaJ (RRID:

IMSR_JAX:009122） and NOG mice were purchased from Vital River Laboratory Animal Technology Co. (Beijing, China). Another $\mathrm { N O D / S C I D / I L 2 r g ^ { - / - } }$ strain NDG mice were purchased from Biocytogen (Beijing, China).NSI mice were generated by our group at $\mathrm { G I B H } ^ { 1 5 - 1 9 }$ . NSIN mice were generated using CRISPR-Cas9,as detailed in this paper. All of the mice were bred and maintained in specific pathogen-free (SPF)-grade cages and provided with autoclaved food and water.

# DNA constructs

The guide sequence was incorporated into the first exon of the mouse Foxnl locus. A pair ofoligonucleotides for the targetingsite(forward: $5 ^ { \prime } .$ caccggaagtgtgacgtcagactg $3 ^ { \prime }$ ; reverse: $5 ^ { \prime }$ -aaaccagtctgacgtcacacttcc-3') was annealed and ligated into the BbsI site of the gRNA cloning vector (Addgene: 41824).A pcDNA3.3-hCas9 plasmid (Addgene: 41815) was used for in vitro transfection. A pair of oligonucleotides for the targeting site (forward: $5 ^ { \prime }$ ataggaagtgtgacgtcagactg $3 ^ { \prime }$ ：， reverse: $5 ^ { \prime }$ -taaaaccagtctgacgtcacacttcc-3'） were annealed and ligated into the BbsI-linearized pUC57-T7 vector (Addgene: 51306), using a T7 promoter for in vitro transcription. The Cas9 expression vector (MLM3613; Addgene: 42251) was used for in vitro transcription. All of the plasmids were provided by Professor Liangxue Lai at GIBH.

# Cell culture and transfection

PL08 is an immortalized murine fetal liver cell line, as reported previouslyl6. Millicell Hanging Cell Culture Inserts (Millipore, Darmstadt, Germany） were used to culture PL08 cells. PL08 cells $( 1 \times 1 0 ^ { 6 } )$ were transfected with U6-sgRNA vectors ( $( 1 0 ~ \mu \mathrm { g } )$ and pcDNA3.3-hCas9 plasmids $( 3 0 ~ \mu \mathrm { g } )$ . Cells were selected using G418（ ${ 5 0 0 \ \mathrm { \textmu g / m l } }$ ：0 Sigma) after $2 4 \mathrm { h }$ of transduction, and were maintained in selection medium for $^ { 7 2 \mathrm { ~ h ~ } }$ Single clones were collected and seeded in each well of a 96-well plate.Monoclonal cells were expanded. Then, DNA of monoclonal cells was extracted to determine their genotypes using DNA sequencing. Nalm6, Molm13，A549 and H46O cells were cultured in RPMI-164O media (Gibco, Grand Island, NY, USA) with $10 \%$ fetal bovine serum (FBS；Biochrom，Australia） and $1 \%$ penicillin/streptomycin.HEK-293T cells were used for lentivirus production and cultured in DMEM(Gibco） supplemented with $10 \%$ FBS and $1 \%$ penicillin/streptomycin. Lentivirus particles were produced in HEK-293T cells via polyethyleneimine (Sigma-Aldrich， St Louis,MO，USA) transfection. A pWPXLd-based (luciferase-2A-GFP) lentiviral plasmid and two packaging plasmids, psPAX2 and pMD.2G, were co-transduced into HEK-293T cells. The supernatant containing luciferase-2A-GFP was filtered through a $0 . 4 5 \mathrm { - } \mu \mathrm { m }$ filter, and then used to transfect the Nalm6, Molm13,A549 and H46O cells. The Nalm6-GL (GFP-luciferase)，Molm13-GL，A549-GL and H460-GL cells were sorted using FACSAriaTM II (Becton Dickinson, San Jose, CA, USA) for culture.

# Transcription in vitro

The Cas9 expression vector (MLM3613） was linearized with Pme I(Thermo Fisher Scientific, Waltham,MA,USA） and used as the template for in vitro transcription (IVT）using an mMESSAGE mMACHINE T7 ULTRA Transcription Kit （Ambion, AM1345). Cas9 mRNA was purified using an RNeasy Mini Kit (Qiagen, 74104). The gRNA templates for in vitro transcription of RNA were purified PCR products obtained from pUC57-T7-Foxnl-gRNA vectors using a primer pair (T7-F: （204号 $5 ^ { \prime } .$ gaaattaatacgactcactata $. 3 ^ { \prime }$ and T7-R: $5 ^ { \prime }$ aaaaccgactcggtgccacaaagc-3） and a high-fidelity enzyme (Takara). The T7-sgRNA PCR product was gel purified and used as the template for IVT with a MEGAshortscript T7 Transcription Kit (Ambion, AM1354). The gRNA was purified using a MEGAclear Kit (Ambion, AM1908) and concentrated by alcohol precipitation.

# Microinjection of single-cell embryos

NSI and ICR/HaJ mouse strains were used as zygote donors and foster mothers, respectively. Female NSI mice (aged 8-1O w） were superovulatedusing intraperitoneal injections of pregnant mare serum gonadotropin (5 IU; Sigma) and human chorionic gonadotropin (5 IU； Sigma) at $4 8 \mathrm { - h }$ intervals.The superovulated female mice were mated to NSI male mice,and the fertilized embryos were collected from the oviducts. Cas9 mRNA ( $( 2 0 ~ \mathrm { n g / \mu l } )$ and sgRNA $( 2 0 ~ \mathrm { n g / \mu l } )$ were injected into the cytoplasm of fertilized eggs with visible pronuclei in M2 medium (Sigma) using a piezo-driven micromanipulator. The injected zygotes were cultured in KSOM with amino acids at $3 7 ^ { \circ } \mathrm { C }$ under $5 \%$ （2 $\mathrm { C O } _ { 2 }$ in air until the blastocyst stage (3.5 d). The surviving embryos were selected and transferred into the uterus of pseudo-pregnant foster mothers. The resulting offspring were analyzed for edited Foxnl genes.

# DNA extraction and genotyping

Monoclonal cells and genomic DNA from mouse tail samples were extracted using a Universal Genomic DNA Extraction Kit Ver.3.0 (DV811A, Takara, Japan） according to the manufacturer's instructions. Genomic DNA was subjected to PCR amplification, and mutations were identified by direct sequencing. For sequencing, the primer pair Foxn1-F1 ( $5 ^ { \prime }$ -aatttctcaccttggctatc-3')， Foxn1-R1( $5 ^ { \prime }$ caggagtcccaaagtgacgg-3') was used to amplify 644 bp PCR products spanning the target site. After identifying the mutation site, the primer pair Foxnl-F2( $5 ^ { \prime }$ -ttcgaggccaggactgggtg-3')， Foxn1-R2 0 $5 ^ { \prime } { \cdot }$ -ttacgttctgtggggcaggg-3') was used to amplify 123 bp PCR products spanning the mutation site. The PCR products were digested with the Alu I enzyme (Fermentas; Thermo Fisher Scientific).

# FACS analysis

Cells isolated from PB,BM and SP of mice were subjected to flow cytometric analyses. The cellswere labeled with FACSantibodies(anti-mCD19-PE, anti-mCD3-APC, anti-mNKp46-PE, anti-mCD3-FITC, anti-mCD4-PE, anti-mCD8-Percp-cy5.5,anti-mCD45.1-PE,anti-mCD45.2-APC) on ice for $2 0 ~ \mathrm { m i n }$ All of the antibodies were obtained from eBioscience (San Diego， CA, USA).Nalm6-GL cells were labeled with anti-hCD19-APC and Molm13-GL cells were labeled with anti-hCD123-APC. Flow cytometric analyses were performed using FACSAriaTMII. All of the data were analyzed using FlowJo software (Tree Star, Inc., Ashland, OR, USA).

# Bone marrow transplantation

Bone marrow-nucleated cells (BMNCs; $4 \times 1 0 ^ { 6 \cdot }$ ） from C57BL/6J (B6) donor mice $\mathrm { ( C D 4 5 . 2 + ) }$ were injected into the retro-orbital venous sinus of lethally irradiated (2 Gy） NSI $\mathrm { ( C D 4 5 . 1 + ) }$ and NSIN $\mathrm { ( C D 4 5 . 1 + ) }$ mice,respectively. The engraftment of $\mathrm { C D 4 5 . 2 + }$ donor cells were detected every week in the PB of recipients. 4 weeks later, five animals in each group were killed after bone marrow transplantation (BMT） to analyze the repopulation efficiency of donor cells and the development of donor T cells in the recipients.

# Engraftments ofleukemia and solid tumors

To directly compare the efficiency of cancer cell engraftment, $1 \times 1 0 ^ { 4 } \mathrm { ( L ) } , 1 \times 1 0 ^ { 5 } \mathrm { ( M ) } .$ or $1 \times 1 0 ^ { 6 } \mathrm { ( H ) }$ NALM6-GL cells suspended in phosphate buffer solution (PB; $0 . 2 \mathrm { m L }$ were injected into the tail veins of NOG, NSI and NSIN mice. Similarly, $1 \times 1 0 ^ { 4 } \mathrm { { ( L ) } }$ $1 { \times } 1 0 ^ { 5 } \left( \mathbf { M } \right)$ ，or $1 { \times } 1 0 ^ { 6 } \left( \mathrm { H } \right)$ A549-GL cells suspended in PBS $( 0 . 2 ~ \mathrm { m L }$ ） were injected subcutaneously into NOG, NSI and NSIN mice. The end points were based on animal models in widespread use. The engraftment of leukemia was measured by analyzing the $\mathrm { G F P ^ { + } }$ cells in the PB weekly, or when the mice were moribund after grafting.

# Fluorescence live imaging

In vivo whole-body imaging of GFP-labeled cells was performed using a cooled CCD camera system (IVIS 1OO Series Imaging System, Xenogen， Alameda, CA, USA). Before in vivo imaging，mice burdened with H46O-GL cells or Molm13-GL cells were anaesthetized with isofurane. Quantification of total and average emissions was performed using Living Image software (Xenogen).

# Quantification of mouse immunodeficiencies (TEI)

The hematologic TEI was calculated using the following equation: $I _ { \mathrm { s t r a i n \cdot } }$ hematologic （204 $\scriptstyle { \mathrm { { t u m o r } } } - n = { \mathrm { { G i } } } / { \mathrm { { D i } } }$ ， where “strain” is the immunodeficient mouse strain，“hematologic tumor” is the type of tumor cells, $n$ is the number of individuals,“Gi” is the sum of the percentages of tumor cells in the peripheral blood (GPB), bone marrow (GBM), and spleen (GSP) of an individual mouse either during morbidity or after death,and “Di” is the lifespan of the individual mouse after injection of the tumor cells. The solid TEI was calculated using the following equation: $I _ { \mathrm { s t r a i n - s o l i d t u m o r } - n } { \equiv } \mathrm { W i / D }$ ，where “strain” is the immunodeficient mouse strain,“solid tumor” is the type of transplanted tumor cells, $n$ is the number of individuals,“Wi” is the weight of the graft in an individual mouse either during morbidity or after death, and “D” is the survival time of the mouse after injection of the tumor cells.The total TEI score was determined as follows: ${ \cal I } _ { \mathrm { s t r a i n - x e n o g r a f t } - n } { = } ~ ( I _ { \mathrm { s t r a i n - N a l m } 6 - n } + I _ { \mathrm { s t r a i n - A } 5 4 9 - n } ) / 2$ . Calculations were performed using tools on the website of In Vivo Biomedical (http://www.nsitei.com)15.

# Statistical analyses

Data are presented as means $\pm$ standard deviation. Student's $t \cdot$ -test was used to determine the statistical significance of differences between samples. $P < 0 . 0 5$ was considered to be significant. All statistical analyses were performed using Prism software, version 5.O (GraphPad, Inc., San Diego, CA, USA).

# Acknowledgements

This study was supported by National Natural Science Foundation of China (815220O2)， the Strategic Priority Research Program of the Chinese Academy of Sciences (XDB19030205)， the Natural Science Fund for Distinguished Young Scholars of Guangdong Province (2014A030306028), the Guangdong Provincial Applied ScienceandTechnologyResearch&DevelopmentProgram (2016B02O237006), the Guangdong Provincial Outstanding Young Scholars Award (2014TQO1RO68), the Frontier and key technology innovation special grant from the Department of Science and Technology of Guangdong province, (2O15B020227003, 2014B020225005，2016B030229006)， the Guangdong Provincial Researchand Commercialization Program (Grant No． 2014B090901044)，and the Guangzhou Science Technology and Innovation Commission Project (201504010016).

# Author Contributions

XW contributed to the conception and design, collection and/or assembly of data, data analysis and interpretation，and manuscript writing. YL，BL， LQ， YX and QL contributed to the provision of study material, collection and/or assembly of data. BL, SL， SW,and QW provided animal care and administrative supports. YY and DP contributed to the conception and design and provided financial support. GH, QD, PL, DW, and LL contributed to the conception and design. XW and PL contributed to the conception and design, data analysis and interpretation, manuscript writing,and final approval of manuscript and provided financial support. All authors read and approved the final manuscript.

# Conflict ofInterest Statement

The authors declare no competing financial interests.

# Figure legends

Figure 1.Foxnl gene targeting in PLO8 cells using a type II CRISPR system in vitro.(A） Schematic of the Cas9/sgRNA-targeting site in exon 1 ofFoxnl. The sgRNA-targeting and protospacer-adjacent motif (PAM) sequences are labeled in red and blue,respectively. (B) Schematic of Foxnlgene targeting in PLO8 cells in vitro. PL08 cells were transfected with U6-Foxnl-sgRNA vector and pcDNA3.3-hCas9 plasmid. Abbreviations: $\mathrm { U } 6 = \mathrm { U } 6$ polymerase II promoter; $\mathbf { C M V } =$ cytomegalovirus promoter; ${ \mathrm { N L S ~ } } =$ nuclear localization signal; TK $\mathbf { \tau } = \mathbf { \tau }$ thymidine kinase; $\mathbf { p A } _ { \mathbf { \alpha } } = \mathbf { \alpha }$ polyadenylation signal. In the plate wells, living cells are shown in blue and dying cells are shown in gray. (C) Genotyping of Foxnl-targeted and wild type (WT) PL08 cells. Clones 1 and 9 are shown. The target exon 1 of Foxnl was amplified by PCR and then analyzed using DNA sequencing. Double sequencing peaks in clone 1 showed mutant alleles. The three sequencing peaks in clone 9 may originate from a mixture of two different clones. (D) Efficient knockout of the Foxnl gene in PL08 cells by co-transfection with U6-Foxnl-sgRNA vectors and pcDNA3.3-hCas9 plasmids. The percentages indicate Foxnl-knockout clones in different groups. Data were representative of three independent experiments. 2O clones were picked and analyzed in each experiment.

#

Figure2. Generation and characterization of Foxnl deleted NSIN mice. (A) Schematic of the generation of NSIN mice. Cas9 mRNA and gRNA were co-injected into the cytoplasm of the pronuclear-stage mouse embryos. The injected embryos were then transferred into pseudopregnant surrogate mothers. The mouse pups were genotyped and screened for Foxnl mutations. The in vitro transcription of both vectors was controlled by T7 promoters. (B) Sequence analysis of the founder mice. The target exon 1 of Foxnl was amplified using PCR and then analyzed by DNA sequencing. The PCR products were 644 bp long. The WT sequence of Foxnl was shown at the top of the figure. Double sequencing peaks indicated the presence of mutant alleles in the somatic cells of the mice. The targeting and PAM sequences are labeled in red and blue,respectively. A new AluI enzyme locus was generated by deleting a thymidine.(C) PCR genotyping of the F2 generation offspring. Primers were designed to amplify 123 bp PCR products spanning the mutation site. The PCR products were digested using the Alu I enzyme. Results from seven offspring mice were shown: WT NSI (lane2 and 5), Foxnl+/-NSI (lane 1, 6 and 7),and Foxnl- NSI (NSIN) mice (lane 3 and 4). (D) Photographs of thymus of ICR and NSIN mice. (E) Flow cytometric analysis indicating the absence of B,Tand NK cells in the peripheral blood (PB) of B6, NOG, NSI, NSIN and NDG mice. Cells were gated from FSC/SSC for the detection of T,B and NK cells. B6 mice were used as a positive control.

Figure 3. Allogeneic T cell development was affected in NSIN mice. (A) Schematic of the bone marrow transplantation (BMT). Bone marrow-nucleated cells (BMNCs; 4 $\times 1 0 ^ { 6 } )$ from C57BL/6J (B6) donor mice $\mathrm { ( C D 4 5 . 2 + ) }$ were injected into the retro-orbital venous sinus of lethally irradiated (2 Gy) NSI $\mathrm { ( C D 4 5 . 1 + ) }$ and NSIN $\mathrm { ( C D 4 5 . 1 + ) }$ mice, respectively. The engraftment of $\mathrm { C D 4 5 . 2 + }$ donor cells were detected every week in the PB of recipients. 4 weeks later, five animals in each group were killed BMT to analyze the repopulation efficiency of donor cells and the development of donor T cells in the recipients.(B） At 4 weeks,BMNCs from B6 mice reconstituted in both NSI and NSIN mice at similar levels in PB. $\mathrm { C D 4 5 . 2 + }$ cells were derived from B6 mice.(C)Therepopulationpercentageof $\mathrm { C D 4 5 . 2 + }$ donorcells(left）and $\mathrm { C D 4 5 . 2 + C D 3 + \mathrm { ~ I ~ } }$ cells (right) in PB,SP and BM. $\mathrm { n } { = } 5$ for each group.Error bars denote standard errors of the means (SEM). Groups were compared using the two-tailed unpaired $t$ -test. $^ { * } P < 0 . 0 5$ .\*\* $: P < 0 . 0 1$ .\*\*\* $P { < } 0 . 0 0 1$ ·

Figure 4. Engraftment efficiencies of Nalm6-GL cells in NDG, NOG, NSI, and NSIN mice. (A) Establishment of Nalm6-GL cell line. Nalm6 cells were transfected with luciferase-GFP-expressing lentiviral vectors， and sorted on $\mathrm { G F P + }$ cells. Nalm6-GL cells were labeled human $\scriptstyle \mathrm { C D 1 9 + }$ and $\mathrm { G F P + }$ . (B-D） Percentages of Nalm6-GL cells in PB, SP, and BM of NDG, NOG, NSI, and NSIN mice injected with (B) low ( $\mathrm { 1 \times 1 0 ^ { 4 } }$ ， $\mathrm { n } { = } ~ 5$ for each strain),(C) medium ( $\mathrm { 1 \times 1 0 ^ { 5 } }$ ， $\mathrm { n } \mathrm { = } \mathrm { ~ } 5$ for each strain),and (D） high numbers( $\mathrm { 1 \times 1 0 ^ { 6 } }$ ， $\mathrm { { n } } \mathrm { { = } } ~ 5$ for each strain) of Nalm6-GL cells. Nalm6-GL cells were injected into each mouse through the tail vein, and analyzed on day 20,24,or 32 for the low，medium，and high dose groups，respectively. (E) Engraftment efficiencies of Nalm6-GL cells were further confirmed by labeling with human $\scriptstyle \mathrm { C D 1 9 + }$ in the BM from the high dose $( 1 \times 1 0 ^ { 6 } )$ ） group.Error bars denote standard errors of the means (SEM). Groups were compared using the two-tailed unpaired $t$ -test.\* $P { < } 0 . 0 5$ .\*\* $P < 0 . 0 1$ .\*\*\* $P { < } 0 . 0 0 1$ ：

Figure 5. Engraftment effciencies of A549-GL cells in NDG,NOG,NSI, and NSIN mice. (A） Establishment of A549-GL cell line.Lung adenocarcinoma A549 cells were transfected with luciferase-GFP-expressing lentiviral vectors, and sorted on $\mathrm { G F P ^ { + } }$ cells.(B-D） Weight of A549 tumors in NDG,NOG,NSI and NSIN mice subcutaneously injected with (B) low ( $\mathrm { 1 \times 1 0 ^ { 4 } }$ ， $\mathrm { n } { = } 5$ for each strain), (C) medium ( $\mathrm { ~  ~ \cdot ~ } ^ { \mathrm { ~ \scriptsize ~ 1 ~ } \times }$ $1 0 ^ { 5 }$ ， $\scriptstyle \mathrm { n = 5 }$ for each strain),and (D) high numbers ( $\mathrm { 1 \times 1 0 ^ { 6 } }$ ， $\mathrm { n } { = } 5$ for each strain） of A549-GL cells and analyzed on day 30. (E-F) Photographs of NOG, NSI, and NSIN micebearing subcutaneous A549-GL tumors from the (E) medium ( $\mathrm { \acute { n } } { = } 5$ for each strain)

and (F) high groups ( $\mathrm { \acute { n } } { = } 5$ for each strain) on day 3O. The mice in the low group bore no visible tumors and are not shown. Error bars denote SEM. Groups were compared using the two-tailed unpaired $t$ -test. $^ { * } \mathrm { P } < 0 . 0 5$ .\*\* $\mathbf { P } < 0 . 0 1$ .\*\*\* $\mathrm { P } < 0 . 0 0 1$ ： Figure 6. NSIN mice showed higher tumor engraftment index (TEI) scores than NDG,NOG,NSI mice. (A) TEI scores of NDG, NOG, NSI and NSIN micefor Nalm6 cells. (B) TEI scores of NDG, NOG, NSI and NSIN mice for A549 cells. (C) Overall TEI scores of NDG,NOG, NSI and NSIN mice for both leukemia and solid tumors.Xenograft TEI scores as indicators of immunodeficiency. Figure7. Improved fluorescence live imaging in NSIN mice.(A) Fluorescence live imaging of H460-GL burdened mice. $1 \times 1 0 ^ { 6 } \ \mathrm { H } 4 6 0  – \mathrm { G L }$ ， cells were subcutaneously injected into NSIN and NSI mice ( $\mathrm { \acute { n } } { = } 3$ for each group) and luciferase and GFP were analyzed on day 30. Total flux of luciferase-derived and GFP-derived fluorescence was compared between NSI and NSIN mice (right panel). H46O-GL, a human large cell lung cancer cell line,expressed GFP and luciferase (GL). (B) Fluorescence live imaging of Molm13-GL burdened mice. $1 \times 1 0 ^ { 6 } \mathrm { M o l m } 1 3 – 0$ GL cells were injected into NSIN and NSI mice ( $\scriptstyle \mathbf { n = 4 }$ for each group) through the tail vein and GFP fluorescence was analyzed on day 30. Left,the AML cell line Molm13-GL expressed CD123 and GFP; middle, fluorescence live imaging of Molm13-GL cells; right, comparison of the total flux of GFP fluorescence between NSI and NSIN mice. Error bars denote SEM. Groups were compared using the two-tailed unpaired $t$ -test. $^ { * } \mathrm { ~ P ~ } { < } 0 . 0 5$ .\*\* $\mathbb { P } < 0 . 0 1$ ： \*\*\* $\mathrm { P } < 0 . 0 0 1$ ： ，

Supplementary Figure 1. NSIN mice are more resistant to allogeneic T cell development than NSI mice.(A） Donor BMNCs reconstitution in the T ( $\mathrm { \ C D 4 5 . 2 + }$ $\mathrm { C D } 3 + \mathrm { \ i }$ cell lineage in PB of NSIN(upper) and NSI (lower) mice ( $\mathrm { \tilde { n } } { = } 5$ for each group) 4 weeks after bone marrow transplantation (BMT). (B) Donor T cells were analyzed for CD4 and CD8 expression by flow cytometry. (C) Loss of Foxnl inhibited the regeneration of thymuses after BMT in NSIN mice. (D) Phenotype of cells from the regenerated thymuses in NSI mice.

Supplementary Figure 2. Representative FACS analysis of Nalm6-GL cells in NOG,NSI, and NSIN mice. (A-C) Percentages of Nalm6-GL cells in PB, SP, and BM of mice intravenously injected with (A） low $\left( 1 \times 1 0 ^ { 4 } \right)$ )，(B）medium $( 1 \times 1 0 ^ { 5 } )$ ）， and(C) high $( 1 \times 1 0 ^ { 6 } )$ ） numbers of Nalm6-GL cells. $\mathrm { G F P ^ { + } }$ cells represent Nalm6-GL cells.

# References

v   
9 1 Shultz,L.D., Brehm, M.A., Bavari, S.& Greiner, D.L. Humanized mice as a preclinical tool   
10 forinfectious disease and biomedical research. Ann Ny Acad Scil245， 50-54,   
11 doi:10.1111/j.1749-6632.2011.06310.x (2011).   
12 2 Shultz,L.D., Brehm, M.A., Garcia-Martinez,J.V.& Greiner, D.L.Humanized mice for   
13 immune system investigation: progress, promise and challenges. Nat Rev Immunol12, 786-798,   
14 doi:10.1038/nri3311 (2012).   
15 3 Shultz,L.D., Ishikawa, F. & Greiner,D.L. Humanized mice in translational biomedical   
16 research.Nat Rev Immunol7,118-130,doi:10.1038/nri2017 (2007).   
17 4 Flanagan, S.P.'Nude', a new hairless gene with pleiotropic effects in the mouse. Genet Res8,   
18 295-309 (1966).   
19 5 Rygaard,J.& Povlsen, C. O. Heterotransplantation of a human malignant tumour to "Nude"   
20 mice.Acta Pathol Microbiol Scand77,758-760(1969).   
21 6 Bosma, G. C., Custer, R.P.& Bosma, M.J.A severe combined immunodeficiency mutation in   
22 the mouse. Nature301, 527-530 (1983).   
23 7 Mombaerts，P. et al. Rag-1-Deficient Mice Have No Mature Lymphocytes-B and   
24 Lymphocytes-T. Cell68,869-877, doi:Doi 10.1016/0092-8674(92)90030-G (1992).   
25 8 Shinkai, Y. et al. Rag-2-Deficient Mice Lack Mature Lymphocytes Owing to Inability to   
26 Initiate V(D)J Rearrangement. Cell68, 855-867，doi:Doi 10.1016/0092-8674(92)90029-C   
27 (1992).   
28 9 Shultz,L. D.et al. Human lymphoid and myeloid cell development in NOD/LtSz-scid IL2R   
29 gamma(null) mice engrafted with mobilized human hemopoietic stem cells. J Immunol174,   
30 6477-6489 (2005).   
31 10 Ito,M.et al. NOD/SCID/gamma(c)(null) mouse: an excellent recipient mouse model for   
32 engraftment of human cells. Blood100,3175-3182,doi:10.1182/blood-2001-12-0207 (2002).   
33 11 Covassin, L. et al. Human immune system development and survival of non-obese diabetic   
34 (NOD)-scid IL2rgamma(null) (NSG) mice engrafted with human thymus and autologous   
35 haematopoietic stem cells. Clin Exp Immunol174, 372-388,doi:10.1111/cei.12180 (2013).   
36 12 Agliano,A. et al. Human acute leukemia cells injected in NOD/LtSz-scid/IL-2Rgamma null   
37 mice generate a faster and more efficient disease compared to other NOD/scid-related strains.   
38 Int J Cancer123,2222-2227,doi:10.1002/ijc.23772 (2008).   
39 13 Hiramatsu,H.et al. Complete reconstitution of human lymphocytes from cord blood $\mathrm { C D } 3 4 +$   
40 cells using theNOD/SCID/gammacnullmicemodel. Blood102， 873-880,   
1 doi:10.1182/bl00d-2002-09-2755 (2003).   
2 14 Ishikawa, F., Livingston, A. G.， Wingard, J. R.， Nishikawa, S.& Ogawa, M. An assay for   
3 long-term engrafting human hematopoieticcellsbasedon newborn   
4 NOD/SCID/beta2-microglobulin(nul) mice. Exp Hematol30, 488-494 (2002).   
5 15 Ye,W. et al. Quantitative evaluation of the immunodeficiency of a mouse strain by tumor   
6 engraftments. J Hematol Oncol8,59,doi:10.1186/s13045-015-0156-y (2015).   
7 16 Xiao,Y. et al. ANGPTL7 regulates the expansion and repopulation of human hematopoietic   
8 stem and progenitor cells. Haematologica100, 585-594, doi:10.3324/haematol.2014.118612   
9 (2015).   
10 17 Jiang,Z. et al. Heterogeneity of CD34 and CD38 expression in acute B lymphoblastic   
11 leukemia cells is reversible and not hierarchically organized. J Hematol Oncol9，94,   
12 doi:10.1186/s13045-016-0310-1 (2016).   
13 18 Ye, W. et al. GZD824 suppresses the growth of human B cell precursor acute lymphoblastic   
14 leukemia cells by inhibiting the SRC kinase and PI3K/AKT pathways. Oncotarget,   
15 doi:10.18632/oncotarget.10881 (2016).   
16 19 Jiang，Z. et al. Anti-GPC3-CAR T Cels Suppress the Growth of Tumor Cells in   
17 Patient-Derived Xenograftsof HepatocellularCarcinoma. FrontImmunol7， 690,   
18 doi:10.3389/fimmu.2016.00690 (2016).   
19 20 Takahashi， Y. et al. Mapping of the Nu-Gene Using Congenic Nude Strains and Insitu   
20 Hybridization.J Exp Med175,873-876, doi:DOI 10.1084/jem.175.3.873 (1992).   
21 21 Kaestner,K.H.，Knochel，W.& Martinez,D. E. Unified nomenclature for the winged   
22 helix/forkhead transcription factors. Gene Dev14,142-146 (2000).   
23 22 Nowell, C. S. et al. Foxnl regulates lineage progression in cortical and medullary thymic   
24 epithelial cells but is dispensable for medullary sublineage divergence. PLoS Genet7,   
25 e1002348, doi:10.1371/journal.pgen.1002348 (2011).   
26 23 Gordon, J., Bennett, A. R., Blackburn, C. C.& Manley, N. R. Gcm2 and Foxn1 mark early   
27 parathyroid- and thymus-specific domains in the developing third pharyngeal pouch. Mech   
28 Dev103,141-143 (2001).   
29 24 Chen,L. Z., Xiao, S. Y.& Manley, N.R. Foxn1 is required to maintain the postnatal thymic   
30 microenvironment in a dosage-sensitive manner. Blood113, 567-574,   
31 doi:10.1182/bl00d-2008-05-156265 (2009).   
32 25 Zuklys, S. et al. Foxnl regulates key target genes essential for Tcell development in postnatal   
33 thymic epithelial cels. Nat Immunol17,1206-1215,doi:10.1038/ni.3537 (2016).   
34 26 Cheng,L. et al. Postnatal tissue-specific disruption of transcription factor FoxN1 triggers   
35 acute thymic atrophy. J Biol Chem285,5836-5847, doi:10.1074/jbc.M109.072124 (2010).   
36 27 Pantelouris,E.M.Absence of thymus in a mouse mutant. Nature217,37O-371 (1968).   
37 28 Wang，H. et al. One-step generation of mice carrying mutations in multiple genes by   
38 CRISPR/Cas-mediated genome engineering. Cell153, 910-918, doi:10.1016/j.cell.2013.04.025   
39 (2013).   
40 29 Cong,L. et al. Multiplex Genome Engineering Using CRISPR/Cas Systems. Science339,   
41 819-823,doi:10.1126/science.1231143 (2013).   
42 30 Ma，X.etal.CRISPR/Cas9-mediated gene manipulation to create   
43 single-amino-acid-substituted and floxed mice with a cloning-free method. Sci Rep7, 42244,   
44 doi:10.1038/srep42244 (2017).   
1 31 Tu, Z.et al. Promoting Cas9 degradation reduces mosaic mutations in non-human primate   
2 embryos. Sci Rep7,42081, doi:10.1038/srep42081 (2017).   
3 32 Raveux,A., Vandormael-Pournin, S. & Cohen-Tannoudji, M. Optimization of the production   
4 of knock-in alleles by CRISPR/Cas9 microinjection into the mouse zygote. Sci Rep-Uk7,   
5 doi:ARTN 4266110.1038/srep42661 (2017).   
6 33 Goto,T.,Hara,H.,Nakauchi,H.,Hochi, S.& Hirabayashi,M.Hypomorphic phenotype of   
7 Foxn1 gene-modified rats by CRISPR/Cas9 system. Transgenic Res25， 533-544,   
8 doi:10.1007/s11248-016-9941-9 (2016).   
9 34 Kanaji， N. et al. Higher susceptibility of NOD/LtSz-scid $\mathrm { I l 2 r g ( - / - ) }$ NSG mice to   
10 xenotransplanted lung cancercell lines. Cancer Manag Res6, 431-436,   
11 doi:10.2147/Cmar.S71185 (2014).   
12 35 Machida, K. et al. Higher susceptibility of NOG mice to xenotransplanted tumors.J Toxicol   
13 Sci34,123-127 (2009).   
14 36 Manz, M. G. Human-hemato-lymphoid-system mice: opportunitiesand challenges.   
15 Immunity26, 537-541, doi:10.1016/j.immuni.2007.05.001 (2007).   
16 37 Blais,M. E.， Louis,I. & Perreault,C. T-cell development: an extrathymic perspective.   
17 Immunol Rev209,103-114, doi:10.1111/j.0105-2896.2006.00341.x (2006).   
18 38 Metildi, C.A. et al. Fluorescently labeled chimeric anti-CEA antibody improves detection and   
19 resection of human colon cancer in a patient-derived orthotopic xenograft (PDOX） nude   
20 mouse model. J Surg Oncol109, 451-458, doi:10.1002/jso.23507 (2014).   
21 39 Kiyuna, T. et al. Labeling the Stroma of a Patient-Derived Orthotopic Xenograft (PDOX)   
22 Mouse Model of Undifferentiated Pleomorphic Soft-Tissue Sarcoma With Red Fluorescent   
23 Protein for Rapid Non-Invasive Imaging for Drug Screening. J Cell Biochem118,361-365,   
24 doi:10.1002/jcb.25643 (2017).   
25 40 Hiroshima, Y. et al. Metastatic Recurrence in a Pancreatic Cancer Patient Derived Orthotopic   
26 Xenograft (PDOX） Nude Mouse Model Is Inhibited by Neoadjuvant Chemotherapy in   
27 Combination with Fluorescence-Guided Surgery with an Anti-CA 19-9-Conjugated   
28 Fluorophore.Plos One9,doi:ARTN e11431010.1371/journal.pone.0114310 (2014).   
29 41 Manuel, J. A. & Gawronska-Kozak, B. Matrix metalloproteinase 9 (MMP-9) is upregulated   
30 during scarless wound healing in athymic nude mice. Matrix Biol25， 505-514,   
31 doi:10.1016/j.matbio.2006.07.008 (2006).   
32 42 Gawronska-Kozak, B., Bogacki, M., Rim, J. S., Monroe, W. T. & Manuel, J. A. Scarless skin   
33 repair in immunodeficient mice. Wound Repair Regen14, 265-276,   
34 doi:10.1111/j.1743-6109.2006.00121.x (2006).   
35 43 Gawronska-Kozak,B. Scarless skin wound healing in FOXN1 deficient (nude） mice is   
36 associated with distinctive matrix metalloproteinase expression. Matrix Biol30, 290-300,   
37 doi:10.1016/j.matbio.2011.04.004 (2011).