# MolecularCell

# Structural Basis of the Differential Function of the Two $0 .$ elegans Atg8 Homologs,LGG-1 and LGG-2, in Autophagy

Graphical Abstract

![](images/1d6a89b17051743fe336dd29514b0c4a5a9028b7a04fe40c3af7340e52de15ee.jpg)

# Authors

Fan Wu, Yasunori Watanabe, Xiang-Yang Guo,..., Jun-Jie Hu, Nobuo N. Noda, Hong Zhang

# Correspondence

nn@bikaken.or.jp (N.N.N.), hongzhang@sun5.ibp.ac.cn (H.Z.)

# InBrief

Wu et al. present high-resolution crystal structures of LGG-1 and LGG-2, two C.elegans homologs of the yeast autophagy protein ATG8. Structural differences between the two proteins reflect theirdifferential functions in degradation of protein aggregates during development.

# Highlights

The N-terminal tails of LGG-1 and LGG-2 adopt unique closed   
and open forms   
lgg-2 exhibits temporal-specific and cargo-specific functions   
in autophagy   
LGG-1 and LGG-2 differentially interact with autophagy   
substrates and ATG proteins   
Lipidated LGG-1 and LGG-2 exhibit different membrane   
tethering and fusion activity

# Accession Numbers

5AZF   
5AZG   
5E6N   
5E60   
5AZH

# Structural Basis of the Differential Function of the Two C. elegans Atg8 Homologs, LGG-1 and LGG-2, in Autophagy

FanWu,15Yasunori Watanabe,2,5 Xiang-YangGuo,3.5 XinQi,1.5Peng Wang,4 Hong-Yu Zhao,1 Zheng Wang,1 Yuko Fujioka,2HuiZhang,1 Jin-QiRen,Tian-Cheng Fang,3Yu-XianShen,4WeiFeng,1Jun-JieHu,1NobuoN.Noda, and Hong Zhang1,6,\*

1NationalLaboratoryof Biomacromoecules,InstituteofBiophysics,ChineseAcademyofSciences,Beijing10001,P.R.Cina   
2Institute of Microbial Chemistry,Tokyo 141-0021, Japan   
3College of Life Sciences, Nankai University,Tianjin 30oo71,P.R.China   
4School of Basic Medical Sciences, Anhui Medical University, Hefei 230032, P.R.China   
$^ { 5 } { \mathsf { C o } }$ -firstauthor   
$^ { 6 } { \mathsf { C o } }$ -senior author   
\*Correspondence: nn@bikaken.or.jp (N.N.N.), hongzhang@sun5.ibp.ac.cn (H.Z.)   
http://dx.doi.org/10.1016/j.molcel.2015.11.019

# SUMMARY

Multicellular organisms have multiple homologs of the yeast ATG8 gene，but the differential roles of these homologs in autophagy during development remain largely unknown. Here we investigated structure/function relationships in the two C. elegans Atg8 homologs, LGG-1 and LGG-2. Igg-1 is essential for degradation of protein aggregates,while Igg-2 has cargo-specific and developmental-stage-specific roles in aggregate degradation. Crystallography revealed that the N-terminal tails of LGG-1 and LGG-2 adopt the closed and open form， respectively. LGG-1 and LGG-2 interact differentially with autophagy substrates and Atg proteins,many of which carry a LIR motif. LGG-1 and LGG-2 have structurally distinct substrate binding pockets that prefer different residues in the interacting LIR motif, thus influencing binding specificity. Lipidated LGG-1 and LGG-2 possess distinct membrane tethering and fusion activities,which may result from the N-terminal differences.Our study reveals the differential function of two ATG8 homologs in autophagy during C.elegans development.

selectively removedamagedorganellesand proteinaggregates by tightly enwrapping the cargo and excluding other materials (Stolz et al., 2014; Rogov et al., 2014). The mechanisms by which cargosare selectively recognized and enclosed byautophagosomal membranes remain largelyunknown.

In yeast,about18 autophagy-related (ATG) genes are essential for autophagosome formation (Nakatogawa et al.， 2009; Feng et al., 2014). ATG8 encodes a ubiquitin-like protein that is important for multiple steps in autophagy.Atg8 is conjugated to phosphatidylethanolamine (PE) by the enzymes Atg7 and Atg3.Atg8-PE regulates IM expansion and autophagosome closure,most likely through its membrane tethering and fusion activity (Nakatogawa et al., 2007; Weidberg et al., 2011). Atg8 on the outer autophagosomal membrane interacts with the Rab effectors FYCO1 and PLEKHM1 to facilitate autophagosome transport and fusion with lysosomes (Birgisdottir et al., 2013; McEwan et al., 2015). Atg8 recruits the Atg1/ULK1 complexto the IMto promoteautophagosome formation (Birgisdottir et al.，2013). In selective autophagy，Atg8-PE on the concave face of the IM interacts with autophagy receptor proteins (e.g., p62and NBR1)，which recognize different types of cargo for degradation (Stolz etal., 2014). Atg8/LC3 (the mammalian Atg8 homolog) interacts with proteins via the [W/F/Y]xx[l/LV] motif, known as the Atg8-family interacting motif (AlM)/LC3-interacting region (LIR). In Atg8 family members,two hydrophobic binding pockets termed the“W-site”and“L-site,"recognize thearo matic and [I/LV] residues in the LIR motif,respectively (Noda et al., 2010; Birgisdottir et al., 2013; Wild et al., 2014).

# INTRODUCTION

Duringmacroautophagy (hereafter called autophagy),aportion of cytosol is engulfed bya double-membrane autophagosome and delivered to the lysosome for degradation (Nakatogawa et al.，2009; Feng et al.，2014).Upon autophagy induction，a cup-shaped isolation membrane (IM) (or phagophore) is nucleated,which expands and seals to form the autophagosome. Under stress conditions,autophagosomes non-selectively engulf cytoplasmic contents for degradation.Autophagosomes also

At least seven Atg8 homologs，belonging to the LC3 and GABARAP/GATE-16 subfamilies,associate with autophagosomal structures in mammals.LC3 and GATE-16 have membrane tetheringand fusionactivityand are involved in the elongation of IMs and closure of autophagosomes，respectively (Weidberg et al., 2010). Different Atg8 homologs bind to distinct autophagy receptors to provide specificity for cargo recruitment.For example,LC3C,but not LC3A or LC3B,recognizes a noncanonical LIR in the receptor NDP52 during anti-bacterial autophagy (vonMuhlinen et al.,2012).The differential function of distinct

Table1.DataCollectionandRefinementStatistics   

<html><body><table><tr><td></td><td>LGG-1</td><td>LGG-1</td><td></td><td>LGG-2</td><td>LGG-2</td></tr><tr><td>Ligands</td><td>WEEL Peptide</td><td>UNC-51LIR Peptide</td><td>LGG-2</td><td>WEEL Peptide</td><td>EEEWEEL Fusion</td></tr><tr><td>Data statistics</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Beamline</td><td>NW12A</td><td>BL32XU</td><td>BL19U</td><td>BL19U</td><td>BL32XU</td></tr><tr><td>Wavelength (A)</td><td>1.0000</td><td>1.0000</td><td>0.9785</td><td>0.9785</td><td>1.0000</td></tr><tr><td>Space group</td><td>P21212</td><td>C2221</td><td>C2</td><td>P21</td><td>P212121</td></tr><tr><td>Cell parameters a (A)</td><td>95.3</td><td>63.7</td><td>123.2</td><td>49.9</td><td>35.5</td></tr><tr><td>Cell parameters b (A)</td><td>112.1</td><td>80.1</td><td>24.6</td><td>62.4</td><td>55.2</td></tr><tr><td>Cell parameters c (A)</td><td>35.9</td><td>111.5</td><td>94.8</td><td>67.6</td><td>72.6</td></tr><tr><td>Cell parameters β (°)</td><td></td><td></td><td>130.3</td><td>109.5</td><td></td></tr><tr><td>Resolution range (A)</td><td>29.4-1.60</td><td>55.8-1.80</td><td>40.0-2.10</td><td>50.0-1.80</td><td>43.9-2.20</td></tr><tr><td>Outer range (A)</td><td>1.63-1.60</td><td>1.83-1.80</td><td>2.18-2.10</td><td>1.83-1.80</td><td>2.24-2.20</td></tr><tr><td>Observed reflections</td><td>369,791</td><td>179,541</td><td>12,543</td><td>36,467</td><td>136,004</td></tr><tr><td>Unique reflections</td><td>51,705</td><td>26,489</td><td>2,459</td><td>5,363</td><td>7,374</td></tr><tr><td>Completeness (%)</td><td>99.9 (100.0)</td><td>99.9 (100.0)</td><td>95.7(98.2)</td><td>99.0(100.0)</td><td>98.3 (85.9)</td></tr><tr><td>Rmerge ()</td><td>0.067 (0.463)</td><td>0.088 (0.878)</td><td>0.055(0.270)</td><td>0.103(0.462)</td><td>0.104 (0.708)</td></tr><tr><td><l/σ()></td><td>24.7 (5.3)</td><td>18.1 (3.0)</td><td>24.3(5.1)</td><td>18.1(4.1)</td><td>29.3 (4.5)</td></tr><tr><td>Refinement statistics</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Resolution range (A)</td><td>29.41-1.60</td><td>55.75-1.81</td><td>28.3-2.10</td><td>47.07-1.80</td><td>43.94-2.30</td></tr><tr><td>No. of protein atoms</td><td>2,099</td><td>2,117</td><td>1,865</td><td>3,986</td><td>1,038</td></tr><tr><td>No. of sulfate groups</td><td>1</td><td></td><td>二</td><td>-</td><td>-</td></tr><tr><td>No. of ions</td><td>Cd²+ × 13</td><td>Cd²+ × 14</td><td>-</td><td>-</td><td>Mg²+ × 1</td></tr><tr><td>No. of water</td><td>411</td><td>107</td><td>94</td><td>471</td><td>35</td></tr><tr><td>R/ Rfree</td><td>0.201/0.218</td><td>0.212/0.246</td><td>0.219/0.267</td><td>0.177/0.241</td><td>0.236/0.288</td></tr><tr><td>Rmsd bond length (A)</td><td>0.006</td><td>0.008</td><td>0.009</td><td>0.008</td><td>0.008</td></tr><tr><td>Rmsd bond angle (°)</td><td>1.2</td><td>1.2</td><td>1.2</td><td>1.1</td><td>1.2</td></tr></table></body></html>

Values in parentheses refer to the outer shell.

Atg8 homologs during animal development, however,remains largelyunexplored.

During C.elegans development，autophagy selectively removes superfluous organelles and protein substrates (Yang and Zhang， 2014)． Several differentprotein aggregates, including the p62 homolog SQST-1 and PGL granule components (PGL-1/-3 and SEPA-1),are degraded by autophagy，a process known as aggrephagy (Zhang et al., 20o9; Tian et al., 2010).Scaffold proteins are important for conferring cargo selectivityand promotingdegradation by linking cargo/receptor complexes with autophagic structures (Lin et al.，2013).The scaffold protein EPG-7 links SQST-1 aggregates with LGG-1 (the C.elegans Atg8 homolog) structures (Lin et al.，2013). lgg-1is essential for aggrephagy and other autophagy-regulated processes (Tian et al.，201O). lgg-2，which encodes another Atg8 homolog，actsnon-redundantly with Igg-1 in removal of paternally inherited mitochondria in embryos by regulating autophagosome maturation (Manil-Ségalen et al.. 2014).In this study，we show that Igg-2 has cargo-specific and developmental-stage-specific functions in aggrephagy. The structural differences in the substrate binding regions and the N-termini of LGG-1 and LGG-2 provide insights into the differential function of two ATG8 homologs in autophagy during C.elegans development.

# RESULTS

# Crystal Structures ofLGG-1 and LGG-2

We determined the structures of LGG-1 and LGG-2 by crystallography.LGG-1was crystallized withatypical AlM/LIR motif, Trp-Glu-Glu-Leu (WEEL) (Noda et al., 2010). The structure was determined at $1 . 6 \mathring { \mathsf { A } }$ resolution (Table 1). The N-terminal 15 residues of LGG-2,which are not conserved in other Atg8 members and are dispensable for LGG-2 function (see below), were predicted to be intrinsically disordered and thus were truncated for crystallization.The crystal structures of LGG-2 and LGG-2 in complex with the WEEL peptide were refined to 2.1 A and $1 . 8 \mathring { \mathsf { A } }$ ，respectively (Table 1). Both LGG-1 and LGG-2 exhibit a typical Atg8-family fold,consisting of two N-terminal $\alpha$ -helixes $( a 1 , \ a 2 )$ followed by a ubiquitin fold composed of a four-stranded $\beta$ sheet and two $\alpha$ -helixes (Figures 1A,1B, S1A,and S1B).

# TheStructuresofLGG-1andLGG-2Differin the N-Termini

The overall structures of LGG-1and LGG-2 resemble each other with a RMSD of $1 . 0 \mathring { \mathsf { A } }$ for 99 Ca atoms (Figure S1C). Comparison of the apo- and peptide-bound states of LGG-2 indicates that peptide binding induces no significant conformational changes (A) Overall structure of LGG-1.(Left) ribbon model of LGG-1.N,N terminus;C,C terminus. (Middle) surface model of LGG-1.Pink residues are conserved in the GABARAP family but not the LC3 family. (Right) electrostatic potential representation of LGG-1. Red, negative charges; blue,positive charges.The three figures are in the same orientation.The bound WEEL peptide is shown as a stick model,in which C,N,and O atoms are colored yellow，blue，and red，respectively．All structural figures were prepared using PyMOL (Delano, 2002).

![](images/df6fc77bd4d8502fe2efdc38ed08e3c8b1a9a968ef60e7c316e029a4bd30a472.jpg)  
Figure 1. Crystal Structures of LGG-1 and LGG-2   
(Figure S1D).The peptide-bound structure of LGG-2 was used for furtheranalysis due to its higher quality.Structural differences are mainly located at the N-and C-terminal tails.Superimposition of the structures of LGG-1 and LGG-2 with those of other Atg8 members showed that the C-terminal tail is intrinsically flexible and divergent (Figure 1C). In the N terminus of LGG-1 and GABARAP family proteins,Met1 and the aromatic residue at position 3(Trp in LGG-1) form hydrophobic interactions with Ala108 in the ubiquitin fold,creating a rigid N-terminal conformation known as the closed form (C-form) (Figures 1D, S1E,and S1F). Transgenes expressing mutant Igg-1,witha Trp3 to Glymutation (W3G)and/oran Ala108 to Val mutation (A108V),showed greatly reducedactivityin rescuingthedefectivedegradationof protein

(B) Overall structure of LGG-2,displayed as in (A). Cyan residues in the middle figure are conserved in the LC3 family but not the GABARAP family.

(C) Overlay of the main-chain traces of seven Atg8- family proteins.Superimposition was performed bysecondary structure matching in the program COOT.PDB codes are in parentheses.

(D) N-terminal tails of LGG-1 and the GABARAP family.The side-chain of Met1,the aromatic residue at position 3,and Ala1o8 are shown as stick models.Residues are numbered according to LGG-1.

(E) N-terminal tails of LGG-2 and the LC3 family. The side-chain of the Val/Thr residue at the position equivalent to Ala108 in LGG-1 is shown as a stick model. Residues are numbered according to LGG-2.

(F)The WEEL peptide and the interacting sidechains of LGG-1 are shown as stick models. Broken lines indicate possible hydrophilic interactions. Coloring is as in Figure 1A.

(G) LGG-2/WEEL peptide interactions.

(H)Arg26-mediated interactions in LGG-2.Arg26 and its interacting residues (Arg21，Pro22，and Asp116) are shown as stick models.Possible hydrophilic interactions are shown with broken lines. Seealso Figure S1and Table 1.

substrates in Igg-1(bp500) mutants (Figures 2A-2D,S1G,and S1H), indicating that the closed N-terminal configuration isessential for lgg-1 function.The N-termini of LGG-2 and the LC3 family adopt the open form (O-form)，which is detached from the ubiquitin fold (Figures 1E，S1l,and S1J). The N-terminal region is often not modeled due to poor electron density.All the LGG-1 and LGG-2 N-terminal structures determined in this study are in the C-form and O-form,respectively (Figures S1Eand S1l);thus, their N-terminal conformational difference is not due to crystal packing artifacts.

# Structural Basis of Canonical LIR Recognition byLGG-1 and LGG-2

As Atg8-family proteins,LGG-1 and LGG-2 have the W-site and the L-site. In LGG-1, the W-site is composed of Glu17, Ile21, Pro30,Ile32，Lys48，Leu50,and Phe104，while the L-site is composed of Tyr49,Val51，Pro52,Leu55,Phe60,Leu63,and

![](images/b0d93cc83e2a1b25d3a9dc54553a9b7f881eb8e0202cd150b9f9805fa42a6a97.jpg)  
\`igure 2. Igg-2 Plays a Cargo-Specific and Developmental-Stage-Specific Role in Autophagy

(A)Seqensosddfoteddd LC3 family members are colored cyan. Secondary structure elements are shown at the top.

(B)SEPA-1::GFPaggregatesaccumulate incommastage embryos inIgg-1mutantsbutnotwild-type (Zhang etal.,2009). (C and D)SEPA-1:GFPaggregates in Igg-1mutants expressing wild-type LGG-1(C)and mutantLGG-1(W3G,A108V) (D). (E)ColocalizationofSEPA-1aggegates withLGG-1punctainwild-typeandIgg-1(bp523)embryosatthe64-to100-cel stage. (FanG)Lcaeleb

Arg67 (Figure 1F). In LGG-2, the W-site is composed of Ile33, Pro42,Lys61,Leu63,and Phe118,while theL-site is composed of Phe62,Val64,lle68,Leu73,lle76,and Arg80 (Figure 1G).In LGG-1/WEEL and LGG-2/WEEL complexes,the WEEL peptide hasan extended conformation and forms an intermolecular $\beta$ sheet with $\beta 2$ of LGG-1/2 (Figure S1K). Trp and Leu of the WEEL peptide bind to the W-site and L-site,respectively. The two glutamates in WEEL $( \mathsf { E } _ { + 1 }$ and $\mathsf { E } _ { + 2 }$ in Figures 1F and 1G) formionic interactions with Arg67 and Arg28 in LGG-1 and Arg80 and Lys40 in LGG-2,respectively. Igg-1(bp523),a mutant identified in genetic screens for defective degradation of protein aggregates (Figures S1L-S1N), contains an Arg28 to Cys mutation (Figure 2A).This mutation does not affect lipidation of LGG-1 or temporal formation of LGG-1 puncta (Figure S1O). However, SEPA-1 and SQST-1 aggregates are separable from LGG-1 puncta in Igg-1(bp523) embryos (Figures 2E-2G，S1P，and S1Q).Thus, the interaction mediated by Arg28 is essential for degradation of autophagy substrates.

# lgg-2Playsa Cargo-Specificand Developmental-StageSpecific Role in Autophagy

Two mutant alleles of Igg-2,bp556 and bp595,were identified in genetic screens for mutants with defective degradation of SQST-1 aggregates during embryogenesis (Figures 2H-2K and S2A).Degradation of PGL granules in somatic cells,including PGL-1/-3 (cargo)，SEPA-1 (receptor)，and EPG-2 (scaffold), was also defective in Igg-2 mutant embryos (Figures S2BS2H).bp556 contains an Arg to Cys mutation at amino acid 26 (Figure 2A).Arg26 in LGG-2 forms hydrophilic interactions with the main-chain of Arg21, located between α1 and $\boldsymbol { \mathfrak { a } } 2$ ,andthe side-chain of Asp116,located in the ubiquitin fold, thus contributing to the correct positioning of α2 relative to the ubiquitin fold (Figure 1H).These interactions were abolished by the R26C mutation in LGG-2(bp556).Arg26 and Asp116 are conserved in all Atg8 family members (Figure 2A). LGG-2 with the mutation R26A or D116A failed to rescue the autophagy defect in Igg-2 mutants (Figures 2L and 2M).The corresponding mutations in LGG-1，R14A and D102A,also disrupted Igg-1 function (Figure S2l).Thedeletionallele of Igg-2,tm6474,which creates an early stop codon and is a putative genetic null，was used in this study.

In lgg-2 mutants,accumulation of SQST-1 aggregates before the ${ \sim } 2 0 0$ -cell stage was similar to that in Igg-1 mutants (Figures 2H-2K and 2N). However, the number of SQST-1 aggregates at the comma stage and onward,and levels of SQST-1in extracts of late embryos,were lower than in Igg-1 mutants (Figures 2N,

2O,and S2J).The number of SQST-1 aggregates in Igg-1(RNAi) mutants was not further increased by simultaneous Igg-2 depletion (Figure 2N).At larval stages,a large number of SQST-1::GFP aggregates persisted in multiple tissues in Igg-1 mutants throughout development (Figures 2Q and 2S).In Igg-2 mutants, SQST-1:GFP aggregates accumulated in many tissues in L1 larvae but rapidlydisappearedat the L2 larval stage and onward (Figures 2R and 2T).The intestinal reporter sqst-1:gfp formed numerous aggregates in Igg-1 mutant intestine,but not in Igg-2 larval intestine (Figure S2K)．Therefore， Igg-2 is differentially required for SQST-1 degradation at different developmental stages.In contrast to SQST-1,accumulation of PGL granules and EPG-2aggregates exhibited no evident difference atlate embryonic stages in lgg-2 and lgg-1 mutants (Figures S2B and S2C). The scaffold protein EPG-7,which mediates SQST-1 degradation,is also removed by autophagy (Lin et al., 2013). EPG-7 aggregates accumulate dramatically and persist throughout embryogenesis in autophagy mutants (Figures 2U-2W)(Lin et al.,2013). In lgg-2 mutants,EPG-7 accumulated in 100-cell stage embryos but disappeared at the 2oo-cell stage onward as development proceeded (Figures 2P，2X,and S2L).Thus, lgg-2 acts ina cargo-specific manner in aggrephagy.

The role of epg-7 in degradation of SQST-1 is distinct at different embryonic stages.Compared to other autophagy mutants,epg-7mutants accumulate fewer SQST-1 aggregates prior to the $\sim 1 0 0$ -cell stage.The number of SQST-1 aggregates in epg-7 mutants is similar to other autophagy mutants at the ${ \sim } 2 0 0$ stage and onward (Lin et al.,2013), but the fluorescence intensity of SQST-1 aggregates was weaker (Figure 3A).The intensity of SQST-1 aggregates and levels of SQST-1 in Igg-2; epg-7 double mutants at late embryonic stages,however,reached a level comparable to that in lgg-1 mutants (Figures 3B-3H).At larval stages,small and weak SQST-1aggregatesaccumulated in epg-7 single mutants (Figure 3l) but were more numerous and more intense in Igg-2; epg-7 double mutants (Figure 3J). Thus,Igg-2 functions in parallel to EPG-7to promote degradation of SQST-1 aggregates at late embryonic and larval stages.

# lgg-2and Igg-1ActatDistinctGeneticStepsinthe Aggrephagy Pathway

Based on a series of genetic phenotypes (including the formation,morphology and distribution of protein aggregates,LGG-1 puncta,and ATG-9:GFP puncta),autophagy genes have been placed in a hierarchical order in the aggrephagy pathway (Lu et al.，2011). Lipidation of LGG-1 act at the most upstream step (Lu et al.，2011). In Igg-2 mutants，SQST-1 aggregates

![](images/73ef47e5bad622ebf153b8e800b41f687ba3710ff5923c4cccf7045cad6f53c8.jpg)  
Figure 3. Igg-2 and Igg-1 Act at Distinct Genetic Steps in the Aggrephagy Pathway

(A)Total fluorescence intensity of SQST-1 per focal plane in various mutants.   
(B)SQST-:GFPlevels in lte stage embryosofvarious mutants.TheamountofSQST-1divided byactinin ep-7mutantsisseas1.   
(-H)Ct (H). (C) DIC image of (D).   
( and J)Atlarval stages,more SQST-1::GFPaggregatesareformedin lgg-2;epg-7(J)thanepg-7single mutants(l)   
(K) SEPA-1 and SQST-1 aggregates are largely separable in Igg-2 mutants.   
(L) LGG-1 levels in autophagy mutants.   
(M) Number of LGG-1 puncta in wild-type and Igg-2 embryos.   
(N) LGG-1 puncta largely colocalize with SEPA-1 aggregates at the ${ \sim } 6 4$ -cell stage in lgg-2 mutants.

and PGL granules were spherical in shape,dispersed in the cyto plasm,and largely separable (Figure 3K). Levels of LGG-1-Il (lipidated LGG-1) remained unchanged,while the unlipidated form was decreased compared to wild-type animals (Figure 3L). lgg-2 mutants exhibited a wild-type pattern of LGG-1 puncta formation (Figure 3M).LGG-1 puncta colocalized with SEPA-1 at the ${ \sim } 6 4$ -cell stage in Igg-2 mutants (Figure 3N).Formation and colocalization of enlarged clusters of SEPA-1, SQST-1 aggregates,and ATG-9::GFP structures in epg-4 and epg-6 mutants was disrupted by loss of function of lgg-1,but not by lgg-2 depletion (Figures 3O-3R and S2M-S2P). Partial colocalization of SQST-1 aggregates with NUC-1::Cherry-labeled lysosomes in epg-5 mutants was disrupted by simultaneous loss of Igg-2 function (Figures S2Q-S2T).LGG-3 punctawere barelydetected in wild-type embryos (Figures 3S and 3T). In Igg-2 mutants, LGG-3 formed a large number of small spherical structures (Figure 3U),which associated with or localized closely with LGG-1 puncta (Figure S3A). Igg-1,atg-5,epg-9,and epg-6 mutants or their double mutants with Igg-2 showed no accumulation of LGG-3 puncta (Figures 3V and S3B-S3E). LGG-3 puncta also accumulated and colocalized with LGG-1 in epg-5 mutants (Figures S3F and S3G). Thus, Igg-2 acts downstream of epg-4 and epg-6,but upstream of epg-5,in the aggrephagy pathway.

# LGG-1 and LGG-2 Interact Differentially with Autophagy Substrates

LGG-1 directly binds to SQST-1 and SEPA-1 (Zhang et al., 2009; Lin et al., 2013). LGG-2 was co-immunoprecipitated by SEPA-1 and SQST-1 from extracts of embryos expressing GFP::LGG-2 (Figures 4A and S3H). Higher levels of GFP:LGG-2 than GFP::LGG-1 were co-precipitated by SEPA-1 and SQST-1 (Figures 4A-4C,S3H, and S3l).SQST-1 contains multiple LGG-1 binding sites (Lin et al.，2013). In vitro GST pull-down assays showed that LGG-1 and LGG-2 bound to SQST-1(418-499) (Figure 4D).LGG-1 also bound to SQST-1(640-630),but this was abolished by LGG-1(bp523) (Figures S3J and S3K). Multiple SEPA-1 fragments bound to LGG-1 and/or LGG-2 (Figure S3L). We furtherexamined the interaction ofLGG-1and LGG-2with EPG-7 and EPG-2. In vivo co-immunoprecipitation assays revealed that GFP::LGG-1 bound to EPG-7 (Figures 4E and 4G), while GFP::LGG-2 failed to be precipitated (Figure 4F). Both LGG-1-l and LGG-1-Il were precipitated with anti-EPG-7 (Figure S3M)． In pull-down assays，EPG-7 bound strongly to LGG-1 but weakly to LGG-1(bp523) (Figures 4H and S3N). LGG-2 did not bind to EPG-7 (Figure 4H). EPG-2 bound strongly to LGG-1 and weakly to LGG-2 (Figure S3O).

# LGG-1and LGG-2 Interact Differentially with the UNC51/EPG-1 CompleX and the LGG-3-ATG-5/ATG-16 Complex

The interaction of Atg8 with the Atg1/ULK1 complex (mediated preferentially by GABARAP family members in mammalian cells)

andwiththe Atg12/Atg5/Atg16 complexisessential forautophagy (Birgisdottir et al.， 2013；Kaufmann et al.， 2014). AntiLGG-1 precipitated EPG-1/Atg13 from extracts of embryos expressing EPG-1:GFP (Figure 4I). Pull-down assays showed that LGG-1, but not LGG-2,strongly interacted with EPG-1 (Figure 4J). EPG-1:GFP was also precipitated by anti-LGG-1 in eXtracts of atg-3 mutant embryos,suggesting that lipidation of LGG-1 is not essential for EPG-1 binding (Figure S3P). UNC-51 also directly interacted with LGG-1, but not LGG-2 (Figure 4K). UNC-51 binding was impaired by the LGG-1(bp523) mutation (Figure S3Q).Thus,components of the UNC-51/EPG-1 complex preferentially interactwithLGG-1.

In vitro pull-down assays showed that LGG-3 bound to LGG-2，but not LGG-1 (Figure 4L). Consistent with this, GFP::LGG-2， but not GFP::LGG-1，was co-precipitated by anti-LGG-3 from embryonic extracts，and less LGG-2 was precipitated when atg-5 mutant embryonic extracts were used (Figures S3R-S3T). LGG-2 was still co-precipitated by anti-LGG-3 from atg-3 mutant embryonic extracts (Figure S3U). LGG-2,but not LGG-1, also exhibited strong binding to ATG16.1 and ATG-16.2 (Figure S3V)，and this was delineated to the fifth and sixth WD repeat (Figure S3W).Therefore, LGG-2，but not LGG-1，directly interacts with the LGG-3/ ATG-5/ATG-16 complex.

# LGG-1 and LGG-2 Bind to Interacting Proteins in LIRDependentand-lndependentManners

Atg8 family members bind to substrates via the $\mathsf { X } _ { - 3 } \mathsf { X } _ { - 2 } \mathsf { X } _ { - 1 }$ $[ \mathsf { W } / \mathsf { F } / \mathsf { Y } ] _ { 0 } \mathsf { x } _ { + 1 } \mathsf { x } _ { + 2 } [ \mathsf { I } / \mathsf { L } / \mathsf { M } ] _ { + 3 }$ motif.By delineating the interacting fragments,searchingfor putative LIRs (iLIRWeb resource，Kalvari etal.,2014) and mutating individual amino acids,we determined the interaction motifs in several proteins that bind to LGG-1and LGG-2. SQST-1(470-499) bound to LGG-1 and LGG-2 (Figures 5A and S4A).Two candidate LIRs,491DFVLYDLYG,are present in this fragment.Mutating individual residues in this region revealed that $ { ^ { 4 9 1 } }  { \mathsf { D F V } }$ and $^ { 4 9 6 } \mathsf { D }$ were crucial for LGG-1 binding, while $^ { 4 9 1 } \mathsf { D }$ and $\mathsf { 4 9 5 } \mathsf { Y D L Y }$ were important for LGG-2 binding (Figures 5A and S4B). The 624FSDV LIR motif was responsible for binding of SQST-1(610-630) to LGG-1 (Figure 5B). Mutating the aromatic and other residues in the LIR motifs of SEPA-1 (101-156，107FVEV)， SEPA-1(243-288, $^ { 2 4 7 } \mathsf { F Q K I } )$ ，SEPA-1(289- 328, $^ { 2 9 8 } \mathsf { F G F V }$ ,and SEPA-1(450-480,469YQEL) greatly impaired orabolished bindingwith LGG-1and/orLGG-2 (Figures 5C and S4C-S4E). Binding of EPG-7(852-1142) to LGG-1 was mediated by the LIR motif 1015YRDV (Figure S4F). The binding of EPG-2 (1-250) and EPG-2(676-690) with LGG-1 was reduced by mutating the LIR motif in each fragment ('YSTL and 684WEMV) (Figures_S4G and S4H). The mutations F358A and F360A in the 358FTFL motif dramatically blocked the binding of UNC-51 to LGG-1,while the L361A mutation had a less severe effect (Figure 5D).Two candidate LIR motifs are present in LGG-3 (Figure S4l). The mutations F90A,E91K,and L93A in

Figure 4. LGG-1 and LGG-2 Interact Differentially with Autophagy Substrates and ATG Proteins   
![](images/aac6a89a945c259039eb0079635be51f8198a2c039a32ccac974f5ce0cd89b71.jpg)  
(A-C)GFP:-recsaiia shown in (C). Levels of GFP:LGG-1, GFP:LGG-2,and actin in extracts serve as loading controls. (D)LGG-1 and LGG-2 directly interact with the SQST-1(418-499)fragment.Lowerpanel: protein levels usedintheassay. (E-G)sciai shown in (G). (H) In a pull-down assay,LGG-1 but not LGG-2 interacts with EPG-7(852-1142). () In a colP assay using embryonic extracts,EPG-1:GFP is precipitated by anti-LGG-1. (J) LGG-1 and LGG-1(G116), but not LGG-2,interact strongly with EPG-1. (K) LGG-1 but not LGG-2 interacts with UNC-51(352-366). (L) LGG-2 but not LGG-1 directly interacts with LGG-3 in a pull-down assay. See also Figure S3.

90FETL strongly blocked binding to LGG-2 (Figure 5E).The Asp preceding_the LIRs in SEPA-1_(105DCFVEV and 246DFQKI), UNC-51(357DFTFL),and LGG-3(87DTTFETL) was also important for binding (Figures 5C-5E and S4D).The LIR motifs in the LGG-1 interacting protein AIN-1 also mediated its binding to LGG-1 and/or LGG-2 (Figure S4J).Thus, LGG-1 and LGG-2 interact with proteins via the LIR motif.Analysis of 11 LGG-1-binding LIR motifs revealed that LGG-1 binds to “W,”"F,”and “Y" type LIRs.LGG-2 prefers Phe as the aromatic residue (4/5 LIR motifs) and Asp and Thr in the positions preceding the aromatic residue (Figure 5F).The 6 LIRs that bind LGG-1 but not

LGG-2 prefer acidic residues (Asp or Glu) at the “ $+ 2 "$ position (Figure S4K).

We next investigated how the LIR sequence determines the binding specificity for LGG-1 and LGG-2.LGG-3(87-101) binds only to LGG-2,but when 9FETL was replaced with the LGG-1/ LGG-2-binding motif FVEV found in SEPA-1, the fragment bound strongly to LGG-1 (Figure 5G). The L93V mutation in 9FETLwas essential for LGG-1 binding,which was further enhanced by mutations in Eand Tat the‘ $+ 1 ^ { , , }$ and" $+ 2 "$ positions.The L93V mutation,however, greatly reduced binding to LGG-2 (Figure 5G). SQST-1(610-630) bound to LGG-1 but not LGG-2 (Figure 5H).

Mutating $^ { 6 2 4 } \mathsf { F S D V }$ to 624FVDV enabled interaction with LGG-2 (Figure 5H). Replacement of 684WEMV in EPG-2(676-690) with FVEV, however,abolished binding to LGG-1 and did not enable binding to LGG-2 (Figure S4L). Thus, both the core sequence andsurrounding residues of LIR confer binding specificity for LGG-1 or LGG-2.

LGG-1 and LGG-2 also bind to substrates independent of the canonical LIR motif.The last 14 amino acids of ATG-7，which contains no canonical LiR motif,mediated its binding to LGG-1 and LGG-2 (Figure S4M). Mutating the LIR motifs in EPG-1 and ATG-16.2 did not affect their binding to LGG-1 (Figures S4N and S4O).

# Structural Basis for the Binding Specificity of LGG-1 and LGG-2

To establish the structural basis of the binding specificity of LGG-1 and LGG-2,we determined the crystal structures of the LGG-I-UNC-51 LIR （ $^ { 3 5 6 }$ TDFTFL) complex and EEEWEEL-fused LGG-2 at resolutions of 1.8 and $2 . 3 \mathring { \mathsf { A } }$ ，respectively (Table1). The N-terminal conformations are C-form and O-form, respectively (Figures S5A and S5B). $^ { 3 5 8 } \mathsf { P h e }$ and 361Leu from the UNC-51 LIR (FTFL) bind to the W-site and L-site,respectively, similar to the WEEL peptide (Figures 5l and S5C). Compared to Phe104 in LGG-1 bound to FTFL, the benzene ring of Phe104 in LGG-1 bound to WEEL is rotated $\sim 4 5 ^ { \circ }$ to avoid the Trp of the WEEL peptide (Figure 5J).However，in WEEL-bound LGG-2，the benzene ring of Phe118 (equivalent to Phe104 in LGG-1) hasa conformation similar to Phe104 in FTFL-bound LGG-1 (Figure 5J),indicating that the W-site of LGG-1 is more plastic than that ofLGG-2. In the LGG-1W-site,Phe104 is adjacent to Gly18,while in LGG-2,Phe118 is adjacent to a more bulky amino acid,Val30,which more severely restricts the conformation of Phe118 (Figure 5J). Thus, the W-site of LGG-1 canaccommodate more divergent aromatic residues than that of LGG-2. In the case of the L-site,most of the constituent residues differ between LGG-1 and LGG-2 (Figure 5L). Thus, the L-sites differ in size and shape,which explains their different preference for Leu and Val at the‘ $\cdot _ { + 3 } \cdot \cdot$ position in the LIR (Figure 5F).

Phe360 of the UNC-51 LIR ${ \binom { 3 5 8 } { 5 } } \mathsf { F T F L } )$ forms hydrophobic interaction with Tyr25 and Leu50 of LGG-1 (Figure 5l), suggesting that LGG-1 can accommodate a hydrophobic residue at the $" + 2 "$ position of the LIR.However,isothermal titration calorimetry (ITC) assays showed that an R28C mutation in LGG-1 weakened the affinity with UNC-51 LIR and SQST-1 LIR1,even though they havea hydrophobic residue at the“ $+ 2 "$ position (Figures 5M, S5E,and S5F).Since Arg28 forms an electrostatic interaction with the carboxy terminus of Leu361 (Figure 5l)，Arg28 may contribute to LIR binding by recognizing its main-chain if an acidic residue isabsentat the‘ $+ 2 "$ position. Gln37 in LGG-2, corresponding to Tyr25 in LGG-1,does not favor binding of a hydrophobic residue at the“ $+ 2 "$ position.The ITC data showed that LGG-1 has $\sim 1 0 \mathrm { - }$ and ${ \sim } 4 0$ -foldhigheraffinitythan LGG-2 to FVLY-and FTFL-containing peptides while it has only 2-fold higher affinity than LGG-2 to WEEL- and FSDV-containing peptides (Figures 5M, S5E-S5G,and S5l).

The EEEWEEL peptide has much higher affinity than WEEL to both LGG-1 and LGG-2 in ITC assays (Figure 5M, S5G,and

S5H),indicating that the N-terminal extension of WXXL is also importantfor the interaction.In the structure ofEEEWEEL-fused LGG-2,the EEEWEEL moiety is bound to the binding pockets of another LGG-2 molecule in an intermolecular manner (Figure S5D),which is similar to the structure of LIR-fused LC3 (Suzuki etal., 2014).The N terminus of WEEL is surroundedby four basic residues，Arg20，Arg21，Arg59，and Lys61 (Figure 5K), which may recognize acidic residues in the N terminus of the LIR.Indeed,Arg21 forms adirect ionic interactionwith Gluat the“ $- 2 "$ position of EEEWEEL,and Glu at the“-1”position is close to Arg59 (Figures 5K and S5J).In contrast，when the UNC-51 LIR is bound to LGG-1, no ionic interaction is observed and instead one hydrogen bond is formed between Lys48and the main-chain of $\mathtt { T h r 3 5 6 }$ at the“ $- 2 "$ position of the LIR (Figure 5l).How LGG-1，which has no equivalent of Arg20 and Arg21in LGG-2，recognizes acidic residues in the N-terminal LIR extension remains unknown.Asn9,Lys13,Lys46,and Lys 47mightbe candidates for this interaction.Deleting the Nterminus of LGG-1 greatly impaired its binding with interacting proteins (Figure S5K).LGG-2 with a deletion of the N-terminal two $\alpha$ helixes also showed greatly reduced binding to SQST-1(418- 499)，but binding to LGG-3 and ATG-16s was not severely affected (Figures S5L and S5M).Therefore， the N-termini of LGG-1 and LGG-2 contribute differentially to substrate binding. Thus,the differences in the hydrophobic pockets and surrounding regions in LGG-1 and LGG-2 are responsible for their binding specificity to interacting proteins.

# Formation ofLGG-2 Puncta Is Regulated by LGG-1

Compared to LGG-1 puncta,far fewer LGG-2 punctawere detected in wild-type embryos at the 64-to 200-cell stage，and LGG-2 puncta were separable from LGG-1 puncta and SEPA-1 puncta (Figures 6A-6D and S6A). Formation of LGG-2 puncta required the activityof atg-3,atg-7,atg-5,and atg- $\eta _ { 6 s }$ (Figures S6B-S6E). Loss of lgg-1 activity dramatically increased the number of LGG-2 puncta that were separable from SEPA-1 aggregates (Figures 6E and S6F). In epg-3,atg-2,and epg-6 mutants, LGG-2 formed enlarged punctate structures that colocalized with LGG-1 and also with SQST-1 aggregates (Figures 6F, S6G,and S6H).Simultaneously depleting lgg-1 in epg-6 mutants resulted in the formation of small protein aggregates that are separable from LGG-2 puncta，suggesting that LGG-1 is required for the recruitment of LGG-2 to protein aggregates (Figure 6G). In epg-5 mutants, LGG-2 puncta accumulated and colocalized with LGG-1 (Figures 6Hand S6J).The lipidated form of GFP::LGG-2, like LGG-1-PE,was detected in the membrane fraction but not the soluble fraction of embryonic extracts (Fig ure S6l),indicating that lipidated LGG-2 and LGG-1 are membrane-associated.

In an in vitro reconstitution assay,LGG-1/-2\~ATG-7 and LGG$1 / - 2 \sim \mathsf { A T G } - 3$ intermediates were visible (Figure 6l). Compared to LGG-2, LGG-1(G116), in which LGG-1 ends with the glycine at position 116,was more efficient at forming an intermediate with ATG-7 (Figure 6J) but was less efficient at forming an intermediate with ATG-3 (Figure 6K).The lipidation-resistant mutant LGG-1(A116),in which the C-terminal glycine residue is mutated to Ala, still formed intermediateswith ATG-7,but not with ATG-3 (Figures 6J and 6K). Incubating ATG-7 and ATG-3 with

A 491DFVLYDLY498 693 B 624FSDV627 C 246DFQKI250 693 _702 SQST-1(470-499) 冷 Y NA SQST-1(610-630) SEPA-1(243-288) DA DFA DF D JO   
1 Y B L- L O MB Ponceau -26 Ponceau \* -26 Ponceau -26   
D 357DFTFL361 E 87DTTFETL93 F 856 118 3.0 11LGG-1bindingLIR motifs LGG-3 UNC-51(352-366) Y H 心 2.0 O B B p5yeBWePYPs58   
His- LGG-2   
LGG-1 Ponceau 次\*\*\*\* -26 Ponceau -43 -34 0.0- -26 X4XXXWXXL XX5   
G 1 90 FETL93 118 H 624FSDV 627 693 3.0 5 LGG-2 binding LIR motifs mutate to 90 FVEV93 mutate to 624FVEV627 1 LGG-3(87-101) SQST-1(610-630) 2.0 MW S MW stq T 7   
His- (kDa) His- (kDa)   
LGG-2 -15 LGG-1 -15 1.0-   
LGG-1 -15 LGG-2 -15 Ponceau -26 Ponceau -26 0.0 G18 LGG-1-UNC-51 K LGG-1-UNC-51 K13 LGG-1-WEEL GG-2-WEEL Y25 F360 LGG-2-WEEL Q37 N9 358 R21 K40 K48 L50 R28 /3 K61 F62 168 T356 E8 D35 F118 R20 Y49 F60 176 L55 K47 K46 L.36 F358 R59 R80 I64 L63 LGG-1 LGG-2 V77 M Peptide name Sequence LGG-1WT LGG-1R28C \*\*LGG-2WT \*\*LGG-2RR20,21AA Canonical LIR WEEL 36.1 ±6.2 800.0 ± 119.7 76.9 ± 5.7 N.D. Canonical LIR2 EEEWEEL 1.2 ± 0.1 20.0 ± 2.1 5.6±0.7 77.5 ± 15.9 SQST-1LIR1 EENIADFVLY 3.0± 0.3 40.8 ± 7.4 124.1 ± 25.5 N.D. SQST-1 LIR2 \*YDDNLLDFSDV 45.0±7.8 N.D. 94.3 ± 12.6 N.D. UNC-51 LIR \*YQESTDFTFL 2.9±0.1 27.3 ± 1.4 29.8± 1.7 184.5 ± 17.2 (A) Mutations in $\mathsf { \Pi } ^ { 4 9 1 } \mathsf { D F V L }$ in SQST-1(470-499) affct binding to LGG-1 and LGG-2 in a GST-pull-down assay.   
(B) Mutating “F"or $" \boldsymbol { \mathsf { V } } ^ { \flat }$ in 624FSDV abolishes binding of SQST-1(610-630) to LGG-1.   
(C) Mutating“D,”“F,”and “l" in $^ { 2 4 6 } \mathsf { D F Q K I }$ in SEPA-1(243-288) blocks its interaction with LGG-1 and LGG-2.   
(D) Mutations in the LIR motif 357DFTFLimpair binding of UNC-51(352-366)to LGG-1.   
(E) Mutations in the LIR motif 87DTTFETL reduce binding of LGG-3 to LGG-2.   
(F) Sequence logo of LIR motifs that bind to LGG-1 and/or LGG-2 using WebLogo 3(Crooks etal.,2004).   
(G) Mutating $^ { 9 0 } F E T L$ to FVEV affects binding of LGG-3(87-101) to LGG-1 and LGG-2.   
(H) Mutating $^ { 6 2 4 } \mathsf { F S D V }$ to FVEV affects binding of SQST-1(610-630) to LGG-2.   
() InteractisetweGntU-peptideBokeinicatepoiberolicinteactio.ColoingissinFgue (J) W-site structure in LGG-1-UNC-51,LGG-1-WEEL,and LGG-2-WEEL complexes.

Figure 5. LGG-1 and LGG-2 Bind to Interacting Proteins via the LIR Motif

liposomes and ATP resulted in lipidation of LGG-1 and LGG-2, which had higher mobility in gel shift assays (Figure 6L).The lipidation efficiency of LGG-1 and LGG-2 was similar under the conditions examined (Figure 6L). Lipidation of LGG-2 was unaffected by wild-type LGG-1 in this assay,but addition of LGG-1 (A116) severely reduced LGG-2-PE formation (Figures6M and 6N).

We next examined whether levels of LGG-1 lipidation can regulate LGG-2 puncta formation. In embryos expressing Plgg-1::LGG-1(G116), both LGG-1-l and LGG-1-Il accumulated, and LGG-1 formed enlarged structures at the 100- to 200-cell stages (Figures 6O and 6P) (Wu et al., 2012). PGL granules and SQST-1 aggregates ectopically accumulated and colocalized with the enlarged LGG-1 puncta,which together disappeared at the comma stage and onward (Figures S6K and S6L). Plgg-1::lgg-1(G116) dramatically reduced the number of LGG-2 puncta and also the number and intensity of LGG-2 puncta in epg-6 mutant embryos (Figures 6Q and 6R). In embryos expressing Plgg-1::LGG-1(A116)，LGG-1 was diffusely localized anda large numberofSQST-1aggregatesaccumulated (Figures S6M-S6O). LGG-2 puncta formation in epg-6; Plgg-1:LGG-1 (A116) embryoswas completelysuppressed (Figure S6P).In embryos expressing Plgg-2::LGG-2 or Plgg-2:LGG-2(A130)，no SQST-1 aggregates accumulated,and the formation of LGG-1 puncta was unaffected (Figures S6Q-S6S). These results suggest that LGG-1 regulates LGG-2 puncta formation.

# LipidatedLGG-1 andLGG-2 ShowDistinctMembrane Tetheringand Fusion Activities

We further determined whether lipidated LGG-1 and LGG-2 tether and/or fuse membranes in vitro.The lipidation process ismimicked by replacing the C-terminal glycine residue of Atg8/LC3by cysteine,which is subsequently conjugated to a maleimide group that is attached to PE (PEmaleimide)in liposomes (Figures S7A and S7B) (Weidberg et al., 2011, Nair et al., 2011). Vesicle tethering causes an increase in turbidity,which can be monitoredbymeasuring lightabsorbance at $4 0 5 \mathsf { n m }$ (Figure 7A). A fluorescence dequenching assay was used to determine fusion activity based on lipid mixing (Figure 7A). The donor vesicles contain lipids labeled with nitrobenzoxadiazole (NBD) and rhodamine at quenching concentrations； fusion with the unlabeled acceptor vesicles leads to fluorophore dilution and dequenching.We found that conjugated LGG-1 caused membrane tetheringand fusion inadose-dependent manner (Figures 7B and 7C). $5 \%$ PEmaleimide in liposomes was suffcient to trigger tethering， whereas at least $10 \%$ PEmaleimide was needed for fusion mediated by LGG-1 (Figures S7C and S7D). A physiologically relevant concentration of PE $( 2 0 \ : \mathrm { m o l } \ : \%$ was used for subsequent experiments.No fusion occurred if only the donor or acceptor vesicles contained PEmaleimide (Figure S7E), indicating that fusion requires LGG-1 molecules on both fusing membranes,and is therefore of the homotypic type.We performed an inner leaflet mixing assay to determine whether LGG-1 mediates hemi or full fusion (Xu et al., 20o5). In this assay, the membrane impermeable reductant sodium dithionite is used to quench the NBD fluorescence of the outer leaflet (Figure S7F). The increase of NBD fluorescence caused by LGG-1-mediated fusion was still detectable when sodium dithionite was added but was ${ \sim } 5 0 \%$ lower than untreated samples (Figure S7G), suggesting that LGG-1 mediates full membrane fusion.

We showed above that formation of the N-terminal closed conformation of LGG-1 depends on Trp3. Compared to wildtype LGG-1,mutant LGG-1(W3G,A4G) had slightly impaired tetheringactivity $( \sim 1 3 . 5 \%$ reduction) and moderately impaired fusion activity $( \sim 4 0 . 3 \%$ reduction) (Figures 7D and 7E). This indicated that the “C”form of LGG-1 is essential for membrane fusion. Mutating the two positively charged amino acids (Arg 14 and Arg15) in the $a 2$ helix of LGG-1 had a much more severe effect on membrane tethering $( 4 5 . 8 \%$ reduction）and fusion activity $( 8 7 . 3 \%$ reduction) (Figures 7D and 7E).A transgene expressingmutant lgg-1(R14A,R15A) failed to rescue the autophagy defect in Igg-1(bp500) mutants (Figures 7F-7J). Mutating the charged amino acids Lys2and Glu7 ofLGG-1had avery mild effect on tethering and fusion activity and only weakly reduced its rescuing activity (Figures S7H-S7J).

Since recombinant full-length LGG-2 was prone to precipitation,we examined the tethering/fusion activity of LGG-2 using an N-terminal truncation (residues 16-13O),which was functional in rescuing the autophagy defect in Igg-2 mutants (Figures 7K and S7L-S7N). Compared to LGG-1, LGG-2(Del-N15)G130C displayed weaker membrane tethering activity (Figures 7L and S7K).To confirm this difference,we performed a visual tethering assay by includinga small amount of rhodamine-labeled PE in liposomes (Figure 7M).Vesicles conjugated with LGG-1 formed big clusters (Figure 7N)，whereas vesicles conjugated with LGG-2 yielded small dots (Figure 7O). LGG-2 did not mediate membrane fusion under any of the conditions examined for LGG-1 (Figure 7P). The Arg10 and Arg11 in the α1 helix are important for the tethering and fusion ability of LC3 (Weidberg et al.,2011).Mutations of the corresponding residues in LGG-2, R20A，and R21A impaired its tethering ability (Figure 7Q). A transgene expressing mutant Igg-2(R20A，R21A) failed to rescue the lgg-2 phenotype (Figures 7K and S7O),suggesting thattetheringactivityis important for lgg-2 function.

# lgg-2RegulatesAutophagosomeSizeduring Embryogenesis

Autophagosomes are barely detected in wild-type embryos. In embryos with a mutation in epg-5,which functions downstream ofautophagosome formation,autophagosomesaccumulated and their mean diameter was ${ \sim } 5 1 0 \ \mathsf { n m }$ at the 64- to 100-cell stage (Figures 7R and 7T). In epg-5; Igg-2 double mutants,autophagosomesalso accumulated,but the diameterwas ${ \sim } 2 8 0 \mathsf { n m }$ diameter at the same stage (Figures 7S and 7T). Thus, loss of

![](images/57c60937d76d240a3fba0be675b865009bd13e5b2ad0f12ee981a0462b11fc00.jpg)  
Figure 6. Formation of LGG-2 Puncta Is Regulated by LGG-1

(A-D)Iwild-tybecaadololG(Cd.(atb (E)LossofIgg-1 function increases thenumberofLGG-2 puncta thatareseparable from SEPA-1aggregates.(A)-(E)show ${ \sim } 6 4 { - } 1 0 0$ cell embryos.   
(F) LGG-1 and LGG-2 form enlarged clusters and colocalize in epg-6 mutants.   
(G) SEPA-1 and LGG-2 puncta are separable in Igg-1; epg-6 mutant embryos.   
(H) LGG-1 and LGG-2 puncta accumulate and colocalize in epg-5 mutant embryos.   
() Purified LGG-1 or LGG-2 forms intermediates with ATG-7 and ATG-3 in the presence of ATP.   
(JanQt) are the means of three independent experiments.Error bars indicate SD.   
(L)LipidationofGG-1oGG-2thatascicubatedithG-TG-3andlposomes.HobilityansarelidatedLGG-dG-2.   
(M and N) Lipidation of LGG-2 in the presence of an equal amount of GST,LGG-1(G116)or LGG-1(A116).   
(O and P) LGG-1 forms enlarged clusters in $\sim 1 0 0 -$ to 200-cell stage embryos expressing Igg-1(G116). (O) DAPl image of (P).   
(Q Inepg-6 mutantembryos expressing Plgg-1:lgg-1(G116),thenumberandintensityofLGG-2puncta is greatlyreduced.   
(R) Number of LGG-2 puncta per focal plane in ${ \sim } 1 0 0 \ –$ to 200-cell stage embryos.   
Scale bars: ([A]-[H] and [O]-[Q]) $5 \mu \mathrm { m } ;$ (insets in[B]- $\cdot [ H ] )$ （204 $2 . 5 ~ { \mu \mathrm { m } }$ . See also Figure S6.

lgg-2 activity does not block formation of autophagosomes but reduces their size.

# DISCUSSION

# Role of LGG-1 and LGG-2 in the Aggrephagy Pathway

We show that Igg-2 is differentially involved in aggrephagy in a manner dependent on both the cargo type and developmental stage in C.elegans (Figure 7U). LGG-1and LGG-2 may act coordinately to determine cargo selectivity and degradation efficiency inaggrephagy.Different proteinaggregates vary in their size,aggregation state and in their interactions with ATG proteins,many of which bind differentially to LGG-1 and LGG-2. Aggregates of SEPA-1 and EPG-2,but not SQST-1 and EPG-7,are observed inearlyembryos.EPG-7 directly interactswithavariety of autophagy proteins (Lin et al., 2013),which may bypass the requirement for LGG-2 in degradation.LGG-1-PE and LGG-2-PE on the autophagic precursor membrane may tether the cargo/receptor complex to the membrane to facilitate engulfment.In Igg-1(bp523) mutants,in which the binding of LGG-1 with interacting proteins is impaired,LGG-1 puncta are separable from protein aggregates.Levels of lipidated LGG-1 are tightly controlled to ensure effective progression of autophagic flux.Excess lipidated LGG-1 in embryos expressing LGG-1 (G116) mediates the formation of enlarged protein aggregates, which impede the degradation process.The removal of larger ormore abundant cargoes duringembryogenesismayalso be impaired by the small autophagosomes in Igg-2 mutants.

LGG-1 and LGG-2 interact differentially with Atg proteins. LGG-1 binds to components of the UNC-51/EPG-1 complex. Loss of function of unc-51 and epg-1 impairs the association of EPG-7 with SQST-1 and also with ATG-9 during C.elegans embryogenesis (Lin et al., 2013),suggesting that recruitment of the UNC-51/EPG-1 complex by LGG-1 regulates cargo sequestration as well as autophagosome formation.LGG-2 binds to LGG-3 and ATG-16. In Igg-2 mutants,lipidation of LGG-1 is unaffected and LGG-1 still forms puncta.Thus,LGG-2 is unlikely to recruit the LGG-3-ATG-5/ATG-16 complex to facilitate LGG-1 lipidation and puncta formation.LGG-2-PE on the convex face ofthe IM may form a membrane scaffold with LGG-3-ATG-5/ ATG-16,which has been suggested to mediate expansion of IMsand tightwrapping of the membrane around the cargo (Kaufmann et al.,2014).Differential interaction of LGG-1 and LGG-2 with different Atg proteins ensures the ordered recruitment of Atgproteins to allow them to function at different steps of the autophagy pathway.

# Binding of LGG-1 and LGG-2 to the LIR Motif

LGG-1 and LGG-2 bind to the $[ \mathsf { W } / \mathsf { F } / \mathsf { Y } ] _ { 0 } \mathsf { X } _ { + 1 } \mathsf { X } _ { + 2 } [ \mathsf { I } / \mathsf { L } / \mathsf { V } ] _ { + 3 } \mathsf { L } \mathsf { I } \mathsf { R }$ motif. Acidic residues (Glu or Asp) are preferred in the positions surrounding the aromatic residue.The aromatic and $[ | / \mathsf { L } \mathsf { M } _ { + 3 }$ residues in LIR are positioned in the characteristic “W-site”and "L-site," respectively.The acidic residues adjacent to LIR form electrostatic interactions with Arg residues at the N-termini of LGG-1 and LGG-2，such as Arg28 in LGG-1 and Arg21 in LGG-2.The acidic residues at the N terminus of the LIR motif in mammalian p62 also form ionic interactions with Arg10 and Arg11 in LC3 (Birgisdottir et al.,2013). Ser orThr residuesare frequently present at the“-1,”“ $^ { - 2 }$ ,”or“ $- 3 "$ position. Phosphorylation of optineurin at Ser directly preceding the LIR enhances its interactionwith LC3B (Wild etal., 2014).

LGG-1 and LGG-2 bind differentially with autophagy substrates and Atg proteins.Binding specificity is determined by the residues both within and surrounding the LIR.Distinct from LGG-1, Atg8,and LC3 family members,LGG-2 prefers Phe as thearomatic residue in LIR,because its W-site is less flexible. The structure of the L-site pocket differs in LGG-1 and LGG-2, and the N-terminal arms of the two proteins exhibit distinct modes of interaction with the residues preceding the aromatic residue in LIR.The residues involved in substrate binding in LGG-1 and LGG-2 are conserved in the GABARAP and LC3 families，respectively.Thus，the specificity of LGG-1 and LGG-2 characterized in this study should apply to GABARAP and LC3 familymembers in other organisms.The presence of multiple Atg8homologs greatly expands theselectiveinteraction network that provides specificity for cargo and ATG protein recruitment.

# Membrane Tetheringand Fusion Activity ofLGG-1and LGG-2

Atg8-PE conjugated to liposomes has membrane tethering activityat physiologicallyrelevantPEconcentrations,but hemifusionisobservedonlyatarelatively highPE level $( 5 5 \% )$ (Nakatogawa et al., 2007; Nair et al.， 2011). Another study showed that GATE-16,Atg8,and LC3,when anchored to the membrane by PEmaleimide，mediate membrane fusion when the total PE content is $3 5 \%$ (Weidberg et al.，2011). The difference could be due to the amount of maleimide-lipid and the lipid composition of the tested liposomes.We showed that at a physiologicallyrelevantPE level $( 2 0 \% )$ ，LGG-1 possesses tetheringand fusion activity，while LGG-2 possesses only weak tethering but no fusion activity.

These differences appear to result from the unique N-terminal regions of LGG-1/GABARAP and LGG-2/LC3/Atg8.The hydrophobic amino acid Trp3 in GATE-16 and LGG-1，which is required for the C-form conformation,is essential for lipid mixing activity (Weidberg et al.,2011),suggesting that the C-form is relatedtomembrane fusionactivity.The two basicaminoacids in the $\boldsymbol { \mathfrak { a } } 2$ helix in LGG-1are also required for tetheringand fusion. The tethering and fusionactivity of LGG-1-PEappear to mediate formation of enlarged and clustered LGG-1 puncta and protein aggregates inatg-2，epg-3,epg-4,and epg-6 mutants(Tian et al.,2010; Lu et al., 2011). LGG-1 lipidation acts at the most upstream step in the aggrephagy pathway (Lu et al.,2011),which masks its role in later steps of autophagosome formation, such asautophagosome closure,a process known to be regulated by GATE-16 in mammalian cells (Weidberg et al., 2010).LC3, Atg8,and LGG-2 have an open conformation at the N terminus, and theyall control autophagosome size (Xie etal., 2oo8;Weidberg et al.,2010). LGG-2 and Atg8 show negligible fusion activity but have tethering activity,which is mediated by the N-terminal two basic residues (Arg10 and Arg11) of LC3 and the equivalent residues of LGG-2 (Arg20 and Arg21) (Weidberg et al., 2011). The tethering activity of LGG-2 is essential for its function,but its role in IM expansion is unknown. SNARE proteins are also involved in autophagosomebiogenesissuch as Atg16L1 precursor

A B C D   
Low absorbance High absorbance 1 20%PE 20uM 82 20%PE W 20uM 1.0 20%PE y\~ 0.8 Tethering \~\~ 。 。 0 2M uu   
。o 405   
No fluorescence Fluorescence 00.4 freeL-11 freeLuG-1 0. LGG-1   
NBD Rho NBI Rho 0.2- NRB 0.2 LGG-1(W3G, A4G) Fusion 0 Donor ceptor -0 20 Tme(min) 80100 20 Tme(min) 80 100 0 20 -5J 20%PE F SEPA-1::GFP G SEPA-1:GFP H SEPA-1::GFP wm M T><sis> LGG-1 LGG-1(W3G,A4G) LGG-1(R14A,R15A) H 20 40 60 80100 lgg-1 lgg-1; Plgg-1:LGG-1 Igg-1; Plgg-1:LGG-1(W3G, A4G) Time (min) SEPA-1:GFP L L LGG-2 m 0.2-   
lgg-l; Plgg-1::LGG-1(R14A,R15A) 0 2Time (min) 60   
M N o LGG-2 W 發 5   
+Buffer +LGG-1 +LGG-2 0 20 40 6080100   
Q0.6] 1 \*\*\* Time (min) 工 LGG-2 LGG-2(R20A,R21A) % 20 Time(min)60 80 epg-5 epg-5: lgg-2 0epg-5 epg-5 gg-2 U Embryonic Stage Larval Stage early stage late stage + S SQST-1 lgg-2 LGG-1-II LGG-2-II EPG-7 epg-7

(A) Illustration of the liposome tethering and fusion assays. (B and C) LGG- $1 ^ { \mathsf { G 1 1 6 C } }$ was incubated with $2 { \mathsf { m M } }$ liposomes containing $20 \%$ PEmaleimideLhtsati()andlidmiing (C)wreexamdat $3 7 ^ { \circ } \mathsf { C }$ (D and E)The tethering (D)orlipid mixing (E)activityof mutant LGG-1. $2 0 ~ \mu \mu$ of LGG- $1 ^ { \mathsf { G 1 1 6 C } }$ and mutant LGG-1 were used. $( { \mathsf { F } } - { \mathsf { J } } )$ Aclati abundantinmutantsexpresinglg(G,4G)(H)nd-45A)().QuantificatioofA-1:Gegatesisin(. (K)Numberof-1:GFregatsintantsarigtraseepringl-2g(Del-5)gg-).

(legendcontinuedonnextpage)

homotypic fusion (Nair et al., 2011; Moreau et al., 2011). The tethering and fusion activity of ATG8 family members may act coordinately with SNARE proteins to mediate distinct steps of membrane expansion and closure.

The differential and non-redundant function of Igg-1 and lgg-2 inautophagy provides insights into the distinct function of GABARAP and LC3 subfamily members in autophagy in mammalian cells. It has been largely assumed that during multicellular organism development,an autophagy gene is equally employed for removal of distinct cargo at different developmental stages (Zhang and Baehrecke, 2015). Our study demonstrates that an autophagy gene can exhibit temporal-specific and cargo-specific functions to accommodate the characteristics of different types of cargo and distinct developmental stages during multicellular organism development.

# EXPERIMENTALPROCEDURES

# Identificationof Igg-1and Igg-2Mutants

Animals carrying the sepa-1:gfp (bpls131) or sqst-1::gfp (bpls151) reporter were treated with ethyl methanesulfonate.Embryos derived from F2 animals were screened for aggregate accumulation. Igg-1(bp50o) and Igg-1(bp523) accumulated SEPA-1:GFP aggregates in late-stage embryos.Igg-2(bp556) and Igg-2(bp595)accumulated SQST-1::GFPaggregates in embryos.

# In Vitro Pull-Down Assays

cDNAs encoding full-length,fragments,or mutants of various proteins were cloned into pGEX-6p-1 (for GST tagging)，pET-28a (for His tagging)，pET21b (for His tagging),or pMal-C2X (for MBP tagging).GST fusion proteins immobilized on glutathione Sepharose beads were incubated with His-tagged proteins and then washed five times.Bound proteins were analyzed by western blot with anti-His antibody.

# In Vivo Coimmunoprecipitation Assays

Extracts of embryos carrying gfp::lgg-1,gfp::lgg-2 or epg-1:gfp were immunoprecipitated with primary antibodies and incubated with $4 0 \mu \mu \up$ protein G Sepharose beads.Rabbit IgG or anti-SEA-2 antibody was used as control IgG. After extensive washes,the immunoprecipitates were subjected to western blotwithappropriate antibodies.

# ACCESSIONNUMBERS

The atomic coordinates of the crystal structures of LGG-1-WEEL complex, LGG-1-UNC-51 LIR complex， LGG-2， LGG-2-WEEL complex，and EEEWEEL-fused LGG-2 were deposited in the Protein Data Bank under, respectively,accession codes PDB:5AZF,5AZG,5E6N,5E6O,5AZH.

# SUPPLEMENTALINFORMATION

Supplemental Information includes seven figures and Supplemental Experimental Procedures and can be found with this article online at http://dx.doi. org/10.1016/j.molcel.2015.11.019.

# AUTHORCONTRIBUTIONS

F.W.,W.F.,J.-J.H.,N.N.N.,and H.Z.designed the experiments,interpreted the data,and prepared the manuscript.F.W. performed most of the experiments. X.Q., J.-Q.R.,and W.F.determined the structures of LGG-2 and LGG-2-WEEL complex.Y.W.,Y.F.,and N.N.N.determined other structures of LGG-1 and LGG-2 and performed ITC assays.X.-Y.G.,T.-C.F.,and J.-J.H.performed membrane tethering and fusion analysis.P.W.,Z.W.，and Y.-X.S.helped with pull-down and co-immunoprecipitation (colP)assays.H.-Y.Z.contributed to the electron micrographs.H.Z. designed and supervised the study.

# ACKNOWLEDGMENTS

We are grateful to Dr. Isabel Hanson for editing work and Saori Goda for technical assistance.H.Z.was supported by grants from the NSFC (31421002, 31225018),the National Basic Research Program of China (2013CB910100), and in part by an International Early Career Scientist grant from the Howard Hughes Medical Institute;N.N.N.by Grants-in-Aid for Scientific Research on Priority Areas (25111004),Platform for Drug Discovery,Informatics,and Structural Life Science,and by CREST,JST;J.-J.H.by the NSFC (31225006); and W.F.by the National Basic Research Program of China (2014CB910202).

Received: June 22,2015   
Revised: October 16,2015   
Accepted:November12,2015   
Published:December17,2015

# REFERENCES

Birgisdottir, A.B., Lamark, T. and Johansen, T. (2013). The LIR motif - crucial for selective autophagy. J. Cell Sci. 126, 3237-3247.   
Crooks,G.E., Hon, G., Chandonia, J.M.,and Brenner, S.E. (2004). WebLogo: a sequence logo generator. Genome Res. 14， 1188-1190.   
Delano，W.L. (2002)．The PyMOL Molecular Graphics System (DeLano Scientific LLC).   
Feng,Y., He, D., Yao, Z., and Klionsky, D.J. (2014). The machinery of macroautophagy. Cell Res. 24, 24-41.   
Kalvari, I., Tsompanis, S., Mulakkal, N.C., Osgood， R., Johansen,T., Nezis, I.P.，and Promponas，V.J. (2014). iLIR: A web resource for prediction of Atg8-family interacting proteins. Autophagy 10, 913-925.   
Kaufmann,A., Beier, V.,Franquelim, H.G.,and Wollert, T. (2014).Molecular mechanism of autophagic membrane-scaffold assembly and disassembly. Cell 156, 469-481.   
Lin,L., Yang, P., Huang, X., Zhang, H., Lu, Q., and Zhang, H. (2013). The scaffold protein EPG-7 links cargo-receptor complexes with the autophagic assembly machinery. J. Cell Biol. 201,113-129.   
Lu, Q., Yang, P., Huang, X., Hu, W., Guo, B., Wu, F., Lin,L., Kovacs, A.L., Yu, L.,and Zhang, H. (2011). The WD40 repeat Ptdlns(3)P-binding protein EPG-6 regulates progression of omegasomes to autophagosomes. Dev. Cell 21, 343-357.   
Manil-Ségalen, M., Lefebvre, C., Jenzer, C.,Trichet, M., Boulogne, C., Satiat

Jeunemaitre, B.,and Legouis,R. (2014). The C. elegans LC3 acts downstream

of GABARAP to degrade autophagosomes by interacting with the HOPS subunit VPS39. Dev. Cell 28, 43-55.   
McEwan, D.G., Popovic, D., Gubas, A., Terawaki, S., Suzuki, H., Stadel, D., Coxon，F.P.，Miranda de Stegmann，D.，Bhogaraju，S.，Maddi， K.，et al. (2015). PLEKHM1regulates autophagosome-lysosome fusion through HOPS complex and LC3/GABARAP proteins. Mol. Cell 57, 39-54.   
Moreau, K., Ravikumar, B., Renna, M., Puri, C.,and Rubinsztein, D.C. (2011). Autophagosome precursor maturation requires homotypic fusion. Cell 146, 303-317.   
Nair,U., Jotwani,A., Geng,J.,Gammoh,N.,Richerson, D., Yen,W.L.,Griffith, J., Nag, S.,Wang,K., Moss,T., et al. (2011). SNARE proteins are required for macroautophagy. Cell 146, 290-302.   
Nakatogawa,H., Ichimura,Y.,and Ohsumi,Y. (2007). Atg8,a ubiquitin-like protein required for autophagosome formation, mediates membrane tethering and hemifusion. Cell 130,165-178.   
Nakatogawa, H., Suzuki, K., Kamada, Y.,and Ohsumi, Y. (2009). Dynamics and diversity in autophagy mechanisms: lessons from yeast. Nat. Rev. Mol. Cell Biol. 10, 458-467.   
Noda, N.N., Ohsumi, Y., and Inagaki, F. (2010). Atg8-family interacting motif crucial for selective autophagy. FEBS Lett. 584, 1379-1385.   
Rogov, V.,Dotsch, V., Johansen,T.,and Kirkin, V. (2014). Interactions between autophagy receptors and ubiquitin-like proteins form the molecular basis for selective autophagy. Mol. Cell 53, 167-178.   
Stolz,A., Ernst, A.，and Dikic, I. (2014). Cargo recognition and traficking in selective autophagy. Nat. Cell Biol. 16,495-501.   
Suzuki， H.,Tabata，K.， Morita， E.， Kawasaki， M., Kato，R.，Dobson，R.C., Yoshimori, T.,and Wakatsuki, S. (2014). Structural basis of the autophagyrelated LC3/Atg13 LIR complex: recognition and interaction mechanism. Structure 22, 47-58. Tian, Y., Li, Z., Hu, W., Ren, H., Tian, E., Zhao, Y., Lu, Q., Huang, X., Yang, P., Li, X., et al. (2010). C. elegans screen identifies autophagy genes specific to multicellular organisms. Cell 141, 1042-1055.   
von Muhlinen，N.，Akutsu，M.，Ravenhill，B.J.，Foeglein，A.，Bloor，S., Rutherford, T.J., Freund, S.M., Komander, D.,and Randow,F. (2012). LC3C, bound selectively by a noncanonical LIR motif in NDP52, is required for antibacterial autophagy. Mol. Cell48, 329-342.   
Weidberg, H., Shvets, E., Shpilka, T., Shimron, F., Shinder, V., and Elazar, Z. (2010). LC3 and GATE-16/GABARAP subfamilies are both essential yet act differently in autophagosome biogenesis. EMBO J.29,1792-1802.   
Weidberg,H., Shpilka, T., Shvets, E., Abada,A., Shimron, F.,and Elazar, Z. (2011). LC3 and GATE-16 N termini mediate membrane fusion processes required for autophagosome biogenesis. Dev. Cell 20, 444-454.   
Wild,P., McEwan, D.G.,and Dikic, I. (2014). The LC3 interactome at a glance J. Cell Sci. 127, 3-9.   
Wu, F.,Li,Y.,Wang,F., Noda, N.N., and Zhang, H. (2012). Differential function of the two Atg4 homologues in the aggrephagy pathway in Caenorhabditis elegans. J. Biol. Chem. 287, 29457-29467.   
Xie, Z., Nair, U., and Klionsky, D.J. (2008). Atg8 controls phagophore expansion during autophagosome formation. Mol. Biol. Cell 19, 3290-3298.   
Xu,Y., Zhang, F., Su, Z., McNew, J.A.,and Shin, Y.K. (2005). Hemifusion in SNARE-mediated membrane fusion. Nat. Struct. Mol. Biol. 12, 417-422. Yang, P.,and Zhang, H. (2014). You are what you eat: multifaceted functions of autophagy during C. elegans development. Cell Res. 24,80-91.   
Zhang, H., and Baehrecke, E.H. (2015). Eaten alive: novel insights into autophagy from multicellular model systems. Trends Cell Biol. 25, 376-387. Zhang, Y., Yan, L., Zhou, Z., Yang,P., Tian, E., Zhang, K., Zhao, Y.,Li,Z., Song, B., Han, J., et al. (20o9). SEPA-1 mediates the specific recognition and degradation of P granule components by autophagy in C. elegans. Cell 136, 308-321.