# Temporal and spatial dynamics of scaling-specific features of a gene regulatory network in Drosophila

Honggang $\mathsf { W u } ^ { 1 , 2 , 3 }$ ，Manu4， Renjie Jiao2.5 & Jun Ma1.6

A widely appreciated aspect of developmental robustness is pattern formation in proportion to size.But how such scaling features emerge dynamically remains poorly understood. Here we generate a data set of the expression profiles of six gap genes in Drosophila melanogaster embryos that differ significantly in size.Expression patterns exhibit size-dependent dynamics both spatially and temporally.We uncover a dynamic emergence of under-scaling in the posterior,accompanied by reduced expression levels of gap genes near the middle of large embryos.Simulation results show that a size-dependent Bicoid gradient input can lead to reduced Kruppel expression that can have long-range and dynamic effects on gap gene expression in the posterior.Thus,for emergence of scaled patterns,the entire embryo may be viewedas a single unified dynamic system where maternally derived size-dependent information interpreted localy can be propagated in space and time as governed by the dynamics of a gene regulatory network.

individual's body size is an intriguing feature of animal development1-4.'There are two aspects directly relevant to this developmental scaling problem-scaled tissue specification and coordinated tissue expansion5,6. In many developmental systems,the events controlling these two aspects are connected both temporally and mechanistically7-10, and recent studies have investigated mechanisms controlling the formation and action of morphogen gradients whose profiles are scaled with the expanding sizeofatissuel1-14Inother developmental systems, tissue specification and tissue expansion can take place in a temporally sequential manner.For example,formation of scaled patterns in the chick and mouse neural tube is controlled in two sequential phases,morphogen-induced progenitor specification followed by cell-type-specific growth15. In Drosophila embryos, patterning along the anterior-posterior (AP）axis can be viewed purely as a patterning problem because the physical size of the patterning system,the embryo,is predetermined at an earlier stage of the life cycle (that is,oogenesis) and does not change during the time of pattrn formation6,16. This system thus provides a unique window to probing how scaled patterns emerge from the dynamic operation of a gene regulatory network without the entanglement of growth.

Gap genes are situated at the top of the zygotic regulatory hierarchy for instructing the AP patterning outcomein Drosophila17-24Thesegenesrespondinitillytomaternal inputs such as the Bicoid (Bcd) morphogen gradient and, subsequently，rely on extensive cross-regulatory mechanisms that form an intricately wired regulatory network19,25-27. Previousstudies have generated useful insights into the regulatory mechanisms controlling the spatial and temporal dynamics ofgapgene expressio252830But howthese dyamics are related to,and impacted by,embryo size remains poorly understood. In addition，while the relationship between the scaling properties of the Bcd gradient input and the expression of the gap gene hunchback $( h b )$ has been studied quantitatively6,31, whether (and if so how） this maternal input may impact the dynamic operation of the entire gap gene network in relation to embryo size remains to be resolved.

The concept of gene regulatory networks provides a mechanistic view of how the precise and reproducible patterning and cell fate specification are controlled24.In addition to the scaling problem in embryonic pattern formation that we investigate in the current study， developmental robustnesscan also be understood in the context of other types of variability that a developmental system must face, such as molecular noise32-35 environmental fluctuations18,36,37and genetic variations38,39 While the term robustness(also often referred toas canalization） projects an effective image of developmental systems producing reliable outcomes in spite of the various uncertainties,robustness may not be viewed as an absolute concept.There is a‘sensitive' side of developmental systems that is exemplified by concentration-dependent actions of regulatory proteins24,3132461orhumandiseasescausedbyeitherhavingan extra copy or mutating a single copy of a regulatory gene42,43. Thus experimental and theoretical studies that probe molecular origins and limits of robustness in well-defined systems can lead to fundamental insights into how developmental programs are controlled.With respect to the scaling problem in Drosophila AP patterning that we investigate here, accumulating evidence suggeststhatsaingispefec5uttee molecular mechanisms are not fully resolved,particularly for the posterior part of the embryo6. Since gap genes are the earliest responders to maternal inputs and form a well-characterized regulatory network,systematically evaluating their expression properties in relation to embryo size will likely lead to new insights into how size-dependent (that is, scaling-specific) features of a patterning system emerge. Such new insights can strengthen our knowledge of the concept of developmental robustness in terms of its underlying molecular origins and limits.

Here we perform quantitative studies in Drosophila embryos that differ significantly in egg length and permit reliable extraction of scaling-specific information. We quantify the expression properties of six gap genes,orthodenticle (otd),hb, giant (gt),Kriippel $( K r )$ ，knirps $( \dot { k } n \dot { i } )$ and tailless $( t l l )$ in these embryos. Our results reveal that scaling-specific features of gap gene expression are highly dynamic in both space and time.We show that gap expression boundaries in the anterior exhibit a general over-scaling (over-scaling is defined as an excessive compensation for embryo length to give rise to a more posteriorly located boundary in large embryos relative to small embryos), which is consistent with the scaling properties of the Bcd gradient input6,31,46Theexpression boundaries inthe posterior trunk region of the embryo undergo a spatially and temporally concerted dynamic under-scaling (under-scaling is defined as an insuffcient compensation for embryo length to give rise to a more anteriorly located boundary in large embryos relative to small embryos).We perform simulation studies to probe the origins and impacts of an experimentally detected divergence in gap gene expression levels between large and small embryos. Our results support a view that the divergence in $K r$ expression level is a 'localized’ scaling-specific feature that can originate from the size-dependent properties of the Bcd gradient and exert longrange and dynamic effects on gap_gene expression behaviour in the posterior. Our study suggests that the molecular interactions (of the gap gene network) that control gene expression behaviour within individual cells (nuclei) also mediate the propagation of size-dependent ‘information’ in space and time throughout the embryo. Thus,the entire embryo may be viewed as a single unified dynamic system,in which a ‘localized' interpretation of a size-dependent morphogen input can impact the dynamics and outcome of the patterning landscape as a whole. Such a ‘holistic' view may be of general importance to studies of problems in developmental biology.

# Results

Experimental design and data set. To facilitate experimental investigation of scaling-specific properties of AP patterning, we took advantage of two Drosophila inbred lines that had beenartificiallyselectedtolaylargeandsmalleggs39,46 (hereafter referred to as the large-egg and small-egg line, respectively).Embryos from these two lines are referred to as the large and small embryos,respectively. Their mean lengths, $4 8 2 . 0 \pm 2 0 . 9 \mu \mathrm { m }$ $( \mathrm { m e a n } \pm \mathrm { s . d . } )$ and $4 0 8 . 6 \pm 1 6 . 8 \mu \mathrm { m }$ ，differ by $\sim 1 5 \%$ 1 $( P = 2 . 9 4 \times 1 0 ^ { - 2 6 }$ ； length measurements are from T3 embryos for $h b$ FISH (fluorescence in situ hybridization)； see Supplementary Table 1 for $n$ and below for further details).We used whole mount mRNA FISH to analyse the expression patterns of six gap genes that form boundaries distributed across the entire length of the embryo. Our experimental and imaging procedures were designed to avoid any nonlinear steps of signal amplification or adjustments.Under our experimental and analytical framework4l，the extracted intensities of the fluorescent signals preserve a linear relationship with the cytoplasmic mRNA concentrations (in arbitrary units,a.u.). To permit direct comparisons,both sets of embryos were treated side-by-side at both experimental and imaging steps.

To facilitate the reconstruction of the temporal evolution of gene expression patterns,we sorted embryos into well-resolved temporal classes using morphological markers.This study focuses primarily on nuclear cycles (nc) 13 and 14, during which gap gene transcripts have begun to accumulate to levels suitable for reliable quantitative measurements.This is also the time during which gap genes have begun to cross-regulate among themselves in an intensifying wayl7.For embryos at ncl4,we divided them into nine time classes ina manner to ensure that each time class had a sufficient sample_size for each and every gene analysed (Supplementary Table 1； see Methods for estimated time durations).In total, the data set generated in this study consists of data from 2,202 embryos,with 20 expression boundaries spanning across the entire AP length and across 1O time internals. Our data set permitted us to extract quantitative information about gap gene expresson with regards to (1） their spatial features (that is,expression domains and boundary positions at a given time),(2) their temporal dynamics (that is,movements of domain boundaries and threshold-crossing positionsasa function of time） and (3） their scaling-specific features (that is, relationships between the above mentioned properties and embryo size).

Spatial registry and movements of gap expression boundaries. Figure la,b shows superimposed profiles of gap gene expression asa function of relative AP position in large and small embryos,respectively. Throughout this report,relative positions along the AP axis are expressed as values of fractional embryo length, $\xi = x / L$ ，with O denoting the anterior pole and 1 the posterior pole.Our results shown in Fig. la,b yield the following spatial registry of expression boundaries for both large and small embryos along the AP axis: $k n i 1 > o t d 1 > t l l 1 >$ $h b 1 / g t 3 > t l l 2 > o t d 2 > g t 4 > h b 2 / K r 1 > K r 2 / k n i 2 > k n i 3 / g t 5 > g t 6 /$ $h b 3 > t l l 3 > h b 4$ (seeSupplementary Fig.1 for boundary nomenclature). Two additional boundaries,gtl and $g t 2$ ，which emerge at later times in ncl4,can be included in this spatial registry that would remain identical between the two sets of embryos (Supplementary Fig.1).These results show that, despite the considerable difference between their average lengths, embryos from the two inbred lines have an identical spatial registry of the expression boundaries of gap genes. They indicate that,at this initial level of analysis and consistent with the wellacknowledged framework of developmental robustness,expression patterns_of gap genes are generally robust to variations in embryo lengths.

Toanalyse quantitatively the expression boundariesof gapgenesand characterize their temporal movements during development,we measured their relative positions in individual embryos (Fig.l; Methods). Figure lc,d shows the mean positions of each of the analysed boundaries in the large and small embryos，respectively，at each of the time classes (see Supplementary Fig.2 for s.d.).These results provide a visual representation summarizing how the gap expression boundaries move along the AP axis as a function of time.We can see readily that the boundaries located in the anterior and posterior parts of the embryos tend to exhibit movements toward mid-embryo (for example,otd2 and $h b 3$ ).This is in contrast to boundaries near mid-embryo (for example,gt4,hb2 and $K r 1$ ）ornear the anterior cap $( k n i 1 )$ ,whose positions remain relatively stable as a function of time. The overall dynamic movements of gap expression boundaries in embryos from the two inbred lines are generally similar to those in other characterized wild-type strains28-30.

![](images/c50a46771f09ee46d477f36caa2e70aa10e09e96e488797d5ef468a48d67ca14.jpg)  
FigureSatilfpdoild inlarge(a)orsmall(b)embryosatthepeaktime(inexpression）ofnc14（(T2andT3combined)asafunctionofrelativeAPposition $\xi = x / L$ (0 and 1 denotenteriadsteriespetively)Fachne,tetesttaetractedfrombrlongtiserebckgroudubacted withoutanyfurtsnts.emeadosbsofttesitaoftcaedgeeotedfrachofo fmbryos.(cdoemeanpositiosofgexpresiooudariesigoalldembratthndicatedtiaeAnterid posteriorboundeoftdigptirgsspeielotetatldettllis. Colour code from a to d: red,hb;magenta,gt: green, $K r ;$ cyan,kni; blue,otd; black,tll. See Supplementary Table 1 for n.

Location-dependent scaling of gap expression boundaries.To evaluate AP patterning properties in relation to embryo length, we compared directly the positions of gap expression boundaries in large and small embryos.Our analysis reveals measurable differences between large and small embryos in either the boundary positions at a given time or their dynamic movements over time(Supplementary Fig.2,and see legend for discussions of specific examples).In addition, the moving span of a boundary (that is,the shift of a boundary's relative AP position during a period of time) can also differ between large and small embryos (Supplementary Fig.2 legend).

To quantify the relationship between AP patterning properties and embryo size,we employed a parameter,the scaling coefficient S,which was measured by pooling large and small embryos for a given time class (Methods; Supplementary Fig.3).Under our current definition6, $S = 0$ denotes perfect scaling，that is,a boundary position (expressed as a relative AP position $\xi$ ）exhibits a sample-wide insensitivity to variations in embryo length44. When $\scriptstyle { S < 0 }$ or $\scriptstyle { S > 0 }$ ，the boundary is under- or over-scaled, respectively,indicating that an increase in embryo lengthis either inadequately (under） or excessively(overly) compensated at the boundary position of a gene expression domain.Therefore,S quantifies exclusively boundary position variations derived from imperfect scaling despite the existence of other inevitable sources of noise that can also contribute to boundary variations (see Supplementary Fig.4 for results supporting the conclusion that scaling contributes to the overall precision of gap expression boundaries).

Figure 2a-t shows the S values for individual expression boundaries plotted as a function of time,with the $9 5 \%$ confidence intervals (CI） shown as shaded bands.These profiles reveal diverging scaling properties in the anterior and posterior parts of the embryo. In particular， while the gene expression boundaries in the anterior have a tendency of being over-scaled (that is, $s { > } 0$ ),those in the posterior tend to be under-scaled (that is, $S { < } 0 _ { { } , { } }$ ).In addition,expression boundaries located near the middle of the embryo (for example, $g t 4 _ { \cdot }$ ）orclose to either pole (for example,gtl,hb4) have a tendency of exhibiting a nearly perfect scaling,particularly at later times of ncl4. To provide a visual representation of these scaling trends,we generated an illustrative,superimposed plot of the S profiles for gap expression boundaries (Fig.2u).Here the over- and under-scaled boundaries become readily_perceivable. These resultsshowthat gap expression boundaries located in different parts of the embryo have different scaling-specific properties,that is,scaling is not a spatially uniform feature of the gap gene network.

Spatially and temporally concerted posterior under-scaling. A noted feature of the results shown in Fig.2 is that,for expression boundaries in the posterior trunk region of the embryo (for example,kni2,kni3 and gt6),the degree of under-scaling is the greatest around mid-ncl4 (that is, for kni3 atT5, $\bar { S } = - 0 . 1 9 \pm 0 . 0 6 ;$ error bar is $9 5 \%$ CI).At earlier times,many of these boundaries are not under-scaled, with some even exhibiting over-scaling.An example is gt6,which has the highest degree of over-scaling at T1 $\mathrm { ' } S = 0 . 3 3 \pm 0 . 1 8 \mathrm { ) }$ .But this over-scaling vanishes precipitously( $\prime S = 0 . 0 2 \pm 0 . 1 3$ at T2） to become underscaled as embryos progress further into ncl4 (for example, $S = - 0 . 1 4 \pm 0 . 0 7$ at T4).Figure 3 shows S profiles for expression boundaries detected along the AP axis at individual time classes (see Supplementary Fig.5 for a three-dimensional illustration). Here,under-scaling of gap expression boundaries in the posterior trunk region of the embryo at mid-to-late times of ncl4 exhibits itself as one of the most visible features.This feature can be further appreciated when evaluated against the S profles at earlier times (ncl3,T1 and T2) before the emergence of under-scaling for these boundaries.

Another feature displayed in Fig.3 isa‘wave-like’underscaling that emerges near mid-embryo and propagates toward the posterior. Specifically，a sudden dip (S amplitude trough) develops around middle embryo at T2.This S amplitude trough moves towards the posterior to cause a broader swath of the embryo become under-scaled.By the time of T6,the S amplitude trough has reached alocation near $\xi = \sim 0 . 8$ ，while at the same time the region near mid-embryo has been fully recovered.The $s$ amplitude trough never reaches the posterior pole region marked by the $h b 4$ boundary (see also Fig. 2t)，and portions of the posterior trunk region never recover fully.Together,our analyses of the S profiles show that the scaling-specific features of the AP patterning network emerge dynamically with regard to both space and time.

Large embryos have reduced levels of gap expression in the middle. The location- and time-dependent scaling properties of gap expression boundaries described above represent new discoveries that have not been documented previously. They suggest that,despite extensive studies,our current knowledge about how this regulatory network operates remains incomplete. To probe potential mechanisms,we gauged the effects of embryo size on spatiotemporal patterning by analysing the alterations to gap expression domain widths in time (see Fig.4a for a cartoon depicting a moving expression domain that has a ‘leading' boundary and a‘trailing' boundary in a shifting landscape).Here we measured the widths of each expression domain (expressed as values relative to embryo length) in the large and small embryos to obtain a difference,△W (Methods; Supplementary Fig. 6). Figure $4 \mathrm { b - d }$ shows that $\Delta W$ for $K r$ ，kni and gt domains in the posterior trunk region declines as a function of time,suggesting that these domains experience a greater degree of contraction in large embryos relative to small embryos.This effect for the $K r$ domain is most striking,where an initial positive $\Delta W$ becomes a negative valueatlater times of ncl4 (Fig.4b).Figure 4e,f shows that,for expression domains closer to the posterior pole (that is, the posterior $h b$ and tll domains)，△W remains generally a positive value throughout ncl4 (Supplementary Fig. 6).

To further probe potential mechanisms or events associated with the emergence of under-scaling in the posterior trunk region, we compared the expression levels of gap domains in large and small embryos (see Methods).Figure 5a shows that, relative to small embryos,large embryos exhibit properties of either decreased or progressively decreasing expression levels in the posterior trunk region.In particular, throughout nc14, $K r$ level is persistently lower in large embryos than in small embryos (Fig.5a; Supplementary Fig.1).While the expression levels of both the kni and hb domains at earlier times of ncl4 are comparable between large and small embryos, their levels in large embryos progressively become lower relative to small embryos (Fig.5a).The posterior gt domain in large embryos exhibitsa particularly dynamic decrease in expression output,with a higher level at earlier times but a lower level at later times (Fig.5a). The fact that large embryos do not have universally (gene- and timewise) low expression levels also rules out systematic experimental errors as the origin ofdifferential expression levels.Together, the dynamic differences in gap domain widths (Fig.4) and expression levels(Fig.5a） suggest that,relative to small embryos,large embryos tend to have a weaker gene expresson output particularly near the middle at the $K r$ domain.

Evaluating potential originators of posterior under-scaling. To further evaluate which part of the embryo may play an originator role in the emergence of posterior under-scaling,we compared the positions at which a mean expression profile crosses given thresholds in large and small embryos.Unlike measurements of boundary position or domain width,which utilize normalized intensity data (Fig. 4a; Methods),threshold-crossing analysis was based on unnormalized data,thus also taking into account the expression level of a gene at a time.Figure 5b shows scatter plots of threshold-crossing positions of the trailing boundaries in the posterior trunk region as well as the leading tll3 boundary (Supplementary Fig.7).Here we focus on time classes T2-T4, during which the observed under-scaling begins to emerge (Figs 2 and 3; Supplementary Fig. 5).

![](images/d1ed08e6fbee881da0fd97dcd6ca48c7f8717725a27e7bf42fdb78ebf33dc5e8.jpg)  
Figure 2丨 Temporal evolution of scaling coefficient of gap expression boundaries. $\left( \mathsf { a } \mathsf { - } \dagger \right)$ Shown are scaling coefficient S profiles for individual gap expresionutatiesdodttstdoteo posterior).A dotted line denotes $S = 0$ and shaded bands are $9 5 \%$ Cl of the linear regression slope (Supplementary Fig.3).(u) Superimposed plot of S prflesforldfrototctsaingiolouotigdaaite posterior trunk region (purple).

Our analysis of threshold-crossing positions of gap expression profiles leads to the following findings.First,when differences between large and small embryos are considered, the trailing $K r$ boundary (that is,the posterior boundary of the central $K r$ domain) moves ina manner that is earliest in time and largest in moving span (see above for definition） among all the trailing boundaries in the posterior trunk region (Fig. 5b). Second, the threshold-crossing positional differences (between large and small embryos） for leading boundaries are generally less than trailing boundaries (Supplementary Fig.7). Third and importantly, the threshold-crossing positions of the tll3 boundary do not expand appreciably more from the posterior pole in large embryos than in small embryos (Fig. 5b). These results argue against the posterior pole as the originator of under-scaling.

Long-range and dynamic impacts ofalowered $K r$ level. Our results described thus far suggest that the size-dependent divergence in gap expression levels near mid-embryo (Fig. 5a) is itself a scaling-specific feature that could potentially account for some of the observed patterning behaviour in the posterior trunk region.To directly evaluate this possibility,we performed simulation studies based on a gene circuit model of the gap gene network that correctly simulates dynamic domain boundary shifts25as_well asthe effectsof embryo-to-embryo Bcd variation19,47Herewespecificallyreducedthemaximal synthesis rates of gap proteins without any other adjustments to the model14 (Methods). In essence,our analysis was specifically designed to simulate how a reduction in gap protein levels-as a given—affects the behaviour of the operation of the gap gene network.In our analysis,we reduced the maximal synthesis rates of one protein at a time (Kr,Kni,Gt or Hb) or reduced all four simultaneously.

![](images/b01629f85375a3a4a6f466c5d2d555e72d46f0c8ed3e2c315d6526ba46d99a6c.jpg)  
Figure3calingoentprofilsunctioposiemeasuedSpofifgexpreiooudaiepotdctiof AP position at each of the 1O time classes,with blue bands representing $9 5 \%$ Cl of S.The two dotted lines denote mid-embryo (relative AP position $\xi = 0 . 5 )$ and perfect scaling $( S = 0 )$ ，respectively.

Our results are shown in Supplementary Fig. 8a-e,where protein expression profiles of different gap genes are colour-coded and plotted as a function of relative AP position.They reveal the following findings.First,individually reducing the synthesis rate of Hb or Gt in the embryo only had modest and locally restricted effctson gap geneexpression patterns inthe_posterior (Supplementary Fig. 8a,b). Second, simultaneously reducing the synthesis rates of all four proteins did not cause a strong alteration in expression boundaries (Supplementary Fig.8c).Third and importantly,individually reducing synthesis rate of either Kr or Kni led to long-range effects on gene expression boundaries in the posterior (Supplementary Fig. 8d,g).However,unlike Kr level reduction that could recapitulate qualitatively the anterior shifts of several expression boundaries (for example,Gt5,Gt6 and Hb3), reducing Kni level caused them to shift posteriorly (compare panels e and h in Supplementary Fig. 8).

Reducing Kr level on its own can dynamically affect the expression levels of other gap proteins in the posterior. While the Kni expression level is lowered throughout ncl4,Gt expression is higher at earlier times only to become lower at later times (Supplementary Fig. 8f).This simulated dynamic property is qualitatively very similar to the experimentally observed mRNA expression behaviour of these two genes in the posterior trunk region (Fig. 5a). Furthermore,reducing the synthesis rate of Kr can lower the Kni level but not the other way around (Supplementary Fig. 8f,i).Together， these results suggest that the experimentally observed Kr mRNAlevel divergence between large and small embryos is a scaling-specific feature that on its own can exert long-range and dynamic efects on gap_gene expression behaviour in the posterior (Supplementary Fig.8 legend).

![](images/c9fd67d4e7b028fd8a63053dfbf007c6c3d32f51ef251d3ce86674ff22a61a7f.jpg)  
Figure4|Damicgsingexpoats.oisaticgofeiailtratigiof widthW),totdi (expressedasvauesrlativetombrngt)etwengeadallmbssfunctiooftiSadedandforachpelepresentsdof

![](images/d6df98258d501e95d34b23c18da285cefa1e7f8e65c2eb6a63d603618e691b78.jpg)  
Figure5Largeebsebitweakeroeakenggexpesionlevelsinthposterio.aGapeneexpresiolevelsintotetrunk regionatbpielistiadsastsitesitsfinddte largeand small embryos,respectively. Error bars are s.d.Results of Student's $t$ -tests are given,with n.s.,\*and \*\* denoting not significant, $\textstyle P < 0 . 0 5$ and $P { < } 0 . 0 1$ ,respectively.(b)ShownareAPpositionsatwhichamean intensityprofilecrossesgiven thresholds.Thetrailingboundariesfor $K r ,$ kni, gt and hb as well as the leading boundary of posterior $t I I$ domainare shown.Measured boundary positions of the large and smallembryos are indicatedat the bottomof each panel for reference. Blue,large embryos; red,smal embryos.See Supplementary Table 1 for n.

Bcd input properties can impact gap gene network through $K r$ The long-range effects of Kr protein level reduction documented by our simulation studies raised an important mechanistic question. If the experimentally observed divergence in $K r$ mRNA level (Fig.5a) was indeed responsible,at least partially,for some of the scaling-specific features in the posterior trunk region,what properties might have led to such a divergence? To address this question,we turned our attention to Bcd (refs 6,31,46).One important property of the Bcd gradient is that the large-embryo profile intersects with the small-embryo profile at a critical position,such that large embryos have higher concentrations anterior to the_critical position and lower concentrations posterior to $\mathrm { i t } ^ { 3 , 4 6 }$ . The critical position lies anterior to the $K r$ domain,so that large embryos have lower Bcd concentrations at the location of $K r$ expression than small embryos46.Bcd is known to activate $K r$ reporter constructs48,49, a result supported by in silico reconstructions of the gap gene network from expression data19,25.50A positiverole forcd in $K r$ transcription suggests that Bcd gradient properties may contribute to (or be responsible for）the experimentally observed $K r$ mRNA level divergence between large and small embryos.To directly test this possibility, weperformed simulation studies where we adjusted the amplitude and the relative length scale of the Bcd gradient in ways that are consistent with experimentally observed gradient properties in large embryos.In essence,we compared two otherwise identical embryo systems except for a difference in the Bcd gradient input that is designed to mimic a difference in embryo size.

Our simulation results (Fig. 6; Supplementary Fig.9) show that the scaling properties of the Bcd input can cause Kr level reduction in the simulated large embryos.Importantly,as we moved the critical position further anterior and away from the Kr expression domain,Kr level became progressively lower in the simulated large embryos (Supplementary Fig.9).This documents a causal effect.In addition, the impact of the scaling properties of the Bcd gradient is not restricted to Kr expression level in the simulated large embryos. Other gap proteins in the posterior trunk region also exhibit reduced levels (Fig. 6j; Supplementary Fig. $| \mathfrak { g } _ { \mathrm { c } , \bar { \mathrm { f } } , \mathrm { i } } \rangle$ .Importantly，while Kr has a lower level in the simulated large embryos throughout ncl4,other gap proteins (most notably Kni and Gt） exhibit dynamic weakening in a manner that resembles the experimentally observed behaviour (Fig. 5a).

The expression boundaries of gap genes in the posterior trunk region of the simulated large embryos also exhibit differences relative to their small counterparts in their spatiotemporal dynamics.The properties of two boundaries,Kni2 and Gt5,are particularly noteworthy.These two boundaries in large embryos exhibit an anterior shift relative to small embryos (that is,they are under-scaled)at early times,but such anunder-scalingbecomes 'recovered' at later times. These simulated dynamic properties are reminiscent of the‘wave-like' initiation and recovery of underscaling observed experimentally (Fig.3; Supplementary Fig. 4). Together these simulation studies provide important mechanistic insights into how the regulatory logic of the gap gene network can permit a ‘localized' scaling-specific feature,namely $K r$ mRNA level divergence near mid-embryo, to exert its long-range effects dynamically.In addition,under the framework of the gene circuit modell9,this 'localized' scaling-specific feature near midembryo can originate from the scaling properties of the Bcd gradient.

# Discussion

Anterior-posterior patterning in Drosophila embryos represents an excellent experimental paradigm for studying regulatory network operation and developmental robustness. Despite extensive studies, it is poorly understood how the AP patterning network adapts its operation to the size of an embryo in achieving developmental robustness.Here we perform experimental and simulation studies to gain mechanistic insights into this question. We use two inbred lines that together offer an enhanced spread in embryo length within a species to allow an efficient extraction of scaling-specific features of gap expression properties.We show that scaling is neither absolute in space nor constant in time.We uncover a spatially and temporally concerted dynamic emergence of under-scaling of gap expression boundaries in the posterior trunk region,in general agreement with even-skipped (eve) expression behaviour in this part of the embryo.An important design feature of our experimental data set,which is the first of its kind, is to permit direct comparisons of gap expression levels in relation to embryo size.Our experimental and simulation results support a view that the dynamic emergenceof under-scaling in the posterior trunk originates ‘locally' as a result of lowered expression levels of gap genes near the middle of large embryos.

Our results show that gap expression boundaries located in the anterior are generally over-scaled (Fig.2),which is consistent with the scaling properties of the Bcd gradient input in this part of the embryo,46.In the anterior, the tll2 and otd2 boundaries are also sensitive to input from the terminal system51,52. These boundaries exhibit posterior movements(Fig. lc,d; Supplementary Fig.2） that coincide with an intensification of over-scaling as embryos progress into ncl4 (Fig.2).These results are consistent with the suggestion that Bcd and terminal inputs interact to,specify patterning properties in the anterior of the embryo21-23Butthescalingpropertiesof theBcd gradientare inadequate to explain the scaling-specific features in the posterior6.It thus remained an open question as to how such features might emerge dynamically and how they might be controlled. The extensive interactions between gap genes in the posteriortrunkregion195,26555 suggestthatregulatory networks and/or maternal inputs other than Bcd may play critical roles in patterning this part of the embryo.But our observed dynamic emergence of under-scaling is a new finding and,thus,a fresh conceptual framework is required to gain mechanistic insights. Our simulation results suggest that the scaling properties of the Bcd gradient input can lead to the experimentally observed $K r$ mRNA level divergence between large and small embryos. Thus long-range propagation of localized' effects may represent an important and general mechanism in dynamically guiding the emergence of size-dependent features of patterning systems.

Our current simulation studies were not designed to systematically and quantitatively fit all parameters to experimental data, an analysis that will require future efforts and data.In addition, all of the parameters used in our current simulation studies had been derived from fitting to training data sets that did not have the enhanced spread in embryo size.Naturally our simulated results are not expected to match perfectly with experimental observations for all the individual aspects even on a qualitative basis.In simulated large embryos,both the Hb2 and Krl boundaries shift posteriorly (Fig. 6a-h)，whereas either an anterior shift or no significant shift is observed at early and later times of ncl4,respectively,in the experimental data (Fig. 2k,l; Supplementary Fig.2).The posterior Hb domain is expressed at lower levels in simulations compared with experimental data. Bothof these disagreements with data occur at the edges of the modelled region,and might stem from regulators omitted in the model-the head gap genes and huckebein in the anterior and posterior，respectively.In addition， both empirical56-58and recent modelling studies59,60 suggest a complex regulatory relationship between Hb and Kr in forming the Hb2 and Krl boundaries.

![](images/d4f42c06bb5d56d442772ba39ce219cfd3c3997890886f04504dcf826cf93c3d.jpg)  
Figure6ilatifdaiineitodfgedetpufoda (earlyT),9.75(ate).6l)5(ate))4.4).6)d6(ese nucleardioSmulatiosusingontrolaeddaresoaslidodasedlinesespectivelyealingofdingeebsa based on data from Cheung et $a I . ^ { 4 6 }$ (see Methods for details).The control Bcd profile was the median profile from wild-type embryos used by Manu et $a { \cal I } . ^ { 1 9 }$ tofitthemodel.The invariance ofthe Bcd length scale in absolute units was represented by reducing the length scale by $20 \%$ -from O.1651ELtoO3nteelatieositioitstildteodelFurtherergerBdsouceusedythreasdotf bcdRNingeebasoeldyuiplgouregth)dedcetratiesofi and length scale imply a critical position at $\xi = 0 . 3 1 1$ (see Supplementary Fig.9 for simulation experiments of adjusting the critical position).Control and scaledsiulatiosredaedinedbe.eerofondassinodelutsboforetst posteriorundeepeedftdigttirgssetielyousointrodaldd areoispifpiefo Gtandabdoatrodaduaieodsdctielyt shftotntductoaaklaisaieoa Bcd simulations for Gt and Kni increases in time.

The value of our current simulation studies lies in providing afresh conceptual framework that bridges two disparate experimental observations-the long-range effects of low Kr expression and the under-scaling of Bcd in the posteriorthrough the well-documented and modelled regulatory properties of the gap_ gene network to explain the emergence of sizedependent features in the posterior. Our simulation results show that a reduced Kr level can on its own exert long-range and dynamic impacts on gap expression properties (in terms of both expression level and boundary position） in the posterior. The simulated long-range impact of $\mathrm { K r }$ level reduction is reminiscent of the long-range effects on eve patterning observed in $K r$ and $K r / +$ ,but not kni, mutant embryos61. A gene circuit model for $K r$ mutant embryos5O inferred that the anterior shift of the Gt boundaries was caused bya lack of Kr repression; our simulations suggest that a $\sim 1 . 6$ dosage of the $K r$ gene has similar effects, albeit of modest size.It is notable that the gene circuit model exhibits the Kr-mediated effects of size-dependent Bcd alteration without any additional training. Additional training with expression data from the size-selected lines would probably change the genetic interaction matrix,a zygotic property. Our simulations therefore suggest that the patterning differences originate maternally and do not require any divergence of the zygotic gap genes between the two lines.A related final point concerns the ability_of the gene circuit model to both canalize highBcdvariation21,50andexhibit posteriorunder-scalingas reported here. These apparently contradictory behaviours may be reconciled by noting that the perturbations are qualitatively different. The simulations in Manu et al. correspond to random Bcd variation in nominally isogenic embryos,whereas the perturbation modelled here is a systematic difference stemming from a large_change in embryo length between two genetically distinct lines21. That the same gene circuit underlies both types of the simulated—and empirically observed—behaviours highlights the complexity and plasticity of gene regulatory network dynamics under different perturbations.

An important suggestion of our current study is that the $K r$ mRNA level divergence in relation to size can stem from the scaling properties of the Bcd gradient at the location of the $K r$ expression domain.Thus the AP patterning behaviour in the posterior part of the embryo with respect to scaling could be viewed/explained as being controlled primarily by the dynamic operation of the gap gene network, where the system's initial state is determined by the Bcd input and mediated by $K r$ .Therole of Bcd as a positive input for $K r$ expression is well documented both experimentally and in silico19,25,48-50,and our current simulation results further support this relationship.Importantly,our study suggests that this relationship has a role in mediating the longrange and dynamic efects on gap gene expression to guide the dynamic emergence of size-dependent features in the posterior trunk regionof the embryo.This supports the view that the entire AP axis is patterned as a single unified dynamic system in which the size-driven changes in maternal positional information are interpreted locally in individual nuclei,but ina globally coordinated manner.It remains to be elucidated whether our suggested Bcd- $K r$ relationship is the sole mediator for size-dependent patterning behaviour in the posterior at a quantitative level.

Our experimental design takes advantage of two inbred lines that were selected from wild populations for laying large and small eggs39.Thefact that thespatial registry of gap expression boundaries and their temporal movements in these lines exhibit an overall similarity to one another and to other wellcharacterized wild-type strains suggests that neither of these lines is genetically‘defective' with respect to AP patterning. To further evaluate whether our observed $K r$ expression level divergence is indeed a property that is associated with embryo size (as opposed to other genetic differences between the inbred lines used in this study)，we analysed $K r$ expression level in relation to embryo length within individual lines separately. Our analysis revealsa significant (inverse） correlation in embryos from the large-egg line, though not in embryos from the smallegg line (Supplementary Fig.1Oa,b).These results document that Krlevel can diverge in embryos where size differences are not caused by genetic differences.These results also further support our suggestion that Bcd gradient properties associated with embryo size, that is,its scaling properties,can impact $K r$ expression level. We note that, for extracting scaling-specific features,within-line analyses are inherently less sensitive than between-line analyses due to the limited size variations within a line (Supplementary Fig.10).We attribute our inability to detect a significant correlation between $K r$ level and embryo length in the small-egg line to this reduced experimental sensitivity. This also further underscores the value of the experimental strategy used in our current study.

Developmental robustness is an important concept that can guide how we view and interpret the relevant events that take place on different time scales.It has been proposed that withinspecies scaling properties of the Bcd gradient can be traced fundamentally to a dynamic relationship between bcd gene copy number expansion and nurse cell size expansion during oogenesis6.Under the framework of a recently proposed model of Tissue Expansion-Modulated Maternal Morphogen Scaling $( \mathrm { T E M } ^ { 3 } \mathrm { S } ) ^ { 6 }$ ,this dynamic relationship also imposes constraints on Bcd gradient properties in relation to embryo size in different dipteran species during evolution.Relative to $D$ ，melanogaster embryos,the larger-sized $D$ ：yakuba embryos appear to exhibit both weakened gap expression levels (for $K r$ and posterior kni and gt domains） and under-scaled characteristics in the posterior45.This suggests a possibility that scaling-specific features and their mechanistic underpinnings within a species may also be applicable to evaluations of AP patterning properties across species.The early Drosophila embryo undergoes rapid nuclear divisions and morphological changes24,41,62,63.How cells (nuclei） in thisdynamic environment decodepositional information remainsanopen question27,31,32,34,64,65.The observed wave-like initiation and recovery of under-scaling in the posterior trunk region suggest that,to fully understand developmental robustness with respect to scaling,knowledge of the process of achieving the final outcome (that is,phenotype) is as important as that of the outcome itself.In addition,the temporal constraints on molecular processes imposed by the rapid morphological progression of the embryo may also be relevant to events and changes that take place on the evolutionary time scale.

# Methods

Embryo collectionand RNA FISH.The two inbred lines (#2.46.4and $\# 9 . 1 7 . 1 \rangle$ used in this study were gifts of Drs Cecelia Miles and Martin Kreitman and they had been selected from wild populations for laying large or small eggs39.Flies were reared at $2 5 ^ { \circ } \mathrm { C }$ and O-4-h embryoswere collected from 5-10-day-old females.For convenience,we refer to the embryos collected from the two inbred lines as large and small embryos,respectively.For mRNA FISH,digoxigenin (dig)-labelled RNA probes were synthesized in vitro using PCR products or plasmid (for $h b$ as templates as before66. Gene-specific PCR primers are listed in Supplementary Table 2.The primers for kni (forward: $5 ^ { \prime }$ -CGGAATTCAGTCCTTCTTTGGCCGC TCTTAC- $3 ^ { \prime }$ ；reverse: $5 ^ { \prime }$ -TAATACGACTCACTATAGGGTCTAGATTAGACA CACACGAATATTC $3 ^ { \prime }$ ）and the plasmid for synthesizing $h b$ probes (CZ4001) were as described6.In our mRNA FISH analysis (refs 33,66),the hybridization step was carried out for $4 0 \mathrm { { h } }$ at $6 0 ^ { \circ } \mathrm { C } .$ Afterhybridizationwith dig-labelled RNA probes,embryos were incubated overnight at $4 ^ { \circ } \mathrm { C }$ with sheep anti-dig primary antibody (Roche,l:4oo),followed by a 1-hour incubation at room temperature with goat anti-sheep secondary antibody conjugated with Alexa Fluor 594 (Life technologies,1:40o).DNA counterstain with 4,6-diamidino-2-phenylindole

(DAPI) (Sigma,1:1,250) was performed for $1 0 \mathrm { m i n }$ at room temperature before mounting using the Prolong Gold medium (Invitrogen).Coverslip ‘bridges'were used to minimize physical distortions and the mounted embryos on slides were le to equilibrate for at least $1 0 \mathrm { { h } }$ at $4 ^ { \circ } \mathrm { C }$ before imaging.

Image acquisition.To allow comparisons of the expression profiles of a gene between embryos from the two inbred lines,we ensured that embryo collection, mRNA FISH and imaging were all performed on a side-by-side basis.During imaging,selection criteria were used to ensure that an embryo to be imaged: (1)was at the stage of interest (ncl3 or ncl4 before gastrulation);(2)was judged to be laterally oriented with the anterior and posterior poles aligned horizontally (that is,without obvious tilt along the AP axis);and (3) did not exhibit gross deformations due to experimental manipulations or morphological defects. Image acquisition was performed on Zeiss Imager Z1 ApoTome microscope in conjunction with the AxioVision 4.8 software.All images were captured on the mid-sagittal plane of the embryo under linear settings without any normalizations or other nonlinear adjustments31.To avoid pixel intensity saturation,we first selecteda stained embryo that was judged to have the highest fluorescence signals (according to software determination) for calibrations.In particular,we used this embryo to set an exposure time such that the highest intensity pixels remained unsaturated,and this chosen exposure time,together with all the other microscopic and software settings,was fixed throughout an entire imaging cycle for all images. Therefore,absolute intensity levels for each gene could be compared between embryos from the two inbred lines,but not between genes irrespective of line.For each embryo,two images were taken,the first of which captured the fluorescent signals ( $\times 1 0$ objective,capturing the entire embryo at its mid-sagittal section), followed bythatfortheDICchannel $\mathbf { \nabla } \cdot \times 2 0$ objective,focused on the dorsal side for quantifying the progression of membrane invagination).All images (in TIF format) had $1 3 8 2 \times 1 0 4 0$ resolution and 8-bit pixel depth.

Time class classification.The nuclear cycle (ncl3 or ncl4) of an embryo was first determined by evaluating the number of nuclei on the dorsal side6.For embryos at ncl4 before gastrulation ( $\sim 6 0 \mathrm { m i n }$ into the interphase),we divided them into nine time classes based on nuclear morphology and membrane invagination.Nuclear morphology provides a sensitive measure of time for embryos at early times of ncl4 (refs 63,67),whereas membrane invagination is a reliable measure of later $\mathrm { t i m e } ^ { 2 0 , 2 9 , 6 8 , 6 9 }$ Inouranalysis,thenuclearlengthofanembryowasobtained by averaging measurements of five nuclei on the dorsal side captured by the DAPI channel.Membrane invagination ratio was calculated as a percentage of invagination depth relative to cortex length.We made measurements at three different locations on the dorsal side of an embryo and used the average for ranking embryos.Distance measurements for both nuclear length and membrane invagination depth were conducted using ImageJ.

Embryos that had $< 1 5 \%$ membrane invagination ratio on the dorsal side, estimated not to have exceeded $\sim 2 5 \mathrm { m i n }$ into the interphase67, were grouped into the early time classes Tl and T2,whose division was based on the simple ranking of the measured nuclear lengths.Embryos that had $> 1 5 \%$ membrane invagination ratio were divided into the remainder of the time classes (T3-T9) according to the simple ranking of the percentage of membrane imagination at arbitrary cut-offs (25,35,45,55,65 and $7 5 \%$ ).The chosen method led to both a fine temporal resolution and a sufficient number of embryos at individual time classes for each of the two inbred lines and for each of the genes analysed (see Supplementary Table 1).We estimated that the time classes of T3-T5 represent mid-ncl4 embryos （ $\sim 2 5 \mathrm { - } 4 0 \mathrm { m i n }$ into ncl4) and those of T6-T9 represent late ncl4 embryos 0 $\sim 4 0 { - } 6 0 \operatorname* { m i n }$ into ncl4)69.The nine time classes have an approximate time of: 0-13.5,13.5-25,25-31,31-35,35-38,38-42,42-45,45-48 and $4 8 { - } 6 0 \operatorname* { m i n }$ after the thirteenth nuclear division,respectively (see Fig.6 for further details).

Image processing to obtain gene expression profiles. Image processing was performed using Matlab(MathWorks)as follows31.Briefly,an embryo was first oriented along its AP axis with the anterior end to the left and dorsal side up using the software with manual supervision when applicable.In our analysis,the length of an embryo was first recorded and divided into 50 equal-sized bins along the AP axis; this allows the extraction of a bin-location-based intensity profile on the dorsal side of the embryo such that the extracted profile has effectively been ‘projected'onto the AP axis and expressed asa function of relative AP position irrespective of embryo size or shape.Specifically,the mRNA FISH intensities from the image of the embryo were extracted by moving a scanning circle (of 61 pixels) along the dorsal side of an embryo immediately below (basal to) the nuclear layer as detected by DAPI signals.Background signals on the DAPI channel were also used for defining the embryo bounds with manual assistance whenever necessary. Intensity values for each scanning location were expressed,in arbitrary units,as the mean of all pixels within the scanning circle and an embryo's raw intensity profile smoothed by applying a Gaussian filter.Fluorescent signals at non-expression regions were then used in an embryo-specific background subtraction across all positions,without any further adjustments unless otherwise specified.

Boundary position and movement and domain width. To determine the boundary position of a gap expression domain, the mean of three highest intensity values of a domain was set as 1 for normalizing intensity values at all positions for this domain. Boundary position was defined as the relative AP position $\langle \xi = x / L \rangle$ at which the normalized intensity value is O.5 (see Fig.4a).For the posterior tll domain,we avoided the most posterior three bins for normalization and boundary calculation to prevent capturing erroneous intensities from pole cells.We also avoided boundary calculations whenever prohibited because of low mRNA abundance that rendered such measurements unreliable.In particular,the boundaries of gtl and $g t 2$ become detectable only after T4,whereas the mRNAlevel of the anterior $h b$ expression domain decreases so rapidly during early ncl4 that the anterior boundary $( h b 1 )$ no longer exists after T4 (Supplementary Fig.1).Our analysis also did not include mRNA expression domains that never reached an appreciable level suitable for reliable boundary measurements (such as the posterior $K r$ domain).Thus the data shown in Fig.lc,d and Supplementary Fig.2 do not have boundaries for certain genes at certain times.

The width of an expression domain was defined (see Fig.4a) as the distance between its anterior and posterior boundaries (both expressed as relative AP positions).Specifically,for each time class,we first measured the domain width of an expression domain in individual embryos to obtain a mean $( \pm \mathrm { s . d . } )$ of the large or small embryos as a group, $\langle W _ { \mathrm { l a r g e } } \rangle$ and $\langle W _ { \mathrm { s m a l l } } \rangle$ ,respectively.We then calculated the width difference at each time class as $\Delta W \mathrm { { = } } \langle W _ { \mathrm { l a r g e } } \rangle - \langle W _ { \mathrm { s m a l l } } \rangle$ The s.d. of $\Delta W$ was estimated by $\sqrt { \sigma _ { 1 } ^ { 2 } / n _ { 1 } + \sigma _ { 2 } ^ { 2 } / n _ { 2 } }$ ，where $n _ { 1 }$ and $n _ { 2 }$ are sample numbers for the two groups of embryos,and $\sigma _ { 1 }$ and $\sigma _ { 2 }$ are their respective s.d. The $g t$ ant2 domain is composed of two sub-domains upon its split at later ncl4.For the posterior $^ { t l l }$ domain,the domain width is defined as the distance between the $t l l 3$ （204号 boundary and the posterior end.When calculating boundary movements as a function of time,s.d.of the difference (distance) of the means was also estimated by $\sqrt { \sigma _ { 1 } ^ { 2 } / n _ { 1 } + \sigma _ { 2 } ^ { 2 } / n _ { 2 } }$

Scaling coefficient and threshold-crossing analysis.To analyse the scaling property of an expression boundary,we pooled the large and small embryos at a given time class to generate a scatter plot of boundary position $( \xi )$ against normalized embryo length $L / \langle L \rangle$ ,where $\langle L \rangle$ is the mean of the two inbred lines'mean embryo lengths in this study.The slope of the fitted linear regression in such a scatter plot is defined as $s$ (see Supplementary Fig.3).Threshold-crossing analysis was performed using the interpolated mean (unnormalized) expression profiles of a group of embryos to obtain positions at arbitrary thresholds.Data analyses and figure generation were performed using Matlab (MathWorks) or Microsoft Excel. Unless otherwise specified,all values shown are mean $\pm \ : s . \mathrm { d }$ and all $P$ values are from two-tailed Student's t-test.

Simulations. The simulations were performed using the gene circuit model of Manuet al.19,47.hemodelsimulates thegeneregulationofHb,Kr,GtandKni in nuclei lying between 0.35 and $_ { 0 . 9 2 ~ \mathrm { E L } }$ on the AP axis of the embryo. The simulation begins after the twelfth nuclear division and lasts until gastrulation,a duration of $\mathrm { \sim } 7 1 \mathrm { m i n }$ (ref.62).The statevariablesare the concentrations of each gap protein in each nucleus, $\nu _ { i } ^ { a }$ ,where $^ a$ is the gap gene and $i$ is the nucleus. The concentrations evolve according to the coupled differential equations

$$
\begin{array} { l } { { \displaystyle { \frac { \mathrm { d } \nu _ { i } ^ { a } } { \mathrm { d } t } } = R ^ { a } g \Biggl ( \sum _ { b } T ^ { a b } \nu _ { i } ^ { b } + m ^ { a } \nu _ { i } ^ { B c d } + \sum _ { b } E ^ { a b } w _ { i } ^ { b } ( t ) - h ^ { a } \Biggr ) } } \\ { { \displaystyle ~ + D ^ { a } \left( \nu _ { i + 1 } ^ { a } + \nu _ { i - 1 } ^ { a } - 2 \nu _ { i } ^ { a } \right) - \delta ^ { a } \nu _ { i } ^ { a } , } } \end{array}
$$

where the first term represents synthesis,the second represents the diffusion of gap proteins,and the third represents first-order degradation. $R ^ { a }$ $D ^ { a }$ and $\delta ^ { a }$ are the maximum synthesis rate,diffusion coefficient,and degradation coefficient, respectively.Inthesynthesis termg(u)=1 is a sigmoidal thresholding function. $T$ is the genetic interconnectivity matrix between the gap genes,in which positive and negative elements signify activation and repression, respectively. $\nu _ { i } ^ { \mathrm { B c d } }$ is the concentration of Bcd and $m ^ { a }$ is the strength of regulation by Bcd.The model also includes the time-dependent effects of upstream regulators Cad and Tll, where $w _ { i } ^ { b } ( t )$ is the concentration and $E ^ { a b }$ is the strength of regulation. $h ^ { a }$ is a threshold term.The initial conditions for the gap genes were from ncl2 data in theFlyExdatabase (http://urchin.spbcas.ru/flyex/) $\bar { \mathbf { \Lambda } } ^ { 6 1 , 7 0 }$ Kr, Gtand Kni protein is first detected in ncl3 (ref.29),and hence their initial concentrations were zero. Cad and Tll data were from21.Except as noted below,we utilized the model reported in Manu et al.19 (model oo7)without modification,and refer the readers to that paper for the details of numerical simulation and parameter inference.The parameter values are listed in Supplementary Table 3.

We designate the unmodified model,which utilizes a Bcd profile $\nu ^ { \mathrm { B c d } } ( x ) = A$ $\exp ( - x / \lambda )$ with $A = 3 5 1$ (in a.u.) and $\lambda = 0 . 1 6 5 1$ EL,as the control simulation.For the simulation of reduced synthesis rates (Supplementary Fig.8),one of $R ^ { H b }$ $R ^ { K r }$ $\boldsymbol { R } ^ { G t }$ $R ^ { K n i }$ or all together were reduced by $2 0 \%$ For the simulations of Bcd scaling in largerembryos (Fig.6 and Supplementary Fig.9),we considered two effects.First, the length scale of Bcd does not change with embryo length46,a fact we represent byreducing the relative-unit length scale of Bcd by $2 0 \%$ --the approximate ratio of embryo lengths of large and small embryos-to 0.1321 EL in the model. Second, largerembryos were observed to have more bcd mRNA and higher protein expression at the anterior46.This was represented in the model by increasing the value of $A$ .The simultaneous reduction of relative length scale and increase in $A$ （20 causes the scaled Bcd to intersect with the control Bcd at a critical position6,46.We investigated the dependence of simulated gap gene expression patterns on the location of the critical position by varying A (Supplementary Fig.9).Larger values of $A$ shift the critical position toward the posterior and vice versa.

We measured the boundary positions and peak expression level in domains by fiting a smoothed cubic spline to the model output using the CSAPS function of MATLAB (MathWorks Inc.).Boundary positions were calculated as the position where the simulated expression level is at the mean of the consecutive extrema.

# References

1.Waddington,C.H. Canalization of development and the inheritance of aquired characters.Nature 150,563-565 (1942).   
2．Patel,N.H.& Lall, S.Precision patterning. Nature 415,748-749 (2002).   
3.Umulis,D.M.& Othmer,H.G.Mechanisms of scaling in pattern formation. Development 140,4830-4843 (2013).   
4.Lander,A.D.Pattern,growth,and control. Cell144, 955-969 (2011).   
5.Cheung,D.,Miles,C.,Kreitman,M.& Ma,J.Adaptation of the length scale and amplitude of the Bicoid gradient profile to achieve robust patterning in abnormally large Drosophila melanogaster embryos.Development 141,124-135 (2014).   
6.He,F.et al.Fundamental origins and limits for scaling a maternal morphogen gradient. Nat. Commun.6,6679 (2015).   
7.Crickmore,M.A.& Mann,R.S.The control of size in animals: insights from selector genes. Bioessays 30,843-853 (2008).   
8.Ben-Zvi, D., Shilo,B. Z.& Barkai,N. Scaling of morphogen gradients. Curr. Opin Genet.Dev.21,704-710 (2011).   
9.Wartlick,O.,Mumcu,P.,Julicher,F.& Gonzalez-Gaitan,M. Understanding morphogenetic growth control-lessons from flies.Nat.Rev.Mol. Cell Biol. 2, 594-604 (2011).   
10.Rogers,K.W.& Schier,A.F.Morphogen gradients: from generation to interpretation.Annu.Rev. Cell Dev. Biol.27,377-407 (2011).   
11. Wartlick,O.etal. Dynamics of Dpp signaling and proliferation control. Science 331,1154-1159 (2011).   
12.Hamaratoglu,F.,de Lachapelle,A.M.,Pyrowolakis,G.,Bergmann, S.& Affolter,M.Dpp signaling activity requires Pentagone to scale with tissue size in the growing Drosophila wing imaginal disc.PLoS Biol.9,el001182 (2011).   
13.Ben-Zvi,D.,Pyrowolakis,G.,Barkai,N.& Shilo,B.Z. Expansion-repression mechanism for scaling the Dpp activation gradient in Drosophila wing imaginal discs.Curr. Biol.21,1391-1396 (2011).   
14.Fried,P.& Iber,D.Dynamic scaling of morphogen gradients on growing domains.Nat. Commun.5,5077 (2014).   
15.Kicheva,A.et al. Coordination of progenitor specification and growth in mouse and chick spinal cord. Science 345,1254927 (2014).   
16.Spradling,A.C.in The Development of Drosophila melanogaster. (eds Bates,M. & Martinez-Arias,A.) 1-70 (Cold Spring Harbor Press,1993).   
17.Jaeger,J.,Manu & Reinitz,J. Drosophila blastoderm patterning. Curr. Opin. Genet. Dev.22,533-541 (2012).   
18.Houchmandzadeh,B.,Wieschaus,E.&Leibler,S.Establishment of developmental precision and proportions in the early Drosophila embryo. Nature 415,798-802 (2002).   
19.Manu et al. Canalization of gene expression in the Drosophila blastoderm by gap gene cross regulation.PLoS Biol.7,e1000049 (2009).   
20.Dubuis,J.O.,Samanta,R.& Gregor,T.Accurate measurements of dynamics and reproducibility in small genetic networks.Mol. Syst. Biol.9,639 (2013).   
21.Porcher,A.& Dostatni,N.The bicoid morphogen system. Curr. Biol. 20, R249-R254 (2010).   
22.Lohr, U., Chung,H.R.,Beller,M.& Jackle,H. Bicoid-morphogen function revisited.Fly 4,236-240 (2010).   
23.Chen,H.,Xu, Z.,Mei, C.,Yu,D.& Small,S.A system of repressor gradients spatially organizes the boundaries of Bicoid-dependent target genes.Cell 149, 618-629 (2012).   
24.Gebelein,B.& Ma,J.Regulation in the early Drosophila embryo.Rev.Cell Biol. Mol.Med.(in press) (2015).   
25.Jaeger,J.et al.Dynamic control of positional information in the early Drosophila embryo.Nature 430,368-371 (2004).   
26.Bieler,J.,Pozzorini, C.& Naef,F.Whole-embryo modeling of early segmentation in Drosophila identifies robust and fragile expression domains. Biophys.J.101,287-296 (2011).   
27.Bergmann,S.et al.Pre-steady-state decoding of the Bicoid morphogen gradient.PLoS Biol.5,e46 (2007).   
28.Fowlkes,C.C.et al.A quantitative spatiotemporal atlas of gene expression in the Drosophila blastoderm.Cell 133,364-374 (2008).   
29.Surkova,S.et al. Characterization of the Drosophila segment determination morphome.Dev.Biol. 313,844-862 (2008).   
30.Crombach,A.,Wotton,K.R.,Cicin-Sain,D.,Ashyraliyev,M.& Jaeger,J. Efficient reverse-engineering of a developmental gene regulatory network.PLoS Comput. Biol.8,e1002589 (2012).   
31.He,F.et al. Probing intrinsic properties of a robust morphogen gradient in Drosophila.Dev.Cell15,558-567 (2008).   
32.Gregor,T.,Tank,D.W.,Wieschaus,E.F.& Bialek,W.Probing the limits to positional information. Cell 130,153-164 (2007).   
33.He,F.,Ren,J,Wang,W.& Ma,J.Evaluating the Drosophila Bicoid morphogen gradient system through dissecting the noise in transcriptional bursts. Bioinformatics 28,970-975 (2012).   
34.Tostevin,F.,ten Wolde,P.R.& Howard,M.Fundamental limits toposition determination by concentration gradients.PLoS Comput.Biol.3,e78 (2007).   
35.He,F.et al. Shaping a morphogen gradient for positional precision. Biophys. J. 99,697-707 (2010).   
36.Cheung, D.& Ma, J. Probing the impact of temperature on molecular events in a developmental system. Sci. Rep.5,13124 (2015).   
37.Kuntz, S.G.& Eisen,M.B.Drosophila embryogenesis scales uniformlyacross temperature in developmentally diverse species.PLoS Genet.10,el0o4293 (2014).   
38.Lott,S.E., Kreitman,M.,Palsson,A.,Alekseeva,E.& Ludwig,M. Z. Canalization of segmentationand its evolution in Drosophila.Proc.Natl Acad. Sci.USA 104,10926-10931 (2007).   
39.Miles, C.M.etal.Artfialseletionongszeperturbsearltter formation in Drosophila melanogaster.Evolution 65,33-42 (011).   
40.Driever,W.& Nuslein-Volhard, C.The bicoid protein determines position in the Drosophila embryo in a concentration dependent manner. Cell 54,95-104 (1988).   
41.Liu,J.& Ma,J.Dampened regulates the activating potency of Bicoid and the embryonic patterning outcome in Drosophila. Nat. Commun.4,2968 (2013).   
42.Conrad, B.& Antonarakis,S.E. Gene duplication: a drive for phenotypic diversity and cause of human disease.Annu.Rev. Genomics Hum. Genet. 8, 17-35 (2007).   
43.Semina,E.V.et al.Cloning and characterization of a novel bicoid-related homeobox transcription factor gene,RIEG,involved in Rieger syndrome. Nat. Genet.14,392-399 (1996).   
44.de Lachapele, A. M. & Bergmann, S. Precision and scaling in morphogen gradient read-out.Mol. Syst. Biol.6,351 (2010).   
45.Fowlkes,C.C.et al.A conserved developmental patterning network produces quantitatively diferent output in multiple species of Drosophila.PLoS Genet.7, e1002346 (2011).   
46.Cheung,D.,Miles,C.,Kreitman,M.& Ma,J.Scaling of the Bicoid morphgen gradient by a volume-dependent production rate.Development 138,2741-2749 (2011).   
47.Manu et al. Canalization of gene expression and domain shifts in the Drosophila blastoderm bydynamical atractors.PLoS Comput. Biol.5, e1000303 (2009).   
48.Hoch,Mhoder,Ceifert,E.&Jackle,H.Cisacingotroleleetsfor Kruppel expression in the Drosophila embryo.EMBO J. 9,2587-2595 (1990).   
49.Hoch,M.,Seifert,E.& Jackle,H. Gene expression mediated bycis-acting sequences of the Kruppel gene in response to the Drosophila morphogens bicoid and hunchback.EMBO J.10,2267-2278 (1991).   
50.Kozlov,Kk.soEez,Jeli of gap gene expression in Drosophila Kruppel mutants.PLoS Comput. Biol. 8, e1002635 (2012).   
51. Kim, Y.et al. Context-dependent transcriptional interpretation of mitogen activated protein kinase signaling in the Drosophila embryo.Chaos 23,025105 (2013).   
52.Lohr,U., Chung,H.R.,Beller,M.& Jackle,H.Antagonistic action of Bicoid and the repressor Capicua determines the spatial limits of Drosophila head gene expression domains.Proc.Natl Acad.Sci. USA 106,21659-21700 (2009).   
53.KrotovD,ubis,J.O.Gregor.&ialek,W.ogeesisatiialit Proc. Natl Acad.Sci. USA 111,3683-3688 (2014).   
54.Papatsenko,D.& Levine, M.The Drosophila gap gene network is composed of two parallel toggle switches.PloS ONE 6,e21145 (2011).   
55.Vakulenko,S.,Manu,Reinitz,J.& Radulescu,O.Size regulation in the segmentation of Drosophila: interacting interfaces between localized domains of gene expression ensure robust spatial patterning.Phys.Rev. Lett.103,168102 (2009).   
56.Struhl,G.,Johnston,P.& Lawrence,P.A.Control of Drosophila body pattern by the hunchback morphogen gradient. Cell 69,237-249 (1992).   
57.Hulskamp,M.,Pfeifle, C.& Tautz,D.A morphogenetic gradient of hunchback protein organizes the expression of the gap genes Kruppel and Knirps in the early Drosophila embryo.Nature 346,577-580 (1990).   
58.Schulz, C.& Tautz,D. Autonomous concentration-dependent activation and repression of Kruppel by hunchback in the Drosophila embryo.Development 120,3043-3049 (1994).   
59.Holloway,D.M.& Spirov,A. V. Mid-embryo paterning and precision in Drosophila segmentation: Kruppel dual regulation of hunchback. PloS ONE 10, e0118450 (2015).   
60.Papatsenko, D.& Levine, M. S. Dual regulation by the Hunchback gradient in the Drosophila embryo.Proc.Natl Acad.Sci.USA 105,2901-2906 (2008).   
61.Surkova,S.et al. Quantitative dynamics and increased variability of segmentation gene expression in the Drosophila Kruppel and knirps mutants. D:127 n 11712)   
62.Foe,V.E.& Alberts,B.M.Studies of nuclear and cytoplasmic behaviour during the five mitotic cycles that precede gastrulation in Drosophila embryogenesis.J. Cell Sci.61,31-70 (1983).   
63.Liu,J.&Ma,J. Uncovering a dynamic feature of the transcriptional regulatory network for anterior-posterior patterning in the Drosophila embryo.PloS ONE   
8,e62641 (2013).   
64.He,F.& Ma,J.A Spatial point pattern analysis in Drosophila blastoderm embryos evaluating the potential inheritance of transcriptional states.PloS ONE   
8,e60876 (2013).   
65.Porcher,A.et al.The time to measure positional information:maternal hunchback is required for the synchrony of the Bicoid transcriptional response at the onset of zygotic transcription.Development 137,2795-2804 (2010).   
66.Liu,J.& Ma,J.Fates-shifted is an F-box protein that targets Bicoid for degradation and regulates developmental fate determination in Drosophila embryos.Nat. Cell Biol.13,22-29 (2011).   
67.Fung,J.C.,Marshall,W.F.,Dernburg,A.,Agard,D.A.& Sedat,J.W. Homologous chromosome pairing in Drosophila melanogaster proceeds through multiple independent initiations.J. Cell Biol.14l,5-20 (1998).   
68.Merrill,P.T.,Sweeton,D.& Wieschaus,E.Requirements for autosomal gene activity during precellular stages of Drosophila melanogaster.Development104,   
495-509 (1988).   
69.Royou,A.,Field, C.,Sisson,J.C.,Sullivan,W.&Karess,R.Reassessing the role and dynamics of nonmuscle myosin II during furrow formation in early Drosophila embryos.Mol.Biol.Cell15,838-850 (2004).   
70.Pisarev,A.,Poustelnikova,E.,Samsonova,M.&Reinitz,J.FlyEx,the quantitative atlas on segmentation gene expression at cellular resolution. Nucleic Acids Res.37,D560-D566 (2009).

# Acknowledgements

We thank Drs Cecelia Miles and Martin Kreitman for providing the inbred lines used in this study.We thank members of the Ma lab,in particular Dr Feng He,for discussions

and technical assistance.This work was supported in part by1RoiGMlo1373 from NIH and IOS-0843424 from NSF (to J.M.),and by IIA-1355466,project UND0019821 from NSF ND EPSCoR (to M.).

# Author contributions

H.W.designed and performed experiments,analysed data, interpreted results,generated figures and wrote and approved the paper.M.designed and performed simulation studies,analysed data,interpreted results,generated figures,contributed to writing and approved the paper.R.J.contributed resources and approved the paper. J.M.conceived the study,designed experimental and simulation studies,interpreted results and wrote and approved the paper.

# Additional information

Supplementary Information accompanies this paper at http://www.nature.com/ naturecommunications

Competing financial interests: The authors declare no competing financial interests.

Reprints and permission information is available online at http://npg.nature.com/ reprintsandpermissions/

How to cite this article:Wu,H.et al. Temporal and spatial dynamics of scaling-specific features of a gene regulatory network in Drosophila.Nat.Commun.6:10031 doi: 10.1038/ncomms10031 (2015).

This work is licensed under a Creative Commons Attribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license,visit http://creativecommons.org/licenses/by/4.0/