the homologies between NLRC4 oligomerizationand prion replication may help to better understand their underlying mechanisms.It remains unknown whether the prion-like mechanism is conserved in other immune NLR sensors for oligomerization.However,one apparent advantageofthismechanismisthata single PAMP or a host-derived danger molecule is sufficient for inducing formation of a fiber assembly,which in principle contains endless ASC because of its self-perpetuation property (39) and thus generates an all-or-none response (4O).The requirement of ligand for initiation of this activity could serve to minimize inadvertent activationofanNLR proteinthrough intermolecular collisions.

# KEFEKENUES AND NUIES

1.K. Schroder,J.Tschopp, Cel/140,821-832 (2010).   
2.L.Franchi, R. Munoz-Planillo,G. Nunez, Nat. Immunol.13, 325-332 (2012).   
3.V.A. Rathinam, S.K.Vanaja, K.A. Fitzgerald, Nat. Immunol.13, 333-342 (2012).   
4.J.von Moltke,J.S.Ayres,E.M. Kofoed,J.Chavarra-Smith, R.E.Vance,Annu. Rev. Immunol.31,73-106 (2013).   
5.B. K. Davis, H.Wen,J.P.Ting,Annu. Rev. Immunol. 29, 707-735 (2011).   
6．M.Lamkanfi, V.M. Dixit,Annu. Rev. Cell Dev. Biol.28,137-161 (2012).   
7.T.Strowig,J.Henao-Mejia,E.Elinav,R.Flavell,Nature 481, 278-286 (2012).   
8．H.Wen,J.P.Ting,L.A. O'Neill,Nat. Immunol. 13,352-357 (2012).   
9.F.Martinon,K.Burns,J.Tschopp,Mol. Cell10,417-426 (2002).   
10.O.Danot,E.Marquenet,D.Vidal-lngigliardi,E.Richet,tructure 17, 172-182 (2009).   
11.N.Yan,Y.Shi,Annu.Rev.Cell Dev.Biol.21,35-56 (2005).   
12.W. Chuenchor,T.Jin, G. Ravilious,T.S. Xiao, Cur. Opin. Immunol. 26,14-20 (2014).   
13.Z.Hu et al.,Science 341,172-175 (2013).   
14.X. Cai et al., Cell156,1207-1222 (2014).   
15.A. Lu et al., Cell156,1193-1206 (2014).   
16.L.Franchi et al., Nat. Immunol.7, 576-582 (2006).   
17.K.L.Lightfield et al., Nat.Immunol.9,1171-1178 (2008).   
18.E.A. Miao et al., Nat.Immunol.7,569-575 (2006).   
19.E.A. Miao etal.,Proc.Natl.Acad.Sci.U.S.A.107,3076-3080 (2010).   
20.A.B.Molofsky et al.,J.Exp.Med.203,1093-1104 (2006).   
21.T.Ren,D.S.Zamboni,C.R.Roy,W.F.Dietrich,R.E.Vance, PLOS Pathog.2,e18 (2006).   
22.F.S. Sutterwala et al.,J.Exp.Med.204,3235-3245 (2007).   
23.J.Yang,Y. Zhao,J.Shi,F. Shao,Proc. Natl.Acad. Sci.U.S.A. 110, 14408-14413 (2013).   
24.E.M. Kofoed,R.E.Vance,Nature 477,592-595 (2011).   
25.Y. Zhao et al.,Nature 477, 596-600 (2011).   
26.J.L.Tenthorey,E.M.Kofoed, M.D.Daugherty，H.S.Malik, R.E. Vance,Mol. Cell 54,17-29 (2014).   
27.E.F. Halff et al.,J.Biol.Chem.287,38460-38472 (2012).   
28.S.Yuan,M.Topf,T.F.Reubold,S.Eschenburg,C.W.Akey, Biochemistry52,2319-2327(2013).   
29.S.Yuan et al.,Structure 19,128-140 (2011).   
30.Y.Pang et al., Genes Dev.29,277-287 (2015).   
31. S. Qi et al., Cel/141, 446-457 (2010).   
32.T.F.Reubold,S.Wohlgemuth,S.Eschenburg,Structure 19, 1074-1083 (2011).   
33.R.E.Vance,Cur. Opin.Immunol.32,84-89 (2015).   
34.L. Zhang et al.,Science 350,404-409 (2015).   
35.P.Chien,J.S.Weissman,A.H.DePace,Annu.Rev.Biochem. 73,617-656 (2004).   
36.S.W. Canna et al.,Nat. Genet.46,1140-1146 (2014).   
37.N. Romberg et al., Nat. Genet. 46,1135-1139 (2014).   
38.A.Kitamura,Y. Sasaki,T. Abe,H. Kano,K. Yasutomo,J. Exp. Med.211,2385-2396 (2014).   
39.X.Cai,Z.J.Chen,Trends Immunol.35,622-630 (2014).   
40. Q. Yin,T. M.Fu, J. Li, H. Wu,Annu. Rev. Immunol.33,393-416 (2015).

# ACKNOWLEDGMENTS

We thank Y. Xu and T. Yang at Tsinghua University Branch of China National Center for Protein Sciences Beijing for data collection and computational support,and X. Li for technical instructions of using the K2-Summit camera.We acknowledge the computational facility support on the Explorer lOO cluster system of Tsinghua National Laboratory for Information Science and Technology.The data presented in this paper are tabulated in the main paper and in the supplementary materials.The EM maps have been deposited in the Electron Microscopy Data Bank with accession codes EMD-3139/3140 for the 12-or 11-protomer PrgJ-NAIP2-NLRC4F- complex, EMD-3141/3142 for the 1l- or 10-protomer PrgJ-NAIP2-NLRC4CARDcomplex,and EMD-3143 forthe

FliCDO-NAI5- was funded by the Chinese Ministry of Science and Technology (2014CB910101 to J.C.,2011CB910501 to S.-F.S.,and 2012CB917303 to H.-W.W.) and the National Natural Science Foundation of China (31230016 to S.-F.S.). Z.H. was supported by a China Postdoctoral Science Foundation-funded project and the Center for Life Sciences (CLS) Postdoctoral Fellowship Foundation. Q.Z. was supported by the CLS Postdoctoral Fellowship Foundation.

# SUPPLEMENTARYMATERIALS

www.sciencemag.org/content/350/6259/399/suppl/DC1   
Materialsand Methods   
Figs. Sl to S18   
Table S1   
References (41-56)   
1l May 2015;accepted 14 September 2015   
Published online 8 October 2015   
10.1126/science.aac5489

# INNATEIMMUNITY

# Cryo-EM structure of the activated NAIP2-NLRC4 inflammasome reveals nucleated polymerization

Liman Zhang,1,2\* Shuobing Chen,3,4\* Jianbin Ruan,1,2 Jiayi Wu,3,4 Alexander B. Tong,1,² Qian Yin,1,2 Yang Li,12 Liron David,1,2 Alvin Lu,1,2 Wei Li Wang,4,5 Carolyn Marks,6 Qi Ouyang,' Xinzheng Zhang,7 Youdong Mao,3,4,5 Hao Wu1,2+

The NLR family apoptosis inhibitory proteins (NAlPs) bind conserved bacterial ligands, such as the bacterial rod protein PrgJ,and recruit NLR family CARD-containing protein 4 (NLRC4)as the inflammasome adapter to activate innate immunity.We found that the PrgJ-NAlP2-NLRC4 inflammasome is assembled into multisubunit disk-like structures through a unidirectional adenosine triphosphatase polymerization,primed with a single PrgJ-activated NAlP2 per disk. Cryo-electron microscopy (cryo-EM) reconstruction at subnanometer resolution revealed a $\sim 9 0 ^ { \circ }$ hinge rotation accompanying NLRC4 activation. Unlike in the related heptameric Apaf-1 apoptosome,in which each subunit needs to be conformationally activated by its ligand before assembly,a single PrgJ-activated NAIP2 initiates NLRC4 polymerization in a domino-like reaction to promote the disk assembly. These insights reveal the mechanism of signal amplification in NAlP-NLRC4 inflammasomes.

he nucleotide-binding domain (NBD) and   
1 leucine-rich repeat (LRR)-containing protein (NLR)family participates in the formation of inflammasomes that activate caspase-1 for cell death induction and cy  
tokine maturation.NLR family apoptosis inhib  
itoryproteins (NAIPs)are so far the only NLR   
familymemberswith specificallydefinedligands   
(1-4).NAIP2 detects the inner rod protein of the   
bacterial type III secretion system,including   
SalmonellatyphimuriumPrgJ,whereasNAIP5   
andNAIP6 detect bacterial flagellin such as   
Salmonella typhimurium FliC (2,4,5).NLR   
family caspase recruitment domain (CARD)-   
containing protein 4 (NLRC4) was initially found   
to participate in caspase-1 activation and inter

leukin (IL)- $\cdot 1 \beta$ secretion in response to cytoplasmic flagellin (6)and was only recently shown to be the common adapter for NAIPs (2,4). The NAIP-NLRC4 inflammasomes perform effector functions against intracellular bacteria (7, 8), playprotective roles in mouse models of colitisassociated colorectal cancer(9,IO),and serve asa potential strategy in tumor immunotherapy $( I I )$ Mutations in NLRC4 also induce auto-inflammatory diseases in humans (1o,12-14).

WeassembledtheFliC-activatedNAIP5-NLRC4 complex and the PrgJ-activated NAIP2-NLRC4 complex with the use of CARD-deleted NLRC4 (NLRC4△) to avoid potential CARD-mediated aggregation (Fig.1A).Either full-length NAIP2 orN-terminal baculovirus inhibitor of apoptosis protein repeat (BIR) domain-deleted NAIP2 (NAIP2△BIR) was used.During purification, sucrose gradient fractions showed variable molar ratios between NAIP2 and NLRC4△ (Fig. IB).The fraction with the highest amount of NLRC4△ relative to NAIP2, by approximately one order of magnitude,contained mostly single,complete disk-like particles (Fig.1C and fig.SiA).Similarly,overstoichiometry of NLRC4△ to NAIP5 was observed in the reconstituted FliC-NAIP5-NLRC4△ complex (Fig.1D), also with mostly complete disks under electron microscopy (fig. SiB).In contrast, our earlier PrgJNAIP2-NLRC4△ preparations exhibited much lower NLRC4△/NAIP2 molar ratios (fig. S1C), with mostly incomplete disks (fig. SiD).Labeling of NAIP2 with 5-nm Ni-NTA gold particles in the PrgJ-NAIP2△BIR-His-NLRC4△ inflammasome revealed singularly labeled complexes (Fig.1E). These data, together with the previous report thatNAIP5 did not oligomerize in the presence of flagellin (15),demonstrated that a single NAIP exists in each complex,whether ina full disk ora partially assembled disk (Fig.1F).

We collected cryo-electron microscopy (cryoEM) data on the PrgJ-NAIP2-NLRC4△ complex (Fig.1G and figs.S2and S3A).Reference-free two-dimensional(2D)classification revealed mostlyI1-bladed,but also I2-and 1o-bladed inflammasome complexes (fig. S3B),implying conformational flexibility.From the top or bottom view,an inflammasome disk comprises an inner ring and an outer ring (Fig.1H); 3D classification yielded models with apparent $C _ { 1 0 }$ ， $C _ { \mathrm { 1 1 } } ,$ and $C _ { 1 2 }$ symmetries (fig. S2).The individual blades did not show observable differences to indicate the single PrgJ-NAIP2 complex in eachdisk,probably due to similar domain organizations of NAIP2 and NLRC4 (Fig.1A). Upon imposing the apparent symmetry, the ${ \cal C } _ { 1 1 } ,$ $\begin{array} { r } { C _ { 1 2 } , } \end{array}$ and $C _ { 1 0 }$ reconstructions were refined to resolutions of $4 . 7 \mathrm { \AA } ,$ $7 . 5 \mathrm { \AA } ,$ and $1 2 . 5 \mathrm { \AA } ,$ respectively (Fig.2,Ato F,and fig. S3, C to G).Local resolution estimation of the $C _ { 1 1 }$ reconstruction suggests that the inner ring possesses resolutions of 4.0 to $6 . 0 \textup { \AA }$ (Fig. 2A and fig.S4),with secondary structural features consistent with a resolution of at least $6 . 0 \textup { \AA }$ (Fig. 2, Ato D,and fig. S5).Using the crystal structure of NLRC4△ in the inactive conformation (PDB ID 4KXF) (i6),we built and refined an atomic model of the active NLRC4△ (table S1).All structures have a domed center and a prominent inner hole(Fig.2,B, $\operatorname { E } ,$ andF).The inner ring of the disk contains the NBD,helical domain 1(HD1), and the winged helix domain (WHD); the outer ring comprises helical domain 2(HD2) and the LRR domain (Fig.IA and Fig. 2B).

![](images/0eb348d5ed5226323083a0da47335a8d3c8bcbbb0850fbf7638a7f5e4edc88ae.jpg)  
Fig.1.Preparation and characterization of NAIP-NLRC4△ complexes. (A) DO main organizations of Salmonella typhimurium PrgJ,mouse NAlP2,mouse NLRC4,and mouse caspase-1. Domain size is drawn approximately to scale; residue numbersare labeled.(B) SDS-polyacrylamide gel electrophoresis (PAGE) of different fractions of the sucrose gradient ultracentrifugation during the purification of the PrgJ-NAlP2-NLRC4△ complex.Locations of the three component proteins are labeled.The asterisk indicates a contaminating band.(C)A representative negative-stain EM image from fraction7in (B).(D) SDS-PAGEof amylose resin elution (lane 1),anti-Flag flow-through (lane 2),and anti-Flag   
elution (lane 3) fractions during the purification of the coexpressed His-FliCFlag-NAIP5-His-MBP-NLRC4△ complex. Anenlarged image of lane3 is shown. (E)Ni-NTA gold labeling $( 5 ~ \mathsf { n m } )$ ）ofpurified His-Sumo-PrgJ-NAIP2△BIR-HisHis-Sumo-NLRC4△ complex upon removal of the His-Sumo tag.(F) Schematic diagram of partial and complete inflammasome particles that contain variable ratios between NAIP2 (yellow) and NLRC4 (cyan).(G) Representative cryo-EM micrograph of PrgJ-NAIP2-NLRC4△ particles.(H) Anaveraged 2D classof the 11-bladed PrgJ-NAIP2-NLRC4△ inflammasome complex.The dimensions of the imageare $4 3 . 5 \ : \mathrm { n m } \times 4 3 . 5 \ : \mathrm { n m }$

We focus our further discussions on the llbladed structure with the highest resolution. The conformation of active NLRC4△ in the 11- bladed inflammasome exhibitsdifferences from thatof inactive NLRC4△(I6).When the NBD

HD1 regions of NLRC4 in the two states are aligned, the WHD-HD2-LRR module needs to rotate $8 7 . 5 ^ { \circ }$ along an axis at the junction between HD1 andWHD to turn from the inactive state to the active state (Fig.2,G andH).The pivot point of the long-range hinge motion is where the inactive and active conformations of the αl4 helix of WHD intercept (Fig.2H).The rotation rearranges the intramolecular interactionsbetween WHD and the NBD-HD1 module.

Several missensemutations in human NLRC4 that are associated with auto-inflammatory conditions $( I 2 \mathrm { - } I 4 ) \mathrm { - } \mathrm { T h r } ^ { 3 3 7 } \to \mathrm { S e r , V a l } ^ { 3 4 1 } \to \mathrm { A l a } _ { 2 }$ and $\mathrm { H i s ^ { 4 4 3 } \to P r o { - } l }$ ocalize to this highly dynamic region (fig. S6A).

We do not know whether NLRC4 conformational transitionis accompaniedby exchange of adenosine diphosphate (ADP) in the inactive state toadenosine triphosphate(ATP)in the active state,as observed for apoptosome assembly by the related adenosine triphosphatase (ATPase) Apaf-1 (I7,I8).Lackofnucleotidedensityin the cryo-EM map,local conformational changes,a modifiedWalkerBmotif,andabsenceofa conserved Arg in the sensor I motif (I9) may all suggest alternative mechanisms (fig.S6,B and C). Consistently, theNLRC4WalkerAmotifmutant $\mathrm { L y s } ^ { 1 7 5 } $ Arg induced cell killing almost as effectively as did the wild type when coexpressed withNAIP5, flagellin, and caspase-1 (2).

![](images/ec59dbfe78f00c17fa57113dfeb4f35b6fd37d5397fda4cd5ec2734a7383415b.jpg)  
Fig.2. Cryo-EM structure determination and conformational activation of NLRC4. (A) Cryo-EM map of the $C _ { 1 1 }$ PrgJ-NAIP2-NLRC4△ complex colored with local resolution calculated by ResMap using two separately refined half maps.(B) Superimposed ribbon diagram and transparent surface of the $C _ { 1 1 }$ NLRC4△ structure.(C) Cryo-EM density superimposed with one NLRC4△ subunit.(D) A close-up view of the structure of the NBD of NLRC4△ superimposed with the cryo-EM density.(E and F) Cryo-EM maps and fitted NLRC4△ models for the $C _ { 1 0 }$ reconstruction at $1 2 . 5 \textup { \AA }$ resolution (E) and the $C _ { 1 2 }$   
reconstruction at $7 . 5 \mathrm { ~ \AA ~ }$ resolution (F). (G) The WHD-HD2-LRR domain of NLRC4 swings $8 7 . 5 ^ { \circ }$ to transit from the inactive conformation (left, PDB ID 4KXF） to the active conformation (right).NBD and HDl are shown in superimposed ribbon diagram and transparent surface,and the WHD-HD2-LRR module is shown in ribbon diagram.(H) Superimposed inactive (colored) and active(gray,except for αl4 helix,which is in dark blue) conformations of NLRC4△.The αl4 helices in the two conformations are labeled to show the relative rotations and the rotational pivot point.

Tofacilitate analysis on NAIP2-NLRC4 interactions,we generated a homology model of NAIP2△BIR based on the NLRC4△ structure (20) (Fig.3A).By replacing one of the NLRC4 molecules in the initially fitted $C _ { 1 1 }$ structure,we generated a NAIP2-NLRC4 inflammasome model with oneNAIP2and1ONLRC4 molecules,inwhicha singlePrgJ-activatedNAIP2initiatesNLRC4activation and polymerization in a domino-like reaction (Fig. 3B).This mechanism differs from other oligomeric ATPases such as apoptosome proteinsApaf-1, CED-4,andDARK,which need to be activated before assembly,either conformationally (by its ligand cytochrome c) or by removal of the inhibitory protein CED-9 (21-23).

The NAIP2-NLRC4 interactions are extensive, with a total surface area of ${ \sim } 1 0 0 0 \mathrm { ~ } \mathrm { \AA } ^ { 2 }$ per subunit per interaction,and containamixture of hydrophobic,hydrophilic,and charged interactions.For brevity,we named one surface as A and the opposing surface as B (Fig.3C),each comprising a large patch near the NBD and a small patch at the LRR(Fig.3D). The LRR does appear to play a minor role in strengthening the oligomerization interactions because an LRRdeleted NLRC4 showed attenuation,not abrogation,ofkillingand oligomerization in comparison with ligand-activated wild-type NLRC4 (2). Calculation of surface electrostatics shows that the NAIP2-A surface, formed by NBD and WHD,is largely basic,whereas the NLRC4-B surface, formedbyNBD,HD1,andWHD,islargelyacidic; this would suggest that the NAIP2-A surface interacts with the NLRC4-B surface to initiate thedirectional formationof aninflammasome disk (Fig.3D). Consistently, the NAIP2-B surface islargely incompatible with either NLRC4-A (fig. S7,A andB) or NAIP2-A(fig. S7,C and D). We thus named NAIP2-A the nucleating surface.

Interaction analysis revealed that residues $\mathrm { L y s } ^ { 6 1 3 }$ $\mathrm { T y } ^ { 6 1 5 }$ $\mathrm { A r g } ^ { 6 3 1 }$ $\mathrm { A r g } ^ { 6 3 4 }$ $\mathrm { P r o } ^ { 6 3 5 }$ ${ \mathrm { T y r } } ^ { 6 3 6 } ;$ and $\mathrm { G l n } ^ { 7 7 8 }$ of NAIP2-A and residues $\mathrm { P r o } ^ { \mathrm { 1 1 2 } }$ $\mathrm { A s n } ^ { 1 1 6 }$ $\mathrm { { G l u } ^ { 1 2 2 } }$ H $\mathrm { { A s p } ^ { 1 2 3 } }$ Asp125, $\mathrm { { I l e } ^ { 1 2 7 } }$ Leu220,Leu31，et4n $\mathrm { A s p } ^ { 3 6 8 }$ of NLRC4-Bburya large surfaceareaatthe interface (Fig.3E).To test this interface,we used the HisFliC-Flag-NAIP5-His-MBP-NLRC4△ inflammasome system because of the availability of differentially tagged constructs for coexpression in 293T cells andbecause of the sequence similarity between NAIP2 and NAIP5 (Fig. 3F).For NAIP5-A,we mutated $\mathrm { A r g } ^ { 5 8 7 }$ and $\mathrm { A r g } ^ { 5 9 0 }$ ,which correspond to $\mathrm { A r g } ^ { 6 3 1 }$ and $\mathrm { A r g } ^ { 6 3 4 }$ of NAIP2-A, to Asp (mutation RR2D).For NLRC4-B,we mutated the three acidic residues, $\mathrm { { G l u } ^ { 1 2 2 } }$ $\mathrm { { A s p } ^ { 1 2 3 } }$ ,and $\mathrm { A s p } ^ { 1 2 5 }$ to Ala (mutation EDD2A).Whereas the wild-type constructs showed robust copurification of NAIP5 andFliCbyNLRC4,mutations on eitherNAIP5-A or NLRC4-B reduced the amount of copurified NAIP5 and FliC (Fig. 3F).

For NAIP2 to initiate NLRC4 polymerization, wehypothesized thatitmust have weak affinity with the inactive NLRC4.MappingNLRC4-A and NLRC4-B residues onto the inactive NLRC4 conformation showed that NLRC4-B,but not NLRC4-A,isalready largely formed,with onlya small part of the WHD in clash with an interacting NAIP2-A (Fig. 3G and fig. S7E). The bound NAIP2-A at this site would push the WHD of NLRC4 exactly at αl4,the hinge for conformational changes to occur (Fig.2,G and H).We propose that the active NAIP2-A surface makes aninitial encounter with the NLRC4-B surface in the inactive conformation to initiate the activating conformational change (fig. S7F).

![](images/74f2e962f1a7e9191ab6473098e00a11f20924d9dfe77fa5c0405d21a19d024d.jpg)  
Fig. 3.Conformational activation of NLRC4 by activated NAlP2. (A）Superposition of the NLRC4△ structure and the NAIP2△BIR homology model in the active conformations.(B)A ribbon diagram of the ll-bladed NAIP2-NLRC4 inflammasome disk,with the single NAlP2 molecule in yellow and NLRC4 molecules in cyan.(C) Locations of A and B surfaces,in particular NAIP2-A and NLRC4-B,in the PrgJ-NAIP2-NLRC4△ inflammasome.(D) Mapped interactions at the NAlP2-A surface (pink) and the NLRC4-B surface (green) and their surface electrostatic potentials.Dotted ovals show the approximate locations of the interface on the electrostatic surfaces.(E) Detailed interactions between the NAlP2-A surface and the

NLRC4-B surface.Those on the A surface are labeled in boldface.(F) Mutations at the NAlP5-A and NLRC4-B surfaces impaired complex formation.The mutated residues in NAlP5 are completely conserved in NAIP2 and NLRC4.The three proteins were coexpressed in 293T cells; the MBP tag was used to pull down the complex,and the component proteins were detected using Western blots.(G) Ribbon diagram of NLRC4△ in inactive conformation.The region of WHD at the tip of the αl4,αl5,and αl6 helices, in slight clash with an interacting NAlP2,is shown in yellow.Amino acid abbreviations: D,Asp; E,Glu; H, His: I, Ile;K,Lys: L,Leu; M,Met;N,Asn; P, Pro; Q, GIn;R,Arg; S,Ser;V, Val; Y,Tyr.

Calculation of surface electrostatics revealed charge complementarity between the mostly basic NLRC4-A surface and the opposing, largely acidic NLRC4-B surface (Fig.4,Aand B); this finding supports unidirectional polymerization nucleatedbyNLRC4-A.Structuralanalysisidentified interfacial residues including His144, $\mathrm { A r g } ^ { 1 4 5 }$ $\mathrm { H i s ^ { 2 6 9 } }$ $\mathrm { A r g ^ { 2 8 5 } }$ ，His286， $\mathrm { A r g ^ { 2 8 8 } }$ $\mathrm { H i s ^ { 2 8 9 } }$ $\mathrm { G l n ^ { 4 3 3 } }$ and $\mathrm { A r g ^ { 4 3 4 } }$ of NLRC4-A, and Asn116, $\mathrm { { G l u } ^ { 1 2 2 } }$ Asp123， $\mathrm { A s p } ^ { 1 2 5 }$ Ie127，Leu22，Leand $\mathrm { A s p } ^ { 3 6 8 }$ of the opposing NLRC4-B (Fig. 4C).We mutated NLRC4-A residues $\mathrm { A r g ^ { 2 8 8 } }$ and His289 to Asp (mutation RH2D)and $\mathrm { A r g } ^ { 2 8 5 }$ to Asp (mutation R285D) and tested their interactions using the same 293T cell coexpression system ofHis-FliC,Flag-NAIP5,and His-MBP-NLRC4△.Both RH2D and R285D mutations reduced the amount ofNLRC4△ in the inflammasome complex (Fig.4D).Therefore,like a ligand-activatedNAIP2,a newlyactivatedNLRC4 triggers activation of anotherNLRC4 molecule by inducing conformational changes (fig. S7F). The NBD-NBD interactions betweenadjacentNLRC4 subunits differ from those in the heptameric apoptosome,with distinct angular relationships that may have explained the existence of more subunits in each NLRC4 disk (2I,22) (fig. S8,A to C). ASC,like NLRC4,is an inflammasome adapter protein.We showed previously that ASCdependent inflammasomes activate caspase-1 by ASCCARD-mediated caspase $- 1 ^ { \mathrm { C A R D } }$ polymerization

A NRT4 B Complementary NLRC4-A and NLRC4-B surfaces C H2697 NLRC4-A= NBD 180° NBD Nucleating t H289 Surface H286 116 E 145 H144 LRR LRR 433 R434   
NLRC4 NLRC4 Direction of Largely basic NLRC4-A Largely acidic NLRC4-B NLRC4-A NLRC4-B   
Polymerization   
D His-FIC WT WT WT 套 L $( 2 . 5 \mu \mathrm { M } )$   
Flag-NAIP5 WT WT WT   
MBP-NLRC4△ WTRH2D R285D r ←[NLRC4]   
NEAIA P rrreeeeeee 202020 T 建 ForMBP   
NLRC4-A RH2D:R288D+H289D NLRC4-A R285D 200 0 20 40 60 40 60 Time (min) Time (min)   
G NLRC4CARD   
X   
PrgJ Inactive Inactive NLRC4 NLRC4 Inactive NAIP2 Active NAIP2 Active Active NAIP2 Active NAIP2 Active NAIP2 Active NLRC4 NLRC4 NLRC4CARD cluster Caspase-1CARD Inactive Active NLRC4 Active Caspase NAIP2 NAIP2 domain   
NAIP2 Active NLtive NLRC4 Active NLRC4 Active NLRC4 Active NLRC4 Active NLtic4 NLRC4 Active PrgJ/NAIP2/NLRC4disk PrgJ/NAIP2/NLRC4/Caspase-1

detected using Western blots. (E) $N L R C 4 ^ { \mathsf { C A R D } }$ instead of $N L R C 4 ^ { \mathsf { F L } }$ , nucleates filamentformationof labeledcaspase-1CARD,asshown byincrease in fluorescence polarization. (F) The PrgJ-NAIP2-NLRC4F- inflammasome nucleates filament formation of labeled caspase $- 1 ^ { \mathsf { C A R D } }$ at substoichiometric ratios, asshown by increase in fluorescence polarization.(G) Schematic diagram for mechanism of PrgJ-NAlP2-nucleated polymerization of NLRC4,followed by caspase-1 dimerization and activation.

(24).To examine whether the PrgJ-NAIP2-NLRC4 inflammasome may directly activate caspase1 through the CARD in NLRC4,we adopted the same fluorescence polarization assay of caspase$1 ^ { \mathrm { C A R D } }$ (24).Wereconstitutedthefull-length PrgJ-NAIP2-NLRC4 inflammasome,whichshowed stacked disk-like structures similar to the previously reconstituted FliC-NAIP5-NLRC4 complex (15) (fig. S8D),and expressed $\mathbf { N L R C 4 } ^ { \mathbf { C A R D } }$ fused to green fluorescent protein (GFP-NLRC4CARD), which was filamentous under EM (fig. S8E).GFP$\mathrm { N L R C 4 } ^ { \mathrm { C A R D } }$ augmentedtherateofcaspase-1CARD polymerization,whereas full-lengthNLRC4 from either the monomeric or the void fraction had little effect (Fig.4E).The PrgJ-NAIP2-NLRC4 inflammasome robustlypromoted filament formation of caspase $. 1 ^ { \mathrm { C A R D } }$ even at a 1:100 substoichiometric ratio (Fig. 4F).

Given that $\mathrm { N L R C 4 } ^ { \mathrm { C A R D } }$ alone is filamentous, we expect that the\~10 molecules of NLRC4CARD in each NAIP2-NLRC4 inflammasome form a short filament at the center of the disk.The observed stacking in full-length NAIP-NLRC4 inflammasomes (i5)is likely due to interactions between unengaged NLRC4 CARDs.In the presence of caspase-l, the inflammasome may change into singledisks,just like the transitionof theDrosophila apoptosome from double rings to single rings in the presence of the caspase (21). Curiously, the central hole of the inflammasome has a diameter just a bit smaller than that of CARD filaments at ${ \sim } 9 \mathrm { n m }$ which may provide a perfectly sized“basin”to cradle the protruded CARD filament.These studies demonstrate thatASC-independentNAIP-NLRC4 inflammasomesmakeuse of a similar mechanism for caspase-1 activation,as shown for ASCdependent inflammasomes (24).

Our studies suggest that activation of NAIPNLRC4 inflammasomes may proceed through the following steps (Fig.4G),a conclusion also reached independently by the accompanying study (25): (i) Because of the domain similarity of NAIPs to NLRC4,we propose that the NAIP resting state is similar to the NLRC4 inactive conformation. Aftera cell is infected and bacterial products appear in the cytosol,a NAIP recognizes its specific bacterial ligand, likely through a surface on the HD1,WHD,and HD2 region (26). The specific ligand drives the NAIP into the open,activated conformation.(ii) The ligand-bound NAIP uses its nucleating surface to interact with the adapterNLRC4 thatisyet to be activated.The interactionforcesthe WHD and its linked C-terminal region to change into the activated conformation, overcoming NLRC4 auto-inhibition. The activated NLRC4 uses its newly exposed nucleating surface to repeat recruitment and activation of additional NLRC4 molecules,until a complete disk is formed oruntil the NLRC4 concentration falls below the dissociation constant of the interaction. (iii) NLRC4 clustering induces oligomerization of the CARD of NLRC4, enabling the recruitment of caspase-1 through CARD-CARD interactions and triggering caspase-1 dimerization,autoproteolysis, and activation.The activation mechanism ensures signal amplification from the receptor to the adapter,and then to the effector.

As the most abundant energy source in living organisms,ATP isused widely in enzymes to mediate force generation, conformation change,oligomerization,and transport. The ATPase-mediated nucleated polymerization througha domino-like chain reaction identified here adds an important, elegantmechanismtothisuniversalandalready complexenzymefamily.Nucleatedpolymerization in NAIP-NLRC4 inflammasomes also presents yet another mode of higher-order oligomerization, which mayplayarole in facilitatingproximityinduced enzyme activation, threshold response, and prion-like propagation in immune signaling (27-30).

# REFERENCESANDNOTES

1．J. D. Growney, W.F.Dietrich, Genome Res.10,1158-l171 (2000).   
2.E.M. Kofoed, R.E.Vance, Nature 477,592-595 (2011).   
3．E. Diez et al.,Nat. Genet.33,55-60 (2003).   
4.Y. Zhao et al., Nature 477, 596-600 (2011).   
5．J.Yang,Y. Zhao,J. Shi,F. Shao, Proc. Natl. Acad. Sci. U.S.A. 110,14408-14413 (2013).   
6.E.A. Miao et al., Nat.Immunol.7,569-575 (2006).   
7．E.A. Miao et al., Nat. Immunol. 11,1136-1142 (2010).   
8.L.Franchi et al., Nat. Immunol.13, 449-456 (2012).   
9.B.Hu et al.,Proc.Natl.Acad.Sci. U.S.A.107,21635-2164O (2010).   
10.A.M. Janowski, R. Kolb,W. Zhang,F.S. Sutterwala,Front. Immunol.4,370 (2013).   
11.J.Garaude, A. Kent, N.van Rooijen, J. M. Blander, Sci. Transl. Med.4,120ra16 (2012).   
12.S.W. Canna et al., Nat. Genet. 46,1140-ll46 (2014).   
13.A. Kitamura, Y. Sasaki, T.Abe,H. Kano,K. Yasutomo,J. Exp. Med.211, 2385-2396 (2014).   
14.N. Romberg et al., Nat. Genet. 46,1135-1139 (2014).   
15.E.F.Halff et al.,J.Biol. Chem.287,38460-38472 (2012)   
16. Z. Hu et al., Science 341,172-175 (2013).   
17.T.F. Reubold, S.Wohlgemuth,S.Eschenburg,J. Biol. Chem. 284,32717-32724 (2009).   
18.Q.Bao,W.Lu,J.D.Rabinowitz,Y.Shi, Mol. Cel/25,181-192(2007).   
19. M.Proell, S.J. Riedl,J. H.Fritz, A. M. Rojas, R.Schwarzenbacher,PLOS ONE3,e2119 (2008).

20.M.A. Marti-Renom et al., Annu. Rev. Biophys. Biomol. Struct.   
29,291-325 (2000).   
21.S.Yuan,C.W.Akey, Structure 21, 501-515 (2013).   
22.S.Yuan et al., Structure 19,128-140 (2011).   
23.S. Qi et al., Cell141,446-457 (2010).   
24.A.Lu et al., Cel/156.1193-1206 (2014).   
25.Z. Hu et al., Science 350.399-404 (2015).   
26.J.L.Tenthorey, E. M. Kofoed, M. D. Daugherty, H. S. Malik,   
R.E.Vance,Mol.Cell 54,17-29 (2014).   
27.H.Wu, Cel/153,287-292 (2013).   
28.B. S. Franklin et al., Nat. Immunol.15,727-737 (2014).   
29.X. Cai et al.,Cell 156,1207-1222 (2014).   
30.A.Baroja-Mazo et al., Nat.Immunol.15,738-748 (2014).

# ACKNOWLEDGMENTS

# SUPERCONDUCTIVITY

We thank G.Bozkurt for technical assistance; H. Ploegh for providing engineered, ${ \mathsf { C } } \mathsf { a } ^ { 2 + }$ -independent sortase and the peptide-fluorophore conjugate Gly-Gly-Gly-TAMRA (GGG-TAMRA); and M. Ericsson for use of the Harvard Medical School EM facility.The data presented in this manuscript are tabulated in the main paper and in the supplementary materials. Cryo-EM maps and atomic coordinates have been deposited in the Electron Microscopy Data Bank (EMDB IDs EMD-6458, EMD-6459,and EMD-6460 for $C _ { 1 1 }$ $\begin{array} { r } { C _ { 1 2 } , } \end{array}$ and $\boldsymbol { C } _ { 1 0 }$ reconstructions, respectively) and Protein Data Bank (PDB ID 3JBL for the $C _ { 1 1 }$ reconstruction).Supported by NIHK99 grant All08793(Q.Y.),a Cancer Research Institute postdoctoral fellowship (L.D.),an Intel academic grant, research funds at Peking University, and NiH grant 1DP1HDO87988.The cryo-EM facility was funded through the NIH grant AI100645 Center for HIV/AIDS Vaccine Immunology and Immunogen Design (CHAVl-ID).The experiments were performed in part at the Center for Nanoscale Systems at Harvard University,a member of the National Nanotechnology Infrastructure Network (NNIN), which is Supported by NSF award ECS-0335765.

# SUPPLEMENTARYMATERIALS

www.sciencemag.org/content/350/6259/404/suppl/DC1   
Materials and Methods   
Figs. S1 to S8   
Table S1   
References (31-48)   
14 May 2015;accepted 14 September 2015   
Published online 8 October 2015   
10.1126/science.aac5789

# Metallic ground state in an ion-gated two-dimensional superconductor

Yu Saito,1 Yuichi Kasahara $^ { 1 , 2 \ast } \dag$ Jianting Ye,1,3,4+ Yoshihiro Iwasa, ${ } ^ { 1 , 4 } \ddag$ Tsutomu Nojima5‡

Recently emerging two-dimensional (2D) superconductors in atomically thin layers and atheterogeneous interfaces are attracting growing interest in condensed matter physics. Here,we report that an ion-gated zirconium nitride chloride surface,exhibiting a dome-shaped phase diagram with a maximum critical temperature of 14.8 kelvin,behaves as a superconductor persisting to the 2D limit.The superconducting thickness estimated from the upper critical fields is $\cong 1 . 8$ nanometers,which is thinner than one unit-cell.The majority of the vortex phase diagram down to 2 kelvin is occupied by a metallic state with a finite resistance, owing to the quantum creep of vortices caused by extremely weak pinning and disorder. Our findings highlight the potential of electric-field-induced superconductivity,establishinga new platform foraccessing quantum phases in clean 2D superconductors.

R ecent technological advances of materials fabrication have led to discoveries of a variety of superconductors at heterogeneous interfaces and in ultrathin films; examples include superconductivity at oxide interfaces (1,2), electric-double-layer interfaces (3),and mechanically cleaved (4),molecular-beam-epitaxy-grown (5,6),or chemical-vapor-deposited $( 7 )$ atomically thin layers.These systems are providing opportunities for searching for superconductivityat higher temperatures,as well as investigating the intrinsic nature of two-dimensional (2D)