# Molecular characteristics and evolutionary analysis of a very virulent infectious bursal disease virus

LI Zan14, QI XiaoLe², REN XianGang², CUI Lei³, WANG XiaoMei2\* & ZHU Ping

National Laboratoryof Bomacromolecules,InstituteofBiophysics,ChineseAcademyofSiences,BeijingOoo,China; DivisionofAvifectiosseasStateKeyboratoryofeterinryotechologHarbineterinarysearchsiute Academy of Agricultural Sciences,Harbin15000l, China; 3CAS Key Laboratory of RNA Biology, Institute of Biphysics,Chinese Academy of Sciences,Beijing l0olo1,China; 4University of Chinese Academy of Sciences, Beijing 10o049, China

ReceivedMarch6,2015;accepted April13,2015

Infectious bursal disease virus (IBDV)poses asignificantthreattothe poultry industry.Viral protein2(VP2),the majorstructural proteinofIBDV,hasbeensubjected to frequent mutations thathave impartedtremendous geneticdiversitytothe virus. To determine how amino acid mutations mayaffect the virulenceof IBDV,we builtastructural modelof VP2ofa very virulent strainof IBDVidentified in China,vvIBDVGx,and performedamolecular dynamics simulationof the interactionbetween virulence sites.The study showed that the aminoacid substitutions that distinguish vvIBDVfrom attenuated IBDV (H253Q and T284A) favor a hydrophobic and flexible conformation of $\beta$ -barrel loops in VP2,which could promote interactions between thevirus and potential IBDV-specificreceptors.Population sequence analysis revealed that the IBDV strains prevalent inEast Asiashow a significant signalof positive selectionatvirulence sites253and284.Inadition,asignalof co-evolutionbetween sites 253and 284 was identified.These results suggestthat changes in thevirulenceofIBDV mayresult from both the interaction and the co-evolution of multiple amino acid substitutions at virulence sites.

# infectious bursal disease virus,veryvirulentstrain,virulence,molecularcharacterization,molecular evolution

Citation: LiZ,QiXL,XGiagXZuaactestdluioaalisltes Sci China Life Sci,2015,58:731-738,doi:10.1007/s11427-015-4900-x

Infectious bursal disease virus (IBDV) is an avian virus that specifically infects the bursa of Fabricius in young chickens.It induces the formation of acute lesions of lymphatic tissues and the destruction of lymphocytes in the bursa.Because of its immunosuppressive effects,IBDV severely impairs the immune response to many vaccines and causes a general failure of immunity [1-3]. The virus first emerged in the 195Os [1] and has caused egregious economic ruin to the modern poultry industry [4]. Two serotypes of IBDV, serotype 1 and serotype 2,have been characterized,but only serotype 1 is pathogenic to young chickens [5]. IBDV serotype 1 has undergone tremendous genetic diversification since it Was firstidentified such as classical virulent strains (cvIBDV) [6],antigenic variant strains [7],and very virulent strains(vvIBDV)[8].The vvIBDV strains,which have emerged and spread in the last 2O years,result in mortality rates as high as $60 \%$ in young chickens, thus posing severe challenges to avian epidemic control and prevention in China and throughout the world.

IBDV belongs to the genus Avibirnavirus,in the family Birnaviridae,and contains two double-stranded RNA segments,segment A $( 3 . 2 ~ \mathrm { k b } )$ and segment B $( 2 . 8 ~ \mathrm { k b } )$ [6]. Segment A encodes four viral proteins:VP2,VP3,VP4,and VP5 [9-11]. VP2,the major structural protein [11],assembles into trimers,and then 26O of these trimers aggregate to form the IBDV capsid.The folding of the amino acid chain of VP2 forms three domains,the protrusion domain (P domain),the shell domain(S domain),and the helical base domain (B domain).The P domain is located on the viral surface,perpendicular to the capsid [12,13].According to the crystal structure of VP2 of an attenuated IBDV strain [12], neutralization escape epitopes and cell culture adaptation/virulence sites are located in the flexible loops of the $\mathrm { \bf P }$ domain,which are highly exposed and away from the virion surface.Several viral infection studies have shown that the hypervariable region of VP2 is involved in virulence changes and that the amino acid residues at positions 253 and 284 of VP2 are the main determinants of the virulence and cell tropism of IBDV[14,15].In other studies,residues 253,279,and 284 were found to be important for cell culture adaptation/virulence [16,17]. Virulence declined rapidly when vvIBDV with amino acid mutations at positions 253 and 284 in the VP2 hypervariable region was passaged serially in chicken embryo fibroblast cells [18],while the veryvirulent pathogenicity wasrestored after a reverse mutation assay.Recent virulence and immunology studies have shown that the amino acid residue at position 253 is critical to the alteration of virulence of IBDV,while the residues at positions 279 and 284 are not directly related to viral virulence but are involved in cell adaptability and replication efficiency [19,2O].All these studies suggest that a few amino acid mutations at key positions in the VP2 hypervariable region can have significant impacts on the virulence of IBDV.

High-resolution structures of both VP2 and IBDV strains with different virulences are critical for a full understanding of how amino acid mutations may change the virulence of the virus.In 2OO5,the structure of IBDV $\mathrm { C t }$ ，an attenuated vaccine strain of IBDV, was resolved at $7 \mathrm { ~ \AA ~ }$ resolution by X-ray crystallography [12]. Later, the cryo-electron microscopy structure of Soroa IBDV,another attenuated strain of IBDV, was reported at $1 0 \mathrm { ~ \AA ~ }$ resolution [13]. Amino acid substitutions in a structural protein can markedly affect virulence,which is an important factor in viral adaptive evolution.Nevertheless,no structure of vvIBDV with substitutions at virulence sites in the hypervariable region of VP2 (e.g.,Q253 and A284) is currently available. Gene sequence and evolutionary relationship analyses can provide a novel understanding of virulence proteins [21] and help to predict potential virulence and adaptive selection sites of variant viruses [22,23].

In this study,we built a pseudo-atomic structural model of VP2 from vvIBDV Gx,a very virulent strain of IBDV identified in China,andcharacterized the interactionof two virulence sites，Q253 and A284，by molecular dynamics simulation to reveal the molecular basis of IBDV virulence. In addition,we performed a population sequence analysis of the hypervariable region of VP2 to analyze the positive selection and co-evolution of virulence sites during the geneticdiversification ofIBDV.These results could shed new light on the molecular mechanisms of IBDV virulence and evolution.

# 1Materials and methods

# 1.1 Virus propagation and histopathology

Groups of 3-week-old specific-pathogen-free (SPF) chickens,1O chickens per group,were inoculated with vvIBDV Gx $( 1 , 0 0 0 \mathrm { E L D } _ { 5 0 }$ (median embryo lethal dose) per $0 . 1 ~ \mathrm { m L }$ 门 in the nasal mucosa and conjunctiva.Bursae of Fabricius were collected three to five days later，when chickens showed clinical lesions or were dying.Lesions were fixed and embedded in paraffin blocks.Cross sections were cut with a microtome,stained with hematoxylin and eosin,and examined under a light microscope.DMEM (Dulbecco's Modified Eagle's Medium) and an attenuated strain,IBDV Gt, were used to infect SPF chickens as negative controls. All animal experiments were approved by the Harbin Veterinary Research Institute (HVRI) of the Chinese Academy of Agricultural Sciences (CAAS）and were performed in accordance with animal ethics guidelines and approved protocols.

# 1.2 Viruspurification

Fresh bursae were ground in 1O volumes $( \mathrm { w } / \mathrm { v } )$ of phosphate-buffered saline $\mathrm { ( p H ~ 7 . 2 ) }$ . The cell lysate was centrifuged at $4 { , } 8 0 0 { \times } g$ for $3 0 ~ \mathrm { m i n }$ at $4 ^ { \circ } \mathrm { C }$ to remove cell debris. The supernatants were pelleted through a $20 \%$ glycerol cushion at $1 8 6 , 0 0 0 \times g$ for $2 \mathrm { ~ h ~ }$ at $4 ^ { \circ } \mathrm { C }$ .Thevirus was then purified by density gradient centrifugation on $30 \% - 4 5 \%$ sucrose at $1 2 5 , 0 0 0 \times g$ for $1 2 \mathrm { { h } }$ at $4 ^ { \circ } \mathrm { C }$ and $27 \%$ cesium chloride at $1 2 5 , 0 0 0 \times g$ for $2 4 \mathrm { ~ h ~ }$ at $4 ^ { \circ } \mathrm { C }$ .Fractions containing virions were collected and dialyzed against phosphatebuffered saline $\mathrm { ( p H } 7 . 2 \mathrm { ) }$ ：

# 1.3 vvIBDV $\mathbf { G x }$ VP2 homology modeling

The structure of vvIBDV Gx VP2 was built by the I-TASSER (Iterative Threading ASSEmbly Refinement) server [24].The full-length vvIBDV $\operatorname { G x }$ VP2 model was constructed by multiple-threading alignment and iterative template fragment assembly simulations.

# 1.4Molecular dynamics simulation

Molecular dynamics (MD） simulations of the effects of amino acid mutations at positions 253 and 284 on the IBDV VP2 structure were performed with nanoscale molecular dynamics (NAMD) [25],based on the crystal structure of cvIBDV VP2(253H,284T) [26].Four systems,i.e., the wild type,two single mutations (H253Q，T284A)，and a double mutation (H253Q/T284A)，were subjected to structural dynamics simulations over a 1OO ns simulation period after a 14 ns pre-equilibration period.

# 1.5IBDV VP2 sequence analysis and phylogenetic tree construction

The cDNA sequences for VP2 of variant strains of IBDV (i.e.,cvIBDV,vvIBDV) deposited in GenBank as of October 2O14 were collected and aligned using ClustalW (http://www.genome.jp/tools/clustalw/). Duplicate sequences were detected with DAMBE [27] and discarded prior to analyses. Strains were grouped by geographic origin: Europe，Africa,West Asia,South Asia,East Asia,North America,or South America.The consensus sequences in each group were obtained separately and the pairwise distances by proportion of nucleotide substitutions were calculated using MEGA [28].The nucleotide substitution matrix was used to construct the phylogenetic relationships among IBDVs from sevenregions.Analysis of amino acid variabilityin the hypervariable regionof IBDVVP2 was performed using DAMBE software.

# 1.6Positive selection and co-evolution analyses

The maximum likelihood method was used to estimate the ratio of the non-synonymous substitution rate (dN) to the synonymous substitution rate $( d S )$ codon by codon（ $\omega =$ dN/dS) [29,3O].An @ value greater than one indicates that the codon is under positive selection.Two different algo rithms,FEL (fixed effects likelihood） and SLAC(single likelihood ancestor counting)，were used to calculate the likelihood [31]. All analyses were performed using the software HyPhy (http://www.hyphy.org) and the online tool Datamonkey (http://www.datamonkey.org/). Spidermonkey[32],a component ofHyPhy,was used to identifypairs of sites that co-evolved during IBDV evolution. Summing the probabilities of all branches gives the overall probability $\boldsymbol { Q }$ of co-occurrence of a pair of amino acid substitutions.A threshold probability of O.9 was set,which means apair of amino acid substitutions was regarded as co-evolving if $\boldsymbol { Q }$ was larger than 0.9.

# 2 Results

# 2.1 Virulence of vvIBDVGx

Young SPF chickens were infected with purified vvIBDV Gx virions and the bursae of infected chickens were subjected to histopathology after3 to 5 d[18,33].In comparison with infection by attenuated IBDV Gt, infection by vvIBDV $\mathbf { G } \mathbf { x }$ induced severe bursal lesions and resulted in ${ \sim } 1 0 0 \%$ morbidity and ${ \sim } 6 0 \%$ mortality (Figure 1). Pathological ex

A GXVP2.PRO SSQYQAGGVTITLFSANIDAITSLSIGGELVEQTSVQGLI 256 GTVP2.PRO SSQYQPGGVTITLFSANIDAITSLSVGGELVERTSVHGLV 256 Consensus ssqyq ggvtitlfsanidaitsls ggelvf tsv gl GXVP2.PRO LGATIYLIGFDGTAVITRAVAADNGLTAGTDNLMPFNIVI 296 GTVP2.PRO LGATIYLIGFDGTTVITRAVAANNGLTTGTDNLMPFNLVI 296 Consensus lgatiyligfdgt vitravaa nglt gtdnlmpfn vi GXVP2.PRO PTSEITQPITSIKLEIVTSKSGGQAGDQMSWSASGSLAVT 336 GTVP2.PRO PTNEITQPITSIKLEIVTSKSGGQAGDQMSWSARGSLAVT 336 Consensus pt eitqpitsikleivtsksggqagdqmswsa gslavt GXVP2.PRO IHGGNYPGALRPVT 350 GTVP2.PRO IHGGNYPGALRPVT 350 Consensus ihggnypgalrpvt   
B vvGx Gt DMEM

amination of infected tissues showed severe lymphocyte depletion, bursal atrophy, connective tissue hyperplasia, and regional fibrosis.In contrast,infection with IBDV Gt induced only subclinical pathological changes,including slight hemorrhage and ecchymosis.No bursal lesions were observed in the negative control (DMEM） group (Figure 1B).These results show that the purified vvIBDV Gx virions were highly infectious to bursae of young chickens.

VP2,the major capsid protein of IBDV,is a key determinant of virulence,cell tropism，and pathogenic phenoltype [16].We compared the VP2 sequences of vvIBDV $\mathbf { G } \mathbf { x }$ and attenuated IBDV Gt and found that the homology between them is greater than $9 7 \%$ ，while two amino acid substitutions，H253Q and T284A，exist in the hypervariable region of vvIBDV $\mathbf { G } \mathbf { x }$ VP2 (Figure 1A). In a previous study in which vvIBDV Gx was attenuated to IBDV Gt by blind passage,we identified residues 253 and 284 as possible key virulence sites of vvIBDV Gx [18]. These two amino acids were selected for the molecular dynamics simulation and structural analysis in this study.

# 2.2Interactions between virulence sites

We built a pseudo-atomic structural model of vvIBDV Gx VP2by homology modelingusingI-TASSER,based on the crystal structure of cvIBDV VP2 (PDB: 2DF7) (Figure 2). The overall architectures of VP2 in these two strains of IBDV are quite similar (Figure 2A). Most of the conformational differences are found in four flexible loops $( \mathrm { P _ { B C } , P _ { H I } } ,$ （204 $\mathrm { \Delta P _ { D E } }$ ,and $\mathrm { P } _ { \mathrm { F G } } \backslash$ in the jellyroll $\beta$ barrels of the P domain (Figure 2B).Both virulence sites of vvIBDV $\mathbf { G } \mathbf { x }$ VP2,Q253 and A284,are exposed at the tips of loops $\mathrm { \Delta P _ { D E } }$ and $\mathrm { P } _ { \mathrm { F G } }$

To determine how the amino acid mutations at positions 253 and 284 in VP2 may contribute to virulence change, we performed a simulation of the dynamics of a VP2 trimer based on the crystal structure of cvIBDV VP2.Four systems,including the wild type (253H,284T),two single mutations (H253Q，T284A),and a double mutation $( \mathrm { H } 2 5 3 \mathrm { Q } / \$ T284A),were subjected to long-timescale MD simulations. A total simulation time of 114 ns was used for each system, allowing 14 ns for equilibration and 1OO ns for the subsequent isothermal-isobaric simulation.

According to the homology model, the distance between virulence sites 253 and 284 is slightly greater in vvIBDV Gx VP2 than in cvIBDV VP2 (Figure 2B). In the MD simulation，residues H253 and T284 of cvIBDV VP2 were found to interact with each other via a hydrogen bond (Figure 3A),which could stabilize the connection between loops $\mathrm { \Delta P _ { D E } }$ and $\mathrm { P } _ { \mathrm { F G } }$ .In contrast, this hydrogen bond is remarkably weakened when VP2 carries the mutation H253Q (Figure 3B and 3C)and is completely abolished by the mutations $\mathrm { T 2 8 4 A }$ and H253Q/T284A (data not shown). These results suggest that the mutations at positions 253 and 284 in vvIBDV VP2 could disrupt the hydrogen bond between these two sites and lead to the relaxation of loops $\mathrm { \Delta P _ { D E } }$ and $\mathrm { P } _ { \mathrm { F G } }$ ，which may contribute to the receptor binding and infection process of vvIBDV.

# 2.3Positively selected and co-evolving sites in IBDV VP2

We collected 726 VP2 coding sequences of variant IBDV strains reported in GenBank (including 93 strains of vvIBDV）and performed a positive selection analysis to identify positively selected sites.All sequences were initially pooled together and non-synonymous/synonymous substitution rate ratios(@=dN/dS）were calculated,but no significant signal of positive selection was found.We then classified the strains into seven geographic regions,i.e., Europe(78 strains),East Asia(China,Korea,and Japan; 114 strains),West Asia (33 strains), South Asia (38 stains),

![](images/12150828ead415a7913f262d30f4cfd6be4339e6c04dab0df5462b57b56c6d39.jpg)  
FigureorpologcaldierecesbetweetheX-ratrucralodelofcBDVVB:2FndteomologodelofvGVP2A AlignmentfthecBVP2X-ramodel(cya）ndthevBDVGxVPomologmodel(uple).B,Enlargedviewof tesquarein(Ashowing details of amino acid residues 253 in loop $\mathrm { \Delta P _ { D E } }$ and 284 in loop $\mathrm { P } _ { \mathrm { F G } }$

![](images/597b9bde66423ad5a7c78c49d8ee5ff2dded35002091f2ae82c82b29a55b8444.jpg)  
Figure3MDsimlatooftherogenodbetweesidus23nd284ineDVP2trer.A,ThydrogodtweenH5d84in cvIBDVVP2AstablehrogdisdetenH5ad84GBnCerogdbwesiusd4iss ble nvIBDVVP2withtheutationH25Q(BQ253OE1vs.84G1;CQ53NE1s.84OG)TheoztaldverticaaesidicatetheD simulation time and the distance between H-bond acceptor and donor $( \mathrm { \AA } )$ ,respectively.The red line indicates a threshold of $3 . 5 \mathrm { \AA }$ ,below which an H-bond is formed.

North America (285 strains),South America (1O1 strains), and Africa(77 strains),and performed the positive selection analysis again.This time, one to three significant positively selected sites were identified in each group except West Asia.Residue 253 $\scriptstyle ( P = 0 . 0 3 3 )$ ）and residue 284 $\scriptstyle ( P = 0 . 0 8 5 )$ were identified as positively selected sites in East Asia at a significance level of O.1 (Table 1).Analysis of the phylogenetic relationships among IBDV strains by geographic region indicates that the strains in Asia are more closely related to those in Europe than to those in other areas such as the Americas (Figure 4A).

All of the positively selected sites identified by this analysis arelocatedin the $\mathrm { ~ \bf ~ P ~ }$ domain ofVP2.Most of them are present in the $\beta$ -barrel loops,i.e., $\mathrm { P _ { B C } } .$ $\mathrm { \bf P _ { H I } }$ $\mathrm { P _ { D E } } .$ and $\mathrm { P } _ { \mathrm { F G } }$ which are found on the exterior surface ofIBDV and thus come into contact with exogenous proteins (Figure 4B). Interestingly,although the amino acid variability of the hypervariable region of VP2 is similar among vvIBDV in East Asia,cvIBDV in East Asia,and vvIBDV in other regions (Figure 5),two sites important for the virulence of IBDV, Q253 and A284，were found to be positively selected in East Asia (Table 1).

Table1 Positively selected sites in the VP2 hypervariable region of IBDV strains from different geographic regions   

<html><body><table><tr><td>Region</td><td>Site</td><td>dNIdS</td><td>P-value</td></tr><tr><td rowspan="2">Europe</td><td>249</td><td>4.163</td><td>0.067</td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan="2">East Asia</td><td>253</td><td>1.767</td><td>0.033</td></tr><tr><td>284</td><td>1.294</td><td>0.085</td></tr><tr><td>South Asia</td><td>222</td><td>5.723</td><td>0.080</td></tr><tr><td rowspan="3">Africa</td><td>222</td><td>2.384</td><td>0.060</td></tr><tr><td>253</td><td>2.906</td><td>0.088</td></tr><tr><td>300</td><td>3.230</td><td>0.045</td></tr><tr><td rowspan="2">South America</td><td>253</td><td>1.275</td><td>0.063</td></tr><tr><td>270</td><td>1.171</td><td>0.031</td></tr><tr><td>North America</td><td>222</td><td>13.50</td><td>0.063</td></tr></table></body></html>

We then performed a co-evolution analysis of the VP2 sequences of vvIBDV and cvIBDV strains in East Asia. The analysis identified three significant pairs of co-evolved sites:253 and 284（ $\scriptstyle ( Q = 0 . 9 7 )$ ，284and294 $\scriptstyle ( Q = 0 . 9 5 )$ ，and 242 and 279( $\scriptstyle ( Q = 0 . 9 3 )$ (Figure 4C). Interestingly, the pair of amino acid sites with the largest $\boldsymbol { Q }$ value,253 and 284,is the same as that identified in the positive selection analysis. These results suggest a direct relationship or contact between these two virulence sites.

# 3Discussion

Inthis study,we purified vvIBDVGx virions from the bursae of infected young SPF chickens and analyzed the virulence of this strain ofIBDV.Compared with the attenuated IBDV Gt, the purified vvIBDV Gx virions are highly infectious to bursae; they have mutations at amino acids 253 and 284 in VP2,which were previously reported to be important for the virulence of IBDV[15]. To explore the molecular basis of the contribution of these two mutations to virulence change， we built a pseudo-atomic structural model of vvIBDV Gx VP2 by homology modeling and compared it with the structure of cvIBDV VP2.The overall architectures of VP2 from vvIBDV and cvIBDV are quite similar. Nevertheless,structural differences,mostly in the P domain of VP2,were found. Although the amino acid sequences of variant strains of IBDV are generally well conserved, mutations are frequently observed,especially in the hypervariable region of VP2.

To further explore the effects of the H253Q and T284A mutations on the virulence of IBDV,we performed a molecular dynamics simulation of the structural changes that occur in VP2 with different mutations. One amino acid mutation,H253Q,replaces a relatively bulky side chain (histidine）with a more linear side chain(glutamine),which may increase the flexibility of loop $\mathrm { \Delta P _ { D E } }$ in the P domain of VP2. The other amino acid mutation,T284A,involves a change from a hydrophilic residue (threonine）to a hydrophobic residue (alanine),which could increase the hydrophobicity of the surface of loop $\mathrm { \Delta P _ { F G } }$ .A more flexible conformation of loops and a more hydrophobic environment could facilitate the interaction between virus and target cell and promote stronger binding to IBDV-specific receptors.

![](images/90e3469462297c4037e5fcf6ce575b1159004411b09500babf114e36141e0409.jpg)  
Figure4ocaiofiielyeeteddevlingsiin2.Agetcoftrasyoacio sentOdo signal in the VP2 protein.

![](images/1bd863fb46ea907d885888d4ac35154f84e994e42f2b085a3804129e70618eb6.jpg)  
Figure5Amnocidrabilitofteearablegioof2iVstraisThtalispoiosineocidced the vertical axis shows the entropy (a measure of amino acid variability) at each site.

Our MD simulation showed that there is a hydrogen bond between H253 and T284 in cvIBDV.This hydrogen bond is weakened by the mutation H253Q and completely abolished in vvIBDV VP2 with both mutations (H253Q and T284A). This disruption of the hydrogen bond between residues 253 and 284 by amino acid substitutions could affect the conformational flexibility of the loops in thePdomainof VP2 and likelyalter the virulence of IBDV.

The degree of virulence could be important for the evolution and adaptation of IBDV in different hosts or environments.To determine whether the amino acid substitutions at the virulence sites were positively selected during the molecular evolution of IBDV，we classified the VP2 coding sequences of variant IBDV strains into different groups by geographic origin and performed a positive selection analysis.Amino acid sites with significant signals of positive selection were found in individual geographic regions.Most of these sites are in the exposed $\beta$ -barrelloops of VP2 and thus could contribute to the interaction between IBDV and its specific receptors.Interestingly, the two virulence sites,253 and 284，were found to be positively selectedinEastAsia.Theseresultsindicatethatthevirulence sites could be critical in the adaptive evolution of the virus.

Our positive selection analysis suggested that the virulence sites of IBDV may be subjected to different pressures in different geographic regions.We identified positively selected sites when we performed the analysis for six of the seven individual geographic regions,but no positively selected sites with obvious signals were identified when we performed a similar analysis using pooled vvIBDV VP2 sequences from all six geographic areas.Although the different regions exhibit similarlevels of VP2 amino acid variability (Figure 5),positively selected sites could be detected only when the IBDV strains were grouped by geographic region (Table 1). This could be due to diverse adaptations to the specific environments found in the different geographic areas.

In our positive selection analysis,amino acid 253 was identified as a target of positive selection in most geographic regions (four of six),while site 284 was identified only in East Asia;moreover, site 284 had a lower dN/dS ratio (lower positive selection pressure) than site 253(Table1).These results suggest that the mutation at site 253 plays a key role in determining the virulence of IBDV,while the mutation at site 284 could be population specific and may play a minor role in virulence change.We calculated the frequencies of amino acid residues at positions 253 and 284 in variant strains ofIBDV.Interestingly，most vvIBDVstrains $( 9 6 . 6 \% )$ have Q and A at sites 253 and 284,respectively, while most IBDVstrains with intermediate virulence $( 8 2 . 4 \% )$ have Qand $\mathrm { \Delta T }$ at these two sites,respectively. Furthermore,all avirulent strains have H and $\mathrm { \Delta T }$ at these two sites (Table 2). These results also suggest that site 253 plays a more critical role in the virulence of IBDV.

A single amino acid substitution can have a great influence on virulence; however,virulence can be affected by multiple factors as well as multi-site interactions [34].An example of a multi-site interaction that may play a role in determining virulence is the co-evolution of different sites. Three pairs of amino acids,including pair 253/284,were identified in our co-evolution analysis as co-evolving sites that may promote changes in the virulence of IBDV.

In summary,in this study we analyzed the structure of VP2 and characterized the virulence sites 253 and 284 in vvIBDVGx,a strain identified in China,and explored the molecular mechanism of shifts in the virulence of IBDV caused by amino acid substitutions.Amino acids 253 and 284 in the hypervariable region of VP2 were found to be both positively selected and co-evolving，which suggests a diversity of virulence determination factors.Two mutations in VP2，H253Q and T284A，increase the virulence of vvIBDV Gx in comparison with avirulent strain,but amino acid 253 plays a more critical role.A future study combining structure and sequence analysis of different strains of IBDV will provide more insights into the mechanism of virulence and molecular evolutionof thisvirus.

Table 2Summary of amino acid frequencies at the IBDV virulence sites 253 and 284 VP2 gene sequences and IBDV genotypes were obtained from GenBank   

<html><body><table><tr><td>Genotype</td><td>253</td><td>284</td><td>Proportion</td></tr><tr><td>Very virulent</td><td>Q</td><td>A</td><td>96.6%</td></tr><tr><td></td><td>Q</td><td>T</td><td>3.4%</td></tr><tr><td>Intermediate</td><td>Q</td><td>A</td><td>82.3%</td></tr><tr><td></td><td>Q</td><td>T</td><td>5.7%</td></tr><tr><td></td><td>H</td><td>T</td><td>9.7%</td></tr><tr><td></td><td>N</td><td>T</td><td>1.7%</td></tr><tr><td></td><td>L</td><td>T</td><td>0.6%</td></tr><tr><td>Avirulent</td><td>H</td><td>T</td><td>100%</td></tr></table></body></html>

This work was supported by the National Natural Science Foundation of China (31230018,31430087),and the National Science and Technology MajorProject for infectious disease of China (20l3ZXlooo4606).The moleculardynamics simulationwasperformed intheHighPerformance Computing Centeratthe InstituteofBiophysics,ChineseAcademyofSciences.We thank Zhu HongTao for help with homology modelingand Ling LunJiangfortechnical assistanceattheHPC-Service Station.

1 Lasher HN,Davis VS.History of infectious bursal disease in the USA—the first two decades.Avian Dis,1997,41:11-19

2Becht H.Infectious bursal disease virus.Curr Top Microbiol Immunol,1980,90: 107-121   
3 Muller H,Islam MR,Raue R.Research on infectious bursal disease—the past,the present and the future.Vet Microbiol, 2003,97: 153-165   
4 van den Berg TP,Eterradossi N,Toquin D,Meulemans G.Infectious bursal disease (Gumboro disease).Rev Sci Tech,200O,19: 509-543   
5 Ismail NM, Saif YM,Moorhead PD.Lack of pathogenicity of five serotype 2 infectious bursal disease viruses in chickens.Avian Dis, 1988,32: 757-759   
6 Kibenge FS,Dhillon AS,Russell RG.Biochemistry and immunology of infectious bursal disease virus.JGen Virol,1988,69:1757-1775 7 Jackwood DH, Saif YM.Antigenic diversity of infectious bursal disease viruses.Avian Dis,1987,31: 766-770   
8 Eterradossi N,Arnauld C,Tekaia F,Toquin D,Le Coq H,Rivallan G, Guittet M,Domenech J,van den Berg TP, Skinner MA.Antigenic and genetic relationships between European very virulent infectious bursal disease viruses and an early West African isolate.Avian Pathol,1999,28:36-46 9 Mundt E,Beyer J,Muller H.Identification of a novel viral protein in infectious bursal disease virus-infected cells.J Gen Virol,1995,76: 437-443   
10 Da Costa B,Chevalier C,Henry C,Huet JC,Petit S,Lepault J,Boot H,Delmas B.The capsid of infectious bursal disease virus contains several small peptides arising from the maturation process of pVP2. J Virol,2002,76: 2393-2402   
11 Caston JR,Martinez-Torrecuadrada JL,Maraver A,Lombardo E, Rodriguez JF,Casal JI, Carrascosa JL.C terminus of infectious bursal disease virus major capsid protein VP2 is involved in definition of the T number for capsid assembly.JVirol,2001,75:10815-10828   
12 CoulibalyF,Chevalier C,Gutsche I,Pous J,NavazaJ,Bressanelli S, Delmas B,Rey FA.The birnavirus crystal structure reveals structural relationships among icosahedral viruses.Cell,2005,12O:761-772   
13 Luque D,Saugar I, Rodriguez JF,Verdaguer N, Garriga D, Martin C, Velazquez-Muriel JA,Trus BL,Carrascosa JL,Caston JR.Infectious bursal disease virus capsid assembly and maturation by structural rearrangements of a transient molecular switch.J Virol,2O07,81: 6869-6878   
14 Mundt E.Tissue culture infectivity of different strains of infectious bursal disease virus is determined by distinct amino acids in VP2.J Gen Virol,1999,80: 2067-2076   
15 van Loon AA,de Haas N, ZeydaI,Mundt E.Alteration of amino acids in VP2 of very virulent infectious bursal disease virus results in tissue culture adaptation and attenuation in chickens.J Gen Virol, 2002,83:121-129   
16Brandt M, Yao K,Liu M,Heckert RA,Vakharia VN.Molecular determinants of virulence,cell tropism,and pathogenic phenotype of infectious bursal disease virus.JVirol,2001,75:11974-11982   
17 LimBL,Cao YC,Yu T,Mo CW.Adaptation of very virulent infectious bursal disease virus to chicken embryonic fibroblasts by site-directed mutagenesis of residues 279 and 284 of viral coat protein VP2.J Virol,1999,73:2854-2862   
18Qi XL,Gao HL,Gao YL,Qin LT,Wang YQ,Gao L,Wang XM. Naturally occurring mutations at residues 253 and 284 in VP2 contribute to the cell tropism and virulence of very virulent infectious bursal disease virus.Antivir Res,2009,84: 225-233   
19 Ben Abdeljelil N,Khabouchi N,Kassar S,Miled K,Boubaker S, Ghram A,Mardassi H. Simultaneous alteration of residues 279 and 284 of the VP2 major capsid protein of a very virulent Infectious Bursal Disease Virus(vvIBDV) strain did not lead to attenuation in chickens.VirolJ,2014,11:199   
20Jackwood DJ,Sreedevi B,LeFever LJ, Sommer-Wagner SE.Studies on naturally occurring infectious bursal disease viruses suggest that a single amino acid substitution at position 253 in VP2 increases pathogenicity.Virology,2008,377:110-116   
21 Sikosek T,Chan HS.Biophysics of protein evolution and evolutionary protein biophysics.JR Soc Interface,2014,11: 20140419   
22 Pond SL,Frost SD,Grossman Z,Gravenor MB,Richman DD, Brown AJ.Adaptation to different human populations by HIV-1 revealed by codon-based analyses.PLoS Comput Biol，2006，2: 530-538   
23 Sunyaev SR,Lugovskoy A, Simon K,Gorelik L.Adaptive mutations in the JC virus protein capsid are associated with progressive multifocal leukoencephalopathy (PML).PLoS Genet,2009,5: e1000368   
24 Roy A,Kucukural A,Zhang Y.I-TASSER:a unified platform for automated protein structure and function prediction. Nat Protoc, 2010,5: 725-738   
25Phillips JC,Braun R,Wang W,Gumbart J, Tajkhorshid E,Villa E, Chipot C, Skeel RD,Kale L, Schulten K. Scalable molecular dynamics with NAMD.JComput Chem,2005,26:1781-1802   
26Lee CC,Ko TP,Chou CC,Yoshimura M,Doong SR,Wang MY, Wang AH.Crystal structure of infectious bursal disease virus VP2 subviral particle at 2.6 angstrom resolution:implications in virion assembly and immunogenicity. J Struct Biol, 2006,155: 74-86   
27 Xia XH. DAMBE5:a comprehensive software package for data analysis in molecular biology and evolution.Mol Biol Evol,2013, 30: 1720-1728   
28 Tamura K,Stecher G,Peterson D,Filipski A,Kumar S.MEGA6: molecular evolutionary genetics analysis version 6.O.Mol Biol Evol, 2013,30:2725-2729   
29Nielsen R,Yang ZH.Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene. Genetics,1998,148:929-936   
30 Suzuki Y,Gojobori T.A method for detecting positive selection at single amino acid sites.Mol Biol Evol,1999,16:1315-1328   
31 Pond SL,Frost SD,Muse SV.HyPhy: hypothesis testing using phylogenies.Bioinformatics,2005,21: 676-679   
32 Poon AF,Lewis FI, Pond SL,Frost SD.An evolutionary-network model reveals stratified interactions in the V3 loop of the HIV-1 envelope.PLoS Comput Biol,2007,3: e231   
33 Qi X, Gao W, Gao HL,Deng XY,Bu ZG,Wang XY,Fu CY,Wang XM.An improved method for infectious bursal disease virus rescue using RNA polymerase II system. J Virol Methods,2007,142: 81-88   
34Bull JJ,Lauring AS.Theory and empiricism in virulence evolution. PLoS Pathog,2014,10: e1004387