# SCIENTIFIC REPORTS

# OPEN

# Integrative analysis of human protein, function and disease networks

Received: 10 May 2015   
Accepted: 26 August 2015   
Published: 24 September 2015

Wei Liu¹, Aiping $\boldsymbol { \mathsf { W } } \boldsymbol { \mathsf { U } } ^ { 2 , 3 , 4 }$ , Matteo Pellegrini5 & Xiaofan Wang1

Protein-protein interaction (PPl networks serve as a powerful tool for unraveling protein functions, disease-gene and disease-disease associations. However,a direct strategy for integrating protein interaction,protein function and diseases is stillabsent.Moreover, the interrelated relationships among these three levels are poorly understood. Here we present a novel systematic method to integrate protein interaction,function,and disease networks.We first identified topological modules in human protein interaction data using the network topological algorithm (NeTA) we previously developed.The resulting modules were then associated with functional terms using Gene Ontology to obtain functional modules.Finally,disease modules were constructed by associating the modules with OMlM and GWAS.We found that most topological modules have cohesive structure, significant pathway annotations and good modularity. Most functional modules $( 7 0 . 6 \% )$ fully cover corresponding topological modules,and most disease modules $( 8 8 . 5 \% )$ are fully covered by the corresponding functional modules.Furthermore,we identified several protein modules of interest that we describe in detail, which demonstrate the power of our integrative approach.This approach allows Us to link genes,and pathways with their corresponding disorders,which may ultimately help us to improve the prevention, diagnosis and treatment of disease.

Network methods are powerful tools for unraveling protein functions,protein-pathway associations, disease-gene and disease-disease associations. However, these disparate types of networks are more often studied independently of each other.To date, there has been great progressin the study of protein interaction networks.Previous research on protein networks $^ { 1 - 9 }$ mainly focused on analyzing the associations between genes, functional modules, and pathways.Using these approaches, usually only a fraction of detected protein modules have good mapping to biological functions or pathway annotations. Similarly, previous studies of disease networks10-24 mainly focused on disease classification and the prediction of disease genes.Recently, several groups have studied human disease networks25,26,to shed light on the relationship between disease genes and disease networks,as well as disease gene modules and their functional analysis.These methods start from diseasome²7,which is a bipartite gene-disease network, from which we can derive two diffrent disease networks: disease-disease networks and disease gene networks.Disease networks may help us to understand phenotype associations between proteins and diseases. Thus,a direct strategy for integrating protein interactions, protein function and disease pattrns is still absent,and the interrelated relationships among these three levels have been poorly investigated.

To better understand the relationships between these three network types,we present a multi-network systematic analysis method. Using our approach, protein modules are determined directly from topological modules using the network topological algorithm we previously developed (NeTA28). Traditionally,

1Department of Automation, Shanghai Jiao Tong University, Shanghai, $2 0 0 2 4 0 ,$ China.²Institute of Biophysics, Chinese AcademyofSciences,Beijing,1ooo8o.3Center forSystems Medicine,Instituteof Basic MedicalSciences, Chinese Academy of Medical Sciences & Peking Union Medical Collge，Beijing 1oooo5.4Suzhou Institute of Systems Medicine, Suzhou,Jiangsu 215123,China.5Department of Molecular, Celland Developmental Biology, Universityof California,Los Angeles,CA,goo55.Correspondenceand requests formaterials should beaddressed to X.W. (email: xfwang@sjtu.edu.cn)

![](images/bf3614954aebbcdbb885400190052df5d3a678f2b04f6ca46d104feb3d70f81e.jpg)  
Figure l. The schematic of multi-networks mapping method.There are three steps in multi-networks mapping method, including clustering,mapping and integrative analysis.

a protein module is defined as a group of proteins that carry out similar functions. These functions are associated with the same pathway,and could be associated with a particular disease.Here we focus on three distinct protein modules: topological, functional and disease modules25,26.Topological modules represent a locally dense structure in protein-protein interaction (PPI) networks; function modules represent the aggregation of proteins of related function in a function network; disease modules represents a group of proteins that share a common disease phenotype within a disease network. Though the three types of modules are derived from three different types of networks, they can be closely interrelated and highly overlapping25.

Starting with the protein interaction dataset from Hippie29,we identified 136 topological modules with NeTA,136 corresponding functional modules (annotated using Gene Ontology30),and 139 disease modules annotated using $\mathrm { O M M M } ^ { 3 1 }$ and $\mathrm { G W A S ^ { 3 2 } }$ . To our surprise,most functional modules $( 7 0 . 6 \% )$ are highly consistent with the corresponding topological modules,and most disease modules $( 8 8 . 5 \% )$ are fully covered by the corresponding functional modules,and have significant pathway annotations. By systematically integrating the threelevels of networks and protein modules,we found that our multi-level method for biological interpretation has distinct advantages over approaches that only consider subsets of data and annotations.Many interesting modules are found that could not be easily discovered by only one data type.For example,we identified several protein interaction modules that allowed us to connect inflammatory responses to Alzheimer's disease, suggesting that this pathology may have a strong inflammatory component. Moreover, in many modules, we found that a subset of genes is associated with specific functions or diseases,allowing us to identify genes and pathways with their corresponding disorders.The approach we present here not only provides an avenue for network integration,but also promises to shed light on the prevention, diagnosis and treatment of complex diseases.

# Results

The integrated multi-networks mapping method. Figure 1 shows a schematic of our overall approach,the framework of the integrated multi-networks mapping method,which consisted of three steps.First we determined the topological modules from a human PPI network. Next, we annotated all topological modules using Gene Ontology (GO), to obtain functional modules. Finall, we included OMIM and GWAS data to obtain disease modules. Thus,three levels of networks were constructed and modules were identified at each level, including a protein network and its topological modules,a function network and its functional modules,and a disease network and its disease modules.Finally, we integrate the three types of networks and modules,to discover modules that have coherent function

90L 中云 号国 5 9中 中 0  
40 0 0 学 O 中中出  
20 山 中心 0 珍珍 泰巷  
10 5 你 ） 费 00 山20 40 60 80 100 120 140 专 中 中中The index of condensed modules 中市中(a) (b)

and disease interpretation, leading to new associations that are not evident when analyzing only a single type of network.

Protein Networks and Topological Modules. The human PPI network was constructed based on the HIPPIE29 and IRefWeb3 databases, which results in a network of 2484 direct physical interactions among 1830 proteins.We detected 136 large modules and 185 smallmodules (most of which only contain two proteins)by applying the network topology algorithm NeTA28.Here we analyze the l36 larger topological modules (as shown in Supplementary Table 1).This PPI network contains 1390 proteins, and 2228 interactions (Fig.2b),which results in $7 6 \%$ of the proteins being associated with $8 9 . 7 \%$ of the interactions of the PPI network.As Fig.2a shows,the size of larger modules runs from 3 to 88. In Fig.2b,different colors represent different modules,and we can clearly see that this network has a modular structure. The modularity $\mathrm { Q } ^ { 3 4 }$ is 0.91385,which means these modules have significantly more community structure than a random zero-model.

Function Networks and Functional Modules. To build a function network, we mapped each topological module into Bingo5,and analyzed the GO enrichments of each module at three levels of Gene Ontology（GO） slim,using annotation from Biological Process,Celllar Component, and Molecular Function ontologies30.A functional module was defined as a group of genes in a topological module that is associated with a specific GO term.In total,we found 136 functional modules (as shown in Supplementary Table 2) with at least three proteins.If we don't considertheunannotated proteins (there is only one protein in our network that has no annotations in Bingo),we found that 96 $( 7 0 . 6 \% )$ of our topological modules are fully covered by functional modules (i.e. allthe proteins map to the same function term).For example, topological module 3 consists of 8 proteins, COPA, COPB,COPD, COPE, COPB2, COPZ1, COPG2,and TMEDA (Fig.3a).Al eight proteins share the same BP function “Golgi vesicle transport"(p-value is 2.4E-16),as wellas the same CC function “cytoplasmic vesicle membrane" (p-value is 9.71E-15).In general,allother modules are covered byat most two function modules.An example of this is topological module 9, which has four genes: BL1S1,BL1S2,BL1S3 and SNAPN(Fig.3b),that are associated with two functional modules: BL1S1,BL1S2, BL1S3("celular pigmentation", p-value is 4.01E-7) and BL1S1,BL1S3 and SNAPN("vesicle-mediated transport", p-value is 3.64E-3).

Furthermore,each topological module was annotated using the DAVID36,37 online analysis tool to identify pathway enrichment (see Methods),and construct protein-pathway networks.We found 88 topological modules (as shown in Supplementary Table 4) significantly associated with a pathway,and pathway genes are closely related with corresponding functional module genes.For example,as Fig.3c shows, topological module 23 has 17 proteins,allof which are annotated as“DNA Replication pathway"(p-value is 1.09E-25),as well as “nucleoplasm"(p-value is 3.06E-20).

Disease Networks and Disease Modules.To build the relationship between proteins and diseases, we mapped each topological module to the $\mathrm { O M I M } ^ { 3 1 }$ and $\mathrm { G W A S ^ { 3 2 } }$ databases.In total, 109 topological modules have disease genes,and 139 significant disease modules (as shown in Supplementary Table 3) were identified. One topological module may corresponds to one or more than one disease module. For example,topological module 6 has six genes,of which EGLN,TGFB1, TGFR1,and TGFR2 are disease genes associated with Bone and Cardiovascular diseases,which we therefore label as a disease module (Fig. 4a).Another example,topological module 11, with 4 genes, contains MEIS1,MEIS2 and

![](images/c9199f5dc02221fe033717a2c47ba4503be2a4804d0228ae88cab30cf08dc297.jpg)  
Figure 3.Function network. (a) The protein-function network of topological module 3.(b)The proteinfunction network of topological module 9.Round denotes proteins,and rectangles denote functions.Green denotes BP,cyan denotes CC,and deep purple denotes MF. (c) The protein-Pathway network of topological module 23. Round denotes proteins,and diamond denotes pathway. Orange denotes KEGG pathway, and light green denotes Reactome pathway.

![](images/cfcbb243cea90a203290aff5928005357d139d6b53da723f6feeb5fa2446ca77.jpg)  
Figure 4.Disease network. (a) The disease-gene network of topological module 6.(b) The disease-gene network of topological module 11.The Round denotes proteins,and elipse denotes diseases.Red denotes OMIM diseases,and purple denotes GWAS diseases. (c) The Disease-disease network of topological module 33.Nodes denote diseases and link between two diseases denote they share at least one disease gene.

PBX1,which are disease genes associated with Cardiovascular,Neurological, Psychiatric,Endocrine and Respiratory diseases,and these were also defined as a disease module (Fig.4b).

The study of associations between diseases is also potentially interesting,as it could help understand relationships between complex syndromes. We constructed a disease-disease network for each topological module. Nodes are diseases that are associated with one gene or multiple genes in the topological module,and edges between two diseases denote that they share at least one disease gene. Closely related diseases may be associated with complex syndromes.For example,Fig. 4c shows the disease-disease network of topological module 33: Thyroid carcinoma (papillary),carney complex (type 1),Adrenocortical tumor (somatic), Pigmented adrenocortical disease (primary) and Myxoma (intracardiac), which are all cancers,and besides Myxoma,are also endocrine pathologies.

Integrative Analysis. Considering protein interaction, function,and disease networks independently significantly limits our ability to carry out a systematic study of the data.As a result,we integrated protein,function and disease networks,in order to annotate protein modules according to their function and disease associations,to gain a systematic view of these relationships. In addition,to view the relationship between different types of modules, we also integrated topological modules with functional and disease modules. If a disease module is highly overlapping (over half of proteins) with a functional module,then we defined its corresponding topological module as a non-trivial protein module; if a disease module is highly overlapping (over half of proteins) with a pathway module, then we defined its corresponding topological module as a significant protein module. Using this integrative analysis,we identified 69 non-trivial protein modules,and 47 significant protein modules in our PPI network. We discuss a few examples below.

Figure 5a shows an intriguing non-trivial protein module (Topological module 55） that connects leptin and the leptin receptor to the inflammatory cytokine receptor IL6RB. There is extensive literature implicating leptin to obesity and diabetes38.However, this module shows us that these disorders are also associated with inflammation (through IL6). There is increasing recognition that many metabolic disorders,such as diabetes,are also associated with higher levels of inflammation39.Thus this module suggests that anti-inflammatory treatments could be coupled with weight loss regimes to address metabolic disorders.

Figure 5b shows another non-trivial protein module (Topological module 82) with a number of factors that likely playa significant role in hematopoietic development. Specifically, Tall is amaster regulator of Tcelldevelopment,and inhibits the production of cardiac cells40.It is terefore interesting to see that several of the genes in this module are associated not only with Tcell development,but also with heart disease and heart rate.

Figure 5c describes a complex of proteins associated with NfkB (Topological module 113),a master regulator of inflammatory responses. One interesting observation is that several genes in this module are associated with Alzheimer's disease.This is of interest,as there is growing recognition that Alzheimer's disease may be associated with inflammation,and its risk is elevated by metabolic disorders,such as diabetes41.Thus this non-trivial protein module allows us to make the critical connection between these two important disorders,and the basal inflammatory responses of cells.

Figure 6 shows a significant protein module (Topological module 24) with four genes, SYN1, SYN2, SYN3 and CAPON. Of these SYN1, SYN2,and SYN3 are allassociated with psychiatric disease,and the synaptic transmission and synaptic vesicle trafficking pathway.

In addition, topological module 3 and 51 are also interesting non-trivial protein modules. All the proteins of module 3 are involved in Golgi vesicle transport,and most are also involved in the membrane traffcking pathway,and associated with Alzheimer's disease.Allthe proteins of module 51 are associated with translation initiation factor activity,and in the Metabolism of proteins pathway,and most are also associated with liver disease.

Comparison with existing methods. In recent years,a number of methods have been developed to identify functional modules $^ { 1 - 9 }$ and disease modules10-24 in PPI networks.Most methods to identify disease modules are disease protein prioritization methods.To evaluate the relative performance of our method, we compare our results with two representative methods that can identify functional and disease modules.One is the Markov Cluster Algorithm $( \mathrm { M C L } ) ^ { 4 2 }$ ，which is based on random flow(We use the default settings,inflation parameter $\mathbf { r } = 2$ ）and the other is random walker $( \mathrm { R W } ) ^ { 4 3 }$ that wanders from node to node along the links of the network.After every move the walker is reset to a randomly chosen seed gene with a given probability r (we use $\mathbf { r } = 0 . 4$ ）

Figure 7(a) shows the modularity results from NeTA, MCL and RW, which can qualify the clustering quality of the topological modules.NeTA performs better than the other two methods.Figure 7(b) shows the mapping frequency of the three methods based on the topological modules that were identified. As expected,among the three kinds of modules,no matter what method was used,we identify more functional than disease modules.NeTA identified more functional modules than MCL and RW,and also identified more pathway and disease modules.Figure $7 ( \mathrm { c } )$ shows the average mapping frequency of the three methods based on topological modules that were identified.Fore each method, we count each functional/disease module mapping frequency, and take the mean value, which measures the mapping accuracy of each method. We can see that the mapping accuracy of NeTA is higher than other two methods.Figure 7(d) describes the mapping frequency of non-trivial protein modules and significant protein modules.Again,NeTA finds more non-trivial and significant protein modules.Overall, we find that our method performs better than the other algorithms for all kinds of protein modules.

![](images/4a5cade697c066749fd91b2e70cf9ed9b695d6ed4dd874c64f39d00f5d05d430.jpg)  
Figure5. Integrative analysis of networks and non-trivial protein modules. (a) The integrative analysis of networks and modules of topological module 55. (b)The integrative analysis of networks and modules of topological module 82. (c) The integrative analysis of networks and modules of topological module 113.On the left is the integration of protein networks,function network and disease network,and on the right is the integration of corresponding topological module,functional module and disease module.

![](images/5c6d5c391cd1d136ae7dd34c739fe20388e1e118765edbc54559435e3b351000.jpg)  
Figure 6. Significant protein modules.The Disease network and protein-pathway network of topological module 24.Round denotes proteins,diamond denote pathway and ellipse denotes diseases.

![](images/76539eeb4a3d82d170b0dd97631e9ef0d009ad7b0821f5f1e0f36aa0d42aab1b.jpg)  
Figure 7. Performance evaluation of NeTA. (a) The modularity of three representative clustering algorithms.(b) We mapped the detected topological modules to function,pathway and disease database respectively,and computed the mapping frequency for each algorithm. (c) The average mapping frequency of all detected modules are counted for the three algorithms. (d) The mapped frequency of detected nontrivial protein modules and significant protein modules.

Systematic evaluation analysis.To systematically evaluate the power of our method to infer function-disease associations,we constructed a benchmark network based on the $\mathrm { O M I M } ^ { 3 1 }$ and MIPS human complex database44 We fltered human complex PPI with disease genes from OMIM,and constructed a network with 1460 proteins and 4107 protein-protein interactions.In this setting,there is at least one disease gene in each interaction. We use this as a benchmark network, to identify disease, protein complex and disease-complex modules,and compare our results with the MCL and RW algorithms.

Figure 8(a) shows the resulting modularity of NeTA, MCL and RW against this network. Among them,NeTA has the highest modularity, which shows it can obtain beter module structure than the other two methods.Figure 8(b) shows the number of different modules identified by the three methods.RW identified the most topological modules,and MCL identified the most complex modules,and NeTA identified the most disease modules and disease-complex modules. Figure 8(c) shows the mapping frequency of different modules identified by the three methods.Disease and protein complex modules can only map to approximately $20 \%$ of topological modules,and even fewer disease-complex modules. Overall, we find that our method performs competitively with the other algorithms.

# Discussion

Protein interaction,function and disease networks can be clustered into cohesive groups. Accordingly, these cohesive groups can be defined as topological, functional and disease modules.Most previously published approaches that analyze these datasets only focus on a subset of the three levels.For example, most work on PPI networks only focus on topological modules and their corresponding functional modules1-9. Other approaches analyze pathway enrichment of modules. Similarly, most of the work on disease networks focuses on disease genes and their classification10-24. As a result, an integrative analysis of all three levels of modules and networks has yet to be performed.

Here we present a systematic method for combining protein interactions,functions and disease networks,resulting in an integrative analysis that yields topological, functional and disease modules. Other integrative approaches start from Diseasome²7 to detect disease modules,and then identify functional and topological modules based on these. In contrast, we start from a human PPI network and detect 136 topological modules (as shown in Supplementary Table 1) using NeTA.We then annotate these topological modules using GO, OMIM and GWAS,and find corresponding functional and disease modules, leading to the construction of networks for each of the three levels. To visualize the associations among the three levels of modules and networks,we integrated the three levels together,and found that they generate new insights into protein network analysis.This approach alowed us to identify many interesting modules,which can't be fully annotated only using a single type of data.For example, we identified several protein interaction modules that alowed us to connect inflammatory responses to Alzheimer's disease, suggesting that this pathology may have a strong inflammatory component.

![](images/f6d5b11986da6c71bc27833ee6c8b3cd46785df032f7cfb14350b2b22b400e7f.jpg)  
Figure 8.Systematic evaluation of NeTA based ona benchmark network.(a)The modularity of three clustering algorithms.(b)We mapped the detected topological modules to OMIM disease database and MIPS human complex database respectively,and counted the mapped number of disease modules,complex modules and disease-complex modules respectively,and compared them with the other two algorithms. (c) The mapped frequency of disease modules,complex modules and disease-complex modules respectively, in comparison with the other two algorithms.

In topological module 3,which includes eight proteins,we found that all the proteins are involved in Golgi vesicle transport, and that COPA, COPB1, COPB2, COPD, COPE,COPG2 and COPZ1 belong to an octamer protein complex45-47. In addition, COPA,COPB1, COPB2, COPD, COPE and COPZ1 are involved in the membrane traffcking pathway,and COPA, COPB1, COPB2, TMED10 and COPG2 are associated with Alzheimer's disease45-47.Moreover, COPD is associated with increased risk for Mild Cognitive impairment,the earliest phase of Alzheimers disease,and COPZ1 is involved in intracelular trafficking45-47. Impairment of intracellular traffcking has been implicated in the pathogenesis of Alzheimer's disease,so COPZ1 may be associated with Alzheimer's disease. COPE is associated with depressive disorder, which is similar to the later phase of Alzheimer's disease, suggesting that COPE may also be associated with Alzheimer's disease.

Topological module 5 includes 11 genes, which are all located in the membrane.DMD,DTNA, NOS1, SNTA1,MAST2 and VAC14 are Type2 diabetes disease genes45-47,SCN5A is a diabetes melitus disease gene, SNTB2 and UTRN are type 1 diabetes disease genes46,47; SNTB1 controls glucose levels,and could be a potential diabetes disease gene.MAST1 is an important paralog of MAST245,and phosphorylation of DMD or UTRN may modulate their affnities for associated proteins,and thus may also be associated with diabetes mellitus.

Topological Module 51 includes 11 genes,which are components of the eukaryotic translation initiation factor3 (eIF-3) complex, which is required for several steps in the initiation of protein synthesis45. All these genes are related with translation initiation factor activity,and in the Metabolism of proteins pathway.In fact,the eIF-3 complex is composed of13 subunits,and EIF3Jand EIF3Mare not included in this module45.The most interesting observation is that all these genes are associated with liver diseases: EIF3A,EIF3B,EIF3C,EIF3D and EIF3G are all associated with Liver Failure47,Acute Hepatitis; EIF3E,EIF3F and EIF3K are associated with Liver Neoplasms47;EIF3H is associated with Carcinoma Hepatocelllar47; EIF3I is associated with clonorchiasis47.Furthermore,EIF3L hasa lower level of expression in liver cancer45-47. Therefore,the eIF-3 complex may be associated with liver disease as well.

As these examples illustrate,our work has the potential to inform the prevention,diagnosis and treatment of disease.It is often difficult to accurately identify potential gene targets based on GWAS,even though many GWAS variants are strongly associated with diseases. Although GWAS to protein associations affct the number of disease modules we can identify, we do not expect that these uncertainties significantly change the analysis results we obtained.In conclusion,our integrative analyses are still far from providing important therapeutic breakthroughs, which require substantial folow-up investigation. Nonetheles,they provide a wealth of hypothesis that could lead to clinical improvements in the future.

To make these hypotheses more robust, in subsequent work we intend to improve the method of noise reduction and data integration,split bigger modules into smallr ones,and integrate more levels of data together to improve our system level understanding of these complex diseases.

# Methods

Data source.HIPPIE29 is a human PPI database,and currently contains more than 156,0o0 interactions of $\sim 1 4 { , } 5 0 0$ human proteins.It integrates multiple major expert-curated experimental PPI databases, and all interactions have an associated normalized confidence score.Here we selected six public human PPI databases: BioGrid48, $\mathrm { D I P ^ { 4 9 } }$ ,HPRD50,IntAct51, $\mathrm { M I N T } ^ { 5 2 }$ and $\mathrm { B I N D } ^ { 5 3 }$ as our data sources based on the HIPPIE database,and identified high-confidence interactions based on the HIPPIE scoring system. To obtain more reliable interactions,we only keep those that are found in at least two pubic databases and are classified as high-confidence interactions.

Network Construction. Protein Network. We extracted high-confidence interactions from the HIPPIE database,and took direct physical interactions that cross multiple species based on the IRefWeb33 database to construct the final human PPI network. IRefWeb is a web interface to protein interaction data consolidated from 10 public databases.It can automatically crop the PPI dataset to produce a subset of higher-quality interactions, which aids the generation of more meaningful organism-specific interaction networks. In this network a node denotes a protein,and a link represents a protein-protein interaction.

Function Network.There are two kinds of function networks: one is a Protein-function network,and the other is a protein-pathway network. Protein-function networks are obtained by connecting proteins of each topological module (defined below) with corresponding GO biological processes,cellular localizations and molecular functions.In what follows we only used the third level under of GO slim terms30. In these networks nodes are proteins or GO terms.Edges are drawn between a protein and a term when a significant association between them exists (based on a hypergeometroc test P value between the functional modules and protein module). Protein-pathway networks are constructed by connecting proteins of each topological module with corresponding pathway annotations (pathway sources see below).

Disease Network.By mapping each topological module into the OMIM and GWAS database, we constructed two types of disease networks: Disease-gene networks and disease-disease networks.Disease-gene networks connect the genes in each topological module with their associated diseases.Disease-disease networks connect pairs of diseases if they share at least one disease gene.

Protein Module Detection. Topological Modules. High aggregation is an essential characteristic of biological networks,and it reflects high modularization of gene networks.The network we use was first clustered into different sizes of topological modules before further analysis.Accurately identifying topological modules of a biological network is still challnging. Here we detected topological modules based on a network topological algorithm NeTA28 (NeTA can detect sparse and small modules,and is competitive with other methods28),and we only consider those topological modules that contain at least three proteins.

Functional Modules.To evaluate the biological significance of these topological modules,we analyzed Gene Ontology)30 enrichment of each topological module with the Bingo35 plugin in Cytoscape54 with a threshold P-value $< 0 . 0 5$ based on the Hypergeometric test,and corrected by the Benjamini & Hochberg False Discovery Rate (FDR). Bingo generates hierarchical functional annotations based on GO slim. To obtain coherent functional modules,we only chose functions in the third level of GO $\mathrm { s l i m } ^ { 3 0 }$ . We consider a group of proteins in a single topological module as a functional module if and only if at least one function can cover all these proteins.

Disease Modules. Online Mendelian Inheritance in Man $( \mathrm { O M I M } ) ^ { 3 1 }$ is a comprehensive,authoritative compendium of human genes and genetic phenotypes. Genome-wide association studies $( \mathrm { G W A S } ^ { 3 2 } ,$ examine common genetic variants in populations to see if they are associated with a trait. GWASdb55 is a database that combines collections of GVs from GWAS together with their functional annotations and disease clasifications.MalaCards56 is an integrated searchable database of human maladies and their annotations,modeled on the architecture and richness of the popular GeneCards database of human genes. We detected disease modules based on known disease-gene associations extracted from OMIM and GWASdb and the disease classification of MalaCards online annotations.If more than two proteins have associations with the same disease type within certain topological module,then we take these proteins as a disease module. Here we classify diseases into 15 kind of phenotypes: Neurological, Ophthamological, Cardiovascular, Bone,Dermatological, Endocrine,Metabolic, Cancer, Immunological, Psychiatric, Hematological， Renal,Respiratory,_Ear, Nose,Throat and Gastrointestinal by integrate MalaCards database and Barabasi et al.method²7.

Pathway enrichment analysis.Information on the biological pathways that the module-related genes are involved in for each topological module was retrieved from DAVID36,37 online analytical tools.

We set a corrected P-value $< 0 . 0 5$ as the threshold used for enrichment analysis of pathways. The pathway databaseswe used are $\mathrm { K E G G ^ { 5 7 } }$ andREACTOME58,PANTHER5and BIOCARTA.

Systematic Analysis Method. Here we use a systematic analysis method to discover significant modules.The specific steps are shown in Fig.1.First we integrate 6 different human PPI databases; second,we integrate the HIPPIE database with IRefWeb database to obtain human protein“interactome” network; third,we divided the network into PPI sub-networks (topological modules) based on NeTA algorithm; fourth,construct corresponding function networks (based on detected functional modules) and disease networks (based on detected disease modules); lastly,to view the relationshipof diferent types of modules more clearly, we integrate topological modules,functional modules and disease modules together,to generate an integrative analysis of diffrent network levels,including protein,function and disease networks.We annotated the proteins within each module with the third level of GO slim.

# References

1.Pinkert,S.,chultz,J.&Reichardt,J.ProteinInteractionNetworks-MoreTanMereModules.PLoSComputiol.,000659 (2010). 2.Schaefer,M.H.etal.Adding Protein Contexttothe Human Protein-ProteinInteraction Network to Reveal Meaningful Interactions.PLoS Comput. Biol.9,el002860 (2013). 3.Juyong,Lee,Steven,PGross &Jooyoung,L Improved networkcommunitystructureimprovesfunctionprediction.SciRep. 3,srep02197 (2013). 4.Sharan,R.,UlitskyI.& Shamir,R.Network-based predictionof protein function.Mol.Syst.Biol.3,88,(2007). 5.Cho,Y-R.,Lei,Shi&Aidong,Zhang Functional moduledetectionbyfunctionalflowpaternmining inprotein interaction networks.BMC Bioinformatics 9,S10/O1 (2008). 6.Yook,S.H,Olv,Z.N.&arabsi,A.L.Functionalandtopologicalcaracterzationofproteninteractionetworks.oteomics 4,928-942 (2004). 7.Chen,J.&Yuan,BDetectingfunctialmodulesinheyastprote-proteininteractiontworkioformatics22,22-229 (2006). 8.Pu,S.Vasblom,J,Emili,,Grenblat,J.&Wodak,S.J.dentifyingfuncionalmodulesithepyicalinteractoeof Saccharomyces cerevisiae.Proteomics 7,944-960 (2007). 9.Anna,C.FLewisetal.Thefunctionofcommunities inproteininteraction networksatmultiple scales.BMCSyst.Biol.4,100 (2010). 10.Yunpeng,Zhanget al.Network Analysis RevealsFunctional Cros-linksbetwee Diseaseand Inflammation Genes.Sci.Rep.3, srep03426 (2013). 11.Bauer-Mehren,A.etal.Gene-Disease Network AnalysisRevealsFunctional Modules in Mendelian,ComplexandEnviromental Diseases.PLoS ONE6,e20284 (2011). 12.Gustafsson etal.Modules,networksand systems medicineforunderstandingdiseaseandaidingdiagnosis.Genome Med.6,82 (2014). 13.Marinka,Zitniketal.Discoveringdisease-diseaseassociationsbyfusingsystems-level moleculardata.Sci.Rep.3,srep03202 (2013). 14.Davis,D.A.& Chawla,N.V.Exploringand Exploiting Disease Interactions from Multi-Relational Geneand Phenotype Networks.PLoS ONE6,e22670 (2011). 15.Bauer-Mehren,A.etal.Gene-Disease Network AnalysisRevealsFunctional Modules in Mendelian,ComplexandEnviromental Diseases.PLoS ONE 6,e20284 (2011). 16.Reyes-Palomares,A.,Rodriguez-López,R.，Ranea,J.A.G.,Jimenez,FS.,Medina,M.A.Global Analysisof theHuman Pathophenotypic Similarity Gene Network Merges Disease Module Components.PLoS ONE 8,e56653 (2013). 17.Yang,P,Li,X,Wu,M,Kwoh,C-K.&Ng,S.-K.InferingGene-PhenoteAssociationsiaGlobalProteinComplextwok Propagation. PLoS ONE 6,e21502 (2011). 18.Marc,Vidalchael,E.usick&bert-szl,rabsiIteractoetworksandHumasease.Cell4,98-8). 19.Suthram,S.etal.Network-BasedElucidationof Human DiseaseSimilarities Reveals CommonFunctional Modules Enriched for Pluripotent Drug Targets.PLoS Comput.Biol.6,el000662 (2010). 20.Chan,S.Y.&Loscalzo,JTheemerging paradigmof networkmedicineinthestudyofhumandisease.Circ.Res.1,359-74 (2012). 21.Kwang-Il,Goh&In-Geol,ChoiExplorngthemadiseasoe: theumndseasenetwork.BriefunctGenomics,3-4 (2012). 22. Zhou,X. Z.et al. Human symptoms-disease network. Nat. Commun.5, ncomms5212 (2014). 23.Frank,E.-SShailesh,T,icardode,.S.Ahmed,FH.&Mathias,D.Thehumandiseasenetwork.ystBiomed.1,228 (2013). 24.Xiujuan,W,Natali,G.&Haiyuan,Y.Network-basedmethdsforumandiseasegenepredictionBriefFunct.Genomics10,280 -293 (2011). 25.Barabasi,A.L,Natali,G.&Joseph,L.Network medicine:anetwork-based approach tohuman disease.Nat.Rev.Genet.1, 56-68 (2011). 26.Laura,I.Furlong.Human diseases through the lens of network biology.Trends Genet.29,150-159 (2013). 27.Kwang-Il, Goh et al. Human disease network.Proc. Natl.Acad.Sci. USA104,8685-8690 (2007). 28.Wei,Liu,Matto,Pelegrini& Xiaofan,Wang.DetectingCommunities BasedonNetworkTopology.Sci.Rep.4,rep05739 (2014). 29.Schaefer,M.H.etal.HIPPIE:Integrating Protein InteractionNetworks withExperiment Based Quality Scores.PLoSONE7, e31826 (2012). 30.Ashburner,M.etal.Geneontology:tolfortheunificationofbiology.TheGeneOntologyConsortium.Nat.Genet.25,25-29 (2000). 31.Hamosh Ada,A.SAmegerJana,cni,arolVall,vid&Kusick,VictorOnlieendelianitacein Man (OMIM),a knowledgebase of human genes and genetic disorders.Nucl. Acids Res.30,52-55 (2002). 32.Beck,T,asgs,RKlapudi,F,R.C.&rokes,A.JGral:mreeiveresoueforeo and interrogation of genome-wide association studies.Eur.J. Hum. Genet.7,949-952 (2014). 33.Turinsky.Lick,SBoaldo,.&oda,.Jvigatigthegloblprote-proteteracioa using iRefWeb.Methods Mol. Biol.1091,315-331 (2014). 34.Newman,M.E.J,Girvan,M.Findingand evaluating community structure innetworks.Phys.Rev.E69,02613 (2004).

35.Maere,S.,Heymans,K.&Kuiper,M.BiNGO:aCytoscapepugin toasessoverepresentationofgeneontologycategories in biological networks.Bioinformatics 21,3448-3449 (2005).   
36.Huang,D.Weran,B.T&Lempicki,R.Astematicandintegrativeanalysisofarge genelistsuingVfatis Resources.Nature Protoc.4,44-57 (2009).   
37.Huang,D.W,Sherman,B.T.&Lempicki,R.A.Bioinformatics enrichmenttols:paths toward thecomprehensivefunctional analysis of large gene lists.Nucl.Acids Res.37,1-13 (2009).   
38.JeffreyS.Fier.Hormoneresistanceindiabetesandobesity:insulin,Leptin,andFGF21.YaleJBiol Med.85,45-414(012).   
39.Donath,M.Y.&Shoelson,S.E.Type2diabetes as an inflammatory disease.Nat.Rev.Immunol.11,98-107(2011).   
40.Porcher,Ctal.ecelleukeacoproteinCL/tal-issstialforevelopntofallematopeticlineags.ll 47-57 (1996).   
41.Akiyama,H.et al.Inflammation and Alzheimer's disease.Neurobiol.Aging.21,383-421 (2000).   
42.Enright,A.J,Vanongen,SOuzounis,C.A.AnecientalgoritforlargescaledetectionofproteinfamilesNucl.Acids Res.30,1575-1584 (2002).   
43.Kohler,S.,Bauer,.,Horn,D.&Robinson,PN.Walkingtheinteractomeforpriritizationofcandidatediseasegenes.Am.J. Hum. Genet.82,949-958 (2008).   
44.Ruepp,A.etal.CORUM:thecomprehensiveresourceofmammalianproteincomplexes—2009.Nucl.Acids Res.38,D497-D501 (2010).   
45.Barrett,T.etal.NCBIGEO:archiveforfunctioalgenomics datasets-update.NucleicAcidsRes.41,D991-D95 (2013).   
46.Safran,M.et al.GeneCards Version 3:the human gene integrator. Database 2010,baq020 (2010).   
47.Hruz,T.etalRefGenes:identificationofreliableandconditionspecificreference genesforRT-qPCRdata normalationBMC Genomics 12,156 (2011).   
48.Stark,C.et al.The BioGRID Interaction Database: 201l update.Nucl.Acids Res.39,D698-D704 (2011).   
49.Salwinski,L.et al.The Databaseof Interacting Proteins:2004 update.Nucl.Acids Res.32,D449-D451 (2004).   
50.Keshava,PrasadT.S.etal.HumanProtein Reference Database-209update.Nucl.AcidsRes.37,D767-D72(009).   
51.Aranda,B.et al.The IntAct molecular interaction database in 20l0.Nucl.Acids Res.38,D525-D531 (2010).   
52.Ceol,A.et al.MINT,the molecular interactiondatabase:2009 update.Nucl.Acids Res.38,D532-D539 (2010).   
53.Bader,G.D.,Betel,D.&Hogue,C.W.BIND:the BiomolecularInteractionNetwork Database.Nucl.AcidsRes.31,248-250 (2003).   
54.Shannon,P.etal.Cytoscape:asoftware environmentforintegratedmodelsofiomolecularinteractionnetworks.Genome Res. 3,2498-2504 (2003).   
55.Mulin,JunLietal.GWASdb:adatabaseforhumangeneticvariantsidentifedbygenome-wideasociationstudies.Nucl.Acids Res.40,D1047-D1054 (2012).   
56.Noa,Rappaportetal.MalaCards:anintegratedcompendiumfordiseasesandtheirannotation.Database(Oxford)ol,bat018 (2013).   
57.Kanehisa,M.& Goto,S.KEGG: kyoto encyclopedia of genes and genomes.Nucl.Acids Res.28,27-30 (2000).   
58.David,Croftetal.Reactome:adatabaseofreactionspathwasandbologicalproceses.Nucl.AcidsRes.39,91-D697(011).   
59.MiH.&Thomas,P.PANTHER pathway:anontology-based pathwaydatabasecoupled with dataanalysis tools.Methods Mol. Biol. 563,123-140 (2009).   
60.Darryl,N. BioCarta.Biotech Software & Internet Report.2,117-120 (2001).

# Acknowledgements

This work was supported by the National Natural Science Foundation of China under Grant (No. 61374176),and the Science Fund for Creative Research Groups of the National Natural Science Foundation of China (No.6l221003).

# Author Contributions

W.L.analyzed data,designed and performed research.M.P.,W.L.,A.P.W.and X.FW.discussed the results and wrote the manuscript.All authors reviewed the manuscript.

Additional Information Supplementary information accompanies this paper at http://www.nature.com/srep

Competing financial interests: The authors declare no competing financial interests.

How to cite this article: Liu,W.et al. Integrative analysis of human protein,functionand disease networks. Sci.Rep.5,14344; doi:10.1038/srep14344 (2015).

This work is licensed under a Creative Commons Atribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license,users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/