# A Covalent Approach for Site-Specific RNA Labeling in Mammalian Cells\*\*

Fahui Li, Jianshu Dong, Xiaosong Hu, Weimin Gong, Jiasong Li,Jing Shen, Huifang Tian, and Jiangyun Wang\*

Abstract:AdvancesinRNAresearchandRNAnanotechnolOgydepend on the abilitytomanipulate and probeRNAwith highprecisionthroughchemical approaches,bothin vitroand inmammaliancells.However,covalentRNAlabelingmethods with scope and versatility comparable to those of current protein labeling strategiesareunderdeveloped.A method is reported for the site- and sequence-specific covalent labeling of RNAs in mammalian cells by using $t R N A ^ { I l e 2 }$ -agmatidine synthetase(Tias)andclickchemistry.Thecrystal structureof Tias in complexwith an azide-bearing agmatine analogue was solved to unravel the structural basis for Tias/substrate recognition.TheuniqueRNAsequencespecificity and plastic Tias/substrate recognition enable the site-specific transfer of azide/alkyne groups toanRNA moleculeof interestinvitro and inmammaliancells.Subsequentclickchemistryreactions facilitate the versatile labeling,functionalization,and visualization of target RNA.

It has recently emerged that RNA has functions as diverse and important as those of proteins.[1] To unravel the dynamic trafficking,localization,and interactions of RNA in living cells,numerous methods have been developed for the noncovalentlabelingofRNAsinmammaliancells,anddramatic progress has been made.[2] Compared to the noncovalent approaches,[2] bioorthogonal chemistry[3] relies upon the specific and covalent attachment of probe molecules to the biomacromolecule of interest (BOI).Asa result,it offers several advantages:1) the affinity can be considered to be infinity, therefore stringent wash conditions can be applied to rigorously purify the BOI;2) the high affinity allows for the visualization of low-abundance BOIs,which is particularly important for non-coding RNAs;[1a]3) this methodis extremely versatile;probe molecules harboring fluorescence, NMR,IR,or EPR functional groups can be conveniently conjugated to the BOI.l3] Because of these unique features, bioorthogonal chemistry has been successfully employed in protein and glycan functional studies to visualize their expression,localization,and interaction partners in mammalian cells.[3]

While engineering of the translational4] or post-translational machinery[5]in combination with bioorthogonal chemistry has enabled protein labeling with unprecedented precision and versatility,6] comparable methods for the siteand sequence-specific tagging of RNAs in mammalian cells is currently underdeveloped.[7] To achieve this,we need to engineer unique components of the post-transcriptional machinery and employ bioorthogonal chemistry. First,an RNA-modifying enzyme must recognize a specific site in a unique RNA sequence in the transcriptome. Second, this enzyme must be able to transfer a small molecule bearing a unique chemical handle (such as an azide or alkyne group) to the specific site.Subsequent covalent labeling through bioorthogonal chemistry then leads to the site-specific conjugation of reporter groups,thus enabling fluorescence, NMR,EPR,or IR spectroscopy measurements (Figure 1).

To search for a post-transcriptional modification module that can be used for precise RNAlabeling in mammalian cells, we turned to the tRNA-modifying enzymes,which are known to catalyze around1OO kinds of tRNAmodification. Faithful translation of the genetic code relies on the chemical modification of tRNA,[9] especially at the wobble position 34 of the anticodon.[10] Many tRNA-modifying enzymes acting on this position are found only in specific domains,thus implying that they may have evolved independentlyl1] Notably, the C34 position of the AUA-decoding tRNAe² is modified with lysine by the enzyme lysidine synthetase (Tils) in bacteria[12] but with agmatine (AGM,1; Scheme 1) by the enzyme tRNAIle2-agmatidine synthetase (Tias） in archaea.13,14] In eukaryotes, tRNAle bearing pseudouridine $( \Psi )$ （204号 or inosine (I） at the wobble position decodes the AUA codon.9,15] As shown previously,Archaeoglobus fulgidus (Af) Tias recognition of tRNAIle requires seven nucleotides: G1, G2，C34，U36，A37,C71，and C72(Figure S13A in the

![](images/6162d9c3fdba24e953dd11a9c793b1840d1a08ab49bea2f6265604874a99cae3.jpg)  
Figure1.A) In the presence of ATP,Tias catalyzes the conjugation of agmatine to C34 of tRNAle2,thereby resulting in the formation of agmatidine.B) Labeling of an RNA of interest (ROl).Tias facilitates the site-specific transfer of an agmatine analogue bearingan azide group to the ROl-tRNAle2 fusion RNA. Subsequent strain-promoted azide-alkyne cycloaddition allows the conjuration of biophysical probes.

![](images/3129bd0a33f42582145c40c19a413d994462240c388e3ad7a1fb4f9a5097f257.jpg)  
Scheme1. Structure of agmatine (AGM) and agmatine analogues.

Supporting Information).13,14] Since neither Tias nor tRNA bearing this set of identity elements is present in mammalian cells (Figure S13),16] we envisioned that the introduction of Tias and archaeal tRNAIle2 into mammalian cels could facilitate the sequence- and site-specific labeling of RNA if Tias can transfer small molecule probes bearing azide or alkyne functional groups to an RNA of interest.

Asa first step to developing a covalent method for labelingRNAs,we synthesized three AGManalogues: N-(4 aminobutyl)-2-azidoacetamide (AGN,2),2-propynylamine (3)，and(but-3-yn-1-yl)(4-aminobutyl)carbamate(4; Scheme 1). Modification of Af $\mathrm { \ t R N A } ^ { \mathrm { I l e } 2 }$ was performed with Tias $( 8 ~ \mu \mathrm { M } )$ ，Af $\mathrm { \ t R N A } ^ { \mathrm { I l e } 2 }$ $( 1 ~ \mu \mathrm { M } )$ ,Tris-HCl ( $\mathrm { 1 0 0 ~ m M } ,$ ， $\mathrm { p H } 8 . 0$ KCl ( $\mathrm { \Delta } ^ { \mathrm { \prime 1 0 \ m M } } ,$ ， $\mathrm { { M g C l } } _ { 2 }$ 0 $\mathrm { \Delta } 5 \mathrm { m m }$ ，DTT $\left. { 5 \mathrm { m M } } \right.$ ,ATP(1 mM),and substrates1,2,3,or4( $\left( 1 \mathrm { m M } \right)$ .AGM(1)wasusedasapositive control.The modified and unmodified RNAs were then separated by $6 . 5 \%$ acid-urea polyacrylamide gel electrophoresis (PAGE),as previously described.[17] When tRNAIle2 was modified,the modified RNA exhibited lower mobility (Figure 2A).To our delight,we found thatin the presence of substrates 2, 3,or 4, tRNAIle2 was covalently modified, thus resulting in lower mobility in the acid-urea polyacrylamide gel.Perhaps owing to the lower molecular weight of compound 3, tRNAIe2 modified with compound 3 exhibited higher mobility than tRNA"e² modified with compound 2. In the presence of compound 4, two major bands were present, thus indicating that tRNAIe2 was only partially modified with compound 4. The much bulkier carbamate and alkyne functional groups may result in inefficient recognition of compound 4 by Tias.We therefore choose compounds 2 and 3 for further investigation.

![](images/7cde793c2c0f509a22c24469c544bae277ce1118abb083b02af180b069ab51cc.jpg)  
Figure 2.A) Acid-urea PAGE with $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ modified by using Tias and AGM analogues.The gel was stained by ethidium bromide (EB). B) While Tias catalyzes the efficient conjugation of AGM (l) and AGN (2) to $\mathsf { A f } \mathsf { t R N A } ^ { \mathsf { \tiny { I l e 2 } } }$ ,the reaction does not occur for the Af tRNAle2 C34U mutant, thus indicating that the small-molecule probes are transferred specifically to the C34 position.The reaction mixtures were analyzed by acid-urea PAGE.MALDl-TOF analyses are shown for unlabeled Af $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ (C),Af tRNAle² modified with 1(D),Af tRNAle² modified with 2 (E),and Af tRNAle2 modified with 3 (F).

We then investigated whether Tias catalyzes the specific modification of nucleotide C34 of Af tRNAIe2. While Tias catalyzed the efficient ligation of compounds1 and 2 to Af $\mathrm { \ t R N A } ^ { \mathrm { I l e 2 } }$ ，this activity was abolished for the C34U mutant (Figure 2B).These results indicate that Tias can target both the natural substrate AGM and the unnatural AGM analogues specifically to nucleotide C34 of Af tRNAle2.To further demonstrate that compounds 2 and 3 conjugate precisely with nucleotide C34 of Af tRNAle2，we digested Af tRNAIe² with RNase T1 (Figure S15)[18] and then analyzed the RNA fragment containing the anticodon loop by MALDI-TOF. The unmodified tRNAle2 fragment gave an observed average mass of $3 1 8 4 . 0 8 \mathrm { D a }$ ,whichclosely matches the calculated mass of 3184.4 Da (Figure 2).After AGM or 2- propynylamine modification, the tRNAIle2 fragment gave observed average masses of 3295.3 and $3 2 2 0 . 3 \mathrm { D a }$ ，respectively,which match the calculated masses of 3295.5 and $3 2 2 0 . 3 \mathrm { D a }$ ，After AGN modification, the tRNAIle2 fragment gave an observed average mass of $3 3 1 1 . 2 \mathrm { D a }$ ，while the calculated mass is 3337.5 Da.This can be explained by the loss of two nitrogen atoms and the addition of two hydrogen atoms during the laser irradiation used for MALDI-TOF,as previous observed.[19] Taken together, these results demonstrate that Tias can facilitate the conjugation of azide/alkynebearing AGM analogues 2 and 3 to nucleotide C34 in Af tRNAIle2.

To unravel the molecular basis for AGN recognition by Tias,we solved the structure of Tias with AGN bound to its active site.The main chain carbonyl group of Asp193 and the side chain of Asn194 form two hydrogen bonds with the azide group of AGN,while the guanidine group of Arg217 forms a hydrogen bond with the carbonyl group of AGN (Figure 3).

![](images/b5947878a83ab31bba87d003d17a170b018c601eecdf131271faa2a4922cede2.jpg)  
Figure 3.A) Overall structure of Tias in complex with AGN.The overall structure of Tias resembles a tennis racket.The AGN binding site is highlighted by a red box.B) Structure of the AGN binding site of Tias. The Fo-Fc electron density map of AGN was contoured at $2 . 7 ~ \sigma$ Nitrogen atoms are shown in blue,oxygen atoms in red,and carbon atoms in yellow.

Meanwhile,the sidechain ofVal2O3 stabilizesthe carbon chain of agmatine through hydrophobic interactions. Similar hydrophobic interactions may also be needed to facilitate Tias recognition of 2-propynylamine. Since Tias can also use compounds 3 and 4 as substrates,although these are either much smaller or much bulkier than compound 2,it appears that Tias has a somewhat plastic binding pocket for primary amine substrates.We are currently testing whether Tias can also use primary amines bearing cyclopropene groups as substrates,which may facilitate photoclick reactions or inverse electron demand Diels-Alder reactions.[20]

We then tested whether Tias can facilitate RNA fluorescence labeling through bioorthogonal chemistry. tRNAlle2 with or without AGN (2)modification was incubated with BCN-FITC and then analyzed by $6 . 5 \%$ acid-urea PAGE, followed by fluorescent imaging. tRNAle2 modified with AGN reacted with BCN-FITC selectively,whereas tRNAlle2 without AGN modification did not react (Figure 4A and 4B).

![](images/04a6f9282abc706c77dc8a3f6b6f1df8a13a35d76e81b7e23c9df3e93a04a703.jpg)  
Figure 4. A) Structure of BCN-FITC. B) AGN-modified $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ was visualized through strain-promoted click reaction with BCN-FITC.The bottom panel (EB staining) indicates that equivalent amounts of tRNAle2 with and without AGN modification were loaded.C) The structure of Sulfo-Cy5-azide. D) $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ modified with 2-propynylamine was visualized through ${ \mathsf { C u } } ^ { \iota }$ -catalyzed click reaction with Sulfo-Cy5- azide.The bottom panel (EB staining) shows that equivalent amounts of $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ with and without 2-propynylamine modifications were loaded.

These results indicate thatBCN-FITC reacts specificallywith the azide group present on tRNAIle2. Under similar conditions and in the presence of $\mathrm { C u } ^ { \mathrm { I } }$ catalyst $( 1 \mathrm { m M } )$ ,Sulfo-Cy5-azide reacts specifically with tRNAle2²modified with 2-ropynylamine (3), but not with $\mathrm { \ t R N A } ^ { \mathrm { I l e 2 } }$ without the modification (Figure 4C and 4D).

Once we had demonstratedthat Tias can facilitate the site-specific incorporation of azide and alkyne groups into $\mathrm { \ t R N A } ^ { \mathrm { I l e { 2 } } }$ ,we then asked whether this strategy could be used to label any RNA of interest (ROI).Through in vitro run-off transcription,we synthesized tRNAIle2 harboring $3 ^ { \prime }$ and $5 ^ { \prime }$ extensions (termed tRNAlle2-3-5,Figure 5A).We then incubated tRNA $\mathrm { I l e } 2 _ { - 3 - 5 }$ with Tias in the presence or absence of AGN. After removing excess AGN,BCN-FITC was added. Acid-urea PAGE was then used to analyze the reaction.Only inthe presence of AGN did BCN-FITC selectively label tRNA $\mathrm { I l e } 2 _ { - 3 - 5 }$ (Figure 5B)，thus indicating that Tias can catalyze the conjugation of AGN to $\mathrm { t R N A } ^ { \mathrm { I l e 2 } } – 3 – 5$ in the presence of $3 ^ { \prime }$ and $5 ^ { \prime }$ extensions. This result is consistent with previous reports that the agmatidine modification occurs early in the tRNA maturation process to ensure high-fidelity translation.[13-14]

We then asked whether our method can be used to label a specific RNA in the mammalian cell transcriptome.293T cells expressing Tias with or without tRNAIle²-5S fusion RNA expression were grown in medium containing AGN, followed bycell lysis. Subsequently, total cellularRNA was incubated with BCN-Cy5.In the absence of $\mathrm { t R N A } ^ { \mathrm { I l e 2 } } – 5 \mathrm { S }$ expression, BCN-Cy5 did not label any RNA in the 293T transcriptome (Figure 5C and 5D). When cells expressed tRNAIle2-5S, however,BCN-Cy5 specifically labeled tRNAIle2-5S.Reverse transcription polymerase chain reaction (RT PCR）experiments confirmed that tRNAlle2-5S existed as a full-length fusion RNA in the 293T cells (Figure S15).To further demonstrate the utility of our method for RNA imaging in mammalian cells,U2OS cells expressing Tias with or without tRNAIe2-5S fusion RNA expression were grown in the presence of 2-propynylamine and then fixed,permeabilized, and incubated with Sulfo-Cy5-azide and a ${ \mathrm { C u } } ^ { \mathrm { I } }$ catalyst.We found that only the cells expressing tRNA $\mathrm { I l e } 2 _ { - 5 \mathrm { S } }$ fusion RNA could be labeled and imaged with Sulfo-Cy5-azide (Figure 6). These results demonstrate that in mammalian cells,Tias can recognize the unique Af $\mathrm { \ t R N A } ^ { \mathrm { I l e { 2 } } }$ sequence in the transcriptome,thus enabling the sequence- and site-specific conjugation of an azide/alkyne group to an RNA of interest bearing the $\mathrm { \ t R N A } ^ { \mathrm { { I l e 2 } } }$ tag. Strain-promoted or $\mathrm { C u ^ { I } }$ -catalyzed click chemistry then enables the specific labeling and imaging oftheRNA of interest in the cells.

![](images/f7eeb603da9f7031d6f58175b8e82629c2dddb1b9e91c94e3ba644ddced863fc.jpg)  
Figure 5.A) Sequence and secondary structure of $\mathsf { t R N A } ^ { \mathsf { I I e 2 } }$ in the presence of $3 ^ { \prime }$ and $5 ^ { \prime }$ extensions (denoted tRNAle2-3-5).B) AGNmodified $\tan A ^ { 1 1 0 2 } - 3 . 5$ is visualized through strain-promoted click reaction with BCN-FITC.The bottom panel (EB staining) shows that equivalent amounts of $\tan A ^ { 1 1 e 2 } - 3 - 5$ with and without AGN modificatior were loaded.C) Structure of BCN-Cy5.D) Tias specifically modify $\mathrm { t R N A } ^ { 1 1 \mathrm { e } 2 } { \cdot } 5 5$ with AGN in the 293T cells transcriptome,which allows subsequent strain-promoted click reaction with BCN-Cy5.The bottom panel (EB staining) shows that equivalent amounts of total RNA with and without tRNAlle2-5S expression were loaded.

A great deal of modern RNAbiologyand RNAbiotechnology involves the site-specific incorporation of modified nucleotidesintoRNA moleculesthrough chemical approaches.[1] However, the length of synthetic RNA is typically less than 3O-5O nucleotides owing to limitations in coupling efficiency. Our new method relies on the RNA transcription machinery and the high sequence specificity of the RNA modifying enzyme Tias to enable the highly specific conjugation of biophysical probes to any RNA of interest in mammalian cells. This new technique should be useful for investigating RNA folding,RNA/protein interactions,RNA transportation,and RNA modification,as well as for constructing RNA-based nanomaterials,in vitro and in mammalian cells.[1]

![](images/5f046a351b477a5b2bd959508719f1d93619f37e9bef3904c5704e3978b093fa.jpg)  
Figure 6. Fluorescent imaging of Af tRNAle2-5S in U2OS cells. Upper panel: Cy5 fluorescence (left), DAPl fluorescence (nuclear staining; middle),and differential interference contrast (DIC) imaging (right) of U2OS cells harboring the pCMV-Myc-Tias plasmid.Lower panel: Cy5 fluorescence (left),DAPl fluorescence (nuclear staining; middle),and DIC imaging (right) of U2OS cells harboring the pCMV-Myc-5S-Tias plasmid. $\mathsf { D A P l } = 4 ^ { \prime } ,$ 6-diamidino-2-phenylindole.

In comparison to noncovalent RNA imaging methods such as use of the Spinach aptamer,[2] the major advantages of the Tias strategy are its specificity,highly affinity,and versatility. The Tias labeling strategy relies on the selective recognition of Af tRNAIle² by Tias (which are both absent in mammalian cells). Since a covalent approach is then employed after an azide or alkyne group is introduced, a covalent bond is created between the fluorophore and target RNA.This would potentially allow the imaging of lowabundance RNA.By contrast, the noncovalent interaction between the GFP chromophore analogue and the Spinach aptamer has afinity only in the micromolar range.[2] Finally, the Tias strategy enables versatile site-specific RNA labeling with fluorescent/IR/NMR/EPR agent through click chemistry, without changing the Tias protein or $\mathrm { \ t R N A } ^ { \mathrm { I l e } 2 }$ sequence. The challenge,however,is to develop fluorogenic cyclooctyne dyes that are membrane permeable,reasonablyhydrophobic, and suitable for live cell RNA imaging. We are currently working to achieve this goal.

# Experimental Section

In vitro modification of Af tRNAle2: Modification of Af tRNAe² was performed with Tias $( 8 ~ \mu \mathrm { M } )$ ,Af $\mathrm { \ t R N A } ^ { \mathrm { n e 2 } }$ $( 1 ~ \mu \mathrm { M } )$ ,Tris-HCl $\mathrm { \Omega } ^ { \mathrm { ( 1 0 0 ~ m M ) } }$ ， $\mathrm { p H } 8 . 0$ ，KCl $\mathrm { 1 0 ~ m M } ,$ 一， $\mathbf { M g C l } _ { 2 }$ 0 $\mathrm { \zeta }$ ，DTT $\left( 5 \mathrm { m M } \right)$ ，ATP( $\mathrm { \Delta } ^ { \mathrm { ~ \tiny ~ 1 ~ m M } }$ ,and substrates1,2,3,or4(1 mm).AGM(1) wasused as a positive control. Each reaction mixture was incubated at $3 7 ^ { \circ } \mathrm { C }$ for $^ { 2 \mathrm { h } }$ A ${ 5 \mu \mathrm { L } }$ aliquot from each reaction mixture was analyzed by $6 . 5 \ \%$ acid-urea PAGE (0.1M sodium acetate, $\mathrm { p H } 5 . 0$ ， $8 \mathbf { M }$ urea).In-gel fluorescence images were takenata KFBIO-28oO gel imager and the gel was then stained with ethidium bromide (EB).

Specific fluorescent labeling of Af tRNAle² in vitro: The AGN (2)-modified Af tRNA $^ { \mathrm { I l e 2 } } \left( 1 \mu \mathrm { M } \right)$ was mixed with BCN-FITC( ${ \bf \nabla } ^ { \mathrm { 1 0 0 } } \mu \bf { M } )$

in the buffer ( $2 0 \mathrm { m M }$ Tris, $\mathrm { p H } 7 . 4$ ,150 mM NaCl) and incubated for $3 0 \mathrm { m i n }$ at room temperature. The 2-propynylamine (3)-modified Af $\mathrm { t R N A } ^ { \mathrm { I l e 2 } } ( 1 \mu \mathrm { M } )$ was mixed with Sulfo-Cy5 azide (Lumiprobe,B3330) $( 1 0 0 ~ \mu \mathrm { M } )$ ，and $\mathrm { C u S O _ { 4 } }$ 0 $\mathrm { ~ \textmu ~ m m }$ ，ascorbicacid $( 2 \mathrm { m M } )$ and BTTPS $( 2 \mathrm { m M } )$ in the buffer( $2 0 \mathrm { m M }$ Tris, $\mathrm { p H } 7 . 4$ ，150 mM NaCl）and incubated for $3 0 \mathrm { m i n }$ at room temperature. The unmodified Af $\mathrm { \ t R N A } ^ { \mathrm { n e 2 } }$ was used as a negative control. The reaction mixtures were analyzed by $6 . 5 \%$ TAE-urea PAGE( $\mathrm { \Delta 4 0 ~ m M }$ Tris, $\mathrm { p H } 8 . 0$ ， $1 \mathrm { m m }$ EDTA, $8 \mathbf { M }$ urea),and in-gel fluorescence images were taken at a Typhoon 94oo variable mode imager (Amersham Biosciences). BCN-FITC:Excitation at $4 8 8 \mathrm { n m }$ ，emission at $5 2 0 \mathrm { n m }$ (520 BP 40 filter);Sulfo-Cy5-azide:Excitation at $6 3 3 \mathrm { n m }$ ,emission at 670 (670 BP 30 filter).

RNA imaging of tRNAIe2-5S in mammalian cell: The U2OS cells were allowed to grow to $5 0 – 7 0 \%$ confluence on a $2 0 \mathrm { m m }$ culture dish in DMEM supplemented with $10 \%$ FBS,and transfected with $2 . 2 ~ { \mu \mathrm { g } }$ of the plasmids pCMV-Myc-Tias or pCMV-Myc-5S-Tias, respectively，by using Lipofectamine 2Ooo Reagent (Invitrogen) according to the manufacturer's instructions. $3 6 \mathrm { h r s }$ after transfection, the cells were cultured in the presence of $1 \mathrm { m m } \ 3$ for $4 \mathrm { h r s } .$ Thecells were then washed twice with PBS and fixed in PIPES( $\mathrm { i } 2 5 \mathrm { m M }$ ）， $\mathrm { p H } 6 . 8$ EGTA $( 1 0 \mathrm { m M } )$ ), $\mathrm { { M g C l } } _ { 2 }$ (1 mM),Triton X-100 $( 0 . 2 \% )$ ，and formaldehyde $( 3 . 7 \% )$ for $3 0 \mathrm { m i n }$ at room temperature.The fixed cells were rinsed with TBS and stained for $3 0 \mathrm { m i n }$ at room temperature with TBSbuffer $\mathrm { ( p H } 7 . 5 )$ containing $\mathrm { C u S O _ { 4 } }$ (1 mM),ascorbic acid $( 1 0 0 \mathrm { m M } )$ ，Sulfo-Cy5-azide $( 2 0 ~ \mu \mathrm { M } )$ and BTTPS $( 1 \mathrm { m M } )$ [21]After staining,the cells were washed three times with TBS with $0 . 5 \%$ TritonX-1Oo.The cels were then imaged by fluorescence microscopy and differential interference contrast microscopy (DIC). ${ \bf D } { \bf T } { \bf T } = { \bf \Delta }$ dithiothreitol, $\mathbf { P B S } =$ phosphate-buffered saline,DMEM $\ c =$ Dulbecco's modified Eagle's medium, $\mathrm { F B S = }$ fetal bovine serum, ${ \mathrm { P I P E S } } =$ 1,4-piperazinediethanesulfonic acid,EGTA $\ c =$ ethylene glycol tetraaceticacid, $\mathrm { B T T P S } = 3$ -[4-({bis[(1-tert-butyl- $1 H$ -1,2,3-triazol-4-yl) methyl]amino}methyl)-1H-1,2,3-triazol-1-yllpropyl hydrogensulfate, $\mathrm { T B S = }$ Tris-buffered saline.

Keywords:bioorthogonal chemistry $\cdot \cdot$ biotechnology· RNA labeling $\mathbf { \nabla } \cdot \mathbf { \varepsilon }$ RNA modification $\cdot \cdot$ tRNA

Howto cite:Angew. Chem. Int. Ed. 2015,54,4597-4602 Angew. Chem. 2015,127,4680-4685

Natori,M. Sato,Y. Umezawa,Nat. Methods 2007,4,413-419; h) K. Y. Han,B.J. Leslie,J. Fei, J. Zhang, T. Ha,J. Am. Chem. Soc.2013,135,19033-19038；i) K.Sharma,J. J. Plant,A.E. Rangel, K.N. Khoe,A. J. Anamisis, J. Hollien, J. M. Heemstra, ACS Chem. Biol. 2014,9,1680-1684;j) E. V. Dolgosheina, S. C. Jeng, S. S.Panchapakesan, R. Cojocaru,P.S. Chen,P.D. Wilson, N. Hawkins,P.A. Wiggins,P. J. Unrau,ACS Chem. Biol. 2014,9, 2412-2420；k)A.Samanta，A.Krause，A. Jaschke,Chem. Commun. 2014,50,1313-1316.   
[3] a) R.K.Lim,Q. Lin,Acc. Chem. Res.2011,44,828-839; b)E. M. Sletten, C.R.Bertozzi,Angew. Chem. Int. Ed. 2009,48, 6974-6998; Angew.Chem.209,121,7108-7133;c)J.C.Jewett, C.R.Bertozzi,Chem. Soc. Rev. 2010,39,1272-1279.   
[4] a)L. Wang, J. Xie,P.G. Schultz, Annu.Rev. Biophys. Biomol. Struct. 2006,35,225-249; b) C.C.Liu,P.G. Schultz,Annu. Rev. Biochem. 2010,79,413-44;c) K.Lang,J. W.Chin,Chem. Rev. 2014,114, 4764-4806.   
[5] a) D.A. Zacharias, J. D. Violin, A. C. Newton, R.Y. Tsien, Science 2002,296, 913-916; b) C. Uttamapinant, A. Tangpeerachaikul, S. Grecian, S. Clarke,U. Singh,P. Slade,K.R. Gee, A.Y. Ting, Angew. Chem. Int. Ed. 2012,51,5852-5856; Angew. Chem. 2012,124,5954-5958;c) Y. Zou,J.Yin,J. Am. Chem. Soc.2009,131,7548-7549;d) C.Jing,V.W. Cornish,Acc. Chem. Res.2011,44,784-792;e) K.Johnsson,Nat. Chem. Biol.2009,5, 63-65; f) X. Shi, Y. Jung,L.-J. Lin, C. Liu, C. Wu,I. K. O. Cann, T.Ha,Nat. Methods 2012,9,499-U128.   
[6] C.Hu,S.I. Chan,E.B. Sawyer,Y.Yu,J. Wang, Chem. Soc. Rev. 2014,43, 6498-6510.   
[7] a) D. Schulz, J. M. Holstein,A. Rentmeister, Angew. Chem. Int. Ed.2013,52,7874-7878; Angew. Chem. 2013,125,8028-8032; b) D. Schulz,A.Rentmeister,ChemBioChem 2014,15,2342- 2347.   
[8] a) M.A. Machnicka,K. Milanowska, O.O. Oglou,E. Purta,M. Kurkowska,A.Olchowik,W. Januszewski, S.Kalinowski,S. Dunin-Horkawicz,K.M. Rother,M. Helm,J.M. Bujnicki, H. Grosjean, Nucleic Acids Res. 2013,41,D262-D267.   
[9] a)A. Ambrogelly, S. Palioura,D. Soll, Nat. Chem. Biol. 2007,3, 29-35;b) B. El Yacoubi, M. Bailly, V. de Crecy-Lagard, Annu. Rev.Genet.2012,46,69-95；c)K. Selvadurai,P.Wang,J. Seimetz,R.H.Huang,Nat.Chem. Biol.2014,10,810-812;d) L. Randau, B.J. Stanley,A.Kohlway,S. Mechta,Y. Xiong,D.Sol, Science 2009,324,657-659.   
10] P.F.Agris,F.A. Vendeix,W.D.Graham,J. Mol.Biol. 2007,366, 1-13.   
11] N. Terasaka, S.Kimura,T. Osawa, T. Numata,T. Suzuki, Nat. Struct. Mol. Biol.2011,18,1268-1274.   
12] a)K.Nakanishi,L.Bonnefond, S.Kimura,T.Suzuki,R.Ishitani, O.Nureki,Nature 2009,461,1144-1148.   
13] a) D.Mandal, C.Kohrer,D. Su,S.P.Russell,K.Krivos,C.M. Castleberry,P.Blum,P.A. Limbach, D.Soll, U.L.RajBhandary, Proc.Natl.Acad.Sci. USA 2010,107,2872-2877;b) Y.Ikeuchi, S. Kimura,T.Numata,D.Nakamura,T. Yokogawa,T. Ogata,T. Wada,T. Suzuki, Nat. Chem.Biol. 2010,6,277-282.   
14] T.Osawa, S.Kimura,N. Terasaka,H. Inanaga,T. Suzuki,T. Numata,Nat. Struct. Mol.Biol. 2011,18,1275-1280.   
15] B. Senger， S.Auxilien,U. Englisch,F. Cramer,F.Fasiolo, Biochemistry 1997,36,8269-8275.   
16] F. Juhling,M.Morl,R.K.Hartmann,M. Sprinzl,P.F Stadler,J. Putz,Nucleic Acids Res. 2009,37,D159-162.   
17] C.Kohrer, U.L. Rajbhandary, Methods 2008,44,129-138.   
18] C.N. Pace, U. Heinemann, U. Hahn,W. Saenger,Angew. Chem. Int.Ed.Engl.1991,30,343-360; Angew. Chem.1991,103,351- 369.   
19] a)X. Chen,W.F. Siems, G.R.Asbury, R.G. Yount,J. Am. Soc. Mass Spectrom.1999,10,1337-1340；b)M.A.Nessen，G. Kramer, J. Back,J. M. Baskin,L.E. Smeenk, L.J. de Koning, J.H.van Maarseveen,L.de Jong,C.R.Bertozzi,H.Hiemstra, C.G.de Koster,J. Proteome Res. 2009,8,3702-3711.   
[20] Z. Yu, Q.Lin,J. Am. Chem. Soc.2014,136,4153-4156.   
[21]C.Besanceney-Webler，H.Jiang，T. Zheng，L.Feng，D.S. del Amo,W.Wang,L.M.Klivansky,F.L.Marlow,Y.Liu,P.Wu, Angew.Chem.Int.Ed. 2011,50,8051-8056；Angew. Chem.   
2011,123,8201-8206.

Received:October 24,2014   
Revised:December8,2014   
Published online:February 18,2015