# SARS-coronavirus-2 nspl3 possesses NTPase and RNA helicase activities

Ting Shu1,2, Muhan Huang², Di $\mathrm { W u } ^ { 2 }$ , Yujie Ren2,3, Xueyi Zhang², Yang Han1,2, Jingfang $\mathrm { M u } ^ { 2 }$ , Ruibing Wang5, Yang Qiu2.6, Ding-Yu Zhang1,\*, Xi Zhou1,2.5,\* 1 Center for Translational Medicine, Wuhan Jinyintan Hospital, Wuhan, Hubei, 430023 China 2 State Key Laboratory of Virology, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences,Wuhan, Hubei, 430O71 China 3 Center for Precision Translational Medicine of Wuhan Institute of Virology and Guangzhou Women and Children's Medical Center， Guangzhou Women and Children's Medical Center, Guangzhou, Guangdong, 510120, China 4 State Key Laboratory of Quality Research in Chinese Medicine, Institute of Chinese Medical Sciences, University of Macau, Macau SAR 999078, China 5 University of Chinese Academy of Sciences, Beijing,100049 China \*Correspondence: zhouxi@wh.iov.cn (X. Zhou), zhangdy63@hotmail.com (D. Zhang) Ting Shu and Muhan Huang have contributed equally to this work.

# Abstract

The ongoing outbreak of Coronavirus Disease 2O19 (Covid-19） emerged from Wuhan,Hubei, since December 2O19 has become a global public health emergency. SARS-coronavirus-2 (SARS-CoV-2)， the causative pathogen of Covid-19,is a positive-sense single-stranded RNA virus belonging to the family Coronaviridae. For RNA viruses, virus-encoded RNA helicases have long been recognized to play pivotal roles during viral life cycles by facilitating the correct folding and replication of viral RNAs. Here, our studies show that SARS-CoV-2-encoded nonstructural protein 13 (nspl3) possesses the nucleoside triphosphate hydrolase (NTPase) and RNA helicase activities that can hydrolyze all types of NTPs and unwind RNA helices dependently of the presence of NTP,and further characterize the biochemical characteristics of these two enzymatic activities associated with SARS-CoV-2 nsp13.Moreover, we found that some bismuth salts could effectively inhibit both the NTPase and RNA helicase activities of SARS-CoV-2 nspl3 in a dose-dependent manner. Thus,our findings demonstrate the NTPase and helicase activities of SARS-CoV-2 nspl3, which may play an important role in SARS-CoV-2 replication and serve as a target for antivirals.

# Keywords

SARS-CoV-2 nspl3,NTPase, helicase, antiviral target

# Introduction

Coronaviruses (CoVs) are a large family of single-stranded, positive-sense RNA viruses that belong to the family Coronaviridae in the order Nidovirales (Cui et al., 2019; Chen et al.,2O2Ob). Coronavirus has a large RNA genome of 26-31 kb in length, and its replicase gene consists of two open reading frames (ORFs） 1a and 1b. These two ORFs yield two polyproteins, ppla and pplab, which are proteolytically cleaved into 16 nonstructural proteins (nspl-16) that play pivotal roles in the life cycle of CoVs (Chen et al., 2020b).

Coronaviruses can infect a variety of animals, including humans, bat, mice and birds (Cui et al., 2O19).Previously, six different coronaviruses had been found to infect humans, including CoV-229E, CoV-OC43, CoV-NL63, CoV-HKU1, SARS-CoV, and middle east respiratory syndrome coronavirus (MERS-CoV) (Cui et al., 2019). Among them, CoV-229E, CoV-OC43, CoV-NL63, and CoV-HKU1 primarily cause mild and self-limiting disease such as common cold, while SARS-CoV and MERS-CoV have caused severe respiratory illness.

Since late December 2O19,an outbreak of mysterious viral pneumonia emerged from Wuhan, the metropolitan city in central China,and rapidly spreads to many other regions in China and the world (Zhou et al.; Zhu et al.; Chan et al.,2O2O; Lu et al., 2020; Rothe et al., 2O2O), which has been declared as a global public health emergency by the World Health Organization(WHO) (WHO,2O2O). The causative pathogen was soon identified as a novel coronavirus by several institutions at Wuhan and Beijing,and was initially named as 2019-novel CoV (2019-nCoV) and later as SARS-CoV-2, while the disease caused by this virus is named as Covid-19 by WHO. SARS-CoV-2 is thought to have a zoonotic origin, and highly contagious through human-to-human transmission (Zhou et al.). Human infections by SARS-CoV-2 cause a wide spectrum of diseases, ranging from mild to severe symptoms such as pneumonia, acute respiratory distress syndrome,and even death (Chen et al.,2O2Oa; Guan et al.,2O2O; Huang et al., 2020; Wang et al.， 202O). The ongoing outbreak of SARS-CoV-2 has caused tremendous losses in human lives and economy in Wuhan, Hubei,and across the globe, while there is no approved drug or vaccine available for this virus.

For most RNA viruses including coronaviruses, viral RNAs (vRNAs) usually form a variety of cis-acting elements within the un-translational regions (UTRs) or ORF regions. These vRNA elements play indispensable roles in replication, translation and encapsidation,and need to be folded into correct secondary or tertiary structures to functional. In addition， viral double-stranded RNAs (dsRNAs） produced during replication must be unwound to allow next round of vRNA replication. However, the proper folding of RNA elements or unwinding of dsRNAs is usually challenging, which requires the aid of RNA remodelling proteins including RNA helicase and RNA chaperones (Lorsch,2002; Bleichert and Baserga, 2O07; Musier-Forsyth,201O). Thus far, numerous RNA viruses have been found to encode their own RNA helicases, which are usually indispensable components of vRNA replication complexes (Yang et al., 2014; Xia et al., 2015; Jain et al., 2016; Li et al., 2018; Shu et al.,2019) and recognized as ideal targets for developing antivirals (Pfister and Wimmer, 1999; Li et al., 2018; Shu et al., 2019).

Previous studies have shown that SARS-CoV nsp13 has an NTPase activity as well as an RNA helicase activity belonging to helicase superfamily-1 (Tanner et al., 2003; Ivanov et al., 2OO4). Considering the high homology of the amino acid sequences among coronaviral nspl3 proteins, it is intriguing to examine whether SARS-CoV-2 nsp13 also possesses the NTPase and RNA helicase activities. More importantly, we can re-examine the drugs that were found to target other coronavirus nspl3 for their effects on SARS-CoV-2 nspl3,which may accelerate the development of antivirals against Covid-19 in a timely manner. In this study，we expressed and purified recombinant nsp13 of SARS-CoV-2,and examined its biochemical activities. Our data show that SARS-CoV-2 nspl3 does possess the NTPase and RNA helicase activities that can hydrolyze all types of NTPs and unwind RNA helices in an NTP-dependent manner,and some bismuth salts can effectively inhibit both the NTPase and RNA helicase activities of SARS-CoV-2 nsp13.

# Results

# SARS-CoV-2 nsp13 contains NTPase activity

Previous studies have reported SARS-CoV nspl3 contains NTPase and helicase activities (Tanner et al., 2OO3; Ivanov et al., 2OO4). To characterize the biochemical activities of SARS-CoV-2 nspl3,we expressed and purified it as a maltose-binding protein (MBP-nspl3) from E. coli prokaryotic expression system (Fig. S1).

The RNA-helix unwinding of helicases usually require NTP binding and hydrolysis to provide energy. Therefore,we first sought to examine whether SARSCoV-2 nsp13 has the NTPase activity to hydrolyze four kinds of NTP by measuring the released inorganic phosphate via a sensitive colorimetric assay. We found that the recombinant SARS-CoV-2 nspl3 could hydrolyze all four types of NTPs,with a preference for ATP and GTP (Fig.1A). We used ATP in the subsequent assays,as it is the major energy source in cells. Further investigation showed that the amount of hydrolysed ATP by nsp13 was increased with the increasing concentrations of nsp13 used in the reaction mix (Fig. 1B). Moreover,we found that the NTPase activity of SARS-CoV-2 nspl3 requires the presence of divalent metallic ions,as our data showed that $2 \mathrm { m M } \mathrm { M g } ^ { 2 + }$ $\mathrm { M n } ^ { 2 + }$ $\mathrm { C a } ^ { 2 + }$ or $Z \mathrm { n } ^ { 2 + }$ could support the ATPase activity of nspl3, and their efficiencies were as follow: $\mathrm { M g ^ { 2 + } { > } M n ^ { 2 + } { > } Z n ^ { 2 + } { > } C a ^ { 2 + } }$ (Fig. 1C). Besides, SARSCoV-2 nsp13 displays its optimal ATPase activity in the presence of $2 \mathrm { m M } \mathrm { M g } ^ { 2 + }$ , while higher concentrations of $\mathrm { M g } ^ { 2 + }$ showed certain inhibitory effect on the ATPase activity (Fig.1D). Together, our data show that SARS-CoV-2 nspl3 possesses an NTPase activity, which is dependent on the presence of certain divalent metallic ions.

# SARS-CoV-2 nsp13 has the RNA helix unwinding activity

After establishing that SARS-CoV-2 nspl3 possesses the activity to hydrolyze NTP, we sought to examine whether it has the RNA helix unwinding activity. To this end,we constructed an RNA helix substrate with both $5 ^ { \prime } .$ and $3 ^ { \prime } { \mathrm { . } }$ -single-stranded protrusions by annealing a 24-nt non-labeled RNA with a 42-nt Hexachloro-Fluorescein hosphoramidite (HEX)-labeled RNA (as illustrated in Fig. 2A). This RNA helix substrate was commonly used to characterize the helix unwinding activity of RNA helicases (Li et al., 2018; Shu et al.,2O19). The helix unwinding assay was performed by incubating the RNA helix substrate with MBP-nspl3 in a standard unwinding reaction mix containing ATP and $\mathrm { M g C l } _ { 2 }$ ，followed by the separation of the RNA substrate strands via electrophoresis. As shown in Fig. 2B, HEX-labeled RNA strand was efficiently released from the RNA helix substrate in the presence of MBP-nsp13 (lane 4), whereas the same substrate was stable when MBP alone was supplemented in the reaction as negative control (lane 3). Of note, the boiled helix substrates were used as the positive control (lane 2). Moreover, when increasing concentrations of MBPnsp13 were incubated with the RNA helix substrate, MBP-nsp13 can efficiently unwind RNA helix in a dose-dependent manner (Fig. 2C). Besides, our data showed that the released HEX-labeled RNAs were gradually increased along with the increasing reaction time (Fig. 2D).

We further characterized the helix unwinding activity by MBP-nspl3 via incubating it with the RNA helix substrate (Fig. 3A) under different conditions. We found that MBP-nsp13 could only efficiently unwind the the RNA helix substrate in the presence of ATP and GTP, whereas the efficiency of RNA helix-unwinding by nsp13 was limited in the presence of CTP and UTP (Fig. 3B),consistent with the NTP preference of the NTPase activity of nspl3. Besides, our data showed that the RNA helix-unwinding by MBP-nsp13 required the presence of divalent metallic ions. This protein showed optimal helix unwinding activity in the presence of $\mathbf { M } \mathbf { g } ^ { 2 + }$ (Fig. 3C, lane 4), while the presence of $ { \mathbf { M } }  { \mathrm { n } } ^ { 2 + }$ also supported nspl3 to unwind RNA duplex in a lesser extent than that of $\mathbf { M } \mathbf { g } ^ { 2 + }$ (Fig. 3C, lane 5). On the contrast, the presence of $\mathrm { C a } ^ { 2 + }$ or $Z \mathrm { n } ^ { 2 + }$ failed to support the helix unwinding activity of SARS-CoV-2 nspl3 (Fig. 3C, lanes 6 and 7). And this result is consistent with the observation that the nspl3 NTPase activity is only minimal in the presence of $\mathrm { C a } ^ { 2 + }$ or $Z \mathrm { n } ^ { 2 + }$ (Fig. 1C). Moreover, SARS-CoV-2 nsp13 displayed the optimal helicase activity in the presence of $2 { \mathrm { ~ m M ~ M g } } ^ { 2 + }$ ，while higher concentrations of $\mathrm { M g } ^ { 2 + }$ even showed inhibitory effect (Fig. 3D), consistent with that ofnsp13 NTPase activity (Fig. 1D). Interestingly, high concentration of multivalent metallic cations, such as $\mathrm { M g } ^ { 2 + }$ or $Z \mathrm { n } ^ { 2 + }$ , exhibited inhibitory effects on multiple viral RNA helicases, which is probably due to stabilizing RNA conformation or competing with RNA helicases for nonspecific RNA binding.

Taken together, our findings show that SARS-CoV-2 nspl3 has the RNA helicase activity in the presence of ATP and $\mathrm { M g } ^ { 2 + }$

# SARS-CoV-2 nsp13 unwinds RNA helix in the $\bar { \mathbf { \omega } } _ { 5 } \mathrm { ^ , }$ to ${ \mathbf { } } ^ { 3 ^ { \circ } }$ directionality

The directionality of helix-unwinding is one of the fundamental characteristics of RNA helicases (Musier-Forsyth, 2010). After establishing that SARS-CoV-2 nsp13 displays the RNA helix-unwinding activity, we sought to characterize its helix unwinding directionality. To this end, we constructed three different RNA helix substrates with $5 '$ -protruded, $3 ^ { \circ }$ -protruded and blunted ends,respectively (Fig. 4A-C). Then,we reacted these helix substrates with the purified MBP-nspl3 in the standard helix-unwinding assay. Our data showed that SARS-CoV-2 nsp13 efficiently unwound the RNA helix substrate with $5 ^ { \circ }$ -protrusion (Fig. 4D, lane 3), but not the substrate with $3 ^ { \circ }$ -protrusion (Fig. 4D,lane 6). Moreover, the blunt ended helix substrate could not be unwounded by MBP-nspl3 (Fig. 4E). Together, our findings indicated that SARSCoV-2 nspl3 is able to unwind RNA helix in the directionality of $5 ^ { \circ }$ to $3 ^ { \circ }$ ，which is consistent with other coronaviral RNA helicases.

# Bismuth salts inhibits the NTPase and RNA helix-unwinding activities of SARSCoV-2 nsp13

As aforementioned, we have observed that certain divalent cations, like $\mathrm { C a } ^ { 2 + }$ and $Z \mathrm { n } ^ { 2 + }$ ， did not support RNA helix unwinding by SARS-CoV-2 nsp13,while high concentrations of $\scriptstyle \mathrm { M g 2 + }$ showed inhibitory effect (Fig. 3C-D). Moreover, previous studies reported that some bismuth salts could inhibit the NTPase and RNA helicase activities of SARS-CoV nspl3 as wellas the replication of SARS-CoV in cells (Yang et al.,2007a; Yang et al., 2O07b). Therefore,we sought to examine whether NTPase and RNA helix-unwinding activities of SARS-CoV-2 nsp13 can also be inhibited by the bismuth salts.Here,we used three different bismuth salts,including bismuth potassium citrate (BPC),ranitidine bismuth citrate (RBC),and bismuth citrate (BC). BPC and RBC are originally used to treat the gastrointestinal diseases,and BC is the intermediate of BPC. Our data showed that all these bismuth salts inhibited the ATPase activity of SARS-CoV-2 nsp13 at $1 0 \mu \mathrm { M }$ , as BPC and RBC showed stronger inhibitory effects than BC (Fig. 5A). Moreover,we found that $1 0 ~ \mu \mathrm { M }$ BPC or RBC almost abolished the RNA helix unwinding activity of nspl3 (Fig. 5B,lanes 4 and 5); on the other hand, although BC treatment also effectively inhibited the RNA helicase activity of nsp13,its inhibitory efficiency was in much less than BPC or RBC (Fig. 5B,lane 6). To further characterize the inhibitory effect of BPC or RBC on the NTPase and RNA helicase activities of SARS-CoV-2, we treated recombinant nspl3 with increasing concentrations of either bismuth salts. Our data show that either BPC or RBC could inhibit the ATPase and helicase activities of MBP-nspl3 in a dose-dependent manner (Fig. 6).

In conclusion, our data show that the bismuth salt, BPC or RBC,can effectively inhibit both the NTPase and RNA helicase activities of SARS-CoV-2 nsp13 dosedependently.

# Discussion

The ongoing Covid-19 outbreak in China and across the globe has become a significant threat to the health and lives of humans. Given the emergency and threat caused by the SARS-CoV-2 epidemic, better understanding all aspects of SARS-CoV2 virology and uncovering potential antiviral drug targets are urgently needed. Virusencoded RNA helicases are a class of viral enzymes that are parts of viral RNA replication complex and play indispensable roles in viral propagation and pathogenesis. Therefore，virus-encoded RNA helicases are also recognized as ideal targets for antiviral drugs. In this study, we found that SARS-CoV-2 nspl3 possesses the NTPase and RNA helicase activities that can hydrolyze all types of NTPs and unwind RNA helix in an ATP-dependent manner. Moreover, our data show that two bismuth salts, BPC and RBC,which are used in clinical treatment of gastrointestinal diseases, can effectively inhibit the NTPase and helicase activities of SARS-CoV-2 nsp13.From this perspective,our findings provide insights into the key viral replicative enzyme that probably represents an attractive target for developing antivirals.

The finding that SARS-CoV-2 nsp13 is able to hydrolyze NTPs and to destabilize RNA helix consistent with the previous observation that other coronaviral nsp13 proteins also possess the NTPase and RNA helicase activities (Tanner et al.,2003; Ivanov et al., 2OO4), implying the conserved and indispensable role of nspl3 in the life cycles of coronaviruses. Indeed, coronavirus nspl3 is a multi-functional protein with a Zinc-binding domain (ZBD） in N-terminus and a helicase domain with typical conserved motifs of superfamily-1 (SF1) helicases is present in the C-terminal half (Subissi et al., 2O14). The N-terminus of nsp13 are highly conserved and predicted to form a $Z \mathrm { n } ^ { 2 + }$ -binding cluster,which conserved in all coronaviruses and nidoviruses (Seybert et al., 2005).

RNA helicases contain NTPase activity and utilize the energy of hydrolyzing ATP to melt base pairings,and are generally believed to play important roles in all processes involving RNAs during viral life cycles, including replication, transcription, translation as well as encapsidation (Bleichert and Baserga, 2OO7; Musier-Forsyth, 2010). For example, during the process of vRNA replication of RNA viruses, viral dsRNA replicative intermediates must be efficiently unwound to release nascently synthesized progeny vRNAs from template vRNAs (Yang et al., 2014; Xia et al., 2015). Therefore,virus-encoded RNA helicases have long been considered as the potential targets for antiviral therapy development. A serial of potential drugs, including flavonoids，myricetin， scultellarein， dihydroxychromones，aryl diketoacids,3-[(2- nitrophenyl)sulphanylmethyl]-4prop-2-enyl-1H-1,2,4-triazole-5-thione. and bismuth salts had been found by different groups to target the ATPase/helicase activities of SARS-CoV nsp13 (Seybert et al., 2005; Yang et al., 2007a; Yang et al., 2007b; Lee et al., 2009; Adedeji et al., 2012; Yu et al., 2012). Similarly,we also found that bismuth salts BPC and RBC could effectively inhibit with the ATPase and helicase activities of SARS-CoV-2 nspl3 in a dose-dependent manner. Because bismuth cation $( { \mathrm { B i } } ^ { 3 + } )$ can bind strongly to metallothionein through the formation of Bi-S bonds (Sun et al.,1999), it is possible that BPC or RBC inhibits the enzymatic activities of SARS-CoV-2 nsp13 via binding to the N-terminal ZBD of nsp13.

One of the symptoms of Covid-19 is diarrhoea and gastrointestinal disorder (Chen et al., 2020a; Guan et al., 202O; Huang et al.,2020; Wang et al., 2020). Besides, SARSCoV-2 could be found in the rectal swabs and feces from many Covid-19 patients. More importantly, the rectal swabs can detect SARS-CoV-2 in patients even when the testes for throat swabs turned negative, implying that SARS-CoV-2 is more persistent in gastrointestinal tract compared to that in respiratory tract (Guan et al., 2O2O; Zhang et al., 202O). These findings imply that the gastrointestinal tract is also the target of SARSCoV-2 and the risk of further dissemination of this virus to healthy population through fecal-oral route cannot be omitted. Since BPC and RBC are already used to treat disorders in gastrointestinal tract, they may have the potential to be further developed for the prevention and treatment of Covid-19. Future study should evaluate the inhibitory effect of BPC or RBC on SARS-CoV-2 replication in cells and in vivo.

In summary， SARS-CoV-2 nspl3 possesses the NTPase and RNA helicase activities that can hydrolyze all types of NTPs and unwind RNA helices dependently of the presence of NTP. Moreover, bismuth salts, such as BPC and RBC, have been found to inhibit both the NTPase and RNA helicase activities of SARS-CoV-2 nsp13 in a dose-dependent manner. In a timely manner, our study extends our knowledge about one of the key replicative enzymes of SARS-CoV-2 and suggests that SARSCoV-2 nspl3 can be a valuable target for antivirals, which may be helpful in the efforts to control this life-threatening virus.

# Materials and Methods

# Plasmid construction

MBP fusion expression vector pMAL-SARS-CoV-2 nsp13 was generated using a standard cloning protocol. The cDNA fragment of SARS-CoV-2 nsp13 was generated by reverse transcription from the RNA template of SARS-CoV-2-infected cells. The SARS-CoV-2 nspl3 genome fragment was inserted into the pMAL-c2X vector at BamH I /Sal I restriction sites. The primers used in this study are shown in Table S1.

# Expression and purification of recombinant protein

E.coli (BL21-DE3) were transformed by introduction of pMAL-SARS-COV-2 nsp13. In briefly, $E$ .coli were grown at $3 7 ^ { \circ } \mathrm { C }$ and induced with $3 0 0 ~ \mu \mathrm { M }$ IPTG when the OD value reached ${ \sim } 0 . 6$ .Thereafter, the induced $E$ .coli were transferred to $2 2 ^ { \circ } \mathrm { C }$ to grow for 12 hrs. Cells were harvested and lysed by ultra-sonication. After centrifugation at 12 000 g, the proteins in the supernatant was purified using amylase afinity chromatography(New England BioLabs， Ipswich， MA） according tothe manufacturer's protocol. All the purified proteins were concentrated using Amicon Ultra-30 filters (Millipore, Schwalbach, Germany). After that, the store buffer was exchanged to 50 mM 2-[4-(2-hydroxyethyl)-1-piperazinyl] ethane sulfonic acid (HEPES)-KOH (pH8.O). All proteins were quantified by the Bradford method and stored at $\mathbf { \nabla } _ { - 8 0 ^ { \circ } C }$ in aliquots. Proteins were separated on $10 \%$ SDS-PAGE and visualized by Coomassie blue.

# NTPase assay

NTPase activities were performed by measuring the released inorganic phosphate during NTP hydrolysis via a direct colorimetric assay as previously described (Yang et al., 2017). The concentrations of inorganic phosphate were determined by matching the A620 in a known standard inorganic phosphate curve. All of the data given by this quantitative assay were averages of three independently repeated experiments.

# Preparation of RNA helix substrate

The RNA helix substrates consist of two complementary nucleic acid strands, one of which was labeled at $5 ^ { \prime } { \cdot }$ -end with HEX and the other was unlabeled.The labeled strands were purchased from TaKaRa (Dalian, China). All the unlabeled RNA strands were transcribed in vitro using T7 RNA polymerase (Promega, Madison, WI). The postranscriptional RNAs were purified by using Poly-Gel RNA extraction kit (Omega Bio-Tek, Guangzhou, China） according to the manufacturer's instructions. The oligonucleotide helices were generated by annealing the labeled strand and unlabeled strand, at which mixed a 1:1 ratio in a $1 0 \mathrm { - } \mu \mathrm { l }$ reaction mixture containing $2 5 ~ \mathrm { m M }$ HEPES-KOH $\mathrm { ( p H ~ 8 . 0 ) }$ and 25 mM NaCl. The mixture was heated to $9 5 ^ { \circ } \mathrm { C }$ for 5 min and was then cooled gradually to $2 5 ^ { \circ } \mathrm { C }$ to produce helical duplexes. Two RNA helix substrate with both $5 ^ { \prime } .$ and $3 ^ { \prime } { \mathrm { . } }$ -protrusions was annealed with RNA1 and RNA2 (24-nt non-labeled) or RNA3 (28-nt non-labeled). The $5 ^ { \prime }$ -protruded RNA helix was annealed with RNA1 and RNA4. The $3 ^ { \prime }$ -protruded RNA helix was annealed with RNA1 and RNA5. The blunt RNA helix was annealed with RNA1 and RNA6. All the

oligonucleotides are listed in Table S2.

# RNA helix unwinding assay

The standard helix destabilizing assay was performed as previously described (Shu et al., 2019). Briefly, the indicated recombinant protein and O.1 pmol of HEX-labeled helix substrate were added to the standard reaction mix containing $5 0 ~ \mathrm { m M }$ HEPESKOH $\mathrm { \Phi _ { p H 8 . 0 ) } }$ , 50 mM NaCl, 2 mM MgCl2, 5 mM ATP and 20 U RNasin (Promega). After incubation at $3 7 ^ { \circ } \mathrm { C }$ , the reaction was terminated by adding $1 0 \times$ loading buffer[100 mM Tris-HCl, $1 \%$ SDS, $50 \%$ glycerol,and bromophenol blue $\left( \mathrm { p H } 7 . 5 \right) \dot { }$ ]. The mixtures were then electrophoresed on $1 5 \%$ native-PAGE gels, followed by scanning with a Typhoon 9500 imager (GE Healthcare,Piscataway, NJ).

# Bismuth salts

Bismuth potassium citrate (CAS No. 57644-54-9） was purchased from MedChemExpress. Ranitidine bismuth citrate (CAS No.128345-62-O) was purchased from Changzhou Lanling Pharmaceutical Co., Ltd. Bismuth citrate (CAS No. 813-93- 4) was purchased from Sigma-Aldrich. These compounds were serially diluted to the indicated concentrations and added to the ATPase and helix unwinding reactions, respectively.

# Acknowledgments

We wish to thank Prof.P. Zhou (Wuhan, China) for kindly provided materials.

This work was supported by the Strategic Priority Research Program of CAS

(XDB29010300 to X.Z.), National Natural Science Foundation of China (81873964 to Y.Q., 31800140 to J.M. and 31670161 to X.Z.), the National Science and Technology Major Project (2018ZX10101004 to X.Z.).

# Abbreviations

CoV, coronavirus； SARS-CoV-2, severe acute respiratory syndrome coronavirus 2;Covid-19，coronavirus disease 2019； nspl3,nonstructural protein 13； NTPase,nucleoside triphosphate hydrolase, ORF, open reading frame; dsRNA, double-strandedRNAs; $E$ ，coli,Escherichia. Coli; BPC, bismuth potassium citrate； RBC,ranitidinebismuth citrate; BC,bismuth citrate; ZBD, zinc-binding domain.

# Conflicts of interest

The authors declare that there are no conflicts of interest.

# References

1.Adedeji, A.O., Singh, K., Calcaterra,N.E.,DeDiego,M.L.,Enjuanes,L., Weiss,S.,and Sarafianos, S.G. (20l2).Severe acute respiratory syndrome coronavirus replication inhibitor that interferes with the nucleic acid unwinding of the viral helicase.Antimicrob Agents Chemother 56, 4718-4728.   
2. Bleichert, F. and Baserga, S.J. (2007). The long unwinding road of RNA helicases. Mol Cell 27,   
339-352.   
3. Chan, J.F., Yuan, S., Kok, K.H.,To, K.K.,Chu,H., Yang,J., Xing,F.,Liu,J., Yip, C.C.,Poon,R.W., et al. (2020). A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet.   
4. Chen, N., Zhou, M.,Dong, X., Qu, J., Gong,F., Han, Y., Qiu, Y., Wang, J., Liu, Y., Wei, Y., et al. (2020a). Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet.   
5. Chen, Y., Liu,Q.,and Guo,D. (202ob). Emerging coronaviruses: genome structure, replication, and pathogenesis. J Med Virol.   
6. Cui,J.,Li, F.,and Shi, Z.L. (2019). Origin and evolution of pathogenic coronaviruses. Nat Rev Microbiol 17,181-192.   
7.Guan, W.-j.,Ni,Z.-y.,Hu,Y.,Liang,W.-h.,Ou,C.-q.,He,J.-x.,Liu,L.,Shan,H.,Lei,C.-1.,Hui, D.S., et al. (2020). Clinical characteristics of 2019 novel coronavirus infection in China. medRxiv.   
8． Huang, C., Wang, Y., Li, X., Ren, L., Zhao, J., Hu, Y., Zhang, L., Fan, G., Xu, J., Gu, X., et al. (2020). Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet.   
9. Ivanov, K.A.,Thiel, V.,Dobbe,J.C., van der Meer, Y., Snijder, E.J., and Ziebuhr, J. (2004).Multiple enzymatic activities associated with severe acute respiratory syndrome coronavirus helicase. J Virol 78, 5619-5632.   
10. Jain, R., Coloma, J., Garcia-Sastre,A., and Aggarwal, A.K. (2016). Structure of the NS3 helicase from Zika virus. Nat Struct Mol Biol 23, 752-754.   
11. Lee, C.,Lee, J.M.,Lee, N.R., Kim, D.E., Jeong, Y.J.,and Chong, Y. (2009). Investigation of the pharmacophore space of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) NTPase/helicase by dihydroxychromone derivatives. Bioorg Med Chem Lett 19， 4538-4541.   
12.Li, T.F., Hosmillo,M., Schwanke, H., Shu, T., Wang, Z., Yin,L., Curry, S., Goodfellow, I.G., and Zhou, X. (2018). Human Norovirus NS3 Has RNA Helicase and Chaperoning Activities. J Virol 92, e01606-01617..   
13．Lorsch, J.R. (2002). RNA chaperones exist and DEAD box proteins get a life. Cell 109, 797-800.   
14. Lu, R., Zhao, X.,Li,J., Niu,P., Yang,B., Wu, H., Wang, W., Song,H., Huang, B., Zhu,N., et al. (2020). Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet.   
15.Musier-Forsyth, K.(2010). RNA remodeling by chaperones and helicases.RNA Biol 7, 632-633.   
16.Pfister, T.,and Wimmer, E. (1999). Characterization of the nucleoside triphosphatase activity of poliovirus protein 2C reveals a mechanism by which guanidine inhibits poliovirus replication. J Biol Chem 274，6992-7001.   
17.Rothe, C., Schunk, M., Sothmann, P., Bretzel, G., Froeschl, G., Wallrauch, C., Zimmer, T., Thiel, V.， Janke, C.,， Guggemos,W.，et al. (2020). Transmission of 2019-nCoV Infection from an Asymptomatic Contact in Germany.N Engl J Med.   
18.Seybert, A., Posthuma, C.C., van Dinten, L.C., Snijder, E.J. Gorbalenya, A.E.,and Ziebuhr, J. (2005). A complex zinc finger controls the enzymatic activities of nidovirus helicases. J Virol 79, 696-704.   
19.Shu,T., Gan,T.,Bai,P., Wang, X., Qian,Q., Zhou, H., Cheng, Q., Qiu, Y., Yin,L.,Zhong, J., et al. (2019). Ebola virus VP35 has novel NTPase and helicase-like activities. Nucleic Acids Res 47, 5837-5851.   
20.Subissi,L.,Imbert,1.,Ferron,F., Collet,A., Coutard,B., Decroly,E., and Canard,B. (2014). SARSCoV ORF1b-encoded nonstructural proteins 12-16: replicative enzymes as antiviral targets. Antiviral Res101，122-130.   
21.Sun, H.,Li, H.， Harvey,I.，and Sadler,P.J. (199). Interactions of bismuth complexes with metallothionein(II). JBiol Chem 274，29094-29101.   
22.Tanner, J.A., Watt, R.M., Chai, Y.B.,Lu,L.Y., Lin, M.C.,Peiris, J.S.,Poon,L.L., Kung, H.F.,and Huang, J.D. (2003). The severe acute respiratory syndrome (SARS) coronavirus NTPase/helicase belongs to a distinct class of $5 "$ to $3 ^ { \prime }$ viral helicases. JBiol Chem 278,39578-39582.   
23.Wang,D., Hu,B.,Hu, C., Zhu,F.,Liu, X., Zhang,J., Wang,B., Xiang,H., Cheng, Z., Xiong, Y., et al. (2020). Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel CoronavirusInfected Pneumonia in Wuhan, China. JAMA.   
24.WHO(2020). Coronavirusdisease2019(COVID-19) SituationReport-26. https://wwwwhoint/docs/default-source/coronaviruse/situation-reports/20200215-sitrep-26-covid19pdf?sfvrsn=a4cc6787_2 (accessed Feb 15, 2020).   
25. Xia, H., Wang,P., Wang, G.C., Yang, J., Sun, X., Wu, W., Qiu, Y., Shu,T., Zhao, X., Yin,L., et al. (2015). Human Enterovirus Nonstructural Protein 2CATPase Functions as Both an RNA Helicase and ATP-Independent RNA Chaperone.PLoS Pathog 11， e1005067.   
26.Yang,J., Cheng, Z., Zhang, S., Xiong, W., Xia, H., Qiu, Y., Wang, Z., Wu,F., Qin, C.F.,Yin,L.,et al. (2014). A cypovirus VP5 displays the RNA chaperone-like activity that destabilizes RNA helices and accelerates strand annealing. Nucleic Acids Res 42, 2538-2554.   
27． Yang, J., Qian, Q.,Li,T.F., Yang,X., Won, S.J.,and Zhou, X. (2017). Cypovirus capsid protein VP5 has nucleoside triphosphatase activity. Virol Sin 32， 328-330.   
28.Yang,N.,Tanner,J.A., Wang,Z.,Huang,J.D., Zheng,B.J.,Zhu,N.,and Sun, H. (20o7a). Inhibition of SARS coronavirus helicase by bismuth complexes. Chem Commun (Camb)， 4413-4415.   
29.Yang,N.,Tanner,J.A., Zheng,B.J., Watt,R.M., He, M.L.,Lu,L.Y.,Jiang, J.Q.,Shum, K.T.,Lin, Y.P., Wong, K.L., et al. (2007b). Bismuth complexes inhibit the SARS coronavirus. Angew Chem Int Ed Engl 46, 6464-6468.   
30. Yu, M.S., Lee, J., Lee, J.M., Kim, Y., Chin, Y.W., Jee, J.G., Keum, Y.S.,and Jeong, Y.J. (2012). Identification of myricetin and scutellarein as novel chemical inhibitors of the SARS coronavirus helicase, nsP13. Bioorg Med Chem Lett 22， 4049-4054.   
31. Zhang, H., Kang, Z., Gong,H., Xu,D., Wang,J.,Li,Z., Cui, X., Xiao,J., Meng,T., Zhou, W., et al. (2020).The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes. bioRxiv.   
32.Zhou,P.,Yang, X.L., Wang, X.G., Hu,B., Zhang,L., Zhang, W.,Si, H.R.,Zhu,Y.,Li,B.,Huang, C.L., et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. LID 10.1038/s41586-020-2012-7[doi].   
33. Zhu, N., Zhang, D., Wang, W.,Li, X., Yang, B., Song, J., Zhao, X., Huang, B., Shi, W., Lu,R., et al.A Novel Coronavirusfrom Patients with Pneumonia in China， 2019.LID 10.1056/NEJMoa2001017 [doi].

![](images/f129d65928f12a7e81198b0605369f4bf30872d0516ce738558ca318e682b130.jpg)  
Fig 1

Figure 1. SARS-CoV-2 nsp13 has NTPase activity. (A) 1O pmol MBP-nsp13 was reacted with the indicated NTPs ( $2 . 5 \mathrm { m M }$ for each). The NTPase activity Was measured as nanomoles of released inorganic phosphate by using a sensitive colorimetric assay. (B) $2 . 5 \ \mathrm { m M }$ ATP was incubated with MBP alone or MBP-nspl3 at the increasing concentrations.(C) 1O pmol MBP-nspl3 was reacted with $2 . 5 \ \mathrm { m M }$ ATP at $2 ~ \mathrm { m M }$ indicated divalent metal ions. (D) 1O pmol MBP-nsp13 was reacted with $2 . 5 \mathrm { m M }$ ATP at the indicated concentrations of MgCl2. MBP alone was used as the negative control. Error bars represent standard deviation (SD) values from three separate experiments.

![](images/b30ee0fc73def00423ff164e7ad107a35b82a2d4b35551e076a34bb3393e43d6.jpg)  
Fig 2   
Figure 2. SARS-CoV-2 nspl3 has RNA helix unwinding activity. (A) Schematic

illustration of the RNA helix substrate $( \mathrm { R } / \mathrm { R } ^ { * } )$ ．Asterisks indicate the HEX-labelled strands. (B) The RNA helix substrate (O.1 pmol) was reacted with each indicated protein (20 pmol). And the unwinding activity was assessed via gel electrophoresis and scanning on a Typhoon 95oO imager. Non-boiled reaction mixture (lane 1) and reaction mixture with MBP alone (lane 3) were used as negative controls,and boiled reaction mixture (lane 2) was used as positive control. (C) The RNA helix unwinding assay was performed in the presence of increasing concentrations of MBP-nsp13. (D) MBP-nsp13 (20 pmol) was reacted with the RNA helix substrate (O.1 pmol) in reaction mixture at different reaction time.

![](images/e01abb01a77fc420e032cd90342460d7ddeefcbed6af1f166263e1110c878724.jpg)  
Fig 3

Figure 3. Optimal biochemical reaction conditions for the RNA helix unwinding activity of SARS-CoV-2 nsp13.(A) Schematic illustration of the RNA helix substrate $( \mathrm { R } / \mathrm { R } ^ { * } )$ . Asterisks indicate the HEX-labelled strands. (B-D) MBP-nsp13 (2O pmol) was reacted with the RNA helix substrate (O.1 pmol) in the presence of the indicated NTPs （204号 $( 5 \mathrm { m M } )$ (B）each indicated divalent metal ion $\mathbf { \Omega } ^ { 2 } \mathbf { \Omega } ^ { \mathrm { m M } }$ for each) (C)，increasing concentrations of $\mathrm { M g C l } _ { 2 }$ (D).

![](images/2747cf15acb87428e2e673c33bce8ea400a844b4e5f807d4d8bb93cc9ac775cd.jpg)  
Fig 4   
Figure 4. SARS-CoV-2 nsp13 unwinds RNA helix in the $\mathbf { 5 ^ { \circ } }$ to $\mathbf { 3 ^ { \circ } }$ directionality. (A

C) Schematic illustrations of the RNA helix substrates with $5 '$ -tailed (A), $3 ^ { \circ }$ -tailed (B), and blunt ends (C). Asterisks indicate the HEX-labelled strand. (D) MBP-nsp13 (20 pmol) was reacted with O.1 pmol $5 ^ { \circ }$ -tailed (lane 3) or $_ { 3 } ,$ -tailed (lane 4) RNA helix substrate.(E) MBP-nsp13 (2O pmol) was reacted with O.1 pmol helix substrate with blunt ends.

![](images/bebf18b8e0d95355cd74986852fba0d4799523ce4a5a1ccf62a6cc2106faa7a5.jpg)  
Fig 5   
Figure 5. Bismuth salts inhibit the ATPase and RNA helix unwinding activities of

SARS-CoV-2 nsp13. (A) The ATPase activity of MBP-nspl3(1O pmol） in the presence of the indicated bismuth salts ( $1 0 \mu \mathrm { M }$ for each). X-axis were expressed as the different bismuth salts. Values (Y-axis） were expressed as percentages of those of mock-treated (-） ATPase activity of MBP-nspl3 (1O pmol). The error bars represent SD values from three separate experiments. (B) Upper panel: schematic illustration of the RNA helix substrate $( \mathrm { R } / \mathrm { R } ^ { * } )$ ；asterisks indicate the HEX-labelled strand. Lower panel: The RNA helix unwinding assays were performed by incubating O.1 pmol RNA helix substrate with MBP-nspl3 (2 pmol) in presence of the indicated bismuth salts (10 $\mu \mathrm { M }$ for each).

Fig 6

![](images/49823a46df0e8dc52532ceb8956f06f28ebafeead8b00be1a04cd9e59e48ddd6.jpg)

Figure 6. BPC and RBC inhibit the ATPase and RNA helix unwinding activities of SARS-CoV-2 nsp13 in a dose-dependent manner. (A and D) The ATPase activity of MBP-nsp13 (1O pmol) was performed in the presence of increasing concentrations of BPC(A) or RBC (D) as indicated. (B and E) The RNA helix unwinding activity of MBP-nspl3 (2 pmol) was performed in the presence of increasing concentrations of BPC(B) or RBC (E) as indicated. (C and F) The unwinding activities in (B and E) were plotted as percentages of the released HEX-labelled RNA from the total RNA helix (Y-axis) at indicated concentrations (X-axis) of BPC (C) or RBC (F). The error bars represent SD values from three separate experiments.

# Supplementary Information

![](images/b56d5758ee4c076c5c41a383a5640b75fa826e88bd6c00b07a6e596c995278da.jpg)

Figure S1. Expression of recombinant SARS-CoV-2 nsp13. The purified MBP alone (lane 2） and MBP-fusion SARS-CoV nspl3 (lane 3) were subjected to $10 \%$ SDSPAGE followed by Coomassie brilliant blue R25O staining. Lane 1, protein marker.

Table S1 The primers used in this study.   

<html><body><table><tr><td>Primer</td><td>Sequence (5`-3")</td></tr><tr><td>nsp13-F-BmaH1</td><td>CGGGATCCATGGCTGTTGGGGCTTGTGTTCTTTGC</td></tr><tr><td></td><td>AATTCACAGACTTC</td></tr><tr><td>nsp13-R-SaL1</td><td>GCGTCGACTCATTGCAACTTGTCATAAAGGTCTCT</td></tr><tr><td></td><td>ATCAGACATTATGC</td></tr></table></body></html>

Table S2 The oligonucleotids used in this study   

<html><body><table><tr><td colspan="2">Oligonucleotide</td></tr><tr><td>RNA1</td><td>CAUUAUCGGAUAGUGGAACCUAGCUUCGACUAUCGGAUAAUC</td></tr><tr><td>RNA2</td><td>AUAGUCGAAGCUAGGUUCCACUAU</td></tr><tr><td>RNA3</td><td>CGAUAGUCGAAGCUAGGUUCCACUAUCC</td></tr><tr><td>RNA4</td><td>GAUUAUCCGAUAGUCGAAGCUAGG</td></tr><tr><td>RNA5</td><td>GCUAGGUUCCACUAUCCGAUAAUG</td></tr><tr><td>RNA6</td><td>GAUUAUCCGAUAGUCGAAGCUAGGUUCCACUAUCCGA</td></tr><tr><td></td><td>UAAUG</td></tr></table></body></html>

\*HEX-labeled strands are in boldface.