# Structural View and Substrate Specificity of Papain-like Protease from Avian Infectious Bronchitis Virus\*

Recei   
Lingying Kong\*1, Neil Shaws11,Lingming Yan\*, Zhiyong Lou+,and Zihe Rao\*512   
Fromthe\*Laboratoryoftructuraliologychoolofedicine,TsingaUiversityBeijingo084,China,Cgeoe   
Sciences,NaiUversityanjiindNatioaboatoryfcromleculesstitutei   
Chinese Academy of Science,Beijing 100101,China

Background: Whether papain-like protease (PLpro)from avian infectious bronchitis virus (IBV) that infects poultry, causing huge losses, can perform deubiquitination is unknown.

Results: IBVPLpro cleaves Lys48-and Lys6-linked polyubiquitin.Thecrystal structure reveals a markedly diferentsetof residues for binding substrates.

Conclusion: IBVPLpro performs deubiquitination with catalytic eficiencies thatare different from other PLpros.   
Significance: Results provide a framework for developing antivirals.

Papain-like protease (PLpro) of coronaviruses (CoVs) carries out proteolytic maturation of non-structural proteins that play a role in replication of the virus and performs deubiquitination of host cell factors to scuttle antiviral responses.Avian infectious bronchitis virus (IBV),the causative agent of bronchitis in chicken that results in huge economic losses every year in the poultry industry globally,encodes a PLpro.The substrate specificitiesof thisPLpro are not clearly understood.Here,we show that IBV PLpro can degrade Lys48- and Lys63-linked polyubiquitin chains to monoubiquitin but not linear polyubiquitin. To explain the substrate specificities,we have solved the crystal structure of PLpro from IBV at 2.15-A resolution.The overall structure is reminiscent of the structure of severe acute respiratory syndrome CoVPLpro.However,unlike the severe acute respiratory syndrome CoVPLpro that lacks blocking loop (BL) 1 of deubiquitinating enzymes, the IBVPLpro has a short BL1-like loop.Access to a conserved catalytic triad consisting of $\mathbf { C y s ^ { 1 0 1 } }$ $\mathbf { H i s ^ { 2 6 4 } }$ ,and $\mathbf { A s p } ^ { 2 7 5 }$ is regulated by the flexible BL2.A model of ubiquitin-bound IBV CoVPLpro brings out key differences in substrate binding sites of PLpros.In particular,P3 and P4 subsitesas well as residues interacting with the $\pmb { \beta }$ -barrelofubiquitin are different, suggesting different catalytic efficiencies and substrate specificities.We show that IBVPLpro cleaves peptide substrates KKAG-7-amino-4-methylcoumarin and LRGG-7- amino-4-methylcoumarin with different catalytic efficiencies. These results demonstrate that substrate specificities of IBV PLpro are different from other PLpros and that IBVPLpro might target different ubiquitinated host factors to aid the propagation of the virus.

Avian infectious bronchitis virus $\left( \mathrm { I B V } \right) ^ { 3 }$ is an enveloped, single-stranded RNA virus belonging to the Coronavirus genus (1, 2). The virus infects the upper respiratory tract of chickens, spreading progressively to the digestive and urinary tracts during the later stages of the infection (3).IBVis highly contagious, and each year outbreaks of IBV infections result in huge economic losses to the poultry industry globally. Currently available vaccines do not provide adequate protection against IBV because the strains show variability (4).There are no IBV-specific therapeutics available on the market.Therefore,it is important to study proteins of the virus that aid its replication and assist in circumventing the host immune surveillance machinery.Selective abrogation of function of such proteins could help prevent the spread of the virus.The papain-like protease (PLpro) of IBV is one such protein that could be targeted fordevelopment of effective antivirals.

IBV replicates in the cytoplasm of the cell using endogenous replicase proteins translated from the genomic RNA (5,6). Initially,two large viral polypeptides are translated that are proteolytically processed into 15 non-structural proteins (nsp1- 15) that play a role in the replication of the viral genome (7). Two viral proteases,aPLpro and a main protease,are known to perform the proteolytic maturation of the nsps of IBV (8-10). In particular,PLpro is known to cleave at sites between nsp2 and nsp3 as well as between nsp3 and nsp4 (10-12).So far, genomic RNA of IBV has been shown to encode only one functional PLpro that shows similarity to PL2pro from other viruses (13,14).In addition to proteolytic maturation of nsps,PLpro of coronaviruses is known to perform deubiquitination and removal of interferon stimulated gene 15 (ISG15) from of host proteins (15-19).For instance,the PLpro of SARS CoV is known to deubiquitinate interferon regulatory transcription factor 3,which stalls its migration to the nucleus,preventing the initiation of host antiviral responses (15,20,21). Similarly, the PLpro from MERS CoV has been shown to degrade polyubiquitin chains associated with MDA-5 and RIG-I, preventing activation of host antiviral signaling pathways (14,22). Consequently,inhibition of PLpro by small molecule inhibitors is considered a viable strategy to stall the life cycle of the virus (23-25).Whether thePLpro from IBV can perform similar dual functions (proteolytic maturation of viral nsps and deubiquitination of host cell factors) is currently not known.

Here,we describe the crystal structure of PLpro from IBV refined to 2.15-A resolution. Comparison of the structure with PLpros from other viruses reveals conservation of catalytic residues.However,differencesareobserved inaminoacids of the substrate binding site,suggesting differences in substrate specificities.We show that,unlike the PLpro from SARS CoV, the PLpro from IBV prefers Lys6³-linked polyubiquitin over Lys48 for degradation.However,both PLpros do not cleave linear polyubiquitin chains.Furthermore,thePLpros differin kinetics of degradation of peptide substrates. The reasons underlying these differences are discussed.

# EXPERIMENTALPROCEDURES

ExpressionandPurificationofIBVPapain-likeProteaseIBV PLpro (polyprotein residues 1174-1483） cloned into pGEX-6p-1 vector was expressed in Escherichia coli BL21(DE3) cells (Novagen). Cells were grown at $3 7 ^ { \circ } \mathrm { C }$ to an $A _ { 6 0 0 \mathrm { n m } }$ of ${ \sim } 0 . 5$ . The culture was cooled,and recombinant protein production was initiated by inducing the culture with $1 \mathrm { m M }$ isopropyl D-thiogalactoside for $^ { 4 \mathrm { h } }$ at $2 8 ^ { \circ } \mathrm { C }$ under shaking conditions. At the end of the incubation time,cells were pelleted by centrifugation and resuspended in $4 0 \mathrm { m l }$ of ice-cold lysis buffer (50 mM Tris-HCl, $\mathrm { p H } 7 . 5$ ,200 mM NaCl, $5 ~ \mathrm { m M }$ DTT). Cells were lysed using an ultra-high pressure cell disrupter (JNBIO, Guangzhou, China). Following centrifugation to remove cell debris and unbroken cells,recombinant protein was purified from the supernatant by affinity chromatography using GST resin.After thorough washing to remove contaminating protein,recombinant protein bound to the resin was eluted using a buffer containing $5 0 \mathrm { m M }$ Tris-HCl, $\mathrm { p H } 7 . 5$ $5 0 0 \mathrm { m M N a C l } _ { \cdot }$ $5 \mathrm { m m }$ DTT, $1 5 ~ \mathrm { m M }$ reduced glutathione.Recombinant GST-human rhinovirus 3c protease fusion protein was added to the eluted protein,and the mixture was dialyzed overnight against $5 0 \mathrm { m M }$ Tris-HCl, $\mathrm { p H } 7 . 5$ $5 0 \mathrm { m M N a C l }$ $1 \mathrm { m M }$ DTTbuffer.The dialyzed solution was passed over GST resin to remove the GST-human rhinovirus 3c and free GST polypeptide.Recombinant IBV PLpro was then further purified using an ion exchange chromatography step.Recombinant protein bound to the Mono Q column was eluted using a linear gradient of NaCl.Fractions containing the protein were pooled,concentrated,and loaded onto a Superdex 75 gel filtration column(GE Healthcare).IBV PLpro eluted in a single peak.Fractions containing the protein were pooled,concentrated to $\mathrm { 1 0 - 1 5 ~ m g / m l } ,$ and stored at $- 2 0 ^ { \circ } \mathrm { C }$ until further use.Selenomethionine-labeled IBV PLpro(1174- 1483）was produced following a procedure published previously(26) and purified asdescribed above.The PLpro catalytic domains from SARS CoV and MERS CoV were expressed and purified separately according to methods published previously (14).All purification procedures were carried out at $4 ^ { \circ } \mathrm { C }$

Crystallization-CrystallizationofIBVPLpro was performed at $1 8 ^ { \circ } \mathrm { C }$ using the hanging drop vapor diffusion technique. Crystals were obtained by mixing $1 \mu \mathrm { l }$ of protein solution (10 $\mathrm { m g / m l } )$ with an equal volume of reservoir solution.The crystallization drops were equilibrated against $2 0 0 ~ \mu \mathrm { l }$ of reservoir solution.Crystals of native IBVPLpro were obtained ina solution containing $1 0 0 \mathrm { m M }$ Bis-Tris, pH 6.5, $2 5 \mathrm { - } 3 0 \%$ polyethylene glycol monomethyl ether 2oo0.Crystals grew to maximum dimensions of $1 0 0 \times 5 0 \times 1 5 0 \mu \mathrm { m }$ in 3days.Selenomethioninederivatized IBVPLpro crystallized under similar conditions.

Data Collection, Processing, and Structure DeterminationCrystals were transferred into a harvesting solution containing the respective precipitant solutions plus $5 \%$ (v/v）glycerol before being flash frozen in liquid nitrogen.Data were collected under cryogenic conditions at $1 0 0 ~ \mathrm { K }$ ，The selenomethionine single wavelengthanomalous dispersion data set of IBVPLpro at the selenium peak wavelength was collected at Shanghai Synchrotron Radiation Facility(China) beamline BL17U1 to $1 . \dot { 5 } { - } \mathring \mathrm { A }$ resolution.All data sets were processed using the HKL-3000 package (27). Crystals belonged to space group $I 2 2 2$ with cell parameters $a = 7 1 . 3 \mathring { \mathrm { A } } , b = 9 5 . 1 \mathring { \mathrm { A } } , c = 1 1 8 . 4 \mathring { \mathrm { A } } , \alpha = \beta = \gamma =$ $9 0 ^ { \circ }$ .The only selenium atom waslocated and refined,and the single wavelength anomalous dispersion data phases were calculated and substantially improved by solvent flattening using the PHENIX program (28).A model was manuallybuilt into the modified experimental electron density using Coot (29) and furtherrefined inPHENIX.Refinement statistics for the final model are summarized in Table1.Model geometry was verified using the program MolProbity (3O). Structural figures were drawn using the program PyMOL (31).

Steady-stateKinetic Studies-The ratesof IBVPLpro-catalyzed reactions were determined usinga modified version of methods described previously (18,22,32).Briefly，the rate ofhydrolysisofpeptidesubstratescarboxybenzyl-LeuArg-Gly-Gly-7-amino-4-methylcoumarin (Z-LRGG-AMC), carboxybenzyl-Lys-Ala-Gly-Gly-7-amino-4-methylcoumarin (Z-KAGG-AMC),and carboxybenzyl-Lys-Lys-Ala-Gly-7- amino-4-methylcoumarin (Z-KKAG-AMC) (Life Sensors,Inc.) was determined by measuring the increase in fluorescence of the AMC group released (excitation $\lambda = 3 4 0 \mathrm { n m }$ emission $\lambda =$ $4 6 0 ~ \mathrm { n m } ,$ ）at different time points.The reaction mixtures consisted of various concentrations of peptide substrate (10-160 $\vert \mu _ { \mathrm { { M } } } \rangle$ suspended in $2 0 \mathrm { m M }$ Tris-HCl, pH7.5, $0 . 1 \mathrm { m g / m l }$ BSA,150 mM NaCl, $2 \mathrm { m M }$ DTT.Assays were initiated by the addition of $0 . 2 \ \mu \mathrm { M }$ pure IBVPLpro,and the incubation was carried out at $2 5 ^ { \circ } \mathrm { C } .$ .The intensity of the fluorescence observed was converted to the amount of substrate hydrolyzed by comparing it with the fluorescence of known standards.The ability of SARS CoV PLpro and MERS CoV PLpro to hydrolyze Z-LRGG-AMC was tested using the same assay with the addition of $0 . 2 ~ \mu \mathrm { M }$ SARS CoVPLpro or $1 . 6 \mu \mathrm { M }$ MERS CoVPLpro instead of IBVPLpro. Active sites of all three enzymes could not be saturated using the peptide substrates.Therefore,initial rates of the reactions fordifferent concentration of substrate were fit to the Michaelis-Menten equation using linear regression with SigmaPlot (Systat Software,Inc.) to obtain the apparent $k _ { \mathrm { c a t } } / K _ { m }$ values.

# Structure of Papain-like Protease from IBV

Ubiquitin Chain Cleavage Experiments-Proteolytic cleavage of linear $\mathrm { U b } _ { 4 } ,$ Lys48 $\mathrm { \Delta U b _ { 4 } }$ and $\mathrm { L y s ^ { 6 3 } – U b _ { 4 } }$ (all from UBPBio) was carried out as follows. $5 0 \mathrm { n M }$ purified protein (IBVPLpro) was incubated with $4 0 \mu \mathrm { g }$ of linear $\mathrm { U b } _ { 4 } , \mathrm { L y s } ^ { 4 8 } – \mathrm { U b } _ { 4 } ,$ or $\mathrm { L y s } ^ { 6 3 } – \mathrm { U b _ { 4 } }$ at $2 5 ^ { \circ } \mathrm { C }$ in a $5 0 \mathrm { - } \mu \mathrm { l }$ volume containing $5 0 \mathrm { m M }$ Tris-HCl, $\mathrm { p H } 7 . 5 .$ $1 0 0 \ \mathrm { m M \ N a C l _ { \mathrm { : } } }$ $2 ~ \mathrm { m M }$ DTT.A control reaction was incubated under identical conditions with the exclusion of enzyme.At various time points, $8 ~ \mu \mathrm { l }$ were removed,and the reaction was quenched with the addition of SDS-PAGE sample loading buffer.The samples were analyzed by electrophoresis on a $4 - 1 5 \%$ SDS-polyacrylamide gel and stained with Coomassie dye.

# RESULTS

Overall Structure-IBV PLpro (polyprotein residues $1 1 7 4 -$ 1483) could be overexpressed as soluble protein in $E _ { \ast }$ coli.The protein was purified to homogeneity using GST affinity,ion exchange,and size exclusion chromatography steps.Pure IBV PLpro crystallized in a number of crystallization conditions. The structure was solved by the selenomethionine single wavelength anomalous dispersion method by collectinga data set of selenomethionine-labeled IBV PLpro crystal (I222 space group) at the selenium peak wavelength.Clear electron density permitted unambiguous modeling of residues.The final model consisting of residues 4-304 was refined to 2.15-A resolution $( R _ { \mathrm { w o r k } } = 1 7 . 5 \%$ and $R _ { \mathrm { f r e e } } = 2 1 . 6 \%$ . Statistics pertaining to data collection and refinement are listed in Table1.

The overall structure of the IBV PLpro consists of four distinctdomains(Fig. $1 , A$ and $B$ ).Thefirst 56 residuesfoldintoa ubiquitin-like(Ubl) domain. The Ubl domain is well separated from the other three domains that contact each otherand form a compact globular structure.This Ubl domain adopts a $\beta$ -grasp fold similar to ubiquitin and is followed by three other domains representing the thumb,fingers,and palm of an extended right hand,an arrangement previously observed for deubiquitinating enzymes.The thumb domain is predominantly $\alpha$ -helical.It contains six $\alpha$ -helices and two short $\beta$ -strandsthatarearranged ina $\beta$ -hairpin.The thumbdomain makes contact with the palm domain using this $\beta$ -hairpin. Amino acids 173-31O fold into the finger and palm domains. This region of the protein consists of mostly $\beta$ -strands. Although the finger domain is made up of three long (β6, $\beta 7$ and $\beta 1 0 ^ { \cdot }$ )and two short $\beta$ -strands,the palm domain is made up of seven $\beta$ -strands. One $Z \mathrm { n } ^ { 2 + }$ ion coordinated by four cysteine residues originating from two $\beta$ -hairpins is located between strands $\beta 6$ and $\beta 1 0$ of the finger domain. The conformation of the side chains of cysteines and the arrangement of the $\beta$ -hairpins are similar to those observed for members belonging to the zinc ribbon fold group. The three long $\beta$ -strands of the finger domain protrude out of the protein.This facilitates formation of a hydrophobic cavity at the junction of thumb and finger domains,which has implications for the function of the protein.

Comparison of the Overall Fold-Despite sharing only $1 2 \mathrm { - } 1 5 \%$ sequence identity with ubiquitin-specific proteases (USPs) for which crystal structures are available,the overall architecture and domain organization of the IBVPLpro is similar to USPs.Unlike USPs,PLpros are dual functional enzymes and therefore their substrate binding sites are designed to accommodate two different substrates.For instance,in addition to cleaving ubiquitin, the IBV PLpro cleaves the nascent viral polypeptide between nsp2 and nsp3 as well as between nsp3and nsp4.Comparison of the structures ofUSPs with IBV PLpro reveals differences in the region connecting the finger domain with the palm domain that permits promiscuity in catalysis by IBV PLpro.In particular,“blocking loop"(BL）1, connecting $\beta 8$ with $\beta 9$ ，which is known to regulate the deubiquitinating activity of USP14,is made up of 22 residues and is part of the substrate-binding surface.The corresponding loop in IBV PLpro is much shorter $( \mathrm { A s p } ^ { 2 4 5 } \mathrm { - V a l } ^ { 2 5 1 } )$ and connects $\beta 1 1$ with $\beta 1 2$ .Asa consequence, the substrate binding site is larger in IBVPLpro than in USPs.This probably enables the enzyme to not only bind ubiquitin but also viral polyprotein and operate asa dually functional enzyme.

TABLE1 Datacollection and refinement statistics Numbers in parentheses are corresponding values for the highest resolution shell. r.m.s.d., root mean square deviation; Res., residues; Se-SAD,selenomethionine single wavelength anomalous dispersion.   

<html><body><table><tr><td>Parameters</td><td>Se-SAD peak</td></tr><tr><td>Data collection statistics</td><td></td></tr><tr><td>Cell parameters</td><td></td></tr><tr><td>a(A)</td><td>71.3</td></tr><tr><td>b(A)</td><td>95.1</td></tr><tr><td>c(A)</td><td>118.4</td></tr><tr><td>α,β,γ(）</td><td>90,90,90</td></tr><tr><td>Space group</td><td>I222</td></tr><tr><td>Wavelength used (A)</td><td>0.97855</td></tr><tr><td>Resolution (A)</td><td>50.00-2.00 (2.07-2.00)</td></tr><tr><td>No.of all reflections</td><td>52,949 (5,254)</td></tr><tr><td>No.of unique reflections</td><td>8,146 (809)</td></tr><tr><td>Completeness (%)</td><td>100.0 (100.0)</td></tr><tr><td>Average I/σ(I)</td><td>26.0 (2.8)</td></tr><tr><td>Rmerge(%)</td><td>12.4 (73.5)</td></tr><tr><td>Refinement statistics</td><td></td></tr><tr><td>No.of reflections used (o(F)>0)</td><td>22,249</td></tr><tr><td>Rwork (%)</td><td>17.5</td></tr><tr><td>Rfree(%)</td><td>21.6</td></tr><tr><td>r.m.s.d. bond distance (A)</td><td>0.008</td></tr><tr><td>r.m.s.d. bond angle ()</td><td>1.027</td></tr><tr><td>Average B-value (A²)</td><td>37.1</td></tr><tr><td>No.of protein atoms</td><td>2,504</td></tr><tr><td>No.of ligand atoms</td><td>1</td></tr><tr><td>No. of solvent atoms</td><td>128</td></tr><tr><td>Ramachandran plot</td><td></td></tr><tr><td>Res.in favored regions (%)</td><td>97.6</td></tr><tr><td>Res.in allowed regions (%)</td><td>2.4</td></tr><tr><td>Res.in outlier regions (%)</td><td>0</td></tr></table></body></html>

（ $\overline { { ^ { \iota } R _ { \mathrm { m e r g e } } } } = \Sigma _ { h } \Sigma _ { l } | I _ { i h } - I _ { h } | / \Sigma _ { h } \Sigma _ { I } I _ { h }$ where $I _ { h }$ is the mean of the observations $I _ { i h }$ of reflection h. （204号 $^ b R _ { \mathrm { w o r k } } = \Sigma ( \| F _ { p } ( \mathrm { o b s } ) | - | F _ { p } ( \mathrm { c a l c } ) | | ) / \Sigma | F _ { p } ( \mathrm { o b s } ) | .$ $R _ { f r e e }$ is an $R$ factor for a preselected subset $( 5 \% )$ of reflections that was not included in refinement.

![](images/e91ed714b6a2dfe24db868dead91142b67ee6ef34d9e9b7a6c44807e65f08283.jpg)  
FIGURE 1. Overall structure of PLpro from IBV.A, diagrammatic representation of the organization of domains of IBV PLpro inferred from the crystal structure. $B ,$ schematic representation of the crystal structure of IBV PLpro. The locations of BL1 and BL2 as well as the termini of the protein are marked. $Z n ^ { 2 + }$ ion bound by the protein is shown as a sphere.The $Z n ^ { 2 + }$ ion is coordinated by four cysteine residues (inset) shown as sticks.

We searched the Protein Data Bank for structural homologues of IBV PLpro using the DALI server.The search retrieved the structure of PLpro from SARS CoV as the top match ( $Z \operatorname { s c o r e } = 2 8 . 6 ;$ with a root mean square deviation of 2.4 A for 292 overlapping $\ b { \mathrm { C } } \ b { \alpha }$ atoms. Although the overall structure is similar, superimposition studies revealed four regions with significant differences between the two coronaviral PLpros (Fig. 2, $A$ and $B$ ).The first structural difference is observed in the Ubl domain.Strands $\beta 1$ and $\beta 2$ move inward in IBVPLpro. As a consequence, the loop connecting $\beta 2$ with $\beta 3$ is displaced and does not align well with a similar region of SARS CoV PLpro.The second structural difference involves the α1 helix of the thumb domain.This helix is much shorter than the corresponding helix in the SARS CoV PLpro.Because of this, the loop connecting the Ubl domain with the thumb domain is much longer in IBVPLpro.The third structural difference is observed in the palm domain.Although PLpro from IBV exhibits a BL1-like loop connecting strands $\beta 1 1$ with $\beta 1 2$ ,thePLpro from SARS CoV shows a complete lack of a BL1-like loop. Lastly, strand $\beta 1 6$ of the palm domain of SARS CoV PLpro is missing in IBVPLpro.In addition,the strand following this region (β16) in IBVPLpro is much shorter (Fig.2B).

Comparison of the structure of IBVPLpro with that of MERS CoVPLpro revealed differences in the thumb domain.MERS CoVPLpro has an extra $\beta$ -hairpin in this region.Anadditional difference involves the zinc finger motif. Specifically, the $Z \mathrm { n } ^ { 2 + }$ ion ofPLpro from MERS CoVhas moved inward from itslocation in PLpros from IBV and SARS CoV (Fig.2A).Interestingly, a feature shared between IBVand MERS CoVPLpros but missing in SARS CoVPLpro is theBL1-like loop.MERS CoVPLpro has aBL1-likeloop ata location similar to that observed in IBV PLpro.Thus,the PLpro from IBV,SARS,and MERS CoVs shows some differences that probably impart different substrate specificities and possibly govern the ability of the virus to target different signaling pathways of host cells to assist propagation of the virus.

UbiquitinBindingSiteofIBVPLpro-SimilartoPLpro from SARS and MERS CoVs, IBV PLpro also probably targets ubiquitinated host factors to aid viral replication.To gain information about the mode of binding of substrate,in particular ubiquitin,to IBV PLpro,we superimposed the structure of SARS CoV PLpro bound with ubiquitin over the structure of IBV PLpro.Because the overall structures of the two PLpros are highly similar,it is not surprising to note that in this model ubiquitin binds at a similar location, the junction of thumb, palm,and fingerdomains,in IBVPLpro (Fig.3A).The $\beta$ -barrel of the ubiquitin is primarily engaged by the finger domain, whereas the C-terminal tail containing the diglycine motif extends intoa cavity formedat the junctionof thumb and palm domains.More importantly,the diglycine is in proximity of BL2 and key catalytic residues that are conserved across PLpros (Fig. 3A).Although the overall mode of engaging the ubiquitin is similarinPLpros from IBV,SARS,and MERSCoVs, the nature of amino acids ofPLpro from these CoVs interacting with ubiquitinis different.These differences are illustrated in the following examples involving one salt bridge and several other hydrogen bonding interactions.The side-chain guanidinium group of $\mathrm { A r g ^ { 4 2 } }$ of ubiquitin forms a salt bridge with the side-chain carboxylate group of $\mathrm { G l u } ^ { 1 6 8 }$ of SARS CoV PLpro. A tryptophan $( \mathrm { T r p } ^ { 1 5 6 } )$ occupies the position of $\mathrm { G l u } ^ { 1 6 8 }$ in IBV PLpro,whereas anarginine is locatedata similarpositioninMERS CoVPLpro 2 (Fig. 3, $B$ and $C$ ).Three hydrogen bondsare formed between ubiquitin and the PLpro from SARS CoV.1) The main-chain amide nitrogen atom of $\mathrm { A l a } ^ { 4 6 }$ of ubiquitin hydrogen bonds with the side chain of $\mathrm { G l n } ^ { 2 3 3 }$ from SARS CoV PLpro. 2) The mainchain carbonyl oxygen atom of $\mathrm { G l y ^ { 4 7 } }$ hydrogen bonds with the main-chain amide of Met2o9.3) The side chain of $\mathrm { G l n } ^ { 4 9 }$ forms a hydrogen bond with the side chain of $\mathrm { A r g } ^ { 1 6 7 }$ . The equivalent residues of IBV PLpro interacting with ubiquitin, $\mathrm { A s p } ^ { 2 2 3 }$ (arginine n MERS CoV PLpro), Val199, and $\mathrm { A s n } ^ { 1 5 5 }$ (serine in MERS CoVPLpro),respectively,are all different (Fig. 3, $B$ and $C$ ).In addition, substitution of $\mathrm { H i s } ^ { 1 7 2 }$ ， $\mathrm { T y r ^ { 2 0 8 } }$ ,and $\operatorname { T y r } ^ { 2 6 9 }$ of SARS CoV PLpro with Asn16o (threonine in MERS CoV PLpro), $\mathrm { V a l } ^ { 1 9 9 }$ (valine in MERS CoV PLpro), and $\mathrm { T h r } ^ { 2 6 0 }$ (threonine in

![](images/255f97ead2f34b4a1dbefbfef1948f248dfb02d9e04b53dba09feee54c6f33d2.jpg)  
FIGURE 2.Comparison of the overall structures of PLpros from diferent CoVs.A,crystal structures of PLpros from SARS CoV(blue ribbon; Protein Data Bank code 4MoW)(19)and MERS (green ribbon; Protein Data Bank code 4RF1) CoVs (36) were superimposed over the structure of PLpro from IBV (red ribbon).The location of BL2 and the primary sequences of the PLpros corresponding to this region are shown. $\dot { Z } \mathsf { n } ^ { 2 + }$ ion is shown asa sphere.B,superimposition of the structures of PLpro from SARS CoV(blue schematic) over IBV PLpro (red schematic).The viewis the sameas in Abut rotated by $5 5 ^ { \circ }$ around the x axis.Regions that show significant deviation are marked by a red arrow. The N and C termini of the protein are marked as $N$ and $C _ { \prime }$ respectively.

![](images/210746774bfa7f52a5338343d6e3bcd8d70a76992dd016468e8fe82e1f16443e.jpg)  
FIGURE3SubtrateingieprAitinUueaticorucuodituiinotea Bancodepaticifisa triad (shown as sticks)and BL2isshown (inset).Interaction between thecatalyticresidues isshown bydashed lines. $B ,$ comparison of amino acids of different PLpros interacting with ubiquitin (orange color).Interactions of amino acids of ubiquitin such $\mathsf { A r g } ^ { 4 2 }$ $\mathsf { A i a } ^ { 4 6 }$ ,and $\mathsf { G l n } ^ { 4 9 }$ (orange sticks) with corresponding amino acidsft bindingsiterC of LprofromBVareshownattetop.AbsolutelyonservedesiduesrhighlightedithedbackgroundConservedresiduesaresoinred

MERS CoVPLpro), respectively, further illustrates that PLpros use different amino acids to tether ubiquitin and possibly peptide substrates.Alternatively,differences in substrate binding residues are likely to help occlude non-substrate ligands, increase affinity for canonical peptide ligands,and impact catalytic efficiencies.

PLpros from IBV and SARS CoV ExhibitDifferent Catalytic Efficiencies-Because superimpositionstudies revealed that the amino acids of IBVPLpro interacting with ubiquitin are different from those of SARS CoV PLpro interacting with ubiquitin, suggesting different substrate specificities,we tested the ability of IBV PLpro to degrade linear,Lys48- and $\mathrm { L y s } ^ { 6 3 }$ -linked polyubiquitin chains.Similar to SARS CoVPLpro,IBVPLpro could degrade Lys48- and $\mathrm { L y s } ^ { 6 3 }$ -linked polyubiquitin chains but not linear ubiquitin chains (Fig.4).Remarkably,but consistent with the inference gained from the structural studies that the substrate specificity of IBV PLpro could be different from that of SARS CoVPLpro, the protease from IBV showeda preference for Lys63-linked polyubiquitin chains over Lys48-linked polyubiquitin chains.Specifically,Lys63-linked tetraubiquitin could be degraded almost completely within $5 \mathrm { m i n }$ of the incubation time of the assay mixture to lower order oligomers such as triubiquitin,diubiquitin,and monoubiquitin.Within $3 0 \mathrm { m i n }$ of the reaction time,most of the oligomers were degraded to monoubiquitin. In contrast, the degradation of Lys48-linked tetraubiquitin underidentical conditions was relatively slower. At the end of 5 min of incubation time,a significant amount of substrate remained uncleaved.In fact, even after $2 4 0 ~ \mathrm { { m i n } }$ of incubation, the substrate could not be completely degraded to monoubiquitin.In addition to the substrate, tri- and diubiquitin products persisted throughout the incubation period.Thus, lower order oligomers of Lys48-linked ubiquitin seem to impose product inhibition on IBV PLpro.No cleavage of linear tetraubiquitin chains was observed even after prolonged incubation $( 4 \mathrm { h } )$ under identical conditions of the assay.Previously, SARS CoV PLpro has been shown to degrade Lys48-linked polyubiquitin chains more rapidly than Lys63-linked polyubiquitin chains.Thus,results of enzymatic assays are consistent with inferences gained fromstructural studies and confirm that PLpros from IBV and SARS CoVs have different substrate specificities.

PLpro from IBV has been shown to cleave peptides at KAGG motifs(11,12),whereas SARS CoVhas been shown to cleave at LXGG ( $X$ represents any amino acid) sites (32,33).We tested the catalytic ability of IBV PLpro to cleave at these motifs using peptide substrates KAGG-AMC,KKAG-AMC,and LRGGAMC. Intriguingly, IBV PLpro could not cleave KAGG-AMC, suggesting that this peptide substrate is not a close mimic of the physiological substrate.Either a larger peptide substrate is essential for correctly orienting the site of cleavage to release AMC,orat least a larger side chain like lysine or arginine is required at the P3 position.Previous studies have shown that IBV PLpro can cleave polypeptide with an alanine at the P2 position instead of a glycine (11).Inagreement with this observation,we could monitor activity of IBVPLpro using KKAGAMC as the peptide substrate.The PLpro from IBV could cleave KKAG-AMC and LRGG-AMC.However, the catalytic efficiency of thePLpro toward KKAG-AMC was5 times higher than that for LRGG-AMC (Table 2).We next compared the catalytic efficiencies of PLpros from IBV and SARS CoVs against their preferred peptide substrates.IBVPLpro exhibited a catalytic efficiency of $3 . 2 3 \pm 0 . 7 2 { \mathrm { \ s } } ^ { - 1 } { \mathrm { m } } { \mathrm { m } } ^ { - 1 }$ against KKAGAMC,which was comparable with that of the PLpro from SARS CoVthat showed a catalytic efficiency of $4 . 4 1 \pm 0 . 5 0 \mathrm { s } ^ { - 1 } \mathrm { m } \mathrm { M } ^ { - 1 }$ against LRGG-AMC when assayed under identical conditions (Table 2).Thus,the catalytic efficiencies of the PLpros from IBVand SARS CoVsare similaragainst their preferred peptide substrates.Because the peptide substrates preferred by both the PLpros are markedly different, the results further support the fact that PLpros from different sources differ in substrate specificities.

![](images/3766544513045ed6e1392e9ad3611d90f2312c288039bb5097797718067537a9.jpg)  
FIGURE4.Cleavage of polyubiquitin chains by PLpro from IBV.A,picture of the gel from SDS-PAGE analysis of reaction products of Lys48-linked tetraubiquitin chains treated with IBVPLpro at different time intervals.B,analysis similar to that in Aexcept the substrate used was $\mathsf { L y s } ^ { 6 3 }$ -linked tetraubiquitin. C, SDS-PAGE gel picture of reaction products of linear ubiquitin chains treated with PLpro from IBV.The protease could not cleave linear ubiquitin under reaction conditions similar to those used for cleaving $\mathsf { L y s } ^ { 4 8 } \mathsf { . }$ andLys63_ linked tetraubiquitin.M,molecular mass markers;',minutes.

# DISCUSSION

In this study,we show that the PLpro from IBV can degrade Lys48-and Lys63-linked polyubiquitin chains completely to monoubiquitin. The PLpro could not cleave ubiquitin molecules from linear polyubiquitin chains even after prolonged

# Structure ofPapain-likeProtease from IBV

incubation.Furthermore, the PLpro fromIBV could cleave two different peptide substrates,KKAG-AMC and LRGG-AMC. The nature of the P3 and P4 amino acids of the peptide substrates KKAG-AMC and LRGG-AMC cleaved by IBVPLpro is significantly different; in particular, the amino acid occupying the P4 position of the substrate that is buried in the protease during catalysis is very different in both peptide substrates. Despite this, the PLpro from IBV can cleave both the peptides, conceivably with different efficiencies.Thus,the active site pocket offers some promiscuity but at the cost of catalytic efficiency.The peptide substrate LRGG-AMC was degraded less efficiently when compared with KKAG-AMC.Examination of the structure ofIBVPLpro with one molecule ofubiquitin from the structure of SARS CoVPLpro modeled in the active site by superimposition reveals that substrate selection is probably governed by three regions:1) the region around BL2 that positions and orients the diglycine region (P1-P2 residues) of substrates for proteolysis,2) the region between the finger and palm domain of the PLpro that binds the $\beta$ -barrelofubiquitin or the viral peptide substrate,and 3) the region behind BL2 that holds the part of the peptide downstream of the cleavage site. The diglycine binding region of PLpros from IBV and SARS CoVs is quite similar, suggesting a shared mechanism for optimally orienting the peptidebond and itscleavage.In SARS CoV PLpro,P1( $\mathrm { G l y } ^ { 7 6 }$ of ubiquitin) forms ahydrogen bonding interaction with $\mathrm { G l y ^ { 2 7 2 } }$ . In additional, the C-terminal carboxylate group of $\mathrm { G l y } ^ { 7 6 }$ interacts with $\mathrm { T r p } ^ { 1 0 7 }$ $\mathrm { C y s ^ { 1 1 2 } }$ ,and Tyr113 (16,19). In IBV PLpro, except for $\mathrm { T y r } ^ { 1 1 3 }$ that is replaced bya Trp,other spatially equivalent residues, $\mathrm { G l y ^ { 2 6 3 } }$ $\mathrm { T r p } ^ { 9 6 }$ ,and $\mathrm { C y s ^ { 1 0 1 } }$ ,respectively,are all conserved. $\mathrm { P h e } ^ { 1 5 1 }$ （20 $\left( \mathrm { L e u } ^ { 1 6 3 } \right.$ in SARS CoV PLpro), $\mathrm { A s p } ^ { 1 5 3 }$ (conserved), $\mathrm { G l y ^ { 2 6 3 } }$ (conserved), and $\mathrm { C y s } ^ { 2 6 5 }$ $( \mathrm { T y r } ^ { 2 6 6 }$ in SARS CoVPLpro) are involved in shaping the equally restricted S2 subsite,which again is specific for glycine.

TABLE2 Comparison of the kinetic parameters of PLpros from different CoVs   

<html><body><table><tr><td>PLpro</td><td>Peptide</td><td>Cleavage</td><td>kcat/Kma</td></tr><tr><td>IBV</td><td></td><td></td><td>s−1mm-1</td></tr><tr><td></td><td>Lys48-Ub4</td><td>Yes</td><td></td></tr><tr><td></td><td>Lys63-Ub</td><td>Yes</td><td></td></tr><tr><td></td><td>Linear Ub4</td><td>No</td><td></td></tr><tr><td></td><td>LRGG-AMC</td><td>Yes</td><td>0.67 ± 0.11</td></tr><tr><td></td><td>KKAG-AMC</td><td>Yes</td><td>3.23 ± 0.72</td></tr><tr><td></td><td>KAGG-AMC</td><td>No</td><td>NDb</td></tr><tr><td>SARS CoV</td><td>LRGG-AMC</td><td>Yes</td><td>4.41 ± 0.50</td></tr><tr><td>MERS CoV</td><td>LRGG-AMC</td><td>Yes</td><td>0.05 ± 0.01</td></tr></table></body></html>

号 $k _ { \mathrm { c a t } } / K _ { m }$ values shown here are apparent $k _ { \mathrm { c a t } } / K _ { m }$ valuesobtained by fittingdata to the Michaelis-Menten equation using linear regression.The active sites of the enzyme could not be saturated with substrate. b Not detected.

Compared with the S1 and S2 subsites that are mostly buried, the residue of the peptide substrate occupying the S3 subsite is partially solvent-exposed,and the side chain at this position makes hardly any contact with the PLpro.Examination of the region around the P3 of ubiquitin-bound SARS CoV PLpro shows the main-chain amide of Arg at this position forming two hydrogen bonds,one each with $\mathrm { G l y ^ { 2 7 2 } }$ and the other with the side-chain hydroxyl of $\mathrm { T y r } ^ { 2 6 5 }$ ， $\mathrm { G l y } ^ { 2 7 2 }$ is conserved in PLpro from SARS CoV as well as IBV, but $\mathrm { T y r } ^ { 2 6 5 }$ is replaced by $\mathrm { P h e } ^ { 2 5 6 }$ in IBV PLpro.More importantly, the side chain of $\mathrm { A r g } ^ { 7 4 }$ of ubiquitin (P3 site) is involved in only one intermolecular interaction with the backbone carbonyl oxygen of Leu163 (Fig.5). A smaller side chain at the P3 position of the substrate,for instance replacement of arginine with alanine as observed in the KAGG motif cleaved by IBV PLpro,could prevent steric clashes with Leu163 and facilitate easy access to catalytic residues during binding of substrate.

In contrast to the S3 site,the S4 site is buried,and the corresponding P4 residues occupying this site during proteolysis of viral peptide substrates are verydifferent in IBVand SARS CoV PLpros.Therefore,the P4 residue probably has a profound effect on the catalytic efficiencies.Although SARS CoV PLpro has a Leu at this position, the IBV PLpro has a Lys at this position.Accordingly,the amino acids forming the S4 subsite are different in both PLpros to accommodate these two very different side chains. Amino acids such as Asp165, $\mathrm { P r o } ^ { 2 4 9 }$ P $\mathrm { T y r } ^ { 2 6 5 }$ $\operatorname { T y r } ^ { 2 6 9 }$ $\mathrm { T y r } ^ { 2 7 4 }$ ,and $\mathrm { T h r ^ { 3 0 2 } }$ of SARS CoV PLpro are observed to forma pocket for entrenching $\mathrm { L e u } ^ { 7 3 }$ of ubiquitin (Fig.5A).Residues of IBV PLpro occupying equivalent positions include Asp153, $\mathrm { P r o } ^ { 2 4 1 }$ ,Phe256, $\mathrm { T h r } ^ { 2 6 0 }$ $\mathrm { C y s } ^ { 2 6 5 }$ ,and Ileu290, respectively (Fig.5B).Notably,the substitution of $\mathrm { T h r } ^ { 3 0 2 }$ with Ile is likely to obstruct the placement of the side chain of a lysine residue of theKAGG site cleaved by IBVPLpro in the pocket.To offset this steric clash,the PLpro from IBVhas BL1 located immediately adjacent to Ie29，creating a larger pocket possibly to accommodate the side chain of a P3 lysine (Fig. 5C).The equivalent region in SARS CoV PLpro is rigid and made up of long $\beta$ -strands.Thus,differences are observed in the active site pocket,especially in the S3-S4 subsite region between the two

![](images/84c12f5059434e092a499edf68621ace998733a450ef0806f97648ebd99b6d85.jpg)  
FIGURE5.PutativeoefdeoddfdsndnftAs (orange sticks) of ubiquitin tethered by SARS CoV PLpro.Amino acidsof PLpro within $4 \mathbb { A }$ of the ligand are shownas sticks.Band $C ,$ mode of binding of P1-P4 residuesfubiangtisprfedyupeimpotbitifroructeroditot Data Bank code 4MoW) on the structure of IBV PLpro (red schematic).Amino acids within $4 \mathbb { A }$ of the ligand are shown (magenta sticks).The location of the BL1-like loop and its proximity to the P4 residue are shown in C.

PLpros.These differences coupled with differences in amino acids involved in tethering the region of the substrate adjacent to the P4-P1 residues,i.e. the $\beta$ -barrelof ubiquitin during deubiquitination or the viral polypeptide adjacent to the proteolytic site during proteolytic maturation of nsps,probably give rise to differences in substrate specificities and catalytic efficiencies observed for the PLpros.

The differences in substrate specificities observed probably confer different abilities on the PLpros and consequently the virus to scuttle different aspects of the host antiviral responses. For instance, SARS CoV PLpro shows a preference for Lys48_ linked polyubiquitin over $\mathrm { L y s } ^ { 6 3 }$ -linked ubiquitin.This PLpro cleaves the Lys48-linked polyubiquitin chains of interferon regulatory transcription factor 3 and degrades the Lys48-linked polyubiquitin chains tagged on $\operatorname { I } \kappa \mathrm { B } \alpha$ that prevent its proteasomal degradation (20,21).Both these actions block the activation of host antiviral signaling pathways.In contrast,the MERS CoV PLpro has been shown to process Lys48- and Lys6³-linked polyubiquitin chains with similar rates,producing monoubiquitin species (22).This PLpro has been shown to degrade Lys63_ linked polyubiquitin chains associated with RIG-I and MDA-5, preventing activation of antiviral responses mediated by both these proteins.Thus,differences in substrate specificitiesofPLprosprobably seemto help the viruses target different aspects of host immune responses.Further studies,for example comparison of the abilities of differentPLpros in targeting the same as well as different proteins for deubiquitination and evaluation of their effect on immune responses,are likely to shed more light on the roles of different PLpros. Because PLpros modulate immune responses, they also probablyimpact the virulence of viruses.

The structure of IBV PLpro presented here provides a clear viewof the protease at the molecular level and brings out several interesting featuresof thisPLpro that need to be investigated further.In particular, it would be interesting to know the choice of ubiquitinated host factors targeted by this PLpro and the outcome of degradation of these ubiquitinated substrates. Proteomics studies have identified thousands of proteins involved in diverse signaling pathways that are modified by ubiquitin including Lys48-and Lys63-linked polyubiquitin chains that have been shown to be degraded in vitro by IBV PLpro in this study (34).In particular,activation of nuclear factor $\kappa \mathrm { B }$ transcription factor by Toll-like receptors,nucleotide oligomerization domain-like receptors,RIG-I,and stimulator of interferon genes among othersisdependent on modification of proteins participating in these antiviral signaling pathways by polyubiquitin chains (35).Perhaps IBVPLpro derailsactivation of host antiviral signaling pathways by degrading Lys48- and Lys63_ linked polyubiquitin chains associated with these proteins.In addition,ubiquitination is not limited to immune responses. Therefore,PLpros are probably capable of modulating other aspectsof the host cell regulated byubiquitination,for example the cell cycle,to assist survival of the virus that await discovery.

# REFERENCES

1.Stern,D.F.,and Sefton,B.(1984) Organization of the IBV genome.Adv. Exp.Med.Biol.173,11-23   
2.Stern,D.F.,and Sefton,B.M. (1984) Coronavirus multiplication: locations of genes for virion proteins on the avian infectious bronchitis virus genome.J. Virol. 50,22-29   
3.Cavanagh,D.(2oo7) Coronavirus avian infectious bronchitis virus.Vet. Res. 38,281-297   
4.Cavanagh,D.,Casais,R.,Armesto,M.,Hodgson,T.,Izadkhasti,S.,Davies, M.,Lin,F.,Tarpey,I.,and Britton,P. (2oo7) Manipulation of the infectious bronchitis coronavirus genome for vaccine development and analysis of the accessory proteins. Vaccine 25,5558-5562   
5.Hodgson,T., Britton,P.,and Cavanagh,D.(2006) Neither the RNA nor the proteins of open reading frames 3a and 3b of the coronavirus infectious bronchitis virus are essential for replication.J. Virol. 80,296-305   
6.Casais,R.,Davies,M.,Cavanagh,D.,and Britton,P.(2oo5) Gene 5 of the avian coronavirus infectious bronchitis virus is not essential for replication. J. Virol.79,8065-8078   
7.van Ginkel,F.W.,van Santen, V.L.,Gulley,S.L.,and Toro,H. (2008) Infectious bronchitis virus in the chicken Harderian gland and lachrymal fluid: viral load,infectivity,immune cell responses,and effects of viral immunodeficiency. Avian Dis. 52, 608-617   
8.Ng,L.F.,and Liu,D.X.(20o2) Membrane association and dimerization of a cysteine-rich,16-kilodalton polypeptide released from the C-terminal region of the coronavirus infectious bronchitis virus la polyprotein.J. Virol. 76,6257- 6267   
9.Lim,K.P.,and Liu,D.X.(1998) Characterisation of a papain-like protein ase domain encoded by ORFla of the coronavirus IBVand determination of the C-terminal cleavage site of an 87 kDa protein.Adv.Exp.Med.Biol. 440,173-184   
10.Lim,K.P.,andLiu,D.X. (1998)Characterizationof thetwooverlapping papain-like proteinase domains encoded in gene 1 of the coronavirus infectious bronchitis virus and determination ofthe C-terminal cleavage site of an 87-kDa protein.Virology 245,303-312   
11.Lim,K.P.,Ng,L.F.,andLiu,D.X. (20oo) Identification ofa novelcleavage activity of the first papain-like proteinase domain encoded by open reading frame la of the coronavirus avian infectious bronchitis virus and characterization of the cleavage products.J. Virol.74,1674-1685   
12.Ziebuhr,J., Thiel, V.,and Gorbalenya,A.E.(200l) The autocatalytic release of a putative RNA virus transcription factor from its polyprotein precursor involves two paralogous papain-like proteases that cleave the same peptide bond.J.Biol. Chem. 276,33220-33232   
13.Han,Y.S., Chang, G. G.,Juo, C.G.,Lee, H.J.,Yeh, S.H.,Hsu,J. T.,and Chen,X. (20o5) Papain-like protease 2 (PLP2) from severe acute respiratory syndrome coronavirus (SARS-CoV): expresson, purification,characterization,and inhibition.Biochemistry 44,10349-10359   
14.Mielech,A.M.,Kilianski,A.,Baez-Santos,Y.M.,Mesecar,A.D.,and Baker, S.C. (2014) MERS-CoV papain-like protease has deISGylating and deubiquitinating activities.Virology 450-451,64-70   
15.Chaudhuri,R., Tang, S., Zhao,G.,Lu,H., Case,D.A.,and Johnson, M.E. (2011) Comparison of SARS and NL63 papain-like protease binding sites and binding site dynamics: inhibitor design implications.J. Mol. Biol. 414, 272-288   
16.Ratia,K.,Kilianski,A.,Baez-Santos,Y.M.,Baker,S.C.,and Mesecar,A.(2014) Structural Basis for the Ubiquitin-Linkage Specificity and deISGylating activity of SARS-CoV papain-like protease.PLoS Pathog.10,e1004113   
17.Ratia,K.,Pegan, S.,Takayama,J.,Sleeman,K.,Coughlin,M.,Baliji, S., Chaudhuri,R.,Fu,W.,Prabhakar,B.S.,Johnson,M.E.,Baker, S.C., Ghosh,A.K.,and Mesecar,A.D. (2oo8)A noncovalent class of papain-like protease/deubiquitinase inhibitors blocks SARS virus replication.Proc. Natl. Acad. Sci. U.S.A.105,16119-16124   
18.Lei,J., Mesters,J.R.,Drosten, C.,Anemüller,S.,Ma, Q.,and Hilgenfeld,R. (2014) Crystal structure of the papain-like protease of MERS coronavirus reveals unusual, potentially druggable active-site features.Antiviral Res. 109,72-82   
19.Chou,C.Y.,Lai, H.Y.,Chen,H.Y.,Cheng,S.C.,Cheng, K. W.,and Chou, Y.W.(2014) Structural basis for catalysis and ubiquitin recognition by the severe acute respiratory syndrome coronavirus papain-like protease.Acta Crystallogr.DBiol.Crystallogr.70,572-581   
20.Chen,X., Yang,X.,Zheng,Y., Yang,Y., Xing,Y.,and Chen,Z. (2014) SARS coronavirus papain-like protease inhibits the type I interferon signaling pathway through interaction with the STING-TRAF3-TBK1 complex. Protein Cell 5,369-381   
21.Clementz,M.A.,Chen, Z.,Banach,B.S.,Wang,Y., Sun,L.,Ratia,K, Baez-Santos,Y.M.,Wang,J.,Takayama,J.,Ghosh,A.K.,Li,K.,Mesecar, A.D.,and Baker,S.C.(2o1o) Deubiquitinating and interferon antagonism activities of coronavirus papain-like proteases.J. Virol.84,4619-4629   
22.Baez-Santos,Y.M.,Mielech,A.M.,Deng,X.,Baker,S.,and Mesecar,A.D. (2014) Catalytic function and substrate specificity of the papain-like protease domain of nsp3 from the Middle East respiratory syndrome coronavirus.J. Virol.88,12511-12527   
23.Lee,H.,Cao, S.,Hevener,K.E., Truong,L.,Gatuz,J.L., Patel,K.,Ghosh, A.K.,and Johnson,M.E.(2013) Synergistic inhibitor binding to the papain-like protease of human SARS coronavirus: mechanistic and inhibitor design implications.ChemMedChem8,1361-1372   
24.Ghosh,A.K.,Takayama,J.,Rao,K.V.,Ratia,K., Chaudhuri,R.,Mulhearn, D.C.,Lee,H., Nichols,D.B.,Baliji, S., Baker,S.C., Johnson,M.E.,and Mesecar,A.D.(20lo) Severe acute respiratory syndrome coronavirus papain-like novel protease inhibitors: design,synthesis,protein-ligand xray structure and biological evaluation.J.Med. Chem. 53,4968-4979   
25.Ghosh,A.K., Takayama,J.,Aubin, Y.,Ratia,K.,Chaudhuri, R.,Baez, Y., Sleeman,K.,Coughlin,M.,Nichols,D.B.,Mulhearn,D.C.,Prabhakar, B.S.,Baker, S.C.,Johnson,M.E.,and Mesecar,A.D.(2Oo9) Structurebased design,synthesis,and biological evaluation of a series of novel and reversible inhibitors for the severe acute respiratory syndrome-coronavirus papain-like protease.J.Med.Chem. 52, 5228-5240   
26.Van Duyne,G.D., Standaert,R.F.,Karplus,P.A., Schreiber, S.L.,and Clardy,J.(1993)Atomic structures of the human immunophilin FKBP-12 complexes with FK506 and rapamycin.J. Mol. Biol. 229,105-124   
27．Minor,W.,Cymborowski,M.,Otwinowski, Z.,and Chruszcz,M.(2006) HKL-3ooo: the integration of data reduction and structure solutionfrom diffraction images to an initial model in minutes.Acta Crystallogr.D Biol. Crystallogr.62,859-866   
28.Adams,P.D.,Grosse-Kunstleve,R.W.,Hung,L.W.,Ioerger,T.R.,McCoy,A.J.,Moriarty,N.W.,Read,R.J., Sacchettini,J.C.,Sauter,N.K.,and Terwilliger,T.C.(2oo2) PHENIX: building new software forautomated crystallographic structure determination.Acta Crystallogr.D Biol. Crystallogr. 58,1948-1954   
29.Emsley,P.,and Cowtan,K.(2oo4) Coot: model-building tools for molecular graphics.Acta Crystallogr.D Biol. Crystallogr.60,2126-2132   
30.Chen,V.B.,Arendall,W.B.,3rd,Headd,J.J.,Keedy,D.A.,Immormino, R.M.,Kapral,G.J.,Murray,L.W.,Richardson,J.S.,andRichardson,D.C. (2010) MolProbity:all-atom structure validation for macromolecular crystallography.Acta Crystallogr.D Biol.Crystallogr.66,12-21   
31.DeLano，W.L.(2002）The PyMOL Molecular GraphicsSystem, Schrodinger,LLC,New York   
32.Ratia,K., Saikatendu,K.S., Santarsiero,B.D.,Barretto,N.,Baker, S.C., Stevens,R.C.,and Mesecar,A.D. (2006) Severe acute respiratory syndrome coronavirus papain-like protease: structure of a viral deubiquitinating enzyme.Proc.Natl.Acad.Sci.U.S.A.103,5717-5722   
33.Barretto,N.,Jukneliene,D.,Ratia,K., Chen,Z.,Mesecar,A.D.,and Baker, S.C.(2005) The papain-like protease of severe acute respiratory syndrome coronavirus has deubiquitinating activity.J. Virol.79,15189-15198   
34.Kim,W.,Bennett,E.J.,Huttlin,E.L.,Guo,A.,Li,J.,Possemato,A., Sowa, M.E.,Rad,R.,Rush,J., Comb,M.J.,Harper,J.W.,and Gygi, S.P.(2011) Systematic and quantitative assessment of the ubiquitin-modified proteome.Mol. Cell 44,325-340   
35.Shaw,N., Ouyang, S.,and Liu,Z.J.(2o13) Binding of bacterial secondary messenger molecule c di-GMP is a STING operation.Protein Cell 4, 117-129   
36.Bailey-Elkin,B.A., Knaap,R.C.,Johnson, G.G., Dalebout,T.J.,Ninaber, D.K.,van Kasteren,P.B.,Bredenbeek,P.J.,Snijder,E.J.,Kikkert,M.,and Mark,B.L.(2014) Crystal structure of the Middle East respiratory syndrome coronavirus (MERS-CoV) papain-like protease bound to ubiquitin facilitates targeted disruption of deubiquitinating activity to demonstrate its role in innate immune suppression.J. Biol. Chem. 289,34667-34682