# Haplotype hitchhiking promotes trait coselection in Brassica napus

Lunwen Qian1,Wei Qian² and Rod J. Snowdon1'\*

DepartmentoflantBreedingIFZResearchCentreforosytemsLandUseandutritionJustusLebigUniversityGeenny 2ColegeofAgronomyand BiotechnologySouthwest University，Chongqing，China

Received15September2015;   
revised 16 November2015;   
accepted 25 November 2015.   
\*Correspondence (Tel +49 641 9937420;   
fax $+ 4 9$ 641 9937429;email rod.   
snowdon@agrar.uni-giessen.de)

# Summary

Keywords: linkage disequilibrium,LD, haplogroups,selection,oilseed rape, breeding.

Local haplotype patterns surrounding densely spaced DNA markers with significant trait associations can reveal information on selective sweeps and genome diversity associated with important crop traits.Relationships between haplotype and phenotype diversity,coupled with analysis of gene content in conserved haplotype blocks,can provide insight into coselection for nonrelated traits.We performed genome-wide analysis of haplotypes associated with the important physiological and agronomic traits leaf chlorophylland seed glucosinolate content, respectively,in the major oilseed crop species Brassica napus.A locus on chromosome A01 showed opposite efects on leaf chlorophyll content and seed glucosinolate content,attributed to strong linkage disequilibrium (LD) between orthologues of the chlorophyllbiosynthesis genes EARLY LIGHT-INDUCED PROTEIN and CHLOROPHYLL SYNTHASE,and the glucosinolate synthesis gene ATP SULFURYLASE 1.Another conserved haplotype block,on chromosome A02,contained a number of chlorophyll-related genes in LD with orthologues of the key glucosinolate biosynthesis genes METHYLTHIOALKYMALATE SYNTHASE-LIKE 1and 3.Multigene haplogroups were found to have a significantly greater contribution to variation for chlorophyll content than haplotypes for any single gene,suggesting positive effects of additive locus accumulation. Detailed reanalysis of population substructure revealed a clade of ten related accessions exhibiting high leaf chlorophylland low seed glucosinolate content.These accessions each carried one of the above-mentioned haplotypes from A01 or AO2,generallyin combination with furtherchlorophyll-associated haplotypes from chromosomes A05 and/or C05.The phenotypic rather than pleiotropic corelations between leaf chlorophyll content index and seed GSL suggest that LD may have led to inadvertent coselection for these two traits.

# Introduction

Chlorophyll is a green photosynthetic pigment with which plant chloroplasts generate energy from light.Leaf chlorophyll content relates to photosynthetic capacity and is thus one of the important physiological traits influencing crop yield (CzyczytoMysza et al.,2013;Wang et al.,2008).On the other hand, the presence of chlorophyll in mature seeds can be an undesirable trait that can affect seed maturation,seed oil quality,meal quality and germination (Delmas et al.，2013). Chlorophyll pigments remaining in processed vegetable oils are also associated with increased oxidation causing rancidity (Tautorus and Low,1993) and difficulties in hydrogenation (Abraham and Deman,1986). Recent studies suggested a previously unknown relationship between chlorophyll content and metabolism of glucosinolates,a class of sulphurous secondary metabolites expressed in vegetative and generative tissues throughout almost all plants of the Brassicales.Forexample,a total of 11 light-harvesting chlorophyll (LHC)protein complex proteins (including eight LHCB and three LHCA proteins) were identified to be down-regulated in RNAi lines that suppressed molecular networks controlling glucosinolate metabolism in Arabidopsis (Chen et al.,2012).Yang and Zhu (2009) reported apotential negative correlationbetween chlorophyll and glucosinolate content under abiotic stresses in cabbage plants.These results suggest potential genetic and/or metabolic associations between chlorophyll content and glucosinolate metabolism in Brassica crops.A molecular basis for such a pleiotropic relationship might be associated with the role of plastids in sulphate reduction and cysteine/methionine synthesis inthe chloroplasts，providing sulphuric amino acids that are exported across the chloroplast membrane and play an important roleas precursors for glucosinolate synthesis (Takahashi et al., 2011).Such a relationship is potentially relevant both evolutionarilyand agronomically.Leaf glucosinolates have demonstrated positive nutritional value in vegetable Brassica crops (Murillo and Mehta,2oo1) and playan important role,either antagonistically or mutualistically,in plant interactions with insect pests.On the other hand，glucosinolatesare highlyundesirable in Brassica oilseed meals fed to livestock (Friedt and Snowdon，201O).The huge global importance of oilseed rape and canola (B.napus), the world's second most important oilseed crop,imparts enormous agroeconomic relevance on these compounds.

Genome-wide association studies (GWAS） examine cotransmission of phenotypes with genetic markers,normally based on linkage disequilibrium (LD) analysis in genetically diverse populations using panels of markers spanning the entire genome at high density. Besides providing high mapping resolution by incorporating historical recombination events，LD analyses can also provide important insight into the history of both natural and artificial selection (breeding)and give valuable guidance to breeders seeking to diversify crop gene pools.

Natural and artificial selection can cause conservation of haplotype blocks，comprising specific combinations of nucleotides on the same chromosome，in genome regions carrying genes under positive or negative selection. Haplotypes can thereforeprovidemore information than any singleSNP regarding the complex relationship between DNA variation and quantitative phenotypes (Stephens et al.，2oo1). Elucidatingthe evolutionary relationships among local haplotypes can further improve the detection power of GWAS scans (Buntjer et al.，2oo5). For example,detailed analysis of LD surrounding majorQTL revealed strong signatures ofartificial selection associated with important traits in different breeding pools of rapeseed and bread wheat (Qian et al.，2014;Voss-Fels and Snowdon，2015;Voss-Fels et al.，2015). In such cases，haplotypes reveal the extent to which genetic variation in a given chromosome region is described by clustering markers.Comparing haplotype diversity can help to understand the effects of natural and artificial selection on genome-scale and single-gene variation，as shown recently in grapevine (Fernandez et al., 2014)，maize (Yang etal.，2013)and Arabidposis (Li et al., 2014).New high-density genome screening tools provide an unprecedented level of insight into local LD patterns in even complex crop genomes (Edwards et al.，2013;Voss-Fels and Snowdon，2015)．For example，strongly selected haplotype patterns detected in high-density population genomic studies havebeen associated with domestication，adaptation and breedingin sorghum(Mace etal.，2013)，rapeseed (Qian et al.，2014) and bread wheat (Voss-Fels etal.，2015)，respectively.

Evolutionary selection pressures frequentlyact on entire pathWays or their functional subnetworks.Multiple interacting genes may change in the same fitness direction,at a similar evolutionary rate and across the same timescale，to achieve a common phenotypic outcome.Associations in evolutionary patterns may therefore simply reflect parallel selection of different genes in the same pathway with shared functionality.On the other hand, artificial selection in plant breeding targets recombinationssurrounding chromosome regions that carry variants conferring traits ofagricultural or economic interest.Different traits that interact via molecular networks may be unintentionally coselected due to pleiotropy,whereas traits controlled by distinct networks may be coselected via LD between functionally independent genes. Haplotype blocks can provide powerful insight into the causes of correlations between different, quantitative trait phenotypes and their associations with responsible genes within haplotype regions.

In this study,we used a high-density SNP genotyping array to identify haplotype blocks associated with leaf chlorophyll content index(CCl) and seed glucosinolate content in a diverse B.napus population.Gene content within haplotype blocks for these two traits suggests they have been coselected during breeding of high-yielding，high-quality，modern oilseed rape cultivars.Our resultssuggest theirinterrelationshipin B.napus canbe explained by hitchhiking selection due to LD between functionally nonrelated genes.

# Results

# Variation and correlations for leaf chlorophyll and seed glucosinolate content

Broad variation was observed in leaf CCl across the diversity panel，in different field and glasshouse environments，and across different plant developmental stages.Highly significant $\mathit { \ ' } P < 0 . 0 0 1$ ）positive correlations were seen between different developmental stages in the same location (Figure 1).Weaker butneverthelesssignificantcorrelationsweremeasured between field and glasshouse，while leaf CCl at bolting stage in the glasshouse was not significantly correlated with bolting stage，flowering stage and mature period in the field.A low heritability of $H ^ { 2 } = 0 . 2 4$ was calculated for leaf CCl,reflecting the strong $\mathsf { G } ^ { * } \mathsf { E }$ interaction (Table S1).As expected from previous findings，seed GSL was highly heritable $( H ^ { 2 } = 0 . 8 6 )$ (TableS1).Correspondingly，highlysignificantpositive correlations $' P < 0 . 0 0 1$ ）were found among the GSL values from the diversity panel among the three different environments (Figure 1).

# Homoeologous haplotype blocks containing chlorophyll-related genes associated with leaf CCl

Manhattan plots and quantile-quantile plots describing significant SNP associations for leaf CCl in glasshouse and field experiments are shown in Figure S1.A total of 35 and $3 2 ~ \mathsf { S N P S }$ distributed throughout the genome were detected with the significance threshold of $- \mathsf { l o g } 1 0 ^ { ( P ) } = 4$ using the glasshouse and field data，respectively.Associations to CCl that could be corroboratedinmultipleenvironmentsandtissueswere selected as candidate loci for further investigation.Candidate regions containing SNPs associated with leaf CCl were investigated at high resolution by assayinghaplotype blocks $\begin{array} { r l } { ( r ^ { 2 } } & { { } > 0 . 6 5 ) } \end{array}$ inflanking chromosome segments. Detailsof SNPsand candidate genes in haplotype blocks with significant associations to leaf CCl are provided in Figure S2 and Table S2.

Two SNPs (Bn-A05-p19777231 and Bn-A05-p19777547 with $P = 8 . 6 1 \times 1 0 ^ { - 7 }$ and $8 . 5 9 \times 1 0 ^ { - 5 }$ ,respectively)with significant associations to CCl were located in a $1 1 3 \mathrm { ~ k b }$ haplotype region on chrom0s0me A05 (p0sition 17 873 133-17 986 390 bp; $r ^ { 2 } = 0 . 7 4 )$ (Figure2；Table S3).As shown in Figure2，comparativeanalysis via synteny alignments revealed that this region is homologous with a $6 9 2 ~ \mathsf { k b }$ haplotype block on chromosome C05 (p0sition 36 453 492-36 605 598 bp; $r ^ { 2 } = 0 . 6 8 )$ thatalso showed significant associations to leaf CCl (Table S3).Both homologous regions contain multiple B.napus orthologues of theArabidopsischloroplastmembranegenePALMITOYLMONOGALACTOSYLDIACYLGLYCEROLDELTA-7 DESATURASE (FAD5；BnaA05g23670D，BnaA05g23680D，BnaA05g23690D, BnaC05g37420D， BnaC05g37450D andBnaC05g37460D), along with two copies of the photosynthesis gene POSTILLUMINATION CHLOROPHYLLFLUORESCENCE INCREASEPROTEIN (PIFl;BnaA05g23700D and BnaC05g37470D) (Table S3). Three and nine haplogroups were observed for the Ao5 and C05haplotyperegions，respectively.Twohaplogroups, A05_Hap2 and Co5_Hap5，were found to exhibit higher CCI than the remaining two and eight haplogroups on chromosomes Ao5 and Co5，respectively (t-test and mean values; Figure S3;Table S3).

![](images/d2ef8b06fad5fbdedd0dabb0317e66998f71cefc98ef68e14418107849c3e9e6.jpg)  
Figure1oeafitdstrtisfooltetditesdci ss_GH:chteiiasoepts);tidias experits);d)e choop);) content $^ { \prime \star } P \leq 0 . 0 5$ $^ { \star \star } P \leq 0 . 0 1$ 1 $^ { \star \star \star P } \leq 0 . 0 0 1 .$ ）

# Chlorophyll-associated haplotypes carry chloroplast membrane protein genes

Three SNPs (Bn-A05-p9118240,Bn-A05-p9079762 and Bn-A05- p9078313)in a 147 kb haplotype block $( r ^ { 2 } = 0 . 6 6 )$ ，from 12 869 710 to 13 017 024 bp on chrom0some A02，showed significant associations $\left( P = 2 . 0 8 8 \times 1 0 ^ { - 5 } . \right.$ ， $3 . 6 3 \times 1 0 ^ { - 5 }$ and $5 . 1 7 \times 1 0 ^ { - 5 }$ ，respectively) to leaf CCl (Fig 3a;Table S4).The haplotype block containing this gene also contains B.napus orthologues of the Arabidopsis genes TRANSLOCON AT THE OUTERENVELOPEMEMBRANEOFCHLOROPLASTS159(TOC159; BnaA02g20610D)and AT4G02530 (BnaA02g20650D).TOC159 encodes an essential component of the TOC-protein complex, responsible for recognition and translocation of photosynthetically active proteins through the chloroplast envelope membrane (Bauer et al.，2000)，while BnaA02g20650D encodesa chloroplast thylakoid lumen protein involved in photosynthesisand chlorophyll biosynthesis (Ferro et al.,2010;Friso et al.,2004).By comparing leaf CCl phenotypes of the haplogroups for this haplotype block,we found that the haplogroup Ao2_Hap1 was associated with higher chlorophyll levels than the other five haplogroups (t-test and mean value; Figure 3b; Table S4). A02_Hap1 differs strongly differs from all other haplogroups except for AO2-Hap4,which has differences at only the first two bases.Genotypes carrying AO2-Hap4 also have higher median CCI in later developmental stages (Figure 3,Table S4).

In addition，six SNPs in this haplotype region were located within the gene BnaA02g20650D.Two of these SNPs (Bn-A05- p9079762 and Bn-A05-p9078313)， located within a $2 1 \times 6$ haplotypeblock inside intron 3ofBnaA02g20650D，exhibited significant associations with leaf CCl (Figure $4 a$ ：TableS5).We used t-test and mean values to compare phenotype values for threehaplogroups identified in the 2 kb haplotype block.

Haplogroup AO2_Hap1_1 was found to have higher leaf CCl than other two haplogroups (Figure 4b;Table S5).Comparison of A02_Hap1_1 with A02_Hap1 showed that A02_Hap1 (including both BnaA02g20650D and BnaA02g20610D) has higher leaf CCl than A02_Hap1_1 (comprising only BnaA02g20650D) (Figure 5). Thisresult suggests that the multigene haplogroup hasa significantly greater contributionto higher leaf chlorophyll content than the single gene haplogroup.

# Haplotype conservation associates with coselection for leaf CCl and seed GSL

Inanumber of CCl-associated haplotype blocks,we observed strong LD between genes involved in chlorophyll biosynthesis, photosynthesis or chloroplast membrane fatty acid synthesis,and further genes implicated in GSL synthesis.We therefore mapped SNP-trait associations for seed GSL,using phenotype data from three different environments,to test whether the same haplotype block regions are also associated with GSL.A genome-wide significance threshold of $- { 1 0 9 } ^ { 1 0 ( P ) } = 4 . 4$ was applied to determine SNPs with significant associations to GSL.

Conserved LD blocks containing significant GWAS hits for both seed GSL content and leaf chlorophyll content in chromosome A01are shown in Figure 6 and Table S6.The SNP markers BnA01-p12454306 $( P = 6 . 0 0 \times 1 0 ^ { - 5 } )$ and Bn-A01-p12314813 $( P = 4 . 5 7 \times 1 0 ^ { - 6 } )$ were significantly correlated to CCl and GSL, respectively (Figure 6a;Table S6).These two markers share conserved LD withina $3 3 8 ~ { \textk } { \mathrm b }$ haplotype block $( r ^ { 2 } = 0 . 6 7 )$ containing B.napus orthologues of three chloroplast-associated Arabidopsis genes with functional annotations associated to chlorophyll and glucosinolates,respectively (Figure 6a; Table S6). ThegeNeSEARLY LIGHT-INDUCEDPROTEIN(ELIP2; BnaA01g19110D） and CHLOROPHYLL SYNTHASE(CHLG: BnaA01g19280D）are involved in photosynthesis via regulation of chlorophyll biosynthesis,Whereas ATP SULFURYLASE1 (APS1; BnaA01g19120D） encodes the first enzyme in the sulphate assimilation pathway and therefore has a potential direct impact onsynthesis of sulphuric glucosinolates (Table S6)．For this haplotype block,the haplogroup AO1_Hap5 showed higher leaf CCl,especially at seedling and bolting stages in the glasshouse experiments ${ ' P } < 0 . 0 5 / 0 . 0 1 _ { . }$ ),whereasA01_Hap1 showed lower CCI than the other four haplogroups( $t$ -test and mean value; Figure 6b;Table S6). Conversely,in all three test environments, haplogroup A01_Hap5 showed significantly lower GSL and haplogroup AO1_Hap1 significantly higher GSL than the other four haplogroups (t-test and mean value; Figure 6b; Table S6). In almost all cases, the CCl in the glasshouse experiments showed a negative correlation between leaf CCl and seed GSL between haplogroups A01_Hap1 and A01_Hap5，suggesting that this haplotype may be associated with inadvertent coselection for high leaf chlorophyll in breeding materials with low seed GSL.

![](images/0fdc8dec405a5b683f09c744885285d150befeeed0e2d5670176c5ef28bfa254.jpg)  
Figure 2 Association mapping for leaf chlorophyll content index at bolting stage in glasshouse (bs_GH) and field (bs_F) to homoeologous haplotype regions on of Brassica napus chromosomes A05 (17 873 133- 17 986 390 bp)and C05 (36 453 492-37 145 290 bp).The heatmaps span the SNP markers that show linkage disequilibrium with the most strongly associated SNPs.Three B.napus orthologues of the gene FAD5 and one of PlF/ are located in each of the homoeologous segments (indicated by the red bar).

# Introgressions from winter oilseed rape contribute additively to chlorophyll content

On chromosomes A01,A02,A05 and C05,the high-chlorophyl haplogroups A01_Hap5，A02_Hap1，A05_Hap2 and C05_Hap5 were found in 22 of the 203 accessions (Figures 6b,3b and S3). Haplotype network analysis across these four haplotype regions showed that A01_Hap5,A02_Hap1 and C05_Hap5 correspond to accessions distributed throughout the subpopulations Q1 and 'mixed'，respectively (Figure 7a，b and d)，whileaccessions carrying Ao5_Hap2 were found only in Q1 (Figure 7c).Seven of the 22 accessions carried more than one of the high-chlorophyl haplogroups,and a further 7 also exhibited elevated leaf CCl (Table S7)．11 of these 14 high-chlorophyll accessions were assigned to subpopulation Q1，which has been shown to be associated with a strong winter rapeseed genetic background Qian et al. (2014,Table S7). Collectively，these results suggest that multiple introgressions from winter rapeseed may have elevated chlorophyll content in this group of accessions.

# Haplogroups have additive effects on chlorophyll content

Among the 14 accessions with higher leaf CCl related to haplogroups A01_Hap5,A02_Hap1,A05_Hap2 and C05_Hap5, seven carried combinations of two or three of these haplogroups (designated group A in Table S8).The remaining seven accessions were (designated group B in Table S8) each carry only one of the haplogroups. Comparative phenotype analysesshowed the accessions with multiple haplogroups (group A) have higher leaf CCI than those with only one haplogroup (B group; Figure S4; Table S8).This indicates putative additive effects of the selected haplogroups on leaf chlorophyll content.

# Higher chlorophyll content accessions relate to lower GSL

The reanalysis of detailed substructure in subpopulations Q1 and 'mixed'by PCA and UPGMA revealed three clear subgroups (clades Q1_1，Q1_2 and Q1_3)，comprising 69，16 and 59 accessions,respectively (Figure 8a,b;Table S7).The results of the UPGMA tree analysis corresponded with around $84 \%$ similarity to the PCA.Haplotype A01_Hap5 (associated with low seed GSL and high leaf CCl)was unique to four accessions distributed across the clade Q1_2.This clade (Figure 8b) also included six accessions with higher leaf CCl associated with the haplotypes A02_Hap1,A05_Hap2 and C05_Hap5,respectively.

Extending the haplotype block Ao2_Hap1by only oneadditional upstream SNP,corresponding to a slightly relaxed LD threshold （2 $( r ^ { 2 } = 0 . 6 2$ ；extended haplotype block from 12 869 710 to

![](images/b7a472a42cbdf89442966bf3fecbdb9dab8b0abf2d55ed7b1832bf5039d32afd.jpg)  
Figure3ssoatioainfoeafcopttiexomosesenteadceis. plotshowaaplotatldifadattodf wide significance at a $P$ value of $1 . 0 \times 1 0 ^ { - 4 }$ .The heatmaps span the SNP markers that showlinkage disequilibrium (LD)with the most strongly associated SNPs.PositionsofBrasicanapusorthologuesoftheArabidopsisgnesTOC59(BnaAg0610D)andachoroplastthylakoidlumenprotein (BnaA02g2065DaremarkdTesixSNslabeledinedarelocatedwithinthgenBnaAg265D.(b)BoxplotssowingleafCvauforsix haplogroups with frequency ${ > } 0 . 0 1$ .Symbols show significant differences of haplogroups compared with AO2_Hap1: $^ { \star } P \leq 0 . 0 5$ $^ { \star \star } P \leq 0 . 0 1$ 1 $^ { \star \star \star P } \leq 0 . 0 0 1$ ；Red triangles:not significant $( P > 0 . 0 5 )$ 一

![](images/0383fc834bbdbd664b59b0addb4304c697eca67e7a554c3dda7b2de471fa75f1.jpg)  
Figure 4 (a)The SNPs Bn-A05-p9079762 and Bn-A05-p9078313,located in intron 3 of BnaA02g20650D within a $2 ~ \mathsf { k b }$ haplotype block,exhibited significantasscatiosithleafcloroyllontentindex(C).(b)BoxplotsshowingleafCvauesforteeaploroupsithfrequency ${ > } 0 . 0 1$ within the BnaA02g20650D gene-haplotype region.Symbols showsignificantdiferencesof haplogroupscompared withA02_Hap1_1: $^ { \star } P \leq 0 . 0 5$ $^ { \star \star } P \leq 0 . 0 1$ 1 $^ { \star \star \star P } \leq 0 . 0 0 1$ ; Red triangles: not significant $( P > 0 . 0 5 )$

131667 O84 bp),resulted in detection of significant associations with seed GSL(Figure S6;Table S9).As was the case for AO1_Hap5, haplotypes associated with lowest GSL overlapped with the haplotypes for high CCl.Within the extended A02_GSL_Hap1 haplotype，we located B.napus orthologues of the Arabidopsis glucosinolatebiosynthesisgenes METHYLTHIOALKYMALATE

SYNTHASE-LIKE 3 (MAM3； BnaA02g20830D)and MAM1 (BnaA02g20840D)． In Arabidopsis，MAM1 and MAM3 play important roles in the biosynthesis of aliphatic glucosinolates (Kroymann et al.,2001;Field et al.,2004; Textor et al.,2007).The very close proximity of these genes (approximately $1 0 0 ~ \mathsf { k b } )$ to the CCl-associated haplotype block confirms the presence of strong LD between putative causal loci for GSLvariation and loci with strong effects on CCl.Four accessions belonging to clades Q1_2 and Q1_3 carried the low-GSL haplotype associated with A02_GSL_Hap1, but not A01_Hap5.

![](images/4492eff15ee90b38cbcc72037d7bfdaa9b49d7d8511baefc8b1281390db76114.jpg)  
Figure 5 Boxplots showing phenotypic values for leaf chlorophyl content index (CCl) related to two haplogroups. Comparison between the haplogroups show higher leaf CCl in A02_Hap1 than A02_Hap1_1 （204 $( { } ^ { \star } P \leq 0 . 0 5$ $^ { \star \star } P \leq 0 . 0 1$ ）

A similar situation to that described for Ao2_Hap1 was also observed for the haplotype A05_Hap2.Within 200 kb from the $1 1 3 \mathrm { ~ k b }$ CCl-associated haplotype block $( r ^ { 2 } = 0 . 7 4 )$ ，we found additional genes with a putative function in GSL biosynthesis.This extended haplotype block $( r ^ { 2 } = 0 . 4 1$ )also showed associations to seed GSL content.

Interestingly，phenotypic comparison among the three clades Q1_1，Q1_2and Q1_3 revealed higher leaf CCland lower seed GSL in Q1_2 than in the other two clades (t-test and mean value; Figure S5; Table S7).This suggests that coselection for these two traits have occurred particularlywithin clade Q1_2.All individuals belonging to clade Q1_2 carried either A01_Hap5 or A02_Hap1, eitheralone or in combination with Ao5_Hap2.

# Discussion

Selection for specific agronomic traits during plant domestication and breeding has strong influences on the genetic diversity and population structure within available gene pools for further crop improvement. High-density genotyping tools today provide ameans for unprecedented insight into patterns of diversity associated with breeding in major crop species (Qian et al., 2014;Snowdon et al.，2015;Voss-Fels et al.，2015)，enabling identification of loci under strong selection and potentially allowing islands of depleted diversity to be addressed by targeted，marker-assisted introgressions without compromising desirable adaptation，yield or quality traits(Voss-Felsand Snowdon，2015).

Besides their negative influence on genetic diversity，strong signatures of selection associated with key traits can also cause coselection of loci with undesirable effects,resulting in what is commonly known as linkage drag.In this study,we present an interesting example in oilseed rape for coselection of increased leaf chlorophyll content along with reduced seed glucosinolate content， two traits with no immediately obvious biological relationship.We demonstratethat introgressionsbetween ecogeographicallydistinctgenepoolsresulted inindirect selection of plants with elevated levels of leaf chlorophyll, due to coselection of beneficial haplotypes at four independent haplotype blocks.Within one of these haplotype blocks,a gene involved in chlorophyll synthesis showed almost complete LD witha gene associated with seed glucosinolate content.The relationship between the observed haplotypes and the respective phenotypic behaviour suggests that introgression of this locus during breeding，to reduce seed glucosinolate levels in nutritionallyvaluablevarieties， causeshitchhikingselection resulting in increased chlorophyll content.Elevated chlorophyll may result in improved photosynthetic performance.While this does not necessarily relate per se to improved agronomic performance，stay-greentraitsassociated withincreased chlorophyll production，or suppression of chlorophyll degradation，are implicated inimprovedadaptation ofcropsto abiotic stresses like water or nutrient deficiency (Thomas and Ougham，2014).

In this study,a total of nine haplotype blocks were found to harbour significant associations $( r ^ { 2 } \geq 0 . 6 5 )$ with leaf CCl.Within these regions,we found eighteen genes implicated in chlorophyll synthesis or catabolism，respectively.These included six orthologues of FAD5，which was found in Arabidopsis to influence chlorophyll biosynthesis (Heyndrickx and Vandepoele, 2012）and restore leaf chlorophyll content (Heilmann et al., 2004),along with two orthologues of the gene PIFl,which has anindirect effect on photosynthesis (Gotoh et al.，201O).The observationsofhomoeologous， trait-associatedhaplotypes, carrying multiple duplicated genes，provide a further example forgene amplification by polyploidization． In the complex allopolyploid crop B.napus，this phenomenon has been shown to influence numerous agronomically important traits (Chalhoub et al., 2014).

Additional chlorophyll-associated haplotypes carried B.napus orthologues of the genes ELiP2 and CHLG,which in Arabidopsis aredirectly involved in chlorophyllbiosynthesis (Oster and Rüdiger，1998;Tzvetkova-Chevolleau et al.，2007)，and the chloroplast preprotein transporter protein gene TOC159.The latter encodes an essential component of the chloroplast assemblymechanism，which acts as a selective import receptor for preproteins required in chloroplast development (Smith et al., 2004).Collectively,the genetic associations we found,involving multiple loci carrying these very different contributors to the photosynthesis apparatus,suggest a broad genetic variation for factors involved in photosynthesis in crop plants.Given the key roleof photosynthetic activity in ecogeographical and stress adaptation,it is interesting that multiple loci with positive additive influences on chlorophyll content are coselected in introgressions between different breeding pools in this major crop.

The close genetic relationship among the clade of accessions with simultaneously elevated leaf CCl and low seed GSL suggests that this clade may be the product of artificial selection from a common genetic background.Of the four CCl-associated haplotypes，two (on chromosomes A01 and A02） carry well-known glucosinolate biosynthesis genes either within or directly adjacent to the LD block responsible for increase chlorophyll concentration.The third haplotype block,on chromosome A05,is closely linked to genes that have a putative function in GSL synthesis, while its homoeologous locus on chromosome Co5 can be expected to carry a corresponding repertoire of homologous genes. Collectively, these examples represent an accumulation of experimental evidence (Ao2_Hap5 and A02_Hap1) and putative support (Ao5_Hap2 and Co5_Hap5) fora general genetic linkage between loci involved in expression of seed GSL and leaf chlorophyll content in B.napus.Leaf CCl and seed GSL exhibit significant associations to different SNP loci that in turn do not appear to show pleiotropic associations to the different traits.The presence of highly plausible positional and functional candidate genes for both traits,and their strong LD to most of the traitassociated SNPs，provides additional support for the hypothesis that the phenotypic associations we observed are caused by linkage rather than pleiotropy.

![](images/7affd67a8e901a7169c0a7a72d643b9edc193d574c00d5befa61644b1256fe0a.jpg)  
Figure6Associationmappingforlefcorophyllontntidex(CC)ndsedGSonomsome1Greeotsepresentaplotyeegio (364534toaiddGsivbtalda wide significance at a $P$ value of $1 . 0 \times 1 0 ^ { - 4 }$ and $4 . 5 \times 1 0 ^ { - 5 }$ for CCl and $\mathsf { G S L } ,$ respectively.The SNPs with highest p values for each trait were used to defineaaplotiioitaisolosftbss chlorophyll (ELIP2 and $C H L G )$ and one related to GSL(APS1).(b) Five haplogroups with frequency ${ > } 0 . 0 1$ were found in this haplotype region. Boxplots show thatA0Hap5ashighereafCandoweredGStantheotherfouralogros.ymbolsshosinifantdiferencesofapogropsad with A01_Hap5: $^ { \star } P \leq 0 . 0 5$ 5 ${ \star \star } ; P \leq 0 . 0 1$ 1 $^ { \star \star \star P } \leq 0 . 0 0 1$ ； Red triangles:not significant $( P > 0 . 0 5 ^ { \prime }$ 一

The genetic determinants of these two traits appear to differ among the three subclades we identified,however,with overlapping associations among some loci but not others.This suggests that the apparent coselection of chlorophyll content,which in contrast to seed GSL,is not a breeding target in rapeseed,most likely occurred inadvertently (through linkage to loci conferring low seed GSL) rather than intentionally.As such,this represents an interesting example for local enrichment of a trait with specific adaptation potential as an indirect consequence of intensive breeding for seed quality characters.

It is likely that the expression of many genes is jointly controlled by the actions of multiple regulatory alleles.Despite this,GWAS studies rarely consider the possibility that，at a given locus, multiple genesmay impacta phenotype by interactions between more than one regulatory allele in across a gene-haplotype region (Corradin et al.,2014).Particularly for complex,multigenic traits such as chlorophyll content，a spatial proximity of interacting genes can potentially facilitate coordinated expression in certain tissues,developmental timepoints or in response to environmental stimuli. This may help explain why multigenic haplogroups associate more strongly with CCl phenotypes than single-gene haplotypes.The results of the present study underline the benefit of combining haplotype diversity analysis with GWAS studies to dissect additive effects of quantitative trait loci in crops and understand theirunderlying biology (Buntjeret al.， 2005). Detailed investigations of trait relationships at the local haplotype level,using high-density SNP markers in large populations,also provide plant breeders with a means to distinguish between genetic and pleiotropic trait correlations.

![](images/6c4c79bbb04f49b0d7c4cb9f9c79755cb333f44949cc05e5e11e2dd1479a9c0f.jpg)  
Figure 7 Haplotype networks in the four selected haplotype regions on chromosomes (a) A01 (10 493 351-10 830 942 bp),(b)A02 (12 869 710-13 017 024 bp), (c) A05 (17 873 133-17 986 390 bp) and (d) C05 (36 453 492-37 145 290 bp). Each circle represents a haplogroup,and the size of the circle is proportional to the number of lines within the haplogroup.Colours represent four different subgroups.The arrows indicate the four haplogroups from the Q1 and mixed subpopulations,respectively,which consistently exhibit higher leaf chlorophyll content index.

# Materialsandmethods

# Plant materials, genotype and phenotype data

A diversity panel of 2o3 homozygous B.napus inbred lines was constructed to broadly represent variability in Chinese semiwinter rapeseed,an intermediate form of oilseed B.napus that isbroadly grown in China and also commonly used to enrich gene pools of European winter oilseed rape and Australian or North American spring canola. The plant population (Table S1) and high-density SNP data,generated by genotyping with the Brassica 6Ok SNP Ilumina consortium genotyping array (Illu mina，San Diego,CA)，were described in detail by Qian et al. (2014).

The association panel was evaluated in separate glasshouse and field experiments.A CClwas calculated based onabsorbance measurements at 653 and 931 nm with a CCM-200 chlorophyll content metre (Opti-Sciences, Inc.，Hudson,NH).Measurements of leaf CCl were performed at seedling and bolting stage in the glasshouse experiment in 2012.At each developmental stage two independent measurements were taken from each side of a single young leaf on three individual plants per accession.In the field，the accessions were sown in single rows with two replications.Chlorophyll content index measurementswere performed on five plants per accession per replication at seedling stage，bolting stage，flowering stage and maturity，using the same measurement procedure as in the glasshouse experiment Additional field trials for seed quality analysis were performed at theexperimental farm ofSouthwestUniversity inBeibei, Chongqing,China,in 2012 and 2013.Glucosinolate content in harvested，fully mature seeds from all 203 accessions was measured by near-infrared spectroscopy on seeds grown from all three field environments,recording mean values in glucosinolate $( \mu \mathsf { m o l } )$ per seed dry weight (g) from at least two technical and two biological replicates per accession and environment.

![](images/9c5300021bbc5fe46c892abebc9d8975eb25bcf0af20d3e0aeef67e15a38105a.jpg)  
Figure 8 One hundred and forty-four accessions belonging to the Q1 and mixed subpopulations were reanalysed for detailed population structure by(a) PCA and (b) UPGMA.The 144 accessions clustered into the three clades Q1_1,Q1_2and Q1_3.The blue dots represent accessions carrying haplogroups,A02_Hap1,A05_Hap2 and C05_Hap5,while the red dots represent accessions carrying haplogroup A01_Hap5.

# Statistical analysis

Heritability $( H ^ { 2 } )$ for the two traits was calculated using the statistical software package sPss Statistics for Windows Version 22.0 (IBM Corp.，Armonk,NY).Distributions and correlations among the traits and environments were analysed using the R package psych (Revelle,2014) and HMisc (Harrell and Dupont,2012).

# Genome-wide association analysis

A total of 24 338 high-quality，single-locus single-nucleotide polymorphism (SNP) markers with minor allele frequency (MAF) ${ \geq } 0 . 0 5$ were used for the GWAS and LD analyses.The mixed linear model was as follows:

$$
\begin{array} { r } { y = X \mathsf { \alpha } \mathsf { \alpha } + P \mathsf { \beta } + K \mathsf { \mu } + \mathsf { e } } \end{array}
$$

It was used to test associations between the SNPs and phenotypes,where $y$ is the vector of phenotypic observations, $\mathfrak { a }$ is the vector of SNP effects, $\beta$ is the vector of population structure effects, $\mu$ is the vector of kinship background effects,e is the vector of residual effects, $P$ is the PCA matrix relating y to $\beta ,$ and $\boldsymbol { X }$ and $K$ are incidence matrices of 1s and Os relating y to $\mathfrak { a }$ and $\mu ,$ respectively(Yu et al.,2006).The observed $P$ values from markertraitassociations were used to display $_ { \mathsf { Q - Q } }$ plotsand Manhattan plots,using R.Kinship analysis was performed using the software TASSEL 5.0 (Bradbury et al.,20o7),while detailed information on population structure was imported from the previous analysis described by Qian et al. (2014).The critical $P .$ value forassessing thesignificance of SNP-trait associations was calculated separately for CCl based ona false discovery rate (FDR;Benjamini and Hochberg,1995).An FDR $< 0 . 0 5$ was used to identify significant associations for CCl at cut-off values of $- \mathsf { l o g } _ { 1 0 } \mathsf { \Pi } _ { 0 } ^ { ( P ) } = 4$ To simplify the procedure， we used the uniform Bonferroni-corrected thresholds at ${ \mathfrak { a } } = 1$ as the cut-offs,so,the Bonferroni threshold $( - | \boldsymbol { 0 } \boldsymbol { 9 } ^ { ( 1 / 2 4 3 3 8 ) } = 4 . 4 )$ wasused toidentifysignificant associations for GSL.

# Phenotypic correlations to haplotype diversity groups

Significanthaplotypeblockswere identifiedusingtheR package LDheatmap (Shin et al.，20o6),with haplotypes being defined across regions of homozygous markers LD $( r ^ { 2 } ) > 0 . 6 5$ between the first and last markers in the block.We use the term haplogroup to refer to groups of individuals carrying a common haplotype across a specific haplotype block．Haplogroups with frequency ${ \tt > } 0 . 0 1 \$ were used for comparative phenotype analysis.A two-sample t-test (assuming unequal variances) was used to test for significant phenotypic differences between haplogroups with regard to leaf CCl and seed GSL.Haplotype networks were constructed based on the SNPs number ofhaplotype regionusing the program Tcs1.21 (Clement et al.,2000).

# Gene content in homoeologous haplotype blocks

A chromosome-scale alignment of syntenic haplotype block regionsonhomoeologouschromosomesAo5(position 17 873 133-17 986 390 bp， $r ^ { 2 } = 0 . 7 4 )$ and C05 (36 453 492-37 145 290 bp， $r ^ { 2 } = 0 . 6 8 ,$ ）wasperformed usingthe large-scale genome synteny tool sYMAP version 4.2 (Soderlund et al.，2011).All annotated genes within the corresponding haplotype regions were extracted from the B.napus Darmorbzh reference genome v.4.2 (Chalhoub et al.，2014;accessed from https://genomevolution.org/CoGe/).For verification of the most likely gene functions，we accessed annotations of the closest orthologous Arabidopsis thaliana gene by blasting to theArabidopsis genome database http://www.arabidopsis.org/.

# Population structure analysis

The general population structure of the diversity panel was described previously by Qian et al. (2014).Here，we further elucidated the detailed substructure within the largest subpop ulations,Q1 and ‘mixed',by reanalysing with a random selection of 11 910 polymorphic,single-copy SNPs that have MAF ${ \geq } 0 . 0 5$ across these subpopulations.This enabled us to accurately distinguish a small clade of 1O related individuals,with high leaf chlorophyll and low seed glucosinolate content，that form a subcluster (clade)within subpopulation Q1.The R package SelectionTools(http://www.uni-giessen.de/population-genetics/ downloads) was used to perform a principal component analysis, while an unweighted pair group matrix algorithm tree was calculated by POWERMARkER version3.25 (Liuand Muse,2005)and drawn using the software FIGTREE version 1.3.1 (Rambaut,2009).

# Competing interests

The authors declare no competing interests.

# Authors'contributions

LQ and RS conceived the study, generated the genome-wide SNP data and drafted the manuscript.WQ generated the plant population.LQ and WQ performed field phenotyping,while LQ performed glasshouse trials and was responsible for the data analysis and interpretation.All authors read and approved the final manuscript.

# Acknowledgements

The authors thank Benjamin Wittkop,Petra Degen，Malte Luh, Mechthild Schwarte and Stavros Tzigos for excellent technical assistance.Funding was provided by DFG grants 14/16-2 and 14/ 17-1 to RS and a doctoral fellowship to LQ from the Chinese Scholarship Council.

# References

Abraham, V.and Deman, J.M.(1986) Hydrogenation of Canola Oil as effected by chlorophyll.J.Am. Oil Chem. Soc.63,1185-1188.   
Bauer,J.,Chen，K.，Hiltbunner,A.，Wehrli,E.，Eugster，M.，SchnellD.and Kessler,F. (2ooo) The major protein import receptor of plastids is essential for chloroplast biogenesis.Nature,403,203-207.   
Benjamini,Y.and Hochberg,Y.(1995) Controlling the false discovery rate:a practical and powerful approach to multiple testing.J.R. Stat. Soc.B,57, 289-300.   
Bradbury，P.J.，Zhang,Z.，Kroon,D.E.，Casstevens,T.M.，Ramdoss,Y.and Buckler,E.S. (2O07) TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics,23,2633-2635.   
Buntjer, J.B.,Sgrensen,A.P.and Peleman,J.D.(2oo5) Haplotype diversity: the link between statistical and biological association. Trends Plant Sci.10,466- 471.   
Chalhoub,B,Denoeud,F.,Liu,S., Parkin,I.A.P.,Tang,H.,Wang,X., Chiquet,J. et al. (2014) Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome.Science,345,950-953.   
Chen,Y.Z.,Pang,Q.Y.,He,Y., Zhu,N., Branstrom,I.,Yan,X.F.and Chen,S. (2012） Proteomics and metabolomics ofArabidopsisresponsesto perturbation of glucosinolate biosynthesis. Mol. Plant,5,1138-1150.   
Clement, M., Posada, D.and Crandall,K.A.(2ooo)TCS:a computer program to estimate gene genealogies. Mol. Ecol.9,1657-1660.   
Corradin,O.，Saiakhova,A.,，Akhtar-Zaidi，B.,Myeroff,L.,Wilis,J.,CowperSallari, R.,Lupien,M.et al. (2014) Combinatorial effects of multiple enhancer variants in linkage disequilibrium dictate levels of gene expression to confer susceptibility to common traits. Genome Res.24,1-13.   
Czyczyto-Mysza,I.,Tyrka,M.,Marciska,I., Skrzypek,E., Karbarz,M., Dziurka, M.，Hura，T.etal.(2013)Quantitative trait loci for leaf chlorophyll fluorescence parameters,chlorophyll and carotenoid contents in relation to biomass and yield in bread wheat and their chromosome deletion bin assignments.Mol. Breed.32,189-210.   
Delmas,F.,Sankaranarayanan,S.,Deb,S., Widdup,E., Bournonville,C.,Bolier, N.,Northey,J.G.et al. (2013)ABl3 controls embryo degreening through Mendel's I locus. Proc.Natl Acad.Sci. USA,110,3888-3894.   
Edwards，D.，Batley，J.and Snowdon，R.J.(2013）Accessing complex crop genomes with next-generation sequencing.Theor. Appl. Genet.126, 1-11.   
Fernandez,L.,Le Cunff,L.,Tello,J.,Lacombe,T., Boursiquot,J.M.,Fournier-Level, A., Bravo,G.et al. (2014) Haplotype diversity of VvTFL1A gene and association with cluster traits in grapevine (V.vinifera).BMC Plant Biol.14,209.   
Ferro,M.，Brugiere,S.，Salvi,D.,Seigneurin-Berny,D.，Court,M.，Moyet,L., Ramus,C.et al. (2010)AT_CHLORO,a comprehensive chloroplast proteome   
database with subplastidial localization and curated information on envelope proteins. Mol. Cell Proteomics,9,1063-1084.   
Field,B., Cardon,G.,Traka,M.,Botterman,J., Vancanneyt,G.and Mithen,R. (2004）Glucosinolate and Amino Acid Biosynthesis in Arabidopsis.Plant Physiol. 135,828-839.   
Friedt,W.and Snowdon，R.J.(201O) Oilseed rape.In Handbook of Plant Breeding,Vol.4(Vollmann,J.and Istvan,R.,eds)，pp.91-126.，Oil Crops Breeding.Berlin/Heidelberg: Springer Verlag.   
Friso,G.,Giacomelli,L.,Ytterberg,A.J.,Peltier,J.B.，Rudella,A.,Sun,Q.and Wijk,K.J.(20o4) In-depth analysis of the thylakoid membrane proteome of Arabidopsis thaliana chloroplasts: new proteins,new functions,and a plastid proteome database.Plant $C e I I ,$ 16,478-499.   
Gotoh，E.，Kobayashi，Y.and Tsuyama，M.(2010） The post-illumination chlorophyll fluorescence transient indicates the RuBP regeneration limitation of photosynthesis in lowlight in Arabidopsis.FEBS Lett.14,3061-3064.   
Harrell,F.E.and Dupont,M.C.(2012) Package ‘Hmisc'.R:A Language and Environment for Statistical Computing.Vienna,Austria:R Foundation for Statistical Computing.   
Heilmann，I.，Mekhedov，S.，King，B.，Browse，J.and Shanklin,J.(2004) Identification ofthe Arabidopsispalmitoyl-monogalactosyldiacylglycerol delta7-desaturase gene FAD5，and eects of plastidial retargeting of Arabidopsis desaturases on the FAD5 mutant phenotype.Plant Physiol. 136,4237-4245.   
Heyndrickx，K.S.and Vandepoele，K. (2012） Systematic identification of   
functional plant modules through the integration of complementary data sources.Plant Physiol.159,884-901.   
Kroymann,J.,Textor,S., Tokuhisa,J.G.,Falk,K.L., Bartram,S., Gershenzon,J. and Mitchell-Olds，T.(2o01）A gene controlling variation in Arabidopsis glucosinolate composition is part of the methionine chain elongation pathway.Plant Physiol.127,1077-88.   
Li,P.,Filiault,D., Box,M.S.,Kerdaffrec,E.,van Oosterhout, C.,Wilczek,A.M., Schmitt,J.et al. (2014) Multiple FLC haplotypes defined by independent cisregulatory variation underpin life history diversity in Arabidopsis thaliana. Genes Dev.15,1635-1640.   
Liu,K.J.and Muse，S.V.(2Oo5)PowerMarker:an integrated analysis environment for genetic marker analysis. Bioinformatics,21,2128-2129.   
Mace,E.S.,Tai,S.,Gilding,E.K.,Li,Y.，Prentils,P.J.，Bian,L.,Campbell, B.C. et al. (2013) Whole-genome sequencing reveals untapped genetic potential in Africa's indigenous cereal crop sorghum. Nat. Commun.4,2320.   
Murillo，G.and Mehta，R.G.(2001） Cruciferousvegetables and cancer prevention. Nutr. Cancer,41,17-28.   
Oster,U.and Rudiger,W.(1998)The G4 gene of Arabidopsis thaliana encodes a chlorophyllsynthase of etiolated plants.Bot. Acta,110,420-423.   
Qian,L., Qian,W.and Snowdon,R.J.(2014) Sub-genomic variation patterns as a signature of breeding in rapeseed. BMC Genom.15,1170.   
Rambaut，A.(2009)Fig Tree version 1.3.1.[computer program].[Hattp:// tree.bio.ed.ac.uk].   
Revele,W.(2014) Psych:Procedures for Personalityand Psychological Research. Evanston,Illinois，USA: Northwestern University.htp://RAN.R-project.org/ package $\mathrel { \mathop : } = \mathfrak { p }$ -sych Version=1.4.8.   
Shin,J.-H.，Blay,S.，McNeney，B.and Graham,J. (20O6) LDheatmap:an R function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms.J.Stat. Soft.16,1-9.   
Smith，M.D.，Rounds，C.M.，Wang，F.，Chen，K.，Afitlhile，M.and Schnell,   
D.J.(2004)atToc159isa selective transit peptide receptor for the import of nucleus-encoded chloroplast proteins.J.Cell Biol.165, 323-334.   
Snowdon,R.J.，Abbadi,A.，Kox,T.,Schmutzer,T.and Leckband,G.(2015) Heterotic haplotype capture:precision breeding for hybrid performance. Trends Plant Sci.20,410-413.   
Soderlund,C.,Bomhoff, M.and Nelson,W. (2011) SyMAP: a turnkey synteny   
systemwithapplication to multiple large duplicated plant sequenced genomes. Nucleic Acids Res.39, e68.   
Stephens,J.C., Schneider,J.A.，Tanguay,D.A.,Choi,J.,Acharya,T.,Stanley S.E., Jiang,R.et al. (2oo1) Haplotype variation and linkage disequilibrium in 313 human genes.Science,293,489-493.   
Takahashi, H.，Kopriva,S., Giordano,M.，Saito,K.and HellR.(2011) Sulfur assimilationinnhotosvntheticoraanisms:molecularfunctionsand regulations of transporters and assimilatory enzymes.Annu.Rev. Plant Biol. 62,157-184.   
Textor, S.,de Kraker, J.W., Hause,B.,Gershenzon,J.and Tokuhisa,J.G.(2007) MAM3 catalyzes the formation of all aliphatic glucosinolate chain lengths in Arabidopsis.Plant Physiol.144,60-71.   
Tautorus，C.L.and Low, N.H.(1993) Chemical aspects of chlorophyll breakdown products and their relevance to canola oil stability.J.Am.Oil Chem. Soc.70,843-847.   
Thomas, H.and Ougham,H. (2014) The stay-green trait.J. Exp. Bot.65,3889- 3900.   
Tzvetkova-Chevolleau,T.,Franck,F.,Alawady,A.E.,Dall'Osto,L.,Carriere,F., Bassi,R.,Grimm,B.et al. (20o7) The light stress-induced protein ELiP2 is a regulatorof chlorophyllsynthesis in Arabidopsis thaliana.PlantJ.,50,795-809.   
Voss-Fels，K.and Snowdon，R.J.(2015) Understanding and utilizing crop genome diversity via high-resolution genotyping.Plant Biotechnol. J.doi: 10.1111/pbi.12456.   
Voss-Fels,K.,Frisch,M.，Qian,L.,Kontowski,S.,Friedt,W.,Gottwald,S.and Snowdon,R.J. (2015) Subgenomic diversity patterns caused by directional selection in bread wheat gene pools.Plant Genome，8.doi: 10.3835/ plantgenome2015.03.0013.   
Wang,F.,Wang,G.,Li,X.,Huang,J.and Zheng,J.(2Oo8) Heredity,physiology and mapping of a chlorophyll content gene of rice (Oryza sativa L.).J.Plant Physiol.165,324-330.   
Yang,J.and Zhu,Z.J. (2oo9) Glucosinolates in Pakchoi (Brassica Campestris L. ssp.Chinensis) as Affected by Nutrient Status and Postharvest.PhD thesis, Zhejiang University,China.http://www.dissertationtopic.net/down/1524719.   
Yang,Q.,Li,Z.,Li,W.,Ku, L.,Wang, C.,Ye,J.,Li, K et al. (2013) CACTA-like transposable element in ZmCCT attenuated photoperiod sensitivityand accelerated the postdomestication spread of maize.Proc.Natl Acad.Sci. USA,42, 16969-16974.   
Yu,J.，Pressoir，G.，Briggs,W.H.，Vroh Bi,I.，Yamasaki，M.，Doebley,J.F., McMullen,M.D.et al.(2oo6) Aunified mixed-model method for association mapping that accounts for multiple levels of relatedness. Nat. Genet. 38, 203-208.

# Supporting information

Additional Supporting information may be found in the online version of this article:

Figure S1 Manhattan and quantile-quantile plots of MLM showing genome-wide associations for leaf chlorophyll content index in two different environments (glasshouse and field) in 203 Chinese semi-winter rapeseed accessions.

Figure S2 Genome-wide associations for leaf chlorophyll content index on chromosomes A07,A08,C03 and C08,respectively. Figure S3 Boxplots showing phenotypic values for leaf chlorophyll content index in nine and three haplogroups，with frequency ${ > } 0 . 0 1$ ，found in haplotype regions on chromosomes A05 and Co5,respectively.   
Figure S4 Comparative analysis of leaf chlorophyll content index (CCI) between groups of accessions carrying combinations of two or three of CCl-associated haplogroups (group A)and accessions carrying only one CCl-associated haplogroup (group B).   
Figure S5 Boxplots showing leaf chlorophyll content index and seed GSL content in the three different subgroups.   
Figure S6 Association mapping for seed GSL on chromosome A02 in 203 Chinese semi-winter rapeseed accessions.   
Table S1 Source，population structure and heritability (leaf chlorophyll content index and seed GSL) in 203 Chinese semiwinter rapeseed accessions   
Table S2 Detailed description of five haplotype regions significantly associated with leaf chlorophyll content index   
Table S3 Comparative analysis of haplogroups related to leaf chlorophyll content index,along with gene content in homologous haplotype regions on chromosomes Ao5 and C05   
Table S4 Comparative analysis of six haplogroups related to leaf chlorophyll content index,along with gene information in the A02 haplotype region   
Table S5 Comparative analysis of three haplogroups corresponding to leaf chlorophyll content index in BnaA02g20650D gene region   
Table S6 Comparative analysis of five haplogroups corresponding to leaf chlorophyll content index and seed GSL and gene information in chromosome A01 coselection haplotype region Table S7 The distribution of haplogroups related to higher leaf chlorophyll content index accessions and comparative analyses of three clades corresponding to leaf CCl and seed GSL in subpopulations‘Q1'and'mixed',respectively   
Table S8 Comparative analysis of leaf chlorophyll content index between A and B groups   
Table S9 Gene information in the AO2 haplotype region