# High resolution structures of Plasmodium falciparum GST complexes provide novel insights into the dimer-tetramer transition and a novel ligand-binding site

Markus Perbandt a,c,e,\*,1, Raphael Eberle a,1, Lena Fischer-Riepeb, Huaixing Cangd, Eva Liebaub, Christian Betzel a,e

InstituteofBochemistrndMolecularBologyUniversityofHmburgLaboratoryofStructualBologyfIfectionndIfaatio   
c/o DESY,Notkestr.85,Build.22a,D-22603 Hamburg,Germany   
b Departmentof MolecularPhysiology,University of Miinster,Schlossplatz 8,D-48143 Muinster,Germany   
InstituteofdiclodesticlterdfU)   
d Institute of Biophysics, Chinese Academy of Sciences,Beijing,China   
eThe Hamburg Centre for Ultrafast Imaging,Luruper Chaussee 149,D-22761 Hamburg,Germany

# ARTICLE INFO

Article history:   
Received 4 March 2015   
Received in revised form 8 June 2015   
Accepted 9 June 2015   
Available online 1O June 2015

Keywords:   
Malaria   
Glutathione S-transferase   
GST   
Tetramerization   
Ligandin   
Non-substrate binding site

# ABSTRACT

Protection from oxidative stressand efficient redox regulation are essential for malarial parasites which have to grow and multiply rapidly in pro-oxidant rich environments.Therefore,redox active proteins currently belong to the most atractive antimalarial drug targets.The glutathione S-transferase from Plasmodium falciparum (PfGST) is a redox active protein displaying a peculiar dimer-tetramer transition that causes fullenzyme-inactivation.This distinct structural feature is absent in mammalian GST isoenzyme counterparts.A flexible loop between residues 113-119 has been reported to be necessary for this tetramerization process. However,here we present structural data of a modified PfGST lacking loop 113- 119 at $1 . 9 \mathring { \mathsf { A } }$ resolution.Our results clearly show that this loop is not essential for the formation of stable tetramers.Moreover we present for the firsttime the structures of both,the inactive and tetrameric state at $1 . 7 \mathring { \mathsf { A } }$ and the active dimeric state in complex with reduced glutathione at $2 . 4 \mathring { \mathsf { A } }$ resolution. Surprisingly,the structure of the inactive tetrameric state reveals a novel non-substrate binding-site occupied by a 2-(N-morpholino) ethane sulfonic acid (MES)molecule in each monomer. Although it is known that the PfGST has theability to bind lipophilic anionic ligands,the location of the PfGST ligand-binding site remained unclear up to now.

$\circledcirc$ 2015 Elsevier Inc.All rights reserved.

# 1. Introduction

Malaria is among the most prevalent infectious diseases in the developing world,imposing a vast burden of mortality and perpetuating cycles of poverty(WHO,2O1O).The control of malariais challenged by drug resistance and new antimalarial drugs are urgently needed (Perez etal.,2O12).The parasite Plasmodium falciparum causes malaria tropica,the most prevailing parasitic disease worldwide.The emergence of strains resistant to drugs used for prophylaxis and treatment and no vaccine available makes the characterization of alternative drug targets an urgent requirement (Greenwood and Mutabingwa, 2002; Olliaro, 2001).

Sophisticated defense strategies have evolved in parasitic organisms that enable them to deal with a broad range of foreign and endogenously toxic compounds.P.falciparum parasitesutilize host proteins as food source during their erythrocytic stage and large quantities of hemoglobin are digested,confronting the parasite with high amounts of toxic hemin.Thus,the discovery of compounds selectively inhibiting the detoxification of hemin is of high importance for the development of novel antimalarial drugs.

Glutathione S-transferases (GSTs)[EC 2.5.1.18] play a key role in detoxification processes.On the basis of amino acid sequence homology，immunological cross reactivity，substrate specificity and variation of active site residues,GSTs can be grouped into different classes (Frova,2Oo6).They catalyze the nucleophilic addition of the tripeptide glutathione (GSH) to a large variety of nonpolar or toxic compounds and thereby enhance the solubilityand excretion of the resulting products.GSTs are functional and stable dimers and the active site is composed of two neighboring substrate-binding sites.The so called G-site binds reduced glutathione and the hydrophobic H-site can accommodate a variety of electrophilic co-substrates (Hayes et al., 2Oo5；Mannervik and Danielson,1988).Several crystal structures of the wild-type GST from P. falciparum(PfGST）are already known(Fritz-Wolf et al., 2003；Hiller et al.，2O06；Perbandt et al.，2004；Tripathi et al., 2007).Although the overall three-dimensional structure is homologue to other known GST isoforms,thePfGST could not be clearly assigned to any of the specified GST classes.

Previous studies revealed that the PfGST is an important intracellular ligandin and is able to eficiently bind large lipophilic molecules like bilirubin,hemin and certain drugs (Axarli et al. 2004;Deponte and Becker, 2005;Harwaldt et al.,2002). Distinct ligand-binding siteshave been described for GSTs from different organisms (Ji et al.，1996；McTigue et al.，1995；Oakley et al., 1999;Prade etal.,1997; Reinemer etal.,1996). While the localization of the PfGST ligand-binding site(L-site） has not been structurally described so far,it was proposed to be part of the H-site or the inter-subunit cleft (Liebau et al., 2Oo2.,2005).

Two properties of the PfGST never observed in other members of the GST superfamily are of particular interest and underline the outstanding significance of this enzyme.Firstly,the enzyme shows a strong positive homotropic behavior that modulates the hemin affinity of the two subunits (Liebau et al.,2oo5). Secondly, in the absence of GSH the PfGST gets inactivated ina relative short period of time and loses its ability to bind hemin(Liebau et al, 2005).

This inactivation process is known to be related to a dimer-tetramer transition of the protein (Fritz-Wolf et al., 2Oo3;Hiller et al., 2006;Tripathi et al.,2OO7). The dimer-tetramer transition has not been observed for any other GST (Liebau et al.，2OO9). In the absence of reduced GSH or other ligands,two active dimers form a tetramer.The homodimers are interlocked with each other by loop 113-119 which occupy the H-site of the other monomer and vice versa.Loop 113-119 connects two conserved $\textsf { \textsf { q } }$ -helices and bears the TNLFKQN sequence.The PfGST is the only enzyme of the GST superfamily,which is present as a tetramer rather than a dimer.Itwasalready reported that truncation,increased rigidity or even a simple point mutation of this loop causes a dramatic decrease in the tetramerization kinetics (Liebau et al.,2oo9). It was also proposed that an interaction between Asn112 and Lys117 plays a key role in keeping the tetramer structure stable (Quesada-Soriano et al., 2014).

Here we report for the first time the high resolution structure of PfGST in the active dimeric statein complex with GSH. Furthermore,based on the structural analysis of three different crystal structures of PfGST in both,the active and inactive state, our obtained results extend the knowledge concerning the requirements for the dimer-tetramer-transition and provide novel insightswith respect to the ability of the enzyme to bind lipophilic and toxic substances,thereby securing the inactivation and immobilization of these compounds.

# 2.Material and methods

# 2.1.Recombinant expression and purification of the PfGST

PfGST wascloned,expressed and purified as reported previously (Burmeister et al., 2OO3;Liebau et al., 2OO2;Perbandt et al.. 2004). After $^ { 4 \mathrm { h } }$ growth at $3 7 ^ { \circ } \mathsf C$ ，bacterial cells were harvested by centrifugation; the cell pellet was dissolved in $1 0 0 \mathrm { m M }$ Tris/HCl buffer $\mathsf { p H } 8 . 5$ and sonicated. The cell lysate was centrifuged at $1 6 . 0 0 0 \times g$ and $4 ^ { \circ } C$ for $^ { 1 \mathrm { h } }$ .The supernatant was mixed with Superglu glutathione superflow resin (Generon,Berkshire, UK)and incubated for $^ { 1 \mathrm { h } }$ at $4 ^ { \circ } \mathsf C$ Unspecific bound proteins were removed by washing the matrix with $1 0 \times$ bed volumes of $1 0 0 \mathrm { m M }$ Tris/HCl buffer, $\mathsf { p H } 8 . 5$ and the recombinant protein was eluted with reduced GSH.Cloning,recombinant expression and purification of the mutant $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ was carried out as previously described (Liebau etal., 2009).

# 2.2. Crystallization

Crystallization of the wild type $P f G S \mathrm { T }$ in the tetrameric state $( P f G S \mathrm { T } ^ { \mathrm { a p o } } )$ was performed using the hanging-drop vapor-diffusion method.Hanging droplets were prepared by mixing $2 \mu \mathrm { l }$ of the protein solution at a concentration of $1 8 ~ \mathrm { m g / m l }$ together with $2 \mu \mathrm { l }$ of precipitating solution $( 2 . 1 \mathrm { M }$ ammonium sulfate, $1 0 0 \mathrm { m M }$ $\mathsf { M E S } \mathsf { p H } 6 . 0$ )and $0 . 5 \mu \mathrm { l }$ of a seed stock(1:10oo diluted),containing micro-seed crystals.The preparation of the seed stock is described below.

The setups to grow the active form in complex with GSH $( P f G S \mathrm { T } ^ { \mathrm { G S H } } )$ were performed and stored under an oxygen free environment using phosphate buffer instead of MES buffer ！ $( 2 . 1 \mathrm { M }$ ammonium sulfate, $1 0 0 \mathrm { m M }$ phosphate buffer $\mathsf { p H } \ 6 . 5 \mathsf { \cdot }$ .Crystals appeared after1week at $2 0 ^ { \circ } \mathsf C$ with maximum seize of $5 0 0 \mu \mathrm { m }$

Crystallization of the $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ was performed using the same method with the following modifications:Hanging droplets were made by mixing $3 \mu \mathrm { l }$ of the protein solution with a concentration of $1 0 \mathrm { m g / m l }$ with $3 \mu \mathrm { l }$ of precipitating solution( $( 2 . 3 \mathrm { M }$ ammonium sulfate, $1 0 0 \mathrm { m M T r i s / H C l p H } 8 . 5 )$ .Droplets were equilibrated against $1 ~ \mathrm { m l }$ of precipitating solution. The crystallization trials werekept forabout1week at $2 0 ^ { \circ } \mathsf C$

# 2.3.Preparation of a micro-seed-stock

Microgravity experiments were performed in terms of the DLR SIMBOX experiment (Preu and Braun,2O14) onboard of the chinese space mission Shenzhou8(30.10.2011-17.11.2011).The SIMBOX counter diffusion crystallization experiments were coordinated bythe Institute of Biophysics of the Chinese Academy of Sciences,Beijing.Prior to the microgravity experiments,crystallization conditions applying the compact counter diffusion flight hardware were extensively optimized under laboratory conditions, including transport simulations.The final protein solutions were analyzed by dynamic light scattering and showed a stable monodisperse peak for several days.The capillary length of the SIMBOX crystallization hardware was $1 0 \mathrm { m m }$ with an inner diameter of $1 \mathrm { m m }$ .The capillary was filled with a PfGST solution using a concentration of $1 8 ~ \mathrm { m g / m l }$ and $2 \% ( \boldsymbol { \mathrm { w } } / \boldsymbol { \mathrm { v } } )$ lowmeltingpointagarose (Serva) was used to seal the capillaries.Subsequently,the capillarieswere inserted carefully into a plastic casing,containing four separated sections filled with foam material and sealed with a metal lid.The foam was saturated with approximately $3 0 0 \mu \mathrm { l }$ of precipitant solution $( 2 . 3 \mathrm { M }$ ammonium sulfate, $1 0 0 \mathrm { m M }$ Tris/HCl $\mathsf { p H } 8 . 5 \mathsf { \cdot }$ ).For the microgravity experiment 4 capillaries were positioned in the DLR SIMBOX.The microgravity experiment was performed ata stable temperature of $2 0 ^ { \circ } \mathsf C$ for 17 days(30.10.2011- 17.11.2o11）onboard the Shenzou8 space mission.Following the space mission the capillaries were inspected directly after arrival in Beijing.The capillaries were opened at one end and the agarose was carefully removed.The obtained micro-crystals were used to prepare seed-stock solutions following the manufacturer's instructions of Jena BioScience Beads-for-Seeds.

# 2.4.Data collection and structure determination

Before data collection the crystals were soaked in the precipitating solution supplemented with $1 0 \% \ ( \mathrm { v / v ) }$ glycerol for stabilization ofthe crystalsduringthefreezingprocess. Diffraction data of $P f G S \mathrm { { T } ^ { a p o } }$ were collected at $1 0 0 \mathrm { K }$ at the P14 beamline at a wavelength of $1 . 2 4 \mathring { \mathsf { A } }$ (Petra III, EMBL-Hamburg, Germany）usinga PILATUS $6 { \sf M }$ detector.Data to $1 . 7 \mathring { \mathsf { A } }$ resolution were indexed,integrated,and scaled using XDS (Kabsch). The obtained crystals belong to the orthorhombic space group $\mathsf { P } 2 _ { 1 } 2 _ { 1 } 2$ with cell dimensions of $a = 8 6 . 4$ = $b = 6 0 . 9$ = $c = 7 4 . 8 \mathring { \mathrm { A } }$

Diffraction data of wild-type PfGST in the dimeric and active state in complex with GSH $( P f G S \Gamma ^ { \mathrm { G S H } } )$ were collected at $1 0 0 \mathrm { K }$ at the consortiums beamline X13(DORIS/DESY) on a MAR CCD detectorata wavelength of $0 . 8 0 4 \mathring { \mathsf { A } } .$ Data to $2 . 4 \mathring { \mathsf { A } }$ were indexed,integrated,and scaled by using HKL2ooO (Otwinowski and Minor, 1997).The space group and unit cell parameters were assigned to the space group $\mathsf { P } 2 _ { 1 } 2 _ { 1 } 2 _ { 1 }$ with unit cell dimensions of $a = 5 7 . 4$ $b = 6 8 . 4$ $c = 1 2 2 . 7 \mathring { \mathsf { A } } .$

Diffraction data of the $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ were collected at $1 0 0 \mathrm { K }$ at the consortiums beamline X13 (EMBL-Hamburg/DESY)at a wavelength of $0 . 8 0 4 \mathring { \mathsf { A } }$ using a MAR CCD detector.Data to $1 . 9 \mathring { \mathsf { A } }$ were indexed,integrated,and scaled by using HKL2ooo(Otwinowski and Minor，1997).Crystals belong to the same space group $\mathsf { P } 2 _ { 1 } 2 _ { 1 } 2$ with almost the identical cell dimensions to the wild-type enzyme initstetrameric state.

For all datasets the asymmetric unit contains two monomers. The structures were determined by molecular replacement,using the program package MOLREP(CCP4,1994) and the PfGST structure (PDB code:1PA3). The structure Was carefully built using a bootstrapping procedure involving multiple model building rounds interspersed using maximum likelihood refinement in REFMAC5 (Murshudov et al.,1999) and the progress was monitored by a continuous decrease of the free R-value (Brunger, 1993).The CCP4 program suite(CCP4,1994) and the program COOT(CCP4,1994) were used for calculations and model building,respectively.Solvent molecules were automaticallyadded at the end of the refinement process using the program wARP(Perrakis et al.，1999）and checked to confirm chemically reasonable positions where difference electron density exceeded a $3 \sigma$ level.

For all three structures the final electron density maps were of very good quality and the models indicated good stereochemistry. The quality of the final model was assessed by PROCHECK (Laskowski et al.,1993). The data collection and refinements statistics are summarized in Table 1.The coordinates for the structures, aswell as the experimental diffraction amplitudes have been deposited at the Protein Databank (http://www.rcsb.org）with entry codes 3FR3,3FR9 and 4ZXG.The protein interfaces and the protein ligand interactions were analyzed in detail using PDBsum (Laskowski et al.,1997) and LigPlot (Wallace et al., 1995).

# 2.5.Near UV measurements using CD spectroscopy

CD spectra in the near UV region were recorded on a JASCO J-815 spectrometer (JAScO,Easton,USA). Experiments were implemented at wavelengths between 340 and $2 4 0 \mathrm { n m }$ using a $1 . 0 \mathrm { c m }$ quartz cuvette;the protein concentration was $4 \mathrm { m g / m l }$ . All data were obtained from two independent experiments.For each experiment the protein measurements were repeated 15 times,the blank(buffer）measurements were repeated 5 times.Subtraction of the buffer data from the protein data resulted ina clear CD spectra.By conducting near UV experiments in $2 0 \mathrm { m M }$ ${ \mathrm { K } } _ { 2 } { \mathrm { H P O } } _ { 4 } .$ $1 0 0 \mathrm { m M }$ MES, $\mathsf { p H } 7 . 5$ ，thePfGSTbinding of MESwasanalyzed in solution.Control-measurements were done in absence of MES.

The CD spectra $\left( \pm \mathrm { M E S } \right)$ were compared and analyzed by superposition to follow the decrease in the appropriate spectra of the aromatic amino acids and to identify the interaction between proteins and MES molecules in solution.Results are expressed as molarellipticity $[ \theta ] ( \deg { \cos ^ { 2 } { \mathrm { d m o l } ^ { - 1 } } } )$ ,calculated from the following equation $[ \theta ] = \theta / 1 0 ^ { * } c ^ { * } l ^ { * } n _ { \mathrm { r } } ,$ where $\theta$ is the measured ellipticityinmillidegree, $c$ is the protein concentration in molar units,l thelight path length in $\mathsf { c m }$ ,and $n _ { \mathrm { r } }$ is the number of the residues. The overall mean value for the experimental error of the molar ellipticity $[ \theta ]$ was O.5 and 0.4( $\cdot \pm \boldsymbol { \mathrm { M E S } } )$ .All data points presented in Fig.7 are expressed as mean $\pm$ standard deviation (SD),according to the following Eq.(1).

Table 1 Data collection and refinement statistics.   

<html><body><table><tr><td></td><td>PfGSTapo</td><td>PfGSTGSH</td><td>PfGST△114-118 + GSSG</td></tr><tr><td>Space group</td><td>P2,22</td><td>P2221</td><td>P222</td></tr><tr><td>a (A)</td><td>60.9</td><td>57.4</td><td>60.9</td></tr><tr><td>b (A)</td><td>86.4</td><td>68.4</td><td>87.1</td></tr><tr><td>c (A)</td><td>74.8</td><td>122.7</td><td>74.8</td></tr><tr><td>VM (A³/Da)</td><td>2.0</td><td>2.4</td><td>2.0</td></tr><tr><td>Solvent content (%)</td><td>38.0</td><td>49.3</td><td>39.7</td></tr><tr><td>Resolution (A)</td><td>30-1.7</td><td>20-2.4</td><td>20-1.9 (1.9-1.95)</td></tr><tr><td>No.of total refections</td><td>(1.7-1.75)</td><td>(2.4-2.46)</td><td></td></tr><tr><td>Redundancy</td><td>273,546 6.2</td><td>128,383</td><td>235,301</td></tr><tr><td>Completeness of data (%)</td><td></td><td>7.1</td><td>7.9</td></tr><tr><td></td><td>98.3 (78.2)</td><td>97.2 (94.0)</td><td>98.6 (98.0)</td></tr><tr><td>Average I/sigma Intensity No.of reflections used in</td><td>30.5 (3.6) 43,889</td><td>14.3 (2.9) 17,966</td><td>20.1 (3.6) 29,646</td></tr><tr><td>refinement</td><td></td><td></td><td></td></tr><tr><td>Rmerge (%)a</td><td>2.6 (42.1)</td><td>8.6 (44.6)</td><td>8.6 (54.2)</td></tr><tr><td>Rwork (%)b</td><td>18.3 (25.6)</td><td>24.3 (26.9)</td><td>21.1 (25.0)</td></tr><tr><td>Rfree (%)</td><td>22.9 (30.8)</td><td>29.6 (31.4)</td><td>25.2 (34.9)</td></tr><tr><td>Protein atoms</td><td>3718</td><td>3308</td><td>3261</td></tr><tr><td>Solvent atoms</td><td>218</td><td>122</td><td>212</td></tr><tr><td>Ligand atoms</td><td>56</td><td>40</td><td>80</td></tr><tr><td>Mean B value (A²)</td><td>21.9</td><td>26.7</td><td>33.9</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>Rootmean square deviation</td><td></td><td></td><td></td></tr><tr><td>Bonds (A)</td><td>0.02</td><td>0.006</td><td>0.02</td></tr><tr><td>Bond angels (°)</td><td>1.7</td><td>0.89</td><td>1.9</td></tr></table></body></html>

□ $\mathrm { R } _ { \mathrm { m e r g e } }$ is defined as $\sum _ { h k l } \sum _ { i } \ | I _ { i } ( h k l ) - [ I ( h k l ) ] | / \sum _ { h k l } \sum _ { i } I _ { i } ( h k l )$ where $\mathrm { { I } _ { i } ( \ h k l ) }$ is the intensity of an individual measurement and [I(hkl)] is the mean intensity from multiple observations of this reflection. b $R _ { \mathrm { { w o r k } } } = \sum | \mathrm { F _ { \mathrm { { o b s } } } } | - | \mathrm { F _ { \mathrm { { c a l c } } } } | / \sum | \mathrm { F _ { \mathrm { { o b s } } } } |$ where $\mathrm { F _ { o b s } }$ and $\mathrm { F _ { c a l c } }$ are observed and calculated structure factors. ： $R _ { \mathrm { f r e e } }$ was calculated using a randomly selected $5 \%$ of the reflections.

![](images/b196edf0198c5c51075f13189fcfa677b40d7a53ff15fd99e5237ffe4f00c188.jpg)  
Fig.1.The PfGST monomer,emphasizing the secondary structure elements.Helices are colored yellow, $\beta$ -strands are colored blue.PfGST is composed out of two domains.The first domain consists of four $\beta$ -sheetswiththree flanking $\textsf { \textsf { a } }$ -helices. Domain 2 is built up entirely from five $\mathfrak { a }$ -helical segments connected bya variety of loops.The GSH molecule bound to the G-site is shown in stick representation.The $\mathsf { H }$ -site is located next to the G-site flanked by the C-terminal part of $\beta$ -sheet $\beta 1$ and $\beta 2$ and the N-terminal part of helix α1.Loop 113-119 connects helix α4 and α5.

![](images/58e32bfea566dd2191f9f1378844a98237f4097a25e669a5620f469742443d71.jpg)  
Figo ildtp cyan.The characteristic Mu-loop,located between $\beta$ -sheet2and $\mathfrak { a }$ -helix 2iscolored in greenand loop 113-119 is colored inred.(C)Two active dimers form the enati Mu-loopand loop113-119arecoloredasin (B).(C) $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ thatismissing majorpartsof loop113-119isalsoabletoformstable tetramers.MoreoverfourGSG molecules,shown in stick representation and colored in cyan, bind tightly to all G-sites.

$$
s = { \sqrt { { \frac { 1 } { N - 1 } } \sum _ { i = 1 } ^ { N } ( x _ { i } - { \bar { x } } ) ^ { 2 } } }
$$

# 3.Results and discussion

# 3.1.Overall structures ofwild-type PfGSTinthe dimeric and tetrameric state

For the first time we have crystallized the enzymatically active form in complex with reduced GSH in the dimeric state.The structure was refined to a resolution of $2 . 4 \mathring { \mathsf { A } } .$ The overall structure of the monomers (Fig.1) coincides with previously reported PfGST structures (Fritz-Wolf et al., 2O03；Hiller et al.,2006;Perbandt et al.,2Oo4).A functional active homodimer (Fig.2） forms the asymmetric unit (Fig.3A).The overall fold of the monomers within the asymmetricunit is very similarand can be superimposed with an r.m.s deviation of less than $0 . 5 \mathring { \mathsf { A } }$ for all C-alpha positions (Fig.2A). The space group and unit cell axis differ in comparison to the tetrameric state,even if the crystallization conditionsare quite comparable (Table 1).

The intermolecular interactions at the subunit interface of the active dimersarecharacterized byseveral hydrophilicinteractions (Table2).Particularly,thesidechainof Arg77 servesasa hydrophilic anchor and forms two hydrogen bridges to the side chain of Asp97fromthe other subunit of thehomodimer. This arginine-mediated intersubunit contact is class-specificfor Mu-class GSTs (Dirr et al.，1994).Besides this specific feature, hydrophobic interactions are the dominating force of the dimerization.These“lock and key" motifs are common for several GST-classes and offer a good possibility to distinguish between the classes(Sinning et al.,1993). Here Phe56 acts as the“key", which fitsinto the hydrophobic“lock"formedby Trp131,Phe135 and Tyr134 of the other subunit.This structural feature is also unique for Mu-class GSTs.Active dimers(Fig.2B) reveal an interaction interface of $1 . 3 4 7 \mathring { \mathsf { A } } ^ { 2 }$ ,corresponding to $12 \%$ of the total monomer surface of $1 1 . 4 8 9 \mathring { \mathsf { A } } ^ { 2 }$ ：

The inactive tetrameric state (Fig.2C) of the wild-type PfGST was determined at a resolution of $1 . 7 \mathring \mathrm { A } .$ Here again a dimer forms the asymmetric unit.However these dimers do not match to the active homodimers.Therefore theyare named non-active dimers. Theinactive tetrameris formed bya crystallographic 2-foldaxis from two non-active dimers (Fig.3B). The non-active dimers show an interaction interface of $1 . 1 2 1 \mathring { \mathsf { A } } ^ { 2 }$ ,corresponding to $9 . 7 \%$ of the total monomer surface.The interface is stabilized by hydrogen bonds,but dominated by hydrophobic interactions.The intersubunit interactions are summarized in Table 3.Particular residues located within loop 113-119 are involved in these direct hydrogen bonds and further mediate almost all of the hydrophobic interactions.In the tetrameric state two homodimers are interlocked with eachotherby loop113-119 and form the tetramer.The active site is blocked by loop 113-119 and therefore the tetrameric state remains enzymatically inactive.

Table 2 Characteristic direct intersubunit interactions at the interface of the active dimer in PfGSTaPo, PfGSTGSH and $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$   

<html><body><table><tr><td>Residue subunit 1</td><td>Residue subunit 2</td></tr><tr><td>Hydrogen bonds</td><td></td></tr><tr><td>Glu57 OE1Acc</td><td>Tyr134 OHDon</td></tr><tr><td>Arg77 NEDon</td><td>Asp7 OD2Acc</td></tr><tr><td>Arg77 NH2Don</td><td>Asp7 OD1Acc</td></tr><tr><td>Tyr134 OHDon</td><td>Glu57 OE1Acc</td></tr><tr><td>Asp97 OD2Acc</td><td>Arg77 NEDon</td></tr><tr><td>Asp97 OD1Acc</td><td>Arg77NH2Don</td></tr><tr><td>Hydrophobic interactions</td><td></td></tr><tr><td>Phe56</td><td>Trp131,Tyr134,Phe135</td></tr><tr><td>Trp131,Tyr134,Phe135</td><td>Phe56</td></tr></table></body></html>

Interestingly,for the inactive tetrameric state we identified a ligand-binding site occupied by one MES molecule in each of the four monomers.This novel non-substrate binding site will be described later.

![](images/7b30470b39c843f0a26be5ea2c715aa2e168857d77c7bdd7d82ffc9eff2460f8.jpg)  
Fige groups.Crystals of the active dimers belong to space group $\mathsf { P } 2 _ { 1 } 2 _ { 1 } 2 _ { 1 }$ and crystals of the inactive tetramers to space group $\mathrm { P } 2 _ { 1 } 2 _ { 1 } 2$ .Molecules in the asymmetric unit are colored inmagentacidal

Table3 Characteristic direct intersubunit interactions at the interface of the tetrameric structures of PfGSTapo and PfGST△114-118.   

<html><body><table><tr><td colspan="2">PfGSTapo</td><td colspan="2">PfGST△114-118</td></tr><tr><td>Residue subunit 1</td><td>Residue subunit 2</td><td>Residue subunit 1</td><td>Residue subunit 2</td></tr><tr><td>Hydrogen bonds</td><td></td><td></td><td></td></tr><tr><td>Asn111 oAcc</td><td>Asn114 ND2Don</td><td>None</td><td>None</td></tr><tr><td>Asn112 OD1Acc</td><td>Lys117 NZDon</td><td></td><td></td></tr><tr><td>Asn114 ND2Don</td><td>Asn111 OAcc</td><td></td><td></td></tr><tr><td>Lys117 NZDon</td><td>Asn112 OD1Acc</td><td></td><td></td></tr><tr><td>Hydrophobic interactions</td><td></td><td>Phe10,Phe42,</td><td>Leu124,Tyr176</td></tr><tr><td>Phe10,Phe42, Phe45</td><td>Leu124, Tyr176</td><td>Phe45'</td><td></td></tr><tr><td>Tyr,Phe10,Ala12</td><td>Leu115</td><td>Leu124,Tyr176</td><td>Phe10,Phe42, Phe45</td></tr><tr><td>Lys15,His107, Tyr108</td><td>Phe116</td><td></td><td></td></tr><tr><td>Gln58,Val59</td><td>Gln118</td><td></td><td></td></tr><tr><td>Leu124,Tyr176 Asn111</td><td>Asn114,Phe116</td><td></td><td></td></tr><tr><td></td><td>Phe10,Phe42, Phe45'</td><td></td><td></td></tr><tr><td>Leu115</td><td>Tyr9,Phelo,Ala12</td><td></td><td></td></tr><tr><td>Phe116</td><td>Lys15,His107, Tyr108</td><td></td><td></td></tr><tr><td>Gln118</td><td>Gln58,Val59</td><td></td><td></td></tr><tr><td>Asn114,Phe116</td><td>Asn111</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr></table></body></html>

Residues that are part of $\mathsf { L o o p } ^ { 1 1 3 - 1 1 9 }$ are underlined.

# 3.2.Structural characteristicsof the G-site and H-site

The comparison of $P f G S \mathrm { T }$ with the human Mu-class enzyme (Raghunathan et al.,1994) revealed a sequence identity of $24 \%$ with an r.m.s. deviation of $2 . 6 \mathring { \mathsf { A } }$ for $\mathsf { C a }$ -atoms.The highest differences are in the regions of the Mu-loop, the C-terminal tail,and loop113-119 formed by helix $\mathtt { \Gamma } \mathtt { \backslash } \mathtt { a } 4$ and helix α5 (Figs.1 and 2). The sequence comparison of PfGST with GSTs from other organisms revealed a rather low sequence similarity.The highest sequence identity is observed with the GST from Wuchereria bancrofti GST(PDB entry code 3T2U,to be published) with only $31 \%$

As previously reported in detail by us (Perbandt et al.,2O04), in typical Mu-class GSTs,the Mu-loop and C terminus are connected by interchain hydrogen bonds and form a characteristic deep cleft shielding the G-and H-sites from one side.However,since the Gand H-sites of the PfGSTare not shielded by the Mu-loop and the C terminus they are much more exposed to the solvent (Fig.4). The additional space between the Mu-loop and the C terminus found inthe plasmodial enzyme may support the designof parasite-specific synthetic inhibitors.Although the G-site of PfGST and mammalian Mu-class GSTsis structurally similar,the H-site of the GSTs differs significantly.The most significant difference is the replacement of the typical Mu-loop by a short turn in PfGST.This structural feature is also described for the GST from Schistosoma japonicum,a trematode that causes schistosomiasis (Cardoso et al.,20o3).A further comparison of the H-site from PfGST with that of other GST classes indicates a higher similarity to a typical Pi-class enzyme.In principle,the G- and the H-sites ofPfGST combine structural features of the Pi-and the Mu-classes. Therefore,we can consider thePfGSTasa“chimeric enzyme."This general conclusion is supported by the biochemical characterization of the enzyme (Harwaldt et al., 2oo2;Liebau etal., 2002).

The comparison of the wild-type $P f G S T$ in an inactive tetrameric state,the wild-type $P f G S \mathrm { T }$ ina dimeric state and the loop mutant formin a tetrameric state reveal no significant structural differences or conformational changes for the $G -$ and H-site.The binding sitesare rigid and not changed by the dimer-tetramer transition. The superposition of the monomers in the dimeric and tetrameric state shows a low r.m.s. deviations of $0 . 3 8 \mathring { \mathsf { A } }$ (for $P f G S \mathrm { { T } ^ { G S H } }$ and $P f G S \mathrm { T } ^ { \mathrm { a p o } } )$ and $0 . 5 1 \mathring { \mathsf { A } }$ (for $P f G S \mathrm { T } ^ { \mathrm { G S H } }$ and $\mathrm { P } f \mathrm { G S T } ^ { \Delta 1 1 4 - 1 1 8 } ,$ for all corresponding C-alpha positions.

![](images/29d75fad0b7354eb779bf101b30f1fb3f85989235c875a8368017e350ba8a53c.jpg)  
Fig4Vietdi dimer)Thelectoliteisoadroobcsidsee4de）dkdyoolod

# 3.3. The quaternary structure of PfGST4114-118

Sincea structural knowledge about the dimer-tetramer transition is important to develop new potential inhibitors and since we are interested to elucidate the structural basis of this observation, we carried out an X-ray crystallographic analysis of a mutant with a truncated loop region. The loop region 114-118 (Asn114,Leu115, Phe116, Lys117 and $\mathsf { G l u } ^ { 1 1 8 }$ ) was replaced by two alanine residues.

We expected that the modification of $\log ^ { 1 1 3 - 1 1 9 }$ stabilizes the dimer species as indicated by the recent kinetic data (Liebau etal.,2oo9)and the aim was to crystallize the active dimer in complex with reduced GSH.

Surprisingly,our $\mathsf { X } .$ -ray structural analysis revealed the opposite. The obtained structure of $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ at $1 . 9 \mathring { \mathsf { A } }$ resolution resembles a consistent tetrameric architecture as observed for the wild-type PfGST in the inactive,GSH-free state (Fig. 2D). Accordingly,the intermolecular interactions at the subunit interface of the active dimer of $P f G S \Gamma ^ { \Delta 1 1 4 - 1 1 8 }$ are preserved in comparison to the wild-type PfGST in the inactive,GSH-free state(Table 2). This results in an almost identical interaction interface of $1 . 3 5 0 \mathring { \mathsf { A } } ^ { 2 }$ corresponding to $1 3 \%$ of the total monomer surface of $1 0 , 6 9 0 \mathring { \mathsf { A } } ^ { 2 }$ Moreover,all G-sites and parts of the H-sites are occupied by oxidized glutathione (GSSG) instead of reduced glutathione (GSH).

![](images/abfcf5ae5a861736e6025ecfb5b182b0cb8e089dd7bf680358c5e581a849eb94.jpg)  
ig（ thtetr Phe42oftheMu-loopiscoloredingreen.(B)Acoseuviewtothecorresponingeionof $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ .GSSGisboundtotheactivesiteandstabiizedbyirecthrogen bodsostd e for the truncated loop in $P f { \bar { \mathrm { G S T } } } ^ { \Delta 1 1 4 - 1 1 8 }$ ,also enabling the binding of GSSG in the tetrameric state.

In one of our previous studies we found that GSSG prevents the inactivation process,thus stabilizing the dimeric structure of the wild-type enzyme and behaving like an unambiguous competitive inhibitor(Liebau et al.,2oo9). Furthermore,another group reported that the interaction between Asn112 and Lys117 from neighboring subunits is most essential to keep the tetramer structure stable (Quesada-Soriano et al., 2014).

However,the data we summarize here show that loopl13-119 is not required to form stable tetramers and GSSG is not preventing the tetramerization process for $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ at high protein concentrations.At lower concentrations the interactions through the loop can influence the oligomerization kinetics as reported byus previously (Liebau et al.,2OO9).Neither the wild-type PfGST nor PfGSTA114-118 have been purified or crystallzed in presence of any phosphate ions.Therefore we cannot support the formerly made hypothesis that the formation of tetramers is caused bya phosphate- or pyrophosphate-induced process (Quesada-Soriano et al., 2014).

![](images/2dc7242950bae699f3a4e03ffa1f0d40c4f810bb5e7637e2a71c20b2da220001.jpg)  
ig6 positionsof Phel16revealamaximal displacement of more than $1 0 { \mathring { \mathsf { A } } } .$ Theconformationof thetruncatedloopin $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ is shown in green.

![](images/42dd7fe55e8862ee8c0d8bf6082f8320ff3be5ebbbb175424fe1ecf1151dbf93.jpg)  
Fig.7.The near UV spectrum of the PfGST with $1 0 0 \mathrm { m M }$ MES (blue)shows adecrease in the tyrosine $( 2 7 5 - 2 9 0 \mathrm { n m }$ )and phenylalanine $( 2 6 0 - 2 7 5 \mathrm { n m }$ )region compared to the defined near UV spectrum of the PfGST (red),while the tryptophan region $( 2 9 0 - 3 2 0 \mathrm { n m }$ )remains unaffected.The corridor(in light blue and light red) represents the standard deviation of the data points.

Our data may also explain why the tetramerization in absence of GSH is 100 times slower for $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ than observed for the wild-type enzyme (Liebau et al.,2Oo9).The interface between two active dimers forming the tetramer of $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ differs significantly in comparison to the tetrameric state of the wild-type PfGST (Fig. 5). The truncation of $\log ^ { 1 1 3 - 1 1 9 }$ results in a drastic reduction of direct intramolecular interactions within the tetramer (see Table 3). For the $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ tetramer the surface area between the non-active dimers covers only 331 $\mathring { \mathsf { A } } ^ { 2 }$ in comparison to $1 . 1 2 1 \mathring { \mathsf { A } } ^ { 2 }$ for the wild-type enzyme. No direct hydrogen bonds are formed between the subunits (Table 3)，and the hydrophobic interactionsare mediated byresiduesPhe1o,Phe42 and Phe45 from the first subunit and Leu124 and Tyr176 from the corresponding subunit (Fig. 5B).

Remarkably,Phe42 and Phe45 are part of,or flank the most characteristic Mu-loop(Fig.4),which is located between $\beta \cdot$ -sheet 2 and $\textsf { \textsf { a } }$ -helix2 (Ji et al.,1992).For PfGST,the Mu-loop is formed byresidues 35-42 and is significantly smaller in comparison with more typical Mu-class GSTs.The side chains of the Mu-loop form part of the electrophilic substrate-binding H-site.Only the truncated Mu-loop in PfGST enables the formation of the tetrameric inactive complex.An extended loop would probably sterically interfere with the tetramer-complex formation. One can propose that the replacement of the typical Mu-loop by a short turn in PfGST is an evolutionary development to enable the unique dimer-tetramer transition observed for PfGST.

For $P f G S \mathrm { T } ^ { \Delta 1 1 4 - 1 1 8 }$ the substantial reduction of direct interactions is mainlycaused by the truncated loop13-19(Fig.2).For the wild-type PfGST the intact loop mediates the majority of hydrophobic interactions between the subunits (Fig.5and

Table3).However the reduced number of interactions is partially compensated by GSSG-mediated interactions between the subunits.Expectably,GSSG is tightly bound by hydrogen bonds and hydrophobic interactions mediated by G-site residues of one subunit(Fig.3).Additional hydrogen bonds formed by Thr121 and Lys175 from the other subunit stabilize the tetrameric state.In summary one can postulate that in tetramer formation GSSG has theeffectofanadhesive.

We can assume that the oxidation of GSH to GSSG facilitates the tetramerization of the mutant. Further,only the truncated loop provides sufficient space to allow GSSG binding in the tetrameric state.The direct structural comparison(Fig.5）of a tetramer of the mutant with the tetramer of the wild-type enzyme reveals that GSH or other specific ligands are not able to bind to tetramers of thewild-tyeenbecauseaminacdsofo319ouy major parts of the G-site as well as H-site in the neighboring molecule.

The comparison of all three structures $( P f G S \mathsf { T } ^ { \mathsf { a p o } }$ $P f G S \mathrm { T } ^ { \mathrm { G S H } }$ and $P f G S T ^ { \Delta 1 1 4 - 1 { \bar { 1 } } 8 }$ reveals that loop 113-119 exhibit diferent conformations in the active dimeric state to the inactive tetrameric state (Fig.6). This observation is in good agreement with other structures already deposited in the Protein Data Bank (PDB entry codes 2AAW and 1OKT). Since major parts of the loop are missing in $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ but nevertheless stable tetramers are formed, we conclude that conformational changes in the loop region113-119 are the effect but not the reason of tetramerization.

In summary,it is obvious that inhibitors tightly bound to the PfGST will prevent the dimer-tetramer transition due to sterical hindrance and inhibitor binding to the G- or H-site and tetramerization are therefore competitive processes.

# 3.4.Identificationofanon-substrate-bindingpocket

Surprisingly the obtained wild-type PfGST structure determined at a resolution of $1 . 7 \mathring { \mathsf { A } }$ clearly revealed a non-substrate binding pocket occupied bya MES molecule forall monomers.The localization of ligand-binding-sites for several classes of GSTs is controversially discussedand remainsunclearand inconsistent (Axarli etal., 2004; Ji et al., 1996; McTigue et al., 1995; Oakley et al., 1999).

![](images/0ebc5756d04bce89c1c04bd7e26404a0e392b6b2b3074a4517f2377eecca255d.jpg)  
Fig8（ido et T5 superimposed on a $F _ { \mathrm { o } } - F _ { \mathrm { c } }$ difference electron density map,countered at $2 . 5 \sigma$ ,indicating that it is tightly bound within the binding pocket.

The specific binding of the MES molecule to GSTs in solution was further analyzed by spectroscopic fluorescence methods (Prade et al.,1997). Indeed, the near UV measurements by CD spectroscopy showed a decreased absorbance in the tyrosine and phenylalanine spectra compared to a defined near UV spectrum of the PfGST without MES(Fig.7).PfGST altogether possesses 16 tyrosine residues，but the crystal structure reveals that only Tyr25 is shielded by the MES molecule.This circumstance explains the weak difference of the absorbance in the near UV spectra. Therefore,the solution experiments are in good agreement with the results of the crystallographic analysis.

The binding pocket is formed by the residues Tyr25,Leu26, Leu196,Pro197 and Asn198.The binding mode of the MES moleculesis particular stabilized bya hydrogen bond mediated by the amide nitrogen from Asn198(Fig.8).For PfGST the region around the amino acids Leu196,Pro197,Asn198 and Ile199 is strongly positively charged.The negativelycharged ethane sulfonic moiety of the MES molecule forms a hydrogen bond to Asn198.Additional hydrophobic interactionsare mediated by Leu26,Leu196,Pro197 and Tyr25.

The novel L-site is partof the outer end of a large hydrophobic groove,spanning transversely through the surface of the PfGST monomer (Fig.8). This hydrophobic groove is composed of 11 hydrophobic residuesand flanked by1O hydrophilic residues and extends to about $2 0 \mathring { \mathsf { A } }$ and a width of $1 2 { \mathring { \mathsf { A } } } .$ This accessible mainly hydrophobic groove is also present in PfGST△114-118.

# 4. Conclusions

The high resolution structural analysis of $P f G S T ^ { \Delta 1 1 4 - 1 1 8 }$ in complex with GSSG clearly showed that loop 113-119 is not essential to form stable tetramers as reported before.The loop region 114- 118 (Asn114,Leu115,Phe116, $\mathsf { L y s } ^ { 1 1 7 }$ and $\mathsf { G l u } ^ { 1 1 8 }$ )was replaced and shortened by two alanine residues $( P f G S T ^ { \Delta 1 1 4 - 1 1 8 } ,$ . The structural data shown here indicate that the loop reduction can be partially compensated by GSSG mediated interactions between the subunits.In summary,GSsG hasan adhesive effect and supports the tetramer formation. Evidently conformational changes of the loop region are not the reason but the cause of the tetramerization. The binding of ligands to the $G -$ orH-site and the dimer-tetramer transition seem to be competitive processes.Furthermore,we identified a novel non-substrate-binding site of the wild-type PfGST structure in complexwith 2-(N-morpholino)ethanesulfonic acid (MES).The localization of ligandin-binding-sites is still controversially discussed and remains unclear and inconsistent.The identified binding site reveals new structural information about the so farunexploited PfGST L-site.

# Acknowledgements

This work was supported by grants from the Deutsche Forschungsgemeinschaft (German-African Cooperation Projects in Infectiology-PAK296)，by the excellence cluster ‘The Hamburg Centre for Ultrafast Imaging - Structure,Dynamics and Control of Matter at theAtomicScale'oftheDeutsche Forschungsgemeinschaft,by the Hamburg Ministry of Science and Research and Joachim Herz Stiftung as part of the Hamburg Initiative for Excellence in Research,the Hamburg School for Structure and Dynamics in Infection (SDI) and by DLR via grant 50WB1017.

References   
Axarli,I.A., Rigden, D.J., Labrou, N.E., 2Oo4. Characterization of the ligandin site of maize glutathione S-transferase I. Biochem. J. 382, 885-893.   
Brunger, A.T.,1993. Assessment of phase accuracy by cross validation: the free R value. Methods and applications. Acta Crystallogr. D Biol. Crystallogr. 49, 24-36.   
Burmeister,C.,Perbandt,M, Betzel, C.,Walter,RD,Liebau,E.,2003.Crystalliation and preliminary X-ray diffraction studies of the glutathione S-transferase from Plasmodium falciparum. Acta Crystallogr. D Biol. Crystallogr. 59, 1469-1471.   
Cardoso, R.M., Daniels, D.S., Bruns, C.M., Tainer, J.A., 2oo3. Characterization of the electrophile binding site and substrate binding mode of the 26-kDa glutathione S-transferase from Schistosoma japonicum. Proteins 51,137-146.   
CCP4,1994.The CCP4 suite: programs for protein crystalography. Acta Crystalogr. D Biol. Crystallogr. 50, 760-763.   
Deponte, M., Becker, K., 2005. Glutathione S-transferase from malarial parasites: structural and functional aspects. Methods Enzymol. 401, 241-253.   
Dirr,H.，Reinemer，P.,Huber，R.，1994. X-ray crystal structures of cytosolic glutathione S-transferases. Implications for protein architecture, substrate recognition and catalytic function. Eur. J. Biochem. 220, 645-661.   
Fritz-Wolf,K.,Becker,A., Rahlfs,S., Harwaldt,P.,Schirmer,R.H, Kabsch,W.,Becker K., 2003. X-ray structure of glutathione S-transferase from the malarial parasite Plasmodium falciparum. Proc. Natl. Acad. Sci. U.S.A. 100, 13821-13826.   
Frova, C., 2oo6. Glutathione transferases in the genomics era: new insights and perspectives. Biomol. Eng. 23, 149-169.   
Greenwood, B., Mutabingwa, T.,2002. Malaria in 2002. Nature 415, 670-672.   
Harwaldt, P.,Rahlfs, S., Becker, K., 2Oo2. Glutathione S-transferase of the malarial parasite Plasmodium falciparum: characterization of a potential drug target. Biol. Chem. 383,821-830.   
Hayes, J.D., Flanagan, J.U., Jowsey,LR.,2Oo5. Glutathione transferases. Annu. Rev. Pharmacol. Toxicol. 45, 51-88.   
Hiller,N.,Fritz-Wolf,K.,Deponte,M., Wende,W.,Zimmermann,H.,Becker,K.,2006. Plasmodium falciparum glutathione S-transferase-structural and mechanistic Studies on ligand binding and enzyme inhibition. Protein Sci. 15, 281-289.   
Ji,X.，Zhang，P.，Armstrong，R.N.，Gililand,G.L.，1992. The three-dimensional structure of a glutathione S-transferase from the mu gene class. Structural analysis of_the_binary complex_of isoenzyme 3-3 and glutathione at 2.2-A resolution. Biochemistry 31, 10169-10184.   
Ji, X.,von Rosenvinge,E.C., Johnson,W.W.,Armstrong,R.N., Giliand,G.L,1996. Location of a potential transport binding_site in a_sigma class glutathione transferase by x-ray crystallography. Proc. Natl. Acad. Sci. U.S.A. 93, 8208-8213.   
Kabsch, W.,0000. Acta Crystallogr. D Biol. Crystallogr.66,125-132.   
Laskowski, R.A.,MacArthur, M.W., Moss, D.S., Thornten, J.M.,1993. PROCHECK: a program tocheck the stereochemical quality of protein structures.J. Appl. Cryst. 26, 283-291.   
Laskowski, R.A., Hutchinson, E.G., Michie, A.D., Wallace, A.C., Jones, ML., Thornton, JM., 1997. PDBsum: a Web-based database of summaries and analyses of all PDB structures. Trends Biochem. Sci. 22, 488-490.   
Liebau,E.,Bergman,B.Cmpbel,A.M,dale-Spitle,P.rophy,P, K.Walter，R.D.， 2002.TheglutathioneS-transferasefromPlasmodium falciparum. Mol. Biochem. Parasitol. 124, 85-90.   
Liebau,E.，De Maria,F.， Burmeister,C.， Perbandt, M., Turela，P., Antonini， G., Federici, G., Giansanti, F.,Stela,L.,LoBello,M, Caccuri, A.M, Ricci, G.,2005. Cooperativity_and pseudo-cooperativity in_the glutathione S-transferase from Plasmodium falciparum. J. Biol. Chem. 280, 26121-26128.   
Liebau,E.,，Dawood，K.F.，Fabrini，R.，Fischer-Riepe,L.，Perbandt，M.， Stella,L., Pedersen,J.Z,Bocedi,A,Petrarca,P.,Federici,G., Ricci,G.,20o9.Tetramerization and_cooperativity in Plasmodium falciparum glutathione S-transferase are mediated by atypic loop 113-119. J. Biol. Chem. 284, 22133-22139.   
Manervik， B.， Danielson, U.H.，1988. Glutathione transferases-structure and Catalytic activity. CRC Crit. Rev. Biochem. 23, 283-337.   
McTigue, M.A., Wiliams,D.R.,Tainer,JA.,1995. Crystal structures of a schistosomal drug_and vaccine target: glutathione S-transferase from Schistosoma japonica and its complex with the leading antischistosomal drug praziquantel. J. Mol. Biol. 246, 21-27.   
Murshudov, G.N, Vagin, A.A.,Lebedev, A.,Wilson, K.S.,Dodson,E.J.,1999.Eficient anisotropicrefinementofmacromolecularstructuresusingFT.Acta Crystallogr. D Biol. Crystallogr. 55,247-255.   
Oakley，A.J,Belletelli,,azei，A.akerW999 ligandin (non-substrate) binding site of human Pi class glutathione transferase is located in the electrophile binding site (H-site). J. Mol. Biol. 291, 913-926.   
Olliaro,P.,2001. Mode of action and mechanisms of resistance for antimalarial drugs. Pharmacol. Ther. 89, 207-219.   
Otwinowski, Z., Minor, W.,1997. Processing of X-ray diffraction data collected in oscillation mode. Methods Enymol. 276, 307-326.   
Perbandt, M., Burmeister, C., Walter,R.D, Betzel, C., Liebau,E.,2004. Nativeand inhibited structure of_aMu_clas-related_glutathione S-transferase from Plasmodium falciparum. J. Biol. Chem. 279,1336-1342.   
Perez，B.C.， Teixeira,C.， Aibuquerque,I.S.， Gut, J.,， Rosenthal， P.J.， Gomes， JR., Prudencio, M., Gomes, P. 2012. N-cinnamoylated chloroquine analogues as dual-stage antimalarial leads. J. Med. Chem.   
Perrakis,A.Morrs,，RLamzin, V.S.1999.Automated_proteinmodelbuilding combined with iterative structure refinement. Nat. Struct. Biol. 6, 458-463.   
Prade,L., Huber,R.,Manoharan,T.H.,Fahl, W.E.,Reuter,W.,1997.Structures ofclass pi glutathione S-transferase from human placenta in complex with substrate, transition-state analogue and inhibitor. Structure 5,1287-1295.   
Preu,P.， Braun，M.，2O14. German SIMBOX on Chinese mission Shenzhou-8: Europe's first bilateral cooperation utilizing China's Shenzhou programme. Acta Astronaut. 94, 584-591.   
Quesada-Soriano,I., Baron, C., Tellez-Sanz, R., Garcia-Maroto,F., Garcia-Fuentes, L., 2014.Asn112 in Plasmodium falciparum glutathione S-transferase is essential forinduced reversible tetramerization by phosphate or pyrophosphate. Biochim. Biophys. Acta.   
Raghunathan, S., Chandross,R.J., Kretsinger, R.H., Allison, T.J., Penington, C.J., Rule, G.S.,1994. Crystal structure of human class mu glutathione transferase GSTM2- 2. Effects of lattice packing on conformational heterogeneity. J. Mol. Biol. 238, 815-832.   
Reinemer,P.,Prade,L., Hof,P., Neuefeind,T., Huber, R., Zettl, R., Palme,K., Schell,J, Koelln,I., Bartunik, H.D.，Bieseler,B.，1996. Three-dimensional structure of

glutathione S-transferase from Arabidopsis thaliana at 2.2 A resolution: structural characterization of herbicide-conjugating plant glutathione Stransferases and a novel active site architecture. J. Mol. Biol. 255,289-309. Sinning, I., Kleywegt, G.J., Cowan, S.W., Reinemer,P.,Dirr, H.W., Huber, R., Gilliland, G.L., Armstrong, R.N., Ji, X., Board, P.G., et al.,1993. Structure determination and refinement of human alpha class glutathione transferase A1-1，and a comparison with the Mu and Pi class enzymes. J. Mol. Biol. 232,192-212. Tripathi,T., Rahlfs,S., Becker, K., Bhakuni,V.,2oo7. Glutathione mediated regulation of oligomeric structure and functional activity of Plasmodium falciparum glutathione S-transferase. BMC Struct. Biol.7, 67. Wallace, A.C., Laskowski, R.A., Thornton,J.M.,1995. LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions. Protein Eng. 8, 127-134. WHO.2010.World Malaria Report.www.who.int.