# Proteomic Comparison and MRM-Based Comparative Analysis of Metabolites Reveal Metabolic Shift in Human Prostate Cancer Cell Lines

Qingbo Shu,‡⊥ Tanxi Cai, Xiulan Chen, Helen He_Zhu,§ Peng Xue, Nali Zhu, Zhensheng Xie,† Shasha WeiQing Zhang Lili NiuWei-Qiang Gaolland FuquanYang

+Laboratory of Proteinand Peptide Pharmaceuticals &Laboratory of Proteomics, Institute of Biophysics， Chinese Academy of   
Sciences,Beijing 100101,China   
\*University of Chinese Academy of Sciences, Beijing 100049, China   
\$State KeyLaboratoryofcogenesandRelatedGenes,RenjedXCliicalStemCellesearchCnterRenJispitalhlf   
Medicine, Shanghai Jiao Tong University， Shanghai 200127,China   
ISchool of Biomedical Engineeing& Med-XResearch Institute,Shanghai Jiao Tong University,Shanghai 20oo30China

S Supporting Information

ABSTRACT: One of the major challenges in prostate cancer therapy remains the development of effective treatments for castration-resistant prostate cancer (CRPC),as the underlying mechanisms for its progression remain elusive.Previous studies showed that androgen receptor (AR) is crucially involved in regulation of metabolism in prostate cancer (PCa) cells throughout the transition from early stage, androgen-sensitive PCa to androgen-independent CRPC. AR achieves such metabolic rewiring directively either via its transcriptional activity or via interactions with AMP-activated protein kinase (AMPK). However, due to the heterogeneous expression and activity status of AR in PCa cells,it remains a challenge to investigate the links between AR status and metabolic alterations. To this end， we compared the proteomes of three pairs of androgen-sensitive (AS) and androgenindependent (AI) PCa cell lines， namely， PC3-AR+/PC3, 22Rv1/Dul4S,and LNCaP/C42B, using an iTRAQ labeling approach. Our results revealed that most of the differentially expressed proteins between each pair function in metabolism, indicatinga metabolic shift between AS and AI cells,as further validated by multiple reaction monitoring (MRM)-based quantification of nucleotides and relative comparison of fatty

![](images/371cc140ba619f1c60d89d16a181e2f625b22e3c07f85f5278af3a5b8a0fdcda.jpg)

acidsbetweenthesecelines.Furthermore,increasedadenylateinaseisoenze1(AK1)inASrelativetoIcellsmasultir activation of AMPK, representing a major regulatory factor involved in the observed metabolic shift in PCa cels.

KEYWORDS: quantitative proteomics， metabolite, metabolic shift, androgen receptor， AMPK, CRPC

# INTRODUCTION

Prostate cancer (PCa) is the most prevalent cancer among males.' PCa patients are initially treated primarily with androgen-deprivation therapy (ADT).However，most cases of advanced prostate cancer progress to castration-resistant prostate cancer(CRPC） after a period of ADT.Due to the limited use of chemotherapy in the management of CRPC, there remains an urgent need for effective therapy that delays or prevents the landmark events that characterize the morbidity associated with this cancer.2

PCa is very heterogeneous in terms of grade, genetics, ploidy, and oncogene/tumor suppressor gene expression， and its biological，hormonal，and molecular characteristics are extremely complex.34 Previous studies on clinical samples and xenograft models of PCa indicate that abnormal activation of androgen signaling plays a pivotal role in the progression of CRPC.' As a result, the sensitivities of different PCa cellines to androgen are heterogeneous.The differing androgen sensitivities of PCa cell lines are well-documented (http://capcelllines. $\mathsf { c a / }$ ） and represent the multiple stages of PCa progression.4 Cai et al. suggested that， in CRPC，partial restoration of the transcriptional activity of the androgen receptor (AR）allowed active expression of the genes that mediate androgen biosynthesis and maintained intracellular androgen level as well as AR activity.Recently,abiraterone acetate,an androgen biosynthesis inhibitor,was shown to improve overall survival in patients with CRPC after chemotherapy and also showed a trend toward improved overall survival in patients who had not received previous chemotherapy.

Advanced CRPC patients treated with abiraterone acetate generally relapse within a year,and AR appears to be active in the relapsed tumors. AR signaling has been shown to cross-talk with alternative survival pathways，such as PI3K/AKT and MAPK pathways.9 This may partially explain the resistance to AR-targeted therapies in CRPC.However,interactions between AR signaling and these survival pathways are cell-type- and context-dependent.1° Hence, conclusions based on one PCa cell line could not be directly transferred to all other PCa cell lines or applied to clinical situations.

Here，we set out to obtain a systematic and unbiased understanding of the mechanisms underlying CRPC progression.To this end,we used the iTRAQ labeling approach and performed pairwise comparisons between three androgensensitive (AS）and three androgen-independent (AI） PCa cell lines.Also,multiple reaction monitoring(MRM) analyses were applied to targeted metabolites in order to validate the metabolic changes assumed.Our results indicate that a metabolic shift and distinct forms of cell adhesion existed between AS and AI PCacells.

# EXPERIMENTALSECTION

# Cell Lines and Reagents

Prostate cancer cell lines PC3，Dul45，C42B，22Rvl，and LNCaP were purchased from the American Type Culture Collection (USA).LNCaP cells were maintained in RPMI1640 supplemented with $1 0 \%$ fetal bovine serum (FBS)，100 $\mathrm { { U } / \mathrm { { m L } } }$ penicillin, and $1 0 0 ~ \mathrm { { m g / m L } }$ streptomycin; the other cells were maintained in DMEM supplemented with $1 0 \%$ FBS,100 $\mathrm { U } / \mathrm { m L }$ penicillin, $1 0 0 ~ \mathrm { \ m g / m L }$ streptomycin，and $2 ~ \mathrm { m M }$ glutamine.The $\mathrm { P C } 3 { \cdot } \mathrm { A R } ^ { + }$ cell line，constructed by stable expressing of GFP-tagged AR in PC3 cells,was kindly provided by Prof. Gao Wei-qiang. Cell lines were cultured at $3 7 ~ ^ { \circ } \mathrm { C }$ in a humidified atmosphere containing $5 . 0 \%$ （204号 $\mathrm { C O } _ { 2 }$

Forty two methyl ester standards of fattyacids and 24 kinds of fatty acids were purchased from Cayman (Michigan, USA) and Sigma (Missouri, USA) as listed in Supporting Information Table S1.Eight standards for nucleotides and acetyl-CoA were purchased from Sigma and are listed in Table S1，and commonly used chemical reagents were purchased from Sigma. All chromatographic reagents were from Avantor Performance Materials (Pennsylvania，USA).Trimethylsilyl diazomethane was purchased from Acros Organics (Massachusetts, USA).

# Protein Extraction and Denaturing

Cells were washed with cold phosphate-buffered saline (PBS) twice and harvested by scraping in ice-cold lysis buffer ( $\mathbf { \delta } _ { \left\{ 8 \mathbf { \delta } \right. } \mathbf { M } $ urea, $1 0 0 ~ \mathrm { m M }$ Tris-HCl $\mathrm { \tt p H } 8 . 5 .$ ，and protease inhibitor cocktail (Roche,Basel, Switzerland)).Cells were sonicated on ice,and cell lysates were centrifuged at $1 6 0 0 0 g$ for $1 5 ~ \mathrm { m i n }$ at $4 ~ ^ { \circ } \mathrm { C }$ to remove cell debris. Supernatants were collected and stored at $- 8 0 ~ ^ { \circ } \mathrm { C }$ until use.Cell lysates were denatured according to the FASP procedure with minor modifications.1 Briefly, the cell lysates were transferred to Amicon-O.S ultrafiltration units,and the buffer was exchanged with UA buffer(8 M urea, $0 . 1 \mathbf { M }$ Tris, $\mathrm { p H } ~ 8 . 5 ) \$ ）three times.Protein samples were reduced with 10 $\mathbf { m } \mathbf { M }$ dithiothreitol at $3 7 ^ { \circ } \mathrm { C }$ for $^ \textrm { \scriptsize 1 h }$ and then alkylated with 50 mM iodoacetamide at room temperature for $^ \textrm { \scriptsize 1 h }$ (in the dark). The buffer of the denatured protein samples was then exchanged with UA buffer three times and exchanged with 50 mM tetraethylammonium bromide (TEAB）twice.Proteins were recovered byreverse centrifugation at $1 0 0 0 g$ for $2 ~ \mathrm { m i n }$ and resolved in $5 0 ~ \mathrm { ~ m M }$ TEAB.Protein concentrations were determined in triplicate using the bicinchoninic acid method (Thermo Scientific Pierce,Massachusetts,USA） according to the manufacturer's instructions.

# Protein Digestion and iTRAQ Labeling

Equal amounts of protein $( 3 2 0 ~ \mu \mathrm { g } )$ were digested with trypsin at $3 7 ~ ^ { \circ } \mathrm { C }$ overnight. Then, $8 0 ~ \mu \mathrm { g }$ of tryptic peptide was labeled with 8-plex iTRAQ reagent according to the manufacturer's instructions.The six cell lines were labeled as follows:PC3 with 113 tag, ${ \mathrm { P C } } 3 { \mathrm { - } } { \mathrm { A R } } ^ { + }$ with 114 tag,C42B with 115 tag,LNCaP with 116 tag,22Rvl with 117 tag and 119 tag (technical replicate)，and Dul45 with 118 tag and 121 tag (technical replication)．Following labeling，equal volumesof labeled samples were aliquoted and mixed to test the labeling efficiency by LC-MS/MS analysis.

# Peptide Fractionation Using High pH Reversed-Phase Liquid Chromatography

The labeled peptide samples were pooled and desalted using Sep-Pak Vac $\overline { { 1 \ : \mathrm { ~ c m } ^ { 3 } } }$ 0 $\mathrm { ( 1 0 0 ~ m g ) }$ C18 SPE cartridges (Waters, Massachusetts，USA).Briefly，the column was washed with bufferB ( $8 0 \%$ ACN,50 mM $\mathrm { N H _ { 4 } H C O _ { 3 } } ,$ )three timesand $1 \mathrm { m L }$ each time,then washed with buffer A $( 5 0 \ \mathrm { \ m M \ N H _ { 4 } H C O _ { 3 } } )$ three times and $1 \mathrm { m L }$ each time. Samples were loaded onto the column three times; the column was washed with bufferA three times and $1 ~ \mathrm { m L }$ each time,then eluted with $0 . 5 { \mathrm { ~ m L } }$ of $4 0 \%$ ACN, $5 0 ~ \mathrm { m M }$ （20 ${ \mathrm { N H } } _ { 4 } { \mathrm { H C O } } _ { 3 } ,$ and $0 . 5 \mathrm { \ m L }$ （ $6 0 \%$ ACN, $5 0 \ \mathrm { \ m M }$ ${ \mathrm { N H } } _ { 4 } { \mathrm { H C O } } _ { 3 } ,$ sequentially. The combined elute was concentrated in a CentriVap benchtop centrifugal vacuum concentrator (Labconco,Missouri, USA) at $3 0 ~ ^ { \circ } \mathrm { C }$ .The volume was taken to $3 3 ~ \mu \mathrm { L }$ with solution A used in peptide fractionation. Thirty microliters of the sample was loaded onto a YMC-Triart C18 basic reversed-phase liquid chromatography column $2 5 0 \times 4 . 6$ mm, ${ \mathsf { 5 } } \mu \mathrm { m }$ particles) (cat. no. TA12S05-2546WT,YMC,Kyoto, Japan).Peptides were separated ina binary buffer system of $2 \%$ ACN, $\mathrm { p H } 1 0$ 0 $\mathrm { \Phi _ { \mathrm { p H } } }$ adjusted with ammonium hydroxide,solution A),and $9 8 \%$ ACN (without $\mathrm { \tt p H }$ adjustment,solution B）in an Ultimate-30oo LC system (Thermo Scientific,Massachusetts, USA).The gradient of buffer B was set as follows: $0 - 5 \%$ for 1 min, $5 \%$ for $\mathsf { S } \operatorname* { m i n }$ P $8 \mathrm { - } 1 8 \%$ for $3 5 ~ \mathrm { { m i n } }$ P $1 8 - 3 2 \%$ for $2 2 { \mathrm { ~ m i n } } _ { , }$ $3 2 - 9 5 \%$ for $2 ~ \mathrm { m i n }$ ，and $9 5 \%$ for $7 ~ \mathrm { m i n }$ .Eluateswere collected every $8 4 \ \mathbf { s } ,$ and a total number of SO fractions were obtained. The first five fractions were discarded because most of the excess labeling reagent was eluted during these fractions.The remaining 45 fractions were combined into 15 fractions by merging fraction 1,16,31; fraction 2,17,32;and so on. Then, all fractions were dried ina vacuum concentrator and stored at $- 2 0 ~ ^ { \circ } \mathrm { C }$ until further analysis.

# LC-MS/MS Analysis

For each fraction, an $8 ~ \mu \mathrm { L }$ sample was injected under trapping conditions using an ekspert nanoLC40o system (AB Sciex,

Massachusetts，USA）equipped with the ChromXP nanoLC trap column (cat.no.805-00121，AB Sciex, Massachusetts, USA)and a ChromXP 3C18-CL-120 column ( $7 5 \mu \mathrm { m } \times 1 5 \mathrm { c m } ,$ （204号 $3 \ \mu \mathrm { m }$ particles）(cat.no.805-00120,AB Sciex, Massachusetts, USA).Liquid chromatography was interfaced to a tripleTOF $5 6 0 0 +$ instrument (AB Sciex，Massachusetts，USA）via a nanoelectrospray ionization source.After beingloaded,peptide trappingand desalting were carried out at $2 \mu \mathrm { L } / \mathrm { m i n }$ for $1 5 \mathrm { m i n }$ with $1 0 0 \%$ buffer A( $2 \%$ $\begin{array} { r l } { \mathit { \Pi } _ { 0 } } & { { } \mathrm { A C N } / 0 . 1 \% } \end{array}$ formic acid)．The analytical separation was established by running a linear gradient of buffer B( $9 8 \%$ $\mathbf { A C N } / 0 . 1 \%$ formic acid）ata flow rate of $3 0 0 \mathrm { \ n L } / \mathrm { m i n }$ from 10 to $2 7 \%$ in $8 0 \ \mathrm { m i n }$ .Then the gradient of buffer B was increased to $5 0 \%$ in $2 4 \ \mathrm { m i n }$ and decreased to $20 \%$ in $6 ~ \mathrm { m i n }$ . Initial chromatographic conditions were restored in $_ \mathrm { ~ 6 ~ s ~ }$ and maintained for $1 0 \ \mathrm { m i n }$ Data were acquired using an ion spray voltage of $2 . 3 \mathrm { k V } _ { ; }$ ,curtain gas of 30, gas 1of 4,gas 2 of O,and an interface heater temperature of 150 $^ { \circ } \mathrm { C }$ ，A rolling collision energy voltage was used for CID fragmentation for MS/MS spectra acquisitions. The option “adjust collsion energy(CE） for iTRAQ”was enabled.Each cycle consisted of a TOF MS spectrum acquisition for $2 5 0 ~ \mathrm { m s }$ (mass range $3 5 0 \mathrm { - } 1 5 0 0 ~ \mathrm { D a } ,$ ),followed by acquisition of up to 30 MS/MS spectra ( $\mathrm { { \dot { 1 } 0 0 } ~ m s }$ each, mass range $1 0 0 { - } 1 5 0 0 ~ \mathrm { D a }$ ).The total cycle time was fixed to $3 . 3 \ \mathsf { s } .$ Dynamic exclusion was set for $1 / 2$ of peak width.

cell cultureand sample preparation 8-plex iTRAQ labeling and mixing   
PC3 PC3-AR+ C42B LNCaP 22Rv1 Du145 22Rv1 Du145   
113 114 115 116 117 118 119 121 highpHRP fractionation RPLC-MS/MS   
PC3-AR+/PC3 LNCaP/C42B 22Rv1/Du145 Technical replicates (114/113) (116/115) (117/118,119/121)(117/119,118/121) Target Decoy differential proteome

# Protein ldentification and Quantification

The LC-MS/MS data for the identification and quantification of the global proteome were analyzed using ProteinPilot Software 4.5 (AB Sciex，Massachusetts，USA）against the Uniprot KB human database (released on June，2O13）with 88 771 entries,to which 175 commonly observed contaminants and all the reverse sequences were added. iTRAQ 8-plex labeling and cysteine carbamidomethylation was set as modifications.A $9 5 \%$ confidence (unused protein score $> 1 . 3$ ） was used for protein identification. The PDST_V3.708pE_- TEMPLATE(AB Sciex,Massachusetts，USA）was used to calculate $p$ values for each ratio.Ratio channels of $1 1 4 / 1 1 3$ 116/115，117/118，and 119/121 were assigned as“target". Ratio channels of 117/119 and 118/121 (technical replicates) were assigned as“decoy” (Figure 1).By assigning each ratio channel as either“target”or“decoy"，the PDST estimates target-decoy global false discovery rate (FDR） for the ratios according to their $p$ values.The $p$ value limit was set on the basis of target-decoy FDR analysis.Principally,a protein with a quantitative ratio differing from 1 and the ratio's $p$ value less than O.O1 is declared as differentially expressed between a pair of AS and AI cells.

# Method Validation forMRM Analysis

Method performance was evaluated for 42 fatty acid methyl ester(FAME） standards，eight nucleotides，and acetyl-CoA (Supporting Information, Table S1)，with respect to linearity and reproducibility. For each FAME standard, $\ge 0 . 1 \ \mathrm { \ m g / m L }$ stock solution was prepared in methanol and stored at $- 2 0 { } ^ { \circ } \mathrm { C }$ 车 From these stock solutions,mixtures of 42 FAMEs dissolved in hexane were prepared at 0.05,0.1,0.5,1,5,10, 50,and $1 0 0 ~ \mathrm { { n g / } }$ mL and analyzed using GC-MS.For each nucleotide and acetylCoA standard, $\mathbf { 1 } \mathbf { m } \mathbf { M }$ stock solution was prepared in water and stored at $- 2 0 ~ ^ { \circ } \mathrm { C }$ ，From these stock solutions,mixtures of the ninemetabolites dissolved in waterwere prepared at O.17,0.42, 0.83，1.67，8.33，and $8 3 . 3 ~ \mu \mathrm { M }$ and analyzed using LC-MS. Linearity was evaluated using the linear regression of peak areas with respect to concentration.In addition,FAME standards at

$0 . 5 ~ \mathrm { \ n g / m L }$ were run two times to determine inter-run quantitative reproducibility，as measured by the relative standard deviation (RSD).

# MRM-Based Relative Quantification of Fatty Acids

Fatty acids were extracted from PC3,Dul45,22Rvl,LNCaP, and C42B using a liquid-liquid extraction method.12 Briefly, cells were grown to confluence in $6 \mathrm { c m }$ dishes under conditions same as those in the proteomic experiment.After the medium wasremoved, cells were washed with ${ \mathfrak { s } } _ { \mathrm { m L } }$ of cold PBS before $3 0 0 ~ \mu \mathrm { L }$ of $0 . 9 \%$ KCl solution was added,and cells were then detached witha cell scraper.Deuterium-labeled C17:O fattyacid wasadded as an internal standard.Cells were transferred to an Eppendorf Safe-Lock $2 . 0 ~ \mathrm { \ m L }$ tube(cat.no.0030 120.094, Eppendorf, Hamburg， Germany).After adding $3 0 0 ~ \mu \mathrm { L }$ of methanol and the antioxidant reagent butylated hydroxytoluene 中 $0 . 0 1 \%$ final concentration),cells were incubated overnight at $- 2 0 ~ ^ { \circ } \mathrm { C }$ Then $1 ~ \mathrm { m L }$ of methyl tert-butyl ether (MTBE） was added,and the samples were vortexed thoroughly ona VortexGenie 2 mixer (cat.no.SI-0236,Scientific Industries，New York, USA) for $^ \textrm { \scriptsize 1 h }$ at room temperature for phase separation. After centrifugation at $1 8 0 0 0 g$ for $5 ~ \mathrm { m i n }$ at $4 \ ^ { \circ } \mathrm { C } ,$ the upper phase was collected and divided into two parts.One part (200 $\mu \mathrm { L }$ ）wasused for analysis of phospholipid-derived fatty acids (PFs).The other part was used for analysis of free fatty acids (FFs).Both parts were dried under vacuum.PFs were derivatizated according to the analytical procedure on the Web site of Avanti Polar Lipids company (http://www. avantilipids.com/index.php?option $_ { . = \mathrm { c o m } }$ _content&view $\ c =$ article&id $\lfloor = \rfloor$ 1693&Itemid $\scriptstyle \lfloor = 4 1 2$ ).Samples were dissolved in 1 mL of hexane.Fifty microliters of sodium methoxide $\left( 1 \mathbf { M } \right)$ was added and vortexed thoroughly. Samples were incubated at room temperature for $ { 5 }  { \mathrm { m i n } }$ and centrifuged at $1 6 0 0 ~ \mathrm { r p m }$ at 20 $^ { \circ } \mathrm { C }$ for $ { 5 }  { \mathrm { m i n } }$ . The upper phase was transferred to another SafeLock tube and vacuum-dried and then dissolved in $5 0 ~ \mu \mathrm { L }$ of hexane for analysis.For FF derivatization，trimethylsilyl (TMS)-diazomethane was used.55 Samples were resolved in methyl esterification buffer( $5 0 0 ~ \mu \mathrm { L }$ of MTBE/150 μL of $\mathrm { M e O H } / 6 . 5 \mu \mathrm { L }$ of $1 ~ \mathrm { ~ M ~ } \mathrm { ~ H C l } _ { , } ^ { \cdot }$ ）ina Safe-Lock tube.Fifty microliters of TMS-diazomethane ( $^ { 2 \mathrm { ~ M ~ } }$ in hexane）was added and mixed at a mild speed on a wheel for $3 0 ~ \mathrm { m i n }$ at room temperature.Sixmicroliters of glacial aceticacid wasadded to quench the reaction.After $1 ~ \mathrm { m i n }$ ， $1 2 5 ~ \mu \mathrm { L }$ of water was added and mixed thoroughly. Samples were centrifuged at $1 5 0 0 g$ for 3 min for phase separation. The upper phase was collected and dried under vacuum. Samples were then dissolved in $5 0 ~ \mu \mathrm { L }$ of hexane for analysis.Derivatized FFs and PFs were separated using Agilent 6890N gas chromatograph （Agilent, California, USA) equipped with a J&W HP-88 column ( $\mathrm { 1 0 0 ~ m \times 0 . 2 5 ~ m m }$ i.d. $\times ~ 0 . 2 ~ \mu \mathrm { m }$ film thickness）(cat.no.112-88A7E，Agilent, California, USA). The flow of helium carrier gas was set to 1 $\mathrm { m L } / \mathrm { m i n }$ $\left( 2 0 \ \mathrm { c m } / \mathrm { s } \right)$ ； the injector temperature was set to 260 ${ } ^ { \circ } \mathrm { C } ,$ and the temperature of the sample transfer line was set to $2 4 0 ~ ^ { \circ } \mathrm { C }$ ，Samples were analyzed in splitless mode，and the gradient started from $5 0 \ { } ^ { \circ } { \mathrm { C } } ,$ held for $1 ~ \mathrm { m i n }$ ，thenincreased by $2 5 ~ { ^ \circ } \mathrm { C } / \operatorname* { m i n }$ to ${ 1 7 5 } ^ { \circ } \mathrm { C } ,$ further by $1 \ { ^ { \circ } { \bf C } } / { \mathrm { m i n } }$ to 216 and $4 ~ ^ { \circ } \mathrm { C } /$ min to $2 4 0 \ ^ { \circ } \mathrm { C } ,$ then held for $1 0 \ \mathrm { \ m i n }$ . Mass spectra were collected on an Agilent 70ooB mass selective detector (Agilent, California, USA)，using a chemical ionization source in the positive mode.Each cell line was analyzed in biological triplicate.A washing run was added with every three samples.In order to eliminate the influence of fatty acid contaminations, three solvent blanks were treated in parallel with samples,and the mean values of the peak areas of solvent blanks were subtracted from each sample.In addition,a mixture of 24 fatty acid standards(Table S3)dissolved in methanol was prepared at ${ \mathfrak { s n g } } / { \mathrm { m L } } ,$ derivatized with TMS-diazomethane,and used for recoveryrate evaluation.

# MRM Quantitation of Nucleotidesand Acetyl-CoA

Nucleotides and acetyl-CoA were extracted from five cell lines (PC3，Dul45，22Rvl，LNCaP，and C42B)，according to a method published earlier.13Briefly，cels were grown to confluence ( ${ \bf \Pi } _ { \sim 1 0 ^ { 7 } }$ cells）in $1 0 \ \mathrm { c m }$ dishes under conditions that were the same as those in the proteomic experiment,and cells were washed twice with ice-cold PBS. Then $1 0 0 \mu \mathrm { L }$ of icecold water was added into the $1 0 \ \mathrm { c m }$ dish,which was then flash frozen at $- 8 0 { } ^ { \circ } \mathrm { C }$ for about $3 0 ~ \mathsf { s }$ .Cellsweredetached witha cell scraper on ice and transferred into $1 . 5 ~ \mathrm { m L }$ Eppendorf tubes. Cell suspensions were lysed by two rapid freeze-thaw cycles at $- 8 0 ^ { \circ } \mathrm { C } ,$ followed by $3 0 ~ \mathsf { s }$ of sonication on ice. The volume was made up to $4 0 0 ~ \mu \mathrm { L }$ with ice-cold water,and each sample was then divided equally into two Eppendorf tubes.Five hundred microliters of cold methanol and $2 5 0 ~ \mu \mathrm { L }$ of chloroform were added to the homogenates and vortexed. For safety consideration，addition of chloroform as well as other toxic reagents and sample transfer were always performed ina fume hood. Samples were mixed for $1 0 ~ \mathrm { m i n }$ at $4 ~ ^ { \circ } \mathrm { C }$ and centrifuged for $5 ~ \mathrm { m i n }$ at $2 0 0 0 g$ at $4 ~ ^ { \circ } \mathrm { C }$ to remove debris and precipitate protein. The pellet was dissolved in $^ { 8 \mathrm { ~ M ~ } }$ urea, $0 . 1 \mathrm { ~ M ~ }$ Tris, $\mathrm { \tt p H }$ 8.5,and protein content was quantified using a BCA kit.The supernatant was transferred to a new tube,and $2 5 0 ~ \mu \mathrm { L }$ of chloroform and $2 5 0 ~ \mu \mathrm { L }$ of water were added. Samples were vortexed for $^ { 6 0 \ s , }$ kept on ice for $1 0 ~ \mathrm { m i n }$ for phase separation, and centrifuged for $1 0 \ \mathrm { m i n }$ at $2 0 0 0 g$ at $4 ~ ^ { \circ } \mathrm { C } .$ ，The resulting upper polar and lower nonpolar fractions were separately transferred to Eppendorf tubes，vacuum-dried,and combined after being dissolved in $3 0 ~ \mu \mathrm { L }$ of LC buffer A $\mathit { \Omega } ^ { \prime } 5 \ : \mathrm { \ m M }$ ammonium acetate/ $0 . 1 \%$ FA).A combined volume of a ${ \sim } 6 0$ $\mu \mathrm { L }$ sample was obtained and centrifuged，and a $5 \mu \mathrm { L }$ centrifuged samplewasanalyzed onan Agilent 646OLC-Triple

Quadruple MS(Agilent,California,USA) with an Acclaim 120 C18 RP column ( $( 3 \times 1 5 0 ~ \mathrm { m m } ,$ $3 \ \mu \mathrm { m }$ particles）(cat.no. 063691,DIONEX,California,USA).Metaboliteswere eluted from the column with buffer B $( \mathrm { A C N } / 5 \ \mathrm { \ m M }$ ammonium acetate/ $0 . 1 \%$ FA),and buffer A was $\boldsymbol { 5 } \mathrm { m M }$ ammonium acetate/ $0 . 1 \%$ FA.The gradient of buffer B started at $2 \%$ and was kept for $1 \mathrm { m i n }$ and then increased from 2 to $3 0 \%$ in $9 \mathrm { m i n }$ and from 30 to $8 0 \%$ in $\textit { 5 } \operatorname* { m i n }$ . Then，the gradient of buffer B was decreased to $2 \%$ in $2 ~ \mathrm { m i n }$ and kept for $1 3 ~ \mathrm { m i n }$ . The total LC time was $3 0 ~ \mathrm { \ m i n }$ ，with a flow rate of $0 . 5 ~ \mathrm { \ m L / m i n }$ .Gas temperature was set at $3 2 5 ^ { \circ } \mathrm { C } ,$ and gas flow was $7 ~ \mathrm { L } / \mathrm { m i n }$ Sheath gas temperature was set at $3 2 5 ^ { \circ } \mathrm { C } ,$ and sheath gas flow was $1 1 ~ \mathrm { L / m i n }$ .Capillaryvoltagewas $3 5 0 0 ~ \mathrm { V } ,$ and the nozzle voltage was $5 0 0 \mathrm { V } .$ Delta $\mathrm { E M V } ( + )$ was 20o.Each cell line was analyzed in biological triplicate.

# Mitochondrial Staining and Activity Analysis

Mitochondrial morphology imaging was conducted with Mitotracker Green (Life Technologies,Massachusetts,USA) as described by Chen et al.14 Cells were seeded in $3 5 ~ \mathrm { m m }$ Petri dishes,and after reaching $5 0 \mathrm { - } 7 0 \%$ confluence,the cells were stained with $5 0 \ \mathrm { ~ n M }$ Mitotracker Green diluted in the corresponding culture media for $2 0 ~ \mathrm { \ m i n }$ at $3 7 ^ { \circ } \mathrm { C }$ After incubation，cells were washed twice with PBS，stained with Hoechst 33342(Life Technologies,Massachusetts,USA) for 5 min，and washed three times with PBS. Cell images were obtained with an Olympus FV1ooo laser scanning confocal microscope (OLYMPUS,Tokyo, Japan).

XF24 Extracellular Flux analyzer (Seahorse Bioscience, Massachusetts，USA） was used to detect the extracellular acidification rate (ECAR） and oxygen consumption rate (OCR)of different cell lines.The medium used for the OCR assay was XF assay medium (cat.no.l02352-0o0,Seahorse Biosciences)，with $2 5 ~ \mathrm { m M }$ glucose and $2 ~ \mathrm { m M }$ pyruvate added, $\mathrm { p H } 7 . 3 5 \pm 0 . 0 5$ （20 $( 3 7 ^ { \circ } \mathrm { C } )$ . The medium used for the ECAR assay was DMEM (cat. no.D5030, Sigma), with $1 4 3 \mathrm { m M N a C l } ,$ （20 $3 \mathrm { m g }$ phenol red,and $2 ~ \mathrm { m M }$ L-glutamine added, $\mathrm { p H } 7 . 3 5 \pm 0 . 0 5$ (37 ${ } ^ { \circ } \mathrm { C } \ '$ ）

# Data Analysis

Enrichment analysis for gene ontology(GO) terms and protein interaction network construction was performed using STRING. The $p$ values were corrected for multiple testing using the method by Benjamini and Hochberg,and the top 10 significant categories were presented according to their $p$ values.Pathway mapping was executed using the Reactome database.The Matlab software (version 20iOb,MathWorks) was used for hierarchical clustering analysis.Proteins were clustered using unsupervised hierarchical average linkage clustering and a Euclidian distance metric，based on their logarithm-transformed ratios.

For fatty acids,peak areas were integrated with MassHunter software (Agilent, California, USA).56 After blank subtraction, internal standard correction (this correction was only applied for FFs),and protein content correction, the values were log transformed and $z$ -score-normalized and then clustered using unsupervised hierarchical average linkage clustering and a Euclidian distance metric. Fatty acids with a signal-to-noise ratio less than 1O in any sample were excluded for further analysis.For nucleotides and acetyl-CoA，the concentrations were calculated directly via the standard curves built with MassHunter software.After correction for protein content, the contents of nucleotides and acetyl-CoA among different samples were compared using the student's t-test with a $p$ value of $< 0 . 0 5$ as the significance level.

![](images/d9feacdd7e15f6a38186aef8b677241ce491e726ce0f325dd5f79bf8a2c3e0b2.jpg)  
Figure2.Dcsoffeallexpdprosioclsdootatfte) Hierarchicalsebrredoftialsdoteisf fourclusterssoilsetd.）Veitstistioftellxpdos threpirsotsaosst which a is $\mathrm { P C } 3 { \cdot } \mathrm { A R } ^ { + } / \mathrm { P } \bar { \mathrm { C } } 3$ ，b is 22Rvl/Du145,andcis LNCaP/C42B.(C）Enrichment analysis of gene ontology annotations for the 191 difereialysrotstatgditsttsofCacelllocalpossCaaio

# Safety Considerations

All procedures involving toxic reagents should be performed in a fume hood,and the waste produced must be recovered under specific recovery processing. The TMS-diazomethane was originally developed as a less toxic substitute for the compound diazomethane.However，inhalation of TMS-diazomethane may cause lung damage or central nervous system depression and should therefore be handled with the appropriate safety procedures in place,that is,insidea fume hood,with adequate personal safety equipment.Excess TMS-diazomethane is to be neutralized using acetic acid. Care must be taken if handled in large volumes,as it is highly volatile and produces nitrogen gas.

# RESULTSANDDISCUSSION

# Differentially Expressed Proteins between AS and Al Cells

Inorder to identify the differentially expressed proteins between the three pairs of PCa cels，we performed 8-plex iTRAQ labeling and used a target-decoy global FDR method to set a cutoff $p$ value for the ratios of quantified proteins.A total number of 4953 proteins were identifed with $1 \%$ global FDR at the protein level, among which 4847 proteins were quantified (Table S2).When setting the cutoff $p$ value below 0.01, the target-decoy global FDR for quantification was $2 \%$ .In total, 528 proteins showed at least one quantitative ratio with a $p$ value less than O.O1 (Figure S1). By applying hierarchical clustering to the quantitative ratios of the S28 proteins,we found that the changes of these differentially expressed proteins were heterogeneous among the three pairs of AS and AI cell lines (Figure 2A).Among the S28 proteins,191 were differentially expressed in at least two pairs,with 72 proteins increased,76 decreased,and 43 proteins changed in both directions in AI relative to AS cells (Figure 2B). The other 337 proteins were found to be differentially expressed in only one pair (Table S4). Only 19 proteins among the differentially expressed proteins were changed in the same direction in all of the three pairs, among which 1O were increased and 9 were decreased in AI relative to AS cells (Figure 2B and Table 1).Two of the 19 proteins have been shown previously to be critically involved in cancer development.They are major vault protein (MVP） and poly[ADP-ribose]polymerase 1 (PARP-1).MVP was increased, andPARP-1 was decreased in AI relative to AS cells.MVP has been reported to promote cancer progression by mediating epidermal growth factor receptor (EGFR)/PI3K-mediated migration and survival pathways in human glioblastoma.15 Intriguing，abnormal activation of the PI3K pathway isa common event during PCa progression.1 Here,we also found higher expression of EGFR in AI relative to AS cells (Table S4). Thus，we speculate that MVP plays a similar role in PCa progression as well as in glioblastoma.Previously, PARP-1 was found to promote AR transcriptional activity in AR-positive prostate cancer cells.17 This could explain the higher expression level of PARP-1 in AS relative to AI cells that we observed because AS cells exihibit higher AR transcriptional activity relative to AI cells.Among the other proteins that were found to be changed in the same direction in all three pairs, isoform 3 of A kinase anchor protein 12 (AKAP12)，tyrosine-tRNA ligase,cytoplasmic (YARS),isoform3of L-lactate dehydrogenase A chain (LDHA)，and glutamine-fructose-6-phosphate aminotransferase [isomerizing]1 (GFPT1）were reported earlier as products ofandrogen-regulated genes,18 inagreement with the differences of androgen sensitivity between AS and AI cells used in our study. Taken together, these results revealed the heterogeneity of PCa cells at the proteomic level.

Table 1.Proteins Showing Consistently Expressional Changes in PCa Cells   

<html><body><table><tr><td colspan="2">accession protein name</td><td>gene name</td><td>PC3-AR+/PC3 (114/113)</td><td>LNCaP/C42B (116/115)</td><td>22Rv1/Du145 (117/118)</td><td>22Rv1/Du14S (119/121)</td></tr><tr><td colspan="2">proteins with increased expressions in AI relative to AS cells</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Q02952-3</td><td>isoform 3 of A-kinase anchor protein 12</td><td>AKAP12</td><td>0.48</td><td>0.08</td><td>0.07</td><td>0.09</td></tr><tr><td>P04406</td><td>glyceraldehyde-3-phosphate dehydrogenase</td><td>GAPDH</td><td>0.49</td><td>0.16</td><td>0.54</td><td>0.54</td></tr><tr><td>Q9Y2J8</td><td> protein-arginine deiminase type-2</td><td>PADI2</td><td>0.16</td><td>0.36</td><td>0.11</td><td>0.20</td></tr><tr><td>Q14764</td><td>major vault protein</td><td>MVP</td><td>0.31</td><td>0.22</td><td>0.31</td><td>0.37</td></tr><tr><td>P06737-2</td><td>isoform 2 of glycogen phosphorylase,liver form</td><td>PYGL</td><td>0.42</td><td>0.20</td><td>0.08</td><td>0.10</td></tr><tr><td>P15121</td><td>aldose reductase</td><td>AKR1B1</td><td>0.19</td><td>0.06</td><td>0.02</td><td>0.03</td></tr><tr><td>Q9BUF5</td><td>tubulin β-6 chain</td><td>TUBB6</td><td>0.24</td><td>0.14</td><td>0.20</td><td>0.14</td></tr><tr><td>Q14152</td><td>eukaryotic translation initiation factor 3 subunit A</td><td>EIF3A</td><td>0.50</td><td>0.33</td><td>0.67</td><td>0.41</td></tr><tr><td>P54577</td><td>tyrosine-tRNA ligase,cytoplasmic</td><td>YARS</td><td>0.12</td><td>0.16</td><td>0.41</td><td>0.24</td></tr><tr><td>P00338-3</td><td>isoform 3 of L-lactate dehydrogenase A chain</td><td>LDHA</td><td>0.69</td><td>0.08</td><td>0.11</td><td>0.09</td></tr><tr><td></td><td>proteins with decreased expressions in AI relative to AS cells</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>P09874</td><td>poly[ADP-ribose] polymerase 1</td><td>PARP1</td><td>4.33</td><td>4.09</td><td>7.38</td><td>6.43</td></tr><tr><td>P40926</td><td>malate dehydrogenase, mitochondrial</td><td>MDH2</td><td>4.09</td><td>10.09</td><td>6.37</td><td>8.71</td></tr><tr><td>P07954</td><td>fumarate hydratase, mitochondrial</td><td>FH</td><td>6.61</td><td>6.85</td><td>5.45</td><td>4.79</td></tr><tr><td>Q86VP6</td><td>Cullin-associated NEDD8-dissociated protein 1</td><td>CAND1</td><td>2.13</td><td>2.27</td><td>3.02</td><td>2.91</td></tr><tr><td>K7EJE8</td><td>Lon protease homologue,mitochondrial</td><td>LONP1</td><td>5.25</td><td>3.66</td><td>4.41</td><td>5.06</td></tr><tr><td>P10515</td><td>dihydrolipoyllysine residue acetyltransferase component of pyruvate dehydrogenase complex,mitochondrial</td><td>DLAT</td><td>2.21</td><td>3.16</td><td>2.78</td><td>2.13</td></tr><tr><td>P10809</td><td>60 kDa heat shock protein,mitochondrial</td><td>HSPD1</td><td>1.03</td><td>1.15</td><td>1.17</td><td>1.16</td></tr><tr><td>Q06210</td><td>glutamine-fructose-6-phosphate aminotransferase [isomerizing] 1</td><td>GFPT1</td><td>3.08</td><td>3.19</td><td>1.80</td><td>2.81</td></tr><tr><td>P06744</td><td>glucose-6-phosphate isomerase</td><td>GPI</td><td>2.36</td><td>1.71</td><td>1.56</td><td>2</td></tr></table></body></html>

# Differentially Expressed Proteins Mainly Function in Central Metabolism

To investigate the functions of these differentially expressed proteins as well as their roles in CRPC progression, the 191 proteins whose expressions changed in at least two pairs of AS and AI PCa cells were chosen for GO annotation enrichment analysis.The result showed that most of these proteins were involved in cell metabolism and located in exosomes (Figure 2C).Exosomes play a pivotal role in cancers,19-22 we thus searched the exosome database Exocarta and Vesiclepedia and classified these exosome-located proteins based on their reported origins (Figure S2). These proteins likely play important roles in PCa progression.

Our results showed that nearly all of the enzymes involved in the TCA cycle showed a decrease in expression,while glycolytic enzymes including ATP-dependent 6-phosphofructokinase, platelet-type (PFKP)，glyceraldehyde-3-phosphate dehydrogenase(GAPDH), $\alpha$ -enolase(ENO1)，fructose-bisphosphate aldolase A (ALDOA)，and the rate-limiting enzyme LDHA showed increased expressions in AI relative to AS cells (Figure S3 and Table S4).Specifically，we observed that different isozymes of PFK changed in both directions.PFKP was increased while PFKM wasdecreased inAI relative to AS cells (Table S4).Previously，it was shown that during neoplastic transformation both quantitative increase and isozymic alterations of PFK occurred secondary to altered gene expression.23 In particular, PFK controls the entry of glucose metabolites into the downstream steps of glycolysis.24 Recently, Ros et al. found that the glycolytic enzyme PFKFB4 was essential for PCa cell survival,²5 confirming the importance of glycolysis regulation in PCa.

# Key Enzymes Involved in Glutamine Metabolism Change in PCa Cells

Since glucose is mainly utilized during glycolysis，glutaminedriven TCA turning powers oxidative phosphorylation and accounts for the majority of ATP production in most cancer cells.26 A recent report showed that PC3 cels use glutamine as their main energy source and revealed a key role of glutaminase (GLS1） in the growth of PCa cells.27 In agreement with these studies，we also observed increased levels of GLS1 in C42B relative to LNCaP and in ${ \mathrm { D u l } } 4 5$ relative to $2 2 \mathrm { R v 1 }$ ，andits increase in PC3 relative to $\mathrm { P C } 3 { \cdot } \mathrm { A R } ^ { + }$ was not significant(Table S4)．However， glutamate dehydrogenase 1 (GLUD1） was decreasedin C42BrelativetoLNCaPandinDul45relative to 22Rvl (Table S4)，suggesting a more complex regulation of glutamine metabolism in these cells.Recently，GLUD1- dependent reductive glutamine metabolism was shown to increase in PCa cells when glucose oxidation was decreased by metformin.28 The same study observed a more obvious effect of metformin in the LNCaP cells,indicating that AS cells were dependent more on reductive glutamine metabolism than were AI cells.In support of this observation,another key enzyme for reductive glutamine metabolism，isocitrate dehydrogenase [NADP] cytoplasmic (IDH1),was also shown to be increased in AS relative to AI cells in our results (Table S4).In addition, we found the increase of the mitochondrial aspirate-glutamate carrier SLC2SA13 in AS relative to AI cells (Table S4).These results, together with the information mentioned above, likely reflect that glutamine enters multiple metabolic pathways for distinct utilities in these PCa cells (Figure 5B).

![](images/608c8863fef4689eaf6243f81603b5474cd7209aaa3e8e02e8359b33b3c13dee.jpg)  
Figure3nalysfitoodrallgdfuctiCcll.（A)itoodraestadihtoackrGreendee stainedith.is4lee and LNCaP cells were plated at 40 000 cells/well in XF24 V7 cell culture plates $2 4 { - } 4 8$ h prior to the assays. For OCR assay,oligomycin $( 1 \mu \mathbf { M } )$ 5 FCCP $( 0 . 5 \ \mu \mathrm { M } )$ ，and rotenone ${ \bf \Pi } ( 1 \mu { \bf M } )$ with antimycin $( 1 ~ \mu \mathbf { M } )$ were added at time points indicated by arrows.For the ECAR assay, glucose （10 mM),oligomycin $( 1 \mu \mathbf { M } )$ ，and2-DG( $\mathrm { { 1 0 0 } ~ m M } ,$ were added at time points indicated by arrows.Each data point represents mean $+ \ S \mathrm { D }$ $\left( n = 3 { - } 4 \right)$ ： Mediumusedfordiferentaaswasdescrbedinteexperimentalmetod.ForOCResposedataerepresentedspercentagesofaselie values (measurement of 3).

# Functional Changes of Mitochondria Occur between AS and Al Cells

To investigate whether the decrease of enzymes in the TCA cycle in AI cells was caused by loss of mitochondrial mass, staining of mitochondria and mitochondrial activity analysis were conducted in each of the five cell lines (Figure 3).We excluded ${ \mathrm { P C } } 3 { \mathrm { - } } { \mathrm { A R } } ^ { + }$ cells because they showed distinct changes of the described TCA enzymes relative to PC3 when compared to the other two pairs.We did not observe a significant loss of mitochondrial mass in the AI PCa cells from the staining result, although the morphology and distribution of mitochondria varied considerably (Figure 3A).Moreover, expressions of the mitochondria electron transport chain (ETC） components, including electron transfer flavoprotein (ETF） and ATP synthase，were increased in AS relative to AI cells except for those in the $\mathrm { P C } 3 { \cdot } \mathrm { A R } ^ { + } / \mathrm { P C } 3$ pair in our results (no change of ETF and decrease of ATP synthase,Figure S4).However， no changes were observed in the other ETC components as well as many quantified structural proteins of mitochondria in our results，though it may be partially explained by a lack of sufficient quantitative peptides. Therefore， functional alterations rather than mass loss of mitochondria could eventually lead to such a complex change of proteins located in this organelle among different pairs of PCa cells.To further prove this,two mitochondrial proteins showed opposite changes in our results. They are the Lon protease homologue mitochondrial LONP1 that increased and calcium-binding mitochondrial carrier protein SCaMC-1 (SLC25A24）that decreased in AS relative to AI cells (Table S4).LONP1 plays a key role in metabolic reprogramming by remodeling OXPHOS complexes.29 SLC25A24 regulates the contents of adenosine nucleotides in the mitochondrial matrix and increases the antiapoptotic activity of mitochondria.30 Both of them are involved in timely response of mitochondria to cellular metabolic dynamics. An even more complex change was observed for dynamin-like $1 2 0 ~ \mathrm { \ k D a }$ protein，mitochondrial OPA1,which was increased in $\mathrm { P C } 3 { \cdot } \mathrm { A R } ^ { + }$ relative to PC3 and in LNCaP relative to C42B,but was decreased in 22Rvl relative to Dul45 (Table S4).OPA1 is the key effector for fusion of the mitochondrial inner membrane.Importantly， the rate of mitochondrial fusion responds to changes in metabolism.31

![](images/cccda8ad9367ac10986e96597c423cd515d4bb9ed93b9b7565c58716ff9391cf.jpg)  
Figure4.fciesfcleideseieelteofe are shown as mean $+ \ S \mathrm { D }$ 1 ${ \binom { n } { n } } = 3 )$ in (a),and their ratios to ATP are shown as mean $+ \ S \mathrm { D }$ （204号 $\left( n = 3 \right)$ in (b).The cellular content of acetyl-CoA is shown as mean $+ \ S \mathrm { D }$ $\left( n = 3 \right)$ in (c).The cellular content of NAD and NADH is shown as mean $+ \ S \mathrm { D }$ ${ \mathit { \acute { n } } } = 3 { \mathit { \dot { \Psi } } }$ )in (d),and the cellular content of NADP and NADPH is shown as mean $+ \ S \mathrm { D }$ （ $\left( n = 3 \right)$ in (e). The ratios of NADH/NAD,as well as NADPH/NADP,are shown as mean $+ \ S \mathrm { D }$ （204号 $\left( n = 3 \right)$ in (f). The letters above the columns denote significant differences $\left( p < 0 . 0 5 \right)$ for comparison,among which $\mathbf { \ddot { a } } ^ { \prime \prime }$ represents comparison between C42B and LNCaP and $\mathbf { \ddot { b } } ^ { \prime \prime }$ represents comparison between $\mathrm { D u l } 4 5$ and 22Rv1.

Together，these results reveal that metabolism-related functional alteration rather than mass loss of mitochondria occurs between AS and AI cells.

# High Rate of Glycolysis Serves the High Demand for a Biosynthetic Precursor in Al Cells

Cancer cels are known to rewire their metabolism and energy production networks to support rapid proliferation.3² In contrast to resting cells,proliferating cancer cells are in much greater need of building blocks for reductive biosynthetic reactions.33-35

To further investigate the metabolic signatures of PCa cells, we compared the contents of several metabolites in PCa cells by the MRM approach.In Table S1,we summarize the results of MRM method vaidation for all targeted metabolites in this study.All metabolites gave a linear response $\mathbf { \Phi } ^ { \prime } R ^ { 2 } > 0 . 9 7 )$ .As shown in Figure 4 and Table S5,the contents of ADP,AMP, cAMP,NAD,NADH,and NADP,as well as the ratios of ADP/ ATP，AMP/ATP,cAMP/ATP，NADH/NAD，andNADPH/ NADP,showed significant differences in one or two pair of PCa cells.Since the decrease of TCA enzymes in AI relative to AS cells was observed in our proteomic results，the increased content of NADH and the increased ratio of NADH/NAD in AIrelative to AS cells might be caused by other mechanisms. One possible mechanism is the regeneration of NAD via the lactate generation step in glycolysis,as a high rate of glycolysis marks AI cells (Figure 3B).The malate-aspartate shuttle (Figure SB） is another pathway that regenerates NAD but is kinetically much slower than pyruvate-to-lactate conversion.24 Therefore，elevated glycolysis will lead to a higher ratio of NADH/NAD in AI relative to AScells.

Since the activity of PFK was reported to be sensitive to the AMP/ATP ratio,24 and we also observed a higher ratio of AMP/ATP in AS relative to AI cells,PFK could control the rate of glycolysis via isoenzymatic regulation or activity change according to AMP/ATP ratio in PCa cells.

More importantly, the elevation of the AMP/ATP ratio as well as the increased content of AMP in AS cells could lead to

A B MDH 店 GlutamineGLS GIu a-KGAA →Mal Glucose Lac Asp α-KG PyrLDHA] SLC25A13 SLC25A11 LNCaP-3 GlutamineGLS Glu\~ Asp Mal Pyr LNCaP-2 GLUD AST LOAA a-KG PDH LNCaP-1 Mitochondrion α-KG IDH2 Cit Ac-CoA C42B-3 C42B-2 Glu GLUD，a-KG IDH1 Cit ACLY Ac-CoA GLS C42B-1 ACC FDFT   
0 Glutamine ！ 22Rv1-3 Malonyl-CoAt Cholesterol   
-1 22RV1-2 PC3-3 exotaknouf = lipid FA oxidation CPT Palmitate FASN ELOVL SCD MUFA PUFA ACS   
-2 1 PC3-2 ACS PC3-1 Cholesterol Fattyacyl-CoA Lipid   
-3 Du145-3 droplet GPAT   
中 >MAMAGL Phospholipids PLFA

Figure5.RelativecomparsooftcidsndthangsofelatedmetabolicezesinCacell.(A)eatmasofthefrefatcds(FF) andpospholdds(）eeditClsarsfochl'septca replicates.The contents of short-chain PFs (less than 20 carbons),as wellas polyunsaturated $C 2 0 { : } 3 \mathrm { n } 6$ and C20:4n6 PFs,were higher in AS relative toAcelsetagtsots weredecreasdielaieteltagddsters)Teotosstetangeeedfoeealoouauated namely, $C 1 8 { : } 1 \mathrm { n } 9 \mathrm { c }$ ,C20:1n9,C2:n9,andC24:1(theredcuster),whichwereincreasedinAIrelativetoAScels.PolyunsaturatedC22nd C22:6n3PFsweseifllyregadC4llartageofC(eraeuster).(B)eatieabc pathwaysndthelteredzessowinolorsinCaTegreedashdoxglightsattacidthesisandetaoxidatiotaeeste majorenergylyinAellUFoousaturatedfattcid;A,lsauratedfattcidFAosoliidfatyid monoacylglycerol; ChE,cholesteryl ester.

AMPK activation.AMPK is a core regulator of cellular metabolism.A previous study using human PCa specimens showed thatAMPK is highlyactivated in $4 0 \%$ of the cases,and it serves as a downstream effector of AR-induced PCa cell proliferation.36 In androgen-treated PCa cels，the androgenmediated AMPK-PGC $1 \alpha$ signaling cascade is responsible not only for increasing the rate of glycolysis but also for increasing glucose and fatty acid oxidation.'7 Furthermore, $\mathrm { C a } ^ { 2 + } / \mathrm { c a l m \bar { - } }$ odulin-dependent protein kinase kinase $\beta$ $\left( \mathbf { C a M K K } \beta \right)$ ，an AMPK kinase，was found to be highly expressed in normal prostate tissue and elevated in prostate cancer.38 One recent study reported that a novel AMPK activator inhibited the growth of PCa cells.39 It is worth mentioning that the study includes different AR phenotypes,and the basal level of AMPK activationand the functionsofAMPKactivatoraredifferent in these PCa cells.However, the role of AMPK in prostate cancer is controversial40hetinglelndelaroet AMPK activation may have profound effects on the functional consequences of AMPK signaling.42 A more complicated negative feedbackloop existsbetween AR andAMPK in PCa cells.43

In our proteomic results,an increased expression of AK1 in AS relative to AI cells was observed (Table S4).The highly reversible adenylate kinase reaction can be partially displaced toward synthesis of AMP and cause a large rise in AMP,44 representing an alternative route for AMPK activation in AS cells (Figure 6).Further studies are required to confirm these results.

# Simultaneous Increase of Fatty Acid Synthesis and Fatty Acid Oxidation in AS Cells

A metabolic shift,in which PCa cells use alternative enzymes and pathways to facilitate the production of fatty acids，is proposed to contribute to the well-established lipogenic phenotype of PCa.45 In agreement with this,we observed an increase of fatty acid synthase (FASN) in AS relative to AI cells (Table S4).Importantly, FASN has been reported to be a candidate oncogene in PCa.46 However， changes of other enzymes in fatty acid synthesis，such as ATP-citrate synthase (ACLY）and acetyl-CoA carboxylase 1 (ACC)，were complex among the different pairs (Table S4).ACLY was decreased in LNCaP relative to C42B but increased in 22Rvl relative to Dul45.ACC was only increased in 22Rvl relative to $\mathrm { D u l } 4 5$ and showed no difference in the other two pairs.In agreement with the changes observed for ACLY and ACC,the content of acetyl-CoA was higher in C42B cells compared to LNCaP as well as to those in the other PCa cells，but showed no difference between Dul45 and 22Rvl (Figure 4c)，suggesting that acetyl-CoA accumulated in C42B but was consumed in 22Rvl through fatty acid synthesis.Another function of acetylCoAis to serve as substrate for cholesterol synthesis.We observed that squalene synthase (FDFT1),the key enzyme for cholesterol synthesis,was present at an elevated level in AS relative to AI cells (Table S4),a fact that is in agreement with the increased de novo synthesis of cholesterol in PCa cells.47 Recently,AI cells were shown to bear aberrant accumulation of esterified cholesterol in lipid droplets，which arises from an uptake of exogenous lipoproteins and is independent of androgen signaling.48Thus, AI cellsappear toobtain cholesterol through exogenous uptake rather than de novo synthesis,which is in contrast to AS cells.

![](images/4a83bdc45d3a1652df5d3351368e18e8afe8c220054c984270df3e9dffb9508c.jpg)  
Figure6.PropododeloftAsgaetoknditsoleinellaetaliglaioettatuoctie transcriptionosticttdgeoeges(Gs)sulsisCacelatis(sol cells).Amolfisistodtsetraceladgelld negativefeedbckoiactiyegaioTisatiedbackoupsesAsigalingatigdrgenlevelstlscesd ARandadrogentssiCAucestngoftracellarognevelwicgtialctivatiodaly contributes to androgen independence.AR activation will increase the expression of $\mathbf { C a M K K } \beta$ and activate AMPK,and the latter will conversely inhibitAiAalfabasabCac ARactivitystatusalsocontributetothisshiftAK1caconvertADPtoAMPandincreasetheAMP/ATatio,leadingtoAMPKactivation. DiferentmetabolicathayshgightedintecoloredctagoalboxesaredominantindistictCacelsandtherelatedensthate diferentially expressed in PCa cells are highlighted in the colored boxes above each pathway.

After synthesis,fatty acids can be oxidized for energy supply or converted into the carbon skeleton of triglyceride for lipid storage as well as for phospholipid synthesis.49 We observed that carnitine O-palmitoyltransferase (CPT）changed in the isoform context among different pairs of PCa cells (Table S4). CPT1A and peroxisomal carnitine O-octanoyltransferase (CROT)，which are located on the outer membrane of mitochondria and peroxisomes,respectively,were found to be increased in LNCaP relative to C42B，while showing no difference in the other pairs.CPT2,which locates on the inner membrane of mitochondria,was increased inLNCaP relative to C42B and in $2 2 \mathrm { R v 1 }$ relative to Dul45.One reason for this observation could be that oxidation for different fatty acids in mammalian cells is compartmentalized according to the lengths of their acyl chains.In comparison with AI cells，AS cells overexpressed several mitochondrial enzymes that are responsible for fatty acid $\beta$ -oxidation,including long-chain fatty acidCoA ligase 3 (ACSL3)，enoyl-CoA hydratase (ECHS1), Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase，mitochondrial (ECH1),trifunctional enzyme subunit alpha (HADHA)，and acetyl-CoA acetyltransferase(ACAT1)(Table S4).Consistent with the increased levels of these enzymes,most of the 17 FFs reliablydetected byMRM were found to be increased in AS relative to AI cells (Figure ${ \displaystyle 5 \mathrm { A } , }$ blue cluster).Together, these results indicate that fatty acid $\beta$ -oxidation in mitochondriais increased in AS cells. This may result from the androgenmediated AMPK-PGC- $1 \alpha$ signaling cascade，which directly causes the increase of fatty acid $\beta$ oxidation.37 In order to investigate the fates of fatty acids in PCa cells，we also compared the changes of PFs in different PCa cells relatively using MRM. Consistent with the complex changes of lipogenic enzymes observed here (data not shown)，the contents of different PFs showed complex changes between AS and AI cells (Figure SA).Our results were consistent with a recent MS study on phospholipid composition in PCa cels.50 That study reported that AS cells contain more polyunsaturated phospholipids but less saturated and monounsaturated phospholipids than AI cells.Moreover，we observed that the contents of short-chain PFs as well as FFs were higher in AS relative to AI cells，indicating that synthesized FFs were partially converted to short-chain PFs in AS cells.Long-chain PFs，especially the monounsaturated counterparts，displayed changes in opposing directions in our results， suggesting preferential uptake from the environment rather than de novo synthesis in AI PCa cells.

Thus,we suggest that a simultaneous increase in synthesis and $\beta$ -oxidation of fatty acids occurs in AS cells，which may serve asa major energy supply (Figure SB).In agreement with our hypothesis，a previous study showed that,in PCa cells, androgen could induce the activation of both lipogenesis and fatty acid degradation in a coordinated fashion.51 Further studies will be required to clarify the regulation of lipid metabolism in CRPC cells.

# AIPCa Cells Favor Cell-ECM Focal Adhesion over Cell-Cell Adhesion

Another category of proteins that was observed to show changes in expression between AS and AI is that of factors associated with cell adhesion.These factors included tubulin $\beta \mathrm { . }$ 6,talin-1,Fermitin family homologue 2,IQGAP1, fascin,actinrelated protein 3, CAP1,TWF2,coronin,and profilin-1.All of these proteins were found to be present at elevated levels in AI relative to AS cells.In addition, $\alpha$ -actinin-1 and isoform 1 of vinculin changed in the same direction between AS and AI cells (Table S4).AI cells also expressed more integrin $\beta 1$ and less Ecadherin relative to AS PCa cells (Table S4).It is wellestablished that integrin $\beta 1$ mainly mediates cell adhesion to extracellular matrix (ECM） components，whereas E-cadherin mainly mediates epithelial cell-cell adhesion.52 According to the model of adhesion dynamics proposed by Parsons et al.,53 we conclude that AI PCa cells favor focal adhesion to ECM overcell-cell adhesion based on the changes of these adhesionrelated molecules observed here.Adhesion-mediated signaling through integrin $\beta 1$ and integrin-linked kinase has a critical role in caveolae formation.54 In addition to caveolae formation, caveolin-1 _plays many other roles in CRPC progression.10 Consistently,we observed that caveolin-1 was increased in AI relative to AS cells, though the $p$ valueswere above O.O1 (data not shown),which may be a result of insufficient numbers of peptides for quantification.Another key cell-ECM adhesion molecule,CD44,was also increased in AI relative to AS cells (Table S4).Previously,CD44 was shown to connect to the actin cytoskeleton，bind to growth factors，and enhance signaling through cis interactions with EGFR 52 Whether CD44 links to EGFR for PCa cell adhesion needs further investigation.

Overall,AI cells favor integrin $_ { \cdot \beta 1 }$ -mediated cell adhesion to ECM over E-cadherin-mediated cell-cell adhesion.Like the metabolic enzymes,most of these adhesion-related molecules also locate in the exosome (Figure S2). Therefore, elucidating the internal relations between cell adhesion and metabolism may finally uncover the role of exosomes in PCa.

# CONCLUSION

An increasing number of studies reveal the heterogeneity of PCa as a problem for the identification of common features in its biological properties.In this study，we took this heterogeneity of PCa into consideration and compared the proteomes of three pairs of PCa cells. We identified 19 differentially expressed proteins that changed in the same direction in all three pairs,among which four proteins have been reported to be the products of androgen-responsive genes. Our results suggest that the 19 proteins are pivotal in CRPC progression.Furthermore,isozymic alteration of PFK, together with the resulting aberrant glycolysis,may accompany CRPC progression,based on the established role of glycolysis in rapid proliferating cancer cels.24 Apart from altered glycolysis, glutamine and fatty acid metabolisms are also different between

AS and AI cells according to our proteomic results as well as MRM analysis of fatty acids，nucleotides，and acetyl-CoA These changes suggest a metabolic shift between AS and AI cells and reveal a central place of mitochondria in this shift. Increased glycogen mobilization is likely to represent an additional sign for the metabolic shift，based on our observations that two isoforms of glycogen phosphorylase, namely,PYGL and PYGB,were elevated in AI relative to AS cells (Table1 and Table S4).The simultaneous increases of AMPandAK1likelycontributeto AMPKactivationinAScells, suggesting a potential mechanism underlying the metabolic shift(Figure 6).Furthermore，integrin $\cdot \beta 1 ^ { \circ }$ -mediating cell adhesion to ECM was preferentially utilized in AI relative to AScells. Because many differentially expressed metabolic enzymes as well as adhesion molecules observed in our study are located in exosomes，the synergistic action between cell adhesion and metabolism in exosomes confirmatively contributes to CRPC progression and deserves further investigations.

# ASSOCIATEDCONTENT

# $\bullet$ Supporting Information

Figures S1-S4 and Tables S1-S5.The Supporting Information is available free of charge on the ACS Publications website at DOI:10.1021/acs.jproteome.5b00464.

# AUTHORINFORMATION

# Corresponding Authors

$^ { * } \mathrm { E }$ -mail: fqyang@sun5.ibp.ac.cn. Tel: +86-10-6488-8581. Fax: +86-10-6488-8581.   
$^ * \mathrm { E }$ mail: gao.weiqiang@sjtu.edu.cn. Tel: +86-21-68383917. Fax: +86-21-68383916.   
Author Contributions   
$^ { \perp } { \sf Q . S } .$ and T.C.contributed equally to this work.   
Notes   
The authors declare no competing financial interest.

# ACKNOWLEDGMENTS

We thank Xiang Ding, Xiaojing Guo,Lili Niu, and Jifeng Wang (Laboratory of Proteomics,Institute of Biophysics，Chinese Academy of Sciences） for excellent technical assistance;Camilla Thygesen (Sino-Danish Center for Education and Research, University of Chinese Academy of Sciences） for critical reading of the manuscript;Yanjing Guo and Yongfu Pan (Shanghai Jiaotong University） for providing cells and helpful discussion; Bo Zhang (Institute of Biophysics，Chinese Academy of Sciences） for seahorse technical assistance；and Chunli Jiang (Center for Biological Imaging, Institute of Biophysics, Chinese Academy of Science） for mitochondrial image.This research was supported by the National Basic Research Program of China (973）(Grant Nos.2012CB966803，2012CB966801, 2014CBA02003).

# REFERENCES

(1)DeSantis,C.E.; Lin, C.C.; Mariotto,A.B.; Siegel, R.L.; Stein, K. D.; Kramer,J.L.;Alteri,R.;Robbins,A.S.; Jemal,A. Cancer treatment and survivorship statistics,2014.Ca-Cancer J.Clin.2014,64,252-271. (2) Scher,H.I.; Halabi, S.; Tannock, I.; Morris,M.; Sternberg, C.N.; Carducci,M.A.;Eisenberger,M.A.; Higano,C.; Bubley, G.J.; Dreicer, R;Petrylak,D.; Kantoff,P.; Basch,E.; Kelly,W.K.; Figg,W.D.; Small, E.J.;Beer, T.M.; Wilding, G.; Martin,A.; Hussain,M.Design and end points of clinical trials for patients with progressive prostate cancer and castrate levels of testosterone:Recommendations of the prostate

cancer clinical trials working group. J. Clin. Oncol. 2008,26,1148- 1159.   
(3) Barlow,L.J.; Shen,M.M.SnapShot: Prostate cancer.Cancer Cell 2013,24, 400.   
(4) Russell,P.； Kingsley, E. Human Prostate Cancer Cell Lines. In Prostatee E.,Eds.; Springer: New York,2003；pp 21-39.   
(5)Chen, C.D.; Welsbie, D.S.； Tran, C.； Baek, S.H; Chen， R; Vessella,R.； Rosenfeld, M. G.； Sawyers, C.L.Molecular determinants of resistance to antiandrogen therapy.Nat. Med.2004,10,33-39. (6) Cai, C.M.; He,H. S.H. S.; Chen,S.; Coleman,L.; Wang, H. Y.; Fang, Z.; Chen, S.Y.; Nelson,P.S.; Liu,X.S.; Brown,M.; Balk, S.P. Androgen Receptor Gene Expression in Prostate Cancer Is Directly Suppressed bythe Androgen Receptor Through Recruitment of Lysine-Specific Demethylase 1. Cancer Cell2011, 20,457-471. (7)Ryan, C.J.; Smith,M.R.; de Bono,J.S.; Molina,A.; Logothetis, C.J.；de Souza,P.； Fizazi,K.；Mainwaring,P.；Piulats,J.M.；Ng, S.; Carles,J.;Mulders,P.F.A.; Basch,E.;Small,E.J.; Saad,F.; Schrijvers, D.; Van Poppel, H.; Mukherjee,S.D.; Suttmann, H.; Gerritsen, W.R; Flaig,T.W.； George,D.J.；Yu, E.Y.；Efstathiou,E.； Pantuck,A; Winquist,E.;Higano, C.S.; Taplin,M.E.; Park,Y.;Kheoh,T.; Grifin, T.； Scher, H.I.；Rathkopf, D.E. Investigators, C.-A.-. Abiraterone in Metastatic Prostate Cancer without Previous Chemotherapy.N. Engl. J.Med.2013,368,138-148.   
(8) Yuan,X.; Cai, C.; Chen,S.; Chen,S.; Yu,Z.; Balk, S.P.Androgen receptor functions in castration-resistant prostate cancer and mechanisms of resistance to new agents targeting the androgen axis. Oncogene 2014,33,2815-2825.   
(9)Edwards,J.； Bartlet, J.M. Theandrogen receptorand signaltransduction pathways in hormone-refractory prostate cancer. Part 2: Androgen-receptor cofactors and bypass pathways.BJU Int. 2005, 95, 1327-1335.   
(10）Karantanos,T.; Corn, P.G.； Thompson, T.C.Prostate cancer progression after androgen deprivation therapy: mechanisms of castrate resistance and novel therapeutic approaches. Oncogene 2013, 32, 5501-5511.   
(11）Wisniewski, J.R.；Zougman，A.；Nagaraj,N.；Mann，M. Universal sample preparation method for proteome analysis. Nat. Methods 2009,6,359-362.   
(12）Matyash,V.；Liebisch,G.； Kurzchalia,T.V.； Shevchenko,A.; Schwudke,D.Lipid extraction by methyl-tert-butyl ether for highthroughput lipidomics.J. Lipid Res. 2008,49,1137-1146.   
(13) Gielisch, L; Meierhofer, D.Metabolome and proteome profiling of complex I deficiency induced by rotenone. J. Proteome Res. 2015,14, 224-235.   
(14) Chen, X.L.; Wei, S.S.; Ma, Y.; Lu,J.; Niu, G.; Xue,Y.H.; Chen, X.Y.；Yang，F.Q. Quantitative Proteomics Analysis Identifies Mitochondria as Therapeutic Targets of Multidrug-Resistance in Ovarian Cancer. Theranostics 2014,4,1164-1175.   
(15）Lotsch,D.； Steiner，E.； Holzmann,K.；Spiegl-Kreinecker， S.; Pirker, C.；Hlavaty,J.; Petznek,H.; Hegedus,B.; Garay,T.； Mohr,T. Sommergruber，W.；Grusch，M.；Berger，W.Major vault protein supports glioblastoma survival and migration by upregulating the EGFR/PI3K signalling axis. Oncotarget 2013,4,1904-1918.   
(16)Mulholland,D.J.; Tran,L.M.;LiY.;Cai,H; Mrim,A;Wang S.; Plaisier, S.; Garraway, I.P.; Huang,J.; Graeber,T.G.; Wu,H. Cell autonomous role of PTEN in regulating castration-resistant prostate cancer growth. Cancer Cell 2011,19,792-804.   
(17)Schiewer,M.J.; Goodwin,J.F.; Han, S.; Brenner,J.C.; Augello, M.A.；Dean，J.L.；Liu，F.；Planck，J.L.；Ravindranathan，P.; Chinnaiyan,A. M.; McCue,P.;Gomella, L.G.; Raj, G.V.; Dicker,A. P.; Brody,JR;salJ;Ceteea.;ButleL.lley W.D.; Feng,F.Y.； Knudsen,K.E. Dual roles of PARP-1 promote cancer growth and progression. Cancer Discovery 2012, 2, 1134-1149. (18) Jin, H.J.; Kim,J.; Yu,J.ndrogenreceptor genomicregulation. Translational andrology and urology 2013,2,157-177.   
(19) Hosseini-Beheshti, E.; Pham, S.;Adomat,H.; Li, N.; Tomlinson Guns,E.S.Exosomes as biomarker enriched microvesicles: characterization of exosomal proteins derived from a panel of prostate cell lines with distinct AR phenotypes.Mol. Cell.Proteomics 2012,11,863-885. (20) Ramteke,A.; Ting,H.; Agrwal,C.; ateen,S.;SosgaR; Hussain，A.；Graner，M.；Frederick，B.；Agarwal，R；Deep，G. Exosomes secreted under hypoxia enhance invasiveness and stemness of prostate cancer cells by targeting adherens junction molecules. Mol. Carcinog. 2015,54, 554.   
(21) Boelens,M.C.; Wu, T.J.; Nabet,B.Y.; Xu,B.; Qiu, Y.; Yoon, T.；Azzam,D.J.; Victor, C.T.S.;Wiemann,B.Z.; Ishwaran,H.；ter Brugge,P.J.; Jonkers,J.； Slingerland,J.;Minn,A.J.Exosome Transfer from Stromal to Breast Cancer Cells Regulates Therapy Resistance Pathways. Cell 2014,159,499-513.   
(22）Melo， S.A.； Sugimoto，H；OConnell, J. T.；Kato，N.; Villnueva,A.; VidalA;QuL; Vitkin,E.;Perelman,L.T;MeloC. A.；Lucci，A.； Ivan,C.； Calin,G.A.；Kalluri，R. Cancer Exosomes Perform Cell-Independent MicroRNA Biogenesis and Promote Tumorigenesis. Cancer Cell 2014, 26, 707-721.   
(23) Vora, S.; Halper,J.P.; Knowles,D.M.Alterations in the activity and isozymic profile of human phosphofructokinase during malignant transformation in vivo and in vitro:transformation- and progressionlinked discriminants of malignancy. Cancer Res.1985,45,2993-3001. (24) Lunt,S.Y.; Vander Heiden,M.G.Aerobic Glycolysis: Meeting the Metabolic Requirements of Cell Proliferation.Annu.Rev. Cell Dev. Biol. 2011,27,441-464.   
(25）Ros,S.； Santos, C.R.； Moco,S.； Baenke,F.; Kelly,G.; Howell, M.; Zamboni, N.; Schulze,A. Functional metabolic screen identifies 6- phosphofructo-2-kinase/fructose-2,6-biphosphatase 4 as an important regulator of prostate cancer cell survival. Cancer Discovery 2012，2, 328-343.   
(26)Fan,J.;Kamphorst,JJ;athew,R;Chung,.K.; White,E.; Shlomi， T.； Rabinowitz, J. D.Glutamine-driven oxidative phosphorylation is a majorATP source in transformed mammalian cells in both normoxia and hypoxia. Mol. Syst.Biol. 2014,9,712.   
(27) Pan,T.; Gao,L.; Wu, G.; Shen, G.; Xie, S.; Wen, H.; Yang, J.; Zhou,Y.; Tu, Z.; Qian, W.Elevated expression of glutaminase confers glucose utilization via glutaminolysis in prostate cancer.Biochem. Biophys. Res.Commun.2015,456,452-458.   
(28)Fendt,S.M.; Bell,E.L.; Keibler, M.A.; Davidson, S.M.; Wirth, G.J.；Fiske,B.；Mayers,J.R； Schwab,M.；Belinger，G.；Csibi，A.; Patnaik，A；Blouin，M.J.；Cantley，L.C.；Guarente,L.；Blenis,J.; Pollak,M. N.; Olumi, A.F.; Vander Heiden,M. G.； Stephanopoulos, G.Metformin Decreases Glucose Oxidation and Increases the Dependency of Prostate Cancer Cells on Reductive Glutamine Metabolism. Cancer Res.2013,73,4429-4438.   
(29）Quiros,P. M.；Espanol, Y.；Acin-Perez，R；Rodriguez，F.; Barcena, C.; Watanabe,K.; Calvo,E.; Loureiro,M.; Fernandez-Garcia, M. S.； Fueyo,A.;Vazquez,J； Enriquez, J.A.;Lopez-OtinC.- dependent Lon protease controls tumor bioenergetics by reprogramming mitochondrial activity. Cell Rep.2014,8, 542-556.   
(30）Traba，J.；Del Arco，A.；Duchen，M.R.；Szabadkai，G.; Satrustegui,J.SCaMC-1 promotesancercellsurvivalbysesitng mitochondrial permeability transition via ATP/ADP-mediated matrix $\mathrm { C a } ( 2 + )$ buffering. Cell Death Differ.2012,19,650-660.   
(31)Youle,R.J.;van der Bliek,A.M. Mitochondrial fission, fusion, and stress. Science 2012, 337,1062-165.   
(32）Phan, L. M.； Yeung, S.C.； Lee,M. H. Cancer metabolic reprogramming: importance,main features,and potentials for precise targeted anti-cancer therapies. Cancer Biol. Med.2014,11,1-19. (33）Ward,P. S.； Thompson，C.B. Metabolic reprogramming:a cancer hallmark even warburg did not anticipate. Cancer Cell 2012, 21, 297-308.   
(34) Cairns,R A; Harris,I..; Mak, T.W.Regulation ofcancercell metabolism. Nat.Rev.Cancer 2011,11,85-95.   
(35）Vander Heiden，M.G.； Cantley，L.C.；Thompson，C.B. Understanding the Warburg effect: the metabolic requirements of cel proliferation. Science 2009,324,1029-1033.   
(36) Park,H. U.； Suy,S.; Danner,M.; Dailey,V.; Zhang,Y.; Li,H.; Hyduke,D.R.;Collins,B.T.;Gagnon,G.; Kallakury,B.; Kumar, D.; Brown,M.L.; Fornace,A.; Dritschilo,A.; Collins, S.P.AMP-activated protein kinase promotes human prostate cancer cell growth and survival. Mol. Cancer Ther. 2009,8,733-741.   
(37)Tennakoon,J.B.; ShiY.;Han,J.J;souko,E.;Wite,.A Burns,A.R.； Zhang,A.； Xia,X.; Ilkayeva, O.R.；Xin,L.; Ittmann,M. M.; Rick,F. G.; Schally,A. V.; Frigo,D.E. Androgens regulate prostate cancer cell growth via an AMPK-PGC-lalpha-mediated metabolic switch. Oncogene 2014,33, 5251-5261.   
(38)Frigo,D.E.；Howe,M.K; Wittmann,B.M.； Brunner,A.M.; Cushman,L; Wang,Q.B.; Brown, M.; Means, A.R.; McDonell,D.P. CaM Kinase Kinase beta-Mediated Activation of the Growth Regulatory Kinase AMPK Is Required for Androgen-Dependent Migration of Prostate Cancer Cells. Cancer Res.2011,71,528-537. (39) Zadra, G.; Photopoulos, C.; Tyekucheva, S.; Heidari,P.; Weng, $\mathrm { \Delta Q }$ P.；Fedele,G.；Liu，H.；Scaglia，N.；Priolo，C.；Sicinska，E.; Mahmood,U.； Signoreti, S.；Birnberg N.；Loda，M.A novel direct activator of_AMPK inhibits prostate cancer growth by blocking lipogenesis.EMBO Mol.Med.2014,6,519-538.   
(40) Park, H. U.; Suy,S.; Danner,M.; Dailey,V.; Zhang, Y.; LiH; Hyduke,D.R.；Collins,B. T.; Gagnon,G.; Kallakury,B.;Kumar,D.; Brown,M.L.; Fornace,A.; Dritschilo,A.; Collins,S.P.AMP-activated protein kinase promotes human prostate cancer cell growth and survival. Mol. Cancer Ther.2009,8,733-741.   
(41) Zhou,J.; Huang,W.; Tao,R.; Ibaragi, S.;Lan,F.; Ido,Y.; Wu, X.; Alekseyev, Y. O.; Lenburg,M.E.; Hu, G.F.; Luo, Z.Inactivation of AMPK alters gene expression and promotes growth of prostate cancer cells.Oncogene 2009,28,1993-2002.   
(42） Massie, C.E.; Lynch,A.； Ramos-Montoya,A.; Boren,J.； Stark, R;Fazli, L.;Warren,A.； Scot,H.;Madhu,B.; Sharma,N.; Bon,H; Zecchini,V.; Smith, D.M.; DeNicola, G.M.； Mathews, N.; Osborne, M.；Hadfield, J.; MacArthur, S.；Adryan, B.; Lyons, S.K.； Brindle,K. M.; Griffths,J.; Gleave,M.E.; Rennie,P.S.; Neal,D.E.;Mills,I. G. The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis.EMBO J.2011,30,2719-2733.   
(43) Jurmeister, S.; Ramos-Montoya,A.; Neal, D.E.; Fryer,L.G.D. Transcriptomic analysis reveals inhibition of androgen receptor activity by AMPK in prostate cancer cells. Oncotarget 2014, S,3785-3799. (44)Hardie,D.G.; Ross,F.A.;Hawley, S.A.AMP-ActivatedProtein Kinase:A Target for Drugs both Ancient and Modern.Chem. Biol. 2012,19, 1222-1236.   
(45）Suburu，J.；Chen，Y. $\scriptstyle { \mathrm { \mathrm { Q } } } .$ Lipids and prostate cancer. Prostaglandins OtherLipid Mediators 2012,98,1-10.   
(46)Migita,T.；Ruiz,S.；Fornari，A.； Fiorentino，M.；Priolo,C.; Zadra, G.； Inazuka,F.; Grisanzio,C.； Palescandolo,E.; Shin,E.; Fiore, C.; Xie,W.; Kung,A.L.; Febbo,P.G.; Subramanian,A.; Mucci,L.; Ma, J.； Signoretti, S.； Stampfer,M.; Hahn,W.C.; Finn,S.; Loda,M.Fatty Acid Synthase:A Metabolic Enzyme and Candidate Oncogene in Prostate Cancer.J. Natl.Cancer Inst.2009,101,519-532.   
(47） Zadra,G.; Photopoulos, C.; Loda,M. The fat side of prostate cancer.Biochim. Biophys.Acta,Mol. Cell Biol.Lipids 2013,1831,1518- 1532.   
(48)Yue, S.H.; Li,J.J.; Lee,S.Y.; Lee,H.J.； Shao,T.；Song,B.; Cheng, L.；Masterson, T.A.； Liu, X. $\mathrm { Q . ; }$ Ratliff, T.L.； Cheng, J.X. Cholesteryl Ester Accumulation Induced by PTEN Lossand PI3K/ AKT Activation Underlies Human Prostate Cancer Aggressiveness. Cell Metab.2014,19,393-406.   
(49)Currie,E.; Schulze,A； Zechner,R.;Walther, T.C.; Farese,R. V., Jr. Cellular fatty acid metabolism and cancer. Cell Metab.2013,18, 153-161.   
(50）Rysman, E.； Brusselmans，K.； Scheys,K.；Timmermans,L.; Derua,R; Munck, S.; Van Veldhoven,P.P.; Waltregny,D.; Daniels,V. W.；Machiels，J.；Vanderhoydonc，F.；Smans，K.；Waelkens，E.; Verhoeven,G.； Swinnen, J.V. De novo lipogenesis protects cancer cellsfromfree radicalsand chemotherapeutics bypromoting membrane lipid saturation. Cancer Res. 2010, 70, 8117-8126.   
(S1）Lin,H.L.；Lu, J.P.； Laflamme,P.； Qiao,S.J.； Shayegan,B.; Bryskin,I.; onardo,L.; Wilson,B.C.; Singh, G.; Pinthus,J.H.Interrelated in vitro efectsofandrogens,fatty acids and oxidative stress in prostate cancer: A mechanistic model supporting prevention strategies. Int. I. Oncnl. 2010.37. 761-766. (52)Friedl,P.;Alexander, S. Cancer invasion and the microenvironment:plasticity and reciprocity. Cell 2011,147,992-1009.   
(53)Parsons,J.T.;Horwitz,A.R.；Schwartz,M.A. Cell adhesion: integrating cytoskeletal dynamics and cellular tension.Nat.Rev. Mol. Cell Biol. 2010,11, 633-643.   
(S4)Wickstrom, S.A.; Lange,A.; Hess,M.W.; Polleux, J.； Spatz, J. P.；Kruger，M.；Pfaller，K.；Lambacher，A.；Bloch，W.；Mann，M.; Huber,L.A.；Fassler,R. Integrin-linked kinase controls microtubule dynamics required for plasma membrane targeting of caveolae.Dev. Cell 2010,19,574-588.   
(S5）Kuhnel,E.; Laffan,D.D.；Lloyd-Jones,G.C.；Martinez Del Campo,T.； Shepperson,I.R.； Slaughter,J.L.Mechanism of methyl esterification of carboxylic acids by trimethylsilyldiazomethane.Angew. Chem., Int.Ed.2007,46,7075-7078.   
(56)Benjamini, Y.Controlling the False Discovery Rate:APractical and Powerful Approach to Multiple Testing.J.R.Stat.Soc. SeriesB. 1995,57 (1),289-300.