# CellStemCel

# The Long Noncoding RNA IncTCF7 Promotes SelfRenewal of Human Liver Cancer Stem Cells through Activation of Wnt Signaling

Graphical Abstract

# Authors

Yanying Wang, Lei He.., Runsheng Chen, Zusen Fan

![](images/0f8c8d264e537b7f8cf64cdaff09eb2cdd0193b746f8ac019fd16d5392956f5f.jpg)

Correspondence crs@sun5.ibp.ac.cn (R.C.), fanz@moon.ibp.ac.cn (Z.F.)

# In Brief

Wang et al. have identified a long noncodingRNA,IncTCF7,thatactivates Wnt signaling to promote liver cancer stem cell self-renewal and tumor propagation.Targeting this pathway could help address the high recurrence and heterogeneity of liver cancer.

# Highlights

The long noncoding RNA IncTCF7 is highly expressed in liver   
cancer tissues and CSCs   
LncTCF7 is important for self-renewal of liver CSCs   
LncTCF7 activates the Wnt signaling pathway through TCF7   
expression   
LncTCF7 recruits the SWI/SNF complex to activate the TCF7   
promoter   
Accession Numbers   
GSE66515   
GSE66529

# The Long Noncoding RNA IncTCF7 Promotes Self-Renewal of Human Liver Cancer Stem Cells through Activation of Wnt Signaling

Yanying Wang,1.5Lei He,2.5YingDu,1Pingping Zhu,1 Guanling Huang,13JianjunLuo,4 XinlongYan,1 Buqing Ye,   
Chong Li,1 Pengyan Xia,1 Geng Zhang,1 Yong Tian,4 Runsheng Chen,4,\*and Zusen Fan1,3,\*   
1CAS KeyLaboratoryof IfectionandImmunity,IsituteofBiophysics,ChineseAcademyofSciences,Beijing00,China   
2Department of Hepatobiliary Surgery,PLA General Hospital,Beijing 100853,China   
3University of Chinese Academy of Sciences,Beijing 10o049,China   
4CAS Key Laboratory of RNA Biology,Institute of Biophysics,Chinese Academy of Sciences,Beijing 100101,China   
$^ { 5 } { \mathsf { C o } }$ -first author   
\*Correspondence: crs@sun5.ibp.ac.cn (R.C.), fanz@moon.ibp.ac.cn (Z.F.)   
http://dx.doi.0rg/10.1016/j.stem.2015.03.003

# SUMMARY

Hepatocellular carcinoma (HCC) is the most prevalent subtype of liver cancer,and it is characterized bya high rate of recurrenceandheterogeneity.Liver cancer stem cells (CSCs) may well contribute to both ofthese pathological propertiesï¼Œbut the mechanisms underlying theirself-renewal and maintenance are poorly understood. Hereï¼Œusing transcriptome microarray analysis,we identified a long noncoding RNA (IncRNA) termed IncTCF7 that is highly expressed in HCC tumors and liver CSCs.LncTCF7 is required for liver CSC self-renewal and tumor propagation.Mechanistically, IncTCF7 recruits the SWI/ SNF complex to the promoter of TCF7 to regulate its expression, leading to activation of Wnt signaling. Our data suggest that IncTCF7-mediated Wnt signaling primes liver CSC self-renewal and tumor propagation.In sumï¼Œthereforeï¼Œwe have identified an IncRNA-based Wnt signaling regulatory circuit that promotes tumorigenic activity in liver cancer stem cells,highlighting the role that IncRNAs can play in tumor growth and propagation.

# INTRODUCTION

Liver cancer is the fifth most commonly diagnosed cancer and the second most frequent cause of cancer death in men worldwide (Jemal etal.,2011). Hepatocellular carcinoma (HCC) represents the major histological subtype,accounting for ${ \sim } 7 0 \% { \sim } 8 5 \%$ of cases of primary liver cancer.Unfortunately, the 5-yearsurvival rate of HCC subjects remains poorï¼Œand more than 750,000 HCC subjects die each year.Although previous studies identified manyaberrantly expressed protein-coding genes in HCC,novel molecular markers that can help in early diagnosis and risk assessmentare still urgently needed (Ji etal.,20o9; Yamashita and Wang,2013).New therapeutic approaches are likely to derive froman improved understanding of the molecular basis of HCC.

The high rate of recurrence and heterogeneity are the two major featuresof HCC (Visvaderï¼Œ2011).Recent studies have suggested that heterogeneity isa result of the hierarchical organization of tumor cells bya subset of cellswith stem/progenitor cellfeaturesknownas cancerstemcells (CSCs) (Easwaranetal., 2014).These CSCs within tumor bulk display the capacity to selfrenew,differentiate,and give rise to a new tumor (Visvader and Lindemanï¼Œ2012),accounting fora hierarchical organization of heterogeneous cancer cels and a high rate of cancerous recurrence.Liver CSCscanbeenrichedwithseveraldefinedsurface markers,including epithelial cell adhesion molecule (EpCAM), CD133,CD13,CD90,CD44,CD24,and calcium channel $\alpha 2 \delta 1$ subunit, among others (Haraguchi et al., 2010; Lee et al., 2011; Ma et al., 2007; Zhao et al.,2013). However, how liver CSCs sustain their self-renewal remains largely unknown.

Long noncoding RNAs (IncRNAs) are defined as transcripts longer than 2oo nucleotides (nt) thatare $5 ^ { \prime }$ capped and $3 ^ { \prime }$ polyadenylated,yet this class of transcripts has limited coding potential.LncRNAs function in a wide range of biological processes and can regulate gene expression in cis or in trans by diverse mechanisms (Batista and Changï¼Œ2013ï¼› Cech and Steitz, 2014; Ulitsky and Bartel, 2013). LncRNA-mediated biology has been implicated in many cellular processes (Ulitsky and Bartel, 2013).In cancer, IncRNAs have been reported to act as a prominent layer of transcriptional regulation,often by collaborating with chromatin remodeling complexes (Gupta et al., 2010; Prensner et al.ï¼Œ 2013; Yang et al.ï¼Œ 2013; Yuan et al.ï¼Œ 2014). The SWI/SNF complex is an evolutionally conserved multisubunit complex that mobilizes nucleosomes and remodels chromatin using the energy of ATP hydrolysis (Helming et al., 2014; Wilson and Roberts,2011).The SWI/SNF complexassociateswith transcription factors,coactivators,or coexpressors to regulate gene expression (Tolstorukov et al., 2013; You et al., 2013). Inactivating mutations in several SWl/SNF subunits have been identified invarious human cancers (Jones et al.ï¼Œ 2010;Kadoch et al., 2013;Li et al., 2011), suggesting that mutations of some individual subunits promote tumorigenesis.Additionally, the SWl/SNF complex has been implicated in being involved in controlling mammalian stem cell self-renewal and differentiation (Eroglu et al., 2014; Yu et al., 2013; Zeng et al., 2013). However, how the SWl/SNF complex functions in CSCs remains elusive.In this study,we found that IncTCF7 is required for the self-renewal maintenance of liver CSCs.LncTCF7 recruits the SWI/SNF complex to trigger TCF7 expression, leading to activation of Wnt signaling for priming liver CSC self-renewal.

# RESULTS

# LncTCF7IsHighly Expressed in LiverCSCs

Although CD13 (Haraguchi et al.ï¼Œ 2010) and CD133 (Ma et al., 2007)have been widely used as liver CSC surface markers, CD133 or CD13 alone could enrich more cell populations in HCC cell lines or HCC primary samples (Figures S1A and S1B). Haraguchi et al. (201O) demonstrated that the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cell fraction exhibited slow growth compared with theircounterpart CD $1 1 3 ^ { - } \mathsf { C D } 1 3 3 ^ { - }$ cells.Moreover,the $\mathtt { C D 1 3 ^ { + } C D 1 3 3 ^ { + } }$ cell fractionwas highly resistant to chemical drug treatments compared with their $\mathsf { C D 1 3 ^ { - } C D 1 3 3 ^ { - } }$ subset. Notably,we observed that the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cell subsets were successfully enriched in seven cell linesoutofnine HCCcelllineswe testedand in over $90 \%$ of $3 0 { \mathsf { H C C } }$ primary samples we examined (Figure S1B). Thus,we combined CD13 and CD133 to enrich the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cell fraction and thereby identify themas hepatic CSCs in this study.

We then sorted $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cellsï¼Œpresumptively liver CSCsï¼Œfrom Hep3Bï¼ŒHuh7ï¼Œand PLC/PRF/5 HCC cell lines. Actually, the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ subset derived from Hep3B cells significantly enhanced oncosphere formation compared to $\mathsf { C D 1 3 ^ { - } C D 1 3 3 ^ { - } }$ cells ( $1 0 . 3 \% \pm 1 . 3 \%$ versus $2 . 6 \% \pm 0 . 3 \%$ ${ \mathsf p } =$ 0.003) (Figure S1C).In additionï¼Œthe tumor sphere formation frequencies of $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ and $\mathtt { C D 1 3 ^ { + } C D 1 3 3 ^ { - } }$ subsets of Hep3B cells were $1 0 . 3 \% \pm 1 . 3 \%$ versus $6 . 1 \% \pm 1 . 1 \%$ $( { \mathsf p } =$ 0.0253)ï¼› $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ versus $\mathsf { C D 1 3 ^ { - } C D 1 3 3 ^ { + } }$ fractionsof Hep3B cells were $1 0 . 3 \% \pm 1 . 3 \%$ to $5 . 2 \% \pm 1 . 3 \%$ $( \mathsf { p } = 0 . 0 2 2 6 )$ (Figure S1C).Similar observations were obtained using Huh7 and PLC/PRF/5 cells (Figure S1C). Consistently, the tumor-initiating ability of the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ subset derived from Hep3B cells was significantly higher than that of $\mathtt { C D 1 3 ^ { + } }$ or $\mathsf { C D 1 3 3 ^ { + } }$ cells alone (Figures S1Dand S1E).Similar resultswereachieved with Huh7 cells.We concluded that the tumorigenic capacity of $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cells was much higher than that of ${ \mathsf { C D } } 1 3 ^ { + }$ or $\mathsf { C D 1 3 3 ^ { + } }$ cells alone.Of note, there was no significant difference in the proliferative capacity between $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cells and $\mathsf { C D 1 3 ^ { - } C D 1 3 3 ^ { - } }$ cells through cell-cycle analysis (Figure S1F). To identify IncRNAs involved in liver CSCs,we conducted transcriptomemicroarrayanalysis of $\mathsf { C D 1 3 ^ { + } C D 1 3 3 ^ { + } }$ cells (hereafter called liver CSCs) and $\mathsf { C D 1 3 ^ { - } C D 1 3 3 ^ { - } }$ cells (hereafter referred to as non-CSCs) sorted from the three above mentioned HCC cell lines.286 noncoding RNA transcripts were aberrantly expressed in liver CSCs compared with non-CSCs (Figure 1A), with some of those differential IncRNAs validated in liver CSCs when we detected ten randomly selected IncRNAs (Figure S1G).

Since IncRNAs can act in cis to regulate expression of neighboring genes or in trans by diverse mechanisms(Rinn and Chang, 2012),we concentrated on intergenic IncRNAs,which were highlyexpressed in liver CSCs and located in the nearbystem transcription factors and coding genes related to stem signaling pathways.Among these highly expressed intergenic IncRNAs, we focused on an uncharacterized IncRNA,termed IncTCF7 (gene symbol TCONS_00009511-XLOC_004555).LncTCF7 is one of the most highly expressed IncRNAs in liver CSCs (Figure1Aand Figure S1G), residing on chromosome 5 in humans between the heat shock70 kDa protein 4(HSPA4) andT cell factor 7 (TCF7) genes (Figure 1B).LncTCF7 was composed of three exonsand spanned nearly 3.6 kilobases (kb),identifying itasa modestlyconserved locus.We further confirmed that IncTCF7 was highly expressed in liver CSCs (Figure 1C),as wellas in oncosphere cells derived from HCC cell lines and HCC primary samples (Figures 1D and 1E).We next examined a cohort of 37 paired HCC tumor and peri-tumor tissues,along with 14 hepatitis B virus (HBV)-infected liver cirrhotic tissuesand8 normal liver tissues (Table S1).We observed that /ncTCF7was significantly highly expressed in HCC tumors (Figure 1F and Figure S1H) and HCC cell lines (Figure S1l),whereas it was almost undetectable in liver cirrhosis samples and healthy liver tissues (Figure 1F and FigureS1H).Inaddition, IncTCF7wasnotdetectable in some other normal primary tissues we examined (Figure S1J),suggesting a more restricted fashion of IncTCF7 expression.

We further examined the transcripts of IncTCF7 in liver CSCs bynorthern blot.Only one transcriptvariant of IncTCF7was in liver CSCs,with a length between 500 to 1,0o0 bases (Figure 1G). However, IncTCF7 expression was much lower in non-CSCs.A totallength of 683nt of IncTCF7transcriptwasdetermined by a rapid amplification of cDNA ends (RACE) assay (Figure S1K), which displayed no coding potentiality (Figures S1L and S1M). Furthermore,IncTCF7was localized in the nuclei of HCC tumor cells through RNA fluorescence in situ hybridization (RNAFISH) and cellular fractionation assays (Figures 1H and 1l). LncTCF7 was highly expressed in oncosphere cells,but it was poorly expressed in non-sphere tumor cells.These data indicate that IncTCF7,as an IncRNA,was highly expressed in HCC tumor tissues and liver CSCs.

# LncTCF7IsRequiredforSelf-RenewalMaintenanceof Liver CSCs

Todetermine the role of IncTCF7in liver CSC self-renewal,we silenced IncTCF7 against either exon 1 or 3 in liver CSCs using lentivirus-mediated short hairpins RNAs (shRNAs) (Figure 2A). shRNA-2 (against exon 3) achieved more effective knockdown efficiency.Notablyï¼ŒIncTCF7depletionsignificantlyreduced the expression of pluripotent transcription factors $\mathsf { S o x } 2$ ,Nanog, and Oct4 compared with scrambled control (shCtrl) cells (Figure 2B). By contrast, IncTCF7 knockdown did not affect cMyc expression.Actually,c-Myc was highly expressed in both IncTCF7 silenced and shCtrl liver CSCsï¼Œwhose expression was not significantly altered in loss-of-function experiments (Figure2B).Additionally, IncTCF7knockdownremarkably impaired generation of the fraction of $\mathsf { C D 1 3 ^ { + } C D 1 3 3 ^ { + } }$ cells (CSCs) (Figure 2C). Importantlyï¼Œ IncTCF7 depletion dramatically reduced primary(1st),secondary (2nd),and tertiary (3rd) oncosphere formation of HCC cellines and HCC primary cells (Figure 2D).

Consistently, IncTCF7 silencing also impaired 2nd and 3rd oncosphere formation derived from their primary oncosphere cells Similar observations were obtained in IncTCF7 silenced PLC/ PRF/5 cells (data not shown).Consequently, IncTCF7 depletion in HCC primary tumor cells significantly reduced the expression of $S o x 2$ and Nanog (Figure 2E). More importantly,we conducted 2nd and 3rd generation xenograft tumor growth for the $\mathsf { C D 1 3 ^ { + } C D 1 3 3 ^ { + } }$ cell subsets derived from HCC cellines and primary samples.We noticed that $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cell populations (A) Geometricmean-centered,hierarchical cluster heat map from microarray data.286 annotated noncoding RNAs $( \mathsf { p } < 0 . 0 5 )$ wererepresented in liver CSC $( \mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + } )$ compared with nonCSC $( \mathsf { C D } 1 3 ^ { - } \mathsf { C D } 1 3 3 ^ { - } )$ cells sorted from HCC cell lines,Hep3B,Huh7,and PLC/PRF/5 (PLC).Black arrowheaddenotes IncTCF7.

![](images/177694280249ee4497f87c54ed8832f7e98f27c8d525392f3652159fc62d7f54.jpg)  
Figure 1.LncTCF7 Is Highly Expressed in Liver CSCs

(B) Schematic annotation of IncTCF7 genomic locus on chromosome 5.Blue rectangles represent exons (upper panel). Sequence conservation was analyzed by Phylop software (lower panel).

(C) LncTCF7 was detected in liver CSCs and nonCSCs sorted from HCC cell lines by quantitative real-time PCR analysis.Relative gene expression valueswere normalized to endogenous 18S rRNA unless noted otherwise in this study.Results are shown as means $\pm \mathsf { S D }$

(D and E) LncTCF7 was analyzed in oncosphere and non-oncosphere cells derived from three HCC cell lines and HCC primary cells.Results are shownasmeans $\pm \mathsf { S D }$

(F) LncTCF7 was detected in HCC tumor tissues paired with adjacent tumor tissues $( \mathsf { n } = 3 7 )$ ï¼Œ liver cirrhosis tissues $( \mathsf { n } = 1 4 )$ ,and normal liver tissues $( \mathsf { n } = 8 )$ .Data were normalized to endogenous 18S rRNA expression and normal liver tissues were assigned a value of 1.

$( \mathsf { G } )$ A 363nt probe of IncTCF7 was labeled for northern blot analysis.RNAs were extracted from sphere and non-sphere cells derived from HCC primary specimens.U6 RNA was used as a loading control.

(H) LncTCF7 intracellular localization was visualized in HCC oncosphere and non-sphere cells by RNA-FISH assays.Representative images of IncTCF7 in Huh7 oncosphere (Sphere) and nononcosphere cells (Non-sphere) are shown.DAPl, 4',6-diamidino-2-phenylindole.Probes,IncTCF7. Scale bar, $1 0 \mu \mathrm { m }$

(l) Fractionation of HCC oncosphere cells followed by quantitative real-time PCR (left panel)and fractionation controls by immnoblotting (right panel).U1 RNA served as a positive control for nuclear gene expression. EEA1,endosome antigen 1; N, Nuclear fraction; C, Cytoplasmic fraction. Data are shown as means $\pm \mathsf { S D }$ $\mathsf { \Pi } ^ { \star } \mathsf { p } < 0 . 0 5$ ${ \star } { \star } _ { \mathsf { p } } <$ 0.01,and $^ { \star \star \star } \mathsf { p } < 0 . 0 0 1$ by two-tailed Student's t test.Data are representative of at least three independent experiments.See also Figure S1 and Table S1.

were still enriched from the 2nd and 3rd generation xenograft tumors (data not shown).The C $\mathtt { D 1 3 ^ { + } C D 1 3 3 ^ { + } }$ cell populations from the 2nd and 3rd generation displayed stronger tumor-initiating capacities (Figure S2A) and oncosphere forming abilities (Figure S2B)ï¼Œindicating a long-term self-renewal capacity of the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ cell population. Moreoverï¼ŒKi67 signals were not apparently altered in liver CSCs compared with non-CSCs (Figure S2C). However, the amount of Sox2 was significantly increased in liver CSCsï¼Œbut not in non-CSCs.By contrast, IncTCF7 overexpression remarkably enhanced Sox2 signals in liver CSCs,whereas Ki67 was not altered in IncTCF7-overexpressing CSCs compared to empty-vector-treated cells (Figure S2D).Similarobservations were obtained in xenograft tumors (Figure S2E)ï¼Œsuggesting IncTCF7-mediated tumorigenicity in liver cancer is mainly caused by the self-renewal capacity of CSCs.

We next explored the role of IncTCF7 in tumor-initiating formation.We performed subcutaneous injection of BALB/c nude mice with stable knockdown of IncTCF7 and shCtrl cells. LncTCF7 depletion resulted in a much weaker tumor presence compared with shCtrl primary tumor cells as assessed bya limiting dilution xenograft analysis (Figure 2F),suggesting that IncTCF7 knockdown reduced tumor initiating capacity.Moreoverï¼Œ IncTCF7 depletion significantly suppressed xenograft

A shIncTCF7-1 shIncTCF7-2 B Hep3B Huh7 FT shntTCF7.1 1 å‡º 300 shlncTCF7-1 shCtrl S S sh ssd Sox2 ï¼ˆppo) â– shlncTCF7-2 Hep3B 0.1 0.025 T Î²-actin 0.5- \* %0) \* \* Sox2 0 Huh7 Hep3B Huh7 Î²-actin â˜… \*\* shteTCF7-1 D shCtr shlncTCF7-1 shlncTCF7-2 sos :NETCF7A1 çŽ‹I T shlncTCF7-2 Sample#1 o çŽ‹EçŽ‹   
G0 å·¥ 8 .g % 1 å§   
% Hep3B i.988 01 0 Hep3B Huh7 \*@ã€‚ #1 Sample   
E F   
u0ssd 00TP1 500 ï¿¥ ð‘˜\* \* 1 \* TF1 \* 5025 Sample#1 hneTCF7-1 25 50 Sample#19 TCF7-1 0 2 9 0 2 59 0J 0 S0 Numberof graftedells Numerof gratedels Sample#1 Sample#19   
G H Frequencyof Samshe#192 Cell population tumrigenic cels Pvalue è‰² ç¦ å›­ Â® 8 ? #1-shCtrl 1/32522 + äºŒ I é™„ #1-shinCLC57 2.3E-.06 0- 030405060708090 030405060708090 #19-shCtrl 1/44810 Days Days (1/93187-1/21547) #19-shlncTCF7-1 1171888-1/92130) 1.3E-04 #19-shlncTCF7-2 1/202700 2.8E-05 (1/381715-1/107639) (A)LncTC7eseillia shownasmeans $\pm \mathsf { S D }$   
(BPluripotentic Quantitative real-time PCR results are shown as means $\pm \mathsf { S D }$   
(C) $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ (CSC) subpopulations were detected in IncTCF7-depleted cels by FACS analysis.Resultsare shownas means $\pm \mathsf { S D }$ =   
(D)Losilt means $\pm \mathsf { S D }$ .Scale bar, $1 0 0 ~ { \mu \mathrm { m } }$   
(E)PluripotencytranscriptsereanalyedinIcTCF7-silenedHCCprimarycellsbyquantitativealtimePCRanalysisDataresows means $\pm$ SD.   
(F)LncTC7-sincedosCtrHCCprimarytumorcelseredilutedandsubutaneslymplantedintoB/cudemiceTmorserebseeder 4 months. ${ \mathfrak { n } } = 1 2$ for each group.

(legend continued on next page)

Arsesdenrrren B Sample#1 #19   
302() \*\*\* æ°” (S 1081 OeTTTE OerTTE 1 äºº 0.1 \*\* ä¸€ Sox2 1   
0 Î²-actin   
mple#1 mple#19   
C oeVec oelncTCF7 D â–¡oeVec Sample#1 Sample#19 151oelncTCF7 100   
Sample è´¢ rrrrereeee 75 0eVeTCF7 75 0eeCF7   
#1 é«˜ 50 50 å·¥ \* 25 25 0 0   
Hep3B Y 10Â²103104105 10Â²103104105 Number of grafted cells Number of grafted cells   
E F   
200g 0eVeTCF7 H Fnongnis Cell population Pvalue 0mm #1-oeVec (1/325215-1/16268)   
1 T HIII 0 III #1-oelncTCF7 11/102562-1/2936) 2.3E-06 #19-oeVec 1/47334   
03040 50 60 70 030 40 50 60 70 (1/98695-1/22701) Days Days #19-oelncTCF7 1/5979 4.7E-09 (1/12388-1/2886) (A)LncTC7erexpringprarsreblisceexprsitto;expf $T C F 7$ Dataare shownasmeans $\pm \mathsf { S D }$   
(B)Oveexp western blot (right) analyses.Quantitative real-time PCR data are shown as means $\pm \mathsf { S D }$ ï¼š   
(C) LncTCF7 overexpression enhances the capacity of oncosphere formation.Scale bar, $1 0 0 \mu \mathrm { m }$ .oe,overexpression.The right panel represents statistical results shown as means $\pm \mathsf { S D }$   
(D)LTCFvexptctceasdduali BALB/c nude mice.Tumors were observed over 4 months. ${ \mathsf n } = 1 2$ for each group.   
(E)LncTF7verexpreictorotVeceaedprlsebtaeslecdtBdcfose growth. Results are shown as means $\pm \mathsf { S D }$ ${ \mathsf n } = 1 0$ for each group.   
(F)rig edu.au/software/elda/). #1, sample #1; $\yen 19$ sample $\yen 19$ oe,overexpression; Cl,confidence interval.   
$\star _ { \mathsf { p } } < 0 . 0 5$ $^ { \star \star } \mathsf { p } < 0 . 0 1$ ,and $^ { \star \star \star } | \mathsf { p } < 0 . 0 0 1$ by two-tailed Student's ttest. Data represent at least three independent experiments.See also Figure S3.

tumor growth and tumorigenic cell frequency (Figures 2G and 2H).We observed xenograft growth for 4 months and defined a threshold tumor size for detectability of tumors as 30- $4 0 ~ \mathsf { m m } ^ { 3 }$ .Similar observations were achieved in IncTCF7- silenced HCC cell lines (Figures S2F-S2H). Overall, IncTCF7 silencing abrogates the tumorigenic capacity of liver CSCs.

# LncTCF7Overexpression Enhances Tumorigenic CapacityofLiverCSCs

Wenext overexpressed IncTCF7 in HCC primary tumor cells and cell lines and established IncTCF7 stably overexpressing cell lines (Figure 3A and Figure S3A). LncTCF7 overexpression dramatically increased expression of pluripotent transcription

TC è‡ª å¤§ L å¤§ NCTCFTA WON2 \* TCF20 NANOG APC2 WNT4 WNT2B 0 SNAI2 2a TCF7 HSPA4 CAMLG CDKL3 C D DDX46 shlncTCF7 shIncTCF7 ro 2 CoND2 2 E Hep3B Hep3B TCF7 Huh7 CCND2äºŒ_ Huh7 TTTT FZD4 -1 0 log2 Sample E F oeTCF7+ G #3#4#6#7#8#9 shlncTCF7-2 shCtrl shlncTCF7-2 shincTCF7-2 8 PTPTPTPTPTPT shCtrl oeVec oeTCF7 8 8 TCF7 = TCF7 Espobos Î²-actin -.---- å›½ ï¼š Î²-actin TCF7 Huh7 Inc TTTI æ–° 0 MiAPAAb Peri-tumor Tumor H DAPI anti-TCF7 Merge 1 Sample#18 Sample#36 J 82 3 2.0 Sphere Tumor -1.0 sphene tuerir -20Aive Dead Five-year follow-up Huh7 K L M N F shTCF7-1 Hep3B Huh7 Frequency of S +ShCtrl +shCtr tumorigenic cels 4 g100 + ShTCF7-1 8100 + ShTCF7-1 Cell population (95% CI) Pvalue Sox2 TCF7 Hep3B A Hep3B-shCt1579-17567 Hep3B-shTCF7 1/61865 1.4E-04 rcF7 8 Huh7-shCtrl (187 HePHU Sox2 Huh7 sa Huh7-shTCF719226091) 7.1E-06 Î²-actin

factors (Figure 3B)ï¼ŽNotablyï¼Œ IncTCF7 overexpression significantly enhanced oncosphere formation and tumor-initiating capacity (Figures 3C and 3D). Consequently, IncTCF7 overexpression dramaticallyaugmented xenograft tumor growth (Fig ure 3E) and tumorigenic cell frequency (Figure 3F).We tested atleast six HCC primary samples and achieved similar observations.Similar results were also observed in IncTCF7-overexpressed HCC cell lines such as Hep3B and Huh7 cells (Figure S3).All together, these data indicate that IncTCF7 playsa critical role in the self-renewal maintenance of liver CSCs.

# LncTCF7 Triggers TCF7 Expression to Activate Wnt Signaling

Toexplore the target genes of IncTCF7,we established IncTCF7- silenced HCC primary CSC cells and conducted transcriptome microarrayanalysis.LncTCF7knockdown resulted indifferential expression of 2,491 genes (Figure S4A),suggesting that IncTCF7 regulates many genes that relate to important signaling processes including the Wnt signaling pathway (Figure S4B). Notablyï¼Œ IncTCF7knockdown significantly reduced its nearby protein-coding gene TCF7and some major Wntsignaling targets (Figure 4A). By contrast, HSPA4 and the other neighboring genes displayedno significant changes.We then validated these observations in HCC cell lines (Hep3B,Huh7,and PLC) or HCC primary cells (at least six samples). Consistently, IncTCF7 silencing remarkably reduced the expressionofTCF7and majorWnt targets in HCC cell lines and HCC primary cells we examined (Figures 4B and 4C and Figures S4C and S4D). In contrast, IncTCF7 depletion did not alter the expression of HSPA4 and the other neighboring genes (Figures S4C and S4E). In addition, IncTCF7 overexpression in HCC cell lines or HCC primarycells remarkablyenhancedtheexpressionof TCF7andmajorWnttargets (Figure S4F).These data suggest that IncTCF7 activates the Wnt signaling pathway in HCC.

We further tested the expression levels of IncTCF7 and TCF7 in HCC subjects by quantitative real-time PCR.We noticed that IncTCF7 expression was positively correlated with the expression of TCF7 (Pearson correlation coefficient $\mathsf { r } = 0 . 6 9 3$ $\mathsf { p } < 0 . 0 0 1 ^ { \cdot }$ (Figure 4D).To determinewhether IncTCF7 functions upstream of TCF7 in the regulation of liver CSC self-renewal, we rescued TCF7 expression in IncTCF7-silenced HCC cells (Figure4E). Interestingly,TCF7 restoration rescued the oncosphere formation ability reduced by IncTCF7 depletion (Figure 4F). Additionally,TCF7 restorationalso rescued IncTCF7-depletionreduced tumor-initiating capacityand xenograft tumor growth (data not shown). LncTCF7 silencing dramatically downregulated expression of some major Wnt moleculesï¼Œincluding Wnt7aï¼ŒWnt4,and Wnt2b (Figures 4A and 4B).Among these threeWntmolecules,IncTCF7depletion reduced expression of Wnt7a the most.To further validate whether Wnt7a is a downstream effectorof IncTCF7ï¼Œweincubated IncTCF7-silenced HCC primary CSCs with recombinant Wnt7a for a sphere formationassay.We found thatWnt7a rescued the oncosphere formation ability reduced by IncTCF7 depletion (Figure S4G). Additionallyï¼ŒWnt7a also restored the expression levels of TCF7ï¼ŒSox2ï¼Œand Nanog in IncTCF7-depleted HCC primary CSCs (Figure S4H). DKK1 (Dickkopf-related protein 1) was reported to be an extracellular inhibitor of Wnt signaling (Cui et al.,2013).The presence of DKK1 was able to abolish the oncosphere formationabilityenabledby IncTCF7overexpression (Figure S4l).Consistentlyï¼ŒDKK1 treatment also impeded the expression levelsofTCF7,Sox2,andNanog in IncTCF7-overexpressing HCC primary CSC cells (Figure S4J).These data suggest that IncTCF7 sustains the self-renewal of liver CSCs upstream of TCF7.

To further explore the clinical implications of TCF7 in HCC,we analyzed the expressionof TCF7 in HCC tumorand peri-tumor tissues.We observed thatTCF7 was highly expressed in HCC tumors (Figure 4G). Additionallyï¼ŒTCF7 was mainly localized in the nuclei of HCC oncosphere cells,whereas itwas almost undetectable in non-sphere cells (Figure 4H).High expression ofTCF7 in HCC samples was further confirmed by immunohistochemistrystaining (Figure 4l).Importantly,high expression of TCF7 predicteda lower rate of 5-year survival in members of a cohort analyzed by Hoshida et al. (20o9) (Figure 4J). Next, we silenced TCF7 in HCC cells and established stably silenced cell lines (Figure 4K).Notably, TCF7 depletion suppressed Sox2 expression.We observed that TCF7 silencing dramatically impaired oncosphere formation, tumor initiating capacity,and xenograft growth in HCC cell lines and primary cells (Figures 4L-4N,and data not shown). In sum, IncTCF7 initiates TCF7 expression to activate the Wnt signaling pathway, leading to priming of liver CSC self-renewal and tumor propagation.

# LncTCF7 Recruits the SWI/SNF Complex

LncRNAs are considered to exert their functions through RNAinteractingproteins that regulate geneexpression byvarious mechanisms (Geisler and Coller,2013).Therefore,we performed anRNA pull-downassaywith biotin-labeled IncTCF7 to search forpotential IncTCF7-associated proteins.BRG1, BAF170,and SNF5,three core subunits of the SWl/SNF complex,were identified to bind IncTCF7 in liver CSCs (Figures 5A and 5B and Figures S5A-S5C).The interactionof IncTCF7with the three SWI/ SNF components was further validated by RNA immunoprecipitation (RIP) (Figure 5C and Figure S5D).Howeverï¼Œ IncTCF7 depletion did not influence expression levels of BAF170, BRG1,and SNF5 (Figure 5D), suggesting that IncTCF7 was not involved in the post-translational regulation of the SWl/SNF complex.Moreoverï¼ŒIncTCF7 colocalized with BAF170 in the nuclei of HCC oncosphere cells,whereas IncTCF7 was poorly expressed in the nuclei of non-sphere cells (Figure 5E). These dataindicate that IncTCF7 associateswith the SWl/SNF complex in the nuclei of liver CSCs.

We next constructed a series of IncTCF7truncations to map its binding fragment with the SWl/SNF complex.We found that the $3 ^ { \prime }$ -end fragment of IncTCF7 (nt 468 to 683) was sufficient to bind BAF170, BRG1,and SNF5 (Figure 5F). A stable stem-loop structure of the $3 ^ { \prime }$ -end fragment was predicted by RNA folding analysis (Figure S5E). The binding of the $3 ^ { \prime }$ -endregionof IncTCF7 with BAF170 was further confirmed by an RNA electrical mobility shift assay (EMSA) (Figures 5G and 5H).Unlabeled IncTCF7 probes competitivelydisrupted this bindingcapacity.Additionally,IncTCF7showed high bindingaffinity(dissociationconstant [Kd] $3 6 . 2 4 ~ \mathsf { n M } )$ (Figure 5l).These data indicate that IncTCF7 directly binds to the SWl/SNF complex with high affinity.

# LncTCF7 Triggers TCF7 Expression to Activate Wnt Signaling

Given that the SWl/SNF complex regulates gene transcription by binding to promoter loci and refoldingchromatin (Robertsand Orkinï¼Œ2004)ï¼Œwe then detected whether IncTCF7 influenced BAF170 occupancy of the promoter locus of the TCF7 gene. Weanalyzeda3kb locus upstream from the transcription start sites(TSSs) of the TCF7 gene.We found that IncTCF7 depletion abrogated the binding capacityof BAF170with (approximately)a -1,160 to -1,048 bp segment of TCF7 promoter (Figure 6A), suggesting that this segment was the binding site for IncTCF7. Through luciferase reporter assaysï¼Œwe identified (approximately)a -1,200 to -1,100 bp segment of TCF7 promoter asasufficient binding site for BAF170 (Figures 6B-6D).Finally, we observed via a DNase ä¸¨ digestion assay that IncTCF7 or BAF170 knockdown significantly impaired the chromatin accessibility of the TCF7 locus (Figure 4E).By contrast,BAF170was not recruited on the promoters of the stem pluripotent factors, asassessed throughchromatin immunoprecipitation (ChiP) analyses (Figures S6A-S6D),suggesting that BAF170 had an indirect effect on the expression of stem pluripotent factors.These data suggest that IncTCF7 recruits the SWI/SNF complex to the TCF7 promoter, thereby leading to its activation.

Notably,the SWl/SNF complex was highly expressed in HCC tumors and liver CSCs (Figure 6F and data not shown), suggesting that the SWl/SNF complexmay be implicated in the regulation of liver CSC self-renewal. Consequently, BAF170 depletion dramatically reduced expression of TCF7 and Nanog (Figure 6G)ï¼Œas well as Wnt downstream targets such as Sox2, CCND1,and CCND2.Additionally,BAF170 knockdown significantly impaired the ability of oncosphere formation derived from HCC cellines and HCC primary cells (Figure 6H). Finally, BAF170 knockdown also remarkably reduced tumor-initiating capacityand xenograft tumor growth (Figures 6l-6K).Similar results were observed by depleting SNF5 and BRG1 (data not shown)ï¼ŽAll togetherï¼Œour data show that IncTCF7 triggers TCF7 expression through recruitment of the SWl/SNF complex, leading to priming of the self-renewal of liver CSCs and tumor initiation.

# DISCUSSION

Recentlyï¼ŒCSCs have been identified in many solid tumors, including breast, lung,liver,brain,colon,prostate,and bladder cancers (Haraguchi et al., 2010; O'Brien et al.ï¼Œ 2007;Visvader andLindeman,2012).CSCs have stemcharacteristics such as self-renewal and differentiation.CSCs may account for cancer relapse and metastasis due to their invasive and drug-resistant capacities (Gupta et al.,20o9).Several surface markers have beenidentified in liver CSCs.Howeverï¼Œdetails of liver CSC biology remain largely unknown.In this study,we isolated the $\mathsf { C D } 1 3 ^ { + } \mathsf { C D } 1 3 3 ^ { + }$ subpopulation of cells from HCC cell lines and HCC primary cells and identified themas liver CSCs.Through transcriptome microarray analysis,we identified 286 differentially expressed noncoding RNAs in liver CSCs.Among these highly expressed IncRNAs in liver CSCs,we defined an IncRNA termed IncTCF7 that plays a critical role in priming the selfrenewal of liver CSCs.

LncRNAs have been reported to play widespread roles in gene regulation and other cellular processes (Batista and Chang, 2013; Flynn and Changï¼Œ2014).LncRNAs exert their functions via diverse mechanisms,including cotranscriptional regulation, modulation of gene expression, scaffolding of nuclear or cytoplasmic complexes,and pairing with other RNAs (Ulitsky and Bartelï¼Œ2013)ï¼ŽNotablyï¼Œaccumulating evidence shows that IncRNAs modulate gene expression as epigenetic modifiers (Kaneko et al., 2014; Klattenhoff et al., 2013; Prensner et al., 2013; Tsai et al., 2010; Zhu et al., 2013). Here we found that IncTCF7 activates TCF7 expression in cis through recruitment of the SWI/SNF complex. TCF7 expression triggers Wnt signaling to initiate self-renewal of liver CSCs.Although IncTCF7 isexpressed in the antisense direction,it fails to form an IncTCF7- TCF7 mRNA complex for the regulation of TCF7 expression. The Wntsignalingplaysapivotal role inself-renewal and differentiation of CSCs (Hoffmeyer et al., 2012; Myant et al., 2013). Aberrant activation of Wnt signaling is implicated in liver cancer and other chronic liver diseases (Huch et al., 2013;Wang et al., 2013a).Here we show that depletion of TCF7 and Wnt downstream target genes impairs the self-renewal of liver CSCs and their tumor-initiating capacity,suggesting that TCF7-mediated

![](images/e26476e0ae422761381a2db05328148cc4f04c03379e8289f50ba4600b66db81.jpg)  
Figure 5. LncTCF7 Associateswith the SWI/SNF Complex

(A)Biotin-pdcssitss IncTCF7 intron controls.This was followed by mass spectrometry.Banda,BRG1; band b,BAF170;bandc,SNF5.   
(B)Three core components of the SWl/SNF complex were confirmed by immunoblotting. $\beta$ -actin was used as a loading control.   
(C)TheinteractioofF7ithBF17,G1ndsverfidynNAimunoprepitatio(I)ssaResulteoas $\pm$ SD. \*\*p < 0.01bytwo-tailed Student's ttest.   
(D) LncTCF7 depletion does not alter the protein levels of BAF170,BRG1,and SNF5.   
(E)LcTCF7asaiddnofesesngofB7incspdoheesspefoda $1 0 ~ \mu \mathrm { m }$   
(F)Mappinganalsisfomplebgomasofoellaticaffutdd frats(p IncTCF7 fragments (bottom panel).   
(G)BAF170 protein and biotin-labeled IncTCF7(468-683 nt) probes were incubated for an EMSA assay.   
(H)DiferentconcentrationsofBAF170proteinwereicubatedwithIncTCF7probes (5nM)andfollowedbyEMSAassaysasabove (G).   
()ndiiss See also Figure S5. A BAF170 ChIP in TCF7 promoter B å±± C 70- 170- shCtrl Y ShBAF170 eirneeele shCtrl shincTCF7-1 BF â–¡shBAF170-1 \*\* â–¡shincTCF7-2 1.5 \*\* BAF170 Hep3B Î²-actin 0 446 8 2 BAF170 2 vec 5 930 8 Huh7 L 000 Î²-actin   
D E PL3-100 11000 reeeeree 1.51 hBAF170-1 TCF7 promoter -1200-1000 pGL3-1200 Luc E egelee 0.5 ector ååååå sINTDE 1 -1100\~ 1200\~ -1300 400\~ -1500 -80 oU 1u   
F G   
1 \* H â–¡shCtrl shctg u \*\* Â·Â·Â· I h TCF7 æ³° 20 ä¸­ 2 NND2 L s soa Huh7 Î²-actin   
H â–¡shCtrl 1 shCtrl shBAF170-1 æ‰€ + shCtrl ç¦ ShCAF170-1 5 Â® un ç¦ Sample#1 10Â² 103104105 10Â²103104105 Number of grafted cells Number of grafted cells   
J K Sample#6 Huh7 Frequencyof shCrI snBAF10-1 SshCr BAF1701 Celpopaion Pvalue â†’ shCAF170-1 shCAF170-1 #6-shBAF170-1 1992-1/620) 7.3E-03 1000- ï¼š 1000- 0 030405060708090 8 030405060708090 Huh7-shCtrl (1/18864-1/4896) 1/9611 Days Days Huh7-shBAF170-1 1/41901 3.1E-05 (1/78446-1/22381)

Wntsignaling is critical forsustaining liver CSC self-renewal and tumorigenicity.

CSCs harbor the stem cell properties of self-renewal and differentiation (Visvader and Lindeman,2012).In this studyï¼Œwe found that there wasno significant difference in the proliferative capacity between liver CSCs and non-CSCs,as assessed by cell-cycle analysis and immunofluorescence staining. Actually, CSCs exist mainly in the G1/G0 phase or G0 phase with dormant or slow-growing properties (Haraguchi et al.,201O). As such, it might be hard to distinguish the proliferation differences between liver CSCs and non-CSCs via Ki67 staining or DNA contentsassayed by propidium iodide (Pl).Also,the self-renewal properties of different cancer typesmay have different proliferation characteristics.The maintenance of self-renewal isan extremely complicatedbiological processï¼Œwhich is controlled byepigenetic states,pluoripotent transcription factors,epigenetic complexesï¼Œand many developmental pathways (Boumahdi et al., 2014; Ohnishi et al., 2014; Visvader and Lindeman, 2012).Our data support the idea that IncTCF7-mediated Wnt signaling activation in hepatic CSCs plays a critical role in the regulation of liver CSC self-renewal.

TCF7,as a T cell factor,is best known for its function in T lymphocyte development and multipotential hematopoietic cell self-renewal (Weberet al.ï¼Œ2011).TCF7 can initiate the Wnt signaling cascade as an upstream trigger (Cui et al., 2013; Weber et al., 2011). $T C F 7 ^ { - / - }$ mice developed intestinal and mammary adenomas,implying a role for TCF7 in tumor suppression (Yu et al.ï¼Œ2012). In this studyï¼Œwe demonstrate thatTCF7 is required for the self-renewal of liver CSCs and tumor propagation and acts as a tumor promoter.TCF7 is regulated by IncTCF7 in liver CSCs to trigger the Wnt signaling pathwayï¼Œwhich primes the self-renewal of liver CSCs.A recentstudy reported thatTCF7 functionsasatumorenhancer intumorigenesis after ocular transplantation of embryonic stem cell-derived retinal progenitor cells (Cui et al.ï¼Œ2013). Consistentlyï¼Œour findings reveal that IncTCF7mediated TCF7 expression,which initiates the Wnt signaling cascade to prime self-renewal of liver CSCs and hepatic tumorigenicity as a tumor promoter.CSCs are analogous to tissue-specific stem cells in that they contribute to long-term growth and are responsible for the maintenance and growth of tumors (Beck and Blanpainï¼Œ 2013)ï¼Ž Transcriptional regulationis oneof the key regulatory mechanisms determining cell fate, which includes cis-regulatory elements and transacting factors. Amongall transacting factors,chromatin remodeling regulation plays a pivotal role in gene transcription (Krasteva et al., 2012; Narlikar et al., 2013),which depends on ATP-dependent chromatin remodeling complexes.The SWl/SNF complex is composed of 12-15 subunits containing one of the two catalyticATPase subunitsï¼ŒSMARCA4/BRG1 or SMARCA2/BRM, and several core components such as SMARCB1/SNF5, BAF170,and BAF155 (Helming et al., 2014; Wilson and Roberts,2011).The SWI/SNF complex regulates gene expression through mobilizing nucleosomes and remodeling chromatin using the energy of ATP hydrolysis,suggesting its critical role in gene expression. Inactivating mutations in several SWl/SNF subunits cause various human cancers (Helming et al.,2014; Jones et al.ï¼Œ2010ï¼› Kadoch et al.ï¼Œ2013)ï¼Œsuggesting that some components act as tumor suppressors.A recent study reported that the long noncoding RNA SChLAP1 associates with the SNF5 subunit to promote prostate cancer progress by antagonizing the function of the SWl/SNF complex (Prensner et al.,2013),which implies that SNF5 functions as a tumor suppressor.Here we found that IncTCF7 can recruit the SWI/SNF complex to trigger TCF7 expression, leading to priming of liver CSC self-renewal.Additionallyï¼Œwe observed that BAF170,BRG1,and SNF5 are highly expressed in HCC specimens and liver CSCs,whose depletion impairs tumor-initiating capacity and tumor propagation (Figure 6,and data not shown), suggesting that these three components act as tumor promoters.Thereforeï¼Œhow the SWI/SNF complex components function in tumorigenesis remains to be further investigated.

In conclusion, IncTCF7 can promote liver CSC self-renewal and tumor propagation through activation of Wnt signaling by recruiting the SWI/SNF complex to the TCF7 promoter. Our findings reveal that IncRNAs may represent an additional layer of regulation of oncogenesis.

# EXPERIMENTALPROCEDURES

# Northern Blot Analysis

Total RNA,extracted as described (Wang et al.,2013b) from human CSC oncospheres and control cells,was subjected to northern blot assays.An IncTCF7-specific DNA sequence (nt320-683) was cloned into pcDNA4/mychis B Vector.Radioactive RNA probe with a length of 363 nt was generated using [Î±- $^ { 3 2 } \mathsf { P l }$ CTP (Perkin Elmer) and the Riboprobe in vitro transcription labeling system (Promega).

# RNA-FISH

Fluorescence-conjugated IncTCF7 probes were used for RNA-FISH. RNA-FISH was performed as previously described (Wang et al.ï¼Œ2014). Hybridization was carried out using DNA probe sets (Biosearch Technologies)according to the protocol of Biosearch Technologies.Oncosphere and control cells were observed with a FV1oo0 confocal laser microscope (Olympus).

# RNA Pull-down and Mass Spectrometry Assay

RNA pull-down was performed as described (Klattenhoff et al., 2013). In vitro biotin-labeled RNAs (IncTCF7,itsantisense RNA,and two control RNAs from two introns of IncTCF7)were transcribed with the biotin RNA labeling mix (Roche) and T7 RNA polymerase (Roche) treated with RNase-free DNase I (Promega) and purified with RNeasy Mini Kit (QlAGEN).Biotinylated RNA was incubated with HCC oncosphere cell nuclear extracts,and pull-down proteins were run on SDS-PAGE gels as described (Xia et al.,2013).Mass spectrometry followed.

# RNA-EMSA Assay

EMSA experiments were performed using a LightShift Chemiluminescent RNA EMSA Kit (Thermo Scientific).The shifted signals were quantified and plotted against the amount of BAF170 protein with GraphPad Prism 6 (GraphPad).

# Statistical Analysis

Data were analyzed with a double-sided Student's t test using the SPSS13.0 software and GraphPad Prism 6.Tumorigenic cellfrequency was calculated based on extreme limiting dilution analysis (ELDA) (http://bioinf.wehi.edu.au/ software/elda/). $\mathsf { p } < 0 . 0 5$ was considered statistically significant.For other methods see the Supplemental Information.

# ACCESSIONNUMBERS

Microarray data have been deposited in the NCBl GEO under accession numbers GSE66515 and GSE66529.

# SUPPLEMENTALINFORMATION

Supplemental Information for this article includes six figures,one table,and Supplemental Experimental Procedures and can be found with this article onlineat http://dx.doi.org/10.1016/j.stem.2015.03.003.

# AUTHORCONTRIBUTIONS

Y.W.designed and performed experiments,analyzed data,and wrote the paper,and L.H. provided clinical specimens and analyzed data.P.Z., G.H.,and J.L.constructed plasmids and analyzed data; Y.D.,X.Y., C.L.,P.X.,and G.Z. performed some experimentsï¼›B.Y.and Y.T.performed some experiments and analyzed data; R.C.initiated the study;and Z.F.initiated the study and organized,designed,and wrote the paper.

# ACKNOWLEDGMENTS

We thank Dr. Zeguang Han (Shanghai Jiaotong University School of Medicine, Shanghai) for providing hepatocellular carcinoma cell lines.We thank Drs. Xinlu Wang,Zhensheng Xie,Junying Jia,Yan Teng,and Jing Li (Cnkingbio Company Ltd,Beijing,China) for technical support.This work was supported by the National Natural Science Foundation of China (81330047,91419308, 81402459,81272270,and 81101531); the State Projects of Essential Drug Research and development (2012ZX09103301-041)ï¼›973 Program of the MOST of China (2015CB553705 and 2010CB911902);and the Strategic Priority Research Programs of the Chinese Academy of Sciences (XDA01010407).

Received:November19,2014   
Revised:February 15,2015   
Accepted:March7,2015   
Published: April 2, 2015

REFERENCES   
Batista,P.J.,and Chang, H.Y.(2013). Long noncoding RNAs: celular address codes in development and disease. Cell 152,1298-1307.   
Beck,B.,and Blanpain, C. (2013). Unravelling cancer stem cell potential. Nat. Rev. Cancer13,727-738.   
Boumahdi, S., Driessens,G., Lapouge,G., Rorive,S., Nassar,D.,Le Mercier, M., Delatte,B., Caauwe,A.,Lenglez,S., Nkusi,E., et al. (2014).SOX2 controls tumour initiation and cancer stem-cell functions in squamous-cell carcinoma. Nature 511,246-250.   
Cech,T.R.,and Steitz, J.A. (2014).The noncoding RNA revolution-trashing old rules to forge new ones. Cell157,77-94.   
Cui,L., Guan,Y., Qu,.,Zhang,J.,Liao,B.,Ma,B.,Qian,J.,Li,D.,Li,W.,Xu, G.T.and Jin,Y. (2013).WNT signaling determines tumorigenicity and function of ESC-derived retinal progenitors.J. Clin.Invest. 123,1647-1661.   
Easwaran,H.,Tsai, H.C.,and Baylin,S.B.(2014). Cancer epigenetics: tumor heterogeneityï¼Œplasticity of stem-like states,and drug resistance.Mol. Cell 54ï¼Œ716-727.   
Eroglu,E.,Burkard,T.R.,Jiang,Y.,Saini,N., Homem, C.C.,Reichert,H.and Knoblich,J.A. (2014).SWI/SNF complex prevents lineage reversion and induces temporal patterning in neural stem cels. Cell 156,1259-1273.   
Flynn,R.A.,and Chang,H.Y. (2014).Long noncoding RNAs in cel-fate programming and reprogramming. Cell Stem Cell 14,752-761.   
Geisler,S.,and Coler,J.(2013). RNA in unexpected places: long non-coding RNA functions in diverse cellular contexts.Nat.Rev.Mol. Cell Biol.14, 699-712.   
Gupta,P.B.,Chaffer, C.L.,and Weinbergï¼ŒR.A. (2009). Cancer stem cells: mirage or reality? Nat.Med.15,1010-1012.   
Gupta,R.A.,Shah,N.,Wang, K.C., Kim,J., Horlings, H.M., Wong,D.J.,Tsai, M.C.,Hung,T.,Argani,P.,Rinn,J.L.,et al. (2010).Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis.Nature 464,1071-1076.   
Haraguchi,N.,Ishi, H., Mimori,K.,Tanaka,F.,Ohkuma,M.,Kim, H..,Akita, H.,Takiuchi,D., Hatano,H., Nagano,H.,et al. (2010).CD13 is a therapeutic target in human liver cancer stem cells.J. Clin.Invest.120,3326-3339. Helming,K.C.,Wang,X.,and Roberts,C.W. (2014).Vulnerabilities of mutant SWI/SNF complexes in cancer. Cancer Cell 26,309-317.   
Hoffmeyer,K.,Raggioli,A.,Rudloff,S.,Anton,R.,Hierholzer,A.,Del Valle,l., Hein,K., Vogt, R.,and Kemler,R. (2012).Wnt/Î²-catenin signaling regulates telomerase in stem cells and cancer cells.Science 336,1549-1554.   
Hoshida,Y.,Nijman,S.M.,Kobayashi, M.,Chan,J.A.,Brunet,J.P., Chiang, D.Y.ï¼ŒVillanuevaï¼ŒA.ï¼ŒNewel,P.ï¼ŒIkedaï¼ŒK.ï¼ŒHashimotoï¼ŒM.ï¼Œet al.(2009). Integrative transcriptome analysis reveals common molecular subclasses of human hepatocellular carcinoma. Cancer Res.69,7385-7392.   
Huch,M.,Dorrel,C.,Boj,S.F.,van Es,J.H.,Li,V.S.,van de Wetering,M.,Sato, T.,Hamer, K., Sasaki, N.,Finegold, M.J.,etal. (2013).In vitro expansion of singleLgr5 $\Game$ liver stem cells induced by Wnt-driven regeneration.Nature 494, 247-250.   
Jemal,A.,Bray,F.,Center,M.M.,Ferlay,J.,Ward,E.,and Forman,D. (2011). Global cancer statistics.CA Cancer J. Clin.61,69-90.   
Ji,J.,Shi,J.,Budhu,A.,Yu,Z.,Forgues,M.,Roessler,S.,Ambs,S., Chen,Y., Meltzer,P.S., Croce,C.M.,et al. (2009). MicroRNA expression,survival,and response to interferon in liver cancer.N.Engl.J. Med.361,1437-1447. Jones,S., Wang,T..,Shih,leM., Mao,T..,Nakayama,K.,Roden,R.,Glas, R., Slamon,D., Diaz, L.A.,Jr., Vogelstein, B.,et al. (2010).Frequent mutations of chromatin remodeling gene ARID1A in ovarian clear cell carcinoma.Science 330,228-231.   
Kadoch,C., Hargreaves,D.C., Hodges,C.,Elias,L.,Ho,L., Ranish,J.,and Crabtree,G.R. (2013).Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy. Nat. Genet.45,592-601.   
Kaneko,S.,Bonasio,R.,Saldana-Meyer,R.,Yoshida,T.,Son,J.,Nishino,K., Umezawa,A.,and Reinberg,D.(2014). Interactions between JARID2 and noncoding RNAs regulate PRC2 recruitment to chromatin.Mol. Cell 53,290-300. Klattenhoff, C.A.,Scheuermann,J.C.,Surface,L.E., Bradley,R.K., Fields,P.A., Steinhauser,M.L.,Ding,H.ï¼ŒButtyï¼ŒV.L.,Torrey,L.,ï¼ŒHaas,S.,et al. (2013). Braveheartï¼Œa long noncoding RNA required for cardiovascular lineage commitment. Cell 152,570-583.   
Krasteva,V.,Buscarlet,M.,Diaz-Tellez,A.,Bernard,M.A.,Crabtree,G.R.,and Lessard,J.A.(2012).The BAF53a subunit of SWI/SNF-like BAF complexes is essential for hemopoietic stem cellfunction. Blood 120,4720-4732.   
Lee,T.K., Castilho,A., Cheung, V.C.,Tang, K.H., Ma, S.,and Ng,I.O. (2011). ${ \mathsf { C D } } 2 4 ( + )$ liver tumor-initiating cells drive self-renewal and tumor initiation through STAT3-mediated NANOG regulation. Cell Stem Cell9,50-63. Li,M,Zhao,H.,Zhang,X.,Wood,L.D.,Anders,R.A.,Choti,M.A.,Pawlik,T.M., Daniel,H.D.,Kannangai, R., Offerhaus,G.J.,et al. (2011). Inactivating mutations of the chromatin remodeling gene ARID2 in hepatocellular carcinoma. Nat. Genet.43,828-829.   
Ma,S.,Chan, K.W.,Hu,L.,Lee,T.K., Wo,J.Y.,Ng,I.O.,Zheng,B.J.,and Guan, X.Y.(2007). Identification and characterization of tumorigenic liver cancer stem/progenitor cels.Gastroenterology 132,2542-2556.   
Myant,K.B.ï¼ŒCammareriï¼ŒP.ï¼ŒMcGheeï¼ŒE.J.ï¼ŒRidgwayï¼ŒR.A.ï¼ŒHuelsï¼ŒD.J., Cordero,J.B., Schwitalla,S., Kalna,G., Ogg,E.L.,Athineos,D.,et al. (2013). ROS production and NF- $\kappa \mathsf { B }$ activation triggered by RAC1 facilitate WNTdriven intestinal stem cell proliferation and colorectal cancer initiation.Cell Stem Cell 12,761-773.   
Narlikar,G.J.,Sundaramoorthy,R.,and Owen-Hughes,T.(2013).Mechanisms and functions of ATP-dependent chromatin-remodeling enzymes.Cell 154, 490-503.   
O'Brien,C.A.,Pollett,A., Galinger,S.,and Dick,J.E.(2007).A human colon cancer cell capable of initiating tumour growth in immunodeficient mice. Nature 445,106-110.   
Ohnishi, K., Semi, K.,Yamamoto,T., Shimizu,M.,Tanaka,A.,Mitsunaga,K., Okita,K., Osafune,K.,Arioka,Y.,Maeda,T.,et al. (2014).Premature termination of reprogramming in vivo leads to cancer development through altered epigenetic regulation. Cell 156,663-677.   
Prensner,J.R.,lyer,M.K.,Sahu,A.,Asangani,I.A.,Cao,Q.,Patel,L.,Vergara, I.A.,Davicioni, E.,Erho,N.,Ghadessi, M.,et al.(2013).The long noncoding RNA SChLAP1 promotes aggressive prostate cancer and antagonizes the SWI/SNF complex. Nat. Genet.45,1392-1398.   
Rinn,J.L.,ï¼Œand Chang,H.Y. (2012).Genome regulation by long noncoding RNAs.Annu. Rev. Biochem.81,145-166.   
Roberts, C.W.,and Orkin, S.H.(2004).The SWl/SNF complex-chromatin and cancer. Nat. Rev.Cancer 4,133-142.   
Tolstorukov,M.Y.ï¼ŒSansam,C.G.ï¼ŒLu,P.ï¼ŒKoellhofferï¼ŒE.C.ï¼ŒHelmingï¼ŒK.C., Alver,B.H.,Tillman,E.J.,Evans,J.A.,Wilson,B.G.,Park,P.J.,and Roberts, C.W.(2013).Swi/Snf chromatin remodeling/tumor suppressor complex establishes nucleosome occupancy at target promoters. Proc.Natl. Acad. Sci. USA 110,10165-10170.   
Tsai, M.C., Manor,O., Wan,Y., Mosammaparast, N., Wang,J.K.,Lan,F., Shi, Y., Segal, E.,and Chang,H.Y. (201O). Long noncoding RNA as modular scaffold of histone modification complexes. Science 329, 689-693.   
Ulitsky,I.,and Bartel, D.P.(2013).lincRNAs: genomics,evolution,and mechanisms. Cell 154,26-46. Visvader, J.E.(2011).Cells of origin in cancer. Nature 469,314-322.   
Visvader,J.E.,and Lindeman,G.J.(2012).Cancer stem cells: current status and evolving complexities. Cell Stem Cell 10,717-728.   
Wang,J.,Park,J.S.,Wei,Y.,Rajurkar,M.,Cotton,J.L.,Fan, Q.,Lewis,B.C.,Ji, H.,and Mao,J. (2013a).TRlB2 acts downstream of Wnt/TCF in liver cancer cells to regulate YAP and C/EBPÎ± function.Mol. Cell 51,211-225.   
Wang,S., Xia,P.,Ye,B., Huang,G., Liu,J.,and Fan,Z. (2013b).Transient activation of autophagy via Sox2-mediated suppression of mTOR is an important early step in reprogramming to pluripotency.Cell Stem Cell 13,617-625. Wang,P.,Xue,Y., Han,Y., Lin,L.,Wu,C.,Xu,S., Jiang,Z.,Xu,J., Liu,Q.,and Cao,X. (2014).The STAT3-binding long noncoding RNA Inc-DC controls human dendritic cell differentiation.Science 344,310-313.   
Weber, B.N., Chi,A.W., Chavez,A.,Yashiro-Ohtani, Y.,Yang, Q.,Shestova, O.,and Bhandoola,A. (2011).A critical role for TCF-1 in T-lineage specification and differentiation. Nature 476, 63-68.   
Wilson,B.G.,and Roberts,C.W. (2011). SWl/SNF nucleosome remodellers and cancer. Nat. Rev. Cancer 11,481-492.   
Xia,P.,Wang,S., Du,Y.,Zhao,Z.,Shi,L.,Sun,L., Huang,G.,Ye,B.,Li,C.,Dai, Z.,et al.(2013).WASH inhibits autophagy through suppression of Beclin 1 ubiquitination.EMBO J.32, 2685-2696.   
Yamashita,T.,and Wang, X.W.(2013). Cancer stem cels in the development of liver cancer.J. Clin.Invest.123,1911-1918.   
Yang,F., Huo,X.S., Yuan,S.X.,Zhang,L.,Zhou,W.P.,Wang,F.,and Sun,S.H. (2013). Repression of the long noncoding RNA-LET by histone deacetylase 3 contributes to hypoxia-mediated metastasis.Mol. Cell 49,1083-1096. You,J.S.,De Carvalho,D..,Dai,C.,Liu,M.,Pandiyan,K.,Zhou,X.J.,Liang, G.,and Jones,P.A. (2013).SNF5 is an essential executor of epigenetic regulation during differentiation.PLoS Genet.9,e1003459.   
Yu, S.,Zhou,X., Steinke,F.C.,Liu, C., Chen, S.C., Zagorodna,O., Jing,X., Yokota,Y.,Meyerholz,D.K.,Mullighan,C.G.,et al.(2012).The TCF-1 and LEF-1 transcription factors have cooperative and opposing roles in T cell development and malignancy.Immunity 37,813-826.   
Yu,Y.,Chen,Y.,Kim,B., Wang, H., Zhao, C., He,X.,Liu,L.,Liu,W.,Wu,L.M., Mao,M.,et al. (2013). Olig2 targets chromatin remodelers to enhancers to initiate oligodendrocyte differentiation. Cell 152,248-261.   
Yuan,J.H.,Yang,F.,Wang,F.,Ma,J.Z.,Guo,Y.J.,Tao,Q.F.,Liu,F.,Pan,W., Wang,T.T., Zhou, C.C.,et al.(2014).A long noncoding RNA activated by TGF$\beta$ promotes the invasion-metastasis cascade in hepatocellular carcinoma. Cancer Cell 25,666-681.   
Zeng,X.,Lin,X.,and Hou, S.X. (2013).The Osa-containing SWl/SNF chromatin-remodeling complex regulates stem cell commitment in the adult Drosophila intestine. Development 140,3532-3540.   
Zhao,W.,Wang,L., Han,H.,Jin,K.,Lin, N.,Guo,T., Chen,Y., Cheng, H.,Lu, F., Fang,W.,et al. (2013).1B50-1,a mAb raised against recurrent tumor cells, targets liver tumor-initiating cells by binding to the calcium channel Î±2Î´1 subunit. Cancer Cell 23,541-556.   
Zhu,Y.,Rowley,M.J.,Bohmdorfer,G.,and Wierzbicki,A.T.(2013).A SWI/SNF chromatin-remodeling complex acts in noncoding RNA-mediated transcriptional silencing.Mol. Cell 49,298-309.