# PTEN deficiency reprogrammes human neural stem cells towards a glioblastoma stem cell-like phenotype

Shunlei Duan1\*， Guohong Yuan1,\*， Xiaomeng $\mathsf { L i u } ^ { 2 , \star }$ Ruotong $\mathsf { R e n } ^ { 1 , 3 , 4 } .$ Jingyi ${ \mathsf { L i } } ^ { 2 }$ ，Weizhou Zhang5, Jun Wu6, Xiuling $\mathsf { X } \mathsf { u } ^ { 1 }$ Lina $\mathsf { F u } ^ { 1 }$ Ying Li1,Jiping Yang1,Weiqi Zhang1,Ruijun Bai3,4,Fei $\mathsf { Y i } ^ { \top }$ ，Keiichiro Suzuki6,8,Hua Gao9, Concepcion Rodriguez Esteban6, Chuanbao Zhang1o, Juan Carlos lzpisua Belmonte6， Zhiguo Chen1, Xiaomin Wang10，Tao Jiangl0，Jing ${ \mathsf { Q } } { \mathsf { u } } ^ { 4 } .$ ,Fuchou Tang2,2,13,14&Guang-Hui Lu1

PTEN is a tumour suppressor frequently mutated in many types of cancers. Here we show that targeted disruption of PTEN leads to neoplastic transformation of human neural stem cells (NSCs),but not mesenchymal stem cels. PTEN-deficient NSCs display neoplasm-associated metabolic and gene expression profiles and generate intracranial tumours in immunodeficient mice.PTEN is localized to the nucleus in NSCs,binds to the PAX7 promoter through association with cAMP responsive element binding protein 1 (CREB)/CREB binding protein (CBP） and inhibits PAX7 transcription. PTEN deficiency leads to the upregulation of PAX7,which in turn promotes oncogenic transformation of NSCs and instates ‘aggressveness'in human glioblastoma stem cels.Ina large clinical database，we find increased PAX7 levels in PTENdeficient glioblastoma.Furthermore,we identify that mitomycin Cselectively triggers apoptosis in NSCs with PTEN deficiency. Together，we uncover a potential mechanism of how PTEN safeguards NSCs,and establish a cellular platform to identify factors involved in NSC transformation,potentially permitting personalized treatment of glioblastoma.

suppressor whose loss-of-function mutations are often encountered in human cancers. PTEN mutationsare observed in $6 0 \%$ of glioblastoma multiforme (GBM) and are among the most frequent genetic alterations linked to $\mathrm { { G B M ^ { 1 } } }$ GBMs bearing PTEN loss-of-function mutations are usually associated with increased invasive behavioursand drug resistance2-4.Glioblastoma stem cells (GSCs)，the tumorigenic component of GBM,represent a rare cell population that are resistant to conventional radio- or chemo-therapy，and are presumably involved in cancer relapse5,6. Evidence from mouse tumour models reveals neural precursor/stem cells as the cell-oforigins for GBM or GSCs7-9, ，and GBM is postulated to be derived from transformed neural stem cells (NSCs) that undergo carcinogenic hits10. The high mutation rate of PTEN in GBM suggests its potential as one of the initiating oncogenic events or a key factor in promoting cancer aggressiveness,similarly as seen in endometrial cancerl1. The correlation between PTEN deficiency and poor prognosis suggests a more complex role of PTEN loss in GBM progression. These observations raise an interesting question,that is,how PTEN loss leads to GBM initiation or promotes its progression?

Mouse models have been successfully used to investigate the roles of genetic mutations in triggering oncogenic NSC transformation and/or mediating GBM pathogenesis12,13.The known differences between mouse and human cancer biology, including differential telomere length,distinct utilization of $\mathrm { p } 1 6 ^ { \mathrm { I N K 4 \breve { a } } }$ RB versusp53 signallig and diferent sensitvity to anti-tumour drugs,however,have limited the degree to which insights derived from mouse models can be directly translated to human applications14-16. The advances in human stem cells and targeted gene editing technology have opened a new avenue fordisease modellingand drugdiscoveryl7.Although many genetic disease models that are linked to development and ageing have been developed using human embryonic stem cells (ESCs) or induced pluripotent stem cells $( \mathrm { i } \mathrm { P S C s } ) ^ { 1 7 - 2 4 }$ ,very few human cancer models employing targeted genetic mutations in adult stem cells have been established for gaining mechanistic insights or testing drug efficacies25,26.

Considering the potential of NSCs being the cell-of-origin for human GBM,and PTEN deletion has frequently been reported in GBM,we hypothesize that PTEN functions as a gatekeeper to protect human NSCs from neoplastic transformation.Accordingly，we generated PTEN-deficient human NSCs by targeted gene editing. PTEN deficiency resulted in a reprogramming of NSCs towards a GSC-like phenotype in ahighlylineage-specificmechanismprimarilythrough transcriptional activation of PAX7,a pathway that is also linked to the aggressive characteristics of GSCs in GBM patients.

# Results

PTEN deficiency leads to neoplastic features in NSCs. To generate isogenic PTEN-null human NSC lines,we first utilized transcription activator-like effector nuclease (TALEN)-mediated homologous recombination(HR) to delete the exon 1 of PTEN gene in human ESCs (Fig.la). Successful gene targeting at PTEN locus was verified by genomic PCR (Fig. ib). Immunofluorescence staining revealed a punctate staining pattern of PTEN in the nucleus of wild-type (WT) ESCs,which was absent in PTEN homozygous knockout $( P T E N ^ { - / - } )$ （204号 ESCs (Fig.1c).Western blot further confirmed the loss of PTEN protein in $P T E N ^ { - / - }$ ESCs (Fig. 1d). The defciency of PTEN,however,did not alter ESC-associated features such as morphology,DNA hypo-methylation at OCT4 promoter and the expression of pluripotency markers OCT4,SOX2,NANOG and

TRA-1-81 (Supplementary Fig. la-c). $P T E N ^ { - / - }$ ESCs also maintained the capability of differentiating into endoderm, mesoderm and ectoderm lineages in vivo,as did WT ESCs (Supplementary Fig.1d).

To study the role of PTEN in NSCs,we derived NSCs from both WT and $\dot { P } T E N ^ { - / - }$ ESCs.We confrmed the loss of PTEN in $P T E N ^ { - / - }$ NSCs (Fig. le,f and Supplementary Fig.2a). Both the WT and $P T E N ^ { - / - }$ NSCs exhibited typical features of neural progenitors,including the expression of neural progenitor markers PAX6,SOX2 and Nestin，the absence of pluripotency-related antigen TRA-1-81,the hyper-methylation of OCT4 promoter,as wellas the enrichmentofH3K4me3levelsatPAX6andNestin loci (Fig.lgand Supplementary Fig.2b-f).More importantly both WT and $P \bar { I } E N ^ { - / - }$ NSCs were able to efficiently differentiate into neurons and astrocytes in vitro (Fig. 1g and Supplementary Fig. $2 \mathrm { g } )$ ,confirming their NSC identity.

Next,we investigated whether PTEN-deficient NSCs could have acquired neoplastic potentials. $P T E N ^ { - / - }$ NSCs exhibited accelerated growth under suspended neurosphere culturing condition (Supplementary Fig. 3a)，accompanied by a mild increase in the population of cells retained in the S and G2/M phases and a concomitant decrease in the population of cells in the $\mathrm { { G 0 } / \mathrm { { G 1 } } }$ phases (Supplementary Fig.3b).Consistent with an acquired growth advantage, $P T E N ^ { - / - }$ NSCs exhibited enlarged nuclei size,increased numberof nucleoli and elevated rRNA transcripts (Supplementary Fig.3c-e).It has been reported that PTEN-loss is associated with _genomic instability in mouse embryonic fibroblasts (MEFs)27.Weperformed karyotyping and genome-wide copy number variation (CNV） analyses by deepsequencinganddidnotobserveanydiscernible chromosomal aberrations in $P T E N ^ { - / - }$ NSCs compared with the WT NSCs (Supplementary Fig. 3f, g).In agreement with a neoplastic potential, PTEN-/- NSCsshowed resistance to stresses-induced cell death (Supplementary Fig. 3h,i)28,29，as well as significant elevation in cell migration and clonal expansion abilities (Fig. 2a,b and Supplementary Fig. 4a,b).Re-introduction of PTEN into the $P T \dot { E } \dot { N } ^ { - / - }$ NSCs partially rescued these neoplastic phenotypes (Fig. $^ { 2 \mathrm { c } , \mathrm { d } }$ and Supplementary Fig. 4c,d). In addition,knockdown of PTEN using small hairpin(sh) RNA in either ESC- or iPSC-derived NSCs recapitulated the aggressive phenotypes observed in $P T E N ^ { - / - }$ NSCs(Supplementary Figs 4e,f and 5a-e).Altogether，these in vitro experiments demonstrated that PTEN deficiency endowed NSCswith neoplastic potential.

Nextwe explored whether PTEN deficiency is linked to tumour initiation in NSCs using an in vivo mouse model.We implanted WT or PTEN-deficient NSCs expressing luciferase into the brains of immunocompromised NOD/SCID mice.PTEN-deficient NSCs were able to grow efficiently and form intracranial tumours in vivo as evidenced by both positive luminescence and magnetic resonance imaging (MRI） signals (Fig. 2e and Supplementary Fig. 6a-c). Similar to primary human $\mathrm { \Delta N S C s } ^ { 3 0 }$ ,WT ESC-derived NSCs did not expand in the brain (Fig. 2e).Haematoxylin and eosin(H&E） staining of brain slices showed a substantial expansion of the grafted $P T E N ^ { - / - }$ NSCs,with characteristics of neoplastic cells intruding adjacent normal brain tissue and histological resemblance of intracranial tumour derived from the grafted human GSCs (Fig. 2f and Supplementary Fig. 6e)30-33. Occasionally we observed invasion of neoplastic cells into other brain regions,intra-tumour vasculatures and non-classic necrosis in the mouse brain implanted with $P T E N ^ { - / - }$ NSCs for a longer course ( $\mathord { \sim } 7 0$ days; Supplementary Fig. 6e). $P T E N ^ { - / - }$ NSCderived neoplasm also expressed typical mixed markers of GBM including Nestin，SRY box 2 (SOX2)，glial fibrillary acidic protein (GFAP),neuron-specific beta -III tubulin (Tuj1) and microtubule-associated protein 2 (MAP2）(Fig. $2 \mathrm { g } ) ^ { 3 3 , 3 4 }$

![](images/64e31693c159dbb1e4ad6a481aaea409ce78b738ed01745e70f06f45274eb313.jpg)  
Figure1eeatiaracteatifdictN.ahaeprestatfNsedrgeigtrated forbareshroosisaeeiveas and $P T E N ^ { - / - }$ ESCs using primer pairs indicated (P1+P2:1,744 bp; $\mathsf { P 3 } + \mathsf { P 4 }$ ：1,872bp; P5+P6:1,O35 bp).(c) Immunofluorescence analysis performed on WT and $P T E N ^ { - / - }$ ESCs with an anti-PTEN antibody.PTEN was absent in the $P T E N ^ { - / - }$ ESCs. Nuclei were stained with Hoechst 33342.Scale bars, $1 2 . 5 \mu \mathrm { m }$ .(d) Immunoblotting verified the absence of PTEN protein in $P T E N ^ { - / - }$ ESCs with anti-PTEN and anti-phospho-PTEN(Ser38O) antibodies. $\beta$ -Tubulin was used as loading control.(e) Immunostaining of PTEN in WT and $P T E N ^ { - / - }$ NSCs with an anti-PTEN antibody. Scale bars, $1 0 \mu \mathrm { m }$ $( \pmb { \uparrow } )$ Immunoblotting analysis of PTEN and phospho-PTEN in WT and $P T E N ^ { - / - }$ NSCs. $\mathbf { \sigma } ( \mathbf { g } )$ Immunostaining of neural progenitor- (left) and neuron- (right) specific markers in WT and $P T E N ^ { - / - }$ NSCs (left)and their neuronal derivatives (right).Scale bars, $2 5 \mu \mathrm { m }$ (NSC) and $5 0 \mu \mathrm { m }$ (neuron).

Inagreement with several studies using $\mathrm { G S C s } ^ { 3 5 , 3 6 }$ ，the neoplastic phenotype in $P T E N ^ { - / - }$ NSCs was mediated by their CD133-positive,rather than CD133-negative population (Supplementary Fig. 6f). The in vivo aggressiveness of $\stackrel { \bullet } { P } \dot { T } \dot { E } N ^ { - / - }$ NSCs was able to be suppressed by expressing exogenous PTEN(SupplementaryFig.6d),suggestingadirect involvement of PTEN in suppressing the neoplastic phenotype in NSCs.

A full-fledged cancer phenotype normally needs a combination of multiple oncogenic mutations.To test whether the neoplastic phenotype in PTEN-deficient NSCs can be aggravated by inactivation of another tumour suppressor,we knocked down one of the most well-known tumour suppressor p53in PTEN-deficient $\mathrm { N S C s } ^ { 3 7 }$ . As expected,we observed dramatic neoplastic transformation in NSCs when p53 and PTEN were simultaneously inactivated (Supplementary Fig. 7a-e).Next,we examined whether the intracranial niche of the brain was required for tumorigenesis mediated by PTEN deletion in

NSCs. When $P T E N ^ { - / - }$ NSCs were implanted into NOD/SCID mice via the subcutaneous route,we did not detect any neoplasmlike growth， suggesting that $P T E N ^ { - / - }$ NSCsrequired a supportive brain environment for tumour initiation.

We also investigated whether PTEN deletion had impact on neoplastic transformation in other human adult stem cells. To this end,we differentiated $P T E N ^ { - / - }$ ESCs to mesenchymal stem cells $( \mathrm { M S C s } ) ^ { 2 2 , 2 3 }$ .The $P T E N ^ { - / - }$ MSCsexpressed specific markers，such as CD73，CD90，CD105，and were absent of MSC-irrelevant markers,including CD34,CD43 and CD45 (Supplementary Fig. 8a,b). The $P T E N ^ { - \prime - }$ MSCs exhibited typical featuresofpremature senescence, includingthe positive SA- $\beta$ -Gal staining，expression of senescence-related gene transcripts and the compromised migration and clonal expansion (Supplementary Fig. 8c-g).Likewise,knockdown of PTEN using shRNA led to senescence in WT human MSCs (Supplementary Fig. $\mathrm { 8 h , i }$ ).Similar accelerated ageing phenotypes were also observed in the fbroblast-like cells derivedfrominvivodifferentiated $P T E N ^ { - / - }$ ESCs (Supplementary Fig. 8j).The pro-senescence effect observed in MSCs appears to be specific to PTEN deficiency,as knockdown of p53 in MSCs led to strengthened proliferation and migration instead (Supplementary Figs7e and $\mathbf { 8 k } \mathbf { , l }$ ).Herein，these observations'indicate that PTENdeficiencypreferentially induces neoplastic transformation in NSCs by promoting proliferation and invasiveness; however， in thecaseof mesodermal cell types such as MSCs and fibroblasts,PTEN deletion primarily triggers cellular senescence.

![](images/e3ff918be1a3e8e27cdf0445a2df30d2031ab37c12dbeb0c616434208acb23e7.jpg)  
Figure2|PTEN-deficient NSCs demonstratedneoplastic features in vitro and in vivo.(a)Migrationabilitiesof WTand $P T E N ^ { - / - }$ NSCs were evaluated by Transwell assays.Relative cell migration efficiency was determined.Data are shown as mean $\pm \mathsf { s . e . m }$ $\scriptstyle n = 3$ $^ { \star \star \star } P < 0 . 0 0 1$ 0 $\mathit { \Omega } _ { : } t -$ -test).Scale bars, $1 \mathsf { m m }$ (b) Clonal expansion analysis in WTand $P T E N ^ { - / - }$ NSCs.Crystal violet staining-positive cels were calculated and presented as fold induction using Image J. Data are shown as mean± s.e.m. $n = 4$ $^ { \star \star \star } P < 0 . 0 0 1$ (t-test). (c) Cell migration analyses of NSCs transduced with a PTEN expression vector (Lenti-PTEN) or a control vector (Lenti-Luc).Data are shown as mean± s.e.m. $n = 6$ $^ { \star \star \star } P < 0 . 0 0 1$ (t-test);NS,not significant. Scale bars,1mm.(d) Clonal expansion analysis of NSCs transduced with Lenti-PTEN or Lenti-Luc. Data are shown as mean $\pm$ s.e.m. $n = 3$ $^ { \star \star \star } P _ { < } 0 . 0 0 1$ (t-test); NS, not significant. (e) Representative photon flux images from the brain of NOD/SCID mice implanted with WT(left)and $P T E N ^ { - / - }$ (right) NSCs expressing luciferase (also see Supplementary Fig.6a for details).Images were taken $2 0 \mathrm { m i n }$ after L-luciferin was injected intraperitoneally (i.p.). $n = 3$ .(f) H&E staining and photomicrograph of a section of entire brain showing dramatic neoplastic expansion of $P T E N ^ { - / - }$ NSCs relative to WTcontrol $( 2 \times 1 0 ^ { 6 }$ cells per injection), 35daysaftercelredralaar.epretaunaftixase and P represents mouse brain parenchyma. Scale bars, $5 0 0 \mu \mathrm { m }$ $\mathbf { \sigma } ( \pmb { \mathsf { g } } )$ Immunostaining of brain sections described in f with antibodies against the indicated antigens.Human nuclei, human nucleus antigen.DNA was stained with Hoechst 33342. Scale bars, $2 5 \mu \mathrm { m }$

![](images/655862175303b6bd07f3579c655e5d6bd4ed92cfe1f6c8d8eaa4c4a5361170a7.jpg)  
Figure3ranscitdgeealsistcell.lifualysasuedacellcio (ECARs) in $\mathsf { P T E N } ^ { + / + }$ and PTEN $- / -$ NSCs. $n = 5$ .(b)GO biological process analysis of genes upregulated (q-value $< 0 . 0 5$ ,FC[PTEN-/-/PTEN+/ $^ + ] > 2 \rangle$ after PTEN knockout in NSCs and genes downregulated ( $\mathbf { \dot { \varphi } } _ { \cdot } q$ -value $< 0 . 0 5$ ,FC[PTEN $- / - / \mathsf { P T E N } ^ { + / + } ] < 0 . 5 )$ after PTEN knockout in MSCs.(c) Heat map displaying the upregulated genes $\mathbf { \dot { \xi } } _ { q }$ -value $< 0 . 0 5$ FC[PTEN $- / - / \mathsf { P T E N } ^ { + / + } ] > 2 )$ in PTEN-deficient NSCs (left）and the downregulated genes (q-value $< 0 . 0 5$ $\mathsf { F C } [ \mathsf { P T E N } ^ { - / - } / \mathsf { P T E N } ^ { + / + } ] < 0 . 5 )$ in PTEN-deficient MSCs (right).The representative enriched GO terms of biological processes were shown at the left side of each panel.(d) Boxplots showing H3K4me3,H3K27me3, $5 \mathsf { m C }$ and 5hmC levels in the region of 1kb upstream and 1kb downstream of TSS of differentially expressed genes between $P T E N - 1 -$ and WT NSCs including the upregulated genes (left panels, $\boldsymbol { q }$ -value $< 0 . 0 5 ,$ FC[PTEN $- 1 - / \mathsf { P T E N } ^ { + / + } ] > 2$ 号 $n = 3 2 6$ ）and downregulated genes (right panels, $\boldsymbol { q }$ -value $< 0 . 0 5$ FC[PTEN-/-/PTEN+/+]<0.5； $n = 2 8 1 ,$ $^ { \star } P$ value $< 0 . 0 5$ (Wilcoxon signed-rank test); $^ { \star \star } P$ value $< 0 . 0 1$ (Wilcoxon signed-rank test); NS,not significant.

$P T E N ^ { - / - }$ NSCs display oncogenic gene expression changes. As PTEN is a suppressor of PI3K-AKT pathway primarily via its activity asa lipid phosphatase38,39， ，we next examined the downstream signalling pathways that are regulated by PTEN. $P T E N ^ { - / - }$ NSCs exhibited an activated AKT pathway as evidenced by the phosphorylation of AKT and its downstream targets such as mammalian target of rapamycin (mTOR) and glycogen synthase kinase 3 beta $( \mathrm { G S K } 3 \beta )$ ）when compared with the WT NSCs (Supplementary Fig. 9a,b).In contrast,AKT pathway activities were not affected by PTEN status in MSCs (Supplementary Fig. 9a)， suggesting a different regulatory mechanism for AKT activation in MSCs.

Given that the AKT and mTOR activations are important for metabolic regulation of cancer cells,we further investigated the metabolic changes conferred by PTENdeficiency in NSCs.Mass spectrometry-based screening identified 16 metabolites were significantly upregulated in $P T E N ^ { - / - }$ NSCs compared with WT NSCs (Supplementary Fig. 9c,d and Supplementary Data 1). Among the 16 metabolites some belong to glycolytic pathway including glucose, glucose-6-phosphate and fructose-6-phosphate.Seahorse analysis indicated an increased glycolysis activity in $P T E N ^ { - / - }$ NSCs (Fig. 3a and Supplementary Fig. 9e). Consistently, $P T E N ^ { - / - }$ NSCs produced threefold more lactate acid content than that produced by WT NSCs (Supplementary Fig.9f),and the ratio between ADP or AMP and ATP were increased in $P T E N ^ { - / - }$ NSCs relative to that of WT NSCs (Supplementary Fig. 9c and Supplementary Data 1). These data strongly support that $P T E N ^ { - / - }$ NSCs have dysregulated glucose metabolismand favouraerobic glycolysis,a feature characteristic ofGBMandmostcancercls441ThePTEN-deficiency induced metabolic change was specific to NSCs,because these metabolites did not increase in $P T E N ^ { - / - }$ ESCs when compared with WT ESCs (Supplementary Figs 9d.f and Supplementary Data 1).

Next,we investigated the impact of PTEN loss on global transcriptomic landscape of NSCs. We performed RNA-seq analyses for both WT and $P T E N ^ { - / - }$ NSCs as well as MSCs. Using a threshold of twofold change with $q$ -values less than 0.05, we identified 6O7 genes differentially expressed between WT and $P T E N ^ { - / - }$ NSCs,amongwhich326wereupregulated (Supplementary Fig.10a-e and Supplementary Data 2).Gene ontology(GO) terms enriched in upregulated genes are linked to GSC-related functions (Fig. 3b and Supplementary Data 2).

Notably，most of these genes belong to signalling pathways important for cell motion,migration and proliferation (Fig. $^ { 3 \mathrm { b } , \mathrm { c } , }$ Supplementary Fig. $^ { 1 0 \mathrm { f , g } }$ and Supplementary Data 2). Consistent with the metabolic changes,a number of enzymes involved in glycolysis were upregulated in $P T E N ^ { - / - }$ NSCs (Fig. 3c and Supplementary Fig. $\bar { 1 0 1 , \mathrm { g } } ,$ ). These gene expression changes were partially rescued by re-introduction of PTEN in $P T E N ^ { - }$ NSCs (Supplementary Fig.10h).In contrast,PTEN deletion in MSCs resulted in differential expression of 1,854 genes (Supplementary Fig. 10a-e)．Consistent with the non-neoplastic phenotype in° $P T E N ^ { - / - }$ MSCs，PTEN deficiency led to a coordinated downregulation of genes required for cell cycle process (Fig. $^ { 3 \mathrm { b } , \mathrm { c } }$ and Supplementary Data 2),which is in agreement with a molecular signature of accelerated senescent $\mathrm { M S C s } ^ { 2 3 }$ . Of note is that only few gene expression changes caused by PTEN deficiency overlap between NSCs and MSCs (Supplementary Fig.10d and Supplementary Data 2),serving a possible explanation for the dramatically different cellular phenotypes observed between PTEN-deficient NSCs and MSCs.

To explore whether global transcriptome change in $P T E N ^ { - / - }$ NSCs are linked to epigenetic alterations,we performed chromatin immunoprecipitation sequencing (ChIP-seq） to map genome-wide distribution of H3K4me3,H3K27me3,as well as reduced representationbisulfitesequencing(RRBS） and Tet-assisted RRBS(TA-RRBS) for 5-methylcytosine $( 5 \mathrm { m C } )$ and 5-hydroxymethylcytosine (5hmC） in WT or $P T E N ^ { - / - }$ NSCs (Fig.3d and Supplementary Fig.1la,b).Notably,the upregulated genes showed significant increase of H3K4me3 and decrease of H3K27me3,whereas the downregulated genes were associated with diminished H3K4me3 and elevated $\mathrm { H } 3 \mathrm { K } 2 7 \mathrm { m e } 3$ ，attheir promoter regions (Fig. 3d). We did not observe any global associations between 5mC (or 5hmC） and gene expression changes (Fig.3d).These observations suggested that changes in histone modifications including H3K4me3 and H3K27me3 could account for PTEN-loss-induced transcriptional dysregulation in $P T E N ^ { - / - }$ NSCs.

PAX7 mediates oncogenic features in $P T E N ^ { - / - }$ NSCs. Among the upregulated genes,transcription factor PAX7 was at the top of the list and expressed 155-fold higher in $P T E N ^ { - / - }$ NSCs than in WT NSCs (Supplementary Data 2).Also it is known that elevated levels of PAX7 in embryonal rhabdomyosarcoma cells are associated with increased migration and invasiveness42,43. Thus, we focused on a potential role of PAX7 in regulating neoplastic transformation of NSCs.

We confirmed the elevated PAX7 expression in $P T E N ^ { - / - }$ NSCs relative to WT NSCs，using reverse transcriptionquantitative PCR(RT-qPCR) and western blotting (Fig.4a and

Supplementary Fig.12a).In contrast, the expression of PAX7 in $P \dot { T } \dot { E } N ^ { - / - }$ MSCs did not increase when compared with WT MSCs (Supplementary Fig.12b). The upregulation of PAX7 in

$P T E N ^ { - / - }$ NSCs was directly caused by the absence of PTEN as reintroduction of PTEN in these cells repressed PAX7 expression (Fig.4b and Supplementary Fig.12c).

a b C d □NSC PTEN+/+ 150- 1.5 Lenti-PTEN 81 7 sh-PAX7 1.5_NSC PTEN-/- \*\*\* Ped eilar 1.0 \*\*\* prebieeieled (pjgsls \*\*\*\* ph-gs 500 0.01 20 d4 0 H NSC: PTEN+ TEN NSC: PTEN-/ NSC: TEN S Ct sh-C-PAY   
e f g h 1.5 8 PAX7 promoter 0.3 NSC PCR TSS iJ0A kDa PTEN+/+ PTENT 1.0 工 6 工 -3,075 -2,098 0.2- 50- 又 IP:PTEN PTEN+/+ PTEN-/- 0.1 PREE e M 50- . 2 s 501 β-Actin 0.0 0 Lenti: Luc DN1DN2 Lest:PTEPAX7EN Lane 1 2 3 4 5 6 NSC-PTEN-/-   
_uoieiog k piggg \*\* p1 I \*\*\* \*\*\* 1 0.5 G ↓ 0 ↑PTEN→ TSS 12345678 -3,075 binding_2.098 mmm Position P1 P2 P3 P4 P5 P6P7 P8 T 中 T \* E C C P<1e-4 reielee Ns NS NS 108 (PTEN3K4mPTEN) >2 0 P1 P2 P3 P4 P5 P6 P7 P8 P1 P2 P3 P4 P5 P6 P7 P8 PTEN+/+ PTEN-/- Masse noieiain Luc Luc PTEN PTEN(G129E) PTEN(Y138L) 61\*\* PPT/NNIN 0 7

To investigate whether PAX7 mediates the neoplastic effect of PTEN deficiency during the NSC transformation, $P T E N ^ { - / - }$ NSCs were transduced with lentiviral particles encoding two independentPAX7 shRNAs,which resulted in downregulation of PAX7in the $P T E N ^ { - / - }$ NSCs (Supplementary Fig. 12d). Silencing PAX7 effectively inhibited the clonal expansion and migratory capabilities of $\mathop { P T E N } ^ { - / - }$ NSCs in vitro,as well as the intracranial neoplasm formation in vivo (Fig. 4c,d and Supplementary Fig. 12d-h). Ectopic expression of PAX7 dominant-negative mutants, which do not containthe transactivation domain,also inhibited the neoplastic features of $P T E N ^ { - / - }$ NSCs (Fig. 4e)， suggesting a critical role of the transcriptional activity of PAX7 in promoting neoplastic phenotypes in NSCs.To examine if elevated level ofPAX7alone is able to cause the neoplastic changes in WT NSCs,we overexpressed PAX7 in WT NSCs,to a similar level as in $P T E N ^ { \frac { \mathbf { 1 } } { \mathbf { \rho } } \mathbf { \Sigma } / \mathbf { \Sigma } - }$ NSCs，and found that ectopic PAX7 expression recapitulated the major phenotypes caused by PTEN depletion, including an increase of in vitro cellular migration and in vivo neoplastic expansion (Fig. 4f and Supplementary Fig. 13a-c). In addition, overexpression of PAX7 in WT NSCs resulted in a gene expression signature resembling that of $P T E N ^ { - / - }$ NSCs and distinct from WT NSCs (Supplementary Fig. $^ { 1 3 \mathrm { d } , \mathrm { e } }$ and Supplementary Data 3).We found that majority of genes (19/23) whose transcripts were co-upregulated by PAX7 overexpression and PTEN-knockout are associated with tumour-promoting or increased cellularmobility(SupplementaryFig. 13f and Supplementary Data 3). Therefore,PAX7 appears to be one of the key factors mediating neoplastic potential of PTEN-deficient NSCs.

Given that PTEN was predominantly localized to the nucleus in NSCs(Fig. le)，and emerging evidence suggests that PTEN associates with chromatin DNA and is involved gene transcription regulation27,44，we thus reason that PTEN may repress PAX7 transcription through direct binding to its promoter.A ChIP-based promoter mapping revealed a 981-bp DNA region upstream of the transcription starting site (TSS) of PAX7 as being a docking site for PTEN in WT NSCs,but not in WT MSCs (Fig. 4g and Supplementary Fig.14a). When cloned in a luciferase reporter，this region showed higher cis-activation ability when PTEN was depleted in NSCs (Fig.4h).Bioinformatics analysis predicted several binding sites in this region for transcription factor CREB and its co-activator CBP/p300(http://www.cbrc.jp/research/db/TFSEARCH.html). Given that CREB isa direct phosphatase substrateof nuclear PTEN45, we hypothesized that interplay between PTEN and CREB account for the PTEN-mediated transcriptional repression of PAX7 as well as other genes in NSCs. Supporting thishypothesis, $P T E N ^ { - / - }$ NSCsdemonstrated elevated levels in the phosphorylated form of CREB( $\mathrm { { \dot { p } } }$ -CREB;Fig.4i). Bioinformatics predicted that_167 $( 5 1 . 2 \% )$ of 326 genes that were upregulated in $P T E N ^ { - / - }$ NSCs are potentially CREB targets $\stackrel { \cdot } { (} P < 1 0 \mathrm { E } ^ { - 4 }$ ； Fig. 4j).Among these genes,the promoters of108 genes showed clear enrichments of the active histone mark H3K4me3, suggesting a permissive/active gene transcription state(Fig. 4j, Supplementary Fig.l4b and Supplementary Data 4).As for PAX7 gene,ChIP-quantitative-PCR showed that in WT NSCs PTEN and CREB/CBP co-occupied within the 981-bp DNA region at the PAX7 promoter (Fig. 4k). Co-immunoprecipitation(IP） and ChIPanalysesfurther indicated that PTEN formed a protein complex with CREB and CBP (Supplementary Fig.14c,d)45,46.Moreover, depletion of PTEN in WT NSCs resulted in an elevated occupation of p-CREB and/or CBP as well as H3K4me3 at the PAX7 promoter (Fig. 4k and Supplementary Fig. 14b).Using a GAL4-based reporter assay，we found that the co-activator activity of CBP was elevated in PTEN-deficient NSCs,which could be rescued by reintroduction of PTEN (Supplementary Fig.14e).These results support a model that PTEN deficiency led to a de-repression of CREB/CBP machinery at the PAX7 promoter，resulting in activationofPAX7 transcription (Supplementary Fig. 14f).

Next,we investigated whether over-activation of CREB/CBP upon PTEN deficiency could underlie the neoplastic transformation of NSC.For this purpose,we first repressed CBP/p300 activity in $P T E N ^ { - / - }$ NSCs by shRNA. Silencing CBP/p300 effectively decreased the migration of PTEN-deficient NSCs (Supplementary Fig. $^ { 1 4 \mathrm { g , h ) } }$ .In addition，we compared the abilities of the WT,protein phosphatase-dead mutant (Y138L) and lipid phosphatase-dead PTEN mutant(G129E) of PTEN in rescuing the malignant phenotypesof $P T E N ^ { - / - }$ NSCs. Although complementation with either the WT or the G129E mutant PTEN effectively rescued the neoplastic phenotypes in $P T E N ^ { - / - }$ NSCs,mutation of PTEN at Y138L resulted in loss of rescue ability (Fig.4l),which is consistent with its inability to dephosphorylate CREB (Supplementary Fig.14i).These observations indicated that the neoplasm-suppressing activity of PTEN in NSCs is at least in part dependent on its ability of transrepressing CREB/CBP.

Together, these data support a model that PTEN functions as a trans-repressorat the PAX7 genepromoter in NSCs; PTEN deficiency relieves the repression and activates CREB/ CBP-mediated transcription of PAX7 during the neoplastic transformation of NSCs.

The PTEN-PAX7 axis regulates malignant features of GBM. Our results support the view that human NSCs are the potential cell-of-origin for GBM，especially in the context of PTEN deficiency.It is perceivable that GSCs,the tumorigenic population within GBM,inherit genetic trait from mutated NSCs that potentially mediates the malignancy of GBM.We established three lines of GSCs from primary patient GBM specimens based onthe established methods30,47.These GSCsexhibited features characteristic of NSCs and could be cultured under neurosphere or monolayer conditions (Fig. 5a,b, Supplementary Fig. 15a,b and Supplementary table 1)30.Western blotting and immunofluorescence showed that these GSCs expressed normal levelsof PTEN protein (referred to asparental GSCs; Supplementary Fig.15c).To create isogenic GSCs that_are deficient for endogenous PTEN (referred to as GSCs-PTENlow), the parental GSCs were transduced with a lentiviral vector encoding PTEN shRNA.Silencing PTEN in the parental GSCs resulted in a marked increase of migration capability (Fig.,5c and Supplementary Fig.15d). Correspondingly, GSCs-PTENlow exhibited increased levels ofphosphorylated AKT， phosphorylated CREB and PAX7 (Supplementary Fig. 15e,f). Knockdown of $\mathrm { C B P / p } 3 0 0$ in GSCs-PTENlow not only impaired PAX7 expression but also compromised cellular migration potential (Supplementary Fig.15g,h),whereas overexpression of PAX7 in the parental GSCs resulted in a more aggressive migratoryphenotype(theresultingcellsreferredto_as GSCs-PAX7high)，similar to that observed in GSCs-PTENlow (Fig.5d and Supplementary Fig.15d,i).

Toestablisha connection between PTENand PAX7 in human GBMs,we examined the Cancer Genome Atlas (TCGA) data set of 532 GBMs48，and found that GBMs with decreased PTEN expression demonstrated increased PAX7 transcript(Spearman correlation coefficient $= - 0 . 2 8$ ， $P$ value $= 1 . 0 \times 1 0 ^ { ^ { - } - 1 1 }$ ;Fig.5e), therefore providing a strong evidence that PTEN expression negativelycorrelateswithPAX7transcriptsinclinical GBM settings.

![](images/494ea0291d9b94da82e44532a2b3770d67e4797de7f85bc177eb1eb8d9b4de5a.jpg)  
Figure5|PTEN-PAX7axisfunctionsinGSCsandGBMs.(ab)lmmunofluorescenceanalysisoftheindicatedantigens inGSCs(a）andtheir spontaneously differentiated derivatives (b). Scale bars, $1 \mathsf { m m }$ (phase)and $2 5 \mu \mathrm { m }$ (IF).(c,d) Cell migration assay indicating increased cellular mobility in PTEN-knocked down $\mathbf { \eta } ( \bullet )$ or PAX7-overexpressed (d) GSC lines. $\scriptstyle n = 3$ $^ { \star \star \star } P < 0 . 0 0 1$ (t-test).(e) GBM microarray data were obtained from TCGA website andspearmancorelatioscalculatedbyalltesampleswichwerepottedbyredandbluedots.Tegeeexpressonlevelwaslglowes normalized.PAX7 expression negatively correlated with PTEN (Spearman correlation coefficient $= - 0 . 2 8$ Pvalue $= 1 . 0 \times 1 0 ^ { - 1 1 } )$ .(f,g) FACS (f) and TdT-mediated dUTP nick end labeing $\mathbf { \sigma } ( \mathbf { g } )$ analyses of cell apoptosis in MMC( $7 . 5 \mu \mathsf { M } ,$ 24 h)or vehicle-treated WTand $P T E N ^ { - / - } \mathsf { N S C s }$ $n = 6$ $^ { \star } P < 0 . 0 5$ (ttest); $^ { \star \star \star } P < 0 . 0 0 1$ $\mathit { \Omega } _ { t }$ -test);NS,not significant.(h) Representative bioluminescent images of $P T E N ^ { - / - }$ NSCs implanted animals in the presence or absence of MMCtreatment are shownat day7O after injection (left).Right panel indicates average bioluminescent signals. $n = 5$ for dimethylsulphoxide (DMSO) treatment,and $\scriptstyle n = 8$ for MMC treatment. $^ { \star \star \star } P < 0 . 0 0 1$ (t-test).

Mitomycin C (MMC) triggers apoptosis in PTEN-deficient NSCs. Given that the PTEN deficiency is associated with increased metastasis,recurrence and drug resistance in GBM patients, discovery of compounds selectivelykilling PTEN-deficient GSCs may represent an important step forward towards precision therapy on GBM patients with known PTEN mutation(s). We therefore sought to utilize the isogenic NSC platform to screen compounds specifically kill PTEN-deficient NSCs while not the WT NSCs. By testing compounds with reported tumour-killing activity(Supplementary Table2),we identifiedMMC,aclinically usedanti-tumouragentwith DNA-crosslinkingactivity49, asa potential candidate compound.MMC selectively killed

$P T E N ^ { - / - }$ NSCs while showed minimal effect on WT NSCs,as evidencedbypropidiumiodide stainingand TdT-mediateddUTP nick end labelling assays (Fig. $^ { 4 \mathrm { f } , \mathrm { g } }$ and Supplementary Fig.16a). Temozolomide,a clinically used anti-GBM drug,however,had a comparable effect on WT and $P T E N ^ { - / - }$ NSCs (Supplementary Fig.l6b). MMC was also found cytotoxic to NSCs overexpressing PAX7 (Supplementary Fig. 16c).

To evaluate the efficacy of MMC in vivo,we performed intracranial administration of MMC to mice harbouring neoplasm derived from $P T E N ^ { - / - }$ NSCengrafts. MMCtreatment effectively repressed neoplastic growth from the $P T E N ^ { - / - }$ NSC engrafts (Fig. 5h).

# Discussion

GBM is one of the most deadly forms of human cancers and has a median survival time of only $\sim 1 4 . 6$ months.Up to date,most studies of GBM pathogenesis rely on GBM cell lines,primary GSCs or GBM cells from fresh specimens,or mouse GBM models. These conventional source materials have undoubtedly aided in our understanding of GBM pathogenesis and facilitated development of novel therapies,however,have their drawbacks. For instance,GBM or GSC lines bear significant variations from line to line,such as marker expressions,differences in cellular behaviours in vitro or in vivo,and sensitivities to drugs.These differences can be attributed to different culture conditions, pathological stages or,more likely,from their distinct genetic backgrounds. The complex genetic mutations, including the primary,secondary or evolved ones in tumour suppressors and oncogenes,as well as their interactions constitute a roadblock for studying the oncogenic identity and cell-of-origin for GSCs，as well as for aetiological understanding of GBM.Although it has been well established that GSCs are directly linked to NSCs,it has been challenging to study how genetic mutations may contribute to malignant transformation of NSCs as well as initiation and progression of GBM. The primary GBM subtype is acute and of high-grade and frequently harbours loss-of-function mutations in PTEN gene. Mouse models have been employed to study the role of PTEN deficiency in $\mathrm { N S C s } ^ { 1 3 , 5 0 , 5 1 }$ Giventhatspecies-difference exists between mouse and human,a human cellular platform to study GBM origin or pathogenesis associated with PTEN deficiency or other oncogenic events is thus urgently needed. It was not until very recently that two elegant human intestine tumour models generated using CRISPR/Cas9-based gene editing werereported25,26Nevertheless,none of above-mentioned models investigated tumorigenesis in a genuine tumourinitiating niche. Our study for the first time uses genetically engineered human NSCs to study a specific genetic mutation on neoplastic transformation both in vitro and in intracranial tumour-initiating niches,and provided the proof-of-concept evidence that NSC is a most susceptible human adult stem cell type for PTEN-deficiency-induced oncogenic transformation. Nevertheless,it should be emphasized that GBM are frequently associated with more than one genetic mutations. Indeed, compared with the GSCs derived from clinically obtained GBM samples that potentially harbour multiple genetic mutations52 PTEN-deficient human NSCs show milder phenotypes in intracranial implantation assay，and do not show aggressive growth in non-neural microenvironment. We have shown that p53 insufficiency and PTEN deficiency resulted in a cooperative effect on NSC transformation,which is consistent with a recent mouse study indicating that simultaneous inactivation of Pten and $\mathsf { p } 5 3$ in mouse neural stem/progenitor cells led to malignant transformation and tumour formation13. From this point of view, the isogenic NSC platform also provides a useful model to study other individual neoplastic mutations,or their combinations in mediating malignant human NSC transformation.

Accumulating evidence indicated thatPTENplaysa role in the nucleus through protein-protein interactions27,44 We showed here that PTEN is a nucleus-localized protein in human NSCs. It associates with and inactivates CREB/CBP at PAX7 promoter, and mediates transcriptional repression of PAX7. To our knowledge， this is the first report to show that PTEN can trans-repress gene expression via the association with transcriptional activators on gene promoters.Notably,a recent report suggests that PTEN binds_to Rad51 promoter and is required for its expression in $\mathrm { M E F s } ^ { 2 7 }$ . In a separate study, PTEN is identified to be associated with histone Hl and accounts for maintenance of a condensed chromatin configuration44. Our findings,together with these observations,support the notion that nuclear PTEN may act as a key transcriptional regulator in a context-dependent manner.Given that PAX7 is among the earliest genes involved in neural progenitor commitment in the embryo,the elevated expressionofPAX7inPTEN-deficient GSCs may account for the low-differentiated histology of GBM,a common feature for more aggressive cancers.Together with the finding that PTEN is negatively correlated with PAX7 in clinical human GBM samples,it is likely that the overexpression of PAX7 in GSCs is one of the key factors mediating GBM tumorigenesis or malignancy.

Owing to its prevalence in GBM and other cancers,PTEN mutations have been regarded as a major target for drug discovery. Cell-based systems are advantageous for automate chemical screening or drug evaluation. Most of previous studies rely on cancer cells or primary cultures and use non-isogenic cells as control,which are limited by confounding effect of different genetic backgrounds. When it comes to screening for specific drug to kill GSCs,human fetal NSCs from a different genetic background are the common cells to be used as a control.Our isogenic PTEN-knockout NSCs recapitulated certain features of GSCs,thus holding the potential to be used as a platform for screening‘precision drugs’against GSCsbased on‘PTEN deficiency'.We found that MMC is a candidate compound,and preferentially targeting PTEN-deficient NSCs while sparing their PTEN-proficientcounterparts.Wefurthershowedthat transcranial injection of MMC could diminish neoplasia from PTEN-deficient NSCs.Nevertheless,although MMC is used in the clinic for other cancers,it does not cross the blood-brain barrier and has not been used for GBM.Thus,developing advanced MMC derivatives being capable of passing through blood-brain barrier and with reduced toxicity may represent a potential therapeutic strategy for GBM harbouring PTEN mutations, despitethattheunderlyingpharmacological mechanism warrants further investigations.To our best knowledge,this is the first study using isogenic NSCs bearing _a genetically engineered carcinogenic mutation as a potential drug discovery strategy.We propose here a simplified approach to examine‘one mutation'at a time,using human ESC(or iPSC)-derived NSCs with a well-known genetic background to screen or evaluate drugs being capable of targeting certain genetic mutations.

# Methods

Antibodies and reagents.The following commercial primary antibodies were used: anti-PTEN (9559;Western blotting (WB),1:1,0o0; immunofluorescence (IF), 1:200),anti-Phospho-PTEN (pSer380,9551,1:1,000),anti-Phospho-AKT(pSer473, 4060,WB,1:1,000;IF,1:200),anti-Phospho-AKT (pThr308,2965,1:1,000),antiPhospho-GSK- $3 \beta$ (pSer9,5558,1:1,000),anti-Phospho-mTOR (pSer2448,5536, 1:1,000),anti-phospho-CREB (pSer133,9198,1:1,000),anti-CREB (9197,1:1,000) antibodies were from Cell Signaling Technology; anti-NANOG (ab21624,1:250), anti-Ki67 (ab16667,1:1,000),anti-Nucleolin (ab22758,1:200),anti-CBP (ab2832, 1:1,000)antibodies were from Abcam; anti-OCT4 (sc-5279,1:100),anti-SOX2 (sc17320,1:100),anti-Actin (sc-130301,1:2,000),anti-GFAP (sc-9065,1:1,000),antiTRA-1-81 (sc-21706,1:20) antibodies were from Santa Cruz Biotechnology; antiNestin (MAB5326,l:500),anti-Human nuclei (MAB1281,1:1,000)antibodies was from Millipore;anti-MAP2 (m4403,1:500),anti-Tuj1 (T2200,1:500),anti-Flag (M2,1:2,000),anti-SMA(A5228,1:100),anti-AFP(A8452,1:200),anti-tubulin (T5168,1:2,000) antibodies were from Sigma; anti-CD34 (555822,1:50),anti-CD43 (560198,1:50),anti-CD45 (555482,1:50),anti-CD73 (550741,1:50),anti-CD90 (555595,1:100) antibodies were from BD Biosciences;anti-CD105(17-1057, 1:100)antibody was from eBioscience;anti-PAX6 (PRB-278P,1:500) antibody was from Covance;anti-PAX7 antibody(WB,1:50o; IF,1:50) was from Atsushi Kawakami; anti-CD133(130-080-801,1:10) was from Miltenyi Biotec;MMC was purchased from Biomol International.Temozolomide was purchased from Sigma.

Cell culture.H9 ESCs were purchased from WiCell Research Institute and cultured as previously described54. Briefly, human fibroblast-derived iPSCs,H9 ESCs and theirPTEN knockout derivatives were maintained on MMC inactivated MEFs in human ESC culture medium; whereas the feeder-free cultures were maintained on Matrigel-(BD Biosciences) coated plates in mTeSR medium(STEMCELL

Technologies). The human ESC culture medium contains: $8 0 \%$ DMEM/F-12 (DMEM; Invitrogen), $2 0 \%$ knockout serum replacement (Invitrogen), $1 \%$ GlutaMAX (Invitrogen), $1 \%$ non-essential amino acids (Invitrogen), $1 \%$ PenicillinStreptomycin (PS,Invitrogen), $5 5 \mu \mathrm { M }$ $\beta$ -mercaptoethanol and $1 0 \mathrm { { n g } \mathrm { { m l } ^ { - 1 } } }$ of human basic fibroblast growth factor (bFGF; Joint Protein Central).MSCs were grown on Gelatin-coated plates in culture medium containing $9 0 \%$ $\textsf { \textsf { Q } }$ -MEM (Invitrogen), $1 0 \%$ fetal bovine serum (AusGeneX), $1 \%$ non-essential amino acid $1 \%$ PS and $\mathrm { i n g } \mathrm { m l } ^ { - 1 }$ of bFGF.NSCs were cultured in Neural Stem cell Maintenance Medium containing $5 0 \%$ Advanced DMEM/F12, $5 0 \%$ Neurobasal, $1 \times \ \mathrm { N } 2 , 1 \times \ \mathrm { B } 2 7 , 2 \mathrm { m M }$ GlutaMAX, $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of human leukemia inhibitory factor (hLIF), $3 \mu \mathrm { M }$ CHIR99021 and $2 \mu \mathrm { M }$ SB431542. GSCs were grown in medium containing $90 \%$ Neurocult NSC Basal Medium (STEMCELL Technologies), $1 0 \%$ NeuroClut NSC Proliferation Supplements (STEMCELL Technologies), $2 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of EGF and $2 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of bFGF and $2 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ of heparin55.HEK293T cells were maintained in high-glucose DMEM containing $1 0 \%$ fetal bovine serum (FBS,Gemini).No mycoplasma contamination was observed during cell culture.

GenerationofPTEN knockout $( P T E N ^ { - \prime - \prime - } )$ human ESCs. TALENs against the   
human PTEN gene (PTEN-TALEN_Left and PTEN-TALEN_Right) were pur  
chased from Addgene (Plasmid $\# 3 6 7 6 4$ Plasmid $\# 3 6 7 6 5 \$ . The donorplasmid for   
HR was described previously22.The following PCR primers were used for cloning   
the homology arms into the donor vector (PTEN-HR-Neo-pCR2.1): Left-F: $5 ^ { \prime }$ -CGGGGTACCGACCGTCCCTGCATTTCCCTCTACACT- $3 ^ { \prime }$ Left-R: $5 ^ { \prime }$ -CGCGGATCCCGGAATGGGGAGAAGACGAATAATCCTCC GAAC- $3 ^ { \prime }$ ; Right-F: $5 ^ { \prime }$ -ACGCGTCGACCCTGGTTGCAAGTGTCAA GCCACCGAT- $3 ^ { \prime }$ ：5 Right-R: $5 ^ { \prime }$ -ATAAGAATGCGGCCGCCCCCATCCCTAATCAAAACCAATG TGTTGTACCTG- $3 ^ { \prime }$ H9 ESCs were pre-treated with ROCK inhibitor Y-27632 (Sigma) 24h before   
plasmid electroporation. On the day of electroporation,H9 ESCs were   
individualized with TrypLE (Invitrogen) and filtered through a $4 0 \mathrm { - } \mu \mathrm { m }$ cell strainer   
to remove cell clumps.For TALEN-mediated engineering of $P T E N ^ { - / - }$ ESC lines,   
${ 1 0 } ^ { 6 }$ cells were counted and resuspended in $\mathrm { 1 m l }$ of MEF-conditioned medium   
containing $1 0 \mu \mathrm { M }$ ROCK inhibitor Y-27632,then mixed with plasmids $( 8 \mu \mathrm { g }$ （20   
PTEN-TALEN_Left, ${ 8 \mu \mathrm { g } }$ PTEN-TALEN_Right, $2 \mu \ g$ Neo-PTEN-HR-pCR2.1   
donor vector) and electroporated.After electroporation,cells were plated on MMC   
inactivated G4l8-resistent MEF cells.Two days later,cells were treated with   
$5 0 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$ of G418 (Gibco) for selection.After 14-21 days of selection,   
G418-resistant clones were manually picked and transferred to 96-well plates and   
expanded for genotyping.Gene-targeting was determined by PCR-screening of   
genomic DNA from the drug-resistant clones with the following primers: P1: $5 ^ { \prime }$ -AGGCTGTTACAGTCAAATCTCTGCGAACGAT- $3 ^ { \prime }$ P2: $5 ^ { \prime }$ -CCCCAAAGGCCTACCCGCTTCCATTGCTCA $3 ^ { \prime }$ ： P3: $5 ^ { \prime }$ -CTACCTGCCCATTCGACCACCAAGCGAAACATC $3 ^ { \prime }$ P4: $5 ^ { \prime }$ -TGTCCACTGCCACAATTCGCATTACCAAACTCA- $3 ^ { \prime }$ Homozygous deletion of exon l of the PTEN gene was verified by a two-step   
PCR with the following primers: P5: $5 ^ { \prime }$ -CCCAGACATGACAGCCATCATCAAAGAGATCG- $3 ^ { \prime }$ ： P6: $5 ^ { \prime }$ -AAAACGGTGAGCAGCAATTCTGGCATCACA- $3 ^ { \prime }$ P7: $5 ^ { \prime }$ -TTTGGGCCCCACACAGTGCAAGCTAAAGCAACCCCGTCT $3 ^ { \prime }$ P8: $5 ^ { \prime }$ -CCGCTCGAGAACATCACTGGTCCTGGGCAAAATCGAGA- $3 ^ { \prime }$ The PTEN locus was further examined by Southern bloting after extracting   
genomic DNA from PCR-positive clones.The neomycin-resistance cassette was   
removedaccordingtothepeviouslydescibedmethod223rieflyinodert   
remove the neomycin-resistance cassette,PTEN knock-out hESCs $( 5 \times 1 0 ^ { 6 }$ cells)   
were electroporated with pCAG-Flpo-2A-puro vector $( 1 0 \mu \mathrm { g } )$ ,and then plated on   
MMC-inactivated DR4-MEF cells.Two days later,puromycin $( 1 \mu \mathrm { g } \mathrm { m l } ^ { - 1 }$   
Invitrogen) was added to the medium to enrich Flpo recombinase-expressed cells.   
The emerging colonies were expanded,and the removal of the neomycin-resistance   
cassette was verified by PCR using the following primers: P1: $5 ^ { \prime }$ -AGGCTGTTA   
CAGTCAAATCTCTGCGAACGAT $3 ^ { \prime }$ and P4: $5 ^ { \prime }$ TGTCCACTGCCACAAT   
TCGCATTACCAAACTCA- $3 ^ { \prime }$

Flow cytometry analysis.For TRA-1-8l analysis,each $1 0 0 \mu \mathrm { l }$ of cell suspension $( 1 0 ^ { 6 }$ cells) was used.Cells were incubated with the primary antibody mouse anti-TRA-1-81 (Santa Cruz) and then the secondary Alexa Fluor 568-labelld Goat anti-Mouse IgM (Invitrogen).For CD133 analysis,each $1 0 0 \mu \mathrm { l }$ of cell suspension( $\mathrm { 1 0 ^ { 6 } }$ cells) was used. Cells were incubated with the primary antibody CD133/1(AC133)-PE (Miltenyi Biotec).For cell cycle analysis,cells were processed with the Click-iT EdUFlow Cytometry Assay Kits (Invitrogen) according to the manufacturer's instructions. Briefly,the cells were harvested after incubation with EdU $( { 1 0 } \mu \mathrm { { M } ) }$ in medium. $1 0 0 \mu \mathrm { l }$ of cell suspension ( $1 0 ^ { 6 }$ cells) for each sample were used.For the detection of cell apoptosis,each sample $\cdot 1 0 ^ { 6 }$ cells)was subjected to Annexin V-PE staining2o.Cells were examined by fluorescence-activated cell sorting (FACS)using a flow cytometer (FACSort; Becton,Dickinson and Company),and the cell cycle populations were determined using ModFit software (Verity Software House,Inc.).

Generation and characterization of NSCs.Neural induction was based on previous reports20,22.Inbrief,ESCs cultured onMEF feeder cells were challenged with

NIM-1 medium ( $5 0 \%$ Advanced DMEM/F12 (Invitrogen), $5 0 \%$ Neurobasal (Invitrogen), $1 \times \ \mathrm { N } 2$ (Invitrogen), $1 \times \ \mathrm { ~ B 2 7 ~ }$ (Invitrogen), $2 \mathrm { m M }$ GlutaMAX (Invitrogen) and $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of hLIF (Millipore), $4 \mu \mathrm { M }$ CHIR99021 (Cellagentech), $3 \mu \mathrm { M }$ SB431542 (Cellagentech), $2 \mu \mathrm { M }$ Dorsomorphin(Sigma) and $0 . 1 \mu \mathrm { M }$ Compound E (EMD Chemicals Inc.).Two days later,medium was switched to NIM-2 $5 0 \%$ Advanced DMEM/F12, $5 0 \%$ Neurobasal, $1 \times \ \mathrm { N } 2$ $1 \times \ \mathrm { ~ B 2 7 ~ }$ $2 \mathrm { m M }$ GlutaMAX, $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of hLIF, $4 \mu \mathrm { M }$ CHIR99021, $3 \mu \mathrm { M }$ SB431542 and $0 . 1 \mu \mathrm { M }$ Compound E) for additional 5 days.The cultures were then split onto Matrigelcoated plates and maintained in neural stem cell maintenance medium (NSMM) containing $5 0 \%$ Advanced DMEM/F12, $5 0 \%$ Neurobasal, $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of hLIF, $1 \times$ N2, $1 \times$ B27,2 mM GlutaMAX, $3 \mu \mathrm { M }$ CHIR99021 and $2 \mu \mathrm { M }$ SB431542.

Neuronal differentiation assay. $1 \times 1 0 ^ { 5 }$ NSCs cultured on Matrigel-coated one well of six-well plate were switched to neuronal differentiation medium containing DMEM/F12, $1 \times \ \mathrm { N } 2$ $1 \times \ \mathrm { ~ B 2 7 ~ }$ ， $2 0 0 \mu \mathrm { M }$ Ascorbic acid (Sigma), $4 0 0 \mu \mathrm { M }$ dbcAMP (Sigma), $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of GDNF (Peprotech)and $1 0 \mathrm { n g } \mathrm { m l } ^ { - \mathrm { \bar { 1 } } }$ of BDNF (Peprotech). Laminin (Sigma) was used to facilitate differentiation.Cells were cultured for 21 days,and then characterized by MAP2 and Tujl immunostaining20,22.

Bisulfite sequencing of the OCT4 promoter.Bisulfite conversion of DNA was carried out using CpGenome Fast DNA Modification Kit (Millipore) following the manufacturer's recommendation.About $1 \mu \ g$ of DNAwasused as starting material. A genomic fragment of the OCT4 promoter was amplified using LA Taq Hot Start Version (TAKARA)as previously described56.In brief,PCR products were purified by gel extraction using QIAquick columns (Qiagen),and subsequently cloned into the pEasy-T1 vector(Transgen).Thirteen clones from each sample were sequenced with the universal primer M13.

Derivation of MsCs from ESCs.MSCs were differentiated from ESC lines as previouslydescribed,23.Inbrief,groupsof1-14EBswereplatedonMatrigelcoated six-well plates in $\mathfrak { a }$ -MEM(Invitrogen） medium with $1 0 \%$ FBS (Hyclone), $1 \%$ PS, $1 0 \mathrm { n g } \mathrm { m l } ^ { - 1 }$ of bFGF and $5 \mathrm { n g } \mathrm { m l ^ { - } } \mathbf { \breve { l } }$ of $\mathrm { T G F } \beta$ (HumanZyme). Cells were left forabout 10 days until confluent MSC-like populations occurred.The resulting cells were passaged once and then analysed and sorted by FACS using various antibodies related to the MSC signatures.The cells after sorting were set to passage 0 (PO).Expressions of MSC-specific markers were verified by immunostaining with antibodies against CD73 (BD),CD90 (BD) and CD105 (eBioscience).

Isolation and culture of human GsCs.Three glioblastoma samples were obtained in the Beijing Tiantan Hospital,following approval from the institutional review board,and written informed consent was obtained from all patients.GSCs were isolated according to previously described methods with minor modifications30.,55. Briefly,tumour tissues were dissociated into single cells with Accumax Cell Dissociation Solution (Millipore) for $3 0 \mathrm { m i n }$ at $3 \breve { 7 } ^ { \circ } \mathrm { C } . 2 \times 1 0 ^ { 6 }$ single cells were initially seeded into one well of a six-well plate to allow formation of spheres/aggregates.In addition to sphere culture,GSCs were also cultured as monolayer in Matrigel-coated culture dishes.Cells were stably maintained either as spheres or as monolayer culture.

Neurosphere assay. $2 \times 1 0 ^ { 4 }$ single NSCs were seeded into one-well of an ultralow attachment 96-well plate (Corning) to allow neurosphere formation.Medium was changed every two days. The diameters of neurospheres were measured using Image J.

Luciferase reporter assays.A 981-bp genomic DNA fragment containing the PTEN docking site upstream of the TSS of PAX7 was cloned into the pGL3- Promoter vector (Promega).The DNA fragment was amplified by PCR using the following primers: $5 ^ { \prime }$ -CGGGGTACCCAAGTCTGGCCCAGCCCACACACTCA $\mathbf { G } \mathbf { - } 3 ^ { \prime }$ (forward primer); $5 ^ { \prime }$ -CCGCTCGAGGCCCTCCCCCGGGAGAGATTGG GAGATC- $3 ^ { \prime }$ (reverse primer).Transient transfection with the luciferase reporter plasmids was performed in 12-well cell culture plates according to the manufacturer's protocol. Cells were lysed $^ { 4 8 \mathrm { h } }$ post transfection,and firefly and Renilla luciferase activity were measured with a Synergy H1 Hybrid Reader(BioTek),and Renilla was used as an internal control.

Immunofluorescence microscopy. Cells were fixed with $4 \%$ formaldehyde in PBS at room temperature (RT) for $2 0 { - } 3 0 \mathrm { m i n }$ .After fixation,cellswere permeabilized with $0 . 4 \%$ Triton X-100 in PBS for $5 \mathrm { { m i n } }$ at RT.After incubation with $1 0 \%$ FBSor BSA in PBS for $3 0 \mathrm { m i n }$ ,cellswere incubated atRT for $^ { \mathrm { 1 h } }$ orat $4 ^ { \circ } \mathrm { C }$ overnight with the primary antibody. Subsequently,incubation at RT for $2 0 \mathrm { m i n }$ with the corresponding secondary antibody was carried out.Nucleiwere stained with Hoechst 33342 (Invitrogen).A Leica SP5 confocal was used for Immunofluorescence microscopy.

Teratoma analysis. Teratoma assay was performed as described57.In brief, $3 \times 1 0 ^ { 6 }$ ESCs were administrated subcutaneously into NOD/SCID mice (male,6-8 weeks).Mice were killed 6-12 weeks after injection,and teratomas were analysed by immunofluorescence staining.All animal experiments were conducted with approval of the Institute of Biophysics,Chinese Academy of Science.

SA- $\mathbf { \beta } \mathbf { \cdot } \mathbf { \beta }$ Gal staining. Staining was conducted as described previously23.In brief, cells were fixed at RT in $2 \%$ formaldehyde and $0 . 2 \%$ glutaraldehyde for $3 \mathrm { m i n }$ Fixed cells were then stained with fresh staining solution at $3 7 ^ { \circ } \mathrm { C }$ overnight. The percentage of cells positive for SA- $\cdot \beta$ -Gal signal were quantified and statistically analysed.

Lentivirus production.Expression and purification of lentiviruses were performed as previously described with minor modifications23.For generation of lentiviral vectors encoding shRNA targeting PTEN,PAX7 or $\mathrm { C B P / p } 3 0 0$ ，corresponding shRNA oligos(Supplementary Table 3) were cloned into the pLVTHM vector pre-cleaved by ClaI and MluI.For lentiviral vectors expressing PTEN,PAX7or luciferase (control),corresponding cDNAs were cloned into the pLE4 vector (a kind gift from Dr Tomoaki Hishida).For lentiviral packaging,HEK293T cells were co-transfected with lentiviral expression or shRNA vectors,as well as psPAX2 (Addgene,12260) and pMD2.G (Addgene,12259).Lentiviral particles were collected after $4 8 \mathrm { h }$ and concentrated by ultracentrifugation at $^ { 1 9 , 4 0 0 } \mathrm { g }$ for $2 . 5 \mathrm { h }$ Concentrated viruses were used for titre determination and cell transduction.

Western blotting, IP and RT-qPCR.For Western blotting, cells were lysed in RIPA buffer ( $2 5 \mathrm { m M }$ Tris-HCl pH7.6,150 mMNaCl, $1 \%$ NP-40, $1 \%$ sodium deoxycholate, $0 . 1 \%$ SDS)with protease inhibitor cocktail (Roche).A BCAkit (Thermo Fisher Scientific) was used for protein quantification.About $2 0 \mu \ g$ of protein lysate was subjected to SDS-PAGE electrophoresis and electrotransferred to a PVDF membrane (Milipore).Then primary and HRP-conjugated secondary antibodies were used to visualize the target proteins.The quantification of western blot was performed with Image Lab software for ChemiDoc XRS system (Bio-Rad).For IP, $\mathrm { \sim } 1 \times 1 0 ^ { 7 }$ cells was lysed on ice for $^ \mathrm { 1 h }$ in NP40 lysis buffer ( $5 0 \mathrm { m M }$ Tris-HCl, $\mathrm { p H } 8 . 0 , 1 \%$ NP-40,150 mMNaCl,2 mMEDTA, $1 \mathrm { m M }$ PMSF,Complete EDTA-free Protease Inhibitor Cocktail).After adjusting protein concentrations to about $4 \mathrm { m g } \mathrm { m l } ^ { - 1 }$ ,inputs were aliquoted,and lysates were incubated with anti-Flag agarose (Sigma) overnight at ${ 4 ^ { \circ } \mathrm C } .$ After three washes in NP-4O lysis buffer,bound proteins were eluted with $1 \times ~ \mathrm { S D S }$ loading buffer,and samples were analysed by western blotting with the indicated antibodies.For $\operatorname { R T - q P C R } ,$ cellular total RNA was extracted by TRIzol (Invitrogen),and genomic DNA was removed by DNA-free Kit (Ambion),followed bycDNA synthesis with GoScript Reverse Transcription System (Promega).RT-qPCR was performed using iTaq Universal SYBR Green Supermix (Bio-Rad).Primer sequences are given in Supplementary Table 3.The uncropped images of the western blots and gels are shown in Supplementary Fig. 17.

Cell viability assay.Cell viability was determined with CellTiter 96 AQueous One Solution Cell Proliferation Assay MTS (3-(4,5-dimethylthiazol-2-yl)-2.5- diphenyltetrazolium).

In vitro cell migration assay.The cell migration of NSCs,MSCs and GSCs through the Matrigel (BD Bioscience) in vitro was performed as previously described58.First, the top chamber of the Transwells (24-well insert; pore size, $8 \mathrm { m m }$ ：Corning Costar) pre-coated with Matrigel was seeded with $2 . 5 \times 1 0 ^ { \overline { { 4 } } }$ cells in $1 0 0 \mu \mathrm { l }$ basal medium L $5 0 \%$ Advanced DMEM/F12, $5 0 \%$ Neurobasal forhNSCs,αMEM forhMSCs and GSC culture medium without EGF,FGF2 and Heparin for GSCs).The corresponding complete culture medium for each cell type was put into the bottom chambers. Cells that migrated through the Transwells were stained and counted.

Single-cell clonal expansion assay. To determine the clonal expansion effciency of NSCs and MSCs,2,0o0 cells were seeded in a Matrigel-coated 12-well plate,and cultured until clear cell colonies formed.The relative colony number was then determined by crystal violet staining.

Metabolomics evaluation.Metabolite profiling was carried out at Metabolon with LC-MS and GC-MS methods as described previously59.Briefly, $1 0 ^ { 6 }$ ESCs $( P T E N ^ { + / + }$ versus $P T E N ^ { - / - }$ and NSCs $( P T E N ^ { + / + }$ versus $\dot { P } T E N ^ { - / - }$ were pelleted,and five replicates were stored at $\mathrm { ~ - ~ } 8 0 ^ { \circ } \mathrm { C }$ .Samples were prepared and analysed by using Metabolon's standard solvent extraction methods. $1 0 0 \mu \mathrm { l }$ of each sample was extracted by methanol.After centrifugation,the supernatant consisting of metabolites was used for MS and LC-MS analyses.The extracts were equally split into two parts for analysis on LC/MS/MS (Thermo Fisher linear Ion Trap mass spectrometer with a Fourier Transform) and GC/MS (Mat-95 XP mass spectrometer) platforms.Data were analysed with Welch's two-sample t-test.

Lactate measurements.Lactate secreted in the culture medium was collected and measured by a colorometric L-Lactate assay kit (Abcam) following the manufacturer's instructions.

Mouse brain in vivo imaging.Before intracranial implanting,NSCs were transduced with lentivirus expressing luciferase. $2 \mu \mathrm { l }$ of a $1 \times 1 0 ^ { 7 }$ cells per ml cell suspension in high-glucose PBS were injected to the hemi-striatum using stereotactic injection. The following coordinate parameters were used: antero-posterior $= 0$ medio-lateral $= + 2 . 5 \mathrm { m m }$ dorso-ventral $= - 3 . 5 \mathrm { m m }$ 、About 10 weeks post injection,mice were injected into peritoneal cavity with luciferin to trace tumour cells in vivo.The animals were next anaesthetized,and bioluminescence was imaged using the IVISLumina system (Perkin-Elmer).Images were taken every $2 \mathrm { m i n }$ for a course of $3 0 \mathrm { m i n }$ (10 s exposure per image).The mice brains were also imaged 1 and $^ { 2 \mathrm { h } }$ post injection (10 s exposure per image). Images were processed using Living Image software.Regions of interest were drawn based on each cell mass,and the total number of photons was recorded for each region of interest. Immunofluorescence staining and H&E staining were carried out on $5 \mu \mathrm { m }$ cryostat sections.

Magnetic resonance imaging.MRI for mouse brain was performed on a Bruker's PharmaScan 7T MRI imaging system. The system is equipped with standard actively shielded gradient system.A $7 2 \mathrm { - m m }$ birdcage resonator was used for excitation,and detection was done with a $3 0 \mathrm { - m m }$ surface coil. T2-weighted images were consecutively acquired by using a rapid-acquisition relaxation enhanced sequence.Animals were anaesthetized before MRI.

RNA isolation and sequencing library construction.A total of $5 \mathrm { - } 1 0 \times 1 0 ^ { 5 }$ cells were used to isolate RNA with RNeasy Mini Kit (Qiagen). $1 . 5 \mathrm { - } 2 \mu \mathrm { g }$ of RNA was qualified by Fragment Analyzer (Advanced Analytical) and was used to construct sequencing libraries using the TruSeq RNA Sample Preparation Kit (Illumina) following the manufacture's protocol.

ChIP and sequencing. ChIP-seq was done following a previous protocol with minor modifications. Briefly,cells were cross-linked by $1 \%$ (vol/vol) formaldehyde for $8 \mathrm { { m i n } }$ at RT and then quenched by $1 2 5 \mathrm { m M }$ glycine for $5 \mathrm { { m i n } }$ at RT.Then,the cellswere lysed inlysis buffer( $\mathrm { 5 0 m M }$ Tris-HCl, $1 0 \mathrm { m M }$ EDTA, $1 \%$ SDS,pH8.0),which was followed by chromatin shearing using a Covaris S2 instrument.The samples were then incubated overnight with Protein A-bead bound with the following antibodies separately:anti-PTEN(CST,9559),anti-CBP (Abcam,ab2832),anti-phospho-CREB (CST,9198)anti-H3K4me3 (CST,9751 s) and anti-H3K27me3(CST,9733).Normal rabbit IgG (Santa Cruz,SC-2027) or input was used as a negative control. To obtain the DNA of interest,chromatin with the target modification was eluted in elution buffer ( $2 0 \mathrm { m M }$ Tris-HCl, $5 \mathrm { m M }$ EDTA,50 mM NaCl, pH7.5),and then incubated with Proteinase K on a thermomixer at $1 { , } 3 0 0 \mathrm { r . p . m }$ .for $^ { 2 \mathrm { h } }$ at $6 8 ^ { \circ } \mathrm { C } .$ Next, the DNA was extracted and purifed via phenol-chloroform-isoamyl alcohol extraction and ethanol precipitation. Samples for sequencing including H3K4me3,H3K27me3 and IgG were constructed into libraries with NEBNext DNA Library Prep Reagent Set for Illumina (NEB) as described by the manufacturer.

ChIP-PCR mapping of PTEN-associated genomic region. We mapped the PTEN-binding site at PAX7 promoter spanning from $\phantom { - } - 5 , 0 0 0$ to $^ { + 3 , 2 5 1 }$ with ChIP-PCR using different primer sets.In brief,to obtain the DNA region of interest, lysed samples were incubated overnight with Protein A-bead bound to anti-PTEN antibody.Then, extracted DNAs were subjected to PCR amplification of the different genomic regions by using different primer sets (Supplementary Table 3).

Whole genome sequencing. Genomic DNA was extracted from $1 \times 1 0 ^ { 6 }$ cells by DNeasy Blood & Tissue Kit(Qiagen) and sheared into 150-20o bp fragments by Covaris S2 and the quality was analysed via Fragment Analyzer (Advanced Analytical).The sequencing libraries were constructed using NEBNext DNA Library Prep Reagent Set for Illumina (NEB).

RRBS and TA-RRBS.RRBS and TA-RRBS were performed as we previously reported61.For RRBS,after digested with MspI, genomic DNA was size-selected and spiked inwith $0 . 5 \%$ fullymethylated lambdaDNA.DNAwas then treatedwith bisulfite after ligation with pre-methylated adapters and amplified with PfuTurbo $\operatorname { C x }$ Hotstart DNA polymerase (Agilent Technologies).For TA-RRBS,after MspI digestion,50-6o0 bp DNA fragments were first gel-selected,and then glucosylated with $\beta$ -glucosyltransferase and oxidized by recombinant mouse Tetl (mTetl) subsequently.The final TA-RRBS libraries were generated after bisulfite treatment andamplificationwithPfuTurbo $\operatorname { C x }$ HotstartDNA polymerase as stated above.

RNA-Seq data analysis.We aligned reads from RNA-Seq using TopHat6² and Cufflinks63 and differentially expressed genes wereanalysed as previously described64.A total of 14,797,14,782,14,030 and14,322 genes with $\mathrm { F P K M } > 0 . 1$ were detected in $P T E N ^ { + / + }$ NSC, $P T E N ^ { - / - }$ NSC, $P T E N ^ { + 7 + }$ MSC and $P T E N ^ { - / - }$ MSC,respectively.281,705 genes( $\dot { \boldsymbol { q } }$ -value $< 0 . 0 5$ $P$ -value $< 0 . 0 5$ $\operatorname { F C } > 2$ ）were significantly downregulated and 326,1,139 genes $\mathbf { \chi } _ { [ { q } ] } ^ { \prime }$ -value $< 0 . 0 5$ $P$ -value $< 0 . 0 5$ $\mathrm { F C } > 2$ ）were significantly upregulated in $P \breve { T } E N ^ { - / - }$ NSCs and MSCs,respectively. GO for enrichment of genes was assessed by $\mathrm { D A V I D } ^ { 6 5 }$ ：

ChIP-seq data analysis. Sequencing reads from the ChIP-Seq experiment were mapped to the human reference genome (hgl9) by Burrows-Wheeler Aligner (BWA)66,retaining only unique non-duplicate reads without mismatches in the first 15 bp.Peaks were then called by MACS by default parameters, using IgG as control7.Histone modification level was defined as non-duplicate reads in peaks (PR) normalized by total non-duplicate reads (NR), $D = \mathrm { P R } / \mathrm { N R }$ Modification level variation $( P T E N ^ { - / - }$ versus $\dot { P } T E N ^ { + / + } > 2$ ）wasassessed by the following command: macs2 bdgdiff -t1 k4wt.bdg -t2 k4ko.bdg -cl k4wt_control_lambda.bdg -c2 k4ko_control_lambda.bdg-d1 619i314 -d2 6191314 -outdir./ $- \mathrm { ~ C ~ } 0 . 3 \mathrm { ~ - g ~ } 2 0 0 \mathrm { ~ - 1 ~ } 5 0 .$

CREB-binding sites analysis.Promoter for CREB-binding sites analysis was defined as $3 \mathrm { k b }$ upstream of TSS and 300 bp downstream of TSS.CREB-binding sites with $P _ { - }$ -value less than lE-4 among the promoter regions were found by $\mathrm { F I M O } ^ { 6 8 }$ （20 using CREB motif downloaded from MEME database (MEME version 4.4).

Genome-wide CNV analysis.Reads from whole-genome DNA sequencing were mapped to the human reference genome (hgl9 from UCSC) by BWA. Signal of $5 0 0 \mathrm { k b }$ window CNV analysis was calculated by the following formula, $C = 5 ^ { * } d ^ { * } 1 0 ^ { 9 } / ( ( 5 0 0 , 0 0 0 - t ) ^ { * } \dot { b } ) .$ in which $d$ is the base number in $5 0 0 \mathrm { k b }$ that mapped to the reference, $t$ is the number of $_ \mathrm { N }$ in the $5 0 0 \mathrm { k b }$ window and $b$ is the total bases that mapped to the reference.

RRBS and TA-RRBS data analysis.First, sequencing reads were trimmed using trim_galore_v0.3.3 by the following command,trim_galore--quality 20 --phred33 --stringency 3--gzip --length 50 --rrbs --paired --triml --output_dir \$out \$rawl \$raw2.Then trimmed reads were mapped to the hgl9 reference genome by Bismark9 and Bowtie70 using default parameters.Levels of 5hmC were determined by $\mathrm { C / C } + \mathrm { T }$ of TA-RRBS and $5 \mathrm { m C }$ were determined by $( \mathrm { C } / ( \mathrm { C } + \mathrm { T } ) - 5 \mathrm { h m } \mathrm { C } \% )$ . In our study,only genome regions with coverage $> = 5$ were taken into account.

Statistical analysis.Data were analysed with Student's t-tests and were presented asmean $\pm$ standard deviation (s.d.or s.e.m.). $P { < } 0 . 0 5$ was considered as significant.

# References

1．Furnari,F.B.et al.Malignant astrocytic glioma: genetics,biology,and paths to treatment. Genes Dev.21,2683-2710 (2007).   
2.Bianco,R.et al. Loss of PTEN/MMAC1/TEP in EGF receptor-expressing tumor cells counteracts the antitumor action of EGFR tyrosine kinase inhibitors.Oncogene 22,2812-2822 (2003).   
3．Mellinghoff,I.K.et al.Molecular determinants of the response of glioblastomas to EGFR kinase inhibitors.N. Engl.J.Med.353,2012-2024 (2005).   
4.Nagata,Y.et al.PTEN activation contributes to tumor inhibition by trastuzumab,and loss of PTEN predicts trastuzumab resistance in patients. Cancer cell 6,117-127 (2004).   
5.Dean,M.,Fojo,T.& Bates,S.Tumour stem cells and drug resistance.Nat. Rev. Cancer.5,275-284 (2005).   
6．Eramo,A.et al.Chemotherapy resistance of glioblastoma stem cells.Cell Death Differ.13,1238-1241 (2006).   
7.Liu, C.et al.Mosaic analysis with double markers reveals tumor cell of origin in glioma.Cell 146,209-221 (2011).   
8.Visvader,J.E.Cells of origin in cancer.Nature 469,314-322 (2011).   
9.Wang,Y.et al. Expression of mutant p53 proteins implicates a lineage relationship between neural stem cells and malignant astrocytic glioma in a murine model. Cancer Cell15,514-526 (2009).   
10.Modrek,A.S.,Bayin,N.S.& Placantonakis,D.G.Brain stem cells as the cell of origin in glioma.World J.Stem Cells 6,43-52 (2014).   
11.van der Zee,M.et al.Alterations in Wnt-beta-catenin and Pten signalling play distinct roles in endometrial cancer initiation and progression.J.Pathol.230, 48-58 (2013).   
12.Alcantara Llaguno,S.et al.Malignant astrocytomas originate from neural stem/progenitor cells in a somatic tumor suppressor mouse model. Cancer Cell 15,45-56 (2009).   
13.Zheng,H.et al. p53 and Pten control neural and glioma stem/progenitor cell renewal and differentiation.Nature 455,1129-1133 (2008).   
14.Chen,J.,McKay,R.M.& Parada,L.F.Malignant glioma: lessons from genomics,mouse models,and stem cells. Cell 149,36-47 (2012).   
15.Rangarajan,A.&Weinberg,R.A. Opinion:Comparative biology of mouse versus human cells: modelling human cancer in mice.Nat.Rev. Cancer 3, 952-959 (2003).   
16.Wright,W.E.& Shay,J.W.Telomere dynamics in cancer progression and prevention: fundamental differences in human and mouse telomere biology. Nature Med.6,849-851 (2000).   
17.Grskovic,M.,Javaherian,A., Strulovici, B.& Daley,G.Q.Induced pluripotent stem cells--opportunities for disease modelling and drug discovery.Nat.Rev. Drug Discov.10,915-929 (2011). 1155-1156 (2012).   
19.Liu,G.H.et al. Recapitulation of premature ageing with iPSCs from Hutchinson-Gilford progeria syndrome. Nature 472,221-225 (2011).   
20.Liu,G.H.et al.Progressive degeneration of human neural stem cells caused by pathogenic LRRK2.Nature 491,603-607 (2012).   
21.Liu,G.H.,Ding,Z.& Izpisua Belmonte,J.C.iPSC technology to study human aging and aging-related disorders. Curr. Opin. Cell Biol. 24,765-774 (2012).   
22. Liu, G. H. et al. Modelling Fanconi anemia pathogenesis and therapeutics using integration-free patient-derived iPSCs.Nat. Commun.5,4330 (2014).   
23. Zhang, W.et al. A Werner syndrome stem cell model unveils heterochromatin alterations as a driver of human aging. Science 348,1160-1163 (2015).   
24. Zuckermann, M.et al. Somatic CRISPR/Cas9-mediated tumour suppressor disruption enables versatile brain tumour modelling.Nat. Commun.6,7391 (2015).   
25.Drost, J. et al. Sequential cancer mutations in cultured human intestinal stem cells.Nature 521,43-U329 (2015).   
26.Matano,M.et al.Modeling colorectal cancer using CRISPR-Cas9-mediated engineering of human intestinal organoids.Nature Med.21,256-262 (2015).   
27.Shen,W.H.et al. Essential role for nuclear PTEN in maintaining chromosomal integrity.Cell128,157-170 (2007).   
28. Weng,L., Brown,J.& Eng, C.PTEN induces apoptosis and cell cycle arrest through phosphoinositol-3-kinase/Akt-dependent and -independent pathways. Hum. Mol. Genet.10,237-242 (2001).   
29. Zhao,H.,Dupont,J,Yakar,S.,Karas,M.& LeRoith,D.PTEN inhibits cell proliferation and induces apoptosis by downregulating cell surface IGF-IR expression in prostate cancer cells.Oncogene 23,786-794 (2004).   
30.Pollard,S.M.et al. Glioma stem cellines expanded in adherent culture have tumor-specific phenotypes and are suitable for chemical and genetic screens. Cell stem cell 4,568-580 (2009).   
31. Suva, M. L. et al. Reconstructing and reprogramming the tumor-propagating potential of glioblastoma stem-like cells. Cell157,580-594 (2014).   
32.Louis,D.N.et al. The 2007 WHO classification of tumours of the central ervous system.Acta Neuropathol.114,97-109 (2007).   
33.Zhu,Z.etal.Targetigselfrenealinigadebaintumosleadstolossof brain tumor stem cells and prolonged survival. Cell Stem Cell 15,185-198 (2014).   
34. Cheng,L.et al. Glioblastoma stem cells generate vascular pericytes to support vessel function and tumor growth. Cell153,139-152 (2013).   
35.Singh,S.K.et al.Identification of human brain tumour initiating cells.Nature 432,396-401 (2004).   
36.Bao,S.et al. Glioma stem cells promote radioresistance by preferential activation of the DNA damage response. Nature 444,756-760 (2006).   
37. Kawamura,T.et al. Linking the p53 tumour suppressor pathway to somatic cell reprogramming.Nature 460,1140-U1107 (2009).   
38.Chin,Y.R., Yuan,X.,Balk,S.P.& Toker,A.PTEN-deficient tumors depend on AKT2 for maintenance and survival. Cancer Discov.4,942-955 (2014).   
39.Chow,L. M.& Baker,S.J.PTEN function in normal and neoplastic growth. Cancer Lett.241,184-196 (2006).   
40.Saga,I.et al. Integrated analysis identifies different metabolic signatures for tumor-initiating cells in a murine glioblastoma model. Neuro Oncol. 16, 1048-1056 (2014).   
41.Vander Heiden, M. G. Cantley,L. C.& Thompson, C.B. Understanding the Warburg efect: the metabolic requirements of cell proliferation. Science 324, 1029-1033 (2009).   
42.Chiappalupi, S.,Riuzzi,F.,Fulle,S.,Donato,R.& Sorci, G. Defective RAGE activity in embryonal rhabdomyosarcoma cells results in high PAX7 levels that sustain migration and invasiveness.Carcinogenesis 35,2382-2392 (2014).   
43.Riuzi,F.etal.AGEsignalingdeficiencyinhabdomyosarcomacells causes upregulation of PAX7 and uncontrolled proliferation.J. Cell Sci.127, 1699-1711 (2014).   
44.Chen,Z.H.et al.PTEN interacts with histone Hl and controls chromatin condensation. Cell Rep.8,2003-2014 (2014).   
45.Gu,T.T.et al. CREB is a novel nuclear target of PTEN phosphatase. Cancer Res.71,2821-2825 (2011).   
46.Ikenoue,T.,Inoki,K., Zhao,B.& Guan, K.L.PTEN acetylation modulates its interaction with PDZ domain. Cancer Res. 68,6908-6912 (2008).   
47.Gunther,H.S.et al. Glioblastoma-derived stem cell-enriched cultures form distinct subgroups according to molecular and phenotypic criteria. Oncogene 27,2897-2909 (2008).   
48.Filbin, M.G.et al. Coordinate activation of Shh and PI3K signaling in PTEN-deficient glioblastoma: new therapeutic opportunities. Nature Med.19, 1518-1523 (2013).   
49. Deans, A.J. & West,S.C.DNAinterstrand crolinkrepaiandcancer. Nat. Rev. Cancer11,467-480 (2011).   
50.Amiri,A.et al.Pten deletion in adult hippocampal neural stem/progenitor cells causes cellular abnormalities and alters neurogenesis.J.Neurosci 32,5880-5890 (2012).   
51.Groszer,M.et al.PTEN negatively regulates neural stem cell self-renewal by modulating G0-Gl cell cycle entry.Proc.Natl Acad.Sci. USA 103,111-116 (2006).   
52.Chin,L.et al. Comprehensive genomic characterization defines human glioblastoma genes and core pathways.Nature 455,106l-1068 (2008).   
53.Muller,F.J., Snyder,E.Y.& Loring,J.F.Gene therapy: can neural stem cells deliver?Nat.Rev.Neurosci.7,75-84 (2006).   
54.Liu,G.H.et al. Targeted gene correction of laminopathy-associated LMNA mutations in patient-specific iPSCs.Cell Stem Cell 8,688-694 (2011).   
55.Azari, H.et al. Isolation and expansion of human glioblastoma multiforme tumor cells using the neurosphere assay.J.Vis.Exp.56,e3633 (2011).   
56.Yu,J.et al. Induced pluripotent stem cell lines derived from human somatic cells. Science 318,1917-1920 (2007).   
57.Lensch,M.W.,Schlaeger,T.M., Zon,L.I.& Daley,G.Q.Teratoma formation assays with human embryonic stem cells: a rationale for one type of humananimal chimera. Cell Stem Cell 1,253-258 (2007).   
58.Chiu, Y.W.et al. Baicalein inhibits the migration and invasive properties of human hepatoma cells. Toxicol. Appl.Pharmacol.255,316-326 (2011).   
59.Reitman,Z.J.et al.Profiling the effects of isocitrate dehydrogenase 1 and 2 mutations on the cellular metabolome.Proc.Natl Acad.Sci.USA 108, 3270-3275 (2011).   
60.Dahl,J.A.& Collas,P.A rapid micro chromatin immunoprecipitation assay (microChIP).Nat.Protoc.3,1032-1045 (2008).   
61.Wen,L.et al.Whole-genome analysis of 5-hydroxymethylcytosine and 5- methylcytosine at base resolution in the human brain. Genome Biol.15,R49 (2014).   
62.Trapnell, C.,Pachter,L.& Salzberg, S.L.TopHat: discovering splice junctions with RNA-Seq.Bioinformatics 25,1105-1111 (2009).   
63.Roberts,A.,Pimentel,H.,Trapnell, C.& Pachter,L.Identification of novel transcripts in annotated genomes using RNA-Seq.Bioinformatics 27, 2325-2329 (2011).   
64.Trapnell,C.et al. Differential gene and transcript expression analysis of RNAseq experiments with TopHat and Cuflinks.Nat.Protoc.7,562-578 (2012).   
65.Huang,da,W.,Sherman,B.T.& Lempicki,R.A.Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucleic Acids Res.37,1-13 (2009).   
66.Li,H.& Durbin,R.Fast and accurate long-read alignment with BurrowsWheeler transform.Bioinformatics 26,589-595 (2010).   
67.Zhang,Y.et al.Model-based analysis of ChIP-Seq (MACS).Genome Biol.9, R137 (2008).   
68.Grant, C.E., Bailey,T.L.& Noble,W.S.FIMO: scanning for occurrences of a given motif.Bioinformatics 27,1017-1018 (2011).   
69.Krueger,F.& Andrews,S.R.Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.Bioinformatics 27,1571-1572 (2011).   
70.Langmead,B., Trapnell,C.,Pop,M.& Salzberg,S.L. Ultrafast and memoryefficient alignment of short DNA sequences to the human genome. Genome Biol.10,R25 (2009).

# Acknowledgements

We are grateful to Jianfeng Lei from the Capital Medical University and Yihui Xu from Institute of Biophysics for technical assistance,and Yang Zhao for administrative assistance. This work was supported by the National Basic Research Program of China (973 Program,2015CB964800;2014CB910503;2014CB964600;2012CB966704; 2011CB504lo0),the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA01020312),the Beijing Natural Science Foundation (7141005;5142016), the National High Technology Research and Development Program of China (2015AA020307),the National Natural Science Foundation of China (NSFC: 81330008; 31222039;31201111; 81371342; 81300261; 81300677; 81271266; 81471414; 81422017; 81401159; 31322039),the Program of Beijing Municipal Science and Technology Commision (Z151100003915072),the Key Research Program of the Chinese Academy of Sciences (KJZDEW-TZ-Lo5),the Thousand Young Talents program of China,National Laboratory of Biomacromolecules (012kf02;2013kf05;2013kf11;2014kf02;2015kf10),the State Key Laboratory of Drug Research (SIMMl302KF-17) and the China Postdoctoral Science Foundation Grant (2013M530751).J.C.I.B.was supported by the Salk Institute Cancer Center Support Grant, Universidad Catolica San Antonio de Murcia (UCAM),the G.Harold and Leila Y.Mathers Charitable Foundation and the Leona M.and Harry B. Helmsley Charitable Trust (PG-2012-MED002).W.Z.was supported by NIH grants K99/ R00 CAl58055,the V Scholar award,American Cancer Society seed grant, Breast Cancer Research Award and Oberley Award (National Cancer Institute Award P30CA086862) from Holden Comprehensive Cancer Center at the University of Iowa,and startup funds from the Department of Pathology, University of Iowa.

# Author contributions

S.D., G.Y.and X.L. performed the majority of the experiments.F.T.,X.X.,X.L.and J.L. carried out the metabolomics,genomic and epigenomic analyses. S.D.,G.Y.,X.X.,J.Q., L.F.,C.R.E.,J.Y.and K.S. generated iPSC,ESC/iPSC characterization and gene editing. S.D., G.Y.and J.Q. performed cell culture and differentiation.R.R. S.D.,G.Y.and Y.L. performed animal experiments. S.D.,G.Y.and R.B.X.W.,T.J., C.Z.and Z.C. performed sample collection and data analyses. J.C.I.B.,J.W.,Weizhou Zhang,F.Y.,H.G.and Weiqi Zhang performed data analysis and wrote the manuscript. S.D., G.Y.,X.L., J.Q.,F.T.and G.-H.L.conceived this study and wrote the manuscript.

# Additionalinformation

Accession codes: The RNA-seq, ChIP-seq,RRBS,TA-RRBS sequencing data sets have been deposited in NCBI Gene Expression Omnibus (GEO) under accession code GSE61794.And the whole-genome DNA sequencing data for CNV analysis are deposited to the Sequence Read Archive (SRA) under accession number of SRP048488.

Supplementary Information accompanies this paper at http://www.nature.com/ naturecommunications

Competing financial interests:The authors declare no competing financial interests.

Reprints and permission information is available online at http://npg.nature.com/ reprintsandpermissions/

How to cite this article: Duan, S.et al.PTEN deficiency reprogrammes human neural stem cells towards a glioblastoma stem cell-like phenotype.Nat. Commun. 6:10068 doi: 10.1038/ncomms10068 (2015).

This work is licensed under a Creative Commons Attribution 4.0 International License.The images or other third party material in this article are included in the article's Creative Commons license,unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license,visit http://creativecommons.org/licenses/by/4.0/