# Biochemical Characterization of Recombinant Enterovirus 71 3C Protease with Fluorogenic Model Peptide Substrates and Development of a Biochemical Assay

-uqingShangabumeiZangabYangabixueSun,LnfengLiZhengjieCuabQuhongHe,uGuoYunun Zheng Yina,b

Colegtsi i fite People's Republic of Chinad

Enterovirus71(EV71),aprimaryathogenofhand,foot,andmouthdisease (HMD),ects primarilyinfantsandildren. Currently,therearenoeffectivedrugsagainstHFMD.EV713Cproteaseperforms multipletasksintheviralreplication,which makesitanidealantiviral target.We synthesizedasmallsetoffluorogenicmodelpeptidesderivedfromcleavage sitesofEV71 polyproteinand examinedtheirefcienciesofcleavagebyEV713Cprotease.The novel peptideP08[(2-(N-methylamino)benzoyl) (MA)-IEALFQGPPK(DNP)FR]wasdetermined tobethe most eficientlycleavedbyEV713Cprotease,withakineticconstant $k _ { \mathrm { c a t } } / K _ { m }$ of $1 1 . 8 \pm 0 . 8 2 \mathrm { m M } ^ { - 1 } \mathrm { m i n } ^ { - 1 }$ . Compared with literature reports, Po8 gave significant improvement in the signal/ background ratio,which makes itan attractive substratefor assaydevelopment. A Moleculardynamics simulationstudy elaboratedtheinteractionsbetweensubstratePO8andEV713Cprotease.Arg39,whichislocatedatthebottomoftheS2pocket ofEV713Cprotease,mayparticipateinthe proteolysis process ofsubstrates.WithanaimtoevaluateEV713Cproteaseinhibitors,a reliable and robust biochemical assay with a $\pmb { Z } ^ { \prime }$ factor of ${ \bf 0 . 8 7 \pm 0 . 0 5 }$ was developed. A novel compound (compound 3) 1 $5 0 \%$ inhibitory concentration $[ \bf { I C _ { 5 0 } } ] = 1 . 8 9 \pm 0 . 2 5 \mu \bf { M } )$ was discovered using this assay, which effectively suppressed the proliferation ofEV71 (strain Fuyang) in rhabdomyosarcoma (RD) cells with a highly selective index ( $5 0 \%$ effective concentration $[ \mathbf { E C _ { 5 0 } } ] = 4 . 5 4 \pm 0 . 5 1 \mu \mathbf { M }$ $5 0 \%$ cytotoxic concentration $[ \mathbf { C C _ { 5 0 } } ] > 1 0 0 \mu \mathbf { M } )$ .This fast and efficient assay for lead discovery and optimization provides an ideal platform for anti-EV71 drug development targeting 3C protease.

nterovirus 71(EV71) is the causative agent of hand,foot,and mouth disease (HFMD),which typically affects infants and children (1).HFMD usually presents as a mild febrile disease with a localized rash,but some patients may develop infection of the central nervous system (CNS）with illness ranging from aseptic meningitis through fatal encephalitis (2).In the last decade,outbreaks of HFMD have regularly reoccurred through Asia (3,4). According to data from the Chinese Center for Disease Control and Prevention (CDC),more than 420,000 cases of HFMD,with 70 deaths,were reported in China inApril 2Ol4.Currently,there is no antiviral therapy available for treatmentofHFMD.

EV71 belongs to the genus Enterovirus in the family Picornaviridae (5-7). Similar to other picornaviruses,EV71 contains a single-stranded, positive-sense RNA encodinga large polyprotein precursor (8,9). The polyprotein is further cleaved into four structural proteins (VPl to VP4) to form the viral capsid and seven nonstructural proteins (2A to 3D) for virus replication via the 2A protease and 3C protease (10,11). Except for the cleavage of VP1/2Aby the 2A protease (12) and the RNA-dependent cleavage of VP2/4(13),the 3C protease is absolutely required for the cleavage of other junction sites within the polyprotein (14-16). Meanwhile,EV71 3C reportedly interferes with the polyadenylation ofhost cellRNAby digesting CstF-64,a critical host factor for $3 ^ { \prime }$ pre-mRNA processing,suggesting a novel mechanism by which picornaviruses utilize $3 C ^ { \mathrm { p r o } }$ to impair host cell function (17).In addition, the 3C protease can also cleave numerous factors and regulators associated with cellular DNA-dependent RNA polymerases I, I,and III, such as the octamer-binding protein (OCT

1),TATA box-binding protein(TBP),cyclic AMP-responsive element-binding protein (CREB)，transcription activator p53, histone H3,and DNA polymerase III (18-21). The pivotal role of 3C protease in EV71 replication makes it an attractive target for antiviral discovery (22).

The crystal structure of unliganded EV71 3C protease showed thatEV71 3C protease folded into two domains that are related to other picornaviral 3C protease structures (23).The complex structures of EV71 mutants Hl33G,E71A,E71D with the inhibitor rupintrivir are similar to that of the unliganded protease structure (24).Luet al. thoroughly characterized the 3C proteases from EV7l and CVAl6 and reported a series of structures of both enzymes in free,peptide-bound,or inhibitor-bound form (25). These findings provided precise molecular insights into the substrate recognition and inhibition of 3C protease.

Profiling of the EV71 3C protease substrate could not only provide in-depth knowledge of catalytic mechanism at a molecular level, which would facilitate the design of potent protease inhibitors,but also lead to development of a reliable and robust biochemical assay for screening.In 2oo8,Kuo etal.synthesized six dodecapeptide substrates derived from the EV71 protease cleavage site and one dodecapeptide substrate (TSAVLQSGFRKM) from the severe acute respiratory syndrome coronavirus (SARSCoV）protease autoprocessing site for biochemical characterization of the EV71 3C protease by determining their specificities using high-performance liquid chromatography (HPLC). The results showed that EV71 3C protease cleaved TSAVLQSGFRKM more efficiently than the other six substrates (26).Eight peptides derived from CVAl6 polyprotein and three peptides derived from EV71 polyprotein were also investigated for their susceptibilities to 3C cleavage via HPLC assay, and the peptide (IGNTIEALFQG PPKER) corresponding to 2C-3A junction site of CVA16 could be efficientlyprocessed by both proteases( $\mathit { ' } V = 8 . 3 7 ~ { \mu \mathrm { M / m i n } }$ for EV71 and $1 0 . 7 2 ~ \mu \mathrm { M / m i n }$ for CVA16) (25).However, the HPLC assay is limited to conveniently determine the substrate specification,and the technique of fluorescence resonance energy transfer (FRET) is commonly used for the preparation of fluorogenic substrates for biochemical characterization of proteases and protease inhibitor screening (27).A fluorogenic peptide, $\boldsymbol { p }$ （ $\dot { P }$ -dimethylaminophenylazo)benzoic acid (Dabcyl)-KTSAVLQSGFRKME5-[(2-aminoethyl)amino]naphthalene-1-sulfonic acid (Edans) $\mathit { \Delta } ^ { \prime } K _ { m } = 5 . 8 ~ \mu \mathrm { M }$ 0 $k _ { \mathrm { c a t } } = 0 . 4 5 \ \mathrm { m i n } ^ { - 1 }$ ）was synthesized for the EV71 3C protease inhibitor screening(26). Cui et al. (23) also examined three fluorogenic substrates corresponding to the EV71 polyprotein autoprocessing site 3B-3C, the internal cleavage site (amino acids [aa] 251/252 with the Q-G junction） of CstF-64,and the SARS-CoV autoprocessing site. Contrary to the results of Kuo et al.,Dabcyl-TSAVLQSGFRKM-Edans,corresponding to the autoprocessing site of SARS-CoV protease,was demonstrated to be an unfavorable substrate for EV71 3C protease (the $K _ { m }$ and $V _ { \mathrm { m a x } }$ could not be precisely determined by Michaelis-Menten equation fitting because of its poor enzyme activity).In their hands, though the peptide substrate Dabcyl-RTATVQGPSLDFKE-Edans,corresponding to the EV71 3B-3C junction,displayed the best EV71 3C protease proteolytic activity, it still had a low cleavage rate and exhibited a low kinetic constant $k _ { \mathrm { { c a t } } } / K _ { m }$ of $7 . 1 \times \mathrm { i } \mathrm { \bar { 0 } ^ { - 4 } \ \mu M ^ { - 1 } }$ $\mathrm { m i n } ^ { - 1 }$ (23),indicating that the peptide Dabcyl-RTATVQGPSLD FKE-Edans may not be the optimal substrate for EV7l protease inhibitor screening.

Theaimof this study was to biochemically characterize EV71 3C protease with an aim to develop a reliable and efficient assay for the screening of EV71 3C protease inhibitors.The substrate specificity was defined using a series of fluorogenic peptides mapped from the cleavage sites on the EV71 polyprotein.A peptide [P08, NMA-IEALFQGPPK(DNP)FR] with FRET groups corresponding to the cleavage site of the 2C-3A junction was determined to be the most efficientlycleaved by EV71 3C protease,witha kinetic constant $k _ { \mathrm { { c a t } } } / K _ { m }$ of $1 1 . 8 \pm 0 . 8 \dot { 2 } \mathrm { m M } ^ { - 1 } \mathrm { m i } \bar { \mathrm { n } } ^ { - 1 }$ . Compared with the substrates reported previously,PO8 gave the highest signal-tobackground ratio,which makes it an ideal substrate for assay development.The impacts of pH, temperature,and dimethyl sulfoxide（DMSO）concentration were systematically studied.With a goal to evaluate EV71 3C protease inhibitors,a reliable and robust biochemical assay with a $Z ^ { \prime }$ factor of $0 . 8 7 \pm 0 . 0 5$ was developed. This fast and efficient assay for lead discovery and optimization provides an ideal platform for anti-EV71 drug development targeting 3C protease.

# MATERIALSANDMETHODS

Expressionand purification of the EV713C protease.The cDNA encoding the EV71 3C protease (aa1 to 183) was amplified by reverse transcription-PCR(RT-PCR) using the genomic RNA of EV71 (EV71 isolate BJ/ CHN/2008;GenBank accession no.ADT7l698.1）as the template and cloned into the NdeI and NotI sites of the pET28a expresson vector(GE Healthcare) (28-30).The plasmid was transformed into Escherichia coli strain BL21(DE3),and transformed cells were cultured at $3 7 ^ { \circ } \mathrm { C }$ in LB medium containing lOo mg/liter kanamycin.Protein expression was induced with $0 . 5 \mathrm { m M }$ isopropyl- $\cdot \beta$ -D-1-thiogalactopyranoside for $^ { 1 4 \mathrm { h } }$ after the optical density at $6 0 0 ~ \mathrm { { n m } }$ $( \mathrm { O D } _ { 6 0 0 }$ ）of the culture reached 0.6.Cells harvested by centrifugation were resuspended in lysis buffer ( $5 0 \mathrm { m M }$ TrisHCl $\mathrm { [ p H 7 . 0 ] }$ ， $1 5 0 \mathrm { m M N a C l }$ $1 0 \%$ glycerol) and then homogenized with aultra-high-pressure cell disrupter (JNBIO,Guangzhou,China) at $4 ^ { \circ } \mathrm { C }$ 业 The lysate was clarified by centrifugation at $2 0 { , } 0 0 0 \times g$ for $3 0 \mathrm { m i n }$ at $4 ^ { \circ } \mathrm { C }$ The supernatant was loaded onto a $1 5 \mathrm { - m l }$ Ni-nitrilotriacetic acid (NTA) column equilibrated in lysis buffer.The nonspecific contaminants were then removed by washing with 20 column volumes of lysis buffer.The 3C protease was subsequently eluted with $2 0 0 \mathrm { m M }$ imidazole in lysis buffer. The eluate was further purified with a Superdex 2Oo gel filtration column (GE Healthcare).The purified 3C protease was then concentrated to approximately10 to $1 2 ~ \mathrm { m g / m l }$ in a buffer containing $2 0 ~ \mathrm { m M }$ Tris-HCl (pH 7.0), $1 5 0 \mathrm { m M N a C l , 1 m M }$ EDTA, $5 \mathrm { m M }$ dithiothreitol[DTT],and $1 0 \%$ glycerol.

Synthesis of substrate peptides.A fluorogenic peptide library,P0l to P17,has been developed by solid-phase peptide synthesis using standard Fmoc [N-(9-fluorenyl) methoxycarbonyl] techniques (31). The cleavage of the peptide from 2-chlorotrityl chloride resin and removal of all of the side chain protecting groups were achieved in trifluoroacetic acid solution.Peptides were purified by semipreparative HPLC(Waters,USA) usinga Waters 515 device equipped with a 2489 dual-absorbance detector and analyzed on an Thermo $\mathrm { C _ { 1 8 } } 1 0 0 { - } \mathrm { \AA } ,$ 号 $5 - \mu \mathrm { m }$ ,250-by $1 0 \mathrm { - m m }$ column (unless otherwise specified) using gradients of $\mathrm { C H } _ { 3 } \mathrm { C N }$ （ $0 . 1 \%$ trifluoroacetic[TFA])- $\cdot \mathrm { H } _ { 2 } \mathrm { O }$ 0 $0 . 1 \%$ TFA) as the elution system with a flow rate of 1 $\mathrm { { m l } } / \mathrm { { m i n } }$ .Here,5-[(2-aminoethyl)amino]naphthalene-1-sulfonic acid (Edans)and 2-(N-methylamino)benzoyl (NMA) are the fluorescent donor moieties; $\boldsymbol { p }$ （ $p$ -dimethylaminophenylazo)benzoicacid(Dabcyl) and 2,4-dinitrophenyl (DNP) are the quencher moieties.Peptide homogeneityand identity were analyzed by analytical HPLC(Dionex,China） and liquid chromatography-mass spectrometry (LC-MS) (Shimadzu, Japan), respectively.

In vitro cleavage assay.For each fluorogenic peptide substrate, the cleavage activity of purified EV71 3C protease was determined in a reaction volume of $1 0 0 ~ \mu \mathrm { l }$ ,using $1 0 ~ \mu \mathrm { l }$ enzyme $( 5 0 ~ \mu \mathrm { M } )$ and $1 0 ~ \mu \mathrm { l }$ peptides (maximum concentration, $5 \mathrm { m M }$ )in the reactionbuffer ( $5 0 \mathrm { m M }$ Tris [pH 7.0],150 mM NaCl, $1 ~ \mathrm { m M }$ EDTA, $2 ~ \mathrm { m M }$ DTT, $1 0 \%$ glycerol) to yield a final concentration of $5 ~ { \mu \mathrm { M } }$ enzyme and substrate peptide at concentrations ranging from $5 \mu \mathrm { M }$ to $5 0 0 ~ { \mu \mathrm { M } }$ .Cleavage assay mixtures were incubated ina 96-well flat-bottom black microplate at $3 0 ^ { \circ } \mathrm { C }$ for $^ { 2 \mathrm { h } }$ ,and then reactions were terminatedby the addition of equal volume of acetonitrile. Once the protease cleaved the peptide bond, resulting in separation of the fluorophore and fluorescence quencher moiety,the fluorescence of the fluorophore was recovered by disappearance of the FRET effect in proportion to the degree of the protease reaction.The relative fluorescence of the reactive mixture was measured every $1 0 ~ \mathrm { s }$ over a period of $^ { 2 \mathrm { ~ h ~ } }$ ona Tecan Safire2 (Switzerland） multimode microplate reader (excitation wavelength $[ \lambda _ { \mathrm { e x } } ] = 3 4 0 \mathrm { n m }$ and emission wavelength $[ \lambda _ { \mathrm { e m } } ] = 4 9 0 \mathrm { n m }$ for Edans/Dabcyl; $\lambda _ { \mathrm { e x } } = 3 4 0 ~ \mathrm { n m }$ and $\lambda _ { \mathrm { e m } } = 4 4 0 ~ \mathrm { n m }$ for NMA/DNP). The ratio of the final fluorescence signal of the mixture to the background of the substrate (S/B）was calculated to quickly characterize the cleavage efficiencies of substrates,which are classically defined as ${ \mathrm { { S / B } } } =$ mean signal/mean background (32).

To determine the velocities of product formation,the instrument was precalibrated with the free fluorescent moiety Edans or NMA standard to calculate the relationship between relative fluorescence unit and substrate concentration. The initial velocities were calculated for allof the cleavage products from three independent experiments.The kinetic parameters $K _ { m } , V _ { \operatorname* { m a x } } ,$ and catalytic efficiency $( k _ { \mathrm { c a t } } / K _ { m } )$ were calculated assuming Michaelis-Menten kinetics for all of the cleavage peptides.No significant hydrolysis of the peptide substrates was observed in the absence of EV71 3C protease.

In order to determine the optimal reaction conditions for 3C protease, various salt concentrations, $\mathrm { \ p H }$ values,temperatures,and dimethyl sulfoxide (DMSO) concentrations were used.The protease and substrate were incubated at different combinations of $\mathrm { \Delta N a C l }$ concentration and $\mathrm { \Delta p H }$ = The pHvalues selected for the protease proteolysis were 5,6,7,and 8,and each pHwas carried out with different $\mathrm { \Delta N a C l }$ concentrations (1oo,150, 200,and $5 0 0 \mathrm { m M }$ ).To determine the temperature condition,the protease reaction sample was incubated at different temperatures (10,20,30,40, and $5 0 ^ { \circ } \mathrm { C }$ ).The effect ofDMSO on enzymatic activity of the EV713C protease was determined by assays in buffer containing DMSO concentrations varying from $0 . 5 \%$ to $1 0 \%$ (vol/vol).Fluorescence was measured following incubation at $3 0 ^ { \circ } \mathrm { C }$ for $^ { 2 \mathrm { ~ h ~ } }$ .All reactions were performed in triplicate to determine the effects of salt concentration, $\mathrm { \ p H }$ ,temperature, and DMSO on protease enzymatic activity.

MD simulation.To run molecular dynamics (MD) simulations,enzyme-substrate complexes were first prepared. The receptor enzyme (PDB 4GHQ) was obtained from the Protein Data Bank(29).The ligand peptides were positioned in the protein similar to the position in the work byLu et al.(25),which solved the crystal structure of the CVA16 3C protease with the peptide FAGLRQAVTQ (PDB 3SJ9).We aligned the structure of 4GHQ,maintained the coordinates of the backbone and $\beta$ -carbon of 3SJ9's ligand,and replaced the residues to substitute for our ligand substrates.The Leap module of Amberl2 with the standard Amber99SSB force field was then used to build the enzyme-substrate complexes.The side chains of ligand peptides are added automatically in Leap.The existence of the $\beta$ -carbon ensures the correct direction of side chains (33).Chloride ions were added as counterions to neutralize the complexes (34).TIP3PBOX(35) was used to add an explicit water box, and the distance between the edges of the box and the closet atoms of the complexes was set to $1 0 \mathrm { \AA }$

There are two steps in the minimization procedure to eliminate bad contacts.Every step was using steepest-descent minimization,followed by conjugate gradient minimization with the same steps.First,with the complex fixed,solvent and ions were subjected to a 2,0oo-step optimization (1,000 steps of steepest-descent minimization and 1,ooo steps of conjugate gradient minimization). Second, the whole system was optimized using 5,ooo steps of minimization.After the minimization procedure, every solvated complexwas equilibrated by 6O ps of heating from $0 \mathrm { K }$ to $3 0 0 \mathrm { K }$ and 6O ps of density equilibration.Both simulations were run with a 2-fs time step, $1 0 { \mathrm { - k c a l ~ } } \cdot { \mathrm { m o l } } ^ { - 1 } ~ .$ $\textup { \AA } ^ { - 2 }$ constraints on the complex,and Langevin dynamics (36) for temperature control.Before molecular dynamics simulation,an additional 1-ns equilibration was run to refine the interactions of receptor and ligand.In the first $0 . 5 \mathrm { n s }$ ,the distances of all the hydrogen bonds between residues of receptor and all the main chains of ligand and the side chain of the second peptide from the C terminus were restrained.In the remaining $0 . 5 \mathrm { n s }$ ,a $2 \mathrm { - k c a l \cdot m o l ^ { - 1 } \cdot \& ^ { - 2 } }$ constraint was set on the complex to avoid a significant structure change.Then, the final phase of equilibration of every system was used to run a 3-ns molecular dynamics simulation.The conditions were the same as for the equilibration procedure but without restraints or constraints.The SHAKE algorithm (44) was applied to keep all bonds involving hydrogen atoms rigid.

$\boldsymbol { Z ^ { \prime } }$ factorcalculation.The most efficient substrate,NMA-IEALFQGP PK(DNP)FR $( 2 0 \mu \mathrm { M }$ final concentration),was added to 96-well flat bottom black microplates containing EV71 3C protease at a $1 \mu \mathrm { M }$ final concentration (positive control） or buffer without protease (negative control). Since the $Z ^ { \prime }$ factor is dependent on means and standard deviations (SD),each combination had 60 wells.The plate was incubated at $3 0 ^ { \circ } \mathrm { C }$ for $^ { 2 \mathrm { ~ h ~ } }$ ,and fluorescence readings were obtained. The $Z ^ { \prime }$ factor is calculated from the means and SD of fluorescence from positive (in the presence of protease) and negative (in the absence of protease) control plates as previously described (32): $Z ^ { \prime }$ factor $= 1 - [ ( 3 \sigma _ { s } + 3 \sigma _ { c } ) / \mu _ { s } -$ ${  { \vert { \boldsymbol { \mu } } _ { c } \vert } } )$ ],where ${ \bf { \sigma } } _ { \mathbf { { \sigma } } } \mathbf { { \sigma } } _ { \mathbf { { s } } }$ and $\mathbf { \sigma } _ { \mathbf { c } }$ are the standard deviations for the positive and negative controls,respectively,and ${  { \vert \mu _ { s } \vert } }$ and ${ \mu } _ { \mathrm { c } }$ are the means for the positive and negative controls,respectively.

Enzyme inhibition assay.In the in vitro cleavage assay,the peptide NMA-IEALFQGPPK(DNP)FR was demonstrated to be most efficiently processed by EV71 3C protease,which was used as the substrate for the enzyme inhibition assay.Here,three tested compounds (see Table 3) were synthesized as described previously(26).The assays were performed with $1 0 0 \mathrm { - } \mu \mathrm { l }$ samples containing $1 ~ { \mu \mathrm { M } }$ protease, $2 0 \mu \mathrm { M }$ substrate,and various concentrations ( $\mathrm { \Delta } _ { \mathrm { \cdot } 1 0 \mu \mathrm { M } }$ to $0 . 1 5 6 \mu \mathrm { M }$ )of the tested compounds inabuffer of $5 0 ~ \mathrm { m M }$ Tris, $\mathrm { p H } 7 . 0$ .To determine the inhibitory effects of the various compounds,the compound and the enzyme were preincubated at $3 0 ^ { \circ } \mathrm { C }$ for $3 0 ~ \mathrm { m i n }$ prior to addition of the substrate.The reaction mixture was then incubated for $^ { 2 \mathrm { h } }$ at $3 0 ^ { \circ } \mathrm { C }$ .The reaction was terminated by the addition of acetonitrile.The initial velocities of the enzymatic reactions were determined and fitted to a sigmoidal dose-response equation with nonlinear regression analysis using the program GraphPad Prism.The data from three independent assays were used as input for Prism to determine $5 0 \%$ inhibitory concentrations $( \mathrm { I C } _ { 5 0 } s ) \$ ！

Viruses and cells.Human rhabdomyosarcoma (RD) cells were grown in Dulbecco's modified Eagle's medium (DMEM) (GIBCO） supplemented with $1 0 \%$ fetal bovine serum (FBS) (GIBCO) at $3 7 ^ { \circ } \mathrm { C }$ in a $5 \% \mathrm { C O } _ { 2 }$ humidified incubator.

The EV71 reporter virus system [EV71(FY)-Luc pseudotype virus system] containing pcDNA6-FY-capsid and pEV71-Luc-replicon lacking the Pl region were kindly supplied by Wenhui Li,National Institute of Biological Sciences,Beijing, China.

Antiviral assay.The EV7l pseudotype virus [EV71(FY)-Luc] was produced by using a method developed as previously described (11,37). Briefly,the pEV71-Luc subgenomic replicon was linearized through digestion with SalI restriction enzyme and was used as a template for RNA transcription.The EV71 replicon RNA transcripts were prepared in vitro by using Ambion MEGAscript kits.The pcDNA6-FY-capsid plasmid was transfected into HEK-293T cells at 60 to $8 0 \%$ confluence.After $2 4 \mathrm { h }$ posttransfection,EV71 subgenomic replicon RNA was then transfected using Lipofectamine 2ooo (Invitrogen).EV71 pseudotype virus was harvested at $2 4 \mathrm { h }$ after RNA transfection with two freeze-thaw cycles.To quantify the EV71 pseudotype virus,the stocks were serially diluted 10-fold and incubated with RD cells for $2 4 \mathrm { h }$ at $3 7 ^ { \circ } \mathrm { C }$ .The cells were then harvested,and the luminescence was detected as described in the manufacturer's protocol for the Bright-Glo luciferase assay system (Promega).

Fifty percent effective concentrations $( \mathrm { E C } _ { 5 0 } \mathrm { s } )$ ）of tested compounds were determined on RD cells.RD cells ( $3 \times 1 0 ^ { 4 }$ per well in $1 0 0 \mathrm { ~ \textmu l ~ } 1 0 \%$ FBS-DMEM) were seeded in 96-well plates and cultured at $3 7 ^ { \circ } \mathrm { C }$ and $5 \%$ $\mathrm { C O } _ { 2 }$ overnight.Cells were treated with a dilution gradient of different compounds,ranging from 0.39 to $1 0 0 \mu \mathrm { M }$ .EV71(FY)-Luc virus at a multiplicity of infection (MOI) of1 was added to 96-well plates.The luminescence was measured at $4 9 0 ~ \mathrm { n m }$ after $2 4 \mathrm { ~ h ~ }$ postinfection,and $\mathrm { E C } _ { 5 0 } s$ of compounds were calculated using log(concentration） versus luminescence $( 4 9 0 \mathrm { n m } )$ ）in GraphPadPrism.

Cytotoxicity.The cytotoxicity was measured by WST-1 cell proliferation using a cytotoxicity assay kit according to the manufacturer's protocol (catalog no. 05015944001; Roche,USA).Briefly,RD cells $( 3 \times 1 0 ^ { 4 }$ per well in $1 0 0 \mu \mathrm { l } 1 0 \%$ FBS-DMEM) were cultured at $3 7 ^ { \circ } \mathrm { C }$ under $5 \% \mathrm { C O } _ { 2 }$ in 96-well plates,followed by the addition of $5 0 ~ \mu \mathrm { l }$ compound solution with a concentration ranging from 0.78 to $2 0 0 ~ \mu \mathrm { M }$ . The cells were incubated at $3 7 ^ { \circ } \mathrm { C }$ for $2 4 \mathrm { h }$ ,and then $1 0 \mu \mathrm { l }$ WST-1 reagent was added to each well.The luminescent signals were read at $4 9 0 ~ \mathrm { n m }$ with a microplate reader (Bio-Rad,USA).The percentage of viable cells after treatment with different concentrations of compound was calculated as follows: percentage of inhibition $= 1 0 0 \times \mathrm { O D } _ { 4 9 0 }$ of treated sample $/ \mathrm { O D } _ { 4 9 0 }$ of cell control sample. The $5 0 \%$ cytotoxic concentration $\left( \mathrm { C C } _ { 5 0 . } \right)$ )was defined as the concentration of compound that reduced the treated RD cells by $5 0 \%$ relative to the untreated control cells and was calculated using GraphPad Prism.

![](images/733d20f61e9813a08e7907082b6cd816850dde4cee35df525c05d519ddc81b60.jpg)  
FIG1 Sizeexclusionchromatography(SEC)ofEV713Cprotease.TheEV713Cprotease waspurifiedbyHPLCwitharetentionvolumeof $1 7 \mathrm { m l }$ ,and samples (Tlto T5)obtaiedduringthepurificationprocesswereanalyzedbySDS-PAGEandstainingwith CoomassiebriliantblueR-250.

# RESULTS

Expression and purification of EV71 3C protease.EV71 3C protease was expressed and purified as described previously (29).No amino acid substitutions were introduced in the recombinant protease sequence.After the first induction at $3 7 ^ { \circ } \mathrm { C } .$ ,the majority of 3C-like protease was found in the fraction of the cell lysate.The lysate was induced with isopropyl- $\cdot \beta$ -D-1-thiogalactopyranoside for an additional $1 4 \mathrm { h }$ .As a result,most of the EV71 3C protease was found in the soluble fraction. The protease was purified by HPLC with a retention volume of $1 7 \mathrm { m l }$ ,and SDS-PAGE analysis revealed the presence of a major protein with molecular mass of 22 kDa, which indicates that recombinant EV71 3C protease was obtained with over $9 5 \%$ purity (Fig. 1).

Substrate specificity ofEV7l protease.The cleavage activity of the EV71 3C protease was assayed by fluorescence release from a set of synthesized fluorogenic peptide substrates (Table 1) derived from the junctions of polyprotein (TW/2231/98） that were cleaved by 3C protease (26).The ratios of signal to background were used to quickly evaluate the cleavage efficiency of EV71 3C protease against the peptide substrates.Two reported substrates withFRET pairDabcyl/Edans,P01 (Dabcyl-RTATVQGPSLDFK E-Edans）(23）and P02(Dabcyl-TSAVLQSGFRKME-Edans) (26),were prepared as the controls.The signal-to-background (S/B) ratios for P0l and P02 were 5.7 and 2.2,respectively,which indicated that the reported substrate POl could be more efficiently cleaved by EV71 3C protease than PO2.A shorter-wavelength FRET pair,NMA/DNP,was chosen for our study to improve the signal-to-background ratio (38). The FRET peptides P03 (NMARTATVQGPSLDFK-DNP)and P04[NMA-TSAVLQSGFRK(DN P)M],which linked the NMA group and the quencher group DNP to the ε-amino function ofLys,were obtained to test in vitro cleavage activity against EV71 protease compared with the reported substrates.A slightly improved S/B ratio was observed for the peptide P03 (S/B,6.5） compared to the corresponding peptide P01 (S/B,5.7).A similar result was observed in the comparison of substrate PO4(S/B,2.8)and the corresponding substrate PO2(S/B, 2.2).Based on this result, the peptides corresponding to the cleavage sites of EV71 3C protease with the NMA/DNP pair (P05 to P09)were synthesized and evaluated for their specificity(Table1).

TABLE1 Synthesized peptides derived from the polyprotein of EV71   

<html><body><table><tr><td></td><td></td><td></td><td></td><td colspan="3">Mutational peptides</td></tr><tr><td>Peptide</td><td>Sequence</td><td>Cleavage site</td><td>S/B</td><td>Peptide</td><td>Sequence</td><td>S/B</td></tr><tr><td>P01</td><td>Dabcyl-RTATVQGPSLDFKE-Edans</td><td>3B-3C</td><td>5.7</td><td>P11</td><td>NMA-IAALFQGPPK(DNP)FR</td><td>4.6</td></tr><tr><td>P02</td><td>Dabcyl-TSAVLQSGFRKM-Edans</td><td>SARS-CoV</td><td>2.2</td><td>P12</td><td>NMA-IELLFQGPPK(DNP)FR</td><td>8.2</td></tr><tr><td>P03</td><td>NMA-RTATVQGPSLDFK(DNP)</td><td>3B-3C</td><td>6.5</td><td>P13</td><td>NMA-IEAAFQGPPK(DNP)FR</td><td>9.7</td></tr><tr><td>P04</td><td>NMA-TSAVLQSGFRK(DNP)M</td><td>SARS-CoV</td><td>2.8</td><td>P14</td><td>NMA-IEALAQGPPK(DNP)FR</td><td>1.9</td></tr><tr><td>P05</td><td>NMA-RQAVTQGFPTELK-DNP</td><td>VP2-VP3</td><td>7.7</td><td>P15</td><td>NMA-IEALFQSPPK(DNP)FR</td><td>1.9</td></tr><tr><td>P06</td><td>NMA-QTGTIQGDRVADK-DNP</td><td>VP3-VP1</td><td>1.8</td><td>P16</td><td>NMA-IEFLFQGPPK(DNP)FR</td><td>2.0</td></tr><tr><td>P07</td><td>NMA-DEAMEQGVSDYIK-DNP</td><td>2A-2B</td><td>3.6</td><td>P17</td><td>NMA-IEALKQGPPK(DNP)FR</td><td>1.9</td></tr><tr><td>P08</td><td>NMA-IEALFQGPPK(DNP)FR</td><td>2C-3A</td><td>15.3</td><td></td><td></td><td></td></tr><tr><td>P09</td><td>NMA-LFAGFQGAYSGAK-DNP</td><td>3A-3B</td><td>1.8</td><td></td><td></td><td></td></tr><tr><td>P10</td><td>Dabcyl-IEALFQGPPKFRE-Edans</td><td>2C-3A</td><td>6.8</td><td></td><td></td><td></td></tr></table></body></html>

Though these peptides had the same Gln-Gly pair interaction at the P1-P1'position,the proteolysis efficiencies of the initial set of peptides (PO5 to Po9) in the cleavage assay varied significantly (S/B,1.8 to 15.3)(Fig.2A).Among all substrates,the highest cleavage efficiency was observed for the substrate peptide P08 [NMA-IEALFQGPPK(DNP)FR] (S/B,15.3)，which represents the native EV7l protease cleavage sites at the 2C-3A junction.Mo derate proteolytic efficiency was observed for peptide PO5 (S/B, 7.7),which corresponds to VP2-VP3.The substrate P07 showed much lower cleavage efficiency,with an S/B value of 3.6,which corresponds to the 2A-2B junction.Poor enzymatic activity was observed against the substrate peptides PO6 and PO9(S/B,1.8), representing the VP1-VP3 and 3A-3B junction sites,respectively.

![](images/8c5686cf36764c34ced04bd9f865e918a628c308682ae702b5c1509e44f22e97.jpg)  
FIG2(A)InvitroeavaeactiiyfEroteaseithierentpetidesubstratesositieelatiefuoreceitesityfchbstrat $( 5 0 ~ \mu \mathrm { M } )$ （2 incubated with EV71 3C protease $( 5 ~ { \mu \mathrm { M } } )$ ; negative,background ofeach peptide.(B) Substrate specificity ofEV713C protease.The proteolytic reactions were performed using substrates P01and Po8.O,control substrate PO1 derived from the 3B-3C junction; $\mathbf { \delta m }$ novelsubstrate $\mathrm { P 0 8 }$ derived from the 2C-3A junction.The data presented are mean values fromexperiments in triplicate,and the errorbars represent the standard deviations.

As the substrate PO8 exhibited the highest cleavage efficiency, peptide P10,with the same sequence as Po8 but with the FRET pair Dabcyl/Edans,was also prepared to study the impact of the FRET pair.Pi0 was onlymoderatelyhydrolyzedby 3C protease (S/B,6.8),with cleavage being less effcient than that of Po8 (S/B, 15.3).

To further elucidate the substrate specificities of the 3C proteases,seven site-directed peptide variants,P11 to P17,with substitutions of amino acids in the P5 to $\mathrm { \Delta P l ^ { \prime } }$ region of PO8 were examined (Fig.2A).Peptide P1l,with a P5 site variant E to A within the context of the PO8 sequence,showed a relatively lower activity(S/B,4.6).Peptides P12 andP13,withaP4 site variant A to L and P3 site variantL to A,respectively,represented moderate cleavage efficiency (S/B,8.2 and 9.7),whereas peptide P16,with the variant A to F at the P4 position,exhibited poor cleavage efficiency(S/B,2.0).Moreover,itis notable that EV713C protease showed strong preference for the P2 and $\mathrm { \Delta P l ^ { \prime } }$ residues.Variants at the P2 and $\mathrm { { } ^ { P l { ' } } }$ positions dramatically reduced the cleavage efficiency by EV7l protease,as observed by substrates P14,P15,and P17,withthe $\mathrm { { \cal S } / B }$ valuesofless than 2.

Optimization of cleavage assay conditions.Peptide PO8,with the highest signal-to-background ratio,was selected as the substrate for optimization of cleavage assay conditions.The effect of $\mathrm { \Delta p H }$ on the enzymatic activity of EV713C protease was determined by assays using buffers in the pH range from5 to 8 (Fig.3A). In agreement with earlier reports on EV71 3C protease (23),the pH optimum for reaction for the peptide substrate was 7.0.It is noteworthy that the protease almost lost the enzymatic activity when the pH was decreased to 5.Changes in salt concentration from 100 mM to $5 0 0 \ \mathrm { m M \ N a C l }$ produced little effect on the enzymatic activity.The combination of salt concentration and pHvalue that gave the highest proteolytic activity was 150 mMNaCl and $\mathrm { p H } 7 . 0$

These reaction conditions were then applied throughout the downstream assay development.

The effects of the temperature on proteolysis are shown in Fig. 3B.The maximum fluorescence was observed within $^ { 2 \mathrm { h } }$ of incubation at $3 0 ^ { \circ } \mathrm { C } ,$ while the protease activity decreased with increasing or decreasing temperature beyond $3 0 ^ { \circ } \mathrm { C }$ .Hence,the optimal temperature for protease activityis considered $3 0 ^ { \circ } \mathrm { C }$

The effect of DMSO on the proteolytic activity of EV71 3C protease was also determined (Fig.3C).No significant difference in protease cleavage efficiency was observed with DMSO concentrations of $0 . 5 \%$ to $1 0 \%$ (vol/vol),indicating that this assay is not affected byup to $1 0 \%$ DMSO.

Kinetic analysis of EV71 3C protease substrates.Thekinetic parameters for proteolysis of substrates were determined in order to quantify the cleavage efficiency and substrate specificity of EV713C protease (Table2).For the control substrates, the $K _ { m }$ and $k _ { \mathrm { { c a t } } }$ values of PO1 were determined to be $8 2 . 5 \mu \mathrm { M }$ and $0 . 1 2 \mathrm { m i n } ^ { - 1 }$ respectively,while for the substrate P02,the $K _ { m }$ and $k _ { \mathrm { c a t } }$ could not be precisely determined because of its low activity. Of all 13 peptides,the kinetic parameters for most of the substrates could be calculated assuming Michaelis-Menten kinetics,except for P06, P09,P14,P15,P16,and P17,which had low S/B values (no more than2).The substrateP08[NMA-IEALFQGPPK(DNP)FR]had the lowest $K _ { m }$ 0 $( 4 3 . 1 \pm 2 . 3 \mu \mathrm { M } )$ and the highest turnover number （20 $( 0 . 5 1 \pm 0 . 1 2 \operatorname* { m i n } ^ { - 1 } )$ of all the peptide substrates,resulting in the highest $k _ { \mathrm { c a t } } / K _ { m }$ 1 $( 1 1 . 8 ~ \pm ~ 0 . 8 2 ~ \mu \mathrm { { M } } ^ { - 1 } \operatorname* { m i n } ^ { - 1 } )$ . Thisclearlydemonstrated that P08 was the most suitable substrate for EV71 3C protease cleavage.Compared with the control substrate PO1(Fig.2B), EV71 3C protease displayed much higher enzymatic activity against the substrate $\mathrm { P 0 8 }$ due to the cumulative effects of change in both $K _ { m }$ and $k _ { \mathrm { { c a t } } }$

Molecular dynamics.In order to gain in-depth understanding of the catalytic mechanism of EV71 3C protease at the molecular level,molecular dynamics simulations and molecular docking were applied to build structural models of the complexes and furtherdetermine the relative contributions ofdifferent residues to the overall binding affinity.We docked the P6 to $\mathrm { P l ^ { \prime } }$ portions (such as RTATVQG for POl and IEALFQG for P08) of the peptides into the substrate-binding groove of EV71 3C protease for molecular dynamics simulation (Fig. 4A).

EV713C protease is a specific protease that preferentially recognizes substrates with the Q-G junction.In the molecular model, $\mathrm { \Delta P l ^ { \prime } }$ -Gly was oriented toward the $\mathbf { \Delta } S 1 ^ { \prime }$ subsite constituted by the hydrophobic parts of residues His24,Phe25,His108,Glyl45,and Cysl47.In the natural cleavage sites within the EV71 polyprotein precursors,the P1 position is invariably occupied by Gln.This residue is accommodatedby theHisl6land Thrl42 at the bottom of the S1 pocket. The side chain of P1-Gln formed three strong hydrogen bonds (2.5 to $3 . 0 \mathrm { \AA }$ ）with these two residues (Fig. 4B).

![](images/476d9c5009270937820934f224f839001206de3bdd2db619a65d00991d68cee3.jpg)  
FIG 3 (A) Optimization of salt concentration and $\mathrm { \Delta p H }$ for protease assay.The fluorescence of proteolytic reactions were measured in different reaction buffers with various combinations of salt (100 to $5 0 0 \mathrm { m M } ,$ andpH $\mathrm { ( p H } 5$ to 8).(B) Effect oftemperature (10 to $5 0 ^ { \circ } \mathrm { C }$ )on enzymaticactivity.Positive,relative fluorescence intesiubbcgbe concentrations (0.5 to $1 0 \%$ )ofDMSOwereusedinthereactionbufer.Thedatapresentedaremeanvalues fromexperiments in triplicate,withtheerrorbars representing the standard deviations.

In contrast to the high conservation of P1-Gln,the hydrophobic S2 pocket is large enough to accommodate various side chains of the substrate.Consistently,the P2 residues of substrates (P03 and P05 to Po9) have certain flexibility.The best selectivity is for the relative large hydrophobic residue Phe,as observed with P08, which has the highest cleavage activity.A variant with Phe to Ala orLys at the P2 position resulted in a dramatic reduction of the cleavage site(Po8 versus P14 and P17),indicating the pivotal role of the P2 residue.The Phe side chain mainly hydrophobically contacted with Lysl30 and Glu71at the S2 specificity pocket of the EV713C protease.The introduction of the small residue Ala at P2 left most of the specificity pocket unoccupied.Likewise,the introduction of the basic amino acid Lys may not be appropriate for the hydrophobic S2 pocket,as itled to dramatically decreased cleavage efficiency.Interestingly,Arg39,located at the bottom of S2 pocket, provided different orientations depending on the appearance of a benzene ring in the P2 side chain when we stimulated these peptides.In the molecular model ofPo8,as well as peptide variantsP1l,P12,andP13,Arg39 strikinglyrotated about $7 0 ^ { \mathrm { o } }$ toward to the Phe residue and interacted with the benzene ring, while no such rotation was observed in other substrates without Phe at the P2 position, such as POl (Fig.4C). Consistently,Arg39 has been demonstrated to play a key role in the EV71 protease proteolysis process (24).This should partially account for the observed dramatically reduced activity with the variant of Phe to Ala or Lys at the P2 position.

TABLE 2 Cleavage efficiencies of fluorogenic peptide substratesa   

<html><body><table><tr><td>Peptide</td><td>Sequence</td><td>Km(μM)</td><td>kcat (min-1)</td><td>kcat/Km (mM-1min-1)</td></tr><tr><td>P01</td><td>Dabcyl-RTATVQGPSLDFKE-Edans</td><td>82.5 ± 3.8</td><td>0.12 ± 0.01</td><td>1.45 ± 0.1</td></tr><tr><td>P02</td><td>Dabcyl-TSAVLQSGFRKM-Edans</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P03</td><td>NMA-RTATVQGPSLDFK(DNP)</td><td>58.2 ± 4.7</td><td>0.21 ± 0.08</td><td>3.6 ± 0.72</td></tr><tr><td>P04</td><td>NMA-TSAVLQSGFRK(DNP)M</td><td>75 ± 3.2</td><td>0.02 ± 0.008</td><td>0.27 ± 0.07</td></tr><tr><td>P05</td><td>NMA-RQAVTQGFPTELK-DNP</td><td>92.4 ± 5.3</td><td>0.2 ±0.02</td><td>2.2 ± 0.13</td></tr><tr><td>P06</td><td>NMA-QTGTIQGDRVADK-DNP</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P07</td><td>NMA-DEAMEQGVSDYIK-DNP</td><td>73.5 ± 6.1</td><td>0.03 ± 0.01</td><td>0.4 ± 0.11</td></tr><tr><td>P08</td><td>NMA-IEALFQGPPK(DNP)FR</td><td>43.1 ± 2.3</td><td>0.51 ± 0.12</td><td>11.8 ± 0.82</td></tr><tr><td>P09</td><td>NMA-LFAGFQGAYSGAK-DNP</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P10</td><td>Dabcyl-IEALFQGPPKFRE-Edans</td><td>63.2 ± 3.6</td><td>0.17 ± 0.03</td><td>2.69 ± 0.15</td></tr><tr><td>P11</td><td>NMA-IAALFQGPPK(DNP)FR</td><td>75 ± 5.3</td><td>0.16 ± 0.03</td><td>2.13 ± 0.24</td></tr><tr><td>P12</td><td>NMA-IELLFQGPPK(DNP)FR</td><td>48.2 ± 3.6</td><td>0.24 ± 0.06</td><td>5±0.38</td></tr><tr><td>P13</td><td>NMA-IEAAFQGPPK(DNP)FR</td><td>66.7 ± 5.3</td><td>0.3 ± 0.05</td><td>4.5 ± 0.14</td></tr><tr><td>P14</td><td>NMA-IEALAQGPPK(DNP)FR</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P15</td><td>NMA-IEALFQSPPK(DNP)FR</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P16</td><td>NMA-IEFLFQGPPK(DNP)FR</td><td>ND</td><td>ND</td><td>ND</td></tr><tr><td>P17</td><td>NMA-IEALKQGPPK(DNP)FR</td><td>ND</td><td>ND</td><td>ND</td></tr></table></body></html>

“Dataaremeansandstandarddeviationsfromexperimentsperformedin triplicate.ND,notdeterminedduetopooreavageeiency

![](images/d63be1d3a5322a4d83d1c4a33bdf5b0d734d782529ba2044f4134432798477fc.jpg)  
FIG 4 (A).Surface representation of EV71 3C protease complexed with the P6 to $\mathrm { { \ P 1 ^ { \prime } } }$ portions of POl(yellow) and PO8 (green).(B) Detailed interaction between $\mathrm { { \ P 1 ^ { \prime } } }$ andlfpeptidedhproteasHrogdeosss.ticodeooteaseteactigii ofpeptideelde)etsreotatooelsopofofed (magenta).

The side chain of P3-Leu was oriented to the bulk solvent region. The backbone of Leu interacted with Glyl64 of the protease via two hydrogen bonds.The P4-Ala residue was a relatively conserved residue in the natural joint sequences (Table1).Two backbone hydrogen bonds were formed between Ala and the mainchain groups of Asnl26 and Serl28,respectively.The size and hydrophobic characters of ahydrophobic pocket of the S4 subsite well accommodated the methyl group of Ala.SubstrateP12,which contained an Ala-to-Leu mutation at the P4 position,represented moderate cleavage potency,but P16,containing an Ala-to-Phe mutation, showed weak cleavage effciency,indicating that the S4 pocket is tolerant to a relatively small residue.There were no apparent pockets and interactions in the protease for the P5 and P6 residues of the substrate.P5 Glu contacted with the protease only by a hydrogen bond formed with the backbone of Asnl26 (Fig. 4D).With a Glu-to-Ala mutation,P11 showed a 5-fold reduction of cleavage efficiency.We speculated that this accounts for the location of the P5 position at the entry of the substrate-binding site.

$\boldsymbol { Z } ^ { \prime }$ factor analysis.In order to assess the quality of this fluorogenic model substrate-based assay for the screening of EV71 3C protease inhibitors,the value of the $Z ^ { \prime }$ factor was determined.A $Z ^ { \prime }$ factor of ${ > } 0 . 5$ means a robust and reliable indicator for highthroughput screening assay (32).A scatter plot of fluorescence units is shown in Fig.5,which shows the assay range and data derivation for the positive control (substrate fluorescence after incubation with EV71 3C protease) and negative control (background of the substrate).The $Z ^ { \prime }$ factor was obtained from 60 independent positive controls containing $2 0 \mu \mathrm { M }$ P08[NMA-IEA LFQGPPK(DNP)FR] and $1 ~ \mu \mathrm { M }$ protease and negative controls containing $2 0 \mu \mathrm { M }$ P08 without protease.The $Z ^ { \prime }$ factorwasdetermined as $0 . 8 7 \pm 0 . 0 5$ ,which indicates a good overall quality of the assay and the required robustness for further high-throughput screening application.

![](images/979ae57dc68da0ed45409b0c1e914cc1a0e60a71f455d9c3a64a9fc79466f9cd.jpg)  
FIG 5 Validation of the quality of the EV71 3C protease assay.The assay was performed using $2 0 \mu \mathrm { M }$ substrateP08[NMA-IEALFQGPPK(DNP)FR]with1 $\mu \mathrm { M }$ protease (positive control) $( n = 6 0$ ,circles) and without protease (negative control) ${ \bf \zeta } _ { n } = 6 0$ ,triangles).Experiments were repeated more than three times with consistent results.

Evaluation of inhibitors.To validate the application of the established fluorogenic substrates forEV713C protease inhibitor screening,two published compounds(compounds 1 and 2) and one novel compound (compound 3) were synthesized to evaluate their inhibitory activities against EV71 3C protease.The inhibitory activity of each compound was examined using peptide P08 as the substrate.Cleavage of the substrate byEV71 3C protease in the presence of various concentrations of the inhibitor was monitored with a96-well fluorescence plate reader,as described above. Aldehyde 1,which was previously reported to inhibit EV71 protease (26),showed efficacy with an $\mathrm { I C } _ { 5 0 }$ of $4 . 9 7 \pm 0 . 2 8 \mu \mathrm { M }$ in this study.With a dimethylamine substitution at the benzene group of the cinnamoyl fragment,derivative 2 demonstrated a small increase of anti-EV71 activity $( \mathrm { I C } _ { 5 0 } = 3 . 6 6 \pm 0 . 4 3 \mu \mathrm { M } )$ ,which is consistent with the previous report (26).As shown in Table 3,we synthesized a novel compound (compound 3) with a 2-chloridephenylacetyl substitution instead of the cinnamoyl of compound 1.The biological analysis revealed that the efficacy of compound 3 $( \mathrm { I C } _ { 5 0 } = 1 . 8 9 \pm 0 . 2 5 \mu \mathrm { M } )$ is twice that reported for compounds 1 and 2.

Wenext determined the inhibitory activities of the tested compounds after treatment with EV71(FY)-Luc pseudotype virus-infected RD cells.Consistently,the results indicated that the virus proliferation was also significantly attenuated by the treatment with the three compounds, with $\mathrm { E C } _ { 5 0 } s$ of $\cdot 3 . 3 8 \pm 0 . 4 7 , 3 . 0 7 \pm 0 . 5 6$ and $4 . 5 4 \pm 0 . 5 1 ~ \mu \mathrm { M }$ for tested compounds 1,2,and 3,respectively. The results of the cytotoxicity assay revealed that up to 100 $\mu \mathrm { M }$ ,all three compounds had no cytotoxic effect on the viability of RD cells.

# DISCUSSION

EV71 encodes a polyprotein in a single open reading frame,where the polyprotein is processed by 3C protease and 2A protease into individual functional proteins.Thus,3C protease serves as an excellent target for developing inhibitors against EV71.In previous studies,together with the research on EV7l protease structure,a number of substrates were investigated by HPLC assay,and two fluorogenic peptides,P01 (Dabcyl-RTATVQGPSLDFKE-Edans) andP02(Dabcyl-TSAVLQSGFRKME-Edans),were alsoused to studyinhibitor activity (23,25,26).In this study,we report the substrate specificities and complete kinetic parameters of the EV71 3C protease for a small set of synthesized fluorogenic peptide substrates.We have established a robust fluorogenic substrate model for the screening of EV71 protease inhibitors.

TABLE 3 Peptidomimetic inhibitors against EV71 3C protease"   

<html><body><table><tr><td>Compound no.</td><td>IC50 (μM)</td><td>EC50 (μM)</td><td>CC50 (μM)</td></tr><tr><td></td><td>4.97 ± 0.28</td><td>3.38 ± 0.47</td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>2</td><td>3.66 ± 0.43</td><td>3.07 ± 0.56</td><td>>100</td></tr><tr><td>3</td><td>1.89 ± 0.25</td><td>4.54 ± 0.51</td><td>>100</td></tr></table></body></html>

¹ Data are means and standard deviations from triplicate experiments.

In analogy to a number of specificity studies on the viral proteases (26,39-43),we assayed the control fluorogenic peptide substrates POl and PO2 and 15 novel peptide substrates based on the cleavage sites of EV71 polyprotein.The substrates POl and P02 exhibited inconsistent cleavage efficiency in the previous studies (23,26). Our data suggested that the substrate PO2 and its analogue PO4 [NMA-TSAVLQSGFRK(DNP)M]couldbecleavable but with a lower relatively enzymatic activity (Fig.2A).The binding affinityand cleavage efficiency for POl were higher than those for P02 for EV71 3C protease,and this result is consistent with the report by Cui et al. (23).More importantly, the novel peptide substrate P08 [NMA-IEALFQGPPK(DNP)FR],derived from the 2C-3A junction of the EV71 polyprotein,was determined to be the most eficient substrate among all tested peptides.It represented an approximately 8-fold-higher $k _ { \mathrm { { c a t } } } / K _ { m }$ than that for the control substrate POl (Fig.2B),indicating effective and fast separation of the P3 region from the P2 region of the viral polyprotein.Supportive evidence comes from the report byLu et al.(25),who demonstrated that the peptide IGNTIEALFQGPPKFR,corresponding to 2C-3A junction site of CVAl6,was most efficiently processed by CVA16 protease via the HPLC assay $\mathit { \Delta } ^ { \prime } V = 8 . 3 7 \pm 1 . 9 3 \mu \mathrm { M / m i n } ,$ ） It is also noteworthy that a single variant of the $\mathrm { { } ^ { P l { ' } } }$ -P5 residues within the peptide PO8 could lead to a decrease in cleavage efficiency.When the relatively large hydrophobic amino acid Phe at P2 position was replaced by the relatively small hydrophobic amino acid Ala or the basicamino acid Lys,Pl4 orP17 showed a very lowcleavage rate,for which the precise kinetic parameters $K _ { m }$ and $k _ { \mathrm { { c a t } } }$ cannot be determined.Despite the fact that Ser also exists in the natural cleavage sites,replacement ofGly at the $\mathrm { P l ^ { \prime } }$ position of P08 by Ser led to dramatic decrease in the cleavage efficiency. The variant study revealed that the P2 and $\mathrm { { } ^ { P l { ' } } }$ residues would allow the substrate Po8 to bind more tightly to EV7l protease.

Based on the enzymatic assay,we noted that the same peptide with different fluorescence-quenching groups (PO1 versus P03, P02 versus PO4,and PO8 versus P1O) showed different cleavage efficiencies.The most effcient substrate,PO8,and its analogue P10,with the same peptide sequence but different FRET pairs, showed significantly different kinetic constants $k _ { \mathrm { c a t } } / K _ { m }$ of $1 1 . 8 \pm$ 0.82 and $\bar { 2 } . 6 9 \pm 0 . 1 5 ~ \mathrm { m M } ^ { - 1 } \mathrm { m i n } ^ { - 1 }$ ,respectively. We suspected that the N-terminal group or C-terminal fluorescent/quencher groups favorably interactwith EV713C protease via van der Waals interactions.Further experiments are needed to confirm this hypothesis.

Molecular dynamics simulations were done to study the substrate-enzyme interactions. Similar to the previously determined binding mode in the substrate FAGLRQAVTQGFPTEL complexed with CVAl6 protease (25), the P6 to $\mathrm { P l ^ { \prime } }$ potions of substrates bind to the active site of EV71 3C protease.We observed that a conserved residue Arg39 provided a striking rotation toward to the substrate in the MD model of substrate Po8 and EV71 3C protease.This rotation afforded an increased interaction (3.4 A) between Arg39 and Phe at the P2 position of PO8.In contrast, Arg39 still maintained the original conformation in other MD modes of substrates without Phe at the P2 position.We believe that Arg39 may assist the process of cleavage of the substrate by the EV71 3C protease.

Finally,two reported compounds as references (compounds 1 and 2) and a novel compound(compound 3) were synthesized to validate the assay for the screening of EV7l protease inhibitors.In addition to the two reference compounds,the novel compound 3 was found to inhibit EV713C protease,with an $\mathrm { I C } _ { 5 0 }$ of $1 . 8 9 \pm 0 . 2 5$ $\mu \mathrm { M }$ .Moreover,the inhibitor suppressed the proliferation of EV71 in RD cells without cytotoxicity at the highest tested concentration $( \mathrm { E C } _ { 5 0 } = 4 . 5 4 \pm 0 . 5 1 \mu \mathrm { M }$ $\mathrm { C C } _ { 5 0 } { > } 1 0 0 \mu \mathrm { M }$ ,indicating that itcould be a leading compound for anti-EV71 drug discovery.

In conclusion,we describea systematic biologicalcharacterization of EV71 3C protease with fluorogenic peptide substrates.A fluorogenic peptide,PO8 [NMA-IEALFQGPPK(DNP)FR],with a high substrate specificity constant ( ${ \mathrm { \tilde { k } } _ { \mathrm { c a t } } } / { K _ { m } }$ of $1 1 . 8 \pm 0 . 8 2 \mathrm { m M } ^ { - 1 }$ $\mathrm { m i n } ^ { - 1 }$ )and which undergoes considerable proteolysis in the presence ofEV71 3C protease was identified.Po8 is an ideal substrate for biochemical characterization of EV71 3C protease.A highquality screening assay of EV71 3C protease inhibitors with a high $Z ^ { \prime }$ factor $\left( 0 . 8 7 \pm 0 . 0 5 \right)$ was established.A novel compound (compound 3) $( \mathrm { I C } _ { 5 0 } = 1 . 8 9 \pm 0 . 2 5 \mu \mathrm { M } )$ that effectively suppressed the proliferation ofEV71 (strain Fuyang)in RD cellswitha highly selective index $( \mathrm { E C } _ { 5 0 } = 4 . 5 4 \pm 0 . 5 1 ~ \mu \mathrm { M }$ $\mathrm { C C } _ { 5 0 } > 1 0 0 ~ \mu \mathrm { M }$ was discovered using this assay. The biochemical assay was further validated by the determination of low $\mathrm { I C } _ { 5 0 } s$ of three synthesized EV71 3C protease inhibitors.

# ACKNOWLEDGMENTS

This work was supported by the National Natural Science Foundation of China (grant no.21202087 and 81102374),the National Basic Research Program of China (973 program,grant no.2013CB911104),the Fundamental Research Funds for the Central Universities (grant no.65l24002), the Tianjin Science and Technology Program (grant no.13JCYBJC24300 and13JCQNJC131o0),the Specialized Research Fund for the Doctoral Program ofHigher Education Ministry of Education of China (grant no. 2120031120049),and the $^ { \mathfrak { a } } 1 1 1 ^ { \mathfrak { n } }$ Project of the Ministry of Education of China (project no.B06005).

# REFERENCES

1.Chan KP,Goh KT,Chong CY,Teo ES,Lau G,Ling AE.20O3.Epidemic hand,foot and mouth disease caused by human enterovirus 71,Singapore. EmergInfectDis 9:78-85.http://dx.doi.org/10.3201/eid1301.020112.   
2. Yang Y,Wang H,Gong E,DuJ, Zhao X,McNutt MA,Wang S, Zhong Y,Gao Z, Zheng J.20o9.Neuropathology in 2 cases of fatal enterovirus type 71 infection from a recent epidemic in the People's Republic of China:a histopathologic,immunohistochemical,and reverse transcription polymerase chain reaction study.Hum Pathol 40:1288-1295.http://dx .doi.org/10.1016/j.humpath.2009.01.015.   
3.Chen S-C,Chang H-L,Yan T-R,Cheng Y-T,Chen K-T.2007.An eight-year study of epidemiologic features of enterovirus 71 infection in Taiwan.AmJ Trop Med Hyg 77:188-191.   
4.ChuaK,ChuaB,Lee C,Chem Y,Ismail N,Kiyu A,Kumarasamy V.   
2007.Genetic diversity of enterovirus 7l isolated from cases of hand, foot and mouth disease in the 1997,2000 and 2005 outbreaks,Peninsular Malaysia.Malays JPathol 29:69-78.   
5.Chen C,WangY,Shan C,Sun Y,Xu P,Zhou H,Yang C,Shi P-Y,Rao Z,Zhang B.20l3.Crystal structure of enterovirus 71 RNA-dependent RNA polymerase complexed with its protein primer VPg: implication for a trans mechanism of VPg uridylylation.JVirol 87:5755-5768.http://dx .doi.org/10.1128/JVI.02733-12.   
6.Ming Z,WangW,XieY,DingP,ChenY,JinD,SunY,XiaB,YanL, Lou Z.2014. Crystal structure of the novel di-nucleotide cyclase from Vibrio cholerae (DncV) responsible for synthesizing a hybrid cyclic GMPAMP.Cell Res 24:1270-1273.http://dx.doi.org/10.1038/cr.2014.123.   
7.Sun Y,Guo Y,Lou Z.2014.Formation and working mechanism of the picornavirus VPg uridylylation complex. Curr Opin Virol 9:24-30.http: //dx.doi.org/10.1016/j.coviro.2014.09.003.   
8.Gong P,Kortus MG,Nix JC,Davis RE,Peersen OB.2O13.Structures of coxsackievirus,rhinovirus,and poliovirus polymerase elongation complexes solved by engineering RNA mediated crystal contacts.PLoS One   
8:e60272. http://dx.doi.org/10.1371/journal.pone.0060272.   
9.McMinn PC.2002.An overview of the evolution of enterovirus 7l and its clinical and public health significance.FEMS Microbiol Rev 26:91-107. http://dx.doi.org/10.1111/j.1574-6976.2002.tb00601.x.   
10.Wu Y,Lou Z,Miao Y,Yu Y,Dong H,Peng W,Bartlam M, Li X, Rao Z.2010.Structures ofEV71 RNA-dependent RNA polymerase in complex with substrate and analogue provide a drug target against the hand-footand-mouth disease pandemic in China. Protein Cell1:491-50o. http://dx .doi.org/10.1007/s13238-010-0061-7.   
11. QingJ, Wang Y,Sun Y,Huang J, Yan W, Wang J,Su D,NiC,Li J, Ra Z. 2014. Cyclophilin a associates with enterovirus-71 virus capsid and plays an essential role in viral infection as an uncoating regulator. PLoS Pathol 10:e1004422. http://dx.doi.0rg/10.1371/journal.ppat.1004422.   
12.Hwang Y-C,Chen W,Yates MV.2006.Use of fluorescence resonance energy transfer for rapid detection of enteroviral infection in vivo. Appl Environ Microbiol 72:3710-3715.http://dx.doi.0rg/10.1128/AEM.72.5 .3710-3715.2006.   
13.Bishop NE,Anderson DA.1993.RNA-dependent cleavage of VP0 capsid protein in provirions ofhepatitis A virus.Virology 197:616-623.http://dx .doi.org/10.1006/viro.1993.1636.   
14. Marcotte $\scriptstyle { \mathrm { ~ L L } } ,$ Wass AB,Gohara DW,Pathak HB,Arnold JJ,Filman DJ, Cameron CE,Hogle JM.2007. Crystal structure of poliovirus 3CD protein: virally encoded protease and precursor to the RNA-dependent RNA polymerase. J Virol 81:3583-3596.http://dx.doi.org/10.1128/JV1.02306-06.   
15.Matthews DA,Smith WW,Ferre RA,CondonB,BudahaziG,Slsson W, Villafranca J, Janson CA,McElroy H,Gribskov C.1994. Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide fold, RNA-binding site,and means for cleaving precursor polyprotein. Cell 77:761-771. http://dx.doi.0rg/10.1016/0092-8674(94)90059-0.   
16.Sun Y,Wang Y, Shan C,Chen C,XuP,Song M, Zhou H, Yang C,Xu W, Shi P-Y.2012.Enterovirus 71 VPg uridylation uses a two-molecular mechanism of 3D polymerase.J Virol 86:13662-13671. http://dx.doi.0rg /10.1128/JVI.01712-12.   
17.Weng K-F,Li M-L,Hung C-T,Shih S-R.2009.Enterovirus 71 3C protease cleaves anovel target CstF-64 and inhibits cellular polyadenylation.PLoS Pathol 5:e1000593. htp://dx.doi.org/10.1371/journal.ppat.1000593.   
18. Clark ME,Hammerle T,Wimmer E,Dasgupta A.1991. Poliovirus proteinase 3C converts an active form of transcription factor IIIC to an inactive form: a mechanism for inhibition of host cell polymerase III transcription by poliovirus.EMBO J10:2941.   
19.Falk M, Grigera P,Bergmann I, Zibert A,Multhaup G, Beck E.1990. Foot-and-mouth disease virus protease 3C induces specific proteolytic cleavage of host cell histone H3.JVirol 64:748-756.   
20.Weidman MK,Yalamanchili P,Ng B,Tsai W,Dasgupta A.2001. Poliovirus 3C protease-mediated degradation of transcriptional activator p53 requires a cellular activity. Virology 291:260-271. http://dx.doi.org /10.1006/viro.2001.1215.   
21.Yalamanchili P,Datta U,Dasgupta A.1997. Inhibition of host cell transcription by poliovirus: cleavage of transcription factor CREB by poliovirus-encoded protease 3Cpro.J Virol 71:1220-1226.   
22.Shang L,Xu M,Yin Z.2013.Antiviral drug discovery for the treatment of enterovirus 71 infections. Antiviral Res 97:183-194. http://dx.doi.org/10 .1016/j.antiviral.2012.12.005.   
23. Cui S,Wang J,Fan T, Qin B,Guo L,Lei X, Wang J, Wang M, Jin Q. 2011.Crystal structure of human enterovirus 71 3C protease.JMol Biol 408:449-461.http://dx.doi.org/10.1016/j.jmb.2011.03.007.   
24.WangJ, Fan T, Yao X,Wu Z,Guo L,Lei X,Wang J,Wang M, Jin Q, Cui S.2011. Crystal structures of enterovirus 71 3C protease complexed with rupintrivir reveal the roles of catalytically important residues.J Virol 85:10021-10030.http://dx.doi.0rg/10.1128/JV1.05107-11.   
25.Lu G,Qi J, Chen Z,Xu X,Gao F,Lin D,Qian W, Liu H,Jiang H, Yan J. 2011. Enterovirus 71 and coxsackievirus A16 3C proteases: binding to rupintrivir and their substrates and anti-hand,foot,and mouth disease virus drug design. J Virol 85:10319-10331.http://dx.doi.org/10.1128/JVI .00787-11.   
26.Kuo C-J,Shie J-J, Fang J-M, Yen G-R,Hsu JT-A, Liu H-G, Tseng S-N, Chang S-C,Lee C-Y,Shih S-R.20o8.Design,synthesis,and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents. Bioorg Med Chem 16:7388-7398. http://dx.doi.0rg/10.1016/j.bmc.2008.06.015.   
27.Haugland RP,Yguerabide J,Stryer L.1969.Dependence of the kinetics of singlet-singlet energy transfer on spectral overlap.Proc Natl Acad Sci U S A 63:23-30. htp://x.doi.org/10.1073/pnas.63.1.23.   
28.Li B,Wang Q,Pan X,de Castro IF,Sun Y,Guo Y,Tao X,Risco C, Sui S-F,Lou Z.2ol3.Bunyamwera virus possesses a distinct nucleocapsid protein to facilitate genome encapsidation.Proc Natl Acad Sci U S A   
110:9048-9053.http://dx.doi.org/10.1073/pnas.1222552110.   
29.Wu C, Cai Q,Chen C,LiN,PengX,CaiY,Yin K, Chen X, Wang X,Zang R.2013.Structures of enterovirus 71 3C proteinase (strain E2004l04-TW-CDC) and its complex with rupintrivir.Acta Crystallogr SectD Biol Crystallogr 69:866-   
871.http://dx.doi.0rg/10.1107/S0907444913002862.   
30. Guo Y,Wang W,Ji W, Deng M, Sun Y,Zhou H, Yang C,DengF, Wang H,Hu Z. 20i2. Crimean-Congo hemorrhagic fever virus nucleoprotein reveals endonuclease activity in bunyaviruses.Proc Natl Acad Sci U S A   
109:5046-5051. http://dx.doi.org/10.1073/pnas.1200808109.   
31.Hirata IY,Cezari MHS,Nakaie CR,Boschcov P,Ito AS,Juliano MA, Juliano L.1995.Internally quenched fluorogenic protease substrates: solid-phase synthesis and fluorescence spectroscopy of peptides containing ortho-aminobenzoyl/dinitrophenyl groups as donor-acceptor pairs.Lett Pept Sci 1:299-308.http://dx.doi.org/10.1007/BF00119771.   
32.Zhang J-H, Chung TD,Oldenburg KR.1999.A simple statistical parameter for use in evaluation and validation of high throughput screening assays.J Biomol Screen 4:67-73.htp://dx.doi.0rg/10.1177/108705719900400206.   
33.Klauda JB,Brooks BR.2008.CHARMM force field parameters for nitroalkanes and nitroarenes.J Chem Theor Comput 4:107-115. http://dx .doi.org/10.1021/ct700191v.   
34.Roche O, Kiyama R, Brooks CL. 2001.Ligand-protein database: linking protein-ligand complex structures to binding data.JMed Chem 44:3592-   
3598. http://dx.doi.0rg/10.1021/jm000467k.   
35.Jorgensen WL,Chandrasekhar J,Madura JD,Impey RW,Klein ML.   
1983.Comparison of simple potential functions for simulating liquid water. J Chem Phys 79:926-935.http://dx.doi.org/10.1063/1.445869.   
36.Pastor RW,Brooks BR,Szabo A.1988.An analysis of the accuracy of Langevin and molecular dynamics algorithms.Mol Phys 65:1409-1419. http://dx.doi.0rg/10.1080/00268978800101881.   
37.Shang L,Wang Y,Qing J,Shu B,Cao L, Lou Z,Gong P, Sun Y, Yin Z.   
2014.An adenosine nucleoside analogue NITDo08 inhibits EV71 proliferation.Antiviral Res 112:47-58. http://dx.doi.org/10.1016/j.antiviral .2014.10.009.   
38．Zhang L,Lawson $\mathrm { H L } ,$ Harish VC,Huff JD,Knovich MA,Owen J.2006. Creation of a recombinant peptide substrate for fluorescence resonance energy transfer-based protease assays.Anal Biochem 358:298-300. http: //dx.doi.0rg/10.1016/j.ab.2006.06.022.   
39.Gouvea I, Izidoro M, Judice W,Cezari M, Caliendo G, Santagada V, Dos Santos C,Queiroz M, Juliano M, Young P.2007. Substrate specificity of recombinant dengue 2 virus NS2B-NS3 protease: influence of natural and unnatural basic amino acids on hydrolysis of synthetic fluorescent substrates.Arch Biochem Biophys 457:187-196.http://dx.doi.org /10.1016/j.abb.2006.11.005.   
40.Lou Z, Sun Y,Rao Z. 2014. Current progress in antiviral strategies. Trends Pharmacol Sci 35:86-102. http://dx.doi.org/10.1016/j.tips.2013 .11.006.   
41. Shiryaev SA,Ratnikov BI, Aleshin AE, Kozlov IA, Nelson NA, Lebl M, Smith JW,Liddington RC,Strongin AY.2007.Switching the substrate specificity of the two-component NS2B-NS3 flavivirus proteinase by structure-based mutagenesis.J Virol 81:4501-4509. http://dx.doi.0rg/10 .1128/JVI.02719-06.   
42. Tsai M-T,Cheng Y-H,Liu Y-N,Liao N-C,Lu W-W,Kung S-H. 2009. Real-time monitoring ofhuman enterovirus (HEV)-infected cels and anti-HEV   
3C protease potency by fluorescence resonance energy transfer.Antimicrob Agents Chemother 53:748-755.http://dx.doi.0rg/10.1128/AAC.00841-08.   
43.Zhou H, Sun Y,Guo Y,Lou Z.2013. Structural perspective on the formation of ribonucleoprotein complex in negative-sense singlestranded RNA viruses. Trends Microbiol 21:475-484. htp:/dx.doi.org /10.1016/j.tim.2013.07.006.   
44.RyckaertJ-P,Ciccoti G,Berendsen HJC.1977. Numerical integrationof the Cartesian equations of motion of a system with constraints: molecular dynamics of $n$ -alkanes. J Comput Phys 23:327-341. http://dx.doi.org/10 .1016/0021-9991(77)90098-5.