# Phylogenetic and morphological profile of Cladophora fracta (Cladophorophyceae, Chlorophyta) from karst spings, in northern China

HU Bianfang1, JILi²\*, CHEN Le ³,FENG Jia 4, SHI Shengli4

(1.Departmentofiology,Jinzhong University,JinzhongO3o6o,Shanxi,China;2.CollgeofEnvironmentand Safety,Taiyuan UniversityofScienceand Technology,Taiyuan O3oo24,China;3.SchoolofPharmaceutical   
Science,Shanxi MedicalUniversityTaiyuan O3oo1,China;4.choolofLifeScience,ShanxiUniversityTaiuan 030006,China)

Abstract: Cladophora fracta,a filamentous green macroalgal epiphyte on rhodoliths, is described from five karst springs in North China. Although Cladophora species frequently appear in karst system, their genetic diversity, biogeographical affnities and physiological properties have not been well investigated in these environments. The specific objectives of this study were to: 1) describe the habitat of the Cladophora-like algae form the five karst springs; 2)identify the thallus to species level based on a combination of morphological characteristics and molecular sequence; and 3) explore the morphological influence of habitat.To elucidate the biogeographical paterns in Cladophora， both morphological and molecular evidence were compared of Cladophora specimens across five study sites. Analyses of partial small subunit (SSU) and large subunit (LSU) genes revealed that the studied 5O Cladophora specimens were genetically identical species and a total of 13 ribotypes were detected. The molecular sequencing results indicated that the examined species was highly homologous with Cladophora vagabunda， though they shared few morphological features. The genus didn't form a monophyletic clade but in three different clades both in SSU and LSU trees. The microscopic structure was more consistent with that of $C .$ fracta. The Cladophora from the five karst springs did not show significant variation in cell dimensions. However, the species exhibited larger cell diameters than those reported from lakes. In addition, the rhizoid-like branches are only observed in two locations (XA and ST). Considering the morphological characteristics,we therefore hold our species as Cladophorafracta.

Key words: Cladophora, karst spring, phylogeny, green algae, Ribosomal DNA, Cladophoraceae

# 我国北方岩溶泉域刚毛藻的系统发育及形态学研究

胡变芳¹，吉莉²\*，陈乐³，冯佳4，史胜利4（1.晋中学院生物科学与技术学院，山西 晋中030600；2.太原科技大学环境与安全学院，太原030024;3．山西医科大学药学院，太原030001；4．山西大学生命科学学院，太原030006)

摘要：脆弱刚毛藻(Cladophora fracta)是一种大型丝状绿藻，生境分布广泛，然而对于岩溶泉域分布的刚毛藻研究较少，它们的遗传多样性、生物地理亲缘性和生理特性都有待深入研究。本论文对我国北方地区五个典型的岩溶泉域的 50个脆弱刚毛藻样本进行了形态和分子系统学描述。文章的研究目标是：1）对我国北方地区五个典型岩溶泉的刚毛藻生境进行描述；2）根据形态学特征和分子序列对藻体进行鉴定；3）探究生境对藻体生理特性的影响。基于 SSU和LSU序列的结果显示，所分析的50株刚毛藻个体为同一种，并发现了13个不同的核糖体基因型。同时，基于SSU和LSU的系统发育树，刚毛藻属均未能形成单系分支，分布在三个不同的分支上。13个样本基因型在 SSU和LSU树中的位置相似，与Cladophora vagabunda有很高的序列同源性，但是形态特征却差异很大。从显微结构结果看，五个岩溶泉域采集到的刚毛藻在细胞直径上无显著差异，藻体的形态特征与脆弱刚毛藻相一致。但是，岩溶泉域采集的藻体细胞直径比文献报道中采集在湖泊和河流中的脆弱刚毛藻直径要大。另外，仅有两个地点（XA和 ST）采集的标本中发现有假根状分枝。因此，基于形态学和分子序列的结果，将这五个泉域的刚毛藻鉴定为脆弱刚毛藻Cladophorafracta。

关键词：刚毛藻属，岩溶泉，系统发育，绿藻，rDNA，刚毛藻科 中图分类号：Q949.2 文献标志码：A

Cladophora is a large and common green macroalgal genus belonging to the Cladophoraceae in the Cladophorales,whose members consist of branched or unbranched uniseriate filaments,and absence of akinete. Species of the green algal genus Cladophora (Cladophorales, Chlorophyta) are widely distributed from marine to freshwater habitats worldwide (Leliaert et al,2Oo9) and extend into cold temperature and polar waters (Boedeker et al, 2O12).Most species grow attached to rocky substrate by rhizoidal cells,but they also can form extensive free-floating masses in eutrophic water.

The taxonomy of Cladophora species has been problematic because of the wide degree of morphological variation in response of different environmental conditons (Hayakawa et al., 2012). The Cladophorales is polyphyletic with representatives in three main clades (Siphonocladus clade, Cladophora clade and Aegagropila clade) (Hanyuda et al, 2OO2; Leliaert et al, 2Oo3; Yoshii et al, 2004),and the fourth lineage (Okellyaceae including only one species） has been proposed as a sister to the three main clades (Leliaert et al, 2009；Boedeker et al, 2012).More recently, molecular phylogenetic studies based on SSU and LSU rDNA sequences have provided the insight into the relationships within the Cladophorales (Hanyuda et al, 2OO2; Leliaert et al, 2003, 2007; Boedeker et al, 2O12; Ichihara et al, 2O13). However, the systematic relationship of genera remain poorly understood. Molecular phylogeny revealed that the traditional family and genus level classifications did not reflect the phylogenetic relationships (Leliaert et al,2007,2009).The Cladophora species are distributed in all three lineages rather than form a monophyletic group.

Genus Cladophora contains a heterogeneous group of species that are very hard to tell apart and classify, mainly because of the high morphological plasticity and cryptic diversity (Gestinari et al, 201O). Cladophora is one of the most species-rich genera of green macroalgae and the species of the genus are morphologically highly variable (van den Hoek & Chihara, 2OOO). It is also difficult to define stable taxonomic characteristics,since these characters are influenced by habitat,age and environmental conditions. However, studies on the habitat and ecology of genus Cladophora remain rare, especially about those in karst springs.

In this study, we colected Cladophora-like filamentous green algae from five karst springs in northern China. We performed molecular phylogenetic analyses based on nuclear-encoded small subunit (SSU), partial large subunit (LSU) sequences and the combined SSU and LSU sequences, as well as morphological observation to reveal the diversity of these filamentous green algae from karst springs. The specific objectives of this study were to: 1） describe the habitat of the Cladophora-like algae form the five karst springs; 2) identify the thallus to species level based on a combination of morphological characteristics and molecular sequence; and 3） explore the morphological influence of habitat.

# 1 Materials and methods

# 1.1Plantmaterials

A field survey was conducted at five typical karst springs in Shanxi Province, China in June and July 2O15 to identify the habitat of Cladophora. Figure 1 showed the sampling locations of Cladophora in Shanxi province. Water temperature and pH were measured on site with a pH/EC/TDS/Temperature Tester (HI98129,HHANNA Instruments Inc.， Italia). Specimens of Cladophora were collected from the five streams and Table 1 gave detail information on the sampling sites. Voucher species were deposited at Herbarium of Jinzhong University (JZU) and the voucher information was also given in Table 1. Specimens used for the morphological studies were preserved in freshwater containing $4 \%$ formalin or kept alive in freshwater, and those used for the molecular studies were frozen at $- 2 0 \mathrm { ~ } ^ { \circ } \mathrm { C }$ . External features of thalli were observed under an Olympus BX51 bright field microscope (Olympus Co., Tokyo, Japan).

Table 1 Collection information and GenBank accession numbers of SSU and LSU sequences for Cladophora fracta populations   

<html><body><table><tr><td rowspan="2">Code</td><td rowspan="2">Collection information</td><td rowspan="2">Vocher</td><td colspan="2">Accession No.</td><td rowspan="2">No.of</td></tr><tr><td>SSU</td><td>LSU</td></tr><tr><td>XA</td><td>Xin'an Spring,Lucheng City, Shanxi</td><td>JZU150702</td><td>KX421221</td><td>KX421233</td><td>haplotypes 2</td></tr><tr><td></td><td>Province,North China (36.27 N,</td><td>JZU150706</td><td>KX421222</td><td>KX421234</td><td></td></tr><tr><td rowspan="5">SQW</td><td>113.26 E).Water temp.15-16 C,pH 7.15-7.7.TDs 463 mg·L1. Coll.Hu BF.6</td><td></td><td></td><td></td><td></td></tr><tr><td>July. 2015 Shuiquanwan Spring,Xinzhou City, Shanxi</td><td>JZU150601</td><td>KX421215</td><td>KX421228</td><td>4</td></tr><tr><td>Province, North China (38.51 N,</td><td>JZU150603</td><td>KX421216</td><td>KX421229</td><td></td></tr><tr><td>113.29 E).Water temp.10.4-15.6 C,pH</td><td>JZU150604</td><td>KX421217</td><td>KX421230</td><td></td></tr><tr><td>6.3-7.6. TDs 395 mg·L-1. Coll. Shi SL.21</td><td>JZU150608</td><td>KX421218</td><td></td><td></td></tr><tr><td rowspan="4">ST</td><td>June. 2015</td><td></td><td></td><td></td><td></td></tr><tr><td>Shen'tou Spring, Shuozhou City, Shanxi Province, North China (39.39 N,</td><td>JZU150716</td><td>KX421219 KX421220</td><td>KX421231</td><td>2</td></tr><tr><td>112.61 E). Water temp.14.6-17 C,pH</td><td>JZU150719</td><td></td><td>KX421232</td><td></td></tr><tr><td>7.3. TDs 485 mg · L1. Coll.Shi S L.3 July.</td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="4">JC</td><td>2015 Source of Nanlao Spring, Jinci Temple,</td><td></td><td></td><td></td><td></td></tr><tr><td>Taiyuan City, Shanxi Province,North</td><td>JZU150722 JZU150723</td><td>KX421210 KX421211</td><td>KX421223</td><td>3</td></tr><tr><td>China(37.42 N,112.26 E). Water temp.</td><td>JZU150727</td><td>KX421212</td><td>KX421224 KX421225</td><td></td></tr><tr><td>11.6-16.2 ℃,pH6.5-7.3.TDs 552</td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="5">LC</td><td>mg·L−1. Coll. Hu B F.25 July. 2015</td><td></td><td></td><td></td><td></td></tr><tr><td>Longci Spring,Linfen City, Shanxi</td><td>JZU150736</td><td>KX421213</td><td>KX421226</td><td>2</td></tr><tr><td>Province,North China (36.08 N,</td><td>JZU150740</td><td>KX421214</td><td>KX421227</td><td></td></tr><tr><td>111.38 E).Water temp.15.1-18.4 C,pH</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>6.5-7.2. TDs 506 mg·L1. Coll. Shi S L.5</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>July. 2015</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr></table></body></html>

![](images/71ebe5abc5ce38f30c9bd234e03699077f0ebca3e7e96060e7f9a52ca9b82f60.jpg)  
Note: Location abbreviations were as given in Table 1.   
Fig.1 Map of Shanxi Province showing the sampling locations of Cladophora fracta

# 1.2DNA methods

Specimens were frozen in liquid nitrogen until use and the total DNA was extracted with Aqua-SPIN Plant gDNA Isolation Mini Kit (Watson Biotechnologies， Inc） following the manufacturer's instructions. Molecular phylogenetic analyses were carred out based on nuclear-encoded small subunit (SSU） and partial large subunit (LSU)，and both genes were combined in partitioned alignment. The primer pairs used for amplifying SSU were based upon Teng (2011) (18S rDNAF: 5'-AAT GGC TCG GTA AAT CAG TT-3’ and 18S rDNAR: $5 ^ { \circ }$ -AGT TGA TGA CTC GCG CTT AC-3’). LSU primers were based upon Leliaert et al (2007) (C1: $5 ^ { \circ }$ -ACC CGC TGA ATT TAA GCA TATC-3' and D2: 5'-TCC GTG TTT CAA GAC GG-3'). The standard polymerase chain reaction (PCR) analyses were carried out on a thermocycler (MyCycler Thermal, BIO-RAD, USA). For SSU sequences, PCR amplification was carried out as follows: initial denaturation at $9 5 \mathrm { ~ \ } ^ { \circ } \mathrm { C }$ for 5 min,35 cycles at $9 5 \mathrm { ~ \ } ^ { \circ } \mathrm { C }$ for 30 s, 56.9 $\mathrm { { ^ \circ C } }$ for 30 S and $7 2 \mathrm { ~ } ^ { \circ } \mathrm { ~ C ~ }$ for $1 \mathrm { m i n } 3 0 \mathrm { s }$ ,and a final extension at $7 2 \mathrm { ~ \textdegree C }$ for 7 min. For LSU, PCR amplification was carried out as follows: initial denaturation at $9 5 \mathrm { ~ \ ^ ~ { ~ \circ ~ } C ~ }$ for $5 ~ \mathrm { { m i n } }$ , 35 cycles at $9 5 \mathrm { ~ \ ^ ~ { ~ \circ ~ } C ~ }$ for 30s,63.8 ℃ for $3 0 ~ \mathrm { s }$ and $7 2 \mathrm { ~ \textdegree C }$ for $3 0 ~ \mathrm { s }$ ,and a final extension at $7 2 ^ { \circ } \mathrm { C }$ for 7 min. PCR product was purified with Gel Extraction Mini Kit (Watson Biotechnologies，Inc） according to the manufacturer's recommendation for direct sequencing. The PCR products were commercially sequenced by Shanghai Personal Biotechnology Co,Ltd.Additonal taxa from which obtained from Genbank were shown on Figure 2.

# 1.3 Sequence analyses

The Clustal-X 2.O software was used to align the sequences (Thompson et al, 1997),and the program jModeltest (Posada, 2O08; Guindon & Gascuel, 2003) was used to determine parameters for the maximum likelihood and Bayesian analyses.For the SSU,the model was as follows: $\mathrm { G T R + I + G }$ distance model,portion of invariable sites $= 0 . 5 9 2 \ 0$ ； gamma distribution $= 0 . 4 8 6 \mathrm { ~ 0 ~ }$ base frequencies $\mathrm { A } = 0 . 2 4 6 7 , \mathrm { C } = 0 . 2 2 1 0 , \mathrm { G } = 0 . 2 8 6 9 , \mathrm { T } = 0 . 2 4 5 5$ ; and rate matrix $\mathbf { A - C } = 0 . 7 8 3$ 2, $\operatorname { A - G } = 2 . 4 3 4 ~ 9$ $, \mathrm { A } \mathrm { - } \mathrm { T } = 2 . 0 9 6 ~ 1 , \mathrm { C } \mathrm { - } \mathrm { G } = 0 . 6 1 3 ~ 3 , \mathrm { C } \mathrm { - } \mathrm { T } = 6 . 1 0 1 ~ 0 .$ For the LSU gene,the model was as follows: $\mathrm { G T R } + \mathrm { G }$ distance model, gamma distribution $= 0 . 3 7 8 \mathrm { ~ 0 ~ }$ ；base frequencies $\mathbf { A } =$ （204号 $0 . 2 0 8 ~ 6 , \mathrm { C } = 0 . 2 8 4 ~ 6 , \mathrm { G } = 0 . 3 0 5 ~ 6 , \mathrm { T } = 0 . 2 0 1 ~ 2 ;$ and rate matrix $\mathrm { A { \mathrm { - } C } } = 0 . 8 4 2 \ 7$ ， $\mathrm { A \mathrm { - } G } = 2 . 3 4 7 \ 0 .$ $\mathrm { A { - } T = 1 } . 5 6 9 \ 8 , \mathrm { C { - } G = 0 . 4 7 8 \ 6 , C { - } T = 4 . 4 0 4 \ 0 . }$ For the combination of the genes,the model was as follows: for the SSU partition portion, the model was as follows: $\mathbf { G } \mathrm { T } \mathbf { R } { + } \mathbf { I } { + } \mathbf { G }$ distance model, portion of invariable sites $= 0 . 5 9 2 \ 0$ ; gamma distribution $= 0 . 4 8 6 \ : 0$ ; base frequencies $\mathbf { A } = 0 . 2 4 6 7$ $\mathbf { C } = 0 . 2 2 1 \ 0 , \mathbf { G } = 0 . 2 8 6 \ 9 , \mathbf { T } = 0 . 2 4 5 \ 5$ ；and rate matrix $\mathrm { A - C } = 0 . 7 8 3 \ 2 , \mathrm { A - G } = 2 . 4 3 4 \ 9 , \mathrm { A - T } =$ $2 . 0 9 6 \ 1 , \mathrm { C \mathrm { - } G } = 0 . 6 1 3 \ 3 , \mathrm { C \mathrm { - } T } = 6 . 1 0 1 \ 0 .$ For the LSU partition, the model was as follows: GTR $^ +$ G distance model, gamma distribution $= 0 . 3 9 4 \ 0$ ； base frequencies $\mathrm { A } = 0 . 2 1 1 ~ 5 , \mathrm { C } = 0 . 2 8 5 ~ 0 , \mathrm { G } =$ 0.299 5, $\mathrm { T } = 0 . 2 0 3 9$ ; and rate matrix A $\mathtt { i - C } = 0 . 9 0 0 \ 2$ $, \mathrm { A } { - } \mathrm { G } = 2 . 9 1 7 ~ 3 , \mathrm { A } { - } \mathrm { T } = 1 . 8 5 6 ~ 0 , \mathrm { C } { - } \mathrm { G } = 0 . 5 6 9$ 4, $\mathrm { C - T } = 5 . 4 3 7 \ 1 .$ ，The Maximum Likelihood (ML） analyses were conducted using PhyML 3.0 (Guindon & Gascuel 2Oo3). The robustness of trees obtained from ML analyses was estimated using bootstrap resampling with 1OoO replicates (Felsenstein,1985). The Bayesian analyses of the combined data were conducted using MrBayes version 3.1.2 (Huelsenbeck et al,1996; Ronquist &

Huelsenbeck, 2003).A MCMCMC (Metropolis-coupled Markov chain Monte Carlo） algorithm running four Markov chains simultaneously was used to estimate the posterior probability of the phylogenetic trees. The Markov chains were started from a random tree and run for 1O 000 000 generations sampling every 1 OOO generations for a total of 10 OOO samples each run. The first 2 500 samples from each run were discarded as burn-in. The consensus tree was reconstructed after burn-in of $2 5 \%$ generations.

# 2 Results

# 2.1 Habitat

Karst spring, part of a karst system, is ususlly the end of a cave system at the place where a river cave reaches the Earth's surface. Shanxi Province is the most abundant and typical region of karst system in northern China, where the exposed karst region was $2 . 6 { \times } 1 0 ^ { 4 } ~ \mathrm { k m } ^ { 2 }$ ，occupying nearly $1 7 . 5 \%$ of the total area of the whole province (Han et al, 1993). There were 18 large karst springs with the average discharge being more than $1 . 0 \mathrm { m } ^ { 3 } / \mathrm { s }$ , including the five investigated karst springs. Water temperature of the five streams ranged from 10.4 to $1 8 . 6 ^ { \circ } \mathrm { C }$ in July (except that for SQW was in June). As expected the water temperature showed some seasonal and geographical fluctuations.The $\mathsf { p H }$ of the five collecting sites was more consistent over the sampling period, ranging from 6.3 to 7.7. Total dissolved solids (TDS） showed not much fluctuation over the sampling sites, varying from a minimum of $3 9 5 ~ \mathrm { m g / L }$ at SQW where the water temperature was the lowest to a maximum of $5 5 2 \mathrm { m g / L }$ at JC where the water temperature was the highest.

# 2.2 Morphological observations

The morphological analyses were based on three specimens of each location.Plants form bright green, clustered, floating or mostly attached to the rock substrates,and measured at about 10cm in height. Thalli are composed of irregularly branched,uniseriate filaments.,. Cells are almost pear shaped with irregular swelling. The main axes of the cells were about 250.00-466.70 $\mu \mathrm { m }$ in length,and the length/width ratio was highly variable,in the range of 2.63-6.99 in regions with few cell divisions. The apical cells were about $3 7 8 . 6 4 \mathrm { - } 9 6 6 . 6 7 ~ \mu \mathrm { m }$ in length，and the length/width ratio was also highly variable,in the range of 3.84-13.33.The rhizoid-like branches were only observed in XA and ST. Although there were morphological variances in the samples from five sites, when we carefully checked the diagnostic characters and compared to the previous

reports (Hu & Wei,2OO6), the other features were common in all individuals detected in this study.   
Therefore, the samples were identified as Cladophora fracta.

Table 2 Morphological comparison of cells from samples with Cladophora fracta   

<html><body><table><tr><td>Code</td><td colspan="2">Main axes cells (n=30)</td><td colspan="2">Apical cells(n=30)</td><td>Rhizoid-like branches(n=30)</td></tr><tr><td></td><td>Length(μm)</td><td>L/D ratio</td><td>Length(um)</td><td>L/D ratio</td><td>Diameter (um)</td></tr><tr><td>XA</td><td>250.00-402.60</td><td>3.32-4.23</td><td>433.33-966.67</td><td>8.13-13.33</td><td>416.67-900.00</td></tr><tr><td></td><td>(306.67±36.76)</td><td>(3.91±0.32)</td><td>(757.99±214.24)</td><td>(11.03±1.75)</td><td>(619.52±179.77)</td></tr><tr><td>SQW</td><td>254.09-400.00</td><td>3.45-4.31</td><td>443.65-936.67</td><td>7.64-12.67</td><td></td></tr><tr><td></td><td>(302.58±38.66)</td><td>(4.02±0.37)</td><td>(727.67±197.36)</td><td>(9.69±1.23)</td><td></td></tr><tr><td>ST</td><td>315.00-406.00</td><td>3.61-4.71</td><td>475.00-860.00</td><td>6.33-9.59</td><td>75.00-130.00</td></tr><tr><td></td><td>(358.00±24.82)</td><td>(4.05±0.45)</td><td>(680.00±141.39)</td><td>(8.22±1.11)</td><td>(119±36.39)</td></tr><tr><td>JC</td><td>266.67-466.70</td><td>2.63-6.99</td><td>378.64-683.33</td><td>3.84-7.33</td><td></td></tr><tr><td></td><td>(367.96±51.99)</td><td>(4.61±1.08)</td><td>(580.67±87.19)</td><td>(6.16±0.99)</td><td></td></tr><tr><td>LC</td><td>255.40-405.70</td><td>3.67-4.25</td><td>425.00-897.67</td><td>8.04-13.00</td><td></td></tr><tr><td></td><td>(304.67±35.76)</td><td>(3.98±0.36)</td><td>(689.69±203.49)</td><td>(10.97±1.67)</td><td></td></tr><tr><td>C.oligoclona</td><td>90-330</td><td>2-6</td><td>300-400</td><td>1-10</td><td></td></tr><tr><td>C. fracta</td><td>60-360</td><td>1-3</td><td>60-240</td><td>3-6</td><td></td></tr><tr><td>C.vagabunda</td><td>80-140</td><td>4-12</td><td>20-60</td><td>4-6</td><td></td></tr></table></body></html>

# 2.3 Datasets and alignments

The SSU and partial LSU regions of the isolated material were deposited in to the GenBank and the accession numbers are given in Table 1. Fifty Cladophora specimens examined,a total of 13 ribotypes were detected (Table 1).The alignments of the two ribosomal genes, SSU and partial LSU, were 1 417 bp and 495 bp in length, respectively. The SSU fragment was approximately 3 times as long as the partial LSU region but it contained about the same number of variable and parsimony-informative characters (SSU: 322/144; LSU: 330/155). It seems that the pairwise sequence divergence in the SSU was significantly lower than in the LSU (Leliaert et al, 2007). The alignment of SSU sequences and partial LSU sequences was 1912 bp in length, including 282 parsimony-informative sites among 651 variable sites. The pairwise distances between studies haplotypes were less than $2 \%$ both in SSU and partial LSU sequences. No significant saturation was tested in either SSU or LSU regions,according to the $I _ { s s }$ statistic (Xia & Xie,2001).

# 2.4 Phylogenetic analyses

Phylogenetic analyses using BI for the combined SSU-partial LSU data (Fig.2) showed better performance in terms of resolution than the separate SSU and partial LSU analyses (trees not shown).The overalltrees had similar topology for the analyzed taxa, but difered in the placement of a few taxa. The Cladophora haplotypes occupied a separate well-supported (0.99/0.91) position in most analyses sister with Cladophora vagabunda. The placement of C. sterrocladi, Siphonocladus tropicus and Valonia aegagropila differed among different analyses. Without forming a monophyletic clade, the S. tropicus and $V .$ aegagropila clade positioned on the main Cladophora branch in the SSU trees (O.88) or sister with Pithophora clade in the LSU trees (0.82/0.78).

1 JC1KX421210 B JC1KX421223 JC3KX421212 ST2KX421232 JC2KX421211 JC3KX421225 XA2KX421222 LC2KX421227 LC1KX421213 JC2KX421224 epaeeroropela epaeerrdosera LC1KX421226 SQW3KX421217 SQW3KX421230 ST2KX421220 ST1KX421231 XA1KX421221 SQW1KX421228 LC2KX421214 XA1KX421234 SQW4KX421218 0.91/- SQW2KX421229 0.99/- SQW1KX421215 XA2KX421234 0.99/- SQW2KX421216 0.76/- CladophoravagabundaAj544760 0.81/860 ST1KX421219 CladophoraalbidaAM503433 0.96/- CladophoravagabundaAB062710 Rhizocloniumsp.AB807607 1.00/956 CladophoraalbidaZ35317 Cladophora rupestrisAJ544764 1.00/937 Rhizocloniumsp.AB259958 Cladophora feredayiFM205030 0.99/843 CladophorarupestrisZ35319 CladophoraradiosaFM205042   
1.00/801 CladophoramirabilisFM205050 adophorapygmaeaFM205051 CladophorapygmaeaFM205040 0.88/- Pithophoracf.polymorphaFR873096 SiphonocladustropicusAM498761 1.00/995 1.00/819 PithophoraroettleriFR719942 ValoniaaegagropilaAM498762 Aegagropila linnaeiEU655697 Pithophoracf.polymorphaFR873097 0.95/ 1.00/991 1.00/997 CladophorakosteraeFR719945 PithophoraroettleriFR719930 0.98/- CladophoraramulosaFR719948 1.00/- Cladophora sterrocladiFR873099 CladophorasterrocladiFR719940 1gagropilalinnaeiFR719925 uso4 CladophorakosteraeFR719933 ValoniaaegagropilaAJ544748 0.98/- 0.90/- CladophoraramulosaFR719935 ValoniopsispachynemaAJ544741 UlvalactucaAB425960 0.1 0.1

Cladophorales were downloaded from GenBank. Numbers at the nodes are BI/ML bootstrap values.Location

abbreviations were as given in Table 1.

Fig.2 Hypothesized phylogenetic trees based on Bayesian analysis for Cladophora fracta and relatives

# 3Discussion

Combining sequences from different data sets has long been questioned in phylogenetic analyses (Huelsenbeck et al, 1996; Leliaert et al, 2007),while the combination of multiple-gene data sets show better resolved and supported trees,compared with single-gene partitions (Leliaert et al,20O7; Boedeker et al, 2O12). In the present study, the combined ${ \mathrm { S S U } } +$ LSU data were found to be superior to the individual partition trees,on account of the SSU fragment contained less variable and parsimony-informative characters and as its length was approximately 3 times as long as the partial LSU region. The taxonomy of genus Cladophora is problematic (Leliaert et al, 2007;

Leliaert et al,2OO9 ). Phylogenetically, allthe ribotypes did not correspond with any previously described Cladophora fracta specimens,because we didn't get any available sequence data of Cladophora fracta from genbank. However, our molecular phylogenetic analysis showed a well supported Cladophora clade.

The inability to match our plants with a described taxon is confirmed by morphological data. Our phylogenetic analyses show that all the ribotypes formed a single clade sister with Cladophora vagabunda. However， C. vagabunda has distinct morphological features. For instances,the filament cells are cylindrical,with $8 0 - 1 4 0 ~ { \mu \mathrm { m } }$ in diameter and $4 { \cdot } 1 2 \ \mu \mathrm { m }$ in length; the branch lets taper to $4 0 \mu \mathrm { m }$ in diameter, slightly constricted at the junction with main axes; the apices are straight above,curved or sickle-shaped below; and the apical cells are $2 0 – 6 0 ~ \mu \mathrm { m }$ in diameter, and $5 { \mathrm { - } } 1 1 ~ { \mu \mathrm { m } }$ long (Russell & Balazs,2OoO), but it shares similar length/diameter ratios in the main axes cells with our samples. It was reported that $C$ .vagabunda shares morphological similarity with C. glomerata,while differing from each other on the cell diameter and cell wall thickness which are easily affected by the salinity regimes. C. vagabunda and C. glomerata also showed an incredibly close relationship in the phylogenetic trees (Hayakawa et al, 2O12),and both have been the most often mentioned species in eutrophic freshwaters (Whiton,1970). C. fracta is separated from $C$ ： glomerata by the typically more slender filaments and the more pronounced tendency of the akinetes to swellinto the pear-shaped structures. Considering the morphological characteristics,we therefore hold our species as Cladophora fracta.

Table 2 summarizes a morphological comparison between the analyzed taxa and similar Cladophora species. The cell sizes were similar among/within the analyzed ribotypes; however, the ST samples distinguished from the XA individuals by a thinner rhizoid-like branch (75-130 $\mu \mathrm { m } )$ ，while XA samples exhibited the largest length/diameter ratio in the apical cells. The rhizoid-like branches were not observed in the SQW, JC and LC specimens. Although Cladophora species frequently appear in karst springs，their genetic diversity， phenological patterns and physiological properties have not been well investigated in these environments.In this study, collections from the five karst springs showed a unique morphological characteristic. Our samples collected from the five karst springs did not exhibit significant variation in cell dimensions, but the cell diameters of the individuals from the five locations were larger than those reported for the cells from lakes (Whitton,197O),indicating that the habitat conditions may have an effect. A similar tendency was observed in cell length and cell wall thickness.Perhaps these differences may have been caused by regional differences. Sinha (1968) had concluded that the number of nucleoli and chromocentre like bodies can be correlated with the degree of polyploidy which makes species larger in size. Moreover, morphological plasticity is related to diverse habitats and different salinity levels (Nienhuis,1975；Hayakawa et al, 2012; Ichihara et al, 2O13). Some frameworks，however, still lack enough sequence data to support. It is clear that additional sampling of Cladophora species from different salinity levels as well as karst springs will be needed to further clarify the diversity and plasticity of the species.

# Acknowledgements

We thank Dr. Don Zhao (Professor and Elton Z.and Lois G. Huff Chair of Environmental Engineering,Auburn University,Auburn, USA） for his critical reviews of the manuscript and editorial assistance with the English. Also we want to acknowledge the anonymous reviewers that enhanced the quality of this manuscript.

# References

BORDEKER C,O'KELLYCJ,STAR W,etal,2012.Molecular phylogenyand taxonomyof the Aegagropila clade (Cladophorales, Ulvophyceae), including the description of Aegagropila gen,nov.and Pithophora gen. nov[J]. JPhycol, 48(3): 808-825.   
FELSENSTEIN J,1985. Confidence limits on phylogenies: an approach using the bootstrap[J].Evolution, 39(4): 783-791.   
GESTINARI LMDS,PEREIRA SMB,YONESHIGUE-VALENTIN Y,2010. Distribution of Cladophora Species (Cladophorales,Chlorophyta) along the Brazilian Coast[J].Phytotaxa,14(3): 22-42.   
GUINDON S,GASCUEL O,2003.A simple,fast and accurate algorithm to estimate large phylogenies by maximum likelihood[J]. Systematic Biol, 52(5): 696-704.   
HANYUDA T, WAKANAI, ARAI S,et al,2002.Phylogenetic relationships within Cladophorales (Ulvophyceae, Chlorophyta) inferred from 18S rRNA gene sequences,with special reference to Aegagropila linnaei[J]. J Phycol,38(3): 564-571.   
HAYAKAWA Y,OGAWA T, YOSHIKAWA S,etal,2012. Genetic and ecophysiological diversityof Cladophora (Cladophorales,Ulvophyceae) in various salinity regimes[J].Phycol Res,6O(2): 86-97.   
HAN XR,LU R A,LI QS,et al,1993. Karst system—research on large karst springs in Shanxi[M]. Beijing : Geological Publishing House.[韩行瑞，鲁荣安，李庆松，等,1993．岩溶水系统--山西岩溶大泉研究[M]. 北京：地质出版社.]   
HU HJ,WEI YX,2O06.The freshwater algae of China - systematics,taxonomy and ecology[M]. Beijing: Science Press:770.[胡鸿钧，魏印心,2006．中国淡水藻类：系统，分类及生态[M]．北京：科学出版社:770.]   
HUELSENBECK JP, BULL JJ, CUNNINGHAM CW,1996. Combining data in phylogenetic analysis[J]. Trends ECUI EVUI, 11(4): 1J∠-1J0.   
ICHIHARA K， SHIMADA S，MIYAJI K, 2013. Systematics of Rhizoclonium-like algae (Cladophorales, Chlorophyta) from Japanese brackish waters,based on molecular phylogenetic and morphological analyses[J]. Phycologia, 52(5): 398-410.   
LELIAERT F, BOEDEKER C，PENA V,et al， 2009.Cladophora rhodolithicola sp. nov.(Cladophorales, Chlorophyta),a diminutive species from European maerl beds[J]. Eur JPhycol, 44(2):55-169.   
LELIAERT F, ROUSSEAU F, DE REVIERS B,et al,2003. Phylogenyof the Cladophorophyceae (Chlorophyta) inferred from partial LSU rRNA gene sequences: is the recognition of a separate order Siphonocladales justified[J]? Eur JPhycol, 38(3): 233- 246.   
LELIAERT F, DECLERCK O,VERBRUGGEN H, et al, 2007. Molecular phylogeny of the Siphonocladales (Chlorophyta: Chlorophyceae)[J].Mol Phylogenet Evol, 44(3): 1237-1256.   
NIENHUIS PH,1975. Biosystematics and ecology of Rhizoclonium riparium (Roth） Harvey (Chlorophyceae: Cladophorales） in the estuarine area of the rivers Rhine,Meuse,and Scheldt[M].Bronder Offset B.Y., Rotterdam: 240.   
POSADAD,2008. jModelTest: Phylogenetic Model Averaging[J]. Mol Biol Evol,25(7): 1253-1256.   
RONQUISTF, HUELSENBECK JP, 2003.MRBAYES 3: Bayesian phylogenetic inference under mixed models[J]. Bioinformatics,19(12): 1572-1574.   
RUSSELL DJ,BALAZS GH,2OOo.Identification manual for dietary vegetation of the Hawaiian green turtle, Chelonia mydas[M]. NOAA TM-NMFS-SWFSC-294: 49.   
SINHA JP,1968.Cytotaxonomical studies on Cladophora glomerata,four freshwater forms[J].Int JCytol, 32(3-4): 507-518.   
TENG LH, 2011. Study on morphology and molecular phylogeny of Cladophorales (Chlorophyta) along China sea coast, with Its DNA Barcoding Based on ITS and 18S rDNA Sequences[D]. Beijing: Master Dissertation, Graduate University of Chinese Academy of Sciences: 17.   
THOMPSONJD, GIBSON TJ,PLEWNIAK F, et al,1997.The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools[J]. Nucleic Acids Res, 25(25): 4876-4882.   
VAN DEN HOEK C，CHIHARA M， 2000.A taxonomic revision of the marine species of Cladophora (Chlorophyta) along the coast of Japan and the Russan Far-East. National Science Museum Monographs, vol. 19[M]. Tokyo: National Science Museum:242.   
WHITTON BA,1970. Biology of Cladophora in freshwaters[J]. Water Res,4(7): 457-476.   
XIA X,XIE Z,2001.DAMBE: Software package for data analysis in molecular biology and evolution[J]. JHered, 30(7): 1720-1728.   
YOSHII Y，HANYUDA T，WAKANA I，et al，2004.Carotenoid compositions of the Cladophora balls (Aegagropila linnaei) and some members of the Cladophorales (Ulvophyceae, Chlorophyta): their taxonomic and evolutionary implication[J]. J Phycol, 40(6): 1170-1177.