# Genome-wide association analysis and differential expression analysis of resistance to Sclerotinia stem rot in Brassica napus

Lijuan Wei1,Hongju Jian',Kun Lu1, Fiona Filardo², Nengwen Yin1, Liezhao Liu1,Cunmin ${ \sf Q u } ^ { 1 }$ ,Wei $\mathsf { L i } ^ { 1 }$ ，Hai ${ \sf D u } ^ { 1 }$ and Jiana ${ \mathsf { L i } } ^ { 1 , \star }$

1ChongqingEgrinsearchnterfoaeedolofroodteholutsterstyi   
2Queensland Departmentof Agricultureand Fisheries (QDAF),Ecosciences Precinct,Brisbane,Old,Australia

Received29 March 2015;   
revised 8 October 2015;   
accepted 13 October2015.   
\*Correspondence (Tel $+ 8 6$ 23 68251950;   
fax $+ 8 6$ 23 68251950;   
email Ijn1950@swu.edu.cn)

# Summary

Keywords:association mapping, differential expression,Sclerotinia sclerotiorum,Brassica napus.

Brassica napus is one of the most important oil crops in the world,and stem rot caused by the fungus Sclerotinia sclerotiorum results in major losses in yield and quality.To elucidate resistance genes and pathogenesis-related genes,genome-wide association analysis of 347 accessons was performed using the Ilumina 6OK Brassica SNP (single nucleotide polymorphism) array. In addition,the detached stem inoculation assay was used to select five highly resistant (R)and susceptible (S) B.napus lines, $4 8 \mathrm { ~ h ~ }$ postinoculation with S.sclerotiorum for transcriptome sequencing.We identified 17 significant associations for stem resistance on chromosomes A8 and C6,five of which were on A8 and 12 on C6.The SNPs identified on A8 were located in a 409-kb haplotype block,and those on C6 were consistent with previous QTL mapping efforts. Transcriptome analysis suggested that S.sclerotiorum infection activates the immune system, sulphur metabolism,especially glutathione (GSH) and glucosinolates in both R and S genotypes. Genes found to be specific to the R genotype related to the jasmonic acid pathway,lignin biosynthesis,defence response,signal transduction and encoding transcription factors.Twentyfour genes were identified in both the SNP-trait association and transcriptome sequencing analyses,includinga tau class glutathione S-transferase(GSTU gene cluster.This study provides useful insight into the molecular mechanisms underlying the plant's response to S.sclerotiorum.

# Introduction

Sclerotinia sclerotiorum (Lib.) de Bary isa necrotrophic pathogen that has no specific host and infects more than 4oo different plant species,including sunflower (Hellianthus annuus),chickpea (Cicerarietinum),soybean (Glycine max) and rapeseed (Brassica napus) (Bolton et al.，2oo6).Brassica napus is one of the most economically important oil crops，and stem rot，caused by S.sclerotiorum,results in major losses in yield and quality (Purdy, 1979;Sharma et al.,2015).

Ascospores of S.sclerotiorum infect plants through senescing, old or injured plant tissue,lower stems and flower petals (Garg et al.,201ob).The mycelium then spreads throughout the stem and leaves (Jamaux et al.,1995) producing enzymes that degrade keratin and cell wall components and leading to a build-up in toxic substances， such as oxalic acid (Bolton et al.， 2006). Infection causes pale or dark lesions followed by white patches of fungus,which eventually leads to stem rot and the collapse of the plant.

When a plant and pathogen interact,a number of plant genes are involved in eliciting a response.Resistance (R) genes detect the pathogen attack and activate a signal transduction pathway that in turn triggers the transcription of a large number of defence response (DR) genes (Gururani et al.,2012; Singh et al., 2015).The expression patterns of defence genes in B. napus have mainly been studied using B.napus or Arabidopsis microarrays to compare resistant versus susceptible B.napus cultivars.Gene differential expression levels were found to peak $4 8 \mathrm { ~ h ~ }$ postinoculation，and differentially expressed genes included those involved in oxidative burst，molecular transport,signal transduction,cell maintenance and development;as well as biosynthesis genes for jasmonic acid，ethylene，auxin，glucosinolate and phenylpropanoid metabolism (Yang et al.，2oo7； Zhao et al., 2007，2009)． In addition，Garg et al. (2013)using protein profiling identified a number of B.napus proteins related to resistance to S.sc/erotiorum，such as pathogenesis-related proteinsand proteins involved in primary metabolism and ethylene biosynthesis.However，to date no study has investigated the defence mechanism of B.napususing an RNA sequencing (RNASeq)approach,which ismore sensitive and accurate，offers higher throughput and is more cost-effective than microarray analysis.RNA-Seq has emerged asa powerful tool to examine differences in global gene expression.

Resistance to S.sc/erotiorum showed quantitative inheritance withadditive effectsand medium heritability (Fusari et al.,2012). Early studies looking for resistance genes in B.napus were conducted through QTL mapping,and some pathogen-resistant QTLswere identified on chromosomesA1，A2，A3，A5,A6，A8, A9,C2,C4,C6,C7,C8 and C9 (Behla et al.,2009;Wei et al., 2014;Wu et al.,2013;Yin et al.,2010; Zhao and Meng,2003;

Zhao et al.,2oo6). However,due to the lack of common markers, overlapping QTLs were difficult to identify.With the recent release of the B. napus genome sequence (Chalhoub,2014),the physical positions occupied by QTLs can be identified by aligning the QTL primers with the B.napus genome. Li et al. (2015) integrated and compared QTLs for resistance to S.sclerotiorum from previous mapping efforts，and conserved QTLs were identified on chromosomes A9 (22.5-27.5 Mb) and C6 (29.5- 36.1 Mb)．Nevertheless，QTL mapping based on a biparental population is limited due to fewer recombination events.

With the development of high-throughput sequencing technology, genome-wide association study(GWAS) based on linkage disequilibrium (LD) has emerged as the main tool for gene mapping in plants.GWAS can take advantage of the phenotypic variation and historical recombination in natural populations without the need to construct a mapping population (Nordborg and Weigel,2oo8).In recent years,GWAS has been widely used to identify genes involved in complex traits in B.napus. Cai et al. (2014) identified 43 loci related to yield traits using 451 short sequence repeats (SSR)and 740 amplified fragment length polymorphism (AFLP) markers in192 B.napus inbred lines using the $\mathsf { Q } + \mathsf { K }$ model.Li et al. (2014a) found two loci associated with erucic acid content,four SNP clusters associated with glucosinolate content and two loci associated with seed weight using the Illumina $6 0 ~ \mathsf { K }$ Brassica SNP microarray and 472 B.napusaccessions through GWAS.Lu et al. (2014) identified 26 genes related to glucosinolate metabolism using a GWAS method termed 'associative transcriptomics'analysis.In addition,GWAS has been used to identify resistance genes in crops infected with stem rust (Yu et al.,2014; Zhang et al.,2014a), tan spot in wheat (Triticum aestivum; (Kollers et al.,2014),southern leaf blight (Kump et al., 2011),head smut in maize (Zea mays; (Wang et al.,2012a),blast disease in rice (Oryza sativa;(Wang et al.，2014a)and stem canker in B.napus (Fopa Fomeju et al.，2014).However,there are no reports of GWAS being used to study resistance to S.sclerotiorum in B.napus.

In this study，we carried out a GWAS of S.sclerotiorum resistance in the stems of 347 B.napus accessions genotyped with the $6 0 ~ \mathsf { K }$ Illumina Brassica SNP microarray and identified significant associated SNP loci. In addition，we performed transcriptome sequencing of resistant (R)and susceptible (S) accessions inoculated with S.sclerotiorum.Defence genes, expressed in both accessions，and those specific to the R accessions only were identified.Among the genes identified in both the GWAS and transcriptome analysis，a tau class glutathione S-transferase (GSTU) gene cluster was included,which Was verified by quantitative real-time PCR (qRT-PCR). Our results reveal candidate genes involved in stem rot resistance and provide aninsight into the genetic mechanisms of resistance to S.sc/erotiorum in B.napus.

# Results

# Phenotypicvariation

We evaluated stem resistance to S.sclerotiorum in 347 B. napus accessions during 2013 and 2014，with two replications performed each year. Extensive phenotypic variations were observed inrelative susceptibility (Table 1).Relative susceptibility was calculated according to the formula: the lesion length on the tested accession divided by the control,‘Zhongshuang $9 ^ { \prime }$ The relative susceptibility，stem resistance assessment (RA)，ranged from 0.58 to 1.36,with an average of 0.918,in 2013 and from

Table1 Phenotypic variation in Brassica napus   

<html><body><table><tr><td>Trait</td><td>Mean</td><td>SD*</td><td>Min</td><td>Max</td><td>Cv+(%)</td></tr><tr><td>2013RA</td><td>0.918</td><td>0.144</td><td>0.58</td><td>1.36</td><td>15.73</td></tr><tr><td>2014RA</td><td>0.917</td><td>0.144</td><td>0.59</td><td>1.44</td><td>15.71</td></tr></table></body></html>

\*Standard deviation. +Coefficient of variation.

0.59 to 1.44,with an average of 0.917,in 2014.ANOVA of stem RA revealed significant differences among the genotypes,but no significant differences between the data collected in the two years ${ ' } F = 0 . 0 2$ 1 $ { P } = 0 . 8 8 )$ (Table S3).Stem resistance assessment inB.napus showed continuous variation and approximated a normal distribution (Figure 1)，suggesting that resistance was controlled by multiple genes.The broad sense heritability of stem resistance was $84 \%$ ，which is consistent with the results of Mei et al. (2013) $( 7 8 \% )$ ：

# Relative kinship

SNPs from the $6 0 ~ \mathsf { K }$ Illumina Brassica SNP array were used as queries for BLASTn searches against the B.napus genome; 10 112 of the 52 157 SNPs were not mapped and were excluded.An additional 11 113 SNPs (with call frequencies of $< 9 0 \%$ orminor allele frequencies (MAFs)of $< 0 . 0 5$ ）were removed.Therefore，30 932 polymorphic SNPs were used for further analysis (Table S4).

We evaluated the relative kinships among the 347 accessions using SPAGeDi,and found that $8 0 . 9 \%$ of the kinship coefficients between lines were $< 0 . 0 5$ and $5 5 \%$ were equal to O (Figure S1). Furthermore, $1 4 . 6 \%$ of the coefficients were between O.05 and 0.15．Therefore，the kinship analysis results show that the majority of the accessions have a null or weak relationship with the other accessions in this population of B.napus.

# Genome-wide association analysis

Trait-SNP association analysis was performed using the $P + K$ model (Price et al.，20o6).The QQ plot displayed in Figure 2a shows that the model could be used to identify association signals.A total of 17 significant associations were detected on A8 and C6 at $P < 3 . 2 3 \times 1 0 ^ { - 5 }$ 0 $P = 1 / 3 0 9 3 2$ $- \mathsf { I o g } _ { 1 0 } P = 4 . 4 9 ;$ ,with a false discovery rate (FDR)of $1 . 2 \%$ and $0 . 8 \%$ (Figure 2b).The location and MAF of these 17 loci are listed in Table 2.In addition,one SNP was found on the C3-random chromosome, but this was not analysed further.

![](images/c4127f7d0a3955d2d248403b0f30dafefc9ac5b7bd1503255902df9e2eb87a28.jpg)  
Figure 1 Frequency distribution of stem resistance in Brassica napus. 2013RA and 2014RA refers to the stem resistance assessment (RA) evaluated in 2013 and 2014,respectively.The frequency represents the proportion of accessions.

![](images/6d035c0513878a113ae380e60c605d20b408f4180ea5c3c39613822764d40462.jpg)  
Figure 2 Manhatttan and quantile-quantile plots resulting from the GWAS results for stem resistance in Brassica napus.(a) Quantile-quantile plot for stem resistance;(b) Manhattan plot for stem resistance.The dashed horizontal line indicates the Bonferroni-adjusted significance threshold $( 3 . 2 3 \times 1 0 ^ { - 5 } )$ ：

Table 2 Markers with significant association for resistance to Sclerotiniasclerotiorum   

<html><body><table><tr><td>Chromosome</td><td>Position</td><td>SNP Allele</td><td>MAF</td><td>P value</td></tr><tr><td>A8</td><td>15094063</td><td>T/G</td><td>0.485</td><td>1.32E-05</td></tr><tr><td>A8</td><td>15095383</td><td>A/G</td><td>0.499</td><td>2.74E-05</td></tr><tr><td>A8</td><td>15096961</td><td>C/T</td><td>0.477</td><td>2.15E-05</td></tr><tr><td>A8</td><td>15101546</td><td>A/G</td><td>0.46</td><td>2.34E-05</td></tr><tr><td>A8</td><td>15101600</td><td>A/G</td><td>0.465</td><td>2.28E-05</td></tr><tr><td>C3_random</td><td>4568807</td><td>T/C</td><td>0.276</td><td>1.83E-05</td></tr><tr><td>C6</td><td>30784152</td><td>T/G</td><td>0.357</td><td>2.20E-05</td></tr><tr><td>C6</td><td>30791903</td><td>A/G</td><td>0.388</td><td>5.98E-06</td></tr><tr><td>C6</td><td>30811671</td><td>A/G</td><td>0.377</td><td>8.93E-06</td></tr><tr><td>C6</td><td>30812107</td><td>A/C</td><td>0.371</td><td>9.06E-06</td></tr><tr><td>C6</td><td>30812416</td><td>T/C</td><td>0.38</td><td>8.93E-06</td></tr><tr><td>C6</td><td>30812429</td><td>A/G</td><td>0.385</td><td>9.06E-06</td></tr><tr><td>C6</td><td>30813043</td><td>A/G</td><td>0.393</td><td>9.08E-06</td></tr><tr><td>C6</td><td>30813372</td><td>A/G</td><td>0.356</td><td>1.38E-05</td></tr><tr><td>C6</td><td>30813377</td><td>T/C</td><td>0.354</td><td>2.30E-05</td></tr><tr><td>C6</td><td>30813548</td><td>T/C</td><td>0.364</td><td>1.66E-05</td></tr><tr><td>C6</td><td>30815877</td><td>A/G</td><td>0.38</td><td>8.93E-06</td></tr><tr><td>C6</td><td>31344734</td><td>T/C</td><td>0.392</td><td>3.11E-05</td></tr></table></body></html>

Haplotype block structures were investigated for chromosomes A8 and C6.A total of 31 and 13 haplotype blocks were found on A8 and C6,respectively. Significant SNPs on A8 were located in a haplotype block of 409 kb (from 15.01 Mb to $1 5 . 4 2 \ \mathsf { M b }$ Figure 3).For C6,the region of significant association ranged from 30.78 Mb to 31.34 Mb,which was consistent with previous QTL mapping results (Figure 4) (Wu et al.，2013; Zhao et al., 2006).

![](images/577b2afcef1164d47b4c636beff0d825ca64e9885dbe99bd4da06e2fab1e663a.jpg)  
Figure 3 Genomic location of SNP loci associated with stem resistance (15.09-15.10 Mb) and pairwise LD estimates in A8 haplotype block (15.01-15.42 Mb).The red arrows denote the significantly associated SNPs located in the haplotype block.

# Prediction of stem resistance

The ability to predict stem resistance using genome-wide markers wasassessed.The predictive ability represented the correlation between the predicted and measured field data.We used 20- $90 \%$ of the training population and all of the 30 932 SNP markers，and found that the predictive ability and standard deviation increased with the size of the training population. Higher predictive ability and lower standard deviation were optimum; therefore, $60 \%$ of the training population was selected basedonboth predictive abilitiesand standarddeviation (Table S5).

When using $60 \%$ of the training population (145 accessions), andall SNP markers for the prediction,the predictive abilitywas only O.27,indicating that these makers were not suitable for stem resistance prediction.In addition，300 and 12oo randomly selected markers were also used and the predictive ability of bothsetsof markerswas low (0.17) (Table S5,Figure 5).Markers significantly associated with the trait could be detected in GWAS by setting a certain $P$ value,such as $P < 0 . 0 0 1$ 1 $P < 0 . 0 1$ and $P < 0 . 0 5$ (Zhang et al.，2014b).We then selected significant association markers identified in GWAS with a certain threshold for the prediction,including markers with $P < 0 . 0 0 1$ 1 $( 6 9 \mathsf { S N P S } )$ $P < 0 . 0 1$ (288),and $P < 0 . 0 5$ (1203),and the predictive abilities were 0.58,O.77and O.80,respectively.Therefore,locidetected using GWAS improved the predictive ability of stem resistance.

# Transcriptome sequencing

RNA from five R and five S genotypes was pooled $4 8 \mathrm { ~ h ~ }$ postinoculation to generate $\mathsf { R } _ { 4 8 }$ and $\mathsf { S } _ { 4 8 }$ ：while mock-inoculated B.napus was dubbed ${ \sf R } _ { 0 }$ and $\mathsf { S } _ { 0 }$ .A total of 22.0,20.2,22.1 and 19.2 million raw sequence reads were generated from four libraries of ${ \sf R } _ { 0 } .$ $, \mathsf { R } _ { 4 8 } , \mathsf { S } _ { 0 }$ and $\mathsf { S } _ { 4 8 }$ (Table S6).After the removal of low-quality reads and adaptor sequences,18.4,16.9，18.6 and 16.0 million clean reads were obtained,respectively. Eighty per cent of the reads were successfully mapped to the reference genome,B.napus,in which unique reads that only aligned with one loci of the genome accounted for about $89 \%$ ：

A total of 6,821 significant differentially expressed genes (DEGs)were identified in the two genotypes based on the criterion, $\scriptstyle \lVert \mathsf { o g } _ { 2 } ( \mathsf { F P K M } _ { 4 8 } / \mathsf { F P K M } _ { 0 } ) \rVert \geq 2$ and $\mathsf { F D R } < 0 . 0 1$ .Of the 5384 DEGs in the R genotype,2356 $( 4 3 . 8 \% )$ were up-regulated and 3028 $( 5 6 . 2 \% )$ ：were down-regulated.A total of 5386 differentially expressed genes were identified in the S genotype, $4 1 . 4 \%$ (2229) of which were up-regulated and $5 8 . 6 \%$ (3157) of which weredown-regulated (Figure S2a)． In addition，for the upregulated genes,1784 were common to both genotypes (R and S),572 genes were specific to the R genotype only and 445 were specific to the S genotype.For the down-regulated genes,2,166 were commonly down-regulated (both R and S)，862 were specific to R and 991 specific to the S genotype (Figure S2b). Finally,hierarchical cluster analysis was performed based on the expression levels of the DEGs (Figure S2c).The R and S genotypes had similar expression patterns,but the main differences in gene expression were quantitative.

![](images/3076099256963c333a3780856874859478a55a24adf75283edfcaed7f65bf3c0.jpg)  
Figure 4 Comparison of the GWAS results with those of previous QTL mapping reported in Zhao et al. (2006) and Wu et al. (2013).The markers that lie within the confidence interval of the Sclerotinia stem resistance QTLare in blue for Zhao et al. (2o06) and red for Wu et al. (2013).

![](images/6477dae3b9a98c9d915117c2e26a55890015ff10d6029aa4662e76a4a9ae8c34.jpg)  
Figure 5Predictive abilities with different training populations and SNP markers.‘All',means all of the 30932 SNP markers;‘Random $3 0 0 ^ { \prime }$ randomly selected 300 markers;'Random $1 2 0 0 ^ { \prime }$ ,randomly selected 1200 markers; $^ { \prime } P < 0 . 0 0 1$ 5 $P < 0 . 0 1$ 1 $P < 0 . 0 5 ^ { \prime }$ ，significant association markers identified in GWAS with the certain $P$ value, $P < 0 . 0 0 1$ 1 $P < 0 . 0 1$ and $P < 0 . 0 5$ ，respectively.

Transcription factors (TFs)play an important role in plant resistance response (Seo and Choi，2015).In this study，495 $( 7 . 3 \% )$ genes encoding TFs were identified as being differentially expressed in the two genotypes,of which 84 were differentially expressed only in the R genotype,133 only in the S genotype and 278in both genotypes.These genes were divided into 36 families,including ERF (64)，MYB (63)，NAC (61)，bHLH (55), WRKY (48) and $C _ { 2 } H _ { 2 }$ (37) (Tables3 and S7).Nearly all $E R F$ and WRKY genes were up-regulated,while $78 \%$ of the bHLH genes were down-regulated,and MYB,NAC and $C _ { 2 } H _ { 2 }$ genes were upand down-regulated to similar extents.As numerous ERF and WRKY genes were found to be mostly up-regulated,our results indicate that these two families play a role in the response of B.napus to S.sclerotiorum.

Mitogen-activated protein kinase (MAPK) is a type of serine/ threonine protein kinase，and the MAPK cascade signalling pathway is important for plant defence response (Zhang and Klessig,20o1).In our study, genes encoding for the common MAPK cascade signalling pathway MAPKKK19-MKK9-MPK5 (Sun et al., 2014)were up-regulated in both genotypes,suggesting that this pathway isassociated with response to S.sclerotiorum in B.napus.

# Functional classification of common and specifically expressed genes

To monitor the gene expression pattern after infection，Gene Ontology (GO) enrichment analysis of commonly expressed genes in both genotypes (R and S)was conducted (Figure S3).The analysis showed that genes involved in metabolism，cellular processes and defence response were up- or down-regulated after infection with S.sclerotiorum in B.napus.Most genes located in organelles，mainly thechloroplast，weredownregulated in both genotypes，suggesting that photosynthetic damage was occurring during infection.In addition，KEGG pathway analysis of genes that were differentially expressed in both genotypes (common genes) was performed (Table S8). The KEGG analysis showed that after B.napus was infected with S.sclerotiorum,photosynthesis,glyoxalic acid and carbon metabolism were suppressed，while the biosynthesis of secondary metabolites such as sulphur metabolism (especially GSH and glucosinolates) and 2-oxocarboxylic acid metabolism was activated.

Todecipher the resistance mechanism of B.napus，we functionally classified and determined the metabolic pathways of 1434 genes differentially expressed only in the R genotype (Figure S2b) using MapMan.Genes involved in the jasmonic acid (JA)pathway，lignin biosynthesis,signal transduction，defence response and encoding transcription factors were identified and divided into groups.Group one contains specific genes related to the JA pathway and lignin biosynthesis (Table S9).

The second group contains genes involved in signal transduction including MAPK cascade signaling pathway，calcium signalling and G protein-coupled receptors.The 12 specific genes involved in calcium signalling and 5 genes encoding for G proteincoupled receptors were all induced.Genes involved in the signallingmoduleMEKK1-BnaMKK4-BnaMPK3-BnaWRKY26 (Liang et al.，2013)were all up-regulated in the R genotype. Another gene family,the MLO genes,MLO1 and MLO11,was down-regulated.Previous research shows that loss-of-function of MLO results in broad-spectrum resistance to powdery mildew in Hordeum vulgare (barley) (Piffanelli etal.,2004).

The third group includes a total of 15 genes involved in defence response,suchaschitinase genes (CHl and CHiB),BGLU16,seven disease-resistant protein genes and five receptor protein genes (RLP6,32，41,54and56).

Table3 Differentially expressed genes belonging to transcription   

<html><body><table><tr><td colspan="6">factor families</td></tr><tr><td>TF family</td><td>Total number</td><td>Change</td><td>Common</td><td>R specific</td><td>Ssp</td></tr><tr><td>AP2</td><td>6</td><td>Up</td><td>2</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>3</td><td>1</td></tr><tr><td>ARF</td><td>6</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>4</td><td>0</td><td>2</td></tr><tr><td>ARR-B</td><td>5</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>3</td><td>1</td></tr><tr><td>B3</td><td>3</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>1</td><td>2</td></tr><tr><td>BES1</td><td>1</td><td>Up</td><td>1</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>0</td><td>0</td></tr><tr><td>bHLH</td><td>55</td><td>Up</td><td>4</td><td>4</td><td>4</td></tr><tr><td></td><td></td><td>Down</td><td>20</td><td>11</td><td>12</td></tr><tr><td>bZIP</td><td>13</td><td>Up</td><td>1</td><td>1</td><td>1</td></tr><tr><td></td><td></td><td>Down</td><td>6</td><td>0</td><td>4</td></tr><tr><td>C2H2</td><td>37</td><td>Up</td><td>15</td><td>1</td><td>3</td></tr><tr><td></td><td></td><td>Down</td><td>6</td><td>6</td><td>6</td></tr><tr><td>C3H</td><td>7</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>4</td><td>1</td><td>2</td></tr><tr><td></td><td></td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td>CO-like</td><td>6</td><td>Down</td><td>2</td><td>1</td><td>3</td></tr><tr><td>CPP</td><td></td><td>Up</td><td>0</td><td>1</td><td>0</td></tr><tr><td></td><td>1</td><td>Down</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td>DBB</td><td>6</td><td>Down</td><td>3</td><td>1</td><td>2</td></tr><tr><td></td><td></td><td>Up</td><td>3</td><td>0</td><td>0</td></tr><tr><td>Dof</td><td>10</td><td>Down</td><td>3</td><td>1</td><td>3</td></tr><tr><td></td><td></td><td>Up</td><td>37</td><td>7</td><td>15</td></tr><tr><td>ERF</td><td>64</td><td>Down</td><td>1</td><td>1</td><td>3</td></tr><tr><td></td><td></td><td>Up</td><td>3</td><td>0</td><td>0</td></tr><tr><td>G2-like</td><td>8</td><td></td><td>2</td><td>1</td><td>2</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>0</td><td>2</td></tr><tr><td>GATA</td><td>7</td><td>Up</td><td>2</td><td>0</td><td>3</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>0</td><td>0</td></tr><tr><td>GeBP</td><td>1</td><td>Up Down</td><td>1</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td></td><td>0</td><td>0</td><td>1</td></tr><tr><td>GRAS</td><td>8</td><td>Up</td><td>2</td><td>1</td><td>4</td></tr><tr><td></td><td></td><td>Down</td><td>2</td><td>0</td><td>0</td></tr><tr><td>HD-ZIP</td><td>18</td><td>Up Down</td><td>10</td><td>3</td><td>3</td></tr><tr><td></td><td></td><td></td><td>5</td><td>0</td><td>0</td></tr><tr><td>HSF</td><td>8</td><td>Up</td><td>1</td><td>0</td><td>2</td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>0</td><td>0</td></tr><tr><td>LBD</td><td>3</td><td>Up</td><td>1</td><td>0</td><td></td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>0</td><td>1</td></tr><tr><td>MIKC</td><td>6</td><td>Up</td><td>1</td><td>0</td><td>2 2</td></tr><tr><td></td><td></td><td>Down</td><td>18</td><td>5</td><td>6</td></tr><tr><td>MYB</td><td>63</td><td>Up</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td>Down</td><td>22</td><td>4</td><td>8</td></tr><tr><td>MYB_related</td><td>10</td><td>Up</td><td>0</td><td>1</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>4</td><td>1</td><td>4</td></tr><tr><td>NAC</td><td>61</td><td>Up</td><td>23</td><td>5</td><td>11</td></tr><tr><td></td><td></td><td>Down</td><td>13</td><td>6</td><td>3</td></tr><tr><td>NF-YB</td><td>1</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>0</td><td>0</td></tr><tr><td>Nin-like</td><td>1</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>0</td><td>1</td></tr><tr><td>RAV</td><td>1</td><td></td><td>0</td><td>0</td><td></td></tr><tr><td></td><td></td><td>Up Down</td><td>0</td><td>0</td><td>1 0</td></tr></table></body></html>

Table3 Continued   

<html><body><table><tr><td>TF family</td><td>Total number</td><td>Change</td><td>Common</td><td>R specific</td><td>S specific</td></tr><tr><td>SBP</td><td>5</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>2</td><td>2</td><td>1</td></tr><tr><td>SRS</td><td>2</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>1</td><td>1</td></tr><tr><td>TALE</td><td>10</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>6</td><td>3</td><td>1</td></tr><tr><td>TCP</td><td>1</td><td>Up</td><td>0</td><td>0</td><td>1</td></tr><tr><td></td><td></td><td>Down</td><td>0</td><td>0</td><td>0</td></tr><tr><td>Trihelix</td><td>7</td><td>Up</td><td>2</td><td>1</td><td>3</td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>0</td><td>0</td></tr><tr><td>WOX</td><td>4</td><td>Up</td><td>2</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>2</td><td>0</td><td>0</td></tr><tr><td>WRKY</td><td>48</td><td>Up</td><td>31</td><td>3</td><td>2</td></tr><tr><td></td><td></td><td>Down</td><td>4</td><td>1</td><td>7</td></tr><tr><td>ZF-HD</td><td>2</td><td>Up</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>Down</td><td>1</td><td>1</td><td>0</td></tr></table></body></html>

Group four contains genes encoding transcription factors.A total of 82 TFs were found in the R genotype,of which 29 were up-regulated in response to inoculation,including seven ERFs, five MYBs (two MYB108s,two MYB37s and MYB84),five NACs,four bHLHs and three WRKYs (two WRKY61s and WRKY8) (Table S9).

# Combined analysis of GWAS and transcriptome sequencing

The candidate gene regions on chromosomes A8 and C6 as determined by GWAS analysis were 15.01-15.42 Mb and 30.78- 31.34 Mb,respectively.When we combined these results with those of transcriptome sequencing analysis,21 genes that were differentially expressed in the R or S genotype were found in this interval. Eight of these genes were on A8 and 13 were on C6, including four genes with unknown function (Figure 6, Table 4). Inaddition，a GSTU gene cluster，BnaC06g31020D (W22), BnaC06g31030D (W23) and BnaC06g31040D (W24)，that occurred $3 3 0 ~ \mathsf { k b }$ downstream of the peak SNP on C6 was upregulatedabout64-fold.BothBnaC06g31020Dand BnaC06g31030D showed homology with GSTU12 from Arabidopsisthaliana，and BnaC06g31040Dwas homologouswith AtGSTU11 (Table 4,Figure 6).These genes have been shown to play an important role in the defence response of plants (Jiang and Deyholos,2009;Liao et al.,2014).

# Verification of transcriptome sequencing

To validate our transcriptome sequencing results,qRT-PCR was used to determine differential gene expression for the genes that were commonly identified.The real-time PCR results confirm and verify the expression patterns obtained with transcriptome sequencing (Figure 7).

# Discussion

# Culture of Sclerotinia sclerotiorum and the screening method for stem resistance

The isolate of S.sclerotiorum used in this study (kindly provided by Dr.Mei）was collected from the field，incubated and grown in the laboratory. Different B.napus accessions vary in theirpathogenicity and resistance to different isolatesof S.sclerotiorum (Garg et al.,2010a).In this study, the resistant accessions could be correctly and effectively identified using the same criteria as would be expected when there is a single isolate infection of S.sclerotiorum.If mixed populations of S.sclerotiorum were used，the efficiency of screening for resistance Would have been reduced and GWAS mapping could not have beenperformed.The QTL identification for resistanceto S.sclerotiorum of B.napus also used the isolate of S.sclerotiorum (Mei etal.,2013;Wu et al.,2013).However, there are still biological limitations when using a single isolate，in that it cannot represent the diversity that would generally be observed inthe field.For example，a number of S.sclerotiorum isolates can be found in a field. Sexton et al. (2oo6) identified 55 unique S.sclerotiorum haplotypes with genotypic diversity ranging from 0.40 to O.71 in a rapeseed field.Furthermore，the aggressiveness of the different isolates was variable due to physiological specialization (Taylor et al.，2015)．Due to this variability，it would be difficult to identify all of the QTLs in B.napus that are associated with resistance to all of the S.sclerotiorum isolates. Wei et al. (2014) showed that QTLs identified using the detached stem inoculation assay overlapped with those found inadiseased field;however,some QTLs were found in the field but not in the laboratory.

![](images/0f1e41e5cf5e49d0170d69ad2a7a85a59da26dfec9d26d0a6c0b6dc0d6e0a828.jpg)  
Figure 6 The differentially expressed genes in the interval identified through the GWAS mapping. (a) A8; (b) C6.Top, the candidate gene regions on chromosomes A8 and C6 as determined by GWAS analysis.Middle,the position of differentially expressed genes within the interval. Bottom,the expression levels in the resistant (R) and susceptible (S) genotype.Red denotes up-regulated genes; green represents down-regulated genes;and gray represents the missing value.

In this study,the detached stem assay was used to assess the resistance of B.napus to S.sclerotiorum.Previous methods for screening stem resistance to S.sc/erotiorum have included petal inoculation，toothpick inoculation and plug inoculation in the field (Yin et al.，2010; Zhao and Meng，2003).However，the efficiency of these methods greatly depends on environmental conditions,such as rain and/or temperature.Screening could be better controlled in the glasshouse，but this involves using expensive infrastructure to maintain temperature and humidity. Inaddition，field and glasshouse experiments for screening are fairly time-consuming.The detached stem assay is a flexible,fast and reliable identification method that can easily be performed in acontrolled uniform environment (Mei et al.，2012).Very little space was needed，and a large-scale estimation，about 150 stems,only needed a $2 - m ^ { 2 }$ space (Mei et al.,2012).Furthermore, there was no correlation between lesion length and the diameter of stems (Mei et al.，2012),which meant that stem resistance could be accurately evaluated regardless of different sizes.

The PDA media that S.sc/erotiorum is cultured on affects its virulence. Guo and Stotz (2o07)showed that using PDA relative to minimal medium，disease symptoms occurred 1 day earlier in leaves of Arabidopsis.Thisis advantageous when usingthe detached stem inoculation technique,as it allows for quick and accurate evaluation of stem resistance with minimized water loss from the stem.In addition,using isolates with increased virulence, it may be easier to identify the most significant QTLs.As such,we identified two QTLs on A8 and C6 in this study.Future genetic research should take into consideration the virulence of the pathogen used for screening. Minimal media (relative to PDA) would be better used for inoculation trials to identify howan isolate，with different virulence，performs in plant-pathogen interactions.Furthermore,additional QTLs canalso be identified.

In this study,we wounded the plant stem to allow S. sc/erotiorum infection.The pathogen does not need to penetrate the

Table 4 Common candidate genes identified by GWAS and transcriptome sequencing   

<html><body><table><tr><td>Code</td><td>Gene</td><td>Arabidopsis homologue</td><td>Description</td><td>R log2 (fold change)</td><td>S log2 (fold change)</td></tr><tr><td>W1</td><td>BnaA08g19770D</td><td>AT1G26560</td><td>Beta glucosidase 40 (BGLU40)</td><td></td><td>-4.86**</td></tr><tr><td>W2</td><td>BnaA08g20100D</td><td>AT1G27110</td><td>Tetratricopeptide repeat-like superfamily protein</td><td></td><td>8.32***</td></tr><tr><td>W3</td><td>BnaA08g19850D</td><td>AT1G35350</td><td>EXS family protein</td><td>5.54***</td><td>4.58**</td></tr><tr><td>W4</td><td>BnaA08g19990D</td><td>AT1G26920</td><td>Unknown protein</td><td>-5.04**</td><td>-3.88*</td></tr><tr><td>W5</td><td>BnaA08g20190D</td><td>AT1G27210</td><td>ARM repeat superfamily protein</td><td>-3.95**</td><td>-5.67***</td></tr><tr><td>W6</td><td>BnaA08g20360D</td><td>AT1G23560</td><td>Domain of unknown function (DUF220)</td><td>3.71*</td><td>4.19**</td></tr><tr><td>W7</td><td>BnaA08g19560D</td><td>AT1G26100</td><td>Cytochrome b561</td><td>-4.19*</td><td></td></tr><tr><td>W8</td><td>BnaA08g19580D</td><td>AT1G26120</td><td>Isoprenyl cysteine methylesterase-like 1 (ICME-LIKE1)</td><td>-5.18*</td><td></td></tr><tr><td>W9</td><td>BnaC06g30020D</td><td>AT1G68430</td><td>Unknown protein</td><td></td><td>3.67*</td></tr><tr><td>W10</td><td>BnaC06g30560D</td><td>AT1G69390</td><td>Homologue of bacterial MinE 1 (MINE1)</td><td></td><td>-3.88*</td></tr><tr><td>W11</td><td>BnaC06g29990D</td><td>AT1G69160</td><td>Unknown protein</td><td>-3.62*</td><td>-3.94**</td></tr><tr><td>W12</td><td>BnaC06g30030D</td><td>AT1G68440</td><td>Unknown protein</td><td>3.58*</td><td>3.05*</td></tr><tr><td>W13</td><td>BnaC06g30160D</td><td>AT1G68560</td><td>Alpha-xylosidase 1 (XYL1)</td><td>-4.37**</td><td>-4.91***</td></tr><tr><td>W14</td><td>BnaC06g30180D</td><td>AT1G68570</td><td>Major facilitator superfamily protein (MFS)</td><td>-4***</td><td>-4.5**</td></tr><tr><td>W15</td><td>BnaC06g30250D</td><td>AT4G39670</td><td>Glycolipid transfer proteinfamily protein (GLTP)</td><td>6.48***</td><td>5.23***</td></tr><tr><td>W16</td><td>BnaC06g30370D</td><td>AT1G68740</td><td>PHO1</td><td>-4.04**</td><td>-4.53**</td></tr><tr><td>W17</td><td>BnaC06g30470D</td><td>AT1G26450</td><td>Carbohydrate-binding X8 domain superfamily protein</td><td>-5.84***</td><td>-7.09***</td></tr><tr><td>W18</td><td>BnaC06g30580D</td><td>AT1G69420</td><td>DHHC-type zinc finger family protein</td><td>-5.39***</td><td>-3.13*</td></tr><tr><td>W19</td><td>BnaC06g30610D</td><td>AT1G33590</td><td>Leucine-rich repeat (LRR) family protein</td><td>6.78***</td><td>7.01***</td></tr><tr><td>W20</td><td>BnaC06g30040D</td><td>AT1G68470</td><td>Exostosin family protein</td><td>-3.62*</td><td></td></tr><tr><td>W21</td><td>BnaC06g30280D</td><td>AT1G68590</td><td>Ribosomal protein PSRP-3/Ycf65</td><td>-4.08**</td><td></td></tr><tr><td>W22</td><td>BnaC06g31020D</td><td>AT1G69920</td><td>Glutathione S-transferase TAU 12 (GSTU12)</td><td>5.96***</td><td>6.05***</td></tr><tr><td>W23</td><td>BnaC06g31030D</td><td>AT1G69920</td><td>Glutathione S-transferase TAU 12 (GSTU12)</td><td>8.31***</td><td>8.57***</td></tr><tr><td>W24</td><td>BnaC06g31040D</td><td>AT1G69930</td><td>Glutathione S-transferase TAU 11 (GSTU11)</td><td>8.96***</td><td>7.85***</td></tr></table></body></html>

$* * * P < 0 . 0 0 1$ $* * P < 0 . 0 1$ $^ { * } P < 0 . 0 5$

10- ORS 86 (7n1gre piln) cnog 4 2 024 广T 里 1 -6 -8 -10- W1W2 W3 W4W5W6W7W8W9W10W11W12W13W14W15W16W17W18W19W20W21W22W23W24 natural structural defence barriers during infection.This provides favourable conditions for pathogen attack，but may circumvent anymechanical defence mechanisms of the plant or any defence mechanisms specific to the plant epidermal tissue during the initial infection.As a consequence,some defence-associated QTLs might not be detected.Many similar QTLs were detected in wounding and nonwounding tests examining S.sclerotiorum resistance in common bean,but one specific QTL probably related to initial defence was identified in the nonwounding assay (Soule et al.，2011).Therefore，future research could also include nonwounding assays to identify QTLs associated with the initial defence mechanism.

# The identification of candidate genes

S.sclerotiorum is one of the most damaging agricultural diseases in China and can lead to considerable yield loss in B. napus. Previous studies into the resistance of B.napus to S.sclerotiorum were mainly conducted using QTL mapping,but in this study,we used GWAS implemented with GenABEL software for this purpose.We identified 17 SNPs that had significant associations with S.sclerotiorum response on chromosomes A8 and C6. ldentification of SNPs on C6 (30.7-31.3 Mb) is consistent with previous QTL mapping results (Li et al.，2015;Wu et al.，2013; Zhaoetal.，2006).Zhao etal. (2006)evaluatedS.sclerotiorum resistance in the petioles of B.napus DH lines,and a QTL was repeatedly detected in the region of 23.1-35.4 Mb on C6.Wu et al.(2013) detected a QTL from 30.3 Mb to 36 Mb using B.napus stems inoculated with mycelial agar plugs.Li et al. (2015) also identified conserved stem resistance QTLs on C6 from 29.6 to 36.1 Mb through integration analysis of resistance QTLs.

A comparison of our GWAS and transcriptome sequencing results revealed 21 common genes and a GSTU gene cluster, which were verified by qRT-PCR. Eight common candidate genes found on chromosome A8 were analysed: BnaA08g19770D,

BnaA08g20100D, BnaA08g19850D, BnaA08g19990D, BnaA08g20190D，BnaA08g20360D，BnaA08g19560Dand BnaA08g19580D.On chromosome C6,a total of 13 candidate genes were found (Figure 6, Table 4).

Twoofthese 21 genesencoded forglucosidases, BnaA08g19770D (W1）which is homologous to At1 g26560 (BGLU40) and BnaC06g30160D (W13)which is homologous to XYL1. Zhao et al. (2007) found that BGLU40 was expressed $4 8 \mathrm { ~ h ~ }$ after inoculation with S.sclerotiorum in susceptible B.napus. Glucosidases catalyse the cleavage of the glucoside plant defence molecules，such as cyanogenic glucosides and glucosinolates, during the interaction between plant and pathogen (Morant et al.,2oo8). Gucosidases have also been suggested to play a role inactivating phytohormones (Lee et al.,2oo6). Increased expression of BGLU1 has been demonstrated to be associated with resistance to the necrotrophic pathogen, Cochliobolus heterostrophusin maize (Simmons et al.，2oo1).The resistance has been shown to not depend on the antimicrobial compoundor hydroxamic acids,but operates via other mechanisms (Simmons et al.,2001). In our study, BGLU40 expression decreased 29.04- fold in B.napus which may lead to increased susceptibility in plants infected byS.sclerotiorum.

XYL1 (BnaC06g30160D)，which encodes for $\mathfrak { a }$ -xylosidase, has been shown to catalyse the hydrolysis reaction of xylogucan ologosaccharides,a member of glycoside hydrolase (GH) family 31 (Henrissat and Bairoch，1993).Miedes et al. (2014) showed that an increase in xyloglucan content increased resistance to pathogens in Aradidopsis.In Arabidopsis，the x/y1-2 mutant shows enhanced resistance to the necrotrophic fungus P. cucumerina due to the structural changesin xyloglucan (Delgado-Cerezo et al.，2012). However，inour study， the expresson level of XYL1 in the S genotype decreased (30.06-fold) more than that of the R genotype (20.68-fold).Therefore,the role of glucosidases in the defence response ofB.napus to S.sclerotiorum may need tobe investigatedfurther.

The common gene BnaC06g30470D (W17） found on C6 encodes for the pathogenesis-related (PR-2 family) gene，β-1,3 glucanase.PR proteins are strongly induced by pathogens or related factors (Stintzi etal.，1993). β-1,3-glucanase inhibits the growth of fungi by hydrolysing the fungal cell wall，which contains the primary polysaccharide，β-1,3 glucan.In addition, $\beta$ -1,3-glucanase triggers the release of oligosaccharide elicitors, which activate the production of antifungal compounds such as phytoalexin and other PR proteins (Keen and Yoshikawa,1983; Klarzynski et al.,2ooO). β-1,3-glucanase has also been shown to beresponsible for the decreased infection of S.sclerotiorum in B. napus (Fernando et al.，20o7).When the $\beta$ -1,3-glucanase gene (bgn13.1） from the fungus Trichoderma virens-10 was transformed and overexpressed in B.napus,the transgenic lines inhibited hyphal growth of S.sclerotiorum (Kheiri etal.,2014).In ourstudy，the expression of BnaC06g30470D in S genotype decreased (136.24-fold) more than that in R (55.71-fold),which might increase susceptibility to B. napus.

The gene (BnaC06g30610D,W19)encodes for a leucine-rich repeat(LRR) family protein which was induced in both genotypes in our study,and this finding was verified by qRT-PCR.The LRRs are important conserved domains in $R$ genes and have been shown to be involved in the interaction between plants and pathogens (Kohler et al.,20o8;Porter et al.,2009).

The transcription factor BnaC06g30580D(W18) that encodes for the DHHC-type zinc finger protein (ZFP)was significantly down-regulated in the R genotype (41.93-fold, $P < 0 . 0 1$ ).TFs playan important role in plant resistance response (Seo et al., 2015). ZFPs can be classified into several different types such as C2H2,C2C2,CCCH,GATA and DHHC,some of which have been found to be involved in abiotic and biotic stress (Cui et al.,2002; Jan et al.,2013).There are relatively few DHHC types of ZFPs, and very little is known about their function.It has been shown that DHHC-type ZFPs were repressed by Magnaporthe oryzae in rice (Li et al.，2014b).However,the mechanism for this is still unknown.In addition，many of the other TFs identified in this study belonged to ERF and WRKY families and most were upregulatedupon inoculation with S.sclerotiorum (Tables3and S7).The ERF and WRKY transcription factor are known to be involved in biotic and abiotic stress response (Chen et al.,2013a, b; Son et al.,2012;Wang et al.,2014b;Yang et al.,2009; Zhao et al.，2Oo7). In Arabidopsis，ERF5 negatively regulates chitininduced innate immunity response (Son et al.,2012).

Resistance genes have been shown to generally cluster in the genome (Graham et al.,2002).In this study,a GSTU gene cluster was identified 330 kb away from the peak SNP on chromosome C6；BnaC06g31020D and BnaC06g31030Dare homologuesof GSTU12 (AT1G69920),and BnaC06g31040D is a homologue of GSTU11 (AT1G69930).All of these genes were up-regulated in both genotypes upon S.sclerotiorum infection.GSTU plays an important role in the response to biotic and abiotic stress (including S.sclerotiorum) as cellular protectants,reducing free radicals and reactive species (Jiangand Deyholos，2oo9；Liao et al.，2014)． In previous studies，following infectionwith S.sclerotiorum in B.napus，the homologues of AT1G69920 (GSTU12) and AT1G69930 (GSTU11) were also up-regulated and GST proteins were expressed in resistant B.napus (Garg et al., 2013;Zhao et al.,2007,2009).The GSTU gene cluster identified inthis study confirms their importance in response to S.sc/ero tiorum by possibly acting as an antioxidant defence mechanism.

Another gene,BnaC.IGMT5.a,an indole glucosinolate methyltransferase,located on C6 at $3 5 . 5 ~ \mathsf { M b }$ ，about4Mbawayfrom ourpeak SNP,has previously been identified and cloned as a candidate gene for resistance to Sclerotinia (Wu et al.,2013). RTPCR analysis showed that no expression was observed in the susceptible B.napus lines,and increased expression was observed in the resistant (R) lines 24 to $9 6 ~ \mathsf { h }$ postinoculation (Wu et al., 2013).However,in our study,the expression of BnaC.IGMT5.a (BnaC06g37610D)was also increased $4 8 \mathrm { ~ h ~ }$ postinoculation in both R and S genotypes.

It has also been shown that the presence of susceptibility factors (genes) in susceptible genotypes could increase susceptibility to necrotrophic fungal pathogens in wheat (Liu et al., 2006).Therefore，it could be argued that resistance may be governed by the absence of a susceptibility gene in the resistant genotypes as opposed to the presence of a resistance gene. Specific susceptibility genes in the S genotype were detected in this study; however,the presence of these genes needs to be validated in future studies.To follow up on the genes identified in this study,we aim to clone the above candidate genes in order to elucidate the differences observed between the R and S genotypes in order to confirm their roles in the response to Sclerotinia.

# Transcriptome sequencing

Zhao et al. (2oo9) found that the greatest number of differentially expressed genes between the resistant B.napus variety Zhongyou 821and susceptible cultivar Westar occurred 48 hafter inoculation with S.sclerotiorum.Therefore,we conducted our transcriptome sequencing analysis on B.napus stems $4 8 \mathrm { ~ h ~ }$ after inoculation. Genes involved in photosynthesis，glyoxalic acid and carbon metabolism pathways were mostly down-regulated upon inoculation.Soybean and tomato had a similar response to Pseudomonas syringae; genes involved in photosynthesis were down-regulated after inoculation (Berger et al.,2004; Zou etal.,2005).

Conversely，we found that the biosynthesis of secondary metabolites，sulphur metabolism (especially of GSH and glucosinolates) and2-oxocarboxylicacidmetabolismwere increased， suggesting that theplant immune systemwas activated and plays a key role in the defence response to S.sclerotiorum.A total of 81 up-regulated genes were involved insulphur metabolism，and sulphur is an essential macronutrient for plant growth and development.Recent research into sulphur-induced resistance (SiR)or sulphur-enhanced defence (SED) (Rausch and Wachter， 20o5) shows that GSH and glucosinolates induce SIR/SED.GSH isa major antioxidant molecule in plants and is reported to be involved in the interactionbetween plants and microbes，including necrotrophic pathogens (Ball et al.,2004;Frendo et al.，2013;Parisy et al.，2oo7).GSH has been shown to increase after inoculation with S.sclerotiorum in B.napus (Bloem et al.，2012).Glucosinolates have also been shown to be important for the defence response in Brassicaceae (Wittstock and Halkier，2oo2).Glucosinolate was probably associated with S.sclerotiorum resistance in B.napus (Li et al.，1999).And as discussed above, BnaC.IGMT5.a,an indole glucosinolate methyltransferase，isa candidate gene for S.sclerotiorum resistance (Wu et al.,2013). We therefore conclude that the increased metabolism of sulphur-containing compounds,such as GSH and glucosinolate, playsa key role in plant defence.

Specific genes in JA signaling and lignin biosynthesiswere found only in the R genotype,indicating that these two pathways have important functions in the defence response.JA,synthesized from linoleic acid by a series of enzymes (i.e.LOX,AOS AOC，and OPR),is an endogenous hormone involved in plant defence against necrotrophic pathogens (Frendo et al.，2013). Using microarray analysis,Yang et al. (2oo7)and Zhao et al. (2009) found that genes associated with the JA pathway were induced in response to S.sclerotiorum in B.napus.The sequential activation of salicylic acid (SA) signalling and JA signalling has also been shown to be important in the defence response against S.sclerotioruminB.napus (Wangetal.， 2012b)．The JA-mediated defence response is thought to be though the regulation of protease inhibitor and secondary metabolites,such asanthocyanin and glucosinolates (van Dam et al.，2004; Mengiste,2012).

Lignin in the plant cell wall contributes to disease defence,not only by enhancing the mechanical strength of the plant,but also by preventing infection and colonization of the pathogen due to its hydrophobic properties (Vanholme et al.，2oo8).Lignification protects cotton from the fungal necrotrophic pathogen Verticillium dahliae(Shi et al.,2012;Xu et al.,2011).In our study,specific ligninsynthesis-related genes, that is C4H,CCoAOMT,COMTand CAD5,were up-regulated in the R genotype,which shows that lignin contributes to the response to S.sclerotiorum in B.napus. Miedes et al. (2014) summarized and explained the effect of lignin on plant pathogen resistance or susceptibility,which was mainly through the alternation of cell walls and the production of some soluble phenolic compounds.However,a greater understanding of the biochemical mechanisms is still necessary to fully understand the lignin host-pathogen interactions.

The relationship between GSH and other stress response genes has been studied in response to abiotic and biotic stress (Han et al.，2013;Kumar et al.，2015;Wingate et al.，1988). GSH was found to induce biosynthesis of phenylpropanoid, lignin， dihydroflavonolspathway， JA signalling，andthe expression of TFs and various PR proteins (Garg et al.，2013; Han et al.，2013； Kumaret al.，2015). These genesand pathways were also induced in our study.Therefore，it could be speculated that GSH may also play a role in modulating the JA and lignin pathways in response to necrotrophic pathogens such as S.sclerotiorum in B.napus.GSH may act as a master switch in regulating pathogen response in plants.The increased levels of GST could enhance GSH activity (Garg et al.，2013), and GSTU detected above may play a major role in the response to S.sclerotiorum in B.napus.

# Genome-wide prediction

We performed a genome-wide prediction to evaluate the breeding value of a plant based on its genetic loci with rrBLUP. rrBLUP performed better in the prediction accuracies for the complex traits,such as flowering time,oil content,plant height and glucosinolate content in B.napus， than the method of BayesB (Wurschum et al.,2014).The predictive ability was related to training population， the number of markers and broad sense heritability (Owens et al.，2014;Slavov et al.，2014).In our study,prediction abilities also increased with the size of the training population，but the standard deviation increased also. Therefore,it was better to select $60 \%$ of the training population (145)for genome predictions.When all SNP markers were used, the predictive ability was too low (0.27) that the markers failed to predict stem resistance.It is well known that GWAS and genomic prediction are performed on the same phenotype and same genotype data sets,and that GWAS is able to detect markers significantlyassociated with the trait by settinga certain $P$ value, such as $P < 0 . 0 0 1$ 1 $P < 0 . 0 1$ and $P < 0 . 0 5$ (Zhang et al.,2014b). When using the significantlyassociated SNPs with S.sclerotiorum resistance produced by GWAS including $P < 0 . 0 0 1$ (69 SNPs), $P < 0 . 0 1$ (288）and $P < 0 . 0 5$ (1203)，the predictive ability increased from 0.58 $( P < 0 . 0 0 1$ ）to 0.8 $( P < 0 . 0 5 )$ .Therefore, these significant markers detected in GWAS could be used to predict stem resistance in efforts to breed B.napus lines that are resistant to stem rot. Zhang et al.(2014b)also showed that GWAS could improve the accuracy of genomic prediction in animal breeding,rice breeding and human genetics,when the significant SNPs detected by GWAS and located in QTL regions are employed in genomic prediction.With the help of genetic architecture and population structure revealed by GWAS，the efficiency of genomic prediction and further breeding processes is increased (Spindel et al.,2015).

# Experimental procedures

# Plant material and phenotypic evaluation

A total of 347 B.napus lines were collected from spring,winter and semi-winter accessions and cultivated in Southwest University of Beibei, Chongqing,China,in 2012 and 2013 (Table S1).Each line was planted in two rows of 10 plants per row,with $3 0 ~ \mathsf { c m }$ between rows and a distance of $2 0 \ c m$ between plants within each row.

The S.sclerotiorum isolate was provided by Dr. Jiaqin Mei from Southwest University and was cultured on potato dextrose agarmedium( $20 \%$ potato, $2 \%$ dextrose and $1 . 5 \%$ agar）at

$2 2 ^ { \circ } C$ in darkness.The detached stem inoculation assay was used to evaluate resistance to Sclerotinia at the end of flowering time according to Mei et al. (2012)，with ‘Zhongshuang $9 ^ { \prime }$ as control，which is known to be resistant to stem rot.A 40-cm segment of stem that was $1 0 \ \mathsf { c m }$ above the soil Was cut off the plants.The cut ends were sealed in plastic wrap，and the stems were wrapped in moist towels and filter papers and placed on a board.Each stem was wounded at two points with a 4-mm-diameter punch,and Sclerotinia mycelium agar plugs ( $6 ~ \mathsf { m m }$ in diameter) were placed on each wound. The stems were sprayed with water mist,and the board was covered with plastic film.The infection temperature was maintained at $2 2 ^ { \circ } C$ .Lesions first appeared $2 4 \ h$ after inoculation and expanded to the two sides of the stem.After $7 2 ~ \mathsf { h } .$ lesions were watery and encircled the stems.The lesion lengths onthe stems wererecorded $7 2 \mathrm { ~ h ~ }$ afterinoculation．The resistance assessment (RA)was calculated according to relative susceptibility, $\mathsf { R A } = \mathsf { L } \mathsf { L } _ { \mathsf { c o n t r o l } } ,$ where L denotes the lesion length of the tested accession and $\mathsf { L } _ { \mathsf { C O n t r o l } }$ represents that of ‘Zhongshuang $9 ^ { \prime }$ .The resistance was conducted with two replications for each of 347 lines during 2013 and 2014 using the detached stem inoculation assay.

# SNP genotyping and in silico mapping of SNPs

SNP genotyping Was performed using the Brassica 6OK Illumina SNParray (http://www.illumina.com/technology/beadarray-technology/infinium-hd-assay.html)，and SNP data were analysed using Illumina BeadStudio genotyping software.SNP call frequencies of $< 0 . 9$ or minor allele frequencies (MAFs)of $< 0 . 0 5$ were excluded in the study.

Toidentify the physical position of SNPs in the B.napus genome，SNP array sequences were aligned with the genome sequences of B.napus (Chalhoub,2014) using local BLASTn with E values of $< ^ { \prime } 1 . 0 \mathsf { E } ^ { - 1 0 \prime }$ .SNP positions with a minimum E value and a maximum score were selected.SNPs with the same E values and scores were removed.

# Relative kinship and haplotype block construction

The relative kinship matrix of 347 B.napus lines was computed using SPAGeDi software (Hardy and Vekemans,20o2).Negative kinship values were set to zero. Haplotypeblockswere constructed using the four gamete rule (Lorenz et al.，2010) with Haploview software (Barrett et al.，2oo5).The parameters were set as follows: $P$ value cut-off of Hardy-Weinberg was 0.001，MAF was 0.05，and the maximum number of Mendel errorswas 1.

# Genome-wide association analysis

The best linear unbiased prediction (BLUP) of a stem resistance trait with two replicates over two years was estimated using an R script (www.eXtension.org/pages/61oo6)basedon a linear model.An association analysis was performed using the $P + K$ model implemented in GenABEL software (Aulchenko et al., 2007). $P$ values were calculated based on the mm score function of GenABEL.The quantile-quantile plot was displayed with $- \infty _ { 9 1 0 }$ (P)of each SNP and expected $P$ value.The Manhattan plot was displayed using qqman software (Turner,2014).The threshold of GWAS was set to $P < 3 . 2 3 \times 1 0 ^ { - 5 }$ (1/total SNPs used).The false discovery rate (FDR)was calculated based on the formula: $[ ( \mathsf { m } \times P ) / \mathsf { n } ] \times 1 0 0 \%$ ，where m was the number of total SNPs，P was the threshold of GWAS,and n was the number of significant SNPs.

# Stem resistance prediction

Toevaluate the prediction ability of markers for stem resistance among 243 lines (miss rate $< 1 0 \%$ )，prediction accuracywas examined using ridge regression best linear unbiased prediction rrBLUP (Endelman,2O11) in the R package.The prediction ability (r)of the model representing the correlation between the predicted and field data was estimated (Slavov et al.,2014).To assess the performance of genomic prediction, $20 \text{‰}$ of the population was selected at random as the training population and markers $\mathit { \ ' } P < 0 . 0 0 1$ - $P < 0 . 0 1$ 1 $P < 0 . 0 5$ and all SNPs) were also used.

# Transcriptome sequencing and the identification of transcription factor genes

Five highly resistant (R) and susceptible (S) lines were selected from the GWAS population,and two stems were used per line and two inoculation sites on each stem.The R lines were WH-57, 2011-7103,Huayou14,Chuxianbaihua and Youyan2,and the S lines were Yangjian8,P685,Guangde138,SWU69 and 07037, respectively $\left( F = 4 8 . 6 9 \right.$ $P < 0 . 0 0 0 1$ ).Mock-inoculated lineswere treated with agar plug only.The stem tissues extending $1 0 ~ \mathsf { m m }$ beyond the inoculation point and $1 \ \mathrm { m m }$ deep were harvested $4 8 \mathrm { ~ h ~ }$ postinoculation with a razor blade,and immediately placed in liquid nitrogen and stored at $- 8 0 ~ ^ { \circ } C$ .FiveR and S lines (20 inoculation points) before and after inoculation were pooled to yield four RNA samples,that $\mathsf { i s R } _ { 0 } ,$ ${ \sf R } _ { 4 8 }$ $\mathsf { S } _ { 0 }$ and $\mathsf { S } _ { 4 8 }$ Sequencing libraries were generated using the Illumina RNA Library Prep Kit following the manufacturer'srecommendations.The library preparations were sequenced on an Ilumina Hiseq 2ooo platform,and 1oo-bp paired-end reads were generated.

The B.napus reference genome was constructed using bowtie 2 (Broad Institute, Cambridge,MA,USA). Sequencing reads were aligned to the B.napus reference genome and then assembled using TopHat 2.0.0 and Cufflinks (Trapnell et al.，2012).Gene expression levels were estimated using FPKM (fragments per kilobase of exon per million mapped fragments).Differentially expressed genes (DEGs) between two samples were identified with Cuffdiff,based on the criteria ${ \mathsf { F D R } } { < } 0 . 0 1$ and $\| \mathsf { o g } _ { 2 }$ 0 $\mathrm { F P K M } _ { 4 8 } /$ $F P K M _ { 0 } ) | { > } 2$ .To identify possible transcription factors (TFs),DEGs in the R and S genotypes were aligned to TFs in A. thaliana,which were downloaded from the Plant Transcription Factor Database (http://planttfdb.cbi.pku.edu.cn/index.php).

# GO and KEGG enrichment analysis of DEGs

GOenrichment analysis was implemented using BiNGO in Cytoscape v3.1.0 (Maere et al.,2005;Smoot et al.,2011),and significantly enriched GO terms $( \mathsf { F D R } < 0 . 0 1$ )were displayed using the online tool WEGO (Web Gene Ontology annotations Plot, http://wego.genomics.org.cn) (Ye et al.，20o6). The enrichment of DEGs in the KEGG pathway analysis was conducted in the KOBAS2.0 website (http://kobas.cbi.pku.edu.cn/home.do).To analyse the metabolic pathway and functional classification of DEGs,expression data were mapped to metabolic pathways using MapMan software (Thimm et al.,2004).

# qRT-PCR analysis

Total cDNA was synthesized from $1 ~ \mu \ g$ RNA according to the manufacturer's instructions (Bio-Rad,Foster City,CA).Primers are listed in Table S2.PCRs contained $1 0 ~ \mu \iota$ SSoAdvanced SYBR Green Supermix (Bio-Rad), $2 . 0 ~ \mu \ L$ CDNA, $1 ~ \mu \iota$ primer and distilled water to a final volume of $2 0 ~ \mu \iota$ .The PCR program wasas follows: $9 5 ~ ^ { \circ } C$ for $3 0 ~ \varsigma$ and 35 cycles of $9 5 ~ ^ { \circ } C$ for $5 \ 5 ,$ followed by $5 6 { \mathrm { - } } 6 7 ~ ^ { \circ } C$ (depending on the primers used) for $3 0 ~ \mathsf { s }$ Three technical replicates were run for test and reference genes. Relative expression levels were evaluated using the $2 ^ { - \Delta \Delta \mathsf { C T } }$ method.Actin was used as an internal control for normalization, and the inoculated group was compared to the mock-inoculated control group.

# Acknowledgements

We thank Prof. for William John Lucas kindly discussion and comments on the manuscript. This work was supported by the National High Technology Research and Development ('863') Programs of China (2011AA10A104， 2013AA102602)， the National Natural Science Foundation of China (31171619, 31371655 and U1302266)，the Earmarked Fund for Modern Agro-industry Technology Research System (CARS-13) and the '111'Project (B12006).

# References

Aulchenko,Y.S.，Ripke，S., Isaacs,A.and van Duijn,C.M.(20o7) GenABEL: anR library for genome-wideassociation analysis.Bioinformatics，23, 1294-1296.   
Ball, L.,Accotto,G.P., Bechtold, U., Creissen,G.,Funck,D.,Jimenez,A., Kular, B.et al. (2oo4) Evidence for a direct link between glutathione biosynthesis and stress fefense gene expression in Arabidopsis.Plant Cell,16,2448-2462.   
Barrett, J.C.,Fry, B.,Maller,J.and Daly,M.J.(2Oo5) Haploview:analysis and visualization of LD and haplotype maps. Bioinformatics,21,263-265.   
Behla,R.S.,Fernando,W.G.D.and Li,G.(2oo9) Identification of quantitative trait loci for resistance against sclerotinia stem rot in Brassica napus. Can.J. Plant Pathol.31,477-478.   
Berger,S.,Papadopoulos,M.,Schreiber, U.,Kaiser,W.and Roitsch,T.(2004) Complex regulation of gene expression,photosynthesis and sugar levels by pathogen infection in tomato.Physiol. Plant.122,419-428.   
Bloem，E.，Haneklaus，S.，Kesselmeier，J.and Schnug，E.(2012）Sulfur fertilization and fungal infections affect the exchange of H2S and COS from Agricultural Crops.J.Agric.Food Chem.60,7588-7596.   
Bolton，M.D.，Thomma，B.P.H.J.andNelson，B.D．(2006)Sclerotinia sclerotiorum (Lib.)de Bary:biology and molecular traits of a cosmopolitan pathogen.Mol. Plant Pathol.7,1-16.   
Cai,D.F., Xiao,Y.J.,Yang,W.,Ye,W.,Wang, B., Younas,M.,Wu,J.S.et al. (2014)Association mapping of six yield-related traits in rapeseed (Brassica napus L.).Theor. Appl. Genet.127,85-96.   
Chalhoub,B.(2014) Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome.Science,345,1255-1255.   
Chen,L.G.，Zhang,L.P.，Li,D.B.，Wang,F.and Yu,D.Q.(2013a)WRKY8 transcription factor functions in the TMV-cg defense response by mediating both abscisic acid and ethylene signaling in Arabidopsis.Proc. Natl Acad. Sci. USA,110,E1963-E1971.   
Chen,X,Liu,J., Lin,G.,Wang,A., Wang,Z.and Lu,G. (2013b) Overexpression of AtWRKY28 and AtWRKY75 in Arabidopsis enhances resistance to oxalic acidand Sclerotinia sclerotiorum.Plant Cell Rep.32,1589-1599.   
Cui,J.P.,Jander,G.,Racki,L.R., Kim,P.D.,Pierce,N.E.and Ausubel,F.M.(2002) Signalsinvolved in Arabidopsisresistance to Trichoplusia ni caterpillars induced by virulent and avirulent strains of the phytopathogen Pseudomonas syringae.Plant Physiol.129,551-564.   
van Dam，N.M.，Witjes，L.and Svatos，A.(20o4) Interactions between aboveground and belowground induction of glucosinolates in two wild Brassica species. New Phytol.161,801-810.   
Delgado-Cerezo，M.， Sanchez-Rodriguez， C.， Escudero，V.，Miedes，E., Fernandez,P.V.,Jorda,L.,Hernandez-Blanco,C.et al. (2012) Arabidopsis heterotrimeric G-protein regulates cell wall defense and resistance to necrotrophic fungi. Mol. Plant,5, 98-114.   
Endelman,J.B.(2011) Ridge regression and other kernels for genomic selection with R package rrBLUP.Plant Genome.4,250-255.   
Fernando,W.G.D.,Nakkeeran,S., Zhang,Y.and Savchuk,S.(2007) Biological control of Sclerotinia sclerotiorum (Lib.)de Bary by Pseudomonasand Bacilus species on canola petals. Crop Prot.26,100-107.   
Fopa Fomeju，B.，Falentin，C.，Lassalle，G.，Manzanares-Dauleux,M.J.and   
Delourme，R.(2014) Homoeologous duplicated regions are involved in quantitative resistance of Brassica napus to stem canker. BMC Genom.15, 498.   
Frendo，P.，Baldacci-Cresp，F.，Benyamina，S.M.and Puppo，A.(2013) Glutathione and plant response to the biotic environment. Free Radic. Biol.   
Med. 65,724-730.   
Fusari,C.M.，Di Rienzo,J.A.，Troglia,C.,Nishinakamasu,V.，Moreno，M.V., Maringolo,C., Quiroz,F.et al. (2012) Association mapping in sunflower for Sclerotinia Head Rot resistance.BMC Plant Biol.12,93.   
Garg，H.，Kohn,L.M.，Andrew,M.,Li,H.，Sivasithamparam,K.and Barbeti, M.J. (2010a) Pathogenicity of morphologically different isolates of Sclerotinia Sclerotiorum with Brassica napus and B.juncea genotypes.Eur.J.Plant Pathol. 126,305-315.   
Garg,H.,Li，H.,Sivasithamparam,K.,Kuo,J.and Barbetti,M.J.(2O10b)The infection processes of Sclerotinia sclerotiorum in cotyledon tissue ofa resistant and a susceptible genotype of Brassica napus.Ann. Bot.106,897- 908.   
Garg，H.,Li，H.，Sivasithamparam,K.and Barbetti，M.J.(2O13) Diferentially expressed proteins and associated histological and disease progression changes in cotyledon tissue of a resistant and susceptible genotype of Brassica napus infected with Sclerotinia sclerotiorum.PLoS ONE,8,e65205.   
Graham，M.A.，Marek，L.F．and Shoemaker，R.C.(2002)Organization, expression and evolution of a disease resistance gene cluster in soybean. Genetics,162,1961-1977.   
Guo,X.M.and Stotz,H.U.(2007) Defense against Sclerotinia sclerotiorum in   
Arabidopsisisdependent on jasmonic acid，salicylicacid，and ethylene signaling.Mol. Plant Microbe Interact.20,1384-1395.   
Gururani,M.A., Venkatesh,J., Upadhyaya,C.P.，Nookaraju,A.，Pandey，S.K. and Park,S.W. (2O12) Plant disease resistance genes: current status and future directions.Physiol. Mol.Plant Pathol.78,51-65.   
Han,Y.,Mhamdi,A.,Chaouch,S.and Noctor,G.(2013) Regulation of basal and oxidative stress-triggered jasmonic acid-related gene expression by glutathione.Plant, Cell Environ.36,1135-1146.   
Hardy,O.J.and Vekemans, X. (2002) SPAGEDi: a versatile computer program to analyse spatial genetic structure at the individual or population levels. Mol. Ecol. Notes,2, 618-620.   
Henrissat, B.and Bairoch,A.(1993)New families in the clasification of glycosyl hydrolases based on amino-acid-sequence similarities. Biochem.J.293,781- 788.   
Jamaux,I.，Gelie，B.and Lamarque，C.(1995) Early stages of infection of   
rapeseed petals and leaves by Sclerotinia sclerotiorum revealed by scanning electron microscopy.Plant.Pathol.44,22-30.   
Jan，A.，Maruyama，K.，Todaka,D.，Kidokoro,S.，Abo,M.，Yoshimura，E., Shinozaki,K.etal. (2013) OsTZF1，a CCCH-Tandem zinc finger protein, confers delayed senescence and stress tolerance in rice by regulating stressrelated genes.Plant Physiol.161,1202-1216.   
Jiang,Y.and Deyholos,M.K.(2oo9) Functional characterization of Arabidopsis NaCl-inducible WRKY25 and WRKY33 transcription factors in abiotic stresses. Plant Mol.Biol.69,91-105.   
Keen,N.T.and Yoshikawa,M.(1983) Beta-1,3-endoglucanase from soybean   
releases elicitor active carbohydrates from fungus cell-wals.Plant Physiol.71, 460-465.   
Kheiri，H.-R.，Mostafa，M.，Mohammad-Reza，Z.and Ali，D.(2014)Beta glucanase (Bgn13.1）expressed in transgenic Brassica napus confers antifungal activityagainst Sclerotinia sclerotiorum.J. Crop Prot.3, 31-42.   
Klarzynski,O.,Plesse,B., Joubert,J.M.,Yvin,J.C.，Kopp,M.,Kloareg,B.and   
Fritig,B.(2ooo)Linear beta-1,3glucansare elicitorsof defenseresponses in tobacco.Plant Physiol.124,1027-1038.   
Kohler,A.Raldi，C.，uplesis,.，Bucher,.，Geelen,D,uchay. Meyers，B.C.et al. (20o8) Genome-wide identification of NBS resistance genes in Populus trichocarpa.Plant Mol.Biol.66,619-636.   
Kollers,S.,Rodemann, B.,Ling,J.,Korzun,V.,Ebmeyer,E.,Argilier,O.,Hinze,   
M.et al. (2014) Genome-wide association mapping of tan spot resistance (Pyrenophora tritici-repentis)in European winter wheat. Mol. Breeding 34, 363-371.   
Kumar，D.， Datta，R.， Hazra，S.， Sultana，A.，Mukhopadhyay，R.and Chattopadhyay，S.(2015)Transcriptomic profiling of Arabidopsis thaliana Mutant pad2.1 in response to combined cold and osmotic stress. PLoS ONE, 10,e0122690.   
Kump,K.L.，Bradbury,P.J.,Wiser,R.J., Buckler,E.S., Belcher,A.R.,OropezaRosas,M.A.,Zwonitzer,J.C.et al. (2011) Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population. Nat. Genet. 43,163-168.   
Lee,K.H.,Piao,H.L., Kim, H.Y., Choi, S.M., Jiang,F.,Hartung,W., Hwang,I. et al.(2oo6) Activation of glucosidase via stress-induced polymerization rapidly increases active pools of abscisic acid. Cell,126,1109-1120.   
Li,Y., Kiddle,G.,Bennett,R.N.and Wallsgrove,R.M.(1999)Local and systemic changes in glucosinolates in Chinese and European cultivars of oilseed rape (Brassica napus L.) after inoculation with Sclerotinia sclerotiorum (stem rot). Ann. Appl. Biol. 134,45-58.   
Li,F.,Chen,B.,Xu,K.,Wu,J.,Song,W.,Bancroft,I.,Harper,ALetal.(214a) Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (Brassica napus L.). DNA Res.21,355- 367.   
Li,W.,Chen,W., Yang, C.,Wang,J., Yang,L.,He,M.,Wang,J.et al. (2014b) Identification and network construction of zinc finger protein (ZFP) genes involved in the rice-Magnaporthe oryzae interaction.Plant Omics J.7,540- 548.   
Li,J.,Zhao,Z,Hayward,A.,Cheng, H.and Fu,D.(2O15) Integration analysisof quantitative trait loci forresistance to Sclerotinia sclerotiorum in Brassica napus.Euphytica,205,483-489.   
Liang,W.,Yang,B.,Yu,B.J.,Zhou,Z.,Li,C.,Jia,M.，Sun,Y.et al. (2013) Identification and analysis of MKK and MPK gene families in canola (Brassica napus L.).BMC Genom.14,392.   
Liao,W.H.,Ji,L.X.,Wang,J., Chen,Z.,Ye,M.X., Ma,H.D.and An,X.M.(2014) Identification of glutathione S-transferase genes responding to pathogen infestation in Populus tomentosa.Funct. Integr.Genomics,14,517-529.   
Liu,Z.H.,Friesen,T.L.,Ling,H.，Meinhardt, S.W.,Oliver,R.P.，Rasmussen,J.B. and Faris,J.D.(2006) The Tsn1-ToxA interaction in the wheat-Stagonospora nodorum pathosystem parallels that of the wheat-tan spot system.Genome, 49, 1265-1273.   
Lorenz,A.J.，Hamblin,M.T.and Jannink,J.L.(2010) Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley. PLoS ONE,5,e14079.   
Lu,G.,Harper,A.L.,Trick,M.,Morgan,C.,Fraser,F.,O'Neil, C.and Bancroft,I. (2014)Associative transcriptomics study dissects the genetic architecture of seed glucosinolate content in Brassca napus. DNA Res.21,613-625.   
Maere,S.,Heymans,K.and Kuiper,M. (2005) BiNGO:a Cytoscape plugin to assessoverrepresentation of Gene Ontology categories in Biological Networks.Bioinformatics,21,3448-3449.   
Mei, J.Q.,Wei, D.Y., Disi,J.O.,Ding,Y.J.,Liu,Y.and Qian,W. (2012) Screening resistance against Sclerotinia sclerotiorum in Brassica crops with use of detached stem assay under controlld environment.Eur.J.Plant Pathol.134, 599-604.   
Mei，J.，Ding，Y.，Lu,K.，Wei，D.，Liu,Y.，Disi， J.O.，Li，J.et al.(2013) Identification of genomic regions involved in resistance against Sclerotinia sclerotiorum from wild Brassica oleracea.Theor.Appl.Genet.126，549- 556.   
Mengiste,T.(2012) Plant immunity to necrotrophs.Annu. Rev. Phytopathol. 50, 267-294.   
Miedes,E., Vanholme,R., Boerjan,W.and Molina,A.(2014) The role of the secondary cell wallin plant resistance to pathogens.Front. Plant Sci.5,358.   
Morant,A.V.,Jorgensen,K.,Jorgensen,C.,Paquette,S.M.,Sanchez-Perez,R., Moller，B.L.and Bak，S.(2o08) Beta-glucosidases as detonators of plant chemical defense.Phytochemistry,69,1795-1813.   
Nordborg,M.and Weigel,D.(2oo8)Next-generation genetics in plants. Nature, 456, 720-723.   
Owens，B.F.，Lipka，A.E.，Magallanes-Lundback，M.，Tiede,T.,Diepenbrock, C.H.,Kandianis,C.B.,，Kim,E.et al. (2014)A foundation for provitamin A biofortification of maize: genome-wide association and genomic prediction   
Parisy,V.,Poinssot, B.,Owsianowski,L.,Buchala,A., Glazebrook,J.and Mauch, F.(2007) Identification of PAD2 as a gamma-glutamylcysteine synthetase highlights the importance of glutathione in disease resistance of Arabidopsis. Plant J. 49,159-172.   
Pifanell, P.,Ramsay,L.,Waugh,R.,Benabdelmouna,A.,D'Hont,A.,Hollricher, K.,Jorgensen,J.H.et al. (2o04)Abarley cultivation-associated polymorphism conveys resistance to powdery mildew.Nature,430,887-891.   
Porter,B.W.,Paidi,M.,Ming,R.,Alam,M., Nishijima,W.T.and Zhu,Y.J.(2009) Genome-wide analysis of Carica papaya reveals a small NBS resistance gene family. Mol. Genet. Genomics,281,609-626.   
Price,A.L.，Patterson,N.J.，Plenge,R.M.,，WeinblattM.E.，Shadick，N.A.and Reich,D.(2oo6) Principal components analysis corrects for stratification in genome-wide association studies.Nat. Genet. 38, 904-909.   
Purdy，L.H.(1979)Sclerotinia Sclerotiorum: history，diseases and symptomatology, host range,geographic-distribution,and impact.Phytopathology, 69,875-880.   
Rausch,T.and Wachter,A.(2oo5) Sulfur metabolism: a versatile platform for launching defence operations.Trends Plant Sci.10, 503-509.   
Seo，E.and Choi，D.(2015) Functional studies of transcription factors involved in plant defenses in the genomics era.Brief.Funct. Genomics,14, 260-267.   
Sexton，A.C.,Whitten,A.R.and HowlettB.J.(2006) Population structure of Sclerotinia sclerotiorum inan Australian canola field at flowering and steminfection stages of the disease cycle.Genome,49,1408-1415.   
Sharma,P.,Meena,P.D.,Verma,P.R., Saharan,G.S.,Mehta,N., Singh,D.and Kumar,A.(2015) Sclerotinia sclerotiorum (Lib.) de Bary causing Sclerotinia rot in oilseed Brassicas: a review.J. Oilseed Brassica,6,1-44.   
Shi,H.Y.,Liu,Z.H.,Zhu,L.,Zhang,C.J.,Chen,Y.,Zhou,Y.,Li,F.G.etal. (2012) Overexpression of cotton (Gossypium hirsutum） dirigent1 gene enhances lignification that blocks the spread of Verticillium dahliae.Acta Biochim. Biophys. Sin.44,555-564.   
Simmons,C.R.,Grant,S.,Altier,D.J.,Dowd,P.F.,Crasta,O.,Folkerts,O.and Yalpani，N.(2o01)Maize rhm1 resistance to Bipolaris maydis is associated with few differences in pathogenesis-related proteins and global mRNA profiles.Mol.Plant Microbe Interact.14,947-954.   
Singh，S.，Chand，S.，Singh，N.K.and Sharma，T.R.(2015)Genome-Wide distribution,organisation and functional characterization of disease resistance and defence response genes across rice species. PLoS ONE,10,e0125964.   
Slavov,G.T.,Nipper,R.,Robson,P.,Farrar,K.,Allison,G.G., Bosch,M., CliftonBrown,J.C.et al.(2014) Genome-wide association studies and prediction of 17 traits related to phenology，biomass and cell wall composition in the energy grass Miscanthus sinensis.New Phytol.201,1227-1239.   
Smoot,M.E.，Ono，K.，Ruscheinski,J.，Wang，P.L.and Ideker，T.(2011) Cytoscape 2.8:new features for data integration and network visualization. Bioinformatics,27,431-32   
Son,G.H.,Wan,J.，Kim，H.J.，Nguyen,X.C.，Chung,W.S.，Hong,J.C.and Stacey，G.(2012) Ethylene-responsive element-binding factor 5,ERF5，is involved inchitin-induced innate immunity response.Mol.Plant Microbe Interact.25,48-60.   
Soule,M.，Porter,L.，Medina,J.,Santana,G.P., Blair,M.W.and Miklas,P.N. (2011) Comparative QTL map for white mold resistance in common bean, and characterization of partial resistance in dry bean lines VA19 and I9365- 31. Crop Sci. 51, 123-139.   
Spindel,J., Begum,H.,Akdemir, D., Virk,P., Collard,B., Redona,E.,Atlin,G. etal.(2015)Genomic selection and association mapping in rice (Oryza sativa):effect of trait genetic architecture,training population composition, marker number and statistical model on accuracy of rice genomic selection in elite,tropical rice breeding lines.PLoS Genet.11,e1004982.   
Stintzi,A.，Heitz,T.，Prasad,V.，Wiedemannmerdinoglu,S.，Kauffmann,S., Geoffroy，P.,Legrand,M.et al. (1993) Plant pathogenesis-related proteins and their role in defense against pathogens. Biochimie,75,687-706.   
Sun,Y.,Wang, C., Yang,B.,Wu,F.,Hao,X., Liang,W., Niu,F.et al. (2014) Identification and functional analysis of mitogen-activated protein kinase kinase kinase (MAPKKK) genes in canola (Brassica napus L.).J.Exp.Bot.65, 2171-2188.   
Taylor,A.,Coventry,E.,Jones,J.E.and Clarkson,J.P.(2015) Resistance to a highlyaggressve isolate of Sclerotinia sclerotiorum ina Brassca napus divit:>+ Dln+ D+h>l cA ə） An   
Thimm,O., Blasing,O.,Gibon,Y.，Nagel,A.,Meyer,S.，Kruger,P.，Selbig,J. et al. (20o4) MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes. Plant J.37, 914-939.   
Trapnell,C., Roberts,A., Goff,L.,Pertea,G., Kim,D., Kelley, D.R.,Pimentel, H. et al.(2012) Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cuflinks.Nat. Protoc.7,562-578.   
Turner，S.D.(2014) qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots.   
Vanholme,R.,Morree,K.,Ralph,J.andBoerjan,W.(2oo8)Lignin engineering. Curr.Opin.Plant Biol.11,278-285.   
Wang,M.,Yan,J.,Zhao,J.，Song,W.，Zhang,X.，Xiao,Y.and Zheng，Y. (2012a) Genome-wide association study (GWAS) of resistance to head smut in maize.Plant Sci.196,125-131.   
Wang,Z.,Tan,X.L., Zhang, Z.Y., Gu,S.L., Li, G.Y.and Shi, H.F. (2012b) Defense to Sclerotinia sclerotiorum in oilseed rape isassociated with the sequential activations of salicylic acid signaling and jasmonic acid signaling.Plant Sci. 184, 75-82.   
Wang, C.,Yang,Y., Yuan,X.,Xu,Q.,Feng,Y.,Yu,H.,Wang,Y.et al. (2014a) Genome-wide association study of blast resistance in indica rice.BMC Plant Biol. 14, 311.   
Wang,Z.,Fang,H.，Chen,Y.,Chen,K.,Li,G.,Gu,S.and Tan,X.(2014b) Overexpression of BnWRKY33 in oilseed rape enhances resistance to Sclerotinia sclerotiorum.Mol.Plant Pathol.15,677-689.   
Wei,D.,Mei,J.,Fu,Y.,Disi,J.O.,Li,J.and Qian,W. (2014) Quantitative trait loci analyses for resistance to Sclerotinia sclerotiorumand flowering time in Brassica napus.Mol. Breeding,34,1797-1804. Wingate,V.P.M.,Lawton,M.A.and Lamb,C.J.(1988) Glutathione causes a massive and selective induction of plant defense genes.Plant Physiol. 87, 206-210.   
Wittstock, U.and Halkier, B.A.(2oo2) Glucosinolate research in the Arabidopsis era.Trends Plant Sci.7,263-270.   
Wu,J., Cai, G.,Tu,J.,Li,L., Liu,S.,Luo,X., Zhou,L.et al. (2O13) Identification of QTLs for resistance to sclerotinia stem rot and BnaC.IGMT5.aasa candidate gene of the major resistant QTL SRC6 in Brassica napus. PLoS ONE, 8, e67740.   
Wurschum,T.,Abel, S.and Zhao,Y.S.(2014) Potential of genomic selection in rapeseed (Brassica napus L.)breeding.Plant Breeding,133,45-51.   
Xu,L.,Zhu,L.F.,Tu,L., Liu,L.L, Yuan, D.J.,Jin,L.,Long,L.et al. (2o11) Lignin metabolism has a central role in the resistance of cotton to the wilt fungus Verticillium dahliae as revealed by RNA-Seq-dependent transcriptional analysis and histochemistry.J. Exp. Bot.62,5607-5621.   
Yang,B.,Srivastava,S., Deyholos,M.K.and Kav, N.N.V. (20o7) Transcriptional profiling of canola (Brassica napus L.）responses to the fungal pathogen Sclerotinia sclerotiorum.Plant Sci.173,156-171.   
Yang,B.， Jiang，Y.，Rahman，M.H.，Deyholos，M.K.and Kav,N.N.(2009) Identification and expression analysis of WRKY transcription factor genes in canola (Brassica napus L.) in response to fungal pathogens and hormone treatments.BMC Plant Biol.9,68.   
Ye,J.,Fang,L.,Zheng,H., Zhang,Y., Chen,J.,Zhang,Z.,Wang,J.et al. (2006) WEGO:a web tool for ploting GO annotations.Nucleic Acids Res.34, W293-W297.   
Yin,X.R.,Yi,B.,Chen,W., Zhang,W.J.,Tu,J.X.,Fernando,W.G.D.and Fu,T.D. (2010)Mapping of QTLs detected in a Brassica napus DH population for resistance to Sclerotinia sclerotiorum in multiple environments.Euphytica, 173, 25-35.   
Yu,L.X.,Morgounov,A.,Wanyera,R.， Keser,M., Singh,S.K.and Sorrells,M. (2014)Identification of Ug99 stem rust resistance loci in winter wheat germplasm using genome-wide association analysis.Theor. Appl. Genet. 125, 749-758.   
Zhang,S.and Klessig, D.F. (2oo1) MAPK cascades in plant defense signaling. Trends Plant Sci.6,520-527.   
Zhang,D.,Bowden,R.L.,Yu,J.,Carver,B.F.and Bai,G.(2014a) Association analysis of stem rust resistance in U.S.winter wheat.PLoS ONE,9,e103747.   
Zhang,Z.,Ober,U.,Erbe,M.,Zhang,H.,Gao,N.,He,J.L.,Li,J.Q.et al. (2014b) Improving the accuracy of whole genome prediction for complex traits using the results of genome wide association studies.PLoS ONE,9，e93017.   
Zhao,J.and Meng, J. (2oo3) Genetic analysis of loci associated with partial resistance to Sclerotinia sclerotiorum in rapeseed (Brassica napus L.).Theor. Appl. Genet. 106,759-764.   
Zhao,J., Udall,J.A., Quijada,P.A., Grau,C.R., Meng,J.and Osborn,T.C.(2006) Quantitative trait loci for resistanceto Sclerotinia sclerotiorum andits association with a homeologous non-reciprocal transposition in Brassica napus L. Theor. Appl. Genet. 112, 509-516.   
Zhao,J.,Wang,J.,An,L., Doerge, R.W., Chen,Z.J.,Grau,C.R.,Meng,J. et al. (2007）Analysis of gene expression profiles in response to Sclerotinia sclerotiorum in Brassica napus.Planta,227,13-24.   
Zhao,J.W., Buchwaldt, L., Rimmer, S.R.,Sharpe,A.,McGregor,L., Bekkaoui,D. and Hegedus,D. (2oo9) Patterns of differential gene expresson in Brassica napus cultivars infected with Sclerotinia sclerotiorum.Mol.Plant Pathol.10, 635-649.   
Zou,J.,Rodriguez-Zas,S.,Aldea,M.,Li,M.,Zhu,J.,Gonzalez,D.O.,Vodkin, L.O.et al. (2oo5) Expression profiling soybean response to Pseudomonas syringaerevealsnew defense-related genes and rapid HR-specific downregulation of photosynthesis.Mol.Plant Microbe Interact.18,1161- 1174.

# Supporting information

Additional Supporting information may be found in the online version of this article:

Figure S1 Distribution of relative kinship values in 347 Brassica napus accessions.   
Figure S2 Genes that are differentially expressed in the resistant (R)and susceptible (S) Brassica napus lines following Sclerotinia sclerotiorum infection.   
Figure S3 GO categories of DEGs in the stem of Brassica napus inducedbySclerotiniasclerotiorum.   
Table S1 Accessions used in the study.   
Table S2 Primers used for qRT-qPCR verification.   
Table S3 ANOVA of stem resistance in Brassica napus.   
Table S4 SNPsused for genome-wide associationanalysis. Table S5 Mean prediction ability (r) and standard deviation (SD) in 243 Brassica napus lines based on different training populations and markers   
Table S6 Summary of RNA-Seq reads.   
Table S7 Differentially expressed genes in transcription factor families.   
Table S8 List of KEGG pathways mapped by common up- and down- regulated genes.   
Table S9 Specific up-regulated genes involved in jasmonic acid ligninbiosynthesis，signal transduction，stressresponseand transcription factors in resistant (R) Brassica napus genotype.