RESEARCHARTICLE

# FUS Interacts with HSP6O to Promote Mitochondrial Damage

CrossMark click for updates

Citation: Deng J,Yang M,Chen Y,Chen X,Liu J, Sun S,et al. (2015) FUS Interacts with HSP60 to Promote Mitochondrial Damage.PLoS Genet 11(9): e1005357.doi:10.1371/journal.pgen.1005357

Editor: Bingwei Lu, Stanford University School of Medicine, UNITED STATES

Received: February 24,2015

Accepted: June 12,2015

Published: September 3,2015

Copyright: $\circledcirc$ 2015 Deng et al.This is an open accessarticle distributed under the terms of the Creative Commons Attribution License,which permits unrestricted use,distribution,and reproduction in any medium,provided the original author and source are credited.

DataAvailability Statement:All relevantdataare within the paper and its Supporting Information files.

Funding: LZ and JL are supported by MOST China 973Projects (2010CB529603&2013CB917803)and NSFC grant (91132710).YC and QXare supported by MOST China 973 Project (2010CB529603).EHB and MM are supported by NIA P30AG13854.JYW has been supported by ALS Therapy Alliance and National Institutes of Health [R56NS074763, U54 CA151880 (PI: C.Mirkin) and RO1AG033004).The funders had no role in study design,data collection and analysis,decision to publish,or preparation of the manuscript.

Jianwen Deng1,2, Mengxue Yang1,2,3, Yanbo Chen3,4, Xiaoping Chen³, Jianghong Liu1, Shufeng Sun', Haipeng Cheng³,Yang $L i ^ { 3 , 5 }$ ,Eileen H. Bigio, Marsel Mesulam, Qi $\mathsf { X } \mathsf { u } ^ { 4 }$ Sidan ${ \mathsf { D } } { \mathsf { u } } ^ { 5 }$ ,Kazuo Fushimi²,LiZhu1 \*, Jane Y. $\boldsymbol { \mathsf { W } } \boldsymbol { \mathsf { u } } ^ { 1 , 3 * }$ （204号

1StateKey Laboratory of Brainand Cognitive Science,Instituteof Biophysics,Chinese Academy of Sciences,Beijing,China,2 Universityof Chinese Academy of Sciences,Beijing, China,3 Departmentof Neurology,Centerfor Genetic Medicine,Lurie Cancer Center,Northwestern UniversityFeinberg Schoolof Medicine, Chicago,Ilinois,United States of America,4 National Laboratory of Medical Molecular Biology, Institute of Basic Medical Sciences, Chinese Academy of Medical Science and Peking Union Medical College,Tsinghua University,Beijing,China,5 Schoolof Electronic Science and Engineering,Nanjing University,Nanjing,China,6 Departmentof Pathology& Neurology,The Cognitive Neurology& Alzheimer's Disease Center,Northwestern UniversityFeinberg School of Medicine,Chicago,Illinois,United Statesof America

\* zhuli@ moon.ibp.ac.cn (LZ); jane-wu @northwestern.edu (JYW)

# Abstract

FUS-proteinopathies,a group of heterogeneous disorders including ALS-FUS and FTLDFUS,are characterized by the formation of inclusion bodies containing the nuclear protein FUS in the affected patients. However,the underlying molecular and cellular defects remain unclear.Here we provide evidence for mitochondrial localization of FUS and its induction of mitochondrial damage. Remarkably, FTLD-FUS brain samples show increased FUS expression and mitochondrial defects. Biochemical and genetic data demonstrate that FUS interacts with a mitochondrial chaperonin, HSP60,and that FUS translocation to mitochondria is,at least in part, mediated by HSP60. Down-regulating HSP60 reduces mitochondrially localized FUS and partially rescues mitochondrial defects and neurodegenerative phenotypes caused by FUS expression in transgenic flies.This is the first report of direct mitochondrial targeting by a nuclear protein associated with neurodegeneration,suggesting that mitochondrial impairment may representa critical eventin different forms of FUS-proteinopathies and a common pathological feature for both ALS-FUS and FTLD-FUS. Our study offers a potential explanation for the highly heterogeneous nature and complex genetic presentation of different forms of FUS-proteinopathies. Our data also suggest that mitochondrial damage may be a target in future development of diagnostic and therapeutic tools for FUS-proteinopathies,a group of devastating neurodegenerative diseases.

# Author Summary

Amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration (FTLD) are two groups of common and devastating neurodegenerative diseases, characterized by losses of selected groups of neurons. Mutations in the FUS gene have been associated with

Competing Interests:The authors have declared that no competing interests exist.

ALS, whereas inclusion bodies containing the FUS protein have been discovered in both ALS and FTLD patients. However, the underlying pathogenic mechanisms of FUS in these diseases remain unclear. Here, we demonstrate that wild-type or ALS-associated mutant FUS can interact with mitochondrial chaperonin HSP60 and that HSP60 mediates FUS localization to mitochondria,leading to mitochondrial damage. Mitochondrial impairment may be an early event in FUS proteinopathies and represent a potential therapeutic target for treating these fatal diseases.

# Introduction

Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease primarily affecting motor neurons.The $\mathrm { { C u / Z n } }$ superoxide dismutase 1(SOD1) gene was the first ALS-associated gene whose mutations were identified in familial ALS (fALS) patients [1,2]. Subsequently, genetic studies have uncovered more than ten ALS-associated genes [3,4,5]. Among these are genes encoding RNA/DNA binding proteins, including TAR-DNA binding protein of $\underline { { 4 3 } } \mathrm { k D a }$ (TDP-43) and fused in sarcoma/translocated in liposarcoma (FUS/TLS or FUS) [6,7,8,9]. Pathologically,FUS immunoreactive inclusion bodies are detected in a range of neurological diseases classified as FUS-proteinopathies.These disorders are genetically and clinically heterogeneous.Depending on the regions affected,FUS-proteinopathies can manifest as motor neuron disease such as ALS-FUS,or as various forms of dementia including frontotemporal lobar degeneration with FUS pathology (FTLD-FUS),atypical FTLD with ubiquitin pathology (aFTLD-U), neuronal intermediate filament inclusion disease (NIFID),and Basophilic Inclusion Body Disease (BIBD) [4,10,11,12,13,14]. Interestingly,although ${ > } 3 0$ mutations in the FUS gene have been found in patients with ALS,no FUS mutations have been detected in the vast majority of sporadic or familial, pathologically proven cases of FTLD-FUS [15].A recent study has identified several mutations in the ${ \boldsymbol { 3 } } ^ { \prime }$ untranslated region of the FUS gene that are associated with increased FUS expression among ALS patients [16],suggesting that increased FUS expression could be a mechanism contributing to the pathogenesis of ALS.

Several systems have been used to model FUS-proteinopathies,ranging from yeast to vertebrate animals [17,18,19,20,21,22,23,24,25]. A few groups, including Lanson and colleagues as well as our team, have established transgenic flies expressing wild-type (Wt) or ALS-mutant forms of human FUS protein [17,22]. In our transgenic fly model,targeted expresson of either Wt- or ALS-mutant FUS protein in specific neuronal subpopulations leads to age-dependent neurodegeneration with functional deficits,recapitulating the critical features ofFUS proteinopathies [17]. In transgenic mice, simply overexpressing the wild-type FUS led to progressive neurodegeneration [26]. Because no FUS mutation has been detected in most FTLD-FUS patients and because the patient samples that we examined showed elevated FUS protein levels (see below),the work in this study examining the effects of increased Wt-FUS expression is pertinent to understanding FTLD-FUS,whereas the data with ALS-mutant FUS, such as ${ \mathrm { P } } 5 2 5 { \mathrm { L } } ,$ is relevant to ALS-FUS.

To understand the biological function of mammalian FUS in the nervous system, we searched for interaction partners ofFUS. Using a FUS-specific monoclonal antibody (S1 Fig; also see [27])we developed an immunopurification-coupled mass spectrometry approach to identify proteins that interact with FUS in the bovine brain tissue. Among candidate FUS interaction partners,several mitochondrial proteins were identified,including HSP6o (see below; detailed data to be reported in a separate study). Consistent with this finding, the endogenous FUS protein was detected by mass-spectrometry in biochemically-purified mitochondria

(without using the FUS antibody or overexpressing FUS),supporting the idea that FUS interacts with mitochondria. These observations prompted us to carefully examine mitochondria in our models for FUS proteinopathies.

HSP60 proteins are a family of evolutionarily conserved ATP-dependent chaperones that play important roles in stress response, protein folding and cell signaling [28,29,30]. They are expressed constitutively as well as in response to stress signals [31,32]. HSP60 proteins are detected in the cytosol and inside the mitochondrial matrix.It has been reported that HSP60, together with other heat shock proteins such as HSP10 and HSP70,facilitates proper protein folding and assembly of protein complexes imported into mitochondria [30,32,33,34,35,36]. Mutations in the HSPDl gene (encoding the human HSP6O protein) have been found in patients with spastic paraplegia type 13 (SPG13),a late-onset autosomal-dominant neurodegenerative disease characterized by progressive weakness and spasticity of lower limbs [37,38].

Mitochondrial impairment has been extensively investigated in ALS,in particular,in SOD1 animal models [39,40,41,42,43]. Aggregated mitochondria have been reported in transgenic mice overexpressing TDP-43 [44,45]. An EM study of spinal cord samples of two cases of ALS-FUS,including one containing P525L mutation, revealed disorganized mitochondria and endoplasmic reticulum [46]. Cytoplasmic expression of two other ALS-associated FUS mutants, R521G or R521H,was associated with shortened mitochondria in cultured motor neurons [47]. These studies suggest mitochondrial damage may be a common feature in ALS-- FUS.However,no evidence has been reported for the mitochondrial localization ofFUS or for mitochondrial damage in FTLD-FUS patients. Although it remains to be determined if mitochondrial impairment is a direct consequence of FUS expression, our data presented here show that mitochondrial fragmentation was detected not only in vitro in cultured neurons but also in vivo in motor neurons in the FUS-transgenic flies. Both Wt- and ALS-mutant P525L FUS interacted with HSP60.Furthermore, mitochondrial damage was detected in brain samples of FTLD-FUS patients, with FUS levels increased in all 3 FTLD-FUS patient brain samples examined.Remarkably, elevated HSP6O expression was detected in two of these 3 cases of FTLD-FUS patient brain samples. Knocking-down HSP60 led to reduced level of mitochondrial FUS in cultured cels.Interestingly,RNAi-mediated down-regulation of the HSP60 homolog partially rescued the neurodegenerative phenotypes in FUS transgenic flies. Thus, increased FUS expression and mitochondrial impairment appear as a prominent pathological feature in FTLD-FUS. Our data also uncover the previously unknown regulation of FUS subcelllar distribution by chaperon proteins such as HSP60,suggesting a new direction for treating FUS proteinopathies by modulating mitochondrial localization of FUS and protecting against FUS-induced mitochondrial damage.

# Results

# Expression of Wt- or ALS-associated mutant FUS led to mitochondrial fragmentation in mammalian neuron-like cells, cultured neurons and FUS transgenic flies

To examine ifFUS expression affects mitochondria in mammalian cells, we expressed FUS in neuron-like cells,HT-22,an immortalized mouse cell line with neuronal features [48].To monitor changes in the mitochondrial morphology,a plasmid expressing mitochondrion-targeted red fluorescent protein (RFP) (mito-Red) was co-transfected with either a GFP vector control or a plasmid expressing GFP-tagged FUS protein into HT-22 cells (Fig 1A). The majority of cells expressing GFP vector control showed mitochondria with tubular morphology (Fig lA). However,the percentage of the cells with fragmented mitochondria was significantly

![](images/6406eb3f24fb82c01b505a34790859f09d5b5d85b1352a573e2f26682a885894.jpg)  
Fig 1.Expression of Wt-or ALS-mutant FUS in HT22 neuron-like cells led to mitochondrial fragmentation. (A)The mitoRed plasmid was co-transfected together with plasmids expressing either the control GFP vector(Ctr),or Wt-or P525L-mutantFUS into HT22 cells.Cels with tubular,intermediate or fragmented mitochondrial patterns were imaged72 hrs post-transfectionand quantified.Arrowheads mark the cels that expressed exogenous FUS and showed fragmented mitochondria(with their nuclei marked by \*\*"); whereas the arrows mark adjacent non-transfected cels showing tubular mitochondria. Insets show the boxed areasata higher magnification.(B)Quantification of mitochondrialfragmentation from experiments shown in panel A.The data were analyzed using one-way ANOvA with Bonferroni post-test $( ^ { * * } \colon \mathsf { p } { < } 0 . 0 0 1$ \*\*\*: $\mathsf { p } { < } 0 . 0 0 0 1 _ { \cdot }$ ）

doi:10.1371/journal.pgen.1005357.g001

increased when either Wt-FUS or the ALS-mutant P525L was expressed.This mitochondrial change was particularly pronounced in cells expressing the P525L-mutant FUS (Fig 1B).

Next, mito-Red and FUS-GFP were co-expressed in cultured mouse cortical neurons.Fluorescent confocal microscopy revealed a significant increase in the percentage of neurons with fragmented mitochondria when expressing either Wt- or P525L-mutant FUS. The percentage of neurons expressing the P525L-mutant FUS showed fragmented mitochondria is higher than those expressing Wt-FUS (Fig 2A-2C). Some of these neurons expressing FUS also showed condensed or fragmented nuclei,a sign of cell death.It is important to note that in a significant fraction of FUS-expressing HT22 cells or cortical neurons that showed mitochondrial

![](images/844b3291af7c7c37e60d3519a811fc7649062a6122b744e4cc6c8405b5e0c2b7.jpg)  
Fig2.ExpressfWtorLmutantSinpriaryorticaleuros(-C)orflyotoreros(Fedtitodrialfragentatie mitoRedplasdraseetisexpintrFctCtrt2utattu cortiusiatotrodcboedi InpaneArobosolodriasoadskuosongedidrib nucleiwithooo#arktdesdreedcsptois)aia areasareshossatoirseiasinpel.(D-F).USexpesinfotouros(Ms)ldidral fragntati.toaoelipresnsa.foalo fotoreuroodtaced Ctr:D42-Ga4/UAS-mitoGFP/UAS-RFP;Wt:D42-Gal/UAS-mitoGFP/UAS-W-FUS-RFP;P525L:D42-Gal/UAS-mitoGFP/UAS-P525L-FUS-RFP.(E) QuaticationofdazistWutatUxpresssodicantyeadsf smaermiohodriautfweiochondaitrgersssompadithteotroloupenaaeerequantfdinacgoup.() Quatficatioolsodsaexputaota ascompareditthotroieTeeremeasuredinacgrouAlldataereanaldusngoaAitfeot (\*: $\mathsf { p } { < } 0 . 0 5$ $^ { * * }$ ${ \mathsf { p } } { < } 0 . 0 1$ $\ntimes \ntimes \ntimes$ $\mathsf { p } { < } 0 . 0 0 0 1 \$ ！

doi:10.1371/journal.pgen.1005357.g002

fragmentation, there was no detectable sign of nuclear morphological changes characteristic of cell death, suggesting that mitochondrial changes may be an early event, preceding cell death. These data indicate that increased expression of FUS,especially the P525L-mutant, promotes mitochondrial fragmentation and cell death in mammalian neurons.

To test in vivo effects of FUS expression, we examined motor neurons (MNs) expressing human FUS in transgenic flies [17]. The D42-Gal4 driver [49,50] was used to express either Wt- or P525L-mutant FUS specifically in Drosophila MNs.Mitochondria in MN axons were examined with confocal microscopy by the expression of mito-GFP in wandering 3rd instar larvae (Fig 2D). It should be noted that the flies expressing the P525L-mutant FUS used in this study were from a line with a less severe phenotype [17],because other P525L-mutant FUS lines exhibiting more severe phenotypes did not survive to the late larval stage.As compared with the control flies,axonal mitochondria were significantly smaller in MNs expressing the Wt- or P525L-mutant FUS protein. MNs expressing the human FUS protein showed an increase in the percentage of smallermitochondria (length $< 0 . 5 \mu \mathrm { m }$ )andadecrease inthe percentage of larger mitochondria (length ${ > } 1 . 5 { \mu \mathrm { m } }$ 0 $\mathrm { p } { < } 0 . 0 5 \$ ),with P525L-mutant exhibiting more severe defects than the Wt-FUS ( $\mathrm { ( p < 0 . 0 1 }$ ; Fig 2E). Consistently,the average mitochondrial size in the FUS-expressing fly MNs were significantly smaller than that in the control group （204号 $\mathrm { ( p < 0 . 0 1 ) }$ ,with the P525L-mutant expressing flies showing further shortened average mitochondrial length than the Wt- group $( \mathrm { p } { < } 0 . 0 5$ ; Fig 2F). These data indicate that the expression of either Wt- or P525L-mutant FUS induces mitochondrial defects in vivo,with the ALSmutant eliciting a more severe phenotype.This is consistent with our previous observation thal neurodegeneration phenotypes in flies expressing the P525L-mutant were more severe than those expressing Wt-FUS [17].

Interestingly,cells transfected with either the Wt- or P525L-mutant FUS consistently showed a moderate increase in the levels of Fisl and Drpl proteins without affecting mitofusin 2.A decrease in the level of Drpl phosphorylated at its amino acid residue 637 was also detected (S2A-S2E Fig). This prompted us to test whether inhibiting Drpl function by a dominant-negative mutant K38A-Drpl could reverse FUS-induced mitochondrial fragmentation in cultured neurons. As shown in S3A-S3C Fig, we co-transfected mitoRed and GFP control, WtFUS-GFP or P525L-FUS-GFP together with either the vector control, wild type Drp1-Flag (WtDrp1) or K38A mutant Drpl-Flag (K38A-Drp1). Expression of Wt-Drpl moderately increased the percentage of neurons with fragmented mitochondria as compared with the control group, whereas expression of K38A-Drpl mutant decreased the percentage of cells containing fragmented mitochondria, especially in the neurons expressing the P525L-mutant FUS (S3A-S3C Fig). Moreover, the percentage of neurons showing condensed nuclei was also reduced by the expression of K38A-Drpl in those expressing the P525L-mutant FUS. It prompted us to test if inhibiting Drpl in FUS transgenic flies would rescue the neurodegenerative phenotypes. However, expression of Drpl-specific RNAi or a dominant-negative K38A-Drplmutantdid not lead to a significant rescue of neurodegeneration phenotypes in the FUS transgenic flies expressing Wt- or P525L-FUS protein (S3D and S3E Fig). It has been reported that a dominant- negative mutation in Drpl is associated with a severe neurodevelopmental syndrome in a patient and down-regulation of Drp1 leads to developmental defects in flies [51,52]. It is possible that simply suppressing Drpl activity is not sufficient to block the FUS-induced neurotoxic signal(s).

# FUS is associated with mitochondria, with the ALS-associated P525Lmutant showing increased FUS mitochondrial localization

To examine the relationship between FUS and mitochondria, we purified mitochondria from the HEK293 cells expressing the vector control or the ALS-associatedP525L-mutant FUS following published protocols [53,54] (Fig 3). Western blotting experiments demonstrate that our mitochondrial preparations were highly enriched in mitochondrial proteins (such as CoxIV) but devoid of either cytoplasmic proteins (such as GAPDH) or nuclear proteins (e.g., Histone

![](images/b5d201219614dba947f9022ac488f3367b484911b6801f63e783b395918e7e37.jpg)  
Fig 3.TheFUS protein is associated with mitochondria. (A)Highly purified mitochondria were prepared fromthe control or P525L-FUS-expressing stable HEKcelines.The mitochondrial purity was confirmed by the detection of mitochondrial CoxlVand theabsence of the cytoplasmic proteins suchas GAPDHor nuclear protein Histone H3.The endogenous FUSor P525L-mutant FUS localized to mitochondria;and the P525LmutantFUS showed increased levels of mitochondrial localization (lane 6),as compared with the endogenous FUS in mitochondria(lane 3). (B-D)IEM images of the controlorFUS-expressing stable HEK cellines show reduced mitochondrial sizes in cells overexpressing FUS.Arrows,FUS-immunostaining signalsassociated with mitochondria labeled with10-nmimmuno-gold particles;arrowheads,mitochondria showed damaged cristae with “onion-like"structure. Mitochondrial cristae in P525L-mutant FUS expressing cells were significantly more frequently disrupted than the controland Wt-FUS groups,with quantification shown i panel C.More than 50 mitochondria were quantified in each group,analyzed using Chi-square test $( ^ { \ast \ast \ast } { : } \mathsf { p } { < } 0 . 0 0 0 1 )$ .(D) Quantification of mitochondrial size using Image J.Mitochondriain Wt-or P525L-mutant FUS expressing cels were significantly smaller than the control group.At least 50 mitochondria were quantified in each group,analyzed using one-way ANOVA with Bonferronipost test $( { \ast } { * } { * } { * }$ $\mathsf { p } { < } 0 . 0 0 0 1 ;$ .(E) Mitochondrion-associated FUS immunostaining signals were significantly increased in Wtor P525L FUS expressing celsas compared with the Ctr.Atleast 60 mitochondria were quantified in each group,analyzed using one-way ANOVA with Bonferronipost test (\*\*: $\mathsf { p } { < } 0 . 0 1$ $\nrightarrow \ast \ast$ 元 $\mathsf { p } { < } 0 . 0 0 0 1 \$ ）

doi:10.1371/journal.pgen.1005357.g003

H3) (Fig 3A). The endogenous FUS protein was consistently detected in these highly purified mitochondrial preparations (lane3 in Fig 3A). Although the level of P525L-mutant FUS protein detected was higher than the endogenous Wt-FUS protein in both the purified mitochondria and cytosol (see Fig 3A,the upper band was the P525L-mutant FUS; the lower band, the endogenous FUS),the endogenous FUS protein was clearly detected in the purified mitochondria (Fig 3A,lane 3 and lane 6). These data demonstrate that both the endogenous wild-type and transfected ALS-mutant FUS are translocated to mitochondria.

To confirm that the FUS protein is indeed closely associated with mitochondria, we performed immuno-electron microscopy (IEM) using the specific anti-FUS antibody.In the control HEK293 cells,a fraction of FUS-IEM staining signals were associated with mitochondria, whereas in cells overexpressing the Wt- or P525L-mutant FUS, many mitochondria were decorated with immuno-gold particles conjugated to the specific anti-FUS antibody (arrows in Fig 3B and 3E). It should be noted that the anti-FUS antibody was specific, because pre-absorption of the antibody using the purified FUS antigen essentially eliminated the immunoreactive signals in Western blotting (WB) and IEM (see S1A-S1D Fig). Consistently, down-regulation of FUS by specific siRNA (siFUS) or genetic deletion of the FUS gene reduced or eliminated the WB signals (see S1C and S1D Fig, respectively).Quantification of IEM images shows that mitochondria in cell expressing either Wt- or P525L-mutant FUS were smaller with severely damaged cristae,as compared with those in the control group (Fig 3B-3D).“Onion-like" mitochondria with multi-layered structure and damaged cristae were frequently detected in cells expressingP525L-mutant FUS (marked by arrowheads in Fig 3B and 3C),but not in the control cells. Consistent with the data described previously,mitochondrial size was significantly decreased in cells expressing either Wt- or P525L-mutant FUS,with the P525L-mutant showing more pronounced mitochondrial defects (Fig 3D). These data support the notion that increased FUS expression leads to mitochondrial damage.Together,our results demonstrate that increased FUS expression,especially that of the cytoplasm-localized ALS-associated P525L-mutant FUS protein, promotes association of FUS with mitochondria and induces mitochondrial damage.

# Expression of Wt- or ALS-associated mutant FUS reduces the mitochondrial membrane potential and increases the production of mitochondrial reactive oxygen species

A common sign of mitochondrial damage is the change in mitochondrial membrane potential, leading to mitochondrial depolarization.We tested ifFUS expression affects mitochondrial membrane potential. Tetramethylrhodamine methyl ester (TMRM),a fluorescent dye labeling mitochondria in a membrane potential-dependent manner [55] was used to stain HEK293 cells expressing either the GFP vector control or Wt- or P525L-mutantFUS tagged with GFP. In cells expressing the control, the TMRM signals were detected at a similar level as in the neighboring non-transfected cells (Fig 4A). However, the TMRM staining intensity in cells expressing Wt-or P525L-mutant FUS (marked by the white arrows) was reduced as compared with their neighboring non-transfected cells (marked by arrowheads).The control and Wt- or P525L-mutant FUS expressing cells were then analyzed by fluorescence activated cell sorting (FACS) to quantify TMRM florescence intensity in GFP-positive cells. Consistently,TMRM intensity was significantly lower in cels expressing either Wt- or P525L-mutant FUS as compared with the GFP vector-control group, with the P525L-mutant group showing more pronounced effect (Fig 4B and 4C).This indicates that FUS expression,in particular that of P525L-mutant, reduces mitochondrial membrane potential and causes mitochondrial damage.

![](images/79c8f92dd14287eb5523250474323dbb39ead2f5485102a05cf48c15b7e13c5d.jpg)  
Fig 4.Increased expression of Wt-or P525L-mutant FUS induced adecrease in mitochondrial membrane potential and an increase in production of mitochondrial superoxide. (A) Confocal images oflive TMRM-stained HEK293 cels following transfection with corresponding plasmids:the GFPcontrol,Wtor P525L-mutant FUS tagged with GFP.Allimages were acquired in Z-stacksand projected as single images.Cells were stained with TMRM(a mitochondrial membrane potentiometric dye)and Hoechst 33342 before imaging.(B,C)FACS analyses of GFP-positive cels from experiments in panel A with quantification of TMRM signal intensity.Cels expressing either Wt-or P525L-mutant FUS showed decreased TMRMintensity as compared with the control group.The data were analyzed using one-way ANOvA (representing 4 independent experiments;\*: ${ \mathsf p } { < } 0 . 0 5$ $\napprox \ast \ast$ ： ${ \mathsf p } { < } 0 . 0 0 1$ ). (D) Confocal images of mitoSox Red-stained HEK293 cells 24-hrs folowing transfectionas described inpanelAand Hoechst 33342 staining.(E,F)FACSanalyses and quantification of mitoSOX-Red stained cels shown in panel D.Cells expressing P525L-mutant FUS showed significantly increased mitoSOX staining signals as compared with cels expressing the control or Wt-FUS protein. Data from 3 independent experiments were analyzed using one-way ANOVA $( ^ { * * } { \cdot } \mathsf { p } { < } 0 . 0 1 )$ ：

doi:10.1371/journal.pgen.1005357.g004

Next, we measured the mitochondrial production of the reactive oxygen species (ROS) in these cells. Using a specific mitochondrial superoxide indicator, mitoSOX, we compared cells expressing either Wt- or P525L-mutant FUS.Both confocal images and FACS analyses showed that ROS production was increased in the P525L-mutant FUS expressing cells (Fig 4D-4F). It has been reported that ROS could activate Drpl and induce mitochondrial fragmentation [56], consistent with our findings of FUS-induced Drpl activation and mitochondrial damage.

# Increased FUS protein levels and profound mitochondrial damage are detected in FTLD-FUS brain samples

To characterize molecular and cellular damages in patients with FUS-proteinopathies, we collected de-identified post-mortem tissue samples from the Cognitive Neurology & Alzheimer's Disease Center at Northwestern University.After initial testing to exclude cases with non-specific protein degradation, three FUS-proteinopathies brains together with six control samples were identified suitable for biochemical studies and electron microscopy.In all three FUS

proteinopathies cases examined, no FUS mutations were identified (see S1 Table) and the pathological diagnosis was FTLD-FUS with prominent FUS-positive inclusion bodies detected in the brain tissues. Using the specific anti-FUS antibody, we examined FUS protein levels in these FTLD-FUS brain samples together with six control samples.Remarkably,althree FTLD-FUS samples showed increased FUS protein levels as compared to the controls (Fig 5A and 5B).This is consistent with a previous report that the brain samples from patients affected by atypical FTLD with FUS pathology (aFTLD-FUS) showed increased total FUS levels,as compared with the controls or FTLD-TDP-43 samples [10]. It should be noted that 2-3 bands were detected in the human brain samples, with approximate molecular weight of 53-70kD, which is consistent with previously published Western blotting data on FUS proteinopathies patient tissue samples [57,58]. It is possible that FUS may undergo proteolytic cleavage,and the molecular nature of such cleavage remains to be determined by future studies. Because our mass-spectrometry analyses showed mitochondrial HSP60 as a FUS-interacting protein, we also examined HSP60 protein in these brain samples and found that two FTLD-FUS cases (A and C) exhibited elevated HSP60 protein level (Fig 5A and 5B).It is not surprising to see that not all FTLD-FUS samples exhibited increased HSP60 expression. Considering the multi-level regulation of FUS gene expression and the diverse genetic background in different individuals, it is conceivable that different mechanisms contribute to the development of FUS-proteinopathies and that HSP60 is not the only modulator of FUS induced neurotoxicity among different FTLD-FUS patients.

We performed IEM of these samples using the FUS-specific antibody and the secondary antibody conjugated to gold particles.In the control samples, most mitochondria appeared healthy with well-organized cristae as packed-stacks of membrane sheets (see panels C1-C6 in Fig 5C) and with only a few FUS-immunostaining signals detected in the vicinity of the mitochondria.However,in allthree FTLD-FUS brain samples,FUS-immuno-positive signals were frequently detected in close association with mitochondria (arrows in panels C9-C13 in Fig 5C and 5D). Consistently,mitochondria in these FTLD-FUS cases showed a marked loss or disruption of cristae with frequent detection of"onion-like”deformed shape (arrowheads in panels C7-C13 in Fig 5C). Quantification of the EM data indicates that allthree FTLD-FUS cases showed increased FUS-immunostaining signals in mitochondria and increased mitochondrial damage (Fig 5D and 5E). These observations indicate that mitochondrial impairment,accompanied by increased FUS expression,represents a prominent neuropathological feature in FTLD-FUS patients.

# HSP60 interacts with FUS and mediates FUS mitochondrial localization

HSP60,an ATP-dependent mitochondrial chaperone protein associated with neurodegeneration [30] was one of the FUS-interacting mitochondrial proteins identified in our immunoaffinity-coupled mass-spectrometry experiments.We confirmed the FUS-HSP60 interaction in a co-immunoprecipitation (co-IP) assay using HEK293 cells transfected with the Wt- or P525Lmutant FUS tagged with GFP or the GFP vector control. Immunoprecipitation to pull-down FUS followed by Western blotting using the HSP60-specific antibody revealed that both Wtand P525L-mutant FUS interacted with HSP60 (Fig 6A). To examine whether FUS interacts with HSP60 in the cytosol or in mitochondria, we prepared mitochondrial and cytosolic fractions from HEK293 cells expressing the control vector,Wt- or P525L-mutant FUS tagged with 6xMyc tag to perform co-IP assay. Western bloting analyses of the immunoprecipitated pro tein revealed that Wt- or P525L-mutant FUS interacted with HSP60 in the mitochondria as well as in the cytosol (Fig 6B).To test whether FUS directly interacted with HSP60,we performed a cross-linking immunoprecipitation assay (Fig 6D, S4B Fig). Following the treatment buffer (see Methods)and used for Western bloting with anti-FUS or HSP60 antibodies.Beta-actin was used asan internal control.In panel Bis quantificationof datain panelA,representing 3 independentsamples (\* $\mathsf { P } { < } 0 . 0 5$ ;\*\*: $\mathsf { p } { < } 0 . 0 1$ ; ANOVA with studentt test). (C) IEMof FTLD-FUS and control brain samples. Cryosections of corresponding brain samples were immuno-labeled with specific anti-FUS antibody and secondary anti-murine IgGconjugated to10-nmgoldparticles.Mitochondria were labeled with rabbit-antiTomm20 and anti-rabbit IgG conjugated to 25-nm gold particles.Arows mark FUS-immunostaining signals labeled with 10-nm immuno-gold particles;arrowheads,mitochondria showing damaged cristae.(D) Mitochondrion-associated FUS immunostaining signals were significantly increased inFTLD-FUS cases as compared with the controls(\*\*\*: $\mathsf { p } { < } 0 . 0 0 0 1 \$ .(E) Mitochondrial cristae showed significant damage in FTLD-FUS samples as compared with the control samples.Atleast 30 mitochondria were quantified in each group and analyzed using Chi-square test $( ^ { \ast \ast } ; \mathsf { p } { < } 0 . 0 1$ ).Datarepresent three independent experiments.

![](images/11175c1406be7f4cb6bc865db4932512c617e5eddb0e2f004da00bc43aad7381.jpg)  
Fig 5. Increased FUS protein levels and mitochondrial damage detected in 3 independent FTLD-FUS brain samples.(A,B)Fronto-corticaltissues of postmortem brain samples from three patients diagnosed with FTLD-FUS (#A,Band C) or 6 control cases (Ctr; #C1-#C6) were lysed in $1 \%$ SDS-containing hot lysis

doi:10.1371/journal.pgen.1005357.g005

with formaldehyde (a cross-linking reagent) of the live cells stably expressing 6xMyc-His tagged FUS,FUS protein was purified by pulling-down with Ni-NTA resin. A $1 3 0 \mathrm { k D a }$ species was detected by anti-HSP60 antibody,representing FUS-HSP60 cross-linked product, because Myc-His tagged FUS and HSP60 were detected as 70 and 60kDa proteins, respectively (marked by the arrow,lane 2 and lane 8 in Fig 6D). Because live cels were treated with the cross-linking reagent immediately before immunoprecipitation in this assay,our data support the notion that FUS directly interacts with HSP60 in cells.In addition, protein purification was performed under the denaturing condition in the presence of guanidine-HCl using Ni-NTA resin, thus it is unlikely that the 130kDa cross-linked product was a result indirect interaction of HSP60 with FUS.To further demonstrate that FUS directly interacts with HSP60 and to map the region in FUS responsible for FUS-HSP60 interaction, we performed glutathione S-transferase (GST) pull-down experiments using purified GST-tagged proteins of either Wt- or P525Lmutant or fragments of the Wt-FUS protein and His-tagged HSP60 protein (see Fig 6C). Although the interaction was not detectable when the N-terminal fragments containing 285 or 370 amino acid residues (aa),the full-length Wt- or P525L-mutant or the carboxyl terminal fragment containing aa371-526 clearly interacted with the purified HSP60 protein (Fig 6C). To test if the FUS-HSP60 interaction is RNA-dependent, we performed GST pull-down experiments in the presence of RNaseA. RNase A treatment did not affect the interaction between FUS and HSP60, although there was partial degradation of FUS and HSP60 proteins following RNaseA treatment (S4A Fig).Together,these data indicate that Wt- or P525L-FUS directly interacts with HSP60 in cells and in vitro.

To test whether HSP60 mediates FUS mitochondrial localization, we down-regulated HSP60 expression using specific siRNA in HEK293 cells and then examined the level of mitochondrion-localized FUS (the endogenous wild-type FUS) using purified mitochondria followed by Western blotting. Although the total FUS levels were not altered, the mitochondrial FUS level was significantly decreased in the HSP60-siRNA transfected cells,as compared with the control-siRNA group (Fig 6E-6G).Furthermore,analyses of the nuclear and cytosolic fractions revealed that the nuclear FUS level was increased in the HSP60-siRNA transfected cells, as compared with the control-siRNA group (S4C-S4E Fig). We also performed siRNA knockdown experiments in the stable cells expressing the P525L-mutant FUS.Similarly,reducing HSP60 expression in these cells also decreased mitochondrial localization of both the P525Lmutant and the endogenous Wt-FUS proteins (S4F-S4H Fig). These results suggest that HSP60 may play an important role in mediating FUS translocation to mitochondria.

# Reducing HSP60 expression rescues neurodegenerative phenotypes in FUS transgenic flies

To examine HSP60-FUS interaction in vivo, we tested whether FUS genetically interacted with HSP60 in flies by knocking-down Drosophila homologs of human HSP60, including HSP60A,

A Ctr Wt P525L WB: D x-linkpul down Mt input x-linkiput MWt x-inul dow   
GFPFUS-GFP IP: GFP GFP 250- x-linked(kDa) HSP60250 MWt (KDa) 150- 150- FUS-150- 水 HSP60 HSP60 HSP60 100- 100. 100- 7 8 GFP 75- GFP 75. nduced   
TCL FUS-GFP -HSP60 HSP60 HSP60 50- a-HSP6050. a-FUS IP:a-HSP6C 37- frompurifiedfractior Actin Actin 37. WB:a-FUS 1 23 12 34 56   
B Mito Cyto E Ctr Wt P525L Ctr Wt P525LWB: Total Cell Mitochondria MWt   
IP: FUS ！ Myc Ctr siHSP60 Ctr siHSP60 (KDa)   
Myc FUS -70 HSP60 HSP60 HSP60 -55 FUS Myc PCNA -35 HSP60 HSP60 -25   
TCLGAPDH GAPDH RhoA -15 Tom20 CoxIV CoxIV 12 3 4 5 6 7 8 9101112 1 2 34 5 6 165 267 286 371 422 453 510526 1.5   
） FUSQ/G/S/YGRGG1RRMRGG2ZnRGG3 F 火\* FL 1.0 1-285 1-370 371-526 ndu paeeg 3 M P1555 8 113-0 325-330 Blot: 0.0 Ctrtsi HsP60 MirochinSP60 anti-HSP60 MWt(KDa) G 130 1.5 -70 -100 FPea lee 1.0 \* -40 0.5 -35 -25 0.0 Ctr_siHSP60 CtrsiHSP60 Coomassie Blue Staining Total Cell Mitochondria

were detected in the input cross-linked cellysates (input).HSP60 and the tagged FUS were detectedata size of 60 and $7 0 \models \mathsf { D a }$ ,respectively,consistent with the prediction that the HSP60 containing 130kDa band consists of HSP60 and 6XHis-Myc tagged FUS.(E)Down-regulating HSP60 leads to areduction in FUS mitochondriallocalization.HEK293 cells were transfected with the controlor HSP60-specific siRNAs and harvested for mitochondrialpurification72-hrpost-transfection.The mitochondrialpurity was confirmed by the enrichment of mitochondrial TOM20 and the absence of cytoplasmic protein such as RhoA or nuclear protein PCNA.Triplicates of the experiments are shown. (F) Quantification of the HSP60 levels in the total cell extractsand in themitochondrial fractions. $( \mathsf { G } )$ QuantificationofFUS levels intotalcell extractsand the mitochondrial fractions.Alldata represent 3 independent experiments with statistical analyses using one-way ANOVA with Bonferronipost-test (\* $\mathsf { p } { < } 0 . 0 5$ $^ { * * }$ 元 $\mathsf { p } { < } 0 . 0 1 \$ ）

doi:10.1371/journal.pgen.1005357.g006

B,C or D in FUS transgenic flies using RNA interference (RNAi) (supplementary S5 and S6 Figs). Fly eyes expressing Wt- or P525L-mutant FUS exhibited rough surface and reduced pigmentation. Scanning electron microcopy (SEM) revealed ommatidial loss,ommatidial fusion, ectopic bristle formation and disrupted ommatidial organization,as reported previously [17]. Down-regulating expression of HSP60A, HSP60B orHSP60C in photoreceptors suppressed the FUS-induced retinal degeneration phenotype to various extents,with siHSP60B showing the most robust effect,although knocking-down HSP60 did not affect the expression level of FUS (see S5A and S5B Fig). Therefore,we chose siHSP60B for subsequent experiments. Knockingdown HSP60B in FUS-expressing flies partially restored the eye morphology,rescuing FUSinduced photoreceptor degeneration,although siHSP6oB by itself in the control flies did not show any effects (Fig 7A; S7 Fig). Scanning EM revealed that down-regulating HSP60B in fly eyes expressing Wt- or P525L-mutant FUS partially reversed FUS-induced ommatidial loss or fusion,restoring ommatidial pattern and bristle organization, especially in the peripheral region of the eye (Fig 7B and 7C).The effect of siHSP60 in rescuing FUS-induced neurodegeneration was specific because a number of fly lines tested that expressed siRNAs against other genes did not show any effect (see S3D Fig). When mitochondria were examined in the motor neuron axons, reducing HSP6oB expression by RNAi alone in the control flies did not affect mitochondrial size (Fig 7D and 7E). In contrast, down-regulating HSP60B significantly increased the mitochondrial size in flies expressing P525L-mutant FUS in motor neurons (Fig 7D and 7E), thus rescuing the effect of P525L-mutant FUS induced mitochondrial fragmentation. Importantly,HSP6oB down-regulation in fly MNs significantly rescued the locomotive defects in the P525L-mutant FUS expressng fly larvae, with mitigated tail-paralysis phenotype in these siHSP60B/P525L-FUS expressng animals (Fig 7F and 7G). These results indicate that FUS genetically interacts with HSP60, suggesting a role of HSP60 in FUS-induced neurotoxicity.

Then, we used transmission EM(TEM) to examine the fly retinal structure.Retinas displayed normal differentiation of seven rhabdomeres at day 3,whereas the flies expressing Wtor P525L-mutant FUS exhibited defective rhabdomeres (marked as “Rh" in Fig 8).Although knocking-down HSP60B in files expressing Wt- or P525L-mutant FUS did not completely restore the rhabdomeres to its normal morphology,it improved rhabdomere formation in the siHSP60B-expressingflies (as marked by arrows in Fig 8)(S8D Fig). As compared with the healthy mitochondria in the control flies,the mitochondria in the flies expressing Wt- or P525L-mutant FUS were smaller (S8E Fig). Down-regulating HSP60B expression in these FUS transgenic flies significantly increased the mitochondrial size (S8E Fig).It was noticed that at day 3 the nuclei of photoreceptor cells in the fly eyes expressing either Wt- or P525L-mutant FUS appeared swollen with nearby mitochondria severely damaged,although the cells remained intact. This phenomenon was not detected in the control flies (Fig 8,S8B and S8C Fig). By day 15,a significant photoreceptor loss was observed in flies expressing Wt- or P525Lmutant FUS as compared with the control flies, or with the FUS transgenic flies at day 3 (Fig 8,

![](images/5c0267a22aa49e81e6c7f2f2c719ba340033ff9a0357d1e13fa400af5f0cbf5d.jpg)

Fig7.Down-regulating fly HSP6oB expression partially rescues neurodegeneration phenotypes in FUS transgenic flies. (A)FUS genetically interacts with HSP60B.Co-expressing siHSP60B with FUS in transgenic flies significantly rescued FUS-induced retinal defects as compared with the control. (B,C) Scanning electron microscopic images offly eyes expressing FUS together with the control siRNA or siHSP60B,with the lower panels showing higher magnification images of the corresponding groups.Flies coexpressing siHSP6oB and FUS showed more well-organized bristles and reduced ommatidial fusion as compared with FUS transgenic fies expressng the control siRNA.Fly genotypes: Ctr: GMR-Gal4/UAS-WtFUS-RFP or GMR-Gal4/UAS-P525L-FUS-RFP;siHSP60B:GMR-Gal4/UAS-Wt-FUS-RFP/UAS-siHSP60B or GMR-Gal4/UAS-P525L-FUS-RFP/UAS-siHSP60B.(D)Confocal microscopic images of mitochondria in motor neuron axons in the larval A3 abdominal segments(Z-stacks projected into single images).Fly genotypes,Ctr: D42-Gal4/UAS-mitoGFP/UAS-RFP; siHSP60B:D42-Gal4/UAS-mitoGFP/UAS-siHSP60B; P525L:D42-Gal4/UAS-mitoGFP/UAS-P525L-FUS-RFP; P525L-FUS+siHSP60B:D42-Gal4/UAS-mitoGFP/ UAS-P525L-FUS-RFP/UAS-siHSP60B.(E) RNAi mediated down-regulation of HSP60B in P525L-mutant FUS expressing flies rescued mitochondrialfragmentation phenotype,as shown by quantification of mitochondrial lengths in MN axons.The 3rd instarlarvae expressing P525L-mutantFUS showed significantly smaller mitochondria as compared with the control.Ten larvae were measured in each group. (F,G) Knocking-down HSP6oB inflies expressing P525L-mutant FUS in MNs significantly rescued their larval locomotive defects.The tail paralysis phenotype was examined in third instar larvae,as measured bythe loss of abilty forlarvae toanchor theirtails on the agarsurface whencrawling {as previously published[17]}.Fly genotypes, Ctr: OK371-Gal4/UAS-RFP; siHSP60B: OK371-Gal4/UAS-siHSP60B; P525L: OK371-Gal4/ UAS-P525L-FUS-RFP;P525L-FUS $^ +$ siHSP60B:OK371-Gal4/UAS-P525L-FUS-RFP/UAS-siHSP60B.AII data were analyzed using one-way ANOVA with Bonferroni post-test (\*: $\mathsf { p } { < } 0 . 0 5$ \*\*\*: $\mathsf { p } { < } 0 . 0 0 1 \$ ）

doi:10.1371/journal.pgen.1005357.g007

S8A Fig). These results support the hypothesis that mitochondrial damage may be an early event in FUS proteinopathies and that the mitochondrial damage induced by FUS expression may precede photoreceptor neuronal death.

# Discussion

Originally identified as a gene involved in chromosomal translocation in liposarcoma [59], the human FUS gene encodes a RNA/DNA binding protein involved in multiple cellular processes [60,61]. Although it has been reported that FUS shuttles between the nucleus and cytoplasm [62] and that FUS has many nuclear activities,the cytoplasmic function of FUS is far less clear. For the first time,our study reveals that the FUS can be localized to mitochondria and increased FUS mitochondrial localization is toxic to neurons, contributing to neurodegeneration. While the physiological role of FUS in regulating mitochondrial biology remains to be

![](images/8ab8f21189b902bdfe0b91fb7f9bdbdf2da92e4f6396fcdd7f32674b9c502afc.jpg)  
Fig 8.Down-regulating HSP6oB in fly photoreceptors partially rescues the retinal degeneration phenotype of FUS transgenic mice as shown by transmission electron microscopy(TEM).Electron microscopic images offly retinas atday 3are shown,together with mitochondriaand nuclei (at higher magnifications).Rh: rhabdomere (marked by arrows); Mito: mitochondria; Nu: nuclei.The retinas in the controlflies display normal rhabdomere organizationwith seven photoreceptorcellsand healthy mitochondriaatday 3,buttheflies expressing eitherWt-or P525L-mutantFUS show disrupted rhabdomere organization and smaller mitochondria. Knocking-down HSP6oB expressionby specific siRNA partially rescued these defects in flies expressing Wt-or P525L-mutant FUS.

doi:10.1371/journal.pgen.1005357.g008

elucidated by future studies,our data provide new information about the cytoplasmic platform on which FUS is likely to play an active role.

The vast majority of mitochondrial proteins are encoded by the nuclear genome and synthesized in the cytoplasm, then transported into mitochondria via different mechanisms [63]. Under physiological conditions,FUS possibly plays a functional role in mitochondria.A recent report showed that knocking down of FUS by RNAi affected the expression of a subset of mitochondrion-associated genes 64]. Although the human FUS protein shows a very low probability $( < 0 . 0 0 0 1 )$ in computational prediction of mitochondrial import using a published algorithm [65],our data,ranging from biochemical purification of mitochondria, immunoEM detection of the endogenous FUS to in vivo assays in transgenic flies provide clear evidence that FUS is associated with mitochondria.Previous studies have shown that some mitochondrial proteins are chaperoned by heat shock proteins [66].The mitochondrial chaperonin HSP60,together with HSP70 and HSP10,are important for protein import into mitochondria [30,32,33,34,35,36]. Although it has been reported that HSP60 may promote mitochondrial localization of amyloidbeta peptide and mitochondrial impairment in Alzheimer disease [67],the role of HSP60 in other neurodegenerative diseases remains unclear.Mutations in HSP60 were reported to be associated with spastic paraplegia type 13 (SPG13),a late-onset autosomal-dominant neurodegenerative disease [37,38]. Recently,it was reported that a heterozygous knock-out HSP60 mouse model could recapitulate the features of human disease (SPG13) and increase mitochondrial ROS [68]. Here, we demonstrate that FUS interacts with HSP60 by mass-spectrometry analyses of FUS interacting proteins,co-immunoprecipitation, GST pull-down and cross-linking experiments.Importantly,down-regulation of HSP60 expression decreased mitochondrial FUS levels and partially rescued FUS-induced phenotypes,including mitochondrial fragmentation, neurodegeneration and locomotive deficits.Drosophila has four HSP60 homologs. In our experiments, knocking down expression of each HSP60 homolog by their RNAi led to a partial rescue of the neurodegenerative phenotypes in FUS transgenic flies. It is likely that different HSP60 homologs or HSP6o-like genes may compensate for each other in vivo.

Our data support the hypothesis that HSP60 promotes or mediates FUS translocation from the nucleus to mitochondria.HSP60 has been reported to play a dual role in cell death,as either an anti- or a pro- apoptosis factor [69]. On one hand, HSP60 together with HSP70 may play a protective role in mitochondrial unfolded protein response [33]. HSP60 may form a pro-survival complex with Bcl-2,an anti-apoptotic mediator; and disruption of this complex formation by nutrient deprivation results in celldeath [7O]. On the other hand,HSP60 has been reported to facilitate pre-caspase-3 maturation [71] or induce nitric oxide production by microglia, leading to neurotoxicity [72]. Our results indicate that HSP60 interacts with FUS to promote mitochondrial damage and cell death.Future studies are necessary to determine whether modulating the HSP60-FUS interaction may alter the activity of HSP60 in regulating neuronal survival. For example,it is possble that the interaction of FUS with HSP60 might affect the ability of HSP60 to form complex with Bcl-2,thereby inhibiting Bcl-2 pro-survival function and resulting in celldeath. Our findings provide a new direction for further investigating the multi-facet role ofHSP6O in neurodegenerative disorders.

There are four HSP60 homolog genes in drosophila, named as HSP60A,HSP60B,HSP60C, HSP60D [73,74]. The predicted phylogenetic relationship between these four genes and human HSP60 is shown in S6 Fig. HSP60A is a constitutively expressed chaperonin in drosophila cells and essential for embryogenesis.Mutations in HSP60A lead to embryonic lethal phenotype in Drosophila [75]. HSP6oB shares extensive homology with other drosophila HSP60 proteins and is required for spermatid individualization process.Mutations in HSP60B result in male sterility [76]. HSP60C is required for tracheal development and for early-stage spermatogenesis [77]. Therefore,HSP60A,HSP60B and HSP60C have distinct functional activities in development, whereas HSP6oD is essential for caspase-mediated apoptosis in Drosophila [78]. Our data show that down-regulating the expression of HSP60A,HSP60B or

HSP60C led to a partial rescue of the neurodegenerative phenotypes in FUS transgenic flies to various extent, suggesting that these three genes may share common features in FUS proteinopathies.Interestingly,HSP6oD down-regulation in FUS transgenic flies did not show any rescue effects,suggesting that caspase-mediated apoptosis might not be critical for neurodegeneration in FUS proteinopathies.

A number of missense mutations have been identified in the human FUS gene among ALS-- FUS patients [7,8,14,15,79,80]. In addition,at least four mutations have been detected in the ${ \boldsymbol { 3 } } ^ { \prime }$ UTR of the FUS gene among sporadic or familial ALS patients but not in the control samples [16].Remarkably, fibroblasts from the ALS patients with three different mutations in the 3'UTR showed increased FUS protein expression and accumulation of cytoplasmic FUS [16]. Animal models ofFUS proteinopathies established by over-expressing Wt- or ALS-mutant FUS recapitulate major clinical and pathological features of FUS proteinopathies, providing useful systems to study pathogenic mechanisms underlying these devastating diseases [17,18,19,20,21,22,23,24,25]. These results, together with our observation of increased FUS protein in FTLD-FUS patient samples,support the notion that increased FUS expression and cytoplasmic accumulation of FUS likely contribute to the pathogenesis of FUS proteinopathies.

Characteristic cytoplasmic inclusions containing FUS protein have been detected in the affected neural tissues of sporadic ALS (sALS) and FLTD-FUS patients.However, the pathogenic role of FUS and the underlying molecular mechanisms in ALS and FTLD remain to be elucidated. One common finding in ALS and other neurodegenerative diseases is mitochondrial damage [81,82,83,84].Mitochondrial impairment has been reported in ALS patients as well as animal models for ALS [39,40,43].Disorganized mitochondria and endoplasmic reticulum have been reported in spinal cord tissue samples from two cases of FUS-positive juvenile ALS patients, including one carrying P525L mutation [46]. In another study, expresson of two other ALSassociated FUS mutants,R521G or R521H, was associated with shortened mitochondria in cultured motor neurons in vitro [47]. These studies suggested mitochondrial impairment in FUSpositive ALS,although it was not clear whether such mitochondrial defects were a direct consequence of FUS expression or secondary efects following neurodegeneration in ALS.Neither was itclear if mitochondrial defects could be shared pathological features for both ALS-FUS and FTLD-FUS. Our work provides the first EM evidence of mitochondrial localization of FUS not only in cultured cells but also in the FTLD-FUS patient brain samples. Our study provides direct evidence for mitochondrial damage in motor neurons in our animal model for FUS proteinopathies.This is also the first report of mitochondrial damage in FTLD-FUS at the ultra-structural level. Our data obtained from diferent cellular models and from motor neurons in transgenic flies clearly demonstrate that increased expression ofFUS,either Wt-or ALS-mutant, leads to mitochondrial fragmentation, decrease in mitochondrial membrane potential, increased ROS production and eventually neurodegeneration. Our findings suggest that mitochondrial impairments may be an early or initiating event in FUS proteinopathies.

Previous data have shown that certain ALS-associated FUS mutations,such as P525L, increase the cytoplasmic distribution of FUS protein because of the disruption of its nuclear localization signal [17,85].Interestingly,the P525L-mutation in ALS patients is associated with earlier age-of-onset and faster disease progression (see S1 Table) [7,46,86,87]. Overexpressing either Wt- or ALS-mutant FUS results in a marked increase in mitochondrial fragmentation. In the FUS transgenic flies,FUS expression induced mitochondrial damage and neurodegeneration,both of which were partially rescued by down-regulation of the mitochondrial HSP60 expression. Consistently,brain tissues from at least some FTLD-FUS patients showed increased expression ofboth FUS and HSP60 proteins.

The data presented in this report led us to propose a working model that can be further tested in our future studies (see Fig 9): increased cytoplasmic FUS protein levels triggered by

![](images/30a1e88e121aad9e7fd9e91dc7e006837dd9fdd749d478392f220f75a74e8625.jpg)  
Fig9.Aworking model for FUS-induced neurotoxicity.Eitheranaberrantincrease in the expression of Wt-FUS(WtFUS)or FUS mutations that enhance the cytoplasmic redistribution of the mutant FUS protein (mtFUS)leads to abnormal accumulationof cytoplasmic FUS and increased translocation to mitochondria, resulting in excessive mitochondrialfission and damage that eventually culminate in neuronal death.

doi:10.1371/journal.pgen.1005357.g009

cellular stresses or pathogenic FUS mutations result in an increased interaction of FUS with mitochondrial chaperone protein HSP60,which promotes FUS localization to mitochondria. Elevated mitochondrial localization ofFUS may damage mitochondria,leading to mitochondrial fragmentation. Such mitochondrial impairment may trigger the onset of neuronal damage and neuronal death, eventually resulting in neurodegenerative manifestations of FUS proteinopathies. Recent studies have revealed possible convergence between ALS and FTLD [14]. Our data suggest that one point of such convergence and a critical event in FUS-proteinopathies may be FUS-induced impairment of mitochondria.

Although our data do not rule out the possibility that loss of function caused by sequestration of the wild-type FUS protein into inclusion bodies may also contribute to pathogenesis of FUS-proteinopathies,as previously proposed [88,89], the present study supports the gain-offunction toxicity mechanism. Our work provides strong evidence that mitochondrial damage contributes to FUS-proteinopathies and represents a common molecular pathology shared by ALS-FUS and FTLD-FUS.Moreover,HSP60 signaling pathway may be critical for FUSinduced neurotoxicity,and reducing HSP60 expression or suppressing HSP60 activity may provide therapeutic benefit for FUS-proteinopathies patients with increased HSP60 expression.

# MaterialsandMethods

# Ethics statement

De-identified postmortem brain samples from patients affected by FTLD-FUS and control subjects were obtained from the Cognitive Neurology &Alzheimer's Disease Center (CNADC) at Northwestern University following institutional and NIH guidelines. All experiments involving animal tissue samples were carried out following institutional and NIH guidelines.

# Fly strains and reagents

Fly strains are as follows: D42-Gal4 UAS-mitoGFP/TM6B was from Dr.Y. Zhang [50]. UAS-Drp1, UAS-Drp1-RNAi and UAS-Drp1-K38A were from Dr.B.Lu. UAS-RFP, UAS-Wt

FUS-RFP,UAS-P525L-FUS-RFPwere described previously[17].The HSP60A-RNAi, HSP60B-RNAi, HSP60C-RNAi and HSP60D-RNAi stocks were obtained from Vienna Drosophila RNAi Center (VDRC). OK371-Gal4, GMR-Gal4 and Actin5C-Gal4 were obtained from Bloomington Drosophila Stock Center.Flies were raised according to standard procedures at $2 5 ^ { \circ } \mathrm { C }$

Antibodies used in this study were as follows: rabbit-anti-GFP(Millipore),rabbit-antiHSP60 (BD) and monoclonal murine anti-myc (Covance), monoclonal murine anti-FUs (ProteinTech Group Inc)and following rabbit polyclonal antibodies against corresponding proteins from ProteinTech Group Inc: CoxIV, Histone H3,TOMM20,MFN2, GAPDH, Actin and PCNA. Anti-RhoA is from Santa Cruz and anti-p637Drp1 is from cell signaling technology (CST).

# Axonal mitochondrial imaging

FUS transgenic and control flies were prepared as described [17].The third instar larvae were dissected and fixed with $4 \%$ paraformaldehyde (PFA,Electron Microscopy Science) for $2 0 \mathrm { m i n }$ at room temperature (RT). Following rinse with PBS,larvae were mounted onto coverslips using mounting gel. Confocal images were taken under an Olympus FV1ooo confocal microscope.

# Larval movement assay

The assay was done as described [17,9O].Briefly,the larval movement index was measured as the number of peristaltic waves during the period of 2 min in the late third instar larvae expressing control RFP,siHSP60B,P525L-FUS or siHSP60B+P525L-FUS under the OK371-Gal4 driver in a controlled environment ( $2 5 ^ { \circ } \mathrm { C } ,$ humidity $5 0 \% \pm 5 \%$ ,illumination $2 8 0 0 \pm 1 0 0 \mathrm { l u x } )$ 1

# Immuno-electron microscopy (IEM) or EM

For immuno-EM assay in HEK293 cells, cells stably expressing GFP,Wt- or P525L-mutantas GFP-tagged protein were harvested and fixed with $4 \%$ PFA and $0 . 2 \%$ glutaraldehyde (pH7.2) in PBS for 3 hrs at RT.Following rinses and post-fixation processing, gelatin-embedded blocks were prepared in $2 . 3 \mathrm { M }$ sucrose at $4 ^ { \circ } \mathrm { C }$ Ultrathin sections $( 7 0 - \mathrm { n m } )$ were cut at $- 1 2 0 ^ { \circ }$ Cusing dry diamond knives. Following blocking, the sections were immunostained with monoclonal antiFUS antibody (1:100) and anti-mouse IgG antibody (1:25) conjugated to $1 0 \mathrm { n m }$ -colloidal gold particles.

For human brain tissue immuno-EM,postmortem frontal cortex samples from control cases and patients affected by FUS-proteinopathies were fixed with $2 \%$ PFA and $0 . 2 \%$ glutaraldehyde for 3 hrs at RT.Samples were embedded in $6 \%$ gelatin. Immuno-staining was performed as described above.

For EM study offly eyes,adult fly heads were dissected and fixed in a solution with $4 \%$ paraformaldehyde, $2 . 5 \%$ glutaraldehyde in PBS, $\mathrm { p H } 7 . 4 _ { \mathrm { : } }$ for $1 2 \mathrm { { h } }$ at $4 ^ { \circ } \mathrm { C }$ and in a solution with $1 \%$ osmium tetroxide in PBS, $\mathrm { p H } 7 . 4$ ,for $^ { 2 \mathrm { h } }$ at room temperature.The tissues were then dehydrated in a series of ethanol solutions (30-min washes in 10,25,40,55,70,85 and $1 0 0 \%$ ethanol) and embedded in spurr resin.Thin sections ( ${ \mathrm { 7 0 - n m } } ,$ were prepared and were examined by transmission EM.

All EM images were obtained usinga Tecnai Spirit $( 1 2 0 \mathrm { { k V } ) }$ orFEI Tecnai 20 electron microscope.

Scanning electron microscopy (SEM) was carried out as described before [17].

# Cell culturesand transfection

HEK293T or HT22 cells were cultured ( $3 7 ^ { \circ } \mathrm { C } 5 \%$ CO2) in DMEM(GIBCO) supplemented with $1 0 \%$ FBS (HyClone or Atlanta Biological). Plasmids were transfected using VigoFect (Vigorous Biotechnology)according to manufacturer's instructions or calcium phosphate method as described before [91,92]. Cortical neurons were cultured from E15 embryonic mouse brains following the published protocols [93,94].

HEK293-based T-Rex"293 cells (Invitrogen) were transfected with pcDNA4 TO/myc-His plasmids (Invitrogen) expressing either Wt- or P525L-mutant FUS;and stable expressing cells were selected as individual clones in zeocin $( 4 0 0 \mu \mathrm { g / m l } )$ . To induce FUS expression, 0.5 or $\mathrm { { 1 m g / m l } }$ tetracycline was added to the culture medium,and cells were incubated for different period of time at $3 7 ^ { \circ } \mathrm { C }$

The control siRNA and FUS siRNA or HSP60 siRNA were transfected using lipofectamine (Invitrogen) according to manufacturer's instructions. SiRNA targeting human FUS: $5 ^ { \prime }$ -GGAC AGCAGCAAAGCTATG- $3 ^ { \prime }$ [95]. SiRNA targeting human HSP60: 5'-GTGACAAGGCTC AAATTGA-3'.

# Immunoprecipitation

HEK293T cells were used for the transfections and analyses of protein-protein interactions. The experiments were performed at 48 hours post-transfection.The harvested cells were washed with phosphate-buffered saline (PBS)and lysed for 30 minutes on ice in the lysis buffer [91].The soluble fraction of cellysates was collected and used for immunoprecipitation with specific antibodies and protein A-agarose (Roche) at $4 ^ { \circ } \mathrm { C }$ .The immunoprecipitates were examined using Western blotting(WB) with proper antibodies.

# Mitochondrial purification

Mitochondria were purified by percoll gradient ultracentrifugation following published protocols with minor modifications [53,54]. Briefly,stable FUS-expressing HEK cells were lysed in mitochondrial isolation buffer ( $\mathrm { i 0 m M }$ Tris-MOPS,pH7.4,1mMEGTA, $2 5 0 \mathrm { m M }$ sucrose), homogenized with a Glass/Teflon Potter Elvehjem homogenizer (Bellco Glass Inc) and fractionated by sequential centrifugation.

# Cross-linking pull-down assay

Afterexpression of myc-His tagged P525L FUS mutant was induced with $0 . 5 \mathrm { u g / m l }$ tetracycline for l6 hours,cells were briefly washed with PBS containing $\mathrm { 1 m M M g C l _ { 2 } }$ (PBS, $\mathrm { M g C l } _ { 2 } ^ { \cdot }$ ,and treated with $1 \%$ PFA in PBS $\mathrm { M g C l } _ { 2 }$ for 10 minutes. Cells were washed with $5 0 \mathrm { m M }$ Tris-HCl buffer $\mathrm { ( p H 7 . 4 ) }$ containing $1 2 0 \mathrm { m M N a C l }$ and $\mathrm { 1 m M M g C l _ { 2 } }$ twice and treated with the same buffer for $1 0 \mathrm { m i n }$ .Cells were washed with PBS- $\mathrm { M g C l } _ { 2 }$ once again and lysed with phosphate buffer $\left( \mathrm { { p H 8 . 0 } } \right)$ containing $1 \% \mathrm { N P - } 4 0 , 2 0 0 \mathrm { m M N a C l }$ and1mMPMSF.Soluble fractions were obtained by centrifugation at $2 5 0 0 \mathrm { g }$ for $3 0 \mathrm { m i n }$ and twice volume of 6M guanidine HCl in $5 0 \mathrm { m M }$ phosphate buffer $\mathrm { ( p H 8 . 0 ) }$ was added. FUS proteins were collected with Ni-NTA agarose,washed with 6M guanidine HCl in $5 0 \mathrm { m M }$ phosphate buffer $\left( \mathrm { p H 8 . 0 } \right)$ twice,and then eluted with $1 0 0 \mathrm { m M }$ PIPES $\left( \mathrm { p H } 6 . 6 \right)$ containing $0 . 1 \%$ SDS, $5 \mathrm { m M }$ EDTA. The obtained solutions were diluted ten times with $2 0 \mathrm { m M }$ HEPES $\mathrm { ( p H 7 . 4 ) }$ containing $0 . 5 \%$ Triton X-100, $0 . 0 5 \%$ deoxycholate, $1 0 0 \mathrm { m M N a C l }$ 0 $5 \%$ glycerol and $0 . 5 \mathrm { m M }$ PMSF to carry out immunoprecipitation with anti-myc antibody and protein $\mathrm { A } / \mathrm { G }$ beads.Immunoprecipitated beads were resuspended with 6M guanidine HCl in $5 0 \mathrm { m M }$ phosphate buffer $\left( \mathrm { p H 8 . 0 } \right)$ ,and FUS-myc-His protein was pull-down with Ni-NTA beads (by repurification with Ni-NTA,antibody can be removed so that background signals can be eliminated in WB analyses).Pull-down fractions were analyzed by Western blotting.

# GST pull-down assay

GST pull-down assay was performed as described by [96]. GST tagged Wt or P525L or fragments FUS protein were expressed in bacteria and purified using glutathione 4B Sepharose beads.His tagged HSP60 were similarly expressed in bacteria and purified using nickel beads. Purified GST,or Wt,P525L,fragments FUS-GST was incubated with purified HSP60-His in TNE buffer ( $1 0 ~ \mathrm { m M }$ Tris, $\mathrm { p H } 8 . 0 \$ ,150 mMNaCl, $1 \mathrm { m M }$ EDTA, $1 \% \mathrm { N P - 4 0 }$ )on ice for1 hour. Then,supernatants were incubated with glutathione 4B Sepharose beads at $4 ^ { \circ } \mathrm { C }$ for 3 hours. The glutathione beads were then washed extensively with ice-cold TNE buffer,and bound proteins were subjected to SDS-PAGE followed by immunoblotting analysis.

# RNaseAtreatment

The RNaseA treatment assay was performed as [97]. Briefly,the purified protein were treated with $5 0 \mathrm { { u g / m l } }$ RNaseA for $3 0 \mathrm { m i n }$ at $3 7 ^ { \circ } \mathrm { C }$ . Then GST pull-down assay was performed as previous described.

# Measurementofmitochondrial membranepotential

Mitochondrial membrane potential was measured by staining cels with tetramethylrhodamine methyl ester (TMRM) (Invitrogen).HEK293 cells were transfected with the GFP vector control,Wt- or P525L-mutant FUS as GFP-tagged proteins.Cells were incubated with $2 0 ~ \mathrm { n M }$ （204号 TMRM for $2 0 \mathrm { m i n }$ at $3 7 ^ { \circ } \mathrm { C } ,$ 19 hrs post-transfection.Cells were washed three times with PBS and cultured in opti-MEM(without phenol red) supplemented with $1 0 \%$ FBS and $5 \mathrm { n M }$ TMRM. Confocal images were taken using an Olympus FV1000 microscope. The TMRM intensity in different groups was measured by FACS (BD FACS AriaII) and analyzed using FlowJo software.Data were obtained from four independent experiments,and 20oo cels were examined per group in each experiment.

# Mitochondrial ROS measurement

Mitochondrial ROS production was measured using mitoSOX-Red (Invitrogen). HEK293 cells were transfected with plasmids expressing either the GFP vector control, Wt or P525L FUS-GFP.24 hours post-transfection, cells were stained with $5 \mu \mathrm { M }$ mitoSOX-Red for $2 0 \mathrm { m i n }$ at $3 7 ^ { \circ } \mathrm { C }$ .After washes,cells were fixed with $4 \%$ PFA for $2 0 \mathrm { m i n }$ at room temperature followed by FACS analysis with mitoSOX-Red fluorescence intensity determined using FlowJo software. Confocal images were taken using an Olympus FV10oo microscope.

# RT-PCR

To measure HSP6OB expression levels in the control and siRNA flies, total RNA was prepared from flies using Trizol (Invitrogen),reverse transcribed,and subjected to PCR analysis.For HSP60B,PCR (25 cycles) was carried out using the following primers: $5 ^ { \circ }$ CTGAGGATG CCTTGCCAGACC- ${ \boldsymbol { \cdot 3 } } ^ { \prime }$ and $5 ^ { \circ } .$ -GCAGCACCTTTGTGGGATCAATA-3'.ForActin,PCR analysis (21 cycles) using the specific primers: $5 ^ { \circ }$ -GAGCGCGGTTACTCTTTCAC- ${ \boldsymbol { 3 } } ^ { \prime }$ and $5 ^ { \circ }$ -ATCCCGATCCTGATCCTCTT- ${ 3 ^ { \circ } }$ ：

# Quantitative analyses of mitochondrial morphology

Mitochondria from HT22 cels or cortical neurons were quantified as [98].Briefly,the percentage of cells showed tubular,fragmented or intermediate pattern was counted and statistic analysis. Mitochondria from fly motor neuron were measured their length by using Image J. The mitochondrial length from Wt- or P525L-mutant FUS groups was normalized to the control (Ctr) group and presented as the percentage of the mitochondrial size in the control group.

Mitochondrial size in EM analyses were quantified by their cross sectional areas by using Image J software. The mitochondrial size from Wt- or P525L-mutant group was normalized to the control group and presented as percentage of the mitochondrial size in the control group. Normal or abnormal cristae morphology of mitochondria was defined as previously published [99,100]. Briefly, normal mitochondria with numerous well-organized cristae or abnormal mitochondria showing ring structures or loss of cristae were scored and quantified as the percentage of total mitochondria examined.

Statistical analyses were performed using either one-way ANOVA, followed by student ttest, Chi-square test or Bonferroni multiple comparison for comparing individual groups,as indicated in corresponding figure legends. The bar graphs with error bars represent mean $\pm$ standard error of the mean (SEM).Significance is indicated by asterisks:\*, $P < 0 . 0 5$ \*\*， $P { < } 0 . 0 1$ $* * *$ $P { < } 0 . 0 0 1$ ：

# Supporting Information

# S1 Table.The age of onset and duration of ALS-FUS (published) and FTLD-FUS (this study)patients. (DOCX)

# S1 Fig. Western blotting and IEM assays demonstrating specificity of the FUS antibody.

(A)A monoclonal antibody (ProteinTech Group Inc, USA) detected a single band in the total cell lysates prepared from HEK293 cells (marked by the arrow),and this band is almost eliminated when the antibody was pre-absorbed (FUS-absorbed) in the presence of purified recombinant FUS protein. (B)The immunoEM (IEM) signals were almost reduced to the background level when FUS-absorbed antibody was used in staining the human brain tissue samples.The number of $1 0 \mathrm { n m }$ -gold particles per square $\mu \mathrm { m }$ of the brain section was quantified in the fronto-cortical tissues from postmortem brain of FTLD-FUS case $\# \mathrm { C }$ (left) or control case $\# 3$ (right). All data were analyzed using one-way ANOVA ( ${ \mathrm { ~  ~ n ~ } } = 7 , \cdots { }$ $\mathrm { p } { < } 0 . 0 0 0 1 \$ ). (C) Knocking down FUS by specific siRNA significantly reduced the FUS-specific Western blotting band signals.Two independent knocking-down experiments were carried out (shown in lanes 1-4). (D) Quantification of WB signals shown in panel C.(E) Western-blotting images of brain lysates from the wild-type $( \mathrm { W t } ; + / + )$ ,heterozygous $\mathrm { ( H e t ; + / - ) }$ and homozygous deficientknockout $( \mathrm { K O ; - } / \mathrm { - } )$ FUS mice.   
(TIF)

S2 Fig.Increased expression of FUS alters mitochondrial protein levels. (A,B) HEK293T cells were transfected with GFP,Wt-FUS-GFP or P525L-FUS-GFP,and cell lysates were subjected to Western bloting analyses 24 hrs post-transfection.(C-E) Quantification of protein levels as indicated.Data were analyzed using one-way ANOVA ( $( \mathtt { n } ) 3 ,$ \*： $\mathrm { p } { < } 0 . 0 5$ ） (TIF)

S3 Fig. Expression of wild-typeDrpl or K38A-Drpl in primary cortical neurons expressing Wt- or P525L- mutant FUS and in FUS transgenic flies.(A) The mitoRed plasmid was cotransfected together with plasmids expressing either the control vector (Ctr),or Wt- or

K38A-Drpl into E18 murine cortical neurons expressing P525L FUS.Arrow marks tubular mitochondria,arrowhead marks fragmented mitochondria,“\*” marks condensed or fragmented nuclei (signs of cell death). Insets show the boxed areas at a higher magnification. (B) Quantification of the percentage of cells containing fragmented mitochondria. (C) Quantification of the percentage of cells with condensed nuclei. All data were analyzed using one-way ANOVA with Bonferroni post-test ( \*\*\*: $\mathrm { p } { < } 0 . 0 0 0 1 \$ ). (D) Light microscopic images of eyes of control or Drpl or siDrpl or K38A-Drp1 flies. Fly genotypes: Ctr: GMR-Gal4/UAS-WtFUS-RFPor GMR-Gal4/UAS-P525L-FUS-RFP;Drp1: GMR-Gal4/UAS-Wt-FUS-RFP/ UAS-Drpl or GMR-Gal4/UAS-P525L-FUS-RFP/UAS-Drpl; siDrpl: GMR-Gal4/UAS-WtFUS-RFP/UAS-siDrp1 or GMR-Gal4/UAS-P525L-FUS-RFP/UAS-siDrp1; K38A-Drpl: GMR-Gal4/UAS-Wt-FUS-RFP/UAS-K38A-Drp1 or GMR-Gal4/UAS-P525L-FUS-RFP/ UAS-K38A-Drp1. (E) Scanning electron microscopic images of flies expressing P525L FUS with control or co-expressing K38A-Drpl, with the right panels showing higher magnification images of the corresponding ones.   
(TIF) S4 Fig. HSP60 interacts with FUS and mediates FUS mitochondrial localization.(A) The purified proteins were treated with RNaseA $( 5 0 \mathrm { { u g } / \mathrm { { m l } ) } }$ for $3 0 \mathrm { m i n }$ at $3 7 ^ { \circ } \mathrm { C }$ GST pull-down was performed and bound proteins were analyzed by Western bloting (WB).The lower panel shows Coomasse blue staining of GST fusion proteins used in the assay. (B) A darker exposure of panel 5 and 6 in Fig 6D shows multiple bands including 70 and $1 3 0 \mathrm { { k D a } }$ in the cross-linked cell lysates using anti-FUS antibody. (C) Western bloting(WB) analysis of nuclear or cytosolic fractions from Fig 6E to show endogenous FUS localization. Histone H3 was used as a nuclear marker and GAPDH was used as a cytosolic marker.The nuclear FUS levels were increased when HSP60 expression was knocked down by siHSP60.(D) Quantification of the HSP60 levels in the nuclear fractions and in the cytosolic fractions.(E) Quantification of FUS levels in the nuclear fractions and the cytosolic fractions.(F)P525L-FUS-expressing stable HEK cels were transfected with the control or HSP60 siRNAs and harvested for mitochondrial purification 72-hr post-transfection. The mitochondrial purity was confirmed by the enrichment of mitochondrial protein TOM20 and the absence of cytoplasmic proteins such as RhoA or nuclear protein PCNA. The mitochondrial levels of the P525L-mutant or the endogenous Wt- FUS were decreased when HSP60 expression was down-regulated by siHSP60,as shown by WB. (G) Quantification of the HSP60 levels in the total cell extracts and in the mitochondrial fractions. (H) Quantification ofFUS levels in totalcel extracts and the mitochondrial fractions.All data were analyzed using one-way ANOVA with Bonferroni post-test ( $\mathrm { ^ { * } { : } p { < } 0 } . 0 5 ; \mathrm { ^ { * * } }$ $\mathrm { p } { < } 0 . 0 1$ \*\*\*. $\mathrm { p } { < } 0 . 0 0 0 1 \rangle$ ：   
(TIF)

S5 Fig. Knocking down HSP60 partially rescued FUS-induced retinal generation phenotype without altering the level ofFUS expression. (A) Western blotting experiments using the cell lysates prepared from fly heads in corresponding fly groups demonstrate that siHSP60 expression did not affect the level ofFUS transgene expression (at least 30 fly heads were used in each group). Beta-actin was used as an internal control for total protein loaded.(B) Light microscopic images of fly eyes in the control or siHSP60A or siHSP60B,siHSP60C or siHSP60D groups.Arrows mark the retinal areas with improved ommatidial organization and reduced retinal degeneration when siHSP60A,or siHSP60B or siHSP60C was expressed in photoreceptor cells of the Wt- or P525L-mutant FUS transgenic flies.Fly genotypes: Ctr: GMR-Gal4/ UAS-Wt-FUS-RFP or GMR-Gal4/UAS-P525L-FUS-RFP; siHSP60A: GMR-Gal4/UAS-WtFUS-RFP/UAS-siHSP60Aor GMR-Gal4/UAS-P525L-FUS-RFP/UAS- siHSP60A; siHSP60B: GMR-Gal4/UAS-Wt-FUS-RFP/UAS-siHSP60B or GMR-Gal4/UAS-P525L-FUS-RFP/UAS

siHSP60B; siHSP60C:GMR-Gal4/UAS-Wt-FUS-RFP/UAS-siHSP60Cor GMR-Gal4/ UAS-P525L-FUS-RFP/UAS- siHSP60C; siHSP60D:GMR-Gal4/UAS-Wt-FUS-RFP/UASsiHSP60D or GMR-Gal4/UAS-P525L-FUS-RFP/UAS- siHSP60D.   
(TIF)

S6 Fig. A phylogram showing the evolutionary conservation between the four HSP60 homolog genes in Drosophila Melanogaster $( D m )$ and the HSP60 gene in Homo Sapiens $( H s )$ . The phylogram was generated by ClustalW alignment of amino acid sequences, showing the predicted relationship between four DmHSP60 genes and the HsHSP60 gene. (TIF)

S7 Fig.HSP60B expression in the control and HSP60B-siRNA flies.(A) RT-PCR analysis of the control and siHSP6oB flies using specific primers to determine the expression levels of HSP60B.Actin expression was used as an internal control.(B) Quantification of the HSP60B expression levels in respective groups. Data were collected from 3 independent experiments and analyzed using two-tailed t-test $( ^ { * } \colon \mathsf { p } { < } 0 . 0 5 )$ . (C) Light microscopic images of eyes of control or siHSP60B.No morphological changes were detected in HSP6oB-knock down flies. (TIF)

S8 Fig.TEM sections of fly retina at day 15 and quantification of nuclear size,Rhabdomere (Rh) number, mitochondrial size in fly retina. (A)At day 15, marked photoreceptor cell loss is detected in flies expressng Wt- or P525L-mutant FUS but not in the control flies (scale bar: 5um). (B) The photoreceptors in flies expressing Wt- or P525L-mutant FUS show nuclei of increased size as compared to the control group (scale bar: lum).(C) Quantification of nucleic size in the corresponding fly photoreceptors $( \boldsymbol { \mathrm { n } } \ge 1 6 )$ . (D) Quantification of the number of remaining rhabdomeres (including fragments of Rh, $\mathbf { n } { \geq } 1 2$ for each group).(E) More than 100 mitochondria in each group were quantified using Image J. All data were analyzed using oneway ANOVA with Bonferroni post-test (\* $\mathrm { p } { < } 0 . 0 5$ ；\*\*： $\mathrm { p } { < } 0 . 0 1$ $* * *$ $\mathrm { p } { < } 0 . 0 0 0 1 \$ ） (TIF)

# Acknowledgments

We thank Drs.D. Cleveland, DW. Dickson, A. Gitler, E.Huang, L. Petruceli,R.Rademakers and Y. Zhang for sharing unpublished data and expert suggestions and for generously providing reagents and suggestions.We thank Dr. D.R. Schubert for providing HT-22 cels. We are grateful to Drs.T.Takumi and S. Kurosaka for providing FUS deficient tissues and cels. We thank W.McGee and other members of the Wu laboratory for critical reading of the manuscript, stimulating discussons and helpful suggestions.We are grateful for generous help from Drs. C. Wang, D.Han and J. Xie at National Center for Nanoscience and Technology in our SEM work. EM and confocal microscopy were carried out at the Center for Biological Imaging, IBP of CAS.

# Author Contributions

Conceived the idea and supervised the project: JYW. Designed experimental strategies: JD MY KF LZ JYW.Provided critical tissue samples: EHB MM. Performed the experiments and analyzed the data: JD MY YC XC HC LZ YL SS KF JYW.Supervised the experiments, discussed and analyzed the data: YL SD LZ JL QX JYW.Wrote the paper: JD JYW.

References 1.Aoki M,Ogasawara M,MatsubaraY,Narisawa K,NakamuraS,etal. (1993) MildALSinJapanassociated with novel SOD mutation. Nat Genet 5: 323-324. PMID: 8298637 2. Rosen DR (1993) Mutations in $\mathsf { C u / Z n }$ superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis.Nature 364: 362.PMID: 8332197 3. Andersen PM, Al-Chalabi A(2011) Clinical geneticsof amyotrophic lateral sclerosis: what do we really know? Nat Rev Neurol 7: 603-615. doi: 10.1038/nrneurol.2011.150 PMID: 21989245 4. Da Cruz S,Cleveland DW (2011) Understanding the role of TDP-43 and FUS/TLS in ALS and beyond.Curr Opin Neurobiol 21:904-919.doi: 10.1016/j.conb.2011.05.029 PMID: 21813273 5. Morris HR,WaiteAJ,WilliamsNM,Neal JW,Blake DJ(2012)Recentadvancesin the geneticsof the ALS-FTLD complex. Curr Neurol Neurosci Rep 12: 243-250.doi: 10.1007/s11910-012-0268-5 PMID: 22477152 6. Neumann M,Sampathu DM,Kwong LK,Truax AC,Micsenyi MC,etal. (2006) Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Science 314:130-133. PMID: 17023659 7. KwiatkowskiTJJr.,Bosco DA,Leclerc AL,Tamrazian E,Vanderburg CR,etal. (2009) Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. Science 323: 1205-1208. doi: 10.1126/science.1166066 PMID: 19251627 8. Vance C,RogeljB, HortobagyiT,De Vos KJ,NishimuraAL,etal. (2009) Mutations in FUS,an RNA processing protein,cause familial amyotrophic lateral sclerosis type 6.Science 323: 1208-1211.doi: 10.1126/science.1165942 PMID: 19251628 9. HansonKA,KimSH,TibbettsRS(2012) RNA-binding proteins inneurodegenerativedisease:- 43 and beyond.Wiley Interdiscip Rev RNA 3: 265-285.doi: 10.1002/wrna.111 PMID: 22028183   
10. Neumann M, Rademakers R, Roeber S,Baker M, Kretzschmar HA,et al. (2009)A new subtype of frontotemporal lobar degeneration with FUS pathology. Brain 132: 2922-2931.doi: 10.1093/brain/ awp214 PMID: 19674978   
11. Neumann M, Roeber S, Kretzschmar HA, Rademakers R, Baker M, etal. (2009) Abundant FUSimmunoreactive pathology in neuronal intermediate filament inclusion disease. Acta Neuropathol 118: 605-616. doi: 10.1007/s00401-009-0581-5 PMID: 19669651   
12. Munoz DG,Neumann M,Kusaka H,Yokota O,Ishihara K,etal.(2009) FUS pathology in basophilic inclusion body disease.Acta Neuropathol 118: 617-627.doi: 10.1007/s00401-009-0598-9 PMID: 19830439   
13. Mackenzie IR,Rademakers R, Neumann M (2010)TDP-43and FUS inamyotrophic lateral sclerosis and frontotemporal dementia.Lancet Neurol 9: 995-1007.doi: 10.1016/S1474-4422(10)70195-2 PMID:20864052   
14. Baloh RH (2012) How do the RNA-binding proteins TDP-43and FUS relate to amyotrophic lateral sclerosis and frontotemporal degeneration,and to each other? Curr Opin Neurol 25: 701-707. doi: 10.1097/WCO.0b013e32835a269bPMID:23041957   
15. Mackenzie IR, Munoz DG,Kusaka H,Yokota O,Ishihara K,etal. (2011) Distinct pathological subtypes of FTLD-FUS.Acta Neuropathol 121: 207-218.doi: 10.1007/s00401-010-0764-0 PMID: 21052700   
16. SabatelliM, MoncadaA,Conte A,Lattante S,Marangi G,etal. (2013) Mutations in the 3'untranslated regionof FUS causing FUS overexpressionare associated with amyotrophic lateral sclerosis. Hum Mol Genet 22: 4748-4755.doi: 10.1093/hmg/ddt328 PMlD: 23847048   
17. Chen Y,Yang M,Deng J,Chen X,Ye Y,etal. (2011) Expression of human FUS protein in Drosophila leads to progressive neurodegeneration. Protein Cell 2: 477-486. doi: 10.1007/s13238-011-1065-7 PMID:21748598   
18. FushimiK,Long C,JayaramN,Chen X,LiL,etal. (2011) Expression of human FUS/TLS in yeast leads to protein aggregationand cytotoxicity, recapitulating key features of FUS proteinopathy. Protein Cell 2: 141-149. doi: 10.1007/s13238-011-1014-5 PMID: 21327870   
19. Huang C, Zhou H,Tong J,Chen H,Liu YJ,etal. (2011) FUS transgenic rats develop the phenotypes of amyotrophic lateral sclerosisand frontotemporal lobar degeneration.PLoS Genet 7:e1002011. doi: 10.1371/journal.pgen.1002011 PMID: 21408206   
20. Ju S,TardiffDF, HanH,DivyaK,ZhongQ,etal. (2011)AyeastmodelofFUS/TS-dependentcyt toxicity. PLoS Biol 9: e1001052.doi: 10.1371/journal.pbio.1001052 PMID: 21541368   
21.Kryndushkin D, Wickner RB,ShewmakerF (2011) FUS/TLS forms cytoplasmic aggregates,inhibits cell growth and interacts with TDP-43 in ayeast model of amyotrophic lateral sclerosis.Protein Cell 2: 223-236.doi: 10.1007/s13238-011-1525-0 PMID:21452073   
22. Lanson NA Jr.,Maltare A,King H,Smith R,KimJH,etal. (2011)A Drosophilamodel of FUS-related neurodegeneration reveals genetic interaction between FUS and TDP-43. Hum Mol Genet 20: 2510- 2523. doi: 10.1093/hmg/ddr150 PMlD: 21487023   
23.Murakami T,Yang SP, Xie L, Kawano T,Fu D,et al. (2012)ALS mutations in FUS cause neuronal dysfunction and death in Caenorhabditis elegans by a dominant gain-of-function mechanism. Hum Mol Genet 21: 1-9. doi: 10.1093/hmg/ddr417 PMlD: 21949354   
24.Sun Z, Diaz Z,Fang X, HartMP,Chesi A,etal. (2011) Molecular determinants and genetic modifiers of aggregation and toxicity for the ALS disease protein FUS/TLS. PLoS Biol 9: e1000614. doi: 10. 1371/journal.pbio.1000614 PMID: 21541367   
25. Verbeeck C,Deng Q,Dejesus-Hernandez M,Taylor G,Ceballos-Diaz C,etal. (2012) Expressionof Fused in sarcoma mutations in mice recapitulates the neuropathology of FUS proteinopathies and provides insight into disease pathogenesis.Mol Neurodegener7: 53.doi:10.1186/1750-1326-7-53 PMID:23046583   
26．Mitchell JC,McGoldrick P,Vance C, HortobagyiT,Sreedharan J,etal. (2013) Overexpressionof human wild-type FUS causes progressive motor neuron degeneration inanage-and dose-dependent fashion. Acta Neuropathol 125: 273-288.doi: 10.1007/s00401-012-1043-z PMID: 22961620   
27. Neumann M, Valori CF,Ansorge O,Kretzschmar HA,Munoz DG,et al. (2012) Transportin 1 accumulates specificaly with FET proteins but no other transportin cargos in FTLD-FUS and isabsentin FUS inclusions in ALS with FUS mutations.Acta Neuropathol 124:705-716.doi: 10.1007/s00401-012- 1020-6 PMID:22842875   
28. Ellis RJ (2005) Chaperomics: in vivo GroEL function defined. Curr Biol 15: R661-663.PMID: 16139196   
29. Hartl FU,Martin J,Neupert W(1992) Protein folding in the cel: the role of molecular chaperones Hsp70 and Hsp60. Annu Rev Biophys Biomol Struct 21: 293-322. PMID:1525471   
30. Mayer MP (2010) Gymnastics of molecularchaperones.Mol Cell39: 321-331.doi: 10.1016/j.molcel. 2010.07.012 PMID:20705236   
31. Gupta RS,Ramachandra NB,Bowes T,Singh B (2oo8) Unusual cellular disposition of the mitochondrial molecular chaperones Hsp60,Hsp70and Hsp10.Novartis Found Symp 291: 59-68; discussion 69-73,137-140. PMID: 18575266   
32. Nakamura H, Minegishi H (2013) HSP60 as a drug target. Curr Pharm Des 19: 441-451. PMID: 22920899   
33. HaynesCM,Ron D(2010)The mitochondrial UPR-protecting organelle protein homeostasis.JCell Sci 123: 3849-3855.doi: 10.1242/jcs.075119 PMID: 21048161   
34. HeyrovskaN,FrydmanJ,HohfeldJ,HartlFU(1998)Directionalityofpolypeptide transferinthemitochondrial pathway of chaperone-mediated protein folding.Biol Chem 379: 301-309.PMID:9563826   
35. MartinJ(1997) Molecular chaperones and mitochondrial protein folding. J Bioenerg Biomembr 29: 35-43.PMID:9067800   
36. Ostermann J, Horwich AL,Neupert W,Hartl FU(1989) Protein folding in mitochondria requires complex formation with hsp60 and ATP hydrolysis.Nature 341: 125-130. PMID: 2528694   
37. Hansen J,Svenstrup K,Ang D,NielsenMN,Christensen JH,etal. (2007) Anovel mutation in the HSPD1 gene in a patient with hereditary spastic paraplegia.JNeurol 254: 897-900. PMID: 17420924   
38．Hansen JJ, DurrA, Cournu-Rebeix I, Georgopoulos C,Ang D,etal. (2002) Hereditary spastic paraplegia SPG13 is associated with a mutation in the gene encoding the mitochondrial chaperonin Hsp60.AmJHum Genet 70:1328-1332.PMID:11898127   
39. Menzies FM, Cookson MR,Taylor RW,TurnbullDM, Chrzanowska-Lightowlers ZM,etal. (2002) Mitochondrial dysfunction in a cellculture model offamilial amyotrophic lateral sclerosis.Brain 125: 1522-1533. PMID: 12077002   
40.Raimondi A, Mangolini A,Rizzardini M,Tartari S, Massari S,etal. (20o6)Cellculture models to investigatethe selective vulnerability of motoneuronal mitochondria to familial ALS-linked G93ASOD1.Eur JNeurosci 24:387-399.PMID:16903849   
41. Vande Velde C,MillerTM, Cashman NR, Cleveland DW (2008) Selective association of misfolded ALS-linked mutant SOD1 with the cytoplasmic face of mitochondria.Proc Natl Acad Sci U S A 105: 4022-4027.doi: 10.1073/pnas.0712209105 PMID: 18296640   
42. Vande Velde C,McDonald KK,BoukhedimiY,McAlonis-Downes M,Lobsiger CS,etal. (2011) Misfolded SOD1 associated with motor neuron mitochondria alters mitochondrial shape and distribution prior to clinical onset. PLoS ONE 6: e22031. doi: 10.1371/journal.pone.0022031 PMID: 21779368   
43.Cozzolino M,Ferri A, Valle C,Carri MT (2013) Mitochondria and ALS: implications from novel genes and pathways.Mol Cell Neurosci 55: 44-49.doi: 10.1016/j.mcn.2012.06.001 PMlD:22705710   
44. Shan X,Chiang PM,Price DL,Wong PC (2010)Altered distributions of Gemini of coiled bodies and mitochondriain motor neurons of TDP-43 transgenic mice.Proc Natl Acad Sci US A 107:16325- 16330.doi: 10.1073/pnas.1003459107 PMID: 20736350   
45.Xu YF,Gendron TF, Zhang YJ,Lin WL,D'Alton S,etal. (2010)Wild-type human TDP-43 expression causes TDP-43 phosphorylation,mitochondrial aggregation, motor deficits,and early mortality in transgenic mice.JNeur0sci 30: 10851-10859.doi: 10.1523/JNEUROSCl.1630-10.2010 PMiD: 20702714   
46. Huang EJ,Zhang J,GeserF,Trojanowski JQ,Strober JB,etal. (2010)Extensive FUS-immunoreactive pathologyin juvenile amyotrophic lateral sclerosis with basophilic inclusions.Brain Pathol 20: 1069-1076. doi: 10.1111/j.1750-3639.2010.00413.x PMID: 20579074   
47. TradewellML,YuZ,TibshiraniM,Boulanger MC,Durham HD,etal.(2012) Arginine methylation by PRMT1 regulates nuclear-cytoplasmic localization and toxicity of FUS/TLS harbouring ALS-linked mutations. Hum Mol Genet 21: 136-149.doi: 10.1093/hmg/ddr448 PMlD: 21965298   
48.Aoun P,Watson DG, Simpkins JW (2003) Neuroprotective effects of PPARgamma agonists against oxidative insults in HT-22 cells.EurJPharmacol 472:65-71.PMID:12860474   
49. Pilling AD,Horiuchi D,LivelyCM,SaxtonWM(2006) Kinesin-1andDyneinare the primarymotorsfor fast transport of mitochondriain Drosophila motor axons.Mol Biol Cell 17: 2057-2068.PMID: 16467387   
50. Yao A,JinS,LiX,LiuZ,MaX,etal. (2011) DrosophilaFMRP regulates microtubule networkformationand axonal transport of mitochondria.Hum Mol Genet 20:51-63.doi: 10.1093/hmg/ddq431 PMID:20935173   
51. MitraK,RikhyR,LillyM,Lippincott-SchwartzJ(2012) DRP1-dependentmitochondrialfission initiates follicle cell differentiation during Drosophila oogenesis.JCellBiol197: 487-497.doi: 10.1083/jcb. 201110058 PMID: 22584906   
52. Waterham HR, Koster J,van Roermund CW,Mooyer PA,Wanders RJ,etal. (2007)Alethal defectof mitochondrial and peroxisomal fission.N Engl JMed 356: 1736-1741.PMID: 17460227   
53. Frezza C, Cipolat S, Scorrano L (2007) Organelle isolation: functional mitochondria from mouse liver, muscle and cultured fibroblasts. Nat Protoc 2: 287-295. PMID:17406588   
54.Dunkley PR,Jarvie PE, Robinson PJ(2008)A rapid Percollgradient procedure for preparation of synaptosomes.NatProtoc 3:1718-1728.doi:10.1038/nprot.2008.171PMID:18927557   
55.Verburg J, Holenbeck PJ (2008) Mitochondrial membrane potential in axons increases with local nerve growth factoror semaphorin signaling. JNeurosci 28: 8306-8315.doi: 10.1523/JNEUROSCl. 2614-08.2008 PMID: 18701693   
56. Wu S,Zhou F, Zhang Z, Xing D (2011) Mitochondrial oxidative stress causes mitochondrialfragmentation via diferential modulation of mitochondrial fission-fusion proteins.FEBS J278:941-954.doi: 10.1111/j.1742-4658.2011.08010.xPMID:21232014   
57. Lashley T,Rohrer JD,BandopadhyayR,Fry C,Ahmed Z,etal. (2011)Acomparative clinical,pathological,biochemicaland genetic study of fused insarcoma proteinopathies.Brain134:2548-2564. doi: 10.1093/brain/awr160 PMID: 21752791   
58. Page T, Gitcho MA, Mosaheb S, Carter D,Chakraverty S,et al. (2011) FUS immunogold labeling TEManalysis of the neuronalcytoplasmic inclusions of neuronal intermediate filament inclusion disease: a frontotemporal lobar degeneration with FUS proteinopathy. JMol Neurosci 45: 409-421. doi: 10.1007/s12031-011-9549-8 PMID: 21603978   
59. RabbitsTH,ForsterA,Larson R,NathanP(1993)Fusionof the dominant negative transcription regulator CHOP with a novel gene FUS by translocation t(12;16)in malignantliposarcoma.Nat Genet 4: 175-180.PMID:7503811   
60. Lagier-Tourenne C,Polymenidou M, Cleveland DW (2010)TDP-43and FUS/TLS: emerging roles in RNA processing and neurodegeneration. Hum Mol Genet 19: R46-64.doi: 10.1093/hmg/ddq137 PMID:20400460   
61. Traynor BJ, Singleton AB (2009)What's the FUS! Lancet Neurol 8: 418-419.doi: 10.1016/S1474- 4422(09)70088-2 PMID: 19375659   
62. ZinsznerH,SokJ,ImmanuelD,YinY,Ron D(1997)TLS(FUS)binds RNAinvivoandengages in nucleo-cytoplasmic shutling.J Cell Sci 110 (Pt 15): 1741-1750. PMID: 9264461   
63. TruscottKN,Brandner K,PfannerN(2003) Mechanisms of protein import into mitochondria.Current Biology 13: R326-R337. PMID: 12699647   
64. SchwartzJC,Ebmeier CC,Podell ER, Heimiller J,Taatjes DJ,etal.(2013) FUS binds the CTDof RNA polymerase Iland regulates its phosphorylationat Ser2 (vol 26,pg 2690,2012). Genes & Development 27:699-699.   
65.Claros MG, Vincens P(1996) Computational method to predict mitochondriall imported proteins and their targeting sequences.EurJBiochem 241:779-786.PMID:8944766   
66.Young JC, Hoogenraad NJ, Hartl FU (2003) Molecular chaperones Hsp90 and Hsp70 deliver preproteins to the mitochondrial import receptor Tom70. Cell 112: 41-50.PMlD: 12526792   
67．Wals KC,Coskun P,Gallegos-Perez JL, Zadourian N,Freude K,etal. (2012) Swedish Alzheimer mutation induces mitochondrialdysfunctionmediated by HSP60 mislocalization of amyloid precursor protein (APP)and beta-amyloid.JBiol Chem 287: 30317-30327.doi: 10.1074/jbc.M112.365890 PMID: 22753410   
68. Magnoni R,Palmfeldt J, HansenJ,ChristensenJH,Corydon TJ,etal.(2014) The Hsp60 folding machineryiscrucial formangane roxide dismutase folding and function.Free Radic Res 48: 168-179. doi: 10.3109/10715762.2013.858147 PMID: 24151936   
69.ChandraD, Choy G,Tang DG (2007) Cytosolic accumulation of HSP60 during apoptosis with or without apparent mitochondrial release—Evidence thatits pro-apoptotic or pro-survival functions involve diferential interactions with caspase.Journal of Biological Chemistry 282: 31289-31301.PMID: 17823127   
70. Yang TT, Hsu CT,Kuo YM (2009)Amyloid precursor protein, heat-shock proteins,and Bcl-2 form a complex in mitochondria and modulate mitochondria function and apoptosis in N2a cells.Mech Ageing Dev 130: 592-601.doi: 10.1016/j.mad.2009.07.002 PMID: 19622370   
71.Xanthoudakis S, Roy S, Rasper D, Hennessey T,Aubin Y,et al.(1999) Hsp60 accelerates the maturation of pro-caspase-3by upstreamactivator proteases during apoptosis.EMBOJ18:2049-2056. PMID:10205159   
72.Lehnardt S,Schot E,Trimbuch T,Laubisch D,Krueger C,etal. (20o8)A vicious cycle involving release of heat shock protein 60 from injured celsand activation of tol-like receptor 4 mediates neurodegenerationin the CNS.JNeurosci 28: 2320-2331.doi: 10.1523/JNEUROSCl.4760-07.2008 PMID:18322079   
73. Adams MD,Celniker SE, Holt RA,Evans CA,Gocayne JD,etal. (2000) The genome sequence of Drosophila melanogaster. Science 287: 2185-2195. PMID:10731132   
74.RubinGM,YandellMD,WortmanJR,GaborMiklos GL,NelsonCR,etal. (2000)Comparative geno mics of the eukaryotes.Science 287: 2204-2215. PMID: 10731134   
75. Baena-Lopez LA,Alonso J,RodriguezJ, Santaren JF (2oo8) The expression of heat shock protein HSP60A revealsa dynamic mitochondrial pattern in Drosophilamelanogaster embryos.JProteome Res 7: 2780-2788.doi: 10.1021/pr800006x PMID: 18549261   
76. Timakov B, Zhang P (2001) The hsp60B gene of Drosophila melanogaster is essential for the spermatid individualization process. Cell Stress Chaperones 6: 71-77. PMlD: 11525246   
77. Sarkar S,Lakhotia SC (2005) The Hsp60C gene in the 25F cytogenetic region in Drosophila melanogaster is essentialfor tracheal development and fertility.JGenet 84: 265-281.PMID: 16385159   
78. Arya R,Lakhotia SC (2008) Hsp60D is essentialfor caspase-mediated induced apoptosis in Drosophila melanogaster. Cel Stress Chaperones 13: 509-526. doi: 10.1007/s12192-008-0051-3 PMID: 18506601   
79．Ince PG, HighleyJR,KirbyJ,Wharton SB,TakahashiH,etal.(2011) Molecularpathologyand genetic advances inamyotrophic lateral sclerosis: an emerging molecular pathway and the significance of glial pathology. Acta Neuropathol 122: 657-671.doi: 10.1007/s00401-011-0913-0 PMID: 22105541   
80.Lattante S, Rouleau GA,Kabashi E(2013) TARDBP and FUS mutations associated with amyotrophic lateral sclerosis: summary and update. Hum Mutat 34: 812-826.doi: 10.1002/humu.22319 PMID: 23559573   
81. DiMauro S,Schon EA (20o8) Mitochondrial disorders in the nervous system.Annu Rev Neurosci 31: 91-123. doi: 10.1146/annurev.neuro.30.051606.094302 PMID: 18333761   
82. Lezi E,Swerdlow RH (2012) Mitochondria in neurodegeneration.Adv Exp Med Biol 942: 269-286. doi: 10.1007/978-94-007-2869-1_12 PMID:22399427   
83. Lin MT,Beal MF (2oo6) Mitochondrial dysfunction and oxidative stress in neurodegenerative diseases. Nature 443: 787-795. PMID: 17051205   
84. Spinazzi M,CazzolaS,Bortolozzi M,Baracca A,Loro E,etal. (2oo8)Anovel deletion in the GTPase domain of OPA1 causes defects in mitochondrial morphology and distribution,but notin function. Hum Mol Genet 17: 3291-3302.doi: 10.1093/hmg/ddn225 PMlD: 18678599   
85.Dormann D,Rodde R,EdbauerD,Bentmann E,Fischerl,etal. (2010) ALS-associated fused in sarcoma (FUS) mutations disrupt Transportin-mediated nuclear import.EMBOJ29:2841-2857.doi: 10 1038/emboj.2010.143 PMID: 20606625 86.Baumer D,Hilton D,Paine SM,Turner MR,Lowe J,etal.(2010)Juvenile ALS with basophilic inclusions is a FUS proteinopathy with FUS mutations. Neurology 75: 611-618.doi: 10.1212/WNL. 0b013e3181ed9cde PMID: 20668261   
87.Chio A,Restagno G,Bruneti M,Ossolal,Calvo A,etal. (2Oo9)Two Italian kindreds with familial amyotrophic lateral sclerosis due to FUS mutation.Neurobiol Aging 30:1272-1275.doi: 10.1016/j. neurobiolaging.2009.05.001 PMID: 19450904 88. Kabashi E,BercierV,Lissouba A,Liao M,Brustein E,etal.(2011)FUS and TARDBP but not SOD1 interact in genetic models of amyotrophic lateral sclerosis.PLoS Genet 7:e1002214.doi:10.1371/ journal.pgen.1002214 PMID: 21829392 89. Wang JW,Brent JR,Tomlinson A, ShneiderNA,McCabe BD (2011) The ALS-associated proteins FUS and TDP-43 function together to afect Drosophila locomotion andlife span.JClin Invest 121: 4118-4126.doi: 10.1172/JCI57883 PMID:21881207 90. LiY, Ray P,Rao EJ,ShiC,Guo W,etal. (2010)A Drosophila model forTDP-43 proteinopathy.Proc Natl Acad Sci USA 107: 3169-3174. doi: 10.1073/pnas.0913602107 PMID:20133767 91. Li HS,Chen JH,WuW,FagalyT,Zhou L,etal. (1999) Vertebrate slit,a secreted ligand for the transmembrane protein roundabout,is a repellent for olfactory bulb axons. Cell 96: 807-818.PMID: 10102269 92. Wu W,Wong K,Chen J,Jiang Z, Dupuis S,etal. (1999) Directional guidance of neuronal migration in the olfactory system by the protein Slit. Nature 400: 331-336.PMID: 10432110 93. Yuasa-Kawada J, Kinoshita-Kawada M,Wu G,RaoY,Wu JY (2009) Midline crossing and Slit responsiveness of commissural axons require USP33.Nat Neurosci 12:1087-1089.doi: 10.1038/nn.2382 PMID: 19684588 94. Guo W,Chen Y,Zhou X, Kar A, RayP,etal.(2011) An ALS-associated mutation affecting TDP-43 enhances protein aggregation,fibril formation and neurotoxicity.Nat Struct Mol Biol18:822-830.doi: 10.1038/nsmb.2053 PMiD: 21666678 95. Orozco D,Tahirovic S,Rentzsch K,Schwenk BM, Haass C,etal. (2012) Loss of fused insarcoma (FUS) promotes pathological Tau splicing.EMBO Rep 13:759-764.doi: 10.1038/embor.2012.90 PMID:22710833 96. Yano H,Baranov SV,Baranova OV, Kim J,PanY,etal. (2014) Inhibition of mitochondrial protein import by mutant huntingtin.Nat Neurosci 17: 822-831.doi: 10.1038/nn.3721 PMlD: 24836077   
97. Liu-YesucevitzL,BilgutayA,Zhang YJ, Vanderweyde T,Citro A,et al. (2010) Tar DNA binding protein-43 (TDP-43)associates with stress granules: analysis of cultured cels and pathological brain tissue. PLoS ONE 5: e13250. doi: 10.1371/journal.pone.0013250 PMID: 20948999 98. Song W,Chen J,PetriliA,Liot G,Klinglmayr E,etal. (2011) Mutant huntingtin binds the mitochondrial fission GTPase dynamin-related protein-1and increases its enzymatic activity.Nat Med 17: 377- 382.doi: 10.1038/nm.2313 PMID:21336284 99. Paumard P,Vailier J,Coulary B,SchaefferJ,SoubannierV,etal. (2002)The ATP synthase is involved in generating mitochondrial cristae morphology.EMBOJ21: 221-230.PMID:11823415   
100. Gavin PD,Prescott M,LuffSE, Devenish RJ(2004) Cross-linking ATP synthase complexes in vivo eliminates mitochondrial cristae.JCell Sci 117:2333-2343.PMlD:15126633