# SARS-CoV-2-encoded Nucleocapsid Protein Acts as a Viral Suppressor of RNA

interference in Cells

Jingfang $\mathrm { { M u } ^ { 1 } }$ , Jiuyue $\mathrm { X u } ^ { 1 , 2 }$ ,Ting $\mathrm { S h u } ^ { 1 , 3 }$ ,Di $\mathrm { W u } ^ { 1 }$ , Muhan Huangl, Yujie Ren1,4, Xufang $\mathrm { L i } ^ { 4 }$ , Qing Geng5, Yi $\mathrm { X u } ^ { 4 }$ , Yang Qiu1,2\*,and Xi Zhou1,2,3\*

1 State Key Laboratory of Virology, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences,Wuhan, Hubei, 430071 China   
² The University of Chinese Academy of Sciences, Beijing,100049 China   
3 Center for Translational Medicine, Wuhan Jinyintan Hospital, Wuhan, Hubei, 430040 China   
4 Center for Precision Translational Medicine of Wuhan Institute of Virology and Guangzhou Women and Children's Medical Center, Guangzhou Women and Children's Medical Center, Guangzhou, Guangdong, 51O120, China   
5 Department of Thoracic Surgery, Renmin Hospital of Wuhan University, Wuhan, Hubei, 430060 China   
\* Correspondence: zhouxi@ wh.iov.cn (X.Z.) and yangqiu@wh.iov.cn (Y.Q.); Tel.: +86-27-87197080

Key words: SARS-CoV-2, nucleocapsid, antiviral RNAi, viral suppressor of RNAi

# Abstract

The SARS-CoV-2 outbreak has emerged and is still ongoing in Wuhan and other areas of China and world. Human infections by SARS-CoV-2 lead to diseases ranging from mild symptoms to severe pneumonia and even death. And in the current situation, better understanding of the virology and virus-host interactions of SARS-CoV-2 would be vital for the efforts to control the infections and develop effective therapies. RNA interference (RNAi) is an evolutionarily conserved antiviral immune mechanism in diverse eukaryotic organisms,and numerous viruses have been found to encode their own viral suppressors of RNAi(VSRs) as countermeasures.In this study, we uncovered that the nucleocapsid (N) protein encoded by SARS-CoV-2 effectively suppressed RNAi triggered by either small hairpin RNAs (shRNAs) or small interfering RNAs (siRNAs) in cultured human cells. Furthermore, similar with VSRs encoded by other viruses, SARS-CoV-2 N protein shows double-stranded RNA(dsRNA)-binding activity, as it interacted with in vitro transcribed dsRNAs in human cells. Taken together, our findings showed that SARS-CoV-2 N exhibits the VSR activity in human cells, which could be as a key immune evasion factor for SARS-CoV-2 and contribute to its pathogenicity.

# Introduction

Coronaviruses (CoVs) are large enveloped non-segmented positive-stranded RNA viruses that broadly distribute among humans and other animal species, including bats, mice, birds, etc [1,2]. Among six pathogenic species of coronavirus, CoV-229E, CoVOC43, CoV-NL63 and CoV-HKU1 typically cause mild symptoms,whereas severe acute respiratory syndrome coronavirus (SARS-CoV） and Middle East respiratory syndrome coronavirus (MERS-CoV) have caused severe respiratory disease outbreaks in the past two decades [1,3,4]. Since December 2019, an outbreak of mysterious pneumonia has been reported in Wuhan City, Hubei Province, China. On early January 2020, the causative agent of the mysterious viral pneumonia had been identified and isolated as a novel coronavirus by several Chinese institutions at Wuhan and Beijing. This novel coronavirus was initially named as 2019 novel coronavirus (2019-nCoV) and later as SARS-CoV-2, while the disease caused by SARS-CoV-2 is named as 2019 novel coronavirus disease (Covid-19) by the World Health Organization (WHO). According to clinical observations, SARS-CoV-2 infections can cause diseases ranging from mild symptoms to severe respiratory syndromes, including pneumonia, and even death,and its human-to-human transmission had also been confirmed [5,6]. So far, the Covid-19 outbreak has been reported to cause more than 76,Oo0 confirmed cases, and has been declared by WHO as a global public health emergency.

RNAi is a post-transcriptional gene silencing mechanism that is evolutionarily conserved in all eukaryotes and has been recognized as a cell-intrinsic antiviral immune defense mechanism in diverse eukaryotes including mammals [7]. In antiviral RNAi, viral infection and replication generates virus-derived dsRNA (vi-dsRNA), which could be recognized and cleaved by the host endoribonuclease Dicer into virus-derived siRNAs (vsiRNAs). These vsiRNAs are integrated into the Argonaute protein within the RNA-induced silencing complex (RISC) to direct the destruction of cognate viral RNAs in infected cells in a sequence-specific manner [8]. As a countermeasure, viruses encode viral suppressors of RNAi (VSRs) to antagonize the RNAi pathway at different steps [9]. Nodamura virus (NoV) B2, Influenza A virus (IAV) NS1, human enterovirus A71 (EV-A71) 3A, Dengue virus 2 (DENV2) NS2A,and Semliki Forest virus (SFV) Capsid have been identified to act as VSRs to inhibit vsiRNA production and antiviral RNAi response in the context of authentic viral infections [1O-15]. In addition, Wuhan Nodavirus (WhNV) B2, Ebola virus (EBOV) VP35,HIV-1 TAT, Hepacivirus core and NS2, Yellow Fever virus capsid, Chikungunya virus nsP2 and nsP3,have been found to suppress RNAi in vitro [16-22]. The phenomena of numerous viruses encoding VSRs to suppress the RNAi pathway highlight the importance of antagonizing antiviral RNAi during viral life cycle. Therefore, the understanding of how SARS-CoV-2 interacts with antiviral RNAi is important for the better knowledge of this novel coronavirus,and may contribute to the efforts of controlling the spread of infections and developing effective therapies.

Previous study has reported that SARS-CoV nucleocapsid (N) protein displayed a VSR activity in mammalian cells via a cellular reversal-of-silencing assay [23]. Given the high homology of the amino acid sequences among coronavirus N proteins, it is intriguing to examine whether SARS-CoV-2 N also contains the VSR activity. Therefore,in the current study，we evaluated the role of SARS-CoV-2 N in the suppression of RNAi in cultured human cells. Our finding demonstrated that SARSCoV-2 N does possess the VSR activity to inhibit RNAi response in cells,which probably acts as one of the key immune evasion factors of SARS-CoV-2 and contribute to its pathogenicity, thereby representing a potential target for antiviral therapy.

# RESULTS

# SARS-CoV-2 N suppressed shRNA-induced RNAi in cultured human cells

We first examined whether SARS-CoV-2 N possessed VSR activity via a classic reversal-of-silencing assay, in which enhanced green fluorescent protein (EGFP)- specific shRNA was transfected into EGFP-expressing 293T cells, together with a plasmid encoding SARS-CoV-2 N protein with Flag tag. At 48 hr post-transfection (hpt), EGFP protein levels were examined via fluorescent microscopy and western blotting respectively. EGFP-specific shRNA expression resulted in low EGFP protein levels (Fig.1A, panel “Vec"; Fig. 1B,lane “Vec"), confirming the efficiency of shRNA in this RNAi system. Our data showed that expression of SARS-CoV-2 N markedly restored the protein level of EGFP (Fig. 1A, panel “N";Fig. 1B, lane“N"), indicating that SARS-CoV-2 N displays the VSR activity in cells. Of note, the ectopic expression of EBOV VP35,a well-characterized VSR, suppressed the shRNA-induced RNAi as expected (Fig. 1B, lane “VP35").

Because RNAi directly results in the cleavage and degradation of target mRNAs, we further examined the VSR activity of SARS-CoV-2 N using the reversal-ofsilencing system via northern bloting with a digoxin (DIG)-labeled RNA probe targeting EGFP ORF 1-400 nt. Our results showed that SARS-CoV-2 N markedly restored the EGFP mRNA levels in 293T cells (Fig. 1C). Taken together, our data show that SARS-CoV-2-encoded N protein has the VSR activity in cultured human cells.

# SARS-CoV-2 N sequestrated dsRNAs in cells

Having established that SARS-CoV-2 N contains VSR activity, we sought to examine the mechanism of how SARS-CoV-2 N antagonizes RNAi. During the dsRNA/shRNA-induced RNAi, dsRNA/shRNA is initially recognized and cleaved by

Dicer into siRNA. Previous studies also found that numerous VSRs encoded by different viruses antagonize antiviral RNAi via sequestering viral dsRNAs. Thus, we examined whether SARS-CoV-2 N can sequestrate dsRNA via the RNA-IP assay. In brief, 293T cells expressing Flag-tagged $\mathrm { \Delta N }$ or empty vector, together with EGFPspecific dsRNA (EGFP ORF 1-200nt) were lysed and immunoprecipitated with antiFlag or mouse IgG antibodies, respectively. The RNAs extracted from the RNA-IP precipitates were then examined via northern blotting with RNA probes targeting the 1- 200 nt dsRNAofEGFP. Our results showed that SARS-CoV-2 N does, indeed,associate with dsRNA in 293T cells (Fig.2), implying that the mechanism by which SARS-CoV2 N suppresses RNAi is to sequestrate dsRNA in cells, which probably prevents the recognition and cleavage of viral dsRNA by Dicer.

# SARS-CoV-2 N suppressed siRNA-induced RNAi in mammalian cells

In the process of RNAi, Dicer-cleaved siRNAs are required to assemble siRNAincorporated RISC to direct the degradation of cognate RNAs, which is the effector step of RNAi [24]. After establishing that SARS-CoV-2 N can associate with dsRNA, we further examined whether SARS-CoV-2 N could also suppress siRNA-induced RNAi. To this end, we co-transfected chemically synthesized EGFP-specific siRNA together with the plasmid for SARS-CoV-2 N into EGFP-expressing 293T cells. And the effects of RNAi were determined via fluorescent microscopy and Western blotting to detect EGFP protein expression, or via Northern blotting to detect EGFP mRNA level. EGFPspecific siRNA reduced the protein and mRNA levels of EGFP,while the ectopic expression of SARS-CoV-2 N efficiently restored the expression of EGFP in both the protein and mRNA levels (Fig. 3A-C). Consistent with previous findings, EBOV VP35, which was used as a positive control, also restored the expression of EGFP in both protein and mRNA levels as expected (Fig. 3B and 3C). Our findings indicate that SARS-CoV-2 N can suppress siRNA-induced RNAi in cells, implying that SARS-CoV$2 \mathsf { N }$ antagonizes RNAi in the effector step, either.

#

Discussion   
The emergence of SARS-CoV-2 outbreak has caused and is causing serious threat to human health and tremendous economic loss in China and across the globe,which pushes us to obtain the knowledge about the all aspects of characteristics of this novel coronavirus as quickly and as more as possible.RNAi is an antiviral response conserved in all eukaryotes including mammals, and viruses encode VSRs to antagonize RNAi [24].In this study，we found that the SARS-CoV-2-encoded structural protein N displayed VSR activity in cultured human cells. Our findings showed that SARS-CoV（2 $2 \ : \mathrm { N }$ can antagonize RNAi in both initiation (i.e. siRNA biogenesis) and effector (i.e. RISC assembly and target RNA cleavage) steps. Therefore, our data support the notion that N protein can act as the VSR to facilitate SARS-CoV-2 replication by shielding viral dsRNA and siRNA from Dicer cleavage and RISC assembly, respectively. The finding that SARS-CoV-2 N suppresses RNAi in cells is consistent with the previous observation that SARS-CoV N also displayed VSR activity [23],implying that using N protein as the VSR is a common strategy for coronaviruses to antagonize antiviral RNAi. Moreover, in addition to N protein, previous study has identified SARSCoV 7a could suppress RNAi in mammalian cells [25], suggesting that coronaviruses may antagonize RNAi by encoding multiple VSRs. Having multiple VSRs is not uncommon, as several other RNA viruses, such as EBOV, DENV,and HIV-1 also have been reported to encode more than one VSRs [21, 22, 26-30]. Encoding multiple VSRs may offer these pathogenic viruses extra advantages for efficient inhibition of RNAi, highlighting the importance of antiviral RNAi for host cells in defending viral infection.

Coronavirus N protein contains nonspecific RNA-binding activity[31, 32]. In this study, we also found that SARS-CoV-2 N could associate with dsRNA in cels. Our results that SARS-CoV-2 N suppressed RNAi by sequestrating dsRNA are consistent with the previous findings that coronavirus $\mathrm { \Delta N }$ is directly involved in viral RNA replication [33, 34]. During viral life cycle, coronavirus $\mathrm { \Delta N }$ protein encapsulates viral genomic RNAs to protect the genome and co-enter the host cell with viral genomic RNAs, indicating that $\mathrm { \Delta N }$ is important for viral RNA replication，especially at the initiation step [33, 34].

In summary, SARS-CoV-2 can act as a VSR in cells in both initiation and effector steps od RNAi, thereby probably representing a key immune evasion factor of SARSCoV-2 and contributing to the pathogenicity of this novel coronavirus. Our study extends our knowledge about the interaction between antiviral RNAi immunity and SARS-CoV-2 in a timely manner and may be helpful in the efforts of controlling this dangerous virus.

# MaterialsandMethods

# Cell culture and transfection

Human embryonic kidney 293T (HEK293T) cells were commercially obtained from the American Type Culture Collection and maintained in modified Eagle's medium supplemented with $10 \%$ fetal bovine serum (Gibco), $1 0 0 \mathrm { U / m l }$ penicillin, and $1 0 0 \mu \mathrm { g / m l }$ streptomycin at $3 7 ^ { \circ } \mathrm { C }$ in an incubator with $5 \% \mathrm { C O } _ { 2 }$ . Transfection ofHEK293T cells was performed using Lipofectamine 2Ooo (Invitrogen) according to the manufacturer's instructions.

# Plasmid construction

pRK-Flag-SARS-CoV-2 N was generated using a standard cloning protocol. The cDNA fragment of SARS-CoV-2 N was generated by reverse transcription, the RNA template was isolated from SARS-CoV-2-infected cells. pRK-Flag- EBOV VP35 was described previously [35]. The primers used in this study are shown in Table S1.

# Northern blotting

Total cellular RNAs were extracted using Trizol reagent (Tiangen, Beijing, China) according to the manufacturer's protocol. Northern blotting was performed as previously described [12]. Briefly, $5 \mu \ g$ of total RNAs were electrophoresed on denaturing $1 . 2 \%$ agarose gels containing $2 . 2 \mathrm { ~ M ~ }$ formaldehyde,and then capillary transferred to Hybond-A nylon membrane (GE Healthcare). The membranes were hybridized with the indicated DIG-labeled RNA probes at $6 5 ^ { \circ } \mathrm { C }$ for $1 2 \mathrm { ~ h ~ }$ ，and then incubated with anti-DIG-alkaline phosphatase antibody (Roche). The membranes were then incubated with CDP-STAR (Roche) at $3 7 ^ { \circ } \mathrm { C }$ for $1 0 \mathrm { { m i n } }$ . The signals were detected by radiography on X-ray film (FujiFilm, Tokyo, Japan). The DIG-RNA probe targeting

EGFP was produced via in vitro transcription. Intact 28s and 18s rRNAs were observed using ethidium bromide staining and were used as loading controls. The primers used for the preparation of RNA probes in this study are shown in Table S1.

# Western blotting

Cells were washed twice in cold phosphate-buffered saline and lysed in lysis buffer containing 50 mM Tris-HCl (pH7.4), 150 mM NaCl, $0 . 5 \%$ NP40, $0 . 2 5 \%$ deoxycholate, and a protease inhibitor cocktail (Roche),and the lysates were then subjected to $12 \%$ SDS-PAGE and western blotting according to standard procedures with the relevant antibodies.

# RNA-immunoprecipitation (RNA-IP)

RNA-IP was performed as previously described [12]. Briefly,cells were lysed in lysis buffer containing $2 0 \mathrm { m M }$ Tris-HCl (pH7.4),100 mM NaCl, 2.5 mM MgCl2, $0 . 5 \%$ NP40, $0 . 5 \mathrm { U / \mu L }$ RNase inhibitor (Promega) and a protease inhibitor cocktail (Rochel) at $4 ^ { \circ } \mathrm { C }$ for $3 0 \mathrm { m i n }$ . Lysates were clarified at 12, $0 0 0 \times \mathtt { g }$ for $1 0 \mathrm { m i n }$ at $4 ^ { \circ } \mathrm { C }$ and post-nuclear lysates were incubated with antibodies (anti-FLAG or anti- $\mathrm { \cdot I g G }$ ） together with proteinA/G agarose beads (Roche) at $4 ^ { \circ } \mathrm { C }$ for $4 \mathrm { h }$ . The antibody-bound complexes were washed five times with the same lysis buffer. Finally, proteins or RNAs were extracted from the complexes and analyzed by western or northern blotting as described above.

# Acknowledgments

We wish to thank Prof.P. Zhou (Wuhan, China) for kindly provided materials. This work was supported by the Strategic Priority Research Program of CAS   
(XDB29010300 to X.Z.), National Natural Science Foundation of China (31800140 to   
J.M., 81873964 to Y.Q., and 31670161 to X.Z.), the National Science and Technology   
Major Project (2018ZX10101004 to X.Z.), the Yunde Hou Academician Fund from   
National Institute for Viral Disease Control and Prevention (2019HYDQNJJ10 to J.M.).

# Conflicts of interest

The authors declare that there are no conflicts of interest.

References   
1.de Wit E,van Doremalen N, Falzarano D, Munster VJ. SARS and MERS: recent insights into emerging coronaviruses. Nat Rev Microbiol 2016;14(8):523-534.   
2.Chen Y, Liu Q, Guo D. Emerging coronaviruses: genome structure, replication, and pathogenesis. J Med Virol 2020.   
3．Cui J,LiF, Shi ZL. Origin and evolution of pathogenic coronaviruses. Nat Rev Microbiol 2019;17(3):181-192.   
4.Zhou P, Yang XL, Wang XG, Hu B, Zhang L et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020.   
5.Huang C, Wang Y,Li X,Ren L, Zhao J et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 2020.   
6. Chen N, Zhou M, Dong X,Qu J, Gong F et al. Epidemiological and clinical characteristics of99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet 2020.   
7.Ding SW, Han Q, Wang J, Li WX. Antiviral RNA interference in mammals. Curr Opin Immunol 2018;54:109-114.   
8.Ding SW. RNA-based antiviral immunity. Nat Rev Immunol 2010;10(9):632-644. 9.Guo Z, Li Y, Ding SW. Small RNA-based antimicrobial immunity. Nat Rev Immunol 2019;19(1):31-44.   
10. Qiu Y, Xu YP, Wang M, Zhou H, Xu J et al. Flavivirus induces and antagonizes antiviral RNA interference in both mammals and mosquitoes. Sci Adv 2020;6:eaax7989.   
11． Qian Q, Zhou H, Shu T,Mu J,Fang Y et al. The Capsid Protein of Semliki Forest Virus

Antagonizes RNA Interference in Mammalian Cells.J Virol 2020;94(3).

12. Qiu Y, Xu Y, Zhang Y, Zhou H, Deng YQ et al. Human Virus-Derived Small RNAs Can Confer Antiviral Immunity in Mammals. Immunity 2017;46(6):992-1004 e1005.

13．Li Y, Basavappa M, Lu J, Dong S, Cronkite DA et al. Induction and suppression of antiviral RNA interference by influenza A virus in mammalian cells. Nat Microbiol 2016;2:16250.   
14. Maillard PV, Ciaudo C, Marchais A,Li Y, Jay F et al. Antiviral RNA interference in mammalian cells. Science 2013;342(6155):235-238.   
15. Li Y,Lu J, Han Y,Fan X, Ding SW. RNA interference functions as an antiviral immunity mechanism in mammals. Science 2013;342(6155):231-234.   
16. Zhou H, Qian Q, Shu T, Xu J, Kong Jet al. Hepatitis C Virus NS2 Protein Suppresses RNA Interference in Cells. Virol Sin 2019.   
17. Samuel GH, Wiley MR, Badawi A, Adelman ZN, Myles KM. Yellow fever virus capsid protein is a potent suppressor of RNA silencing that binds double-stranded RNA. Proc Natl Acad Sci U S A 2016;113(48):13863-13868.   
18. Mathur K, Anand A, Dubey SK, Sanan-Mishra N, Bhatnagar RK et al. Analysis of chikungunya virus proteins reveals that non-structural proteins nsP2 and nsP3 exhibit RNA interference (RNAi) suppressor activity. Sci Rep 2016;6:38065.   
19. Qi N, Zhang L, Qiu Y, Wang Z, Si J et al. Targeting of dicer-2 and RNA by a viral RNA silencing suppressor in Drosophila cells. J Virol 2012;86(10):5763-5773.   
20. Chen W, Zhang Z, Chen J, Zhang J, Zhang J et al. HCV core protein interacts with Dicer to antagonize RNA silencing. Virus Res 2008;133(2):250-258.

21. Haasnoot J, de Vries W, Geutjes EJ,Prins M, de Haan Pet al. The Ebola virus VP35 protein is a suppressor of RNA silencing. PLoS Pathog 2007;3(6):e86.

22. Bennasser Y,Le SY, Benkirane M, Jeang KT.Evidence that HIV-1 encodes an siRNA and a suppressor of RNA silencing. Immunity 2005;22(5):607-619.   
23. Cui L, Wang H, Ji Y, Yang J, Xu S et al. The Nucleocapsid Protein of Coronaviruses Acts as a Viral Suppressor of RNA Silencing in Mammalian Cells. J Virol 2015;89(17):9029- 9043.   
24. Maillard PV, van der Veen AG,Poirier EZ, Reis ESC. Slicing and dicing viruses: antiviral RNA interference in mammals. EMBO J2019;38(8):e100941.   
25. Karjee S, Minhas A, Sood V, Ponia SS, Banerjea AC et al. The 7a accessory protein of severe acute respiratory syndrome coronavirus acts as an RNA silencing suppressor. $J$ Virol 2010;84(19):10395-10401.   
26. Aqil M, Naqvi AR, Bano AS, Jameel S. The HIV-1 Nef protein binds argonaute-2 and functions as a viral suppressor of RNA interference. PLoS One 2013;8(9):e74472.   
27. Fabozzi G, Nabel CS, Dolan MA, Sullivan NJ. Ebolavirus proteins suppress the effects of small interfering RNA by direct interaction with the mammalian RNA interference pathway. J Virol 2011;85(6):2512-2523.   
28. Kakumani PK, Rajgokul KS, Ponia SS, Kaur I, Mahanty S et al. Dengue NS3, an RNAi suppressor, modulates the human miRNA pathways through its interacting partner. Biochem J2015;471(1):89-99.   
29. Kakumani PK,Ponia SS,S RK, Sood V, Chinnappan M et al. Role ofRNA interference (RNAi) in dengue virus replication and identification of NS4B as an RNAi suppressor. J Virol 2013;87(16):8870-8883.   
30. Schnettler E, Sterken MG, Leung JY, Metz SW, Geertsema C et al. Noncoding flavivirus RNA displays RNA interference suppressor activity in insect and Mammalian cells. J Virol 2012;86(24):13486-13500.   
31. Tang TK, Wu MP, Chen ST, Hou MH, Hong MH et al. Biochemical and immunological studies of nucleocapsid proteins of severe acute respiratory syndrome and 229E human coronaviruses. Proteomics 2005;5(4):925-937.   
32. Takeda M, Chang CK, Ikeya T, Guntert P, Chang YH et al. Solution structure of the cterminal dimerization domain of SARS coronavirus nucleocapsid protein solved by the SAILNMR method. J Mol Biol 2008;380(4):608-622.   
33. Almazan F, Galan C,Enjuanes L. The nucleoprotein is required for efficient coronavirus genome replication. J Virol 2004;78(22):12683-12688.   
34. Tylor S,Andonov A, Cutts T, Cao J, Grudesky E et al. The SR-rich motif in SARS-CoV nucleocapsid protein is important for virus replication. Can J Microbiol 2009;55(3):254-260. 35. Shu T, Gan T, Bai P, Wang X, Qian Q et al. Ebola virus VP35 has novel NTPase and helicase-like activities. Nucleic Acids Res 2019;47(11):5837-5851.

![](images/d63215480b037407216c26790349043b7bd3c2491a2ede259e022f169d27474f.jpg)  
Figure 1. SARS-CoV-2 N suppressed shRNA-induced RNAi in cultured human

cells.HEK293T cells were co-transfected with plasmids encoding EGFP $( 0 . 1 \ \mu \mathrm { g } )$ and EGFP-specific shRNA $( 0 . 3 ~ \mu \ g )$ ，together with either empty vector or the plasmid encoding SARS-CoV-2 N or EBOV VP35( $1 ~ { \mu \mathrm { g } }$ each).(A) At 48 hpt, cells were observed via fluorescent microscopy. (B) Cell lysates were harvested and analyzed by western bloting with anti-EGFP,anti-FLAG and anti-Tubulin antibodies.(C) Total RNAs were extracted and EGFP mRNA levels were examined by northern blotting with a DIG-labeled RNA probe targeting EGFP ORF 1-400nt.18s and 28s rRNAs were used as loading controls.

![](images/ac0babe62385e7872a9258d8c3d7506d1515e718545a8763cc2510fb7248297e.jpg)

Figure 2. SARS-CoV-2 N sequestrated dsRNAs in cells. HEK293T cells were transfected with the plasmid encoding SARS-CoV-2 N or empty vector, together with EGFP-specific dsRNA. At 24 hpt, the cell lysates were subjected to RNA-IP with antiFLAG or anti-IgG antibodies. Input and precipitated RNAs and proteins were detected by northern blotting and western blotting, respectively.

![](images/bbf1c6025de2a24d6d108e8afe24d48dc753e20ea66ba42349e279028a08cd34.jpg)  
Figure 3. SARS-CoV-2 N suppressed siRNA-induced RNAi in cells. HEK293T cells

were co-transfected with the plasmid encoding EGFP $( 0 . 1 ~ \mu \ g )$ and EGFP-specific siRNA $( 0 . 3 \mu \mathrm { g } )$ , together with either empty vector or the plasmid encoding SARS-CoV$2 \ : \mathrm { N }$ 0r EBOVVP35（ $1 . 0 ~ \mu \ g$ each). (A) At 48 hpt, cells were observed via fluorescent microscopy. (B) Cell lysates were harvested and analyzed by western blotting with antiEGFP, anti-FLAG and anti-Tubulin antibodies. (C) Total RNAs were extracted and EGFP mRNA levels were examined by northern blotting with a DIG-labeled RNA probe targeting EGFP ORF 1-400nt. 18s and 28s rRNAs were used as loading controls.

# Supplementary Information

# Supplementary Table S1.The primers and oligonucleotides used in this study.

# Primers for plasmids construction

<html><body><table><tr><td>Name</td><td>Sequences(5'-3')</td></tr><tr><td>SARS-CoV-2N-FOR</td><td>GACTACAAGGACGACGATGACAAGATGTCTGATAATGGACC CCAAAATCAGC</td></tr><tr><td>SARS-CoV-2N-REV</td><td>GTTCTGCGCCTGCAGGTTTATTAGGCCTGAGTTGAGTCAGCA CTG</td></tr></table></body></html>

355

# 356 Primers for amplification of templates for in vitro transcription dsRNAs

<html><body><table><tr><td>Name</td><td>Sequences(5'-3')</td></tr><tr><td>EGFP-ds200-FOR</td><td>TAATACGACTCACTATAGATGGTGAGCTAGGGCGAGGA</td></tr><tr><td>EGFP-ds200-REV</td><td>TAATACGACTCACTATAGGCGGCTGAAGCACTGCACGC</td></tr></table></body></html>

357

# 358 Primers for amplification of templates for in vitro transcription RNA probes

<html><body><table><tr><td>Name</td><td>Sequences(5'-3')</td></tr><tr><td>EGFP-1-400(+)Probe-F</td><td>ATGGTGAGCAAGGGCGAGGA</td></tr><tr><td>EGFP-1-400(+)Probe-R</td><td>TAATACGACTCACTATAGGCTTGTGCCCCAGGATGTTGC</td></tr><tr><td>EGFP-1-400(-)Probe-F</td><td>TAATACGACTCACTATAGATGGTGAGCAAGGGCGAGGA</td></tr><tr><td>EGFP-1-400(-)Probe-R</td><td>GCTTGTGCCCCAGGATGTTGC</td></tr></table></body></html>